{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "AI-II-HW4.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "toc_visible": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "accelerator": "GPU",
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "a98a9b6071ef4276bd5881c91e236d10": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_7502a5be28d94e32bfea582d4ac2f499",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_5a4b239dfaf14b3b90f0a996138799e2",
              "IPY_MODEL_7cb36365c9924a82a5c5994625f7a000"
            ]
          }
        },
        "7502a5be28d94e32bfea582d4ac2f499": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "5a4b239dfaf14b3b90f0a996138799e2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_f7ae808a81f24ca396097a2648bfe5a6",
            "_dom_classes": [],
            "description": "Downloading: 100%",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 612,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 612,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_b25a7a65d94145abaec76c26dae996ae"
          }
        },
        "7cb36365c9924a82a5c5994625f7a000": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_457b7c8f86d54de8a5d3d2b42496c235",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 612/612 [00:00&lt;00:00, 1.60kB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_deb748b8cfcb473fb17451d100658b19"
          }
        },
        "f7ae808a81f24ca396097a2648bfe5a6": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "initial",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "b25a7a65d94145abaec76c26dae996ae": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "457b7c8f86d54de8a5d3d2b42496c235": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "deb748b8cfcb473fb17451d100658b19": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "9b7fbf48285847f8b780ff10375aed42": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_01636534b6104bfbbe1a3cc4a405c87e",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_6e57c6786ddc4bc39d1c5cb0974da9f6",
              "IPY_MODEL_55dc802e29e44f68a2522b042118b69e"
            ]
          }
        },
        "01636534b6104bfbbe1a3cc4a405c87e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "6e57c6786ddc4bc39d1c5cb0974da9f6": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_35830c8f476447dd9c09daa4d1e753b8",
            "_dom_classes": [],
            "description": "Downloading: 100%",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 267871721,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 267871721,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_c04769aedd15409b8d102ec8685e1f83"
          }
        },
        "55dc802e29e44f68a2522b042118b69e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_27a06abddf4f4d8aa6e99b48dffe7b5d",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 268M/268M [00:06&lt;00:00, 44.0MB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_30b4f99f902048a6a8c3eaff9b18de27"
          }
        },
        "35830c8f476447dd9c09daa4d1e753b8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "initial",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "c04769aedd15409b8d102ec8685e1f83": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "27a06abddf4f4d8aa6e99b48dffe7b5d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "30b4f99f902048a6a8c3eaff9b18de27": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "7e52fbca393842c1bd6d37e036d49bf2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_266e209f7c344e5bb3beab2b7f28fc29",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_10a66e84bc9a4e88b2df1bad9d384b18",
              "IPY_MODEL_e9936d5360044207a304f537195f7b07"
            ]
          }
        },
        "266e209f7c344e5bb3beab2b7f28fc29": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "10a66e84bc9a4e88b2df1bad9d384b18": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_b1fc9319cb094585ae32d98ca62ab3f5",
            "_dom_classes": [],
            "description": "Downloading: 100%",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 231508,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 231508,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_09b6694a2f62457f90279186036a528d"
          }
        },
        "e9936d5360044207a304f537195f7b07": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_94de9741ed944356a7185de896812c32",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 232k/232k [00:00&lt;00:00, 881kB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_6dd29eaa9ede477da606ad9803200cb7"
          }
        },
        "b1fc9319cb094585ae32d98ca62ab3f5": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "initial",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "09b6694a2f62457f90279186036a528d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "94de9741ed944356a7185de896812c32": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "6dd29eaa9ede477da606ad9803200cb7": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "ccd8cac0d9df4c8b914b7059fefde8a1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_d2c1f30b66964c5f8b719ad67d2202bd",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_7bd184a7de7f4fa0af68ced7b7147083",
              "IPY_MODEL_f3f11121a1f84e27be0cbb8d9551177d"
            ]
          }
        },
        "d2c1f30b66964c5f8b719ad67d2202bd": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "7bd184a7de7f4fa0af68ced7b7147083": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_60a064cee5514ef384968f587755d0a9",
            "_dom_classes": [],
            "description": "Downloading: 100%",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 112,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 112,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_f79574abae4b40d4b0a78152d0f00b5c"
          }
        },
        "f3f11121a1f84e27be0cbb8d9551177d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_f28e72efe5914c98b0c29ca60be70271",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 112/112 [00:00&lt;00:00, 838B/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_3724c930b807465e98182efe8bec6198"
          }
        },
        "60a064cee5514ef384968f587755d0a9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "initial",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "f79574abae4b40d4b0a78152d0f00b5c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "f28e72efe5914c98b0c29ca60be70271": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "3724c930b807465e98182efe8bec6198": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "bbfc8697e31d4653be93fa5824786558": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_abefa75157da4d3f961a768c4dd68587",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_30875a174c274e2ca6c69ec10eac41ae",
              "IPY_MODEL_275a5c32b03143b4a0237164fd2916e7"
            ]
          }
        },
        "abefa75157da4d3f961a768c4dd68587": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "30875a174c274e2ca6c69ec10eac41ae": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_422bee0204184139a3fcc609b80c27e7",
            "_dom_classes": [],
            "description": "Downloading: 100%",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 541,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 541,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_05be2a2814e3488fb5d1555af228c898"
          }
        },
        "275a5c32b03143b4a0237164fd2916e7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_c6e8ee5bba5046c8b79a7259e4866369",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 541/541 [00:00&lt;00:00, 9.80kB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_243be1e9d83f4f4bb0809ad0a76e20e0"
          }
        },
        "422bee0204184139a3fcc609b80c27e7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "initial",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "05be2a2814e3488fb5d1555af228c898": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "c6e8ee5bba5046c8b79a7259e4866369": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "243be1e9d83f4f4bb0809ad0a76e20e0": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "9d79b8acfab04f66aa002a39dacb2614": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_46dddac7ec7249dda04d9ca3022d4e2f",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_2d67fbdde70f41ffb26d4309dd2abcb1",
              "IPY_MODEL_31da0f5d98924a9595977b50db7f19b5"
            ]
          }
        },
        "46dddac7ec7249dda04d9ca3022d4e2f": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "2d67fbdde70f41ffb26d4309dd2abcb1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_f13a9a5e328442f6935a67046ff8ab00",
            "_dom_classes": [],
            "description": "Batches: 100%",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 282,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 282,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_9f8e9c0b73ca4b34a287e1db06295238"
          }
        },
        "31da0f5d98924a9595977b50db7f19b5": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_3ee266e550024c009e14f2398ae62c2b",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 282/282 [03:48&lt;00:00,  1.24it/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_2c246241ed5047db96d068d23712d139"
          }
        },
        "f13a9a5e328442f6935a67046ff8ab00": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "initial",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "9f8e9c0b73ca4b34a287e1db06295238": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "3ee266e550024c009e14f2398ae62c2b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "2c246241ed5047db96d068d23712d139": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        }
      }
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "AU8rZVK_dbA1"
      },
      "source": [
        "# 1. Document Retrieval System."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ko8_cA0t_xey"
      },
      "source": [
        "## 1) Data Extracting & Preprocessing."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ZlRGQoc-d_GG"
      },
      "source": [
        "In this section of the first part of this assignment, we will extract all the useful data from the given papers.\n",
        "\n",
        "First, we'll load from our local disk (or from our drive), the ***comm_use_subset.zip*** file, we'll extract it and then we'll create a folder named papers. Our structure will be something like this:\n",
        "\n",
        "```\n",
        "/content/papers/comm_use_subset/.jsonfiles\n",
        "```"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "134UXae7i1Jr"
      },
      "source": [
        "### (1.1): Useful Functions and Libraries."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "l1gyZMjODnpT"
      },
      "source": [
        "import zipfile\n",
        "\n",
        "def createFolder(zip_location):\n",
        "  with zipfile.ZipFile(zip_location, 'r') as zip_ref:\n",
        "    zip_ref.extractall('/content/papers')"
      ],
      "execution_count": 1,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "el8SBtAMI5TA"
      },
      "source": [
        "def writeToCSV(jsonf, data_file):\n",
        "\n",
        "  with open(jsonf) as json_file: \n",
        "\n",
        "      try:    \n",
        "        data = json.load(json_file)\n",
        "\n",
        "        csv_writer = csv.writer(data_file)\n",
        "\n",
        "        row = []\n",
        "\n",
        "        paper_id = data['paper_id']\n",
        "        row.append(paper_id)\n",
        "\n",
        "        title = data['metadata']['title']\n",
        "        row.append(title)\n",
        "\n",
        "        if data['abstract']:\n",
        "          abstract = data['abstract'][0]['text']\n",
        "          text = abstract \n",
        "        else:\n",
        "          text = \"\"\n",
        "\n",
        "        body_text = data['body_text']\n",
        "\n",
        "        text = re.sub(\"\\[\\d*]\",\"\", text)\n",
        "        text = re.sub(\" +\",\" \",text)\n",
        "\n",
        "        for sections in body_text:\n",
        "\n",
        "          sub_text = sections['text']\n",
        "          x = re.sub(\"\\[\\d*]\",\"\", sub_text)\n",
        "          x = re.sub(\" +\",\" \",x)\n",
        "          text += x\n",
        "\n",
        "        row.append(text)\n",
        "\n",
        "        csv_writer.writerow(row)\n",
        "\n",
        "      except:\n",
        "        print('Error in json file:', json_file)"
      ],
      "execution_count": 2,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NBzxQHjHDIpF"
      },
      "source": [
        "import os\n",
        "import csv\n",
        "import json\n",
        "import re\n",
        "\n",
        "import pandas as pd\n",
        "from pandas import read_csv\n",
        "\n",
        "def createCSV(fLocation):\n",
        "  entries = os.listdir(fLocation)                                               # Read all papers\n",
        "  \n",
        "  data_file = open('papers.csv','w')\n",
        "  csv_writer = csv.writer(data_file)\n",
        "  csv_writer.writerow(['paper_id', 'title', 'text'])\n",
        "  json_path = fLocation\n",
        "\n",
        "  for entry in entries:\n",
        "    \n",
        "    json_path = fLocation + \"/\" +entry\n",
        "    writeToCSV(json_path, data_file)\n",
        "\n",
        "  data_file.close()"
      ],
      "execution_count": 3,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "oh6EbrPTjSMB"
      },
      "source": [
        "### (1.2): Explanation of the Data Management."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "LrqH_YhNfXE6"
      },
      "source": [
        "Then, in order to have a more human-friendly data visualisation, we'll convert the .json files into a single .csv file. Ofcourse, we'll keep only the important data from these files:\n",
        "\n",
        "\n",
        "\n",
        "*   paper_id\n",
        "*   title\n",
        "*   body_text\n",
        "\n",
        "It's important to emphasize the fact that we'll do some data cleaning in the text inside of paper. Almost at all the range of the text, there are mentions, like this:\n",
        "\n",
        "\n",
        "    Old version: \" Bolton et al. [19] demonstrated that the LysM protein \". \n",
        "\n",
        "That's something we don't want to include in the text, because it's clearly noise. So, we managed to get rid of it. So, all the sentences like this, we'll be from now and then clear:\n",
        "\n",
        "    Cleared version:  \" Bolton et al. demonstrated that the LysM protein \". \n",
        "\n",
        "\n",
        "Also, we've cleaned all the multiple white spaces. At this point, we are ready to extract all the useful informations from the given papers. The cleaned .csv file will be named: **papers.csv**.\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "KTMNmR3bMetl"
      },
      "source": [
        "#Import .zip file straight from /content\n",
        "\n",
        "# Uncomment if the .zip is loaded\n",
        "# createFolder('/content/cord.zip')"
      ],
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "tfHK-vQTDKHz"
      },
      "source": [
        "#Code to read .zip file from drive\n",
        "\n",
        "!pip install -U -q PyDrive\n",
        "from pydrive.auth import GoogleAuth\n",
        "from pydrive.drive import GoogleDrive\n",
        "from google.colab import auth\n",
        "from oauth2client.client import GoogleCredentials\n",
        "\n",
        "auth.authenticate_user()\n",
        "gauth = GoogleAuth()\n",
        "gauth.credentials = GoogleCredentials.get_application_default()\n",
        "drive = GoogleDrive(gauth)"
      ],
      "execution_count": 5,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Wn4oWXIIDjh0"
      },
      "source": [
        "downloaded = drive.CreateFile({'id':'1shaeus7dCdt6LZxnQu9Au9oOHZrcyWYf'})\n",
        "downloaded.GetContentFile('comm_use_subset.zip')"
      ],
      "execution_count": 6,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "P9VEVsNbNVbe"
      },
      "source": [
        "createFolder('/content/comm_use_subset.zip')\n",
        "createCSV('/content/papers/comm_use_subset')"
      ],
      "execution_count": 7,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 198
        },
        "id": "se-ZIO68_rru",
        "outputId": "005a4ddb-8ed6-47cf-b23a-4cd83763bbe5"
      },
      "source": [
        "LocationCSV = r'/content/papers.csv'\n",
        "papers = pd.read_csv(LocationCSV);                                             \n",
        "papers.head()"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>paper_id</th>\n",
              "      <th>title</th>\n",
              "      <th>text</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0d1448dbed0b123a78907316826d116ad97cfbcf</td>\n",
              "      <td>a section of the journal Frontiers in Pharmaco...</td>\n",
              "      <td>Since its discovery in 2001, the major focus o...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>e1c5d3a82f4296f2867ccca63989530f17a773d7</td>\n",
              "      <td>molecules Using UPLC-MS/MS for Characterizatio...</td>\n",
              "      <td>Yupingfeng (YPF), a famous traditional Chinese...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>b3ad716630b356b1399e9df08cad73b1e92f317d</td>\n",
              "      <td>Changing risk awareness and personal protectio...</td>\n",
              "      <td>Background: Outbreaks of low and high pathogen...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>ae0700fe06361c6d9e2286332ba467940ec9f89e</td>\n",
              "      <td>Citation: A Host Factor GPNMB Restricts Porcin...</td>\n",
              "      <td>Porcine circovirus type 2 (PCV2) is the infect...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>4c16e6d3922d2b800f5f3e2dec0e51b9f7700898</td>\n",
              "      <td>Programmed cell removal by calreticulin in tis...</td>\n",
              "      <td>T he process of viable cell clearance via phag...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "                                   paper_id  ...                                               text\n",
              "0  0d1448dbed0b123a78907316826d116ad97cfbcf  ...  Since its discovery in 2001, the major focus o...\n",
              "1  e1c5d3a82f4296f2867ccca63989530f17a773d7  ...  Yupingfeng (YPF), a famous traditional Chinese...\n",
              "2  b3ad716630b356b1399e9df08cad73b1e92f317d  ...  Background: Outbreaks of low and high pathogen...\n",
              "3  ae0700fe06361c6d9e2286332ba467940ec9f89e  ...  Porcine circovirus type 2 (PCV2) is the infect...\n",
              "4  4c16e6d3922d2b800f5f3e2dec0e51b9f7700898  ...  T he process of viable cell clearance via phag...\n",
              "\n",
              "[5 rows x 3 columns]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 8
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "fSvfVE3yx6DE"
      },
      "source": [
        "## 2) Implementation of 2 different sentence embedding approaches."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "WpO7dRTklhQ4"
      },
      "source": [
        "### (2.1): First approach: SentenceBERT."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Gy6iUr-Rllsl"
      },
      "source": [
        "Sentence-BERT, uses a Siamese network like architecture to provide 2 sentences as an input. \n",
        "\n",
        "These 2 sentences are then passed to BERT models and a pooling layer to generate their embeddings. Then use the embeddings for the pair of sentences as inputs to calculate the **cosine similarity.**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "giy_rDCNl3nl"
      },
      "source": [
        "Let's install Sentence BERT:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MlLR72BMl5ym",
        "outputId": "ab92aa7e-5785-4ca1-bc40-47f0ef4902af"
      },
      "source": [
        "!pip install -U sentence-transformers"
      ],
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting sentence-transformers\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/6a/e2/84d6acfcee2d83164149778a33b6bdd1a74e1bcb59b2b2cd1b861359b339/sentence-transformers-0.4.1.2.tar.gz (64kB)\n",
            "\u001b[K     |████████████████████████████████| 71kB 7.5MB/s \n",
            "\u001b[?25hCollecting transformers<5.0.0,>=3.1.0\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/98/87/ef312eef26f5cecd8b17ae9654cdd8d1fae1eb6dbd87257d6d73c128a4d0/transformers-4.3.2-py3-none-any.whl (1.8MB)\n",
            "\u001b[K     |████████████████████████████████| 1.8MB 24.7MB/s \n",
            "\u001b[?25hRequirement already satisfied, skipping upgrade: tqdm in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (4.41.1)\n",
            "Requirement already satisfied, skipping upgrade: torch>=1.6.0 in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (1.7.0+cu101)\n",
            "Requirement already satisfied, skipping upgrade: numpy in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (1.19.5)\n",
            "Requirement already satisfied, skipping upgrade: scikit-learn in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (0.22.2.post1)\n",
            "Requirement already satisfied, skipping upgrade: scipy in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (1.4.1)\n",
            "Requirement already satisfied, skipping upgrade: nltk in /usr/local/lib/python3.6/dist-packages (from sentence-transformers) (3.2.5)\n",
            "Collecting sentencepiece\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/14/67/e42bd1181472c95c8cda79305df848264f2a7f62740995a46945d9797b67/sentencepiece-0.1.95-cp36-cp36m-manylinux2014_x86_64.whl (1.2MB)\n",
            "\u001b[K     |████████████████████████████████| 1.2MB 52.4MB/s \n",
            "\u001b[?25hRequirement already satisfied, skipping upgrade: importlib-metadata; python_version < \"3.8\" in /usr/local/lib/python3.6/dist-packages (from transformers<5.0.0,>=3.1.0->sentence-transformers) (3.4.0)\n",
            "Requirement already satisfied, skipping upgrade: packaging in /usr/local/lib/python3.6/dist-packages (from transformers<5.0.0,>=3.1.0->sentence-transformers) (20.9)\n",
            "Collecting sacremoses\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/7d/34/09d19aff26edcc8eb2a01bed8e98f13a1537005d31e95233fd48216eed10/sacremoses-0.0.43.tar.gz (883kB)\n",
            "\u001b[K     |████████████████████████████████| 890kB 51.0MB/s \n",
            "\u001b[?25hCollecting tokenizers<0.11,>=0.10.1\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/fd/5b/44baae602e0a30bcc53fbdbc60bd940c15e143d252d658dfdefce736ece5/tokenizers-0.10.1-cp36-cp36m-manylinux2010_x86_64.whl (3.2MB)\n",
            "\u001b[K     |████████████████████████████████| 3.2MB 50.6MB/s \n",
            "\u001b[?25hRequirement already satisfied, skipping upgrade: filelock in /usr/local/lib/python3.6/dist-packages (from transformers<5.0.0,>=3.1.0->sentence-transformers) (3.0.12)\n",
            "Requirement already satisfied, skipping upgrade: requests in /usr/local/lib/python3.6/dist-packages (from transformers<5.0.0,>=3.1.0->sentence-transformers) (2.23.0)\n",
            "Requirement already satisfied, skipping upgrade: dataclasses; python_version < \"3.7\" in /usr/local/lib/python3.6/dist-packages (from transformers<5.0.0,>=3.1.0->sentence-transformers) (0.8)\n",
            "Requirement already satisfied, skipping upgrade: regex!=2019.12.17 in /usr/local/lib/python3.6/dist-packages (from transformers<5.0.0,>=3.1.0->sentence-transformers) (2019.12.20)\n",
            "Requirement already satisfied, skipping upgrade: typing-extensions in /usr/local/lib/python3.6/dist-packages (from torch>=1.6.0->sentence-transformers) (3.7.4.3)\n",
            "Requirement already satisfied, skipping upgrade: future in /usr/local/lib/python3.6/dist-packages (from torch>=1.6.0->sentence-transformers) (0.16.0)\n",
            "Requirement already satisfied, skipping upgrade: joblib>=0.11 in /usr/local/lib/python3.6/dist-packages (from scikit-learn->sentence-transformers) (1.0.0)\n",
            "Requirement already satisfied, skipping upgrade: six in /usr/local/lib/python3.6/dist-packages (from nltk->sentence-transformers) (1.15.0)\n",
            "Requirement already satisfied, skipping upgrade: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata; python_version < \"3.8\"->transformers<5.0.0,>=3.1.0->sentence-transformers) (3.4.0)\n",
            "Requirement already satisfied, skipping upgrade: pyparsing>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from packaging->transformers<5.0.0,>=3.1.0->sentence-transformers) (2.4.7)\n",
            "Requirement already satisfied, skipping upgrade: click in /usr/local/lib/python3.6/dist-packages (from sacremoses->transformers<5.0.0,>=3.1.0->sentence-transformers) (7.1.2)\n",
            "Requirement already satisfied, skipping upgrade: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests->transformers<5.0.0,>=3.1.0->sentence-transformers) (2.10)\n",
            "Requirement already satisfied, skipping upgrade: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests->transformers<5.0.0,>=3.1.0->sentence-transformers) (3.0.4)\n",
            "Requirement already satisfied, skipping upgrade: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests->transformers<5.0.0,>=3.1.0->sentence-transformers) (2020.12.5)\n",
            "Requirement already satisfied, skipping upgrade: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests->transformers<5.0.0,>=3.1.0->sentence-transformers) (1.24.3)\n",
            "Building wheels for collected packages: sentence-transformers, sacremoses\n",
            "  Building wheel for sentence-transformers (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for sentence-transformers: filename=sentence_transformers-0.4.1.2-cp36-none-any.whl size=103068 sha256=aac94c20d4b645f42fa0b42b96891b0d54fb391eb74d89ca9ec84949a6623602\n",
            "  Stored in directory: /root/.cache/pip/wheels/3d/33/d1/5703dd56199c09d4a1b41e0c07fb4e7765a84d787cbdc48ac3\n",
            "  Building wheel for sacremoses (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for sacremoses: filename=sacremoses-0.0.43-cp36-none-any.whl size=893261 sha256=27e41ffaa113572149c62a88f9dcb2805f86447116aa84907154c9597e629134\n",
            "  Stored in directory: /root/.cache/pip/wheels/29/3c/fd/7ce5c3f0666dab31a50123635e6fb5e19ceb42ce38d4e58f45\n",
            "Successfully built sentence-transformers sacremoses\n",
            "Installing collected packages: sacremoses, tokenizers, transformers, sentencepiece, sentence-transformers\n",
            "Successfully installed sacremoses-0.0.43 sentence-transformers-0.4.1.2 sentencepiece-0.1.95 tokenizers-0.10.1 transformers-4.3.2\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "BQzdSHbxmd5N"
      },
      "source": [
        "Then, we will load the pre-trained BERT model. We choose among a variant list of models, the **best-base-nli-mean-tokens**. [Here](https://docs.google.com/spreadsheets/d/14QplCdTCDwEmTqrn1LH4yrbKvdogK4oQvYO1K1aPR5M/edit#gid=0) is a list with the models."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QXJGyWcUm8SL",
        "outputId": "a0648545-2bca-4777-a90d-fb7ed6736a1b"
      },
      "source": [
        "from sentence_transformers import SentenceTransformer\n",
        "sbert_model = SentenceTransformer('bert-base-nli-mean-tokens')"
      ],
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "100%|██████████| 405M/405M [00:15<00:00, 26.9MB/s]\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "T6ZSiloNngik"
      },
      "source": [
        "Then, we will create a list with **sentences**. Each sentences will be the content of each paper individually. \n",
        "\n",
        "We also should mention that from each paper we will subtract the \"noise\", such as emails and url's."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ZLFRxP0JMpE4"
      },
      "source": [
        "def createSentences(dataframe):\n",
        "\n",
        "  sentences = []\n",
        "\n",
        "  url_regex = r\"(?i)\\b((?:https?://|www\\d{0,3}[.]|[a-z0-9.\\-]+[.][a-z]{2,4}/)(?:[^\\s()<>]+|\\(([^\\s()<>]+|(\\([^\\s()<>]+\\)))*\\))+(?:\\(([^\\s()<>]+|(\\([^\\s()<>]+\\)))*\\)|[^\\s`!()\\[\\]{};:'\\\".,<>?«»“”‘’]))\"\n",
        "  email_regex = r'\\S*@\\S*\\s'\n",
        "\n",
        "  for paper in dataframe['text']:\n",
        "\n",
        "      # sentences.append(paper)\n",
        "      paper = re.sub(email_regex, '', paper)\n",
        "      paper = re.sub(url_regex, '', paper)\n",
        "      # text = re.sub(\"\\[\\d\\]\",\"\", paper)\n",
        "      # text = re.sub(\" +\",\" \",text)\n",
        "      sentences.append(paper)\n",
        "\n",
        "  return sentences"
      ],
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "V0HQgFJBiRVc",
        "outputId": "2c66b021-591c-4216-846e-2604583a67d5"
      },
      "source": [
        "sentences = createSentences(papers)\n",
        "print(len(sentences))"
      ],
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "9000\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "v8AqgFfjLtp4"
      },
      "source": [
        "### (2.2): Creation of the Sentence Embeddings.\n",
        "\n",
        "Then, we'll create the sentence embeddings, with the help of our **pre-trained SBERT** model."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "jYUb05Y1KF86"
      },
      "source": [
        "sentence_embeddings = sbert_model.encode(sentences)"
      ],
      "execution_count": 13,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Tk4D3ccCI8Yd"
      },
      "source": [
        "Also, in order to testify how \"simimlar\" are 2 sentences, we'll use the cosine similarity metric."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_1cqc283HuOS"
      },
      "source": [
        "import numpy as np\n",
        "\n",
        "def cosine(u, v):\n",
        "    return np.dot(u, v) / (np.linalg.norm(u) * np.linalg.norm(v))"
      ],
      "execution_count": 14,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Uh2TzDKeJGIm"
      },
      "source": [
        "Below, we have the code that will give us the most \"good\" paper, that will much likely give us the answer that we want. For the assignment's purpose, we'll print the title of the paper."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "77bJo84VNVoq"
      },
      "source": [
        "def get_key(val, my_dict):\n",
        "    for key, value in my_dict.items():\n",
        "         if val == value:\n",
        "             return key\n",
        " \n",
        "    return \"key doesn't exist\"\n",
        "\n",
        "def askQuestion(query):\n",
        "\n",
        "  # query = \"I like salmon and shrimps\"\n",
        "  query_vec = sbert_model.encode([query])[0]\n",
        "  dict_sent   = {}\n",
        "  index_sent  = {}\n",
        "  counter = 0\n",
        "\n",
        "  for sent in range(len(sentences)):\n",
        "\n",
        "    sim = cosine(query_vec, sbert_model.encode([sentences[sent]])[0])\n",
        "    index_sent[sent] = sim\n",
        "    dict_sent[sentences[sent]] = sim\n",
        "    # print(counter)\n",
        "    counter += 1\n",
        "\n",
        "  a = sorted(dict_sent.items(), key=lambda x: x[1], reverse=True)    \n",
        "\n",
        "  print(\"Top-4 Answers: \\n\")\n",
        "  for hit in a[:4]:\n",
        "    print(\"\\t \", hit[1] , \"\\t\", hit[0])\n",
        "\n",
        "  # print(a[:3])\n",
        "  value = a[0][1]\n",
        "  \n",
        "  # print(get_key(value, index_sent))\n",
        "  return(get_key(value, index_sent))\n",
        "\n",
        "def getTitle(dataframe, index, column):\n",
        "\n",
        "  return dataframe[column][index]"
      ],
      "execution_count": 15,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wJ49m87J0Scc"
      },
      "source": [
        "Now, our next task is to create a list of questions in order to feed our model. (You can find this list of question [here](https://drive.google.com/file/d/1XxqGTttbTqcrC88M3ZerpsCS10L0tVME/view)."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "OAmaHdKXrjHG"
      },
      "source": [
        "questions = [\"What are the coronoviruses?\", \n",
        "       \"What was discovered in Wuhuan in December 2019?\", \n",
        "       \"What is Coronovirus Disease 2019?\",\n",
        "       \"What is COVID-19?\",\n",
        "       \"What is caused by SARS-COV2?\",\n",
        "       \"Where was COVID-19 discovered?\",\n",
        "       \"How does coronavirus spread?\"]"
      ],
      "execution_count": 16,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "5T-XVdAA2lSd"
      },
      "source": [
        "Now, for each questions we will print the Top-4 scores and also the text we retrieved from our dataset. Finally for each question we'll return the title of the corresponding paper."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "OCEVB2ox0lZI"
      },
      "source": [
        "def provideQuestions(questions):\n",
        "\n",
        "\n",
        "  for question in range(len(questions)):\n",
        "    \n",
        "    print(question,\"\\b.\")\n",
        "    index = askQuestion(questions[question])\n",
        "    title = getTitle(papers, index, 'title')\n",
        "    print(\"\\n Title: \",title)"
      ],
      "execution_count": 17,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0iJ8zs7U3JKW",
        "outputId": "4fd4a2e6-511b-49a5-e6fa-2f44a8dde7ef"
      },
      "source": [
        "provideQuestions(questions)"
      ],
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "0 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.68208283 \t Porcine deltacoronavirus (PDCoV), a member of genus Deltacoronavirus, is an emerging swine enteropathogenic coronavirus (CoV). Although outstanding efforts have led to the identification of Alphacoronavirus and Betacoronavirus receptors, the receptor for Deltacoronavirus is unclear. Here, we compared the amino acid sequences of several representative CoVs. Phylogenetic analysis showed that PDCoV spike (S) protein was close to the cluster containing transmissible gastroenteritis virus (TGEV), which utilizes porcine aminopeptidase N (pAPN) as a functional receptor. Ectopic expression of pAPN in non-susceptible BHK-21 cells rendered them susceptible to PDCoV. These results indicate that pAPN may be a functional receptor for PDCoV infection. However, treatment with APN-specific antibody and inhibitors did not completely block PDCoV infection in IPI-2I porcine intestinal epithelial cells. pAPN knockout in IPI-2I cells completely blocked TGEV infection but only slightly decreased PDCoV infection. Homologous modeling of pAPN with the S1 C-terminal domain (S1-CTD) of PDCoV or TGEV showed that TGEV S1-CTD adopted β-turns (β1-β2 and β3-β4), forming the tip of a β-barrel, to recognize pAPN. However, only the top residues in the β1-β2 turn of PDCoV S1-CTD had the possibility to support an interaction with pAPN, and the β3-β4 turn failed to contact pAPN. We also discuss the evolution and variation of PDCoV S1-CTD based on structure information, providing clues to explain the usage of pAPN by PDCoV. Taken together, the results presented herein reveal that pAPN is likely not a critical functional receptor for PDCoV, although it is involved in PDCoV infection.Porcine deltacoronavirus (PDCoV) is an emerging swine enteropathogenic coronavirus (CoV) belonging to the genus Deltacoronavirus of the family Coronaviridae within the order Nidovirales 1-4 . Like other CoVs, PDCoV is an enveloped virus that contains positive, singlestranded genomic RNA 5, 6 . PDCoV was first identified in 2012 during molecular surveillance of CoVs in mammals and birds in Hong Kong 6 . The first PDCoV outbreak was reported in 2014 in the United States 7 , causing severe diarrhea, vomiting, and mortality in piglets 3, 4, 8, 9 . Thereafter, PDCoV was also detected in China 10-13 , Canada, South Korea 14, 15 , Lao People's Democratic Republic, Thailand 16 , and Vietnam 17, 18 , gaining considerable attention .In general, CoVs have a limited host range and tissue tropism. The interaction between CoV spike (S) proteins and specific cellular receptors on host cell surfaces mediates viral attachment and fusion of viral and cellular membranes, playing a vital role in successful infection in the host . The CoV S protein is a type I transmembrane glycoprotein with high molecular weight that protrudes from the surface of virions. The amino-terminal S1 domain is responsible for the recognition of cellular receptors, and the carboxy-terminal S2 domain mediates the subsequent membrane fusion process 25 . Recently, two research groups independently resolved the structure of PDCoV S protein by cryo-electron microscopy 26, 27 .PDCoV S protein is a trimer containing three receptorbinding S1 subunits and membrane-fusion S2 subunits 27 . The C-terminal domain (CTD) of the PDCoV S1 subunit that is responsible for receptor binding shares a similar structural fold with alphacoronavirus.To date, a series of cellular receptors for different genera of CoVs have been identified. For example, aminopeptidase N (APN, also called CD13) is the functional receptor for human coronavirus 229E (HCoV-229E) 28 , feline infectious peritonitis virus 29 , canine CoV 30 , and transmissible gastroenteritis virus (TGEV) 31 . Angiotensin-converting enzyme 2 is utilized by HCoV-NL63 and severe acute respiratory syndrome coronavirus (SARS-CoV) 24, 32 . Middle East respiratory syndrome coronavirus (MERS-CoV) S protein employs dipeptidyl peptidase 4 (also called CD26) as its receptor 33 . Carcinoembryonic antigen-related cell adhesion molecule 1 is reported to mediate viral infection by interacting with mouse hepatitis virus (MHV) S protein 34 . However, the relationship between hosts and members of genus Deltacoronavirus remains unknown.In this study, we investigated the role of porcine APN (pAPN) in PDCoV infection. We found that ectopic expression of pAPN rendered non-susceptible cells susceptible to PDCoV infection and promoted PDCoV infection in poorly susceptible cells. However, pAPN knockout or treatment with APN-specific antibody and inhibitor only decreased PDCoV infection to some degree. Additionally, pAPN knockout in porcine intestinal epithelial (IPI-2I) cells, a cell line established from porcine ileum, did not completely block PDCoV infection but significantly affected viral replication. We also demonstrated that the APN enzymatic activity inhibitor did not disrupt PDCoV infection, indicating pAPN enzymatic activity is not involved in this process. Our work suggests pAPN is not a critical receptor but is an important factor during PDCoV infection.Because of the dominant role of CoV S protein in receptor recognition and viral entry, we conducted amino acid sequence alignments of PDCoV S proteins, together with S proteins from three swine CoVs (TGEV; porcine epidemic diarrhea virus, PEDV; and porcine respiratory coronavirus, PRCV) and four representative CoVs from different genera (Betacoronavirus SARS-CoV, MERS-CoV, and MHV; Gammacoronavirus infectious bronchitis virus (IBV)). PDCoV S displayed higher homology with PEDV, PRCV, and TGEV (Fig. 1a) . pAPN acts as a functional cellular receptor for TGEV and PRCV infection 31, 35 . Whether pAPN is indeed a functional receptor for PEDV remains controversial, but it is involved in PEDV infection . Phylogenetic analysis indicated that PDCoV S is close to the cluster containing PEDV, TGEV, and PRCV (Fig. 1b) . Thus, we evaluated the role of pAPN in PDCoV infection.To determine whether pAPN contributes to PDCoV infection, we explored whether ectopic expression of pAPN in non-susceptible cells can cause PDCoV infection. Primary experiments showed that baby hamster kidney (BHK)-21 cells were non-susceptible and HeLa cells were slightly susceptible to PDCoV infection. pAPN overexpression in these cells was induced by transient transfection of the pAPN expression plasmid. PDCoV Sspecific fluorescence was observed in BHK-21 cells transfected with the pAPN expression plasmid, but not in cells transfected with control vector (Fig. 2a) . Moreover, HeLa cells showed greater infection by PDCoV after transfection with pAPN expression plasmid compared with cells transfected with control vector (Fig. 2a) .To further investigate whether pAPN-mediated infection in transfected BHK-21 and HeLa cells is productive, quantitative real-time PCR (RT-qPCR) was performed to detect the PDCoV genome RNA copies at different time points post infection. In BHK-21 cells, ectopic expression Mouse monoclonal antibody against PDCoV S was used to detect PDCoV-infected cells (green). The anti-Flag rabbit polyclonal antibody was used to stain for ectopic expression of APN protein (red). DAPI was applied to detect nuclei (blue). b, c BHK-21 cells (b) or HeLa cells (c) were transfected as described in a and then infected with PDCoV (MOI = 2). At different time points (0, 12, 24 h) post infection, cells were collected for RT-qPCR with primers targeting the PDCoV nsp16 gene to measure viral genome copies. Data are expressed as the mean ± SD for triplicate wells. Statistical significance was determined by Student's t test; ns, P > 0.05; *P < 0.05; **P < 0.01 slight increase in empty vector-transfected HeLa cells at 12 h and 24 h post infection, pAPN-transfected HeLa cells showed a stronger ability to promote PDCoV replication compared with control cells (Fig. 2c) . These results strongly suggest that HeLa cells and BHK-21 cells expressing pAPN are equipped to support PDCoV infection, implying that pAPN does serve as an important factor for PDCoV infection.In previous studies of various CoVs, pretreatment with antibody against receptor neutralized viral infectivity 32, 33 .To determine the effect of pAPN in PDCoV infection, we first examined whether polyclonal APN antibody can affect PDCoV infection in susceptible cells. Porcine IPI-2I cells, established from the ileum of an adult boar, are highly susceptible to PDCoV infection. Thus, we compared PDCoV infectivity in IPI-2I cells treated with APNspecific antibody, control antibody and no antibody. The results of immunofluorescence assay (IFA) showed that PDCoV S-specific fluorescence was reduced in cells treated with APN antibody (Fig. 3a, b) , indicating that treatment with APN-specific antibody reduced PDCoV infection. The viral titer (indicated by TCID 50 ) was also decreased with APN-specific antibody treatment (Fig. 3c) .APN is a zinc-dependent metalloprotease, and previous studies suggested that its enzymatic activity or epitope independent of its enzymatic activity are associated with APN function as a viral receptor 37, 39 . Thus, three APNspecific inhibitors, bestatin (a small inhibitory molecule that competitively binds to the catalytic site of APN), 2,2′dipyridyl, and 1,10-phenanthroline (a zinc-chelating molecule that impairs APN epitope conformation) 36 , were used to determine the role of APN in PDCoV infection. First, MTT assays and western blots were performed to detect the cytotoxicity of the three inhibitors and whether they affected the expression of endogenous APN in IPI-2I cells. Nearly no cytotoxicity could be detected (Fig. 4a) , and APN expression was not affected in cells treated with any of the three inhibitors (Fig. 4b) .Then, IFA and TCID 50 assays were performed to analyze the roles of three inhibitors in PDCoV infection. The results of IFA showed that the numbers of PDCoV Sspecific fluorescence-positive cells were reduced in 2,2′dipyridyl-and 1,10-phenanthroline-treated cells, but not in bestatin-treated cells, compared with control cells (Fig. 4c, d) . The viral titers after inhibitor treatments showed similar results under infection with PDCoV at different MOIs [MOI = 2 and MOI = 0.2] (Fig. 4e, f) , suggesting that pAPN's epitope conformation, not its enzymatic activity, is involved in PDCoV infection.To determine whether pAPN is essential to PDCoV infection, the CRISPR/Cas9 system was applied to establish pAPN knockout cell lines. Two sgRNAs were designed to target exon 1 of pAPN in IPI-2I cells. The surveyor nuclease assay indicated that the indel occurrence rates of the target nucleotide caused by the two sgRNAs were 38.8% and 35.6%, respectively (Fig. 5a) . Isolated pAPN knockout IPI-2I (IPI-2I-APN KO ) cell lines were further confirmed by sequencing and western blot analyses (Fig. 5b, c) . Sequencing analysis demonstrated the alleles of the APN gene in the clonal cell line harbored a 5-nt deletion and a 1-nt insertion in exon 1 of the APN genome, resulting in the production of a truncated peptide with no function (Fig. 5b) . Therefore, IPI-2I-APN KO cell lines were used to functionally assess the role of endogenous pAPN in PDCoV infection. Compared with Fig. 3 Effects of APN-specific antibody on PDCoV infection. a IPI-2I cells were treated with anti-APN rabbit polyclonal antibody, anti-Flag rabbit polyclonal antibody or no antibody (control) for 2 h and then infected with PDCoV (MOI = 2). After 24 h, cells were analyzed by IFA. Mouse monoclonal antibody against PDCoV S was used to detect PDCoV-infected cells (green). DAPI was applied to detect nuclei (blue). b The fluorescence intensity in a was quantified with the software ImageJ. Error bars show standard deviations. c IPI-2I cells were treated with antibody and infected with PDCoV as described in a. Cells underwent three freeze/thaw cycles, and LLC-PK1 cells were used to measure viral titer by TCID 50 assay. Data are expressed as the mean ± SD for triplicate samples. Statistical significance was determined by Student's t test; *P < 0.05 wild-type IPI-2I cells (IPI-2I-APN WT ), the viral titer in IPI-2I-APN KO cell lines was decreased by 1 log (Fig. 5d) . We tested another isolated pAPN knockout cell line, IPI-2I-APN KO2 , and similar results were observed (data not shown). These results demonstrate that knockout of APN expression in IPI-2I cells decreased PDCoV infection.pAPN affects an early step of PDCoV entry rather than viral assembly or release in IPI-2I cells Because pAPN knockout in IPI-2I cells decreased PDCoV infection, we further investigated which step is affected by APN in the replication cycle of PDCoV infection. Considering the cell membrane localization of APN, pAPN in IPI-2I cells is most likely to affect the step of viral adsorption, invasion or release, each of which occurs on the cell membrane. First, the internalization assay was conducted to analyze the earliest viral entry process. RT-qPCR and western blot showed that viral genome RNA copies and protein levels in IPI-2I-APN KO cells were both lower than those in IPI-2I-APN WT cells (Fig. 6a, b) . To further detect the viral assembly or release, IPI-2I-APN KO and IPI-2I-APN WT cell lines were infected with PDCoV, respectively. At 24 h after infection, the cells and supernatants were separately collected for RT-qPCR . At 24 h after incubation, the inhibitors were removed, and MTT reagents (20 μL, 5 mg/mL) were added. After another 4 h incubation, the medium was discarded, and 150 μL of dimethyl sulfoxide (DMSO) solution was added. The OD value at 570 nm was measured. b The expression levels of APN in cells after treatment with the three inhibitors. IPI-2I cells were cultured in six-well plates and treated with 2,2′-dipyridyl (250 μM), 1,10-phenanthroline (15 μM), or bestatin (300 μM). At 24 h after treatment, the expression of endogenous pAPN was detected by western blot with anti-APN rabbit polyclonal antibody. c 2,2′-Dipyridyl (250 μM), 1,10-phenanthroline (15 μM), or bestatin (300 μM) was added to IPI-2I cells for 1 h. Cells were then infected with PDCoV (MOI = 2). At 24 h post infection, cells were analyzed by IFA. Mouse monoclonal antibody against PDCoV S was used to detect PDCoVinfected cells (green). DAPI was applied to detect nuclei (blue). d The fluorescence intensity in c was quantified with ImageJ. e, f IPI-2I cells were treated with the three inhibitors as described in c and infected with PDCoV (MOI = 2 in e or MOI = 0.2 in f). At 24 h post infection, cells were collected, and the TCID 50 was determined in LLC-PK1 cells. Data are expressed as the mean ± SD for triplicate samples. Statistical significance was determined by Student's t test; ns, P > 0.05; *P < 0.05; **P < 0.01 and western blot. The ratios of the viral genome RNA copies and protein levels were almost identical between supernatant and cell lysates (Fig. 6c, d) , suggesting that pAPN has no effect on viral assembly or release. Taken together, these results indicate that pAPN mainly affects an early step of PDCoV entry in IPI-2I cells.pAPN is reported to act as a critical cellular receptor during TGEV entry 31 . In the light of the limited effect of pAPN in PDCoV infection, we compared the difference between PDCoV and TGEV infection in IPI-2I cells to better understand the role of pAPN. IPI-2I-APN WT and IPI-2I-APN KO cell lines were incubated with the same dose of TGEV or PDCoV. Cytopathic effects (CPEs) caused by PDCoV infection were observed in both IPI-2I-APN WT and IPI-2I-APN KO cells, even though IPI-2I-APN KO cells presented a mild CPE (Fig. 7a) . However, loss of the APN gene in IPI-2I-APN KO cells led to the disappearance of CPEs in TGEV infection (Fig. 7a) . PDCoV S protein and TGEV M protein levels in IPI-2I-APN WT and IPI-2I-APN KO cells were also analyzed by IFAs. As expected, PDCoV S-specific fluorescence in IPI-2I-APN KO cells was partially reduced (Fig. 7b) , while TGEV M-specific fluorescence was undetectable (Fig. 7c) . Similar results were demonstrated by western blot using antibodies against viral N proteins (Fig. 7d, e) . These results indicate that pAPN knockout in IPI-2I cells absolutely blocked TGEV infection and only partly decreased PDCoV infection.Homologous modeling of pAPN with S1-CTD of PDCoV or TGEV To further explain the different roles of pAPN in PDCoV and TGEV, we compared the amino acid sequences of TGEV and PDCoV S1-CTD. Although the two proteins shared similar secondary structure elements, the N-terminus of PDCoV S1-CTD was shorter than that of TGEV, which led to a shorter turn of β1-β2 and of β3-β4 in the PDCoV structure (Fig. 8a, b) . Hence, the homologous model showed that S1-CTD of both PDCoV and TGEV adopted β-turns (β1-β2 and β3-β4), forming the tip of the β-barrel to recognize pAPN (Fig. 8c) . For TGEV, residues in the two β-turns both contacted pAPN comfortably, forming a contact network (Fig. 8c) . Conversely, only the top residues in the β1-β2 turn of PDCoV supported an interaction with pAPN, and the β3-β4 turn failed to contact pAPN (Fig. 8c) , indicating that its contribution to the interaction with pAPN is smaller than that of TGEV. Overall, the different lengths of the two The target sequence in isolated clonal cells was amplified through PCR and cloned into the pMD18-T vector. Edited nucleotides in pAPN gene alleles are shown according to sequencing analysis. c Isolated clonal IPI-2I-APN KO cells were cultured in 6-well plates, and the expression of endogenous pAPN was detected by western blot with anti-APN rabbit polyclonal antibody. d IPI-2I-APN KO and IPI-2I-APN WT cells were cultured in 24-well plates and infected with PDCoV (MOI = 2). At 24 h post infection, cells were collected, and the viral titer was determined by TCID 50 assay in LLC-PK1 cells. Data are expressed as the mean ± SD for triplicate samples. Statistical significance was determined by Student's t test; *P < 0.05 β-turns (β1-β2 and β3-β4) between PDCoV and TGEV may determine their variability in pAPN usage during viral entry.As an emerging swine enteropathogenic CoV and the sole member of the genus Deltacoronavirus that has been successfully isolated by cell culture in vitro, PDCoV is a good model to study deltacoronaviruses. However, very little is known about this emerging virus, including its receptor and infection mechanisms. A recent study showed that the β-sandwich core structure of PDCoV S1-CTD, which may be responsible for receptor recognition, is similar to that of alphacoronaviruses, which mainly use APN as a cellular receptor 27 . Furthermore, PDCoV in infected pigs was primarily detected in the villous epithelium of the mid-jejunum to ileum 20 . Large amounts of APN are also expressed on the surface of porcine small intestine enterocytes 31 . Thus, in this study, we investigated the role of pAPN in PDCoV infection. Interestingly, ectopic expression of pAPN successfully reestablished PDCoV infection in non-susceptible BHK-21 cells, and pAPN overexpression effectively promoted PDCoV infection in poorly susceptible HeLa cells expressing pAPN. However, pAPN knockout in IPI-2I cells, a cell line that was established from porcine ileum and is highly susceptible to PDCoV infection, did not completely block PDCoV infection. We also performed pAPN knock-down experiments with pAPN-specific siRNA in LLC-PK1 (porcine kidney) cells, and the viral titers were decreased by only 0.6 log in pAPN-knock-down LLC-PK1 cells compared to the LLC-PK1 cells transfected with control siRNA (data not shown). A recent study found no interaction between purified pAPN and PDCoV S1-CTD by dot-blot assay 27 . Although we detected a possible weak interaction with the purified full-length PDCoV S protein and pAPN through molecular sieve analysis, a direct interaction could not be confirmed by pull-down assay (data not shown). In contrast, another CoV, TGEV, which uses pAPN as a functional receptor, failed to infect IPI-2I-APN KO cells, as demonstrated by CPE, viral protein expression, and viral genome replication. These results indicate that pAPN plays a decisive role in TGEV infection and only a contributing role in PDCoV infection in IPI-2I cells. Coincidentally, pAPN was initially identified as a PEDV-binding protein in swine kidney cells by virus overlay protein binding assay, and non-susceptible cell lines expressing pAPN became susceptible to PEDV 40, 41 . However, more recent evidence showed that pAPN knockout did not decrease PEDV infection, and no interaction between PEDV S1 and pAPN was detected, implying that pAPN is not a functional receptor for PEDV entry 37 . Thus, whether pAPN promotes PDCoV or PEDV infection through direct interaction with S protein or along with other cellular proteins requires further investigation.APN is a 150-kDa type II glycoprotein in the metalloprotease family 31 . As a zinc-dependent metalloprotease, APN has multiple functions. In its membrane-bound form, APN is involved in complex functions in cells, such as peptide cleavage, immune cell chemotaxis, and monocyte cell adhesion 42, 43 . One of the best-studied characterizations of APN is its receptor function in CoVs, including TGEV, HCoV-229E, feline infectious peritonitis virus, canine CoV, and PRCV from the genus Alphacoronavirus 31, 35, 44, 45 . In this study, bestatin, an APN inhibitor that binds to its catalytic site, failed to affect PDCoV infection. However, two inhibitors that alter APN epitope conformation both decreased PDCoV infection in IPI-2I cells. These data suggest that a specific conformation of the pAPN epitopes, not its enzymatic activity, is required in PDCoV infection.CoV S proteins contribute to the first step of viral infection and seem to be a vital determinant of host range and tissue tropism. Considering that pAPN is a functional receptor for TGEV, but not for PDCoV, we analyzed differences between PDCoV and TGEV S protein structures. In general, S protein S1 subunits, which contain two independent, functional subdomains S1-NTD and S1-CTD, mediate viral entry into cells during CoV infection. Although MHV S1-NTD is reported to recognize sugar receptors and a unique protein receptor carcinoembryonic antigen-related cell adhesion molecule 1 34 , protein receptor recognition for most CoVs depends on S1-CTD 46 . In particular, APN receptor in several alphacoronaviruses, such as TGEV and PRCV, interacts with S1-CTD 35 . Understanding the differences between S1-CTD structures of various CoVs is helpful to explain the role of pAPN in viral infection. Therefore, we analyzed the possible recognition of pAPN by PDCoV and TGEV S1-CTD. We found that the β1-β2 and β3-β4 turns in TGEV S1-CTD strongly supported its interaction with pAPN. However, for PDCoV S1-CTD, only residues at the top of the β1-β2 turn contacted pAPN. The markedly shorter β1-β2 and β3-β4 turns in PDCoV S1-CTD may cause insufficient contact with pAPN during viral attachment. These findings may help explain the limited role of pAPN in PDCoV infection. PDCoV can be detected in other tissues of infected pigs 1, 4, 10, 47 , suggesting that another, unidentified protein may contribute to the PDCoV recognition and entry processes. In addition, the S gene of CoV has been shaped by recombination and positive selection that may have led to changes in receptorbinding affinity 48, 49 . For example, isolated MERS-CoV strains from a recent outbreak in South Korea had point mutations in S1-CTD and showed decreased binding ability to the cellular receptor 50 . A highly neurotropic MHV JHMV strain infected host cells through a carcinoembryonic antigen-related cell adhesion molecule 1 receptor-independent manner 51 . However, whether the evolution of PDCoV S1-CTD is responsible for the shorter β1-β2 and β3-β4 turns and impaired ability to use pAPN requires further investigation. Moreover, the receptor-binding region of CTD is believed to bear mainly Fig. 8 Homologous modeling of PDCoV or TGEV S1-CTD with pAPN. a Sequence alignment was conducted with PDCoV S1-CTD (GenBank accession no. ALS54086.1) and TGEV S1-CTD (GenBank accession no. ADY39740.1). The secondary structures of S1-CTD were analyzed using the ESPript website (). b, c The three-dimensional structures of PDCoV S1-CTD (yellow; PDB ID 6B7N), TGEV S1-CTD (red; PDB ID 4F2M), and pAPN (blue; PDB ID 4F5C) obtained from the Protein Data Bank were analyzed with PyMOL software epitopes recognized by CoV neutralizing antibodies 35, 50, 52, 53 . Thus, the receptor-binding regions of CoVs are under selective pressure from the host immune system. We speculate that the shorter β1-β2 and β3-β4 turns in PDCoV S1-CTD compared with TGEV S1-CTD may be the result of positive immune pressure from the host. Conformational changes help PDCoV to evade host immune surveillance, while the ability of PDCoV to hijack pAPN may be impaired. These findings imply that a yet unidentified receptor may be utilized by PDCoV in the process of viral evolution and variation.In this study, PDCoV exhibited a low level of infection in HeLa cells, a human cell line. Purified PDCoV S1-CTD binds to both human and porcine cells with high affinity 27 , suggesting that a co-receptor(s) exists on the surface of both human and porcine cells. We therefore evaluated the role of hAPN in PDCoV infection. PDCoV efficiently infected non-susceptible BHK-21 cells expressing hAPN by transient transfection, as demonstrated by detection of PDCoV S protein-specific fluorescence. However, TGEV failed to infect BHK-21 cells overexpressing hAPN (data not shown). These observations suggest that PDCoV may employ both pAPN and hAPN, while TGEV can only employ pAPN. In addition, a previous study suggested that calves are also susceptible to PDCoV infection 54 . Although no diarrhea or other clinical signs were observed in PDCoV-inoculated calves, persistent fecal viral RNA shedding and serum IgG antibody responses against PDCoV were detected 54 . These phenomena remind us of the potential risk for PDCoV infection with cross-species transmission.In addition to the protein receptor that is responsible for CoV S1-CTD, host sugar receptors also interact with S1-NTD to facilitate initial viral attachment to cells in some alpha-and betacoronaviruses 55, 56 . The structural model of PDCoV S1-NTD presents a similar galectin fold to that of alpha-and betacoronaviruses, and the sugarbinding capability of PDCoV S1-NTD to mucin has been demonstrated through enzyme-linked immunosorbent assay 27 . The specific effects of sugar receptors in PDCoV infection deserve deeper study.In summary, our results illustrate that pAPN could support PDCoV infection, independent of its enzymatic activity. However, whether pAPN promotes PDCoV infection through direct interaction with PDCoV S protein or with the cooperation of other host proteins must be further elucidated. The findings of slight infection in HeLa cells and successful infection in non-susceptible BHK-21 cells expressing hAPN suggest that a possible co-receptor exists in both porcine and human cells, revealing the potential risk for PDCoV cross-species transmission. Therefore, detailed study focused on virus-host interactions and identification of critical functional receptor usage is necessary to increase the understanding of PDCoV infection. pAPN and human APN (hAPN) eukaryotic expression plasmids were constructed by cloning pAPN or hAPN cDNA into vector pCAGGS-Flag with an N-terminal Flag tag. To knockout the pAPN gene in IPI-2I cells, two single-guide RNAs (sgRNAs) targeting exon 1 (5′-GGTAGGCGGTACCGGTTCCA-3′ and 5′-GTCTGTCTGTGGTGTACGCCC-3′) were designed and inserted into the PX459 background to generate sgRNA1 and sgRNA2 expression plasmids. All plasmids used were confirmed by sequencing.APN inhibitors bestatin (TargetMol, USA), 2,2′-dipyridyl, and 1,10-phenanthroline (Sigma, USA) were each dissolved in water at a concentration of 200 mM. The inhibitors were then diluted to various concentrations for use at 300 μM (bestatin), 250 μM (2,2′-dipyridyl), and 15 μM (1,10-phenanthroline) as previously described 36 . IPI-2I cells cultured in 24-well plates were pre-incubated with inhibitors, rabbit polyclonal antibody against APN, or anti-Flag rabbit polyclonal antibody (control) for 1 h and subsequently infected with PDCoV [multiplicity of infection (MOI) = 2] for 1 h. The cells were further washed three times with Dulbecco's modified Eagle medium and maintained with inhibitors, APN antibody, or control antibody in cell culture containing 2.5 μg/ml trypsin. At 24 h post infection, cells were collected for IFA, and PDCoV titers in LLC-PK1 cells were determined by 50% tissue culture infective dose (TCID 50 ).Internalization assay IPI-2I-APN WT and IPI-2I-APN KO cells in 24-well plates with 90% confluence were inoculated with PDCoV (MOI = 30) for virus attachment at 4°C. After 1 h, the infected cells were washed three times with cold PBS and cultured at 37°C for another 1 h to allow virus internalization. Then, the infected cells were washed with citrate buffer solution (pH = 3) to remove the bound but noninternalized virus particles. The cells were harvested for RT-qPCR and western blot to evaluate the effect of pAPN on PDCoV internalization.were seeded on glass coverslips (NEST). Cells were washed three times with phosphate-buffered saline (PBS) and then sealed with 4% paraformaldehyde in methanol for 15 min and methanol for 15 min. Bovine serum albumin (5%) diluted in PBS was used to block cells for 1 h, and mouse monoclonal antibody against PDCoV S, TGEV M, or anti-Flag rabbit polyclonal antibody was then added and incubated for 1 h. After three washes with PBS, Alexa Fluor-conjugated secondary antibodies were added and incubated for 1 h, followed by 0.01% 4′,6-diamidino-2-phenylindole (DAPI) staining for 15 min to detect nuclei. After three washes with PBS, fluorescent images were examined by confocal laser scanning microscopy (LSM 510 Meta, Carl Zeiss).pAPN knockout by CRISPR/Cas9 genome-editing system Knockout of the pAPN gene in IPI-2I cells was performed as previously described 57 . sgRNA1 and sgRNA2 expression plasmids targeting exon 1 of pAPN gene were constructed. The two sgRNA expression plasmids (each 0.5 µg) were mixed and transfected into IPI-2I cells cultured in 24-well plates. After 24 h, 1 µg/ml of puromycin (Sigma, USA) for IPI-2I cells was added to the cell culture for positive cell selection. After 48 h incubation, surviving cells were treated with trypsin and harvested to detect indel mutations by the surveyor nuclease assay. Clonal cell lines were isolated by dilution. Approximately 60 cells in 10 ml of Dulbecco's modified Eagle medium were plated in each 96-well plate. pAPN knockout clonal cell lines were confirmed by sequencing and western blot.Total RNA was extracted from PDCoV-infected cells by TRIzol reagent (Promega, USA). Using AMV reverse transcriptase (Takara, Japan), RNA (1 μg) was then reverse-transcribed into cDNA, which acted as the template in the SYBR Green PCR assay. RT-qPCR primers targeting PDCoV nsp16 (nsp16-F: 5′-GCCCTCGGTGGTTCTATCTT-3′, nsp16-R: 5′-TCCTTAGCTTGCCCCAAATA-3′) were used to measure PDCoV genome RNA copies.Cells cultured in six-well plates were harvested with lysis buffer (Beyotime, China) and boiled for 10 min with sample loading buffer (Beyotime, China). The samples were then resolved by sodium dodecyl sulfatepolyacrylamide gel electrophoresis, and proteins were transferred to polyvinylidene difluoride membranes (Millipore, USA). The membranes were blocked with 5% bovine serum albumin at room temperature for 2 h and incubated with rabbit polyclonal antibody against APN or mouse monoclonal antibody against PDCoV N or TGEV N for 3 h. After washing three times, the membranes were incubated with horseradish peroxidase-conjugated antipolyclonal or -monoclonal antibody for another 1 h and washed three times. Proteins were detected using a western blot analysis system (Bio-Rad).The structure information of proteins evaluated in our study was obtained from the Protein Data Bank (PDB) library. pAPN (PDB ID 4F5C), TGEV S (PDB ID 4F2M), and PDCoV S (PDB ID 6B7N) were chosen for analysis of S1-CTD and pAPN using PyMOL software (https:// ).All experiments were performed in triplicate. Data are shown as the mean ± standard deviation (SD). Student's t test was used to measure significant differences between groups. P values < 0.05 were considered statistically significant.\n",
            "\t  0.6688348 \t Coronaviruses (CoVs), such as human coronavirus NL63 (HCoV-NL63), severe acute respiratory syndrome CoV (SARS-CoV), murine hepatitis virus (MHV), porcine epidemic diarrhea virus (PEDV), and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), encode papain-like (PL) proteases that inhibit Sendai virus-(SeV-) induced interferon (IFN-) production. Recently, the crystal structure of transmissible gastroenteritis virus (TGEV) PL1 has been solved, which was similar to that of SARS-CoV PL2 pro , which may antagonize host innate immunity. However, very little is known about whether TGEV PL1 can antagonize host innate immune response. Here, we presented evidence that TGEV PL1 encoded by the replicase gene could suppress the IFNexpression and inhibit the nuclear translocation of interferon regulatory factor 3 (IRF3). The ability to antagonize IFN-production was dependent on the intact catalytic activity of PL1. Furthermore, TGEV PL1 exerted deubiquitinase (DUB) activity which strongly inhibited the retinoic acid-induced gene I-(RIG-1-) and stimulator of interferon gene-(STING-) dependent IFN expression. Our data collectively suggest that TGEV PL1 can inhibit the IFN-expression and interfere with RIG-1-and STING-mediated signaling through a viral DUB activity. Our study has yielded strong evidence for the TGEV PL1 mechanisms that counteract the host innate immunity.The innate immune system is the first line of defense that protects the host against viral infection, and the induction of IFN-/ is a crucial antiviral mechanism of the innate immune system . The initiation of IFN expression is triggered by pathogen-associated molecular patterns (PAMPs) through host pattern recognition receptors (PRRs) . After viral RNAs are sensed by PRRs, signals are transmitted to different downstream adaptor molecules (such as IFNpromoter stimulator 1 (IPS-1)); and then I B kinase-(IKK-) related kinases are recruited. Next, interferon regulatory factor 3 (IRF3), nuclear factor B (NF-B), and ATF-2/c-jun are activated by the kinase complexes and translocate to the nucleus and directly induce the expression of type I IFNs .TGEV is an enveloped virus belonging to the Coronaviridae (CoV) family and the Nidovirales order. CoVs are positive-strand RNA viruses that replicate in the cytoplasm of infected cells . CoVs encode two types of cysteine proteases, M pro , and papain-like proteases, PL1 and PL2, which contained nonstructural protein 5 (nsp5) and nsp3, respectively. PL pro is served mainly as in processing of the replicase pp1a and pp1ab polypeptides . Other than their role in replicase polyprotein processing, PL2 domains possess an additional but related enzymatic activity, in HCoV-NL63 , MHV , SARS-CoV [8, 9] , and MERS-CoV , through their deubiquitination (DUB) enzymes, which play a key role in antagonizing IFN induction. However, TGEV PL1 processes the nsp2/nsp3 site and is capable of hydrolyzing isopeptide bonds in both Lys48-and 2 BioMed Research International Lys63-linked polyubiquitin chains . Whether TGEV PL1 could antagonize the production of IFNs was unknown.In the present study, we found that TGEV PL1 encoded by the replicase gene could suppress the IFN-expression and inhibit the nuclear translocation of interferon regulatory factor 3 (IRF3) and exerted deubiquitinase (DUB) activity which strongly inhibited the retinoic acid-induced gene I-(RIG-1-) and stimulator of interferon gene-(STING-) dependent IFN expression.Cells and Viruses. HEK293T cells and PK-15 cells were cultured in Dulbecco's modified Eagle's medium (Hyclone, Logan, USA) containing 10% (v/v) fetal calf serum supplemented with penicillin (100 U ml −1 ) and streptomycin (100 g ml −1 ). Sendai virus (SeV) was obtained from the Centre of Virus Resource and Information (Wuhan Institute of Virology, Chinese Academy of Sciences).Plasmids and Agents. IFN--Luc, 4x PRDIII/I-Luc (referred to as IRF3-Luc), 4x AP-1-Luc, and 4x NF-B-Luc luciferase reporter plasmids were constructed according to an earlier protocol . Accession numbers of STING, IRF3, and MAVS were KC860780, KC860781, and KC860782, respectively. Expression plasmids for RIG-1 (p-Flag-RIG-1) and TBK-1 (p-Flag-TBK-1) were generated with the following primers: RIG-1 forward, 5 -TTTGGATCCATGACAGCA-GAGCAGCGGCGGAAT-3 , RIG-1 reverse 5 -TTTAAG-CTTCACTCAAGGTTCGGGATTCCCTG-3 ; TBK-1 forward, 5 -TTTGAATTCATGCAGAGCACTTCTAATCAT-CTTT-3 , TBK-1 reverse 5 -TTTAGATCTTAAAGACAG-TCAACATTGCGAA-3 . To construct the DNA expression vector, pMyc-PL1, pFlag-PL1, and pPL1-Myc, encoding TGEV PL1, standard reverse transcription-(RT-) PCR was applied to amplify cDNA of the total RNA extracted from PK-15 cells infected with the TGEV strain HX, using the following primers: PL1-forward, 5 -GTACAAGAA-GCTGAACAATTTAA-3 (3498-3520 bp), PL1 reverse, 5 -ATCGTTTTTAGGACTTTGAATTT-3 (4249-4271 bp). All constructs were validated via DNA sequencing. pDsRed2-Mito was purchased from Clontech (Tokyo, Japan). Transfection agent was performed with X-tremeGENE HP (Roche, Switzerland) per the manufacturer's instructions.Luciferase Reporter Gene Assay. HEK-293T cells grown in 24-well plates were cotransfected with 0.2 g/well reporter plasmid, 0.02 g/well pRL-TK plasmid (Promega, Madison, USA) as an internal control for normalization of transfection efficiency, and the indicated expression or empty control vector plasmid. Where indicated, cells were also mockinfected or infected with SeV (100 hemagglutinating activity units/well) at 10 h after cotransfection. Cells were subsequently lysed, and firefly and Renilla luciferase activities were determined with the Dual-Luciferase reporter assay system (Promega, Madison, USA), according to the manufacturer's protocol. Data are presented as mean relative luciferase units ± standard deviation from triplicate samples. For statistical analysis, data were compared between empty vector-and TGEV PL1-transfected groups with the unpaired, two-tailed Student's t-test using SPSS 11.0 software. values < 0.05 were considered statistically significant .ELISA. Cell supernatants of transfected PK-15 cells were centrifuged at 3,000 for 5 min to remove cell debris and stored at −80 ∘ C until use. Secreted IFN-in the cell supernatants was determined using commercial Porcine IFN-(Interferon Beta) ELISA Kit (Elabscience, China) according to the manufacturer's instructions.Immunoblot Analysis. HEK293T cells were cultured in 6well plates and 60 mm dishes were transfected with the appropriate plasmids. After 36 h, cells were harvested by the addition of lysis buffer and protein concentrations in whole cell extracts measured. Equal amounts of samples were subjected to SDS-PAGE and analyzed for TGEV PL1, STING, TBK-1, and IRF3 proteins via immunoblotting using HA, Flag, or GFP-tagged antibodies (Sigma, St Louis, USA). Expression of p-IRF3, IRF3, and GAPDH was detected with the rabbit-anti p-IRF3 (ab76493), IRF3 (ab68481) (Abcam, Cambridge, UK), and a mouse anti-GAPDH monoclonal antibody (Sigma, St Louis, USA).Assay of Deubiquitinase Activity in Cultured Cells. HEK293T cells were cotransfected with pcDNA3.1-HA-Ub plus the indicated amounts of TGEV PL1, p-Flag-RIG-1, and p-Flag-STING constructs. The effect of TGEV PL1 on ubiquitinated proteins in cultured cells was assessed by immunoblot analysis.Coimmunoprecipitation Analysis. Coimmunoprecipitation experiments were performed on HEK293T cells transfected with the indicated expression plasmids as described in an earlier report .Immunofluorescence Assay. HEK293T cells were plated on fibronectin-treated glass coverslips in 24-well plates. To evaluate the localization of TGEV PL1, cells were cotransfected with plasmid DNA expressing Flag-PL (500 ng per well) and pDsRed-Mito (500 ng per well) using X-treme GENE HP, according to the manufacturer's protocol. HEK293T cells were cotransfected with IRF3-GFP (500 ng per well) and empty vector (500 ng per well) or IRF3-GFP (500 ng per well) and Flag-PL1 (500 ng per well). 24 h after transfection, cells were infected with SeV (100 hemagglutinating activity units/well) for 16 h. Next, cells were fixed with 4% paraformaldehyde for 30 min and permeated with 0.1% Triton X-100 for 15 min at room temperature. After three washes with PBS, cells were blocked with PBS containing 5% bovine serum albumin for 2 h, followed by incubation with a mouse monoclonal antibody against Flag (1 : 100) for 1 h at room temperature. Cells were treated with fluorescein isothiocyanate-labeled goat anti-mouse (Sigma, St Louis, USA) for 1 h, and subsequently with 4 ,6-diamidino-2-phenylindole (DAPI) for 15 min at room temperature. Samples were washed with PBS, and fluorescent images were acquired under a confocal laser scanning microscope (TCS SP5; Leica, Solms, Germany).To assess the formation of STING dimers, HEK293T cells were transfected with Flag-STING (500 ng per well) and the lysates were subjected to Western blot, as described earlier , with the indicated antibodies. Is an IFN Antagonist. The crystal structure of TGEV PL1 has been determined . The structure of TGEV PL1 is similar to that of SARS-CoV PL2 pro . In order to determine whether TGEV PL1 is capable of blocking IFN-production, we assessed IFN-promoter activity in the presence of PL1 (Figure 1(a) ). HEK 293T cells were cotransfected with TGEV PL1 and IFN-luciferase or Renilla luciferase reporter plasmids for 24 h and subsequently infected with SeV to activate the RIG-1-dependent IFN-expression pathway. We observed the inhibition of SeV-induced IFN-promoter activation in the presence of PL1, similar to the antagonistic function of NL63 PLP2 and porcine epidemic diarrhea virus (PEDV) PLP2, clearly indicating that TGEV PL1 could act as an interferon antagonist. To establish the mechanisms by which PL1 inhibits IFNexpression, transcriptional activities of NF-B, IRF3, and AP-1 were analyzed using the luciferase assay to identify the precise transcription factor involved. Notably, the luciferase activities of all three transcription factors were significantly inhibited by TGEV PL1 in a dose-dependent manner ( Figures 1(b) , 1(c), and 1(d)). Furthermore, Flag-PL1 also significantly inhibited IFN-production in PK-15 cells at protein level ( Figure 1(e) ), which was further confirmed with the result that TGEV PL1 could block the production of interferon. To further establish whether TGEV PL1 affects IRF3 phosphorylation or migration from the cytoplasm to nucleus, HEK293T cells were transfected with TGEV PL1 and/or IRF3-EGFP. Then the result was analyzed using Western blot and confocal microscopy. In Figure 1(f) , the level of p-IRF3 was decreased significantly by TGEV PL1 compared with that of SeV-induced. Furthermore, IRF3-EGFP was located in the cytoplasm compared with mock-infected HEK293T cells but translocated to the nucleus when the cells were inoculated with SeV. In contrast, after being inoculated with SeV, it was found that IRF3-EGFP was translocated from cytoplasm to nuclear in mock infected HEK293T cell, which was not observed in cells transfected with TGEV PL1 (Figure 1(g) ).Our results collectively suggested that TGEV PL1 suppressed IFN-transcription by interfering with NF-B-, IRF3-, and AP-1 signaling-mediated IFN expression.To determine whether TGEV PL1 is capable of blocking STINGmediated activation of the IFN-promoter, we assessed promoter activity in the presence of STING along with increasing amounts of TGEV PL1. Stimulation of HEK293T cells with STING alone resulted in a robust increase in IFNpromoter activity. Coexpression of STING and TGEV PL1 induced a dose-dependent decrease in IFN-activity, clearly indicating antagonistic activity of TGEV PL1 on STINGmediated activation of the IFN-promoter (Figure 2(a) ).STING dimerization is reduced in the presence of HCoV-NL63. STING dimmers are visualized as an 80 kD band on SDS-PAGE. To further determine whether TGEV PL1 inhibits STING-mediated signaling through disrupting the stability of STING dimers, HEK293T cells were cotransfected with plasmid DNA expressing STING in the presence or absence of TGEV PL1 and SeV, and cell lysates were evaluated for dimmers via immunoblotting (Figure 2(b) ). Interestingly, the results indicated that STING dimerization was not affected by TGEV PL1.Inhibiting IFN-Expression. To determine whether catalytic activity is required for TGEV PL1-mediated inhibition of IFN-expression, HEK293T cells were cotransfected with alanine mutants of three conserved catalytic residues of TGEV PL1 (C32A, H183A, and D196A) with or without RIG-1, MAVS, STING, or TBK-1, and IFN--Luc and pRL-TK plasmids, followed by infection with SeV to activate IFNpromoter activity. TGEV PL1 mutation at two of the catalytic sites (C32A and H183A) led to almost complete loss of IFN antagonistic activity, relative to wild-type TGEV PL1, but the D196A mutant showed a little inhibition for IFN-promoter activity (Figures 3(a), 3(b) , 3(c), 3(d), and 3(e)). Based on the results, we conclude that the intact catalytic triad of TGEV PL1 is required to inhibit activation of the IFN-promoter driven by STING and TBK-1. Recent studies have revealed that STING acts as a scaffold protein for TBK-1 and IRF3 and links them to the MAVS complex in mitochondria upon viral infection . Moreover, activation of STING is critical for stimulation of IRF-3 activity. Here, we observed that TGEV PL1 protein inhibits STING-and TBK-1-induced activation of IFN-. Additional localization experiments showed that PL1 existed in mitochondria (Figure 3(f) ).Modification of signaling molecules by ubiquitin (Ub) plays a critical role in activation of the IFN response. TGEV PL1 has been shown to possess DUB activity. Here, we investigated the DUB activities of TGEV PL1 and its catalytic mutants. HEK293T cells were cotransfected with pcDNA HA-Ub and TGEV PL1, and the level of ubiquitinated proteins was assessed via Western blot. The level of Ub-conjugated proteins was reduced dramatically in cells transfected with wild-type TGEV PL1, while the ubiquitinated Ub-HA level was not reduced in the presence of the C32A, H183A, and D196A mutants (Figure 4(a) ). Next, we investigated whether TGEV PL1 recognizes and deubiquitinates the key regulators, RIG-I and STING, in the IFN signaling pathway. TGEV is known to induce robust expression of IFN-at the late step of the replication and is distinct from CoVs [17, 18] . Moreover, TGEV infection activates transcription factors NF-B, IRF3, and AP-1 in porcine kidney cells and a delayed activation of the IFN response in intestinal epithelial cells [19, 20] . However, the mechanism of its evasion of the innate immune system has never been reported. The current study firstly showed antagonistic function of the TGEV PL1 protein against the IRF3 signaling pathway to inhibit IFNinduction through its DUB activity.To combat the host antiviral effects, coronaviruses likely take advantage of PL activity to escape from the host innate antiviral response. HCoV-NL63 (PL2-TM) and SARS-CoV (PLpro-TM) inhibit STING-mediated activation of IRF-3 nuclear translocation and induction of IRF-3-dependent promoters [6, 8] . PL2 of MHV strongly inhibits CARDIF-, TBK1-, and IRF3-mediated IFN-reporter activities and prevented nuclear translocation of IRF3 . PEDV PLP2 negatively regulated RIG-I and STING-mediated IFN-expression . Moreover, TGEV PL1 displays a similar structure to SARS-CoV PL2 and gives rise to the speculation that TGEV PL1 may similarly act as an IFN antagonist. In the present study, we first found that overexpressed TGEV PL1 inhibited STING-and TBK-1-mediated IFNtranscription and antagonized the type I IFN response stimulated by SeV in PK-15 cells. The catalytic activity of TGEV PL1 is essential for inhibiting IFN-transcription. Furthermore, STING dimerization is reduced in the presence of HCoV-NL63 PL2-TM, which was not affected by TGEV PL1. These results suggested that TGEV PL1 acted as an IFN antagonist to negatively regulate host antiviral innate immunity.Ubiquitination and deubiquitination are critically involved in regulation of virus-induced type I IFN signaling pathways [21, 22] . Recently, DUBs have been reported in a variety of viruses, such as foot-and-mouth disease virus, Lpro , human cytomegalovirus, UL48 , herpes simplex virus type 1, UL36 , and porcine reproductive and respiratory syndrome virus, nsp2 [26, 27] . Interestingly, all CoVs have evolved to encode DUB enzymes, which may contribute to modulation of the innate immune response. PLP of HCoV-NL63, SARS-CoV, MHV, PEDV, and MERS-CoV dramatically reduced the levels of ubiquitinated STING, RIG-I, TBK1, and IRF-3 to negatively regulate host antiviral innate immunity. Here, we showed that TGEV PL1 interferes with and significantly inhibits ubiquitination of RIG-1 and STING, which are essential activators of type I IFN signaling. Then, the levels of phosphorylated IRF-3 were reduced, which blocked nuclear translocation of IRF3 to activate the transcript of IFNs. Three catalytically inactive mutants of TGEV PL1 (C32A, H183A, and D196A) found to be defective in DUB activity failed to inhibit virus-induced INF-expression, indicating that the DUB function of TGEV PL1 is directly involved in inhibition of type I IFN induction. However, the membrane protein M and envelope protein E of TGEV were translated at the late step of the replication as the major inducing component of IFNs. Further studies are required to establish the precise functions of PL1 protease/DUB activity in coronavirus interactions with the host innate immune response.Our results are the first report identifying TGEV PL1 that is responsible for inhibiting the induction of IFN-. We found that TGEV PL1 displayed IFN antagonist activity dependent on the intact catalytic triad (C32, H183, and D196) and interfered with RIG-1-and STING-mediated signaling through a viral DUB activity. These characteristics of TGEV PL1 served as a multifunctional protein with a critical regulatory role in TGEV interactions with the host antiviral innate immune response. Moreover, these findings contribute to our understanding of the molecular mechanisms of innate immunity evasion strategies utilized by TGEV.\n",
            "\t  0.66818136 \t Supplementary Figure 1: Schematic representation of the HCV replicons. Schematic representation of subgenomic (top) and full-length (bottom) HCV replicons. The genotype and the name of plasmids are indicated in italic. RLuc: Renilla Luciferase; FLuc: Firefly Luciferase; Neo: neomycine resistance gene; EMCV: internal ribosome entry site of the encephalomyocarditis virus; NS: non-structural; C: core; E: envelope; GT: genotype. Supplementary Figure 2: Cellular toxicity of C31 and ALV in Huh7.5 and MRC5 cells. Cellular toxicity of C31 (black circle) and ALV (open triangle) was evaluated in Huh7.5 cells (A) and MRC5 cells (B) by means of a MTT assay.and CsA (C), as quantified by RT-qPCR. SMCypI compounds (C31, C32, C34, C35 and C36) synthesis was performed as described in our previous report (1) . The synthesis of compound C33 was performed as described in Scheme 1. Ester S1 (330 mg, 0.90 mmol) was dissolved in 10 ml of MeOH and 10 ml of water, LiOH (86 mg, 3.60 mmol, 4.00 equiv) was added and the reaction mixture was heated to 50°C for 2 h. The reaction mixture was concentrated and 100 ml of water was added; it was then extracted twice with AcOEt. The aqueous phase was acidified to pH 3 with concentrated HCl 36%, then extracted twice with AcOEt. The combined organic phase was dried over Na 2 SO 4 , filtered and concentrated to yield S2 (283 mg, 93%) as a white solid. 1 \n",
            "\t  0.6667315 \t Background: Members of the Baculoviridae encode two types of proteins that mediate virus:cell membrane fusion and penetration into the host cell. Alignments of primary amino acid sequences indicate that baculovirus fusion proteins of group I nucleopolyhedroviruses (NPV) form the GP64 superfamily. The structure of these viral penetrenes has not been determined. The GP64 superfamily includes the glycoprotein (GP) encoded by members of the Thogotovirus genus of the Orthomyxoviridae. The entry proteins of other baculoviruses, group II NPV and granuloviruses, are class I penetrenes.The entry of enveloped animal viruses into target cells occurs via fusion of the viral membrane with a cellular membrane. Penetrenes are viral membrane proteins that mediate penetration into the host cell. The penetrenes of enveloped animal viruses can be divided on the basis of common structural motifs into at least three classes. Orthomyxoviruses, retroviruses, paramyxoviruses, arenaviruses, and coronaviruses encode class I penetrenes , which are also known as class I viral fusion proteins or α-penetrenes. Class I penetrenes contain a \"fusion peptide,\" a cluster of hydrophobic and aromatic amino acids located at or near the amino terminus, an amino terminal helix (N-helix, HR1), a carboxyl terminal helix (C-helix, HR2), usually an aromatic amino acid (aa) rich pre-membrane domain and a carboxyl terminal anchor [1, 7, 2, 8, 9] . Envelope glycoprotein (E) and envelope glycoprotein E1 encoded respectively by members of the Flavivirus genus of the Flaviviridae and the Alphavirus genus of the Togaviridae are class II penetrenes (β-penetrenes) . Class II penetrenes possess three domains (I-III) comprised mostly of antiparallel β sheets, a membrane proximal αhelical stem domain and a carboxyl terminal anchor. The fusion loops of class II penetrenes are internal and located in domain II. Members of the two other Flaviviridae genuses, Hepaciviruses and Pestiviruses, appear on the basis of proteomics computational analyses to encode truncated class II penetrenes . Proteomics computational analyses suggest that the carboxyl terminal glycoproteins (Gc) of bunyaviruses, and similar proteins of tenuiviruses and a group of Caenorhabditis elegans retroviruses, are also class II penetrenes . Additional evidence that bunyavirus Gc are class II penetrenes has been provided [15, 16] .Recent studies have provided evidence for a third class of viral penetrenes (class III or γ-penetrenes). The entry glycoprotein (G) of vesicular stomatitis virus (VSV), a rhabdovirus, contains a fusion domain comprised of β sheets, other β sheet domains, an extended α-helical domain, a membrane proximal α-helical stem domain and a carboxyl terminal anchor [17, 18] . On the basis of sequence similarity it is likely that G of other members of the Rhabdoviridae are also class III penetrenes. Although larger, glycoprotein B (gB) of herpes simplex virus type 1 (HSV-1) and by sequence similarity gB of other herpesviruses, were unexpectedly demonstrated to share several structural features with VSV G . The extended α-helices in the post-fusion forms of G and gB are involved in trimerization, as is well documented for α-helices in the postfusion structures of class I penetrenes. The fusion domains of rhabdovirus G and herpesvirus gB are very similar structurally to the fusion domains of class II penetrenes . Therefore, class III penetrenes may share a common progenitor(s) with members of other penetrene classes.Members of the Baculoviridae are enveloped doublestranded DNA viruses of arthropods that are subdivided into two genuses, Nucleopolyhedrovirus (NPV) and Granulovirus (GV). NPV are further subdivided into group I and II. Baculoviruses encode two distinct penetrenes [21, 22] . Entry proteins of group I NPV are all approximately 64 kilodalton glycoproteins (GP64), and are referred to collectively as GP64 superfamily proteins . Group II NPV and GV encode entry proteins referred to as fusion proteins (F) [22, 24] . Group I NPV often encode both GP64 and F homologues, although in these viruses F is nonfunctional. Autographa california multiple NPV (AcMNPV) lacking GP64 can be pseudotyped by the F protein of Spodoptera exigua MNPV , suggesting that F of group II NPVs and GV can serve as a functional analog of GP64. However, GP64 cannot serve as an analog of F . Baculovirus F are class I penetrenes. Structural similarities exist between baculovirus F, the envelope glycoproteins of insect retroviruses (errantoviruses), the envelope glycoprotein of the gypsy retrotransposon of Drosophila mela-nogaster and other class I penetrenes . Like other class I penetrenes, baculovirus F is present in virions as a homotrimer and synthesized as a precursor (F 0 ), which is subsequently cleaved by furin-like proteases into subunits F 1 and F 2 [27, 28] . Prior studies have not revealed structural relationships between baculovirus GP64 proteins and other penetrenes.Thogoto virus (THOV) is a tick-transmitted virus, which is classified in the Thogotovirus genus of the Orthomyxoviridae. The genome of THOV comprises six segments of single-stranded, negative-sense RNA. The fourth largest RNA segment of THOV encodes a glycoprotein (GP) that has significant similarity with corresponding proteins of Dhori, Araguari, and Batken viruses and other thogotoviruses. Thogotovirus GP do not share significant sequence similarities with the class I penetrenes, hemagglutinin 2 (HA2) or hemagglutinin-esterase 2 (HE2), encoded by members of the three influenza virus genuses (types A, B and C) of the Othomyxoviridae or the fusion (F) protein or HE2 encoded by members of the Isavirus genus, the fifth orthomyxovirus genus . However, thogotovirus GP share significant sequence similarity with baculovirus GP64, and are included in the GP64 superfamily [30, 31] . Here, we present the results of proteomics computational analyses that suggest that GP64 superfamily members are class III penetrenes.Sequence and structural comparisons were performed for THOV strain SiAr 126 envelope glycoprotein precursor (THOV GP, accession number P28977), the AcMNPV GP64 superfamily protein (AcMNPV GP64, P17501) and other GP64 superfamily members. Representatives of G from six genera of the Rhabdoviridae were also used for sequence and structural comparisons: Vesiculovirus: VSV strain Indiana (AAA48370); Lyssavirus: rabiesvirus strain street (AAA47211); Ephemerovirus: bovine ephemeral fever virus structural G (P32595) and nonstructural G (P32596); Novirhabdovirus: infectious hematopoietic necrosis virus (CAA61498); Cytorhabdovirus: lettuce necrosis yellows virus glycoprotein (LYP425091); Nucleorhabdovirus: rice yellow stunt virus (AB011257) and an unclassified rhabdovirus: Taastrup virus (AY423355). We also compared GP64 superfamily members to penetrenes of representative members of the Herpesviridae, Flaviviridae, Togaviridae, and Bunyaviridae. Comparisons of F from ISAV strain RPC/NB 98-049-1 (ABE98322) and strain RPC/NB 98-0280-2 (ABE02810), F from Spodoptera exigua MNPV (AAF33539) and retrovirus-related Env polyprotein from transposon gypsy (P10403) were made to HA from influenza A virus strains A/WSN/1933 (H1N1, AAA3209), A/Aichi/2/1968 (H3N2, AAA43178), A/ udorn/1972 (H3N2, ABD79032), A/guinea fowl/Italy/ 330/97 (H5N2, AF194991), A/chicken/Korea/S20/2004 (H9N2, AAV68031) and influenza B virus, strain B/Texas/ 37/1988 (ABN50602). Comparisons were also made amongst HE of influenza C virus strains Yamagata/9/88 (BAA06094) and C/Johannesburg/1/66 (CAL69520), ISAV strain T91/04 (AAY40756), human coronavirus OC43 strain ATCC VR-759 (AAR01014) and human torovirus (AAF00614).Methods developed by William Gallaher and coworkers to derive models of viral surface glycoproteins have been described previously [7, 3, 2, 5] . William Pearson's LALIGN program, which implements a linear-space local similarity algorithm, was used to perform regional alignments. PHD (Columbia University Bioinformatics Center), which is part of the ProteinPredict suite was the preferred method of secondary structure prediction. Domains with significant propensity to form transmembrane helices were identified with TMpred (ExPASy, Swiss Institute of Bioinformatics). TMpred is based on a statistical analysis of TMbase, a database of naturally occurring transmembrane glycoproteins . Sequences with propensity to interface with a lipid bilayer were identified with Membrane Protein eXplorer version 3.0 from the Stephen White laboratory using default settings , which can be used to calculate scores on the Wimley-White interfacial hydrophobicity scale (WWIHS) . MacPymol was used to render 3D models of VSV G (2cmz.pdb) and HSV-1 gB (2gum.pdb) in the post-fusion configurations. These models were extrapolated to THOV GP and AcMNPV GP64 using Photoshop (Adobe) and Freehand (Macromedia).Gallaher and co-workers employed the fusion peptide and other conserved features in combination with computer algorithms that predict secondary structure, to construct working structural models of several viral entry/fusion proteins, collectively referred to here as class I penetrenes [7, 2, 3, 5, 6] . This strategy has proven to be highly predictive of structures solved later by X-ray crystallography [4, 36] . Gallaher's strategy, supplemented with increasingly robust proteomics computational tools, can also be applied to discovery of potential structures of viral penetrenes that belong to class II [13, 14] . Here, we apply these methods to THOV GP and AcMNPV GP64, representative members of the GP64 superfamily.The PHD algorithm predicts protein secondary structure from multiple sequence alignments by a system of neural networks, and is rated at an expected average accuracy of 72% for three states, helix, strand and loop. Application of PHD to VSV G, a prototypic class III penetrene , reveals that predicted secondary structures ( Fig. 1 , α-helix and β-sheets depicted with dashed lines) closely correspond to the structures determined by X-ray crystallography (colored cones and arrows, accuracy; 77.5%). The PHD algorithm predicts that there is an extended α-helix in THOV GP (aa 284-338; blue cone) and AcMNPV GP64 (aa 284-340). With the exception of this extended αhelix, the ectodomains of the GP64 superfamily proteins are comprised mostly of β-sheets (colored arrows). Another domain readily identifiable with computational tools in THOV GP and AcMNPV GP64 is the carboxyl terminal transmembrane anchor. TMpred, an algorithm that identifies possible transmembrane helices, assigns significant scores (> 500 is statistically significant) to THOV GP aa 471-491 (score: 2428) and AcMNPV GP64 aa 470-490 (3030), which suggests that these sequences represent the transmembrane anchors (violet cones). PHD analyses also predict the presence of an α-helical stem domain with several aromatic aa (indigo cones) in THOV GP (aa 442-472) and AcMNPV GP64 (aa 441-471) prior to the transmembrane anchor, a feature present in both class II and III penetrenes 18] .The structural determinations of VSV G and HSV-1 gB were performed by independent groups and although it was established that similar domains/structures are present, a consistent domain nomenclature for these class III penetrenes was not used (compare Fig. 2A with Fig. 2B ) [17, 19] . The fusion domains of class II and III penetrenes have a highly similar structure. Therefore, a class III domain nomenclature is used here that can apply to both rhabdovirus G and herpesvirus gB and assigns domain II (IV in the VSV G nomenclature of Roche et al. , I in the HSV-1 gB nomenclature of Heldwein et al. ) as the class III fusion domain as in class II penetrenes. In addition to minor adjustments in the ends of domains, the current class III penetrene numbering also combines two interacting domains into domain III (I + II in Roche's VSV G nomenclature, III + IV in Heldwein's HSV-1 gB nomenclature). The fusion domains of all class II or III penetrenes contain 1 or 2 prominent fusion loops, which give significant scores on the WWIHS . Sequences with positive WWIHS have a high potential to interface with or disrupt lipid membranes, and therefore are key features of viral penetrenes. Another feature of the fusion domains of class II and III penetrenes is the presence of several dicysteine bonds, which appear to stabilize the overall domain architecture. Regions in THOV GP (aa and AcMNPV GP64 (aa 49-186) with 6 or 8 cysteine residues, plus 1 or 2 sequences with positive WWIHS scores (Fig. 1 , red letters) are likely to represent the fusion domains.A prominent feature of class III penetrenes is an extended α-helix beginning near the carboxyl terminal third of the ectodomain (domain III), which is involved in trimerization of the post-fusion structure [17, 19] . The extended αhelices predicted by PHD in THOV GP and AcMNPV GP64 correspond to this location. As noted previously , the sequence of the predicted helices is consistent with that of a leucine zipper (mostly leucines or isoleucines in the first and fourth positions of seven amino acid repeats), as is the case for both VSV G (Fig. 1 , blue bars) and HSV-1 gB (not shown). The α-helices in the GP64 proteins are predicted to be several helical turns longer than the major helix (helix H) of the post-fusion structure of VSV G, but comparable in length to the major α-helix of HSV-1 gB.Sequence similarities between VSV G, THOV GP and AcM-NPV GP64 do not permit alignment by computational methods alone. However, using the regions of local structural similarity including the putative fusion domain/ loops, extended α-helices and transmembrane domains, all of which are collinear, alignments between VSV G, Collinear arrangement of similarities in THOV GP, AcMNPV GP64 and VSV G Figure 1 Collinear arrangement of similarities in THOV GP, AcMNPV GP64 and VSV G. A common domain nomenclature for class III penetrenes is utilized: domain I (green), domain II (yellow), domain III (blue), and stem domain (indigo). The domain numbering originally proposed is also indicated . UA represents \"hinge\" aa not assigned to domains in VSV G in the prior scheme. Sequences with significant WWIHS scores in the fusion domain (II) were identified by MPeX and colored red. Hydrophobic transmembrane domains (violet) were predicted using TMpred. The post fusion secondary structure of VSV G as solved and numbered by Roche and coworkers is depicted with α-helices as cylinders and β-sheets as arrows. The α-helices predicted by PHD In THOV GP and AcMNPV GP64 are indicated similarly. β-sheets (t) and (u) of VSV G are not present in the protein data base structure (2cmz.pdb). In VSV G, α-helices predicted by PHD are indicated by dashed boxes and predicted β-sheets are identified with dashed arrows. Amino acids are numbered beginning after the putative signal sequences enclosed in parentheses. In the alignments (:) refers to identical amino acids. (.) refers to chemically similar amino acids. Plum amino acids: N-glycosylation sites. THOV GP and BV GP64 are proposed (Figs. 1, 2) . These alignments support assignment of a common domain architecture for these proteins. The proposed domains of these GP64 superfamily members are also collinear with analogous domains of herpesvirus gB, the other prototypic class III penetrene (Fig. 2) .Cysteine residues are usually the most conserved aa within a protein family because disulfide bonds between cysteines are critical determinants of secondary structure. The cysteines of class III (and class II) penetrenes determined by X-ray crystallography are arranged such that most disulfide bonds are formed between cysteine residues within the same domain (Fig 2) . To determine the plausibility of the proposed alignment, models of THOV GP and AcMNPV GP64 scaffolded on the structure of VSV G in the post-fusion (low pH) configuration were constructed (Fig. 3) . The alignments between VSV G, THOV GP and AcMNPV suggest that these penetrenes may have a similar structure. Therefore, putative structures in the GP64 superfamily members are depicted as in VSV G. The proposed THOV GP and AcMNPV GP64 models are based principally on the structural predictions of PHD, the most robust secondary structure prediction algorithm used. These results provide evidence that the 6 or 8Similar linear arrangement of putative domain structures of THOV GP and AcMNPV GP64 compared to domain structures of VSV G and HSV-1 gB Figure 2 Similar linear arrangement of putative domain structures of THOV GP and AcMNPV GP64 compared to domain structures of VSV G and HSV-1 gB. Amino acids are numbered beginning after the putative signal sequences in VSV G, but at the beginning of the signal sequence of HSV-1 gB. Arrows indicate G and gB truncations of the forms used for crystallography. Solid lines represent cysteine bonding in VSV G and HSV-1 gB. Black boxes represent hydrophobic regions, with violet representing the transmembrane anchor (TM) . Dashed lines represent potential cysteine bonding in THOV GP and AcMNPV GP64. Panel A: class III penetrene domain nomenclature and coloring as in Fig. 1 III IV I II III III V The results of these analyses suggest that the locations of the glycosyl residues may be conserved in class III penetrenes. Domain I of VSV contains a consensus glycosylation motif (NXS/T) between β-sheets h and I (Fig. 1) . The other glycosylation site in VSV G is located between βsheets r and s in domain III. THOV GP, AcMNPV and other GP64 superfamily members have similarly located glycosylation sites on or between predicted β-sheets corresponding to VSV G β-sheets h and i and r and s (Figs. 1, 3) .The THOV GP and AcMNPV GP64 structural models are not intended as definitive structural predictions. Rather, there are many possible alternatives to the secondary and tertiary structures and the cysteine linkages of these and other GP64 superfamily members. The modeling does establish that feasible structures exist that are consistent with the secondary structure predictions and with the assignment of GP64 superfamily members as class III penetrenes. The results of this structural modeling also provide further support for the proposed alignments of VSV G with THOV GP and AcMNPV GP64.Our computational analyses and molecular modeling studies suggest that thogotovirus penetrenes are structurally distinct from penetrenes encoded by viruses in the three other genuses of the Orthomyxoviridae, influenza A, B and C viruses. The fifth genus in the Orthomyoviridae, Isavirus, is represented by infectious salmon anemia virus (ISAV). ISAV encodes two glycoproteins, one of which (HE) has hemagglutinin and esterase activities . The other ISAV glycoprotein is a class I penetrene designated the fusion protein (F). Previous studies by Aspehaug and coworkers indicated that like other class I penetrenes,Models of THOV GP and AcMNPV GP64 based on the X-ray crystallographic structure of VSV G Figure 3 Models of THOV GP and AcMNPV GP64 based on the X-ray crystallographic structure of VSV G. The predicted structures of THOV GP and AcMNPV GP64 were fit to the post-fusion structure of VSV G . Secondary structures for THOV GP and AcMNPV GP64 were predicted by PHD or by alignment to VSV G. The structure of HSV-1 gB is shown for comparison. Domain coloring as in Fig. 1 and Fig. 2A . Orange/black lines: dicysteine linkages as in Fig. 2 . Black stick figures: Nglycosylation sites.ISAV F 1 is produced by cleavage of a precursor (F 0 ) that exposes a hydrophobic fusion peptide near the amino terminus . To further investigate the relationship of isavirus F 1 to the class I penetrenes of other orthomyxoviruses sequence comparisons and molecular modeling were conducted. While the alignment we report here is somewhat different than that proposed previously , it is also consistent with the designation of ISAV F 1 as a class I penetrene (Fig. 4) . ISAV F 1 is shorter in overall length than HA2 of either influenza A or B viruses or HE2 of influenza C viruses, but comparable in length to certain class I penetrenes, such as glycoprotein 2 of Ebola virus. PHD reveals the presence of an extended α-helical domain that corresponds to the N-helix of influenza A and B viruses. The N-helix of influenza A and B virus HA2 features a leucine zipper, which is involved in trimerization. Other similarities include the presence of a conserved cluster of three cysteine residues (ISAV F aa 382-390) and an aromatic pre-anchor domain (aa 414-421). Of note is the location of the C-helix in ISAV F 1 . In comparison to most other class I penetrenes, the Chelices of influenza viruses in HE2 or HA2 are shorter and located more distally from the C-terminus. A sequence termed the leash is located between the C-helix and the aromatic domain. The formation of the post-fusion configuration of influenza A, B and C virus HA2 or HE2 is best described by a leash-in-grove mechanism, rather than by a six-helix bundle mechanism as in most other class I pen- etrenes . The location of the C-helix in ISAV F 1 suggests that this penetrene shares a common progenitor with HA or HE of influenza A, B and C viruses, and mediates fusion/penetration by a leash-in-grove mechanism.Proteomics computational analyses suggest that GP64 superfamily proteins are class III penetrenes. Each of the major features common to class III fusion proteins are present in THOV GP and AcMNPV GP64, including internal fusion loops, an extended α-helical domain, a stem domain and a carboxyl terminal transmembrane domain. These features are located collinearly with these features in VSV G, a prototypic class III penetrene [17, 18] . On the basis of sequence similarities among the GP64 superfamily members it is likely that all are class III penetrenes. Previous studies have suggested a role for the putative extended α-helix and the leucine zipper motif in GP64 mediated fusion/entry, but did not assign GP64 to any penetrene class [40, 43] . Our results do not corroborate the previous conclusion that a 6 aa sequence (AcM-NPV aa 209-214 in Fig. 1 ) may be the GP64 fusion peptide. Structural models, which include feasible cysteine linkage maps, could be established for THOV GP and AcMNPV GP64. The fusion domains of THOV GP and AcMNPV GP64 appear to be stabilized by cysteine bonds and to contain one or more loops with positive WWIHS scores, features that are characteristic of the fusion domains of both class II and III penetrenes. Glycosylation sites in THOV GP and AcMNPV GP64 appear in similar model locations to the two glycosylation sites of VSV G. Whether or not the secondary and tertiary folding of GP64 superfamily members conform to the domain structure of class III penetrenes will require x-ray crystallographic or other physical structural determinations.The three penetrene classes for enveloped virus membrane glycoproteins were established based on structural similarities in the post-fusion configurations. Therefore, it is likely that there is a common post-fusion (low pH) configuration of class III penetrenes, and that GP64 superfamily members have a post-fusion structure similar to VSV G. In contrast, the prefusion configurations of class I, II and II penetrenes are highly variable. The virion configuration of VSV G is homotrimer arranged in a tripod shape with the fusion domains corresponding to the legs of the tripod . No structural prediction of the prefusion configurations of GP64 superfamily members is possible.Conversion of the virion configuration of VSV G to the fusion competent form occurs upon exposure to low pH in the infected cell. Current models suggest that low pH may permit reversible bending of VSV G at \"hinge\" regions flanking domain I elevating the fusion loop(s) for insertion into the host membrane . Additional rearrangements of VSV G involve a rotation around the hinge, unfolding of α-helix A 0 and formation of helix C, interactions of the stem with domains I-III, and formation of higher multimers of the trimers. The order in which these steps occur has not been established. These changes in VSV G are hypothesized to drive deformation of the viral and target membranes. Complete cell membrane:virion membrane fusion follows, allowing entry of the ribonucleoprotein containing the viral genomic RNA. It is likely that GP64 superfamily members follow a mechanism of fusion similar to rhabdovirus G. In the case of HSV-1 gB there may be differences in the rearrangements due to size and cysteine bonding patterns of this class III penetrene [19, 20] . Rearrangements involving a hinge region also occur in class II penetrenes during entry [11, 12, 44] . A mechanism involving rearrangement of functional domains has also been proposed for class I penetrenes as well as the penetrenes of non-enveloped viruses . In the case of influenza A virus HA2, the prototypic class I penetrene, the rearrangement results in formation of a trimer of the N-helices stabilized by an internal leucine zipper . The leash sequence interacts with the external groove of the N-helix trimer. For other class I penetrenes the rearrange brings together the N-and C-helices into a six-helix bundle . The F protein of isaviruses appears to utilize a leash-in-the groove mechanism of membrane fusion.Orthomyxoviridae, Retroviridae, Paramyxoviridae, Filoviridae, Arenaviridae, and Coronaviridae and Baculoviridae have members that encode class I penetrenes 36] . Syncytin, encoded by a human endogenous retrovirus (HERV-W), is also a class I penetrene with has a critical role in membrane fusion events involved in placental morphogenesis. Syncytin may also play a pathogenic role in cancer and autoimmunity . Flaviviridae, Togaviridae, and Bunyaviridae family members are known or appear to have members that encode class II penetrenes [10, . If the current analyses are correct, GP64 superfamily members join rhabdovirus G and herpeviruses gB as class III penetrenes. While convergence to common structures is possible, penetrenes of enveloped viruses may have evolved from a limited number of common progenitors. Support for this hypothesis comes from the remarkable similarities in the post-fusion structures of the penetrenes in each class, even though the proteins differ dramatically in aa sequence. While, it is likely that other classes of penetrenes exist for enveloped viruses, there may be a limited number of effective structures for virus-mediated membrane fusion.Similar penetrenes are not present in all contemporary members of the Orthomyxoviridae . Reassortment of segmented viruses is a well-establish phenomenon, and it is possible that orthomyoviruses diverged via the acquisition of segments encoding distinct penetrenes (Fig 5) . The HA proteins of type A and B influenza viruses lack several structural domains present in influenza C virus HE, although the carboxyl terminal proteins (HA2 and HE2) derived from both HA and HE are class I penetrenes .Members of the fifth genus in the Orthomyoviridae, Isavirus, represented by ISAV, encode two glycoproteins, HE with hemagglutinin and esterase activities and F, a class I penetrene . The progenitor of the penetrenes of members of the three influenza virus genuses (A, B, and C) and ISAV may have had a segment encoding an HE-like class I penetrene that diverged to the HA in influenza A and B viruses, HE in influenza C viruses and F in isaviruses. Alternatively, HA or F could have evolved from HE or another penetrene by loss or acquisition of the esterase sequences. ISAV HE shares limited sequence similarities with the more closely related HE of influenza C viruses, coronaviruses and toroviruses . Thogotoviruses appear to have acquired a distinct penetrene possibly from a common progenitor with the GP64 superfamily. It is not possible to root the tree of Orthomyxoviridae with regards to acquisition of penetrenes. The distinct penetrenes, class I or III, appear to have been acquired independently by different orthomyxovirus genuses, but it is unclear which, if any, was present prior to divergence of this family.As previously discussed , GP64 penetrenes seem to have been acquired after divergence of the two main groups of Baculoviridae. Therefore, it is possible to root this tree with regards to penetrenes (Fig. 5) . In this scenario, the baculovirus progenitor acquired F, a class I penetrene. One particular lineage then also acquired GP64, which we suggest are class III penetrenes, after the split into the two distinct groups of NVP and GV. Baculoviruses have large DNA genomes, and mechanisms of genetic exchange are distinct for those of RNA viruses. In contrast, the G gene appears to have been present in the common ancestor of all members of the Rhabdoviridae. The similarities detected between GP64 superfamily members and rhabdovirus G are consistent with divergent evolution from a common progenitor, but sequence similarities are insufficient to establish a phylogenic relationship. It is unlikely that there are any recent common ancesters of rhabdoviruses and baculoviruses, and that the class III penetrenes of these viruses were acquired by independent genetic events. The gB of herpesviruses of birds, mammals and reptiles have a high degree of conservation, and are likely to all represent class III penetrenes . A gB-like progenitor probably was present in the common ancestor of these herpesviruses. Other viral glycoproteins (gC, gD, gH/gL) are involved in herpesvirus fusion and entry . These additional entry proteins are differentially distributed among members of the Herpesviridae, and it is likely that they were acquired after acquisition of gB by the herpesvirus progenitor. Herpesvirus gB is nearly twice as long as VSV G or GP64 superfamily proteins. Assuming that the structure of gB is not an extreme example of convergence to a class III penetrene structure, it appears to have undergone extensive insertions of sequences from a common class III progenitor. Alternatively, the class III progenitor could have been a longer protein that deleted sequences prior to independent acquisitions by rhabdoviruses, thogotoviruses or baculoviruses.The author(s) declare that they have no competing interests.Acquisition of class I and/or III penetrenes by members of the Orthomyxoviridae, Baculoviridae, Rhabdoviridae and Herpesviridae Figure 5 Acquisition of class I and/or III penetrenes by members of the Orthomyxoviridae, Baculoviridae, Rhabdoviridae and Herpesviridae. Thick lines indicate primordial lineages and thin lines are lineages leading to contemporary viruses. The orthomyxovirus tree is unrooted, while the baculovirus, rhabdovirus and herpesvirus trees are rooted. Adapted and updated from Pearson and Rohrmann . \n",
            "\n",
            " Title:  Contribution of porcine aminopeptidase N to porcine deltacoronavirus infection\n",
            "1 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.5795569 \t PCV3/ GDHL3-2017 swine China 12-2017 MH491019 PCV3/GDHL2-2017 swine China 05-2017 MH491018 PCV3/GDHL1-2017 swine China 02-2017 MH491017 PCV3/GDDH2-2018 swine China 03-2018 MH491016 PCV3/GDDH1-2018 swine China 01-2018 MH445394 Nanjing-BALB-C4 \n",
            "\t  0.5454194 \t SEIR susceptible-exposed-infected-recovered model, IDEA incidence decay and exponential adjustment model, ABM agent-based model, BDEI birth-death-exposed-infected model, BD birthdeath model *corresponds to R e for the early period of May 27-August 4, 2014\n",
            "\t  0.50521886 \t On February 7, 2020 , this report was posted as an MMWR Early Release on the MMWR website (https: //) .In December 2019, a cluster of cases of pneumonia emerged in Wuhan City in central China's Hubei Province. Genetic sequencing of isolates obtained from patients with pneumonia identified a novel coronavirus (2019-nCoV) as the etiology (1) . As of February 4, 2020 , approximately 20,000 confirmed cases had been identified in China and an additional 159 confirmed cases in 23 other countries, including 11 in the United States (2, 3) . On January 17, CDC and the U.S. Department of Homeland Security's Customs and Border Protection began health screenings at U.S. airports to identify ill travelers returning from Wuhan City (4) . CDC activated its Emergency Operations Center on January 21 and formalized a process for inquiries regarding persons suspected of having 2019-nCoV infection (2) . As of January 31, 2020, CDC had responded to clinical inquiries from public health officials and health care providers to assist in evaluating approximately 650 persons thought to be at risk for 2019-nCoV infection. Guided by CDC criteria for the evaluation of persons under investigation (PUIs) (5) , 210 symptomatic persons were tested for 2019-nCoV; among these persons, 148 (70%) had travel-related risk only, 42 (20%) had close contact with an ill laboratory-confirmed 2019-nCoV patient or PUI, and 18 (9%) had both travel-and contact-related risks. Eleven of these persons had laboratory-confirmed 2019-nCoV infection. Recognizing persons at risk for 2019-nCoV is critical to identifying cases and preventing further transmission. Health care providers should remain vigilant and adhere to recommended infection prevention and control practices when evaluating patients for possible 2019-nCoV infection (6) . Providers should consult with their local and state health departments when assessing not only ill travelers from 2019-nCoV-affected countries but also ill persons who have been in close contact with patients with laboratory-confirmed 2019-nCoV infection in the United States.As part of CDC's Emergency Operations Center activation, CDC personnel assist state and local health departments with the evaluation of 2019-nCoV PUIs. Public health laboratories were not yet conducting 2019-nCoV testing during the period covered by this report, while awaiting Food and Drug Administration emergency use authorization for the test. (The authorization occurred on February 4*). Therefore, all testing was conducted at CDC. A call center was staffed by a team of physicians and nurses 24 hours per day. During January 17-31, criteria used to determine whether a person was considered to be a PUI included presence of fever and symptoms of lower respiratory tract illness (e.g., cough or difficulty breathing) in addition to epidemiologic risk. Epidemiologic risk factors included history of travel from Wuhan City, close contact with a patient with laboratory-confirmed 2019-nCoV infection, or close contact with an ill PUI. Given the evolving understanding of 2019-nCoV epidemiology, testing was recommended for some persons who did not strictly meet the PUI definition, based on clinical discretion. For clinical inquiries that resulted in 2019-nCoV testing, real-time reverse transcription polymerase chain reaction testing was conducted at CDC using methods developed specifically to detect 2019-nCoV (7).For this report, CDC reviewed inquiries regarding potential 2019-nCoV PUIs received by CDC through January 31, 2020, from state and local health departments, health care providers, and airport health screening personnel. Information was compiled from call logs and PUI forms to assess source of inquiry, PUI demographic data (including age and sex), clinical information, epidemiologic risk factors, and 2019-nCoV test results. To allow for delays in specimen shipping and testing, data for PUIs for whom an initial inquiry was received during January 2020 were collected through February 4, 2020.During January 2020, approximately 30 CDC physicians and nurses responded to inquiries regarding approximately 650 persons. Testing was recommended for 256 persons ( Figure) across 34 jurisdictions (the jurisdictions included states, the District of Columbia, Puerto Rico, and the U.S. Virgin Islands) and was completed for 210 persons. Testing of PUIs was not always performed because alternative diagnoses were made, or symptoms resolved. Among inquiries resulting in testing, six (3%) persons were identified through airport screening, 178 (85%) in a health care setting, and 26 (12%) through contact tracing (Table) . Among 178 persons identified in a health care setting, the type of setting was reported for 125 (70%), including 79 (63%) who were evaluated at an emergency department or hospital, 22 (18%) at a student clinic, and 24 (19%) in other outpatient care settings. A total of 115 (55%) persons tested were male, and median age was 29 years (interquartile range = 21-49 years). Seventeen (8%) were health care workers, and 48 of 129 persons with available information were reported to be college students.All 210 persons who were tested were symptomatic: 143 (68%) had subjective fever or a measured temperature ≥100.4°F (≥38°C), and 189 (90%) had cough or shortness of breath. Upper respiratory tract symptoms (i.e., sore throat, rhinorrhea, or congestion) were common and were present in nine persons who did not have cough or shortness of breath. Thirty persons were reported to test positive for another respiratory viral pathogen, including influenza or respiratory syncytial virus. Forty-two (20%) patients were hospitalized, and four (2%) were admitted to an intensive care unit. One patient was deceased at the time of notification; testing for this person was negative, and an alternative cause of death was established. Travel-related risk was identified for 148 (70%) persons, 42 (20%) had close contact with ill patients with laboratory-confirmed 2019-nCoV infection or PUIs, 18 (9%) had both travel-and contact-related risks, and two (<1%) had possible contact with a laboratory-confirmed 2019-nCoV patient and were therefore tested.Among the 210 persons tested, 11 (5%) were found to have 2019-nCoV infection. Nine of these persons had traveled to Wuhan City; two persons had not traveled but had been in close contact with patients with laboratory-confirmed 2019-nCoV in the United States. All were symptomatic with fever (subjective or measured) or cough. 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 CDC's Emergency Operations Center activated * Confirmed cases were reported as of January 31, 2020. † Public announcements of a confirmed 2019-nCoV case in the United States were made on the following dates: Jan 21, Jan 24, Jan 26, Jan 27 (two cases), Jan 30, and Jan 31. (20) Travel from China and close contact identified † † 18 (9) Other risk § § 2 (<1)Abbreviations: ICU = intensive care unit; IQR = interquartile range; PUI = person under investigation. * Numbers might not sum to total because of missing data. † For this person, testing was negative for 2019-nCoV, and an alternative cause of death was established. § Additional persons who were being followed through contact tracing but initially sought treatment at a health care setting are not included in this category. ¶ Includes 113 persons who traveled from Wuhan City and 35 who traveled from areas of China outside Wuhan within 14 days of symptom onset. ** Includes 33 persons who were close contacts of an ill laboratory-confirmed 2019-nCoV patient and nine who were close contacts of PUIs. All contacts occurred within 14 days of symptom onset. † † Includes four persons who traveled from Wuhan City and were close contacts of an ill laboratory-confirmed 2019-nCoV patient, 11 who traveled from Wuhan City and were close contacts of PUIs, and three who traveled from China and were close contacts of a PUI. § § Had possible contact with a laboratory-confirmed 2019-nCoV patient and were therefore tested.persons suspected of being at risk for 2019-nCoV to public health officials and health care providers throughout the United States. Epidemiologic risk factors among the 210 persons tested for 2019-nCoV were not limited to travel: 20% of PUIs tested had not recently traveled to China but reported close contact with a person being evaluated for 2019-nCoV infection. Because person-to-person transmission is expected to continue, and as further travel restrictions are implemented, it is likely that the proportion of PUIs with such contact risk in the United States will increase among all persons evaluated for 2019-nCoV. CDC mobilized early in the response and state and local health departments similarly increased capacity to provide clinical consultation regarding 2019-nCoV. The collection of clinical and epidemiologic data that described characteristics of persons tested for 2019-nCoV helped to inform changes to criteria for PUI evaluation.On January 31, 2020, CDC published updated PUI guidance (8) in response to the evolving global epidemiology of 2019-nCoV, including the rapid geographic expansion and documentation of person-to-person transmission (9) . Updated guidance emphasizes 2019-nCoV testing for symptomatic persons in close contact with patients with laboratory-confirmed 2019-nCoV infection, persons returning from Hubei province in addition to Wuhan City, and persons from mainland China requiring hospitalization because of fever and lower respiratory tract illness. Additional refinements to this approach likely will be needed in the future as understanding of 2019-nCoV epidemiology continues to improve.The findings in this report are subject to at least three limitations. First, the number of clinical inquiries received by CDC does not represent all inquiries received by health departments. Second, because the primary objective of this effort was to guide a timely public health response, some clinical and epidemiologic risk factor data might be incomplete. Finally, given the limited available epidemiologic information early in the outbreak, to provide some latitude for clinical decision-making regarding diagnostic testing, the PUI definition was not strictly applied in all cases.A coordinated national effort to diagnose 2019-nCoV among persons at high risk for infection is important to facilitate appropriate management and limit transmission in the United States. CDC's website provides guidance for health care professionals on evaluating persons for 2019-nCoV (10) . Clinicians should maintain a high index of suspicion for possible 2019-nCoV illness not only among persons with fever and lower respiratory tract illness who report travel from China in the past 14 days but also symptomatic persons who have had close contact with patients with laboratory-confirmed 2019-nCoV infection. Clinicians should consult their local and state health departments when they suspect possible 2019-nCoV illness to facilitate diagnosis and aid prevention efforts.What is already known about this topic?During a 2020 outbreak of novel coronavirus (2019-nCoV) infection, CDC provided consultation to public health officials and health care providers evaluating persons at risk for 2019-nCoV infection.What is added by this report?During January 2020, CDC responded to clinical inquiries regarding approximately 650 persons in the United States and tested 210 for 2019-nCoV, one fifth of whom reported no recent travel-related risk but had close contact with a 2019-nCoV patient or a person under investigation for 2019-nCoV in the United States.What are the implications for public health practice?Health care providers should remain vigilant regarding possible 2019-nCoV exposure not only among returning travelers, but also among persons in close contact with 2019-nCoV patients in the United States.\n",
            "\t  0.4978631 \t Background: The pathways and mechanism by which associations between the gut microbiome and the brain, termed the microbiome-gut-brain axis (MGBA), are manifest but remain to be fully elucidated. This study aims to use bibliometric analysis to estimate the global activity within this rapidly developing field and to identify particular areas of focus that are of current relevance to the MGBA during the last decade (2009)(2010)(2011)(2012)(2013)(2014)(2015)(2016)(2017)(2018). Methods: The current study uses the Scopus for data collection. We used the key terms \"microbiome-gut-brain axis\" and its synonyms because we are concerned with MGBA per se as a new concept in research rather than related topics. A VOSviewer version 1.6.11 was used to visualize collaboration pattern between countries and authors, and evolving research topics by analysis of the term co-occurrence in the title and abstract of publications. Results: Between 2009 and 2018, there were 51,504 published documents related to the microbiome, including 1713 articles related to the MGBA: 829 (48.4%) original articles, 658(38.4%) reviews, and 226 (13.2%) other articles such as notes, editorials or letters. The USA took the first place with 385 appearances, followed by Ireland (n = 161), China (n = 155), and Canada (n = 144).The overall citation h-index was 106, and the countries with the highest h-index values were the USA (69), Ireland (58), and Canada (43). The cluster analysis demonstrated that the dominant fields of the MGBA include four clusters with four research directions: \"modeling MGBA in animal systems\", \"interplay between the gut microbiota and the immune system\", \"irritable bowel syndrome related to gut microbiota\", and \"neurodegenerative diseases related to gut microbiota\". Conclusions: This study demonstrates that the research on the MGBA has been becoming progressively more extensive at global level over the past 10 years. Overall, our study found that a large amount of work on MGBA focused on immunomodulation, irritable bowel syndrome, and neurodevelopmental disorders. Despite considerable progress illustrating the communication between the gut microbiome and the brain over the past 10 years, many issues remain about their relevance for therapeutic intervention of many diseases.The interaction between gut and brain has been acknowledged by physicians since antiquity . As far back as the sixteenth century, the association between depression and altered bowel function was recognized and in 1978 Manning and his colleagues described the \"irritable bowel syndrome (IBS)\" as a gastrointestinal condition which is strongly associated with psychological stress, some authors reporting 50% of sufferers have comorbid depression or anxiety . The pathways and mechanism by which these associations are manifest remain to be fully elucidated. However, recent developments in genome sequencing, metabolomics, functional imaging and computational biology have increased our understanding considerably .The rapid development of 16S ribosomal RNA and whole genome sequencing analysis has enabled us to understand the diverse nature of the microbial symbionts that inhabit our gastrointestinal tract . Metabolomics is beginning to explain how those microbes produce a range of molecules that impact our behaviors and perceptions. The changes in our microbial diversity, manifest as changes in their metabolic output appear to alter the development of multiple facets of the enteric and central nervous systems including astrocytes, microglial cells and neurons [10, 11] . Functional imaging, functional magnetic resonance imaging and magneto encephalography, have enabled us to identify real time changes in neurological activity and correlate these with changes in behavior or perception . Advances in computational biology are beginning to explain how these multifaceted and complex systems interact with each other [15, 16] .The microbiota interacts with the host through their effect on immune, neuro-hormonal and neural pathways. They have been shown to impact a broad range of disease, including neurodegenerative disorders, such as multiple sclerosis and Parkinson's disease, auto-immune disease and obesity [17, 18] . The gastrointestinal microbiome has also been shown to influence behavior in mammals and man [19, 20] . Transfer of feces from depressed humans to microbiota depleted rats led the recipient rats to display behaviors analogous to depression in the human (anhedonia and anxiety like behaviors) [21, 22] . A strain of bifidobacteria has been demonstrated to increase resilience in people with anxiety . These findings were not observed when healthy people consumed a strain of Lactobacillus . Short chain fatty acids, propionate, butyrate and acetate, are important products of the microbiome and changes in the proportion and quantities of these products alter insulin resistance, ghrelin production and presumably appetite and risk of obesity and diabetes [25, 26] .Bibliometric analyses have been used in various fields to highlight the most influential countries, authors, journals, publications, and institutions . These include research related to microbiota [43, 44] . Worldwide, there are more than 330 clinical studies recorded on clinical trials.gov with a specific focus on the microbiome. This is a growing area of importance in order to better understand the impact of specific strains on individuals, and the interaction with pre-existing microbial symbionts. Currently, there is a lack of research concerning assessment of the current status, hot spots, and future outlook on the theme of the microbiomegut-brain axis (MGBA). This study aims to use bibliometric methods to identify particular areas of research activity in this field and to allow researchers to identify new areas for future development.Although a large number of databases are used for evaluation research at global level , the current study uses the Scopus database which is widely accepted among researchers for the purposes of high quality bibliometric analyses [44, . Scopus is the world's largest abstract and citation database of peer-reviewed research literature, and is an established resource for identifying biomedical research including MEDLINE documents, and includes a higher level of detail than PubMed including the country of origin and citations per document [47, 54] .We used the key terms \"microbiome-gut-brain axis\" and its synonyms because we are concerned with microbiomegut-brain axis per se as a new concept in research rather than related topics. Data mining was conducted on July 12, 2019. The central theme in this study was research articles containing \"microbiome or microbiota and braingut or gut-brain\" to identify items based on their search in the fields title, abstract and keyword simultaneously and the time was 10 years between 2009 and 2018.VOSviewer software (, Van Eck & Waltman version 1.6.11) was used to create a visual representation of collaborations between countries and authors using network maps . Creating a term co-occurrence map in VOSviewer involved only terms that occurred in the title and abstract at least 50 times under binary counting . Terms with the highest relevance score were used to create a term map for network visualization. The algorithm was designed to ensure that terms that co-occurred more frequently had larger bubbles and terms that have a high similarity are located close to each other .Statistical analysis was carried out for the retrieved data by the Statistical Package for the Social Sciences (version 16.0, SPSS Inc., Chicago, IL, USA). Pearson correlation Coefficient was used to test the correlation between some variables (e.g. h-index and number of publications for each country, number of publications and years, and the number of publications related to MGBA and the number of publications related to microbiome in all fields). The analyses carried out in the current study focused largely on the frequencies and percentages of publications for types of documents, countries, journals, and institutes.Between 2009 and 2018, there were 51,504 published documents related to the microbiome, including 1713 articles related to the MGBA: 829 (48.4%) original articles, 658(38.4%) reviews, and 226 (13.2%) other articles such as notes, editorials or letters. English was the most frequently used language (n = 1648), followed by French (n = 16), and Chinese (n = 19), with these accounting for 98.2% of publications related to MGBA. Publications related to MGBA and the microbiome are represented in Fig. 1a and b, respectively. Time trend analyses show rising numbers of publications related to MGBA between 2009 and 2018 (r = 0.950; P value< 0.001), and a correlation between overall numbers of microbiome and MGBA publications (r = 0.991, p < 0.001) during the study period.The term analyses maps are presented in Fig. 2 : the larger circles representing frequently occurring abstract and title terms. Colors used to differentiate between 4 main topic clusters: 1. \"modeling MGBA in animal systems (red cluster)\", 2. \"interplay between the gut microbiota and the immune system (green cluster)\", 3. \"irritable bowel syndrome related to gut microbiota (blue cluster)\", and 4. \"neurodegenerative diseases related to gut microbiota (yellow cluster)\". Table 1 presents the 10 most prolific countries related to MGBA publications, with the top 4 being the USA (n = 385), Ireland (n = 161), China (n = 155), and Canada (n = 144). The overall citation h-index was 106, and the countries with the highest h-index values were the USA (69), Ireland (58) , and Canada (43) . There is a positive modest correlation between h-index and number of published articles (r = 0.817, P-value = 0.004). Figure 3 shows the network visualization map for country collaborations, showing 35 out of a total 86 countries that had more than ten publications; the size of frame represents the number of publications, the thickness of lines signifies the extent of collaboration between the countries.Co-authorship in the field of MGBA is shown in Fig. 4 , with 5 clusters identified; the size of frame represents the number of publications by an author, and the thickness of lines signifies the extent of collaboration between authors. Of the 6054 authors, 25 Research topics clustered by mapping of co-occurrences of terms in title/abstract for publications related to microbiome-gut-brain axis (MGBA). Of the 30,250 terms, 179 terms occurred at least 50 times. For each of the 179 terms, a relevance score was calculated and used to select the 60% most relevant terms. In Fig. 2 , the size of the circles represents the occurrences of terms in title/abstract. The largest set of connected terms consists of 107 terms in four clusters. The four clusters can be broadly interpreted as \"modeling MGBA in animal systems (red cluster)\", \"interplay between the gut microbiota and the immune system (green cluster)\", \"irritable bowel syndrome related to gut microbiota (blue cluster)\", and \"neurodegenerative diseases related to gut microbiota (yellow cluster)\" had at least ten publications including the most active author Cryan, J.F. with 120 (7.0%) publications. The 10 most influential journals covering the MGBA research with their IFs are shown in Table 2 . The three most influential journals from the top 10 influential journals are Brain Behavior and Immunity (49 articles), Plos One (34 articles), and Scientific Reports (33 articles). Table 3 (Table 4 ).This is the first application of bibliometric quantitatively and qualitatively methods regarding the MGBA involving 1713 papers retrieved from Scopus. The results of this bibliometric analysis present a comprehensive overview of the development of the scientific literature in the MGBA field over the past 10 years.The number of articles concerning MGBA research increased rapidly between 2009 and 2018. This increase is likely related to the many experts in psychiatry, neurology and gastroenterology fields (e.g. Cryan J.F., Dinan T.G., Clarke G., Bienenstock J., Forsythe P., Stanton C., Quigley E.M.M., Bercik P., O'Mahony S.M., Shanahan F., Foster J.A., Moloney R.D., and others) developing their interest in the physiological role of the guts' microbiota on brain and behavior as an emerging platform for therapeutic intervention of many diseases. Furthermore, the increased number of publications may relate to several hot topics [56-68, 70-72, 74-77] which were published during this period, revealing novel findings that open the door for new areas of investigation. These studies propose novel concepts for treating several conditions such as IBS, autism, depression, multiple sclerosis, auto-immune disease, Parkinson's disease, and obesity .Since 2012, there has been growing research output in the field of MGBA, which is consistent with increasing research activity related to the microbiome in general. Similar findings have been reported in other bibliometric studies [43, 44, . A possible underlying explanation for the rising publication numbers is that in 2013 the National Institutes of Health (NIH) launched the second phase of Integrative Human Microbiome Project (iHMP) . Our study clearly reveals that the United States is at the forefront of studies on MGBA. The research output from the USA may be associated with the wide range of researchers with an interest within this field and a substantial amount of financial support to researchers. In 2013 the USA launched a special research project on gut microbiotabrain axis . Since then, there has been increasing neuroscience interest in the role of gut microbiota on animal and human brain behavior and cognitive development [92, 93] . Ireland featured as the second most prolific nation and this might be related to Professor John F Cryan and Professor , and has conducted studies in collaboration with several companies including GlaxoSmithKline, Cremo, Suntory, Pfizer, Wyeth and Mead Johnson which consequently provided more funding for conducting research in the field of psychobiotics , thus may contribute to increasing number of publications regarding gut microbiota-brain axis. The number of citations for the top 20 articles in the current study varied from 1490 to 347, which is higher range of citations than in other medical fields such as Table 3 The 20 most influential articles in the microbiome-gut-brain axis research SCR a Authors TitleYear of publication Source title Cited by 1st Nicholson et al. \"Host-gut microbiota metabolic interactions\" 2012 Science 14902nd Cryan and Dinan \"Mind-altering microorganisms: The impact of the gut microbiota on brain and behavior\"2012 Nature Reviews Neuroscience 12043rd Heijtz et al. \"Normal gut microbiota modulates brain development and behavior\" \"Gut-brain axis: How the microbiome influences anxiety and depression\"2013 Trends in Neurosciences 612 7th Bercik et al. \"The intestinal microbiota affect central levels of brain-derived neurotropic factor and behavior in mice\"Gastroenterology 6028th Collins et al. \"The interplay between the intestinal microbiota and the brain\" 2012 Nature Reviews Microbiology 566 8th Berer et al. \"Commensal microbiota and myelin autoantigen cooperate to trigger autoimmune demyelination\" 2011 Nature 56610th De Vadder et al. \"Microbiota-generated metabolites promote metabolic benefits via gut-brain neural circuits\"Cell 52511th Neufeld et al. \"Reduced anxiety-like behavior and central neurochemical change in germ-free mice\" \"The microbiome-gut-brain axis during early life regulates the hippocampal serotonergic system in a sex-dependent manner\" 2013 Molecular Psychiatry 47614th Sampson et al. \"Gut microbiota regulate motor deficits and neuroinflammation in a model of parkinson's disease\"Cell 45515th Tillisch et al. \"Consumption of fermented milk product with probiotic modulates brain activity\"Gastroenterology 44516th Rhee et al. \"Principles and clinical implications of the brain-gut-enteric microbiota axis\"2009 Nature Reviews Gastroenterology and Hepatology 444 17th Braniste et al. \"The gut microbiota influences blood-brain barrier permeability in mice\" \"The microbiome-gut-brain axis: From bowel to behavior\" 2011 Neurogastroenterology and Motility 347 SCR Standard competition ranking a Equal citations have the same ranking number, and then a gap is left in the ranking numbers mobile-health , toxicology , social media in psychology , parasitic diseases [51, 97] , and viral diseases . Additionally, it also reveals that researchers paid great attention on the MGBA mostly in recent years, and published several outstanding articles on top-ranking journals in the medical field such as Science and Nature . The most cited article is \"Host-gut microbiota metabolic interactions\" a review by Nicholson et al., 2012 , published in Science, where the authors suggest that the manipulation of the gut microbiota to optimize new therapeutic strategies could control many diseases and improve health. The second most cited article \"Mind-altering microorganisms: The impact of the gut microbiota on brain and behavior\" was published in the Nature Reviews Neuroscience in 2012 by Cryan and Dinan , where the authors suggest that the concept of a microbiota-gut-brain axis may lead to the development of novel therapeutics for management of several neurological and psychiatric disorders. Finally, there are some limitations for our study findings. First, the search was limited to publications listed in Scopus, which is the largest biomedical database and the most frequently used database for bibliometric analyses, although it might not contain all publications relevant to MGBA research. MGBA publications that do not include this term or its synonyms in the title, abstract or key words might not be taken into account for our analysis. Secondly, a general limitation of the bibliometric approach is that there is no weighting to take account of the quality or scientific rigor of any individual publication. Despite these limitations, we still consider that the findings offer a valid representation of MGBA research output at a global level.The characteristics of the MGBA related publications from 2009 to 2018 are investigated through the bibliometrics analysis based on the Scopus database. This study demonstrates that the research on the MGBA has been becoming progressively more extensive at global level over the past 10 years. Overall, our study found a large amount of work on MGBA, focused on immunomodulation, irritable bowel syndrome, and neurodevelopmental disorders. Despite considerable progress illustrating the communication between the gut microbiome and the brain over the past 10 years, many issues remain to fully realize their relevance for therapeutic intervention of many diseases.Abbreviations IBS: Irritable bowel syndrome; IFs: Impact factors; JCR: Journal citation reports; SCR: Standard competition ranking; SPSS: Statistical package for social sciencesNo funding was received for writing this study.Availability of data and materials Not applicable.Ethics approval and consent to participate Not applicable.Not applicable.The authors declare that they have no competing interests. \n",
            "\n",
            " Title:  nan\n",
            "2 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.59015375 \t OPEN ACCESS Citation: Selinger M, Tykalová H, Štěrba J, Věchtová P, Vavrušková Z, Lieskovská J, et al. (2019) Tick-borne encephalitis virus inhibits rRNA synthesis and host protein production in human cells of neural origin. PLoS Negl Trop Dis 13(9): e0007745.  encephalitis virus (TBEV), a member of the genus Flavivirus (Flaviviridae), is a causative agent of a severe neuroinfection. Recently, several flaviviruses have been shown to interact with host protein synthesis. In order to determine whether TBEV interacts with this host process in its natural target cells, we analysed de novo protein synthesis in a human cell line derived from cerebellar medulloblastoma (DAOY HTB-186). We observed a significant decrease in the rate of host protein synthesis, including the housekeeping genes HPRT1 and GAPDH and the known interferon-stimulated gene viperin. In addition, TBEV infection resulted in a specific decrease of RNA polymerase I (POLR1) transcripts, 18S and 28S rRNAs and their precursor, 45-47S pre-rRNA, but had no effect on the POLR3 transcribed 5S rRNA levels. To our knowledge, this is the first report of flavivirus-induced decrease of specifically POLR1 rRNA transcripts accompanied by host translational shutoff.Tick-borne encephalitis virus (TBEV) is a causative agent of a severe human neuroinfection that threatens Europe and Asia. Little is known about the interaction of this neurotropic virus with neural cells, even though this may be important to better understand why or how TBEV can cause high pathogenicity in humans, especially following neural cell infection. Here, we showed that TBEV induced host translational shut-off in cells of neural origin. In addition, TBEV interfered also with the expression of host ribosomal RNAs. Interestingly, the transcriptional shut-off was documented for rRNA species transcribed by RNA polymerase I (18S rRNA, 28S rRNA and their precursor 45 The Flaviviridae family contains arthropod-borne viruses including medically important pathogens with worldwide distribution and impact, such as dengue virus (DENV), yellow fever virus (YFV), West Nile virus (WNV), Japanese encephalitis virus (JEV), Zika virus (ZIKV), and tick-borne encephalitis virus (TBEV) . TBEV causes a severe neuroinfection known as tick-borne encephalitis, which affects thousands of people across Eurasia annually [2, 3] . In recent years, an increase in TBEV infection rates in affected countries and in its geographical distribution has been observed, involving previously unaffected areas such as Switzerland and northern Germany . Although the disease is not always fatal (mortality rate of 1-2%), a high percentage of patients (35-58%) suffer from permanent sequelae, such as cognitive or neuropsychiatric afflictions, balance disorders, headaches, dysphasia, hearing defects, and spinal paralysis after overcoming the main symptoms [2, 7] . Specific antiviral therapy for tick-borne encephalitis does not exist. Neurons are the primary target for TBEV infection in mice and humans, and according to post mortem studies of TBEV-infected patients, the cerebellum is one of the main foci affected . Understanding the interaction between TBEV and human neural cells is essential as it could lead to possible new treatment targets and a better understanding of why TBEV infection can result in severe neurological symptoms. Like all flaviviruses, TBEV is an enveloped virus with a single-stranded RNA (ssRNA) genome of positive polarity (approx. 11 kb) with a 7-methylguanosine cap at the 5´end. The coding segment is flanked on both ends by untranslated regions (UTR). Viral proteins are encoded in a single open reading frame that is translated in one poly-protein which is then proteolytically processed into three structural (C, prM, E) and seven non-structural proteins (NS1, NS2A, NS2B, NS3, NS4A, NS4B, NS5) . While the structural proteins are the main building units of the viral particle, the non-structural proteins are crucial in the TBEV life cycle. They are essential components of viral replication within the host endoplasmic reticulum or Golgi apparatus-derived membrane compartments and the virion assembly processes and are involved in immune response evasion/counteractions .Virus replication is reliant on the host protein synthesis apparatus and manipulates it in favour of viral requirements. There are various strategies viruses use to accomplish this goal and generally aim at three levels: host translational shut-off, processing of host mRNA, and host transcriptional shut-off [17, 18] . Translation of eukaryotic and viral proteins is often controlled at the rate-limiting step of initiation and viruses such as Bunyamwera virus, influenza A virus or poliovirus were shown to target initiation factors . More specifically for flaviviruses, a recent study documented repression of the host translation initiation step during DENV infection and general translational repression was also recorded for WNV and ZIKV. While inducing host translational shut-off, viral proteins are still synthesised thanks to alternative translation initiation strategies, such as cap-independent translation [20, .Transcription in eukaryotic organisms is carried out by three RNA polymerases: RNA polymerase I, II, and III. Each of the RNA polymerase complexes is responsible for the transcription of different genes. RNA polymerase I (POLR1) yields a single transcription unit 45-47S pre-rRNA, which undergoes a complex maturation process that generates 5.8S, 18S, and 28S rRNA [28, 29] . RNA polymerase III (POLR3) produces 5S rRNA, tRNAs, and specific small RNAs . RNA polymerase II (POLR2) transcribes protein-coding genes and certain small RNAs . Out of all the transcripts synthesised in the eukaryotic cell, ribosomal RNA is the most abundant and a key component of ribosomes. Virus-induced interference with transcription and subsequent processing of host rRNA has been described for influenza A virus , herpes simplex virus type I , human papillomavirus type 8 , human cytomegalovirus , and human immunodeficiency virus (HIV) . However, this was not described for flaviviruses.Given the indications for flaviviruses affecting host translation , we aimed at exploring this topic further in TBEV infection of naturally permissive host cells of neural origin, that represent a key cell type responsible for tick-borne encephalitis manifestation. We found that TBEV triggered host translational shut-off that involved lowered expression of GAPDH and HPRT1 housekeeping genes as well as the interferon-induced protein viperin. TBEV further specifically impaired the production of POLR1-transcribed rRNAs. Therefore, we postulate that TBEV specifically targets POLR1-mediated transcription of rRNA and rate of host translation thus promoting virus replication.The human medulloblastoma (DAOY HTB-186; ATCC), human lung adenocarcinoma (A549; a gift from R. Randall, University of St. Andrews, UK), and Vero (green monkey kidney; Biology Centre, CAS, CZ) cell lines were grown in low glucose DMEM medium supplemented with 10% foetal bovine serum (FBS), 1% antibiotics-antimycotics (amphotericin B 0.25 μg/ml, penicillin G 100 units/ml, streptomycin 100 μg/ml), and 1% L-alanyl-L-glutamine. DAOY HTB-186 cell line is derived from desmoplastic cerebellar medulloblastoma of a 4-year-old Caucasian male . A549s are derived from a lung cancerous tissue (alveolar basal epithelial cells) of a 58-year-old Caucasian male . Vero cells are derived from kidney epithelial cells from African green monkey (Cercopithecus aethiops). PS cells (porcine kidney stable) were grown in L15 medium with 3% new-born calf serum (NCS), 1% antibiotics-antimycotics, and 1% L-alanyl-L-glutamine . The human osteosarcoma cell line MG-63 (Sigma-Aldrich) was grown in RPMI 1640 medium supplemented with 10% FBS, 1% antibiotics-antimycotics, 1% L-alanyl-L-glutamine, and 50 nM β-mercaptoethanol. These were explanted from a 14-yearold Caucasian male .For metabolic labelling experiments, all cell lines were grown in RPMI 1640 medium supplemented with 10% FBS, 1% antibiotics-antimycotics, 1% L-alanyl-L-glutamine, and 50 nM β-mercaptoethanol. All cell lines were grown at 37˚C and 5% CO 2 ; with the exception of PS cells (37˚C without additional CO 2 ).PolyJet In Vitro Transfection Reagent (SignaGen; #SL100688) was used for transfection. The procedure was carried out according to the manufacturer's protocol. For GFP and Renilla luciferase expression, the mammalian expression vectors phMGFP (Promega) and pRL-CMV (Promega) were used, respectively. The wt viperin mammalian expression vector was a kind gift from Lisa F.P. Ng (Singapore Immunology Network, Agency for Science, Technology and Research (A � STAR), Singapore), in which the viperin gene with C-terminal c-myc tag is transcribed under the control of the CMV promoter .Two representatives of the West-European TBEV subtype with different degrees of virulence were used-medium (Neudoerfl) and severe (Hypr). Both strains differ in their coding sequences by only 12 nonconservative amino acid substitutions , and in the length and structure of the 3´UTR . When mice were infected peripherally, the Hypr strain exhibited pronounced neuroinvasiveness and caused shorter survival than strain Neudoerfl . The low passage TBEV strain, Neudoerfl (fourth passage in suckling mice brains; GenBank accession no. TEU27495), was provided by Prof. F.X. Heinz (Medical University of Vienna, Austria) . The low passage TBEV strain, Hypr (fourth passage in suckling mice brains; GenBank accession no. TEU39292), is available at the Institute of Parasitology, Biology Centre of CAS, České Budějovice, Czech Republic . Viruses were handled under biosafety level 3 conditions.TBEV was added to the cells one day post seeding. Cells were then incubated for 2 hours, washed with PBS, and finally fresh pre-warmed medium was added. Brain suspension from uninfected suckling mice was used as a negative control.Viral titres were determined by plaque assay as described , with minor modifications. Briefly, PS cell monolayers (9x10 4 cells per well) were grown in 24-well plates and incubated with 10x serial dilutions of infectious samples for 4 hours at 37˚C. The samples were then covered by 1:1 (v/v) overlay mixture (carboxymethyl cellulose and 2x L15 medium including 6% NCS, 2% antibiotics-antimycotics, and 2% L-glutamine). After five days, medium with overlay was removed, cells washed with physiological solution, subsequently fixed and stained (0.1% naphthalene black in 6% acetic acid solution) for 45 minutes. Virus-produced plaques were counted, and titres are stated as PFU/ml.The following primary antibodies were used: anti-TBEV C polyclonal antibody (produced in- L-azidohomoalanine (Click Chemistry Tools; #1066-25) and 5-ethynyl-uridine (Click Chemistry Tools; #1261-25) were used for metabolic labelling of nascent proteins or RNA, respectively. Biotin-PEG4-Alkyne (Click Chemistry Tools; #TA105-25) and Biotin Picolyl Azide (Click Chemistry Tools; #1167-25) were used for subsequent detection of incorporated L-azidohomoalanine or 5-ethynyl-uridine, respectively. Cycloheximide was purchased from Sigma-Aldrich (#01810-1G).DAOY cells were seeded one day prior to infection in the 12-well plate at a density of 2.5×10 5 cells/well. At the indicated time intervals post-TBEV infection, cells were washed with PBS, trypsinized, and fixed by 4% paraformaldehyde in PBS (Roth) . After permeabilization (0.1% Triton X-100), cells were stained using guinea pig anti-TBEV C antibodies (1:1500 dilution) and anti-guinea pig DyLight 594 (1:500 dilution) secondary antibodies. Flow cytometry was performed on a FACS Canto II cytometer and data analysed using FACS DIVA software v. 5.0 (BD Biosciences).Total cellular RNA was isolated using Trizol-based RNA Blue reagent (Top-Bio; #R013) according to the manufacturer's instructions. RNA pellets were dissolved in DEPC-treated water and directly used for either real-time PCR or analysis on an RNA denaturing gel.The quantity and integrity of rRNA in total RNA samples were analysed on a 2100 Bioanalyzer using Agilent RNA 6000 Nano kit (Agilent Technologies; #5067-1511). The concentration of each sample was determined spectrophotometrically prior the Bioanalyzer measurement and samples were diluted according to the cell number ratio (resulting concentrations were between 10-20 ng/μl). 1 μl of the diluted RNA samples was loaded on the Bioanalyzer chip and the electrophoretic assay was performed according to the manufacturer's instructions. All samples were analysed in technical triplicates. 1.2% agarose MOPS-buffered denaturing gel (with 6.7% formaldehyde) was used for fractionation of isolated total RNA. RNA was visualised by addition of the GelRed dye (Biotium) into the gel. The signal was subsequently quantified using Fiji software.We observed that the viability of TBEV Hypr-infected cells was negatively affected at 36 and 48 hours p.i. (Fig 1D) . Therefore, in order to diminish the effect of this phenomenon on our data, we decided to standardise in our experiments to cell counts.Normalisation to cell numbers was performed for real-time PCR, western blotting, northern blotting, and metabolic labelling analyses. For this, we established a viability-based method using alamarBlue reagent (Thermo Fisher Scientific; #DAL1025). Our data demonstrate that viability measurement is directly proportional to the cell number, and therefore this method is fully suitable for normalisation to the cell number (S1 Fig). The procedure was performed according to the manufacturer's instructions. Briefly, cells were washed with PBS and fresh pre-warmed growth medium with diluted alamarBlue reagent was added (1:10 dilution ratio; v/v). Cells were incubated for 2-2.5 hours and fluorescence of the reduced product was measured on a BioTek plate reader (λ ex = 550 nm; λ em = 590 nm). Growth medium with alamar-Blue without cells was used as a blank. All samples were analysed in technical triplicates. Average fluorescence values for TBEV-treated sample were normalized to the respective mock control cells. The viability factor (f) was subsequently used as a normalisation factor for the calculation of RNA/protein input based on the pre-set mock control input.f ¼ viability sample ½a:u:� viability control ½a:u:For real-time qPCR analyses, the KAPA SYBR FAST Universal One-Step qRT-PCR Kit was used according to the manufacturer's protocol. Data obtained were processed via relative quantification using the delta c t (Δ-c t ) method; the amount of RNA was adjusted to the cell number instead of the c t values of the housekeeping reference gene. All samples were treated with dsDNase and subsequently 5× diluted in RNAse-free water before the real-time PCR analysis. All samples were analysed in technical triplicates. List of primers used can be found in S1 Table. Western blotting Cells were washed with PBS and RIPA buffer (25 mM Tris-HCl pH 7.6, 150 mM NaCl, 1% NP-40, 0.1% SDS, 1% sodium deoxycholate) with protease inhibitors (Thermo Fisher Tick-borne encephalitis virus and host rRNA/protein shut-off Scientific; #78430) was added. Cell lysis was performed for 15 minutes on ice while gently shaking. Sonicated and cleared protein lysates in RIPA buffer were separated on 12% denaturing polyacrylamide gels and blotted onto PVDF membranes. The quantity of proteins was normalised to the cell number. Membranes were blocked (5% skimmed milk in PBS-T) and incubated with primary, secondary, and alternatively also tertiary antibodies; between each staining step, membranes were washed three times in PBS-T. Primary antibodies used were guinea pig anti-C (produced in-house; 1:1500), chicken anti-NS3 (M. Bloom laboratory; 1:5000), rabbit anti-GAPDH (Abcam; 1:1000), anti-HPRT1 (Thermo Fisher Scientific; 1:500), anti-viperin (Hycult Biotech; 1:500). Secondary/tertiary antibodies used were goat anti-rabbit HRP (VectorLabs; 1:1000), rabbit anti-chicken HRP (Thermo Fisher Scientific; 1:1000), and horse anti-mouse HRP (VectorLabs; 1:1000). Chemiluminescent signal was developed using either Novex CDP-Star kit for alkaline phosphatase (Thermo Fisher Scientific) or Wester-nBright Quantum kit for horseradish peroxidase (Advansta; #K-12042-D20). The signal was subsequently quantified using Fiji software . For stripping of antibodies, membranes were incubated with stripping solution (62.5 mM Tris HCl pH 6.8, 2% SDS, 0.8% β-mercaptoethanol) for 45 minutes at 50˚C. Subsequently, membranes were extensively washed six times with PBS. Following this, membranes were blocked, and immunostaining was again performed as described above.For analyses of Renilla luciferase activity in CHX-treated cells, Renilla Luciferase Assay Kit from Promega (#E2810) was used according to the manufacturer's instructions. Briefly, 5×10 4 DAOY cells per well were seeded on a 96-well plate. Cells were transfected with 100 ng of pRL-CMV vector per well using PolyJet transfection reagent and incubated with cycloheximide (50-300 μg/ml) for 2, 4, 6, 14, and 24 hours. At 24 hours post-transfection, the viability of cells was measured using alamarBlue. Subsequently, cells were lysed and Renilla luciferase activity was determined.Cells were seeded in 6-well plates at a density of 1×10 6 (Vero, A549) or 5×10 5 (DAOY, MG-63) cells per well. At indicated time intervals p.i., cells were washed with PBS and starved for 1 hour by addition of complete methionine-free RPMI medium (methionine-free RPMI medium containing 10% FBS, 1% L-alanyl-L-glutamine, 1% antibiotics/antimycotics, and 0.27 mM L-cystine). Subsequently, fresh complete methionine-free RPMI medium was added with 50 μM L-azidohomoalanine (AHA) and 1× AlamarBlue reagent. Metabolic labelling with AHA was performed for 2 hours. Afterwards, cell viability was measured as described earlier. . Between each staining step, membranes were washed three times in PBS-T. Chemiluminescence signal was developed using Novex CDP-Star kit (Invitrogen; #WP20002). Signal was subsequently quantified using Fiji software .DAOY cells were seeded in 6-well plates at a density of 5×10 5 cells per well. At the indicated time intervals p.i., 5-ethynyl uridine (5-EU) was added to the cells (final concentration of 5-EU was 1 mM) as well as alamarBlue reagent. Metabolic labelling with 5-EU was performed for 2 hours. Cell viability was measured as described earlier. Cells were then washed with PBS and lysed using RNA Blue reagent. Total RNA was isolated according to the manufacturer's instructions. Next, RNA was separated in MOPS-buffered denaturing gel, as described above. , and 10 μM picolyl biotin azide) for 1 hour in the dark at room temperature. Blocking and triple labelling using biotin-streptavidin system was performed as described above. The chemiluminescence signal was developed using Novex CDP-Star kit (Invitrogen; #WP20002), and signal was subsequently quantified using Fiji software .DAOY cells were seeded in chamber slides (0,3 cm 2 /well; 5×10 3 cells/well) and at the indicated time intervals p.i. processed as previously described . Rabbit anti-POLR1A (Abcam; 1:200) and chicken anti-NS3 (a kind gift from Dr. M. Bloom, NIAID, NIH; 1:5000) antibodies were used. As the secondary antibodies, anti-rabbit DyLight 594 (Abcam; 1:500) and anti-chicken DyLight 488 (Abcam; 1:500), were used. In the case of metabolic labelling of nascent RNA, the Click reaction was performed in situ before the blocking step. 10 μM Picolyl biotin azide was used for the detection of incorporated 5-EU. For subsequent fluorescent labelling, streptavidin conjugated with DyLight 549 was used (VectorLabs; 1:500). Slides were eventually mounted in Vectashield mounting medium (VectorLabs). The Olympus Fluoview FV10i confocal microscope was used for imaging and subsequent export of images was done in FV10-ASW software (v.1.7).All statistical analyses were performed in MS Excel using one-sample two-tailed Student's ttest. Only in case of qPCR analysis of over-expressed viperin and GFP, an unpaired two-tailed Student's t-test was used. In this case, datasets were first tested for the equality of variances by F-test. If the experiment was performed in technical replicates, the statistics was performed using the means of the independent biological replicates.Recent studies have shown that DENV decreases the rate of de novo protein synthesis in host cells [24, 48] . In order to establish whether TBEV also affects translation, de novo protein synthesis kinetics was measured in TBEV-infected cells using Click chemistry . For this purpose, we utilized a suitable in vitro experimental system of the cerebellum-derived human medulloblastoma cell line DAOY HTB-186 to broaden previous findings . Two closely related members of the European subtype of TBEV with different virulence were used for comparative purposes: a medium virulent prototype strain, Neudoerfl, and a highly virulent strain, Hypr . Initially, we characterized the course of infection for both TBEV strains. DAOY cells were infected at an MOI of 5 with either strain and at 12, 18, 24, 36, 48 hours p.i., replication kinetics, infection rate, viral protein (C, NS3) production and viability of infected cells were determined. Both strains successfully replicated in DAOY cells, with the Hypr strain reaching at least one order of magnitude higher titres during the course of infection until 48 hours p.i., when both strains eventually produced equal titres (Fig 1A) . The infection rate was also considerably higher for the Hypr strain, culminating at 36 hours p.i. (87.5% of infected cells), whereas the Neudoerfl strain infected only 43.6% of cells (Fig 1A) . Relative quantification of genomic RNA at 24 and 48 hours p.i. revealed that Hypr replicated with higher efficiency than Neudoerfl (Fig 1B) . TBEV C and TBEV NS3 protein detection corresponded to replication kinetics and for both strains proteins could be detected earliest at 18 hours p.i., increasing thereafter ( Fig 1C) . While TBEV Neudoerfl affected the viability of the infected cells only mildly (maximal decrease by 16.6% at 36 hours p.i.), TBEV Hypr lowered the viability of the infected cells by 23.8% and 62.5% in comparison to mock-infected control at 36 and 48 hours p.i., respectively ( Fig 1D) . Therefore, in order to compensate the potential bias originating from cell death, we standardised our experiments to viability which is directly proportional to the number of living cells (S1 Fig). In the following experiments we pursued interaction of TBEV with DAOY cells during the period of productive infection for both TBEV strains, ranging from 24 to 48 hours p.i. After this detailed characterization of our in vitro model, de novo protein synthesis and quantification was performed. DAOY cells were infected with either TBEV Hypr or Neudoerfl and metabolic labelling was carried out for 2 hours at 24, 36, and 48 hours p.i. using the methionine analogue L-azidohomoalanine (AHA). At 24 hours p.i., translation levels were comparable in control and infected cells, but infection resulted in a significant decrease of AHAlabelled proteins at 36 and 48 hours p.i. in TBEV Hypr-infected cells and at 48 hours p.i. in TBEV Neudoerfl-infected cells (Fig 2A and 2B; S2A Fig) . Interestingly, the viral NS3 protein levels increased over the course of the infection with both strains (Fig 2A, lower panel) . Furthermore, TBEV-induced host translational shut-off was also documented for cell lines of non-neural origin (A549 cells, Vero cells, and MG-63 cells) at 48 hours p.i., for both TBEV strains (Fig 2C; S2B Fig) . Interestingly, despite the observed host translational shut-off both TBEV strains were able to replicate (Fig 1B) successfully and reached high titres (Fig 1A) in DAOY cells throughout the infection.Since these experiments revealed a significant decrease in host protein synthesis upon TBEV infection on a global level, we evaluated the specificity of this for particular host proteins. First, the effect of TBEV infection on common housekeeping genes GAPDH and HPRT1 was determined by analysing their mRNA and protein levels. Relative quantification of GAPDH and HPRT1 mRNAs revealed a strong inhibition of expression for both genes and TBEV strains at 48 hours p.i. (Fig 3A and 3B; upper panel) . Similar results were observed for their protein levels, although the more virulent strain Hypr elicited a stronger reduction ( Fig 3A and 3B; lower panel) .As the subversion of host translation process can be used as an immune evasion strategy by viruses , we investigated the effect of translational shut-off on the interferon-inducible gene viperin. Viperin has been described so far as an antiviral protein that interferes with TBEV on multiple levels . A time course of viperin mRNA production in response to TBEV infection in DAOY cells was determined. Induction of viperin mRNA expression was detected at 24 hours p.i. and increasing throughout next 24 hours (Fig 3C; upper panel) . Despite significantly increased viperin mRNA levels, none or very small amounts of viperin Tick-borne encephalitis virus and host rRNA/protein shut-off protein were detected in cell lysates from TBEV-infected DAOY cells by western blot analysis (Fig 3C; lower panel) . As a positive control, DAOY cells treated with INF-β (12 hours; 50 ng/ ml) as well as DAOY cells transfected with a human viperin expression vector were used.To assess whether the effect of TBEV on endogenous viperin production can be overcome by artificial over-expression, DAOY cells were first infected (TBEV Neudoerfl and Hypr; MOI 5) and subsequently transfected with a wt-viperin expression construct at 12 hours p.i. Viperin mRNA, as well as protein levels, were analysed at 12 hours post-transfection (S3A Fig). As a control, GFP expression construct was used. S3B Fig shows that viperin protein was produced; however, the protein levels were significantly reduced in TBEV-infected cells compared to control cells. Hypr strain infection also resulted in a statistically significant decrease in mRNA levels of viperin. As expected, GFP production in TBEV infected cells was negatively affected in case of both TBEV strains (S3C Fig). Again, Hypr strain infection also caused a significant Tick-borne encephalitis virus and host rRNA/protein shut-off decrease in GFP mRNA. Consequently, TBEV induces a general translational shut-off, which can negatively affect even the production of overexpressed transcripts. Nevertheless, viral titres were increasing throughout the infection (Fig 1) .Previous data revealed a significant decrease in RNA encoding genes including 5.8S rRNA and 7SL RNA following TBEV infection . Here, we verified the link between the TBEVinduced translational shut-off and production of host rRNAs. We quantified the levels of 18S and 28S rRNAs in total cellular RNA from TBEV-infected DAOY cells at 24 and 48 hours p.i. We found that infection by both TBEV strains significantly decreased the 18S and 28S rRNA (S4 Fig). 18S rRNA levels decreased to 50 ± 6% or 33 ± 1% for TBEV Neudoerfl-or Hyprinfected cells compared to controls, respectively (Fig 4A) . For 28S rRNA, its transcription levels fell to 49 ± 5% or 28 ± 2% for TBEV Neudoerfl-or Hypr-infected cells, respectively ( Fig 4B) . Both 18S and 28S rRNAs are transcripts of POLR1. Interestingly, the POLR3 transcript 5S rRNA levels remained unaffected by TBEV infection (Fig 4C) . These data imply that the effect of TBEV infection on host cells also involves the transcription of specific ribosomal RNA genes.In order to elucidate at which step TBEV interferes with rRNA production, we first analysed the integrity of mature rRNA molecules. No degradation products were observed following infection with either TBEV strains at 24 or 48 hours p.i. in DAOY cells (Fig 5A and 5B ). Next, we investigated the rRNA expression and processing via quantification of de novo synthesised RNA in TBEV-infected DAOY cells. We labelled nascent RNA in TBEV-infected DAOY cells at 24, 36 and 48 hours p.i. with 5-ethynyl uridine (EU). Incorporated EU was visualised using Click chemistry and the biotin-streptavidin detection system. The presence of TBEV Hypr strain resulted in a decreased quantity of 45-47S pre-rRNA transcripts at 36 and 48 hours p.i., whereas infection with TBEV Neudoerfl strain reduced de novo synthesis of 45-47S pre-rRNA at 48 hours p.i. (Fig 5C) . Tick-borne encephalitis virus and host rRNA/protein shut-off . There are several hallmarks typical for nucleolar stress including disruption of nucleolus structure . We, therefore, characterised the localization and production of nascent RNA at the cellular level and also investigated the structure of the nucleolus. DAOY cells infected with TBEV Hypr strain were analysed at 24, 36, and 48 hours p.i. using in situ Click reaction with 10 μM picolyl biotin azide and subsequent visualisation via streptavidin conjugated with DyLight-549. As shown in Fig 5D, the overall production of nascent RNA in TBEV-infected cells started to decrease from 36 hours p.i.; de novo synthesised RNA was exclusively detected in nuclei with foci of nascent RNA molecules localised in nucleoli. In addition, these nascent RNA foci were not structurally altered upon TBEV infection. The specificity of the labelling reaction was determined using EU-unlabelled cells in the Click reaction (S5A Fig). In order to further verify that TBEV did not induce nucleolar re-arrangement due to nucleolar stress, we analysed the nucleolar structure upon TBEV Hypr infection using nucleophosmin (NPM1; a nucleolar marker). As a positive control, cells were treated with 1 mM H 2 O 2 for 45 minutes. No disruption of nucleoli in TBEV-infected cells was observed (S5B Fig) . These data imply that TBEV inhibits 45-47S pre-rRNA production without triggering the nucleolar stress pathway.Based on the observed TBEV interference with rRNA production on the transcriptional level, we sought to investigate if the levels and cellular localization of POLR1 changes in infected cells. As shown in Fig 6A and 6B , POLR1 was localised exclusively to the nuclei, and no translocation occurred in infected cells at any time interval tested. Nevertheless, POLR1 protein levels were impaired in TBEV Hypr-infected cells at 48 hours p.i. This may be a result of the previously mentioned translational shut-off since it coincided at 48 hours p.i. Besides, POLR1 mRNA levels were negatively affected by TBEV infection, too (Fig 6C) . In particular, POLR1A (the largest subunit of the RNA polymerase I complex) mRNA levels dropped to 60 ± 5% or 25 ± 1% in TBEV Neudoerfl-or Hypr-infected DAOY cells at 48 hours p.i., respectively.TBEV-induced translational shut-off and the decrease in production of nascent 45-47S pre-rRNA raised the question whether these processes are casually interconnected. We analysed the rate of rRNA production in DAOY cells after treatment with cycloheximide (CHX), an inhibitor of translation elongation. First, we determined the time-and dosage-dependent effect of CHX in DAOY cells using a Renilla (RL) luciferase-based reporter system. DAOY cells were first transfected with pRL-CMV and treated with CHX (50, 100, and 300 μg/ml). As shown in S6 Fig, all CHX concentrations tested decreased the production of luciferase. Moreover, the inhibition rate of luciferase production increased with longer exposure to CHX. Next, rRNA production in DAOY cells with decreased translational rate was assessed. Cells were treated with CHX (100 μg/ml) for 6 or 14 hours and de novo RNA synthesis in CHXtreated cells was subsequently determined. Fig 7A shows a statistically significant decrease in levels of nascent 45-47S pre-rRNA for both intervals. In particular, the levels decreased to 22 ± 9% or 56 ± 16% during CHX treatment for 14 or 6 hours, respectively. In addition, total levels of mature 18S and 28S rRNAs were quantified in CHX-treated cells. Significant infected cells was calculated using one-sample Student's t-test ( � P<0.05). (D) In situ metabolic labelling revealed TBEV-induced reduction of nascent RNA at 36 hours p.i. without change in RNA localization. DAOY cells were infected with TBEV Hypr strain (MOI 5) and at indicated time intervals incubated for 2 hours with 1 mM 5-ethynyl uridine (5-EU) in order to label nascent RNA. Detection of incorporated 5-EU was performed by Click reaction using 10 μM biotin picolyl azide followed by fluorescent labelling with streptavidin-DyLight549. Cells were co-stained with anti-NS3 antibodies; signal was further visualised using anti-chicken DyLight488 antibodies. Nuclei were stained with DAPI. Scale bar represents 200 μm. encephalitis virus and host rRNA/protein shut-off Tick-borne encephalitis virus and host rRNA/protein shut-off decreases in 18S rRNA levels were observed after a 14-hour incubation (65 ± 9%; Fig 7B) . 28S rRNA levels were reduced to 81 ± 4% compared to control cells; however, this effect was not statistically significant (Fig 7B) . Quantification of 5S rRNA, a POLR3 transcript, revealed a statistically significant decrease even for this rRNA species after 14 hours of CHX treatment (46 ± 11%; Fig 7C) . These data demonstrated that during translation inhibition induced by CHX, the quantity of rRNA transcripts of both RNA polymerases (POLR1 and POLR3) were with CHX (100 μg/ml) for either 6 or 14 hours; for metabolic RNA labelling, 5-EU was added 2 hours before the sample collection. Cell viability was measured before the cell lysis. Isolated total RNA was transferred to a PVDF membrane and nascent RNA quantified using Click chemistry with 10 μM biotin picolyl azide before subsequent chemiluminescent detection. Data are representative of three independent experiments and values in graphs are expressed as mean with SEM, normalised to cell numbers and mock infected cells. Significant difference from the control was calculated using one-sample Student's t-test ( � P<0.05; �� P<0.01). (B) Levels of 28S and 18S rRNA were analysed by in-gel RNA staining with GelRed before blotting. Data are representative of three independent experiments and values in graphs are expressed as mean with SEM, normalised to cell number and mock infected samples. Significant difference from the control was calculated using one-sample Student's t-test ( � P<0.05). Tick-borne encephalitis virus and host rRNA/protein shut-off decreased. In comparison to the general rRNA synthesis shut-down resulting from the action of CHX, TBEV infection induced only a decrease in POLR1 rRNA transcripts (Fig 7D) . This suggests that TBEV infection specifically targeted POLR1, which may subsequently result in translational shut-off.TBEV infection is spreading through Europe, resulting in increased numbers of TBEV cases and emergence in previously unaffected areas. TBEV is known to be able to cause neurological symptoms in some infected patients, though little is known about its interplay with neural cells. The molecular basis of damage to the CNS following TBEV infection is still not fully understood. So far, it seems that it is a complex phenomenon combining multiple factors including host immune system . Therefore, understanding the TBEV interaction with target cells and detailed description of processes of viral or host response can help to reveal new targets and ideas on how to treat this disease more successfully. To what extent the outcome of these infection-induced processes is reflected on longer term sequelae remains unrevealed.Metabolic labelling experiments demonstrated that TBEV infection interferes with the global de novo protein synthesis in infected cells. Surprisingly, the effect of translational arrest was so robust that even the protein levels of two commonly used housekeeping genes, GAPDH and HPRT1, were significantly lowered (Fig 3A and 3B ). Cell lines of both neural and nonneural origin underwent translational shut-off, demonstrating thus the general nature of this phenomenon upon TBEV infection. However, the rate of reduction varied substantially in individual cell lines suggesting cell-dependent effects. TBEV Hypr strain caused a greater translational shut-off in all cell lines compared to the Neudoerfl strain. This may be due to the increased virulence and neuroinvasiveness of the Hypr strain or due to susceptibility and tropism of the virus strains to specific cell types. Recent studies have demonstrated that some flaviviruses can cause translation suppression via diverse mechanisms [24, 48] . These findings together with our results revise the idea of flaviviruses as \"non-host cell protein synthesis influencers\" [25, 54, 55] . Indeed, flaviviruses have been thought to avoid the host-cell protein synthesis shut-off as they replicate at a slower rate and global protein synthesis manipulation might have potentially deleterious effects on cell viability and virus yields [56, 57] . However, reduced synthesis of host proteins had no adverse effect on the production of viral NS3 and C proteins (Fig 1C) , viral gRNA ( Fig 1B) or production of viral progeny (Fig 1A) . This suggests that protein synthesis shut-off does not stop TBEV from successful replication.Viperin is a known interferon-stimulated gene (ISG) and has been described as a potent antiviral protein against members of the Flaviviridae family, especially TBEV [50, . Thereby it is anticipated to see an increase in viperin mRNA levels upon TBEV infection in DAOY cells. However, the absence of endogenous viperin protein in TBEV-infected cells is surprising. Thus, translational shut-off may yield multiple advantages to TBEV. Apart from gearing the host protein synthesis apparatus to the purposes of the virus, it may also perform as an immune evasion strategy by preventing ISG production. A widely used stable overexpression approach in an ISG/viperin study might therefore mask the real interactions among flaviviruses and host cells during the infection. In general, our data highlight the importance of careful experimental design when studying virus-host interactions and ISG function specifically.To our knowledge virus-driven reduction in host rRNA levels has not been described before for any flavivirus. Only scarce information is available regarding the virus-induced reduction of rRNA expression, production, and maturation. For example, murine hepatitis virus directly reduces the levels of mature 28S rRNA ; Autographa californica multiple nucleopolyhedrovirus was shown to decrease both, 18S and 28S rRNAs . Additionally, over-expression of HIV Tat protein in Drosophila melanogaster led to the impairment of 45S pre-rRNA precursor processing . Similarly, herpes simplex virus 1 decreased the rate of rRNA maturation despite unaltered levels of 45-47S pre-rRNA and unchanged POLR1 activity . The reduction of rRNA levels can be associated with the induction of nucleolar stress, which is characterized by several hallmarks including nucleolar and ribosomal disruption eventually leading to the activation of the p53 signalling pathway. A possible link between flaviviral pathogenesis and nucleolar stress was suggested previously. DENV and ZIKV, but not WNV were shown to induce nucleolar stress in infected cells by disruption of nucleoli, which TBEV may interfere directly with host translational processes, leading to decreased host protein levels. This decrease could negatively affect pre-rRNA synthesis and eventually rRNA levels. On the other hand, TBEV may also interfere directly with the synthesis of pre-rRNA first, which results in decreased levels of mature rRNAs. Insufficiency of rRNAs subsequently leads to the impairment of ribosome biogenesis and decrease of the translational rate in infected DAOY cells. TBEV infection could also trigger host defence mechanisms leading to the translational arrest. For example, protein kinase R (PKR) activated by dsRNA or PKR-like endoplasmic reticulum kinase (PERK) activated by ER stress could play a significant role in the observed translational shut-off as well. encephalitis virus and host rRNA/protein shut-off resulted in an increased rate of apoptosis via the p53 signalling cascade . However, no disruption of nucleoli was observed in the case of TBEV-infected DAOY cells (S5B Fig), possibly not surprising as the TBEV infection specifically affects only the POLR1 activity.We propose alternative ways by which TBEV could interfere with transcription and/or translation in DAOY cells: 1) TBEV negatively affects the translation of host proteins, including POLR1, transcription factors, and ribosomal proteins; their lower levels subsequently result in a decline in synthesis of all rRNA species; or 2) TBEV directly interferes with de novo synthesis of 45-47S pre-rRNA (but not 5S rRNA) via a POLR1 specific mechanism, which reduces the levels of 18S and 28S rRNAs and this leads to the decline of translational rate in host cells; 3) transcription and translation can be modified independently by both viral or cellular factors as a result of infection (summarised in Fig 8) . Translational shut-off can otherwise be elicited by host cell defence mechanisms, such as activation of protein kinase R (PKR) or PKR-like endoplasmic reticulum kinase (PERK) . To elucidate the exact mechanism of the inhibition of host protein and rRNA production and actual involvement of viral and host factors further experiments will be needed. These may for example assess whether viral proteins can directly inhibit transcription and/or translation. The present study does not elucidate this question and more work will be required to understand the processes; underlying the effects described here.An overall translational inhibition induced by CHX treatment results in reduced de novo synthesis of 45-47S pre-rRNA precursor as well as the levels of 5S rRNA in DAOY cells. In contrast, TBEV infection only affected the 45-47S pre-rRNA precursor (and mature 18S and 28S rRNA levels) and did not affect 5S rRNA. This suggests TBEV-specific inhibition of POLR1 activity, which could result in reduced production of host proteins. Further analyses are needed to characterise the connection between rRNA production arrests and translational shut-off upon TBEV infection.In summary, our results give new insights into the flavivirus-host interactions at the transcriptional/translational level. Moreover, a virus-induced rRNA decrease was described for flaviviral infection for the first time. The research here can contribute to understanding the mechanisms which determine at least to some extent the subsequent pathological processes. However, the relatively late onset of effects described in this study cannot completely rule out the possibility that our observations are due to cellular responses to TBEV infection rather than virus-mediated, or even combinations of both cellular and viral effects. More work is required to assess these possibilities in detail.DAOY cells were infected with either TBEV Neudoerfl or Hypr strains (MOI 5) or untreated (mock); at indicated time intervals, cells were counted. A two-fold serial dilution was prepared with range from 50000 to 390 cells/well and cell viability was subsequently analysed by using alamarBlue reagent. included. Cells were starved for 1 hour in methionine-free medium and subsequently, nascent proteins were labelled using AHA (incubation for 2 hours; non-labelled negative controls, NC). Cell lysates analysed by SDS-PAGE followed by proteins transfer to PVDF membrane and Click reaction using biotin-alkyne. De novo synthesized proteins were further visualized by using biotin-streptavidin detection system along with conjugated alkaline phosphatase. Developed membranes were then stripped and NS3 viral protein detected. Total protein pattern was visualized using CBB staining of the gels after blotting. Representative images out of three independent experiments are shown. \n",
            "\t  0.57518846 \t The performance of the new ePlex Respiratory Pathogen (RP) panel (GenMark Diagnostics) for the simultaneous detection of 19 viruses (influenza A virus; influenza A H1 virus; influenza A 2009 H1 virus; influenza A H3 virus; influenza B virus; adenovirus; coronaviruses [HKU1, OC43, NL63, and 229E]; human rhinovirus/enterovirus; human metapneumovirus; parainfluenza viruses 1, 2, 3, and 4; and respiratory syncytial virus [RSV] [RSV subtype A and RSV subtype B]R espiratory tract infections are a significant contributor to the global burden of respiratory tract illnesses, with up to 4 million deaths worldwide in 2013 (1) . Several pathogens, including viruses and bacteria, can cause respiratory tract infections. As clinical presentations often overlap, identification of the underlying pathogen based solely on clinical criteria is challenging (2) . Even though most respiratory tract infections are self-limited in immunocompetent hosts, many are treated, as shown in a recent study highlighting the increase in unnecessary antibiotic use in this group (3) . In other patient groups, including immunocompromised patients, elderly patients, and critically ill patients, complications from respiratory tract infections often occur, with poor outcomes including increased morbidity and mortality (4) (5) (6) . Thus, the ability to rapidly and accurately diagnose respiratory tract infections is critical for optimal patient care. Additionally, rapid and accurate diagnosis is essential for the timely administration of antiviral therapy, when available, and the institution of contact and droplet precautions to prevent health care-associated transmission (7, 8) . Several instruments and multiplexed molecular panels are currently approved by the U.S. Food and Drug Administration (FDA) for the rapid and sensitive detection of viral and bacterial respiratory pathogens in nasopharyngeal swab (NPS) specimens. These methods differ based on several criteria, including the degree of multiplexing (4 to 22 targets), complexity of the method (moderate to high), throughput (low to high), and turnaround time (TAT) (ϳ1 h to 8 h) (9) . In general, performance characteristics of these multiplex panels are significantly better than those of conventional methods, including viral culture and antigen tests, and have replaced these methods in many laboratories. On the other hand, the increased cost of multiplexed molecular panels compared to the cost of conventional methods has raised questions on their cost/benefit ratio, particularly as many viral targets on these panels have no specific treatments. However, given their utility in the rapid diagnosis of infections (especially in immunocompromised patients), infection control and prevention practices, and antibiotic stewardship, the interest in multiplexed PCRs remains high.The GenMark Respiratory Pathogen (RP) panel is a qualitative nucleic acid multiplex test that recently received U.S. FDA clearance for use on GenMark's new ePlex instrument. The ePlex RP panel detects 21 of the most common respiratory pathogens in NPS specimens. Viral targets identified are adenovirus, coronavirus (229E, HKU1, NL63, and OC43), human metapneumovirus (hMPV), human rhinovirus/enterovirus (HRV/EV), influenza A virus, influenza A H1 virus, influenza A 2009 H1 virus, influenza A H3 virus, influenza B virus, parainfluenza virus 1 (PIV1), PIV2, PIV3, PIV4, respiratory syncytial virus subtype A (RSV-A), and RSV-B. Bacterial targets identified are Chlamydia pneumoniae and Mycoplasma pneumoniae (10) . The ePlex RP panel is a sample-to-answer multiplex assay that runs on a single-use cartridge that automates all aspects of nucleic acid testing, including extraction, amplification, and detection. It combines electrowetting microfluidics and GenMark's eSensor technology, which is based on the principles of competitive DNA hybridization and electrochemical detection, as previously described for the GenMark XT-8 Respiratory Viral panel (11) .In this multicenter clinical trial study, the performance characteristics of the investigational-use-only (IUO) ePlex RP panel were evaluated by using NPS specimens collected at 13 sites across the United States and Canada. Reproducibility was evaluated at 3 sites. In addition to assay performance, other characteristics, including assay workflow and turnaround time, were evaluated.Study population. The study population included patients of all ages and both genders presenting with signs and/or symptoms of upper respiratory tract infection at 13 clinical sites located in the United States and Canada. Residual NPS specimens were prospectively collected from March 2013 through August 2014 from 5 sites and frozen at Ϫ80°C for future testing and were prospectively collected from September through October 2016 from 4 sites and tested fresh. To supplement the results of the prospective collection, NPS samples positive for low-prevalence pathogens were retrospectively collected and frozen. Frozen samples were thawed, pipetted into separate aliquots, and frozen at Ϫ80°C until they were tested by ePlex RP, the BioFire RP, and/or target-specific PCR/bidirectional sequencing as needed. Also, for low-prevalence pathogens, additional contrived samples were used to evaluate assay performance.Overall study design. NPS specimens were prospectively and retrospectively collected in viral transport medium, handled, and processed according to the NPS kit manufacturer's instructions. Either samples were collected specifically for this study, separately from a standard-of-care (SOC) purpose, or the residual sample was deidentified and provided after SOC was completed. The study was approved by a central quorum review institutional review board and/or individual clinical testing site institutional review board.Samples were tested with the ePlex RP panel at 1 of 5 sites and compared to results of testing with BioFire FilmArray respiratory panel version 1.7 (bioMérieux, Durham, NC) and an analytically validated PCR amplification assay(s) followed by bidirectional sequencing, if necessary (e.g., for resolution of discordant results or virus subtyping) (Laboratory Corporation of America, Morrisville, NC).GenMark ePlex RP panel testing. Testing with the ePlex RP panel was performed according to the manufacturer's instructions, using the materials in the kit. Briefly, after vortexing, 200 l of the primary NPS sample was aspirated into the sample delivery device (SDD) provided with the ePlex RP panel kit and vortexed. The entire volume of the SDD was dispensed into the sample loading port of the ePlex RP panel cartridge, and the cap was depressed to close the port. Each cartridge was bar-coded and scanned with the ePlex instrument and inserted into an available bay (Fig. 1) . Upon the completion of the assay run, the ePlex instrument ejected the cartridge for disposal, and an ePlex RP panel report was generated.Comparator method and method for resolution of discordant results. The comparator method was BioFire FilmArray respiratory panel version 1.7. All collected samples were tested with the BioFire RP, which detects all the respiratory viral and bacterial targets included on the ePlex RP panel but does not differentiate between RSV subtypes A and B (12) . Samples with RSV detected by the BioFire RP were additionally tested with an analytically validated PCR amplification assay(s) followed by confirmation by bidirectional sequencing to determine the subtype (Laboratory Corporation of America, Morrisville, NC).Results from collected samples that were discordant between the ePlex RP panel and the comparator method (i.e., false negative [FN] or false positive [FP]) were tested with an analytically validated PCR amplification assay(s) followed by bidirectional sequencing as described above. For coronaviruses, additional repeat testing by the BioFire RP and ePlex RP panel was also conducted as part of discordantresult resolution.Reproducibility study. Three reproducibility panels, each with 6 pathogens (influenza A H3 virus, RSV-A, PIV1, hMPV, coronavirus OC43, and adenovirus species B) at 3 different concentrations (moderate [3ϫ the limit of detection {LOD}]), low [1ϫ LOD], and negative) in NPS specimens, were tested at 3 sites. LOD studies were performed by the manufacturer (data not shown). Reproducibility testing was performed by 2 operators at each of 3 sites, and each operator tested the reproducibility panels in triplicate over 6 days (including 5 nonconsecutive days) by using 3 different cartridge lots, resulting in 2 days of testing for each of the 3 lots. Statistical methods. Positive percent agreement (PPA), negative percent agreement (NPA), and overall percent agreement (OPA) with the comparator method results or expected results were determined for each target detected by the ePlex RP panel. The PPA was calculated as 100 ϫ no. of TP/(no. of TP ϩ no. of FN), the NPA was calculated as 100 ϫ no. of TN/(no. of TN ϩ no. of FP), and OPA was calculated as 100 ϫ (no. of TP ϩ no. of TN)/(no. of TP ϩ no. of TN ϩ no. of FP ϩ no. of FN), where TP is true-positive results, FN is false-negative results, TN is true-negative results, and FP is false-positive results. The two-sided 95% score confidence interval (CI) was calculated for PPA, NPA, and OPA. Statistical analysis was performed by using SAS version 9.4. Workflow and turnaround time study. A time study was performed to determine the time needed to perform the ePlex RP panel test. Data were averaged from 20 samples tested by 2 operators. Each step of the process from sample preparation to unloading of the instrument was recorded and timed, and an average TAT was determined.Patient demographics. A total of 2,462 prospectively collected, 446 retrospectively collected, and 327 contrived samples were eligible for testing with the ePlex RP panel. Approximately 67% of samples were collected with Copan-manufactured UTM (universal transport medium), and 33% were collected with Remel M4 or M5 medium. Demographic information for the evaluable prospectively and retrospectively collected subject samples is provided in Table 1 . Approximately 50% of the subjects were male, with median ages of 33 years for subjects with prospectively collected samples and 5 years for subjects with retrospectively collected samples. The prevalences of ePlex RP panel targets by age group during the two phases of prospective collection are provided in Tables S1 and S2 in the supplemental material.Assay performance. (i) Accuracy. Of the 3,235 samples eligible for testing with the ePlex RP panel, 154 had invalid results (4.8%). After repeat testing, 8 samples were not evaluable due to final, invalid results (7 prospective samples and 1 contrived sample). The final validity rate was 99.8% (95% CI, 99.5% to 99.9%). The ePlex RP panel detected a potential respiratory pathogen in 1,212 of 2,462 prospectively collected NPS specimens tested, for a positivity rate of 49%. The PPAs and NPAs with 95% CIs of the ePlex RP panel targets with comparator methods are provided in Tables 2 to 4 separately for retrospectively collected samples and prospectively collected samples and with all samples combined. Additional data for fresh and frozen prospectively collected samples are presented in Table S3 in the supplemental material.The overall percent agreement between the ePlex RP and BioFire RP results was Ͼ95% for all targets tested. target but positive for either the 2009 H1 (n ϭ 5) or H3 (n ϭ 1) subtype, and 9 samples were positive for the influenza A virus target without a subtype being identified. Of note, influenza A H1 virus was not detected, and very few Chlamydia pneumoniae (n ϭ 4) isolates were detected during the study period. For these targets, at least 50 specimens with the pathogen spiked into negative NPS specimens were contrived; all were detected by ePlex RP (data not shown). Discordance resolution results (Table 5 ) demonstrate that while ePlex RP had a high level of agreement with the BioFire RP and detected the same targets in nearly all samples, each assay detected targets that the other did not. In Table 5 , samples with targets that were detected by ePlex RP but not detected by BioFire RP (FP) are counted as true-positive samples if the target was detected by discordance testing (TPd). Samples with targets not detected by ePlex RP but detected by the BioFire RP (FN) are counted as true-negative samples if the target was not detected by discordance testing (TNd). Given the number of discordant samples between the two panels for coronavirus detection, additional repeat testing by both the BioFire RP and ePlex RP was performed as part of the discordance analysis. This additional testing demonstrated evidence of Of the 2,462 prospectively collected samples, 164 had codetections by at least one of the assays (data not shown). The ePlex RP panel identified a total of 135 prospective samples with multiple pathogens detected, or 5.5% of all prospectively collected samples, while the BioFire RP detected a total of 116 samples with multiple pathogens, or 4.7% of all prospectively collected samples. Of the 135 samples with multiple pathogens detected by ePlex RP, 118 (4.8%) had two pathogens, 14 (0.6%) had three pathogens, and 3 (0.1%) had four pathogens detected. Of the 135 samples with codetections, 57 samples (2.3%) included one or more pathogens that were not detected by the BioFire RP method. The comparator method identified a total of 32 (1.3%) codetections with one or more pathogens not identified by the ePlex RP panel. Discordance resolution results are included in the analysis presented in Table 5 .(ii) Reproducibility. Each of 3 sites performing the reproducibility study collected 35 to 36 results for each target at each concentration, for a total of 107 to 108 samples per site. For the moderately positive and the negative panels, the percent agreement was 100% (95% CI, 96.6% to 100%) for all 7 panel targets. For the low-positive panel, the percent agreement was 100% (95% CI, 96.5% to 100%) for 6 of the 7 panel targets. For the remaining adenovirus panel target, the percent agreement in the low-positive panel was 91.6% (95% CI, 84.8% to 95.5%). Agreement results are shown in Table 6 . A total of 315/324 (97.2%) samples yielded a valid result upon initial testing. Upon repeat testing, all but one specimen yielded a valid result, for a final validity rate of 99.7% (95% CI: 98.3% to 99.9%). No significant sources of variability were found for any of the targets on the panel across cartridge lots, operation days, technologists/operators, or sites (data not shown).Workflow and turnaround time study. The time to identification from receipt to result was determined to be 1 h 45 min and 57 s, with hands-on times (HOTs) per sample of 1 min and 35 s.In this multicenter clinical study, the performance characteristics of the new ePlex Respiratory Pathogen panel on the GenMark ePlex instrument were estimated by measuring agreement with the results of the BioFire RP, a commercially available multiplex respiratory panel. The ePlex RP panel was recently FDA cleared and expands the available options for laboratories interested in the rapid diagnosis of respiratory tract infections using broadly multiplexed molecular assay on NPS specimens. Similar to the BioFire RP, ePlex RP offers a large panel of both viral and bacterial respiratory pathogens in a simple, sample-to-answer format with minimal hands-on time and a rapid turnaround time to results.As with all molecular diagnostic test methods, both the BioFire RP and ePlex RP have the potential to generate false-positive and false-negative results, as confirmed in this study by additional testing with target-specific PCRs with bidirectional sequencing. The unavailability of an established gold standard prevents a true measure of correctness, requiring the use of agreement measures with a nonreference standard instead to establish the performance characteristics of a new test (13) . Thus, data presented in this study reflect only the agreement between the results of the two platforms. In spite of these limitations, the overall performance of ePlex RP was comparable to that of the BioFire RP, with an overall percent agreement (true-positive and true-negative results) of Ͼ95% for all available targets tested. In line with our results, a recent study by Nijhuis and colleagues evaluating the performance of the ePlex research-use-only (RUO) RP panel compared to laboratory-developed assays found an overall agreement of 97.4%, with discordant results occurring primarily for samples with low pathogen loads, as estimated by using cycle threshold (C T ) values (i.e., high C T value, Ͼ35) (10).The highest numbers of discordant results were observed for adenovirus, coronavirus, and rhinovirus/enterovirus targets. This finding may be explained by the large diversity of genotypes present within each of these groups of pathogens or the prevalence of these pathogens in the population tested. For adenovirus, the performance of diagnostic tests is particularly important for young children and immuno- compromised hosts. However, the detection of adenovirus can be challenging. In this study, ePlex RP detected more adenoviruses than did the BioFire RP. A recent report on the performance of BioFire RP v1.7 for the detection of adenovirus highlighted the decreased sensitivity of the assay for samples with low viral loads or those positive for species A, D, and F (14) . Although the ePlex RP panel detects 4 genotypes of coronaviruses (HKU1, OC43, 229E, and NL63), results are reported without the genotyping information. There are currently no treatments for coronaviruses, and the utility of the genotyping information may lie in its value for epidemiological studies. Reports of severe coronavirus infections have been reported for immunocompromised hosts, but there has not been any indication that severity is associated with a particular genotype (15) .Similar to the BioFire RP, ePlex RP detects a wide range of rhinovirus and enteroviruses genotypes, but given their high degree of genetic similarity, it does not reliably differentiate between these two species. Based on discordance analysis performed by PCR and bidirectional sequencing, most samples were positive for rhinovirus, a trend that is expected given that rhinoviruses are the most common cause of the common cold (16) .The overall agreement for influenza viruses was high. As the prospective collection periods for this clinical study occurred during the 2013 and 2016 respiratory seasons, there was an expected lack of clinical specimens positive for influenza A H1 virus (17) . Most of the influenza virus infections during that time period were due to the 2009 H1 and H3 viruses. Hence, the evaluation of the influenza virus A H1 target was performed entirely by using contrived NPS samples. Similar to the BioFire RP results, instances of samples positive for influenza A virus without subtypes or positive for an influenza A virus subtype only were identified. Such results may be caused by low virus titers or a novel influenza virus subtype and thus require additional testing.Only a limited number of samples were positive for Chlamydia pneumoniae, and the evaluation was conducted primarily with contrived samples. The performance for C. pneumoniae was similar to that reported in the BioFire RP clinical study, which was also conducted by using primarily contrived samples (12) . Both Chlamydia pneumoniae and Mycoplasma pneumoniae are associated with atypical, community-acquired pneumonia (CAP), with no seasonality and nonspecific symptoms (18) . Therefore, the inclusion of both bacteria in large multiplex panels has the potential to significantly improve the diagnosis of CAP.A notable difference between the ePlex RP panel and the BioFire RP is the absence of Bordetella pertussis-Bordetella parapertussis targets among the bacterial targets. It is noteworthy that unlike other bacterial targets on the panel, infection with Bordetella pertussis-B. parapertussis has a relatively specific epidemiology, with higher incidences typically being reported for children less than 2 months old and, more recently, for unvaccinated populations or vaccinated patients with waning immunity (19) . Thus, exclusion from a highly multiplexed respiratory panel may be warranted. In addition, the utilization of the single-copy pertussis toxin promoter target (ptxP) in some multiplex panels has been shown to be less sensitive for the detection of B. pertussis than PCR tests targeted to the multicopy IS481 insertion sequence (20) . As a result, when B. pertussis infection is suspected, an FDA-cleared B. pertussis molecular test may be more appropriate.Similar to other broadly multiplexed respiratory panels, the ePlex RP panel allows the detection of coinfections. In this study, coinfections were identified in 5.5% of samples, with human rhinovirus/enterovirus being the most common virus detected in coinfected specimens. The importance of detecting coinfections was highlighted in a recent study showing that viral-bacterial coinfection during severe CAP in adults resulted in a more complicated disease course and was associated with complications (21) . In other studies, human rhinoviruses were frequently detected in association with bacterial pathogens in a hematopoietic stem cell transplant population, suggesting that this commonly detected virus may be a significant cause of severe pneumonia in immunocompromised patients (22, 23) .The initial rate of invalid results with ePlex RP was 4.8%, with repeat testing resulting in a final rate of invalid results of 0.2%. This value falls in the range of rates of invalid results reported for other multiplexed respiratory panels, including Nanosphere Verigene RV (9.7%), xTAG RVP (1 to 2%), and the BioFire RP (1 to 3%) (24) (25) (26) (27) .The ePlex instrument consists of a stand-alone system, with no additional accessories (e.g., computer) and a touch screen, making it a great option for space-constrained laboratories. The ePlex instrument is available in configurations ranging from one to four towers, with each tower containing six random-access testing bays. This supports single-shift (8 h) testing volumes ranging from 24 samples to 96 samples, with a TAT of less than 2 h and an HOT of less than 2 min, providing a flexible capacity for laboratories of all sizes. Additional features available on the system include the ability to generate customized reports (e.g., statistical and trending reports) and the potential for a bidirectional interface with laboratory information systems.In conclusion, the ePlex RP panel provides a rapid, sensitive, broadly multiplexed, sample-to-results option with minimal hands-on time. The assay performance was equivalent to that of the BioFire FilmArray RP for all targets. This new, fully integrated instrument is easily expandable and offers a great option for the implementation of a multiplex assay in larger-volume laboratories.Supplemental material for this article may be found at  .01658-17.SUPPLEMENTAL FILE 1, PDF file, 0.1 MB.\n",
            "\t  0.57211834 \t When analysing new emerging infectious disease outbreaks, one typically has observational data over a limited period of time and several parameters to estimate, such as growth rate, the basic reproduction number R 0 , the case fatality rate and distributions of serial intervals, generation times, latency and incubation times and times between onset of symptoms, notification, death and recovery/discharge. These parameters form the basis for predicting a future outbreak, planning preventive measures and monitoring the progress of the disease outbreak. We study inference problems during the emerging phase of an outbreak, and point out potential sources of bias, with emphasis on: contact tracing backwards in time, replacing generation times by serial intervals, multiple potential infectors and censoring effects amplified by exponential growth. These biases directly affect the estimation of, for example, the generation time distribution and the case fatality rate, but can then propagate to other estimates such as R 0 and growth rate. We propose methods to remove or at least reduce bias using statistical modelling. We illustrate the theory by numerical examples and simulations.During the last decades, several new disease outbreaks have caused worldwide alarm, e.g. SARS, foot and mouth disease, H1N1 influenza, and, more recently, Ebola. What these outbreaks have in common is the need for estimation of key parameters early on, in order to plan interventions and monitor the progress of the disease. Thus estimation must be performed in the emerging phase of an outbreak, when the number of infected individuals is in the hundreds or at most thousands, while the community fraction of infected is still small. Typically, the early numbers grow exponentially, as also predicted by mathematical epidemic models .There may be many complicating or limiting factors related to incompleteness of data, lack of detailed knowledge about the disease and other issues when analysing data from the early phase of an outbreak. Despite these complicating factors, the conclusions drawn from early analyses, often based on simple models, are usually highly valuable. The aim of the present paper is to identify and highlight some of the potential biases in the statistical analysis of emerging outbreaks and to illustrate how they can propagate to parameter estimates and predictions. A further aim is to give some fairly simple suggestions for how to reduce, or even remove, such biasing effects.The typical available data consist of reported numbers of cases per day or week, some case histories illustrating the course of the disease and some contact tracing data containing information about possible durations between onset of symptoms of infected individuals and their infectors. The epidemic models used in the statistical analyses are often of simple form, neglecting various heterogeneities. The use of simple models in these situations is motivated by the lack of detailed information but has also recently been studied , showing that neglecting population structures when making inference in emerging & 2019 The Authors. Published by the Royal Society under the terms of the Creative Commons Attribution outbreaks has little effect. However, estimation in simple models can still be quite complicated. The complications are mainly due to three factors: (1) important events, such as times of infection, are usually unobserved, but instead some proxy measures such as onset of symptoms are available, (2) estimation of parameters of the epidemic process is based on observations up to some fixed time, implying that events occurring later are censored and (3) the population of infectives is increasing (exponentially) with time.In our investigation, we assume a simple, homogeneously mixing, model of an outbreak (the model would have to be different if one had knowledge of household size distribution, contact network, spatial heterogeneity, etc.) and we first discuss the effects of estimating the generation time distribution from observations of generation times observed backwards in time using contact tracing, i.e. the time between the infection time of an individual (the infectee) and that of his/her infector (rather than the infection time of the individuals he/she infects). The second problem we study is the effect of replacing generation times (the time between infections of an infector and an infectee) with the more commonly observed serial intervals (the time between onset of symptoms of an infector and an infectee). A third problem we discuss is how to treat the common situation, when contact tracing, of finding more than one potential infector of a case, with the implication that the backward generation time or incubation time (time from infection to symptoms) is one out of several possible values. As it turns out, the potential biasing effects of these problems all go in the same direction and, for example, the basic reproduction number R 0 (the average number of infections in a fully susceptible population) can be quite significantly underestimated. We also point out some general stability properties of ratios during exponential growth that may be useful for inference. We then quantify the various biases that can arise in realistic parameter settings, both theoretically and by simulation, mostly using parameters freely adapted from the recent Ebola epidemic in West Africa .In the next five sections, we describe the theory behind our results. In §7, we illustrate our findings with numerical examples and report some interesting simulation results. Section 8 is a brief discussion and, finally, mathematical details and proofs as well as detailed numerical and simulation results are collected in the electronic supplementary material. Computer code for simulations is available in separate files.We start by presenting the basic underlying epidemic model. We assume that individuals are at first susceptible and later may become infected, and that infected individuals may then infect other individuals. The infection ends with death or recovery. The population is assumed to be a homogeneously mixing community of homogeneous individuals.Since we model the initial phase of the outbreak, the depletion of susceptibles is considered as negligible. Also, we assume that individuals do not change their behaviour over the considered time period as a consequence of the ongoing outbreak, nor are there yet any control measures put in place by health authorities or similar. Finally, we assume that there are no seasonal changes in transmission.Predictions are made assuming that the disease spreading mechanism continues unaltered, reflecting what would presumably happen in the absence of control measures (these predictions should then be compared with predictions including various preventive measures). Traditionally, the population effects of such an infection have been modelled using compartmental models with separate states, for example, susceptible, latent (historically called exposed and hence abbreviated by E), infectious or recovered/removed individuals (SI, SEI, SIR and SEIR models; e.g. [1, 4] ). Recently, modelling has reverted to something akin to the original Kermack & McKendrick formulation, with emphasis on one single quantity, b(s), the average rate at which an infected individual infects new individuals s time units after his/her time of infection, denoted the infection rate or infectivity function. The assumption of a homogeneous community implies that b(s) is the same for all individuals, and the assumptions of no depletion of susceptibles, no preventive measures and no seasonal effects imply that b(s) is independent of the time of infection of the individual. The previously mentioned compartmental models can all be translated to this framework. While the original treatment of the Kermack -McKendrick model was deterministic (Volterra-type integral equations), statistical modelling requires a stochastic formulation which, in this case, corresponds to a Crump-Mode -Jagers branching process in the initial phase of spread. It should be noted that the infectivity functions in a stochastic model may be different from individual to individual, although the average behaviour is the same, and that different stochastic models may have the same average behaviour .The average infection rate b(s) completely determines the basic reproduction number R 0 (the average number of new infections caused by one infectious individual in a completely susceptible population), as is well known in epidemic modelling and branching process theory .The basic reproduction number R 0 is given byIt is well known that an epidemic can take off if and only if R 0 . 1, which we from now on assume. Another important quantity is the so-called generation time distribution f G (s), which is simply the infection rate scaled to make it a probability density:The generation time distribution is the probability distribution of the time between the moment of infection of a randomly chosen infective and that of his/her infector.In what follows we will write R 0 f G (s) instead of b(s). Let i(t) denote the expected incidence at t (time since the start of the outbreak, assumed to have been started by one infective individual), i.e. the average community rate of new infections. Since we assume that individuals infected s time units ago (at time t 2 s if present time equals t) will infect new individuals at rate R 0 f G (s) we have the following renewal equation for i(t) (see [1, p. 212] ): J. R. Soc. Interface 16: 20180670 The additive term in the above equation derives from the initial infective that is supposed to have started the outbreak at t ¼ 0. It is well known that the incidence i(t) will quickly approach exponential growth i(t) C e rt , where r is the socalled Malthusian parameter, the exponential growth rate, defined as the solution to the Euler -Lotka equation(2:4)To simplify matters, we will assume that this exponential growth of new cases holds from the start. The validity of this assumption will be shown by subsequent simulations. In figure 1 , 10 simulated epidemics are plotted over time showing the exponentially growing feature (clearly visible on the log-scale). Thus, knowing the generation time distribution f G ( Á ) and one of R 0 and r allows the determination of the other one (cf. ), e.g. using equation (2.4) . For this reason, estimation of the generation time distribution f G ( Á ) becomes paramount in this model formulation and will be extensively discussed in following sections. Also, various rather general conclusions about the effects of varying the components of (2.4) related to the directions of biases in the estimation of these components can be drawn. The mathematical details are given in the electronic supplementary material and the specific results will be discussed in the relevant sections.There are other approaches to the estimation of R 0 and r. The exponential growth rate can be directly estimated from case data and R 0 through modelling approaches (e.g. ), assuming the generation time distribution to be known, or in the so-called 'First Few Hundred' studies, usually restricted to transmission in households (e.g. [13 -15] ). In , the joint estimation of R 0 and the generation time distribution is contemplated, but the authors suggest that these methods do not work well during the early phase of an epidemic. In this paper, we assume that the generation time distribution, as well as the incubation time distribution and also distributions of time from notification to recovery/death, will be estimated by limited contact tracing or specific samples during the initial phase of the disease spread and that, otherwise, just counts of cases in various stages are available.In the model description above, the expected incidence i(t) is a time-continuous deterministic function. The true incidence is, of course, integer-valued and, in most situations, observations are not made continuously but are aggregated in discrete time units such as days or weeks. A related discrete time model is obtained by suitably discretizing equation (2. 3) so that the expected incidence I(t þ 1) in time (interval) t þ 1 is expressed aswhere p G is a discrete probability distribution for the generation time corresponding to the continuous-time distribution f G . A natural statistical model for data collected daily or weekly is then to assume that the number of newly infected I(t þ 1), given previous incidence, follows a Poisson distribution with mean parameter as in equation (2.5) (e.g. ). Finally, the quantitative evaluation of many theoretical results requires explicit assumptions about the involved probability distributions and other parameters typical of the disease under study. As illustrations, we have chosen to use Gamma distributions, where possible, because of their flexibility and analytical properties, and parameters compatible with the recent 2014 West Africa Ebola epidemic . Various formulae related to these assumptions are collected in the electronic supplementary material. 3. Looking backwards rather than forwards in timeThe generation time distribution f G (t) ¼ b(t)/R 0 describes the variability of the (random) time between the moment of infection of an individual and the moments that this individual infects other individuals. When trying to estimate this distribution from outbreak data, the most common situation is where infected cases are contact-traced, i.e. the infectors of cases are identified, and the duration between the infection times of infector and infectee is ascertained (at least in theory, but see also next section). This seemingly innocent choice of looking backwards rather than forwards in time (measuring duration backwards from an infectee rather than forwards from an infector) actually modifies the distribution of observed times in the early stage of an outbreak when the epidemic grows at an exponential rate (e.g. ). The reason is that, by looking backwards in time, long generation times will be underrepresented and short generation times will be overrepresented because exponential growth implies that there are many more recently infected individuals who are potential infectors compared to those infected longer ago. As a consequence, if the generation time distribution is estimated from a sample of backward generation times, the resulting distribution f B (s) will be different from the true generation time distribution f G (s).In fact, it can be shown that the backward generation time has density f B (t) ¼ e 2rt R 0 f G (t) (note that equation (2.4) implies that this function integrates to 1). It can also be shown that this density is stochastically smaller than f G and thus has smaller mean than f G (see the electronic supplementary material). We can in fact say more. One can predict the effect on estimating R 0 of using f B (t) instead of f G (t) assuming that the growth rate r is known or approximately known through observations. Since incidence essentially is R 0 Â a weighted sum of previous incidence (cf. equation (2.5)) and f B (t) attributes too much weight to recent incidence (shorter generation times), which is higher than earlier incidence, there will be a compensatory underestimation of R 0 (see §7 for illustrations). This effect also takes place using the Euler -Lotka equation (2.4) .If instead f B (t) is used to calculate the exponential growth rate in equation (2.4) , assuming that the correct value of R 0 is used, the resulting growth rate r B will always be larger than r. The exact relation is model specific, but as an example one may consider the simple Markovian SIR model, where the infectious period has an exponential distribution with expected value 1/g and the infectious contacts, in the initial phase of the epidemic, occur with intensity b during the infectious period. Theand f B (t) ¼ be 2bt . Then, the resulting r B equals R 0 r. With typical values of R 0 being between 1.5 and 2, this means that the exponential growth rate will be grossly overestimated (50-100%), when using equation (2.4) .Finally, if an estimatef B (t) has been obtained from generation times observed backwards and the exponential increase rate r is known or has been estimated from observations, then, in theory, the density f G (t) could simply be estimated as C e rtf B (t) where C is a suitable normalizing constant. However, since this method risks overweighting large observations, it is maybe preferable to do the correction within a suitable parametric form. For instance, if f G is a Gamma distribution with parameters a and l, then f B is again a Gamma distribution, now with parameters a and l þ r (see the electronic supplementary material). By inverting this relation, a suitable correction procedure can be obtained.The above analysis of the potential bias in ascertaining generation times in the early stage of an outbreak is focused on 'backward' contact tracing, because this is the most common situation studied in theory and used in practice, i.e. finding potentially infectious contacts that have occurred in the past. One could, in theory, consider the case of 'forward' contact tracing and potential sources of bias in that situation, but many assumptions would be needed for, e.g. how infectivity after discovery would be related to infectivity prior to discovery, for how long the tracing would go on, etc. There is also a theoretical problem, namely that the 'forward' generation times do not have the same distribution as 'backward' times (e.g. ), even in the absence of the exponential growth bias illustrated above.As described earlier, the generation time is defined as the time between moments of infection of an infector-infectee pair. However, in real life, the infection times are rarely known. Instead, typically, the onset of symptoms is observed. For this reason, the serial interval, which is defined as the time between symptom onsets in the two individuals mentioned above, is often used as a surrogate for the generation time.We now study the effects of using serial intervals instead of 'true' generation times when estimating the generation time distribution f G ( Á ) and on derived quantities, such as r and R 0 .Considering the disease and infectivity history of an individual, starting from the moment of infection, several time periods are of interest (see also figure 2). We denote the time of infection of this individual by t 0 , there may be a latent period of length ' 0 until start of infectivity followed by an infectious period of length i 0 , and a time from infection Figure 2 . Relationship between generation time G and serial interval S of an infector and its infectee. The infector is infected at time t 0 and then infects the infectee at t 1 . The red circles indicate end of latent period and start of infectious period, the black circles indicate onset of symptoms, and black boxes end of infectious period (either by death or recovery). In the figure, the infectious period starts slightly before onset of symptoms, but, in general, the relationship between these event times is disease-dependent. In the illustration, the serial interval is shorter than the generation time, S , G, but the opposite relation could equally well happen. (Online version in colour.) J. R. Soc. Interface 16: 20180670to symptoms (incubation period) of length s 0 . Assume that another individual is infected by the first one after a time g Ã within the infectious period i 0 , i.e. at timeand that this second individual shows symptoms at time s 1 after infection. Then, the generation time is G ¼ (t 0 þ ' 0 þ g Ã ) À t 0 and the serial interval S ¼ (t 1 þ s 1 ) 2 (t 0 þ s 0 ) (see figure 2 for an illustration).Although much work has been devoted to estimating the distributions of incubation, latent and infectious periods for various diseases, relatively little has been done regarding their joint distribution. It will be seen below that this joint distribution plays an important role for the relationship between G and S. Let us only assume, for a start, that the involved times are independent between different individuals and that corresponding periods have identical marginal distributions for different individuals. We may then rewrite the above expressions asThese representations are quite unnatural, but show the common structure of G and S. For instance, we see that S ¼ G þ (s 1 2 s 0 ) and thus the expected values of G and S will be equal since s 0 and s 1 are assumed to have identical expected values. We also see in equation (4.1) that S is the sum of two independent components (since they regard different individuals) and thus its variance will be the sum of the variances of these components, while the variance of G will consist of the sum of the same two variances and a further term, 2Cov(s 0 , ' 0 þ g Ã À s 0 ), by the rule for variances of sums.Depending on assumptions, we can now have different results. Under quite usual assumptions of independence between s 0 , ' 0 and i 0 and also Var(s 0 ) . 0, the covariance above will be negative, implying that Var(S) . Var(G). If we, instead, want the variances of G and S to be the same, we must require Cov(s 0 , ' 0 þ g Ã À s 0 ) ¼ 0, which is a rather special balance between various parameters in the joint distribution of (s 0 , ' 0 , i 0 ) (to simplify matters slightly, one can note that assuming g Ã to have a uniform distribution in i 0 leads to Cov(s 0 , g Ã ) ¼ Cov(s 0 , i 0 )=2; see the electronic supplementary material). However, it is also theoretically possible to have the reverse relation, e.g. if s 0 ¼ ' 0 and Cov(s 0 , i 0 ) . 0.One may also investigate whether the distributions of S and G can be identical, as argued in , under the assumption that s 0 ¼ ' 0 (i.e. the end of the latent period/start of infectious period is identical with onset of symptoms). However, since one then has G ¼ s 0 þ g Ã and S ¼ s 1 þ g Ã , one must impose further conditions (see the electronic supplementary material). The independence of s 0 and g Ã (or i 0 ) would be sufficient but not necessary for the result. At least, s 0 must be uncorrelated with g Ã , since variances must coincide. However, we have not been able to find a simple sufficient and necessary condition for equality (see the electronic supplementary material for further details).The exact relationship between the generation time G and serial interval S is thus model-dependent, but it always holds that they have the same mean. As for the variances, nothing can be said in complete generality. However, for all existing models we are aware of, it holds that V(S) ! V(G), with equality requiring rather specific assumptions. So, except in specific cases, the observed serial interval distribution will be a biased estimate of the generation time distribution and will have a larger variance. The quantitative effects of using a distribution with equal mean but larger variance than the true one are again model-dependent, but, for example, assuming Gamma distributions, there will again be underestimation of R 0 , given r, and overestimation of r, given R 0 (see the electronic supplementary material for further details).A general method to correct the serial interval distribution to obtain an unbiased estimate of the generation time distribution will of course be very model-dependent and necessitate further information about the various stages of disease history. However, within a specific distribution family, such as the Gamma distributions, a correction based only on the ratio between the serial interval and the generation time variances can be implemented (see the electronic supplementary material). The above results show that the elements needed for the correction from Var(S) to Var(G) are the covariances between incubation period and latent and infectious periods and the variance of the incubation period. These could be estimated from a close observation of some case histories with known time of infection and/or assumptions such as the above-mentioned s 0 ¼ ' 0 .Contact tracing means that reported cases, with known onset of symptoms, are investigated to find out when they have been in contact with infectious individuals, with the aim of finding who the infector was and when the case was infected, thus allowing estimation of the incubation period or, if the symptom onset time of the infector is also known, the serial interval. In practice, when infected individuals are contact traced, certain cases will have one unique possible infection time, but others will have several potential infectors or infection occasions, or no identified exposure. In the first situation, it is clear who the infector was and also how long the incubation period was, and, in the last case, when there is no identified exposure, there is not much to do. But, in the second scenario, it could be any one of the potential exposures that caused the infection, also implying that the incubation period could be one out of several values. In the current section, we describe one way to infer the incubation period distribution of contact traced individuals in this situation, and also to study the effects of not acknowledging the multiple exposures situation. It should be noted that this is not a standard problem in survival data analysis, where it is usually assumed that the time origin of durations is well defined.Most of the literature about the 'uncertain origin problem' in an epidemiological context arose during the 1980s in connection with inference on AIDS data, where the moment of infection of patients was usually not known exactly (e.g. [20 -23] ). Often, analyses were based on the assumption of a known interval within which infection had occurred and some kind of continuous distribution therein for the moment of infection. The problem reemerged during the SARS pandemic in the early 2000s (e.g. [24, 25] ), but again with data limited to single exposures during known time intervals. In this paper, we analyse the situation where individuals may have more than one exposure, the times of these exposures are known and where there is no detailed information about the nature/strength of  J. R. Soc. Interface 16: 20180670 Let us start by considering the problem of estimating the incubation period distribution, i.e. the time from infection to symptoms, in the simplest model possible allowing for multiple infection exposures and to formulate an appropriate likelihood. Consider one infected individual with onset of symptoms at time s that has been traced for previous infectious contacts and assume that these exposures took place at the time points e 1 , . . ., e k , where e 1 . . . e k , s. In order to obtain a likelihood, we introduce some notation and assumptions. Suppose that at time t, the rate of infection exposure equals l(t), and that the probability of infection upon exposure equals p (the same p for all contacts whether with the same or different infected individuals; if more detailed contact information is available it would be possible to have different p's for different types of contacts and/or different individuals). Finally, let g(t) denote the density distribution of the incubation period. For this model, the likelihood for the infected individual with exposures at times e 1 , . . ., e k and onset of symptoms at s is then given by\" # :(5:1)We will discuss the estimation problem arising from equation (5.1), but we start with some general considerations. It is of course possible to also study more complicated models allowing for individual heterogeneity in susceptibility and/ or various types of contacts having different transmission probabilities, both at distinct times and also during extended periods, e.g. household contacts, but here we consider the simplest model still taking multiple exposures into account, compatible with limited data, only number and times of contacts and time of symptoms. Were data to be different, e.g. containing genetic information from whole genome sequencing of the pathogens, different models would be possible (e.g. ). One can imagine several ways to try to avoid the multiple exposures problem. One approach could be to simply assume that the earliest potential infector is the infector, the likelihood contribution related to the incubation time distribution simplifying to g(s 2 e 1 ) (this would approximately be the same as equation (5.1) if p % 1). This would however certainly lead to the duration of incubation periods being overestimated. The opposite approach, to pretend that the most recent contact was the infector, would similarly lead to underestimation. A type of compromise could be to treat all potential contacts as being potential infection times (to different cases). As a consequence, one observation with k multiple potential infectors would then result in k independent incubation periods s 2 e 1 , . . ., s 2 e k , and the likelihood contribution would become Q k i¼1 g(s À e i ). Compared to the likelihood in equation (5.1), where the shorter incubation periods are given relatively lower weight due to the factor (1 2 p) i21 , such an analysis would lead to the incubation periods (and serial intervals) being underestimated and the precision of the biased estimate overestimated because of the apparent higher number of data points. A related assumption, leading to the same conclusion, would be to treat all potential exposures as equally likely (which would approximately hold true if p % 0).An alternative approach to overcome the difficulty of having multiple potential infectors, is to base inference only on individuals having one exposure, i.e. simply leaving out all contact traced individuals having more than one exposure. This clearly increases uncertainty by using fewer data points. However, it also leads to biased estimates, as we now explain. Individuals having only one exposure and then symptoms must have been infected at this exposure and thus their infection history is certain. However, the fact that no other exposures have happened during the incubation period favours shorter than usual intervals. In fact, the observed time interval will be distributed as the minimum of a typical 'inter-exposure time' and a generic incubation time, and will thus have a distribution different from a generic incubation time, leading to underestimation. In order to obtain explicit expressions for the size of the bias, explicit models of the 'exposure process' and the incubation time distribution are required.If we adopt the simple multiple exposure model defined above, it is however possible to estimate the incubation period distribution using the likelihood in equation (5.1). It is reasonable to condition on the number and times of exposure, since these essentially depend on the 'interexposure process', and to base inference on the second part of the likelihood expression only, containing parameters p and the incubation distribution g( Á ). Assuming a parametric form for g( Á ), e.g. a two-parameter gamma distribution, the problem is non-standard but essentially a three-parameter maximum-likelihood problem with natural bounds on parameters. Simulations show that this approach works well if the correct parametric form is adopted (see the electronic supplementary material). It is also possible to find non-parametric (distribution-free) moment estimators of p, the mean and the variance of the incubation time at the cost of assumptions about the contact process, e.g. as a constant rate Poisson process. Details about one set of such moment estimators and their performance are given in the electronic supplementary material.The individual evolution of a disease is often a sequential process of events delayed with respect to some previous event, starting with infection and then followed by, for example, symptoms, notification, admission to treatment, recovery or death, not necessarily passing through all these states nor in that particular order. The prevalence of individuals in some of these disease stages is of public health interest but reliable data may not always be available. During the early phase of an outbreak, information is usually incomplete due to censoring, but also distorted by exponential increase. The estimation problems and possibilities are thus quite different during the initial phase compared to, retrospectively, after or near its end (e.g. [24, 27] for some analyses based on more complete data). However, during the exponentially increasing phase of spread, a well-known result from branching process theory (a special case of 'counting by random characteristics'; e.g. ), implies that ratios between counts within a specific time window can be predicted.Assume that events occur on the time interval [0, T ]. Assume also that each event may be followed, with a certain probability p, by a secondary event after some time having probability density h(s). Then, assuming that the number of  J. R. Soc. Interface 16: 20180670 primary events grows exponentially at rate r, the fraction p(T ) of secondary events to primary events in [0, T ] will quickly approach (for T not too small)e Àrs h(s) ds:(6:1)More details are given in the electronic supplementary material and in §7.5 about the stability of ratios between various events in the simulations. The above formula illustrates the combined effect of censoring and exponential growth on (theoretical) counts. Thus, taking, for example, infection as primary event and notification as secondary event, knowledge about the notification probability and about the distribution of the time to notification allows estimation of not yet notified cases or of total number of infected from the number of notifications in [0,T ].Another useful application of the above result is to countbased estimation of the case fatality rate (CFR). In this case, p represents the probability of dying from the disease. The problem has been treated by many authors under many different assumptions of data availability (e.g. [27, ). It is well known that a crude estimator of the type D/N, D denoting number of dead individuals and N number of notified individuals in [0, T ], will underestimate the 'true' CFR. In theory, disregarding biased reporting, the underestimation is reflected in the integral part of equation (6.1), which is always less than 1, where h now denotes the probability density function of the time between notification and death. Consequently, knowledge of r and of the distribution h could be used to correct the naive estimate D/N. As an illustration, if the distribution h is assumed to be a simple exponential distribution with expected value m, the correction factor would simply be 1 þ rm.In , another approach is used, namely estimating only on cases with a known final destiny (death or recovery) within [0, T ]. Let us denote by N those notified in [0, T ] and, among those, by R the number of cases who are observed as recovering and, as before, by D those that have died. Then, another application of equation (6.1) gives that the fraction R/N will be close to (1 2 p)r, where r is the integral part of equation (6.1) calculated with the probability density of the time from notification to recovery. As before, the fraction D/N will be close to pd, where d is the corresponding expression involving the probability density function of the time from notification to death, and thus the estimator D/(D þ R) will be close toThus, the estimate will be (approximately) unbiased only if r ¼ d. If r , d, then the CFR will be overestimated. This happens if the time to remission is stochastically larger than the time to death, which is the case for many diseases; for instance, this seems to be the case for Ebola (see §7.4). However, the reverse case, i.e. r . d, is also interesting, e.g. for influenza . Of course, this analysis only considers the theoretical effects of censoring and exponential growth, not other effects such as differential reporting, reporting delays or general underreporting.We now illustrate the numerical consequences of our findings based on realistic parameter values and model based formulae and simulations. Mostly, we have chosen parameter values compatible with the recent Ebola epidemic in West Africa as described in . Details about theoretical derivations and about the simulation program and related results are reported in the electronic supplementary material.We assume that the generation time follows a gamma distribution G G(a, l) with (a, l) ¼ (3, 0.2), implying a mean of 15 days and standard deviation 8.66 days, and that R 0 ¼ 1.7. This induces a true exponential growth rate r ¼ 0.0387 ( per day). The generation time when looking backwards in time will also follow a gamma distribution, but with different parameters and mean ¼ 12.6 days and standard deviation ¼ 7.26 days. If the true value r ¼ 0.0387 is used in equation (2.4) in conjunction with the backward generation time distribution, the result is R (B) 0 ¼ 1:57 as compared to the true value R 0 ¼ 1.7, an 8% underestimate. If the exponential growth rate is computed for this (backward) generation time distribution and the true R 0 , the induced exponential growth rate then equals r B ¼ 0.0462. Thus, the growth rate is overestimated by 19%.We illustrate the consequences of overestimating the variance of the generation time distribution by using serial intervals instead of generation time data in the simplified framework where both distributions are of the Gamma type and the difference is represented by the coefficient of variation of the serial interval distribution being larger than that of the generation time distribution by a factor c . 1, while the means are equal, as predicted by theory (see §4). If we assume the same basic parameter values as in the preceding subsection (i.e. the generation time follows a gamma distribution G G(a, l) with (a, l) ¼ (3, 0.2) and R 0 ¼ 1.7), and calculate the biases resulting from, for example, c ¼ 1.1, 1.2, 1.5 and 2, we find that the corresponding R 0 values, assuming the true r value is used in equation (2.4) , are underestimated by 0.9, 1.8, 4.8 and 9.6%, respectively, while the corresponding r values, assuming R 0 ¼ 1.7, are overestimated by 1.9, 4.1, 12.3 and 32.9%, respectively. Thus, sizeable bias can be obtained if the serial intervals are much more variable than the generation times.In , the generation time distribution was estimated from observed serial intervals, under the assumption that the distributions would be equal, assumed to be a consequence of the exact coincidence of onset of symptoms and beginning of infectious period. Under this assumption, there is of course no bias effect due to increased variance. However, it is interesting to note that the simulation results (see the electronic supplementary material) show that the serial intervals suffer the same distortion as generation times, due to exponential growth.In order to estimate the effects of basing the estimates of durations on individuals having only one exposure (see §5),  J. R. Soc. Interface 16: 20180670 some additional assumptions are needed. For the recent Ebola epidemic, it is found that the incubation period distribution f D , assumed to be equal to that of the latent period, is Gamma distributed with mean 11.4 days and s.d. ¼ 8.1 days and that generation times are Gamma distributed with mean 15.3 days and s.d. ¼ 9.3 days. It is also reported that approximately 25% of the contact traced individuals had one unique infector and 75% had more than one potential infector. We will use the above parameter values for this example. With the complete data, it would have been possible to estimate the contact rate l and the probability p to get infected by a close Ebola contact separately. Here we can only use that 25% had a single contact. We simply assume that p ¼ 0.5 and equate P(a single contact) ¼ p Thus, the mean incubation period for infectees with only one potential infector will be 11.4 2 8.1 ¼ 3.3 days shorter than the mean incubation period had all observations been used. This in turn implies that the mean generation time from the same data would be underestimated by 3.3 days, giving a mean of 12 instead of 15.3 days. Assuming that the standard deviation remains unchanged (¼ 9.3 days), the estimated generation time distribution would be Gamma distributed with mean 12 and s.d. ¼ 9.3 days. Using this generation time distribution, instead of the true one, in equation (2.4), assuming r ¼ 0.0383 to be known (e.g. estimated from the observed growth rate), leads to R 0single ¼ 1.50, an underestimation of 12%. Conversely, assuming R 0 ¼ 1.7, the exponential growth rate estimated from the contact traced individuals having only a single unique infector would approximately equal r single ¼ 0.0522, which overestimates the true value by 36%.Instead, to study the performance of maximum-likelihood estimation based on equation (5.1) and on an alternative set of moment estimators, we have simulated observations from 500 individuals (see the electronic supplementary material for details), showing that estimates of p and the parameters of the incubation period distribution seem reasonably unbiased, given the parameter setting and assuming the correct distributional form in the likelihood method. If the incubation period has a distribution differing from the assumed (gamma) model distribution (the log-normal distribution, in our simulation), the moment-estimators still perform well, but the maximum-likelihood estimates of mean and variance derived under the assumption of gamma distributed incubation times now acquire some bias. The speed of convergence of estimates and further properties under misspecification of assumptions need further study, but this initial experiment shows that unbiased estimation based on all observations is possible.The WHO Ebola Response Team report that the average time from symptoms to death is 5 þ 4 ¼ 9 days, while to remission the average time is 5 þ 12 ¼ 17 days.The numerical consequences of the results in §6 can be sizeable. If we assume that we have exponential growth with a doubling time of say 20 days, the growth rate r becomes 0.0347. Under the simplifying assumption that the time from notification to death follows an exponential distribution with mean m ¼ 9 days, the multiplier 1/(1 þ rm) becomes 0.76, i.e. there is an underestimation of about 24% of the CFR, using the simple estimator D/N. However, similarly the factor r will be 0.63 and, assuming a CFR of 70%, say, the estimator relying only on cases with a known final destiny will overestimate the CFR by approximately 5% of its value.In order to better study the behaviour of various observables during the early phase of an outbreak, we have conducted simulations of the basic epidemic model and evaluated various statistics. Parameter values were chosen to be similar to the recent Ebola epidemic in West Africa. Some of the results have already been commented upon in the preceding sections, but further results about the time it takes to reach predefined levels of notified cases, the stability of ratios between numbers of individuals in different disease stages, the relationship between generation times and serial intervals and predicting the size of the epidemic at a later time as well as details about the simulation and its parameters are presented in the electronic supplementary material.In this paper we have, by means of modelling, analysis and heuristics, both theoretical and simulation-based, studied inferential problems in an ongoing epidemic outbreak in its early stage. Our analyses give insights into where biases might 'hide' and also how to avoid these biases. We have studied three potential sources of bias: (1) backward estimation of generation times (contact tracing), (2) using serial intervals instead of generation times and (3) contact tracing leading to several potential infectors thus making the time of infection uncertain. Importantly, all three sources lead to biases in the same direction, causing the basic reproduction number R 0 to be underestimated if the epidemic growth rate r is correctly estimated. The converse is also true, namely that the growth rate will be overestimated if a correct estimate of R 0 is available, but this situation is likely to be less common in practice.The biases have been numerically evaluated, as illustrations, in a setting resembling the recent 2014 Ebola epidemic. To evaluate the combined biasing effect in this setting is a complicated exercise, depending on exactly how estimation is performed and presented but, assuming independence (i.e. multiplicative action), the total effect could be as large as, for example, 23% underestimation of R 0 (assuming a 50% increase in the coefficient of variation of the serial interval). In other settings, the effects could be quite different. In the main sections and in the corresponding sections of the electronic supplementary material, general formulae, if possible, or at least formulae under assumptions of Gamma distributions, if results depend on assumed distributions, are available and applicable in other parameter settings. It should be noted that even moderate errors in R 0 , especially if the true value is not very large, can have large  J. R. Soc. Interface 16: 20180670 consequences. As numerical illustration, assume that the true value of R 0 is 1.7, but that the estimate is negatively biased by 10% or even 23%, depending on how estimation is performed. Then the biased estimate would be around 1.53 or 1.31, respectively. Such a difference can have quite large consequences when planning control measures. For instance, the critical immunization level (both for vaccination and any other measure aimed at reducing infection) is usually calculated as v c ¼ 1 2 1/R 0 . For the true R 0 , this results in v c ¼ 41%, while the lower biased estimates yieldv c ¼ 35% or 24%, respectively. The underestimation of R 0 may hence lead to suggested preventive measures that are insufficient to stop the spread.The focus of the paper has been on studying potential biasing effects originating from a typical set of observables in the initial phase of an outbreak. However, there are also some positive observations. The stability of proportions of individuals in different phases of disease during the increasing phase is one, since quite good estimates of the total number of infected or not yet notified infected could be made, based on number of dead patients or notified ones, if good information about the related stage duration distributions is available. Another positive observation is that accurate inference in the multiple infector problem seems possible, although more research is needed. Finally, many biases can be understood and corrected for if the sampling situation is correctly modelled. It may be difficult to obtain simple analytical results, but simulation can then reveal the performance of various estimation procedures.In the paper, it was assumed that incidence was reported on aggregated level without additional information on household or spatial structure of the reported cases. If more detailed data were available, then more sophisticated models taking such heterogeneities into account may be used. This will improve the statistical analyses in that the model better describes the spreading patterns of the disease. The biases under focus in the present paper will however remain, but it is an open problem to study if they are reduced or increased when compared with the present homogeneous modelling assumptions. The results in this paper also indicate what other types of data would have been useful to avoid or decrease the discussed biases. For instance, more detailed observation of the joint distribution, or at least first and second order moments, of the various phases of the disease history (incubation, latency, infectiousness, etc.) would have improved the conversion of serial intervals to generation times; genotyping of the infectious agent in infected individuals may improve inference about infection chains and thus avoid the multiple infector problem and also allow direct estimation of R 0 in infection trees.Of course, there are also many other problems related to data from an emerging outbreak not treated in the current paper, important ones being underreporting, selective reporting and reporting delays, but also batch-reporting of numbers. A rather different type of potential source of bias, also not studied here, is when model assumptions are violated. For example, it has been assumed that there was no individual or society-induced changing behaviour during the data collection period, and social or spatial effects on spreading patterns have been ignored. Social structures have been shown to have limited effects for estimation in emerging epidemics , but spatial effects clearly play a role in disease spread, and their effect on parameter estimates is yet to be investigated. Changing behaviour probably kicks in early in emerging outbreaks of serious diseases like Ebola, and are hence also important to include in future inferential procedures for emerging epidemic outbreaks.Still, it is our hope that the results can help in improving future analyses of emerging outbreaks and the important efforts to guide health authorities in predictions and identifying possible preventive measures.Data accessibility. Details of proofs and simulation results are available in the file Supplementary Information.pdf, the code (in C) for outbreak simulation in the file simulation.c and the code for simulating multiple infectors data (in R) in the file multiple infectors.R, all available as electronic supplementary material to this paper.\n",
            "\t  0.5676177 \t Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations Yanan Cao 1 , Lin Li 1 , Zhimin Feng 1 , Shengqing Wan 1 , Peide Huang 1 , Xiaohui Sun 1 , Fang Wen 1 , Xuanlin Huang 1 , Guang Ning 1 and Weiqing Wang 1Dear Editor, The ACE2 gene encodes the angiotensin-converting enzyme-2, which has been proved to be the receptor for both the SARS-coronavirus (SARS-CoV) and the human respiratory coronavirus NL63. Recent studies and analyses indicate that ACE2 could be the host receptor for the novel coronavirus 2019-nCoV/SARS-CoV-2 1, 2 . Previous studies demonstrated the positive correlation of ACE2 expression and the infection of SARS-CoV in vitro 3, 4 . A number of ACE2 variants could reduce the association between ACE2 and S-protein in SARS-CoV or NL63 5 . Therefore, the expression level and expression pattern of human ACE2 in different tissues might be critical for the susceptibility, symptoms, and outcome of 2019-nCoV/SARS-CoV-2 infection. A recent single-cell RNA-sequencing (RNA-seq) analysis indicated that Asian males may have higher expression of ACE2 6 . Currently, the clinical reports of 2019-nCoV/SARS-CoV-2 infection from non-Asian populations for comparison are very limited. A study from Munich reported four German cases, all of which showed mild clinical symptoms without severe illness 7 . However, the genetic basis of ACE2 expression and function in different populations is still largely unknown. Therefore, genetic analysis of expression quantitative trait loci (eQTLs) 8 and potential functional coding variants in ACE2 among populations are required for further epidemiological investigations of 2019-nCoV/SARS-CoV-2 spreading in East Asian (EAS) and other populations.To systematically investigate the candidate functional coding variants in ACE2 and the allele frequency (AF) differences between populations, we analyzed all the 1700 variants (Supplementary Table S1) in ACE2 gene region from the ChinaMAP (China Metabolic Analytics Project, under reviewing) and 1KGP (1000 Genomes Project) 9 databases. The AFs of 62 variants located in the coding regions of ACE2 in ChinaMAP, 1KGP, and other largescale genome databases were summarized (Supplementary Table S2 ). All of the 32 variants potentially affecting the amino acid sequence of ACE2 in databases were shown (Fig. 1a) . Previous study showed that the residues near lysine 31, and tyrosine 41, 82-84, and 353-357 in human ACE2 were important for the binding of S-protein in coronavirus 5 . The mutations in these residues were not found in different populations in our study. Only a singleton truncating variant of ACE2 (Gln300X) was identified in the ChinaMAP (Fig. 1a) . These data suggested that there was a lack of natural resistant mutations for coronavirus S-protein binding in populations. The effects of low-frequency missense variants in populations for S-protein binding could be further investigated. The distributions of seven hotspot variants (Lys26Arg, Ile468Val, Ala627Val, Asn638Ser, Ser692Pro, Asn720Asp, and Leu731Ile/Leu731Phe) in different populations were shown (Fig. 1b) . Six low-frequency loci (rs200180615, rs140473595, rs199951323, rs147311723, rs149039346, and rs73635825) were found to be specific in 1KGP database, the AFs of which were also low in the gnomAD and TopMed 10 database. Only two of these six variants (rs200180615 and rs140473595) could be found in CHB (Han Chinese in Beijing) population with the AF < 0.01.Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit . Table S2 ). Taken together, the differences in AFs of ACE2 coding variants among different populations suggested that the diverse genetic basis might affect ACE2 functions among populations.To analyze the distribution of eQTLs for ACE2, we used the Genotype Tissue Expression (GTEx) database (https:// ). We found 15 unique eQTL variants (14 SNPs and 1 INDELs) for ACE2 with q value ≤ 0.05 in 20 tissues from the GTEx database (rs112171234, rs12010448, rs143695310, rs1996225, rs200781818, rs2158082, rs4060, rs4646127, rs4830974, rs4830983, rs5936011, rs5936029, rs6629110, rs6632704, and rs75979613). The AFs of the 15 eQTL variants were compared among different populations. Notably, our results showed most of the 15 eQTL variants had much higher AFs in the ChinaMAP dataset and EAS populations compared to European populations ( Fig. 1c and Supplementary Table S3 ). The AFs of the top 6 common variants (rs4646127, rs2158082, rs5936011, rs6629110, rs4830983, and rs5936029) were higher than 95% in EAS populations, whereas the AFs of these variants in European populations were much lower (52%-65%). All of the 11 common variants (AF > 0.05) and 1 rare variant (rs143695310) in the 15 eQTLs are associated with high expression of ACE2 in tissues (Supplementary Table S3 ). For instance, the eQTL variant rs4646127 (log allelic fold change = 0.314), which locates in the intron of ACE2 gene, has the highest AFs in both of the ChinaMAP (0.997) and EAS (0.994) populations. Comparatively, the AFs of rs4646127 in EUR (0.651) and AMR (0.754) populations are much lower. These findings suggested the genotypes of ACE2 gene polymorphism may be associated higher expression levels of ACE2 in EAS population.Recent reports of the ACE2 expression analysis in lung tissues from Asian and Caucasian populations are still controversial. The single-cell RNA-seq analysis reported that the Asian donor had much higher ACE2 expression cell ratio than white and African-American donors 6 . In contrast, the ACE2 expression analysis using the RNA-seq and microarray datasets from control lung tissues indicated there were no significant differences between Asian and Caucasian, or male and female 11 . The ACE2expressing cells are a very small part of cells in lung tissues 6 . The sample size and the purity of ACE2-positive cells in the selected samples would influence the conclusions. Our analysis showed the differences in distribution and AFs of eQTLs for ACE2 in different populations, indicating the diversity of ACE2 expression pattern in populations (Supplementary Table S3 ). Largescale and multiple tissue-level analysis of single-cell RNAseq would be more accurate for the expression analysis of ACE2 in different populations. In addition, our data showed the moderate difference in AFs of eQTLs between South Asian and EAS, which suggests the potential difference of ACE2 expression in different populations and ethnics in Asia (Fig. 1c) .In summary, we systematically analyzed coding-region variants in ACE2 and the eQTL variants, which may affect the expression of ACE2 using the GTEx database to compare the genomic characteristics of ACE2 among different populations. Our findings indicated that no direct evidence was identified genetically supporting the existence of coronavirus S-protein binding-resistant ACE2 mutants in different populations (Fig. 1a) . The data of variant distribution and AFs may contribute to the further investigations of ACE2, including its roles in acute lung injury and lung function 12 . The East Asian populations have much higher AFs in the eQTL variants associated with higher ACE2 expression in tissues (Fig. 1c) , which may suggest different susceptibility or response to 2019-nCoV/SARS-CoV-2 from different populations under the similar conditions.\n",
            "\n",
            " Title:  Tick-borne encephalitis virus inhibits rRNA synthesis and host protein production in human cells of neural origin\n",
            "3 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.61909384 \t CamF GATATAGATTGAAAAGTGGAT Overlaps between primers vacACamR and CamF, primers vacACamF and CamR, primers ureCamF2 and CamR, primers ureCamR1 and CamF are underlined.\n",
            "\t  0.6109336 \t Heger A, Holm L. 2001 \n",
            "\t  0.60011446 \t Camberwell ------------------------------------T-----------F ----G------------------------P ----T--------A-T-N-- OXF.B5S8.02.UK -------------------------------------T-----------F ----G------------------------P ----T--------A-T-N-- OXF.B5S9.02.UK -------------------------------------T-----------F ----G------------------------P ----T--------A-T-N-- OXF.B5S10.02.UK -------------------------------------T-----------F ----G------------------------P ----T--------A-T-N-- GII.4.02a -------------------------S-----------T-----------F ----G------------------------P ----T--------A-T-N-- FHills.02.US -------------------------------------T-----------F ----G------------------------P ----T--------A-T-N-- Carlow.02.Ire -------------------------------------T-----------F ----G------------------------P ----T--------A-T-N-- OXF.B5S1.202.UK -------------------------------------T-----------F ----G------------------------P ----A--------A-T-N-- OXF.B4S5.02.UK -------------------------------------T-----------F ----G------------------------P ----A--------A-T-N-- OXF.B4S6.02.UK -------------------------------------T-----------F ----G------------------------P ----A--------A-T-N-- OXF.----E --------------------------Y-------------------------N.N----- DenHaag015.00.NL --------N-----------------------M------E --------------------------Y-------------------------N.N----- Dresden153.97.GE --------N-----------------------M------E --------------------------Y-------------------------N.N----- 358.96015107.96.FL.US --------N-----------------------M------E --------------------------Y-------------------------N.N----- WDVN016.00.NL --------N-----------------------M------E --------------------------Y-------------------------N.N----- Beeskow.124.00.DE --------N-----------------------M------E --------------------------Y-------------------------N.N----- BWLND.331.95.US --------N-----------------------M------E --------------------------Y-------------------------N.N----- Berlin.159.98.DE --------N-----------------------M------E --------------------R-----Y-------------------------N.N----- Osaka.98.JPN --------N-----------------------M------E --------------------------Y-------------------------N.N----- Berlin.238.98.DE --------N-----------------------M------E --------------------------Y-------------------------N.N----- Hamburg189.97.GE --------N-----------------------M------E --------------------------Y-------------------------N.N----------E------------V---NGT----- Langen1061.02.DE --------N--G----------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- CS_D1.02.CAN --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- CS_G4.02.US --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- Germanton.02.US --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- CS_G102.US --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- OXF.B5S7.02.UK --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- OXF.B5S8.02.UK --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- OXF.B5S9.02.UK --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- OXF.B5S10.02.UK --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---NGT----- GII.4.02a --------N-------------------R---M------G -----G--------D---------I--N-------E------------V---\n",
            "\t  0.5660083 \t Este protocolo é propriedade dos pesquisadores aqui citados e contém informações confidenciais. O uso de todo o conteúdo é restrito, destinando-se a orientar os pesquisadores, suas equipes e colaboradores nos procedimentos a serem seguidos durante o estudo. Esse material não deve ser revelado ou usado por outros, exceto com autorização dos pesquisadores.Todas as perguntas referentes a este protocolo devem ser enviadas por e-mail para Beatriz Grinsztejn e Luiz Antonio Bastos Camacho Em geral, a resposta deve ser recebida em 24 horas (de segunda a sexta-feira)..Gerenciamento clínico Para questões referentes a critérios de inclusão no estudo, conduta em casos de efeitos tóxicos, medicamentos concomitantes e co-recrutamento entrar em contato com Beatriz Grinsztejn Valdiléa Gonçalves Veloso dos Santos Marilia Santini de Oliveira e Maria Isabel Fragoso da Silveira Gouvêa Favor incluir no assunto da mensagem o título do protocolo, o número de identificação do paciente e um pequeno resumo relevante da história.Para as questões relacionadas especificamente aos exames laboratoriais imunológicos, entrar em contato com os imunologistas do protocolo. Enviar uma mensagem de e-mail para Carmem Beatriz Wagner Giacoia Gripp e para Mariza Gonçalves Morgado Para as questões relacionadas especificamente à sorologia e aos exames laboratoriais de identificação viral, entrar em contato com o imunologista do protocolo. Enviar uma mensagem de e-mail para Marilda Siqueira Para as questões relacionadas especificamente aos exames laboratoriais de segurança, entrar em contato com Marilia Santini de Oliveira e Maria Isabel Fragoso da Silveira Gouvea participante permanecerá no estudo por cerca de 12 meses. A duração total do estudo será de 16 meses, considerando um período de inclusão de 4 meses.Serão incluídos no estudo pacientes infectados pelo HIV, estratificados de acordo com a contagem de CD4 (< 200 cels/mm3 ou > 200 cels/mm3) no momento da triagem para o estudo, que estejam sem receber tratamento anti-retroviral ou em tratamento estável há no mínimo 8 semanas, sem planos de mudança nos próximos 6 meses, com indicação de receber vacina contra influenza A H1N1. O grupo controle será formado por indivíduos HIV negativos, status confirmado por sorologia realizada no momento da triagem, com indicação de receber vacina contra influenza A H1N1.Os pacientes infectados pelo HIV receberão um de dois possíveis esquemas vacinais: 1) 3,75 µg em duas aplicações, com 21 dias de intervalo; 2) 7,5 µg em duas aplicações, com 21 dias de intervalo. Os voluntários do grupo controle receberão uma única aplicação da dose de 3,75 µg da vacina. Adicionalmente os participantes do estudo receberão vacina contra influenza sazonal no dia da segunda dose da vacina contra A H1N1 (portadores do HIV) ou 21 dias após a dose única da vacina contra A H1N1 (grupo controle).Considerando um estudo de não-inferioridade com diferença aceitável de 10% e uma soroconversão de 95% foi determinado um tamanho de amostra de 82 pessoas por grupo, ampliado para 90 para lidar com perdas no processo, tendo em vista que os pacientes portadores do HIV serão selecionados aleatoriamente a partir do conjunto de pacientes em tratamento no IPEC. nas respostas de anticorpos. 1.3.5 Avaliar os efeitos da carga viral inicial de HIV-1 na resposta de anticorpos. 1.3.4 Determinar as respostas imunológicas celulares e suas correlações ao desenvolvimento e à amplitude das respostas ao H1N1 e comparar as respostas imunológicas celulares. 1.3.5 Avaliar os efeitos da vacinação nos níveis de CD4 e carga viral do HIV 1.3.7 Avaliar os efeitos de HAART na resposta imune à vacina. 1.3.8 Avaliar os efeitos da vacina contra influenza sazonal na tolerabilidade e na resposta imune à vacina anti-H1N1 1.3.9 Determinar as respostas imunológicas celulares e suas correlações ao desenvolvimento e à amplitude das respostas ao H1N1 e comparar as respostas imunológicas celulares. 1.3.10 Avaliar o efeito de imunidade pré-existente para H1N1 nas respostas sorológicas a essa vacina 1.3.11 Avaliar o numero de casos de síndrome gripal influenza like entre os pacientes após a vacinação inicial 1.3.12 Caracterizar os tipos de influenza causando síndrome gripal influenza like na coorte em estudo 1.3.13 Analisar a custo-efetividade das diferentes estratégias de vacinaçãoA disseminação dos vírus de influenza A de origem animal em populações humanas [incluindo os subtipos A/H5N1, H7N7, H9N2 e, mais recentemente, o vírus da influenza H1N1 de origem suína (S-OIV), também designado \"Influenza A de 2009 (H1N1)\" neste texto] adicionou urgência aos esforços em andamento para desenvolver planos de resposta a potenciais pandemias de influenza (Subbarao et al, 1998; Peiris et al, 1999; Dawood et al, 2009; ).Houve três pandemias no século passado. Durante a pandemia de influenza A/H1N1 em 1918, estima-se que 40 milhões de mortes ocorreram em todo o mundo (Oxford, 2000; . Foram observados excesso de mortalidade, morbidade e perturbações sociais durante as pandemias de influenza A/H2N2 em 1957 e de influenza A/H3N2 em 1968 (Patriarca & Cox, 1997 (Oxford, 2000; (Couch & Kasel, 1983; Ennis et al, 1977) .A magnitude da resposta imunológica à vacina inativada sem adjuvantes contra a influenza é grandemente afetada pela dosagem dos antígenos (geralmente expressa em peso de HA) contidos na vacina. Estudos avaliando o efeito da dose de HA nas respostas imunológicas às vacinas inativadas sem adjuvantes contra a influenza sazonal realizados nos últimos 35 anos demonstraram aumentos relacionados com a dose do soro e das respostas de anticorpos nas mucosas (Ennis al, 1977; Ruben &, Jackson, 1972; Ruben et al, 1975; Matzkin & Nili, 1984; Palache et al, 1993; Gross et al, 1988; Keitel et al, 1994; Keitel et al, 1996; Keitel et al, 2008 Os pacientes infectados pelo HIV-1 não apenas sofrem a morbidade típica da influenza sazonal e a potencial interferência da doença na resposta à terapia antirretroviral, mas também a possibilidade de a influenza ser mais grave que nas pessoas não infectadas da mesma faixa etária Madhi et al, 2002 Em pacientes que não têm doença progressiva pelo HIV-1 e/ou estão recebendo TARV altamente ativa (HAART), estas respostas são melhores (King et al, 2000; Levin et al, 2008; Vigano et al, 2008) .Os estudos que avaliaram a segurança e a eficácia da vacina contra influenza sazonal mostram que a administração do produto é segura nessa população, porém os resultados de eficácia são contraditórios. Vários estudos clínicos mostraram que as respostas de anticorpos a TIV são diminuídas em crianças e adultos infectados pelo HIV-1 (Staprans et al, 1995; Amendola et al, 2001; Zanetti et al, 2002; Brydak et al, 1999; Kroon et al, 1998; Nelson et al, 1988; Kroon et al, 1998) , especialmente nos indivíduos que não estavam em uso de terapia antirretrovial (Kroon et al, 1994; Miotti et al,1989; Chadwick et al, 1994; Fowke et al, 1997; Nelson et al, 1988; Kroon et al, 1998; Brydak et al, 1999) ), porém outros estudos não obtiveram os mesmos resultados (Tasker et al, 1999; Amendola et al, 2001; Zanetti et al, 2002 Os 10 primeiros voluntários incluídos em cada um dos cinco sub-grupos (dois estratos de CD4, dois diferentes esquemas vacinais e indivíduos HIV negativos) colherão sangue para a avaliação de resposta celular à vacina.Serão incluídos no estudo 360 pacientes infectados pelo HIV e 90 voluntários saudáveis, ambos os grupos com idade entre 18 e 59 anos. Esse estudo só será iniciado após aprovação, pelo Comitê de Ética em Pesquisa do IPEC, desse protocolo, assim como de todo material de divulgação a ser utilizado para o recrutamento de sujeitos.A1. Ambos os sexos, com idade entre 18 e 59 anos;A2. Capacidade de dar consentimento voluntariamente para a participação no estudo;A3. As participantes do estudo com potencial de reproduzir (definido como meninas após a menarca ou mulheres que não chegaram à menopausa pelo menos durante 24 meses consecutivos, ou seja, que menstruaram nos últimos 24 meses, ou não foram submetidas a esterilização cirúrgica) precisam ter resultado negativo ao exame de gravidez, sérico ou urinário, no momento da triagem para o estudo e nos dias de aplicação de vacina. Caso pratiquem atividades sexuais passíveis de promover gestação, as participantes precisam usar algum tipo de contracepção durante o período do estudo. Pelo menos um dos seguintes métodos TEM DE ser usado de modo adequado:• Preservativos (masculinos ou femininos)• Anticoncepcionais hormonaisAs participantes do estudo sem potencial de reproduzir [definido como meninas antes da menarca ou mulheres após a menopausa pelo menos durante 24 meses consecutivos ou mulheres submetidas a esterilização cirúrgica (p. ex., laqueadura tubária bilateral, procedimento Essure® ou qualquer procedimento contraceptivo permanente não-incisional, ooforectomia bilateral ou salpingectomia bilateral)] são elegíveis sem exigência de uso de contracepção. O relato da participante constitui documento aceitável de esterilização, menopausa e potencial reprodutivo das mulheres.A4. Resultados laboratoriais no momento da triagem para o estudo:• Hemoglobina > 8,0 g/dL;• Bilirrubina direta < 2,5 x o limite normal superior;• Aminotransferase alanina, ALT (TGP) e aminotransferase aspartato, AST (TGO) < 3 x o limite superior de normalidade; • Contagem de plaquetas > 100.000/mm 3B1. Infecção pelo HIV-1 (comprovada por teste rápido do HIV ou kit de ELISA aprovado e confirmado pela repetição do ELISA, IFI, Western blot, carga viral plasmática do HIV-1) a qualquer momento antes do ingresso no estudo.A confirmação do resultado do teste inicial deve usar um teste diferente do utilizado para a avaliação inicial. Um teste rápido inicial reagente deve ser confirmado por outro teste rápido ou por um teste ELISA baseado em uma diferente preparação de antígenos e/ou teste com diferente princípio (p.ex., indireto vs. competitivo), ou um Western blot ou carga viral plasmática do HIV-1. Um resultado inicial de ELISA tem de ser confirmado por outro método como teste rápido, Western blot ou carga viral plasmática de HIV-1, mas não pela repetição do próprio ELISA B2. Sem modificação da situação de tratamento antirretroviral nas 8 semanas anteriores ao momento da triagem para o estudo:• Se o voluntário estiver em uso de HAART é necessário que esteja usando o mesmo esquema durante pelo menos 8 semanas antes da triagem. Modificações de posologia dos antiretrovirais nas 8 semanas anteriores ao ingresso no estudo são permitidas. Além disso, é permitida a troca de apresentação farmacológica (p. ex., da apresentação convencional para apresentação em associações). • Se o voluntário não estiver em tratamento antirretroviral regular não deve ter recebido nenhuma dose de nenhum antirretroviral nas 8 semanas que antecedem a triagem, incluindo profilaxia de transmissão vertical do HIV (esquemas profiláticos e terapêuticos prévios são permitidos)B3. Sem planejamento de trocar ou iniciar HAART nos próximos 6 meses. O centro também precisa consultar a bula mais recente do produto do estudo para ter acesso a informações atuais sobre os medicamentos proibidos ou que exijam precauções.6.1 Cronograma de eventos O cronograma de eventos do estudo, a ser seguido para cada voluntário incluído, está descrito no anexo 1 deste protocolo de pesquisa (fluxograma de procedimentos). A descrição detalhada dos procedimentos a serem realizados em cada consulta do estudo está registrada no item 6.2 deste protocolo, assim como a janela permitida entre cada consulta.As consultas interinas para avaliação clínica e coleta de secreção respiratória na vigência de quadros respiratórios agudos serão realizadas a qualquer momento a partir da primeira dose da vacina (dia zero) e não constam do fluxograma de procedimentos (anexo 1).Momentos das avaliações A seguir estão descritas as consultas do estudo e quais procedimentos devem ser realizados em cada uma delas. • Coleta de sangue, processamento e armazenagem para sorologia (H1N1) • Coleta de sangue, processamento e armazenagem para ELI-Spot6.2.9 Consultas de oito semanas (2 meses) após esquema vacinal completo ou após primeira dose contra H1N1: dia 57 (todos) e dia 77 (apenas HIV positivos) Essa consulta deve ser realizada 8 semanas após a aplicação do esquema vacinal completo, idealmente no mesmo dia da semana (por exemplo, na terceira segunda-feira seguinte à primeira dose para os voluntários inicialmente vacinados numa segunda-feira), sendo permitida uma janela de mais ou menos 7 dias. Os seguintes procedimentos devem ser realizados nessa consulta: O sangue para realização da sorologia será coletado nas consultas especificadas na seção 6.2 deste protocolo. Os exames serão realizados ao final do estudo, em bloco.6.3.9 Avaliação da resposta imune celular à vacina anti-H1N1 A resposta imune celular dos indivíduos infectados pelo HIV-1 e dos voluntários saudáveis submetidos ao presente protocolo vacinal será avaliada através do método de ELISpot. Este método consiste na análise indireta da resposta imune funcional de linfócitos T frente a um antígeno específico, através da identificação e enumeração de células produtoras de citocinas, classicamente de interferon-gama (IFN-γ), onde cada spot desenvolvido representa uma única célula responsiva. Para isto, deverão ser coletados cerca de 30ml de sangue venoso, frente ao anticoagulante heparina, para a obtenção das células mononucleares periféricas, através da separação em ficoll, por gradiente de densidade. As células serão, então, contadas e submetidas à criopreservação em soro fetal bovino 10% DMSO, na concentração de 1x10 7 /ml e estocadas em nitrogênio líquido. Em momento oportuno, as células de cada participante, referentes aos dias zero, 21 e 42 serão descongeladas e avaliadas para a produção dos spots frente ao antígeno vacinal e frente à proteína p24 do HIV-1, no caso dos indivíduos infectados, ou ao PPD, para os voluntários saudáveis. O ELISpot será realizado em placas de filtração de 96 poços, cobertas com anticorpos monoclonais de captura anti-IFN-γ (Diaclone, França), na concentração de 1x10 5 células/poço, em duplicadas para cada estímulo e para a ausência de estímulo. As células serão mantidas em estufa por 40 horas e a secreção de IFN-γ detectada através de anticorpo anti-IFN-γ biotinilado (Diaclone, França). Para a revelação dos spots será utilizado o sistema estreptavidina-fosfatase alcalina-NBT-BCIP. Respostas serão consideradas positivas quando o número de células formadoras de spot (CSF) /10 6 células mononucleares periféricas for igual ou superior a 50, após subtração das culturas sem estímulo. O sangue para realização desse teste será coletado nas consultas especificadas na seção 6.2 deste protocolo. Os exames serão realizados ao final do estudo, em bloco. As amostras positivas para Influenza, na etapa anterior, serão submetidas a RT-PCR, utilizando a enzima Superscript III com Taq platinum e iniciadores específicos para amplificar os segmentos genômicos 4 (HA) e 6 (NA) dos vírus influenza A e B Após purificação e quantificação dos produtos de PCR, a reação de sequenciamento será realizada utilizando o kit \"Big Dye Terminator Cycle Sequencing Ready Reaction\" (Applied Biosystems). Os produtos serão seqüenciados em ambas as direções e analisados no sequenciador automático ABI PRISM TM 3130XLGenetic Analyzer (Applied Biosystems).As seqüências serão alinhadas utilizado o Programa Clustal X e, para as análises filogenéticas, utilizaremos o programa MEGA 4.1 (Tamura e cols., 2007). As árvores filogenéticas serão reconstruídas usando o método de junção de vizinhos (\"Neighbour-Joining\")(Saitou e Nei, 1987) e as distâncias evolucionárias serão inferidas pelo método de Kimura 2-parâmetros (Kimura, 1980) -ou outro método que melhor se adeqüe aos dados -, incluindo análise de reamostragem (\"bootstrap\", mínimo de 1000 replicatas). A análise de custo-efetividade a ser conduzida tem como objetivo avaliar comparativamente os benefícios e os custos de diferentes estratégias de vacinação. As estratégias de vacinação sendo comparadas são a vacinação dos pacientes infectados com o HIV com diferentes dosagens. Os benefícios serão medidos em número de casos prevenidos e anos de vida ganhos. Os custos a serem considerados incluem os custos médicos diretos derivados da vacinação assim como os custos médicos não gastos devido aos casos prevenidos. O resultado da análise de custo-efetividade será expresso em termos do custo incremental por unidade de benefício ganho.Os custos e os benefícios determinados pela vacinação são influenciados por diversos fatores que incluem: a efetividade da vacina, o número de casos esperados, o uso de recursos médicos, os custos da vacinação e do cuidado médico. No caso de uma epidemia por H1N1, que supostamente afetará em determinado momento do tempo, assume-se razoável a análise de custoefetividade para aquela epidemia. Assim, o horizonte analítico, ou seja, o período de tempo para o qual os benefícios e custos serão computados, assumido é de uma epidemia. A perspectiva adotada será a do Serviço Único de Saúde (SUS). O software TreeAge Pro Suite será utilizado para conduzir a análise.Os tipos, intensidade e freqüência das reações adversas locais e de hipersensibilidade esperadas estão listados no anexo III deste protocolo. Não há recomendações específicas para o tratamento das reações adversas locais ou sistêmicas ao produto. O manejo dessas reações deve ser feito a critério ds equipe médica do estudo, com o uso de medicações sintomáticas e procedimentos paliativos e/ou de suporte. A vacinação será feita no IPEC, onde estão disponíveis equipamentos, medicamentos e profissionais qualificados para o pronto-atendimento a reações anafiláticas.Ao longo do estudo (que começa quando o TCLE é assinado), os pesquisadores monitorarão cada paciente para o desenvolvimento de qualquer evidência clínica e/ou laboratorial para identificação de um Evento Adverso (AE). Um evento adverso é definido como qualquer ocorrência médica indesejável em um paciente participante da pesquisa e inclui eventos/ experiências mesmo sem relação causal com as vacinas do estudo. Antes da administração das vacinas de estudo, apenas os eventos adversos graves e os eventos adversos que o pesquisador considerar como estando relacionados ao desenho do estudo e/ou procedimentos serão capturados na ficha clínica de estudo.Um evento adverso pode ser um sintoma, sinal, ou achado de laboratório anormal. Qualquer piora de uma condição pré-existente ou doença intercorrente será relatada como um evento adverso. Uma anormalidade de laboratório será informada como um evento adverso quando for necessária uma conduta (p.ex., interrupção ou descontinuação da droga do estudo, ou necessidade de tratamento específico). A natureza do sinal ou sintoma, sua data e horário de aparecimento, duração e severidade, terapia empregada (se alguma) e a opinião do pesquisador dobre a relação de causalidade com a droga do estudo, com uma etiologia alternativa, se apropriado, será documentada.Os Mulheres grávidas no momento da triagem e na consulta de dia zero não serão incluídas neste estudo. Mulheres com teste positivo para gravidez no dia da aplicação da segunda dose de Pandemrix® e ou da vacina contra influenza sazonal não receberão o produto, porém permanecerão no estudo para análise de segurança e de imunogenicidade, cumprindo os demais procedimentos do estudo.Voluntários com eventos adversos atribuídos à vacina de grau maior ou igual a 3, ou que desenvolvam intercorrências clínicas relevantes, não receberão a segunda dose da vacina, porém permanecerão no estudo para análise de segurança e de imunogenicidade, cumprindo os demais procedimentos previstos neste protocolo.Os voluntários que se apresentarem para a segunda dose da vacina após o prazo determinado não receberão essa dose, mas permanecerão no estudo para análise de segurança e de imunogenicidade, cumprindo os demais procedimentos previstos neste protocolo.Qualquer paciente participante desse estudo poderá retirar seu consentimento, a qualquer momento e por qualquer motivo, sem que isso implique em prejuízo para seu tratamento nas instituições envolvidas. Os pesquisadores poderão retirar qualquer indivíduo do estudo, por motivos que julgaram pertinentes (toxicidade, falta de adesão, etc).Esse estudo pode ser prematuramente interrompido pelos pesquisadores, pelo CEP do IPEC ou pelos órgãos regulatórios brasileiros, caso seja demonstrado risco para a segurança dos sujeitos da pesquisa. Para tanto, será necessário aplicar um procedimento seqüencial de recrutamento, principalmente quando os grupos estiverem próximos de atingir o tamanho de sua amostra. Este procedimento sequencial é conhecido como amostragem inversa (Haldane, 1945; Vasconcellos, Silva & Szwarcwald, 2005) . Na realidade, esse é um procedimento seqüencial derivado de uma ordenação aleatória prévia, que termina quando o número de participantes do grupo (90, no caso) for atingido.Para o grupo de controle, serão selecionados os primeiros 90 candidatos que aceitarem participar do estudo e cumprirem os demais critérios de elegibilidade.A grande vantagem desse processo de seleção de candidatos reside no fato de que serão conhecidas as probabilidade de inclusão no estudo dos participantes e, portanto, podem ser estimadas prevalências para combinação de estratos, que podem ser testadas com as do grupo de controle.9.5 Monitoramento Os pesquisadores responsáveis, a equipe de gerenciamento de dados e os estatísticos do estudo avaliarão semanalmente as listas de randomização e a lista de portadores do HIV previamente selecionados para participação no estudo, de acordo com os estratos de CD4, a fim de monitorar a velocidade de inclusão e de preenchimento dos subgrupos do estudo. Serão analisados também o número de participantes que consentiu em participar do estudo e o número de sujeitos que mudaram de estrato devido à contagem de CD4 no momento da triagem para o estudo, além do número de exclusões por critérios de elegibilidade. Essa análise freqüente permitirá eventuais correções nas listas de seleção, mantendo a aleatoriedade do procedimento de seleção.Será feita uma análise demográfica de cada grupo e dos quatro grupos de portadores do HIV, sendo calculadas, por sexo, as médias e desvios padrão de idade, que serão comparados aos mesmos parâmetros do conjunto de pacientes HIV positivos em tratamento no IPEC. Os grupos randomizados e o grupo de voluntários HIV-negativos terão suas características pré-vacinais (sócio-demográficas, clínicas e laboratoriais) comparadas, para demonstrar o sucesso da randomização e a comparabilidade do controle não randomizado. O conjunto de pacientes randomizados poderá ser comparado em algumas características selecionadas com o \"universo\" dos pacientes do IPEC, como um acessório para analisar a validade externa dos resultados.A não-inferioridade da vacina contra gripe A H1N1 em portadores do HIV, em relação ao grupo de controle, será avaliada a partir da proporção de soroproteção obtida com as sorologias nos dias indicados no item 6.2 deste protocolo.Os intervalos de 95% de confiança das diferenças entre as proporções de soroproteção entre os grupos de portadores do HIV e o grupo de controle serão construídos e a não-inferioridade será definida como limite inferior do intervalo de confiança de 95% para a diferença na proporção de soroproteção entre os grupos de HIV positivos e o de controle de -10% (por exemplo, -8%, -5% etc.).De forma análoga, será testada a não-inferioridade entre os dois esquemas vacinais propostos para cada estrato de contagem de células CD4.Como os estratos podem ser grupados e a proporção de soroproteção estimada para esses grupamentos, testes de não-inferioridade podem ser conduzidos para estratos agrupados e para o conjunto de participantes HIV positivos.Similarmente, as razões das médias geométricas dos títulos de anticorpos nos grupos HIV positivo e no grupo de referência e seus respectivos intervalos de 95% de confiança serão estimados. Não-inferioridade será definida como limite inferior da razão de títulos médios geométricos de anticorpos contra gripe A H1N1 ao final do esquema completo em cada par de grupos em comparação, superior a 0,5 (por exemplo, 0,6, 0,7 etc.).A análise de segurança será realizada através da determinação, por dose, da proporção de sujeitos com um relato de qualquer sintoma durante a avaliação de segurança (item 6.2). Para cada grupo e dose, a incidência de cada sintoma observado será apurada. Serão tabuladas as proporções de eventos adversos por dose e por grupo, que serão comparadas através de teste de qui-quadrado com nível de significância de 0,01.A duração da resposta humoral à vacina será avaliada através da comparação da freqüência de sujeitos com títulos protetores de anticorpos em cada momento em que a sorologia for realizada, entre 2 e 12 meses após a vacinação A dispensação dos produtos do estudo deve ser feita diretamente ao profissional de saúde pertencente a equipe do estudo que irá realizar a administração no paciente. A dispensação só poderá ser feita mediante prescrição médica, assinada por um prescritor autorizado pelo investigador principal. O farmacêutico deve assegurar que o termo de consentimento livre e esclarecido foi assinado antes de dispensar os produtos do estudo. Todo procedimento de dispensação deve ser documentado no prontuário do paciente.Para cada paciente que assinar o TCLE e se submeter aos procedimentos de seleção para este estudo, será completada uma ficha clínica de estudo, especificamente desenvolvida para esse estudo. Isto se aplica também aos pacientes que se submeterem aos procedimentos de seleção mas não foram admitidos e aos pacientes que forem descontinuados antes da conclusão do estudo. A razão pela qual um paciente não foi admitido ou a razão para descontinuação será descrita no documento fonte e na FCE.Todas as fichas clínicas de estudo serão completadas com uma caneta esferográfica de cor escura, em letra legível. As correções serão feitas traçando uma única linha na entrada incorreta e escrevendo a entrada correta. Todas as correções serão datadas e rubricadas.Qualquer referência a pacientes será feita usando-se suas iniciais e números específicos do protocolo, nunca pelos nomes.O pesquisador responsável pela pesquisa ou um profissional da equipe do estudo revisará as fichas clínicas de estudo para verificar a precisão dos dados coletados. • se os procedimentos foram realizados e registrados adequadamente -verificação de 20% dos prontuários e FCE • se os TCLE foram adequadamente assinados -revisão de 100% dos termos; • se os eventos adversos graves foram relatados no prazo previsto e de forma completa e acurada -revisão de 100% dos relatos; • se os documentos regulatórios foram gerados e arquivados de forma correta -revisão de 100% dos arquivos do pesquisador Todos os registros do estudo estarão disponíveis para a equipe de pesquisadores, o CEP do IPEC, a CONEP e a ANVISA.11.4 Eventos adversos de notificação imediata para o grupo coordenador médico do estudo Todos os eventos adversos graves deverão ser comunicados em até 24 horas após o conhecimento para o grupo coordenador médico do estudo, através de formulário específico de notificação enviado por e-mail ao grupo. Também deverão ser notificados no mesmo formulário os eventos adversos locais e sistêmicos da vacina de grau maior ou igual a 3.Este protocolo só será iniciado após avaliação e aprovação pelo CEP do IPEC.Será considerada responsabilidade dos pesquisadores assegurar que a cada paciente foi dada uma explicação adequada dos objetivos, métodos, benefícios antecipados e riscos potenciais deste estudo e que cada participante voluntariamente assine e date o TCLE aprovado pelo CEP antes da participação no estudo (anexo III). O pesquisador também deverá explicar que os pacientes têm o direito de se recusar a participar no estudo ou de se retirar a qualquer momento por qualquer razão.Será documentado no registro médico da paciente que o consentimento informado foi obtido antes da realização de qualquer procedimento relacionado ao estudo e que uma via original do TCLE foi mantida com os registros do estudo. Uma via do TCLE assinado e datado será oferecida aos pacientes.Os pesquisadores manterão um diário de seleção/ admissão e informação de identificação completa sobre cada paciente a ser usado com a finalidade de seguimento a longo prazo, se necessário. Este documento será mantido pelos pesquisadores em sigilo estrito.As fichas clínicas de estudo serão identificadas apenas com o número do paciente no estudo. Qualquer divulgação dos resultados desta pesquisa não conterá nomes, fotos ou outras formas de identificação dos sujeitos da pesquisa.O estudo pode ser terminado a qualquer momento pelo CEP, a CONEP, a ANVISA ou outros órgãos do governo como parte de seus deveres de assegurar a proteção dos participantes de pesquisas.Os dados obtidos com esse estudo são de propriedade dos pesquisadores responsáveis. A publicação dos resultados das análises será feita, independente de serem favoráveis ou não, após acordo entre os pesquisadores sobre autoria. Esse estudo não envolve cooperação estrangeira.Todos os procedimentos de manipulação das amostras biológicas dos indivíduos infectados pelo HIV-1 e pelos voluntários saudáveis seguirão as recomendações de boas práticas estabelecidas pela CIBio -IOC/FIOCRUZ. As amostras biológicas obtidas para os exames de acompanhamento e de avaliação imunológica deste protocolo, tanto dos indivíduos infectados pelo HIV-1 como dos voluntários saudáveis, serão avaliados nos laboratórios de segurança biológica de nível 3 (NB3) do IOC ou no laboratório de segurança biológica de nível 2 (NB2) do Laboratório de AIDS e Imunologia Molécula -IOC. Todo o material consumível utilizado em todas as etapas deste protocolo será devidamente autoclavado e descartado segundo as normas de biossegurança regidas na Fiocruz.Nas visitas interinas do estudo para avaliação clínica e coleta de secreção respiratória para identificação da etiologia do quadro respiratório serão seguidos os procedimentos de segurança recomendados pelo Ministério da Saúde do Brasil e padronizados no IPEC.\n",
            "\n",
            " Title:  S1 Table. Primers used for construction of various mutants of H. pylori. Primer Nucleotide sequence (5'→3') Reference\n",
            "4 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.6833411 \t Middle East respiratory syndrome (MERS) coronavirus (MERS-CoV) causes a severe respiratory disease in humans. The MERS-CoV spike (S) glycoprotein mediates viral entry into target cells. For this, MERS-CoV S engages the host cell protein dipeptidyl peptidase 4 (DPP4, CD26) and the interface between MERS-CoV S and DPP4 has been resolved on the atomic level. Here, we asked whether naturally-occurring polymorphisms in DPP4, that alter amino acid residues required for MERS-CoV S binding, influence cellular entry of MERS-CoV. By screening of public databases, we identified fourteen such polymorphisms. Introduction of the respective mutations into DPP4 revealed that all except one (Δ346-348) were compatible with robust DPP4 expression. Four polymorphisms (K267E, K267N, A291P and Δ346-348) strongly reduced binding of MERS-CoV S to DPP4 and S protein-driven host cell entry, as determined using soluble S protein and S protein bearing rhabdoviral vectors, respectively. Two polymorphisms (K267E and A291P) were analyzed in the context of authentic MERS-CoV and were found to attenuate viral replication. Collectively, we identified naturally-occurring polymorphisms in DPP4 that negatively impact cellular entry of MERS-CoV and might thus modulate MERS development in infected patients.Middle East respiratory syndrome coronavirus (MERS-CoV) is an enveloped virus with a single-stranded RNA genome of positive polarity. It belongs to the Coronaviridae family (genus Betacoronavirus), which is part of the order Nidovirales. MERS-CoV was isolated in 2012 from the sputum of a 60 year old man suffering from acute pneumonia and renal failure in Saudi Arabia . Since its discovery, MERS-CoV has caused 2,442 human infections of which 842 (34.5%) had a fatal outcome (as of May, 2019) . Dromedary camels are reservoir hosts of MERS-CoV and display only common cold-like symptoms upon infection but constitute the main source of human infections. Transmission to humans occurs via close contact to animals or contaminated animal products . Human-to-human transmissions seem limited and were mainly observed in health care settings, leading to MERS outbreaks in hospitals . Finally, differences in the tissue specific expression of the cellular receptor for MERS-CoV, DPP4, were recently suggested to account for the differences in MERS-CoV transmission and disease induction in camels and humans, respectively [13, 14] .In order to infect a host (cell) and replicate, MERS-CoV has to deliver its genome into the cellular cytoplasm for gene translation and genome replication. This process is facilitated by the viral spike (S) glycoprotein, a type-I transmembrane protein embedded in the viral envelope. For host cell entry, the surface unit, S1, of MERS-CoV S binds to the cellular type-II transmembrane protein dipeptidyl peptidase 4 (DPP4, CD26) . The structure of the interface between DPP4 and MERS-CoV-S was resolved on the atomic level and fifteen residues in DPP4 were found to make direct contact with residues in the viral S protein . Upon DPP4 engagement, MERS-CoV S undergoes proteolytic activation through the cellular serine protease TMPRSS2 or the endosomal cysteine protease cathepsin L , which allows the transmembrane unit, S2, of MERS-CoV S to fuse the viral membrane with cellular membranes.DPP4 is a prolyl oligopeptidase that is expressed in various tissues and involved in multiple biological processes including T-cell activation , control of the activity of growth factors, chemokines and bioactive peptides , and regulation of the glucose metabolism . Mature DPP4 is embedded in the plasma membrane as a homodimer and each monomer consists of an N-terminal cytoplasmic domain, followed by a transmembrane domain and a large ectodomain, which can be further subdivided into a short stalk domain, a glycosylation-rich and a cysteine-rich region as well as the C-terminal catalytic domain (α/ β-hydrolase domain) . Polymorphisms in the DPP4 gene were implicated in several diseases and conditions, including diabetes [27, 28] and myocardial infarction but their potential impact on MERS-CoV infection has not been analyzed.We asked whether naturally-occurring amino acid polymorphisms in DPP4 residues making contact with MERS-CoV S have an impact on MERS-CoV entry. We identified fourteen polymorphisms by screening public databases and introduced the respective mutations into a DPP4 expression plasmid. We identified four mutations that reduced MERS-CoV S binding to DPP4 and MERS-CoV S-driven host cell entry without affecting DPP4 expression at the cell surface.293T cells were transfected with expression vectors for WT or mutant DPP4, or empty expression vector (negative control). At 16 h post transfection, the culture medium was replaced and the cells were further incubated for additional 32 h. Then, the cells were washed with PBS and mixed with 2x SDS-sample buffer (0.03 M Tris-HCl, 10% glycerol, 2% SDS, 0.2% bromophenol blue, 1 mM EDTA). Cell lysis was achieved by incubating the samples for 10 min at room temperature followed by incubation at 96°C for an additional 10 min. The samples were further loaded on polyacrylamide gels and SDS-PAGE (sodium dodecyl sulfate-polyacrylamide gel electrophoresis) was performed. Next, the proteins were transferred onto nitrocellulose membranes (Hartenstein GmbH) by immunoblotting. The membranes were further blocked by incubation in PBS-T (PBS containing 0.5% Tween 20 and 5% skim milk powder) for 30 min at room temperature. Afterwards, the membranes were incubated overnight at 4°C with undiluted supernatant of a hybridoma cell line secreting anti-cMYC antibody 9E10 (for DPP4 detection) or PBS-T containing anti-ß-actin (ACTB) antibody (mouse, 1:1,000, Sigma Aldrich). Following three washing intervals with PBS-T, the membranes were further incubated with PBS-T containing horseradish peroxidaseconjugated anti-mouse antibody (goat, 1:5,000, Dianova) for 1 h at room temperature before an in house-prepared enhanced chemiluminescent solution (0.1 M Tris-HCl [pH 8.6], 250 µg/ml luminol, 1 mg/ ml para-hydroxycoumaric acid, 0.3% H 2 O 2 ) was added and signals were recorded using the ChemoCam imaging system and the ChemoStar Professional software (Intas Science Imaging Instruments GmbH).In order to quantify the signal intensity of the protein bands, the program ImageJ (FIJI distribution) was used. To account for differences in the total protein content of the samples and variations, we normalized the DPP4 signals against the respective signals of the loading control (ACTB).293T (human kidney cells, DSMZ no. ACC 635), BHK-21 (hamster kidney cells, DSMZ no. ACC 61) and Vero 76 (African green monkey kidney cells, kindly provided by Andrea Maisner, Philipps-University Marburg) were cultivated in Dulbecco's modified Eagle medium (PAN-Biotech) while Caco-2 cells (human colorectal adenocarcinoma cells) were cultivated in Minimum Essential Medium (ThermoFisher Scientific). The media were supplemented with 10% fetal bovine serum (Biochrom), 100 U/ml of penicillin and 0.1 mg/ml of streptomycin (PAN-Biotech). All cell lines were incubated at 37°C and 5% CO2 in a humidified atmosphere. For subcultivation and seeding, cells were washed with phosphate-buffered saline (PBS) and detached by incubation with trypsin/ EDTA solution (PAN-Biotech) (BHK-21, Vero 76 and Caco-2) or by resuspending the cells in culture medium (293T). Transfection of 293T and BHK-21 cells was carried out by calcium-phosphate precipitation or with the help of ICAFectin-441 (In-Cell-Art) or FuGENE HD (Promega).All DPP4 mutants were generated based on a pcDNA3.1/Zeo(+)-based expression vector in which the coding sequence for human DPP4 (GenBank: XM_005246371.3) containing an C-terminal cMYC epitope was inserted into via BamHI/EcoRI restriction sites. The following aa (amino acid) substitutions were introduced via overlap-extension PCR: K267E, K267N, Q286K, T288I, T288S, A289V, A291P, A291V, R317K, Y322H, I346T, I346V and K392N . In addition, a deletion mutant was generated that lacks aa residues 346-348 (Δ346-348). Information on DPP4 polymorphisms was retrieved from the Ensembl database () and the Single Nucleotide Polymorphism Database (dbSNP) of the National Center for Biotechnology Information (NCBI) () , and is based on data provided by the gnomAD database (Genome Aggregation database, . broadinstitute.org/), TOPMed program (Trans-Omics for Precision Medicine, ), ExAC consortium (Exome Aggregation Consortium, ) and the 1000G project (1,000 genomes project, . internationalgenome.org/) (For detailed information see Supplementary Table 1) .We further utilized pCAGGS-based expression vectors for vesicular stomatitis virus (VSV) glycoprotein (G), MERS-CoV S wildtype (WT) and MERS-CoV S (D510G) (the latter two either untagged or equipped with a C-terminal V5 epitope) that have been described elsewhere . In addition, a previously described expression vector for angiotensin converting enzyme 2 was employed . Similar to the strategy used for the generation of DPP4 mutants, we employed the overlap-extension PCR technique to introduce a single mutation into the MERS-CoV S open reading frame, thus generating untagged and V5-tagged MERS-CoV S (D539N).Soluble S comprising the S1 subdomain of MERS-CoV S (aa residues: 1-747) fused to a human IgG Fc tag was generated by inserting the PCR-amplified S1 sequences into the pCG1Fc vector (kindly provided by Georg Herrler, University of Veterinary Medicine Hannover) making use of the BamHI/SalI restriction sites. In addition, we generated an expression vector for the enhanced green fluorescent protein (eGFP) by inserting the eGFP coding sequence, which was PCR-amplified from the pEGFP-C1 vector (Clontech), into the pCAGGS plasmid using the EcoRI/XhoI restriction sites.All PCR-amplified sequences were subjected to automated sequence analysis (Microsynth SeqLab) to verify their integrity. Sequences of primers used for cloning of the different constructs are available upon request.Analysis of DPP4 surface expression by immunofluorescence analysis BHK-21 cells were grown on coverslips and transfected with the different DPP4 constructs or empty expression vector using ICAFectin-441 (In-Cell-Art) at 24 h post seeding according to the manufacturer's instructions. After changing the culture medium at 4 h post transfection, the cells were incubated for additional 20 h. Then, the culture medium was aspirated and the cells were washed with PBS, before they were fixed by incubated with PBS containing 4% paraformaldehyde (PBS/PFA) for 15 min at room temperature. Subsequently, the cells were washed with 0.1 M glycine/ PBS solution followed by a washing step with PBS. Next, the coverslips were incubated with anti-DPP4 antibody (mouse, diluted 1:200 in PBS containing 1% bovine serum albumin [PBS/BSA], Abcam) for 1 h at 4°C. For this, the coverslip was put on a drop (20 µl) of antibody solution that was added on a sheet of parafilm inside a humidity chamber (a glass dish in which the parafilm was placed on wet paper tissue). Thereafter, the cells were washed 3x with PBS before incubation with AlexaFluor568-conjugated antimouse antibody (goat, 1:1000, diluted in PBS/BSA, ThermoFisher Scientific) for 30 min at 4°C was performed. Subsequently, the cells were washed 3x with PBS. Finally, the cells were incubated with DAPI (4',6-diamidino-2-phenylindole, Carl Roth) and mounted in ProLong Gold Antifade Mountant (Ther-moFisher Scientific) before they were analyzed using a Zeiss LSM800 (Zeiss) confocal laser scanning microscope and the ZEN imaging software (Zeiss).Analysis of DPP4 surface expression by flow cytometry BHK-21 cells were transfected with expression vectors for WT or mutant DPP4, or empty expression vector (negative control). At 16 h post transfection, the culture medium was replaced and the cells were further incubated for additional 32 h. Then, the cells were washed with PBS, resuspended in PBS/ BSA and pelleted by centrifugation (600 x g, 5 min, 4°C). After aspiration of the supernatant, the cells were resuspended in PBS/BSA containing anti-DPP4 antibody (mouse, diluted 1:100, Abcam) and incubated for 1 h at 4°C. Next, the cells were pelleted, washed with PBS/BSA, pelleted again, resuspended in PBS/BSA containing AlexaFluor488-conjugated anti-mouse antibody (donkey, diluted 1:500, Thermo-Fisher Scientific) and incubated for 1 h at 4°C. Subsequently, the cells were washed (as described above) and resuspended in PBS/PFA for 2 h at 4°C for fixation. Finally, the cells were washed (as described above) and resuspended in PBS/BSA for flow cytometric analysis using an LSR II flow cytometer and the FACS Diva software (both BD Biosciences). Additional data analysis was carried out using the FCS Express 4 Flow research software (De Novo software). For quantification of DPP4 surface expression, the mean fluorescence intensity (MFI) value of the negative control was subtracted from all samples. For normalization of DPP4 surface expression, values obtained for cells expressing DPP4 WT were set as 100% and the relative surface expression of the respective DPP4 mutants was calculated accordingly.In order to generate soluble MERS-CoV S for binding studies, 293T cells were transfected with an expression vector for the S1 subunit of MERS-CoV S fused to the Fc fragment of human immunoglobulin G (solMERS-S1-Fc). At 24 h post transfection, the culture medium was exchanged and the cells were further incubated for 24 h before culture supernatants were harvested and freed from cellular debris by centrifugation (4,700 x g, 10 min, 4°C). The clarified supernatants were loaded on Vivaspin protein concentrator columns with a molecular weight cut-off of 30 kDa (Sartorius) and centrifuged at 4,700 x g at 4°C until the sample was 10-fold concentrated.For the binding studies with solMERS-S1-Fc, a similar protocol was followed as described for the analysis of DPP4 surface expression with the exceptions that sol-MERS-S1-Fc was used instead of the primary antibody (1:10 dilution in PBS/BSA) and that an AlexaFluor488conjugated anti-human antibody (goat, 1:500 dilution in PBS/BSA, ThermoFisher Scientific) was employed as the secondary antibody. BHK-21 cells transfected with expression vectors for WT or mutant DPP4, ACE2 or empty expression vector (both negative controls) were analyzed by flow cytometry for solMERS-S1-Fc binding using an LSR II flow cytometer and the FACS Diva software (both BD Biosciences). Additional data analysis was carried out using the FCS Express 4 Flow research software (De Novo software). For quantification of solMERS-S1-Fc binding, the MFI value obtained for cells transfected with empty expression vector was subtracted from all samples. Further, binding of solMERS-S1-Fc to cells expressing DPP4 WT was set as 100% and the relative binding efficiencies to cells expressing the respective DPP4 mutants or ACE2 were calculated accordingly.293T cells (grown in 6-well plates) were cotransfected with expression plasmids coding for solMERS-S1-Fc and WT or mutant DPP4. Cells transfected with empty expression vector instead of DPP4 or sol-MERS-S1-Fc (or both) served as controls. At 16 ,400 x g at 4°C, before 400 µl of the supernatant were mixed with 50 µl of protein A-sepharose (1 g protein A-sepharose [Sigma-Aldrich] in 4 ml PBS) while the residual 100 µl of the cell lysate were mixed with 100 µl 2x SDS-sample buffer and incubated for 15 min at 96°C (These samples were later analyzed to confirm comparable total protein levels [via detection of ACTB] as well as comparable DPP4 and solMERS-S1-Fc levels before the co-immunoprecipitation [co-IP] step.). Following incubation of the lysate/protein A-sepharose mixtures for 2 h at 4°C in an overhead shaker, the samples were centrifuged for 5 min at 16,400 x g at 4°C to pellet the protein A-sepharose/solMERS-S1-Fc/DPP4-complexes. After aspiration of the supernatant, 500 µl of NP40 lysis buffer (without protease inhibitors) were added and the cells were mixed by vortexing, before being centrifuged again. This washing routine was repeated three times, before finally 50 µl of 2x SDSsample buffer were added to the pelleted complexes and the samples were further incubated for 15 min at 96°C. Thereafter, the samples were subjected to SDS-PAGE and Western blot analysis (see above). Detection of DPP4 (lysate and co-IP samples) and ACTB (lysate samples) was carried out as described above. solMERS-S1-Fc was detected (lysate and co-IP samples) by incubation with a peroxidase-conjugated anti-human antibody (goat, 1:5,000, Dianova).Signal intensities of the protein bands were quantified as described above. Further, signals obtained for DPP4 were normalized against the respective signals for solMERS-S1-Fc in order to account for variations in transfection efficiency and sample processing.For the binding studies with soluble DPP4, a similar protocol was followed as described for the analysis of binding of solMERS-S1-Fc with the exceptions that a soluble DPP4 fused to the Fc region of human IgG (solDPP4-Fc, Acro Biosystems) was used instead of solMERS-S1-Fc (1:200 dilution in PBS/BSA) and that an AlexaFluor488-conjugated anti-human antibody (goat, 1:500 dilution in PBS/BSA, ThermoFisher Scientific) was employed as the secondary antibody. 293T cells transfected with expression vectors for WT or mutant (D510G and D539N) MERS-CoV S, or empty expression vector (negative control) were analyzed by flow cytometry for solDPP4-Fc binding using an LSR II flow cytometer and the FACS Diva software (both BD Biosciences). Additional data analysis was carried out using the FCS Express 4 Flow research software (De Novo software). For quantification of solDPP4-Fc binding, the MFI value obtained for cells transfected with empty expression vector was subtracted from all samples. Further, binding of solDPP4-Fc to cells expressing MERS-CoV S WT was set as 100% and the relative binding efficiencies to cells expressing the respective MERS-CoV S mutants were calculated accordingly.We employed a previously described protocol for the generation of VSV pseudotype particles (VSVpp) that is based on a replication-deficient VSV vector that lacks the genetic information for VSV-G but instead contains the genetic information for eGFP and firefly luciferase (fLuc) as reporters of transduction efficiency (VSV*ΔG-fLuc, kindly provided by Gert Zimmer, Institute of Virology and Immunology, Mittelhäusern/Switzerland) [37, 40] . In brief, 293T cells transfected with expression vectors for MERS-CoV S, VSV-G (positive control) or empty expression vector (negative control) were inoculated with VSV*ΔG-fLuc for 1 h before being washed with PBS and further incubated for 16 h with culture medium that was supplemented with anti-VSV-G antibody (I1, mouse hybridoma supernatant from CRL-2700; ATCC) (except for cells expressing VSV-G). The produced VSVpp were inoculated onto BHK-21 cells expressing WT or mutant DPP4, or no DPP4 (empty expression vector, negative control) and incubated for 16-18 h before fLuc activity in cell lysates was quantified as an indicator for transduction efficiency using the Beetle-Juice kit (PJK) and a plate luminometer (Hidex) .BHK-21 cells were transfected with expression vectors for wildtype or mutant DPP4 (K267E or A291P), or empty expression vector (negative control) using FuGENE HD (Promega) according to the manufacturer's instructions. At 24 h posttransfection, the cells were infected with MERS-CoV (Human betacoronavirus 2c EMC/2012, MERS-CoV EMC-2012, GenBank accession number: JX869059) at a multiplicity of infection of 0.01 for 1 h. Thereafter, the inoculum was removed and the cells were washed 3x with PBS before fresh medium was added and the first sample (time point 0 h postinfection) was taken. The cells were further incubated and additional samples were taken at 24 and 48 h postinfection. Viral titers in the culture supernatant were analyzed by quantitative reversetranscriptase PCR, using the upE assay according to a published protocol . In brief, viral RNA was isolated from cell culture supernatant using the NucleoSpin RNA Virus kit (Macherey-Nagel), reversetranscribed into cDNA using the Superscript III one step RT-PCR system (ThermoFisher Scientific) and analyzed on a LightCycler 480 qPCR cycler platform (Roche) with primers and conditions as specified for the upE assay . In vitro-transcribed standard samples containing defined amounts of MERS-CoV fragments (10, 100, 1,000 and 10,000 copies) were included for absolute quantification as genome equivalents (GE).The DPP4 protein structure (4PV7) and the structure of the complex formed by the MERS-CoV S receptor binding domain bound to DPP4 (4L72) were retrieved from the Research Collaboratory for Structural Bioinformatics Protein Database (RSCB PDB, ). Structure visualization and colorization was performed using the YASARA software () and UCSF Chimera version 1.14 (developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco) .One-way or two-way analysis of variance (ANOVA) with Dunnett's posttest was used to test for statistical significance. Only p values of 0.05 or lower were con- Identification of polymorphisms in DPP4 that alter amino acid residues which make contact with MERS-CoV S The binding interface between MERS-CoV S and the cellular receptor DPP4 was resolved by Wang and colleagues using crystallography, revealing the interacting amino acid residues for each binding partner : Fourteen residues of MERS-CoV S (Y499, N501, K502, L506, D510, R511, E513, D537 G538, D539, Y540, R542, W553 and V555) interact with a total of fifteen residues in DPP4 (K267, F269, Q286, T288, A289, A291, L294, I295, H298, R317, Y322, R336, Q344, I346 and K392) , which are distributed over the glycosylation-rich domain and the cysteinerich domain ( Figure 1A-C) .In order to identify polymorphic residues in DPP4 that contact MERS-CoV S, we screened public databases that provide information on polymorphic amino acid residues based on data derived from different bio projects (i.e. gnomAD, TOPMed, ExAC, 1000G; more information is given in the Materials and Methods section). By this method we found that nine out of the fifteen DPP4 residues interacting with MERS-CoV S are polymorphic (K267, Q286, T288, A289, A291, R317, Y322, I346 and K392) ( Figure 1C ). While five of these residues can be replaced by only a single different amino acid residue (Q286[K], A289 . Circles with sticks represent glycosylation sites, while small numbers indicate the amino acid residues. Triangles below the domains highlight the positions of amino acid residues that directly interact with MERS-CoV S (grey triangles mark residues for which no polymorphism has been reported, while red triangles indicate polymorphic residues). (B) Side (left) and top (right) view of homodimeric DPP4 (the dotted line indicates the border between the two monomers and the cellular plasma membrane is schematically depicted below the side view model of DPP4). The protein model was constructed on the published crystal structure (4PV7) deposited in RSCB PDB and the binding interface with MERS-CoV S has been highlighted (green). (C) Close-up on the DPP4 residues that directly interact with MERS-CoV S and for which no polymorphic (yellow) or polymorphic (red) residues have been reported. In addition, the specific residues in DPP4 (regular letters and numbers), including the respective polymorphic residues (letters in brackets), and the corresponding interacting residues in MERS-CoV S (italicized letters and numbers) are indicated. (D) Frequency of polymorphic DPP4 residues in the human population. Public databases (see Supplementary Table 1 and the materials and methods section for detailed information) were screened for the frequency of the polymorphic residues under study (y-axis). Error bars indicate standard error of the mean (SEM) and refer to polymorphic residues found in more than one database. Table 1 ). Finally, the frequency of these polymorphisms in the human population is low, ranging roughly from 1:19,000 (A289V) to 1:245,000 (T288I) ( Figure 1D and Supplementary Table 1 ).We next introduced the polymorphisms in a DPP4 expression plasmid. Western blot analysis and signal quantification revealed that all resulting DPP4 variants were robustly expressed and total expression levels were comparable (Figure 2A-B) . As DPP4 needs to be transported to the plasma membrane to be engaged by MERS-CoV S for host cell entry, we next investigated whether the presence of the polymorphic DPP4 residues has an impact on DPP4 cell surface localization. For this, we performed flow cytometry and confocal laser scanning microscopy, using transfected BHK-21 cells and an antibody targeting the DPP4 ectodomain. We found that all DPP4 variants but one, a deletion variant lacking amino acid residues 346-348 (Δ346-348), displayed comparable cell surface expression levels ( Figure 3A-B) .We next investigated whether polymorphic DPP4 residues impact MERS-CoV host cell entry. For this, we made use of vesicular stomatitis virus (VSV) pseudotypes (VSVpp) bearing MERS-CoV S or VSV G, which does not bind to DPP4 and served as negative control . As expected, VSVpp harboring VSV G were able to efficiently transduce BHK-21 target cells irrespective of DPP4 expression. In contrast, transduction of BHK-21 cells mediated by MERS-CoV S critically depended on ectopic expression of human DPP4, in accordance with published findings ( Figure 4) . Notably, four DPP4 polymorphisms -K267E, K267N, A291P and Δ346-348 -severely reduced MERS-CoV S-driven transduction compared to DPP4 WT (Figure 4 ). In order to analyze whether the reduction in MERS-CoV S-driven host cell entry would translate into attenuated MERS-CoV replication, we next investigated two DPP4 polymorphisms (K267E and A291P) in the context of infection with authentic MERS-CoV. When followed over a period of two days post infection it was observed that MERS-CoV replication in BHK-21 cells expressing human DPP4 was significantly reduced when DPP4 contained either K267E or A291P ( Figure 5 ).After the identification of DPP4 polymorphisms that reduce S-driven cellular entry of rhabdoviral vectors as well as MERS-CoV replication, we next sought to investigate whether the attenuating phenotype was due to reduced binding of MERS-CoV S to DPP4. For this, we used soluble MERS-CoV S, produced by fusing the S1 subunit, which contains the DPP4 binding domain, to the Fc portion of human immunoglobulin G (solMERS-S1-Fc). Co-immunoprecipitation analysis demonstrated that DPP4 variants harboring polymorphisms K267E, K267N or A291P, which were not compatible with efficient MERS-CoV S-driven host cell entry, displayed significantly reduced ability to interact with MERS-CoV S as indicated by weaker DPP4 signals upon protein A-sepharosemediated pull-down of DPP4/solMERS-S1-Fc (as compared to DPP4 WT, Figure 6A-B) . Notably, DPP4 variant Δ346-348 could be as efficiently coimmunoprecipitated as DPP4 WT, indicating that its inefficient receptor function was solely due to its defect in proper surface transport. The findings obtained by co-IP analysis were confirmed by flow cytometry. It was revealed that polymorphisms that reduced MERS-CoV S-driven host cell entry (K267E, K267N, A291P and Δ346-348) and spread of authentic MERS-CoV (K267E and A291P) also reduced MERS-CoV S binding to cells expressing DPP4 on the cell surface ( Figure 6C ). In addition, polymorphism A289V, which decreased MERS-CoV S-driven transduction to a lesser extent than the aforementioned polymorphisms (Figure 4) , also reduced MERS-CoV S binding to DPP4. Thus, DPP4 polymorphisms K267E, K267N and A291P reduce MERS-CoV S-driven host cell entry and MERS-CoV infection by diminishing MERS-CoV S binding to DPP4.Host cell entry of MERS-CoV critically depends on the interaction between the viral S protein and the cellular receptor DPP4. A link between obesity or underlying diseases like diabetes mellitus, which both can affect DPP4 expression levels , and the risk of fatal outcome of MERS-CoV infection has been made . Moreover, alanine scanning mutagenesis identified DPP4 residues critical for MERS-CoV entry, including K267, L294, I295, R317 and R336 [50, 51] . However, the impact of natural-occurring variations on host cell entry of MERS-CoV has not been addressed so far. We identified DPP4 polymorphisms that reduce S protein-driven host cell entry and replication of authentic MERS-CoV by lowering the binding efficiency of MERS-CoV S to DPP4, suggesting that the DPP4 phenotype may impact the course of MERS-CoV infection.Western blot analysis, flow cytometry and confocal laser scanning microscopy revealed that none of the polymorphisms studied, except deletion of amino acids 346-348, had a significant impact on total or cell surface expression of DPP4, at least in the context of DPP4 transfected cells. Four polymorphisms located at three different sites in DPP4 (K267E, K267N, A291P and Δ346-348) severely reduced S protein-driven host cell entry. As DPP4 Δ346-348 was shown to be incompatible with robust cell surface transport but able to interact with MERS-CoV S in co-IP analysis, we conclude that the reduction in entry efficiency is solely due to insufficient DPP4 surface levels. In contrast, reduction of host cell entry by K267E, K267N and A291P could not be explained by reduced DPP4 expression and these polymorphisms were thus further investigated. MERS-CoV infection of BHK-21 cells transfected to express DPP4 WT and variants K267E or A291P revealed that K267E or A291P were not compatible with robust MERS-CoV replication. Finally, co-IP analyses and binding studies with soluble MERS-CoV S showed that these DPP4 polymorphisms reduced S protein binding to DPP4.When looking at the crystal structure of the complex consisting of the MERS-CoV S receptor binding domain bound to DPP4, these observations do not come as a surprise. DPP4 residue K267 has been reported to contact MERS-CoV S residues G538 and D539, including a salt bridge interaction with D539 . The exchange of K267 to either glutamate (E) . Whole cell lysates (WCL) were prepared and analyzed for DPP4 expression by SDS-PAGE under non-reducing conditions and WB using a primary antibody targeting the C-terminal cMYC epitope and a peroxidase-conjugated secondary antibody. Further, expression of beta-actin (ACTB) was analyzed as a loading control. Shown are the expression data from a representative experiment. Numbers at the left indicate the molecular weight in kilodalton (kDa). (B) Quantification of total DPP4 expression in WCL. After normalization of DPP4 band intensities with that of the corresponding ACTB bands. DPP4 WT expression was set as 100% and the relative expression of mutant DPP4 was calculated accordingly. Presented are the combined data of three independent experiments with error bars indicating the SEM. No statistical significance for differences in total expression between WT and mutant DPP4 was observed by one-way analysis of variance with Dunnett's posttest (p > 0.05, not significant [ns]). . Surface expressed DPP4 was stained by subsequent incubation of the non-permeabilized cells with a primary antibody that targets the DPP4 ectodomain and an AlexaFluor488-conjugated secondary antibody. Fluorescent signals representing surface-expressed DPP4 were analyzed by flow cytometry and the mean fluorescence intensity (MFI) values for each sample were calculated. For normalization, the MFI value of the negative control was subtracted from all samples. Further, surface expression of DPP4 WT was set as 100% and the relative surface expression of the DPP4 mutants was calculated accordingly. Shown are the combined data of three experiments with error bars indicating the SEM. Statistical significance for differences in surface expression between WT and mutant DPP4 was tested by one-way analysis of variance with Dunnett's posttest (p > 0.05, not significant; p ≤ 0.05, *). (B) DPP4 surface expression was further analyzed by immunofluorescence analysis. For this, DPP4 WT or DPP4 mutants were expressed in BHK-21 cells grown on coverslips (cells transfected with empty expression vector served as negative control). After fixation of the cells, surface expressed DPP4 was stained by subsequent incubation of non-permeabilized cells with a primary antibody that targets the DPP4 ectodomain and an AlexaFluor568-conjugated secondary antibody. In addition, cellular nuclei were stained with DAPI. Finally, images were taken using a confocal laser scanning microscope at a magnification of 80x.or asparagine (N) likely abolishes/decreases the interaction with MERS-S due to the different biochemical properties of K267 (positively charged, basic) versus E267 (negatively charged, acidic) and N267 (not charged, acidic) (Supplementary Figure 1) . For DPP4 residue A291, which has been reported to contact the MERS-CoV S residue E513, no information on the type of interaction is available . Here, we speculate that the bulky and distorted side chain of proline (in comparison to the small side chain of alanine) abolishes/decreases interaction with MERS-CoV S residue E513 (Supplementary Figure 1) . In contrast to that, valine contains a small side chain and also has identical biochemical properties as alanine and thus might be efficiently contacted by E513 of MERS-CoV S, which is why we did not observe any impact of polymorphisms A291V on MERS-CoV S-driven entry and MERS-CoV S MERS-CoV S binding/interaction (Supplementary Figure 1 ).The observation that certain polymorphisms in DPP4 reduced MERS-CoV S binding and viral entry triggered the question whether residues in MERS-CoV S that are in direct contact with the respective DPP4 residues are also polymorphic. Indeed, we obtained initial evidence to support such a concept. Thus, we found that residue 539 in MERS-CoV S which contacts DPP4 residue 267 is polymorphic, with certain MERS-CoV variants harboring an asparagine instead of an . Reduced MERS-CoV S-driven host cell entry is caused by inefficient S protein binding to DPP4 harboring polymorphic amino acid residues. In order to investigate whether reduced MERS-CoV S-driven host cell entry and MERS-CoV replication is due to inefficient MERS-CoV S binding to DPP4 harboring amino acid polymorphisms at the binding interface, we performed co-immunoprecipitation (co-IP) as well as binding experiments with a soluble S protein comprising the S1 subunit of MERS-CoV S fused to the Fc region of human IgG. (A) 293T cells were cotransfected with expression plasmids coding for soluble, Fc-tagged MERS-CoV S1 (solMERS-S1-Fc) and the indicated DPP4 variant containing a C-terminal cMYC-tag. Cells that were transfected only with empty expression vector alone, or empty expression vector instead of either solMERS-S1-Fc or DPP4 served as controls. At 48 h posttransduction, cells were lysed and incubated with protein A sepharose. Next, samples were subjected to SDS-PAGE and Western blot analysis. DPP4 levels were detected via antibodies specific for the cMYC-tag, whereas solMERS-S1-Fc was detected using a peroxidase-coupled anti-human antibody. Similar results were obtained in three individual experiments. Analysis of whole cell lysates (WCL) for expression of solMERS-S1-Fc, DPP4 and ß-actin confirmed comparable ß-actin levels in each sample and comparable expression levels for solMERS-S1-Fc and DPP4. (B) For quantification of MERS-CoV S/DPP4 interaction we first normalized the DPP4 signals against the respective solMERS-S1-Fc signals. Then, MERS-CoV S/DPP4 interaction was set as 100% for wildtype (WT) DPP4 and the relative interaction efficiency for each DPP4 mutant was calculated accordingly. Presented are the mean data from three independent experiments. Error bars indicate the SEM. Statistical significance of differences in MERS-CoV S/DPP4 interaction between WT and mutant DPP4 was analyzed by one-way analysis of variance with Dunnett's posttest (p > 0.05, ns; p ≤ 0.05, *; p ≤ 0.001, ***). (C) Soluble MERS-CoV S1-Fc was incubated with BHK-21 cells expressing wildtype (WT) or mutant DPP4, or cells transfected with empty expression vector or an ACE2-expression plasmid (controls). To detect bound S protein, the cells were subsequently incubated with an AlexaFluor488-conjugated anti-human antibody directed against the Fc-tag. Fluorescent signals representing bound solMERS-S1-Fc were analyzed by flow cytometry and MFI values for each sample were calculated. For normalization, the MFI value of the negative control (empty expression vector) was subtracted from all samples. Further, binding of sol-MERS-S1-Fc to cells expressing DPP4 WT was set as 100% and the relative binding to cells expressing the DPP4 mutants was calculated accordingly. Shown are the combined data of five independent experiments with error bars indicating the SEM. Statistical significance of differences in solMERS-S1-Fc binding to cells expressing WT or mutant DPP4 was analyzed by one-way analysis of variance with Dunnett's posttest (p > 0.05, ns; p ≤ 0.05, *; p ≤ 0.01, **; p ≤ 0.001, ***). aspartate residue at this position. D539N reduced entry into cells expressing relatively low amounts of DPP4 but had no effect on entry into cells expressing high amounts of DPP4 (Supplementary Figure 2) . Moreover, and more interestingly, D539N slightly rescued MERS-CoV S-driven entry from the negative effect exerted by DPP4 polymorphism K267N (Supplementary Figure 2) . Similarly, residue 510 in MERS-CoV S, which is known to interact with DPP4 residues 317 and 322, was found to be polymorphic, and previous studies demonstrated that polymorphism D510G reduced DPP4 binding but also increased resistance to neutralizing antibodies . Notably, D510G slightly increased entry via DPP4 harboring polymorphism Y322H and allowed MERS-CoV S to use DPP4 with polymorphism R317K with the same efficiency as WT DPP4. It should be stated that none of these effects was statistically significant and that DPP4 and MERS-CoV S polymorphisms occur with low frequency. Although it is unlikely that the DPP4 polymorphisms have emerged as a result of evolutionary pressure from MERS-CoV infections, our results suggest that certain existing DPP4 polymorphism(s) might foster the emergence of MERS-CoV variants with altered biological properties.The polymorphisms studied here occur with relatively low frequencies of one per ∼19,000 (A289V) to ∼245,000 (T288I) individuals. However, detailed information on the geographic distribution or incidence in certain ethnical groups is largely missing. Thus, DPP4 polymorphisms could contribute to the perplexing absence of MERS cases in Africa, where the virus circulates in camels . However, recent evidence suggests that sequence variations between African and Arabian MERS-CoV might be a factor [53, 57] . More importantly, it remains to be analyzed how frequent DPP4 polymorphisms that affect S protein binding occur in the Middle East and whether they are associated with the clinical course of MERS.\n",
            "\t  0.680561 \t Rationale: Hemophagocytic lymphohistiocytosis (HLH) is a syndrome that is characterized by an inappropriate hyperinflammatory immune responseprimary, as a consequence of a genetic defect of NK cells and cytotoxic T lymphocytes orsecondary, in the progression of infections, rheumatic or autoimmune diseases, malignancies or metabolic diseases.Hemophagocytic lymphohistiocytosis (HLH) is a syndrome that is characterized by an inappropriate hyper-inflammatory immune responseprimary, as a consequence of a genetic defect of NK cells and cytotoxic T lymphocytes orsecondary, in the progression of infections, rheumatic or autoimmune diseases, malignancies or metabolic diseases. Among HLH-related infections, the most common are the viral infections: Epstein--Barr virus, cytomegalovirus, other herpes viruses, the viruses of hepatitis B and C, etc., followed by bacterial, parasitic, or fungal infections. Association of secondary HLH to Streptococcus pneumoniae infection in a patient splenectomized for spherocytosis has not been described so far.We present the case of a male Caucasian patient, aged 37 years, splenectomized for spherocytosis since the age of 4, with no prophylaxis of meningococcal, and pneumococcal infections through vaccination, that was brought to the emergency room for fever, diarrhea, vomiting, rash skin, myalgia, anuria, and marked alteration of his general condition. At the time of admission, on physical examination, the following changes were noticed: facial erythema, purpura on the legs (see Fig. 1 ), cyanosis of the extremities, jaundice, left basilar crackles, heart rate of 130 beats per minute, blood pressure of 140/100 mm Hg, hepatomegaly, and anuria. The laboratory investigations that were performed revealed the following alterations (at admission and in evolution) that are presented in Table 1 .Peripheral smear revealed normochromic normocytic red blood cells, as well as microspherocytes and spherocytes, frequent polychromatophilic macrocytes; erythrocytes with Howell-Jolly bodies (splenectomy), rare dacryocytes (teardrop cells), and schistocytes. Polymorphonuclears with vacuolated cytoplasm are present (toxic appearance), diplo-, encapsulated, and intra-and extracellular gram-positive cocci. Bone marrow sample was harvest and on hematoxylin-eosin stain an increased number of activated macrophages with prominent hemophagocytosis of hematopoietic elements was revealed. Blood cultures and urine cultures were positive for S pneumoniae, resistant to benzylpenicillin, chloramphenicol, erythromycin, trimethoprim/ sulfamethoxazole, tetracycline, cefotaxime, and intermediate resistance to ceftriaxone, imipenem, sensitive to ofloxacin, vancomycin, moxifloxacin, quinupristin/dalfopristin, levofloxacin, linezolid, rifampicin, sparfloxacin, pristinamycin, amoxicillin, and telithromycin. Viral, parasitic etiologies were excluded, as well as rheumatic diseases, malignant tumors, which may be involved in secondary HLH.A cardiac ultrasound was performed and revealed no suggestive images of infectious endocarditis or valvular heart disease. Initially, the chest radiography revealed no changes, but in evolution, it showed bilateral alveolar condensation and left pleural effusion.The case was interpreted as sepsis due to a multidrug-resistant S pneumoniae associated with consumption coagulopathy (bleeding at venepuncture site and epistaxis), acute liver failure, acute renal failure by myoglobinuria, and HLH. In evolution, acute respiratory failure occurred, for which endotracheal intubation of the patient was performed.The treatment was started with infusions of macromolecular solutions, hydro-electrolytic rebalancing, packed red blood cells, fresh frozen plasma, and antibioticsinitially, with ultrabroadspectrum antibiotics, meropenem 2 g/day associated with linezolid 2 Â 600 mg/day, thereafter treatment continued with linezolid associated with moxifloxacin 400 mg/day according to antibiogram results, dexamethasone, etoposides 150 mg/m 2 /day, 3 days, anidulafungin, intravenous immunoglobulin, and daily hemodialysis sessions throughout hospitalization. The evolution was unfavorable with coma, Glasgow Coma Scale of 3, and quadriplegia, the occurrence of bronchopneumonia required endotracheal intubation. During hospitalization, the patient was anuric. Patient death occurred on day 15 of hospitalization. Hematoxylin-eosin and immunohistochemically stainings of liver This case brings into question the risk of splenectomised patients to develop severe systemic infections with encapsulated bacteria, against which the vaccine prophylaxis is essentialfor S pneumoniae, H influenzae, and Meningococcus. Lack of vaccination in this patient has enabled the development of severe infections with multidrug-resistant S pneumoniae and the induction of secondary HLH, characterized by an uncontrolled inflammatory response, resulting in patient's death. Primary HLH has a family, autosomal recessive transmission, and is present in 50,000 new-borns annually. Secondary HLH may be induced by viral (29%), bacterial, or parasitic infections (20%), autoimmune or rheumatic diseases (7%), cancers (27%), and metabolic diseases or immunodeficiency syndromes (6%). Among viral causes, the most common is associated with Epstein-Barr virus. There are also described associations with cytomegalovirus, herpes simplex, [2, 3] varicella-zoster virus, herpes virus 8, and associated with HIV infection. There were also published associations with hepatitis B and C viruses, [6, 7] influenza viruses, enteroviruses, rotavirus, severe acute respiratory syndrome virus, hemorrhagic fevers, HIV, and so on. Among bacterial infections associated with HLH, there were published associations with Borrelia, -Babesia sp, Bartonella sp, Brucella sp, Q fever, Leptospira sp, Listeria monocytogenes, Mycoplasma pneumoniae, and mycobacteria. The most frequently described parasitic causes responsible for secondary HLH are: Leishmania sp, [33, 34] malaria, and Toxoplasma gondii. [38, 39] Fungal infections are found to be associated with secondary HLH in HIV-infected patients such as Cryptococcus neoformans, Candida spp, or in patients with renal transplantation-association with disseminated histoplasmosis. Of the defining HLH criteria established by the Histiocyte Society, respectively, fever, splenomegaly, cytopenia (on at least 2 lines in peripheral blood), hypofibrinemia, hyperferritinemia, hypertriglyceridemia, presence of hemophagocytosis in bone marrow or lymph node, reduction/absence of NK cells activity, and increase in the concentration of soluble IL-2 receptor, CD25, the patient had 5 defining criteria for HLH. The presence of consumption coagulopathy evidenced by the presence of fibrin degradation products is associated with hypofibrinemia and thrombocytopenia in HLH, as well as the liver damageelevated transaminases, hyperbilirubinemia, activated partial thromboplastin time prolongation. These changes are also Table 2 Clinical and biological characteristics of the 3 cases with HLH due to S pneumoniae. # 1 2 3Our case Dalle et al 1995 Ondruschka et al Although the diagnosis of HLH was early, the administration of dexamethasone, intravenous immunotherapy, and administration of etoposides have not improved prognosis, the patient's death occurring on the 15th day of hospitalization. Renal insufficiency due to bilateral renal papillary necrosis has been associated with myoglobinuria as a result of septic myositis, demonstrated by high levels of creatine phosphokinase. There have been described situations in which S pneumoniae infections are responsible for the death of patients with primary HLH. Two cases of HLH associated with a pneumococcal infection had been published (see Table 2 ). HLH association with hereditary spherocytosis is found in association with viral infections, such as parvovirus B19 or Epstein-Barr virus. Moreover, splenectomy is described as a therapeutic method in refractory HLH cases. 3.1. Informed consent Written informed consent was obtained from the patient's next of kin (from his father) for publication of this case report and any accompanying images. The study was accepted by the Ethics Committee of the hospital and they encouraged publishing the article. A copy of the written consent is available for review by the Editor-in-Chief of this journal.The association of sepsis due to S pneumoniae with secondary HLH in a splenectomized patient for spherocytosis should be taken into consideration in similar cases like ours. To our knowledge, no similar cases had been published in the literature, in which the secondary HLH was the consequence of an invasive pneumococcal infection in a splenectomized patient for spherocytosis, and the association of splenectomy with HLH is surprizing. Other similar observations are necessary in the future.\n",
            "\t  0.6697011 \t Hantaan virus (HTNV) causes hemorrhagic fever with renal syndrome (HFRS). Previous studies have identified interferon-induced transmembrane proteins (IFITMs) as an interferon-stimulated gene family. However, the role of IFITMs in HTNV infection is unclear. In this study, we observed that IFITM3 single nucleotide polymorphisms (SNP) rs12252 C allele and CC genotype associated with the disease severity and HTNV load in the plasma of HFRS patients. In vitro experiments showed that the truncated protein produced by the rs12252 C allele exhibited an impaired anti-HTNV activity. We also proved that IFITM3 was able to inhibit HTNV infection in both HUVEC and A549 cells by overexpression and RNAi assays, likely via a mechanism of inhibiting virus entry demonstrated by binding and entry assay. Localization of IFITM3 in late endosomes was also observed. In addition, we demonstrated that the transcription of IFITM3 is negatively regulated by an lncRNA negative regulator of interferon response (NRIR). Taken together, we conclude that IFITM3, negatively regulated by NRIR, inhibits HTNV infection, and its SNP rs12252 correlates with the plasma HTNV load and the disease severity of patients with HFRS.associates with the severity of disease, indicating the importance of viremia in the pathogenesis of HFRS (2) . Therefore, further studies of host factors limiting HTNV infection and influencing antiviral response as well as disease progression are clinically significant and timely. The human family of interferon-induced transmembrane proteins (IFITMs) was discovered 25 years ago to consist of interferon-stimulated genes (ISGs) (3) . This family includes five members, namely, IFITM1, 2, 3, 5, and 10, among which IFITM1, 2, and 3 possess antiviral activity (4) . Different IFITM proteins have different antiviral spectrum (5) . For example, IFITM3 has been shown to prevent influenza virus infection in vitro and in mice (6, 7) , and it also inhibits multiple viruses, including filoviruses, rhabdoviruses, flaviviruses, and even Ebola and Zika virus (7) (8) (9) (10) (11) . The antiviral mechanism of IFITM3 is thought to be the restriction of viral entry into cells (4, 12) . Single nucleotide polymorphisms (SNPs) are single nucleotide variations in a genetic sequence that occur at an appreciable frequency in the population. Several SNPs has been identified in IFITM3, among which the rs12252 site with C allele results in a N-terminal truncation of IFITM3 protein, leading to impaired inhibition of influenza virus in vitro (13, 14) . Notably, the frequencies of rs12252 C allele and CC genotype correlate with disease severity in patients infected with influenza virus (13, 15) . HTNV has been shown to induce a type I interferon response (though in later time postinfection) (16, 17) . While overexpression of IFITM1, 2, and 3 in Vero E6 cells has been reported to inhibit HTNV infection (18) , however, the effect of IFITMs on HTNV infection in human cell lines and its role in HFRS still remain unknown.LncRNA comprises a group of non-coding RNAs longer than 200 nt that function as gene regulators. Some lncRNAs have been shown to play a role in innate immunity (19) . Among them, negative regulator of interferon response (NRIR) (lncRNA NRIR, also known as lncRNA-CMPK2) is a non-coding ISG that negatively regulates IFITM1 and Mx1 expression in HCV infection (20) . Notably, IFITM3 is largely homologous to IFITM1, but the role of NRIR in the regulation of IFITM3 in HTNV infection remains unclear.In the present study, we investigate the effect of IFTTM3 on the replication of HTNV and its role in the development of HFRS in humans. We provide primary evidence suggesting that IFITM3, regulated by NRIR, can inhibit HTNV infection and its SNP rs12252 correlates with the disease severity and viral load in patients with HFRS. This study expands our understanding of the antiviral activity of IFITM3 and enriches our knowledge of innate immune responses to HTNV infection.This study was conducted in accordance with the recommendations of the biomedical research guidelines involving human participants established by the National Health and Family Planning Commission of China. The Institutional Ethics Committee of Tangdu Hospital approved this study. All subjects gave written informed consent in accordance with the Declaration of Helsinki. Before inclusion, all participants were informed of the study objectives and signed the consent form before blood samples and medical records were obtained.Sixty-nine HFRS patients admitted into the Department of Infectious Diseases, Tangdu Hospital between October 2014 and March 2016 were enrolled in this study. All patients were Han Chinese. The diagnosis of HFRS was made based on typical symptoms and signs as well as positive IgM and IgG antibodies against HTNV in the serum assessed by enzyme linked immunosorbent assay (ELISA) in our department. The classification of HFRS severity and the exclusion criteria were described as follows (21) : white blood cells (WBC), platelets (PLT), blood urea nitrogen (BUN), serum creatinine (Scr), and heteromorphic lymphocytes that were tested by the Department of Clinical Laboratory (shown in Table 1 ).According to clinical symptoms and signs, such as fever, effusion, hemorrhage, edema, and renal function, the severity of HFRS can be classified as previously described (21): (1) mild patients were identified with mild renal failure without an obvious oliguric stage; (2) moderate patients were those with obvious symptoms of uremia, effusion (bulbar conjunctiva), hemorrhage (skin and mucous membrane), and renal failure with a typical oliguric stage; (3) severe patients had severe uremia, effusion (bulbar conjunctiva and either peritoneum or pleura), hemorrhage (skin and mucous membrane), and renal failure with oliguria (urine output, 50-500 ml/day) for ≤5 days or anuria (urine output, <50 ml/day) for ≤2 days; and (4) critical patients exhibited ≥1 of the following signs during the illness: refractory shock, visceral hemorrhage, heart failure, pulmonary edema, brain edema, severe secondary infection, and severe renal failure with oliguria (urine output, 50-500 ml/day) for >5 days, anuria (urine output, <50 ml/day) for >2 days, or a BUN level of >42.84 mmol/l. Due to the sample quantity required for SNP typing, the mild and moderate patients were assessed together in the mild group, and we combined severe and critical patients as severe group.The exclusion criteria for this study were patients with: (1) any other kidney disease, (2) diabetes mellitus, (3) autoimmune disease, (4) hematological disease, (5) cardiovascular disease, (6) viral hepatitis (types A, B, C, D, or E), or (7) any other liver disease. In addition, no patients received corticosteroids or other immunomodulatory drugs during the study period (21) .Genomic DNA was extracted from the peripheral blood of patients using the PureGene DNA Isolation kit (Gentra Systems, Minneapolis, MN, USA). The region encompassing the human IFITM3 rs12252 were amplified by PCR (forward primer, 5′-GGAAACTGTTGAGAAACCGAA-3′ and reverse primer, 5′-CATACGCACCTTCACGGAGT-3′). The PCR products were purified and sequenced using an Applied Biosystems 3730xl DNA Analyzer (Thermo Scientific, Waltham, MA, USA). The allele frequencies and genotypes of healthy Han Chinese and other groups were obtained from the 1,000 genomes project (http:// ).The HTNV load in plasma samples (collected during the acute phase) from 24 age-and sex-matched HFRS patients with different genotypes were measured using previously reported methods (2) . Briefly, viral RNA was extracted from the plasma of HFRS patients using Purelink Viral RNA/DNA Kits (Invitrogen, Carlsbad, CA, USA). The SuperScript III Platinum One-Step Quantitative RT-PCR System kit (Invitrogen, Carlsbad, CA, USA) was employed for the real-time RT-PCR assay. The primers and probe (provided by Sangon Biotech, Shanghai, China) were as follows: forward, 5′-TACAGAGGGAAATCAATGCC-3′, reverse, 5′-TGTTCAACTCATCTGGATCCTT-3′, and probe, 5′-(FAM) ATCCCTCACCTTCTGCCTGGCTATC (TAMRA)-3′. The synthetic S segment of the HTNV standard strain 76-118 RNA transcript was used as the quantitative calibrator. The external standard was the culture supernatant of Vero E6 cells infected with HTNV 76-118, which was quantified using synthetic quantitative calibrators. For each experiment, one aliquot of calibrated 76-118 standard was extracted in parallel with the clinical samples and serially 10-fold diluted with concentrations ranging from 10.56 to 2.56 log10 copies/ml. PCR was performed using an iQ5 Cycler (Bio-Rad, Hercules, CA, USA) with following conditions: 42°C for 15 min, 95°C for 2 min, and 50 cycles of 15 s at 95°C, 30 s at 53°C, and 30 s at 72°C. Fluorescence was read during the 72°C step of the final segment of every cycling program.HUVEC cells (ScienCell Research Laboratories, Carlsbad, CA, USA) were grown in ECM BulletKit (ScienCell Research Laboratories, Carlsbad, CA, USA) in a 5% CO2 incubator. A549 cells (ATCC Cat# CRM-CCL-185, RRID:CVCL_0023) were grown in our laboratory in DMEM with 10% FBS (Thermo Scientific, Waltham, MA, USA) in a 5% CO2 incubator. Cells were used within passage 10 after primary culture. HTNV strain 76-118 was cultured in Vero E6 cells (ATCC Cat# CRL-1586, RRID:CVCL_0574) in our laboratory and titrated using an immunofluorescence staining assay for HTNV nucleocapsid protein (NP) as previously described (22) . The TCID50 was 10 5 /ml, which was calculated using the Reed-Muench method.The recombinant human IFN-α2a was obtained from PBL Interferon Source (Piscataway, NJ, USA) and dissolved in the buffer provided by the manufacturer (composition not disclosed). HUVEC and A549 cells were infected by incubation with HTNV as indicated moi at 37°C for 60 mins. Subsequently, the virus solution was removed and fresh medium added to the cell culture.Cells were transfected with lentiviral vectors of c-myc-tagged IFITM1, IFITM2, IFITM3, and IFITM3 NΔ21 (purchased from GENECHEM, Shanghai, China) at a moi of 10. Puromycin (2 μg/ ml for HUVEC and 6 μg/ml for A549 cells) was used to create cell lines stably expressing IFITMs. Cells were transfected with control (scrambled) short interfering RNA (siRNA), IFITM1 siRNA, IFITM2 siRNA, or IFITM3 siRNA (10 nM) using Lipofectamine 3000 transfection reagent (Invitrogen, Carlsbad, CA, USA). SiRNAs were purchased from Origene (Rockville, MD, USA), and the sequences were not disclosed.Total RNA was extracted using TRIzol reagent (Invitrogen, Carlsbad, CA, USA), and cDNA was synthesized using the K1622 kit (Thermo Scientific, Waltham, MA, USA). Quantitative realtime PCR (qPCR) was performed using SYBR Premix Ex Taq II (Takara Biotechnology Co., Dalian, China) with a Bio-Rad iQ5 cycler (Bio-Rad, Hercules, CA, USA). β-actin was used as the reference gene. The primers (Sangon Biotech, Shanghai, China) were as follows: IFITM1 (forward, 5′-ACTCCGTGAAGTCTAGGGACA-3′ and reverse, 5′-TGTCACAGAGCCGAATACCAG-3′); IFITM2 (forward, 5′-ATCCCGGTAACCCGATCAC-3′ and reverse, 5′-CTTCCTGTCCCTAGACTTCAC-3′); IFITM3 (forward, 5′-GGTCTTCGCTGGACACCAT-3′ and reverse, 5′-TGTCCCTAGACTTCACGGAGTA-3′); IFITM3 pre-mRNA (forward, 5′-CATAGCACGCGGCTCT CAG-3′ and reverse, 5′-CGTCGCCAACCATCTTCCTG-3′); HTNV S segment (forward, 5′-GCCTGGAGACCATCTGA AAG-3′ and reverse, 5′-AGTATCGGGACGACAAAGGA-3′); β-actin (forward, 5′-GCTACGTCGCCCTGGACTTC-3′ and reverse, 5′-GTCATAGTCCGCCTAGAAGC-3′); NRIR (forward, 5′-ATGGTTTTCTGGTGCCTTG-3′ and reverse, 5′-GGAGGTTAGAGGTGTCTGCTG-3′); NRAV (forward, 5′-TCACTACTGCCCCAGGATCA-3′ and reverse, 5′-GGTGGTCACAGGACTCATGG-3′).For detection of miR-130a, cDNA was synthesized using the TaqMan microRNA reverse transcription kit (Invitrogen, Carlsbad, CA, USA) with a specific primer in gene-specific TaqMan assay kit (000454, Invitrogen, Carlsbad, CA, USA). MiR-130a level was determined using the gene-specific TaqMan assay kit (000454, Invitrogen, Carlsbad, CA, USA). U6 (001973, Invitrogen, Carlsbad, CA, USA) was used as an endogenous control (23) . Because the pre-mRNA levels can represent the initial transcription rate (24) , the primers used to detect the pre-mRNA of IFITM3 were designed targeting the intron of IFITM3 as previously described (25) . IFITM3 has two exons and one intron. For qPCR of IFITM3 pre-mRNA, the forward primers were positioned in the intron, and the reverse primer was positioned at the beginning of the second exon. For qPCR of IFITM3 mRNA, the forward primers were positioned in the first exon, and the reverse primer was positioned at the beginning of the second exon (24) . Because the basal expression of IFITM3 is low in A549 cells, we detected IFITM3 mRNA and pre-mRNA in A549 cells following IFN-α2a treatment (20 IU/ml for 12 h) after the overexpression of NRIR.Cell lysates were prepared using Radio Immunoprecipitation Assay (RIPA) buffer (Sigma-Aldrich, St. Louis, MO, USA). Equal amounts of protein (20 μg protein/lane) were electrophoresed on a 10%-SDS-polyacrylamide gel and electrophoretically transferred to a polyvinylidene difluoride membrane (Millipore, Billerica, MA, USA). After blocking with 5% bovine serum albumin in Trisbuffered saline at room temperature for 1 h, the membranes were incubated with antibodies against IFITM1 (Proteintech Group Cat# 60074-1-Ig Lot# RRID:AB_2233405), IFITM2, IFITM3 (Proteintech Group Cat# 66081-1-Ig Lot# RRID:AB_11182821), and β-actin (Proteintech, Wuhan, Hubei, China) or HTNV NP (provided by the Department of Microbiology, The Fourth Military Medical University) overnight at 4°C. The membranes were then washed and incubated with HRP-conjugated IgG antibody (Cell Signaling Technology, Danvers, MA, USA) for 1 h at room temperature. The blots were developed using an enhanced chemiluminescence detection kit (Millipore, Billerica, MA, USA) and visualized using X-ray film. The blot densities were analyzed using the Quantity One software (Bio-Rad, Hercules, CA, USA). In addition, the RIPA buffer contains 50mM Tris (pH = 7.4), 150 mM NaCl, 1% Triton X-100, 1% sodium deoxycholate, 0.1% SDS. Protease inhibitor cocktail (Roche, Basel, Switzerland) was added before use.The cells were cultured on glass coverslips (Millipore, Billerica, MA, USA) until they were semi-confluence and then incubated with HTNV for 60 min (moi = 1). At the indicated times post-HTNV infection, the cells were fixed with 4% PFA, incubated with 0.3% Triton X-100 (Sigma-Aldrich, St. Louis, MO, USA), and blocked with 5% BSA for 1 h. Following incubation with a mouse monoclonal antibody against c-myc-tag (Sigma-Aldrich, St. Louis, MO, USA, Sigma-Aldrich Cat# M5546), IFITM3, lysosome-associated membrane glycoprotein 1 (LAMP1, Cell Signaling Technology, Danvers, MA, USA), or HTNV NP at 37°C for 2 h, the cells were washed and incubated with anti-rabbit Ig conjugated to Alexa 555 and anti-mouse Ig conjugated to Alexa 488 (Abcam, Cambridge, MA, USA) secondary antibodies at room temperature for 1 h. The nuclei were counterstained with DAPI. An Olympus BX51 fluorescence microscope system and FV1000 confocal microscopy system (Olympus, Tokyo, Japan) were used to capture the images. hTnV binding and entry assay Cells transduced with IFITM3 or the empty vector were detached and washed extensively with cold PBS. The cells and HTNV were pre-chilled on ice for 30 min, mixed at a moi of 1 and incubated at 4°C for 1 h with rotation. Part of cells were washed extensively with ice-cold PBS and harvested for binding assay. Another part of cells were switched to 37°C for 2 h to allow HTNV entry. The HTNV that remained on the cell surface was removed by treatment with proteinase K (0.1 mg/ml, Thermo Scientific, Waltham, MA, USA). To achieve direct entry of HTNV into cells by virus-plasma membrane fusion as a positive control, cells were pre-chilled on ice for 10 min with 20 mM NH4Cl. Adsorption of HTNV (moi = 1) was performed at 4°C for 1 h. The cells were then washed, and fusion of the virus with the plasma membrane was triggered by incubation in low pH medium (20 mM sodium succinate, pH = 5.5) for 10 min at 37°C. Infection was followed by incubation for 2 h at 37°C in the presence of 20 mM NH4Cl (26) . qPCR analysis of the HTNV S segment was conducted to evaluate the influence of IFITMs on HTNV cell adhesion and HTNV entry.All data were expressed as the mean ± SEM. Statistical analyses were performed using GraphPad Prism 5 (GraphPad Software, La Jolla, CA, USA). For association analysis of the rs12252 allele and genotype, Fisher's exact test was used. Independent samples t-tests were used for normally distributed data. Differences among groups were determined by one-way analysis of variance (ANOVA) with repeated measures, followed by Bonferroni's post hoc test. P < 0.05 was considered statistically significant.The iFiTM3 snP rs12252 c allele and cc genotype associated with severe hFrs Disease and a higher Plasma hTnV loadTo determine the clinical significance of IFITM3 SNP in HTNV infection, the relationship between rs12252 SNP and the severity of HFRS in 69 patients were examined. We sequenced 300 bp of the IFITM3 locus encompassing SNP rs12252 in all enrolled patients. Then, we stratified these patients into mild and severe, based on the clinical assessment as described in Section \"Material and Methods. \" We found a significantly higher frequency of the C allele among severe HFRS patients compared with the healthy Han Chinese in the 1,000 genomes sequence database (68.29 vs. 52.16%, P = 0.0076). The frequency of rs12252 C in severe patients was also higher than those mild patients (68.29 vs. 46.43%, P = 0.013, Figures 1A,B; Table 2 ). These data suggest that harboring rs12252 C allele increases the risk of suffering severe disease in HTNV-infected individuals, with an odds ratio (95% CI) of 2.124 (1.067-4.230). For genotypes, 43.90% of the severe patients carried the CC genotype, a significantly higher frequency than the control Han Chinese per 1,000 genomes sequence database (26.92% CC genotype, P = 0.03) as well as mildly infected patients (14.29%, P = 0.02, Figures 1A,B ; Table 2 ). However, mildly ill individuals did not exhibit a Fisher's exact test was used to test the association between rs12252 allele/genotype and HFRS severity. (c) The plasma HTNV load in CC genotype patients and CT/TT genotype patients, tested by qRCR analysis. Each symbol represents one individual patient. Independent samples t-test was used to test the difference of HTNV load between groups. *P < 0.05, **P < 0.01. significantly different genotype frequencies compared with the Han Chinese population. In addition, we also found that patients with CC genotype had higher plasma viral load in acute phase ( Figure 1C) . These results support the notion that the normal function of IFITM3 plays a critical role in the immune response to HTNV infection in vivo, which has a substantial influence on the clinical manifestation of HFRS.Previous studies reveal that the truncated IFITM3 protein produced by SNP rs12252 C allele (Figure 2A , the missing part stands for the truncated 21 amino acids from N-terminal of IFITM3, the intramembrane helix, and transmembrane helix was presented as boxes) leads to an impaired anti-influenza activity (14) . To test the functional significance of this polymorphism in HTNV infection, we transfected the majority T or minority C variant IFITM3 alleles that produce full-length or N-terminally truncated (NΔ21) proteins (Figure 2A ) with c-myc-tag to HUVEC and A549 cell using lentivirus vectors ( Figure 2B) . Then, we challenged the cells with HTNV at moi = 1 for 24 h and found that cells with the minority C variant were more susceptible to HTNV infection with higher expression of HTNV S segment ( Figure 2C ) and more positive of HTNV NP ( Figure S3 in Supplementary Material). Indeed, compared with the mock (empty vector)-infected control, the NΔ21 protein almost lost the ability to inhibit HTNV infection in both HUVEC and A549 cells (Figures 2C,D ; Figure S3 in Supplementary Material).To determine the role of HTNV infection in inducing IFITMs, qPCR as well as Western blot of IFITMs were conducted in HUVEC and A549 cells (Figures 3A,B ; Figure S1 in Supplementary Material). While we observed only a moderate upregulation of IFITM1, 2, and 3 mRNA and protein in HUVECs after more than 24 h postinfection; IFITM1, 2, and 3 mRNA, however, were only transiently upregulated in A549 cells and caused no significant change in protein level.We knocked down the IFITM1, 2, and 3 expression by transfection of their siRNAs individually. The effect of siRNAs on the expression of target IFITMs was tested by qPCR in HUVECs ( Figure S2 in Supplementary Material), and the effect of the best oligo against each IFITMs (IFITM1C, IFITM2A, IFITM3B) was tested by Western blot in A549 ( Figure 4A ) and HUVEC cells ( Figure 4B) . To assess the role of IFITMs in anti-HTNV effect of IFN-α2a, IFITM1, 2, and 3 were knocked down respectively by transfecting the above-tested oligoes for 12 h, followed by IFN-α2a treatment (20 IU/ml for another 12 h). The cells were then challenged with HTNV (moi = 1) for 24 h. The HTNV S segment and NP levels were significantly suppressed in both HUVEC and A549 cells in response to IFN-α2a treatment.Notably, knockdown of IFITM3 significantly restored the levels of HTNV S segment and NP in HUVEC and A549 cells. Knockdown of IFITM1 also partially restored the HTNV level in A549 cells (Figures 4C,D) . These results demonstrate that To assess the anti-HTNV effects of IFITMs, we tested the effect of overexpressed IFITM1, 2, and 3 on HTNV infection. c-myc-tagged IFITM1, 2, and 3 were expressed in both HUVEC and A549 cells (Figure 5A) , and the cells were then challenged with HTNV (moi = 1) for 24 h. The HTNV S segment and NP levels were suppressed by IFITM3 overexpression in HUVEC cells (Figures 5B-D) . They were also suppressed by expressing IFITM1 and IFITM3 in A549 cells (Figures 5B-D) .The inhibitory effect of IFITM3 was further confirmed by immunofluorescence analysis of HTNV NP ( Figure S3 in Supplementary Material). These results were in accordance with the above-described RNAi results. To determine whether IFITM3 inhibited HTNV binding or entry, HUVEC and A549 cells were incubated with HTNV (moi = 1) at 4°C for 1 h, unbound virus was washed away, and HTNV RNA collected at this time point represents HTNV bound to the cell surface. After virus binding, the cells were shifted to 37°C for 2 h to allow HTNV internalization, and HTNV RNA collected at this time point represents cell-internalized virus. As a positive control for inhibition of virus entry, we incubated a parallel group of cells with HTNV at pH = 5.5 as described in Section \"Materials and Methods.\" Expression of IFITM3 did not affect HTNV binding ( Figure 6A ) but significantly suppressed HTNV entry in both HUVEC and A549 cells (Figure 6B ). iFiTM3 Was Partially localized to laMP1 + late endosomes in the host cellsTo elucidate the mechanism of IFITM3 function, we investigated the subcellular localization of IFTIM3 in the host cells. IFITM3 was found partially localized to LAMP1 + late endosomes in HUVECs analyzed by confocal microscopy (Figure 6C) . The co-localization of IFITM3 and LAMP1 + late endosomes had also been found in A549 cells (27) . Because the transfer into LAMP1 + late endosomes is a necessary step for HTNV entry (28) , this result provides an evidence for the anti-HTNV mechanism of IFITM3.LncRNA-and microRNA-mediated regulation of IFITM3 has been reported in several studies. We tested the change of previously reported regulators of IFITMs, such as NRAV, NRIR, and miR-130a after HTNV infection, among which NRIR was the only changed one (downregulated) after HTNV infection ( Figure 7A ; Figure S4 in Supplementary Material) in HUVEC. However, the expression of NRIR was unchanged in A549 cells. We overexpressed NRIR in HUVEC and A549 cells using the pcDNA3.1 vector ( Figure 7B) . Importantly, overexpression of NRIR significantly suppressed IFITM3 mRNA and pre-mRNA levels and facilitated HTNV infection in HUVEC and A549 cells (Figures 7C-E) . These data suggest that lncRNA NRIR is a negative regulator of IFITM3 transcription.Hantaan virus is an enveloped, negative-sense RNA virus from the genus Hantavirus within the family Bunyaviridae. It causes HFRS, which is an important threat to public health worldwide. It is also a potential weapon for biological terrorism. Reservoir animals, usually rodents, are asymptomatic during persistent infection. Unlike in rodents, Hantavirus infection leads to HFRS and Hantavirus pulmonary syndrome (HPS) in humans (21) . The major clinical characteristics of HFRS include fever, hemorrhage, hypotension, and renal injury (1, 21) , causing severe manifestations and death in some cases. The current standard of care for HFRS relies on symptomatic and supportive treatment. It has been confirmed that the plasma viral load is associated with the severity of HFRS, implicating the importance of viremia in the pathogenesis of HFRS (2). However, no direct antiviral medications are currently available for this illness. Interferon is the key molecule for the antiviral response and has been used as an antiviral medicine in many diseases. It has been reported that HTNV infection induces a late type I interferon response (16) . However, the set of ISGs required for IFN-mediated inhibition of HTNV has not yet been identified. Therefore, identification of ISGs that are effective against HTNV is an attractive strategy to identify novel therapeutic targets.In this study, we demonstrated a significantly high frequency of the rs12252 C allele and CC genotype among HFRS patients with severe illness compared with mildly infected individuals and the healthy Han Chinese. The rs12252 C allele and CC genotype are also found to be associated with higher plasma viral load in the early stage of HFRS. We also discovered that HTNV infection induces IFITMs, and the truncated IFITM3 produced by rs12252 C allele exhibits significantly decreased anti-HTNV activity. Interestingly, IFITM3 is found to restrict HTNV infection with a mechanism of cellular entry inhibition. Indeed, IFITM3 is localized to the late endosome in the host cells, which is a necessary structure for HTNV entry. In addition, we find that HTNV infection downregulated lncRNA NRIR 48 h post infection, which negatively regulates the transcription of IFITM3. Collectively, these results suggest that IFITM3, regulated by NRIR, inhibits HTNV infection, and its SNP rs12252 correlates with the disease severity and viral load in patients with HFRS.The antiviral properties of IFITM proteins were identified in 2009 in an RNAi screen for host factors that influence influenza virus replication (29) . IFITM1, 2, and 3 have been demonstrated to possess antiviral activity in several studies. Everitt et al. demonstrated that the severity of influenza virus infection was greatly increased in IFITM3-knockout mice compared with wild-type animals (15) . Different IFITM members have also been confirmed to inhibit the cellular entry of multiple virus families (including filoviruses, rhabdoviruses, and flaviviruses) (7, (9) (10) (11) 30) . For example, HIV-1 and HCV infection are inhibited by IFITM1 (31) (32) (33) (34) . It is commonly believed that IFITMs restrict viral infection at the stage of cellular entry (12) . Recent studies suggested that the cellular location of different IFITMs may influence the range of viruses restricted by each protein (5) . IFITM1 prevents HCV entry because it colocalizes with CD81 on the cell membrane, interrupting the endocytosis of HCV particles (32) , whereas IFITM3 confines influenza virus in acidified endosomal compartments (27) . Notably, retrovirus subvirus particles (ISVPs), which do not require endosomal acidification for entry, are not inhibited by IFITM3 expression, suggesting that IFITM3 may function at the stage of endosomal entry (35) . Studies utilizing cell-cell fusion assays have suggested that IFITM3 blocks the entry of enveloped virus by preventing the fusion of the viral membrane with a limiting membrane of the host cell, either the plasma membrane and/or the endosomal membranes. The results obtained using two-photon laser scanning and fluorescence lifetime imaging (FLIM) suggest that IFITM proteins may reduce membrane fluidity and increase the spontaneous positive curvature in the outer leaflet of membranes (36) . In the present study, we demonstrated that IFN-α2a (20 U/ ml) significantly inhibited HTNV infection, siRNA-mediated depletion of IFITM3 alone significantly mitigated the antiviral effect of IFN-α2a in both HUVEC and A549 cells, whereas depletion of IFITM1 alone alleviated the antiviral effect of IFN-α2a in A549 cells. Overexpression of IFITM3 inhibited HTNV infection to HUVEC and A549 cells. IFITM1 overexpression was also effective in inhibition of HTNV in A549 cells. All these results suggest that IFITM3 is an important control factor under natural infection of HTNV. Our results also demonstrate that the effectiveness of IFITM3 is cell type-independent, which is in accordance with the results from similar viruses, such as RVFV (18) . Binding and entry assays, conducted by controlling the temperature and pH, showed that IFITM3 did not significantly influence HTNV binding but inhibited HTNV entry into HUVEC and A549 cells. Indeed, IFITM3 partially localizes to the late endosome of the host cells, which is a necessary site for the HTNV entry. However, we failed in tracking the transportation of HTNV in infected cells possibly due to the lack of fluorescence-labeled virus. In addition, IFITM1 also suppressed HTNV infection in A549 cells. The mechanism underlying anti-HTNV effect of IFITM1 remains undetermined and deserves to be further explored.According to a recent study on the three-dimensional structure of IFITM3, there is a C-terminal transmembrane α-helix and a two-N-terminal intramembrane α-helices (shown in Figure 2A as black boxes) in IFITM3 (14) . There are two splice variants that differ by the presence or absence of the first N-terminal 21 amino acids (deleted part, shown in Figure 2A as red dotted line). Several SNPs including 13 non-synonymous, 13 synonymous, 1 in-frame stop, and 1 splice site acceptoraltering have been reported in the translated IFITM3 sequence (15, 29) . Among them, the rare SNP rs12252C allele of IFITM3 truncates the protein as described above, leading to a reduced inhibition of influenza virus infection in A549 cells (15) . We demonstrated that truncated IFITM3 protein also loses the ability to inhibit HTNV infection in vitro. In Northern European patients hospitalized with seasonal influenza or pandemic influenza A virus, increased homozygosity of the minor C allele of SNP rs12252 in IFITM3 was observed (37) . In Chinese patients infected with influenza A (H1N1) virus, there was also an increased frequency of the C allele and CC genotype of SNP rs12252 (13) . In the present study, we observed an increased frequency of the C allele and CC genotype of SNP rs12252 in severely infected HFRS patients compared with healthy control and mildly affected patients. Patients carrying the CC genotype also had higher plasma viral loads compared with those with the CT/TT genotype. Given the impaired function of the IFITM3 protein produced by the C mutation, and the fact that enrichment of the rs12252 C allele in patients with severe disease and the higher viral load in patients with the CC genotype, this founding suggests that IFITM3 plays a pivotal role in the anti-HTNV response in vivo. We speculate that the much higher level of CC allele at healthy population of Han Chinese compared with Caucasians may place the Chinese at a higher risk for developing severe illness upon HTNV infection, which needs further investigation.LncRNAs are a group of non-coding RNAs longer than 200 nt that function as gene regulators, playing a role in regulating multiple cellular functions, including the innate immunity. For example, lncRNA NEAT1 is reported to be upregulated by influenza virus or PolyI:C stimulation, which promotes IL-8 expression (38) . lncRNA NRAV has been shown to negatively regulate the initial transcription of IFITM3 and Mx1 by affecting the histone modification of these genes (25) . lncRNA NRIR is a non-coding ISG, which has been reported to negatively regulate IFITM1 and Mx1 expression in HCV infection (20) . Mir-130a was also reported as a regulator of IFITM1 (23) . In this analysis, lncRNA NRIR was downregulated in HUVECs after HTNV infection for 48 h, overexpression of NRIR negatively regulates the initial transcription of IFITM3, evidenced by the decreased pre-mRNA as well as mRNA levels. NRIR overexpression also facilitated HTNV infection. These results indicate that the downregulation of NRIR after HTNV infection is possibly involved in the activation of innate immune responses against HTNV infection. We have also evaluated other potential regulators of IFITM3 before we choose NRIR for further study. Another lncRNA that can regulate IFITM3, i.e., NRAV (NR_038854), remained unchanged after HTNV infection ( Figures S4A,B in Supplementary Material). Additionally, miR-130a, which potentially regulate IFITM3, was also unaltered after HTNV infection ( Figures S4C,D in Supplementary Material).In conclusion, this study revealed a critical role for IFITM3 in HTNV infection. We demonstrated, for the first time to our knowledge, that IFITM3 is a newly identified anti-HTNV ISG; its expression is negatively regulated by NRIR; and its antiviral activity seems via a mechanism of inhibiting virus entry into the host cells. In addition, we discovered that the IFITM3 SNP rs12252 C allele and CC genotype correlates with the plasma HTNV load and the severity of HFRS; and the rs12252 C allele produces a truncated IFITM3 protein (NΔ21) that attenuates its anti-HTNV function. These results provide new insights into the role of IFITM3 in regulating innate immunity against HTNV infection, which is the basis for identifying new targets to develop novel agent against this worldwide infectious disease. aUThOr cOnTribUTiOns ZX-y, BP-y, YC-t, and MH-w performed the experiments; WP-z, BX-f, LY-f, ZY, and JZ-s designed the research; HC-x, YW, and WX analyzed the data; TK and ZC-m provided clinical data; ZX-y and BP-y wrote the paper. \n",
            "\t  0.6689262 \t Coronaviruses (CoVs), such as human coronavirus NL63 (HCoV-NL63), severe acute respiratory syndrome CoV (SARS-CoV), murine hepatitis virus (MHV), porcine epidemic diarrhea virus (PEDV), and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), encode papain-like (PL) proteases that inhibit Sendai virus-(SeV-) induced interferon (IFN-) production. Recently, the crystal structure of transmissible gastroenteritis virus (TGEV) PL1 has been solved, which was similar to that of SARS-CoV PL2 pro , which may antagonize host innate immunity. However, very little is known about whether TGEV PL1 can antagonize host innate immune response. Here, we presented evidence that TGEV PL1 encoded by the replicase gene could suppress the IFNexpression and inhibit the nuclear translocation of interferon regulatory factor 3 (IRF3). The ability to antagonize IFN-production was dependent on the intact catalytic activity of PL1. Furthermore, TGEV PL1 exerted deubiquitinase (DUB) activity which strongly inhibited the retinoic acid-induced gene I-(RIG-1-) and stimulator of interferon gene-(STING-) dependent IFN expression. Our data collectively suggest that TGEV PL1 can inhibit the IFN-expression and interfere with RIG-1-and STING-mediated signaling through a viral DUB activity. Our study has yielded strong evidence for the TGEV PL1 mechanisms that counteract the host innate immunity.The innate immune system is the first line of defense that protects the host against viral infection, and the induction of IFN-/ is a crucial antiviral mechanism of the innate immune system . The initiation of IFN expression is triggered by pathogen-associated molecular patterns (PAMPs) through host pattern recognition receptors (PRRs) . After viral RNAs are sensed by PRRs, signals are transmitted to different downstream adaptor molecules (such as IFNpromoter stimulator 1 (IPS-1)); and then I B kinase-(IKK-) related kinases are recruited. Next, interferon regulatory factor 3 (IRF3), nuclear factor B (NF-B), and ATF-2/c-jun are activated by the kinase complexes and translocate to the nucleus and directly induce the expression of type I IFNs .TGEV is an enveloped virus belonging to the Coronaviridae (CoV) family and the Nidovirales order. CoVs are positive-strand RNA viruses that replicate in the cytoplasm of infected cells . CoVs encode two types of cysteine proteases, M pro , and papain-like proteases, PL1 and PL2, which contained nonstructural protein 5 (nsp5) and nsp3, respectively. PL pro is served mainly as in processing of the replicase pp1a and pp1ab polypeptides . Other than their role in replicase polyprotein processing, PL2 domains possess an additional but related enzymatic activity, in HCoV-NL63 , MHV , SARS-CoV [8, 9] , and MERS-CoV , through their deubiquitination (DUB) enzymes, which play a key role in antagonizing IFN induction. However, TGEV PL1 processes the nsp2/nsp3 site and is capable of hydrolyzing isopeptide bonds in both Lys48-and 2 BioMed Research International Lys63-linked polyubiquitin chains . Whether TGEV PL1 could antagonize the production of IFNs was unknown.In the present study, we found that TGEV PL1 encoded by the replicase gene could suppress the IFN-expression and inhibit the nuclear translocation of interferon regulatory factor 3 (IRF3) and exerted deubiquitinase (DUB) activity which strongly inhibited the retinoic acid-induced gene I-(RIG-1-) and stimulator of interferon gene-(STING-) dependent IFN expression.Cells and Viruses. HEK293T cells and PK-15 cells were cultured in Dulbecco's modified Eagle's medium (Hyclone, Logan, USA) containing 10% (v/v) fetal calf serum supplemented with penicillin (100 U ml −1 ) and streptomycin (100 g ml −1 ). Sendai virus (SeV) was obtained from the Centre of Virus Resource and Information (Wuhan Institute of Virology, Chinese Academy of Sciences).Plasmids and Agents. IFN--Luc, 4x PRDIII/I-Luc (referred to as IRF3-Luc), 4x AP-1-Luc, and 4x NF-B-Luc luciferase reporter plasmids were constructed according to an earlier protocol . Accession numbers of STING, IRF3, and MAVS were KC860780, KC860781, and KC860782, respectively. Expression plasmids for RIG-1 (p-Flag-RIG-1) and TBK-1 (p-Flag-TBK-1) were generated with the following primers: RIG-1 forward, 5 -TTTGGATCCATGACAGCA-GAGCAGCGGCGGAAT-3 , RIG-1 reverse 5 -TTTAAG-CTTCACTCAAGGTTCGGGATTCCCTG-3 ; TBK-1 forward, 5 -TTTGAATTCATGCAGAGCACTTCTAATCAT-CTTT-3 , TBK-1 reverse 5 -TTTAGATCTTAAAGACAG-TCAACATTGCGAA-3 . To construct the DNA expression vector, pMyc-PL1, pFlag-PL1, and pPL1-Myc, encoding TGEV PL1, standard reverse transcription-(RT-) PCR was applied to amplify cDNA of the total RNA extracted from PK-15 cells infected with the TGEV strain HX, using the following primers: PL1-forward, 5 -GTACAAGAA-GCTGAACAATTTAA-3 (3498-3520 bp), PL1 reverse, 5 -ATCGTTTTTAGGACTTTGAATTT-3 (4249-4271 bp). All constructs were validated via DNA sequencing. pDsRed2-Mito was purchased from Clontech (Tokyo, Japan). Transfection agent was performed with X-tremeGENE HP (Roche, Switzerland) per the manufacturer's instructions.Luciferase Reporter Gene Assay. HEK-293T cells grown in 24-well plates were cotransfected with 0.2 g/well reporter plasmid, 0.02 g/well pRL-TK plasmid (Promega, Madison, USA) as an internal control for normalization of transfection efficiency, and the indicated expression or empty control vector plasmid. Where indicated, cells were also mockinfected or infected with SeV (100 hemagglutinating activity units/well) at 10 h after cotransfection. Cells were subsequently lysed, and firefly and Renilla luciferase activities were determined with the Dual-Luciferase reporter assay system (Promega, Madison, USA), according to the manufacturer's protocol. Data are presented as mean relative luciferase units ± standard deviation from triplicate samples. For statistical analysis, data were compared between empty vector-and TGEV PL1-transfected groups with the unpaired, two-tailed Student's t-test using SPSS 11.0 software. values < 0.05 were considered statistically significant .ELISA. Cell supernatants of transfected PK-15 cells were centrifuged at 3,000 for 5 min to remove cell debris and stored at −80 ∘ C until use. Secreted IFN-in the cell supernatants was determined using commercial Porcine IFN-(Interferon Beta) ELISA Kit (Elabscience, China) according to the manufacturer's instructions.Immunoblot Analysis. HEK293T cells were cultured in 6well plates and 60 mm dishes were transfected with the appropriate plasmids. After 36 h, cells were harvested by the addition of lysis buffer and protein concentrations in whole cell extracts measured. Equal amounts of samples were subjected to SDS-PAGE and analyzed for TGEV PL1, STING, TBK-1, and IRF3 proteins via immunoblotting using HA, Flag, or GFP-tagged antibodies (Sigma, St Louis, USA). Expression of p-IRF3, IRF3, and GAPDH was detected with the rabbit-anti p-IRF3 (ab76493), IRF3 (ab68481) (Abcam, Cambridge, UK), and a mouse anti-GAPDH monoclonal antibody (Sigma, St Louis, USA).Assay of Deubiquitinase Activity in Cultured Cells. HEK293T cells were cotransfected with pcDNA3.1-HA-Ub plus the indicated amounts of TGEV PL1, p-Flag-RIG-1, and p-Flag-STING constructs. The effect of TGEV PL1 on ubiquitinated proteins in cultured cells was assessed by immunoblot analysis.Coimmunoprecipitation Analysis. Coimmunoprecipitation experiments were performed on HEK293T cells transfected with the indicated expression plasmids as described in an earlier report .Immunofluorescence Assay. HEK293T cells were plated on fibronectin-treated glass coverslips in 24-well plates. To evaluate the localization of TGEV PL1, cells were cotransfected with plasmid DNA expressing Flag-PL (500 ng per well) and pDsRed-Mito (500 ng per well) using X-treme GENE HP, according to the manufacturer's protocol. HEK293T cells were cotransfected with IRF3-GFP (500 ng per well) and empty vector (500 ng per well) or IRF3-GFP (500 ng per well) and Flag-PL1 (500 ng per well). 24 h after transfection, cells were infected with SeV (100 hemagglutinating activity units/well) for 16 h. Next, cells were fixed with 4% paraformaldehyde for 30 min and permeated with 0.1% Triton X-100 for 15 min at room temperature. After three washes with PBS, cells were blocked with PBS containing 5% bovine serum albumin for 2 h, followed by incubation with a mouse monoclonal antibody against Flag (1 : 100) for 1 h at room temperature. Cells were treated with fluorescein isothiocyanate-labeled goat anti-mouse (Sigma, St Louis, USA) for 1 h, and subsequently with 4 ,6-diamidino-2-phenylindole (DAPI) for 15 min at room temperature. Samples were washed with PBS, and fluorescent images were acquired under a confocal laser scanning microscope (TCS SP5; Leica, Solms, Germany).To assess the formation of STING dimers, HEK293T cells were transfected with Flag-STING (500 ng per well) and the lysates were subjected to Western blot, as described earlier , with the indicated antibodies. Is an IFN Antagonist. The crystal structure of TGEV PL1 has been determined . The structure of TGEV PL1 is similar to that of SARS-CoV PL2 pro . In order to determine whether TGEV PL1 is capable of blocking IFN-production, we assessed IFN-promoter activity in the presence of PL1 (Figure 1(a) ). HEK 293T cells were cotransfected with TGEV PL1 and IFN-luciferase or Renilla luciferase reporter plasmids for 24 h and subsequently infected with SeV to activate the RIG-1-dependent IFN-expression pathway. We observed the inhibition of SeV-induced IFN-promoter activation in the presence of PL1, similar to the antagonistic function of NL63 PLP2 and porcine epidemic diarrhea virus (PEDV) PLP2, clearly indicating that TGEV PL1 could act as an interferon antagonist. To establish the mechanisms by which PL1 inhibits IFNexpression, transcriptional activities of NF-B, IRF3, and AP-1 were analyzed using the luciferase assay to identify the precise transcription factor involved. Notably, the luciferase activities of all three transcription factors were significantly inhibited by TGEV PL1 in a dose-dependent manner ( Figures 1(b) , 1(c), and 1(d)). Furthermore, Flag-PL1 also significantly inhibited IFN-production in PK-15 cells at protein level ( Figure 1(e) ), which was further confirmed with the result that TGEV PL1 could block the production of interferon. To further establish whether TGEV PL1 affects IRF3 phosphorylation or migration from the cytoplasm to nucleus, HEK293T cells were transfected with TGEV PL1 and/or IRF3-EGFP. Then the result was analyzed using Western blot and confocal microscopy. In Figure 1(f) , the level of p-IRF3 was decreased significantly by TGEV PL1 compared with that of SeV-induced. Furthermore, IRF3-EGFP was located in the cytoplasm compared with mock-infected HEK293T cells but translocated to the nucleus when the cells were inoculated with SeV. In contrast, after being inoculated with SeV, it was found that IRF3-EGFP was translocated from cytoplasm to nuclear in mock infected HEK293T cell, which was not observed in cells transfected with TGEV PL1 (Figure 1(g) ).Our results collectively suggested that TGEV PL1 suppressed IFN-transcription by interfering with NF-B-, IRF3-, and AP-1 signaling-mediated IFN expression.To determine whether TGEV PL1 is capable of blocking STINGmediated activation of the IFN-promoter, we assessed promoter activity in the presence of STING along with increasing amounts of TGEV PL1. Stimulation of HEK293T cells with STING alone resulted in a robust increase in IFNpromoter activity. Coexpression of STING and TGEV PL1 induced a dose-dependent decrease in IFN-activity, clearly indicating antagonistic activity of TGEV PL1 on STINGmediated activation of the IFN-promoter (Figure 2(a) ).STING dimerization is reduced in the presence of HCoV-NL63. STING dimmers are visualized as an 80 kD band on SDS-PAGE. To further determine whether TGEV PL1 inhibits STING-mediated signaling through disrupting the stability of STING dimers, HEK293T cells were cotransfected with plasmid DNA expressing STING in the presence or absence of TGEV PL1 and SeV, and cell lysates were evaluated for dimmers via immunoblotting (Figure 2(b) ). Interestingly, the results indicated that STING dimerization was not affected by TGEV PL1.Inhibiting IFN-Expression. To determine whether catalytic activity is required for TGEV PL1-mediated inhibition of IFN-expression, HEK293T cells were cotransfected with alanine mutants of three conserved catalytic residues of TGEV PL1 (C32A, H183A, and D196A) with or without RIG-1, MAVS, STING, or TBK-1, and IFN--Luc and pRL-TK plasmids, followed by infection with SeV to activate IFNpromoter activity. TGEV PL1 mutation at two of the catalytic sites (C32A and H183A) led to almost complete loss of IFN antagonistic activity, relative to wild-type TGEV PL1, but the D196A mutant showed a little inhibition for IFN-promoter activity (Figures 3(a), 3(b) , 3(c), 3(d), and 3(e)). Based on the results, we conclude that the intact catalytic triad of TGEV PL1 is required to inhibit activation of the IFN-promoter driven by STING and TBK-1. Recent studies have revealed that STING acts as a scaffold protein for TBK-1 and IRF3 and links them to the MAVS complex in mitochondria upon viral infection . Moreover, activation of STING is critical for stimulation of IRF-3 activity. Here, we observed that TGEV PL1 protein inhibits STING-and TBK-1-induced activation of IFN-. Additional localization experiments showed that PL1 existed in mitochondria (Figure 3(f) ).Modification of signaling molecules by ubiquitin (Ub) plays a critical role in activation of the IFN response. TGEV PL1 has been shown to possess DUB activity. Here, we investigated the DUB activities of TGEV PL1 and its catalytic mutants. HEK293T cells were cotransfected with pcDNA HA-Ub and TGEV PL1, and the level of ubiquitinated proteins was assessed via Western blot. The level of Ub-conjugated proteins was reduced dramatically in cells transfected with wild-type TGEV PL1, while the ubiquitinated Ub-HA level was not reduced in the presence of the C32A, H183A, and D196A mutants (Figure 4(a) ). Next, we investigated whether TGEV PL1 recognizes and deubiquitinates the key regulators, RIG-I and STING, in the IFN signaling pathway. TGEV is known to induce robust expression of IFN-at the late step of the replication and is distinct from CoVs [17, 18] . Moreover, TGEV infection activates transcription factors NF-B, IRF3, and AP-1 in porcine kidney cells and a delayed activation of the IFN response in intestinal epithelial cells [19, 20] . However, the mechanism of its evasion of the innate immune system has never been reported. The current study firstly showed antagonistic function of the TGEV PL1 protein against the IRF3 signaling pathway to inhibit IFNinduction through its DUB activity.To combat the host antiviral effects, coronaviruses likely take advantage of PL activity to escape from the host innate antiviral response. HCoV-NL63 (PL2-TM) and SARS-CoV (PLpro-TM) inhibit STING-mediated activation of IRF-3 nuclear translocation and induction of IRF-3-dependent promoters [6, 8] . PL2 of MHV strongly inhibits CARDIF-, TBK1-, and IRF3-mediated IFN-reporter activities and prevented nuclear translocation of IRF3 . PEDV PLP2 negatively regulated RIG-I and STING-mediated IFN-expression . Moreover, TGEV PL1 displays a similar structure to SARS-CoV PL2 and gives rise to the speculation that TGEV PL1 may similarly act as an IFN antagonist. In the present study, we first found that overexpressed TGEV PL1 inhibited STING-and TBK-1-mediated IFNtranscription and antagonized the type I IFN response stimulated by SeV in PK-15 cells. The catalytic activity of TGEV PL1 is essential for inhibiting IFN-transcription. Furthermore, STING dimerization is reduced in the presence of HCoV-NL63 PL2-TM, which was not affected by TGEV PL1. These results suggested that TGEV PL1 acted as an IFN antagonist to negatively regulate host antiviral innate immunity.Ubiquitination and deubiquitination are critically involved in regulation of virus-induced type I IFN signaling pathways [21, 22] . Recently, DUBs have been reported in a variety of viruses, such as foot-and-mouth disease virus, Lpro , human cytomegalovirus, UL48 , herpes simplex virus type 1, UL36 , and porcine reproductive and respiratory syndrome virus, nsp2 [26, 27] . Interestingly, all CoVs have evolved to encode DUB enzymes, which may contribute to modulation of the innate immune response. PLP of HCoV-NL63, SARS-CoV, MHV, PEDV, and MERS-CoV dramatically reduced the levels of ubiquitinated STING, RIG-I, TBK1, and IRF-3 to negatively regulate host antiviral innate immunity. Here, we showed that TGEV PL1 interferes with and significantly inhibits ubiquitination of RIG-1 and STING, which are essential activators of type I IFN signaling. Then, the levels of phosphorylated IRF-3 were reduced, which blocked nuclear translocation of IRF3 to activate the transcript of IFNs. Three catalytically inactive mutants of TGEV PL1 (C32A, H183A, and D196A) found to be defective in DUB activity failed to inhibit virus-induced INF-expression, indicating that the DUB function of TGEV PL1 is directly involved in inhibition of type I IFN induction. However, the membrane protein M and envelope protein E of TGEV were translated at the late step of the replication as the major inducing component of IFNs. Further studies are required to establish the precise functions of PL1 protease/DUB activity in coronavirus interactions with the host innate immune response.Our results are the first report identifying TGEV PL1 that is responsible for inhibiting the induction of IFN-. We found that TGEV PL1 displayed IFN antagonist activity dependent on the intact catalytic triad (C32, H183, and D196) and interfered with RIG-1-and STING-mediated signaling through a viral DUB activity. These characteristics of TGEV PL1 served as a multifunctional protein with a critical regulatory role in TGEV interactions with the host antiviral innate immune response. Moreover, these findings contribute to our understanding of the molecular mechanisms of innate immunity evasion strategies utilized by TGEV.\n",
            "\n",
            " Title:  Polymorphisms in dipeptidyl peptidase 4 reduce host cell entry of Middle East respiratory syndrome coronavirus\n",
            "5 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.5784194 \t Heger A, Holm L. 2001 \n",
            "\t  0.55438405 \t CamF GATATAGATTGAAAAGTGGAT Overlaps between primers vacACamR and CamF, primers vacACamF and CamR, primers ureCamF2 and CamR, primers ureCamR1 and CamF are underlined.\n",
            "\t  0.54358155 \t For the sequence information of probes and primers, please see Supplementary Table S1 . \n",
            "\t  0.53405803 \t  \"A430078G23RIK\" \"A530064D06RIK\" \"A630033H20RIK\" \"AANAT\" \"AB124611\" \"ACMSD\" \"ACOT5\" \"ACOT7\" \"ACP1\" \"ACRV1\" \"ACSL1\" \"ACTB\" \"ACTG1\" \"ACTL7A\" \"ADAM15\" \"ADAM5\" \"ADAM8\" \"ADCY9\" \"ADRA1B\" \"ADSSL1\" \"AF251705\" \"AGO2\" \"AKR1A1\" \"AKT1\" \"ALAS1\" \"ALDH2\" \"ALDH3B1\" \"ALDH3B2\" \"ALDOA\" \"ALOX12\" \"ALOX5\" \"ALOX5AP\" \"AMICA1\" \"ANK\" \"ANKRD2\" \"ANXA1\" \"ANXA11\" \"ANXA2\" \"APOBR\" \"APOL9B\" \"APP\" \"ARAP3\" \"ARG2\" \"ARHGAP18\" \"ARHGDIB\" \"ARHGEF10L\" \"ARID3A\" \"ARID5A\" \"ARMCX6\" \"ARPC1B\" \"ARPC2\" \"ASB7\" \"ASIC3\" \"ASPRV1\" \"ATF7\" \"ATP11A\" \"ATP1A3\" \"ATP2A3\" \"ATP5K\" \"ATP8B3\" \"ATP9A\" \"AU015791\" \"AW011738\" \"AW011956\" \"AW555355\" \"AZIN1\" \"B430306N03RIK\" \"B4GALT3\" \"BAK1\" \"BC052688\" \"BC094916\" \"BC100530\" \"BC147527\" \"BCL3\" \"BCL6\" \"BCR\" \"BCS1L\" \"BIN3\" \"BMP1\" \"BRD2\" \"BST1\" \"C030037D09RIK\" \"C130093G08RIK\" \"C1GALT1C1\" \"C1QL2\" \"C3\" \"C5AR2\" \"C81489\" \"CALD1\" \"CALN1\" \"CALY\" \"CAMKV\" \"CAPN2\" \"CAPZB\" \"CAR2\" \"CAR7\" \"CARHSP1\" \"CASK\" \"CASP4\" \"CBR3\" \"CCDC153\" \"CCDC167\" \"CCDC82\" \"CCDC92\" \"CCL4\" \"CCL6\" \"CCL9\" \"CCND3\" \"CCR2\" \"CCR6\" \"CD14\" \"CD151\" \"CD27\" \"CD300C\" \"CD300LD\" \"CD300LF\" \"CD302\" \"CD33\" \"CD44\" \"CD55\" \"CD80\" \"CD8B1\" \"CD93\" \"CD97\" \"CDC25B\" \"CDK2AP2\" \"CDKN2D\" \"CDR2\" \"CDS2\" \"CEBPB\" \"CEBPD\" \"CEP68\" \"CFAP74\" \"CFL1\" \"CFP\" \"CHCHD10\" \"CHIL1\" \"CHST12\" \"CHST3\" \"CHSY1\" \"CIDEB\" \"CISD3\" \"CLEC4A1\" \"CLEC4A3\" \"CLEC4E\" \"CLEC7A\" \"CLIC1\" \"CMAS\" \"CMTM6\" \"CMTR1\" \"CNN1\" \"CNN2\" \"CNST\" \"CNTN6\" \"COG5\" \"COTL1\" \"COX6B2\" \"COX6C\" \"CPD\" \"CRISPLD2\" \"CRLF2\" \"CRYGF\" \"CRYZL1\" \"CSF2RA\" \"CSF2RB\" \"CSF2RB2\" \"CSN1S2B\" \"CSPRS\" \"CSRNP1\" \"CSRP1\" \"CST3\" \"CTLA2A\" \"CTLA2B\" \"CTNNBIP1\" \"CTRL\" \"CTSA\" \"CTSC\" \"CTSD\" \"CTSS\" \"CTSZ\" \"CUEDC2\" \"CUL4A\" \"CXCR2\" \"CYBA\" \"CYP21A1\" \"CYP2B10\" \"CYP2D11\" \"CYP4A12B\" \"CYP4B1\" \"CYP4B1-PS2\" \"CYP4F17\" \"CYP4F18\" \"D030029J20RIK\" \"D130012P04RIK\" \"D2HGDH\" \"D430041D05RIK\" \"D4ERTD681E\" \"D630029K05RIK\" \"D830030K20RIK\" \"DAPP1\" \"DAXX\" \"DCAF7\" \"DCLRE1C\" \"DCUN1D1\" \"DEDD2\" \"DEFA-RS2\" \"DEFA1\" \"DEFA23\" \"DEFB9\" \"DENND5A\" \"DFNA5\" \"DGAT1\" \"DGAT2\" \"DHCR24\" \"DHRS11\" \"DHRS7\" \"DIO1\" \"DLGAP4\" \"DNAH17\" \"DNAJA4\" \"DNAJB2\" \"DNAJB6\" \"DNAJC14\" \"DOC2A\" \"DOK3\" \"DPP6\" \"DPYSL4\" \"DSTN\" \"DTNB\" \"DUSP1\" \"DUSP16\" \"DUSP23\" \"DUSP8\" \"E130119H09RIK\" \"E330014E10RIK\" \"EBI3\" \"ECM1\" \"EFHD2\" \"EFS\" \"EHBP1L1\" \"EHD4\" \"EIF2D\" \"EIF4EBP1\" \"ELL2\" \"ELOVL1\" \"EMB\" \"EME1\" \"EMILIN2\" \"ENTPD3\" \"EPHX1\" \"EPOR\" \"ERGIC3\" \"ERICH5\" \"ESD\" \"ETFB\" \"F10\" \"F11R\" \"F13A1\" \"F420015M19RIK\" \"F5\" \"F830002L21RIK\" \"F830014O18RIK\" \"FADS1\" \"FAM110A\" \"FAM216B\" \"FAM71F1\" \"FBXL5\" \"FBXO31\" \"FBXO34\" \"FBXO9\" \"FBXW21\" \"FCER1G\" \"FCGR1\" \"FCGR3\" \"FCGR4\" \"FCNAOS\" \"FERMT3\" \"FES\" \"FGF8\" \"FGFBP3\" \"FGL2\" \"FGR\" \"FHL5\" \"FKRP\" \"FLNA\" \"FLOT1\" \"FN1\" \"FOSL2\" \"FPR1\" \"FPR2\" \"FUOM\" \"FURIN\" \"FYB\" \"G6B\" \"G6PDX\" \"GABARAPL1\" \"GALT\" \"GAPDH\" \"GAS2L1\" \"GATA1\" \"GATA2\" \"GBP6\" \"GBP7\" \"GBP9\" \"GCNT2\" \"GDAP10\" \"GDPD2\" \"GFI1B\" \"GFRA4\" \"GKN3\" \"GLIPR1\" \"GLRX5\" \"GM10272\" \"GM10693\" \"GM10845\" \"GM11127\" \"GM11559\" \"GM11564\" \"GM11937\" \"GM11938\" \"GM12060\" \"GM12070\" \"GM12250\" \"GM12271\" \"GM13375\" \"GM14206\" \"GM14446\" \"GM14548\" \"GM15217\" \"GM15645\" \"GM15800\" \"GM1604B\" \"GM16501\" \"GM16510\" \"GM1966\" \"GM2044\" \"GM2437\" \"GM2694\" \"GM2703\" \"GM2744\" \"GM3035\" \"GM3051\" \"GM3181\" \"GM3224\" \"GM3265\" \"GM3448\" \"GM3455\" \"GM3627\" \"GM3652\" \"GM3806\" \"GM3848\" \"GM4013\" \"GM4326\" \"GM4653\" \"GM4871\" \"GM4951\" \"GM4955\" \"GM5068\" \"GM5069\" \"GM5150\" \"GM5483\" \"GM5523\" \"GM5595\" \"GM5935\" \"GM5938\" \"GM6132\" \"GM6293\" \"GM6498\" \"GM6934\" \"GM6981\" \"GM7193\" \"GM7475\" \"GM813\" \"GM826\" \"GM8709\" \"GM8884\" \"GM8909\" \"GM8995\" \"GM9706\" \"GM9733\" \"GNAI2\" \"GNB2\" \"GNGT2\" \"GP1BA\" \"GP49A\" \"GP5\" \"GP6\" \"GP9\" \"GPR137B-PS\" \"GPR157\" \"GPR31B\" \"GPR56\" \"GRWD1\" \"GSDMC\" \"GSDMD\" \"GSR\" \"GSTT1\" \"GTPBP3\" \"GVIN1\" \"GYG\" \"H13\" \"H19\" \"H2-BL\" \"H2-D1\" \"H2-K1\" \"H2-K2\" \"H2-M2\" \"H2-OB\" \"H2-Q10\" \"H2-Q2\" \"H2-Q6\" \"H2-Q7\" \"H2-Q8\" \"H2-T10\" \"H2-T23\" \"H2-T24\" \"H2-T9\" \"HBB-BH2\" \"HBQ1B\" \"HCAR2\" \"HCK\" \"HCLS1\" \"HDAC4\" \"HDC\" \"HEBP1\" \"HES7\" \"HEXIM2\" \"HEYL\" \"HGSNAT\" \"HHAT\" \"HIPK1\" \"HIST1H3C\" \"HIST2H2AA1\" \"HIST3H2BA\" \"HMGCL\" \"HMGN2\" \"HMOX1\" \"HPCAL1\" \"HSCB\" \"HSD17B11\" \"HSH2D\" \"I830012O16RIK\" \"I830077J02RIK\" \"I830127L07RIK\" \"ICA1L\" \"IFI202B\" \"IFI203\" \"IFI204\" \"IFI27L2A\" \"IFI47\" \"IFITM2\" \"IFITM6\" \"IFNA14\" \"IFRD2\" \"IGF1\" \"IGFBP6\" \"IGHV14-2\" \"IGSF6\" \"IGSF9B\" \"IGTP\" \"IL18BP\" \"IL1B\" \"IL1F9\" \"IL1R2\" \"IL4I1\" \"IL6RA\" \"ILK\" \"IMPG1\" \"INF2\" \"IRF1\" \"IRF6\" \"IRG1\" \"IRGM1\" \"IRGM2\" \"ITGA6\" \"ITGAL\" \"ITGB1\" \"ITGB5\" \"ITM2B\" \"JARID2\" \"JUN\" \"KCND3\" \"KCNN3\" \"KLHL25\" \"KLHL32\" \"KLK1B4\" \"KLRA2\" \"KPNA6\" \"KRT6B\" \"KRT73\" \"LAMP1\" \"LAMP2\" \"LANCL3\" \"LARP1B\" \"LCP1\" \"LEFTY2\" \"LGALS1\" \"LGALS3\" \"LGALSL\" \"LHX9\" \"LIN52\" \"LITAF\" \"LOC100038947\" \"LOC102636217\" \"LOC547349\" \"LOC676689\" \"LOC68395\" \"LRG1\" \"LRRC25\" \"LRRTM1\" \"LSP1\" \"LST1\" \"LTB4R1\" \"LTBP1\" \"LTBR\" \"LY6A\" \"LY6C1\" \"LY6G6C\" \"LY6K\" \"LYZ1\" \"LYZ2\" \"MADCAM1\" \"MAGED2\" \"MARCH7\" \"MARCH8\" \"MARCKS\" \"MARCKSL1\" \"MAST2\" \"MBD3\" \"MBNL2\" \"MBOAT7\" \"MC3R\" \"MCL1\" \"MCUR1\" \"MEF2C\" \"MEFV\" \"MEIS3\" \"MEMO1\" \"MESP2\" \"METAP2\" \"METRNL\" \"METTL7A1\" \"MFSD6\" \"MGAT5B\" \"MGMT\" \"MGST1\" \"MIDN\" \"MLANA\" \"MMD\" \"MMP17\" \"MMP8\" \"MNDA\" \"MOCOS\" \"MOV10\" \"MPC2\" \"MPEG1\" \"MPL\" \"MRAP2\" \"MRAS\" \"MRGBP\" \"MRGPRA2A\" \"MRGPRA2B\" \"MRGPRA6\" \"MRPL33\" \"MRVI1\" \"MS4A4C\" \"MS4A6C\" \"MS4A6D\" \"MS4A8A\" \"MSGN1\" \"MSRA\" \"MSRB1\" \"MTMR14\" \"MTMR3\" \"MTUS1\" \"MVB12A\" \"MYD88\" \"MYL3\" \"MYO1D\" \"MYO1F\" \"NAAA\" \"NADK\" \"NADK2\" \"NAIP2\" \"NAMPT\" \"NCF2\" \"NCF4\" \"NCK2\" \"NCOA1\" \"NDEL1\" \"NES\" \"NFAM1\" \"NFE2\" \"NFIL3\" \"NFKBIA\" \"NGFRAP1\" \"NINJ1\" \"NKIRAS2\" \"NKX2-2\" \"NLRC5\" \"NLRX1\" \"NME1\" \"NOC4L\" \"NOD1\" \"NOD2\" \"NOMO1\" \"NOS1\" \"NPRL3\" \"NPTN\" \"NRGN\" \"NRON\" \"NRROS\" \"NRTN\" \"NUAK1\" \"NUDT3\" \"NUDT4\" \"NUPR1\" \"OAS1A\" \"OAS1C\" \"OAS1F\" \"OASL1\" \"OASL2\" \"OBOX1\" \"ODC1\" \"OGDH\" \"OGFR\" \"OGT\" \"OLFR1211\" \"OLFR1466\" \"OLFR209\" \"OLFR324\" \"OLFR394\" \"OLFR406\" \"OLFR458\" \"OLFR522\" \"OLFR545\" \"OLFR631\" \"OLFR635\" \"OLFR700\" \"OLFR704\" \"ONECUT2\" \"OSER1\" \"OSGIN1\" \"OSTF1\" \"OTOS\" \"OTUD5\" \"OTUD7B\" \"OVCH2\" \"P2RY13\" \"PABPC1\" \"PABPC4L\" \"PACSIN2\" \"PAQR6\" \"PARVB\" \"PCGF5\" \"PCX\" \"PDGFA\" \"PDLIM7\" \"PEX11G\" \"PF4\" \"PFKFB4\" \"PGA5\" \"PGAM1\" \"PGD\" \"PGLYRP1\" \"PGM5\" \"PHF11A\" \"PHF11B\" \"PHF11D\" \"PHLDA3\" \"PI16\" \"PIK3R6\" \"PILRA\" \"PILRB2\" \"PIN1RT1\" \"PIRA11\" \"PIRA2\" \"PIRA6\" \"PIRA7\" \"PIRB\" \"PKM\" \"PLA2G7\" \"PLAUR\" \"PLD4\" \"PLEK\" \"PLEKHG5\" \"PLEKHO2\" \"PLIN2\" \"PLP1\" \"PLXNA4\" \"PLXNB2\" \"PNKP\" \"PNLDC1\" \"PNMA3\" \"PNP\" \"PODNL1\" \"POLR2A\" \"POLR3H\" \"POR\" \"POU3F1\" \"POU6F2\" \"PPBP\" \"PPDPF\" \"PPIF\" \"PPOX\" \"PPP1R14C\" \"PPP1R15A\" \"PPP1R2\" \"PPP1R3D\" \"PRAM1\" \"PRDX2\" \"PRDX5\" \"PRDX6\" \"PREB\" \"PRKAB1\" \"PRKAB2\" \"PRKX\" \"PRL2A1\" \"PRMT7\" \"PRND\" \"PROSER2\" \"PRR5L\" \"PRRG2\" \"PRRT1\" \"PSAP\" \"PSMB10\" \"PSMB5\" \"PSMB8\" \"PTCHD1\" \"PTGS1\" \"PTK2B\" \"PTP4A3\" \"PTPLAD2\" \"PTPN1\" \"PTPN11\" \"PTPN18\" \"PTPRD\" \"PTPRH\" \"PTPRJ\" \"PTRF\" \"PTTG1IP\" \"PYDC3\" \"PYGL\" \"PYHIN1\" \"QK\" \"QSOX1\" \"RAB31\" \"RAB32\" \"RAB3IL1\" \"RAB44\" \"RALB\" \"RAN\" \"RANBP10\" \"RARA\" \"RASGRF2\" \"RASGRP2\" \"RASGRP4\" \"RB1\" \"RBBP4\" \"RBM19\" \"RBM3\" \"RBM33\" \"RBM47\" \"RBPMS2\" \"REN2\" \"REP15\" \"RETNLG\" \"RFK\" \"RFXANK\" \"RGS2\" \"RGS20\" \"RGS21\" \"RGS3\" \"RGS7\" \"RHOG\" \"RHOU\" \"RHOX7\" \"RND1\" \"RNF11\" \"RNF114\" \"RNF149\" \"RNF19B\" \"RNF31\" \"RNPEP\" \"RSU1\" \"RTN3\" \"S100A11\" \"S100A4\" \"SAA1\" \"SAA3\" \"SAMHD1\" \"SBNO2\" \"SCAMP1\" \"SCARB1\" \"SCGB1A1\" \"SCNN1A\" \"SCRG1\" \"SEC61B\" \"SELK\" \"SELPLG\" \"SEMA4A\" \"SEMA6B\" \"SENP3\" \"SERPINA3G\" \"SERPINB6A\" \"SERPINB6C\" \"SFN\" \"SFXN5\" \"SH3BGRL3\" \"SH3BP2\" \"SH3D21\" \"SHISA9\" \"SIGMAR1\" \"SIRPA\" \"SIRPB1A\" \"SIRPB1B\" \"SKAP2\" \"SLA\" \"SLAMF1\" \"SLC10A1\" \"SLC11A1\" \"SLC15A3\" \"SLC16A3\" \"SLC22A1\" \"SLC24A3\" \"SLC25A51\" \"SLC27A4\" \"SLC28A2\" \"SLC2A3\" \"SLC2A6\" \"SLC38A5\" \"SLC39A1\" \"SLC40A1\" \"SLC44A1\" \"SLC4A2\" \"SLC6A1\" \"SLC6A4\" \"SLC7A7\" \"SLFN1\" \"SLFN10-PS\" \"SLFN13L\" \"SLFN2\" \"SLFN3\" \"SLFN4\" \"SLPI\" \"SMC4\" \"SMIM3\" \"SMPDL3A\" \"SMTN\" \"SNAP23\" \"SNRNP25\" \"SNX15\" \"SNX20\" \"SNX32\" \"SNX9\" \"SOCS3\" \"SOD2\" \"SORD\" \"SOWAHA\" \"SPAG17OS\" \"SPEER1-PS1\" \"SPEF1\" \"SPI1\" \"SPOP\" \"SPPL2A\" \"SPRN\" \"SPRR1A\" \"SPTA1\" \"SQRDL\" \"SRM\" \"SRPX2\" \"SRXN1\" \"SSBP2\" \"SSBP4\" \"SSX9\" \"SSXB9\" \"ST3GAL5\" \"STARD3\" \"STAT3\" \"STFA1\" \"STFA2\" \"STFA2L1\" \"STFA3\" \"STK40\" \"SVIL\" \"SYCN\" \"TAAR4\" \"TAB3\" \"TAGLN\" \"TAL1\" \"TAPBP\" \"TAS2R131\" \"TBC1D2\" \"TBC1D22A\" \"TBKBP1\" \"TBXAS1\" \"TCSTV1\" \"TCTE1\" \"TCTEX1D2\" \"TDRP\" \"TEAD1\" \"TENM4\" \"TFAP2D\" \"TFDP2\" \"TFE3\" \"TGFB1\" \"TGFB1I1\" \"TGIF2\" \"TGM2\" \"TGTP2\" \"THBS1\" \"THEMIS2\" \"TIMM10B\" \"TIMM17B\" \"TIMP2\" \"TIRAP\" \"TLDC2\" \"TLN1\" \"TLR13\" \"TLR2\" \"TLR6\" \"TM4SF5\" \"TMA7\" \"TMBIM1\" \"TMBIM6\" \"TMEM116\" \"TMEM178\" \"TMEM219\" \"TMEM246\" \"TMEM256\" \"TMEM29\" \"TMEM40\" \"TMEM50B\" \"TMEM87A\" \"TMEM8C\" \"TMOD4\" \"TMSB4X\" \"TMTC3\" \"TNFAIP2\" \"TNFRSF1A\" \"TNFRSF1B\" \"TNFSF14\" \"TNP1\" \"TOM1\" \"TOR1AIP1\" \"TOR1AIP2\" \"TOR3A\" \"TPD52\" \"TPI1\" \"TPM1\" \"TPM2\" \"TPM3\"[ reached getOption(\"max.print\") --omitted 100 entries ] $Tang \"A1BG\" \"A2M-AS1\" \"ABCA1\" \"ABHD14B\" \"ABLIM1\" \"ACADSB\" \"ACAT1\" \"ACCS\" \"ACKR3\" \"ACOT13\" \"ACOXL\" \"ACRBP\" \"ADAMTS10\" \"ADARB1\" \"ADCY4\" \"ADRB2\" \"AES\" \"AKAP11\" \"AKIRIN2\" \"ALG13\" \"AMIGO1\" \"AMPD2\" \"ANK3\" \"ANKRD22\" \"ANKRD36\" \"ANKRD46\" \"ANKS6\" \"ANXA8L1\" \"AP5B1\" \"APBA2\" \"APOBEC3B\" \"APOBEC3H\" \"AQP1\" \"AQP10\" \"ARG1\" \"ARHGAP5\" \"ARHGEF40\" \"ARL17A\" \"ARL4A\" \"ARL4C\" \"ARNTL\" \"ARPC3\" \"ARPC5\" \"ASGR2\" \"ASPH\" \"ASPM\" \"ATF1\" \"ATP5E\" \"ATP5I\" \"ATP5J\" \"ATP6\" \"ATP6V0E2-AS1\" \"AUTS2\" \"B4GALT5\" \"BAALC\" \"BACH2\" \"BAHD1\" \"BANK1\" \"BATF2\" \"BCL11A\" \"BCL11B\" \"BCL2A1\" \"BIRC3\" \"BIRC5\" \"BLK\" \"BRD1\" \"BTG1\" \"BTLA\" \"BTN3A1\" \"BUB1\" \"BZRAP1-AS1\" \"C10orf105\" \"C12orf57\" \"C16orf74\" \"C1orf52\" \"C20orf141\" \"C2orf40\" \"C2orf88\" \"C4orf33\" \"C5orf45\" \"C6orf48\" \"CA2\" \"CACNA1E\" \"CACNG6\" \"CAMK1D\" \"CAMP\" \"CARD11\" \"CARD16\" \"CARD17\" \"CARD6\" \"CASC15\" \"CASC5\" \"CBLB\" \"CBX7\" \"CCDC102A\" \"CCDC144A\" \"CCDC147-AS1\" \"CCNB1\" \"CCNB2\" \"CCR3\" \"CCR7\" \"CD164\" \"CD177\" \"CD19\" \"CD2\" \"CD22\" \"CD247\" \"CD28\" \"CD300E\" \"CD3E\" \"CD40LG\" \"CD5\" \"CD6\" \"CD7\" \"CD74\" \"CD79B\" \"CD81\" \"CD83\" \"CD8A\" \"CD8B\" \"CD96\" \"CDC20\" \"CDC42SE1\" \"CDC45\" \"CDCA2\" \"CDCA3\" \"CDCA5\" \"CDK1\" \"CDK11A\" \"CDK11B\" \"CDKN2C\" \"CDKN3\" \"CDT1\" \"CDV3\" \"CENPF\" \"CENPM\" \"CENPN\" \"CENPU\" \"CENPW\" \"CEP19\" \"CES1\" \"CFAP97\" \"CHEK1\" \"CHIT1\" \"CHPT1\" \"CHRM3-AS2\" \"CISD2\" \"CIT\" \"CKAP4\" \"CKLF\" \"CKS1B\" \"CKS2\" \"CLEC10A\" \"CLEC11A\" \"CLEC12A\" \"CLUHP3\" \"CMTM5\" \"CNIH4\" \"CNR2\" \"CNTNAP3\" \"COQ10A\" \"COX17\" \"COX7A2\" \"COX7B\" \"CR1\" \"CR2\" \"CREG1\" \"CRHR1-IT1\" \"CRIP2\" \"CROCC\" \"CRTC1\" \"CSGALNACT2\" \"CSTA\" \"CTNNAL1\" \"CTSL\" \"CTTN\" \"CXCR5\" \"CYB561A3\" \"CYSTM1\" \"CYTH3\" \"DCAF16\" \"DDAH2\" \"DEFA8P\" \"DENND2D\" \"DHFR\" \"DIAPH3\" \"DISC1\" \"DLGAP5\" \"DNAJA1P5\" \"DNAJA3\" \"DNAJC30\" \"DNAJC5\" \"DOCK10\" \"DPP4\" \"DPRXP4\" \"DRC1\" \"DTL\" \"DUSP13\" \"DUT\" \"DYRK2\" \"E2F1\" \"E2F7\" \"ECRP\" \"EEF1A1\" \"EEF2K\" \"EIF1AX\" \"EIF3F\" \"EIF4A2\" \"EIF4B\" \"EIF4G3\" \"ENHO\" \"ENO2\" \"ENY2\" \"EOMES\" \"EPHX2\" \"ERN1\" \"ESPL1\" \"ESRG\" \"EVL\" \"EZH2\" \"FABP5\" \"FAIM3\" \"FAM102A\" \"FAM117B\" \"FAM129C\" \"FAM134B\" \"FAM155A-IT1\" \"FAM159A\" \"FAM177B\" \"FAM20A\" \"FAM26F\" \"FAM27C\" \"FANCF\" \"FANCI\" \"FBL\" \"FBLN5\" \"FBXL16\" \"FCAR\" \"FCER1A\" \"FCER2\" \"FCGBP\" \"FCGR1A\" \"FCN1\" \"FCRL1\" \"FCRL2\" \"FCRL3\" \"FCRL6\" \"FCRLA\" \"FFAR3\" \"FGD4\" \"FLT3LG\" \"FOXJ3\" \"FOXM1\" \"FOXO1\" \"FRMD3\" \"FSD1\" \"FSTL3\" \"FTL\" \"G0S2\" \"GADD45A\" \"GADD45G\" \"GALM\" \"GALNT14\" \"GATA3\" \"GCSAM\" \"GDF15\" \"GINS1\" \"GINS2\" \"GLCCI1\" \"GLIPR2\" \"GLS\" \"GLTSCR2\" \"GMFG\" \"GMNN\" \"GNB2L1\" \"GNG10\" \"GOLGA8A\" \"GPA33\" \"GPD2\" \"GPER1\" \"GPR141\" \"GPR174\" \"GPR18\" \"GPR183\" \"GPR68\" \"GPR84\" \"GPRASP1\" \"GRB10\" \"GRPEL2\" \"GSTM3\" \"GTF2H5\" \"GTSE1\" \"GVINP1\" \"GYG1\" \"GZMK\" \"GZMM\" \"H1F0\" \"HABP4\" \"HAPLN3\" \"HBM\" \"HCG18\" \"HELZ2\" \"HES6\" \"HIP1R\" \"HIST1H1B\" \"HIST1H2AC\" \"HIST1H2AD\" \"HIST1H2AE\" \"HIST1H2AH\" \"HIST1H2AI\" \"HIST1H2AJ\" \"HIST1H2AL\" \"HIST1H2AM\" \"HIST1H2BB\" \"HIST1H2BC\" \"HIST1H2BD\" \"HIST1H2BE\" \"HIST1H2BF\" \"HIST1H2BG\" \"HIST1H2BH\" \"HIST1H2BI\" \"HIST1H2BJ\" \"HIST1H2BK\" \"HIST1H2BL\" \"HIST1H2BM\" \"HIST1H2BO\" \"HIST1H3B\" \"HIST1H3D\" \"HIST1H3F\" \"HIST1H3H\" \"HIST1H3J\" \"HIST1H4D\" \"HIST1H4F\" \"HIST1H4K\" \"HIST1H4L\" \"HIST2H2AA4\" \"HIST2H2AB\" \"HIST2H3A\" \"HJURP\" \"HLA-DOA\" \"HLA-DOB\" \"HLA-DPA1\" \"HLA-DPB1\" \"HLA-DPB2\" \"HLA-DQA2\" \"HLA-DQB1\" \"HLA-DQB2\" \"HLA-DRA\" \"HLA-DRB3\" \"HMGB1\" \"HMGB3P1\" \"HMMR\" \"HNRNPA1\" \"HNRNPH1\" \"HOXB2\" \"HP1BP3\" \"HPR\" \"HRASLS2\" \"HS3ST1\" \"HSP90AA1\" \"ICOS\" \"IFFO2\" \"IFI27L1\" \"IFI27L2\" \"IFITM4P\" \"IFT20\" \"IGFBP7\" \"IGJ\" \"IGLL1\" \"IGLL5\" \"IKZF2\" \"IL10RB-AS1\" \"IL11RA\" \"IL18R1\" \"IL18RAP\" \"IL1RAP\" \"IL23A\" \"IL27\" \"IL2RB\" \"IL32\" \"IL7R\" \"IMP3\" \"ING5\" \"INHBA\" \"INPP4B\" \"INPP5E\" \"IRAK3\" \"ISCA1\" \"ISY1\" \"ITGA7\" \"ITIH4\" \"ITK\" \"JDP2\" \"JPH4\" \"KCNG1\" \"KDF1\" \"KIAA0101\" \"KIAA0226L\" \"KIAA0319L\" \"KIAA0355\" \"KIAA1147\" \"KIF11\" \"KIF1B\" \"KIF2C\" \"KLF12\" \"KLHDC7B\" \"KLHDC8B\" \"KLHL3\" \"KLRB1\" \"KLRC3\" \"KLRC4\" \"KLRG1\" \"KRTAP10-6\" \"LAT\" \"LBH\" \"LCK\" \"LDLRAP1\" \"LEF1-AS1\" \"LEPROTL1\" \"LILRA3\" \"LILRA5\" \"LILRB3\" \"LIN7A\" \"LIPN\" \"LLGL2\" \"LMF1\" \"LOXL3\" \"LPIN1\" \"LRFN3\" \"LRIG1\" \"LRRC47\" \"LRRN3\" \"LSM3\" \"LTB4R\" \"LTBP3\" \"LTF\" \"LY9\" \"LY96\" \"MAL\" \"MAMSTR\" \"MAN1C1\" \"MAOA\" \"MAP1LC3B\" \"MAPK14\" \"MATK\" \"MB21D1\" \"MCF2L-AS1\" \"MCM2\" \"MCM4\" \"MCTP1\" \"MEAF6\" \"MEGF6\" \"MEX3C\" \"MEX3D\" \"MGAM\" \"MGC16025\" \"MGLL\" \"MILR1\" \"MIR146A\" \"MIR4435-1HG\" \"MKI67\" \"MLLT6\" \"MME\" \"MOAP1\" \"MPO\" \"MRC2\" \"MRPL22\" \"MRPL27\" \"MRPL51\" \"MRPS18C\" \"MS4A1\" \"MSN\" \"MSX2P1\" \"MT1B\" \"MT1E\" \"MT1HL1\" \"MT1L\" \"MT1M\" \"MTA1\" \"MXD4\" \"MXI1\" \"MXRA7\" \"MYBL1\" \"MYBL2\" \"MYBPC3\" \"MYC\" \"MYCL\" \"MYL6\" \"MYL6B\" \"NAB2\" \"NAIP\" \"NAP1L1\" \"NCR3\" \"ND6\" \"NDST2\" \"NDUFA1\" \"NDUFA4\" \"NDUFA6\" \"NDUFB3\" \"NELL2\" \"NEXN\" \"NKTR\" \"NKX3-1\" \"NLRC3\" \"NLRC4\" \"NMUR1\" \"NOP10\" \"NRIR\" \"NSG1\" \"NT5C3A\" \"NT5E\" \"NT5M\" \"NTNG2\" \"NTSR1\" \"NUSAP1\" \"OBFC1\" \"ODF3B\" \"OIP5\" \"OPLAH\" \"OR52K2\" \"OSCAR\" \"OXNAD1\" \"P2RX5\" \"P2RY10\" \"PASK\" \"PCED1B\" \"PCMT1\" \"PCNA\" \"PCSK1N\" \"PDE4DIP\" \"PDP1\" \"PEBP1\" \"PHC3\" \"PHF19\" \"PID1\" \"PIK3AP1\" \"PIK3C2B\" \"PIN4\" \"PIWIL4\" \"PJA1\" \"PKIA\" \"PKMYT1\" \"PLBD1\" \"PLCG1\" \"PLEKHA1\" \"PLEKHB1\" \"PLEKHG4\" \"PLXDC1\" \"PLXNA1\" \"PMEPA1\" \"PNMA1\" \"PNRC1\" \"POLG2\" \"POLRMT\" \"PP7080\" \"PPARD\" \"PPIB\" \"PQLC1\" \"PRC1\" \"PRDX4\" \"PREPL\" \"PRKAG2-AS1\" \"PRKAR1B\" \"PRKAR2B\" \"PRKCA\" \"PRKCH\" \"PRKCQ-AS1\" \"PRNP\" \"PRPF8\" \"PRRG4\" \"PRUNE2\" \"PSIP1\" \"PSMA2\" \"PSMA3\" \"PSMA4\" \"PSMA6\" \"PSMC2\" \"PSMF1\" \"PTCRA\" \"PTGDR\" \"PTGDR2\" \"PTPRK\" \"PTTG1\" \"PTTG2\" \"PTX3\" \"PURA\" \"PYCR1\" \"RAB11FIP3\" \"RAB13\" \"RABL2A\" \"RAD21\" \"RAD51\" \"RAD54L\" \"RAI1\" \"RALGAPA1\" \"RARA-AS1\" \"RASGRP1\" \"RBM20\" \"RBX1\" \"RCAN3\" \"REC8\" \"RECQL4\" \"RETN\" \"RFTN1\" \"RGL4\" \"RHOH\" \"RMI2\" \"RNA5-8S5\" \"RNASE1\" \"RNF208\" \"RNF32\" \"RNF44\" \"RORC\" \"RPA3\" \"RPL10A\" \"RPL10L\"\n",
            "\n",
            " Title:  nan\n",
            "6 \b.\n",
            "Top-4 Answers: \n",
            "\n",
            "\t  0.7287808 \t Coronaviruses (CoVs), such as human coronavirus NL63 (HCoV-NL63), severe acute respiratory syndrome CoV (SARS-CoV), murine hepatitis virus (MHV), porcine epidemic diarrhea virus (PEDV), and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), encode papain-like (PL) proteases that inhibit Sendai virus-(SeV-) induced interferon (IFN-) production. Recently, the crystal structure of transmissible gastroenteritis virus (TGEV) PL1 has been solved, which was similar to that of SARS-CoV PL2 pro , which may antagonize host innate immunity. However, very little is known about whether TGEV PL1 can antagonize host innate immune response. Here, we presented evidence that TGEV PL1 encoded by the replicase gene could suppress the IFNexpression and inhibit the nuclear translocation of interferon regulatory factor 3 (IRF3). The ability to antagonize IFN-production was dependent on the intact catalytic activity of PL1. Furthermore, TGEV PL1 exerted deubiquitinase (DUB) activity which strongly inhibited the retinoic acid-induced gene I-(RIG-1-) and stimulator of interferon gene-(STING-) dependent IFN expression. Our data collectively suggest that TGEV PL1 can inhibit the IFN-expression and interfere with RIG-1-and STING-mediated signaling through a viral DUB activity. Our study has yielded strong evidence for the TGEV PL1 mechanisms that counteract the host innate immunity.The innate immune system is the first line of defense that protects the host against viral infection, and the induction of IFN-/ is a crucial antiviral mechanism of the innate immune system . The initiation of IFN expression is triggered by pathogen-associated molecular patterns (PAMPs) through host pattern recognition receptors (PRRs) . After viral RNAs are sensed by PRRs, signals are transmitted to different downstream adaptor molecules (such as IFNpromoter stimulator 1 (IPS-1)); and then I B kinase-(IKK-) related kinases are recruited. Next, interferon regulatory factor 3 (IRF3), nuclear factor B (NF-B), and ATF-2/c-jun are activated by the kinase complexes and translocate to the nucleus and directly induce the expression of type I IFNs .TGEV is an enveloped virus belonging to the Coronaviridae (CoV) family and the Nidovirales order. CoVs are positive-strand RNA viruses that replicate in the cytoplasm of infected cells . CoVs encode two types of cysteine proteases, M pro , and papain-like proteases, PL1 and PL2, which contained nonstructural protein 5 (nsp5) and nsp3, respectively. PL pro is served mainly as in processing of the replicase pp1a and pp1ab polypeptides . Other than their role in replicase polyprotein processing, PL2 domains possess an additional but related enzymatic activity, in HCoV-NL63 , MHV , SARS-CoV [8, 9] , and MERS-CoV , through their deubiquitination (DUB) enzymes, which play a key role in antagonizing IFN induction. However, TGEV PL1 processes the nsp2/nsp3 site and is capable of hydrolyzing isopeptide bonds in both Lys48-and 2 BioMed Research International Lys63-linked polyubiquitin chains . Whether TGEV PL1 could antagonize the production of IFNs was unknown.In the present study, we found that TGEV PL1 encoded by the replicase gene could suppress the IFN-expression and inhibit the nuclear translocation of interferon regulatory factor 3 (IRF3) and exerted deubiquitinase (DUB) activity which strongly inhibited the retinoic acid-induced gene I-(RIG-1-) and stimulator of interferon gene-(STING-) dependent IFN expression.Cells and Viruses. HEK293T cells and PK-15 cells were cultured in Dulbecco's modified Eagle's medium (Hyclone, Logan, USA) containing 10% (v/v) fetal calf serum supplemented with penicillin (100 U ml −1 ) and streptomycin (100 g ml −1 ). Sendai virus (SeV) was obtained from the Centre of Virus Resource and Information (Wuhan Institute of Virology, Chinese Academy of Sciences).Plasmids and Agents. IFN--Luc, 4x PRDIII/I-Luc (referred to as IRF3-Luc), 4x AP-1-Luc, and 4x NF-B-Luc luciferase reporter plasmids were constructed according to an earlier protocol . Accession numbers of STING, IRF3, and MAVS were KC860780, KC860781, and KC860782, respectively. Expression plasmids for RIG-1 (p-Flag-RIG-1) and TBK-1 (p-Flag-TBK-1) were generated with the following primers: RIG-1 forward, 5 -TTTGGATCCATGACAGCA-GAGCAGCGGCGGAAT-3 , RIG-1 reverse 5 -TTTAAG-CTTCACTCAAGGTTCGGGATTCCCTG-3 ; TBK-1 forward, 5 -TTTGAATTCATGCAGAGCACTTCTAATCAT-CTTT-3 , TBK-1 reverse 5 -TTTAGATCTTAAAGACAG-TCAACATTGCGAA-3 . To construct the DNA expression vector, pMyc-PL1, pFlag-PL1, and pPL1-Myc, encoding TGEV PL1, standard reverse transcription-(RT-) PCR was applied to amplify cDNA of the total RNA extracted from PK-15 cells infected with the TGEV strain HX, using the following primers: PL1-forward, 5 -GTACAAGAA-GCTGAACAATTTAA-3 (3498-3520 bp), PL1 reverse, 5 -ATCGTTTTTAGGACTTTGAATTT-3 (4249-4271 bp). All constructs were validated via DNA sequencing. pDsRed2-Mito was purchased from Clontech (Tokyo, Japan). Transfection agent was performed with X-tremeGENE HP (Roche, Switzerland) per the manufacturer's instructions.Luciferase Reporter Gene Assay. HEK-293T cells grown in 24-well plates were cotransfected with 0.2 g/well reporter plasmid, 0.02 g/well pRL-TK plasmid (Promega, Madison, USA) as an internal control for normalization of transfection efficiency, and the indicated expression or empty control vector plasmid. Where indicated, cells were also mockinfected or infected with SeV (100 hemagglutinating activity units/well) at 10 h after cotransfection. Cells were subsequently lysed, and firefly and Renilla luciferase activities were determined with the Dual-Luciferase reporter assay system (Promega, Madison, USA), according to the manufacturer's protocol. Data are presented as mean relative luciferase units ± standard deviation from triplicate samples. For statistical analysis, data were compared between empty vector-and TGEV PL1-transfected groups with the unpaired, two-tailed Student's t-test using SPSS 11.0 software. values < 0.05 were considered statistically significant .ELISA. Cell supernatants of transfected PK-15 cells were centrifuged at 3,000 for 5 min to remove cell debris and stored at −80 ∘ C until use. Secreted IFN-in the cell supernatants was determined using commercial Porcine IFN-(Interferon Beta) ELISA Kit (Elabscience, China) according to the manufacturer's instructions.Immunoblot Analysis. HEK293T cells were cultured in 6well plates and 60 mm dishes were transfected with the appropriate plasmids. After 36 h, cells were harvested by the addition of lysis buffer and protein concentrations in whole cell extracts measured. Equal amounts of samples were subjected to SDS-PAGE and analyzed for TGEV PL1, STING, TBK-1, and IRF3 proteins via immunoblotting using HA, Flag, or GFP-tagged antibodies (Sigma, St Louis, USA). Expression of p-IRF3, IRF3, and GAPDH was detected with the rabbit-anti p-IRF3 (ab76493), IRF3 (ab68481) (Abcam, Cambridge, UK), and a mouse anti-GAPDH monoclonal antibody (Sigma, St Louis, USA).Assay of Deubiquitinase Activity in Cultured Cells. HEK293T cells were cotransfected with pcDNA3.1-HA-Ub plus the indicated amounts of TGEV PL1, p-Flag-RIG-1, and p-Flag-STING constructs. The effect of TGEV PL1 on ubiquitinated proteins in cultured cells was assessed by immunoblot analysis.Coimmunoprecipitation Analysis. Coimmunoprecipitation experiments were performed on HEK293T cells transfected with the indicated expression plasmids as described in an earlier report .Immunofluorescence Assay. HEK293T cells were plated on fibronectin-treated glass coverslips in 24-well plates. To evaluate the localization of TGEV PL1, cells were cotransfected with plasmid DNA expressing Flag-PL (500 ng per well) and pDsRed-Mito (500 ng per well) using X-treme GENE HP, according to the manufacturer's protocol. HEK293T cells were cotransfected with IRF3-GFP (500 ng per well) and empty vector (500 ng per well) or IRF3-GFP (500 ng per well) and Flag-PL1 (500 ng per well). 24 h after transfection, cells were infected with SeV (100 hemagglutinating activity units/well) for 16 h. Next, cells were fixed with 4% paraformaldehyde for 30 min and permeated with 0.1% Triton X-100 for 15 min at room temperature. After three washes with PBS, cells were blocked with PBS containing 5% bovine serum albumin for 2 h, followed by incubation with a mouse monoclonal antibody against Flag (1 : 100) for 1 h at room temperature. Cells were treated with fluorescein isothiocyanate-labeled goat anti-mouse (Sigma, St Louis, USA) for 1 h, and subsequently with 4 ,6-diamidino-2-phenylindole (DAPI) for 15 min at room temperature. Samples were washed with PBS, and fluorescent images were acquired under a confocal laser scanning microscope (TCS SP5; Leica, Solms, Germany).To assess the formation of STING dimers, HEK293T cells were transfected with Flag-STING (500 ng per well) and the lysates were subjected to Western blot, as described earlier , with the indicated antibodies. Is an IFN Antagonist. The crystal structure of TGEV PL1 has been determined . The structure of TGEV PL1 is similar to that of SARS-CoV PL2 pro . In order to determine whether TGEV PL1 is capable of blocking IFN-production, we assessed IFN-promoter activity in the presence of PL1 (Figure 1(a) ). HEK 293T cells were cotransfected with TGEV PL1 and IFN-luciferase or Renilla luciferase reporter plasmids for 24 h and subsequently infected with SeV to activate the RIG-1-dependent IFN-expression pathway. We observed the inhibition of SeV-induced IFN-promoter activation in the presence of PL1, similar to the antagonistic function of NL63 PLP2 and porcine epidemic diarrhea virus (PEDV) PLP2, clearly indicating that TGEV PL1 could act as an interferon antagonist. To establish the mechanisms by which PL1 inhibits IFNexpression, transcriptional activities of NF-B, IRF3, and AP-1 were analyzed using the luciferase assay to identify the precise transcription factor involved. Notably, the luciferase activities of all three transcription factors were significantly inhibited by TGEV PL1 in a dose-dependent manner ( Figures 1(b) , 1(c), and 1(d)). Furthermore, Flag-PL1 also significantly inhibited IFN-production in PK-15 cells at protein level ( Figure 1(e) ), which was further confirmed with the result that TGEV PL1 could block the production of interferon. To further establish whether TGEV PL1 affects IRF3 phosphorylation or migration from the cytoplasm to nucleus, HEK293T cells were transfected with TGEV PL1 and/or IRF3-EGFP. Then the result was analyzed using Western blot and confocal microscopy. In Figure 1(f) , the level of p-IRF3 was decreased significantly by TGEV PL1 compared with that of SeV-induced. Furthermore, IRF3-EGFP was located in the cytoplasm compared with mock-infected HEK293T cells but translocated to the nucleus when the cells were inoculated with SeV. In contrast, after being inoculated with SeV, it was found that IRF3-EGFP was translocated from cytoplasm to nuclear in mock infected HEK293T cell, which was not observed in cells transfected with TGEV PL1 (Figure 1(g) ).Our results collectively suggested that TGEV PL1 suppressed IFN-transcription by interfering with NF-B-, IRF3-, and AP-1 signaling-mediated IFN expression.To determine whether TGEV PL1 is capable of blocking STINGmediated activation of the IFN-promoter, we assessed promoter activity in the presence of STING along with increasing amounts of TGEV PL1. Stimulation of HEK293T cells with STING alone resulted in a robust increase in IFNpromoter activity. Coexpression of STING and TGEV PL1 induced a dose-dependent decrease in IFN-activity, clearly indicating antagonistic activity of TGEV PL1 on STINGmediated activation of the IFN-promoter (Figure 2(a) ).STING dimerization is reduced in the presence of HCoV-NL63. STING dimmers are visualized as an 80 kD band on SDS-PAGE. To further determine whether TGEV PL1 inhibits STING-mediated signaling through disrupting the stability of STING dimers, HEK293T cells were cotransfected with plasmid DNA expressing STING in the presence or absence of TGEV PL1 and SeV, and cell lysates were evaluated for dimmers via immunoblotting (Figure 2(b) ). Interestingly, the results indicated that STING dimerization was not affected by TGEV PL1.Inhibiting IFN-Expression. To determine whether catalytic activity is required for TGEV PL1-mediated inhibition of IFN-expression, HEK293T cells were cotransfected with alanine mutants of three conserved catalytic residues of TGEV PL1 (C32A, H183A, and D196A) with or without RIG-1, MAVS, STING, or TBK-1, and IFN--Luc and pRL-TK plasmids, followed by infection with SeV to activate IFNpromoter activity. TGEV PL1 mutation at two of the catalytic sites (C32A and H183A) led to almost complete loss of IFN antagonistic activity, relative to wild-type TGEV PL1, but the D196A mutant showed a little inhibition for IFN-promoter activity (Figures 3(a), 3(b) , 3(c), 3(d), and 3(e)). Based on the results, we conclude that the intact catalytic triad of TGEV PL1 is required to inhibit activation of the IFN-promoter driven by STING and TBK-1. Recent studies have revealed that STING acts as a scaffold protein for TBK-1 and IRF3 and links them to the MAVS complex in mitochondria upon viral infection . Moreover, activation of STING is critical for stimulation of IRF-3 activity. Here, we observed that TGEV PL1 protein inhibits STING-and TBK-1-induced activation of IFN-. Additional localization experiments showed that PL1 existed in mitochondria (Figure 3(f) ).Modification of signaling molecules by ubiquitin (Ub) plays a critical role in activation of the IFN response. TGEV PL1 has been shown to possess DUB activity. Here, we investigated the DUB activities of TGEV PL1 and its catalytic mutants. HEK293T cells were cotransfected with pcDNA HA-Ub and TGEV PL1, and the level of ubiquitinated proteins was assessed via Western blot. The level of Ub-conjugated proteins was reduced dramatically in cells transfected with wild-type TGEV PL1, while the ubiquitinated Ub-HA level was not reduced in the presence of the C32A, H183A, and D196A mutants (Figure 4(a) ). Next, we investigated whether TGEV PL1 recognizes and deubiquitinates the key regulators, RIG-I and STING, in the IFN signaling pathway. TGEV is known to induce robust expression of IFN-at the late step of the replication and is distinct from CoVs [17, 18] . Moreover, TGEV infection activates transcription factors NF-B, IRF3, and AP-1 in porcine kidney cells and a delayed activation of the IFN response in intestinal epithelial cells [19, 20] . However, the mechanism of its evasion of the innate immune system has never been reported. The current study firstly showed antagonistic function of the TGEV PL1 protein against the IRF3 signaling pathway to inhibit IFNinduction through its DUB activity.To combat the host antiviral effects, coronaviruses likely take advantage of PL activity to escape from the host innate antiviral response. HCoV-NL63 (PL2-TM) and SARS-CoV (PLpro-TM) inhibit STING-mediated activation of IRF-3 nuclear translocation and induction of IRF-3-dependent promoters [6, 8] . PL2 of MHV strongly inhibits CARDIF-, TBK1-, and IRF3-mediated IFN-reporter activities and prevented nuclear translocation of IRF3 . PEDV PLP2 negatively regulated RIG-I and STING-mediated IFN-expression . Moreover, TGEV PL1 displays a similar structure to SARS-CoV PL2 and gives rise to the speculation that TGEV PL1 may similarly act as an IFN antagonist. In the present study, we first found that overexpressed TGEV PL1 inhibited STING-and TBK-1-mediated IFNtranscription and antagonized the type I IFN response stimulated by SeV in PK-15 cells. The catalytic activity of TGEV PL1 is essential for inhibiting IFN-transcription. Furthermore, STING dimerization is reduced in the presence of HCoV-NL63 PL2-TM, which was not affected by TGEV PL1. These results suggested that TGEV PL1 acted as an IFN antagonist to negatively regulate host antiviral innate immunity.Ubiquitination and deubiquitination are critically involved in regulation of virus-induced type I IFN signaling pathways [21, 22] . Recently, DUBs have been reported in a variety of viruses, such as foot-and-mouth disease virus, Lpro , human cytomegalovirus, UL48 , herpes simplex virus type 1, UL36 , and porcine reproductive and respiratory syndrome virus, nsp2 [26, 27] . Interestingly, all CoVs have evolved to encode DUB enzymes, which may contribute to modulation of the innate immune response. PLP of HCoV-NL63, SARS-CoV, MHV, PEDV, and MERS-CoV dramatically reduced the levels of ubiquitinated STING, RIG-I, TBK1, and IRF-3 to negatively regulate host antiviral innate immunity. Here, we showed that TGEV PL1 interferes with and significantly inhibits ubiquitination of RIG-1 and STING, which are essential activators of type I IFN signaling. Then, the levels of phosphorylated IRF-3 were reduced, which blocked nuclear translocation of IRF3 to activate the transcript of IFNs. Three catalytically inactive mutants of TGEV PL1 (C32A, H183A, and D196A) found to be defective in DUB activity failed to inhibit virus-induced INF-expression, indicating that the DUB function of TGEV PL1 is directly involved in inhibition of type I IFN induction. However, the membrane protein M and envelope protein E of TGEV were translated at the late step of the replication as the major inducing component of IFNs. Further studies are required to establish the precise functions of PL1 protease/DUB activity in coronavirus interactions with the host innate immune response.Our results are the first report identifying TGEV PL1 that is responsible for inhibiting the induction of IFN-. We found that TGEV PL1 displayed IFN antagonist activity dependent on the intact catalytic triad (C32, H183, and D196) and interfered with RIG-1-and STING-mediated signaling through a viral DUB activity. These characteristics of TGEV PL1 served as a multifunctional protein with a critical regulatory role in TGEV interactions with the host antiviral innate immune response. Moreover, these findings contribute to our understanding of the molecular mechanisms of innate immunity evasion strategies utilized by TGEV.\n",
            "\t  0.705639 \t Background: Middle East respiratory syndrome (MERS) is caused by a coronavirus (MERS-CoV) and is characterized by hypoxemic respiratory failure. The objective of this study is to compare the outcomes of MERS-CoV patients before and after the availability of extracorporeal membrane oxygenation (ECMO) as a rescue therapy in severely hypoxemic patients who failed conventional strategies.Middle East respiratory syndrome (MERS), which was first described in 2012, is caused by a novel coronavirus (MERS-CoV). The World Health Organization (WHO) as of 5 December 2016 reported 1917 confirmed cases of the MERS-CoV infection globally with an overall mortality rate of 35% . The majority of cases were reported in Saudi Arabia, wherein 1567 were confirmed cases, and of which 649 (41%) died . Human coronaviruses were first identified in the mid-1960s and usually cause mild upper-respiratory tract illness. In 2012, the first confirmed case of MERS-CoV was reported from Saudi Arabia .MERS-CoV infection is associated with significant mortality related to the virulence of the virus, nature of the disease, and the lack of effective therapy. Patients with MERS-CoV who develop acute respiratory distress syndrome (ARDS) are at a high risk of dying from refractory hypoxemia, multiorgan failure, and septic shock .Current interventions such as lung protective ventilation, prone ventilation, and neuromuscular blocking agents have been shown in randomized trials to improve mortality in patients with ARDS . However, in some patients, these conventional measures fail to maintain adequate oxygenation; therefore, other rescue therapies are considered, such as different modes of ventilation, inhaled pulmonary vasodilators, and extracorporeal membrane oxygenation (ECMO). Anticipated difficulties in patient recruitment, study design, and ethical concerns affect the feasibility of conducting randomized clinical trials that examine the efficacy of ECMO in this population. Therefore, observational studies are a reasonable alternative. In this study, we aim to describe the effect of ECMO rescue therapy on patient-important outcomes in patients with severe MERS-CoV.In response to the large MERS-CoV outbreak, the Saudi Ministry of Health implemented a national ECMO program in April 2014. The Saudi ECMO program provided a rapid transportation chain system (Medevac system), isolated intensive care unit (ICU) beds, and venovenous (V-V) ECMO machines in selected centers across the country. An ECMO team was created that was available 24 h a day/7 days a week. The team included an intensivist trained in ECMO, a cardiac surgeon, a perfusionist, and ECMO-trained nurses. The intensivist on the ECMO team triaged all calls from other centers based on predefined criteria, wherein patients were predetermined to be candidates to receive ECMO or not. Criteria for eligibility to receive ECMO were based on the Extracorporeal Life Support Organization (ELSO) guidelines and are listed below.We retrospectively identified patients who would have been eligible for ECMO but did not receive it because the ECMO program was not available at that time (prior to April 2014). The intervention (ECMO) group was included from five main ECMO centers in three major cities in Saudi Arabia after the program initiation (April 2014 to December 2015). All participating hospitals were accredited by the Joint Commission International and had closed ICUs with 24-h coverage by trained intensivists. We obtained ethics approval from the Saudi Ministry of Health ethics review board and from individual centers' ethics boards.Patients were candidates to receive ECMO if they have met the following criteria: The ECMO group included patients who met the above criteria and received ECMO after implementing the ECMO program from April 2014 to December 2015. We included all patients with MERS-CoV who received ECMO during that period. The control group were patients who met the above criteria but did not receive ECMO in the period prior to the introduction of ECMO program (prior to April 2014). Weaning from ECMO was primarily based on clinical improvement demonstrated by adequate oxygenation and gas exchange shown in vital signs, blood gases, and chest X-ray. The decision for readiness of a patient to be weaned from ECMO was left to the judgment of treating clinician and the ECMO team. The weaning process followed the ELSO criteria as follow: weaning starts by decreasing the flow to 1L/min while keeping the sweep of 100% (to maintain SPO2 > 95%). If SPO2 remains within target, a trial of clamping the catheters and keeping the patient on the ventilator at appropriate settings was attempted.We designed an electronic pretested data abstraction forms; the forms were pilot tested prior to data collection to ensure accuracy and reproducibility. Trained personnel collected the data at each participating center under the supervision of the local principal investigators. Research personnel collected data on patients' demographics, comorbidities, Acute Physiology and Chronic Health Evaluation II (APACHE II) score, laboratory results (hemoglobin concentration, white blood and platelets counts, kidney function, blood gases), ventilator modes and settings, interventions used to treat refractory hypoxemia (prone ventilation, use of neuromuscular blocking drugs, and pulmonary vasodilators), vasoactive support, antimicrobial and antiviral therapy, steroid use, and primary and secondary outcome data.Data were tested for normality using the Kolmogorov-Smirnov test. A repeated-measures analysis of variance was performed. Fischer's exact test was used for the categorical data. Independent t test was used to compare the continuous variables in the two groups. The Mann-Whitney U test was performed to compare the nonparametric values of the two groups. Data were expressed as median (interquartile range (IQR) [range]), number (proportion), or mean (SD) as appropriate. The volume of cases was not enough to allow a priori power analysis. However, a post hoc power analysis indicated that the current sample size of 35 patients is powered to detect 35% absolute difference in mortality rate, with a type I error of 0.05 and a power of 80%. A value of P < 0.05 was considered statistically significant.Eighty patients with confirmed MERS-CoV infection were admitted to the ICUs of participating centers from April 2014 to December 2015. Thirty-five patients met our eligibility criteria and were included in the analysis, 17 in the ECMO group and 18 in the control group. As shown in Table 1 , the baseline characteristics were similar in both groups; the median ages were (46 vs. 50 years), and mean APACHE II score (28 vs. 31) were not statistically different. (P = 0.48 and P = 0.12; respectively).Adjunctive therapies were used in both groups. Ribavirin was used significantly more often in the ECMO group compared to the control group (82 vs. 24%, P = 0.001), interferon was also used more in the ECMO cohort compared to controls (65 vs. 24%, P = 0.016), and the use of steroids was similar in both groups (53 vs. 72%, P = 0.24). At day one of eligibility to ECMO, more patients in the control group required hemodynamic support with norepinephrine compared to ECMO group; however, both groups had similar use of epinephrine and dobutamine, continuous renal replacement therapy (CRRT), modes of ventilation, positive end-expiratory pressure (PEEP), and neuromuscular blocking agents (Tables 2 and 3 ). Alveolar recruitment maneuver was used in one patient in the ECMO group. None of the patients received prone ventilation. Throughout days 1-14, more patients in the control group developed renal impairment and had significantly lower PaO2/FiO2 ratio (Table 3) . Other laboratory values were similar between both groups (Table 4 ). However, due to the small sample size, it was not feasible to adjust for all confounding factors.In the ECMO group, the V-V mode was used in all patients via the percutaneous cannulation approach for vascular access. Femoral-femoral access was used in 65% of patients, while femoral-jugular access was used in 35% of cases. ECMO access was inserted by a cardiac surgeon in 70% of cases and by a cardiac intensivist in the remaining 30%. Chest X-ray was used to confirm successful cannulation in 16 patients and transesophageal echocardiography (TEE) in one patient. Blood flow (L min −1 ), revolutions per minute, and sweep gas among ECMO patients had a mean (SD) of 3.8 (0.77), 3148.7 ECMO-related mechanical complications occurred in 3 (18%) patients; one patient developed pneumothorax that was treated with chest tube insertion, and two patients had major bleeding immediately after the initiation of ECMO.Compared to the control group, the ECMO group had significantly lower in-hospital mortality (65 vs. 100%; P = 0.02), longer ICU stay (25 vs. 8 days; P = 0.001) ( Table 5 and Fig. 1 ). Less use of norepinephrine at days 1 and 14 (P < 0.05), and better oxygenation (higher PaO 2 / FiO 2 ratio) throughout days 7-14 (Table 2 ).In this retrospective cohort study, we found that ECMO rescue therapy was associated with lower in-hospital mortality, better oxygenation, and fewer organ failures compared to historical control (usual care) in patients with severe MERS-CoV. However, the length of hospital stay was the same and a possible explanation is that during the crisis phase, patients were mechanically ventilated in the ward when ICU beds are full, and it is possible that this could have contributed to similar stay in hospital in both groups.Although ELSO issued guidelines on the use of ECMO in patients with ARDS, these guidelines do not address specific disease context, and are difficult to generalize to the heterogeneous ARDS population. Therefore, we conducted this observational study to report on the efficacy and safety of ECMO in patients with severe MERS-CoV infection.There is a single case report in the literature looking at ECMO in MERS-CoV patients. Guery et al. described the use of ECMO in two patients with acute respiratory failure secondary to MERS-CoV infection in France, where both patients developed severe hypoxia and increasing oxygen requirements, leading to mechanical ventilation and ECMO use. One patient died, and the other survived after approximately 2 months in hospital .ECMO use in respiratory failure has been reported with variable survival rates. The first 2 randomized clinical trials (RCTs) failed to prove superiority of ECMO over conventional management [11, 12] . However, the severe adult respiratory failure (CESAR) trial showed improved 6-month survival in patients who were referred early to an ECMO center . This was the largest clinical trial to investigate the efficacy of early use ECMO in patients with ARDS. Despite concerns about the trial design and possible differences in steroid use and ventilator strategies, these results contributed to the increasing use of ECMO worldwide.In this study, we observed no significant differences in the use of adjunctive therapies except for ribavirin use in the ECMO group. The benefit of antiviral therapy in MERS-CoV infection remains unclear. Recent Korean guidelines published during the 2015 MERS-CoV outbreak in South Korea suggested the use of antiviral therapy in patients with severe MERS-CoV .In patients with respiratory failure from H1N1 infection who required the use of ECMO, the survival rate varied considerably between studies ranging from 35 to 90% . There was a large variation in survival rates, which could be explained by differences in patients' baseline characteristics and severity of illness. In one study, older, obese, diabetic, or immunocompromised patients were found to be at a higher risk of developing severe . In this study, the two groups were comparable at baseline, and there were no significant differences between groups in any of these variables. Another large observational study examined the predictors of death in H1N1 patients who underwent V-V ECMO and found that creatinine and bilirubin levels, systemic arterial pressure, hematocrit, and pre-ECMO hospital length of stay were associated with higher mortality . Another important factor is the center experience and volume of cases; this could have contributed to the variability in survival rates with ECMO use. A recent study by Barbaro et al. demonstrated that centers with > 30 ECMO cases/year had better survival rates than centers with less than 6 cases per year. In Saudi Arabia, ECMO was not available except in one center until the MERS-CoV crisis; thereafter, the ECMO program was implemented as a therapeutic option for patients with refractory hypoxemia. ECMO interventions were run in tertiary centers with equipped ICUs by most experienced intensivists and perfusionists who received training in ECMO prior to the start of the program.Although more ECMO patients received ribavirin and interferon therapy, we do not believe that this difference has an impact to our findings. Published reports on this therapy are limited, but none showed significant improvement with this combination . The largest study to date published in abstract format showed no reduction in mortality. Therefore, we believe that the imbalance of co-interventions between the two groups is unlikely to affect the estimation of treatment effect.In regard to infection control issues, caregivers safety of ECMO patients was organized and maintained by aggressive measures which were applied strictly and monitored closely with all admissions were taken to airborne isolated rooms which impacted the containment of the virus plus applying the universal protective personal measures all the time during the patients encounter. Because of these stringent measures, there were no reports by or about any caregiver of any ECMO patient being affected.To our knowledge, this is the largest study to describe outcomes in patients with MERS-CoV who received ECMO. There are several strengths to our study: the \"before and after\" design allowed us to compare ECMO cases to a control group with similar demographics and within the same institutions. We also collected data on important variables and confounders, and conducted adjusted analyses to assess the impact on the results. We adhered to the Strengthening the Reporting of Observational studies in Epidemiology (STROBE) guidelines .Despite the strengths of our study, it has several important limitations. First, the retrospective nature of this study renders it at risk of bias. All patients in the control group died, which may be explained by the severity of illness, as these were patients who had ARDS and were eligible otherwise. We cannot rule out the possibility of selection bias, as we were unable to track all transfer requests due to the outbreak and crisis at the time, leaving us with limited information. In addition, some patients were transferred from non-participating ECMO centers; therefore, baseline pre-ECMO data such as blood gases and ventilator settings could not be obtained. Furthermore, due to insufficient documentation during the outbreak and crisis circumstances, we were not able to track the ECMO requests to the referral call center.There were differences in some co-interventions (e.g., antiviral therapy), and the influence of unmeasured confounders cannot be excluded. Such concerns can only be addressed in RCTs; however, conducting RCT is likely to be challenging in the context of epidemics. This study was not designed to compare the cost of 2 interventions; although it is an important outcome that could help the clinicians and stakeholders to make decisions. Lastly, the small sample size limited our ability to perform an adequate multivariate analysis. Similar to other ECMO studies, it is difficult to determine if the mortality was the result of refractory respiratory failure or other causes like septic shock or other organs failure.In summary, the use of ECMO was associated with lower mortality in patients with severe MERS-CoV infection and refractory hypoxia. Future randomized trials, although challenging to conduct, are highly needed to confirm or dispute these observations. Until more data are available, ECMO could be considered as a rescue therapy in selected MERS-CoV patients with refractory hypoxemia.\n",
            "\t  0.69676405 \t The SARS (severe acute respiratory syndrome) outbreak was caused by a coronavirus (CoV) named the SARS-CoV. SARS pathology is propagated both by direct cytotoxic effects of the virus and aberrant activation of the innate immune response. Here, we identify several mechanisms by which a SARS-CoV open reading frame (ORF) activates intracellular stress pathways and targets the innate immune response. We show that ORF8b forms insoluble intracellular aggregates dependent on a valine at residue 77. Aggregated ORF8b induces endoplasmic reticulum (ER) stress, lysosomal damage, and subsequent activation of the master regulator of the autophagy and lysosome machinery, Transcription factor EB (TFEB). ORF8b causes cell death in epithelial cells, which is partially rescued by reducing its ability to aggregate. In macrophages, ORF8b robustly activates the NLRP3 inflammasome by providing a potent signal 2 required for activation. Mechanistically, ORF8b interacts directly with the Leucine Rich Repeat domain of NLRP3 and localizes with NLRP3 and ASC in cytosolic dot-like structures. ORF8b triggers cell death consistent with pyroptotic cell death in macrophages. While in those cells lacking NLRP3 accumulating ORF8b cytosolic aggregates cause ER stress, mitochondrial dysfunction, and caspase-independent cell death.In 2002-2003 the SARS-CoV caused a severe respiratory illness affecting more than 8000 individuals with a mortality rate near 10% 1,2 . The subsequent identification of a large pool of coronaviruses circulating in bats and other animals portended the appearance of other highly pathogenic CoVs 3 , and in 2012 the Middle East Respiratory Syndrome-CoV caused a similar outbreak with a higher mortality rate 4 . Severe SARS-CoV infection manifests clinically as acute lung injury associated with high initial virus titers, macrophage/neutrophil accumulation in the lungs. and elevated proinflammatory serum cytokines (IL-1, IL-18, IL-6, IL-8, and MCP-1) . Recent advances have implicated inflammatory monocyte-macrophages (IMMs) in the lungs as critical mediators of SARS-CoV pathology, as a delayed type I interferon response promotes high initial virus titers and aberrant IMM recruitment 9 . Host disease is likely a combination of direct viral damage and consequences of an aberrant immune response promoted by IMMs 9 , underscoring the importance of determining the mechanisms by which the virus targets innate immunity.The SARS-CoV is an enveloped, positive-strand RNA virus that encodes a set of accessory proteins, several of which target the innate immune response. Open reading frame (ORF) 8a and ORF9b trigger cellular apoptosis; ORF7a activates nuclear factor-κB (NF-κB); ORF3b upregulates the expression of several cytokines and chemokines; ORF6 limits interferon production; ORF3a induces necrotic cell death; and ORF8b induces cellular DNA synthesis and suppresses the expression of the viral envelope protein 10, 11 . Recently, we found that ORF9b localizes to mitochondrial membranes and reduces mitochondrial-associated adapter molecule MAVS, severely limiting the interferon response 12 . Of note, ORF8b is of interest as ORF8 encoded a single polypeptide during the early phase of the SARS epidemic, while in the later stages a 29-nucleotide deletion split it into two ORFs, ORF8a and ORF8b. They encode 39-and 84-residue polypeptides respectively. The splitting of ORF8 is thought to confer evolutionary advantage to the virus, and accordingly virus expressing ORF8b is better able to replicate in the presence of interferon 10, 13 .Initially, two studies had difficulty expressing the ORF8b protein, suggesting it may be rapidly degraded in cells 14, 15 . However, antibodies raised specifically against the ORF8b protein detect low levels of expression as early as 8 h (h) post infection, while expression is prominent by 24 h 16 . Immunostaining ORF8b after expression in Vero E6 cells localized it to cytosolic punctate vesicle-like structures reminiscent of intracellular aggregates 17 . Here, we investigate the cellular mechanisms by which ORF8b contributes to SARS-CoV pathology. We report that ORF8b forms intracellular aggregates dependent on a valine at residue 77, which contributes to the induction of lysosomal stress, autophagy, and eventual cell death. Given the importance of IMMs in SARS-CoV pathology, the link between intracellular aggregates and inflammasome activation 18 , and the observation that SARS-CoV infected patients have elevated serum levels of IL-1 and IL-18 19 , we tested whether ORF8b might affect inflammasome activation in macrophages and lung epithelial cells. We found that ORF8b triggers robust NLRP3 inflammasome activation and IL-1β release. NLRP3 activation was accompanied by direct binding of ORF8b to the LRR domain of NLRP3. In sum, this study suggests that ORF8b activates cell stress pathways by forming intracellular aggregates and may be important in the aberrant activation of macrophages during SARS-CoV infection.As previous studies have shown that ORF8b forms vesicle-like puncta in the cytosol, we used the bioinformatics aggregation prediction server PASTA2 (http:// ) 20 to investigate the intrinsic aggregation propensity of ORF8b. Analysis of the ORF8b sequence predicted a tendency to aggregate based on a carboxy-terminal VLVVL motif (aa 75-79), while reanalyzing the ORF8b sequence with V77 mutated to K (V77K) reduced its predicted propensity to aggregate (Fig. 1a) . To empirically test the aggregation tendency of ORF8b, we expressed GFP-ORF8b (GFP-8b) in Hela cells overnight and imaged the transfected cells by confocal microscopy. While some cells had a homogenous pattern of expression, other GFP-8b expressing cells showed intracellular patch and fiber aggregation patterns (Fig. 1b) . Quantification showed a considerably higher percentage of cells with intracellular aggregates in GFP-8b expressing cells than GFP vector (GFP) expressing cells (Fig. 1c) . Presumably, the homogenous expression of GFP-8b precedes the development of intracellular aggregates. These findings were replicated in A549 cells, a human lung epithelial cell line (Fig. 1b) .Aggregated proteins often form detergent insoluble deposits. Thus, we separated the triton soluble and insoluble fractions of HeLa cells expressing either GFP, GFP-ORF8b, or mutant GFP-8b V77K to determine if ORF8b forms insoluble aggregates. To separate these fractions, we lysed the cells in a Triton-X based buffer and spun down the resultant lysates at 14,000 rpm for 10 min (min). The supernatant was collected and considered the triton soluble fraction, while the pellet was reconstituted in an LDS based buffer and heated at 100°C for 30 min. While preparing the triton insoluble fraction, we consistently observed that addition of LDS and heating for 30 min was unable to completely solubilize the pellet from GFP-8b expressing cells, but completely solubilized the pellet from GFP and GFP-8b V77K expressing cells. Accordingly, analysis of each fraction by SDS-PAGE followed by immunoblotting for GFP showed considerable monomeric and oligomerized GFP-8b in the insoluble fraction, while significantly less monomeric and almost no oligomerized protein was seen in the GFP-8b V77K insoluble fraction (Fig. 1d) . The control GFP was undetectable in the insoluble fraction and seen only in the soluble fraction (Fig. 1d) . Moreover, we observed consistently less GFP-8b V77K expression in the soluble fraction compared to GFP-8b, indicating that GFP-8b's ability to aggregate may be protective against degradation (Fig. 1d) .Intracellular protein aggregates and misfolded proteins cause ER stress and induction of the unfolded protein response (UPR), which can be quantified by detecting the upregulation of the UPR protein CHOP 21, 22 . Confocal imaging showed that aggregated GFP-8b caused ER dispersion as assessed by immunostaining for ERp72 (Fig. 2a) . The expression of GFP did not affect the usual juxta nuclear localization of the ER while expressing GFP-8b disrupted the usual ERp72 localization. Immunoblotting showed greater CHOP induction after transfection of GFP-8b than either GFP or GFP-8b V77K (Fig. 2b) , indicating that GFP-8b forms protein aggregates dependent on its valine 77. As intracellular protein aggregation is toxic once the aggregates overwhelm cellular homeostatic mechanisms, we looked at cell death by Trypan blue uptake following expression of GFP, GFP-8b, and GFP-8b V77K. Expression of GFP-8b induced significantly more cell death than did GFP, while the GFP-8b V77K was partially protective (Fig. 2c) . Taken together, this data suggests that GFP-8b forms insoluble protein aggregates and that can cause cell death, which at least partially depends on its ability to form intracellular aggregates.ORF8b induces lysosomal stress, the nuclear translocation of TFEB, and the activation of Transcription factor EB target genes Aggregated proteins are typically cleared from the cytosol by the autophagy-lysosome pathway, which engulfs bulk cargo in the cytoplasm and delivers it to the lysosome for degradation 23, 24 . Formation of intracellular protein aggregates can disrupt this pathway, as aggregated proteins are poorly degraded by lysosomes. Lysosomal protein buildup then results in lysosomal stress and compensatory activation of transcription factor EB (TFEB), the master transcriptional regulator of autophagic and lysosomal machinery 25, 26 . To determine if ORF8b disrupts autophagic and lysosomal homeostasis, we expressed GFP-8b and mCherry-galectin 3 in HeLa cells followed by imaging via confocal microscopy. Under homeostatic conditions galectin-3 is localized diffusely throughout the cytosol, but rapidly forms puncta on lysosomes following lysosomal stress/damage 27 . Accordingly, GFP expressing cells showed diffuse cytosolic galectin-3 distribution, while many GFP-8b expressing cells showed clear galectin-3 puncta formation, some of which co-localized with GFP-8b (Fig. 3a) .Correspondingly, expression of GFP-8b with mCherry-TFEB showed prominent TFEB nuclear localization compared to the control (Fig. 3b) . Immunoblotting endogenous TFEB after separation of the nuclear and cytosolic fractions in GFP-8b expressing HeLa cells showed significantly more TFEB in the nucleus than control cells, and levels of the TFEB target gene LAMP1 were consistently increased (Fig. 3c) . Confocal microscopy of GFP-8b expressing cells showed large, prominent lysosomes as assessed by LAMP1 immunofluorescence (Fig. 3d) . However, we noted only moderate overlap between the LAMP1 immunostaining and the GFP-8b aggregates. TFEB nuclear translocation following both ER and lysosomal stress has been reported to depend upon dephosphorylation of its S211 residue by the phosphatase calcineurin (PPP3CB) . To determine the mechanism of TFEB activation in GFP-8b overexpressing cells, we immunoblotted endogenous TFEB following nuclear fractionation of GFP, GFP-8b, and GFP-8b V77K expressing cells. The GFP-8b expressing cells were treated or not with the calcineurin inhibitor cyclosporin A. Again, GFP-8b induced TFEB nuclear translocation and increased LAMP1 expression, both of which were reduced by treatment with cyclosporin A. Expression of GFP-8b V77K did not increase nuclear TFEB levels or increase the level of LAMP1 in the cytosolic fraction (Fig. 3e) . These data indicate that ORF8b aggregates causes ER and lysosomal stress, which leads to the nuclear translocation of TFEB, and the upregulation of TFEB target genes.Activation of TFEB increases both lysosomal biogenesis and autophagic flux 31 , thus we checked to see whether GFP-8b affects autophagy downstream of TFEB. Expression of GFP-8b induced clear formation of RFP-LC3 puncta, while GFP did not (Fig. 4a) . In some cells the RFP-LC3 defined autophagosomes strongly co-localized with the GFP-8b aggregates while in other cells many of the aggregates did not (Fig. 4a ). An increase in LC3 puncta can be due to an increase in autophagosome formation or a decrease in autophagosome/lysosome fusion 32 . During autophagosome formation, LC3 is lipidated and converted from LC3-I to LC3-II. We expressed GFP or GPF-8b in the presence or absence of Bafilomycin A1, which blocks autophagosome-lysosome fusion, and immunoblotted the cell lysates for LC3. We found that the cell lysates from the GFP-8b and GFP expressing cells had similar LC3-II Fig. 2 ORF8b induces ER stress and cell death. a Confocal microscopy images of HeLa cells transiently transfected with either GFP or GFP-8b and immunostained for the endoplasmic reticulum marker ERp72 (red). Nuclei were counterstained with DAPI and cells were fixed prior to imaging. Individual and merged images are shown, and the scale bar is 5 μm. b Immunoblot looking at the accumulation of the ER stress marker CHOP after transient transfection of GFP, GFP-8b, or GFP-8b V77K in HeLa cells. c Quantification of cell death by Tryptan Blue uptake after overnight transfection of HeLa cells with the indicated proteins. A minimum of three fields with >50 cells per field were quantified. Statistical significance evaluated with Student's t-test (*p < 0.05, **p < 0.01). Each experiment repeated a minimum of three times levels. However, bafilomycin A1 treatment increased the LC3-II levels in the GFP-8b expressing cells more than it did in the control cells (Fig. 4b) . These results indicate that GFP-8b induces autophagic flux. Treatment with bafilomycin A1 increased the GFP-8b levels, but not GFP levels suggesting that ORF8b aggregates are partially degraded by the autophagy-lysosome pathway. Taken together, the GFP-8b intracellular aggregates trigger cellular stress. This results in a calcineurin dependent activation of TFEB and TFEB target genes, which leads to an increase in autophagic flux.Intracellular protein aggregation and lysosomal stress have previously been mechanistically linked to NLRP3 activation 33, 34 . As SARS patients show increased serum IL-1β and IL-18 7,8 , we investigated whether ORF8b affects inflammasome activation in macrophages. To do so, we first attempted to generate a permanent cell line expressing GFP-8b in the human macrophage cell line THP-1 cells. While we easily established GFP expressing THP-1 cells, no cells survived the selection process when we attempted to express GFP-8b. This is likely secondary to the toxicity of the protein, as even attempts at using an inducible promoter failed to generate a useable cell line. Therefore, we switched to a transient transfection system. Evaluating inflammasome activation in macrophage cell lines following transient DNA transfection can be problematic as cytosolic DNA activates DNA sensing inflammasomes. However, we found that that by carefully titering the amount of DNA transfected, we could express the GFP vector and yet trigger minimal Il-1β production. Therefore, we compared IL-1β production by THP-1 cells transiently transfected with low amounts of the vectors producing GFP, GFP-8b, 8b-GFP, or GFP-8b V77K. We Confocal microscopy images of GFP or GFP-8b co-transfected with RFP-galectin 3 (a) or mCherry-TFEB (b). Individual and merged images are shown, and the scale bar is 5 μm. c The percentage of cells with predominantly nuclear TFEB from part b was quantified for each condition (three fields, minimum of 50 cells/field). **p < 0.01 with Student's t-test. d Immunoblot to assess LAMP1 and TFEB expression in nuclear and cytosolic fractions separated by SDS-PAGE. HeLa cells were transiently transfected with GFP or GFP-8b. e Representative confocal images from HeLa cell expressing GFP or GFP-8b and immunostained for LAMP1 expression. Individual and merged images are shown, and the scale bar is 5 μm. f Immunoblot of cytosol and nuclear extracts from GFP, GFP-8b, or GFP-8b V77K transfected HeLa cells treated overnight with or without the calcineurin inhibitor cyclosporin A (CsA-400 nM). Images and blots are representative data from experiments repeated a minimum of 3 times found that both GFP-8b and 8b-GFP increased the amount of IL-1β present in the cell supernatant compared to GFP alone (Fig. 5a) . Surprisingly, the GFP-8b V77K construct also enhanced IL-1β production suggesting that the mechanism did not solely depend upon lysosomal stress (Fig. 5a) . We also checked whether an alternatively tagged version could trigger IL-1β production by expressing Flag tagged version of ORF8b. Similar to the results with the GFP-tagged ORF8b, we found that 8b-Flag induced 3-fold more Il-1β than did the Flag vector (Fig. 5b) . A complementary approach to assess inflammasome activity is to reconstitute the requisite components in HEK293T cells. Therefore, we transfected HEK293T cells with expression vectors for NLRP3, caspase-1, ASC, and full-length IL-1β in the presence of GFP or GFP-8b. A similar experiment was performed using FLAG or 8b-FLAG. In both instances we found an increase in IL-1β in the supernatant and a decrease in fulllength IL-1β in the cell lysate in the presence of the ORF8b constructs versus the controls (Fig. 5c) . These results suggest that ORF8b activates NLRP3 inflammasomes in macrophages and likely in monocytes.In NLRP3 expressing cells, signals that activate the NLRP3 inflammasomes cause the disassembly of the trans-Golgi network (TGN) and the recruitment of NLRP3 to the dispersed TGN, which serves as a scaffold for NLRP3 aggregation 35 . This leads to the polymerization of the adapter protein ASC and recruitment of caspase-1.To assess the impact of GFP-8b on the localization of NLRP3, we expressed NLRP3-DsRed in HeLa cells in the presence or absence of GFP-8b. In the absence of GFP-8b NLRP3 expressed homogenously throughout the cytosol of transfected HeLa cells, however the presence of GFP-8b dramatically altered NLRP3 localization (Fig. 5d) . We found that both proteins co-localized in the cytosol in small cytoplasmic dots as well as in a large perinuclear aggregate. We performed a similar experiment using HEK 293T cells (Fig. 5e ). In the left panel the images show a HEK 293T cell expressing NLRP3 alone and one expressing both proteins, while the right panel shows another cell co-expressing NLRP3 and GFP-8b. We found that GFP-8b forms large aggregates in HEK 293T cells and that its expression disturbed the homogenous cytosolic pattern of NLRP3, and as we had observed with HeLa cells both proteins showed an overlapping expression pattern. We also checked the distribution of the trans-Golgi network in HeLa cells co-expressing GFP-8b and NLRP3-DsRed by immunostaining for TGN38 (Fig. 5f ). We found partial co-localization between the NLRP3-DsRed and GFP-8b cytoplasmic dots and TGN38.In macrophages, NLRP3 also adopts a homogenous pattern until activating signals rapidly trigger the dispersion of the TGN and recruitment of NLRP3 in a punctate pattern. As macrophages also express ASC, the punctate NLRP3 distribution rapidly redistributes into a single speck that contains NLRP3 and ASC. When we expressed GFP-8b in LPS activated THP-1 cells we found that a major portion of the GFP-8b localized in a single speck that co-localized with endogenous NLRP3 and ASC (Fig. 5g) . These results also suggested that ORF8b targets NLRP3.Enterovirus 71 (EV71) 3D protein stimulates NLRP3 inflammasome activation by interacting with NLRP3 to facilitate inflammasome complex assembly. EV71 3D interacted with NLRP3 through the NACHT and LRR domains. To test whether ORF8b might also interact with NLRP3, we transiently expressed 8b-Flag in PMA differentiated and LPS stimulated THP-1 cells and tested whether we could detect an association with endogenous NLRP3. We found that immunoprecipitated 8b-Flag pulled down endogenous NLRP3, while control immunoprecipitates did not (Fig. 6a) . To characterize the region in NLRP3 domain responsible for the interaction, we examined the binding of Myc-tagged truncated NLRP3 proteins to 8b-Flag. In addition to full-length NLRP3, we used constructs that express NLRP3 with a deleted leucine-rich repeat (LRR) domain (NLRP3ΔLRR), a deleted Pyrin domain (NLRP3ΔPYD), or deleted NACHT and LRR domains (NLRP3ΔLRRΔNACHT). Following transient expression, 8b-Flag interacted with both fulllength NLRP3 and the PYD domain deleted protein, but not the LRR and LRR/NACHT domains deleted proteins (Fig. 6b) To directly show that ORF8b interacts with the LRR domain of NLRP3, we expressed the NLRP3 LRR domain as a Myc tagged protein (amino acids 537-991) and collected Myc immunoprecipitates following GFP-8b or GFP expression. We found GFP-8b, but not GFP in the Myc-immunoprecipitates (Fig. 6c) . Next, we expressed the non-functional NLRP3 LRR domain and determined whether its presence affected the induction of mature IL-1β secretion by ORF8b. We found that expression of the NLRP3 LRR domain impaired the production of mature IL-1β after GFP-8b expression in inflammasome reconstituted HEK 293T cells (Fig. 5e) , confirming the functional importance of this interaction. In summary, our data indicates that ORF8b activates the NLRP3 inflammasome and directly targets the NRLP3 LRR domain.Though the exact mechanistic role of ORF8b in SARS-CoV pathogenesis remains unclear, it is well established that the splitting of ORF8 corresponds with the shift from the early to the middle stage of the SARS epidemic, with the majority of affected patients being infected by virus containing the split ORF 36 . Recent advances in mouse models have shown that SARS pathogenesis is driven by high initial virus titers resulting from a late interferon response, which drives aberrant recruitment of IMMs and activation of the innate immune response resulting in cytotoxicity 9 . Given the likelihood that ORF8b is evolutionarily adaptive for the virus 10 , it follows that ORF8b may contribute to SARS-CoV pathogenesis by impairing the host interferon response and/or targeting the innate immune system. While the former has been previously shown 13 , here we show that ORF8b activates intracellular stress pathways upon forming insoluble aggregates and directly targets innate immunity by activation of the NLRP3 inflammasome.Intracellular protein aggregation contributes to the pathogenesis of a variety of diseases and is both propagated by and contributes to inflammation 37, 38 . Protein aggregates have long been observed after viral infection; viruses are known to use aggregates as scaffolds for replication and/or for shielding from host degradation 39 . In our study, ORF8b appears to be degraded by the autophagy-lysosome system, as treatment with bafilomycin A1 increases ORF8b levels. The aggregation ability of ORF8b may be protective against degradation, as we consistently observed that the anti-aggregation V77K mutant is expressed at lower levels than ORF8b. It has previously been reported that ORF8ab and ORF8b induce ER stress 40 , which is consistent with our findings showing accumulation of CHOP by ORF8b expression dependent on V77. We also find induction of lysosomal stress and activation of TFEB by ORF8b. Both ER stress and TFEB have been shown to propagate the inflammatory response, with TFEB directly promoting the transcription of inflammatory cytokines 26, 41, 42 . This may be a cellular mechanism by which ORF8b activates the innate immune response in macrophages and contributes to SARS-CoV pathogenesis.Finally, RNA or DNA virus infection is known to trigger NLRP3 or AIM2 inflammasome activation 43, 44 . The released IL-1β and IL-18 functions to promote host defense, but aberrant inflammasome activation can result in excessive inflammation and enhanced disease. Examples include the influenza encoded proton-selective ion channel M2 which provides signal 2 for NLRP3 inflammasome activation 45 , the encephalomyocarditis virus viroporin 2B which promotes the release of intracellular Ca 2+ from intracellular stores also providing signal 2 46 , and the pathogenic influenza A viruses which use PB1-F2 to activate NLRP3 inflammasomes 47 . Our findings indicate the SARS-CoV can activate NLRP3 inflammasome in macrophages via ORF8b. While SARS-CoV abortively infects macrophages/monocytes, enough ORF8b may be present to impact lysosome integrity, autophagy pathways, and NLRP3 inflammasomes. In contrast to macrophages, the SARS-CoV productively replicates in lung epithelial cells. These cells also express NLRP3 and can assemble NLRP3 inflammasomes. In humans infected with the SARS-CoV the full impact of ORF-8b on the pathways we delineated in this study are likely in the lung epithelium. ORF8b may contribute to the cytokine storm and inflammasome activation that occurs during severe SARS-CoV infection. Moving forward, live virus deletion studies are required to assess the effects of ORF8b mediated intracellular aggregates and ORF8b mediated NLRP3 activation. However, here we identify novel mechanisms through which ORF8b may contribute to SARS pathogenesis.The following primary Abs were used for immuno Alexa 568 conjugated polyclonal antimouse (Thermo Fisher, A11004) and anti-rabbit (Thermo Fisher, A11011) were used as secondary Abs prior to fluorescent imaging. For immunoprecipitation experiments, we used GFP antibodies coupled to magnetic beads (MBL, D153-9) and Flag antibodies coupled to agarose (Sigma, A2220) to pull down tagged proteins prior to immunoblotting. Cyclosporin A (R&D Systems, 1101) and Z-VAD-FMK (Sigma, V116) were used at 10 μM overnight, and Bafilomycin A1 (Sigma, B1793) at 100 nM for 4 h.THP-1, HEK293, A549, and HeLa cells were obtained from the American Type Culture Collection (ATCC) and maintained following ATCC's recommendations. THP-1 monocytes were differentiated into macrophages by treating with PMA (50 nM) for 3 h. To generate stable THP-1 cell lines, constructs expressing 8b-GFP or GFP were transfected followed by G418 (200 μg/ml) selection for 6 weeks. GFP-positive cells were FACS sorted twice and expanded. SARS-CoV ORF8b cDNA was generated by PCR from K14 and J17 cDNA clones by Dr. H.Y. Qi and Dr. James Shelhamer (Critical Care Medicine Department, National Institutes of Health), which were produced from the SARS-CoV genome Tor2 isolate (Michael Smith Genome Sciences Centre, Vancouver, Canada) 48 . PCR primers for cDNA generation were derived from the SARS-CoV accessory gene sequences in accession number NC_004718 (NCBI). Constructs expressing N and C terminal GFP-tagged SARS-CoV ORF8b were made using the pEGFP-N1 and C1 vectors (Clontech). PCR mutagenesis of GFP-8b was used to generate the V77K GFP-8b point mutant, and ORF8b-Flag was made by replacing the 8b-GFP c-terminal GFP with a 3x Flag tag. All constructs were verified by DNA sequencing.Myc tagged NLRP3 plasmids were kindly provided by Dr. Yong-Jun Liu (Baylor). The Myc tagged NLRP3 LRR domain construct spanned amino acids 537-991 in NLRP3. The pcDNA3 NLRP3-Flag construct was kindly provided by Dr. Gabriel Nunez (University of Michigan). To construct the NLRP3 Ds-Red construct, the Ds-Red coding region was PCRed from the pDs-Red-Monomer fluorescent vector (Clontech) and inserted into Flag tagged NLRP3 construct after removal of the c-terminal Flag coding sequence. The TFEB-GFP plasmid was bought from Addgene, (38119) and the TFEB-mCherry plasmid made by moving the TFEB cDNA into the mCherry N1 vector (ClonTech). The ptf-Galectin 3 plasmid was also purchased from Addgene (64149), and the mCherry-Galectin 3 plasmid made by removing the GFP from the ptf-Galectin 3 vector. Pooled siRNAs targeting NLRP3 (sc-45469), ASC (sc-37281), GFP (sc-45924), and a scrambled control (sc-37007) were purchased from Santa Cruz Biotechnology. Plasmids and siRNA were transfected into cells using X-tremeGENE-HP (Roche, 6366236001) following the manufacture's protocol.For standard immunoblotting, cells were lysed in Buffer A containing 20 mM HEPES (pH 7.4), 50 mM β-glycerophosphate, 1% (v/v) Triton X-100, 2 mM EGTA, and cOmplete protease inhibitor cocktail (Sigma, 11836170001) plus PhosStop (Sigma, 04906837001) phosphatase inhibitor tablets for 30 min. Lysates were cleared by centrifugation at 14,000 rpm for 10 min and the supernatant collected. For separation of Triton soluble and insoluble fractions, cells were lysed in Buffer A and centrifuged as before. The supernatant was considered the Triton soluble fraction, while the pellet was directly mixed with 1x NuPage LDS Sample Buffer (Invitrogen, NP0008) and heated at 100°C for 30 min (Triton insoluble fraction). For nuclear fractionation, cells were lysed with Buffer B, which is Buffer A substituting 0.5 (v/v) NP-40 for Triton X-100. After centrifugation, the supernatant was considered the cytosolic fraction. The resultant pellet was washed 4-6 times with Buffer B and then lysed in Buffer B + 0.5% (v/v) SDS for 30 min, this was considered the nuclear fraction. Inflammasome activation was measured by immunoblotting cell culture supernatants for mature IL-1β and cleaved caspase-1 as previously described 49 . For immunoprecipitations, cells were lysed with Buffer C, which is Buffer A + 0.5% (w/v) CHAPS and incubated with either GFP or FLAG beads for 3 h. Immunoprecipitates were washed 4-6 times and resuspended in 1x NuPage LDS buffer to release captured proteins prior to immunoblotting.All lysates were mixed with 4x NuPage LDS Sample Buffer and heated at 100°C for 10 min, followed by separation on a 4-20% Tris-Gylcine Gel (Invitrogen) and transfer to a nitrocellulose membrane using the iBlot Gel Transfer System (Invitrogen). The membrane was blocked with 5% nonfat milk (or 5% BSA) in TBST (25 mM Tris-HCl, 150 mM NaCl, 0.1% Tween-20) for 1 h and incubated at 4°C overnight with the primary Ab in TBST with 5% BSA. The appropriate secondary Abs conjugated to HRP were used to detect the protein of interest via ECL. When necessary, membranes were stripped using Restore Plus Western Blot Stripping Buffer (Thermo, 46430) following the manufacturers protocol, re-blocked, and reblotted. Images were acquired either by exposure on film (Amersham Hyperfilm ECL) or using the iBright 1000FL (Invitrogen). Blots were scanned and imported into Photoshop as unmodified tagged image files; quantification of band intensity was performed using standard methods on ImageJ (NIH) and presented relative to the appropriate housekeeping gene.Immunofluorescence THP-1 cells transiently transfected to express GFP-8b or GFP were seeded on glass-bottom collagen coated microwell dishes (MatTek), differentiated with PMA, and incubated overnight with LPS (50 ng/ml) and a caspase-1 inhibitor (Z-VAD-FMK, 10 μM). The following day, the cells were washed, fixed with 4% paraformaldehyde, permeabilized with 0.1% Triton X-100 in PBS, and blocked with 1% bovine serum albumin for 15 min. For immunostaining, the cells were incubated for 4 h at room temperature or overnight at 4°C with primary antibodies diluted in 1% bovine serum albumin/ PBS. Fluorescent images were collected after secondary antibody incubation using a Leica TCS-SP5-X-WLL confocal microscope equipped with an argon and white light laser (Leica Microsystems) at either 63X or 100X oil immersion objective (NA 1.4). The samples were excited with 488-or 561-nm laser lines. Image analysis was performed using Imaris 8.0.0 (Bitplane AG) and Adobe Photoshop CS3 (Adobe Systems). HeLa cells transiently transfected to express NLRP3-Ds-Red plus or minus GFP-8b were washed 6 h after transfection and plated in glass-bottom microwell dishes overnight. Plated cells were processed and imaged as outlined above the following day. For DNA staining, the cells were incubated with 30 nM DAPI (Sigma) in PBS for 30 min and washed prior to imaging.HEK293 cells were seeded onto 24-well plates in standard media. The following day, the media was changed to OPTI-MEM (Gibco) and the cells transfected with constructs expressing NLRP3 Flag (15 ng), Casp-1 Flag (5 ng), ASC (5 ng), full-length IL-1β Flag (150 ng) and either a control plasmid (GFP or Flag) or GFP-8b or 8b-Flag. Cell supernatants and lysates were immunoblotted for fulllength and mature IL-1β the next day.Cell death assay 0.4% Tryptan Blue (Sigma T8154) was added to cell culture plates and incubated at room temperature for 2 min. Percentage of non-viable cells (cells that had taken up Tryptan Blue) was determined manually by light microscopy and recorded; a minimum of three fields with >50 cells per field were quantified.All immunoblots, and immunofluorescence images shown are representative of reproducible experiments. Differences between groups were determined using Student's t-test (p < 0.05 was considered significant), and all statistical analysis was performed on GraphPad Prism 6.0. All experiments were performed a minimum of three times.\n",
            "\t  0.69135016 \t An epidemic of Severe Acute Respiratory Syndrome (SARS) led to the identification of an associated coronavirus, SARS-CoV. This virus evades the host innate immune response in part through the expression of its non-structural protein (nsp) 1, which inhibits both host gene expression and virus-and interferon (IFN)-dependent signaling. Thus, nsp1 is a promising target for drugs, as inhibition of nsp1 would make SARS-CoV more susceptible to the host antiviral defenses. To gain a better understanding of nsp1 mode of action, we generated and analyzed 38 mutants of the SARS-CoV nsp1, targeting 62 solvent exposed residues out of the 180 amino acid protein. From this work, we identified six classes of mutants that abolished, attenuated or increased nsp1 inhibition of host gene expression and/or antiviral signaling. Each class of mutants clustered on SARS-CoV nsp1 surface and suggested nsp1 interacts with distinct host factors to exert its inhibitory activities. Identification of the nsp1 residues critical for its activities and the pathways involved in these activities should help in the design of drugs targeting nsp1. Significantly, several point mutants increased the inhibitory activity of nsp1, suggesting that coronaviruses could evolve a greater ability to evade the host response through mutations of such residues.Vertebrate cells use an array of sensors to detect infection by viruses and other microorganisms; triggering of such sensors activates signaling cascades that lead to the expression of host defense genes necessary to fight off the infection . Naturally, viruses evolved to encode not only proteins necessary for their replication, such as their polymerase and capsid protein, but also security proteins that counteract the host defenses . As unimpeded induction and action of interferons (IFNs) in response to viral infection leads to a powerful antiviral state that strongly restricts replication of most viruses , viral IFN antagonists constitute a major class of security proteins [4, 5] . IFN antagonists can be further distinguished based on their ability to i) inhibit the virus-dependent signaling necessary for the production of IFNs and other cytokines; ii) inhibit the IFN-dependent signaling necessary for the induction of antiviral genes; or iii) inhibit the activity of antiviral proteins.The Severe Acute Respiratory Syndrome (SARS) Coronavirus (CoV) is an enveloped, positive-stranded RNA viruses that can cause a severe respiratory disease . Its genome consists of a ,30 kb linear, non-segmented, capped, polycistronic, polyadenylated RNA molecule, the first two-third of which is directly translated into two large polyproteins. These two polypeptides are processed into 16 non-structural proteins (nsps), forming the replicase complex, which is active in the cytoplasm in close association with cellular membranes. Transcription and replica-tion of the coronaviral genome leads to the expression of open reading frames located in the remaining third of the genome, which encode structural proteins such as S (spike), E (envelope), M (membrane) and NP (nucleocapsid protein), as well as other open reading frames (ORFs) whose functions remain to be fully characterized .A number of putative IFN antagonists have been identified in the SARS-CoV genome through a variety of screens: nsp1, nsp3, nsp7, nsp15, ORF3a, ORF3b, ORF6, M and NP . However, validation of their role as genuine IFN antagonists in the context of the live virus was obtained only for nsp1 so far [16, 25] . Two mechanisms were identified to account for the IFN antagonist activity of nsp1, general inhibition of host gene expression [13, 16, 25] and inhibition of antiviral signal transduction . Specifically, nsp1 inhibits host gene expression by decreasing translation efficiency and by destabilizing mRNAs [13, 25, 26] . Additionally, nsp1 expression inhibits the three virusdependent signaling pathways that lead to activation of the transcription factors ATF2/c-Jun, IRF3/IRF7, and NF-kB, preventing full induction of virus-inducible genes, such as IFNB ; moreover, nsp1 decreases the levels of STAT1 phosphorylation, inhibiting the induction of genes by IFN-a and IFN-c .The pleiotropic activities of nsp1 could be the result of multiple host factors interacting with nsp1 or of a single factor whose interaction with nsp1 eventually impacts distinct cellular pathways. In this study, we carried out a detailed mutational analysis of SARS-CoV nsp1 to distinguish between these possibilities. We determined that SARS-CoV nsp1 main functional activities, inhibition of host gene expression and inhibition of antiviral signaling, could be genetically separated. Mapping of residues involved in inhibitory functions on the surface of nsp1 suggests it interacts with distinct host factors to inhibit host gene expression and antiviral signaling.293T cells (ATCC CRL-11268) are a SV40 large T antigenexpressing and highly transfectable derivative of 293 cells, which are derived from human embryonic kidney cells transformed with human adenovirus type 5, were grown at 37uC, 5% CO 2 , in Dulbecco's Modified Eagle medium containing 10% Hyclone bovine growth serum, 50 U/mL penicillin and 50 mg/mL streptomycin. Sendai Virus (SeV, from SPAFAS) was used at 25 hemagglutinin units (HAU)/mL. Viruses and cell lines were manipulated according to BSL-2 guidelines under a protocol approved by the Institutional Biosafety Committee.Mutations in SARS-CoV nsp1 were introduced by PCR. All nsp1 coding regions were cloned into pcDßAF 3 m1 downstream of a N-terminal triple flag-tag sequence, as described , and verified by sequencing; primer sequences and plasmid sequences are available upon request. The virus-and interferon (IFN)responsive reporter construct ISREx3CAT contains three copies of the ISG15 gene IFN-Stimulated Response Element (ISRE) and the TATA box of the E1b gene to drive expression of the Chloramphenicol Acetyl Transferase (CAT) gene [27, 28] . Plasmids for expression of IRF3, STAT1a, enhanced green fluorescent protein (eGFP), luciferase (P. pyralis), and b-galactosidase (lacZ) were described previously [16, 28, 29] . All proteins were expressed from the Cytomegalovirus (CMV) enhancer.293T cells (2.0610 6 in 100 mm dishes) were transfected by calcium phosphate coprecipitation . Transfections consisted of either ISREx3-CAT reporter or pcDßA-STAT1a or pcDßA-IRF3, combined with pcDbAF 3 nsp1-wildtype (wt) or pcDbAF 3 nsp1-mutant (m1-m39) plasmid, as well as the following plasmids reporters: pCMV-lacZ, pcDbA-luciferase, and pcDbA-eGFP. Transfection efficiency is routinely .95% as determined by in situ lacZ staining or eGFP expression. Transfected cells were trypsinized and aliquoted for individual treatment conditions (control, hIFNa, SeV). Cell extracts were made with M-Per (Pierce). Standardized CAT, luciferase and b-galactosidase assays were performed, with appropriate dilutions to remain in the linear range for each assay : all extracts for CAT activity were diluted 100-fold and activity is computed as percent conversion, where the counts of mono-acetylated-[ 14 C]-chloramphenicol is divided by total (mono-acetylated+un-acetylated)-[ 14 C]-chloramphenicol counts (di-acetylated-[ 14 C]-chloramphenicol is not formed when the assay is carried out in the linear range); luciferase activity is expressed as relative light units (RLU); all extracts for b-galactosidase activity were diluted 10-fold and activity is expressed in absorbance units at 405 nm that results from the release of ortho-nitrophenol from ortho-nitrophenyl-bgalactoside. Statistical significance was determined using a two-tail t-Test assuming unequal variances. All experiments replicated a minimum of three times. Surface residues that could be important for function were identified and mutated. (B) SARS-CoV nsp1-wt and mutants were cotransfected into 293T cells with reporter plasmids. Plasmid expressing eGFP was used to visually inspect for high transfection efficiency. (C) Standard assays were performed: luciferase and b-galactosidase assays were used as a proxy to measure level of nsp1 inhibition of gene expression; CAT gene under the control of three ISRE copies was used as a proxy for nsp1 inhibition of interferon-and virus-dependent signaling; immunoblots were run to directly measure nsp1 inhibition of STAT1 phosphorylation and IRF3 dimerization, and nsp1 levels. doi:10.1371/journal.pone.0062416.g001Cell extracts were analyzed by sodium dodecyl sulfate (SDS)polyacrylamide gel electrophoresis (PAGE) and immunoblotting with the following commercial primary antibodies: anti-STAT1 (sc-592, Santa Cruz Biotech.); anti-phospho-Tyr701-STAT1 (#9171L, Cell Signaling) to detect levels of total and phosphorylated STAT1; and anti-FLAG-M2 (Sigma) to detect FLAG-tagged nsp1 proteins. IRF3 was detected following deoxycholate (DOC)-PAGE as described ; monoclonal SL-12 (BD Pharmingen) detects both IRF3 monomers and dimers. Primary antibodies were detected by immunoglobulin-horseradish peroxidase conjugated secondary antibody (Promega) and visualized using the PerkinElmer Life Sciences chemiluminescence detection system. Bradford Assay was used to determine protein concentration of extracts, and ten micrograms of total protein per sample was loaded onto gels. Quantitation of immunoblots was done using ImageJ . To calculate relative STAT1 phosphorylation, the quantity of phosphorylated STAT1 was divided by the quantity of total STAT1 for each time point. To calculate the relative IRF3 dimerization, the quantity of IRF3 dimer was divided by the quantity of IRF3 monomer plus the dimer for each time point. All experiments and measurements are replicated a minimum of three times.SARS-CoV nsp1-wt exhibits multiple inhibitory effects on host functions. In addition to inhibiting host gene expression and virusand IFN-dependent signaling, SARS-CoV nsp1 also inhibits cell cycling and decreases cell death in transfected cells . We hypothesized that these pleiotropic activities could be the result of multiple interactions between nsp1 and host proteins. If true, the prediction would be that individual residues in nsp1, which if mutated, would differentially affect the ability of nsp1 to inhibit host activities; if untrue no such mutations would be identified. Map of SARS-CoV nsp1 mutants generated. SARS-CoV nsp1 is a small 180-residue protein whose structure has been solved by NMR for residues 13-129. We hypothesized that surface residues would be important for interaction between nsp1 and host target protein(s) that would mediate the inhibitory effects of nsp1. Three rounds of mutagenesis targeting surface residues were carried out in this study (A-C). Mutant nsp1-m5 was identified to potentially mediate nsp1 inhibition of host IFN-and virus-dependent signaling (A) but not inhibition of host gene expression and was explored further. Mutant nsp1-m16 was generated to attempt to further resolve the nsp1-m5 loss of function, and mutants -m27 through -m30 (C) were generated to refine important residues identified in nsp1-m16 (B). doi:10.1371/journal.pone.0062416.g002To distinguish between these possible modes of action, we carried out a mutational analysis of SARS-CoV nsp1. Before the structure of SARS-CoV nsp1 was determined, we initially selected a few amino acids that were predicted to be solvent exposed (mutants m1-m8); after the NMR structure of SARS-CoV nsp1 was reported , we selected additional solvent exposed residues for replacement (mutants m9-m39). The nature of the selected residues was changed, e.g., positively charged to negatively charged, so that if the amino acid was involved in an interaction with a host factor, that interaction would likely be disrupted. This abrogation of nsp1 to host-protein interactions would be reflected in the attenuation of standard assays measuring the level of nsp1 inhibition of host functions. The nsp1 mutants were generated by PCR, cloned into plasmids ( Figure 1A ), and were co-transfected into 293T cells along with plasmids expressing luciferase, bgalactosidase, eGFP, and a reporter plasmid expressing the CAT gene under the control of a virus-and IFN-inducible ISRE, ISREx3-CAT ( Figure 1B) . Standardized luciferase and b-galactosidase assays were used as a proxy for an nsp1 mutant's ability to inhibit host gene expression; inhibition of luciferase activity by nsp1-wt is mild (,40%) while inhibition of b-galactosidase is strong (,90%) ; other investigators used different reporters as proxy for nsp1 effects on host gene expression [13, 25, 33, 34] . Expression of the ISREx3CAT reporter is stimulated by both IFN treatment and SeV infection and coexpression of SARS-CoV nsp1-wt results in an inhibition of that stimulation, which we previously showed was due to an inhibitory effect on signaling and not on expression of the CAT gene . Thus, CAT expression is used here as a proxy for an nsp1 mutant's ability to inhibit virusand IFN-dependent signaling. Immunoblots were run and quantitated to measure the expression levels of all nsp1 mutants in these experiments ( Figure 1C ). Direct measurement of virusand IFN-dependent signaling was also carried out (see below).Eight nsp1 variants were generated (nsp1-m1 through nsp1-m8) as an initial test of our hypothesis ( Figure 2A ). As expected, variants were discovered that exhibited attenuated (nsp1-m1) or almost complete loss of inhibition (nsp1-m4, -m7 and -m8) of both signaling ( Figure 3A ) and gene expression ( Figure 3B ,C). Expression of all nsp1 mutants was compared to that of nsp1-wt ( Figure 3D ); some variants, such as nsp1-m8, were consistently expressed at such low levels in transfected cells that we removed them from further consideration in our measurements (Table 1) . Of particular interest was nsp1-m5, which exhibited a statistically significant attenuation of inhibition of antiviral signaling while maintaining a strong inhibition of b-galactosidase expression, similar to that obtained with nsp1-wt. Variant nsp1-m5 phenotype of partial inhibition of host signaling with strong inhibition of bgalactosidase expression is consistent with our hypothesis that nsp1 exerts its pleiotropic inhibitory effects by interacting with more than one host protein. The inhibitory functions for all nsp1 mutants are shown in this table expressed as averaged percent compared to nsp1-wt (set to 100 percent). Percent inhibitions were also normalized to expression levels. No values are reported for mutants nsp1-m8, -m25, and -m26 as they had no significant inhibitory activities and were expressed at a level too low to be measured. After expression normalization, many nsp1 mutants exhibited increased inhibitory abilities compared to nsp1-wt; for example, nsp1-m19 exhibits nearly four-fold more inhibitory functions over nsp1-wt. Even after normalization, nsp1-m16 continues to show a loss of inhibition of host IFN-and virusdependent signaling while maintaining strong inhibition of host gene expression. (*indicates significant as compared to empty vector; # indicates significant as compared to nsp1-wt; significant is defined as a t-Test P-value # 0.05). doi:10.1371/journal.pone.0062416.t001Distinct Inhibitory Activities of SARS-CoV nsp1 can be Genetically SeparatedWe carried out a second round of nsp1 mutagenesis (nsp1-m9 through nsp1-m26, Figure 2B ) to identify additional residues that may play a role in nsp1 activities, with particular attention to the region surrounding the residues mutated in nsp1-m5 (D33K, E36K, E37K) that appears to be involved in nsp1 inhibition of antiviral signaling.Analysis of the second round of mutants identified some noteworthy phenotypes (Figure 4) . Mutants nsp1-m14, -m15, -m23 inhibited the virus-and IFN-inducible CAT reporter more strongly than nsp1-wt, with nsp1-m14 essentially abolishing any induction of the CAT reporter and more strongly inhibiting host gene expression than nsp1-wt as determined by both the luciferase and bgalactosidase assays. Variants nsp1-m16 (D33R, E36R, E37R, S40R) and nsp1-m17 (E41R, R43E, E44R, K47E) were selected for their immediate proximity to the residues mutated in nsp1-m5 to attempt to further separate the inhibition of antiviral signaling from the inhibition of host gene expression. Mutants nsp1-m17 and -m18 (incorporating both m16 and m17 mutations), were not significantly different from nsp1-m5 in their ability to inhibit the CAT reporter stimulation by IFN or SeV. By contrast, nsp1-m16 exhibited a complete loss of inhibition of antiviral signaling ( Figure 4A ), while maintaining strong inhibition of b-galactosidase expression ( Figure 4C ). In addition, mutations introduced in the intrinsically disordered C-terminal segment (amino acids 128-180) of nsp1, expressed relatively poorly (mutants nsp1-m12 and -m13) or barely at all (mutants nsp1-m25 and -m26, Figure 4D ). Mutant nsp1-m25 targeted the same residues as nsp1-m8, but instead of replacing the acidic residues with lysines, arginine residues were substituted to test the possibility that the lack of nsp1-m8 expression was due to ubiquitination of those lysines; rather, our results suggest that the charge change was sufficient to interfere with expression of nsp1. Variants nsp1-m25 and -m26 were consistently expressed at such very low levels in transfected cells, displaying no inhibitory activities, that we removed them from further consideration in our measurements (Table 1) .Thus, we found that the disordered C-terminal tail, including a putative a-helix targeted by the m26 mutation, is important for proper expression of nsp1. Importantly, mutations were identified that increased the inhibitory effects of nsp1 on a virus-and IFNinducible reporter, suggesting a potential for the evolution of increased evasion from the host response for SARS-CoV. With the generation of nsp1-m16, which further attenuated the nsp1-m5 phenotype, we confirm the hypothesis that the different inhibitory properties of nsp1 can be genetically separated.A third round of mutations was carried out to further explore the surface of nsp1 and complete the map of functional residues. Mutants -m27 through -m30 were generated in an attempt to tease Table 1 . doi:10.1371/journal.pone.0062416.g004 out residues accounting for the separation of inhibition of signaling and gene expression activities of nsp1-m16; mutants -m31 through -m34 targeted residues located on the surface of nsp1, midway between the functional domains emerging from the analysis of the first two rounds of mutations; and mutants -m35 through -m39 affected residues in the intrinsically disordered C-terminus of nsp1. Intriguingly, mutants -m27 through -m30 had a very different phenotype than nsp1-m5 or -m16 from which they were derived, with nsp1-m27 and -m30 inhibiting the IFN-and virus-inducible CAT reporter and both b-galactosidase and luciferase activities more strongly than nsp1-wt ( Figure 5 ). The remaining mutants, nsp1-m31 through nsp1-m39, displayed no phenotype distinct from any of those already described in this study.With the third round of nsp1 mutagenesis, we completed a map of functional residues important for the inhibition of gene expression and antiviral signaling. We were able to isolate a putative, minimal set of residues involved solely in the inhibition of host IFN-and virus-dependent signaling, but not in the inhibition of host gene expression. No mutations were identified that resulted in a loss of the inhibition of gene expression without also resulting in a loss of the inhibition of antiviral signaling and additional mutations were identified that increased the inhibitory effects of nsp1 on a virus-and IFN-inducible reporter.We next investigated the ability of a representative set of SARS-CoV nsp1 mutants to directly inhibit antiviral signaling by assessing STAT1 phosphorylation and IRF3 dimerization, as described previously . To this end, 293T cells were transfected with a plasmid expressing an nsp1 mutant, a plasmid expressing eGFP for transfection efficiency, and a plasmid expressing STAT1a. Transfected cells were treated with IFNa for 0, 20, and 60 minutes, and extracts were analyzed by immunoblotting to 1) confirm the expression of each nsp1 mutant, and 2) to quantitate the level of phosphorylated STAT1 and the level of total STAT1. Similarly, to determine the level to which each nsp1 mutant inhibited IRF3 dimerization, 293T cells were transfected with a plasmid expressing an nsp1 mutant, a plasmid expressing eGFP for transfection efficiency, and a plasmid expressing IRF3. Transfected cells were treated with SeV for 0, 6, and 9 hours, and extracts were analyzed by immunoblotting 1) confirm the expression of each nsp1mutant, and 2) to quantitate the levels IRF3 monomers and dimers.As shown in Figure 6 , mutants that decreased or lost their ability to inhibit antiviral signaling based on the CAT assay (Figures 3, 4 , 5) indeed displayed partial or complete loss of inhibition of both STAT1 phosphorylation and IRF3 dimerization (nsp1-m4, -m12, -m21, and -m22). Expression of nsp1-m16 resulted in no decrease Figure 5 . Third round of mutagenesis refines map of nsp1 residues important for inhibition of host signaling or gene expression. SARS-CoV nsp1-m27 through nps1-m39 mutants were tested for (A) inhibition of host IFN-and virus-dependent signaling using the ISREx3-CAT reporter, followed by inhibition of host gene expression using (B) luciferase and (C) b-galactosidase assays. CAT activity values correspond to percent chloramphenicol acetylation using cell extracts diluted 100-fold, luciferase activity is determined in straight extracts and is expressed in RLU and bgalactosidase activity corresponds to released ortho-nitrophenol absorption at 405 nm using extracts diluted 10-fold. Mutants nsp1-m27 through nsp1-m30 were derived from nsp1-m16 to define minimal residues important for inhibition of host signaling. Immunoblots of nsp1 mutants are quantitated in (D). Error bars are 6 standard error; significance for all mutants is listed in Table 1 . doi:10.1371/journal.pone.0062416.g005 in either STAT1 phosphorylation or IRF3 dimerization, confirming the loss of antiviral signaling inhibition observed by CAT assay. Since nsp1-m16 retained the ability to inhibit gene expression, this mutation genetically separates the two inhibitory activities of SARS-CoV nsp1. By contrast, nsp1-m14, -m27 and -m30, which showed increased inhibition of the CAT reporter, had wildtype or partially attenuated inhibition of antiviral signaling (Fig. 4) . Therefore, the increased inhibition of the CAT reporter by nsp1-m14, -m27 and -m30 must be accounted for by an inhibitory effect on CAT mRNA expression, consistent with the observed increased inhibition of bgalactosidase and luciferase activities by these mutants. Table 1 is a compilation of all the nsp1 mutants generated in this study. Fold inhibition for nsp1-wt were 6.1, 1.6, 8.4 and 8.6, for b-galactosidase, luciferase, IFNa and SeV, respectively, and were set to 100 for ease of comparison. Point mutations in proteins often affect their expression levels by modulating transcription, mRNA stability, translation and/or protein stability, and we found this to be true for SARS-CoV nsp1 mutants as well. Since SARS-CoV nsp1 inhibitory activities are dose-dependent , we additionally normalized the measured inhibition to the levels of expression for each mutant (Table 1) . Expression-normalized inhibition was used to classify the nsp1 mutants according to their ability to attenuate or increase inhibition of b-galactosidase expression and/or inhibition of antiviral signaling, leading to the identification of six broad groups, A-F, Table 1 (inhibition of luciferase expression was not considered as it was only reduced 1.6-fold by nsp1-wt). Group A mutant displayed partial attenuation of both gene expression and antiviral signaling; group B mutants had loss most inhibition of both gene expression and antiviral signaling; group C mutants; group C displayed differential effects on gene expression and signaling, with strongly reduced inhibition of signaling and maintained inhibition of gene expression; group D had increased inhibition of signaling and maintained inhibition of gene expression; group E displayed increased inhibition of both signaling and gene expression; and group F had no clear phenotype. Some mutants fell somewhat in between categories and are positioned at the edge of their groups.The lower expression levels of some nsp1s observed in transiently transfected cells could be due to self-inhibition. Their expression could be higher during virus infection because of less self-inhibition in that context, and thus their IFN antagonism may be correspondingly higher. Consistent with the possibility of selfinhibition in transfection experiments, it was recently shown that natural SARS-CoV NP subgenomic mRNA escapes nsp1mediated degradation, although translation of this mRNA is still inhibited by nsp1 . This indicates some level of specificity for the targeting of mRNA degradation, but its molecular basis remains elusive. However, many mutants characterized in this study may be useful to elucidate the molecular basis of this function. Some mutations in SARS-CoV nsp1 changed the profile of the genes whose expression was affected: nsp1-wt strongly inhibited b-galactosidase expression but had little effect on the other genes, while other mutants showed increased luciferase inhibition with no change in b-galactosidase inhibition (e.g., nsp1-m22, -m37) and others also very strongly inhibited CAT expression (nsp1-m14, -m27, Figure 4 ).SARS-CoV nsp1 could exert its multiple activities by interacting with a single factor that impinges on multiple pathways or with distinct factors that each account for some of its activities. Here, Figure 6 . SARS-CoV nsp1 mutants show altered inhibition of signaling molecules. To confirm previous results, selected nsp1 mutants were tested for ability to directly inhibit signaling molecules. Cell extracts were separated by SDS-PAGE, and probed for levels of active phosphorylated STAT1 (P-STAT1) and for levels of total STAT1 (STAT1) (A). Cell extracts were separated by native-PAGE (B), and probed with anti-IRF3 antibody to detect both monomeric (inactive) and dimeric (active) forms. Immunoblots were quantitated and the relative activity of signaling molecules was calculated. Immunoblots confirm previous results showing that nsp1-m16 had indeed lost its ability to strongly inhibit both the phosphorylation of STAT1 and the dimerization of IRF3 after cells had been stimulated with either IFNa or SeV. Error bars are standard error; P-values are result of a t-Test and significance for all mutants is shown in (C). doi:10.1371/journal.pone.0062416.g006 we carried out a detailed mutational analysis and identified functional surface residues of SARS-CoV nsp1 that are differentially involved in its various inhibitory functions. A summary of SARS-CoV mutations analysis is presented in Figure 7 , where the mutated amino acid residues are color-coded according to their phenotypes ( Table 1) . None of the mutants tested displayed a significant difference in their ability to target IFN-versus virusdependent signaling. Few mutations had no effect on nsp1 activities (purple, group F); some mutations partially (blue, group A) or almost completely (green, group B) attenuated the ability of nsp1 to inhibit both antiviral signaling and gene expression; other mutations strongly repressed the ability of nsp1 to inhibit antiviral signaling without affecting inhibition of gene expression (yellow, group C); other mutations increased the ability of nsp1 to inhibit antiviral signaling without affecting inhibition of gene expression (orange, group D); finally, some mutations (red, group E) increased both the inhibition of antiviral signaling and gene expression. Thus, our data indicate that inhibition of host gene expression and Figure 7 . Mapping of SARS-CoV nsp1 mutants onto its 3D-structure. The structure of SARS-CoV nsp1 was solved from a.a. 13 to 127 and is displayed using the PyMOL software. Top row (A-C), the backbone structure is displayed in cartoon form, it consists of a mixed parallel/antiparallel six strand b-barrel, with the a-helix (cyan) at one barrel opening and the 3 10 -helix (green) alongside the barrel. Middle row (D-F) displays protein electrostatic surface charge colored blue for positive regions and red for negative regions, with scale (G). Bottom row (H-J), atoms from amino acids that were mutated in this study are shown as space-filling spheres on top of the backbone structure in cartoon form; purple amino acids correspond to mutations that did not affect nsp1 function; blue amino acids correspond to mutations that partially attenuated both inhibition of signal transduction and inhibition of gene expression; green amino acids correspond to mutations that abolished both inhibition of signal transduction and inhibition of gene expression; yellow amino acids correspond to nsp1-m5/2m16, which maintained inhibition of gene expression but lost inhibition of IFN-and antiviral-signal transduction; orange amino acids correspond to mutations that maintained inhibition of gene expression but increased inhibition of signal transduction. Red mutations displayed increased inhibition of ISREx3CAT, b-galactosidase and luciferase reporters. The structures in each row are rotated along the vertical axis by 120u. A putative amphipathic a-helix in the C-terminus of nsp1 (a.a.169-177) is displayed in a wheel representation, a.a. are color coded: hydrophobic, grey; hydrophilic, purple; acid, red; and basic, blue (K). Primary sequence of SARS-CoV nsp1 (L): grayed amino acids correspond to regions disordered in the published NMR structure (a. a. 1-12, a.a. 128-180) ; amino acids were color-coded as in (H-J); the boxed sequence corresponds to the putative amphipathic a-helix displayed in (K). doi:10.1371/journal.pone.0062416.g007 inhibition of antiviral signaling could be genetically separated and suggest that distinct but overlapping surfaces of SARS-CoV nsp1 are involved in interactions with factors that mediate inhibition of gene expression (red, green and blue) and with factors that mediate inhibition of antiviral signaling (red, orange, yellow, green and blue) (Figure 7) .Our results are consistent with SARS-CoV nsp1 targeting different host factors to mediate these inhibitory activities. One of these host factors has been identified as the 40S subunit of the ribosome and can account for nsp1 inhibition of translation . However, the factors responsible for increased mRNA degradation and for inhibition of signaling remain to be discovered. In addition, a recent two-hybrid screen identified a large number of host proteins interacting with SARS-CoV nsp1, including several members of the immunophilin and calcipressin families . This ability of SARS-CoV nsp1 to interact with multiple proteins despite its small size (180 residues) may be mediated in part by its disordered domains (a. a. 1-12, a.a.128-180 ). Intrinsically disordered domains are present in numerous proteins and their structural plasticity allows them to adopt different partial or fully ordered conformations when interacting with different ligands, increasing the size of the repertoire of proteins with which they can interact .A number of viruses encode IFN antagonists that function nonspecifically to target a central cellular function such as transcription or translation to prevent expression of IFNs and IFNinducible genes. By contrast, other IFN antagonists specifically target IFN-or virus-dependent signaling, or IFN-induced proteins with antiviral function [4, 5] . In a first approximation, the mechanism by which IFN production/action are inhibited does not matter; the net result is a deficient IFN response. SARS-CoV nsp1 is unusual in that it inhibits gene expression and it inhibits both virus-and IFN-dependent signaling . A mutant such as nsp1-m16, which lacks any ability to inhibit antiviral signaling while maintaining inhibition of gene expression, may prove useful to determine the relative importance of various inhibitory functions in evasion from the innate immune response and in overall virulence. Inhibition of signaling may be a more efficient strategy to antagonize the antiviral response than inhibition of gene expression due to the relative scarcity of signaling proteins vs. the abundance of components of the gene expression machinery, such as ribosomes. Thus, the levels of nsp1 necessary to inhibit a signaling target will be reached earlier during the replication cycle than those necessary to inhibit translation. The need for IFN antagonists to reach a threshold level before becoming effective is illustrated by the observation that at an early time point during SARS-CoV infection, IRF3 is observed to translocate to the nucleus whereas it is excluded from the nucleus at a later time point .A specific inhibition of the IFN genes as opposed to a general inhibition of gene expression is expected to alter the immune response. Indeed, even subtle changes in IFN basal and induced expression are associated with disease states , and thus more selective inhibition of the induction of IFNs and IFN-inducible genes as observed with SARS-CoV may also impact the severity of the disease it causes. It has been shown that genes encoding inflammatory mediators are expressed at relatively much higher levels in SARS patients and in vitro tissue culture models than IFN genes when compared with other viral infections . SARS-CoV nsp1 inhibitory effects on antiviral signaling and its stimulatory effect on the Calcineurin/NFAT pathway likely contribute to the increased production of inflammatory mediators in SARS patients. Such an exaggerated inflammatory response leading to a ''cytokine storm'' is thought to account at least in part for SARS-CoV morbidity and mortality [42, 43, 45, 36] .Long-term replication of a virus in a host population may result in evolution towards decreased virulence, defined as harm to the host, as less damage to the host may increase the fitness of a virus. Consistent with such an evolutionary successful solution, several human coronaviruses (HCoVs) have an established niche in humans and cause no or mild disease in healthy adults 11, 12] . By contrast, SARS-CoV emergence in humans from another animal niche caused severe disease and even death in a substantial fraction of infected individuals . Significantly, several SARS-CoV nsp1 mutations increased its ability to function as an IFN antagonist (e.g., nsp1-m14, -m27 and -m30, Figures 4&5 ). This increase was not due to increased inhibition of signaling ( Figure 6 ), but was due to increased inhibition of gene expression (b-galactosidase, luciferase and CAT, Figures 4&5) . Thus, our work indicates that a change in a few amino acids of SARS-CoV nsp1 was sufficient to substantially increase its overall IFN antagonism, which may lead to increase virulence. Comparable mutations may occasionally emerge in other human coronaviruses and lead to increased virulence, before selective pressure eventually eliminates such mutants. Other mutations tested decreased SARS-CoV nsp1 IFN antagonism and such mutations may have been part of SARS-CoV adaptation to the human population had it not been eradicated by public health measures.\n",
            "\n",
            " Title:  Transmissible Gastroenteritis Virus Papain-Like Protease 1 Antagonizes Production of Interferon-through Its Deubiquitinase Activity\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1m16uRvn6XVn"
      },
      "source": [
        "Let's sum up how good our results was:\n",
        "\n",
        "\n",
        "\n",
        "1.   With a quick glance, we can see that the possible answers are not such good as expected, and also the best possible answers are not usually at the top of the returned list.\n",
        "\n",
        "2.   The time it takes to run each question, is around ~3.5 min. That's because we need to search across **all the dataset** each time we provide a new question. That's a lot for each question.\n",
        "\n",
        "\n",
        "We are having these bad results because the only thing we did is to compare the similarity of each questions and each text of each paper in the dataset. Below, we have implemented a far more better solution for our Q&A problem.\n",
        "\n",
        "We are keeping this code anyways because it's more like a sentence-similarity program rather than a Q&A implentation.\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jjNYDroDBLNe"
      },
      "source": [
        "### (2.3): Second Approach: Bidirectional & Cross Encoder."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "t6jR-OZgBu1r"
      },
      "source": [
        "For the second sentences embeddings approach, we were inspired from the examples of the [UKPLab](https://github.com/UKPLab/sentence-transformers)."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "y7-F7tYtCbH2"
      },
      "source": [
        "In the end of this section of our notebook, we can input a query or a question. The script then uses semantic search to find relevant passages in the given [papers](https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/historical_releases/cord-19_2020-03-13.tar.gz)."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ScBo824pDQXB"
      },
      "source": [
        "For semantic search, we use SentenceTransformer('msmarco-distilbert-base-v2') and retrieve 100 potentially passages that answer the input query."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "8zEK3BqADS3x"
      },
      "source": [
        "Next, we use a more powerful CrossEncoder (cross_encoder = CrossEncoder('cross-encoder/ms-marco-TinyBERT-L-6')) that scores the query and all retrieved passages for their relevancy. The cross-encoder is neccessary to filter out certain noise that might be retrieved from the semantic search step."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "MwBeqCJiDfZE"
      },
      "source": [
        "import json\n",
        "from sentence_transformers import SentenceTransformer, CrossEncoder, util\n",
        "import time\n",
        "import gzip\n",
        "import os\n",
        "import torch\n",
        "\n",
        "if not torch.cuda.is_available():\n",
        "  print(\"Warning: No GPU found. Please add GPU to your notebook\")"
      ],
      "execution_count": 19,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "IutYVzMgF_Hd"
      },
      "source": [
        "First, we'll initialize our model and we are going to define the number of the papers we want to retrieve."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7VydUE34F95n",
        "outputId": "505ba823-0fa1-405a-e5c1-a7b642b8c935"
      },
      "source": [
        "#We use the Bi-Encoder to encode all passages, so that we can use it with sematic search\n",
        "model_name = 'msmarco-distilbert-base-v2'\n",
        "bi_encoder = SentenceTransformer(model_name)\n",
        "top_k = 100     #Number of passages we want to retrieve with the bi-encoder"
      ],
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "100%|██████████| 245M/245M [00:10<00:00, 23.4MB/s]\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7JAl_i-sH-39"
      },
      "source": [
        "The bi-encoder will retrieve 100 documents. We use a cross-encoder, to re-rank the results list to improve the quality.\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 268,
          "referenced_widgets": [
            "a98a9b6071ef4276bd5881c91e236d10",
            "7502a5be28d94e32bfea582d4ac2f499",
            "5a4b239dfaf14b3b90f0a996138799e2",
            "7cb36365c9924a82a5c5994625f7a000",
            "f7ae808a81f24ca396097a2648bfe5a6",
            "b25a7a65d94145abaec76c26dae996ae",
            "457b7c8f86d54de8a5d3d2b42496c235",
            "deb748b8cfcb473fb17451d100658b19",
            "9b7fbf48285847f8b780ff10375aed42",
            "01636534b6104bfbbe1a3cc4a405c87e",
            "6e57c6786ddc4bc39d1c5cb0974da9f6",
            "55dc802e29e44f68a2522b042118b69e",
            "35830c8f476447dd9c09daa4d1e753b8",
            "c04769aedd15409b8d102ec8685e1f83",
            "27a06abddf4f4d8aa6e99b48dffe7b5d",
            "30b4f99f902048a6a8c3eaff9b18de27",
            "7e52fbca393842c1bd6d37e036d49bf2",
            "266e209f7c344e5bb3beab2b7f28fc29",
            "10a66e84bc9a4e88b2df1bad9d384b18",
            "e9936d5360044207a304f537195f7b07",
            "b1fc9319cb094585ae32d98ca62ab3f5",
            "09b6694a2f62457f90279186036a528d",
            "94de9741ed944356a7185de896812c32",
            "6dd29eaa9ede477da606ad9803200cb7",
            "ccd8cac0d9df4c8b914b7059fefde8a1",
            "d2c1f30b66964c5f8b719ad67d2202bd",
            "7bd184a7de7f4fa0af68ced7b7147083",
            "f3f11121a1f84e27be0cbb8d9551177d",
            "60a064cee5514ef384968f587755d0a9",
            "f79574abae4b40d4b0a78152d0f00b5c",
            "f28e72efe5914c98b0c29ca60be70271",
            "3724c930b807465e98182efe8bec6198",
            "bbfc8697e31d4653be93fa5824786558",
            "abefa75157da4d3f961a768c4dd68587",
            "30875a174c274e2ca6c69ec10eac41ae",
            "275a5c32b03143b4a0237164fd2916e7",
            "422bee0204184139a3fcc609b80c27e7",
            "05be2a2814e3488fb5d1555af228c898",
            "c6e8ee5bba5046c8b79a7259e4866369",
            "243be1e9d83f4f4bb0809ad0a76e20e0"
          ]
        },
        "id": "-cCQvNkbH9Hh",
        "outputId": "19a4dd66-c5e0-477b-8aa8-f66e84c1e0ff"
      },
      "source": [
        "# cross_encoder = CrossEncoder('cross-encoder/ms-marco-TinyBERT-L-6')\n",
        "cross_encoder = CrossEncoder('cross-encoder/ms-marco-TinyBERT-L-6')"
      ],
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "a98a9b6071ef4276bd5881c91e236d10",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=612.0, style=ProgressStyle(description_…"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "9b7fbf48285847f8b780ff10375aed42",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=267871721.0, style=ProgressStyle(descri…"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "7e52fbca393842c1bd6d37e036d49bf2",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=231508.0, style=ProgressStyle(descripti…"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "ccd8cac0d9df4c8b914b7059fefde8a1",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=112.0, style=ProgressStyle(description_…"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "bbfc8697e31d4653be93fa5824786558",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=541.0, style=ProgressStyle(description_…"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "l_9tWpkuIQ6k"
      },
      "source": [
        "As dataset, we use the first version of\n",
        "the COVID-19 Open Research Dataset (CORD-19). We split these papers into paragraphs and encode them with the bi-encoder (**msmarco-distilbert-base-v2 model**)."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 67,
          "referenced_widgets": [
            "9d79b8acfab04f66aa002a39dacb2614",
            "46dddac7ec7249dda04d9ca3022d4e2f",
            "2d67fbdde70f41ffb26d4309dd2abcb1",
            "31da0f5d98924a9595977b50db7f19b5",
            "f13a9a5e328442f6935a67046ff8ab00",
            "9f8e9c0b73ca4b34a287e1db06295238",
            "3ee266e550024c009e14f2398ae62c2b",
            "2c246241ed5047db96d068d23712d139"
          ]
        },
        "id": "YDHESQW0R_xC",
        "outputId": "aad4529e-d053-454b-b78f-5a8172a69dd8"
      },
      "source": [
        "sentence_embeddings = bi_encoder.encode(sentences, convert_to_tensor=True, show_progress_bar=True)"
      ],
      "execution_count": 22,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "9d79b8acfab04f66aa002a39dacb2614",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "HBox(children=(FloatProgress(value=0.0, description='Batches', max=282.0, style=ProgressStyle(description_widt…"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "4xmA26WTUEGR"
      },
      "source": [
        "corpus_embeddings = sentence_embeddings\n",
        "passages = sentences"
      ],
      "execution_count": 23,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "T0ypEuDaUNqL"
      },
      "source": [
        "Next, we have the *search* function. Its utility is to search across all the given dataset to find the best possible answer to each query."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "tfUPzenoUbl3"
      },
      "source": [
        "def search(query):\n",
        "  #Encode the query using the bi-encoder and find potentially relevant passages\n",
        "  start_time = time.time()\n",
        "  question_embedding = bi_encoder.encode(query, convert_to_tensor=True)\n",
        "  question_embedding = question_embedding.cuda()\n",
        "  hits = util.semantic_search(question_embedding, corpus_embeddings, top_k=top_k)\n",
        "  hits = hits[0]  # Get the hits for the first query\n",
        "\n",
        "  #Now, score all retrieved passages with the cross_encoder\n",
        "  cross_inp = [[query, passages[hit['corpus_id']]] for hit in hits]\n",
        "  cross_scores = cross_encoder.predict(cross_inp)\n",
        "\n",
        "  #Sort results by the cross-encoder scores\n",
        "  for idx in range(len(cross_scores)):\n",
        "      hits[idx]['cross-score'] = cross_scores[idx]\n",
        "\n",
        "  \n",
        "  end_time = time.time()\n",
        "\n",
        "  #Output of top-5 hits\n",
        "  print(\"Input question:\", query)\n",
        "  print(\"Results (after {:.3f} seconds):\".format(end_time - start_time))\n",
        "\n",
        "  print(\"Top-5 Bi-Encoder Retrieval hits \\n\")\n",
        "  hits = sorted(hits, key=lambda x: x['score'], reverse=True)\n",
        "  for hit in hits[0:5]:\n",
        "      print(\"\\t{:.3f}\\t{}\".format(hit['score'], passages[hit['corpus_id']].replace(\"\\n\", \" \")))\n",
        "\n",
        "  print(\"Top-5 Cross-Encoder Re-ranker hits \\n\")\n",
        "  hits = sorted(hits, key=lambda x: x['cross-score'], reverse=True)\n",
        "  for hit in hits[0:5]:\n",
        "      print(\"\\t{:.3f}\\t{}\".format(hit['cross-score'], passages[hit['corpus_id']].replace(\"\\n\", \" \")))\n",
        "  \n",
        "  # print(hits[0]['cross-score'])\n",
        "  return hits[0]['corpus_id']\n"
      ],
      "execution_count": 24,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "WwPbTR7SEs4x"
      },
      "source": [
        "For the best answer based on the *Cross-Encoder* , we'll also return the title of the paper."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "fdp8JXsMUeEm"
      },
      "source": [
        "indexes = []\n",
        "\n",
        "def provideQuestions2(questions):\n",
        "\n",
        "  for question in range(len(questions)):\n",
        "\n",
        "    print(question,\"\\b. \\n\")\n",
        "    index = search(questions[question])\n",
        "    indexes.append(index)\n",
        "    title = getTitle(papers, index, 'title')\n",
        "    print(\"\\n Title: \",title)"
      ],
      "execution_count": 25,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "R7-KSNpVGODl",
        "outputId": "8ca79242-5bdb-4913-a836-6c955a799819"
      },
      "source": [
        "provideQuestions2(questions)"
      ],
      "execution_count": 26,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "0 \b. \n",
            "\n",
            "Input question: What are the coronoviruses?\n",
            "Results (after 2.335 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.541\tCoronaviruses (CoVs) are a group of enveloped viruses with a large positive single-stranded RNA genome (∼26-32 kb in length) of the subfamily Coronavirinae under the family Coronaviridae. The complete genome of CoV contains five major open reading frames (ORFs) that encode replicase polyproteins (ORF1ab), spike glycoprotein (S), envelope protein (E), membrane protein (M), and nucleocapsid protein (N) flanked by a 5 -untranslated region (UTR) and a 3 -UTR. Currently, members of the subfamily Coronavirinae are classified into four genera, Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and Deltacoronavirus (Fehr and Perlman, 2015; Su et al., 2016) . CoVs can cause upper and lower respiratory diseases, gastroenteritis, and central nervous system infections in a wide variety of avian and mammalian hosts. Some CoVs are human pathogens that cause mild to severe disease, including NL63 and 229E of the genus Alphacoronavirus, and severe acute respiratory syndrome CoV (SARS-CoV), Middle East respiratory syndrome CoV (MERS-CoV), OC43, and HKU1 of the genus Betacoronavirus (Dijkman and van der Hoek, 2009; Desforges et al., 2014; van den Brand et al., 2015; Lim et al., 2016) . The SARS pandemic originated in Guangdong Province in China between 2002 and 2003 , and spread to 29 countries, resulting in nearly 8,000 cases and 800 deaths worldwide (Donnelly et al., 2004) . Since being discovered in Middle Eastern countries in 2012, MERS-CoV has infected 2,260 people with a current fatality rate of 35.5% (Mackay and Arden, 2015; van den Brand et al., 2015; de Wit et al., 2016) .With the discovery of diverse wildlife-borne CoVs in different regions of the world, previous studies have indicated that bats are the main and original natural reservoirs of Alphacoronavirus and Betacoronavirus (Calisher et al., 2006; Woo et al., 2012) . The special metabolic and immune systems allow bats to tolerate diverse viruses, and the social roosting behavior of many bat species lets them serve as ideal incubators for the occurrence of frequent co-infection, recombination, and intraspecies transmission of CoVs O'Shea et al., 2014; Hu et al., 2015; Lu et al., 2015; Sola et al., 2015) . Although 15 years have passed without a recurrence of the SARS outbreak, the consistent discovery of SARS-related CoVs (SARSr-CoVs) in Rhinolophus bats indicates an ongoing risk of SARS reemergence Balboni et al., 2012; Yang et al., 2013; Wu et al., 2016b; Hu et al., 2017) . Recently, a CoV of Rhinolophus bat origin, swine acute diarrhea syndrome CoV (SADS-CoV), was found to be responsible for the death of 24,693 piglets across four farms in Guangdong Province (Zhou et al., 2018) . Besides similar clinical signs, the outbreak of SADS coincided with infection by another swine enteric CoV, porcine epidemic diarrhea virus (PEDV; Zhou et al., 2018) . It is worth mentioning that a PEDV-related bat CoV, BtCoV/512, was previously identified in Scotophilus kuhlii bats (Tang et al., 2006; Wang et al., 2016; Gerdts and Zakhartchouk, 2017; Zhou et al., 2018) . All of these findings emphasize the importance of continuous monitoring for ecological diversity of CoVs in bats to minimize the impact of potential CoV-related diseases on public health and economic growth.As the original site of the SARS epidemic in humans and SADS outbreak in pigs (Heymann et al., 2013) , Guangdong Province is also the primary region in China in which wildlife is being consumed. Since the supply of wildlife for consumption in Guangdong mainly comes from Yunnan and Guangxi Provinces (Wu et al., 2016b) , understanding the presence and dynamic changes in SARS-or SADS-related bat CoVs within and between these provinces is important for predicting and tracing CoVrelated diseases. Sichuan is next to Yunnan Province, and has high ecological diversity of animal resources, and it is the largest pig-farming province in China. Although PEDV-related pig diseases were reported previously (Cai et al., 2016) , no systematic survey of bat-borne CoVs has been conducted in Sichuan to establish if there are SARS-or SADS-related CoVs.In this study, a survey of bat-borne CoVs was performed in bats of Guangxi, Yunnan, and Sichuan provinces between 2016 and 2017. Samples from Rhinolophus bats in Sichuan and Yunnan Provinces, and S. kuhlii and Cynopterus sphinx in Guangxi Province were found to be CoV-positive, and a large number of sequencing reads classified into diverse alpha-and beta-CoVs were obtained using virome analysis. The characterization of 11 CoVs in bats from different regions provides clues for the evolutionary relationships of PEDV, SARS-CoV, and other batassociated CoVs.Bats were treated according to the guidelines of Regulations for the Administration of Laboratory Animals (Decree No. 2 of the State Science and Technology Commission of the People's Republic of China, 1988) . The sampling was approved by the Ethics Committee of Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College (Approval number: IPB EC20100415). The bat species were initially determined morphologically and subsequently confirmed by sequence analysis of mitochondrial cytochrome b DNA (Tang et al., 2006; Du et al., 2016) . Anal swab samples from captured bats were immersed in virus sampling tubes (Yocon, China) containing maintenance medium and temporarily stored at −20 • C. The samples were then transported to the laboratory and stored at −80 • C. The accurate sampling locations were recorded by place name, latitude, and longitude.The samples of each species were pooled by adding 1 ml from each maintenance medium sample into one new sample tube. The pooled samples, classified by species, were processed with a virusparticle-protected, nucleic acid purification method as described in our previous studies (Wu et al., 2012 (Wu et al., , 2016a (Wu et al., , 2018 . The samples were homogenized and subsequently filtered through a 0.45 µm polyvinylidene difluoride filter (Millipore, Germany). The filtered samples were then centrifuged at 150,000 × g for 3 h at 4 • C. To remove naked DNA and RNA, the pellet was digested in a cocktail of DNase and RNase enzymes. The viral DNA and RNA were simultaneously isolated using a QIAmp MinElute Virus Spin Kit (Qiagen, United States). First-strand viral cDNA was synthesized using the primer K-8N and a Superscript III system (Invitrogen, United States). The cDNA was converted into dsDNA by Klenow fragment (NEB, United States). Sequenceindependent PCR amplification was conducted using primer K. The PCR products were analyzed by agarose gel electrophoresis. All DNA smears larger than 500 bp were extracted by a MinElute Gel Extraction Kit (Qiagen). The extracted nucleic acid libraries were then analyzed using an Illumina HiSeq2500 sequencer, for a single read of 100 bp. The sequence reads were filtered using previously described criteria (Wu et al., 2012 (Wu et al., , 2016a (Wu et al., , 2018 . Reads with no call sites, reads with similarity to the sequencing adaptor and the primer K sequence, duplicate reads, and low-complexity reads were removed.Sequence-similarity-based taxonomic assignments were conducted as described in our previous study (Wu et al., 2012) . Valid sequence reads were aligned to sequences in the NCBI non-redundant nucleotide database and non-redundant protein database using BLASTn and BLASTx, respectively. The taxonomies of the aligned reads with the best BLAST scores (E score <10 −5 ) were parsed by the MEGAN 6 -MetaGenome Analyzer.CoV screening of individual samples was performed by amplifying a 440-bp fragment of the RNA-dependent RNA polymerase (RdRp) gene of CoVs using conserved primers (5 -GGTTGGGACTATCCTAAGTGTGA-3 and 5 -CCATC ATCAGATAGAATCATCATA-3 ), as described previously (Du et al., 2016) .Sequence reads classified into the same virus genus were extracted. The accurate locations of the reads were determined based on the alignment results exported with MEGAN 6. Specific primers were designed from the located sequence reads. Fragments between reads were amplified with nested specific primers and then sequenced. The remaining genomic sequences were determined using 5 -and 3 -rapid amplification of cDNA ends.MEGA6.0 was used to align nucleotide sequences and deduced amino acid sequences using the MUSCLE package and default parameters. The best substitution model was then evaluated by the Model Selection package. Finally, we constructed a maximum-likelihood method using an appropriate model to process the phylogenetic analyses with 1,000 bootstrap replicates. Recombination among CoVs was detected with SimPlot software. All analyses were performed with a Kimura model, a window size of 1000 bp, and a step size of 100 bp.All genome sequences were submitted to GenBank. The accession numbers for the five bat alpha-CoVs were MK211369 to MK211373. The accession numbers for the six bat beta-CoVs were MK211374 to MK211379.Anal swabs from 831 bats of 15 species were collected from Yunnan, Guangxi, and Sichuan Provinces in China between August 2016 and May 2017. All anal swab samples were classified by species and then combined into 27 pools and subjected to virome analysis. CoV-related sequencing reads were detected in ten of the 27 sample pools. These 10 pools related to 22 Rhinolophus spp. samples from Sichuan, 176 Rhinolophus affinis samples from Yunnan, 150 S. kuhlii samples from Guangxi, and 30 C. sphinx samples from Guangxi. Pan CoV screening revealed that 22 individual samples were CoV-positive (Table 1 and Figure 1) . A total of 1,040,901 reads of 100 bp in length showed the best matches with Coronavirinae viral proteins in the NCBI non-redundant database, and these reads showed 40-100% amino acid (aa) identity with known alphaor beta-CoVs.Eleven viruses of representative positive samples were selected for genomic sequencing as quasi-species. There were five viruses in the genus Alphacoronavirus; four from S. kuhlii in Guangxi Province were named as BtSk-AlphaCoV/GX2018A (α-GX2018A), BtSk-AlphaCoV/GX2018B (α-GX2018B), BtSk-AlphaCoV/GX2018C (α-GX2018C), and BtSk-AlphaCoV/GX2018D (α-GX2018D). The other virus from R. sinicus in Yunnan Province was named as BtRs-AlphaCoV/YN2018 (α-YN2018). There were six CoVs in the genus Betacoronavirus. The virus identified from Rhinolophus spp. in Sichuan Province was named as BtRl-BetaCoV/SC2018 (β-SC2018); four CoVs from R. sinicus in Yunnan Province were named as BtRs-BetaCoV/YN2018A (β-YN2018A), BtRs-BetaCoV/YN2018B (β-YN2018B), BtRs-BetaCoV/YN2018C (β-YN2018C), and BtRs-BetaCoV/YN2018D (β-YN2018D). One CoV from C. sphinx in Guangxi Province was named as BtCs-BetaCoV/GX2018 (β-GX2018).As shown in Table 2 and Figure 2 , with the typical genome organization, the full genome sizes of 11 CoVs ranged from 28, 146 (α-GX2018C) to 30, 256 bases (β-YN2018B) and their G + C contents ranged from 0.38 to 0.41. There were several accessory ORFs in each CoV strain. These accessory ORFs were mainly present in the regions between S and E (named ORFs-e), M and N (named ORFm-n), or downstream of N (named ORFn). The accessory ORFs of α-GX2018 (A-D), α-YN2018, and β-GX2018 were mainly present in ORFse and ORFm-n regions; the small ORFs of β-SC2018 and BtSk-AlphaCoV/GX2018CFIGURE 1 | Numbers of bat samples from Sichuan, Yunnan, and Guangxi Provinces. The size of the pie chart is in proportion with the total number of specimens collected from each province. Red color represents samples positive for CoV, and blue color represents negative samples.β-YN2018 (A-D) were mainly present in ORFs-e and ORFmn regions. We named these accessory ORFs according to previous studies. As shown in Table 3 , The core sequences of the leader transcriptional regulatory sequence (TRS; 5 -CUAAAC-3 ) were identified in the 5 untranslated sequences of α-GX2018 (A-D) and α-YN2018, which is unique to Alphacoronavirus (Madhugiri et al., 2014) . The TRS motifs of S, ORF3, E, and M genes in α-GX2018 (A-D) differed from the core sequences of the leader TRS (Supplementary Table 1 ). The TRS motif of S was identified as 5 -CCAAAU-3 or 5 -CCAAAC-3 ; the TRS motif of ORF3 was identified as 5 -CAUUAC-3 ; the TRS motif of E was identified as 5 -CUAGAC-3 ; the TRS motif of M was identified as 5 -AUAAAC-3 . An alternative TRS motif (5 -GUAAAC-3 ) of α-YN2018 was discovered preceding ORF3, which differed by 1 nucleotide (nt) from those of all other genes in α-YN2018. In addition to the E of β-GX2018, the TRS motifs of six betacoronaviruses all conformed to the consensus motif 5 -ACGAAC-3 , which is unique to Betacoronavirus (Woo et al., 2010 ). An alternative TRS motif (5 -UCGAAC-3 ) was discovered preceding the E in β-GX2018.Four Alpha-CoVs of S. kuhlii in Guangxi Province Four alpha-CoVs were identified from S. kuhlii. The full-length genomes of strains α-GX2018A, B, C, and D were sequenced. The genome sequence similarity among these four CoVs, PEDV, and BtCoV/512 was examined by Simplot analysis (Figure 3A ) and pairwise alignment (Supplementary Table 2 ). The α-GX2018A, B, and C were closely related to a known CoV, BtCoV/512, identified in S. kuhlii of Hainan Province with 96.9-100% aa identity for ORF1b, ORF3, E, M, and N. The differences mainly occurred in S (93.4-96.1% aa identity) and ORFx (65.7-67.6% aa identity). There were also significant differences between α-GX2018A and B and BtCoV/512 at the front end of ORF1a (1000-2500 nt) with 81.3-82% nt identity. In addition, although the α-GX2018D was closely related to that of BtCoV/512 with 92.2-99.4% aa identity for ORF1a, ORF1b, ORF3, and M, it was unexpected that the aa identity of α-GX2018D and BtCoV/512 in S1 region was 39.7%, which was lower than that of α-GX2018D and PEDV (42.7% aa identity). Although the aa identity of α-GX2018D and PEDV in S1 region was only 42.7%, we have not found any known viruses with higher identity than this.A novel alpha-CoV (α-YN2018) was identified in R. affinis of Yunnan Province. The genome sequence similarity among α-YN2018, BtRf-alphaCoV/HuB2013 (α-HuB2013), Hipposideros bat CoV HKU10 (Hi-batCov HKU10), Rousettus bat CoV HKU10 (Ro-BatCoV HKU10), and Cardioderma bat coronavirus/Kenya/KY43/2006 (Ca-batCov/Kenya) was examined by Simplot analysis (Figure 3B ) and pairwise alignment (Supplementary Table 3) . This virus showed low sequence similarity with any known alpha-CoVs. α-YN2018showed the highest sequence similarity to α-HuB2013 of Rhinolophus ferrumequinum with 63.9-92.4% aa identity in ORF1a, ORF1b, and N; to Hi-batCov HKU10 with 50-85.7% aa identity in S, M, and ORF7b; to Ca-batCov/Kenya in ORF3a with 60.6% aa identity; and to Ro-BatCoV HKU10 in E with 74.3% aa identity. The sequences of ORF7a from α-YN2018 showed <25% aa identity with those of the afore-mentioned known CoVs. In addition, we did not find any sequences that were similar to ORF3b and ORF3c of α-YN2018 from known viruses. According to the ICTV criteria, CoVs that share <90% aa sequence identity in the conserved replicase domains are considered to belong to different species; thus, α-YN2018 could be considered a new species under the genus Alphacoronavirus.We obtained five beta-CoVs from Rhinolophus spp. from Yunnan and Sichuan Provinces, which were named as β-YN2018A, β-YN2018B, β-YN2018C, β-YN2018D, and β-SC2018. All of a, These four representative bat CoVs are α-GX2018D, α-YN2018, β-YN2018B and β-GX2018, respectively. b, For putative ORFs, we aligned the TRS that preceded the start codon AUG with the leader TRS. The core sequence is indicated with underscores in bold type. The start codons of genes are in bold blue.these beta-CoVs had high sequence identity with SARSr-CoVs. These five SARSr-CoVs showed 92-97% nt identity with human SARS-CoV SZ3. The genome sequence similarity among the five SARSr-CoVs and SARS-CoV SZ3 strain was examined by Simplot analysis (Figure 3C) . These five SARSr-CoVs were highly conserved and shared a uniformly high sequence similarity to SARS-CoV with 94.2-100% aa identity in ORF1a, ORF1b, E, M, and N (Supplementary Table 4 ). There were some differences between the SARSr-CoVs and SARS-CoV SZ3 in the coding regions of ORF3a, ORF3b, ORF7a, and ORF7b. After further analysis, the SARS-CoV SZ3 showed the highest sequence similarity to β-YN2018B with 92-96% aa identity in S, ORF3a, and ORF3b; and to β-YN2018A with 93.2-95.9% aa identity in ORF7a and ORF7b. In contrast, considerable genetic diversity was shown in the S (72.9-92.4% aa identity) and ORF8 (32.6-34.1% aa identity) among the five SARSr-CoVs and SARS-CoV SZ3 strain. β-YN2018B shared 98.8% aa identity with SARSr-CoV WIV1, which was higher than that of the other four SARSr-CoVs (78.8-79.1% aa identity) (Figure 3C and Supplementary Table 4) . Remarkably, WIV1 and WIV16 (or Rs4874) were identified between 2012 and 2013, and they could use the human angiotensin converting enzyme II (ACE2) receptor for entry, like SARS-CoV (Ge et al., 2013; Yang et al., 2015) .ORFx was also found in the genomes of β-YN2018B and β-YN2018D. ORFx of β-YN2018B and β-YN2018D both shared 98.8% aa identity with WIV1. Except for ORF3b (95.6% aa identity), β-YN2018B was highly conserved and shared a uniformly high sequence similarity to SARS-CoV WIV1 in every ORF (98.2-100% aa identity) (Supplementary Table 4 ).A novel beta-CoV, β-GX2018, was identified from this bat species. The genome sequence similarity among β-GX2018 and four known CoVs, BatCoV HKU9-4, BatCoV HKU9-10-1, Ro-BatCoV HKU9, and BatCoV Philippines/Diliman was examined by Simplot analysis (Figure 3D ) and pairwise alignment (Supplementary Table 5 ). The full-length genome sequence of this virus had sequence similarity to that of HKU9related CoVs of Rousettus bats and GCCDC1-related CoVs of Eonycteris bats (Woo et al., 2007; Huang et al., 2016) , but the sequence identity between them was low. The sequence similarity between β-GX2018 and HKU9-and GCCDC1-related CoV s were 51-90.8% aa identity in ORF1a, ORF1b, S, ORF3, E, M, and N. Some partially sequenced genome segments of a recently reported CoV, Philippines/Diliman1525G2/2008 (BatCoV Philippines/Diliman) (Watanabe et al., 2010) , identified in C. brachyotis from the Philippines, showed the highest aa sequence identity with the β-GX2018 in M (96% aa identity), N (91% aa identity), ORF7a (80% aa identity), ORF7b (84.3% aa identity), and ORF7c (80.8% aa identity) (Watanabe et al., 2010) . According to the ICTV criteria, β-GX2018 could be considered a new species under the genus Betacoronavirus.To determine further the evolutionary relationships between these CoVs, phylogenetic trees were constructed using the aa sequences of RdRp, S, S1, E, M and N (Figure 4) . As we have speculated before, α-GX2018A, α-GX2018B, α-GX2018C, α-GX2018D and α-YN2018 clustered with the genus Alphacoronavirus, and β-SC2018, β-YN2018A, β-YN2018B, β-YN2018C, β-YN2018D, and β-GX2018 clustered with the genus Betacoronavirus. The results of the phylogenetic analyses were consistent with those of the sequence identity FIGURE 4 | Phylogenetic trees based on aa sequences of RdRp, S, S1, E, M, and N. The trees were constructed by the maximum likelihood method using appropriate models (WAG + G + I for RdRp and N; LG + G + I for S, S1, and E; rtREV + G for M) with bootstrap values determined by 1000 replicates. Only bootstraps >40% are shown. The scale bars represent 0.05 (RdRp), 0.5 (S and S1), and 0.2 (E, M, and N), respectively.analyses, and confirmed that the identified CoVs could be divided into four lineages.Lineage E α-GX2018A, α-GX2018B and α-GX2018C clustered with BtCoV/512, and their branches were short, reflecting the high sequence similarities. In addition, phylogenetic analysis of the S protein supported our previous analysis that α-GX2018D represented an independent clade between PEDV and BtCoV/512. Further analysis of S1 revealed that α-GX2018D represented a separate evolution distant from all other members of lineage E. From the length of the branch, α-GX2018D had a closer relationship with PEDV, which means that α-GX2018D may have undergone recombination with the ancestors of PEDV and BtCoV/512.α-YN2018 clustered with Hi-batCov HKU10, Ro-BatCoV HKU10, Ca-batCoV/Kenya, and α-HuB2013. However, α-YN2018 had closer relationships with α-HuB2013 of R. ferrumequinum in RdRp and N, and closer relationships with HKU10 of Hipposideros and Rousettus bats in S and M. This means that recombination may have occurred between α-YN2018 and these three related CoVs.β-SC2018, β-YN2018A, β-YN2018B, β-YN2018C, and β-YN2018D clustered with SARS-and SARSr-CoVs under lineage B. In RdRp, E, M, and N trees, these novel SARSr-CoVs and SARS-CoVs had short branches, which indicated that they were closely related in these regions. Phylogenetic analysis of S and S1 revealed that β-YN2018B was closer to SARS-CoV SZ3 and SARS-CoV Tor2 as well as WIV1 and WIV16.Lineage D β-GX2018 of C. sphinx clustered with HKU9-related CoVs of Rousettus and GCCDC1-related CoVs of Eonycteris under lineage D; however, β-GX2018 always represented a separate lineage distinct from other members of lineage D. Although BatCoV Philippines/Diliman of C. brachyotis had a closer relationship with α-YN2018 in M and N, it was only partially sequenced, so we could not analyze its evolutionary relationship with α-YN2018 in RdRp, S, and E, which hindered further analysis.We found that recombinant events had occurred among α-GX2018D, other bat CoVs, and PEDV of lineage E. α-GX2018D showed the highest degree of similarity to BtCoV/512 in the ORF1ab region, and the highest degree of similarity to PEDV in the S1 region. Evolutionary analysis showed that the S of α-GX2018D may have acted as the common ancestors of BtCoV/512 and PEDV. We analyzed possible recombination events in lineage E using SimPlot software. Similarity plot and bootscan results demonstrated that the S-N region of the α-GX2018D had the highest degree of similarity to PEDV, and complicated recombination may have happened between bat CoV and PEDV in the ORF1ab region ( Figure 5A) .For CoVs of lineage F, at least two recombination events had occurred ( Figure 5B) . Similarity plot and bootscan analysis showed that the S1 subunits of the α-YN2018 had the highest degree of similarity to Ro-BatCoV HKU10, while their S2 subunits had the highest degree of similarity to Hi-batCov HKU10 (Fig 5B) . The complicated recombination history between these alpha-CoVs suggests frequent gene transfers, especially of S, among different CoVs, which may be the result of the cross-species transmission of these CoVs.The identification of SARSr-CoV in bats in 2005 first established a genetic relationship between bats and human SARS-CoVs (Lau et al., 2005; Li et al., 2005) . After that, in the following 13 years, the constant discovery of bat SARSr-CoVs suggested the risk of re-emergence of SARS-CoV originating from bat species. Other than that, two CoV-related cases, human MERS in 2012 and SADS in 2017, which had a great impact on health and the economy were identified as being of bat origin (Omrani et al., 2015; Zhou et al., 2018) . In our previous study, a largescale virome analysis was conducted between 2010 and 2013 to understand the ecological diversity of bat viruses and the bat origin of emerging infectious diseases. Rhinolophus bats in Yunnan and Guangxi Provinces have provided genetic clues to the origin of SARS-CoV, and a SADS-CoV-and HKU2related CoV, BtRf-alphaCoV/YN2012, has also been found in Yunnan Province (Wu et al., 2016a) . Sichuan Province has diverse wildlife resources and the largest pig breeding in China, and a MERS-related CoV was reported in bats in this province in 2013 (Yang et al., 2014) . It is necessary for us to conduct continuous surveillance for the prevalence, genetic diversity and geographical distribution of CoVs in some bat species in these potential high-risk regions. Using virome analysis, our study investigated the presence and genetic diversity of CoVs circulating in bat species from Sichuan, Yunnan, and Guangxi Provinces between 2016 and 2017.The close relationship between five alpha-CoVs in S. kuhlii in Guangxi Province and a previously reported CoV in S. kuhlii in Hainan Province reveals that this bat species from different geographic locations contained the same CoV species, but with distinct S proteins. The S1 region of α-GX2018D is more closely related to that of PEDV than that of any other bat CoV, and is phylogenetically located at the root of the lineage of PEDV and S. kuhlii CoVs. This finding suggests the presence of a much closer common ancestor of PEDV in bats CoVs. The recombination analysis supports that the S region of PEDV may come from the ancestor of α-GX2018D. Considering that bat species such as S. kuhlii prefers to circulate around farmhouses and pigsties, and other PEDV-related sequences have been reported in bats in previous reports (Simas et al., 2015; Phan et al., 2018) , the relationship between bat CoVs and PEDV should be further investigated.The discovery of β-SC2018 provides evidence that Rhinolophus in Sichuan Province could harbor SARSr-CoVs, and reveals a broader geographical distribution of these viruses. The discovery of β-YN2018B here, combined with previously reported WIV1 and WIV16 (Ge et al., 2013; FIGURE 5 | Detection of potential recombination events by bootscan analysis. (A) Full-length genome sequence of α-GX2018D was used as a query sequence and BtCoV/512, α-GX2018A, and PEDV as reference sequences. (B) Full-length genome sequence of α-YN2018 was used as a query sequence and Ro-BatCoV HKU10, Hi-batCov HKU10, and α-HuB2013 as reference sequences. All analyses were performed with a Kimura model, a window size of 1000 base pairs, and a step size of 100 base pairs. The gene map of query genome sequences is used to position breakpoints. Yang et al., 2015) , revealed that these ACE2-adaptable SARSr-CoVs were continuously circulating in R. affinis of Yunnan Province at least from 2012 to 2016. These findings emphasize again the importance of continuous longitudinal monitoring for SARSr-CoVs in Rhinolophus over a wide area. Although no SADS and HKU2-related CoV were found in Rhinolophus bats in our study, the surveillance of this CoV type in bats is still needed in the future.In our previous study, by the identification of correlated alpha-CoVs in different Miniopterus species from multiple regions, the characteristics of co-infection, recombination, and hostshifting for alpha-CoVs have been described. Here, the alpha-CoVs of lineage F found in more diverse bat species (including frugivorous and insectivorous bats) extends the known host range of these viruses. However, the recombination events found between α-YN2018, Ro-BatCoV HKU10, and Hi-batCov HKU10, and the multiple origins of the S of α-YN2018, further indicate that a more distant host-shifting in accordance with the recombination of the S region may have happened in the evolutionary history of these viruses among diverse host species.When we collected S. kuhlii in Guangxi Province, we found that there were many C. sphinx at the sampling site. The Pteropodidae bats (C. sphinx bat belongs to Pteropodidae) are natural hosts of several emergent human pathogens such as Hendra virus, Nipah virus, and Ebola virus (Leroy et al., 2005; Halpin et al., 2011; Sendow et al., 2013; Mari Saez et al., 2015; Paez et al., 2017 ). An Ebola-related filovirus was found in the Chinese Pteropodidae bats recently (Yang et al., 2019) , and thus, we collected samples of C. sphinx for virome analysis. The C. sphinx beta-CoV, β-GX2018, represents an evolved independent lineage different from HKU9 of Rousettus and GCCDC1 of Eonycteris under lineage D. Based on the partially sequenced region, CoV strain Philippines/Diliman1525G2/2008 previously identified in C. brachyotis from the Philippines, could also be clustered into this separate β-GX2018 clade. By providing full-length sequence data, this finding reveals a new host range of this virus in an unreported location.YH, QJ, and ZW conceived the experiments, analyzed the results, and wrote the manuscript. YH, JD, HS, and ZW conducted the experiments and analyzed the results. JZ, GZ, and SZ collected the specimens. All authors reviewed the manuscript. \n",
            "\t0.523\tCopyright: Lee et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License 3.0 (CC BY 3.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.Enteroviruses belong to the Picornaviridae family, which is characterized by a single stranded positive-sense RNA genome with about 7500-8000 nucleotides, and have been emerged as the major causative agents of various human diseases. Coxsackievirus B3 (CVB3), one of the most well-studied enteroviruses, causes viral meningitis, myocarditis and pancreatitis [1, 2] . In addition, enterovirus 71 (EV71) is a causative agent of hand-foot-mouth disease and also of severe neurological symptoms, which can lead to even death . However, despite the increasing public threat, no effective therapy is currently available for the treatment of these infections. Enteroviruses have hundreds of distinct viruses, and newly emerging enteroviruses have been increasingly reported in recent years. Moreover, many RNA viruses including influenza, severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV) and zika virus (ZIKV) have become an enormous threat for public health. Therefore, broad-spectrum antiviral drugs are necessary to efficiently control various viral infections. In another aspect ineffectiveness of conventional enzymetargeting drugs due to the rapid development of resistant mutants is another hurdle we need to tackle. In order to achieve the development of broad-spectrum antiviral drug with a low rate of mutation, two strategies have been generally considered. One is targeting host cellular factor that is essentially required for the viral life cycle. This strategy would have a low potential of producing resistant viruses, but undesirable side effects could be accompanied. The other is activating innate immune response such as interferon (IFN) signaling so as to boost host antiviral defense system . Actually, IFN itself or in combination with other antiviral drugs such as ribavirin has been primarily used for the treatment of various RNA virus infections. More recently, a few inhibitors of nucleoside biosynthesis have been shown to induce the innate immunity and suppress a broad range of virus infections . For instance, Wang et al identified a broad-spectrum antiviral compound (Brequinar) targeting DHODH, a key enzyme of the pyrimidine biosynthetic pathway, and subsequently inducing innate immune response .Previously, we identified gemcitabine, a drug currently being used for anti-cancer chemotherapy, as an effective inhibitor of enteroviruses including CVB3, EV71 and human rhinoviruses (HRVs) . Its antiviral activity has been also shown against various RNA viruses including hepatitis C virus (HCV), human immunodeficiency virus (HIV), influenza virus, poliovirus, MERS-CoV and ZIKV . Gemcitabine, as a cytidine analog, was reported to interfere with the pyrimidine biosynthesis . However, the role of pyrimidine inhibition and the involvement of subsequent innate immunity in the antiviral action of gemcitabine have not been explored yet.In this study, we examined the role of pyrimidine inhibition in the antiviral activity of gemcitabine by adding the exogenous nucleosides to CVB3-infected or CVB3 replicon-harboring HeLa cells. As a result, the antiviral effect of gemcitabine was remarkably suppressed by the pyrimidine nucleosides. Further analysis demonstrated that gemcitabine inhibited the salvage pathway of pyrimidine biosynthetic pathway most probably by targeting cytidine and/or uridine synthesis. Moreover, the treatment with gemcitabine activated the expression of several IFN-stimulated genes (ISGs), the major effectors in the innate immunity, which was also suppressed by the supplemented cytidine.Previously, we identified a new indication of gemcitabine as an effective anti-enteroviral inhibitor . As a cytidine analog, gemcitabine is known to have an inhibitory activity on the pyrimidine biosynthesis. Besides, a few inhibitors of the pyrimidine biosynthesis have been reported to show the antiviral activity, at least partly, through activating the innate immune response [10, 11, 14] . Thus, in this study we sought to examine if the anti-enteroviral activity of gemcitabine is also associated with the modulation of pyrimidine biosynthesis and innate immunity. At first, in order to test if the anti-enteroviral activity of gemcitabine is related with the inhibition of pyrimidine biosynthesis, HeLa cells were infected with CVB3 and simultaneously treated with excessive 4 nucleosides (adenosine, guanosine, uridine and cytidine) in the presence of various doses of gemcitabine for 8 hours. Antiviral activity was measured by staining infected cells with an anti-CVB3 3Cpro antibody and secondary antibody conjugated with AF488 fluorescent dye and counting cells with a fluorescent signal. As previously reported, gemcitabine itself exhibited a strong antiviral activity on CVB3 infection, with an estimated IC 50 of ~5 µM and maximal efficacy of > 80 % ( Figure 1A ). When gemcitabine was co-treated with excessive 4 nucleosides each, only two pyrimidine nucleosides (cytidine and uridine) could significantly suppress the antiviral activity of gemcitabine, resulting in the sustained infection. Especially, cytidine had the strongest effect, which can be explained by the fact that gemcitabine is a cytidine analog. The effect of cytidine was evidently demonstrated by a dose-dependent suppression of gemcitabine's antiviral activity (Supplementary Figure 1A) . In contrast, purine nucleosides (adenosine and guanosine) had little effect. As a control experiment, cell viability was analyzed in the same condition by using MTT assay ( Figure 1B ). There were little changes except that the treatment with adenosine without gemcitabine decreased the cell viability by about 20 %, indicating the cytotoxic effect of adenosine. This phenomenon was further confirmed in an independent experiment using increasing concentrations of each nucleosides without CVB3 infection (Supplementary Figure 2) .To further confirm the effect of pyrimidine nucleosides on the antiviral activity of gemcitabine, CVB3 subgenomic replicon system was additionally used. This system contains the firefly luciferase gene in place of structural genes (VP4-VP1) of CVB3 viral genome and allows the quantitative measurement of viral replication [23, 24] . HeLa cells were transfected with in vitrotranscribed CVB3 replicon RNAs and simultaneously treated with 4 nucleosides each in the presence of various doses of gemcitabine for 8 hours. As similar to the results from CVB3 infection experiment ( Figure 1A) , addition of excessive pyrimidine nucleoside (cytidine or uridine) remarkably suppressed the antiviral activity of gemcitabine, thereby protecting the replication of CVB3 replicon (Supplementary Figure 3A) . According to our previous study, gemcitabine has a strong antiviral effect on EV71 as well as CVB3 . Therefore, we also tested pyrimidine nucleosides in EV71 replicon system. As a result, only cytidine and uridine suppressed the antiviral activity of gemcitabine on the replication of EV71 replicon, which is similar to those observed from CVB3based assays (Supplementary Figure 3B) . Collectively, these results strongly indicate the involvement of pyrimidine biosynthesis in the antiviral activity of gemcitabine.The observation that the antiviral effect of gemcitabine is suppressed by the supplemented pyrimidine nucleosides directed us to further define the effect of gemcitabine on the pyrimidine biosynthetic pathway. Pyrimidine biosynthesis occurs by two separate pathways: de novo synthesis and the salvage ( Figure 2C ). To determine which pathway and step were affected by gemcitabine, a few intermediates of both pathways were tested. Dihydroorotate (DHO) and orotic acid were used as intermediates of de novo biosynthetic pathway, and 4 nucleobases (adenine, guanine, uracil and cytosine) were additionally used as intermediates of the salvage pathway. Uridine mono-phosphate (UMP) acting downstream of both pathways was also included. UMP is synthesized from uridine by uridine kinase in the salvage pathway and separately from orotidine mono-phosphate (OMP) by OMP decarboxylase in de novo biosynthetic pathway ( Figure 2C ). Gemcitabine, as a nucleoside analog, may directly affect the activity of enzyme(s) that are involved in the pyrimidine biosynthesis, resulting in the decrease of intermediates acting downstream of its target. Thus, excessive addition of any intermediate acting downstream of gemcitabine's target can rescue the deficit and supposedly shows a suppressive effect on the antiviral activity of gemcitabine. When selected intermediates were co-treated with gemcitabine in CVB3-infected HeLa cells, only UMP significantly showed a suppressive effect with an extent similar to that of cytidine ( Figure 2A ). Other intermediates for de novo pathway (DHO and orotic acid) and for the salvage pathway (4 nucleobases) had marginal effects. Under the same condition, little changes in cell viability were shown ( Figure 2B ). Moreover, similar effect of UMP was also observed in CVB3 replicon-harboring cells (Supplementary Figure 4) , even though it was slightly weaker than that of cytidine. Taken together the results from Figures 1 and 2 , it seems that gemcitabine majorly targets the salvage pathway rather than de novo pathway, particularly uridine and/or cytidine synthesis from uracil and/or cytosine ( Figure 2C ).There are a few antiviral compounds interfering with the nucleoside biosynthesis, which subsequently induce innate immunity involving the upregulation of ISGs 25] . In those studies antiviral compounds induced the expression of IFN-stimulated response element (ISRE)-luciferase reporter even without IFN stimulation. Thus, we also examined if gemcitabine itself has a stimulatory effect on ISRE-luciferase reporter. HeLa cells were transfected with the reporter plasmid and then treated with gemcitabine (0.16 to 100 μM) and IFN-α (0.08 to 50 ng/ml) for 8 or 24 hours. At 8 hours posttreatment gemcitabine increased the luciferase activity up to ~ 200% of the control ( Figure 3A ). The increase of luciferase activity was much more evident at 24 hours post-treatment, maximally reaching ~800% of the control ( Figure 3C ). In contrast, IFN-α induced the luciferase expression up to ~600 and 400 % of the control at 8 and 24 hours post-treatment, respectively, indicating a rapid stimulation ( Figure 3B and 3D) . Both gemcitabine and IFN-α significantly stimulated the ISRE promoter but with apparently different time-kinetics. ISRE promoter more slowly responded to gemcitabine compared to IFN-α. Differential mechanisms of ISRE activation by gemcitabine and IFN-α will be proposed in the discussion section.To further confirm that ISRE activation by gemcitabine is correlated with its antiviral activity, we tested if the antiviral activity is enhanced by the longer treatment of gemcitabine. HeLa cells were treated with gemcitabine for 16 hours prior to CVB3 infection and maintained for another 8 hours, achieving 24 hours of treatment (24h exposure). Antiviral activity was assessed by quantifying the virus-infected cells showing a fluorescent signal of 3Cpro protein at 24 hours after treatment with gemcitabine. As a result, a considerably stronger antiviral effect was exhibited by a longer time treatment (24h exposure) than a shorter time treatment (8h exposure) (Supplementary Figure 5) . The estimated IC 50 of gemcitabine in '24h exposure' cells (~0.2 μM) was much lower than that in '8h exposure' cells (~1 μM). Moreover, the maximum efficacy of gemcitabine in '24h exposure' cells (~100%) was apparently higher than that in '8h exposure' cells (~80%). These results indicate that the enhanced antiviral activity by a longer exposure of cells with gemcitabine is likely correlated with stronger activation of ISRE promoter attained at 24 hours of treatment.IFNs trigger an intracellular signal through the JAK/ STAT pathway and transcriptionally induce numerous ISGs under the control of ISRE . According to our aforementioned results, gemcitabine itself has a stimulatory effect on ISRE promoter like IFNs. However, their modes of activation seem to be quite different. Thus, in order to further confirm the activation of ISREcontaining promoters by gemcitabine and to know how differently these promoters respond to gemcitabine and IFN-α, the transcriptional expression of 5 ISGs (CXCL10, IRF7, IRF9, IFIT1, and DDX58) were analyzed. HeLa cells were treated with increasing doses of gemcitabine or IFN-α for 24 hours and then total RNAs were prepared for quantitative real-time PCRs. As a result, gemcitabine strongly induced two genes (IFIT1 and DDX58) up to and UMP) (100 μM) of pyrimidine biosynthetic pathway were treated as described in Figure 1A (A) and in Figure 1B (B) . Cytidine was used as a positive control. The average and standard deviation were obtained from three independent experiments. *** indicates P < 0.001. (C) Pyrimidine biosynthetic pathway. Tested metabolites were indicated as boxes and metabolites with the significant effect were shown as orange boxes. The salvage pathway affected by gemcitabine was depicted by blue arrows. more than 7-fold of the control and moderately three genes (CXCL10, IRF7 and IRF9) up to less than 4-fold of the control ( Figure 4A ). On the other hand, IFN-α strongly induced 4 ISGs (IRF7, IRF9, IFIT1 and DDX58) up to more than 5-fold of the control ( Figure 4B ). These results indicate that gemcitabine has a different mode of ISG activation from IFN-α. In order to further confirm whether the ISG activation by gemcitabine is mediated by the modulation of pyrimidine biosynthesis, we tested the effect of cytidine on the activation of IFIT1 and DDX58 genes, the two strongest responders to gemcitabine. As expected, the supplemented cytidine considerably suppressed the gemcitabine-induced expression of IFIT1 and DDX58 ( Figure 5 ). In contrast, cytidine had no significant effect on the IFN-induced IFIT1 and DDX58 expression.To further clarify whether the stimulation of innate immune response by gemcitabine is independent of IFN-induced JAK/STAT pathway, we first examined the phosphorylation of STAT1 at Tyr 701, a major event occurred at the early step of IFN signaling. HeLa cells were treated with various doses of gemcitabine or IFN-α for 2 hours or 24 hours and then cell lysates were analyzed by Western blotting. As a result, the levels of phosphorylated STAT1 (pTyr701) proteins was not changed at all by gemcitabine at both 2 and 24 hours of treatment (Supplementary Figure 7A and 7C) . On the contrary, treatment with IFN-α remarkably increased the phosphorylation of STAT1 protein at both time conditions (Supplementary Figure 7B and 7D) , as expected. In addition, we further examined whether the gemcitabineinduced ISG expression involved IRF9, which forms IFNstimulated gene factor 3 (ISGF3) complex by interacting with the phosphorylated STAT1/STAT2 and functions as a transcriptional activator of ISGs . siRNA-mediated knockdown of IRF9 did not affect the gemcitabineinduced expression of DDX58, while it had a significant reducing effect on that induced by IFN-α (decreased by ~45% of the control) (Supplementary Figure 8) .Collectively, these results demonstrate that the antiviral effect of gemcitabine involves the activation of ISGs, the mode of which is different from IFN-dependent conventional way but which is mediated, at least partly, by the inhibition of pyrimidine biosynthesis DISCUSSION Gemcitabine, currently in use for cancer therapy, has the antiviral activity against a broad range of RNA viruses, including poliovirus , HCV , influenza , MERS-CoV and ZIKV . In the previous study, we also identified gemcitabine as an effective and potent anti-enteroviral agent . Herein, we further defined the underlying mechanism of gemcitabine's antiviral action that involved the modulation of pyrimidine biosynthesis and the subsequent activation of ISGs.There have been accumulating reports that nucleoside analogs act as the antiviral agents through the modulation of nucleotide biosynthesis [10, . As a cytidine analog, gemcitabine was also expected to work in this way. Indeed, the antiviral activity of gemcitabine in both CVB3-infected and CVB3 repliconharboring HeLa cells was significantly suppressed by the addition of exogenous pyrimidine nucleosides (cytidine and uridine) but not by purine nucleosides (Figure 1 and Supplementary Figure 3) . Further delineation demonstrated that gemcitabine acted through the salvage pathway rather than de novo pathway and targeted most probably uridine and/or cytidine synthesis from uracil and/or cytosine (Figure 2 and Supplementary Figure 4) . Therefore, it is conceivable that gemcitabine induces the decrease of pyrimidine nucleosides and their subsequent metabolites including UMP.There have been a few reports that the inhibition of pyrimidine biosynthesis induces the innate immune response, especially the activation of ISGs, which is the major antiviral mechanism in the early stage of virus infection [10, 11, 14] . These previous findings were confirmed in our test by observing the activation of ISRE promoter by leflunomide, an inhibitor of dihydroorotate dehydrogenase (DHODH) on the pyrimidine biosynthetic pathway (Supplementary Figure 6) . More importantly, we newly revealed that gemcitabine also activated the ISRE promoter ( Figure 3A and 3C) . The activation of ISRE promoter by gemcitabine was further confirmed by the significant induction of all 5 endogenous ISGs (CXCL10, IRF7, IRF9, IFIT1, and DDX58) tested, which was evaluated by quantitative real-time PCR analysis ( Figure 4A) . Intriguingly, the increase of IFIT1 and DDX58 mRNAs was particularly outstanding, and the increase of those mRNAs was definitely suppressed by the additional treatment with cytidine, suggesting the role of IFIT1 and DDX58 in the antiviral effect of gemcitabine. IFIT1 family members are among the most highly induced ISGs in response to IFN and there are many reports concerning their antiviral functions on various viruses. For instance, IFIT1 proteins exhibit the antiviral effect generally by sequestrating viral genomes having 5'-triphosphates . They also inhibit HCV translation initiation by binding to eIF3, which is required for the efficient ribosome recruitment on HCV IRES [33, 34] . In addition, IFIT's targeting of HPV E1 helicase, which is essential for viral DNA replication , was reported as anti-Human Papillomavirus (HPV) effect of IFIT protein. DDX58, the other ISG strongly stimulated by gemcitabine, encodes retinoic acid-inducible gene I (RIG-I). It is part of the RIG-I-like receptor family, which functions as a pattern recognition receptor that is a sensor for single-or double-strand RNA of viruses such as flavivirus, influenza A and reovirus, and is involved in triggering an antiviral response . Also, IFIT and DDX58 proteins may majorly contribute to the antiviral action of gemcitabine in a way similar to the aforementioned mechanisms. Still, we would not exclude the contribution of other ISGs such as CXCL10, IRF7, and IRF9, and other untested ISGs as effectors to the full antiviral activity of gemcitabine.One of the most interesting observations from our study is that gemcitabine seems to activate the expression of ISGs in a way different from IFN-α. First, ISRE promoter more slowly responded to gemcitabine, compared to IFN-α ( Figure 3 ). Gemcitabine weakly activated the ISRE promoter at 8 hours and almost fully at 24 hours after treatment, which is quite contrary to IFN-α that achieved the full activation even at 8 hours after treatment. Second, the pattern of ISG activation was quite different between gemcitabine and IFN-α (Figure 4 ). Gemcitabine exhibited a relatively strong induction of IFIT1 and DDX58 mRNAs over IRF7 and IRF9 mRNAs, while IFN-α had a similarly strong stimulatory effect on all of IRF7, IRF9, IFIT1 and DDX58 mRNAs. Considering that all 4 ISGs contain STAT1/2-binding element in their promoters, the mode of gemcitabine seems to be weakly dependent on STAT1/2 or independent of them, which is contrary to IFN-α that highly depends on them. Third, the phosphorylation of STAT1 (Tyr 701) was not induced by gemcitabine, while a remarkable induction of STAT1 phosphorylation was observed by IFN-α (Supplementary Figure 7) . Fourth, the expression of DDX58 mRNAs induced by gemcitabine was not affected by siRNA-mediated IRF9 knockdown, which is contrary to the result that the IFN-α-induced expression of DDX58 mRNAs was significantly reduced at the same condition (Supplementary Figure 8) . Fifth, the gemcitabine-induced expression of IFIT1 and DDX58 mRNAs was significantly suppressed by the additional treatment with cytidine, while that by IFN-α was not affected ( Figure 5 ). This observation is of particular significance in that it strongly suggests a crosstalk between the pyrimidine biosynthesis and the innate immune response, which is induced by gemcitabine but not by IFN-α. Decreased pyrimidine pool or inactivation of metabolic enzyme(s) might trigger a signal, which is delivered to certain cis-acting element on a subset of ISGs possibly through the relay of some kinase(s). As mentioned above, this signal is less likely to be dependent on STAT1/2-IRF9 (IFN-stimulated gene factor 3; ISGF3) that is the major transcriptional complex in the IFN-induced JAK/STAT pathway . Rather, the possible role of IRF3 or IRF7 needs to be alternatively evaluated. Further investigation in the following study will clarify the underlying molecular mechanism of gemcitabine's antiviral action.Our results, particularly the suppression of the gemcitabine-induced ISG activation by cytidine ( Figure 5 ), indicate that the gemcitabine-induced innate immune response is mediated, at least partly, by its pyrimidinedepleting effect. Moreover, according to the results in Figure 1A and 2A, the gemcitabine's pyrimidine-depleting effect seems to occur mainly through its targeting of the salvage pathway. However, considering that neither excessive cytidine, uridine nor UMP could achieve the full suppression of gemcitabine's antiviral effect but by up to 50-70% of DMSO control ( Figure 1A and 2A) , there seems to be the other way of contribution that accounts for the residual portion of antiviral effect (~30-50%). Besides the salvage pathway of pyrimidine biosynthesis, gemcitabine is known to inhibit ribonucleotide reductase catalyzing the formation of dNTPs from NTPs , which is another cause of nucleotide depletion. Thus, the residual antiviral activity might be explained by these additional inhibition of nucleotide biosynthesis, seemingly resulting in the severe nucleotide depletion and the subsequent stronger ISG activation. Alternatively, gemcitabine could be incorporated into newly synthesized viral genome or directly inhibit RDRP activity, resulting in the reduced viral replication.In this study, we demonstrated the involvement of pyrimidine inhibition-induced innate immune response in the antiviral activity of gemcitabine, particularly by targeting the salvage pathway. The antiviral action of gemcitabine through innate immunity supported the feasibility of its therapeutic application as a broadspectrum antiviral drug.HeLa cells were purchased from ATCC and cultured as previously described . The stock of CVB3 (Nancy; ATCC VR-30) was also purchased from ATCC and amplified in HeLa cells. Gemcitabine, nucleosides (adenosine, guanosine, cytidine and uridine), nucleobases (adenine, guanine, cytosine and uracil), pyrimidine synthetic intermediates [dihydroorotate (DHO), orotic acid and uridine mono-phosphate (UMP)], IFN-α-2A and leflunomide were purchased from Sigma-Aldrich. Rabbit polyclonal anti-CVB3 3Cpro antibody was generated in house by immunizing with CVB3 3Cpro recombinant protein. Anti-rabbit secondary antibody conjugated with Alexa Fluor 488 (AF488) was purchased from Life technologies.To examine the involvement of pyrimidine biosynthetic pathway in the antiviral activity of gemcitabine, HeLa cells were infected with CVB3 (5 MOI) and simultaneously treated with nucleosides, nucleobases or pyrimidine biosynthetic intermediates (100 μM). At 8 hours post-infection cells were fixed with icecold mixture of methanol-acetone (3:1, v/v). After washing twice with PBS, infected cells were stained with rabbit polyclonal anti-CVB3 3Cpro antibody and anti-rabbit secondary antibody conjugated with AF488. Nuclei were counterstained with Hoechst 33342 (Life technologies). Images were captured and viral infection was quantified as previously described . Infection was calculated as a percentage relative to DMSO control.MTT assay was used for measuring cell viability as described previously . Cell viability was calculated as a percentage relative to the control.Plasmids p53CB3-LUC and pRibFluc-EV71, which contain the firefly luciferase gene in place of the P1 capsid coding region of CVB3 and EV71 viral genomes, were kindly provided by Frank J. M. van Kuppeveld (Utrecht University, The Netherlands) [23, 24] . Synthesis of CVB3 and EV71 replicon RNAs and replicon assays were performed as previously described . Briefly, HeLa cells (3×10 5 cells/well) on a 6-well plate were transfected with 0.4 μg of replicon RNA, split into 96well plates (2×10 4 cells/well), and simultaneously treated with nucleosides, nucleobases or pyrimidine biosynthetic intermediates in the presence of gemcitabine. Eight hours after compound treatment, cells were assayed for the firefly luciferase activity using One-Glo Luciferase Assay system (Promega).HeLa cells (3×10 5 cells/well) in a 6-well plate were transfected with an ISRE-luciferase reporter plasmid using Lipofectamine 2000 (Invitrogen) and split into 96-well plates (2 × 10 4 cells/well). The IFN-stimulated response elements (ISRE)-luciferase reporter plasmid was described previously . Forty-eight hours after transfection, cells were treated with gemcitabine or IFN-α for another 8 or 24 hours. Thereafter, cells were assayed for the firefly luciferase activity using One-Glo Luciferase Assay System (Promega).Total RNAs from cells treated with compounds in a 6-well plate were prepared using RNeasy mini kit (QIAGEN) and then cDNAs were generated using SuperScript IV (Life Technologies) and random hexamers. For the quantification of ISG expression, we performed a real-time PCR using SsoFast Evagreen Supermix (Bio-Rad) and primers described in the previous report . The β-actin mRNA was also quantified and used as the endogenous control .HeLa cells (3×10 5 cells/well) in a 6-well plate were treated with gemcitabine or IFN-α for 2 or 24 hours. Cell lysates were prepared and subjected to Western blot analysis with anti-STAT1 and -phospho-STAT1 (Tyr 701) antibodies. GAPDH proteins were also analyzed as a loading control. More detailed procedures were described previously . Anti-STAT1 and -phospho-STAT1 (Tyr 701) antibodies were purchased from Cell Signaling Technology, and GAPDH antibody was purchased from Santa Cruz Biotechnology.HeLa cells (3×10 5 cells/well) in a 6-well plate were transfected with IRF9 or negative control siRNAs by using Lipofectamine RNAiMAX reagent (Invitrogen) for 24 hours and then treated with gemcitabine or IFN-α for another 24 hours. Thereafter, total RNAs were prepared and analyzed for the quantitative real-time PCR of DDX58 mRNAs. The siRNAs were synthesized by BIONEER (Daejeon, South Korea).In figures, data are presented as the means ± standard deviations obtained from 3 or more independent experiments. * , ** , and *** represent p<0.05, p<0.01, and p<0.001, respectively, which were analyzed by Student's t-test.The authors declare no conflicts of interest.\n",
            "\t0.522\tEight hundred and thirteen European rodents and shrews encompassing seven different species were screened for alphacoronaviruses using PCR detection. Novel alphacoronaviruses were detected in the species Rattus norvegicus, Microtus agrestis, Sorex araneus and Myodes glareolus. These, together with the recently described Lucheng virus found in China, form a distinct rodent/shrew-specific clade within the coronavirus phylogeny. Across a highly conserved region of the viral polymerase gene, the new members of this clade were up to 22% dissimilar at the nucleotide level to the previously described Lucheng virus. As such they might represent distinct species of alphacoronaviruses. These data greatly extend our knowledge of wildlife reservoirs of alphacoronaviruses.Coronaviruses (CoVs) are important pathogens affecting both humans and animals. The first CoV discovery was made in the 1930s . In humans, CoVs are associated with respiratory disease, ranging from mild upper respiratory tract symptoms, to those that are more severe and potentially fatal, associated with infection of the lower-respiratory tract. CoVs are also known to infect mammals and birds, among which they have been associated with enteric and respiratory diseases as well as hepatitis and neurological disorders . The recent emergence of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) in humans and porcine epidemic diarrhoea virus (PEDV) in pigs has highlighted the epizootic and zoonotic risk that these viruses pose (reviewed in [3, 4] ). It is very likely that there are novel unrecognized CoVs circulating in animals that pose a cross-species transmission risk.Bats harbour a plethora of CoV species, some of which are thought to be the initial source of recent spillover events such as SARS, MERS and PEDV [3, 4] . However, the recent discovery of newIn total, 899 tissue samples from 813 wild rodents and shrews from the East Midlands region of the United Kingdom and the Mazury Lake District region of Poland ( Figure 1 , Table 1 ) were collected either as part of routine pest control, as a consequence of predation (Felis catus) or for other studies . The samples from the United Kingdom were collected between 2008 and 2015 and the study was approved by the University of Nottingham Internal Ethics Committee (Nottingham, UK). The study for the Polish samples was approved by the Ethics Commission for Experiments on Animals of the M. Nencki Institute of Experimental Biology of the Polish Academy of Sciences (Warszawa, Poland). The rodent species included: house mouse (Mus musculus), brown rat (Rattus norvegicus), field vole (Microtus agrestis), bank vole (Myodes glareolus), wood mouse (Apodemus sylvaticus) and shrew (Sorexaraneus). A necropsy was performed and tissue samples were stored at´70˝C in ethanol (Polish liver specimens), at´70˝C in RNAlater (Sigma Aldrich, St. Louis, MO, USA; UK liver and gut specimens) or at´20˝C without additives (UK liver specimens collected in 2008). Total animals 813 Total samples 899 ‡ Two animals were positive in liver and gut, one animal was positive in the gut only and two animals had only a liver sample available.Total RNA, from approximately one cubic millimetre sections of liver or intestinal tissue samples, was extracted using the GenElute™ Mammalian Total RNA Miniprep Kit (Sigma Aldrich) and used as template in reverse-transcription PCRs using in-house designed primers targeting the rodent glyceraldehyde 3-phosphate dehydrogenase (GAPDH) gene (F: 5'-CCATCTTCCAGGAGCGAGA-3', R: 5'-GCCTGCTTCACCACCTTCT-3'), cytochrome b gene or a previously published primer set targeting a conserved region of the ORF1ab CoV polymerase gene . Extra sequence for ORF1ab was acquired by using an in-house primer targeting an upstream conserved region (F: 5'-AATCTTAAGTATGCTATTAGTGG-3') in combination with the previously described reverse primer . PCR products of expected size were subject to Sanger sequencing (Source Bio Science, Nottingham, UK) and sequence similarity to Genbank database sequences was determined using BLASTn. Coronavirus ORF1ab and rodent cytochrome b reference sequence sets were downloaded from Genbank and used alongside sequences obtained in this study for phylogenetic analysis using the Molecular Evolutionary Genetics Analysis version 6 (MEGA6) software . Codon-constrained nucleotide sequences were aligned using ClustalW and maximum likelihood phylogenetic trees (utilising a GRT with invariant sites (G+I) model of evolution) were generated, with robustness assessed using bootstrap resampling (1000 pseudoreplicates). CoV sequences generated in this study have been deposited in the Genbank database under accession numbers KU739070-KU739074. Total RNA, from approximately one cubic millimetre sections of liver or intestinal tissue samples, was extracted using the GenElute™ Mammalian Total RNA Miniprep Kit (Sigma Aldrich) and used as template in reverse-transcription PCRs using in-house designed primers targeting the rodent glyceraldehyde 3-phosphate dehydrogenase (GAPDH) gene (F: 5'-CCATCTTCCAGGAGCGAGA-3', R: 5'-GCCTGCTTCACCACCTTCT-3'), cytochrome b gene or a previously published primer set targeting a conserved region of the ORF1ab CoV polymerase gene . Extra sequence for ORF1ab was acquired by using an in-house primer targeting an upstream conserved region (F: 5'-AATCTTAAGTATGCTATTAGTGG-3') in combination with the previously described reverse primer . PCR products of expected size were subject to Sanger sequencing (Source Bio Science, Nottingham, UK) and sequence similarity to Genbank database sequences was determined using BLASTn. Coronavirus ORF1ab and rodent cytochrome b reference sequence sets were downloaded from Genbank and used alongside sequences obtained in this study for phylogenetic analysis using the Molecular Evolutionary Genetics Analysis version 6 (MEGA6) software . Codon-constrained nucleotide sequences were aligned using ClustalW and maximum Eleven of the 813 rodents and shrews tested were found to be positive for CoVs (Table 1) . Five positives were found in Rattus norvegicus, three in Microtus agrestis, one in Sorex araneus and two in Myodes glareolus. The phylogenetic analysis ( Figure 2 ) showed that the new rodent/shrew CoVs, together with the Lucheng CoV reference strain formed a single clade (with 99% bootstrap support) in the Alphacoronavirus genus. Within this cluster one lineage contained a single sequence that was present in five separate samples (UKRn1-5) obtained from Rattus norvegicus, which were most related to the Lucheng CoV (100% bootstrap support). A separate cluster with 100% bootstrap support contained three sequences obtained from Microtus agrestis (UKMa1&2) and Myodes glareolus (UKMg1). A third branch contained a sequence obtained from Myodes glareolus (PLMg1) living in the Mazury lakes region of Poland and a further distinct sequence obtained from Sorex araneus (UKSa1) and Microtus agrestis (UKMa3) (99% bootstrap support). Detailed analysis of the sample archive, including details of the day and method of capture revealed that samples UKSa1 and UKMa3, which were identical across the region of ORF1a1b analysed, were not only obtained from the same geographical area but also captured by the same predator (Felis catus) on the same day. This was also true for samples UKMa2 and UKMg1. Therefore, we are left with three plausible explanations for the CoV phylogeny observed. Either the same or similar rodent coronaviruses were co-circulating and freely infecting different rodent species living in the same geographical area at the time of sampling; cross-species transmission of coronaviruses has been described previously . An alternative explanation is that these carcasses were cross-contaminated during or after predation with coronavirus present in either animal or through predator-contamination from a different infected animal altogether that was not sampled. We do not consider that this result was due to contamination during tissue processing as new sterile dissection equipment was used for each carcass and this effect was observed between animals that were not processed consecutively. The cytochrome b sequence data ( Figure 3 ) clearly showed that the specimens were of different species (arguing against sample mix-up). Importantly, re-extraction of the original rodent tissues gave identical results (data not shown), which strongly suggests that contamination did not occur during nucleic acid extraction or PCR preparation. Finally, it is also possible that these animals were infected with distinct strains of coronavirus but the PCR primers, which were designed to target highly conserved regions of the ORF1ab gene, were unable to discriminate between highly related viruses. The inability of ORF1ab PCR sequencing to resolve different isolates was evident in recent analyses of bat CoVs [20, 21] . Sequencing additional regions of the genome, as has been performed previously with bat CoVs , together with analysis of larger numbers of rodents, will help elucidate the overall diversity and distribution of alphacoronaviruses in European rodents and shrews. Interestingly, the samples UKSa1, UKMa2&3 and UKMg1 were killed by the same predator in a relatively limited sample set (Site 1, Figure 1 ), suggesting coronavirus pathogenesis may have increased susceptibility to predation in these animals. Whether or not coronavirus infection might render rodents more susceptible to predation needs further investigation.The novel UKRn1-5, UKMa1, UKMa2&UKMg1, UKSa1/UKMa3 and PLMg1 sequences were 97.6%, 83.8%, 84.6%, 78.4% and 81.8% similar and 98.6%, 96.2%, 96.2%, 91.9% and 94.3% similar to the corresponding region of the Lucheng Rn CoV, at the nucleotide and amino acid level, respectively ( Table 2) .Many animal and human CoV spillover events can be ultimately traced back to a bat reservoir . Therefore it is important to determine whether or not there is any evidence to suggest that the rodent CoVs described here are related to those circulating in mainland European/UK bats [21, 26] . Although the sequences described in the bat analyses represented a shorter fragment of the ORF1ab gene, we were able to perform phylogenetic analyses with the novel rodent CoV sequence described here, and these comparisons showed that there was no evidence, based on phylogenetic clustering that the UK/European rodents were directly related to those present in bats (Figure 4) . The nearest relative was FIPV, which infects cats, although the rodent/shrew and cat sequences did fall within a larger cluster that contained coronaviruses isolated from pipistrelle bats. Therefore, it is highly unlikely that the novel rodent/shrew CoVs represent a recent and direct epizootic event from bats. However, more wide scale analyses of CoV infection of European mammal populations needs to be performed to fully understand their evolutionary history. phylogenetic clustering that the UK/European rodents were directly related to those present in bats ( Figure 4 ). The nearest relative was FIPV, which infects cats, although the rodent/shrew and cat sequences did fall within a larger cluster that contained coronaviruses isolated from pipistrelle bats. Therefore, it is highly unlikely that the novel rodent/shrew CoVs represent a recent and direct epizootic event from bats. However, more wide scale analyses of CoV infection of European mammal populations needs to be performed to fully understand their evolutionary history. This study has shown the first evidence of alphacoronaviruses present in European rodents and shrews demonstrating that rodent and shrew coronaviruses sampled to date, from worldwide locations, form a discreet clade within this genus. CoV infection of rodents and shrews appears both geographically and temporally widespread and therefore these mammals may pose a threat for cross-species transmission to humans and/or other animals. Further characterization is required for better understanding of their genetic diversity, host range and cellular receptors.\n",
            "\t0.515\tprotein is the most abundant viral protein in CoV-infected cells, and has been identified as an innate immunity antagonist in several CoVs, including mouse hepatitis virus (MHV) and severe acute respiratory syndrome (SARS)-CoV. However, the underlying molecular mechanism(s) remain unclear. In this study, we found that MHV N protein inhibited Sendai virus and poly(I:C)-induced IFN-β production by targeting a moleculeCoronaviruses (CoVs) are a type species of enveloped viruses with a single-stranded positive RNA genome of approximately 26-32 kilobases. CoVs belong to the Coronaviridae family, classified into the order Nidovirales along with Arteriviridae [1, 2] . All CoVs share similarities in the organization and expression of their genome, which is arranged into open reading frame (ORF) 1a, ORF1b, S, ORF3, E, M and N in order, with both termini flanked by 5′-and 3′-untranslated regions. CoVs are separated into four genera based on phylogeny: alpha-CoV, beta-CoV, gamma-CoV and delta-CoV. The beta-CoV genus contains important veterinary and human etiological agents, such as mouse hepatitis virus (MHV) and severe acute respiratory syndrome coronavirus (SARS-CoV). MHV infects mice liver or neuron, but it is a well-studied coronavirus model for similar patterns of pathogenesis . SARS-CoV is the etiological agent behind an outbreak of severe respiratory disease through China during 2002-2003 .Oncotarget 49656  Virus infection can be sensed in host cells through the interaction of pathogen-associated molecular patterns with host pattern recognition receptors . During CoVs infection, viral components or replication intermediates can be detected by retinoic acid-induced gene I (RIG-I) and melanoma differentiation gene 5 (MDA5) . Activated RIG-I and/or MDA5 recruit the mitochondrial adaptor protein MAVS/IPS-1/Cardif through caspase activation and recruitment domain (CARD)-CARD interactions to activate the downstream κB kinase (IKK)related kinases, such as TANK-binding kinase 1 (TBK1) and IKKε. The TBK1-IKKε complex then mediates downstream activation of the critical transcription factors, interferon regulation factor 3 (IRF3) and nuclear factor κB (NF-κB), triggering their nuclear translocation, where they form active transcriptional complexes that bind to IFN promoter elements and activate type I IFN gene expression . Recent studies have identified a new molecule that acts as a protein activator of protein kinase R (PKR), namely protein activator of the interferoninduced protein kinase (PACT) [21, 22] , and also promotes RIG-I signaling in response to viral infection and dsRNA through interacting with the carboxy-terminal domain (CTD) of RIG-I, which in turn stimulates RIG-I ATPase activity [23, 24] .Many CoVs, including MHV, SARS-CoV, Middle East respiratory symdrome (MERS)-CoV, porcine epidemic diarrhea virus (PEDV), and porcine deltacoronavirus (PDCoV), have been reported to cause dysregulated type I IFN expression to counteract the host innate immune defense, and this is mostly due to virus produced-proteins involved in IFN evasion . Nucleocapsid (N) proteins of CoVs are a substantial part of the viral proteins, seemingly dedicated to suppressing the antiviral response . SARS-CoV and PEDV N proteins can inhibit IFN-β synthesis, and this occurs by an unknown mechanism in SARS-CoV . We have previously found that PEDV N protein blocks IFN-β signaling by targeting TBK1 to prevent its interaction with IRF3. We also investigated whether MHV and SARS-CoV N proteins use the same strategy to antagonize IFN-β production; however, the N proteins of MHV and SARS-CoV do not interact with TBK1 .In the present study, we showed that MHV N protein is responsible for IFN-β inhibition, and exerts its IFN-antagonizing function by targeting PACT, thereby impairing PACT-RIG-I/MDA5 interactions and RIG-I/ MDA5 activation for downstream IFN production. This is the same evasion strategy developed by SARS-CoV N protein. However, this mechanism was not shared by PEDV or porcine reproductive and respiratory syndrome virus (PRRSV) N proteins. Our findings thus shed light on the IFN-regulatory mechanism employed by beta-CoV N proteins.The MHV N protein has been reported to block type I IFN-induced antiviral responses, but the effect on IFN-β production is still unclear . To test this, HEK-293T cells were co-transfected with a DNA expression construct pCMV-tag2b-MHV-N encoding MHV N protein and luciferase reporter plasmid IFN-β-Luc, together with the internal control plasmid pRL-TK, and then infected with Sendai virus (SEV) or treated with poly(I:C). As shown in Figure 1A and 1B, the overexpression of MHV N protein blocked the SEV/poly(I:C)-induced IFN-β promoter activities. To further confirm that IFN-β expression was inhibited by MHV N protein, HEK-293T cells were transiently transfected with pCMV-tag2b-MHV-N plasmid and then infected with SEV for 12 h. Cells were collected for detection of IFN-β relative mRNA expression and SEV NH gene copies by real-time RT-PCR. The expression of MHV N protein was detected by immunoblotting with an anti-Flag antibody. As shown in Figure 1C , SEVinduced mRNA expression of IFN-β was significantly reduced in MHV N protein-expressing cells, meanwhile SEV NH RNA in mock cells equaled to cells expressing MHV N protein. This indicated MHV N perturbed IFN-β production not result from inhibiting SEV replication. The coordinated and cooperative actions of transcription factors IRF3 and NF-κB are essential to mediate type I IFN expression [40, 41] , therefore IRF3 and NF-κB were additionally examined to investigate the role of MHV N protein in type I IFN suppression. Luciferase reporter gene assays were performed with luciferase reporter plasmids IRF-3-Luc ( Figure 1D ) or NF-κB-Luc ( Figure 1E ). Our results indicated that SEV-induced promoter activities of IRF-3 and NF-κB are also blocked by MHV N protein.SEV-induced IFNs initiate a series of signaling cascades through the Janus kinase/signal transducer and activator of transcription pathway to direct antiviral and immune gene expression, resulting in the suppression of virus replication . Vesicular stomatitis virus encoding green fluorescent protein (VSV-GFP) is sensitive to this antiviral response , which makes it a good model system to screen IFN-β antagonists. To further confirm the inhibitory effect of MHV N protein on the SEV-induced antiviral response, HEK-293T cells were transfected or mock transfected with pCMV-tag2b-MHV-N plasmid prior to infection with SEV and VSV-GFP. Dramatic reduction of VSV-GFP proliferation was observed after SEV infection, but a robust restorative effect was observed in N protein-expressing cells ( Figure 1F ). These results further confirmed that MHV N protein antagonizes the production of IFN-β.Oncotarget 49657  MHV N protein blocks IFN-β production by targeting molecule(s) upstream of RIG-I and MDA5RIG-I and MDA5 are RNA sensors and can be activated by their respective RNA ligands to induce the downstream cascade pathway [46, 47] . To identify the molecular target of MHV N protein in the IFN-induction signaling pathway, HEK-293T cells were co-transfected with pCMV-tag2b-MHV-N vector and luciferase reporter plasmids containing the IFN-β promoter and Renilla luciferase-expressing plasmid, pRL-TK, and a series of IFN-β promoter stimulators in the RIG-I-like receptor (RLR) signaling pathway, including RIG/RIG-IN, MDA5, IPS-1, TBK1, IKKε, IRF3 and IRF3-5D. Luciferase assays were performed at 28 h after co-transfection. The overexpression of any molecule in the RLR signaling pathway dramatically stimulated IFN-β promoter activation, but interestingly, MHV N had no inhibitory effect on this activation (Figure 2A-2E ). This is coincident with SARS-CoV N protein inhibition of IFN-β production, which also fails to affect the activation of these signaling molecules. We re-confirmed that overexpression of SARS-CoV N protein did not inhibit IFN-β promoter activation induced by RIG-I ( Figure 2F ) and MDA5 ( Figure 2G ) under our experimental conditions. If MHV N protein involved the identical IFN-β-antagonizing mechanisms to SARS-CoV N protein, which has been demonstrated not interact with RIG-I or MDA5 , MHV N protein should not be supposed to target RIG-I or MDA5. To test this hypothesis, expression plasmids of hemagglutinin (HA)-tagged MHV N and Flag-tagged RIG-I ( Figure 2H ) or MDA5 ( Figure 2I ) were co-transfected into HEK-293T cells. At 28 h posttransfection, co-immunoprecipitations were performed with anti-Flag antibodies. As speculated, The results represent the means and standard deviations of three independent experiments. The relative firefly luciferase activity was normalized to the Renilla reniformis luciferase activity, and the uninfected or untreated empty vector control value was set to 1. (C) HEK-293T cells were transfected with MHV-N expression plasmid. After infection with SEV for 12 h, cells were lysed to extract total RNA, which was used for detecting the expression of the IFN-β, SEV HN and GAPDH genes by quantitative real-time RT-PCR. The results are expressed as increases in mRNA levels or SEV HN gene copies relative to those in cells transfected without SEV infection and were normalized to the expression of the GAPDH housekeeping gene. (F) HEK-293T cells were transfected with MHV-N expression plasmid and then infected with SEV for 12 h. The cells were then re-infected with VSV-GFP for 24 h, followed by analysis for fluorescence by microscopy. *P < 0.05; **P < 0.01; ***P < 0.001.Oncotarget 49658  MHV N protein was not detected in neither Flag-tagged RIG-I nor MDA5 precipitates, suggesting that MHV N protein did not associate with RIG-I or MDA5. All together, these data indicated that MHV N and SARS-CoV N proteins suppress the activation of an upstream molecule in the IFN signaling cascade.Previous studies have revealed that optimal activity of RIG-I and MDA5 requires PACT, a cellular dsRNAbinding protein which binds to RIG-I and MDA5 to induce IFN production [24, 48, 49] . MHV and SARS-CoV N proteins with RIG-I-signaling antagonistic and RNA binding properties were speculated to target PACT. To test this possibility, HEK-293T cells were co-transfected with MHV N or SARS-CoV N expression plasmids and PACT, and either RIG-I or MDA5, along with luciferase reporter plasmids. Luciferase assays were performed to assess the suppressive activity in the presence of PACT and either RIG-I or MDA5 at 28 h after co-transfection. As shown in Figure 3 , PACT promoted the activation of RIG-I and MDA5 for induction of IFN-β promoter activity, but this effect was reversed by the N proteins of MHV and SARS-CoV. These findings indicated that MHV ( Figure 3A and 3B) and SARS-CoV ( Figure 3C and 3D) N proteins counteract the activity of PACT.If MHV or SARS-CoV N proteins target PACT, then colocalization between these proteins should occur within cells. To test this association, Flag-tagged MHV or SARS-CoV N proteins and HA-tagged PACT were co-transfected into HEK-293T cells. Indirect immunofluorescence assay (IFA), reciprocal co-immunoprecipitation and immunoblotting experiments were then performed. IFA showed that MHV and SARS-CoV N proteins were both distributed in the cytoplasm, as was PACT ( Figure 4A ). found that PACT was present in the MHV and SARS-CoV N protein precipitates and vice versa, MHV and SARS-CoV N proteins were present in the PACT precipitate ( Figure 4B-4E) . These results indicated that MHV and SARS-CoV N proteins can interact with PACT.PACT physically binds to the C-terminal repression domain of RIG-I and potently stimulates RIG-I-induced type I interferon production, resulting in stimulation of its ATPase activity . N protein-PACT interactions may impair the association between PACT and RIG-I. To assess this hypothesis, HEK-293T cells were transfected with increasing amounts of Myc-tagged MHV or SARS-CoV N proteins, together with constant amounts of pCAGGS-HA-PACT and Flag-tagged RIG-I. Immunoprecipitation was performed with anti-HA antibody, and the coprecipitated N proteins and RIG-I were analyzed by immunoblotting with anti-Myc and anti-Flag antibodies. As shown in Figure 5A and 5B, PACT efficiently pulled down RIG-I. However, the amount of RIG-I present in PACT precipitates gradually decreased as the amount of MHV and SARS-CoV N proteins increased. These results suggested that MHV and SARS-CoV N proteins inhibit the PACT-RIG-I interaction in a concentration-dependent manner.Considering that co-expression of MDA5 and PACT resulted in enhancement of MDA5induced activation of the IFN-β promoter [48, 49] , we hypothesized that the activation of MDA5 might plausibly be mediated by a direct interaction with PACT. We therefore performed co-immunoprecipitation experiments with lysates of co-transfected or single vector HA-tagged PACT and Flag-tagged MDA5 transfected HEK-293T cells. As shown in Figure 5C and 5D, both HA-tagged PACT and Flag-tagged MDA5 proteins were found to co-precipitate upon detection with anti-HA and anti-Flag antibodies. This indicated that PACT also interact with MDA5. To investigate this interaction further, we repeated the Oncotarget 49660  co-immunoprecipitation experiment to detect whether N-PACT interactions could also disrupt the PACT-MDA5 association. Likewise, gradually decreasing levels of MDA5 were detected in the anti-HA precipitate as MHV and SARS-CoV N protein concentrations were increased ( Figure 5E and 5F), indicating that formation of the PACT-MDA5 complex is impaired by the viral N proteins. Since MHV and SARS-CoV N proteins and PACT are dsRNA-binding proteins , we questioned whether the interactions between N proteins and PACT were mediated by RNA. The co-immunoprecipitation experiments were repeated in the presence of increasing amount of RNase A. The amounts of MHV and SARS-CoV N proteins detected in the PACT precipitates were gradually reduced but not vanished with increasing dose of RNase A ( Figure 6A and 6B) , indicating that MHV or SARS-CoV N protein can directly RNA-independently interact with PACT. However, N-PACT interaction was weakened when endogenous RNA was digested. PACT contains three domains (domain 1 to 3). Domain 1 and domain 2 possess dsRNA binding activity, while domain 3 is not capable of binding RNA . Theoretically, PACT mutant with RNA binding domain deletion could also interact with MHV and SARS-CoV N proteins. To test this, PACT mutant (ΔPACT) deleting domain 1 and domain 2 was constructed ( Figure 6C ) and verified no RNA binding activity ( Figure 6D ). Then ΔPACT-N association was assessed with co-immunoprecipitation experiment using anti-HA antibodies. As shown in Figure 6E and 6F, both MHV and SARS-CoV N proteins were detected in ΔPACT precipitates, with a weaker band compared with wild type PACT precipitates. This confirmed that PACT mutant with no RNA binding activity is sufficient for N interaction.We wished to determine whether this N-PACT targeting mechanism was universal for CoV-encoded N proteins, and potentially even commonly used by the N proteins from arteriviruses, which are of the same phylogenetic order as CoVs. To test this possibility, the N proteins of PEDV, a member of the alpha-CoV genus and PRRSV, a member of the arteriviridae family, were chosen for co-immunoprecipitation experiments; both N proteins are also known to be IFN-antagonizing proteins Oncotarget 49662  but function downstream of RIG-I and MDA5 [38, 52] . As shown in Figure 7A and 7B, neither PEDV nor PRRSV N protein was present in the PACT precipitates. Thus, PEDV and PRRSV N proteins were not supposed to impair PACT-RIG-I/MDA5 complex conformation. To test this, Flag-tagged RIG-I or MDA5 expression plasmids, HA-tagged PACT, together with increasing amounts of pCAGGS-Myc-PEDV-N or pCAGGS-Myc-PRRSV-N vectors, were co-transfected into HEK-293T cells for 28 h. Cell lysates were harvested for co-immunoprecipitation experiments using anti-HA antibodies. As shown in Figure 7C -7F, neither PEDV nor PRRSV N proteins disrupted RIG-I or MDA5 solid interaction with PACT. Hence, these supported the notion that MHV and SARS-CoV N proteins are unique in their counteraction with PACT.Previously, MHV N protein was reported to antagonize the IFN response, with limited information about MHV N protein effects on IFN synthesis. In this study, we found that MHV N protein was responsible for this IFN-antagonizing activity and the molecular mechanism was identical to that of SARS-CoV N protein.We further investigated the shared IFN-antagonistic mechanism of MHV and SARS-CoV N proteins, and found that they targeted PACT to impede the interaction of this molecule with RIG-I/MDA5, leading to blockade of PACT-activated RIG-I/MDA5, and ultimately inhibition of IFN-β synthesis.CoV N proteins share a similar overall topology structure and highly conserved function of packaging the viral genome with structural proteins to form ribonucleoprotein complexes for viral assembly . However, their behaviors in host innate immune mediation were quite distinct from this structural function. Among CoV N proteins, MHV, SARS-CoV, and PEDV N proteins were identified as IFN antagonists . MHV and SARS-CoV N proteins were proved by us to target PACT to inhibit IFN-β synthesis, but the N protein of MHV also inhibited RNaseL-mediated host translation shut-off , which has not yet been shown for SARS-CoV N. PEDV N protein was previously reported to target TBK1, thus disrupting the interaction between IRF3 and TBK1 . Surprisingly, MERS-CoV N protein was demonstrated to be unable to suppress IFN production . But recently, SARS-CoV and MERS-CoV N proteins were discrepantly reported to prohibit RIG-IN induced IFN-β production with same methods . In addition, given the mutual RNA binding properties of CoV N proteins , it will be of interest to determine whether coating of dsRNA is another common mechanism by which N proteins antagonize IFN-β expression, whereby dsRNA is masked from immunosurveillance sensing molecules.PACT harbors multiple functions. Among these, PACT heterodimerizes with PKR and activates it via protein-protein interactions between the dsRNAbinding domain present in each protein . All three domains of PACT can interact with but only domain 3 activates PKR . In this study, we demonstrated that domain 3 of PACT is sufficient for N proteins association, but much weakly compared with wild type PACT, which is share similarity to PACT-PKR interaction. PACT can directly interact with the CTD of RIG-I, thus stimulating its ATPase activity and mediating RIG-I activation. Here, we also identified that PACT induced MDA5 activation is via directly interaction. However, MDA5 seems to be lower activated than RIG-I upon PACT stimuli. Even though RIG-I and MDA5 share a highly homologous structure, they are activated by distinct agonists. Considering the domain of PACT which associates with and activates RIG-I/MDA5 has not been mapped, the mechanisms of PACT-mediated RIG-I/ MDA5 activation and N proteins involved in PACT-RIG-I/MDA5 perturbation remain obscure. Besides, PACT associates with TAR RNA binding protein (TRBP) and/or Dicer, thus participating the RNA interference process . In the context of different binding partners, N-PACT counteraction may involve in distinct physiological process. The Ebola virus protein VP35-PACT interaction has been verified to interfere with RIG-I activation. Furthermore, this was also proposed to contribute to functions of RNA silencing and PKR prohibition in the absence of virus infection or IFN . Previous study suggested that SARS-CoV N protein suppresses RNA interference . Whether the CoV N-PACT association can also contribute to the biogenesis of regulatory RNAs requires further study. PACT-mediated IFN production was described by two mechanisms: PACT potentiation of the dsRNAinduced activation of RIG-I, or direct binding of PACT to the CTD of RIG-I to augment ATPase activation [24, 71] . Thereafter, an expanding list of dsRNA-binding, IFN-antagonizing, and PACT-targeting viral proteins were identified. According to the feature of PACT combination, proteins can be classified into two types: RNA dependent and RNA independent. Us11 protein of Herpes simplex virus 1, VP35 of Ebola virus, and NS1 protein of influenza A virus interactions with PACT require an RNA binding motif, and Us11 or VP35-PACT interactions are verified totally RNA independent [49, 72, 73] . This contributes to the hypothesis that Us11 and VP35 have higher affinities for PACT than RNA, to impair direct binding of PACT to RIG-I. However, targeting of PACT by ORF4a of MERS-CoV and the N proteins of MHV and SARS-CoV are mediated by RNA . Mechanistically, these three proteins may compete with RIG-I/MDA5 for dsRNA and inhibit dsRNA recognition. As mentioned above, PACT can augment RNA agonists in RIG-I/MDA5 activation. However, the interaction of MHV or SARS-CoV N proteins with dsRNA and PACT is proposed to disrupt this process. Alternatively, the antagonist-RNA-PACT complex may render RNA-PACT nonfunctional in terms of RIG-I/MDA5 activation, or prevent its subsequent combination with RIG-I/MDA5.Despite the high homology in genetic sequence , IFN antagonists produced by CoVs exploit divergent strategies with complicated characteristics. Although the majority of proteins from the various CoVs display the same phenotypic trends in IFN production, there are exceptions. While nsp1, -3, -7, -14, -15, and -16 from both SARS-CoV and PEDV were found to inhibit IFN production, as was PEDV nsp5, but not SARS-CoV nsp5 [33, 34, . Thus, IFN-antagonizing activity regulated by the same antagonist with similar function from differing CoVs may do so by the same mechanism, or may not. The strategy exploited by both MERS-CoV and SARS-CoV M protein is attenuation of the formation of functional TBK1containing complexes [83, 84] . Alternative examples include the papain-like proteases (PLPs). PLP from SARS-CoV, PEDV, and MHV can deubiquitinate innate immune modulators, but the specific targets are STING/ RIG-I and STING/TBK1 [33, 85, 86] . It would appear that a strategy of tailoring several inhibitors to one target has been adopted. As demonstrated in the present study, both MHV and SARS-CoV N proteins can target PACT. There may be compensatory mechanisms involving other viral proteins to counter PACT-mediated IFN signaling in place of the N protein, such as the inhibition achieved by ORF4a protein of MERS-CoV . Whether the anti-PACT mechanism could be used by other RNA-binding and IFN-antagonistic proteins requires further research, especially with regards to the proteins from the recently emerged severe pathogens MERS-CoV and PDCoV, the IFN-inhibitory effects of which have not yet been thoroughly investigated. The luciferase reporter plasmids IFN-β-Luc, 4×PRDIII/I-Luc (referred to as IRF-3-Luc), and 4×PRDII-Luc (referred to as NF-κB-Luc), and molecules in RLR signaling pathway including RIG-I, MDA5, IPS-1, TKB1, IKKε, IRF3, IRF3-5D have been described previously , as has the plasmid containing the N protein of PRRSV, pCAGGS-Flag-PRRSV-N . pCAGGS-Flag-PEDV-N-encoded PEDV N protein was constructed with specific primers and amplified from pCAGGS-HA-PEDV-N . pCMV-tag2b-SARS-CoV-N was kindly provided by Deyin Guo (Wuhan University, China). pCMV-tag2b-MHV-N was constructed by standard reverse transcription (RT)-PCR assays with specific primers to amplify the cDNA of the MHV N gene from the total RNA extracted from L929 cells infected with MHV A59. MHV, SARS-CoV, PEDV and PRRSV encoded N proteins from pCMV-tag2B or pCAGGS-Flag were constructed into pCAGGS-HA or/and pCAGGS-Myc plasmids, and were named as pCAGGS-HA-MHV-N, pCAGGS-Myc-MHV-N, pCAGGS-HA-SARS-CoV-N, pCAGGS-Myc-SARS-CoV-N, pCAGGS-Myc-PEDV-N and pCAGGS-Myc-PRRSV-N, respectively. pCAGGS-HA-PACT was similarly constructed from HEK-293T cell RNA extracts. pCAGGS-HA-ΔPACT was constructed with overlap extension PCR by using indicated primers. And all primers used in these constructs were shown in Table 1 HEK-293T cells grown in 24-well plates were cotransfected with 0.1 μg/well reporter plasmid, 0.02 μg/ well pRL-TK plasmid (Promega; as an internal control for the normalization of transfection efficiency), and the indicated expression plasmid or an empty control plasmid. Where indicated, the cells were also mock-infected/ treated or infected/treated with SEV (10 hemagglutinating activity units/well)/poly(I:C) 24 h after co-transfection. The cells were lysed 12 h later and the firefly luciferase and Renilla luciferase activities were determined with the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer's protocol. The data are the relative firefly luciferase activities normalized to the Renilla luciferase activities and are representative of three independently conducted experiments. Data are presented as means and standard deviations (SD). Values of P < 0.05 were considered statistically significant, and P values of < 0.01 were considered highly statistically significant.To determine the effects of MHV N protein on the expression of IFN-β, HEK-293T cells in 24-well plates were transfected with 1 µg of empty vector or plasmid encoding MHV N protein. After 24 h, the cells were mockinfected or infected with SEV for 12 h. Total RNA was extracted from the cells with RNA-Solv Reagent (Omega, GA, USA) and an aliquot (1 µg) was reverse transcribed to cDNA using AMV reverse transcriptase (Roche, Basel, Switzerland). The cDNA (1 µl of the 20 µl RT reaction) was then used as the template in a SYBR Green PCR assay (Applied Biosystems). The abundance of the individual mRNA or viral RNA transcript in each sample was assayed three times and normalized to that of the internal control, glyceraldehyde-3-phosphate dehydrogenase (GAPDH) mRNA. The primers were designed using Primer Express software v.3.0 (Applied Biosystems) as shown in Table 2 .For the transient transfection experiments, HEK-293T cells were transfected with the appropriate plasmids for 28 h. The transfected cells were lysed in 200 μL of lysis buffer (4% SDS, 3% DTT, 0.065 mM Tris-HCl, [pH 6.8], 30% glycerine) supplemented with protease inhibitor (PMSF, Sigma-Aldrich). The lysates were boiled at 100°C for 10 min before separation by SDS-PAGE and electroblotting onto a polyvinylidene difluoride membrane (Bio-Rad), and analyzed by immunoblotting with the indicated antibodies. For co-immunoprecipitation analysis, cells were washed with PBS and lysed for 20 min at 4°C in lysis buffer containing 50 mM Tris-HCl (pH 7.4), 150 mM NaCl, 1% NP-40, 10% glycerin, 0.1% SDS, and 2 mM Na 2 EDTA. The lysates were then cleared by centrifugation, and the proteins were immunoprecipitated overnight at 4°C with affinity antibodies and protein A+G agarose beads (Beyotime, Shanghai, China). The immunoprecipitates were washed three times with 1 mL of lysis buffer and then analyzed by standard immunoblotting procedures. \n",
            "\t0.507\tCape 7535, South Africa; E-Mails: (R.M.); (M.Z.) † These authors contributed equally to this work.Coronaviruses (CoVs) have a global distribution and infect a variety of human and animal hosts, causing illnesses that range from mostly upper respiratory tract infections in humans to gastrointestinal tract infections, encephalitis and demyelination in animals; and can be fatal [1, 2] . The International Committee for Taxonomy of Viruses (ICTV) reports four coronavirus genera, namely Alphacoronaviruses, Betacoronaviruses, Gammacoronaviruses and Deltacoronaviruses . CoVs are enveloped single-stranded, positive-sense RNA viruses with genomes ranging between 26.2-31.7 kb, the largest among known RNA viruses . This large, capped and polyadenylated genome contains seven OPEN ACCESS common coronavirus genes in the following conserved order: 5'-ORF1a-ORF1b-S-ORF3-E-M-N-3' . ORF1a/b encompasses two-thirds of the genome and produces a genome-length mRNA (mRNA1) that encodes two overlapping viral replicase proteins in the form of polyproteins 1a (pp1a) and pp1ab .These polyproteins are formed as a result of a -1 ribosomal frame shift that involves a complex pseudoknott RNA structure and are then proteolytically processed by virally encoded proteases into mature nonstructural proteins (nsp1 to nsp16), which assemble to form a membrane-associated viral replicase-transcriptase complex (RTC) [6, 8, 9] . The last third of the genome produces subgenomic (sg) mRNAs that encode the four structural proteins, spike (S), envelope (E), membrane (M), and nucleocapsid (N), as well as a number of accessory proteins [10, 11] .Amino acid sequence comparisons have shown that CoV N proteins have three distinct and highly conserved domains: two structural and independently folded structural regions, namely the N terminal domain (NTD/domain 1) and C-terminal domain (CTD/domain 3), which are separated by a intrinsically disordered central region (RNA-binding domain/domain 2) ( Figure 1 ); all three domains have been shown in different CoVs to bind with viral RNA . [16, 18, 19] .The NTD is divergent in both sequence and length. It has been mapped for Infectious Bronchitis Virus (IBV)-N to aa 19-162 , for Severe Acute Respiratory Syndrome human coronavirus (SARS)-N to aa 45-181 , and for Mouse hepatitis Virus (MHV)-N to aa . The N-termini of these three CoVs have been found to associate with the 3' end of the viral RNA genome, possibly through electrostatic interactions [21, 22] . There are several common characteristics of CoV N protein NTDs, including predicted secondary structures such as a central β-sheet platform flanked by α-helices , with a basic RNA binding groove along the β-platform and an extended β-hairpin. The NTD is enriched in aromatic and basic residues and the folded shape resembles a hand with basic fingers that extend far beyond the protein core, a hydrophobic palm, and an acidic -wrist‖ . It has been proposed that the flexible, positively charged finger-like β-hairpin extension in the NTD of both IBV and SARS-CoV N protein is able to grasp RNA by neutralizing its phosphate groups, while the base moieties can make contact with exposed aromatic residues from the hydrophobic palm [16, 21] .More precise mapping of the RNA-binding site locations has been determined for SARS-and IBV-N protein. Within the NTD of SARS-CoV-N, positively charged lysine and arginine residues have been proposed to bind a 32 nucleotide stem-loop structure located at the 3' end of the SARS-CoV RNA genome . Site-directed mutagenesis studies on IBV-N have identified two residues that are critical for RNA binding; namely Tyr-94 and Arg-76 . Tyr-94 is located in strand β3 of the four-stranded anti-parallel β sheet; Arg-76 is located in the immediate vicinity of Tyr-94, at the base of the extended flexible hairpin loop . It is however likely that, since no single mutation totally disrupts RNA binding, other aromatic/basic residues at the surface of the NTD contribute to nucleic acid binding by creating a broad surface that comes into contact with the viral genomic RNA . The NTD possesses some features similar to those of other RNA-binding proteins that form a RNP. For example, the U1A spliceosomal protein and the coat protein of MS2 bacteriophage bind viral RNA with residues arising from the surface of a four-stranded anti-parallel β sheet. Seemingly, strands β2, β3, and the flexible β-hairpin from the IBV N protein could fulfill a comparable role by interacting with phosphate groups on the viral RNA . The Arg-76 and Tyr-94 residues in the IBV N protein are well conserved across the whole CoV family, and may structurally correspond to the Arg-94 and Tyr-122 residues in the SARS-CoV N protein , meaning that Arg-94 and Tyr-122 may therefore be critical for SARS N-RNA binding.The crystal structure of MHV N197 (residues 60-197) adopts a U-shaped β-platform containing five short β-strands (arranged β4-β2-β3-β1-β5) across the platform with an extended β2'-β3' hairpin similar to NTDs from other CoV N proteins . Interestingly, the crystal structure of the MHV NTD shares a similar overall and topology structure with that of SARS-CoV and IBV but varies in its potential surface, indicating a possible difference in RNA-binding module . It has been shown that N219, an MHV-A59 N domain protein fragment that contains the folded NTD and the immediately adjacent intact linker region (LKR; residues 60-219), binds to the TRS in the viral genome body (TRS-B) and complementary TRS (cTRS) with high affinity to form a N219-TRS duplex . MHV TRS binds across the β-platform of NTD in a defined orientation, with the 5'-end of TRS near β4 and the 3'-end of TRS near β5; this N219 binding to single-stranded RNAs-containing the TRS or cTRS-uses base stacking interactions between aromatic side chains on the β-platform with a triple adenosine motif within the TRS, 5'-gAAUCUAAACU-3' . Furthermore, due to its potent helix-destabilizing activity, N219 is able to efficiently melt an RNA duplex between the template TRS and nascent cTRS strand into component single strands that may be transiently formed during discontinuous transcription of viral sgRNA by the coronaviral replicase complex . Three residues on the β-platform have been shown to play key roles in TRS binding and helix destabilization: Arg-125 and Tyr-127 on the β3 strand and Tyr-190 on the β5 strand, suggesting that the AAA motif in the 3'-end of the TRS is anchored here . These three residues are completely invariant in betacoronavirus N proteins and occupy precisely analogous positions on the fold of each NTD, and are therefore likely to define similar RNA binding grooves in the closely related SARS NTD . The duplex formation and duplex TRS unwinding activity exhibited by N219 therefore implicates MHV NTD in template switching during discontinuous sgRNA transcription [28, 29] . Moreover, the ability of the NTD to melt dsRNA may also play a role in RNA packaging or other steps in the viral life cycle where RNA remodeling is needed . For example, mutations that cripple duplex unwinding are defective in stimulating CoV replication in BHK-R cells, and are lethal, providing evidence of a critical role for NTD in viral replication [18, 26] . CoV N proteins have also been recognized as RNA chaperones [30, 31] , which, as part of their chaperone activities, anneal nucleic acids, and so RNA duplex destabilization activity may be important in CoV N NTDs role in assisting viral RNA in reaching its functional three-dimentional structure. Viral nucleocapsid and replication accessory proteins from other viruses have also been shown to function as RNA chaperones and facilitate helix destabilization, for example HIV-1 NCp7 protein , and adenovirus DNA binding protein .The NTD is separated from the CTD by an intrinsically disordered middle region referred to as the linker region (LKR). The charged LKR is also known as the SR-domain because it is rich in serine and arginine residues , and it is involved in cell signaling [15, 35, 36] . The flexible LKR is capable of direct interaction with RNA under in vitro conditions . Potential phosphorylation sites have been mapped to the Ser/Arg-rich portion of the LKR of SARS-CoV N . These LKR phosphorylation sites are thought to function in binding M protein, heteronuclear ribonucleoprotein (hnRNP-A1) and RNA to the N protein with high binding affinity [14, . There are conflicting reports regarding the involvement of the LKR in N protein oligomerization. Some studies has suggested that the LKR is directly involved and that through electrostatic effects, hyperphosphorylation of the LKR reduces the total positive charge on the SARS-CoV N protein and leads to enhanced oligomerization of di-domain constructs . Other studies have, however, reported that the LKR interferes with oligomerization when the CTD is present or if the LKR is phosphorylated . Despite almost no structural information being available for the LKR, possibly due to its high positive charge and flexible nature , there is evidence in support of the functional importance of intrinsically disordered regions in proteins for modulating transcription, translation, post-translational modifications such as phosphorylation, and cell signaling . RNA chaperones often have structural flexibility because the RNA-protein recognition process often requires conformational changes in the RNA, the protein or both . An interaction between N protein and a subunit of the viral replicase-transcriptase complex, namely non-structural protein 3 (nsp3), has been described and key binding determinants localize to the LKR [50,51], highlighting the importance of this unstructured region for a number of potential interactions, such as viral infectivity . It has also been proposed that nsp3 binding induces a conformational change in the LKR, potentially regulating the intracellular localization of N to the site of replication and/or other RNA binding functions of N.The CTD, which is a hydrophobic, helix-rich terminal, has been mapped for SARS-N to aa 248-365 , and for IBV-N to aa 219-349 [21, 53] . This domain is also referred to as the dimerization domain because it contains residues responsible for self-association to form homodimers, as well as homo-oligomers through a domain-swapping mechanism [16, 17, 42, . Oligomerization of N protein is necessary to produce a stable conformation because in its monomeric form, the CTD folds into an extended conformation with a large cavity in its center, making it unstable . Sequence comparison shows that the dimerization domain of the N protein is conserved at least among the alpha, beta and gamma groups of CoVs, suggesting a common structural and functional role for this domain . The monomer of cSARS-N, a crystalized C-terminal construct of SARS-N that contains residues 270-370, comprises five short α-helices, one 3 10 helix, and two β-strands . The general shape of the monomer resembles the letter C, with one edge formed by a β-hairpin extending away from the rest of the molecule . This structure is similar to the crystalline structure of another SARS N CTD monomer (NP248-365), which consists of eight α-helices and two β-strands .The cSARS-N dimer interface is formed largely by insertion of the β-hairpin of one subunit into the cavity of the opposite subunit, resulting in the four β-strands of the two subunits forming an anti-parallel β-sheet that is superposed by two long alpha helices . Due to the extensive hydrogen bond formation between the two hairpins, together with hydrophobic interactions between the beta-sheet and the alpha helices, this interface is highly stable , and these interactions suggests that the dimeric structure may in fact represent the functional unit of the N protein . The crystal structure of NP248-365, a SARS-CoV CTD spanning residues 248-365, revealed that the N protein dimer has the shape of a rectangular slab in which the four-stranded β-sheet forms one face of the slab and the α-helices form the opposite face . Similarly to cSARS N, the dimerization interface of NP248-365 is composed of four β-strands and six α-helices, with each protomer contributing one β-hairpin and helices α5, α6 and α7. The two β-hairpins form a four-stranded intermolecular β-sheet that is stabilized through extensive hydrogen bonding. The other part of the dimerization interface is composed of helices α5 and α6, where strong hydrophobic interactions involving Trp302, Ile305, Pro310, Phe315 and Phe316 were observed. The dimer is further stabilized by hydrophobic interactions between the longest helix, α7, and the intermolecular β-sheet . Similarly to cSARS-N and NP248-365, a nuclear magnetic resonance (NMR) study has reported secondary structural assignments of a SARS N protein construct whose dimeric interface also consists of a four-stranded anti-parallel β-sheet and two α-helices .Self-association of the N protein has been observed in many viruses, and is required for the formation of the viral capsid which protects the viral genome from extracellular agents . In addition to the ability of N protein to oligomerize, viral capsid formation also requires RNA-binding ability . Studies revealed that SARS-CoV N protein fragments containing the dimerization domain (residues 236-384) can bind to a putative packing signal within the viral RNA, with the most likely RNA binding site being within the basic region between residues 248-280 . NMR studies then showed that the RNA-binding site between residues 248-280 formed part of the complete dimerization domain structure . It was not until the crystal structure of SARS N CTD was resolved that the molecular basis of RNA-binding activity and organization of the CTD octamer was determined. The CTD spanning residues 248-365 (NP248-365) revealed that, due to the presence of the eight positively charged lysine and arginine residues, amino acids 248-280 form a positively charged groove, one of the most positively charged regions of the N protein . This groove is similar to that in IBV-N CTD, except that the positively charged surface area is larger in the SARS-CoV construct than in the IBV , due in part to the presence of additional negatively charged residues in the IBV N protein and in part due to the absence of residues 215-218 from the IBV construct, which contain two lysine residue in the SARS-CoV construct . The NP248-365 construct, which contains both the charge-rich region (residues 248-280) and dimerization core (residues 281-365) of the dimerization domain, is capable of binding to single-stranded RNA (ssRNA), single-stranded DNA (ssDNA) and double-stranded DNA (dsDNA), and NP248-365 has stronger nucleic acid-binding activity than the NTD [55, 57] . The strong electrostatic character of residues 248-280 and the fact that both ssRNA, ssDNA and ddDNA bind to NP248-365, strongly indicates that oligonucleotide binding is based on non-specific charge interactions between the positively charged protein and the negatively charged nucleic acid backbone [55, 60] . By keeping the RNA-binding domains in close proximity to the CTD, the formation of a large helical nucleocapsid core is therefore possible . Association of the N protein dimers is necessary for further assembly of the core. The full-length dimeric N protein has a tendancy to form tetramers and higher molecular weight oligomers in vitro , and a serine/arginine-rich motif (residues 184-196) has been shown to be important for N protein oligomerization . Two dimers arrange themselves into a butterfly-shaped tetramer, while two butterfly-shaped tetramers unite to form an octamer in the asymmetric unit of the CTD crystal . The octamer is held together through hydrophobic interactions and hydrophilic contacts among the four dimers, and networks of inter-dimer hydrogen bonds further help stabilize the octamer . Crystallography studies have demonstrated that CoV-SARS CTD (NP248-365) packs as an octamer which stacks to form a helical supercomplex structure with a continuous positively charged surface that could potentially allow viral RNA strands to bind and wrap around the helical oligomer structure through electrostatic interactions . The existence of transient self-association between dimers in solution was confirmed using a disulphide trapping technique, and it was shown that by neutralization of excessive charges on the protein, either through environmental charge screening or charge modifications, this transient self-association can be regulated . This proposed biophysical mechanism whereby electrostatic repulsion between N protein molecules acts as an oligomerization switch has implications for understanding how nucleocapsid assembly is then subsequently modulated . In addition, the CTD 45 residues of the MHV N protein have been shown to be the major determinant for interaction with the M protein , and so association of the N protein with the M protein may also play a role in the assembly of the nucleocapsid core into a progeny virion .Oligomerization via the CTD has also been reported in human coronavirus 229E (hCoV-229E) and a recent study has shown that the C-terminal tail peptide, an intrinsically disordered domain that flanks the CTD, plays an important role in dimer-dimer association . The C-terminal tail interferes with oligomerization of the CTD and has an inhibitory effect on viral titer of HCoV-229E; and further understanding this mechanism of oligomerization may provide insight into the viral assembly process and could identify additional targets for drugs to combat CoVs through the disruption of the N protein self-association . The CTD of SARS-N (aa 251-422) is also responsible for stress granule localization that occurs as part of an integrated stress response in arsenite-treated HeLa cells . Once sequestered in these granules, the N protein can induce host translational shutoff.The NTD and the CTD are interspersed by intrinsically disordered regions (IDRs) [19, 37] . Intrinsically disordered proteins (IDPs) or IDRs lack a tertiary structure and have no fixed 3-dimentional shape in the native form. However, IDPs and IDRs play a role in various biological functions including DNA, RNA and protein binding with the disordered regions facilitating access to these binding sites .In fact, the three IDRs in SARS-N (aa 1-44, 182-247 and 366-422) have all been shown to modulate the RND-binding activity of the NTD and CTD [19, 37] . Moreover, both the middle and C-terminal IDRs (Figure 1 ) have been implicated in the oligomerization of the N protein [44, 65] , with the middle IDR also associated with N protein functionality and N-M interaction [19, 39, 40, 66] . It would be interesting to determine whether the presence of three disordered regions in SARS N, compared to the one disordered region in HCoV-NL63 N for example, would result in SARS N having a higher binding affinity to viral, as well as host cellular proteins. If indeed so, could this then indicate a probable basis for the increased pathogenicity of SARS-CoV compared to HCoV-NL63?In order for CoV N proteins to package the viral genome with structural proteins to form ribonucleoprotein (RNP) complexes for viral assembly, two key activities are required: the interaction between protein and nucleic acid and the ability of the complex to oligomerize . The N proteins of SARS-CoV, IBV and MHV have all been shown to perform both these functions. SARS-CoV-N protein interacts with RNA at multiple sites, with all three domains having charged regions . The crystal structures of the NTD and CTD domains of the N protein from SARS-CoV, IBV and MHV all share a similar overall and topology structure, which corroborates a conserved mechanism of nucleocapsid formation for CoVs .Furthermore, despite a lack of significant sequence similarity, the cSARS-N had a similar fold to that of the N protein of porcine reproductive and respiratory syndrome virus, a member of Arteriviridae family, suggesting an evolutionary link between Coronaviridae and Arteriviridae in which the N proteins of both viruses have a common origin . In fact, due to their similar genome organization and viral replication mechanisms, the Coronaviridae and Arteriviridae were united to form the relatively new order Nidovirales.In virus-infected cells, CoV N proteins can localize to the cytoplasm alone or to the cytoplasm and nucleolus . Proteins that are able to localize to the cytoplasm, nucleus and/or nucleolus require multiple signals to determine their subcellular localization . CoV N proteins commonly localize in the nucleolus, and although nucleolar localization/retention signals (NoRSs) and pathways are not well characterized, nucleolar localization usually requires regions in the protein that are rich in Arg residues and is likely cell-cycle dependent [34, 69, 70] .The N protein of IBV was found to localize in the cytoplasm alone or to co-localize in both the cytoplasm and nucleolus [67, 68, 70] . IBV N protein contains a functional nuclear export signal (NES) to traffic N protein to the cytoplasm [68, 71] , and an 8 amino acid NoRS motif at its NTD and is necessary and sufficient for nucleolar retention . It is hypothesized that the localization of IBV-N to the nucleolus forms part of a virus strategy to control sgRNA synthesis in both the host cell and virus by associating with ribosomal subunits and interacting with nucleolar proteins, nucleolin and fibrillarin . Importantly, this interaction is not direct, but mediated through RNA and could therefore simply be an artifact of the proteins having RNA-binding domains . Thus, the nucleolar localization could simply be due to a high density of the host RNA attracting a viral RNA-binding protein. Even so, it has been postulated that the nuclear localization of the N protein may interfere with cellular machinery and thus lead to triggering of apoptosis . The localization of N to nucleoli alone might be cell cycle dependent, because the number and size of nucleoli differ at different stages of the cell cycle: at the beginning of G1 phase, multiple nucleoli can be found, but only single nucleoli can be seen at later G1, S and G2 phases [67, 74] . It was also found that domain 2 of IBV-N predominantly localizes in the nucleus, but when fused with domain 3 (CTD) it localizes to the cytoplasm and thus supports the findings of other studies done on IBV-N localization [35, 68] .The ability for nucleolar localization varies between N proteins of different CoVs. Unlike other CoV N proteins, SARS-CoV N protein is mostly distributed to the cytoplasm [34, 71, 75] . This cytoplasmic localization is somewhat unexpected because there is at least one NoRS in domain 2 and eight putative nuclear localization signal (NLS) motifs within domains I and II of the SARS-CoV N protein , of which the short lysine-rich sequence (366-381) near the carboxy-terminus is a putative bipartite NLS that is unique to SARS-CoV N [71, 76] . As a reason for this, it has been suggested that signals for nuclear and nucleolar targeting of SARS-CoV N protein are poorly accessible to nuclear import machinery due to phosphorylation or conformation restraints . A cytoplasmic NES may be involved in also over-riding the NLS, resulting in significantly less N protein (only 10%) being localized to the nucleolus . Shuttling of N protein from the nucleus to the cytoplasm occurs through phosphorylated-dependent binding of SARS-CoV to 14-3-3, with the absence/inhibition of this 14-3-3 molecule resulting in increased nuclear localization of SARS-N . Also, the deletion of the SR-rich domain contained within the middle region of SARS-N can result in dramatic changes in sub-cellular localization of N compared to wild-type N . These results indicate that the localization of N protein to the nucleus or nucleolus is not a conserved property of Nidovirales .The primary role of CoV N protein is to package the genomic viral genome into long, flexible, helical ribonucleoprotein (RNP) complexes called nucleocapsids or capsids (Table 1) . The nucleocapsid protects the genome and ensures its timely replication and reliable transmission. The filamentous nucleocapsids are 10 to 15 nm in diameter and several 100 nm in length, and these macromolecular structures are visible using electron microscopy . Within the nucleocapsid there are both N-RNA interactions as well as intermolecular association between disulfide-linked N protein multimers . The N-RNA interaction is mediated by binding signals contained within the leader RNA sequences . During the virus life cycle, multiple copies of the N protein interact with gRNA and sgRNA molecules, indicating a role for N protein in viral transcription and translation [79, 80] . The basic building block for CoV nucleocapsid formation is a dimeric assembly of N protein , and it is the CTD of N protein that possesses dimerization function .A structural model of CoV proposes that N protein is not only present in the helical nucleocapsid but also in the internal spherical/icosahedral core . The internal core consists of N protein, RNA and the CTD of M protein. The M protein is the main core shell component and a 16 amino acid domain (aa 237-252) on the CTD of M protein binds directly to N protein via an ionic interaction, leading to specific genome encapsidation in the budding viral particle .The N protein therefore plays an essential structural role in the CoV virion through a network of interactions with (i) the genomic RNA; (ii) M protein and (iii) other N proteins.Assembly of virus particles is an essential step for a productive viral replication cycle. CoV virions contain three envelope proteins, M, E and S, and a viral nucleocapsid, which consists of genomic RNA and N protein, within the viral envelope. [87, 88] . It is believed that the interaction of the nucleocapsid with envelope proteins drives the incorporation of the nucleocapsid in enveloped viruses , and such protein-protein interactions are critical for viral assembly, as has been shown for alphaviruses [90, 91] .N and M proteins are the two major structural proteins in CoV virions . The M protein is anchored by its three transmembrane domains to the viral envelope and its large carboxy-terminal tail in the virion interior interacts with the nucleocapsid . The nucleocapsid consists of the positive strand genomic RNA, mRNA 1, helically encapsidated by N protein monomers, and the N protein region that interacts with the C-terminus of the M protein domain seems to be CoV specific. The intracellular sites of virus assembly also vary among different viruses [94, 95] .In MHV, the large carboxy-terminal domain of the M protein interacts with the CTD of the N protein . Newly synthesized, unglycosylated M protein interacts with N protein at the ER membrane, which is a pre-Golgi compartment that is also the site of MHV budding , suggesting that the site of interaction overlays with MHV budding sites . The M protein-nucleocapsid interaction is thought to be initiated by a direct binding of M protein to genomic RNA that is mediated by a 69 nucleotide (nt) packaging signal (ps) present only on the mRNA 1 association [93, 97] . These ps is located about 21kb from the 5' end of MHV mRNA 1 [97, 98] , is necessary and sufficient for packaging RNA into MHV particles , and it has been suggested that the M protein-ps interaction could lead to the association of M protein with N protein, thereby stabilizing the complex between M protein and the nucleocapsid . Although the M protein-nucleocapsid interaction could theoretically also be initiated by direct binding of N protein to genomic RNA, this is unlikely because N protein interacts with all mRNAs [79, 93] , which makes it difficult to explain how the formation of N protein-mRNA 1 RNP complex might lead to specific packaging of genomic RNA, and not sgRNA, into virus particles . It has since been conclusively demonstrated that M protein selectively interacts with ps-containing RNA in the absence of N protein, indicating that the mechanism of M-ps recognition does not require the formation of RNP complex by N protein, and in fact, N protein is not required for RNA packaging in that model . MHV M protein was the first example of a viral transmembrane protein that could bind to a specific viral RNA element in the absence of any other viral structural proteins , and a proposed model for RNA packaging in MHV suggests that once M protein accumulates and oligomerizes in the intermediate compartment between the ER and Golgi complex, M protein binds to ps-mRNA 1, and only thereafter does N protein associated with mRNA 1 interact with oligomerized M protein . Although N protein appears to be dispensable for MHV RNA packaging, N-M interaction might be important in compensating for viral envelope defects that occur due to M protein mutation. The M protein carboxy terminus is extremely sensitive to mutations, and removal of even only the last two amino acid residues from the tail of the M protein appears to be lethal . Interestingly, N protein becomes mutated in its CTD, and these changes then compensate for the loss of the two M protein residues, either by increasing the affinity of an adjacent interaction or by providing a new contact point between N and M to stabilize the virion .For the porcine transmissible gastroenteritis coronavirus (TGEV), an interaction between the carboxy terminus of M and nucleocapsid has been mapped to residues 233-257 of the TGEV M protein . This segment corresponds to residues 201-224 of the MHV M protein , which overlaps with only one of the critical residues identified in the MHV M protein . This region of the two M proteins is, however, poorly conserved and the apparent disagreement between the TGEV and MHV results may relate to differences in the respective folds of the M proteins, or differences in how these residues influence those folds . SARS-CoV is markedly different from other members of the Coronaviridae family in the sense that there is only 20%-30% amino acid identity with other known CoVs, with both the N and M proteins having low sequence homology [76, 101] . One might therefore expect that there could be differences in the mechanism of viral assembly. A mammalian two-hybrid system, which is performed in vivo so that viral proteins will adopt their native state and therefore be more likely to interact in a biologically accurate manner , confirmed that N-M protein interactions occur in vivo . Moreover, this study identified a stretch of amino acids in the middle of the N gene that may be critical for N-M protein interaction . This stretch of amino acids spans the LKR and dimerization domain in the CTD, suggesting that this region may be essential in maintaining correct N protein conformation for both self-association and N-M protein interaction . Despite SARS-CoV having the closest genetic resemblance to MHV, the M proteins of these viruses bind to different domains on the N protein.CoVs assemble and bud intracellularly at the ER-Golgi complex [96, 103] , and association of the nucleocapsid with this organelle may reflect a role in virus budding. The formation of the virion envelope requires expression of only M-and E-protein, and not N protein, as has been observed for MHV , IBV , TGEV , and BCoV . It was recently noted however, that these studies all used vaccinia-based expression systems, where overexpression of viral membrane proteins may lead to release in microvesicles, complicating the interpretation of virus-like particle (VLP) experiments . Subsequent experiments using transient transfection to express the proteins from plasmids have shown that, at least for MHV , SARS-CoV and IBV , the presence of N protein can greatly increase VLP yield. Therefore, while N protein is not necessarily required for envelope formation, N protein plays an important role in forming a complete virion .N protein binds to both full-length genomic RNA (gRNA) as well as all six sgRNAs but displays an increased affinity for gRNA . The gRNA functions as a template for the viral RNA-dependent RNA polymerase as well as a message for translation . During infection, gRNA is initially transcribed by an early polymerase activity into a genome-sized negative-stranded RNA and then a late polymerase activity transcribes the negative-stranded RNA into a full length gRNA that is bound to polysomes and detected in nucleocapsid structures . Numerous studies have demonstrated that N protein is required for optimal CoV replication [31, . The participation of N protein in an early event in RNA synthesis is implied by at least two things: firstly MHV-and SARS-CoV N protein colocalize intracellularly with replicase components at early stages of infection ; and secondly, stimulation of gRNA infection is dependent upon N protein translation [52, 126] .A more clearly defined role for N protein in gRNA synthesis was delineated when an interaction between the SR region of the MHV N protein and a region in the amino terminal segment of the nsp3 subunit of the viral replicase was discovered [51, 52] . The N-nsp3 interaction has been specifically mapped to the ubiquitin-like domain (Ubl1) of nsp3, an essential domain to the virus . Moreover, it was also shown that this N-nsp3 interaction is required for N protein to promote optimal infectivity of gRNA . It was proposed that the formation of an initiation complex at the 3' end of the genome requires N protein to tether gRNA to the newly translated replicase via its interaction with nsp3 [52, 127] . Considering that the start of negative-strand synthesis is the first step in both genomic replication and transcription [128, 129] , it is likely that this N-nsp3 interaction is important in both of these RNA-synthetic processes . The role of N protein in MHV gRNA synthesis may not necessarily be limited to early time points of infection as it has also been proposed that N protein sustains ongoing transcription throughout the course of infection [31, 130] . The role of CoV N protein in RNA synthesis remains controversial as N protein is not essential for TGEV RNA replication but rather for efficient transcription .RNA chaperone proteins assist the proper folding of nucleic acids [131, 132] . Chaperone activity has been postulated as a general activity of all CoV N proteins, and has been demonstrated for TGEVand SARS-CoV N proteins . It has been proposed that amino acids 117-268 in the central disordered domain (the LKR domain) of the TGEV N protein has chaperone activity , and that this region facilitates template switching in vitro by decreasing the energy barrier needed to dissociate the nascent minus RNA chain from the gRNA template during discontinuous RNA transcription [30, 31] . This facilitation of template switching is required for efficient transcription and may explain how the presence of N protein therefore increases RNA synthesis.Deregulation of the cell cycle is a common strategy adopted by many RNA (and DNA) viruses aimed at exploiting host cell machinery in order to create a more favorable environment for their survival. One of the MHV nonstructural proteins, p28, induces G 0 /G 1 cell cycle arrest by inhibiting hyperphosphorylation of retinoblastoma protein (Rb), a step which is necessary for cell cycle progression through late G 1 into the S phase . A model was proposed whereby expressed cytoplasmic p28 induces stabilization of p53, and accumulation of p53 causes transcriptional upregulation of p21, a cyclin-dependent kinase (CDK) inhibitor, leading to suppression of cyclin E/CDK 2 activities, and the resulting reduction in G 1 cyclin-CDK complexes and CDK activities ultimately inhibits Rb hyperphosphorylation . Certain herpes simplex virus gene products [136, 137] , cytomegalovirus gene products [138, 139] , and Zta of Epstein-Barr virus have similarly also been shown to arrest cell cycle progression at the G 1 phase. IBV infection also perturbs cell cycle progression and arrests cells in the S and G 2 /M phases, and this occurs partly through the interaction of IBV nsp3 and DNA polymerase delta .The N protein of SARS-CoV also modulates the host cell cycle by regulating cyclin-CDK activity such that S phase progression is halted. The N protein bears a signature cyclin box-binding region (RXL motif), can be phosphorylated by CDK and is therefore a substrate for the cyclin-CDK complex . N protein has been shown to have an inhibitory effect on S phase cyclins CDK4 and to a lesser extent CDK6 . The inhibition of kinases means that Rb remains hypophosphorylated and cannot release E2F1, and the transcription of S phase genes is halted . Similarly to MHV p28, SARS-CoV N protein also inhibits CDK2 activity, meaning that by blocking the activity of both G 1 and S phase cyclins, N protein doubly ensures the blockage of S phase progression . N protein mediated inhibition of CDK activity; leading to inhibition of Rb phosphorylation, is independent of CDK inhibitors (CDKIs) such as p21, and so it has been suggested that the N protein mimics the role of CDKIs by acting as a competitive inhibitor to CDK4 and CDK2 . The ability of N protein to inhibit CDK4 activity was dependent on the RXL motif, whereas its ability to inhibit CDK2 requires at least two mechanisms: the CDK phosphorylation motif and dependence on host cell signaling pathways . It has been postulated that, p28-mediated (MHV) and N protein-mediated (SARS) blocking of the S phase, allows these virus enough time to utilize the cellular raw materials that have been synthesized ahead of the S phase, for replication of its genome, as well as for assembly and budding of progeny particles . These studies provide evidence that even within the CoV family, different proteins can deregulate the cell cycle and that those proteins, in the case of SARS-N protein, can employ multiple mechanisms to achieve this.In response to environmental stress, eukaryotic cells reprogram their mRNA metabolism to adapt to stress-induced damage. Translationally stalled mRNAs together with a number of translational initiation factors and RNA-binding proteins are selectively deposited in cytoplasmic stress granules (SGs) . CoVs, like many other viruses, induce host translational shutoff, while maintaining synthesis of their own gene products, as a means to interfere with cellular defense mechanisms . MHV-induces a host translational shutoff that is reminiscent of a cellular stress response; SGs appear, numerous mRNAs including protein translation-related factors are downregulated and levels of phosphorylated translation initiation factors increase . It is important to remember, though that this translational shutoff could also simply be the host cell's response to the infection.In SARS-infected HeLa cells that are exposed to arsenite treatment, N protein translocates to cytoplasmic SGs where it colocalizes with poly(A)-binding protein 1 (PARB) and transiently expressed TIA-1 , both of which are components of SGs . This localization and pattern was enhanced when the protein was hypophosphorylated (NΔRS), perhaps because NΔRS induced larger ribonucleoprotein (RNP) complex formation . The sequestration of N protein in SGs possibly indicates a role in translational suppression. The nucleocapsid protein of both porcine and respiratory syndrome and rubella virus also interact with PARB, which promotes protein synthesis by circularizing mRNAs, and inhibit host protein synthesis by sequestration of PARB through a stoichiometric mechanism [146, 147] . It has been proposed that, at least in the rubella virus, that N-dependent inhibition of translation via PARB binding may facilitate the switch from viral translation to packaging RNA into nucleocapsids . More specifically, binding of the capsid to PARB late in the virus cycle, when the levels of newly synthesized genomes and viral proteins are high, and there is little need for additional replicase components, could prevent recruitment of ribosomes to the nascent 40S RNA. This scenario would favour packaging of the 40S genomic RNA into nucleocapsids . Considering that SARS-N and MHV-N protein has been shown to interact with the PARB hnRNP-A1 [41, 148] , it is possible that the N-PARB interaction also acts as a switch that redirects viral activity from RNA synthesis to nucleocapsid formation.The translational suppressive effect of the N protein could also be through its CTD interaction with elongation factor 1α (EF1α), a major translational factor in mammalian cells . N protein of SARS-CoV binds directly to and induces aggregation of EF1α, resulting in inhibition of protein translation . Moreover, EF1α is a multifunctional protein and has unconventional functions related to its association with the cytoskeleton, including interaction with filamentous (F) actin, promotion of F actin bundling and formation of a contractile ring during cytokinesis . N-EF1α therefore blocks EF1α-mediated F actin bundling and the formation of the contractile ring, leading to the formation of multinucleated cells by inhibiting cytokinesis . Multinucleated syncytia of macrophages have been observed in late-phase SARS-CoV but not in other CoV-infected patients . The N protein of hCoV-229E also binds to EF1α, but with significantly lower affinity than SARS-CoV N, and induces multinucleation much more slowly in a substantially smaller fraction of transfected cells . In addition, N-protein mediated inhibition of cytokinesis leads to inhibition of cell proliferation in human peripheral blood lymphocytes (PBLs) . Since actively dividing lymphocytes are a major cell target of SARS-CoV , SARS-CoV N may decelerate lymphocyte proliferation and therefore interfere with immune system function .SARS-N protein has also been shown to cause cytoskeletal changes in response to a different type of cell stress, namely serum deprivation . N protein induces actin reorganization in COS-1 monkey kidney cells by down regulating focal adhesion kinase and fibronectin via p38 MAPK pathway activation .N protein plays an important role in viral pathogenesis since anti-N monoclonal antibodies protect mice from lethal infection of JMHV . Coronaviruses seem to have developed mechanisms to overcome the host innate immune response, including the interferon response, a major component of innate immunity. SARS-CoV infected cells do not allow production of interferon and so it seems likely that SARS-CoV, like many other viruses, has developed mechanisms to subvert the interferon response . SARS-CoV N protein is one of three β interferon (IFβ) antagonists, with ORF3b and ORF6 being the other two, and inhibition of IFβ synthesis by N protein is due to inhibition of IRF-3 and NF-kB, two of the transcription factors required for IF gene expression . SARS-CoV N protein inhibits the synthesis of type-1 interferon (1FN) and the CTD of N has been shown to be critical for antagonism of 1FN induction . This inhibition of the interferon response likely contributes to the pathogenesis of SARS-CoV.CoV infection is likely to activate a diversity of host cell signal transduction pathways and kinases, which would lead to phosphorylation of N protein . N protein phosphorylation would therefore only occur at a relatively late stage of the viral cell cycle and may explain why more phosphorylated N protein is incorporated into IBV virions .It has been demonstrated that the SARS-CoV N protein can bind to DNA in vitro . Recently, the N protein of hCoV-OC43 was shown to interact with the transcription factor nuclear factor-kappa B (NF-kappaB) . It was reported that hCoV-OC43 N protein potentiates NF-kB activation as a direct result of the ability of the nucleocapsid to bind microRNA miR-9, a negative regulator of NF-kB. It was suggested that this previously undescribed interaction between virus and host is a potential mechanism of immune evasion in RNA viruses because NF-kB required for IF gene expression [73, 158] . 148] , which could act as a switch that redirects viral activity from RNA synthesis to nucleocapsid formation.  Interaction of N protein CTD with elongation factor 1α (EF1α), a major translational factor in mammalian cells, can suppress translation . N protein plays an important role in viral pathogenesis. Mice infected with JMHV protected by anti-N monoclonal antibodies .  Synthesis of type-1 interferon (1FN) inhibited by SARS-CoV N .  The CTD of N has been shown to be a critical antagonist of 1FN induction 2.5. Signal transduction  Activation of host cell signal transduction pathways and kinases leads to phosphorylation of N .NF-kB is also involved in SARS-CoV mediated activation of cyclooxygenase-2 (COX-2), more specifically, N protein binds to NF-kB and CCAAT/enhancer binding protein binding sites on the COX-2 gene promoter . Activation of COX-2, a proinflammatory factor, by SARS-CoV N protein is a likely cause of lung inflammation in SARS-CoV infected patients . SARS-CoV N protein is also known to activate the activator protein 1 (AP1) signal transduction pathway by increasing the amount of transcription factors binding to promoter sequences of c-Fos, ATF2, CREB-1, and FosB . SARS-CoV N protein induces apoptosis of COS-1 monkey kidney cells in the absence of growth factor by down-regulating extracellular-signal-regulated kinase (ERK) and up-regulating c-Jun N-terminal kinase (JNK) and p38 mitogen-activated protein kinase (MAPK) pathways, with activation of p38 MAPK also causing actin reorganization in serum-deprived cells .The coronavirus N protein is abundantly produced within infected cells. N has multiple functions, including binding to viral RNA to form the ribonucleocapsid and has also been proposed to have roles in virus replication, transcription and translation. In host cells, N proteins have been shown to cause deregulation of the cell-cycle [70, 142] , inhibit the production of interferon , up-regulate the production of COX2 , up-regulate the activity of AP1 , and induce apoptosis in serum deprived cells -of all which may have possible pathological consequences . Several excellent reviews on the coronavirus N protein have previously been published [165, 166] , including one on the structure and function of the SARS-CoV N and its interaction with nucleic acids . Notwithstanding these reviews, the manner in which coronavirus N proteins carry out its roles during the viral life cycle is still not clearly understood. An important piece of missing information lies in the difficulty in resolving the atomic structure of the RNP complex, which has been hampered by low solubility of the RNP complex and the labile nature of the full-length N protein. In addition, in order to determine whether the RNPs from various coronaviruses share a common structural code, the structure of different coronavirus RNPs need to be resolved .Direct intraviral protein-N interactions identified to date include the interaction between N and M [61, 92, 93] and N and nsp3a, a component of the viral replicase [10, 52] . Additionally, in MHV-infected cells, monoclonal anti-N antibody co-immunoprecipitates both M and S proteins; this N-S interaction is not a direct one though. Rather, it is due to the interaction between S and M protein , where the S protein forms complexes with M protein in the endoplasmic reticulum (ER) [167, 168] . The identification of host proteins targeted by viral proteins during the infection process provides important insights into mechanisms of viral protein function . To date, the interaction of N with numerous host cell proteins have been identified, including hCypA , proteasome subunit p42 , the B23 phosphoprotein [172, 173] , Smad3 , nRNP-A1 , the chemokine CXCL16 , translation elongation factor-1 alpha , cellular pyruvate kinase protein , 14-3-3 and nucleolin [73, 177] . More recently, a study using high-throughput mass spectrometry identified a list of cellular proteins that could potentially interact with the IBV N protein . Comparative studies between various coronavirus N protein interactions could provide valuable information on host specificity and evolution of the interactions between N and host cell proteins. In turn, this may offer insight into the development of novel antiviral therapeutics that target interactions between host cell proteins and the N protein [177, 178] .SARS-CoV N protein is extremely antigenic. SARS-CoV infection causes a highly restricted, immunoglobulin G-dominated antibody response that is directed most frequently and predominantly at the nucleocapsid . DNA vaccines encoding SARS-CoV N protein generate a strong N-specific humoral and T-cell-mediated response and significantly reduce the viral titre of challenging vaccinia virus in C57BL/6-vaccinated mice . Importantly though, another study suggests that N does not induce virus neutralizing antibody and as a result, provides no protection to infection in hamsters . In the diagnosis/screening hCoV-OC43, rabbit poloyclonal antibodies demonstrated greater immunoreactivity to the central (LKR) region and CTD than the NTD of N protein in serum samples, highlighting that LKR region is a strong candidate for use in the design of diagnostic tools . Understanding the role of N in coronavirus infection could lead to the development of novel therapeutics that could potentially be used to combat the threat posed by the emerging lethal human coronaviruses identified in recent times. \n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.083\tCoronaviruses (CoVs) are a group of enveloped viruses with a large positive single-stranded RNA genome (∼26-32 kb in length) of the subfamily Coronavirinae under the family Coronaviridae. The complete genome of CoV contains five major open reading frames (ORFs) that encode replicase polyproteins (ORF1ab), spike glycoprotein (S), envelope protein (E), membrane protein (M), and nucleocapsid protein (N) flanked by a 5 -untranslated region (UTR) and a 3 -UTR. Currently, members of the subfamily Coronavirinae are classified into four genera, Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and Deltacoronavirus (Fehr and Perlman, 2015; Su et al., 2016) . CoVs can cause upper and lower respiratory diseases, gastroenteritis, and central nervous system infections in a wide variety of avian and mammalian hosts. Some CoVs are human pathogens that cause mild to severe disease, including NL63 and 229E of the genus Alphacoronavirus, and severe acute respiratory syndrome CoV (SARS-CoV), Middle East respiratory syndrome CoV (MERS-CoV), OC43, and HKU1 of the genus Betacoronavirus (Dijkman and van der Hoek, 2009; Desforges et al., 2014; van den Brand et al., 2015; Lim et al., 2016) . The SARS pandemic originated in Guangdong Province in China between 2002 and 2003 , and spread to 29 countries, resulting in nearly 8,000 cases and 800 deaths worldwide (Donnelly et al., 2004) . Since being discovered in Middle Eastern countries in 2012, MERS-CoV has infected 2,260 people with a current fatality rate of 35.5% (Mackay and Arden, 2015; van den Brand et al., 2015; de Wit et al., 2016) .With the discovery of diverse wildlife-borne CoVs in different regions of the world, previous studies have indicated that bats are the main and original natural reservoirs of Alphacoronavirus and Betacoronavirus (Calisher et al., 2006; Woo et al., 2012) . The special metabolic and immune systems allow bats to tolerate diverse viruses, and the social roosting behavior of many bat species lets them serve as ideal incubators for the occurrence of frequent co-infection, recombination, and intraspecies transmission of CoVs O'Shea et al., 2014; Hu et al., 2015; Lu et al., 2015; Sola et al., 2015) . Although 15 years have passed without a recurrence of the SARS outbreak, the consistent discovery of SARS-related CoVs (SARSr-CoVs) in Rhinolophus bats indicates an ongoing risk of SARS reemergence Balboni et al., 2012; Yang et al., 2013; Wu et al., 2016b; Hu et al., 2017) . Recently, a CoV of Rhinolophus bat origin, swine acute diarrhea syndrome CoV (SADS-CoV), was found to be responsible for the death of 24,693 piglets across four farms in Guangdong Province (Zhou et al., 2018) . Besides similar clinical signs, the outbreak of SADS coincided with infection by another swine enteric CoV, porcine epidemic diarrhea virus (PEDV; Zhou et al., 2018) . It is worth mentioning that a PEDV-related bat CoV, BtCoV/512, was previously identified in Scotophilus kuhlii bats (Tang et al., 2006; Wang et al., 2016; Gerdts and Zakhartchouk, 2017; Zhou et al., 2018) . All of these findings emphasize the importance of continuous monitoring for ecological diversity of CoVs in bats to minimize the impact of potential CoV-related diseases on public health and economic growth.As the original site of the SARS epidemic in humans and SADS outbreak in pigs (Heymann et al., 2013) , Guangdong Province is also the primary region in China in which wildlife is being consumed. Since the supply of wildlife for consumption in Guangdong mainly comes from Yunnan and Guangxi Provinces (Wu et al., 2016b) , understanding the presence and dynamic changes in SARS-or SADS-related bat CoVs within and between these provinces is important for predicting and tracing CoVrelated diseases. Sichuan is next to Yunnan Province, and has high ecological diversity of animal resources, and it is the largest pig-farming province in China. Although PEDV-related pig diseases were reported previously (Cai et al., 2016) , no systematic survey of bat-borne CoVs has been conducted in Sichuan to establish if there are SARS-or SADS-related CoVs.In this study, a survey of bat-borne CoVs was performed in bats of Guangxi, Yunnan, and Sichuan provinces between 2016 and 2017. Samples from Rhinolophus bats in Sichuan and Yunnan Provinces, and S. kuhlii and Cynopterus sphinx in Guangxi Province were found to be CoV-positive, and a large number of sequencing reads classified into diverse alpha-and beta-CoVs were obtained using virome analysis. The characterization of 11 CoVs in bats from different regions provides clues for the evolutionary relationships of PEDV, SARS-CoV, and other batassociated CoVs.Bats were treated according to the guidelines of Regulations for the Administration of Laboratory Animals (Decree No. 2 of the State Science and Technology Commission of the People's Republic of China, 1988) . The sampling was approved by the Ethics Committee of Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College (Approval number: IPB EC20100415). The bat species were initially determined morphologically and subsequently confirmed by sequence analysis of mitochondrial cytochrome b DNA (Tang et al., 2006; Du et al., 2016) . Anal swab samples from captured bats were immersed in virus sampling tubes (Yocon, China) containing maintenance medium and temporarily stored at −20 • C. The samples were then transported to the laboratory and stored at −80 • C. The accurate sampling locations were recorded by place name, latitude, and longitude.The samples of each species were pooled by adding 1 ml from each maintenance medium sample into one new sample tube. The pooled samples, classified by species, were processed with a virusparticle-protected, nucleic acid purification method as described in our previous studies (Wu et al., 2012 (Wu et al., , 2016a (Wu et al., , 2018 . The samples were homogenized and subsequently filtered through a 0.45 µm polyvinylidene difluoride filter (Millipore, Germany). The filtered samples were then centrifuged at 150,000 × g for 3 h at 4 • C. To remove naked DNA and RNA, the pellet was digested in a cocktail of DNase and RNase enzymes. The viral DNA and RNA were simultaneously isolated using a QIAmp MinElute Virus Spin Kit (Qiagen, United States). First-strand viral cDNA was synthesized using the primer K-8N and a Superscript III system (Invitrogen, United States). The cDNA was converted into dsDNA by Klenow fragment (NEB, United States). Sequenceindependent PCR amplification was conducted using primer K. The PCR products were analyzed by agarose gel electrophoresis. All DNA smears larger than 500 bp were extracted by a MinElute Gel Extraction Kit (Qiagen). The extracted nucleic acid libraries were then analyzed using an Illumina HiSeq2500 sequencer, for a single read of 100 bp. The sequence reads were filtered using previously described criteria (Wu et al., 2012 (Wu et al., , 2016a (Wu et al., , 2018 . Reads with no call sites, reads with similarity to the sequencing adaptor and the primer K sequence, duplicate reads, and low-complexity reads were removed.Sequence-similarity-based taxonomic assignments were conducted as described in our previous study (Wu et al., 2012) . Valid sequence reads were aligned to sequences in the NCBI non-redundant nucleotide database and non-redundant protein database using BLASTn and BLASTx, respectively. The taxonomies of the aligned reads with the best BLAST scores (E score <10 −5 ) were parsed by the MEGAN 6 -MetaGenome Analyzer.CoV screening of individual samples was performed by amplifying a 440-bp fragment of the RNA-dependent RNA polymerase (RdRp) gene of CoVs using conserved primers (5 -GGTTGGGACTATCCTAAGTGTGA-3 and 5 -CCATC ATCAGATAGAATCATCATA-3 ), as described previously (Du et al., 2016) .Sequence reads classified into the same virus genus were extracted. The accurate locations of the reads were determined based on the alignment results exported with MEGAN 6. Specific primers were designed from the located sequence reads. Fragments between reads were amplified with nested specific primers and then sequenced. The remaining genomic sequences were determined using 5 -and 3 -rapid amplification of cDNA ends.MEGA6.0 was used to align nucleotide sequences and deduced amino acid sequences using the MUSCLE package and default parameters. The best substitution model was then evaluated by the Model Selection package. Finally, we constructed a maximum-likelihood method using an appropriate model to process the phylogenetic analyses with 1,000 bootstrap replicates. Recombination among CoVs was detected with SimPlot software. All analyses were performed with a Kimura model, a window size of 1000 bp, and a step size of 100 bp.All genome sequences were submitted to GenBank. The accession numbers for the five bat alpha-CoVs were MK211369 to MK211373. The accession numbers for the six bat beta-CoVs were MK211374 to MK211379.Anal swabs from 831 bats of 15 species were collected from Yunnan, Guangxi, and Sichuan Provinces in China between August 2016 and May 2017. All anal swab samples were classified by species and then combined into 27 pools and subjected to virome analysis. CoV-related sequencing reads were detected in ten of the 27 sample pools. These 10 pools related to 22 Rhinolophus spp. samples from Sichuan, 176 Rhinolophus affinis samples from Yunnan, 150 S. kuhlii samples from Guangxi, and 30 C. sphinx samples from Guangxi. Pan CoV screening revealed that 22 individual samples were CoV-positive (Table 1 and Figure 1) . A total of 1,040,901 reads of 100 bp in length showed the best matches with Coronavirinae viral proteins in the NCBI non-redundant database, and these reads showed 40-100% amino acid (aa) identity with known alphaor beta-CoVs.Eleven viruses of representative positive samples were selected for genomic sequencing as quasi-species. There were five viruses in the genus Alphacoronavirus; four from S. kuhlii in Guangxi Province were named as BtSk-AlphaCoV/GX2018A (α-GX2018A), BtSk-AlphaCoV/GX2018B (α-GX2018B), BtSk-AlphaCoV/GX2018C (α-GX2018C), and BtSk-AlphaCoV/GX2018D (α-GX2018D). The other virus from R. sinicus in Yunnan Province was named as BtRs-AlphaCoV/YN2018 (α-YN2018). There were six CoVs in the genus Betacoronavirus. The virus identified from Rhinolophus spp. in Sichuan Province was named as BtRl-BetaCoV/SC2018 (β-SC2018); four CoVs from R. sinicus in Yunnan Province were named as BtRs-BetaCoV/YN2018A (β-YN2018A), BtRs-BetaCoV/YN2018B (β-YN2018B), BtRs-BetaCoV/YN2018C (β-YN2018C), and BtRs-BetaCoV/YN2018D (β-YN2018D). One CoV from C. sphinx in Guangxi Province was named as BtCs-BetaCoV/GX2018 (β-GX2018).As shown in Table 2 and Figure 2 , with the typical genome organization, the full genome sizes of 11 CoVs ranged from 28, 146 (α-GX2018C) to 30, 256 bases (β-YN2018B) and their G + C contents ranged from 0.38 to 0.41. There were several accessory ORFs in each CoV strain. These accessory ORFs were mainly present in the regions between S and E (named ORFs-e), M and N (named ORFm-n), or downstream of N (named ORFn). The accessory ORFs of α-GX2018 (A-D), α-YN2018, and β-GX2018 were mainly present in ORFse and ORFm-n regions; the small ORFs of β-SC2018 and BtSk-AlphaCoV/GX2018CFIGURE 1 | Numbers of bat samples from Sichuan, Yunnan, and Guangxi Provinces. The size of the pie chart is in proportion with the total number of specimens collected from each province. Red color represents samples positive for CoV, and blue color represents negative samples.β-YN2018 (A-D) were mainly present in ORFs-e and ORFmn regions. We named these accessory ORFs according to previous studies. As shown in Table 3 , The core sequences of the leader transcriptional regulatory sequence (TRS; 5 -CUAAAC-3 ) were identified in the 5 untranslated sequences of α-GX2018 (A-D) and α-YN2018, which is unique to Alphacoronavirus (Madhugiri et al., 2014) . The TRS motifs of S, ORF3, E, and M genes in α-GX2018 (A-D) differed from the core sequences of the leader TRS (Supplementary Table 1 ). The TRS motif of S was identified as 5 -CCAAAU-3 or 5 -CCAAAC-3 ; the TRS motif of ORF3 was identified as 5 -CAUUAC-3 ; the TRS motif of E was identified as 5 -CUAGAC-3 ; the TRS motif of M was identified as 5 -AUAAAC-3 . An alternative TRS motif (5 -GUAAAC-3 ) of α-YN2018 was discovered preceding ORF3, which differed by 1 nucleotide (nt) from those of all other genes in α-YN2018. In addition to the E of β-GX2018, the TRS motifs of six betacoronaviruses all conformed to the consensus motif 5 -ACGAAC-3 , which is unique to Betacoronavirus (Woo et al., 2010 ). An alternative TRS motif (5 -UCGAAC-3 ) was discovered preceding the E in β-GX2018.Four Alpha-CoVs of S. kuhlii in Guangxi Province Four alpha-CoVs were identified from S. kuhlii. The full-length genomes of strains α-GX2018A, B, C, and D were sequenced. The genome sequence similarity among these four CoVs, PEDV, and BtCoV/512 was examined by Simplot analysis (Figure 3A ) and pairwise alignment (Supplementary Table 2 ). The α-GX2018A, B, and C were closely related to a known CoV, BtCoV/512, identified in S. kuhlii of Hainan Province with 96.9-100% aa identity for ORF1b, ORF3, E, M, and N. The differences mainly occurred in S (93.4-96.1% aa identity) and ORFx (65.7-67.6% aa identity). There were also significant differences between α-GX2018A and B and BtCoV/512 at the front end of ORF1a (1000-2500 nt) with 81.3-82% nt identity. In addition, although the α-GX2018D was closely related to that of BtCoV/512 with 92.2-99.4% aa identity for ORF1a, ORF1b, ORF3, and M, it was unexpected that the aa identity of α-GX2018D and BtCoV/512 in S1 region was 39.7%, which was lower than that of α-GX2018D and PEDV (42.7% aa identity). Although the aa identity of α-GX2018D and PEDV in S1 region was only 42.7%, we have not found any known viruses with higher identity than this.A novel alpha-CoV (α-YN2018) was identified in R. affinis of Yunnan Province. The genome sequence similarity among α-YN2018, BtRf-alphaCoV/HuB2013 (α-HuB2013), Hipposideros bat CoV HKU10 (Hi-batCov HKU10), Rousettus bat CoV HKU10 (Ro-BatCoV HKU10), and Cardioderma bat coronavirus/Kenya/KY43/2006 (Ca-batCov/Kenya) was examined by Simplot analysis (Figure 3B ) and pairwise alignment (Supplementary Table 3) . This virus showed low sequence similarity with any known alpha-CoVs. α-YN2018showed the highest sequence similarity to α-HuB2013 of Rhinolophus ferrumequinum with 63.9-92.4% aa identity in ORF1a, ORF1b, and N; to Hi-batCov HKU10 with 50-85.7% aa identity in S, M, and ORF7b; to Ca-batCov/Kenya in ORF3a with 60.6% aa identity; and to Ro-BatCoV HKU10 in E with 74.3% aa identity. The sequences of ORF7a from α-YN2018 showed <25% aa identity with those of the afore-mentioned known CoVs. In addition, we did not find any sequences that were similar to ORF3b and ORF3c of α-YN2018 from known viruses. According to the ICTV criteria, CoVs that share <90% aa sequence identity in the conserved replicase domains are considered to belong to different species; thus, α-YN2018 could be considered a new species under the genus Alphacoronavirus.We obtained five beta-CoVs from Rhinolophus spp. from Yunnan and Sichuan Provinces, which were named as β-YN2018A, β-YN2018B, β-YN2018C, β-YN2018D, and β-SC2018. All of a, These four representative bat CoVs are α-GX2018D, α-YN2018, β-YN2018B and β-GX2018, respectively. b, For putative ORFs, we aligned the TRS that preceded the start codon AUG with the leader TRS. The core sequence is indicated with underscores in bold type. The start codons of genes are in bold blue.these beta-CoVs had high sequence identity with SARSr-CoVs. These five SARSr-CoVs showed 92-97% nt identity with human SARS-CoV SZ3. The genome sequence similarity among the five SARSr-CoVs and SARS-CoV SZ3 strain was examined by Simplot analysis (Figure 3C) . These five SARSr-CoVs were highly conserved and shared a uniformly high sequence similarity to SARS-CoV with 94.2-100% aa identity in ORF1a, ORF1b, E, M, and N (Supplementary Table 4 ). There were some differences between the SARSr-CoVs and SARS-CoV SZ3 in the coding regions of ORF3a, ORF3b, ORF7a, and ORF7b. After further analysis, the SARS-CoV SZ3 showed the highest sequence similarity to β-YN2018B with 92-96% aa identity in S, ORF3a, and ORF3b; and to β-YN2018A with 93.2-95.9% aa identity in ORF7a and ORF7b. In contrast, considerable genetic diversity was shown in the S (72.9-92.4% aa identity) and ORF8 (32.6-34.1% aa identity) among the five SARSr-CoVs and SARS-CoV SZ3 strain. β-YN2018B shared 98.8% aa identity with SARSr-CoV WIV1, which was higher than that of the other four SARSr-CoVs (78.8-79.1% aa identity) (Figure 3C and Supplementary Table 4) . Remarkably, WIV1 and WIV16 (or Rs4874) were identified between 2012 and 2013, and they could use the human angiotensin converting enzyme II (ACE2) receptor for entry, like SARS-CoV (Ge et al., 2013; Yang et al., 2015) .ORFx was also found in the genomes of β-YN2018B and β-YN2018D. ORFx of β-YN2018B and β-YN2018D both shared 98.8% aa identity with WIV1. Except for ORF3b (95.6% aa identity), β-YN2018B was highly conserved and shared a uniformly high sequence similarity to SARS-CoV WIV1 in every ORF (98.2-100% aa identity) (Supplementary Table 4 ).A novel beta-CoV, β-GX2018, was identified from this bat species. The genome sequence similarity among β-GX2018 and four known CoVs, BatCoV HKU9-4, BatCoV HKU9-10-1, Ro-BatCoV HKU9, and BatCoV Philippines/Diliman was examined by Simplot analysis (Figure 3D ) and pairwise alignment (Supplementary Table 5 ). The full-length genome sequence of this virus had sequence similarity to that of HKU9related CoVs of Rousettus bats and GCCDC1-related CoVs of Eonycteris bats (Woo et al., 2007; Huang et al., 2016) , but the sequence identity between them was low. The sequence similarity between β-GX2018 and HKU9-and GCCDC1-related CoV s were 51-90.8% aa identity in ORF1a, ORF1b, S, ORF3, E, M, and N. Some partially sequenced genome segments of a recently reported CoV, Philippines/Diliman1525G2/2008 (BatCoV Philippines/Diliman) (Watanabe et al., 2010) , identified in C. brachyotis from the Philippines, showed the highest aa sequence identity with the β-GX2018 in M (96% aa identity), N (91% aa identity), ORF7a (80% aa identity), ORF7b (84.3% aa identity), and ORF7c (80.8% aa identity) (Watanabe et al., 2010) . According to the ICTV criteria, β-GX2018 could be considered a new species under the genus Betacoronavirus.To determine further the evolutionary relationships between these CoVs, phylogenetic trees were constructed using the aa sequences of RdRp, S, S1, E, M and N (Figure 4) . As we have speculated before, α-GX2018A, α-GX2018B, α-GX2018C, α-GX2018D and α-YN2018 clustered with the genus Alphacoronavirus, and β-SC2018, β-YN2018A, β-YN2018B, β-YN2018C, β-YN2018D, and β-GX2018 clustered with the genus Betacoronavirus. The results of the phylogenetic analyses were consistent with those of the sequence identity FIGURE 4 | Phylogenetic trees based on aa sequences of RdRp, S, S1, E, M, and N. The trees were constructed by the maximum likelihood method using appropriate models (WAG + G + I for RdRp and N; LG + G + I for S, S1, and E; rtREV + G for M) with bootstrap values determined by 1000 replicates. Only bootstraps >40% are shown. The scale bars represent 0.05 (RdRp), 0.5 (S and S1), and 0.2 (E, M, and N), respectively.analyses, and confirmed that the identified CoVs could be divided into four lineages.Lineage E α-GX2018A, α-GX2018B and α-GX2018C clustered with BtCoV/512, and their branches were short, reflecting the high sequence similarities. In addition, phylogenetic analysis of the S protein supported our previous analysis that α-GX2018D represented an independent clade between PEDV and BtCoV/512. Further analysis of S1 revealed that α-GX2018D represented a separate evolution distant from all other members of lineage E. From the length of the branch, α-GX2018D had a closer relationship with PEDV, which means that α-GX2018D may have undergone recombination with the ancestors of PEDV and BtCoV/512.α-YN2018 clustered with Hi-batCov HKU10, Ro-BatCoV HKU10, Ca-batCoV/Kenya, and α-HuB2013. However, α-YN2018 had closer relationships with α-HuB2013 of R. ferrumequinum in RdRp and N, and closer relationships with HKU10 of Hipposideros and Rousettus bats in S and M. This means that recombination may have occurred between α-YN2018 and these three related CoVs.β-SC2018, β-YN2018A, β-YN2018B, β-YN2018C, and β-YN2018D clustered with SARS-and SARSr-CoVs under lineage B. In RdRp, E, M, and N trees, these novel SARSr-CoVs and SARS-CoVs had short branches, which indicated that they were closely related in these regions. Phylogenetic analysis of S and S1 revealed that β-YN2018B was closer to SARS-CoV SZ3 and SARS-CoV Tor2 as well as WIV1 and WIV16.Lineage D β-GX2018 of C. sphinx clustered with HKU9-related CoVs of Rousettus and GCCDC1-related CoVs of Eonycteris under lineage D; however, β-GX2018 always represented a separate lineage distinct from other members of lineage D. Although BatCoV Philippines/Diliman of C. brachyotis had a closer relationship with α-YN2018 in M and N, it was only partially sequenced, so we could not analyze its evolutionary relationship with α-YN2018 in RdRp, S, and E, which hindered further analysis.We found that recombinant events had occurred among α-GX2018D, other bat CoVs, and PEDV of lineage E. α-GX2018D showed the highest degree of similarity to BtCoV/512 in the ORF1ab region, and the highest degree of similarity to PEDV in the S1 region. Evolutionary analysis showed that the S of α-GX2018D may have acted as the common ancestors of BtCoV/512 and PEDV. We analyzed possible recombination events in lineage E using SimPlot software. Similarity plot and bootscan results demonstrated that the S-N region of the α-GX2018D had the highest degree of similarity to PEDV, and complicated recombination may have happened between bat CoV and PEDV in the ORF1ab region ( Figure 5A) .For CoVs of lineage F, at least two recombination events had occurred ( Figure 5B) . Similarity plot and bootscan analysis showed that the S1 subunits of the α-YN2018 had the highest degree of similarity to Ro-BatCoV HKU10, while their S2 subunits had the highest degree of similarity to Hi-batCov HKU10 (Fig 5B) . The complicated recombination history between these alpha-CoVs suggests frequent gene transfers, especially of S, among different CoVs, which may be the result of the cross-species transmission of these CoVs.The identification of SARSr-CoV in bats in 2005 first established a genetic relationship between bats and human SARS-CoVs (Lau et al., 2005; Li et al., 2005) . After that, in the following 13 years, the constant discovery of bat SARSr-CoVs suggested the risk of re-emergence of SARS-CoV originating from bat species. Other than that, two CoV-related cases, human MERS in 2012 and SADS in 2017, which had a great impact on health and the economy were identified as being of bat origin (Omrani et al., 2015; Zhou et al., 2018) . In our previous study, a largescale virome analysis was conducted between 2010 and 2013 to understand the ecological diversity of bat viruses and the bat origin of emerging infectious diseases. Rhinolophus bats in Yunnan and Guangxi Provinces have provided genetic clues to the origin of SARS-CoV, and a SADS-CoV-and HKU2related CoV, BtRf-alphaCoV/YN2012, has also been found in Yunnan Province (Wu et al., 2016a) . Sichuan Province has diverse wildlife resources and the largest pig breeding in China, and a MERS-related CoV was reported in bats in this province in 2013 (Yang et al., 2014) . It is necessary for us to conduct continuous surveillance for the prevalence, genetic diversity and geographical distribution of CoVs in some bat species in these potential high-risk regions. Using virome analysis, our study investigated the presence and genetic diversity of CoVs circulating in bat species from Sichuan, Yunnan, and Guangxi Provinces between 2016 and 2017.The close relationship between five alpha-CoVs in S. kuhlii in Guangxi Province and a previously reported CoV in S. kuhlii in Hainan Province reveals that this bat species from different geographic locations contained the same CoV species, but with distinct S proteins. The S1 region of α-GX2018D is more closely related to that of PEDV than that of any other bat CoV, and is phylogenetically located at the root of the lineage of PEDV and S. kuhlii CoVs. This finding suggests the presence of a much closer common ancestor of PEDV in bats CoVs. The recombination analysis supports that the S region of PEDV may come from the ancestor of α-GX2018D. Considering that bat species such as S. kuhlii prefers to circulate around farmhouses and pigsties, and other PEDV-related sequences have been reported in bats in previous reports (Simas et al., 2015; Phan et al., 2018) , the relationship between bat CoVs and PEDV should be further investigated.The discovery of β-SC2018 provides evidence that Rhinolophus in Sichuan Province could harbor SARSr-CoVs, and reveals a broader geographical distribution of these viruses. The discovery of β-YN2018B here, combined with previously reported WIV1 and WIV16 (Ge et al., 2013; FIGURE 5 | Detection of potential recombination events by bootscan analysis. (A) Full-length genome sequence of α-GX2018D was used as a query sequence and BtCoV/512, α-GX2018A, and PEDV as reference sequences. (B) Full-length genome sequence of α-YN2018 was used as a query sequence and Ro-BatCoV HKU10, Hi-batCov HKU10, and α-HuB2013 as reference sequences. All analyses were performed with a Kimura model, a window size of 1000 base pairs, and a step size of 100 base pairs. The gene map of query genome sequences is used to position breakpoints. Yang et al., 2015) , revealed that these ACE2-adaptable SARSr-CoVs were continuously circulating in R. affinis of Yunnan Province at least from 2012 to 2016. These findings emphasize again the importance of continuous longitudinal monitoring for SARSr-CoVs in Rhinolophus over a wide area. Although no SADS and HKU2-related CoV were found in Rhinolophus bats in our study, the surveillance of this CoV type in bats is still needed in the future.In our previous study, by the identification of correlated alpha-CoVs in different Miniopterus species from multiple regions, the characteristics of co-infection, recombination, and hostshifting for alpha-CoVs have been described. Here, the alpha-CoVs of lineage F found in more diverse bat species (including frugivorous and insectivorous bats) extends the known host range of these viruses. However, the recombination events found between α-YN2018, Ro-BatCoV HKU10, and Hi-batCov HKU10, and the multiple origins of the S of α-YN2018, further indicate that a more distant host-shifting in accordance with the recombination of the S region may have happened in the evolutionary history of these viruses among diverse host species.When we collected S. kuhlii in Guangxi Province, we found that there were many C. sphinx at the sampling site. The Pteropodidae bats (C. sphinx bat belongs to Pteropodidae) are natural hosts of several emergent human pathogens such as Hendra virus, Nipah virus, and Ebola virus (Leroy et al., 2005; Halpin et al., 2011; Sendow et al., 2013; Mari Saez et al., 2015; Paez et al., 2017 ). An Ebola-related filovirus was found in the Chinese Pteropodidae bats recently (Yang et al., 2019) , and thus, we collected samples of C. sphinx for virome analysis. The C. sphinx beta-CoV, β-GX2018, represents an evolved independent lineage different from HKU9 of Rousettus and GCCDC1 of Eonycteris under lineage D. Based on the partially sequenced region, CoV strain Philippines/Diliman1525G2/2008 previously identified in C. brachyotis from the Philippines, could also be clustered into this separate β-GX2018 clade. By providing full-length sequence data, this finding reveals a new host range of this virus in an unreported location.YH, QJ, and ZW conceived the experiments, analyzed the results, and wrote the manuscript. YH, JD, HS, and ZW conducted the experiments and analyzed the results. JZ, GZ, and SZ collected the specimens. All authors reviewed the manuscript. \n",
            "\t0.073\tThe coronaviruses (CoVs) are enveloped viruses of animals and humans associated mostly with enteric and respiratory diseases, such as the severe acute respiratory syndrome and 10-20% of all common colds. A subset of CoVs uses the cell surface aminopeptidase N (APN), a membrane-bound metalloprotease, as a cell entry receptor. In these viruses, the envelope spike glycoprotein (S) mediates the attachment of the virus particles to APN and subsequent cell entry, which can be blocked by neutralizing antibodies. Here we describe the crystal structures of the receptor-binding domains (RBDs) of two closely related CoV strains, transmissible gastroenteritis virus (TGEV) and porcine respiratory CoV (PRCV), in complex with their receptor, porcine APN (pAPN), or with a neutralizing antibody. The data provide detailed information on the architecture of the dimeric pAPN ectodomain and its interaction with the CoV S. We show that a protruding receptorbinding edge in the S determines virus-binding specificity for recessed glycan-containing surfaces in the membrane-distal region of the pAPN ectodomain. Comparison of the RBDs of TGEV and PRCV to those of other related CoVs, suggests that the conformation of the S receptor-binding region determines cell entry receptor specificity. Moreover, the receptorbinding edge is a major antigenic determinant in the TGEV envelope S that is targeted by neutralizing antibodies. Our results provide a compelling view on CoV cell entry and immune neutralization, and may aid the design of antivirals or CoV vaccines. APN is also considered a target for cancer therapy and its structure, reported here, could facilitate the development of anti-cancer drugs.The Coronaviridae is a large family of enveloped, plus-RNA viruses. They are involved in respiratory, enteric, hepatic and neuronal infectious diseases in animals and humans that lead to important economic losses [1, 2] , as well as to high mortality rates in severe acute respiratory syndrome CoV (SARS-CoV) infections . The CoVs are a numerous group of Coronaviridae. They have been clustered in the Coronavirinae subfamily, which includes three approved genera, Alpha-, Betaand Gammacoronavirus, as well as a tentative new genus, the Deltacoronavirus .Representative CoV species in each genus are Alphacoronavirus 1 (comprising transmissible gastroenteritis virus (TGEV), porcine respiratory CoV (PRCV) and related canine and feline CoVs), Human coronavirus (HCoV-229E and HCoV-NL63, genus Alphacoronavirus), Murine coronavirus (including mouse hepatitis virus (MHV), genus Betacoronavirus, cluster A), Severe acute respiratory syndrome-related coronavirus (SARS-related CoV, genus Betacoronavirus, cluster B), Avian coronavirus (including infectious bronchitis virus (IBV), genus Gammacoronavirus), and Bulbul-CoV (tentative genus Deltacoronavirus) .CoV particles display characteristic large surface projections or peplomers (17-20 nm) comprised of homotrimers of the spike glycoprotein (S), a type I membrane protein [1, 5] . The peplomers have a globular portion connected by a protein stalk to the transmembrane domain . The globular region is formed by the N-terminal S1 region, whereas the stalk corresponds to the membrane-proximal S2 region, which mediates virus fusion to host cells and adopts a helical structure characteristic of class I virus fusion proteins . Determinants of CoV tropism locate at the S1 region [1, 8] , which mediates attachment of CoV particles to cell surface molecules, initiating virus entry into cells and infection. There is considerable variability in receptor usage among the CoVs. Most Alphacoronavirus such as TGEV and HCoV-229E use APN [9, 10] , whereas the related HCoV-NL63 uses a distinct cell entry receptor, the human angiotensin converting enzyme 2 (ACE2) ; SARS-CoV also recognizes the ACE2 receptor . SARS and NL63 CoV bind to common regions of the ACE2 protein, although the structures of their receptor-binding domains (RBDs) are quite distinct [11, 13] . MHV uses the cell adhesion molecule CEACAM1a ; a recent crystal structure showed that the MHV RBD adopts a galectin-like fold . The use of alternative receptors that confer extended tropism has been described for SARS-CoV, MHV and TGEV [1, 8] .The mammalian APNs (CD13) are type II cell surface metalloproteases whose large glycosylated ectodomain has a zinc metal ion at the active site . APN is linked to many cell functions, leading it to be termed the ''moonlighting enzyme'' . Animal models confirmed a role for this cell surface enzyme in angiogenesis . Peptides and inhibitors that target APN showed a link between this protein and tumor growth and invasion [17, 18] . APN is a target for cancer chemotherapies; drugs that bind this protein have been developed to treat tumors, some of which are in clinical trials . As mentioned above, APN is also a major CoV cell entry receptor [1, 9, 10] . CoV recognition of APN is species-specific, and specificity is associated with N-linked glycosylations in the APN protein .Cell tropism and immune neutralization have been extensively studied in some porcine Alphacoronavirus, such as the enteropathogenic TGEV and porcine respiratory CoV (PRCV), a nonenteropathogenic virus derived from TGEV . Both viruses use porcine APN (pAPN) for cell entry. The APN-binding domain in TGEV, PRCV and other Alphacoronavirus locates at the C-terminal portion of the S1 region [8, 22, 23] , which bears epitopes recognized by CoV-neutralizing antibodies [22, 23, 24, 25] . Most TGEV-neutralizing antibodies cluster at antigenic site A [25, 26] , comprised within the RBD at the S1 region ( Figure 1A ) ; the other antigenic sites defined in the TGEV S1 region (B through D) are outside the RBD ( Figure 1A ) .To date, there is no structural information available on antibody neutralization and APN recognition by Alphacoronavirus. We determined crystal structures of the PRCV RBD in complex with the pAPN ectodomain, and the TGEV RBD in complex with the neutralizing monoclonal antibody (mAb) 1AF10 . The RBD adopts a b-barrel fold, with a distinct protruding tip engaged in pAPN recognition. The structures show how these porcine Alphacoronavirus recognize its cell entry pAPN receptor and how immune neutralization of these CoVs is achieved by antibody targeting of receptorbinding residues in the S protein. The mechanisms used by TGEV to escape immune neutralization and the evolution of receptor recognition in the CoV family are discussed.APN-binding domain and epitopes for neutralizing mAbs overlap in TGEV and PRCV S proteins APN receptor recognition and envelope S antigenicity are well documented in TGEV and related PRCV. The pAPN-binding Figure 1 . APN-binding domain and epitopes for neutralizing mAbs overlap in TGEV and PRCV S proteins. A. Scheme of TGEV and PRCV S proteins showing the S1, S2, transmembrane (T) and cytoplasmic (Cy) regions. Location of the C, B, D and A antigenic sites , and the pAPN RBD (bar with N and C-terminal residues) are shown. Length is indicated for mature S1 regions. B. A short, soluble S protein variant containing the TGEV RBD region binds to cell surface pAPN. Binding of a bivalent SA-Fc fusion protein (SA) to BHK-pAPN (open histograms), alone (left) or in the presence of the site A-specific 6AC3 mAb (right), as analyzed in FACS. Filled histograms correspond to an unrelated Fc fusion protein. C. Binding of site A-specific TGEVneutralizing mAb to the SA protein. mAb binding to plastic-bound SA protein, monitored by optical density (OD). Site A mAbs are specific for the Aa (1BB1), Ab (1DE7) and Ac (6AC3, 1AF10) subsites . An anti-HA mAb that binds to the HA tag in the SA protein was used as control. D. Site A-specific mAbs prevent SA protein binding to pAPN. Binding of the SA protein to BHK-pAPN cells in panel B was monitored alone or in the presence of site A (shown in C) or site D-specific (1DG3) mAb, and the binding ratio determined (see Materials and Methods). C, D. Mean and standard deviation for three experiments. doi:10.1371/journal.ppat.1002859.g001The cell surface aminopeptidase N (APN), a membranebound metalloprotease target for cancer therapy, is a major cell entry receptor for coronaviruses (CoVs), agents that cause important respiratory and enteric diseases. In some CoVs, the virus envelope spike glycoprotein (S) mediates attachment of the virus particles to the host APN protein and cell entry, which is blocked by antibodies that prevent CoV infections. The crystal structures of the S proteins of two porcine CoV in complex with the pig APN (pAPN) or with a neutralizing antibody shown here, reveal how some CoV bind to its cell surface APN receptor and how antibodies prevent receptor binding and infection. The report uncovers a unique virus-receptor recognition mode that engages a glycan N-linked to the pAPN ectodomain, revealing structural determinants of the receptor-binding specificity in CoVs. Neutralizing antibodies target viral residues used for binding to the APN receptor and entry into host cells, showing that efficient CoV neutralization requires immune responses focused toward key receptor binding motifs in the virus envelope. These structural insights, together with the structure of the APN ectodomain, provide a compelling view of relevant cell membrane processes related to infectious diseases and cancer.domain was mapped within residues 506 to 655 of the mature TGEV S polypeptide , whereas TGEV mAb-resistant (mar) mutants defined four antigenic sites (C, B, D and A) [24, 26] ( Figure 1A) . Antigenic sites C and B are not present in the PRCV S protein. Antigenic site A determinants are located within the pAPN-binding domain at the C-terminal moiety of the TGEV and PRCV S1 regions ( Figure 1A ) [21, 22] .We recently reported the modular dissection of the N-terminal S1 region of TGEV and PRCV, and the preparation of soluble S1 length variants with single antigenic sites . We produced a recombinant short S protein fragment termed SA, which comprises only residues 481 to 650 of the TGEV S protein that binds cell surface pAPN ( Figure 1B ) and displays conformational epitopes for the three antigenic A subsites (Aa, Ab, and Ac) ( Figure 1C ). Antibodies clustered at the Aa (1BB1), Ab (1DE7) and Ac (1AF10 and 6AC3) subsites blocked binding of the soluble SA protein to pAPN ( Figure 1D ). The SA protein therefore includes the pAPN-binding domain of TGEV and epitopes for site Aneutralizing mAb. We applied X-ray crystallography to S protein variants containing the RBD of the related TGEV and PRCV, and have identified how these Alphacoronavirus bind to the cell surface pAPN and its inhibition by neutralizing antibodies.We attempted crystallization of the soluble pAPN-binding SA protein derived from the TGEV S, alone and in complex with several neutralizing mAbs. Crystals were prepared with the SA protein in complex with the Fab fragment of the 1AF10 mAb ; the structure of the complex was determined and refined using diffraction data extending to 3.0 Å resolution (Materials and Methods; Table 1 ). The asymmetric unit of the crystals contains two antibody-RBD complexes, one of which is shown in Figure 2 . Residues Pro507 to Val650 of the TGEV S protein, previously identified as the pAPN-binding domain ( Figure 1A ) , were well defined in the crystal structure. They folded in a single domain structure, the RBD of TGEV (Figure 2A) . The RBD adopts a bbarrel fold formed by two b-sheets with five b-strands each (scheme in Figure S1A ). N-and C-terminal ends are on the same side of the domain (terminal side), which presumably lies close to other S protein domains; at the opposite side, two b-turns (b1-b2 and b3-b4) form the tip of the barrel (Figure 2A) , where the mAb binds to the RBD.The immunoglobulin (Ig) variable domains of the mAb heavy (V H ) and light (V L ) chains contact the b1-b2, b3-b4 and b5-b6 regions of the TGEV RBD ( Figure 2B ), burying a virus protein surface of ,810 Å 2 . The buried surface of the 1AF10 mAb is ,750 Å 2 , with equal contribution by the V H (51%) and V L (49%) Ig domains. Complementarity determining regions (CDR) of the antibody heavy (H3) and light (L1 and L3) chains, the N-terminus of the light chain and the C, C9 and C0 b-strands of the V H domain contact the viral RBD tip ( Figure 2B ). The CDR-H3 of the 1AF10 mAb is relatively long, with two-residue insertion (Tyr103 H and Asp104 H ) relative to other homologous H3 loops in reported mAb structures (Table S1 ).The RBD b1-b2 hairpin with Tyr528 at its tip is at the center of the interacting surface and penetrates between the V L and V H Ig domains of the 1AF10 mAb ( Figure 2B and 2C). Similar antibodyantigen recognition is described for some peptides and is common for small hapten molecules [28, 29] . The RBD b1-b2 region contributed 73% of the RBD surface buried by the 1AF10 mAb, and docked between the 1AF10 mAb variable domains ( Figure 2B ). The b-turn is fully buried between the mAb Ig domains ( Figure 2C ), forming a contact network with mAb residues ( Figure 2D ). The RBD residue Tyr528 at the bottom of the pocket contacts mAb residues Trp47 H and Tyr107 H , whereas its hydroxyl group is hydrogen bonded to the side chain of Gln89 L and main chain carbonyl of Tyr107 H ( Figure 2C and 2D). These structural findings on 1AF10 recognition of the RBD b1-b2 region correlate with 1AF10 mAb binding to peptides (MKRSGYGQPIA533) that include this hairpin region .The RBD b3-b4 and b5-b6 regions are at the periphery of the epitope ( Figure 2B ); their contribution to interaction with 1AF10 is smaller than that of the b1-b2 region, representing respectively ,17% and 10% of the RBD surface buried by the mAb. They contact either the V L or V H Ig domains ( Figure 2B ). RBD residues Leu570 and Trp571 at the b3-b4 loop contact the N-terminus, CDR-L1 and CDR-L3 of the V L domain, whereas the b5-b6 loop contacts the long CDR-H3 loop ( Figure 2B and 2C).To characterize CoV attachment to its APN receptor, we attempted crystallization of the pAPN ectodomain in complex with TGEV and PRCV S protein variants comprising their RBDs (Materials and Methods). Crystals were obtained only with a mixture of a PRCV S protein (S3H) and the pAPN. Using these crystals, we determined the structure of the PRCV RBD-pAPN complex by molecular replacement using previously solved structures of the TGEV RBD shown in Figure 2 (97% sequence identity) and of the pAPN ectodomain (Materials and Methods and Table 1 ). The asymmetric unit of the crystals contained two macromolecular RBD-pAPN complexes ( Figure 3A ). The PRCV RBD adopts a b-barrel fold like the TGEV RBD ( Figure S1 ). Each pAPN molecule was engaged by the tip of a single PRCV RBD molecule, which bears two exposed aromatic residues (Tyr and Trp) ( Figure 3A , in red), and they bound to a membrane-distal region of the pAPN ectodomain ( Figure 3A ). The RBD N-and Cterminal ends and the remaining CoV S are also distant from the pAPN, and are unlikely to contact the receptor molecule. Based on a cryo-EM structure of the SARS-CoV S , the RBD must be also at the viral-membrane distal side of the S and therefore, the receptor binding edge must be accessible for CoV binding to the APN receptor.The pAPN is a type II membrane protein and the N-terminal end of the ectodomain must be near the cell membrane ( Figure 3A ). The 25 N-terminal residues of the crystallized pAPN ectodomain are largely disordered in the structure and they might form a flexible region close to the cell membrane. The pAPN ectodomain is composed of four domains ( Figure 3A ). Domain I (orange) is made of b-strands, domain II (yellow) adopts a thermolysin-like fold bearing a zinc ion at the catalytic site, domain III (red) is a small b-barrel domain, and the C-terminal domain IV (green) is composed of alpha-helices (domain boundaries are shown in Figure S2 ). The pAPN molecule structure is closely related to that of the human endoplasmic reticulum aminopeptidase-1 [30, 31] (root-mean-square deviation of 2.3 Å for 791 residues sharing 33% sequence identity, based on DALI server). Domain II bearing the enzyme active site is the most related domain (47% identity), whereas domain IV is the most distinct (22% identity). The zinc ion is coordinated to conserved residues at the pAPN active site in domain II ( Figure S2 ). The active site conformation is similar to that of other aminopeptidases ( Figure S3 ). The pAPN crystallized in complex with the PRCV RBD had an open conformation [30, 31, 32] , in which domain IV was ,20-25 Å from domains I and II; this creates a central cavity in which the zinc ion at the catalytic site is highly accessible ( Figure 3A ).The mammalian APNs are cell surface metalloproteases that form membrane-bound dimers . The crystallized pAPN ectodomain also behaved as a dimer in solution ( Figure S4 ). The pAPN dimeric assembly showed in Figure 3A buried a large accessible surface (,980 Å 2 ) in each monomer. The dimerization surface comprises 29 residues spread across domain IV, which are distinct from those recognized by CoV ( Figure S2 ). Similar dimeric assemblies were observed in two crystal structures determined for the pAPN ectodomain alone (not shown), crystallized using distinct conditions. The pAPN molecular assembly shown here might thus be representative of the dimer described for mammalian APN on membrane surfaces .In the crystals of the PRCV RBD-pAPN complex, the RBD tip contacts a membrane-distal region of the pAPN ectodomain ( Figure 3A ). The conformations of the receptor-binding loops (b1-b2 and b3-b4) at the tips of the two PRCV b-barrel domains in the structure are identical ( Figure S1B ), suggesting very similar RBD-pAPN interactions in both complexes of the asymmetric unit. The virus-receptor interaction buried ,870 Å 2 of the virus protein, 60% of which corresponded to the b1-b2 region ( Figure 3B ) and 30% to the b3-b4 turn ( Figure 3C ). The size of the pAPN surface buried by the RBD was similar (,770 Å 2 ), and included pAPN residues ranging from alpha helix 19 (a19) to 22 (a22) in domain IV, and a few domain II residues ( Figure S2 , Table S2 ).The end of the pAPN helix a19 and helix a21 contacted the b1-b2 region of the RBD ( Figure 3B ). The Tyr side chain (Tyr528 in TGEV), which protrudes at the b-turn in PRCV and TGEV RBDs ( Figure 3B and 3D) , is almost fully buried in the complex, locating between the first N-acetyl glucosamine (NAG7361) linked to pAPN Asn736, the end of helix a19, and the first half of helix a21 ( Figure 3B ). The hydroxyl group of the RBD Tyr528 was hydrogen bonded to side chains of pAPN residues Glu731 and Trp737, and contributed to virus-receptor binding specificity. The preceding RBD Gly527 residue was at the pAPN proximal side of the b-turn, hydrogen bonded to the pAPN Asn736 main chain; at the opposite side, the RBD Gln530 side chain formed a network of hydrogen bond interactions with pAPN NAG7361 and Asn736 side chain ( Figure 3B ). The N-acetyl moiety of the glycan also interacted with RBD residues at the b2 and b6 strands ( Figure 3B , Table S2 ). The pAPN N-linked glycan and surrounding residues that contact the CoV RBD b1-b2 region in the structure were identified as one of the APN determinants of the CoV host range .The second relevant virus-receptor interacting region engaged a b-turn at the beginning of the RBD b3-b4 loop ( Figure 3C and 3D). The unique RBD Trp571 residue, which protrudes at the turn, docked in a pAPN cavity formed by the coils that precede helices a22 in domain IV and a5 in domain II ( Figure 3C and S2). The bulky side chain of the RBD Trp571 residue packed against pAPN residues His786 and Pro787, and its imino group was hydrogen bonded to the main chain carbonyl of Asn783 ( Figure 3C ). The RBD Trp571 as well as the RBD Tyr528 at the b-barrel tip in TGEV and PRCV appear to be central residues in the virus-receptor interaction, as they contact with many pAPN residues and contribute also to binding specificity by mediating polar interactions with the pAPN (Table S2) .To confirm the contribution of the PRCV or TGEV RBD bbarrel tip in pAPN receptor recognition, we analyzed binding of wild type and mutant TGEV RBD proteins to cell surfaceexpressed pAPN ( Figure 4A ). Mutations in the three regions (b1-b2, b3-b4 and b5-b6) that build the receptor binding edge of the b-barrel decreased RBD binding to pAPN, whereas mutations outside the receptor-binding region (V617Ngly) had no effect on receptor recognition. Deletion of the pAPN Asn736 glycosylation site also abolished TGEV RBD binding to cell surface-expressed pAPN ( Figure 4B ). Deletion of the homologous glycan in feline APN similarly prevents cell infection by feline, canine and porcine CoVs, all of which share the glycan-binding Tyr residue in the b1-b2 turn (see below), whereas addition of this glycan to human APN is sufficient to render it a TGEV receptor .We determined the crystal structures of the related TGEV and PRCV RBDs bound to two distinct ligands. The RBDs adopt bbarrel structures with small differences in the ligand binding loops (Figures S1). In the RBD, each of the two highly twisted b-sheets that build the b-barrel is formed by five b-strands ( Figure 5A ). The bent b-strand 5 (b5) crosses both b-sheets and has a b-bulge at Asn608 ( Figure 5A , magenta). At one side of the b-barrel, all bstrands are antiparallel ( Figure 5A, cyan) , whereas on the opposite A DALI search of structural homologs showed the greatest similarity (Z score of 10) with the RBD of the ACE2 receptorbinding HCoV-NL63 (root-mean-square deviation of 2.4 Å for 103 residues), the other Alphacoronavirus RBD whose structure is known . The cores of the TGEV and HCoV-NL63 b-barrel domains are structurally similar, but the loops at the tips ( Figure 5B and 5D). The tip region of the HCoV-NL63 RBD is the ACE2 receptor-binding edge and has a ''bowl''-shaped conformation ( Figure 5C ) that differs from the TGEV RBD protruding edge. Aromatic residues protrude from the b-turns at the tip of the bbarrel in TGEV, whereas they are partially buried at the center of the ''bowl''-shaped edge in HCoV-NL63 ( Figure 5B and 5C). The distinct RBD tip conformation in ACE2-binding HCoV-NL63 and in APN-binding TGEV might be a determinant of their distinct cell entry receptor specificities.The degree of sequence identity in the RBD region among members in the species Alphacoronavirus 1 (,90% identity) suggests a structure closely related to that of TGEV, including conforma-tion of the receptor-binding loops (b1-b2 and b3-b4) at the bbarrel tip ( Figure 6 ). Therefore, TGEV, PRCV, CCoV and FCoV must recognize the APN receptor in similar fashion. In contrast, the receptor-binding loops at the tip appear to have a different conformation from TGEV in the HCoV-229E RBD, which also binds to the APN. In this CoV, the b1-b2 region has two Cys, as in HCoV-NL63, and lacks the APN-binding Tyr residue in Alphacoronavirus 1, although it preserves the two Gly residues found in the TGEV b-turn ( Figure 6 ). The b3-b4 loop in HCoV-229E is markedly shorter than in TGEV, but it also has a Trp residue. Sequence identities between the RBD of TGEV and IBV (Gammacoronavirus) or the Bulbul-CoV (tentative Deltacoronavirus) are relatively large (,25%), and similarities are found mostly in bstrands and at the RBD C-terminal half ( Figure 6) . These data indicate a conserved RBD fold between Alphacoronavirus and Gammaor Deltacoronavirus. There is less sequence similarity between the Alphaand Betacoronavirus RBD regions (,10%), which correlates with notable structural differences between their Detail of the RBD b1-b2 region with the exposed Tyr residue interacting with the pAPN. Side chains of RBD and pAPN residues engaged in the interaction are shown as sticks with carbons in magenta or green, respectively. NAG7361 glycan N-linked to pAPN Asn736 is shown with carbons in yellow and the electron density map, determined without the glycan, shown as a blue mesh contoured at 3 sigma. C. Detail of the RBD b3-b4 region with the Trp residue interacting with the pAPN. In B and C, RBD residues are numbered following the TGEV sequence shown in D, and intermolecular hydrogen bonds are shown as dashed red lines. D. Structure-based sequence alignment of the TGEV and PRCV RBDs. b-strands are marked with bars. TGEV sequence is numbered. In red, 1AF10 mAb-(for TGEV) and pAPN receptor-binding residues (for PRCV) identified by the structures. Residues absent in the RBD structures are in grey, and the thrombin recognition sequences at the end of recombinant porcine CoV RBDs are in lowercase letters. doi:10.1371/journal.ppat.1002859.g003RBDs [8, 11, 13] . The RBDs of the SARS and MHV Betacoronavirus adopt folds unrelated to the b-barrel shown for Alphacoronavirus.The most TGEV-neutralizing mAbs, including 1AF10, recognize antigenic site A in the S protein, divided into the Aa, Ab and Ac subsites . To further characterize site A antigenic determinants in the TGEV RBD, we mutated RBD residues targeted by the 1AF10 mAb ( Figure 2 ) and some surrounding residues, and analyzed binding to other site A-specific mAbs. The antigenicity of residues in the b1-b2 region, in the center of the epitope for 1AF10 ( Figure 2C ), was determined by monitoring mAb binding to RBD mutants with TGEV residue substitutions Gly527 (G527D), Tyr528 (Y528A) and Gly529 (G529D) ( Figure 7A ). All three substitutions abolished RBD binding by the Ac subsite-specific mAbs 1AF10 and 6AC3. The Y528A RBD mutant was recognized by Aa-(1BB1) and Ab-specific (1DE7) mAbs ( Figure 7A) , and mAb 1DE7 also bound the G529D mutant.In contrast to the antibody binding profile of the Y528A RBD mutant, Ala substitution of the TGEV Trp571 residue (W571A), a pAPN-binding residue in the b3-b4 loop at the periphery of the RBD epitope for 1AF10 ( Figure 2C ), did not affect binding by the Ac-specific mAbs (1AF10 and 6AC3), whereas RBD recognition by 1BB1 and 1DE7 mAbs was greatly reduced ( Figure 7A ). Deletion of the b3-b4 turn (LWD572A mutant) reduced 6AC3 mAb binding to the RBD markedly, with a partial reduction in 1AF10 binding ( Figure 7A ); this indicates that mAb 6AC3 recognizes a broader epitope, which correlates with its higher TGEV neutralization activity . Replacement with Ala of RBD residues Thr631 and Asn632 at the b5-b6 hairpin, which contacts the 1AF10 mAb in the RBD-1AF10 structure ( Figure 2C) , reduced binding by all site A-specific mAb ( Figure 7A ). This might be a result of a conformational effect induced on the nearby b1-b2 region of the RBD.Results for antibody binding to RBD mutants showed that site A epitopes extend across the TGEV RBD tip, although there are some differences among the three A subsites ( Figure 7B ). The epitopes recognized by Aa-and Ab-specific mAbs bear the exposed TGEV Trp571 residue at the b3-b4 loop, whereas epitopes for the Acspecific mAbs center on Tyr528 in the b1-b2 turn. None of the mAb tested simultaneously targeted the two aromatic side chains (Tyr and Trp) at the tip of the TGEV RBD that bind to the pAPN. Subsite-specific residues defined by mar mutants (Lys524 for Aa, Arg577 for Ab and Gly529 for Ac) might be located at the periphery of their respective epitopes ( Figure 7B ). Ab and Ac subsites appear to be relatively far apart, with the Aa epitope in an intermediate position. The RBD tip, shown here as the pAPN-binding edge of the domain (Figure 3) , is the main S protein determinant of antigenic site A, recognized by the most effective neutralizing antibodies of TGEV and related CoV infections [25, 26] .Here we show how a group of CoVs attaches to the cell surface APN metalloprotease for entry into host cells, and how some CoVneutralizing antibodies prevent infection. The RBD-receptor complex structures determined for Alphacoronavirus indicate that the conformation of the receptor binding edge in the envelope S proteins probably determines their receptor-binding specificity. The CoV that bind APN analyzed here have protruding receptorbinding motifs that engage recessed surfaces on the receptor. This mode of receptor recognition is essentially opposite to that reported for CoV binding to the ACE2 receptor, where recessed receptor-binding motifs in the viral RBD cradle exposed surfaces of the ACE2 ectodomain [11, 13] . In the case of pAPN, an Nlinked glycan is also engaged in the virus-receptor interaction. The inherent flexibility of this glycan might facilitate the initial contact of the CoV Tyr residue with APN amino acids, and subsequent virus-receptor interactions could lock the bound Tyr between the glycan and an a-helix ( Figure 3B ). The glycan N-linked to Asn736 in pAPN is also conserved in canine and feline APN proteins ( Figure S2 ), as are the viral S protein residues that interact with this glycan in the RBD b1-b2 and the b5-b6 regions ( Figure 6 ). This unique glycan-virus interaction must thus be conserved among the different CoVs in the species Alphacoronavirus 1, in accordance with the glycan requirement reported for cell infection by CCoV, FCoV, and TGEV/PRCV . The lack of this glycan in human APN ( Figure S2 ) and the absence of the interacting Tyr residue in the b1-b2 region of HCoV-229E RBD ( Figure 6 ) imply distinct virus-APN local contacts in humans. As shown for the Alphacoronavirus 1 group, however, HCoV-229E probably has a protruding receptor-binding edge in the envelope S, responsible for its APN-binding specificity.The structure of the RBD-1AF10 complex, together with structure-guided RBD mutagenesis and mAb binding data, demonstrated that the receptor-binding region is a major antigenic determinant in the envelope S protein of CoV that bind APN. Potent TGEV-neutralizing antibodies, such as the 6AC3 mAb , target key APN-binding residues in the S (Figure 7) , preventing infection. Data from antibody neutralization-resistant TGEV mar mutants nonetheless show that some substitutions can be accommodated in the receptor-binding region of Alphacoronavirus, which confer the ability to escape immune neutralization, while preserving . Substituted residues at the RBD tip that contact pAPN in the PRCV RBD-pAPN structure are shown in Figure 3 , except for the V617Ngly mutant, with a glycan at RBD position 617 in the b5-b5b loop, outside the RBD tip (see Figure 3D) . B. TGEV RBD binding to cell surface pAPN glycosylation mutants. Relative binding of the SA protein and the anti-HA mAb to HA-tagged pAPN proteins with (pAPN) or without the glycan linked to Asn736 (N736A and T738V). Mean and standard deviation for three experiments. doi:10.1371/journal.ppat.1002859.g004 the receptor-binding affinity necessary for cell entry [24, 26] . Our results thus demonstrate that the receptor-binding region in Alphacoronavirus is under selective pressure from the immune system, as described for other viruses [34, 35, 36, 37] . It is tempting to speculate that immune pressure on exposed receptor-binding residues in the CoV S could lead to conformational changes in receptor-binding edges of CoV RBDs. This would result either in changes in the APN-recognition mode observed with HCoV-229E and TGEV, or in conformational changes in the RBD tip that lead to a receptor specificity switch for cell entry, as observed for HCoV-NL63 . Virus use of recessed binding regions, as for HCoV-NL63, is a well-defined strategy for hiding conserved receptorbinding residues from antibodies [34, 36] . Like HCoV-NL63, SARS-CoV uses a recessed, although broader ACE2-binding surface, which can accommodate mutations that permit crossspecies receptor recognition . It remains to be understood why, despite major changes in the receptor-binding region, all these CoV use metalloproteases as cell entry receptors.In the course of our studies, we also determined the crystal structure of the cell surface APN, an important target for cancer therapies. The domain architecture of APN resembles that of related aminopeptidases [30, 31, 32] . Here we show a unique dimer configuration for the APN, mediated by its domain IV, the most divergent domain among M1 aminopeptidases . The implication of these structural findings for APN biology will require further biochemical analysis. Knowledge of the structure is leading to research on the mechanism of action of numerous anti-tumor compounds that target mammalian APN ; these studies will be fundamental for improving drug specificity. The detailed view of the APN-CoV interaction shown here might also lead to development of small molecules to block CoV infection. We have identified the receptor-binding region as the major antigenic site in the Alphacoronavirus envelope S, which could guide the design of immunogens that boost CoV-neutralizing immune responses to key motifs for virus cell entry.Design of soluble S proteins variants of TGEV and PRCV has been described . The SA protein containing the RBD of TGEV was derived from the SC11 strain, and contains residues b-strands are marked (bars) above or beneath their sequences. TGEV sequence is numbered. ACE2 receptor-binding residues reported for HCoV-NL63 , as well as pAPN receptor-binding residues for TGEV (Supplementary Table S2 ) are colored as in C. Residues absent in the RBD structures are in grey, and the thrombin recognition sequence at the end of the TGEV RBD is in lowercase letters. doi:10.1371/journal.ppat.1002859.g005 481 to 650 of the TGEV S, an N-terminal influenza hemagglutinin HA peptide, and either a FLAG mAb epitope (monovalent SA-Flag variant) or the human IgG1 Fc portion (bivalent SA-Fc variant) at the C-terminal end. The engineered soluble pAPN contains residues 36 to 963 (ectodomain) of the cell surface protein fused to HA and FLAG tags at the N and C terminus, respectively . The soluble S protein crystallized in complex with the pAPN was derived from the PRCV HOL87 strain (S3H in ), and contains the N-terminal 426 residues of the PRCV S protein and same C-terminus as the TGEV-derived SA protein . A recombinant membrane bound pAPN with an HA tag at the Cterminal end was engineered for cell surface expression. Thrombin recognition sequences were introduced between the tags and the viral or pAPN protein sequences.Proteins were produced in transiently transfected 293T or stably transfected CHO-Lec 3.2.8.1 (CHO-Lec) cells as described , and concentration in cell supernatants determined by ELISA. Proteins prepared in CHO-Lec cells were used in crystallization experiments. Hybridoma cells secreting the TGEV S mAbs were grown in DMEM supplemented with 10% FCS in roller bottles. Proteins secreted to culture supernatants were initially purified by affinity chromatography. All protein samples were further purified by size exclusion chromatography in HEPES-saline buffer (20 mM HEPES, 150 mM NaCl) pH 7.5.The Fab fragment of the 1AF10 mAb was prepared by papain digestion of the purified antibody. The reaction was terminated by the addition of E64 (Sigma) and the Fab fragment purified by size exclusion and ion exchange chromatography using HEPES-saline buffer pH 8.0. The polypeptide chains of the Ig variable domains of the 1AF10 mAb were determined by sequencing of their cDNA prepared from reverse transcribed mRNA purified from hybridoma cells.Binding of anti-TGEV S or -HA (control) mAb to wild type and mutant SA proteins was tested in 96-well plates, using purified mAb or hybridoma supernatants. The SA-Fc fusion proteins in serum-free (opti-MEM, Invitrogen) cell supernatants were bound to plastic, and mAb binding monitored by optical density (OD 490 nm ). At least four SA-Fc protein concentrations ranging from 10 to 1 mg/ml were used in duplicate and average binding determined in each experiment. Binding ratios were determined after correction for background binding.APN binding assays were also carried out with the SA-Fc fusion protein comprising the TGEV RBD. BHK-pAPN cells constitutively expressing cell surface pAPN were used for binding experiments comparing wild type and mutant RBDs, whereas transiently transfected 293T cells were used for analysis of RBD binding to pAPN glycosylation mutants. Binding was monitored as the percentage of stained cells with the Fc fusion proteins and FITC labeled anti-Fc antibodies by Fluorescence-Activated Cell Sorting (FACS), as shown in Figure 1B . The percentage of cells stained was determined for each protein sample and corrected for background staining. pAPN binding ratios for wild type and mutant RBD proteins shown in Figure 4A were determined from the percentage of BHK-pAPN cells stained with same concentration of wild type and mutant SA-Fc proteins. The binding ratios for wild type and mutant pAPN glycosylation mutants shown in Figure 4B were determined from the percentage of SA-Fc stained 293T cells expressing similar amounts of HA-tagged pAPN proteins. Cell surface expression of the pAPN-HA protein was determined with the HA 12AC5 mAb.The TGEV RBD in complex with the 1AF10 Fab fragment was crystallized using the size exclusion-purified complex of a Table 1 ).Crystallization of the pAPN ectodomain in complex with porcine CoV S proteins was carried out with mixtures of the receptor protein and several TGEV and PRCV protein variants comprising the receptor-binding region (SA, S1H and S3H in ). Crystals appeared only in trials performed with an equimolar mixture of pAPN and the S3H protein derived from the PRCV S at a final protein concentration of 13 mg/ml, and with a crystallization solution of 20% PEG-4K, 0.2 M lithium sulfate and 0.1 M Tris buffer pH 8.5. Crystals were transferred to crystallization solution containing 20% ethylene glycol and frozen for diffraction data collection at the ID29 beamline (PRCV RBD-pAPN in Table 1 ).The structure of the TGEV RBD-1AF10 Fab fragment was initially determined by the molecular replacement (MR) method using the PHASER program , and two search models having either the variable or constant regions of the PDB ID 1AIF mAb structure. The 1AF10 Fab model structure was built manually following electron density maps determined from the MR solution, after improvement with the DM program . The 1AF10 Fab structure was refined with the program phenix.refine , which provided an excellent electron density map for building residues 507 to 650 of the TGEV S, as well as four residues of a thrombin recognition site at the C-terminus. Final structure refinement of the complex was carried out with data extending to 3.0 Å resolution (statistics in Table 1 ). Three cycles of solvent correction, refinement of individual coordinates and atomic displacement parameters combined with TLS were applied in each step of structure refinement with phenix.refine, which was alternated with manual adjustment of the model to the electron density maps. All residues are in allowed regions of the Ramachandran plot. SA protein residues included in the structure of the TGEV RBD are shown in Figure 3D .The structure of the PRCV RBD-pAPN complex was resolved by the MR method using the pAPN structure determined alone (manuscript in preparation) and the TGEV RBD structure as search models. MR solutions were obtained for the two pAPN molecules (chains A and B) of the asymmetric unit and for one RBD molecule (chain E). The three molecules were adjusted manually and refined with the phenix.refine program. The second RBD molecule (chain F) bound to pAPN molecule B was built manually into the electron density map. The 282 residues Nterminal to the PRCV RBD in the S3H protein were largely disordered or degraded during crystallization, and are absent in the structure. The complex structure was refined with the program phenix.refine applying solvent correction, NCS, refinement of individual coordinates and atomic displacement parameters combined with TLS ( Table 1 ). The current model comprises residues 60 to 963 of the pAPN ectodomain with a zinc metal ion at the pAPN enzyme active site, and residues 283 to 426 of the PRCV S, homologous to the TGEV S residues 507 to 650 that defined the TGEV RBD structure ( Figure 3D ). All the residues are in allowed regions of the Ramachandran plot.Coordinates and structure factors have been deposited in the Protein Data Bank with ID codes 4F2M (TGEV RBD-1AF10) and 4F5C (PRCV RBD-pAPN).Buried surfaces and residues at the molecular complex interfaces were determined with the PISA server (. ). Only residues with at least 10% of their surface buried at interfaces in the two independent molecules of the crystal asymmetric units are shown. Figure 2D was prepared with LIGPLOT ( thornton-srv/software/LIGPLOT/), Figure 5A with Ribbons and the other structure representations with PyMOL (pymol.org). Structural alignments were carried out with Modeller using a gap penalty of 3 .Accession numbers of the Alphacoronavirus S proteins mentioned are Q0PKZ5 (TGEV), Q65984 (CCoV), P10033 (FCoV), P15423 Figure 7 . Determinants of TGEV S antigenic site A. A. Binding of TGEV-neutralizing, site A-specific mAbs to RBD mutants. Relative binding (%) of mutants to wild type SA protein is shown for TGEV Sspecific mAbs (top; described in Figure 1C ) and a control anti-HA antibody (see Materials and Methods). RBD regions in which mutations locate are shown (bottom; see also Figure 5D ). Mean and standard deviation of data from at least three experiments. B. Antigenic site A in the TGEV RBD and epitopes for antibodies. Surface and ribbon representation of the RBD with the 1AF10 contact regions colored as in Figure 2B . Three antibody-binding residues (Tyr528, Trp571 and Asn632) in the loops at the RBD tip, as well as TGEV Lys524, Arg577 and Gly529 residues associated with Aa, Ab and Ac subsites , respectively. Lines indicate epitopes for mAbs specific for each of the three antigenic subsites: Aa in yellow, Ab in green and Ac in red. doi:10.1371/journal.ppat.1002859.g007 (HCoV-229), Q6Q1S2 (HCoV-NL63), B6VDW0 (Bulbul-CoV) and Q9Q9P1 (IBV). The PRCV HOL87 S protein sequence is reported in reference . Sequence identities among S proteins were determined with psiblast ( psiblast/). Accession number for the pAPN protein is P15145. Figure S1 Structures of TGEV and PRCV RBDs. A. Secondary structure elements of the RBD structures. b-strands are shown with arrows and colored in blue and cyan, a b-bulge at the b-strand 5 is shown in magenta, helix with a red cylinder, coils with black lines, and disulphide bonds with green lines. B. Stereo view of the superimposed asymmetric unit RBD structures of TGEV (blue and cyan), complex with the 1AF10 mAb, and of PRCV (green and red), complex with the pAPN protein. View as in Figures 2A and 5A . Locations of N and C terminal ends are indicated in lowercase letters. (TIF) Figure S2 Mammalian APN ectodomains. Sequence alignment of the porcine, canine, feline and human APN proteins with conserved residues highlighted in red. Secondary structure elements of the pAPN structure determined in complex with the RBD of PRCV are shown above the sequences. CoV-binding residues and those engaged in pAPN dimerization are highlighted in blue and green, respectively, whereas those at the pAPN catalytic site are in yellow. Residues coordinating the zinc ion are marked with an asterisk, and the N-linked glycosylation site recognized by CoV is marked with a triangle at the pAPN Asn736. The beginning of each of the four APN domains is indicated. (TIF) Figure S3 Aminopeptidases active site. Side chains of residues at the catalytic site of four structurally aligned zinc aminopeptidases based on domain II are shown with stick representation, and with the coordinated zinc ion as a cyan sphere. Human ERAP-1 (PDB code 2XDT) is shown in green, aminopeptidase N of E. Coli (PDB code 2HPT) in magenta, aminopeptidase N of Neisseria meningitidis (PDB code 2GTQ) in blue, and pAPN in yellow. The glutamic acid located in the GAMEN motif is labeled in blue and those located at the conserved HExxHx 18 E motif are in red (sequence in Figure S2 ). (TIF) Figure S4 Dimerization of the pAPN ectodomain in solution. Size exclusion chromatography of the soluble pAPN ectodomain. Continuous line shows optical density (OD) at 280 nm for the elution volume. pAPN protein was run through a Superdex 200 16/60 column (GE Healthcare) with HEPESsaline buffer pH 7.5. Exclusion volume and size (kDa) of molecular weight markers are indicated. Determined molecular weight for the single recombinant glycosylated pAPN ectodomain is about 130 kDa, whereas the protein elutes with a volume corresponding to ,300 kDa. (TIF)Table S1 Sequence of homologous CDR-H3 loops in known mAb structures. Sequence of homologous heavy chain CDR-H3 loops to that of the 1AF10 mAb, identified by a Blast search among protein structures, whose PDB codes are shown. (TIF)Table S2 Intermolecular contacts in the PRCV RBD-pAPN complex structure. RBD and pAPN residues in close contact (#5 Å ) in the two complexes of the crystal asymmetric unit, computed with the program NCONT . RBD residues from the b1-b2, b3-b4 and b5-b6 regions at the tip of the bbarrel domain are shown, with those engaged in hydrogen bonding in red. TGEV/PRCV numbering is given for the RBD residues. (TIF)\n",
            "\t0.058\tCoronaviruses (CoVs) are common human and animal pathogens that can transmit zoonotically and cause severe respiratory disease syndromes. CoV infection requires spike proteins, which bind viruses to host cell receptors and catalyze virus-cell membrane fusion. Several CoV strains have spike proteins with two receptor-binding domains, an S1A that engages host sialic acids and an S1B that recognizes host transmembrane proteins. As this bivalent binding may enable broad zoonotic CoV infection, we aimed to identify roles for each receptor in distinct infection stages. Focusing on two betacoronaviruses, murine JHM-CoV and human Middle East respiratory syndrome coronavirus (MERS-CoV), we found that virus particle binding to cells was mediated by sialic acids; however, the transmembrane protein receptors were required for a subsequent virus infection. These results favored a two-step process in which viruses first adhere to sialic acids and then require subsequent engagement with protein receptors during infectious cell entry. However, sialic acids sufficiently facilitated the later stages of virus spread through cell-cell membrane fusion, without requiring protein receptors. This virus spread in the absence of the prototype protein receptors was increased by adaptive S1A mutations. Overall, these findings reveal roles for sialic acids in virus-cell binding, viral spike protein-directed cell-cell fusion, and resultant spread of CoV infections. IMPORTANCE CoVs can transmit from animals to humans to cause serious disease. This zoonotic transmission uses spike proteins, which bind CoVs to cells with two receptor-binding domains. Here, we identified the roles for the two binding processes in the CoV infection process. Binding to sialic acids promoted infection and also supported the intercellular expansion of CoV infections through syncytial development. Adaptive mutations in the sialic acid-binding spike domains increased the intercellular expansion process. These findings raise the possibility that the lectin-like properties of many CoVs contribute to facile zoonotic transmission and intercellular spread within infected organisms.could be attributed to S proteins. Intriguingly, JHM HEϪ -CoV virus particles agglutinated human erythrocytes, while the related A59-CoV strain did not (Fig. 1A) , even at particle-associated S-protein input levels slightly exceeding the corresponding JHM HEϪ -CoV (Fig. S1B) . These findings suggested that JHM uniquely engages host sialic acids.To validate the presumed JHM virus spike-host cell sialic acid interaction, we developed an MHV-based virus-like particle (VLP) system, in which VLPs contained reporter proteins that can be tracked during virus-cell binding and entry (Fig. 1B ). In this system, VLPs were produced from 293T cells cotransfected with MHV structural genes S, E, M, and N (38) (39) (40) . The N genes were modified by in-frame fusion with dual-split protein (DSP) encoding sequences, the two of which complement to form Renilla luciferase (Rluc) and green fluorescent protein (GFP) reporters. This allowed for the release of VLPs that were intrinsically Rluc and GFP positive.We noted that MHV VLPs incorporated intact JHM spike proteins at levels that were similar to authentic viruses. This was in contrast to JHM spikes that were incorporated into vesicular stomatitis virus (VSV)-based pseudoparticles, most of which were degraded (Fig. 1C) . These findings indicated that the VLPs structurally resemble authentic virus structures, at least with respect to spike stability and particle incorporation. As spike stability and multivalent presentation are frequently required to detect sialate interactions (21, 41) , we considered the VLPs to be well-suited for receptor binding studies. Additionally, VLPs are equally produced either with or without incorporated S proteins (42) , making it possible to use VLPs in quantifying S-protein-dependent virus-cell binding processes.VLPs, either with or without JHM or A59 S proteins, were applied to mCEACAMnegative HeLa cells at equivalent (Rluc) multiplicities. Under these conditions, the S-protein levels on the A59 VLPs exceeded those of JHM VLPs (Fig. 1D ). Yet, in agreement with the hemagglutinating effects of the viruses, the JHM S proteins increased VLP-HeLa cell binding above background levels, while the A59 S proteins did not (Fig. 1E) . Notably, the JHM VLPs did not bind to HeLa cells that were pretreated with neuraminidase ( Fig. 1F, left) , suggesting that the VLP binding was dependent on cellular sialic acids.Surprisingly, VLP binding to cells was not increased by expression of the MHV protein receptor mCEACAM, and furthermore, VLPs did not bind to mCEACAMexpressing cells that had been pretreated with neuraminidase ( Fig. 1F, right) . Together, these data suggest that, during MHV-JHM infection, the initial viral binding process is mediated by S-proteinϪsialic acid interactions, not by the protein receptor mCEACAM.Virus binding to sialic acids facilitates protein receptor-mediated entry. We next aimed to identify the biological importance of the JHM spike-sialate interaction. We reasoned that this interaction could be sufficient for viral entry or may instead be an early step preceding viral engagement with protein (mCEACAM) receptors. To assess virus entry, we utilized VLPs containing N proteins fused to only one of the DSP fragments (DSP ), such that VLP entry into target cells containing DSP 1-7 fragments could be measured by reporter complementation into active Rluc or GFP moieties ( Fig. 2A) . We compared VLP entries in HeLa and HeLa-mCEACAM cells and found that Rluc entry signals required mCEACAM presence (Fig. 2B) . The entry signals were elevated by the presence of TMPRSS2 (Fig. 2B) , a known viral fusion-triggering protease (11, 12, (43) (44) (45) (46) (47) (48) . This facilitating effect of TMPRSS2 supported the contention that the VLP-based entry assay faithfully reflected authentic CoV entry processes. In conjunction with VLP binding data, these findings suggest that JHM-CoV cell entry is a multistep process, with initial virus-sialate binding and subsequent mCEACAM receptor-driven entry.We next determined whether virus binding to sialic acids elevates protein receptordependent virus entry. Here, authentic JHM HEϪ -CoV particles were bound for 2 h to DBT (murine), HeLa-mCEACAM (human), or BHK (hamster) cells, with or without prior cell exposure to neuraminidase. Infection was then measured by quantifying viral firefly luciferase (Fluc) reporter expression. The neuraminidase pretreatments decreased (E) HeLa cells were formalin fixed, rinsed, and incubated for 2 h at 4°C with VLPs bearing A59 or JHM S. Rluc input multiplicities were equal for all VLP inoculations. Media were removed, cells were lysed, and cell-associated Rluc activities were quantified. Data are presented after subtracting background (\"No S\") Rluc-positive (Rluc ϩ ) VLP levels, with each data point representing averages from independent experiments (N ϭ 8 [A59] and 7 [JHM]; n ϭ 4 technical replicates per experiment). Error bars present standard errors (SE) from the mean. Statistically significant deviations from background (\"No S\") binding were assessed by unpaired Student's t test and are indicated as follows: ns, not significant; *, P Ͻ 0.05; **, P Ͻ 0.01; ***, P Ͻ 0.001; ****, P Ͻ 0.0001. (F) HeLa or HeLa-mCEACAM cells were fixed and treated with vehicle (PBS) or neuraminidase (NA) for 3 h at 37°C. JHM-CoV VLPs were then added for 2 h at 4°C, cell-associated Rluc activities were quantified, and data are presented after subtracting background (\"No S\") Rluc ϩ VLP levels. Error bars present standard errors (SE) from the mean. Statistically significant deviations were assessed by unpaired Student's t test and are indicated as follows: ns, not significant; *, P Ͻ 0.05; **, P Ͻ 0.01; ***, P Ͻ 0.001; ****, P Ͻ 0.0001. Qing et al. JHM HEϪ -CoV entry by 38% (DBT), 38% (HeLa-mCEACAM), and 54% (BHK) (Fig. 2C) . After neuraminidase pretreatments, the cells were similarly resistant to IAV pp transduction, but not VSV G protein (VSV G)-containing pseudoparticle (VSVG pp) transduction (Fig. 2C) . Partial resistance to IAV pp, a known sialic acid-requiring virus (49) (50) (51) , is best explained by incomplete removal of sialic acids by neuraminidase. We infer that the partial neuraminidase resistance of CoV VLPs is similarly explained by incomplete sialic acid removal. These results support a model in which initial virus binding to sialic acids facilitates protein receptor-mediated entry. Sialic acid receptors promote JHM viral spread without requiring mCEACAM protein receptors. CoV infections spread through the canonical process of progeny virus release and reinfection, and also through fusion, i.e., syncytial spread, with uninfected cells. JHM S proteins can uniquely mediate syncytia in several conditions, even when mCEACAM protein receptors are absent (33, 35) . To determine whether sialic acids operate independently of mCEACAM receptors in JHM cell-cell membrane fusion and syncytial development, we cultured JHM CoV-infected mouse (DBT) cells with mCEACAM-negative human (HeLa) cells that separately harbored either DSP 1-7 or DSP , such that HeLa cell-cell fusions would permit DSP complementation into active Rluc (Fig. 3A) . Using live-cell Rluc substrate, the development of active Rluc was measured over time, and in the presence of increasing neuraminidase doses. The results (Fig. 3B ) revealed neuraminidase dose-dependent reductions in JHM CoVinduced syncytia, up to 7.4-fold at the highest neuraminidase dose (2 U/ml). Similar, but less pronounced reductions were observed in parallel cultures of CEACAM-positive HeLa cells (Fig. 3C ). Of note, the neuraminidase treatments significantly reduced cell-surface sialic acids, as revealed by adherence of fluorescent wheat germ agglutinin (WGA) to treated cells (Fig. S2 ). From these findings, we conclude that sialic acids can enable a CoV S-mediated cell-cell fusion process without requiring a prototypic proteinaceous receptor.A JHM-CoV spike mutation increases mCEACAM-independent cell binding and membrane fusion. To date, analyses of the multidomain CoV S proteins have revealed that the S1A domains interact with sialic acids (18, 19, 21, 41) and that most of the S1B domains bind to protein receptors (22, 52, 53) . The exceptions are the MHV beta-CoVs, which have S1A domains that bind to protein (CEACAM) receptors (25) . From these findings, we inferred that the JHM-CoV S1A domains contain a novel dual-receptor binding capability, able to bind both sialic acid and CEACAM receptors. Sialic acidbinding sites on beta-CoV S proteins were originally inferred from mutagenesis studies (19) , and most recently, from structural resolution of S proteins in complex with sialosides (18, 54) . Mutations in the inferred site did decrease sialoside binding (19, 41) , even though they are distal from the structurally resolved sialoside binding grooves (18, 54) . Here, we noted a mutation in the original inferred site. Among a stretch of four residues important for sialate-binding activity of bovine CoV (B-CoV) S1A, JHM differed only at residue 176, where there is a Gly on a divergent loop in place of the orthologous B-CoV Glu170 (Fig. 4A ). To evaluate the importance of this divergence to sialic acid binding, we constructed a G176E mutant JHM-CoV S protein. G176E S-protein expression, proteolytic processing, and VLP incorporation were all comparable to the wild type (WT) (Fig. 4B) . Notably, VLPs with G176E spikes bound more tightly to HeLa cells, and G176E spike-containing VLP binding was suppressed by neuraminidase pretreatment of the cells (Fig. 4C ). To correlate this increased cell binding with membrane fusion, we compared the syncytial potency of JHM wild-type and G176E mutant S proteins. Cell-cell fusions were measured by DSP 1-7 ϪDSP 8 -11 complementation (Fig. 4D ), visualized as GFP-positive syncytia (Fig. 4E) , and quantified by measuring Rluc signals (Fig. 4F ). Of note, when mCEACAM receptors were present, synyctia induced by wild type and mutant S proteins developed comparably, but in the absence of mCEACAM, the G176E spikes were significantly more robust in their syncytiuminducing activity (Fig. 4F) . We concluded that a single point mutation in the JHM-CoV S1A domain increases the cell binding and membrane fusion activities of S proteins, independent of the prototype JHM protein receptor.MERS-CoV S1B domains bind to host protein (hDPP4) receptors (22) (23) (24) , and MERS-CoV S1A domains bind to host sialic acids (21) . This raised the question of whether MERS-CoV spikes can catalyze cell fusions after the binding of their S1A domains to cells, and independently of high-affinity protein receptors, similar to the JHM-CoV spikes. To address this question, MERS-CoV S-mediated cell-cell fusions were measured by DSP 1-7 ϪDSP 8 -11 complementation (Fig. 5A ) and quantified by measuring Rluc signals (Fig. 5B ). Of note, syncytial development mediated by MERS-CoV S proteins did not require hDPP4 but did require the S-protein-activating protease human TM-PRSS2 (hTMPRSS2) (Fig. 5B) . The catalytically inactive TMPRSS2(S441A) did not facilitate cell-cell fusion (Fig. 5B) . The MERS S-mediated cell fusions contrasted with those generated by HCoV-229E S proteins, which do not bind sialic acids (21) . 229E S-mediated fusions absolutely required target-cell human aminopeptidase N (hAPN) receptors (Fig. 5B ). Thus, with sufficient cell-surface protease activities, the primary hDPP4 receptor was dispensable, conceivably because the secondary sialate-binding S1A RBDs can operate at a stage in the process.To determine whether the MERS S1A domains operate in the cell fusion process, S1A-Fc proteins (Fig. S5) were added during the assays. Exogenous S1A proteins interfered with hDPP4-independent MERS S-mediated fusion but had no effect on fusion when hDPP4 was present (Fig. 5C ). To determine whether sialic acids operate in the cell fusion process, neuraminidase was added during the assays. Exogenous neuraminidase reduced hDPP4-independent MERS-S-mediated fusion but had no effect on the cell fusion when hDPP4 was present (Fig. S3) . These results further suggest that either of the two S domains, S1A or S1B, can tether S proteins to target cells for subsequent cell-cell fusions, with distinct host receptors for each domain.An analogous MERS-CoV spike mutation similarly increases hDPP4-independent cell binding and membrane fusion. We next determined whether analogous changes in S1A of the related MERS-CoV also modified viral attachment. The S1A interaction with sialic acids has biological significance, as antibodies against S1A protect mice from lethal MERS-CoV infections (55) , and S1A acquires adaptive mutations in humans (56) and in mouse models of human MERS-CoV lung infection (57, 58) . In mice, this selectivity was at Asn222 (57, 58), a known glycan addition site (17, 59, 60) , and a documented hypervariable residue, evident in a group of MERS-CoV-related HKU4 bat viruses but absent in the HKU5 bat virus groups (Fig. 6A) . Remarkably, in the context of CoV S1A structure, the Asn222 position overlaps closely with B-CoV residue Glu170 and MHV-CoV residue Gly176 (Fig. 6B) . We constructed MERS VLPs to determine whether this Asn222 change impacted cell binding. In initial tests, we constructed MERS-CoV VLPs and evaluated their sialate binding properties. Human red blood cells (RBCs) were agglutinated by MERS VLPs, while neuraminidase-treated RBCs were not (Fig. S4) , indicating that VLPs do bind sialosides. We then constructed wild-type and ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) N222D VLPs containing internal Rluc and compared their Rluc and S-protein levels relative to VLPs lacking S proteins (Fig. 6C ). VLPs were applied to human lungderived Calu3 cells, which express the human DPP4 receptor for MERS-CoV, and to mouse lung-derived LET-1 and mouse brain-derived DBT cells, neither of which express human DPP4 and thus do not bind MERS-CoV via a protein receptor. VLP binding was assessed by quantifying cell-associated Rluc. Asn222 change did not change VLP binding to human Calu3 cells but clearly enhanced VLP binding to murine DBT and LET-1 cells (Fig. 6D) . We next considered whether this adaptation for increased cell binding had consequences in MERS S-mediated cell-cell membrane fusion, similar to the JHM-CoV G176E mutant. We found that the N222D mutant S proteins did indeed exhibit enhanced cell fusion, relative to wild-type S proteins (Fig. 6E) , but this was only observed by eliminating hDPP4 (Fig. 6E) , which removes the dominant S1B-protein receptor interaction and demands S1A utilization. To further probe the relationships between S1A-cell binding and cell fusion, we determined whether N222D S1A-Fc proteins would suppress cell fusion more effectively than WT S1A-Fc. Indeed, we found that exogenously added N222D S1A-Fc proteins suppressed cell fusions more potently than the corresponding wild-type S1A-Fc did ( Fig. 6F and Fig. S5) , further reinforcing the direct relationship between S1A binding to cells and resultant cell-cell fusion. Altogether, these findings support the hypothesis that MERS-CoV and JHM-CoV can acquire increased cell binding through S1A mutation, which consequently allows for more robust cell-cell fusion capability. Both of these CoVs can procure these cell binding and cell fusion properties through similarly localized S1A mutations.Viruses frequently begin infection by attaching to cellular sialic acids (61) . Such viruses include several CoVs, which attach to sialates via spike S1A domains (11, 12) , at low affinity (18) , but relatively high multivalent avidity (21, 41) . Atomic resolution structures of 9-O-acetylated sialic acid in complex with the human OC43-CoV S1A domain (18) as well as several sialosides in complex with the MERS-CoV S1A (54) have revealed architectures of CoV-sialate binding sites. Yet even with this detailed understanding, it is not clear whether sialic acids confer susceptibility to CoV infection on their own or whether proteinaceous CoV receptors are also required. In considering this question, we first focused on the murine MHV JHM-CoV strain. This strain can infect cells and mice that lack the proteinaceous MHV receptor, mCEACAM1a (27, 30, 32-35, 62, 63) . This strain is also unusually sensitive to proteolytic activation of spike-mediated membrane fusion (46) , and therefore, we hypothesized that a low-affinity cell binding event, conceivably to cellular sialoglycans, might be sufficient for subsequent JHM-CoV protease-triggered fusion activation and cell entry.We found that JHM-CoV binding to cells depended on host sialic acids. This binding accounted for initial virus attachment to cells, which facilitated subsequent virus engagement with proteinaceous CEACAM receptors. Later in the infection cycle, JHM-CoV spikes facilitated cell fusions and further virus dissemination in a sialic acid-dependent manner. MERS-CoV spike proteins could similarly fuse cells together without requiring prototype protein (hDPP4) receptors, presumably by utilizing sialoside receptors to engage neighboring cells. These findings, summarized in Fig. 7 , suggest that CoV-cell entry and intercellular spread involve the lectin-like activities of spike proteins during both virus-cell attachment and infected-cell expansion into syncytia. green), A59 S1A (PDB accession no. 3JCL, magenta), and MERS S1A (PDB accession no. 5X4R, yellow) using PyMol. MERS S1A residue N222 (stick representation) is located proximally to B-CoV S1A residue E170. (C) MERS VLP binding characterization. MERS-VLPs with equivalent Rluc activities (right) were processed for Western blotting to image S-protein abundance and integrity (left). (D) VLPs were added at equivalent Rluc input multiplicities to human Calu3 or murine LET-1 cells. After 2 h at 4°C, cellassociated Rluc activities were quantified, and data were presented after subtracting background (\"No S\") Rluc ϩ VLP levels. (E) Effector cells expressing the indicated S proteins were cocultured with target cells expressing hTMPRSS2 and/or hDPP4. (F) MERS S-expressing effector cells were mixed with hTMPRSS2expressing target cells, in the presence of the indicated Fc proteins. All Fc proteins were at 10 M concentration throughout the coculture period. Experimental procedures, data acquisition, and processing were performed as described in the Fig. 1 legend.VLPs required the proteinaceous receptors mCEACAM and hDPP4, respectively. Therefore, one possibility is that multivalent CoV-sialate interactions are not sufficient to hold viruses throughout a virus particle-cell membrane fusion process and that sialic acids may instead only tether viruses, transiently, onto host cell surfaces, as they diffuse two-dimensionally and then bind more stably to high-affinity protein receptors. Yet, while insufficient for the initial infection, sialic acid binding may advance the spike-mediated cell-cell membrane fusions that are observed in several CoV infection processes. In cell-cell fusion, spikes on infectedcell surfaces are steadily directed at uninfected (target) cells. They are not liable to diffuse away as cell-bound viruses would, yet they still may require attachment to target cell sialates in order to effect membrane fusion. In support of this contention, we found correlations between sialic acid abundance and membrane fusion activity. This was observed for both JHM-and MERS-CoV spikes, and notably, both the JHMand MERS-CoV spike-directed cell fusions took place independently of any proteinaceous receptors. These correlations suggest that sialic acids on uninfected cells can arrange spikes such that they will then operate cooperatively to pull opposing plasma membranes into coalescence.This differential utilization of sialoside and protein receptors raises questions about CoV evolution. The CoV S1A domains fold into a galectin-like structure (14, 15, 17-19, 25, 64) , and several CoV S1A domains demonstrably bind carbohydrate ligands (18, 21, 41, 54) , but prior to this report, the MHV-CoV S1A domains were considered to bind only protein (mCEACAM) receptors (25) . However, Peng et al. (19) proposed that ancestral MHV-CoVs bound carbohydrates, with adaptive evolution generating the present-day CEACAM-binding sites. This proposal infers the existence, past or present, of evolutionary intermediates capable of binding both mCEACAM and sialic acid. The MHV JHM-CoV strain meets the criteria for such an intermediate-a virus in which a single domain retains two \"receptor\" binding activities, one for carbohydrate and the other for protein. Conceivably, these two receptor sites interact, such that increased affinity for one ligand reduces affinity for the other. Mutants with relatively high affinity for sialic acid may bind poorly to mCEACAM, while strains lacking sialic acid binding may demonstrate strong mCEACAM affinity. An inverse relationship would be consistent with the study of Peng et al., who proposed that acquisition of a CEACAM-binding site concomitantly destroyed a sialate site (19) . Yet perhaps more intriguing relationships are between CoV S and hemagglutinin-esterase (HE) proteins, both of which bind sialosides (65, 66) . Esterase activities within HE proteins deacetylate sialosides, and it has been documented, in HKU1-CoV infections, that HE activity destroys an S-specific sialoside receptor, keeping viruses detached from cells and allowing for virus dissemination (67) . Several MHV strains, including JHM, express a viral HE, which binds and deacetylates 4-O-acetyl sialiate (68) . Similar to HKU-1-CoV HE, MHV HE could destroy MHV sialoside receptors and facilitate dissemination. MHV strain A59 did not bind sialic acids, and it does not retain an intact HE open reading frame (36, 37) , while strain JHM did bind sialic acids and does express HE, suggesting that HE is necessary for those CoVs with relatively high S-protein affinity for sialosides. MERS-CoV binds sialic acids, albeit weakly (21) , and increases in its affinity for sialates may be limited by the absence of a MERS-CoV HE gene.This study revealed S-protein mutations that may endow CoVs with expanded tropisms, beyond that determined by prototype proteinaceous CoV receptors. The JHM-CoV mutation G176E, engineered to reflect sialate-utilizing B-CoV, increased viral S-protein binding to cells, as well as S-protein-mediated membrane fusion, independently of mCEACAM. The MERS-CoV mutation N222D, an adaptation for virus growth in mouse lungs (58) , operated remarkably similar to the JHM-CoV G176E change, with mutant S proteins showing increased hDPP4-independent cell binding and cell fusion. Our findings fit with the hypothesis that viruses with these mutations bind relatively tightly to sialoside receptors. However, it is clear that the changes are not present in currently identified sialoside-binding sites (18, 54) , which raises alternative hypotheses that include mutation-induced allosteric restructuring of sialoside-binding sites. Alternatively, the mutations may restructure distinct receptor-or coreceptor-binding sites, such as presumed binding sites for orthologous CEACAMs on MHV S proteins (69), or sites for CEACAM5 (70) or GRP78 (71) on MERS-CoV S proteins.Finally, this report illuminates understanding of CoV pathogenesis. Among the CoVs, the JHM-CoV strain is known for causing lethal brain infection, even in mice that lack the principal MHV receptor, CEACAM1a (30) . JHM spike was identified to be the major contributor to this phenotype (30) , and JHM-CoV, but not the related A59-CoV, spread interneuronally both in vivo and within in vitro cultures of central nervous system (CNS)-derived cells (89, 90) . Notably, neural cell membranes are known for their abundant sialic acid content (72) . These findings, combined with evidence that cell-tocell syncytial spread correlates with pathogenesis in several infection models (30, 62, 63, (73) (74) (75) , prompts a hypothesis that JHM-CoV sialic acid binding potential accounts for an interneuronal syncytial spread that is rapidly lethal. A prediction is that variants of JHM-CoV exhibiting enhanced sialic acid affinity will have unusually high neurovirulence. Similarly, the MERS-CoV strain causes lethal pneumonia, and here it is significant that antibodies specific for the MERS-CoV S1A domains both neutralize the virus and reduce infection and pathogenesis in a mouse MERS-CoV model system (55, 59, 76) . Conceivably, these antibodies interfere with sialic acid binding, reducing expansion of MERS-CoV that may take place via cell-cell fusion. Variants of MERS-CoV with enhanced cell binding may be useful in assessing the in vivo significance of the findings presented in this report.Cells. HEK293T (ATCC), HeLa (ATCC), and HeLa-mCEACAM (77, 78) cells were maintained in DMEMϪ10% FBS medium (Dulbecco's modified Eagle medium [DMEM] containing 10 mM HEPES, 100 nM sodium pyruvate, 0.1 mM nonessential amino acids, 100 U/ml penicillin G, and 100 g/ml streptomycin, and supplemented with 10% fetal bovine serum [FBS] [Atlanta Biologicals]). BHK-21 cells (ATCC) were maintained in DMEMϪ5% FBS medium. LET-1 cells (BEI Resources) (79) were maintained in DMEMϪ10% FBS medium lacking HEPES, sodium pyruvate and nonessential amino acids. Calu3 cells (ATCC) were maintained in MEMϪ20% FBS medium (minimum essential medium [MEM] supplemented with 20% FBS, 100 U/ml penicillin G, and 100 g/ml streptomycin). DBT cells (80, 81) were maintained in MEMϪ5% FBS medium (MEM supplemented with 5% FBS, 10% tryptose-phosphate broth, 26.8 mM sodium bicarbonate, 2 mM L-glutamine, 100 U/ml penicillin G, and 100 g/ml streptomycin). All cell lines were cultured in a 5% CO 2 incubator at 37°C.Recombinant MHV strains JHM.SD (82) and A59 (83) , both containing a firefly luciferase (Fluc) reporter between the viral E (envelope) and M (matrix) genes, were grown in DBT cells. Media were collected at 24 to 48 h postinfection. JHM HEϪ arose during laboratory passaging of JHM.SD.Virus-like particles. CoV virus-like particles (VLPs) were constructed by cotransfection with equimolar amounts of plasmids encoding CoV S, E (envelope), M (matrix), and N (nucleocapsid). Coding sequences for A59-CoV S, E, M, and N genes are presented in GenBank accession no. AY910861.1; for JHM-CoV, GenBank accession no. AC_000192.1; and for MERS-CoV (EMC 2012 strain , GenBank accession no. JX869059.2, where only the S gene is codon optimized ). The A59/JHM-CoV and the MERS-CoV genes were inserted into pCAGGS and pcDNA3.1 expression vector plasmids, respectively. Recombinant pCAGGS-DSP 1-7 -N and pCAGGS-DSP -N were constructed by fusing the DSP 1-7 or DSP coding sequences (pDSP 1-7 and pDSP [85, 86] provided by Zene Matsuda [University of Tokyo]), followed by a 2ϫ SGGS linker, to the 5= ends of the coding sequences for the N genes.Expression plasmids (1 g) were complexed with polyethylenimine (PEI) (Polysciences) (6 g) in 0.2 ml of Opti-MEM (Life Technologies) for 15 min at room temperature and then added dropwise to 5 ϫ 10 5 HEK293T cells in 1 ml of DMEMϪ10% FBS. For spikeless VLPs, S expression plasmids were replaced with empty vector plasmid DNAs. Six hours later, the cells were replenished with fresh DMEMϪ10% FBS. Media were collected at 48 h posttransfection.Pseudoviruses. Pseudovirus particles (pps) were constructed from a vesicular stomatitis virus (VSV) platform as described in reference 87. Briefly, HEK293T cells were transfected with viral glycoprotein expression plasmids, including pHEF-VSV G-Indiana (BEI Resources), pcDNA3.1-HA5-QH, and pcDNA3.1-PR8 NA1 (obtained from Lijun Rong, University of Illinois-Chicago). Twenty-four hours later, the cells were inoculated for 2 h with VSVdeltaG/Junin GP-luciferase (87, 88) . The cells were rinsed twice with FBS-free DMEM medium and replenished with DMEMϪ10% FBS. Media were collected after a 48-h incubation period.Particle concentration. Media containing viruses, pseudovirus particles, and VLPs were clarified by differential centrifugation (300 ϫ g, 4°C, 10 min; 3,000 ϫ g, 4°C, 10 min). Particles were concentrated from clarified media by overnight centrifugation (SW28, 6500 rpm, 4°C, 18 h) through a cushion comprised of 20% (wt/wt) sucrose in FBS-free DMEM. The resulting pellets were resuspended in FBS-free DMEM, and the resulting 100ϫ concentrated particle stocks were stored at -80°C.Hemagglutination assay. Serial twofold dilutions of 100ϫ concentrated virus and pseudovirus particle stocks were prepared using FBS-free DMEM as diluent. Diluted samples were placed into V-bottom 96-well plates and incubated for 2 h at 4°C. Washed adult human erythrocytes (0.5%) (in phosphate-buffered saline [PBS]) were then added, and scoring was performed after 2-to 12-h incubation at 4°C. For live-cell assays, adherent, confluent cells in 24-or 96-well plates were pretreated with vehicle or neuraminidase (1 U/ml) (catalog no. N2876; Sigma) in FBS-free DMEM for 2 h at 37°C. The cells were rinsed three times with PBS and then used in virus transduction assays.VLP binding assay. VLPs used for binding assays were prepared by cotransfection with plasmids encoding CoV S, E (envelope), M (matrix), and DSP 1-7 -N ϩ DSP 8-11 -N. The resulting VLPs contained complemented DSP Rluc-positive interiors. To quantify VLP-associated Rluc, VLPs were serially diluted into passive lysis buffer (PLB) (catalog no. E194A; Promega) and introduced into opaque microplate wells (50 l per well). Renilla luciferase substrate (1.1 M NaCl, 2.2 mM Na 2 EDTA, 0.22 M KH 2 PO 4 , 1.3 mM NaN 3 , 0.44 mg/ml bovine serum albumin, 2.5 M coelenterazine [pH 5]) was added (100 l per well), and luminescence was read using a Veritas microplate luminometer (Turner BioSystems, Sunnyvale, CA).VLPs were incubated for 2 h at 4°C to adherent, confluent cells, either with or without prior neuraminidase treatments. Incubations were in FBS-free DMEM and at known Rluc VLP/cell ratios. After incubation, unbound VLPs were removed, and cells were rinsed variably with PBS. To measure cell-bound VLPs, adherent cells were dissolved into PLB, and Rluc activity was quantified.VLP entry assay. VLPs used for entry assays were prepared by cotransfection with plasmids encoding CoV S, E (envelope), M (matrix), and DSP 8-11 -N. The resulting VLPs contained DSP fragments in their interiors. To quantify VLP-associated DSP levels, VLPs were mixed with excess DSP 1-7 , in passive lysis buffer, for 30 min at 37°C, to allow for DSP complementation. The excess DSP 1-7 was obtained from HEK293 cells overexpressing these fragments. After postlysis complementation, samples were introduced into opaque microplate wells and luminescence readings were used to infer DSP levels.VLPs in FBS-free DMEM were inoculated onto target cells that were transfected 2 days earlier with pDSP . Indicated experiments included cotransfection of target cells with pDSP 1-7 and pCAGGS-TMPRSS2 FLAG (44) . For VLP inoculations, input multiplicities were normalized to VLP DSP 8-11 levels. After 6 h at 37°C, the cells were rinsed three times with PBS and dissolved in passive lysis buffer, and Rluc activities were quantified.Virus and pseudovirus particle entry assays. Virus particles were incubated with cells, either with or without prior neuraminidase treatments, for 2 h at 4°C. Cells were then rinsed three times with PBS and replenished with FBS-containing DMEM/MEM. After 16 h at 37°C, cells were dissolved in lysis buffer (25 mM Cell-cell fusion assay. Effector and target cells were prepared by introducing expression plasmids, using the PEI transfection method. Effector cells were cotransfected with pDSP 1-7 and the indicated S-expressing plasmids. S-expressing plasmids included pcDNA3.1-229E S C9 (GenBank accession no. AB691763.1, obtained from Fang Li, University of Minnesota). Control effector cells received pDSP 1-7 and empty vector plasmids. Target cells were cotransfected with pDSP and the indicated S receptorexpressing plasmids. These plasmids included pCMV6-Entry-hDPP4 FLAG (GenBank accession no. NM_001935; purchased from OriGene) and pcDNA3.1-hAPN (GenBank accession no. M22324; obtained from Fang Li). Indicated experiments included additional cotransfection of target cells with pCAGGS-TMPRSS2 FLAG (44) .At 6 h posttransfection, effector and target cells were rinsed with PBS, lifted with 0.05% trypsin, and mixed at 1:1 ratios. The cocultures were incubated for 2 to 18 h at 37°C. Fused cells were visualized microscopically as green fluorescent protein-positive (GFP ϩ ) cells, and extents of cell-cell fusion were quantified as Rluc activities present in PLB cell lysates. Samples were chilled and maintained at 4°C during Rluc quantifications to eliminate DSP postlysis complementation. In time course experiments, EnduRen substrate (Promega E6482) was used in place of coelenterazine.Fc constructs. pCEP4-mCEACAM-Fc was constructed previously (31) . Splice overlap extension PCR was used to insert an MreI restriction site and GSGGGG linker codons between the mCEACAM (codons 1 to 142) and the human IgG1 splice donor site. Using this modified construct, the mCEACAM coding region was removed and replaced with MERS-S1A (codons 1 to 357), MERS S1A (N222D), and MERS S1B (codons 1 to 24 from human CD5 [hCD5] signal, followed by MERS S codons 358 to 588).HEK293T cells were transfected using the PEI method and then incubated in FBS-free DMEM containing 2% (wt/vol) Cell Boost 5 (HyClone). Supernatants were collected on days 4 and 7 and clarified by sequential centrifugation (300 ϫ g, 4°C, 10 min; 4,500 ϫ g, 4°C, 10 min). The Fc-tagged proteins were then purified using HiTrap protein A high-performance columns (GE Healthcare) according to the manufacturer's instruction. The resulting purified proteins were quantified spectrophotometrically and stored at -20°C until use.Western blotting. Proteins in sodium dodecyl sulfate (SDS) solubilizer (0.0625 M Tris·HCl [pH 6.8], 10% glycerol, 0.01% bromophenol blue, 2% [wt/vol] SDS, with 2% 2-mercaptoethanol] were heated at 95°C for 5 min, electrophoresed through 8% (wt/vol) polyacrylamide-SDS gels, transferred to nitrocellulose membranes (Bio-Rad), and incubated with mouse monoclonal anti-MHV-HR2 10G (obtained from Fumihiro Taguchi, Nippon Veterinary and Life Sciences, Tokyo, Japan), mouse anti-C9 (EMD Millipore), mouse monoclonal anti-MHV HE, clone 5A11, or goat anti-human IgG (sc-2453; Santa Cruz Biotechnologies). After incubation with appropriate horseradish peroxidase (HRP)-tagged secondary antibodies and chemiluminescent substrate (Thermo Fisher), the blots were imaged and processed with a FluorChem E (Protein Simple).Microscopy and image acquisition. Live-cell images were captured in a z series on an electronmultiplied charge-coupled-device digital camera (EMCCD Cascade 2; Photometrics) and deconvolved using SoftWoRx. Identical conditions were applied to all acquisitions. Deconvolved images were analyzed by an identical algorithm in Imaris 8.3.1 (Bitplane).Statistical analysis. Unless stated otherwise, all experiments were repeated independently at least three times. Each data point graphed represents the mean of an independent repeat (N), generated from three or four technical repetitions (n ϭ 3 or 4). Statistical comparisons were made by unpaired Student's t test. Error bars indicate the standard errors of the data. P values less than 0.05 were considered statistically significant.Supplemental material is available online only. \n",
            "\t0.058\tCoronaviruses (CoVs) are common human and animal pathogens that can transmit zoonotically and cause severe respiratory disease syndromes. CoV infection requires spike proteins, which bind viruses to host cell receptors and catalyze virus-cell membrane fusion. Several CoV strains have spike proteins with two receptor-binding domains, an S1A that engages host sialic acids and an S1B that recognizes host transmembrane proteins. As this bivalent binding may enable broad zoonotic CoV infection, we aimed to identify roles for each receptor in distinct infection stages. Focusing on two betacoronaviruses, murine JHM-CoV and human Middle East respiratory syndrome coronavirus (MERS-CoV), we found that virus particle binding to cells was mediated by sialic acids; however, the transmembrane protein receptors were required for a subsequent virus infection. These results favored a two-step process in which viruses first adhere to sialic acids and then require subsequent engagement with protein receptors during infectious cell entry. However, sialic acids sufficiently facilitated the later stages of virus spread through cell-cell membrane fusion, without requiring protein receptors. This virus spread in the absence of the prototype protein receptors was increased by adaptive S1A mutations. Overall, these findings reveal roles for sialic acids in virus-cell binding, viral spike protein-directed cell-cell fusion, and resultant spread of CoV infections. IMPORTANCE CoVs can transmit from animals to humans to cause serious disease. This zoonotic transmission uses spike proteins, which bind CoVs to cells with two receptor-binding domains. Here, we identified the roles for the two binding processes in the CoV infection process. Binding to sialic acids promoted infection and also supported the intercellular expansion of CoV infections through syncytial development. Adaptive mutations in the sialic acid-binding spike domains increased the intercellular expansion process. These findings raise the possibility that the lectin-like properties of many CoVs contribute to facile zoonotic transmission and intercellular spread within infected organisms.could be attributed to S proteins. Intriguingly, JHM HEϪ -CoV virus particles agglutinated human erythrocytes, while the related A59-CoV strain did not (Fig. 1A) , even at particle-associated S-protein input levels slightly exceeding the corresponding JHM HEϪ -CoV (Fig. S1B) . These findings suggested that JHM uniquely engages host sialic acids.To validate the presumed JHM virus spike-host cell sialic acid interaction, we developed an MHV-based virus-like particle (VLP) system, in which VLPs contained reporter proteins that can be tracked during virus-cell binding and entry (Fig. 1B ). In this system, VLPs were produced from 293T cells cotransfected with MHV structural genes S, E, M, and N (38) (39) (40) . The N genes were modified by in-frame fusion with dual-split protein (DSP) encoding sequences, the two of which complement to form Renilla luciferase (Rluc) and green fluorescent protein (GFP) reporters. This allowed for the release of VLPs that were intrinsically Rluc and GFP positive.We noted that MHV VLPs incorporated intact JHM spike proteins at levels that were similar to authentic viruses. This was in contrast to JHM spikes that were incorporated into vesicular stomatitis virus (VSV)-based pseudoparticles, most of which were degraded (Fig. 1C) . These findings indicated that the VLPs structurally resemble authentic virus structures, at least with respect to spike stability and particle incorporation. As spike stability and multivalent presentation are frequently required to detect sialate interactions (21, 41) , we considered the VLPs to be well-suited for receptor binding studies. Additionally, VLPs are equally produced either with or without incorporated S proteins (42) , making it possible to use VLPs in quantifying S-protein-dependent virus-cell binding processes.VLPs, either with or without JHM or A59 S proteins, were applied to mCEACAMnegative HeLa cells at equivalent (Rluc) multiplicities. Under these conditions, the S-protein levels on the A59 VLPs exceeded those of JHM VLPs (Fig. 1D ). Yet, in agreement with the hemagglutinating effects of the viruses, the JHM S proteins increased VLP-HeLa cell binding above background levels, while the A59 S proteins did not (Fig. 1E) . Notably, the JHM VLPs did not bind to HeLa cells that were pretreated with neuraminidase ( Fig. 1F, left) , suggesting that the VLP binding was dependent on cellular sialic acids.Surprisingly, VLP binding to cells was not increased by expression of the MHV protein receptor mCEACAM, and furthermore, VLPs did not bind to mCEACAMexpressing cells that had been pretreated with neuraminidase ( Fig. 1F, right) . Together, these data suggest that, during MHV-JHM infection, the initial viral binding process is mediated by S-proteinϪsialic acid interactions, not by the protein receptor mCEACAM.Virus binding to sialic acids facilitates protein receptor-mediated entry. We next aimed to identify the biological importance of the JHM spike-sialate interaction. We reasoned that this interaction could be sufficient for viral entry or may instead be an early step preceding viral engagement with protein (mCEACAM) receptors. To assess virus entry, we utilized VLPs containing N proteins fused to only one of the DSP fragments (DSP ), such that VLP entry into target cells containing DSP 1-7 fragments could be measured by reporter complementation into active Rluc or GFP moieties ( Fig. 2A) . We compared VLP entries in HeLa and HeLa-mCEACAM cells and found that Rluc entry signals required mCEACAM presence (Fig. 2B) . The entry signals were elevated by the presence of TMPRSS2 (Fig. 2B) , a known viral fusion-triggering protease (11, 12, (43) (44) (45) (46) (47) (48) . This facilitating effect of TMPRSS2 supported the contention that the VLP-based entry assay faithfully reflected authentic CoV entry processes. In conjunction with VLP binding data, these findings suggest that JHM-CoV cell entry is a multistep process, with initial virus-sialate binding and subsequent mCEACAM receptor-driven entry.We next determined whether virus binding to sialic acids elevates protein receptordependent virus entry. Here, authentic JHM HEϪ -CoV particles were bound for 2 h to DBT (murine), HeLa-mCEACAM (human), or BHK (hamster) cells, with or without prior cell exposure to neuraminidase. Infection was then measured by quantifying viral firefly luciferase (Fluc) reporter expression. The neuraminidase pretreatments decreased (E) HeLa cells were formalin fixed, rinsed, and incubated for 2 h at 4°C with VLPs bearing A59 or JHM S. Rluc input multiplicities were equal for all VLP inoculations. Media were removed, cells were lysed, and cell-associated Rluc activities were quantified. Data are presented after subtracting background (\"No S\") Rluc-positive (Rluc ϩ ) VLP levels, with each data point representing averages from independent experiments (N ϭ 8 [A59] and 7 [JHM]; n ϭ 4 technical replicates per experiment). Error bars present standard errors (SE) from the mean. Statistically significant deviations from background (\"No S\") binding were assessed by unpaired Student's t test and are indicated as follows: ns, not significant; *, P Ͻ 0.05; **, P Ͻ 0.01; ***, P Ͻ 0.001; ****, P Ͻ 0.0001. (F) HeLa or HeLa-mCEACAM cells were fixed and treated with vehicle (PBS) or neuraminidase (NA) for 3 h at 37°C. JHM-CoV VLPs were then added for 2 h at 4°C, cell-associated Rluc activities were quantified, and data are presented after subtracting background (\"No S\") Rluc ϩ VLP levels. Error bars present standard errors (SE) from the mean. Statistically significant deviations were assessed by unpaired Student's t test and are indicated as follows: ns, not significant; *, P Ͻ 0.05; **, P Ͻ 0.01; ***, P Ͻ 0.001; ****, P Ͻ 0.0001.® JHM HEϪ -CoV entry by 38% (DBT), 38% (HeLa-mCEACAM), and 54% (BHK) (Fig. 2C) . After neuraminidase pretreatments, the cells were similarly resistant to IAV pp transduction, but not VSV G protein (VSV G)-containing pseudoparticle (VSVG pp) transduction (Fig. 2C) . Partial resistance to IAV pp, a known sialic acid-requiring virus (49) (50) (51) , is best explained by incomplete removal of sialic acids by neuraminidase. We infer that the partial neuraminidase resistance of CoV VLPs is similarly explained by incomplete sialic acid removal. These results support a model in which initial virus binding to sialic acids facilitates protein receptor-mediated entry. Sialic acid receptors promote JHM viral spread without requiring mCEACAM protein receptors. CoV infections spread through the canonical process of progeny virus release and reinfection, and also through fusion, i.e., syncytial spread, with uninfected cells. JHM S proteins can uniquely mediate syncytia in several conditions, even when mCEACAM protein receptors are absent (33, 35) . To determine whether sialic acids operate independently of mCEACAM receptors in JHM cell-cell membrane fusion and syncytial development, we cultured JHM CoV-infected mouse (DBT) cells with mCEACAM-negative human (HeLa) cells that separately harbored either DSP 1-7 or DSP , such that HeLa cell-cell fusions would permit DSP complementation into active Rluc (Fig. 3A) . Using live-cell Rluc substrate, the development of active Rluc was measured over time, and in the presence of increasing neuraminidase doses. The results (Fig. 3B ) revealed neuraminidase dose-dependent reductions in JHM CoVinduced syncytia, up to 7.4-fold at the highest neuraminidase dose (2 U/ml). Similar, but less pronounced reductions were observed in parallel cultures of CEACAM-positive HeLa cells (Fig. 3C ). Of note, the neuraminidase treatments significantly reduced cell-surface sialic acids, as revealed by adherence of fluorescent wheat germ agglutinin (WGA) to treated cells (Fig. S2 ). From these findings, we conclude that sialic acids can enable a CoV S-mediated cell-cell fusion process without requiring a prototypic proteinaceous receptor.A JHM-CoV spike mutation increases mCEACAM-independent cell binding and membrane fusion. To date, analyses of the multidomain CoV S proteins have revealed that the S1A domains interact with sialic acids (18, 19, 21, 41) and that most of the S1B domains bind to protein receptors (22, 52, 53) . The exceptions are the MHV beta-CoVs, which have S1A domains that bind to protein (CEACAM) receptors (25) . From these findings, we inferred that the JHM-CoV S1A domains contain a novel dual-receptor binding capability, able to bind both sialic acid and CEACAM receptors. Sialic acidbinding sites on beta-CoV S proteins were originally inferred from mutagenesis studies (19) , and most recently, from structural resolution of S proteins in complex with sialosides (18, 54) . Mutations in the inferred site did decrease sialoside binding (19, 41) , even though they are distal from the structurally resolved sialoside binding grooves (18, 54) . Here, we noted a mutation in the original inferred site. Among a stretch of four residues important for sialate-binding activity of bovine CoV (B-CoV) S1A, JHM differed only at residue 176, where there is a Gly on a divergent loop in place of the orthologous B-CoV Glu170 (Fig. 4A ). To evaluate the importance of this divergence to sialic acid binding, we constructed a G176E mutant JHM-CoV S protein. G176E S-protein expression, proteolytic processing, and VLP incorporation were all comparable to the wild type (WT) (Fig. 4B) . Notably, VLPs with G176E spikes bound more tightly to HeLa cells, and G176E spike-containing VLP binding was suppressed by neuraminidase pretreatment of the cells (Fig. 4C ). To correlate this increased cell binding with membrane fusion, we compared the syncytial potency of JHM wild-type and G176E mutant S proteins. Cell-cell fusions were measured by DSP 1-7 ϪDSP 8 -11 complementation (Fig. 4D ), visualized as GFP-positive syncytia (Fig. 4E) , and quantified by measuring Rluc signals (Fig. 4F ). Of note, when mCEACAM receptors were present, synyctia induced by wild type and mutant S proteins developed comparably, but in the absence of mCEACAM, the G176E spikes were significantly more robust in their syncytiuminducing activity (Fig. 4F) . We concluded that a single point mutation in the JHM-CoV S1A domain increases the cell binding and membrane fusion activities of S proteins, independent of the prototype JHM protein receptor.MERS-CoV S1B domains bind to host protein (hDPP4) receptors (22) (23) (24) , and MERS-CoV S1A domains bind to host sialic acids (21) . This raised the question of whether MERS-CoV spikes can catalyze cell fusions after the binding of their S1A domains to cells, and independently of high-affinity protein receptors, similar to the JHM-CoV spikes. To address this question, MERS-CoV S-mediated cell-cell fusions were measured by DSP 1-7 ϪDSP 8 -11 complementation (Fig. 5A ) and quantified by measuring Rluc signals (Fig. 5B ). Of note, syncytial development mediated by MERS-CoV S proteins did not require hDPP4 but did require the S-protein-activating protease human TM-PRSS2 (hTMPRSS2) (Fig. 5B ). The catalytically inactive TMPRSS2(S441A) did not facilitate cell-cell fusion (Fig. 5B ). The MERS S-mediated cell fusions contrasted with those generated by HCoV-229E S proteins, which do not bind sialic acids (21) . 229E S-mediated fusions absolutely required target-cell human aminopeptidase N (hAPN) receptors (Fig. 5B ). Thus, with sufficient cell-surface protease activities, the primary hDPP4 receptor was dispensable, conceivably because the secondary sialate-binding S1A RBDs can operate at a stage in the process.To determine whether the MERS S1A domains operate in the cell fusion process, S1A-Fc proteins (Fig. S5) were added during the assays. Exogenous S1A proteins interfered with hDPP4-independent MERS S-mediated fusion but had no effect on fusion when hDPP4 was present (Fig. 5C ). To determine whether sialic acids operate in the cell fusion process, neuraminidase was added during the assays. Exogenous neuraminidase reduced hDPP4-independent MERS-S-mediated fusion but had no effect on the cell fusion when hDPP4 was present (Fig. S3) . These results further suggest that either of the two S domains, S1A or S1B, can tether S proteins to target cells for subsequent cell-cell fusions, with distinct host receptors for each domain.An analogous MERS-CoV spike mutation similarly increases hDPP4-independent cell binding and membrane fusion. We next determined whether analogous changes in S1A of the related MERS-CoV also modified viral attachment. The S1A interaction with sialic acids has biological significance, as antibodies against S1A protect mice from lethal MERS-CoV infections (55) , and S1A acquires adaptive mutations in humans (56) and in mouse models of human MERS-CoV lung infection (57, 58) . In mice, this selectivity was at Asn222 (57, 58), a known glycan addition site (17, 59, 60) , and a documented hypervariable residue, evident in a group of MERS-CoV-related HKU4 bat viruses but absent in the HKU5 bat virus groups (Fig. 6A) . Remarkably, in the context of CoV S1A structure, the Asn222 position overlaps closely with B-CoV residue Glu170 and MHV-CoV residue Gly176 (Fig. 6B) . We constructed MERS VLPs to determine whether this Asn222 change impacted cell binding. In initial tests, we constructed MERS-CoV VLPs and evaluated their sialate binding properties. Human red blood cells (RBCs) were agglutinated by MERS VLPs, while neuraminidase-treated RBCs were not (Fig. S4) , indicating that VLPs do bind sialosides. We then constructed wild-type and ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) ) N222D VLPs containing internal Rluc and compared their Rluc and S-protein levels relative to VLPs lacking S proteins (Fig. 6C ). VLPs were applied to human lungderived Calu3 cells, which express the human DPP4 receptor for MERS-CoV, and to mouse lung-derived LET-1 and mouse brain-derived DBT cells, neither of which express human DPP4 and thus do not bind MERS-CoV via a protein receptor. VLP binding was assessed by quantifying cell-associated Rluc. Asn222 change did not change VLP binding to human Calu3 cells but clearly enhanced VLP binding to murine DBT and LET-1 cells (Fig. 6D) . We next considered whether this adaptation for increased cell binding had consequences in MERS S-mediated cell-cell membrane fusion, similar to the JHM-CoV G176E mutant. We found that the N222D mutant S proteins did indeed exhibit enhanced cell fusion, relative to wild-type S proteins (Fig. 6E) , but this was only observed by eliminating hDPP4 (Fig. 6E) , which removes the dominant S1B-protein receptor interaction and demands S1A utilization. To further probe the relationships between S1A-cell binding and cell fusion, we determined whether N222D S1A-Fc proteins would suppress cell fusion more effectively than WT S1A-Fc. Indeed, we found that exogenously added N222D S1A-Fc proteins suppressed cell fusions more potently than the corresponding wild-type S1A-Fc did (Fig. 6F and Fig. S5) , further reinforcing the direct relationship between S1A binding to cells and resultant cell-cell fusion. Altogether, these findings support the hypothesis that MERS-CoV and JHM-CoV can acquire increased cell binding through S1A mutation, which consequently allows for more robust cell-cell fusion capability. Both of these CoVs can procure these cell binding and cell fusion properties through similarly localized S1A mutations.Viruses frequently begin infection by attaching to cellular sialic acids (61) . Such viruses include several CoVs, which attach to sialates via spike S1A domains (11, 12) , at low affinity (18) , but relatively high multivalent avidity (21, 41) . Atomic resolution structures of 9-O-acetylated sialic acid in complex with the human OC43-CoV S1A domain (18) as well as several sialosides in complex with the MERS-CoV S1A (54) have revealed architectures of CoV-sialate binding sites. Yet even with this detailed understanding, it is not clear whether sialic acids confer susceptibility to CoV infection on their own or whether proteinaceous CoV receptors are also required. In considering this question, we first focused on the murine MHV JHM-CoV strain. This strain can infect cells and mice that lack the proteinaceous MHV receptor, mCEACAM1a (27, 30, 32-35, 62, 63) . This strain is also unusually sensitive to proteolytic activation of spike-mediated membrane fusion (46) , and therefore, we hypothesized that a low-affinity cell binding event, conceivably to cellular sialoglycans, might be sufficient for subsequent JHM-CoV protease-triggered fusion activation and cell entry.We found that JHM-CoV binding to cells depended on host sialic acids. This binding accounted for initial virus attachment to cells, which facilitated subsequent virus engagement with proteinaceous CEACAM receptors. Later in the infection cycle, JHM-CoV spikes facilitated cell fusions and further virus dissemination in a sialic acid-dependent manner. MERS-CoV spike proteins could similarly fuse cells together without requiring prototype protein (hDPP4) receptors, presumably by utilizing sialoside receptors to engage neighboring cells. These findings, summarized in Fig. 7 , suggest that CoV-cell entry and intercellular spread involve the lectin-like activities of spike proteins during both virus-cell attachment and infected-cell expansion into syncytia. green), A59 S1A (PDB accession no. 3JCL, magenta), and MERS S1A (PDB accession no. 5X4R, yellow) using PyMol. MERS S1A residue N222 (stick representation) is located proximally to B-CoV S1A residue E170. (C) MERS VLP binding characterization. MERS-VLPs with equivalent Rluc activities (right) were processed for Western blotting to image S-protein abundance and integrity (left). (D) VLPs were added at equivalent Rluc input multiplicities to human Calu3 or murine LET-1 cells. After 2 h at 4°C, cellassociated Rluc activities were quantified, and data were presented after subtracting background (\"No S\") Rluc ϩ VLP levels. (E) Effector cells expressing the indicated S proteins were cocultured with target cells expressing hTMPRSS2 and/or hDPP4. (F) MERS S-expressing effector cells were mixed with hTMPRSS2expressing target cells, in the presence of the indicated Fc proteins. All Fc proteins were at 10 M concentration throughout the coculture period. Experimental procedures, data acquisition, and processing were performed as described in the Fig. 1 legend.Sialic Acids Facilitate Coronavirus Dissemination ® While sialic acids affected virus-cell binding, infection by JHM-CoV and cell entry by MERS-CoV VLPs required the proteinaceous receptors mCEACAM and hDPP4, respectively. Therefore, one possibility is that multivalent CoV-sialate interactions are not sufficient to hold viruses throughout a virus particle-cell membrane fusion process and that sialic acids may instead only tether viruses, transiently, onto host cell surfaces, as they diffuse two-dimensionally and then bind more stably to high-affinity protein receptors. Yet, while insufficient for the initial infection, sialic acid binding may advance the spike-mediated cell-cell membrane fusions that are observed in several CoV infection processes. In cell-cell fusion, spikes on infectedcell surfaces are steadily directed at uninfected (target) cells. They are not liable to diffuse away as cell-bound viruses would, yet they still may require attachment to target cell sialates in order to effect membrane fusion. In support of this contention, we found correlations between sialic acid abundance and membrane fusion activity. This was observed for both JHM-and MERS-CoV spikes, and notably, both the JHMand MERS-CoV spike-directed cell fusions took place independently of any proteinaceous receptors. These correlations suggest that sialic acids on uninfected cells can arrange spikes such that they will then operate cooperatively to pull opposing plasma membranes into coalescence.This differential utilization of sialoside and protein receptors raises questions about CoV evolution. The CoV S1A domains fold into a galectin-like structure (14, 15, 17-19, 25, 64) , and several CoV S1A domains demonstrably bind carbohydrate ligands (18, 21, 41, 54) , but prior to this report, the MHV-CoV S1A domains were considered to bind only protein (mCEACAM) receptors (25) . However, Peng et al. (19) proposed that ancestral MHV-CoVs bound carbohydrates, with adaptive evolution generating the present-day CEACAM-binding sites. This proposal infers the existence, past or present, of evolutionary intermediates capable of binding both mCEACAM and sialic acid. The MHV JHM-CoV strain meets the criteria for such an intermediate-a virus in which a single domain retains two \"receptor\" binding activities, one for carbohydrate and the other for protein. Conceivably, these two receptor sites interact, such that increased affinity for one ligand reduces affinity for the other. Mutants with relatively high affinity for sialic acid may bind poorly to mCEACAM, while strains lacking sialic acid binding may demonstrate strong mCEACAM affinity. An inverse relationship would be consistent with the study of Peng et al., who proposed that acquisition of a CEACAM-binding site concomitantly destroyed a sialate site (19) . Yet perhaps more intriguing relationships are between CoV S and hemagglutinin-esterase (HE) proteins, both of which bind (65, 66) . Esterase activities within HE proteins deacetylate sialosides, and it has been documented, in HKU1-CoV infections, that HE activity destroys an S-specific sialoside receptor, keeping viruses detached from cells and allowing for virus dissemination (67) . Several MHV strains, including JHM, express a viral HE, which binds and deacetylates 4-O-acetyl sialiate (68) . Similar to HKU-1-CoV HE, MHV HE could destroy MHV sialoside receptors and facilitate dissemination. MHV strain A59 did not bind sialic acids, and it does not retain an intact HE open reading frame (36, 37) , while strain JHM did bind sialic acids and does express HE, suggesting that HE is necessary for those CoVs with relatively high S-protein affinity for sialosides. MERS-CoV binds sialic acids, albeit weakly (21) , and increases in its affinity for sialates may be limited by the absence of a MERS-CoV HE gene.This study revealed S-protein mutations that may endow CoVs with expanded tropisms, beyond that determined by prototype proteinaceous CoV receptors. The JHM-CoV mutation G176E, engineered to reflect sialate-utilizing B-CoV, increased viral S-protein binding to cells, as well as S-protein-mediated membrane fusion, independently of mCEACAM. The MERS-CoV mutation N222D, an adaptation for virus growth in mouse lungs (58) , operated remarkably similar to the JHM-CoV G176E change, with mutant S proteins showing increased hDPP4-independent cell binding and cell fusion. Our findings fit with the hypothesis that viruses with these mutations bind relatively tightly to sialoside receptors. However, it is clear that the changes are not present in currently identified sialoside-binding sites (18, 54) , which raises alternative hypotheses that include mutation-induced allosteric restructuring of sialoside-binding sites. Alternatively, the mutations may restructure distinct receptor-or coreceptor-binding sites, such as presumed binding sites for orthologous CEACAMs on MHV S proteins (69), or sites for CEACAM5 (70) or GRP78 (71) on MERS-CoV S proteins.Finally, this report illuminates understanding of CoV pathogenesis. Among the CoVs, the JHM-CoV strain is known for causing lethal brain infection, even in mice that lack the principal MHV receptor, CEACAM1a (30) . JHM spike was identified to be the major contributor to this phenotype (30) , and JHM-CoV, but not the related A59-CoV, spread interneuronally both in vivo and within in vitro cultures of central nervous system (CNS)-derived cells (89, 90) . Notably, neural cell membranes are known for their abundant sialic acid content (72) . These findings, combined with evidence that cell-tocell syncytial spread correlates with pathogenesis in several infection models (30, 62, 63, (73) (74) (75) , prompts a hypothesis that JHM-CoV sialic acid binding potential accounts for an interneuronal syncytial spread that is rapidly lethal. A prediction is that variants of JHM-CoV exhibiting enhanced sialic acid affinity will have unusually high neurovirulence. Similarly, the MERS-CoV strain causes lethal pneumonia, and here it is significant that antibodies specific for the MERS-CoV S1A domains both neutralize the virus and reduce infection and pathogenesis in a mouse MERS-CoV model system (55, 59, 76) . Conceivably, these antibodies interfere with sialic acid binding, reducing expansion of MERS-CoV that may take place via cell-cell fusion. Variants of MERS-CoV with enhanced cell binding may be useful in assessing the in vivo significance of the findings presented in this report.Cells. HEK293T (ATCC), HeLa (ATCC), and HeLa-mCEACAM (77, 78) cells were maintained in DMEMϪ10% FBS medium (Dulbecco's modified Eagle medium [DMEM] containing 10 mM HEPES, 100 nM sodium pyruvate, 0.1 mM nonessential amino acids, 100 U/ml penicillin G, and 100 g/ml streptomycin, and supplemented with 10% fetal bovine serum [FBS] [Atlanta Biologicals]). BHK-21 cells (ATCC) were maintained in DMEMϪ5% FBS medium. LET-1 cells (BEI Resources) (79) were maintained in DMEMϪ10% FBS medium lacking HEPES, sodium pyruvate and nonessential amino acids. Calu3 cells (ATCC) were maintained in MEMϪ20% FBS medium (minimum essential medium [MEM] supplemented with 20% FBS, 100 U/ml penicillin G, and 100 g/ml streptomycin). DBT cells (80, 81) were maintained in MEMϪ5% FBS medium (MEM supplemented with 5% FBS, 10% tryptose-phosphate broth, 26.8 mM sodium bicarbonate, 2 mM L-glutamine, 100 U/ml penicillin G, and 100 g/ml streptomycin). All cell lines were cultured in a 5% CO 2 incubator at 37°C. Viruses. Recombinant MHV strains JHM.SD (82) and A59 (83) , both containing a firefly luciferase (Fluc) reporter between the viral E (envelope) and M (matrix) genes, were grown in DBT cells. Media were collected at 24 to 48 h postinfection. JHM HEϪ arose during laboratory passaging of JHM.SD.Virus-like particles. CoV virus-like particles (VLPs) were constructed by cotransfection with equimolar amounts of plasmids encoding CoV S, E (envelope), M (matrix), and N (nucleocapsid). Coding sequences for A59-CoV S, E, M, and N genes are presented in GenBank accession no. AY910861.1; for JHM-CoV, GenBank accession no. AC_000192.1; and for MERS-CoV (EMC 2012 strain , GenBank accession no. JX869059.2, where only the S gene is codon optimized ). The A59/JHM-CoV and the MERS-CoV genes were inserted into pCAGGS and pcDNA3.1 expression vector plasmids, respectively. Recombinant pCAGGS-DSP 1-7 -N and pCAGGS-DSP -N were constructed by fusing the DSP 1-7 or DSP coding sequences (pDSP 1-7 and pDSP [85, 86] provided by Zene Matsuda [University of Tokyo]), followed by a 2ϫ SGGS linker, to the 5= ends of the coding sequences for the N genes.Expression plasmids (1 g) were complexed with polyethylenimine (PEI) (Polysciences) (6 g) in 0.2 ml of Opti-MEM (Life Technologies) for 15 min at room temperature and then added dropwise to 5 ϫ 10 5 HEK293T cells in 1 ml of DMEMϪ10% FBS. For spikeless VLPs, S expression plasmids were replaced with empty vector plasmid DNAs. Six hours later, the cells were replenished with fresh DMEMϪ10% FBS. Media were collected at 48 h posttransfection.Pseudoviruses. Pseudovirus particles (pps) were constructed from a vesicular stomatitis virus (VSV) platform as described in reference 87. Briefly, HEK293T cells were transfected with viral glycoprotein expression plasmids, including pHEF-VSV G-Indiana (BEI Resources), pcDNA3.1-HA5-QH, and pcDNA3.1-PR8 NA1 (obtained from Lijun Rong, University of Illinois-Chicago). Twenty-four hours later, the cells were inoculated for 2 h with VSVdeltaG/Junin GP-luciferase (87, 88) . The cells were rinsed twice with FBS-free DMEM medium and replenished with DMEMϪ10% FBS. Media were collected after a 48-h incubation period.Particle concentration. Media containing viruses, pseudovirus particles, and VLPs were clarified by differential centrifugation (300 ϫ g, 4°C, 10 min; 3,000 ϫ g, 4°C, 10 min). Particles were concentrated from clarified media by overnight centrifugation (SW28, 6500 rpm, 4°C, 18 h) through a cushion comprised of 20% (wt/wt) sucrose in FBS-free DMEM. The resulting pellets were resuspended in FBS-free DMEM, and the resulting 100ϫ concentrated particle stocks were stored at -80°C.Hemagglutination assay. Serial twofold dilutions of 100ϫ concentrated virus and pseudovirus particle stocks were prepared using FBS-free DMEM as diluent. Diluted samples were placed into V-bottom 96-well plates and incubated for 2 h at 4°C. Washed adult human erythrocytes (0.5%) (in phosphate-buffered saline [PBS]) were then added, and scoring was performed after 2-to 12-h incubation at 4°C. For live-cell assays, adherent, confluent cells in 24-or 96-well plates were pretreated with vehicle or neuraminidase (1 U/ml) (catalog no. N2876; Sigma) in FBS-free DMEM for 2 h at 37°C. The cells were rinsed three times with PBS and then used in virus transduction assays.VLP binding assay. VLPs used for binding assays were prepared by cotransfection with plasmids encoding CoV S, E (envelope), M (matrix), and DSP 1-7 -N ϩ DSP 8-11 -N. The resulting VLPs contained complemented DSP Rluc-positive interiors. To quantify VLP-associated Rluc, VLPs were serially diluted into passive lysis buffer (PLB) (catalog no. E194A; Promega) and introduced into opaque microplate wells (50 l per well). Renilla luciferase substrate (1.1 M NaCl, 2.2 mM Na 2 EDTA, 0.22 M KH 2 PO 4 , 1.3 mM NaN 3 , 0.44 mg/ml bovine serum albumin, 2.5 M coelenterazine [pH 5]) was added (100 l per well), and luminescence was read using a Veritas microplate luminometer (Turner BioSystems, Sunnyvale, CA).VLPs were incubated for 2 h at 4°C to adherent, confluent cells, either with or without prior neuraminidase treatments. Incubations were in FBS-free DMEM and at known Rluc VLP/cell ratios. After incubation, unbound VLPs were removed, and cells were rinsed variably with PBS. To measure cell-bound VLPs, adherent cells were dissolved into PLB, and Rluc activity was quantified.VLP entry assay. VLPs used for entry assays were prepared by cotransfection with plasmids encoding CoV S, E (envelope), M (matrix), and DSP 8-11 -N. The resulting VLPs contained DSP fragments in their interiors. To quantify VLP-associated DSP levels, VLPs were mixed with excess DSP 1-7 , in passive lysis buffer, for 30 min at 37°C, to allow for DSP complementation. The excess DSP 1-7 was obtained from HEK293 cells overexpressing these fragments. After postlysis complementation, samples were introduced into opaque microplate wells and luminescence readings were used to infer DSP levels.VLPs in FBS-free DMEM were inoculated onto target cells that were transfected 2 days earlier with pDSP . Indicated experiments included cotransfection of target cells with pDSP 1-7 and pCAGGS-TMPRSS2 FLAG (44) . For VLP inoculations, input multiplicities were normalized to VLP DSP 8-11 levels. After 6 h at 37°C, the cells were rinsed three times with PBS and dissolved in passive lysis buffer, and Rluc activities were quantified.Virus and pseudovirus particle entry assays. Virus particles were incubated with cells, either with or without prior neuraminidase treatments, for 2 h at 4°C. Cells were then rinsed three times with PBS and replenished with FBS-containing DMEM/MEM. After 16 h at 37°C, cells were dissolved in lysis buffer (25 mM Cell-cell fusion assay. Effector and target cells were prepared by introducing expression plasmids, using the PEI transfection method. Effector cells were cotransfected with pDSP 1-7 and the indicated S-expressing plasmids. S-expressing plasmids included pcDNA3.1-229E S C9 (GenBank accession no. AB691763.1, obtained from Fang Li, University of Minnesota). Control effector cells received pDSP 1-7 and empty vector plasmids. Target cells were cotransfected with pDSP and the indicated S receptorexpressing plasmids. These plasmids included pCMV6-Entry-hDPP4 FLAG (GenBank accession no. NM_001935; purchased from OriGene) and pcDNA3.1-hAPN (GenBank accession no. M22324; obtained from Fang Li). Indicated experiments included additional cotransfection of target cells with pCAGGS-TMPRSS2 FLAG (44) .At 6 h posttransfection, effector and target cells were rinsed with PBS, lifted with 0.05% trypsin, and mixed at 1:1 ratios. The cocultures were incubated for 2 to 18 h at 37°C. Fused cells were visualized microscopically as green fluorescent protein-positive (GFP ϩ ) cells, and extents of cell-cell fusion were quantified as Rluc activities present in PLB cell lysates. Samples were chilled and maintained at 4°C during Rluc quantifications to eliminate DSP postlysis complementation. In time course experiments, EnduRen substrate (Promega E6482) was used in place of coelenterazine.Fc constructs. pCEP4-mCEACAM-Fc was constructed previously (31) . Splice overlap extension PCR was used to insert an MreI restriction site and GSGGGG linker codons between the mCEACAM (codons 1 to 142) and the human IgG1 splice donor site. Using this modified construct, the mCEACAM coding region was removed and replaced with MERS-S1A (codons 1 to 357), MERS S1A (N222D), and MERS S1B (codons 1 to 24 from human CD5 [hCD5] signal, followed by MERS S codons 358 to 588).HEK293T cells were transfected using the PEI method and then incubated in FBS-free DMEM containing 2% (wt/vol) Cell Boost 5 (HyClone). Supernatants were collected on days 4 and 7 and clarified by sequential centrifugation (300 ϫ g, 4°C, 10 min; 4,500 ϫ g, 4°C, 10 min). The Fc-tagged proteins were then purified using HiTrap protein A high-performance columns (GE Healthcare) according to the manufacturer's instruction. The resulting purified proteins were quantified spectrophotometrically and stored at -20°C until use.Western blotting. Proteins in sodium dodecyl sulfate (SDS) solubilizer (0.0625 M Tris·HCl [pH 6.8], 10% glycerol, 0.01% bromophenol blue, 2% [wt/vol] SDS, with 2% 2-mercaptoethanol] were heated at 95°C for 5 min, electrophoresed through 8% (wt/vol) polyacrylamide-SDS gels, transferred to nitrocellulose membranes (Bio-Rad), and incubated with mouse monoclonal anti-MHV-HR2 10G (obtained from Fumihiro Taguchi, Nippon Veterinary and Life Sciences, Tokyo, Japan), mouse anti-C9 (EMD Millipore), mouse monoclonal anti-MHV HE, clone 5A11, or goat anti-human IgG (sc-2453; Santa Cruz Biotechnologies). After incubation with appropriate horseradish peroxidase (HRP)-tagged secondary antibodies and chemiluminescent substrate (Thermo Fisher), the blots were imaged and processed with a FluorChem E (Protein Simple).Microscopy and image acquisition. Live-cell images were captured in a z series on an electronmultiplied charge-coupled-device digital camera (EMCCD Cascade 2; Photometrics) and deconvolved using SoftWoRx. Identical conditions were applied to all acquisitions. Deconvolved images were analyzed by an identical algorithm in Imaris 8.3.1 (Bitplane).Statistical analysis. Unless stated otherwise, all experiments were repeated independently at least three times. Each data point graphed represents the mean of an independent repeat (N), generated from three or four technical repetitions (n ϭ 3 or 4). Statistical comparisons were made by unpaired Student's t test. Error bars indicate the standard errors of the data. P values less than 0.05 were considered statistically significant.Supplemental material is available online only. This research was supported by the National Institutes of Health (NIH) under award P01 AI060699.The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.We thank Edward Campbell (Loyola University Chicago) for assistance with fluorescent confocal microscopy. We thank Arlene Barlan and Gautami Galpalli for expert technical support.\n",
            "\t0.048\tThe genus Coronavirus contains about 25 species of coronaviruses (CoVs), which are important pathogens causing highly prevalent diseases and often severe or fatal in humans and animals. No licensed specific drugs are available to prevent their infection. Different host receptors for cellular entry, poorly conserved structural proteins (antigens), and the high mutation and recombination rates of CoVs pose a significant problem in the development of wide-spectrum anti-CoV drugs and vaccines. CoV main proteases (M pro s), which are key enzymes in viral gene expression and replication, were revealed to share a highly conservative substrate-recognition pocket by comparison of four crystal structures and a homology model representing all three genetic clusters of the genus Coronavirus. This conclusion was further supported by enzyme activity assays. Mechanism-based irreversible inhibitors were designed, based on this conserved structural region, and a uniform inhibition mechanism was elucidated from the structures of M pro -inhibitor complexes from severe acute respiratory syndrome-CoV and porcine transmissible gastroenteritis virus. A structureassisted optimization program has yielded compounds with fast in vitro inactivation of multiple CoV M pro s, potent antiviral activity, and extremely low cellular toxicity in cell-based assays. Further modification could rapidly lead to the discovery of a single agent with clinical potential against existing and possible future emerging CoV-related diseases.The genus Coronavirus belongs to the plus-strand RNA virus family of the Coronaviridae and currently contains about 25 species that are classified into three groups according to their genetic and serological relationships . Coronaviruses (CoVs) infect humans and multiple species of animals, causing a variety of highly prevalent and severe diseases [1, 5] . For example, human coronavirus (HCoV) strains 229E (HCoV-229E), NL63 (HCoV-NL63), OC43 (HCoV-OC43), and HKU1 (HCoV-HKU1) cause a significant portion of upper and lower respiratory tract infections in humans, including common colds, bronchiolitis, and pneumonia. They have also been implicated in otitis media, exacerbations of asthma, diarrhea, myocarditis, and neurological disease [2, 3, . A previously unknown HCoV, severe acute respiratory syndrome coronavirus (SARS-CoV), which is most closely related to the group II CoVs , proved to be the etiological agent of a global outbreak of a life-threatening form of pneumonia called severe acute respiratory syndrome (SARS), which, in 2003, was the cause of more than 800 fatalities worldwide . Animal CoVs are mainly associated with enteric and respiratory diseases in livestock and domestic animals. Most of the viruses are highly contagious with significant mortality in young animals, resulting in considerable economic losses worldwide [5, 9] .Although vaccines have been developed against avian infectious bronchitis virus (IBV), canine CoV, and porcine transmissible gastroenteritis virus (TGEV) to help prevent serious diseases, several potential problems remain. Vaccination against IBV is only partially successful due to the continual emergence of new serotypes and recombination events between field and vaccine strains. The development of vaccines against feline infectious peritonitis virus (FIPV) has been frustrated by the phenomenon of antibody-dependent enhancement. No licensed vaccines or specific drugs are available to prevent HCoV infection [6, 9] . Following the SARS outbreak, a series of inhibitors was reported against the helicase and main protease (M pro ) of SARS-CoV to prevent viral replication . However, previous research has only placed emphasis on SARS-CoV, and no structural data are available to confirm the direct interaction between these inhibitors and their targets, or for the further modification of these compounds.In common with other RNA viruses employing RNAdependent RNA polymerases for genome replication, CoVs are generally thought to mutate at a high frequency , although this phenomenon remains to be studied in detail. During the SARS epidemic in China, the emergence of SARS-CoV suggested an animal-human interspecies transmission [22, 23] . The virus continued evolving to adapt to the human host during the course of the outbreak with about onethird the mutation rate of human immunodeficiency virus . The high degree of similarity between genome sequences of bovine CoV and the recently sequenced HCoV-OC43 suggested an earlier animal-to-human interspecies transmission than SARS-CoV . Moreover, a high frequency of RNA recombination is a common feature of CoV genetics and has been demonstrated for representative viruses from all CoV groups, including murine hepatitis virus (MHV), TGEV, and IBV [9, 26] . For instance, the outbreaks caused by variant strains of IBV that arose from recombination of vaccine and wild-type virulent strains in chicken flocks limit the usage of vaccines against IBV [27, 28] . Consequently, it is of concern whether current vaccines or drugs in development will be effective against the next wave of attacks by altered SARS-CoV .In view of the issues posed above, the development of widespectrum drugs against the existing pathogenic CoVs is a more reasonable and attractive prospect than individual strategies for drug design, and thereby could provide an effective first line of defense against future emerging CoVrelated diseases such as SARS. However, some of the key factors controlling the host spectrum and viral pathogenicity are highly variable among CoVs. For instance, CoVs use different host receptors for cellular entry, have poorly conserved structural proteins (antigens), and encode diverse accessory genes in their 3'-terminal genome regions that probably contribute to the pathogenicity of CoVs in specific hosts . Clearly, this structural and functional diversity presents a significant obstacle for designing a versatile compound against all CoVs unless a highly conserved target that is comparatively stable during evolution is identified within the genus Coronavirus. Here we report the discovery of a highly conserved region based on four crystal structures and one homology model of M pro representing all three genetic clusters of the genus Coronavirus, and a uniform inhibition mechanism revealed from the structures of M proinhibitor complexes from SARS-CoV and TGEV. A structureassisted optimization program has yielded compounds with fast in vitro inactivation of multiple CoV M pro s, potent antiviral activity, and extremely low cellular toxicity in cellbased assays. Further modification could rapidly lead to the discovery of a single agent with clinical potential against existing and possible future emerging CoV-associated diseases.Development of wide-spectrum inhibitors is an attractive strategy against CoV-associated diseases; however, it entirely depends on the availability of a conserved target within the whole genus Coronavirus. During the first round of target screening, all structural proteins (including S, E, M, HE, and N proteins) were excluded due to the considerable variations among different CoVs 33, 34] . Subsequently, the RNAdependent RNA polymerase, RNA helicase, and M pro constitute attractive potential nonstructural protein targets for consideration. However, no structural data were available for the former two proteins, increasing the difficulties for rational drug design and downstream modification of possible drug leads.The pivotal roles played by M pro s in controlling viral replication and transcription through extensive processing of replicase polyproteins, together with the absence of closely related cellular homologues, identify the M pro as a potentially important target for antiviral drug design . However, pairwise BLAST of the primary sequences among CoV M pro s showed identities of only 38% in some cases. Since it is acknowledged that three-dimensional structures are more closely conserved than primary sequences, we decided to investigate the conservation among the CoV M pro structures. As the M pro s showed comparatively high sequence similarity within each CoV group, representatives from every group were chosen for comparison. The structures of M pro s from TGEV (group I), HCoV-229E (group I), and SARS-CoV are available . Although the crystal structure of IBV (group III) M pro is currently under refinement by our group, it can nevertheless be used as an experiment-based model. As the structure of MHV M pro (group II) was unavailable, and previous studies have shown that SARS-CoV is related to group II , we constructed a homology model for MHV M pro based on the structure of SARS-CoV M pro . Superposition of the crystal structures and homology model showed approximately 2 Å root mean square deviation for all 300 C a , but the most variable regions were the helical domain III and surface loops. The substrate-binding pockets located in a cleft between domains I and II, and especially the S4, S2, and S1 are highly conserved among CoV M pro s suggesting the possibility for wide-spectrum inhibitor design targeting this region in the M pro s of all CoVs. This hypothesis was further supported by enzyme activity assays (see Table 1 ). Based on the assumption that the substrate-binding sites are highly conserved among CoV M pro s, a fluorescence-labeled substrate MCA-AVLQflSGFR-Lys(Dnp)-Lys-NH2 was synthesized to determine the kinetic parameters of TGEV, HCoV-229E, FIPV, MHV, IBV, and SARS-CoV M pro s. The substrate sequence was derived from residues P4-P59 of the SARS-CoV M pro N-terminal autoprocessing site, which has the sequence AVLQSGFRK. IBV M pro demonstrated an almost identical K m to that of SARS-CoV M pro . An interesting observation was that four other CoV proteases showed marginally stronger binding affinity to the substrate than SARS-CoV M pro itself. These results further support the preliminary biochemical studies on conservation of substrates of CoV M pro s .The structures of TGEV and SARS-CoV M pro s have previously been determined in complex with a substrateanalog chloromethyl ketone (CMK) inhibitor, Cbz-VNSTLQ-CMK. The sequence of this substrate-analog was derived from residues P6-P1 of the N-terminal autoprocessing site of TGEV M pro [36, 38] . However, the two protomers of SARS-CoV M pro each exhibited an unexpected binding mode, possibly resulting from the comparatively weak binding of peptidyl elements derived from the substrate of TGEV M pro and from the highly reactive electrophile CMK. This would suggest that nucleophilic attack might have occurred before a stable noncovalently bound enzyme-inhibitor complex was formed. Accordingly, the single binding mode in the TGEV M pro complex was taken into account when designing possible broad-spectrum inhibitors on the basis of these structures and models. Although the CMK inhibitor is nonselective because of its high chemical reactivity and is susceptible to cleavage by gastric and enteric proteinases, it could provide structural insight into the substrate-binding pocket. Superposition of the structures and model revealed that all these proteases have a His-Cys catalytic dyad with relatively conserved orientations, in which His acts as a proton acceptor and Cys undergoes nucleophilic attack on the carbonyl carbon of the substrate. It is widely accepted that increased inhibitor potency can be achieved provided that a covalent bond is formed between the active Cys residue and the designed compound, resembling the intermediate during substrate cleavage. The Michael acceptors, a class of conjugated carbonyl compounds, were successfully introduced to devise irreversible Cys protease inhibitors, including the antirhinovirus compound ruprintrivir (formerly designated AG7088) , and so the highly reactive electrophile CMK was replaced by a less reactive trans-a, b-unsaturated ethyl ester, which was expected to readily extend into the bulky S19 subsite of CoV M pro s.During our initial round of inhibitor design, we focused on the S1, S2, and S4 subsites crucial for substrate recognition and utilized a strategy for mimicking the substrate side chains of residues P4-P1 to accommodate the corresponding subsites. Since backbones of CoV M pro s constituting this area superimposed particularly well, except for a small segment located on the outer wall of S2, we concentrated on the variation of side chains forming these pockets. In the TGEV M pro complex structure , the side chains of 165-Glu, 162-His, 171-His, and 139-Phe (also conserved in other M pro s) are incorporated with other backbone elements to constitute the S1 site, which has an absolute requirement for Gln at the P1 position via two hydrogen bonds. Modeling showed that a lactam with (S) stereochemistry at the a-carbon might preserve the hydrogen bonds essential for S1 recognition; moreover, a comparatively bulky lactam ring would create additional van der Waals interactions. The side chains of 164-Leu, 51-Ile, 41-His, and 53-Tyr, as well as the alkyl portion of side chains of 186-Asp and 47-Thr, are involved in forming a deep hydrophobic S2 subsite that can accommodate the relatively large side chain of Leu in TGEV M pro . This feature can also be observed in the HCoV-229E M pro . Several conservative substitutions occur in other CoV M pro s (164-Leu ! 165-Met in SARS-CoV and MHV M pro s; 53-Tyr ! 50-Trp in IBV M pro ). Another minor difference was observed in SARS-CoV and MHV M pro s, where the outer wall segment is composed of a short 3 10 -helix from residues 45-50, compared with a less regular structure in HCoV and TGEV M pro . With respect to the structure of IBV M pro undergoing refinement, no clear electron density was observed in the corresponding stretch of residues 44-47. We reasoned that variations in the segment located on the outer wall of S2 should not significantly affect the hydrophobicity of this deep subsite. This is supported by evidence wherein Leu is found at position P2 of substrates for all CoV M pro s. As P2 Phe is present in the C-terminal autocleavage site of SARS-CoV, phenyl was used as a smaller substituent to explore this subsite. The side chain of Thr at P3 is solvent-exposed, so this site was expected to tolerate a wide range of functionality. The side chains of 164-Leu, 166-Leu, 184-Tyr, and 191-Gln that form the S4 hydrophobic subsite of TGEV are conserved in other CoV M pro s, excluding the following conservative substitutions: 184-Tyr ! 184-Phe in HCoV M pro ; 164-Leu ! 165-Met, 184-Tyr ! 185-Phe in SARS-CoV. A tertiary butyloxycarbonyl was introduced at the P4 position as an N-terminal protecting group to enter into the S4 site. Thus, by combining the modifications above, a novel compound designated as I2 (see Figure 1A ) was designed and a series of analogs was synthesized for the inhibition assay (see Protocol S1).Covalent irreversible inactivation of CoV M pro s by Michael acceptor inhibitors proceeds according to the kinetic mechanism presented in the scheme below:The inhibitor initially forms a reversible complex with the protease, which then undergoes a chemical step (nucleophilic attack by Cys) leading to the formation of a stable covalent bond . The evaluation of this series of time-dependent inhibitors requires both the equilibrium-binding constant K i (designated as k 2 /k 1 ) and the inactivation rate constant for covalent bond formation k 3 . We avoided measurement of IC 50 after preincubation to assess the effect of these timedependent inhibitors, since there is a general trend for this value to decrease to zero with prolonged preincubation time, which would lead to an inappropriate evaluation.The Structure of SARS-CoV M pro in Complex with an Inhibitor I2The compounds designed in the first round did not exhibit obvious inhibition on CoV M pro s without preincubation, suggesting a very poor K i . We were able to solve a 2.7-Å resolution crystal structure of SARS-CoV M pro complexed with I2 (see Figure 1B ; Table S1 ) despite the weak noncovalent binding, in order to enhance the inhibitory effect of these compounds. Briefly, compound I2 binds to the shallow cleft formed by a portion of the strand eII and a segment of the loop linking domains II and III. The C b atom of the Michael acceptor forms a covalent bond with Sc of 145-Cys as expected. The lactam P1 inserts favorably into S1 and the side chain of Val at P3 is solvent-exposed. However, the failure of P2 and P4 to be properly accommodated by their corresponding subsites attracted our attention, and might account for the poor inhibitory effect of this series of molecules. First, although phenyl at P2 could enter the S2 site, its rigidity prevents it from reorienting to insert further into this site. Second, the Nterminal protecting group tertiary butyloxycarbonyl did not insert into the S4 subsite, possibly as a result of the planar property of the butyloxyamide group. The other compounds designed in this round are listed in Table S2 .Based on the complex structure of I2, we entered into a second round of optimization focusing on the P2 and P4 recognition sites. For the P2 subsite, the phenyl group was substituted by a more flexible Leu side chain. In order to enhance the binding affinity, a series of residues were utilized as substituents at P4, followed by a heterocycle that should increase the Van der Waals contacts with residues flanking at either side. From this round of modification, two inhibitors designated as N1 and N9, and a more efficacious derivative named N3, were identified with fast in vitro inactivation of all CoV M pro s tested, including those of TGEV, HCoV-229E, FIPV, HCoV-NL63 (representatives from group I); MHV, HCoV-HKU1 (representatives from group II); SARS-CoV (related to group II); and IBV (representative from group III) in preliminary inhibition assays (see Figure S2 ). These inhibitors are not sensitive under 1 mM concentration of dithiothreitol (DTT), which is consistent with a previous report of this type of compound . Subsequently, strict inhibition kinetic parameters were determined and are listed in Table 1 (determination of kinetic parameters of M pro s of HCoV-HKU1 and HCoV-NL63 is underway). These inhibitors showed more powerful inhibition of FIPV M pro than other proteases with high inactivation rates (k obs /[I] ! 23,000 M À1 s À1 ), such that measurement of K i and k 3 proved difficult. In this case, k obs /[I] was utilized to evaluate their inhibition as an approximation of the pseudo second-order rate constant (k 3 /K i ) if very rapid inactivation occurs. The K i of N1 ranges from approximately 1.11-10.7 lM and k 3 ranges from approximately 4.1-50 3 10 À3 s À1 ; the K i of N9 ranges from approximately 0.9-6.7 lM, and k 3 ranges from approximately 2.6-19.5 3 10 À3 s À1 . Compared with N1 and N9, N3 demonstrated more potent inhibition on TGEV, FIPV, MHV, and IBV M pro s with k obs /[I] ranging from approximately 4,700-47,000 M À1 s À1 . We therefore solved the crystal structure of SARS-CoV and TGEV M pro s individually complexed with N1, which revealed a common mechanism of inhibition among CoV M pro s.The Structure of SARS-CoV M pro in Complex with the Inhibitor N1 N1 binds to protomers A and B of SARS-CoV in an identical and normal manner. On binding N1, the S1 subsite in protomer B adopts an active conformation compared with the partially collapsed S1 pocket of protomer B in the native structure , which can be ascribed to inhibitor-induced conformational changes. As a result, discussion will be focused entirely on protomer A (see Figures 1C, 1D, 2A, and 2B ). From the omit map (contoured at 1.2 r), clear electron density showed that N1 binds in an extended conformation with the inhibitor backbone atoms forming an antiparallel sheet with residues 164-168 of the long strand eII on one side, and with residues 189-191 of the loop linking domains II and III. Here we dissect the inhibitor into different parts for further discussion.Gate-regulated switch. Comparison between the molecular surfaces of SARS-CoV M pro complexed with N1 and the native enzyme show that certain residues constituting the S1 and S2 subsites undergo large conformational changes on inhibitor binding (see Figure 2A and 2B). The side chain of 142-Asn flips over with a 6-Å shift to superpose onto the lactam like a lid when P1 inserts into the subsite. This might account for the movement of main chains of residues 141-143 toward the S1 site; 142-Asn, together with the main chains of neighboring residues, covers the P1 site like one half of a gate. On the opposite side, 49-Met protrudes by around 5Å from the hydrophobic S2 site and is situated parallel to the side chain of Leu at P2. The side chain of another residue, 189-Gln, moves upward to form a 3.0-Å hydrogen bond with the backbone NH of P2. These two residues constitute the other half of a gate. Together, these two halves should serve as a gate-regulated switch with an essential role in substrate or inhibitor recognition and binding.Trans-a, b-unsaturated ethyl ester. Clear electron density showed that the S c atom of 145-Cys forms a standard 1.8-Å C-S covalent bond with the C b of vinyl group, which suggests a Michael addition reaction. The S c atom moved slightly (approximately 0.6 Å ) toward the interior of the protein compared with the native enzyme. The Michael acceptor remains in a plane following the Michael addition since it is stabilized by a water molecule. This ordered water molecule donates a long 3.3-Å hydrogen bond to the carboxylate oxygen of the ester and then accepts a 2.8-Å hydrogen bond from the backbone NH of 143-Gly and a 3.0-Å hydrogen bond from the carboxamide nitrogen of 142-Asn. The position of S c in 145-Cys implies that it undergoes nucleophilic attack on C b by approaching the p-electron cloud from above. The carbonyl oxygen occupies the oxyanion hole and is close to backbone NHs of 143-Cys and 145-Cys, mimicking the tetrahedral oxyanion intermediate formed during Ser protease cleavage. However, the standard hydrogen bonds are not formed. The ethyl ester portion extends into the S19 site, with sufficient size, and in an extended conformation, to interact with the alkyl portions of 25-Thr and 27-Leu by van der Waals interaction. P1, P2, and P4 sites. The lactam at P1 inserts favorably into the S1 subsite and forms two stable 2.6-Å hydrogen bonds: one between the lactam oxygen and NE2 of 163-His, and another between the lactam NH and a water molecule at the bottom of this subsite. The C a of Leu at the P2 site in N1 moves into the S2 subsite by approximately 1 Å relative to the corresponding carbon atom in I2, and C b -C c of Leu forms an angle of approximately 408 to the phenyl at P2 in the I2 complex, inserting deeply into the S2 subsite. Another notable difference between N1 and I2 is the insertion of an Ala between P3 and P4 in I2, the latter of which was replaced by an isoxazole to block the N-terminal. As expected, the side chain of Ala at the current P4 position readily enters into the S4 subsite. Simultaneously, the backbone NH of Ala donates a hydrogen bond to the carbonyl oxygen of 190-Thr. The isoxazole at P5 makes Van der Waals contacts with 168-Pro and the backbone of residues 190-191.Further modifications of N1. A variety of substitutions were investigated for P4, P5, and P19 (see Table S3 ). The 1.85-Å crystal structure of SARS-CoV M pro complexed with N9 (see Figure S1 ) showed that Val could serve as a substituent at P4, slightly increasing the hydrophobic interactions. Another derivative N3 with benzyl ester exhibited improved inhibition, which could be seen from inhibition assays of FIPV and MHV M pro s (see Table 1 ). Its co-crystal structure with SARS-CoV M pro indicated that the bulky benzyl group extends into the S19 site, possibly enhancing the Van der Waals interaction with 25-Thr and 27-Leu (see Figure 2C ).There are two molecules per asymmetric unit in the cocrystal structure of TGEV M pro with N1, compared with as many as six molecules per asymmetric unit in the native enzyme structure . N1 binds to TGEV M pro in a similar mode to SARS-CoV M pro with some subtle differences (see Figure 3 ). First, after the nucleophilic addition reaction, the Michael acceptor does not remain in a plane as in the SARS-CoV M pro complex structure, but instead flips over by about 908 to interact with the backbone atoms of residues 141-142. Unlike the SARS-CoV M pro complexed with N1, the TGEV M pro lacks a water molecule connecting the ethyl ester with the side chain of residue 142 (Asn ! Ala in TGEV M pro ). The rate of chemical inactivation presumably depends on how the reactive vinyl group is oriented and on the extent to which the transition-state intermediate can be stabilized by proteases . We suspect that in SARS-CoV M pro , the water molecule prevents the Michael acceptor from reorienting to accept a proton from the imidazole of 41-His in the transition state. Although the intermediate remains to be unveiled, this could partially explain why N1 has a higher inactivation rate constant (k 3 ) against TGEV M pro than SARS-CoV M pro s. Second, another water molecule in the TGEV M pro complex occupies an equivalent position to the 189-Gln side chain, which interacts with the backbone NH of Leu at P2 in SARS-CoV M pro -N1 complex. This water molecule, however, donates a 2.6-Å hydrogen bond to 47-Thr and accepts a 2.7-Å hydrogen bond from the NH backbone of Leu at P2. Third, the isoxazole sways to interact with the backbone atoms of residues 188-189. It is worth mentioning that these slight variations do not notably affect the K i , as the binding modes of P1, P2, and P4 remain the same as in SARS-CoV M pro . lM, and 2.7 lM, respectively (see Figure 4A -4C). The dose response curves all show that N3 was able to penetrate cells derived from different species and tissues to access its targets. Consequently, the results strongly imply that N3 was a widespectrum anti-CoV lead compound. However, we noticed some small discrepancies in the data between enzymeinhibition assays and cell-based assays. This can be explained by the different cells for the inhibitor to enter and by potential incongruities in the dependence of M pro for different CoVs. Furthermore, we cannot exclude the potential existence of differences among the bacterially expressed proteases in enzyme-inhibition assays and subtle differences in activity that were not fully revealed by the SARS-CoVderived substrate used in our in vitro assays.To further substantiate the data and, in particular, to evaluate the ability of this type of compound to prevent cells from being infected by CoVs and their cellular cytotoxicity, a murine delay brain tumor (DBT) cell-based MHV plaquereduction assay was performed for the following reasons: (1) three important human pathogens HCoV-HKU1, HCoV-OC43, and SARS-CoV belong to or relate to group II CoVs; (2) aged mice have been successfully used as a model for increased severity of SARS in elderly humans . The EC 50 of the MHV plaque-reduction assay was 3.4 lM (see Figure 4D ), which was consistent with the IC 50 determined in the MHV inhibition assay. It was observed that when the concentration of N3 increased to 8 lM, the DBT cells could be sufficiently protected. Moreover, 500 lM N3 only displayed 28.3% inhibition of cell growth, suggesting extremely low cellular toxicity (see Figure S3 ). These results demonstrate that N3 is a particularly promising lead compound for further development.Evidence suggests that CoVs may have completed at least two animal-to-human interspecies transmissions to date [22, 24, 25 ]. An alternative hypothesis has been proposed whereby the 1889-1890 pandemic characterized by malaise, fever, pronounced central nervous system symptoms, with a significant increase in case fatality with increasing age, was the result of interspecies transmission of bovine CoV to humans rather than an influenza virus . Although this hypothesis needs more evidence to support, it is widely acknowledged that SARS resulted from animal-to-human transmission of a previously unknown CoV. CoVs, especially those that can infect hosts such as domestic animals and pets, which humans have frequent contact with, remain a potential threat to human health assuming they cross the interspecies barrier again. Hence, the development of wide-spectrum drugs will lead to increased protection of human health, a reduction of the considerable economic costs associated with CoVs, defense against endangered wild animals susceptible to infection, and valuable model animals such as transgenic mice with high mortality rates for CoVs. Identification of the CoV M pro as a conserved target among all CoVs will provide an opportunity for the development of broad-spectrum inhibitors against all CoV-related diseases. Ruprintrivir, whose backbone was also a trans-a, b-unsaturated ester incorporated with the peptidyl portion, has entered clinical trials against rhinovirus infection , although it did not show inhibition of CoVs . This is a compound with poor aqueous solubility and low oral bioavailability in animals. In preclinical animal studies, hydrolysis of this compound produced alcohol and carboxylic acid metabolite, which was 400-fold less active than ruprintrivir and was the predominant biotransformation pathway. Ruprintrivir is formulated as a suspension for intranasal delivery. Phase II studies reported ruprintrivir prophylaxis reduced the proportion of subjects with positive viral cultures and viral titers. Ruprintrivir is well tolerated, and the most common adverse effects of this compound are blood-tinged mucus and nasal passage irritation [45, 46] . This highlights that structure-assisted optimization of N3 could possibly lead to the discovery of a single agent to enter clinical trials against all CoV-associated diseases, although ultimate clinical potential requires more sufficient investigation. Our latest results show that N3 could also strongly inhibit the replication of SARS-CoV and TGEV in cell-based assays (data to be published elsewhere). Furthermore, since this compound was designed against a highly conserved region within the genus Coronavirus, it should have efficient resistance to the high mutation and recombination rates of CoVs. It is noteworthy that N3 also exhibited potent inhibition on the M pro s of HCoV-NL63 and HCoV-HKU1, two recently identified HCoVs associated with bronchiolitis, conjunctivitis, and pneumonia [2, 3] , in preliminary inhibition assays (see Table S2 ). This strongly supports our hypothesis that a single agent developed from N3 could provide an effective first line of defense against future emerging CoVrelated diseases. Moreover, it also suggests that incorporation of Michael acceptor with the peptidyl portion specific for proteases would be a good starting point for the development of inhibitors against viral Cys or Ser proteases. A comprehensive and systematic program of optimization of this class of inhibitors based on CoV M pro -inhibitor complexes is underway. We have so far crystallized MHV M pro , and the crystallization of M pro s of recently identified HCoV-NL63 and HCoV-HKU1 are in progress.Protein cloning, expression, and purification. The preparation of SARS-CoV M pro for structural analysis has been described previously . The method of preparation of SARS-CoV M pro for activity assay is almost identical except that the coding sequence was inserted into BamHI and XhoI sites of the expression vector pGEX-4T-1 (Pharmacia, New York, United States). The cDNA encoding IBV M pro (M41 strain) was a gift from Professor Ming Liao (South China Agricultural University, China); the cDNA encoding M pro of MHV (A59 strain) was a gift from Professor Guangxia Gao (Institute of Biophysics Chinese Academy of Sciences, China); the cDNA encoding M pro of HCoV-HKU1 was kindly provided by Professor Kwok-yung Yuen (University of Hong Kong, China); coding sequences of TGEV, IBV, HCoV-HKU1, and HCoV-NL63 M pro s were inserted into BamHI and XhoI sites of the pGEX-4T-1 plasmid, and the subsequent methods for expression and purification were carried out as for SARS-CoV M pro . After change of a BamHI cleavage site at 429-434 in the sequence coding MHV M pro to GGCTCC, this coding sequence was inserted into BamHI and XhoI sites of pGEX-4T-1 plasmid for expression. FIPV M pro (15 mg/ml) and HCoV-229E M pro (15 mg/ml; two amino acids deleted at C-terminal) were expressed and purified as described previously [39, 47] .Crystallization and data collection. SARS-CoV M pro was crystallized as previously reported . The SARS-CoV M pro inhibitor complexes were prepared as follows. First, the inhibitors were dissolved in 7.5% PEG 6000, 6% DMSO, and 0.1 M Mes (pH 6.0) with a concentration of 10 mM (supersaturation). Then, a 3-ll aliquot of such solution was added to the drop, and the crystals were soaked for approximately 2-6 days. A single crystal was prepared for low-temperature data collection by transfer to a cryoprotectant solution containing 30% PEG 400 and 0.1 M Mes (pH 6.0) and then flash frozen in a stream of N 2 gas at 100 K. The set of SARS-CoV M pro -I2 complex data was collected to 2.7 Å resolution using a Mar345 image plate (Marresearch, Norderstedt, Germany) mounted on a Rigaku RU2000 X-ray generator (Sevenoaks, United Kingdom) operated at 48 kV and 98 mA (k ¼ 1.5418 Å ). Data for SARS-CoV M pro individually complexed with N1 and N3 were collected at 100 K in-house on a Rigaku CuK a rotating-anode X-ray generator (MM007) at 40 kV and 20 mA (k ¼ 1.5418 Å ) with a Rigaku image-plate detector. Data for SARS-CoV M pro -N9 complex were collected at 100 K in-house on a Rigaku CuK a rotating-anode X-ray generator (FR-E) at 45 kV and 45 mA (k ¼ 1.5418 Å ) with a Rigaku image-plate detector.In respect to TGEV M pro co-crystal preparation, TGEV M pro was incubated with a 3-fold molar excess of N1 for 24 h at 4 8C. Crystallization trials were performed by the method published previously . Briefly, the condition for crystal growth is 0.1 M HEPES (pH 8.5), 1.8 M (NH 4 ) 2 SO 4 , 6% MPD, 5 mM DTT, and 5% dioxane. The set of TGEV M pro -N1 complex data was collected according to the method for SARS-CoV M pro -N9 complex All intensity data were indexed, integrated, and scaled with the HKL2000 programs DENZO and SCALEPACK . Data collection statistics are summarized in Table S1 . Since the refinement of the IBV M pro structure is ongoing, the methods of crystallization and structure determination will be published elsewhere. Structure elucidation, model building, and refinement. The methods for structure determination, model building, and refinement were published previously . Briefly, the SARS-CoV M pro -I2 complex structure was determined by molecular replacement from our native structure of SARS-CoV M pro (pH 7.6) (PDB ID: 1UK3). The structures of SARS-CoV M pro in complex with N1, N3, or N9 were determined from the isomorphous SARS-CoV M pro -I2 complex structure. The TGEV M pro -N1 structure was determined by molecular replacement using a single monomer of the native TGEV M pro structure (PDB ID: 1P9U). All cross-rotation and translation searches for molecular replacement were performed with CNS . Adjustments to the models were made in O . Positional refinement, individual B-factor refinement, and water picking were performed with CNS . Validation of the final models was performed with PROCHECK . Detailed refinement statistics are summarized in Table S1 .Enzyme activity assay. The activity of M pro s was measured by continuous kinetic assays, using an identical fluorogenic substrate MCA-AVLQSGFR-Lys(Dnp)-Lys-NH2 (over 95% purity, GL Biochem Shanghai Ltd, Shanghai, China). The fluorescence intensity was monitored with a Fluoroskan Ascent instrument (ThermoLabsystems, Helsinki, Finland) using wavelengths of 320 and 405 nm for excitation and emission, respectively. The experiments were performed with a buffer consisting of 50 mM Tris-HCl (pH 7.3), 1 mM EDTA, with or without DTT. Kinetic parameters, K m and k cat , were determined by initial rate measurements at 30 8C. M pro inhibition assays. As compounds with potent inhibition identified in preliminary inhibition assay, the strict kinetic parameters were determined. Time-dependent inhibitor progress curves were fit to a first-order exponential (equation 2) [43, 52] to yield an observed first-order inhibition rate constant (k obs ). P is the product fluorescence; v 0 is the initial velocity; t is time; D is a displacement term to account for the fact that the emission is nonzero at the start of data collection. The values of K i and k 3 were calculated from plots of 1/k obs obtained from equation 2 versus 1/[I] according to equation 3.[I] is inhibitor concentration; [S] is substrate concentration; K m is the Michaelis-Menten constant for the substrate; k 3 is the rate constant of inactivation, and K i is the equilibrium constant.In the experiment, the K i and k 3 values for the irreversible inhibitors were obtained from reactions initiated by addition of individual M pro , the concentration of which was similar as that for the enzymatic activity assay, containing 10 or 20 lM substrate, which depends on the enzymatic activity. The inhibitors vary from 5-8 different concentrations (10-fold molar excess of the enzyme in most cases). Data from the continuous assays were analyzed with the nonlinear regression analysis program Origin. When fast inactivation occurs, the measurement of K i and k 3 proved difficult. In this case, k obs /[I] was used as an approximation of the pseudo second-order rate constant to evaluate the inhibitors and was measured at approximately 2-4 different inhibitor concentrations. The error associated with this determination (k obs /[I]) is less than 20% of a given value. MHV-A59 plaque-reduction assay. Murine DBT cells (generously provided by Dr. Lishan Su of University of North Carolina) were cultured in Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum (FBS) and antibiotics at 37 8C in 5% CO 2 .DBT cells were suspended in growth medium in triplicate wells in 6-well plates and preincubated with appropriate concentrations of the inhibitor. The next day, the medium was aspirated, and MHV-A59 was added to each well at a titer of 100 PFU/well. After incubation for 1 h, the virus inoculum was aspirated, and 2 ml of a media-agar overlay with appropriate concentrations of inhibitor was added to each well. The plates were further incubated for 24 h and stained with neutral red to visualize plaques.Cytotoxicity assay. DBT cells were suspended in growth medium in 96-well plates. The next day, appropriate concentrations of the inhibitor were added to the medium. Two days later, the relative numbers of surviving cells were measured by MTT (Sigma, St. Louis, Missouri, United States) assay in accordance with the manufacturer's instructions.HCoV-229E, FIPV, and MHV-A59 infection assays. Human embryonic lung fibroblast cells (MRC-5; ATCC [Manassas, Virginia, United States]: CCL 171), Felis catus whole fetus (macrophage) cells (FCWF, ATCC: CRL 2787), and DBT cells were cultured in minimal essential medium (MEM) supplemented with 25 mM HEPES, Glutamax I, nonessential amino acids, 10% FBS, and antibiotics at 37 8C in 5% CO 2 . Nearly confluent monolayers of MRC-5 (incubated at 33 8C following infection), FCWF, and DBT cells, which were grown in 6-well plates, were infected with HCoV-229E, FIPV (strain 79-1146), and MHV-A59, respectively, at a multiplicity of infection of 3 TCID 50 per cell. After 60 min of virus adsorption, the virus inoculum was replaced with cell culture medium containing varying concentrations of N3 or in the absence of inhibitor. At 14 h postinfection, the virus titers in the cell culture supernatants were determined using standard procedures. All experiments were performed in triplicate and mean values were determined. M pro individually complexed with I2, N1, N3 , and, N9, and TEGV M pro in complex with N1 are 1WNQ, 1WOF, 2AMQ, 2AMD, and 2AMP, respectively. The GenBank (. ) accession number for IBV M pro is DQ157446.\n",
            "\n",
            " Title:  Citation: Identification of Diverse Bat Alphacoronaviruses and Betacoronaviruses in China Provides New Insights Into the Evolution and Origin of Coronavirus-Related Diseases\n",
            "1 \b. \n",
            "\n",
            "Input question: What was discovered in Wuhuan in December 2019?\n",
            "Results (after 1.988 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.524\tIn the end of 2019, a novel coronavirus (2019-nCoV) emerged in Wuhan, China and was causing a serious outbreak of acute respiratory illness. 1 Wuhan locates in the centre of mainland China with a population of 14 million and is very conveniently connected to other parts of China through airlines and highspeed rails. 2 As of 31 January 2020 (5:00 p.m., GMT + 8), there were 9809 confirmed 2019-nCoV cases in mainland China, including 213 deaths and 180 discharges. 3, 4 Cases infected in Wuhan were also detected in many foreign countries or regions including Thailand, Japan, Republic of Korea, the United States, Canada and some European countries. 4 The World Health Organization (WHO) has declared that the novel coronavirus outbreak is a public health emergency of international concern. Official reports on the newly confirmed cases are released very rapidly (several times a day) after January 16, 3-5 as the official diagnosis protocol was released by the WHO on January 17. 6 Recent studies indicated the likelihood of travel-related risks of 2019-nCoV spreading both domestically and internationally. 7 , 8 Many major cities in mainland China reported the finding of imported cases, including Beijing, Shanghai, Guangzhou and Shenzhen. The outbreak is still on-going with an increasing trend in daily new cases. 3, 4 Before the Wuhan lockdown (official travel restriction) on January 23, virtually all cases found in other major cities were exported cases from Wuhan. Population flow data between major cities in mainland China are available online due to the rapid development of internet in recent decades, see  (in Chinese). In this work, we quantified the association between the domestic travel load and the number of cases exported from Wuhan to other cityclusters in mainland China. Our city-clusters are the pool of top five cities in the top 10 provinces (in number of cumulative cases). Thus, we included 10 city-clusters in the analysis, and the details of the selected city-clusters can be found in Supplementary Data S1.We examined the association between the load of domestic passengers departed from Wuhan and the number of confirmed cases to the 10 city-clusters (including the three municipalities, Beijing, Shanghai and Chongqing). Data sets of the daily numbers of domestic passengers were obtained from the location-based services database of Baidu company from January 1 to 20. We selected the top 10 provincial regions (except Hubei) with the largest cumulative number of cases, which accounts for 68% of all cases reported outside Hubei, before the implementation of the city lockdown, on 23 January 2020. Cases from other provinces are scattered and showed no clear pattern. The daily numbers of passengers from Wuhan to the city-clusters of each province were adopted to measure the load of domestic passengers departed from Wuhan to that cluster. From now on, province means the city-clusters in that province. The daily number of cases time series are obtained via the online outbreak situation reports. 3 , 4 Daily cases for each cluster are scaled from daily province total. The detail is given in Supplementary Data. The association was formulated as follows:Here, c i,t represents the daily number of new cases in the ith provincial region on day t. The function E(·) is the expectation; the 'province i ' denotes the dummy variable for the ith provincial region in accounting for the heterogenicity among different provinces, and thus, α i is a locality-varying interception term. Term ξ i is the daily number of passengers from Wuhan to the ith province. The time index was denoted by t, and the term τ modelled the delay from exposure to being detected. The ε represented the daily number of new cases in Wuhan. Hence, the product term (ξ ·ε) was proportional to the rate of cases exported from Wuhan to other places, and this setting was consistent with the framework in Imai et al., 9 which estimated the outbreak size through exported 2019-nCoV cases overseas, as well as in more complex frameworks. 10 The β is the regression coefficient to quantify the association between the load of passengers multiplied by the local infectivity in Wuhan and the number of cases reported outside Wuhan. Hence, the change rate, denoted by ∆ = [exp(β × 100) − 1] × 100%, may be interpreted as the expected percentage change in the daily number of cases offsite found associated with per 100 increase in the daily number of passengers departed from Wuhan where there was one new case daily. We estimated β for three means of transportation, and the P-value less than 0.05 was considered as statistical significance.We also considered a baseline version of the model by replacing term 'β·ξ i,t−τ ·ε t−τ ' in Equation (1) with 'β 0 ·ξ i,t−τ ', which ignored the variation in the force of infection in Wuhan, thus a constant, as in Equation (2) logThe likelihood ratio (LR) test was adopted to justify the model structure in Equation (1) against the baseline form in Equation (2) . For the delay term (τ ), it was expected to be equivalent to the incubation period, which was reported to be 5.2 days (95% CI: 4.1-7.0) in. 1 Finally, we examined the association with τ varying at 3, 4, 5, 6 and 7 days in sensitivity analysis.The LR test yielded statistically significant outcomes, which suggests that the model in Equation (1) is more reasonable than the baseline form in Equation (2) . Through the goodness-of-fit in terms of the McFadden's pseudo-R-squared, we found that τ = 5 days attains the best fitting performance to explain the patterns of the cases offsite detected. This well matched the estimate of the mean incubation period of the infection at 5.2 in. 1 We found a statistically significant positive association between the load of passengers multiplied by the local infectivity in Wuhan and the number of cases reported outside Wuhan, see Table 1 . We estimated that per 100 cases increase in the daily number of newly reported cases in Wuhan together with per 100 persons increase in the daily number of passengers departed Wuhan were likely to cause a 16.25% (95% CI: 14.86-17.66%) increase in the daily number of cases offsite detected on average. The sensitivity analysis by varying τ suggested this fundamental relationship holds, and the details can be found in Table 1 and the Supplementary Data S2. This analysis has limitations. The data used in this study are in the early phase of the outbreak and may not represent subsequent waves. We aim at quantifying the association between domestic travel and 2019-nCoV exportation in China. Although the differences in case ascertainment in different cities clusters are considered in the model by the dummy term 'province i ', the association could also bear heterogeneity. Temporal and spatial correlations were not addressed in this simple modelling analysis due to lack of data. The 2019-nCoV surveillance data were too scattered and short at this early stage to consider temporal and spatial correlation. The correlation between population flow, number of cases offsite detected, and source infection prevalence was addressed in this work. Our modelling framework would be extended to a more complex and realistic form for exploring the potential spatial correlations, and benefit from more detailed disease surveillance and travel population flow data.Supplementary data are available at JTM online. \n",
            "\t0.510\tThe situation has continued to evolve rapidly since then and just a few weeks later, as at 23 January, 614 laboratory-confirmed cases and 17 deaths have been reported including some cases detected outside mainland China . Meanwhile, on 7 January 2020, the novel coronavirus, currently named 2019-nCoV, was officially announced as the causative agent by Chinese authorities . In order to support public health action, viral genome sequences were released by Chinese researchers on 10 January and 2 days later, four further sequences were also made available on the Global Initiative on Sharing All Influenza Data (GISAID) (). While more cases are being reported on a daily basis and there is evidence for some human-to-human transmission in China, a number of important questions remain unanswered. For example, there is no certainty about the source of the outbreak, the transmissibility of the virus as well as the clinical picture and severity of the disease.In this issue of Eurosurveillance, we are publishing two articles on different aspects of the newly emerged 2019-nCoV. One is a research article by Corman et al. on the development of a diagnostic methodology based on RT-PCR of the E and RdRp genes, without the need for virus material; the assays were validated in five international laboratories . Before this publication, a description of the assays had already been made publically available on a dedicated WHO webpage to support rapid development of laboratory testing capacities. The other is a rapid communication where researchers based in Hong Kong report on their attempt to estimate the severity among hospitalised cases of 2019-nCoV infection through modelling based on publically available information, mainly from Wuhan health authorities . It also points out the need for more detailed information to make an informed evaluation of the situation as basis for public health decision-making.Today, the WHO Director-General Tedros Adhanom Ghebreyesus, taking into consideration the deliberations of the members of the International Health Regulations (IHR) Emergency Committee on 2019-nCoV in their second meeting, decided not to declare a public health emergency of international concern.International health organisations such as the European Centre for Disease Prevention and Control (ECDC) and the WHO are monitoring the situation and provide regular updates. ECDC has set up a dedicated webpage on which updates and risk assessments with focus on Europe are available:  novel-coronavirus-china.\n",
            "\t0.508\tIt is now 6 weeks since Chinese health authorities announced the discovery of a novel coronavirus (2019-nCoV) causing a cluster of pneumonia cases in Wuhan, the major transport hub of central China. The earliest human infections had occurred by early December 2019, and a large wet market in central Wuhan was linked to most, but not all, of the initial cases . While evidence from the initial outbreak investigations seemed to suggest that 2019-nCoV could not easily spread between humans , it is now very clear that infections have been spreading from person to person . We recently estimated that more than 75,000 infections may have occurred in Wuhan as at 25 January 2020 , and increasing numbers of infections continue to be detected in other cities in mainland China and around the world. A number of important characteristics of 2019-nCoV infection have already been identified, but in order to calibrate public health responses we need improved information on transmission dynamics, severity of the disease, immunity, and the impact of control and mitigation measures that have been applied to date.Infections with 2019-nCoV can spread from person to person, and in the earliest phase of the outbreak the basic reproductive number was estimated to be around 2.2, assuming a mean serial interval of 7.5 days . The serial interval was not precisely estimated, and a potentially shorter mean serial interval would have corresponded to a slightly lower basic reproductive number. Control measures and changes in population behaviour later in January should have reduced the effective reproductive number. However, it is too early to estimate whether the effective reproductive number has been reduced to below the critical threshold of 1 because cases currently being detected and reported would have mostly been infected in mid-to late-January. Average delays between infection and illness onset have been estimated at around 5-6 days, with an upper limit of around 11-14 days [2, 5] , and delays from illness onset to laboratory confirmation added a further 10 days on average .Chains of transmission have now been reported in a number of locations outside of mainland China. Within the coming days or weeks it will become clear whether sustained local transmission has been occurring in other cities outside of Hubei province in China, or in other countries. If sustained transmission does occur in other locations, it would be valuable to determine whether there is variation in transmissibility by location, for example because of different behaviours or control measures, or because of different environmental conditions. To address the latter, virus survival studies can be done in the laboratory to confirm whether there are preferred ranges of temperature or humidity for 2019-nCoV transmission to occur.In an analysis of the first 425 confirmed cases of infection, 73% of cases with illness onset between 12 and 22 January reported no exposure to either a wet market or another person with symptoms of a respiratory illness . The lack of reported exposure to another ill person could be attributed to lack of awareness or recall bias, but China's health minister publicly warned that pre-symptomatic transmission could be occurring . Determining the extent to which asymptomatic or pre-symptomatic transmission might be occurring is an urgent priority, because it has direct implications for public health and hospital infection control. Data on viral shedding dynamics could help in assessing duration of infectiousness. For severe acute respiratory syndrome-related coronavirus (SARS-CoV), infectivity peaked at around 10 days after illness onset , consistent with the peak in viral load at around that time . This allowed control of the SARS epidemic through prompt detection of cases and strict isolation. For influenza virus infections, virus shedding is highest on the day of illness onset and relatively higher from shortly before symptom onset until a few days after onset . To date, transmission patterns of 2019-nCoV appear more similar to influenza, with contagiousness occurring around the time of symptom onset, rather than SARS.Transmission of respiratory viruses generally happens through large respiratory droplets, but some respiratory viruses can spread through fine particle aerosols , and indirect transmission via fomites can also play a role. Coronaviruses can also infect the human gastrointestinal tract [11, 12] , and faecal-oral transmission might also play a role in this instance. The SARS-CoV superspreading event at Amoy Gardens where more than 300 cases were infected was attributed to faecal-oral, then airborne, spread through pressure differentials between contaminated effluent pipes, bathroom floor drains and flushing toilets . The first large identifiable superspreading event during the present 2019-nCoV outbreak has apparently taken place on the Diamond Princess cruise liner quarantined off the coast of Yokohama, Japan, with at least 130 passengers tested positive for 2019-nCoV as at 10 February 2020 . Identifying which modes are important for 2019-nCoV transmission would inform the importance of personal protective measures such as face masks (and specifically which types) and hand hygiene.The first human infections were identified through a surveillance system for pneumonia of unknown aetiology, and all of the earliest infections therefore had Modelling studies incorporating healthcare capacity and processes pneumonia. It is well established that some infections can be severe, particularly in older adults with underlying medical conditions [15, 16] , but based on the generally mild clinical presentation of 2019-nCoV cases detected outside China, it appears that there could be many more mild infections than severe infections. Determining the spectrum of clinical manifestations of 2019-nCoV infections is perhaps the most urgent research priority, because it determines the strength of public health response required. If the seriousness of infection is similar to the 1918/19 Spanish influenza, and therefore at the upper end of severity scales in influenza pandemic plans, the same responses would be warranted for 2019-nCoV as for the most severe influenza pandemics. If, however, the seriousness of infection is similar to seasonal influenza, especially during milder seasons, mitigation measures could be tuned accordingly.Beyond a robust assessment of overall severity, it is also important to determine high risk groups. Infections would likely be more severe in older adults, obese individuals or those with underlying medical conditions, but there have not yet been reports of severity of infections in pregnant women, and very few cases have been reported in children .Those under 18 years are a critical group to study in order to tease out the relative roles of susceptibility vs severity as possible underlying causes for the very rare recorded instances of infection in this age group. Are children protected from infection or do they not fall ill after infection? If they are naturally immune, which is unlikely, we should understand why; otherwise, even if they do not show symptoms, it is important to know if they shed the virus. Obviously, the question about virus shedding of those being infected but asymptomatic leads to the crucial question of infectivity. Answers to these questions are especially pertinent as basis for decisions on school closure as a social distancing intervention, which can be hugely disruptive not only for students but also because of its knock-on effect for child care and parental duties. Very few children have been confirmed 2019-nCoV cases so far but that does not necessarily mean that they are less susceptible or that they could not be latent carriers. Serosurveys in affected locations could inform this, in addition to truly assessing the clinical severity spectrum.Another question on susceptibility is regarding whether 2019-nCoV infection confers neutralising immunity, usually but not always, indicated by the presence of neutralising antibodies in convalescent sera. Some experts already questioned whether the 2019-nCoV may behave similarly to MERS-CoV in cases exhibiting mild symptoms without eliciting neutralising antibodies . A separate question pertains to the possibility of antibody-dependent enhancement of infection or of disease [18, 19] . If either of these were to be relevant, the transmission dynamics could become more complex.A wide range of control measures can be considered to contain or mitigate an emerging infection such as 2019-nCoV. Internationally, the past week has seen an increasing number of countries issue travel advisories or outright entry bans on persons from Hubei province or China as a whole, as well as substantial cuts in flights to and from affected areas out of commercial considerations. Evaluation of these mobility restrictions can confirm their potential effectiveness in delaying local epidemics , and can also inform when as well as how to lift these restrictions.If and when local transmission begins in a particular location, a variety of community mitigation measures can be implemented by health authorities to reduce transmission and thus reduce the growth rate of an epidemic, reduce the height of the epidemic peak and the peak demand on healthcare services, as well as reduce the total number of infected persons . A number of social distancing measures have already been implemented in Chinese cities in the past few weeks including school and workplace closures. It should now be an urgent priority to quantify the effects of these measures and specifically whether they can reduce the effective reproductive number below 1, because this will guide the response strategies in other locations. During the 1918/19 influenza pandemic, cities in the United States, which implemented the most aggressive and sustained community measures were the most successful ones in mitigating the impact of that pandemic .Similarly to international travel interventions, local social distancing measures should be assessed for their impact and when they could be safely discontinued, albeit in a coordinated and deliberate manner across China such that recrudescence in the epidemic curve is minimised. Mobile telephony global positioning system (GPS) data and location services data from social media providers such as Baidu and Tencent in China could become the first occasion when these data inform outbreak control in real time.At the individual level, surgical face masks have often been a particularly visible image from affected cities in China. Face masks are essential components of personal protective equipment in healthcare settings, and should be recommended for ill persons in the community or for those who care for ill persons. However, there is now a shortage of supply of masks in China and elsewhere, and debates are ongoing about their protective value for uninfected persons in the general community.The Table summarises research gaps to guide the public health response identified.In conclusion, there are a number of urgent research priorities to inform the public health response to the global spread of 2019-nCoV infections. Establishing robust estimates of the clinical severity of infections is probably the most pressing, because flattening out the surge in hospital admissions would be essential if there is a danger of hospitals becoming overwhelmed with patients who require inpatient care, not only for those infected with 2019-nCoV but also for urgent acute care of patients with other conditions including those scheduled for procedures and operations. In addressing the research gaps identified here, there is a need for strong collaboration of a competent corps of epidemiological scientists and public health workers who have the flexibility to cope with the surge capacity required, as well as support from laboratories that can deliver on the ever rising demand for diagnostic tests for 2019-nCoV and related sequelae. The readiness survey by Reusken et al. in this issue of Eurosurveillance testifies to the rapid response and capabilities of laboratories across Europe should the outbreak originating in Wuhan reach this continent .In the medium term, we look towards the identification of efficacious pharmaceutical agents to prevent and treat what may likely become an endemic infection globally. Beyond the first year, one interesting possibility in the longer term, perhaps borne of wishful hope, is that after the first few epidemic waves, the subsequent endemic re-infections could be of milder severity. Particularly if children are being infected and are developing immunity hereafter, 2019-nCoV could optimistically become the fifth human coronavirus causing the common cold.None declared.\n",
            "\t0.502\tOn February 7, 2020 , this report was posted as an MMWR Early Release on the MMWR website (https: //) .In December 2019, a cluster of cases of pneumonia emerged in Wuhan City in central China's Hubei Province. Genetic sequencing of isolates obtained from patients with pneumonia identified a novel coronavirus (2019-nCoV) as the etiology (1) . As of February 4, 2020 , approximately 20,000 confirmed cases had been identified in China and an additional 159 confirmed cases in 23 other countries, including 11 in the United States (2, 3) . On January 17, CDC and the U.S. Department of Homeland Security's Customs and Border Protection began health screenings at U.S. airports to identify ill travelers returning from Wuhan City (4) . CDC activated its Emergency Operations Center on January 21 and formalized a process for inquiries regarding persons suspected of having 2019-nCoV infection (2) . As of January 31, 2020, CDC had responded to clinical inquiries from public health officials and health care providers to assist in evaluating approximately 650 persons thought to be at risk for 2019-nCoV infection. Guided by CDC criteria for the evaluation of persons under investigation (PUIs) (5) , 210 symptomatic persons were tested for 2019-nCoV; among these persons, 148 (70%) had travel-related risk only, 42 (20%) had close contact with an ill laboratory-confirmed 2019-nCoV patient or PUI, and 18 (9%) had both travel-and contact-related risks. Eleven of these persons had laboratory-confirmed 2019-nCoV infection. Recognizing persons at risk for 2019-nCoV is critical to identifying cases and preventing further transmission. Health care providers should remain vigilant and adhere to recommended infection prevention and control practices when evaluating patients for possible 2019-nCoV infection (6) . Providers should consult with their local and state health departments when assessing not only ill travelers from 2019-nCoV-affected countries but also ill persons who have been in close contact with patients with laboratory-confirmed 2019-nCoV infection in the United States.As part of CDC's Emergency Operations Center activation, CDC personnel assist state and local health departments with the evaluation of 2019-nCoV PUIs. Public health laboratories were not yet conducting 2019-nCoV testing during the period covered by this report, while awaiting Food and Drug Administration emergency use authorization for the test. (The authorization occurred on February 4*). Therefore, all testing was conducted at CDC. A call center was staffed by a team of physicians and nurses 24 hours per day. During January 17-31, criteria used to determine whether a person was considered to be a PUI included presence of fever and symptoms of lower respiratory tract illness (e.g., cough or difficulty breathing) in addition to epidemiologic risk. Epidemiologic risk factors included history of travel from Wuhan City, close contact with a patient with laboratory-confirmed 2019-nCoV infection, or close contact with an ill PUI. Given the evolving understanding of 2019-nCoV epidemiology, testing was recommended for some persons who did not strictly meet the PUI definition, based on clinical discretion. For clinical inquiries that resulted in 2019-nCoV testing, real-time reverse transcription polymerase chain reaction testing was conducted at CDC using methods developed specifically to detect 2019-nCoV (7).For this report, CDC reviewed inquiries regarding potential 2019-nCoV PUIs received by CDC through January 31, 2020, from state and local health departments, health care providers, and airport health screening personnel. Information was compiled from call logs and PUI forms to assess source of inquiry, PUI demographic data (including age and sex), clinical information, epidemiologic risk factors, and 2019-nCoV test results. To allow for delays in specimen shipping and testing, data for PUIs for whom an initial inquiry was received during January 2020 were collected through February 4, 2020.During January 2020, approximately 30 CDC physicians and nurses responded to inquiries regarding approximately 650 persons. Testing was recommended for 256 persons ( Figure) across 34 jurisdictions (the jurisdictions included states, the District of Columbia, Puerto Rico, and the U.S. Virgin Islands) and was completed for 210 persons. Testing of PUIs was not always performed because alternative diagnoses were made, or symptoms resolved. Among inquiries resulting in testing, six (3%) persons were identified through airport screening, 178 (85%) in a health care setting, and 26 (12%) through contact tracing (Table) . Among 178 persons identified in a health care setting, the type of setting was reported for 125 (70%), including 79 (63%) who were evaluated at an emergency department or hospital, 22 (18%) at a student clinic, and 24 (19%) in other outpatient care settings. A total of 115 (55%) persons tested were male, and median age was 29 years (interquartile range = 21-49 years). Seventeen (8%) were health care workers, and 48 of 129 persons with available information were reported to be college students.All 210 persons who were tested were symptomatic: 143 (68%) had subjective fever or a measured temperature ≥100.4°F (≥38°C), and 189 (90%) had cough or shortness of breath. Upper respiratory tract symptoms (i.e., sore throat, rhinorrhea, or congestion) were common and were present in nine persons who did not have cough or shortness of breath. Thirty persons were reported to test positive for another respiratory viral pathogen, including influenza or respiratory syncytial virus. Forty-two (20%) patients were hospitalized, and four (2%) were admitted to an intensive care unit. One patient was deceased at the time of notification; testing for this person was negative, and an alternative cause of death was established. Travel-related risk was identified for 148 (70%) persons, 42 (20%) had close contact with ill patients with laboratory-confirmed 2019-nCoV infection or PUIs, 18 (9%) had both travel-and contact-related risks, and two (<1%) had possible contact with a laboratory-confirmed 2019-nCoV patient and were therefore tested.Among the 210 persons tested, 11 (5%) were found to have 2019-nCoV infection. Nine of these persons had traveled to Wuhan City; two persons had not traveled but had been in close contact with patients with laboratory-confirmed 2019-nCoV in the United States. All were symptomatic with fever (subjective or measured) or cough. 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 CDC's Emergency Operations Center activated * Confirmed cases were reported as of January 31, 2020. † Public announcements of a confirmed 2019-nCoV case in the United States were made on the following dates: Jan 21, Jan 24, Jan 26, Jan 27 (two cases), Jan 30, and Jan 31. (20) Travel from China and close contact identified † † 18 (9) Other risk § § 2 (<1)Abbreviations: ICU = intensive care unit; IQR = interquartile range; PUI = person under investigation. * Numbers might not sum to total because of missing data. † For this person, testing was negative for 2019-nCoV, and an alternative cause of death was established. § Additional persons who were being followed through contact tracing but initially sought treatment at a health care setting are not included in this category. ¶ Includes 113 persons who traveled from Wuhan City and 35 who traveled from areas of China outside Wuhan within 14 days of symptom onset. ** Includes 33 persons who were close contacts of an ill laboratory-confirmed 2019-nCoV patient and nine who were close contacts of PUIs. All contacts occurred within 14 days of symptom onset. † † Includes four persons who traveled from Wuhan City and were close contacts of an ill laboratory-confirmed 2019-nCoV patient, 11 who traveled from Wuhan City and were close contacts of PUIs, and three who traveled from China and were close contacts of a PUI. § § Had possible contact with a laboratory-confirmed 2019-nCoV patient and were therefore tested.persons suspected of being at risk for 2019-nCoV to public health officials and health care providers throughout the United States. Epidemiologic risk factors among the 210 persons tested for 2019-nCoV were not limited to travel: 20% of PUIs tested had not recently traveled to China but reported close contact with a person being evaluated for 2019-nCoV infection. Because person-to-person transmission is expected to continue, and as further travel restrictions are implemented, it is likely that the proportion of PUIs with such contact risk in the United States will increase among all persons evaluated for 2019-nCoV. CDC mobilized early in the response and state and local health departments similarly increased capacity to provide clinical consultation regarding 2019-nCoV. The collection of clinical and epidemiologic data that described characteristics of persons tested for 2019-nCoV helped to inform changes to criteria for PUI evaluation.On January 31, 2020, CDC published updated PUI guidance (8) in response to the evolving global epidemiology of 2019-nCoV, including the rapid geographic expansion and documentation of person-to-person transmission (9) . Updated guidance emphasizes 2019-nCoV testing for symptomatic persons in close contact with patients with laboratory-confirmed 2019-nCoV infection, persons returning from Hubei province in addition to Wuhan City, and persons from mainland China requiring hospitalization because of fever and lower respiratory tract illness. Additional refinements to this approach likely will be needed in the future as understanding of 2019-nCoV epidemiology continues to improve.The findings in this report are subject to at least three limitations. First, the number of clinical inquiries received by CDC does not represent all inquiries received by health departments. Second, because the primary objective of this effort was to guide a timely public health response, some clinical and epidemiologic risk factor data might be incomplete. Finally, given the limited available epidemiologic information early in the outbreak, to provide some latitude for clinical decision-making regarding diagnostic testing, the PUI definition was not strictly applied in all cases.A coordinated national effort to diagnose 2019-nCoV among persons at high risk for infection is important to facilitate appropriate management and limit transmission in the United States. CDC's website provides guidance for health care professionals on evaluating persons for 2019-nCoV (10) . Clinicians should maintain a high index of suspicion for possible 2019-nCoV illness not only among persons with fever and lower respiratory tract illness who report travel from China in the past 14 days but also symptomatic persons who have had close contact with patients with laboratory-confirmed 2019-nCoV infection. Clinicians should consult their local and state health departments when they suspect possible 2019-nCoV illness to facilitate diagnosis and aid prevention efforts.What is already known about this topic?During a 2020 outbreak of novel coronavirus (2019-nCoV) infection, CDC provided consultation to public health officials and health care providers evaluating persons at risk for 2019-nCoV infection.What is added by this report?During January 2020, CDC responded to clinical inquiries regarding approximately 650 persons in the United States and tested 210 for 2019-nCoV, one fifth of whom reported no recent travel-related risk but had close contact with a 2019-nCoV patient or a person under investigation for 2019-nCoV in the United States.What are the implications for public health practice?Health care providers should remain vigilant regarding possible 2019-nCoV exposure not only among returning travelers, but also among persons in close contact with 2019-nCoV patients in the United States.\n",
            "\t0.489\tWe report two cases of coronavirus disease 2019 in travellers from Wuhan, China to Thailand. Both were independent introductions on separate flights, discovered with thermoscanners and confirmed with RT-PCR and genome sequencing. Both cases do not seem directly linked to the Huanan Seafood Market in Hubei but the viral genomes are identical to four other sequences from Wuhan, suggesting early spread within the city already in the first week of January.In late December 2019, an outbreak with an initially undiagnosed pneumonia was reported in the city of Wuhan, Hubei Province, China, and linked to the Huanan Seafood Market [1, 2] . The causative pathogen was identified as a novel betacoronavirus within the severe acute respiratory syndrome (SARS) coronavirus (CoV) family, recently termed SARS-CoV-2 . In response to the outbreak, several countries including Thailand, established thermal screening at the airport for travellers from Wuhan since 3 January. On 8 January and 13 January, suspected cases of infection with SARS-CoV-2 were identified at Bangkok Suvarnabhumi airport. We report the investigation, basic clinical characteristics and viral genomes derived from these cases.A woman in her early 60s from Wuhan developed a fever with chills, sore throat and headache on 5 January 2020. She went to a local health facility in China and received undisclosed medication. On 8 January 2020, she took a direct ca 4 h flight to Thailand from Wuhan, with five family members, as part of a tour group of 16 (including the case). Her measured temperature at the arrival gate was 38.6 °C by thermoscanner, and confirmed with a tympanic thermometer. After being interviewed by quarantine officers, she was transferred to Bamrasnaradura Infectious Disease Institute (BIDI) Hospital, Nonthaburi, for isolation and laboratory investigations. She reported a runny nose and sore throat but no dyspnea or diarrhoea. Upon admission, her vital signs were normal except for elevated blood pressure. Her physical examination was unremarkable including inconspicuous lung sounds. Her complete blood count suggested a viral infection from relatively decreased neutrophil (48%; norm: 35-75%) to lymphocyte (40%; norm: 20-59%) ratio . The chest X-ray (CXR) results on 8 January were compatible with pneumonia with mild thickening lung marking at both lower lungs possibly because of crowded lung rather than infiltration. It also showed borderline cardiomegaly. Repeat CXR after 7 days showed no remarkable changes.In the interview, the patient explicitly stated that she had not visited the Huanan Seafood Market but she was living near another local market, which she visited daily until illness onset. This market was at ca 7.5 km distance from the Huanan Seafood Market. The patient also reported that she had not purchased live animals and or visited stalls with live animals. She did not visit Jinyintai Hospital or other hospitals in Wuhan. However, she visited local dispensaries in Wuhan to obtain medication. She also reported no contact with persons with respiratory symptoms.Clinical specimens collected on admission included the upper respiratory tract secretions and sputum. These specimens tested positive on 12 January for the CoV family by using a conventional nested RT-PCR . Genomic sequencing analysis included Sanger and next generation sequencing were performed by the Emerging Infectious Diseases Health Science Center, the Thai Red Cross Society and the Thai National Institute of Health, Department of Medical Sciences and the sequence shared via the Global Initiative on Sharing All Influenza Data (GISAID) EpiCoV database (EPI_ISL_403962). The sequencing protocol and details are provided in the Supplementary Materials. The patient recovered after testing negative for SARS-CoV-2, and returned to China without signs and symptoms on 18 January 2020.A woman in her mid-70s from Wuhan landed at Suvarnabhumi airport on 13 January. She travelled to Thailand with three family members as part of a tour group of 20 (including the case). An airport thermoscanner detected a fever of 38.0 °C that was confirmed with a tympanic thermometer. The patient reported a sore throat, that her fever onset date was 13 January and that she had a cough for an undetermined period. The patient was hospitalised at BIDI. Upon admission, she reported mild tachypnoea, and her CXR was compatible with pneumonia. Similar to case 1, the first CXR taken on 13 January, showed thickening interstitial lung marking at both lower lung fields and both perihilar regions because of interstitial infiltration or crowded lung marking, mild cardiomegaly and dilated aorta. Follow-up CXR on 17 January additionally showed recent hazy with reticular opacities at left middle lung field. The patient was not in severe condition but stable.In the patient interview, she reported that she did not visit the Huanan Seafood Market or other markets. She also reported no contact with pigs, camels, other The phylogenetic tree was created from whole genome alignment with MAFFT using the neighbour-joining method with maximum composite likelihood (MCL) model, uniform site rates and 500 bootstrap tests using MEGA X.mammals (or areas with dead birds), or any consumption of raw or undercooked foods. She stated that she was not in contact with persons with respiratory symptoms.A conventional nested RT-PCR test of this patient was positive for the CoV family . Subsequent genome sequencing was again compatible with the SARS-CoV-2 and shared via the GISAID EpiCoV database (EPI_ISL_403963). A nasopharyngeal swab also tested positive by RT-PCR for adenovirus. The patient was no longer febrile as of 17 January, and after testing negative for the CoV family by conventional nested RT-PCR, she was discharged and she returned to China.Comparing the two genome sequences with a nonredundant selection of representatives from all known CoV families by alignment and phylogenetic tree ( Figure 1A ) shows that they belong to the SARS family of betacoronaviruses and while related to SARS-CoV (80% genome identity), they were most closely related to SARS-like bat CoV from China (88% identity) as closest known sequence at the time of emergence. Structural mapping of mutations in the spike glycoprotein between SARS CoV and the two cases of the SARS-CoV-2 reported here shows only 76% identity at the protein level ( Figure 1B ). This surface protein is critical for ACE2 host receptor interaction and is also a target of the immune response [12, 13] . Given several mutations in the binding interface, it may differ in host cell binding efficiency compared with SARS-CoV which could result in differences in virulence and transmission potential [14, 15] .The genomes of the two separate cases of coronavirus disease 2019 (COVID-19) are identical over the full length of close to 30 kb and are furthermore identical to five other sequences (four from Wuhan and one from Zhejiang); together these sequences form the largest cluster of identical cases within the early outbreak, comprising a core of at least indirectly linked cases ( Figure 2 ). Within-outbreak sequence divergence is generally low with 0-9 nt differences over the whole genome and mutations unique to individual strains are possibly related to quality differences of the samples and noise of the methods used for sequencing.Case 1 travelled in a tour group with 15 others and 40 close contacts were identified: 15 members of the tour group, 15 people sitting within two rows in front and back of the seat of case 1 on the same airplane, nine crew members and one port health officer. They wereWithin-outbreak SARS-CoV-2 sequence divergence and clusters, China and Thailand, January 2020 According to the cases' history, the two imported COVID- 19 People travelling out of the city since then may have spread the virus further before travel restrictions were enforced on 23 January .Thailand implemented measures for screening patients travelling from Wuhan since 3 January 2020 at Suvarnabhumi Airport, Don Mueang, Phuket and Chiang Mai airports, and stepped up surveillance at public and private hospitals. The two cases described here who tested positive for SARS-CoV-2 were the first confirmed exported cases from China, suggesting early international spread. Therefore, while one cannot exclude the possibility that asymptomatic cases in their incubation period would be missed and become infectious later, screening of incoming travellers from an affected area proved successful at least in these two instances, when COVID-19 cases were detected, isolated, observed and only discharged once they tested negative for SARS-CoV-2. Rapid response including genome sequencing and sharing via GISAID [18, 19] () enabled fast dissemination of results and provided a glimpse of early transmission patterns in this outbreak.\n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.989\tIn December 2019, a new type viral pneumonia cases occurred in Wuhan, Hubei Province; and then named \"2019 novel coronavirus (2019-nCoV)\" by the World Health Organization (WHO) on 12 January 2020. For it is a never been experienced respiratory disease before and with infection ability widely and quickly, it attracted the world's attention but without treatment and control manual. For the request from frontline clinicians and public health professionals of 2019-nCoV infected pneumonia management, an evidence-based guideline urgently needs to be developed. Therefore, we drafted this guideline according to the rapid advice guidelines methodology and general rules of WHO guideline development; we also added the first-hand management data of Zhongnan Hospital of Wuhan University. This guideline includes the guideline methodology, epidemiological characteristics, disease screening and population prevention, diagnosis, treatment and control (including traditional Chinese Medicine), nosocomial infection prevention and control, and disease nursing of the 2019-nCoV. Moreover, we also provide a whole process of a successful treatment case of the severe 2019-nCoV infected pneumonia and experience and lessons of hospital rescue for 2019-nCoV infections. This rapid advice guideline is suitable for the first frontline doctors and nurses, managers of hospitals and healthcare sections, community residents, public health persons, relevant researchers, and all person who are interested in the 2019-nCoV.In December 2019, the 2019 novel coronavirus (2019-nCoV) was discovered and identified in the viral pneumonia cases that occurred in Wuhan, Hubei Province, China; And then was named by the World Health Organization (WHO) on 12 January 2020. In the following month, the 2019-nCoV quickly spreading inside and outside of Hubei Province and even other countries. What's more, the sharp increase of the case number caused widespread panic among the people.Medical professionals require an up-to-date guideline to follow when an urgent healthcare problem emerging. In response to the requests for reliable advice from frontline clinicians and public healthcare professionals managing 2019-nCoV pandemics, we developed this rapid advance guideline, involving disease epidemiology, etiology, diagnosis, treatment, nursing, and hospital infection control for clinicians, and also for public health workers and community residents.This guideline was prepared in accordance with the methodology and general rules of WHO Guideline Development and the WHO Rapid Advice Guidelines [1, 2] .This guideline development group is multidisciplinary and composed of individuals from health professionals and methodologists. Health professionals included frontline clinical doctors, nurses who work in departments of respiratory medicine, fever clinic, critical medicine, emergency, infectious disease, and experts of respiratory infectious disease and hospital management board. The methodologists included methodologists of guideline development, systematic review, and literature searching professionals.This guideline is suitable for frontline doctors and nurses, managers of hospitals and healthcare sections, healthy community residents, personnel in public healthcare, relevant researchers, and all persons who are interested in the 2019-nCoV management.This guideline is aimed to serve the healthcare professionals to tackle the suspected 2019-nCoV infected cases, confirmed 2019-nCoV infected cases, clustered 2019-nCoV infected cases, and those with close contacts or suspicious exposure to 2019-nCoV infected cases.Oral inquiry for financial interests of relevant personal was conducted at the first meeting while to start this guideline. Relevant financial as well as nonfinancial interests were surveyed and disclosed and subsequently assessed in consensus conference in order to minimize potential bias in guideline development. Finally, there is no conflict of interests for all the personnel participating to prepare this guideline.This guideline is a rapid guideline to responding to the emerging infectious disease of 2019-nCoV. Due to the urgent need and tight work schedule, we conducted no wide-range survey but a discussion meeting with front-line clinicians who managed patients with 2019-nCoV infections to finalize guideline topics and key questions.2.6 Literature searching and preparation of evidence profiles 2.6.1 General notesConsidering the lack of direct evidence for this newly identified 2019-nCoV infection, we searched and referred to the guidelines that related to the SARS (Severe Acute Respiratory Syndrome), MERS (Middle East Respiratory Syndrome), and influenza. We also referred to the guidelines which were newly-issued by the National Health Commission of People's Republic of China and WHO for 2019-nCoV. In addition, we have an independent literature searching team to search available indirect evidence from systematic reviews and/or RCTs (randomized controlled trials), which were for the treatment and/ or chemoprophylaxis of SARS, MERS, or other influenza virus infections.If the existing evidence addressed topics or questions covered by the guideline, then its quality should be assessed. If there is a lack of higher-level quality evidence, our panel considered observational studies and case series. Because of the limited time, we did not perform new systematic review. We identified relevant literature up to 20 January 2020.We searched the bibliographic databases: PubMed, Embase, and Cochrane library.We also searched following websites: the WHO (), CDC (Centers for Disease Control and Prevention, ), NICE (National Institute for Health and Clinical Excellence, ), National Health Commission of the People's Republic of China (. cn/), and National Administration of Traditional Chinese Medicine ().As the 2019-nCoV is a newly identified pathogen responsible for the outbreak of the pandemic disease, there is no sufficient evidence to reveal the whole nature of this virus. In these situations, obtaining evidence from the experts who fighting the disease on the frontline can be efficient and the main source .Until to 24:00 on 29 January 2020, 11,500 persons were screened, and 276 were identified as suspected infectious victims, and 170 were diagnosed (including 33 in critical condition) for 2019-nCoV infection in Zhongnan Hospital of Wuhan University. During this process, frontline clinicians and nurses have accumulated valuable experience in the diagnosis, treatment and nursing for 2019-nCoV infected patients. Hence, these experiences were assessed and then used as \"Expert Evidence\" for our guideline development. We took interviews and group surveys to collect information on treatment evidence during the guideline panel's meeting, so that it could be integrated into the guideline panel in the summary of findings (see Additional files 1 and 2). Experts' evidence can be solicited by the description of case reports, summaries, and reports of topics or questions of all cases they management.We accorded to the Grading of Recommendations Assessment, Development and Evaluation (GRADE) basic approaches and rules [4, 5] , and particularly considered experts' evidence to assess the quality of a body of evidence to make recommendations.The quality of evidence reflects whether the extent to which our confidence estimating the effect is adequate to support a particular recommendation. The level of evidence was categorized as \"high quality\", \"moderate quality\", \"low quality\", or \"very low quality\"; Recommendations were classified as \"strong\" or \"weak.\"The strong recommendation does not always mean there is sufficient intervention effectiveness. Besides the effectiveness of intervention, the forming of recommendations is based on the severity of the disease, patient willingness, safety, and economics . See Tables 1 and 2 [4, 6] .Before meetings, experts' evidence was collected systematically and available to panel members. Once the evidence has been identified and assessed, recommendations were formulated based on the evidence by a face-to-face meeting of panel members and supplemented by experts participating in the panel meeting.Experts' evidence was highly valued in this guideline development. During the consensus process, if the evidence was agreed on by more than 70% frontline clinicians in the consensus meeting, it is considered as highquality evidence.In specific recomendations, we used \"should\" or \"strongly recommend\" for strong recommendations; whereas, \"suggest\" or \"consider\" was used for weak ones.This guideline was published in both Chinese and English versions at the same time. Due to space limitations, the current standard revision does not include evidence descriptions. The full revision will be published in New Medicine (Chinese name: Yixue Xinzhi; . jnewmed.com/), Volume 30 and Issue 1 2020 .Since December 2019, multiple cases occurring unexplainable pneumonia were successively reported in some hospitals in Wuhan city with a history of exposure to a large Hua'nan seafood market in Wuhan city, Hubei province, China. It has been confirmed to be an acute respiratory infection caused by a novel coronavirus. So far, the number of cases without a history of the Hua'nan seafood market exposure is increasing. In addition, clustered cases and confirmed cases without a history of travel to Wuhan emerged. Also, confirmed cases without clear exposure to the Wuhan seafood market have been found in many foreign countries or regions .At 24:00 on 26 January 2020, the National Health Commission of the People's Republic of China has recorded a total of 2744 confirmed cases of pneumonia with 2019-nCoV infection from 30 provinces (districts and cities), including 461 severe cases and 80 deaths. A total of 51 cases have been cured and discharged. At present, 5794 suspected cases were recorded, 32,799 with close contacts to the confirmed patients have been tracked, 583 people were released from medical observation that day, and 30,453 people were still undergoing medical observation. A total of confirmed cases were reported from Hong Kong, Macao and Taiwan of China: 8 cases in Hong Kong, 5 cases in Macao, and 4 cases in Taiwan. In addition, confirmed cases had been reported from abroad: 7 in Thailand, 4 in Australia, 4 in Singapore, 3 in France, 3 in Japan, 3 in Korea, 3 in Wild animal, bats is the most possible host of the 2019-nCoV. It requires further confirmation whether pneumonia infected by the 2019-nCoV is transmitted directly from bats or through an intermediate host. It is believed that clarifying the source of the virus will help determine zoonotic transmission patterns .Up to present, the main infection source was the patients who with pneumonia infected by the 2019-nCoV. Respiratory droplet transmission is the main route of transmission, and it can also be transmitted through contact . Although many details, such as the source of the virus and its ability to spread between people remain unknown, an increasing number of cases show the signs of human-tohuman transmission [8, 13] .The 2019-nCoV isolated from the lower respiratory tract of patients with unexplainable pneumonia in Wuhan, and it is a novel coronavirus belonging to the β genus. The 2019-nCoV has an envelope; its particles are round or oval, often polymorphic, with a diameter from 60 nm to 140 nm. Its genetic characteristics are significantly different from SARSr-CoV (SARS related coronaviruses) and MERSr-CoV (MERS related coronaviruses). Current research shows it has more than 85% homology with SARSr-CoV (bat-SL-CoVZC45). 2019-nCoV can be found in human respiratory epithelial cells 96 h after in vitro isolation and culture, while it takes about 6 days in VeroE6 or Huh-7 cell lines . The source of the virus, the time span of the patients discharging infective virus, and the pathogenesis are still not clear .No evidence of viral mutation has been found so far . It is necessary to obtain much more clinically isolated viruses with time and geographical variety to assess the extent of the virus mutations, and also whether these mutations indicate adaptability to human hosts .Based on currently epidemiological survey, the latency period is generally from 3 to 7 days, with a maximum of 14 days . Unlike SARSr-CoV, 2019-nCoV is contagious during the latency period . The evidence agreed on by more than 70% frontline clinicians in consensus meeting is viewed as high-quality evidenceThe population is generally susceptible to the virus. The elderly and those with underlying diseases show more serious conditions after infection, and children and infants also get infected by the 2019-nCoV. From current knowledge of the cases, most patients have a good prognosis, the symptoms of children are relatively mild, and a few patients are in critical condition. Death cases are more frequently seen in the elderly and those with chronic underlying diseases . The newest study including the first 41 confirmed cases admitted to Wuhan between 16 December 2019 and 2 January 2020 showed the median age of patients was 49 years; and the main underlying diseases were diabetes, hypertension, and cardiovascular diseases. Of them, 12 cases experienced acute respiratory distress syndrome (ARDS), 13 cases were admitted to the intensive care unit (ICU), and 6 cases died . Patients with any 2 of the following clinical features and any epidemiological risk: (1) clinical features: fever, imaging features of pneumonia, normal or reduced white blood cell count, or reduced lymphocyte count in the early stages of the disease onset. (2) epidemiologic risk: a history of travel to or residence in Wuhan city, China or other cities with continuous transmission of local cases in the last 14 days before symptom onset; contact with patients with fever or respiratory symptoms from Wuhan city, China or other cities with continuous transmission of local cases in the last 14 days before symptom onset; or epidemiologically connected to 2019-nCoV infections or clustered onsets .Those with one of the following pathogenic evidence is the confirmed case: (1) positive for the 2019-nCoV by the real-time PCR test for nucleic acid in respiratory or blood samples . 2) viral gene sequencing shows highly homogeneity to the known 2019-nCoV in respiratory or blood samples .Suspected clustering cases are defined when one confirmed case and at the same time, one or more cases of fever or respiratory infection are found in a small area (such as a family, a construction site, a unit, etc.) within 14 days.Under the above circumstances, 2 or more confirmed cases are found, and there is the possibility of human-tohuman transmission due to close contact or infection due to co-exposure, then it is determined as a clustered case [8, 18] .Those who have one of the following contacts after the onset of confirmed cases in the absence of effective protection : (1) those who live, study, work, or have close contact with the confirmed cases, or other close contacts such as working closely with or sharing the same classroom or living in the same house with the confirmed case. (2) medical and nursing staffs and their family members living with them, who treated, nursed or visited the confirmed case, or other personnel who have similar close contact with the case, such as providing direct treatment or care of the case, visiting the case or staying in a closed environment where the cases are located; other patients or caregivers in the same room with the case. (3) people who have close contact with the patients in a same transport vehicle, including those who had taken care of the patients on the vehicle; the person who had companied the patients (family members, colleagues, friends, etc.); other passengers and traffic staff considered having likely close contact with the patients by investigation and evaluation. (4) other circumstances considered to be closely contacted with the person with close contact with the patients by the professional investigation and evaluation.Persons with suspicious exposure are those who are exposed without effective protection to processing, sales, handling, distributing, or administrative management of wild animals, materials, and the environments that are positive for the 2019-nCoV test .Persons with close contacts and suspicious exposure should be advised to have a 14-day health observation period, which starts from the last day of contact with the 2019-nCoV infected patients or suspicious environmental exposure. Once they display any symptoms, especially fever, respiratory symptoms such as coughing, shortness of breath, or diarrhea, they should reach out for medical attention immediately . Contact surveillance should be carried out for those who had exposed to accidental contact, low-level exposure to suspected or confirmed patients, i.e. checking any potential symptoms when carrying out daily activities . See Table 3 for details .Patients with a suspected infection should be isolated, monitored, and diagnosed in hospital as soon as possible. Doctors should make recommendations based on the patient's situation. Patients with mild symptoms and suspected infection may consider in-home isolation and home care (weak recommendation). Suspected infected with severe symptoms and those who need to stay in hospital for observation by doctor's judgment should follow the isolation guidelines for suspected patients (see Tables 4 and 5 for details).It should also be noted that: (1) whether the suspected patients should be given in-home isolation and care or not requires careful clinical evaluation and safety assessment by professionals. (2) if the suspected patients do not get improvement in the symptoms or even worsened in condition during home care, they need to go to the doctor for treatment. (3) during the period of home care, the patients' medication and symptoms should be closely recorded and their caregivers should also monitor their body temperature daily.Throughout the period of home care, healthcare personnel should perform regular (e.g., daily) follow-up through face-to-face visits or phone interviews (ideally, if feasible) to follow the progress of symptoms and, if necessary, specific diagnostic tests should be conducted [14, 19, 21] .International visitors should take routine precautions when entering and leaving the affected areas, including avoiding close contacts with people with acute respiratory infection, washing hands frequently, especially after contacting with the sick or their surrounding environment; following appropriate coughing etiquette; and avoiding close contact with live or dead farming animals or bats or other wild animals [22, 23] . Passengers should avoid unnecessary travel as possible.If they had travelled to Hubei Province (especially Wuhan city) in the past 14 days and is in fever, cough or difficulty in breathing, they should: (1) see a doctor immediately; (2) call the doctor about his/her recent trips and symptoms before going to the doctor's office or emergency room; (3) avoid contact with others; (4) not to travel around; (5) cover mouth and nose with a tissue or sleeve (not hands) when coughing or sneezing; and (6) wash hands with soap and water for at least 20 s. If soap and water are not available, use alcohol-based hand sanitizers .The 2019-nCoV infected cases have symptoms like fever, fatigue, dry cough, dyspnea etc., with or without nasal congestion, runny nose or other upper respiratory symptoms [13, 16] . Despite the atypical symptoms were reported , Nan-Shan Zhong, the academician of Chinese Academy of Engineering in an exclusive interview with Xinhua News Agency on 28 January 2020, pointed out that fever is still the typical symptom of 2019-nCoV infection.Patients with mild symptoms may not present positive signs. Patients in severe condition may have shortness of breath, moist rales in lungs, weakened breath sounds, dullness in percussion, and increased or decreased tactile speech tremor, etc.The imaging findings vary with the patient's age, immunity status, disease stage at the time of scanning, underlying diseases, and drug interventions. When walking on the road or waiting in the hospital, try to stay away from other people (at least 1 m away) and wear a mask.The family members accompanying those for inspection should immediately follow the monitoring recommendations to close contacts, keep the respiratory hygiene and clean their hands properly.The community or street hospital should be informed before the suspected contacts to the hospital. The vehicle used should be cleaned and disinfected with 500 mg/L chlorine-containing disinfectant, and the window should be opened for ventilation. The imaging features of lesions show: (1) dominant distribution (mainly subpleural, along the bronchial vascular bundles); (2) quantity (often more than three or more lesions, occasional single or double lesions); (3) shape (patchy, large block, nodular, lumpy, honeycomblike or grid-like, cord-like, etc.); (4) density (mostly uneven, a paving stones-like change mixed with ground glass density and interlobular septal thickening, consolidation and thickened bronchial wall, etc.); and (5) concomitant signs vary (air-bronchogram, rare pleural effusion and mediastinal lymph nodes enlargement, etc.).Typical CT/X-ray imaging manifestation, including(1) Multiple, patchy, sub-segmental or segmental groundglass density shadows in both lungs. They were classified as \"paving stone-like\" changes by fine-grid or small honeycomb-like thickening of interlobular septa. The thinner the CT scan layers, the clearer the ground-glass opacity and thickening of interlobular septa were displayed. A slightly high-density and ground-glass change with fuzzy edge in the fine-grid or small honeycomb-like thickening of interlobular septa were presented by the high-resolution computed tomography (HRCT), ( Fig. 1: 45 cases, 54.2% in a total of 83 cases). The resolution of X-ray was worse lower than that of CT in the resolution, which was basically manifested as ground-glass When coughing or sneezing, it is necessary to wear a medical mask, or cover with a paper towel and bent elbow, and clean hands immediately after coughing and sneezing.Strong 10 N95 masks should be worn in the same room with patients (preferred strategy).Disposable surgical mask (alternative strategy). Use the mask in strict accordance with the instruction manual.After washing hands with running water, dry them with a paper towel (preferred strategy).Dry with a towel, and wash and disinfect the towel daily (alternative strategy).Home caregivers 1 Clean and disinfect hands after contact with the patient, before leaving patient's room or the house, before and after eating, after using the toilet and after entering house from outside (for visible contaminant on hands, wash hands with running water then use hand disinfection).Avoid direct contact with patient's secretions or discharges, especially oral or respiratory discharges; avoid direct contact with patient's feces. (2) Multiple, patchy or large patches of consolidation in both lungs, with a little grid-like or honeycombshaped interlobular septal thickening, especially in the middle and lower lobes ( Fig. 3: 26 cases, 31.3% in a total of 83 cases). It was more common in the elderly or severe condition patients.Atypical CT/X-ray imaging manifestation, including (1) Single, or multiple, or extensive subpleural grid-like or honeycomb-like thickening of interlobular septum, thickening of the bronchial wall, and tortuous and thick strand-like opacity. Several patchy consolidations, occasionally with a small amount pleural effusion or enlargement of mediastinal lymph nodes, can be seen ( Fig. 4 : 6 cases, 7.2% in a total of 83 cases). This is mostly seen in the elderly.(2) Single or multiple solid nodules or consolidated nodules in the center of lobule, surrounded by ground-glass opacities ( Fig. 5 : 5 cases, 6.2% in a total of 83 cases).Stage based on CT image The CT imaging demonstrates 5 stages according to the time of onset and the response of body to the virus, including:(1) Ultra-early stage. This stage usually refers to the stage of patients without clinical manifestation, negative laboratory test but positive throat swab for 2019-nCoV (2) Early stage.This stage refers to the period of 1-3 days after clinical manifestations (fever, cough, dry cough, etc.). The pathological process during this stage is dilatation and congestion of alveolar septal capillary, exudation of fluid in alveolar cavity and interlobular interstitial edema. It showed that single or multiple scattered patchy or agglomerated ground-glass opacities, separated by honeycomb-like or grid-like thickened of interlobular septa ( Fig. 7 : 45 cases, 54.2% in a total of 83 cases). It mainly should be distinguished from other known viral virus of pneumonia, such as influenza viruses, parainfluenza virus, adenovirus, respiratory syncytial virus, rhinovirus, human metapneumovirus, SARSr-CoV, etc.; and also from mycoplasma pneumonia, chlamydia pneumonia, and bacterial pneumonia. In addition, it should be distinguished from non-infectious diseases, such as vasculitis, dermatomyositis, and organizing pneumonia.In the early stage of the disease, the total number of leukocytes decreased or keeps normal, with decreased lymphocyte count or increased or normal monocytes. High attention should be paid on the situation where the absolute value of lymphocyte is less than 0.8 × 10 9 /L, or the numbers of CD4 and CD8 T cells are significantly decreased, which generally recommend rechecking the blood routine changes after 3 days.(1) Flu antigens. At present, routinely detected flu antigens are A, B, and H7N-subtypes. Sampling of throat swabs is conducive to early rapid screening for flu because of the fast test, but it has a relatively high false negative rate. (2) Respiratory virus nucleic acid. The detection of respiratory virus nucleic acid is commonly used to In the early stage of this disease, the total number of leukocytes in peripheral blood was normal or decreased, and the lymphocyte count decreased. In some patients, liver enzyme (transaminase), creatine kinase (CK) and myoglobin increased. CRP, ESR, and IL-6 increased, and PCT was normal in most patients. The increased Ddimer occurred in severe cases. Compared with 120 healthy check-ups, the absolute value of lymphocyte (0.87 vs 2.13) × 10 9 /L, lymphocyte percentage (19.5% vs 33.7%), eosinophil percentage (0.13% vs 2.16%), and absolute value (0.0061 vs 0.1417) × 10 9 /L in 2019-nCoV patients were significantly reduced (P < 0.05). The absolute number (4.2 vs 3.7) × 10 9 /L and the percentage (72.0% vs 57.0%) increased in 2019-nCoV patients (P < 0.05). The percentage of monocytes increased slightly (8.1% vs 6.8%), while the absolute number of monocytes did not change significantly (0.38 vs 0.44) × 10 9 /L.The next generation sequencing (NGS) and electron microscope technology play a role in the early diagnosis, but their diagnostic values have been weakened by the discovery of specific nucleic acid detection technology. In addition, NGS detection can tell whether the pathogen has mutated or not.Suspected and confirmed cases need to be treated in designated hospitals with effective isolation and protection conditions. Suspected cases need to be treated separately in single room, confirmed cases are admitted to a same ward, and critical cases should be admitted to ICU as soon as possible.(1) The patient should rest in bed, being monitored for vital signs (heart rate, pulse oxygen saturation, respiratory rate, blood pressure) and given supportive treatment to ensure sufficient energy intake and balance for water, electrolytes, acidbase levels and other internal environment factors (Strong recommendation).(2) The patient should be monitored for blood routine, CRP, PCT, organ function (liver enzyme, bilirubin, myocardial enzyme, creatinine, urea nitrogen, Urine volume, etc.), coagulation function, arterial blood gas analysis and chest imaging (Strong recommendation). First, oxygen therapy is the choice for patients with severe respiratory infections, respiratory distress, hypoxemia or shock. The initial flow rate is 5 L/min, and the titration flow rate is to reach the target oxygen saturation (adults: SpO 2 ≥ 90% in non-pregnant patients, SpO 2 ≥ 92-95% in pregnant patients; children: SpO 2 ≥ 94% in children with obstructive dyspnea, apnea, severe respiratory distress, central cyanosis, shock, coma or convulsions, and ≥ 90% in other children).Second, respiratory support should be given to patients with hypoxic respiratory failure and acute respiratory distress syndrome. HFNO or NIV can be selected when nasal cannula or mask oxygen therapy was ineffective or the patient had hypoxic respiratory failure. However, when patients had hypercapnia (acute exacerbation of chronic obstructive pulmonary disease, cardiogenic pulmonary edema), hemodynamic instability, multiple organ failure, and abnormal mental status HFNO oxygen is not the routinely adopted measure. If respiratory failure cannot be improved or worsens continuously within a short time (1 h) after using HFNO or NIV, intubation should be performed immediately. Low tidal volume (4-8 ml/kg) and low suction pressure (platform pressure < 30cmH 2 O) are used for invasive mechanical ventilation. It is suggested that positive endexpiratory pressure (PEEP) with high positive endexpiratory pressure should be used in patients with moderate or severe acute respiratory distress syndrome, and PEEP should be titrated according to FiO 2 to maintain SpO 2 , in order to improve alveolar atelectasis and reduce alveolar hyper-expansion and pulmonary vascular resistance at the end of inspiration. For severe patients with ARDS, it is recommended to ventilate in prone position for more than 12 h/d.should be considered for the patients with refractory hypoxemia that is difficult to be corrected by protective lung ventilation. (Strong recommendation).(1) At present, there is no evidence from RCT to support specific drug treatment against the new coronavirus in suspected or confirmed cases. (2) The α-interferon atomization inhalation can be considered (5 million U per time for adults in sterile injection water, twice a day) (Weak recommendation); lopinavir/ritonavir orally, 2 capsules each time, twice a day, can be also considered (Weak recommendation).Low-level evidences included retrospective cohort, historically controlled studies, case reports, and case series revealed that lopinavir/ritonavir alone or in combination with antivirals produced certain benefits in the treatment of SARS and MERS, such as reducing the incidence or mortality of ARDS . A recently systematic review showed that lopinavir/ritonavir's anti-coronavirus effect was mainly seen in its early application, for reducing patient mortality and reduced glucocorticoid consumption. However, if the early treatment window is missed, there will be no significant effect in their late application . Real-world study stills need to further explore the clinical effects of its early use in 2019-nCoV infected pneumonia.The effectiveness of the combined use of antivirals is still controversial .(1) Principles. Avoid blind or inappropriate use of antibacterial drugs, especially the combination of broad-spectrum antibacterial drugs. Enhancement of bacteriological surveillance should be performed and promptly given appropriate antibacterial drugs when it occurs secondary bacterial infection. (2) According to the clinical manifestations of patients, if the accompanying bacterial infection cannot be ruled out, mild patients can take antibacterial drugs against community-acquired pneumonia, such as amoxicillin, azithromycin, or fluoroquinolones; empirical antibacterial treatment in severe patients should cover all possible pathogens, deescalating therapy until the pathogenic bacteria are clarified.The use of corticosteroids for severe ARDS is controversial; therefore, systemic use of glucocorticoids needs to be cautious. Methylprednisolone can be used as appropriate for patients with rapid disease progression or severe illness. According to the severity of the disease, 40 to 80 mg of methylprednisolone per day can be considered, and the total daily dose should not exceed 2 mg/kg (Weak recommendation). SARS management related researches showed that timely use of non-invasive continuous positive airway pressure and corticosteroids is an effective strategy for increased lung shadows and increased dyspnea. Appropriate use of glucocorticoids is able to significantly improve the clinical symptoms of patients with SARS, reduce the degree of disease progression, and accelerate the absorption of lung lesions; but it cannot shorten the length of hospital stay [35, 36] . Be cautious that hormone therapy has some incidence of adverse reactions .(1) Symptomatic treatment of fever. When the temperature is higher than 38.5 ℃, ibuprofen can be used for Treat the patient based on syndrome differentiation individually. Prevention before illness is better than treatment after getting diseased.(1) Community. Implement relevant national regulations and take great effort to keep away from contaminated materials, disinfect the environment, and improve the healthcare management. i Fumigation with moxa in the room, 1-5 g/m 2 for 30 min per day. ii Wearing perfumed Chinese herb bags (clove, fineleaf schizonepeta herb, Perilla frutescens, atractylodes lancea, cinnamon, biond magnolia flower, asarum sieboldii, and Elettaria cardamomum, 2 g for each, crushed into powder and put it into bags for external use, change a new one every 10 days). iii Prescription of Chinese herbs for feet bath (vulgaris 10 g, carthamus 10 g, and dried ginger 6 g) Soaking the herbs in boiling water and bath the feet into the medical liquid when the temperature is suitable. Soak feet for about 20 min. iv Prescription of Chinese herbs for prophylaxis:Astragalus mongholicus 12 g, roasted rhizoma atractylodis macrocephalae 10 g, saposhnikovia divaricata 10 g, Cyrtomium fortunei 10 g, honeysuckle 10 g, dried tangerine or orange peel 6 g, eupatorium 10 g, and licorice 10 g. Taking the medicine above yielded decoction once a day for adults, and for 5 days as a treatment course. If for children, cutting the dose to half. v Medical tea: perilla leaf 6 g, agastache leaf 6 g, dried tangerine or orange peel 9 g, stewed amomum tsao-ko 6 g, and 3 slices of ginger. Soak the herbs in hot water and drink the water just like enjoying the tea. vi Chinese patent medicine: Huoxiang Zhengqi capsule or Huoxiang Zhengqi Shui (in half dose).In medical observation period There are two clinical symptoms in this period, including:(1) Clinical symptoms 1: hypodynamia accompanied by gastrointestinal upset. And the recommended Chinese patent medicine is the Huoxiang Zhengqi capsules (ball, liquid, or oral liquid).(2) Clinical symptoms 2: hypodynamia and fever. And the recommended Chinese patent medicines is the Jinhua Qinggan granules, Lianhua Qingwen capsules (granules), Shufeng Jiedu capsules (granules), or Fangfeng Tongsheng pills (granules).Clinical treatment period This period involving 7 stages, including:(1) Early-stage, characterized as exterior syndrome of cold-dampness. In this stage, the clinical manifestations presents as follow: aversion to cold without sweating, headache and generalized heaviness, limb pain, glomus and fullness in the chest and diaphragm, thirst with no desire to drink, ungratifying loose stool, yellow urine, frequent micturition and yellow urine. The therapeutic logic is to dissipate cold and eliminate dampness. And the recommended prescription is the Huoxiang Zhengqi powder (Yin dampness injuring superficies case from the National Famous Traditional Chinese Medical Doctor Medical Cases); which comprises of perilla leaf 10 g, atractylodes lancea 15 g, radix angelicae dahuricae 10 g, dried tangerine or orange peel 10 g, notopterygium root 10 g, agastache rugosus 10 g (end addition), mangnolia officinalis 10 g, saposhnikovia divaricata 10 g, poria peel 15 g, and Tetrapanax papyriferus 10 g above yielded decoction. In addition, the recommended Chinese patent medicine is Huoxiang Zhengqi capsules or Huoxiang Zhengqi Shui. (2) Early-stage, characterized as cold-dampness obstructing lung. In this stage, the clinical manifestations presents as follow: aversion to cold with or without fever, dry cough, dry throat, fatigue and hypodynamia, oppression in chest, epigastric fullness, or nausea, loose stool. The tongue is pale or reddish, the tongue fur is slimy white, and soggy pulse. Hence, the therapeutic logic is to dissipate cold and resolve obstruction. And the recommended prescriptions comprises of atractylodes lancea 15 g, dried tangerine or orange peel 10 g, mangnolia officinalis 10 g, agastache rugosus 10 g (end addition), amomum tsao-ko 6 g, ephedra herb 6 g, notopterygium root 10 g, ginger 10 g, areca-nut 10 g (end addition), periostracum cicada 10 g, bombyx batryticatus 10 g, and rhizoma curcumae longae 10 g above yielded decoction. (3) Middle-stage, characterized as epidemic toxin blocking the lung. In this stage, its clinical manifestations includes persistent fever or alternating cold and heat, cough with less phlegm, or yellow phlegm, abdominal distension and constipation; oppression in chest with anhelation, cough with wheezes, panting on exertion; or red tongue, slimy yellow fur or yellow dry fur, slippery and rapid pulse. Therefore, the therapeutic logic is clearing heat and detoxicating. And the recommended prescription comprises of almond 10 g, gypsum 30 g (predecoction), trichosanthes kirilowii 30 g, rhubarb 6 g (end addition), ephedra with honey fried 6 g, semen lepidii 10 g, peach kernel 10 g, amomum tsao-ko 6 g, arecanut 10 g, and atractylodes lancea 10 g above yielded decoction.In addition, the recommended Chinese patent medicine is Xiyanping injection or Xuebijing injection. (4) Severe stage, characterized as heat toxin generating stasis. In this stage, the clinical manifestations is known as high fever, oppression in chest with anhelation, purple-black facial complexion, lips dark and swollen, obnubilation, crimson tongue, yellow dry fur, surging and fine rapid stringlike pulse. Thus, its therapeutic logic is detoxicating and dispersing blood stasis. The recommended prescription is three Yellows and Gypsum decoction, Shang Jiang Powder, and Toxin-Resolving Blood-quickening decoction. Its composition comprises of ephedra with honey fried 10 g, almond 10 g, gypsum 20-30 g, periostracum cicada 10 g, bombyx batryticatus 10 g, rhizoma curcumae longae 10 g, rhubarb stir-fried with wine 10 g, scutellaria baicalensis 10 g, coptis chinensis 5 g, phillyrin 15 g, angelica sinensis 10 g, peach kernel 10 g, radix paeoniae rubra 15 g, and rhizome of rehmannia 15 g above yielded decoction. The recommended Chinese patent medicines is the Xiyanping injection, Xuebijing injection, Qingkailing injection, or Angong Niuhuang pills. (5) Severe-stage, characterized as inner blocking causing collapse. In this stage, the clinical manifestations include dyspnea, panting on exertion or need assisted ventilation, accompanied by coma, and agitation, cold limbs with cold sweating, dark purple tongue, thick or dry thick tongue fur, floating and rootless pulse. The thrapeutic logic is rescuing from collapse by restoring Yang. Hence, the recommended prescription comprises of ginseng 15 g, aconitine 10 g (predecoction), and Cornus officinalis 15 g above yielded decoction, and both taken with fluid Suhexiang pills or Angong Niuhuang pills. The recommended Chinese patent medicines is Xuebijing injection, Shenfu injection, or Shengmai injection. (6) Recovery-stage, presents as lung and spleen Qi deficiency. Its clinical manifestations include shortness of breath, fatigue and hypodynamia, anorexia, nausea and vomiting, glomus and fullness, weak stools, ungratifying loose stool, pale tender-soft enlarged tongue, slimy white tongue fur. Therefore, therapeutic logic is to supplement the spleen and lung. The recommended prescription comprises of rhizoma pinellinae praeparata 9 g, dried tangerine or orange peel 10 g, Codonopsis pilosula 15 g, radix astragali preparata 30 g, poria cocos 15 g, agastache rugosus 10 g, and fructus amomi 6 g (end addition) above yielded decoction. In addition, the recommended Chinese patent medicines is pill of costus and amomum with six noble ingredients. (7) Recovery-stage, characterized as deficiency of Qi and Yin. The clinical manifestations of this stage is generalized heat with sweating, chest heat vexation, Qi counterflow with retching and vomiting, shortness of breath and lassitude of essence-spirit, red tongue and thin tongue fur, vacuous pulse. Hence, the therapeutic logics is boost Qi and nourish Yin. The recommended prescription is Zhuye Shigao decoction with cogongrass rhizome and rhizoma phragmitis; and the composition of this prescription includes bamboo leaf 15 g, gypsum 15 g (predecoction), Codonopsis pilosula 15 g, radix ophiopogonis 10 g, pinellia ternate 9 g, cogongrass rhizome 15-30 g, rhizoma phragmitis 20 g, licorice 10 g, and polished round-grained rice 30 g above yielded decoction. The recommended Chinese patent medicine: Shengmaiyin.6.5 Treatment of severe patients 6.5.1 Hypoxemic respiratory failure and ARDS treatments Treatment principle: treat the patients to improve the symptoms and underlying diseases, actively prevent potential complications and secondary infection; provide timely measures to support organ function.(1) Hypoxic respiratory failure and severe ARDS. Give oxygen therapy immediately to patients with ARDS, and closely monitor them for signs of clinical deterioration, such as rapidly progressive respiratory failure. Consider severe hypoxemic respiratory failure when standard oxygen therapy fails. Conservative fluid management can be adopted for ARDS patients without tissue hypoperfusion. Use vasoactive drugs to improve microcirculation. Empirical antibiotics targeting the suspected potential infection should be used as soon as possible, blind or improper combination of broad-spectrum antibiotics should be avoided. Unless for special reasons, the routine use of corticosteroids should be avoided. Glucocorticoids can be used in a short time (3-5 days) according to the degree of dyspnea and the progress of chest imaging if appropriate and the recommended dose is not more than the equivalent to 1-2 mg/kg methylprednisone per day. Provide intensive standard supportive care to the critically ill patients, including prevention of deep vein thrombosis and stress-induced gastrointestinal bleeding, blood glucose control and so on. Enteral nutrition can be provided. Supplemental nutrition with omega-3 fatty acids and antioxidants is not recommended. Inhaled or intravenous beta-adrenergic agonists are not recommended to promote alveolar fluid clearance and resolution of pulmonary edema.(1) Recognize the septic shock. When infection is suspected or confirmed, and on the basis of full fluid resuscitation, vasoconstrictor drugs are still needed to maintain mean arterial pressure (MAP) ≥65 mmHg with lactate ≥2 mmol/L, the existence of septic shock should be considered. If lactate cannot be monitored for some reasons, the following three manifestations (changes in mental state, oliguria, poor peripheral perfusion and prolonged capillary filling time) should be considered as signs of a combination of infection and hypoperfusion. (2) In resuscitation from septic shock in adults, at least 30 ml/kg of isotonic crystalloid was considered for adults in the first 3 h. In resuscitation from septic shock in children, give 20 ml/kg as a rapid bolus and up to 40-60 ml/kg in first aid. (3) Isosmotic crystal solution is recommended for resuscitation. Do not use hypotonic crystalloids, starches, or gelatins for resuscitation in the first hour. Albumin may be considered as a resuscitation fluid, but this recommendation was based on low-quality evidence under certain conditions. (4) Administer vasoconstrictor is suggested when shock persists after fluid resuscitation, noradrenaline as the first choice. The initial blood pressure target is MAP ≥65 mmHg in adults and age-appropriate targets in children. can also be administered via intraosseous needles.6.6 Condition evaluation and treatment effect evaluation 6.6.1 Criteria to withdraw ECLS(1) Remove VV-ECMO. The oxygen concentration of the ECMO air-oxygen mixer has dropped to 21%, the air flow rate has dropped to 0, and the ventilator is not strong enough. Lasting for 2-3 h, the respiratory rate is within 25 breaths/min, SpO 2 > 92%, PaCO 2 is normal, and withdrawal from VV-ECMO may be considered. (2) Remove VA-ECMO. The blood flow rate is reduced to the rate of (0.2 to 0.5 L / min) every 5 to 6 h from 3 L/min, and the hemodynamic condition is stable. The blood flow rate is reduced to 1.5 L/min within 24 h. If there is a bridging tube, the arteriovenous end can be connected with a bridging tube to form an ECMO circuit for self-circulation, so that the body's hemodynamics is driven by the heart. If hemodynamics is stable for at least 6 h, consider removing the machine.When the patient is well aware, cough reflexes are obvious when sucking the sputum, the hemodynamics is stable, and the ventilator parameters are close to offline parameters, the spontaneous breathing test (SBT) is performed. After the SBT is passed, invasive breathing can be considered to remove the endotracheal tube.Patients do not need advanced respiratory support (HFNO, NIV, MV, ECLS, etc.); stable hemodynamics and tissue perfusion; no significant impairment of organ function; and no need for organ support treatment (CRRT, artificial liver, etc.). Consider transferring the patient out of ICU procedure.The body temperature returned to normal for more than 3 days; respiratory symptoms improved significantly; inflammation of the lungs showed obvious signs of absorption; and respiratory nucleic acid was negative for two consecutive times (one-day sampling time interval at least); and the patient can be released from isolation.7 Prevent and control nosocomial infection 7.1 Restriction and isolation guidelines for patient/ suspected patients See Table 7 . (Strong recommendation).According to the principles of standard prevention and tertiary protection, all personnel entering various zones should be evaluated using individual inventory tables according to the exposure risk level. Chose personal protective equipment of various levels is necessary. Personal protective equipment should be worn strictly in accordance with the instructions and only used for one time ( The patient's home isolation scheme is shown in Table 5 .Patients should monitor their body temperature and illness at home. If your body temperature continues to be higher than 38 ℃, or your breath is getting worse, you should seek medical treatment timely.In addition to taking protective measures, the home caregivers also should monitor their body temperature closely.Mild patients generally use a nasal catheter and a mask for oxygen. Adjust the oxygen flow as appropriate according to the patient's condition and doctor's instruction, and Requirements to the medical staff request 6. Medical personnel enter the isolation area with proper self-protection through designated channels.6.1 Medical staff should perform the personal protection practice under the Personal Protection Guidelines in Table 8 closely monitor the patient's breathing and blood oxygen saturation. If oxygen therapy fails to reach the expected effect, the nurse should analyze the cause comprehensively and be vigilant to notify the doctor.Mild patients generally use antiviral drugs, antibacterial drugs (when bacterial infection exists), and symptomatic treatment. The doctor's advice should be followed accurately and timely. The adverse reactions of oseltamivir mainly include nausea, vomiting, diarrhea, abdominal pain and bronchitis, cough, etc. The adverse reactions of interferon are mainly flu-like symptoms such as fever, fatigue, myalgia, and headache, followed by mild suppression of bone marrow. Attention should be paid to identify the change of clinical manifestations or adverse drug reactions.According to the patients' condition, provide highprotein, high-vitamin, carbohydrate-containing diets (e.g. eggs, fish, lean meat, milk, etc.) for enough nutrition to improve physical condition.Take good care of the patient and respond to the patient's question timely. Positively encourage patients to reduce their anxiety and fear.Dynamically monitor patients' vital signs, waterelectrolytes balance, acid-base balance, and functions of various organs, monitor patients' infection indicators, and determine the occurrence of complications such as acute respiratory distress syndrome, septic shock, stress ulcers, and deep vein thrombosis.The critically illed patients mainly use oxygen therapy such as HFNO, NIV and invasive mechanical ventilation. When using various oxygen treatments in a sequential manner, the airway and breathing circuit need to be kept open, and the effect of oxygen treatment needs to be monitored dynamically. At the same time, skincare products need to be used reasonably to avoid damage to the nose, face and lips by pressure. When using a high-flow nasal catheter to inhale oxygen, the oxygen flow and temperature and humidity should be adjusted appropriately. When using non-invasive mechanical ventilation, patient should receive relevant health education. Patients are instructed to inhale through the nose. The pressure is set from low to high and gradually reaches the target value. The human-machine coordination is maximized. The patient's consciousness and respiratory function are closely observed. Patients with artificial airway established should use a closed suction tube to reduce virus spread. Nurses should wear goggles or a face shield to avoid occupational exposure.If the patient develops moderate to severe ARDS, invasive mechanical ventilation combined with a prone position need to be adopted. Standard operating procedure for prone position needs to be followed. At the same time, be cautious to prevent pressure ulcers, falling bed, tube slippage, and eye damage by pressure and other complications. Patients treated with ECMO should be monitored for the performance of the oxygenator. If the oxygenator changes its color to darker, indicating the possibility of coagulation, the doctor should be notified to adjust the heparin dose as necessary. The oxygenator should be replaced if necessary. The coagulation function need to be monitored dynamically, including the whole set of coagulation and DIC (disseminated intravascular coagulation), and the time of activating partial thromboplastin, etc., the patient should be closely observed for signs of bleeding, such as bruising on the skin and mucous membranes, bleeding in the nasal cavity, oral cavity, bloody sputum, hematuria, blood in the stool, swelling of the abdomen, moving dullness, and the size of bilateral pupils. Make sure that the ECMO pipelines are tightly connected and firmly fixed to prevent air embolism and pipeline slippage.Perform oral care and skin care, assist the patient to use toilet, and take eyes on the indwelling tubes. Rules and regulations for aseptic operation and isolation should be strictly followed to prevent ventilator-related pneumonia, catheter-related sepsis, urinary catheter related urinary tract infections and other secondary infections.Dynamically assess the patients' nutritional risks and timely nutritional support can be given if needed. For the patients who can eat, the diet rich in protein and carbohydrates is recommended. Those patients who cannot eat but are compatible with enteral nutrition should be given enteral nutrition as soon as possible. For the patients incompatible with enteral nutrition, parenteral nutrition should be given timely to meet energy requirement.Psychological and humanistic care should be performed in high priority especially for the awake patients. Psychological techniques like mindfulness -based stress reduction can be adopted to relieve the patients' anxiety and panic by building up their optimistic confidence in overcoming the disease.Our guideline has three major limitations: Firstly, time is so limited that we cannot fully consider all clinical issues for this emergency disease. Secondly, many evidences came from data search is indirect. Thirdly, because some recommendations are based on the evidence from existing guidelines and experts' experience, there are situations where strong recommendations were produced on the base of low-quality evidence or very low-quality evidence, so high-quality evidence, when they appear, is likely to change current recommendations.Supplementary information accompanies this paper at . 1186/s40779-020-0233-6.Additional file 1. A successful treatment case of the severe 2019-nCoV infected pneumonia patient.Additional file 2. Experience and lessons in hospital rescue for 2019-nCoV infections.TB: tuberculosis; TNF: Tumor Necrosis Factor; WBC: White blood cells; WHO: World Health Organization\n",
            "\t0.989\tIn December 2019In December 2019, a cluster of pneumonia of unknown etiology was detected in Wuhan City, Hubei Province of China. The first 27 reported cases were all related to Huanan Seafood Wholesale Market, which sells aquatic products, live poultries, and wild animals . The first batch of cases identified later showed no exposure or even no relation to Huanan Wholesale Market, and the human-to-human transmission was confirmed; moreover, nosocomial infections were reported in some health care workers . The Chinese Center for Disease Control and Prevention (CDC) and Chinese health authorities later identified and announced that a new coronavirus (2019-nCoV) was accountable for the outbreak of this pneumonia in Wuhan . Thereafter, this disease was named Coronavirus Disease 2019 (COVID-19) by World Health Organization (WHO), and the causative virus was designated as SARS-CoV-2 by the International Committee on Taxonomy of Viruses . Within one and a half months, as of midnight of 18 February 2020, the novel coronavirus pneumonia (COVID-19) had spread from Hubei to 34 provinces in China and another 25 countries, resulting in 75,199 confirmed cases with 2009 deaths (Table 1) . At present, the number of cases is increasing rapidly in China and even around the world, which is a big threat to public health. Thirty-one provinces of China have initiated a level-1 public health response. The aim of this article is to provide a timely review of the characteristicsOn 31 December 2019, Wuhan Municipal Health Commission reported a number of unknown pneumonia cases related to Huanan Seafood Wholesale Market, 27 cases were hospitalized, seven of which were in serious condition . On 5 February 2020, Wuhan Municipal Health Committee reported that 59 cases of viral pneumonia with unknown etiology were detected in Wuhan, including seven severe cases, but no clear evidence was found for \"human-to-human\" transmission . On Jan 11, Wuhan Municipal Health Committee issued a new report confirming that the pathogen of the viral pneumonia of unknown cause was initially determined as a new coronavirus . On 20 February 2020, it was officially confirmed that \"human-to-human\" transmission and nosocomial infection had occurred [2, 3] . Since 16 February 2020, the cumulative COVID-19 case number increased quickly; meanwhile, the daily emerging case number increased steadily to 3886 on 4 February 2020, and then fluctuated to 2015 on 11 February 2020. The fatality cases number increased steadily to 2004 cases on 18 February 2020. The cumulative and daily emerged cases number jumped to 59,804 and 15,152, respectively, on 12 February 2020 ( Figure 1 ). This fierce growth of cumulative and daily emerged cases number in one day is due to the improvement of diagnosis standard for confirmed cases in Hubei province, in which the suspected cases with pneumonia imaging characteristics are categorized as clinical diagnosis cases. As a result, the patients can receive standard treatment as soon as possible. All data are from the National Health Commission of the People's Republic of China .The COVID-19 resulted in much lower mortality (about 2.67% up-to-date) among the confirmed cases, compared with Severe Acute Respiratory Syndrome (SARS) at 9.60% (November 2002-July 2003) and Middle East Respiratory Syndrome (MERS) at 34.4% (April 2012-November 2019) ( Table 1) . The median ages for the patients of COVID-19, SARS, and MERS are 55.5, 41.3, and 52.8 years old, respectively. COVID-19 and MERS patients share similargender composition of females (32%) and males (67%), but SARS patients show almost the same proportion of males (46.9%) and females (53.1%).According to the \"Diagnosis &Treatment Scheme for Novel Corona Virus Pneumonia (Trial) 6th Edition\", the source of infection is majorly the COVID-19 patients, even the asymptomatic patients can also be the source of infection. The transmission way is majorly through respiratory droplets and contacting. People are generally susceptible to this virus. respectively, on 12 February 2020 ( Figure 1 ). This fierce growth of cumulative and daily emerged cases number in one day is due to the improvement of diagnosis standard for confirmed cases in Hubei province, in which the suspected cases with pneumonia imaging characteristics are categorized as clinical diagnosis cases. As a result, the patients can receive standard treatment as soon as possible. All data are from the National Health Commission of the People's Republic of China . Coronavirus is a single strand positive RNA (+ssRNA) virus, belonging to order Nidovirales, family Coronaviridae, and subfamily Orthocoronavirinae . According to the characteristics of serotype and genome, the coronavirus subfamily is divided into four genera: α, β, γ, and δ . There are six kinds of coronaviruses known to infect humans, including 229E and NL63 of α genus [15, 16] , OC43, HKU1, Middle East respiratory syndrome-associated coronavirus (MERSr-CoV), and severe acute respiratory syndrome-associated coronavirus (SARSr-CoV) of β genus [16, 17] . The coronavirus isolated from the lower respiratory tract of patients with unidentified pneumonia in Wuhan is a new type of coronavirus (SARS-CoV-2) belonging to genus β, and subgenus sarbe . SARS-CoV-2 is different from the zoonotic MERSr-CoV and SARSr-CoV and becomes the seventh coronavirus to infect humans . The phylogenetic analysis of the coronaviruses based on full-length genome sequences shows that SARS-CoV-2 has the smallest genetic distance from bat coronavirus, but only about 45%-90% similarity with SARSr-CoV, and a lower similarity of 20%-60% with MERSr-CoV . Therefore, a bat is probably the original host of SARS-CoV-2, although the intermediate host may still exist in the process of transmission from bats to human beings.Coronavirus has an envelope, the particles are round or oval, often pleomorphic, with a diameter of 50-200 nm . S protein is located on the surface of the virus and forms a rod-shaped structure. As one of the main antigenic proteins of the virus, the S protein gene is the main target used for typing . Xu et al. also reported that SARS-CoV-2 S-protein supports a strong interaction with human angiotensin-converting enzyme 2 (ACE2) molecules, which means that the virus poses a significant public health risk for human transmission by the S-protein-ACE2 binding pathway .The knowledge of the physical and chemical characteristics of coronaviruses mostly comes from the study of SARS-CoV and MERS-CoV. The coronaviruses are sensitive to heat and can be killed at 4 of 10 56 • C for 30 min. In addition, ether, 75% ethanol, chlorine disinfectant, peracetic acid, and chloroform can effectively inactivate the virus, but not chlorhexidine .According to the \"Diagnosis & Treatment Scheme for Novel Coronavirus Pneumonia (Trial) 6th Edition\" enacted by the National Health Commission of the People's Republic of China on 19 February 2020, the incubation time after exposure is about 1-14 days . Fever, fatigue, and a dry cough are the main manifestations. Nasal obstruction, runny nose, and other upper respiratory symptoms are rare. About half of the patients developed dyspnea one week later, and severe cases developed rapidly into acute respiratory distress syndrome, septic shock, hard-to-correct metabolic acidosis, and coagulation dysfunction. Severe and critical patients may present moderate to low fever, or even no obvious fever. Some patients have mild onset symptoms, no fever, and mostly recovered after one week. Most patients have a favorable prognosis, although some patients are left in a critical condition, or do not survive. The aged patients and the patients with basic diseases have worse prognosis. Children cases are relatively mild.In the early stages of the disease, the total number of leukocytes in peripheral blood is normal or decreased, the lymphocyte count is decreased, and some patients present elevated levels of liver enzyme, muscle enzyme, and myoglobin; some severe cases present elevated troponin level. Most patients show elevated C-reactive protein (CRP) and erythrocyte sedimentation rate (ESR), and normal procalcitonin. In severe cases, the patients present with increased D-dimer and progressively decreased peripheral blood lymphocyte . Compared with non-ICU patients, the plasma levels of IL2, IL7, IL10, GSCF, IP10, MCP1, MIP1a, and TNF-α were higher in ICU patients .All patients suffered from pneumonia, and chest CT scans showed shadows in the lung . In the early stages of the disease, patients present with multiple small patch shadows and interstitial changes, especially in the extrapulmonary zone. It further progresses to multiple ground glass shadows and infiltrative shadows in both lungs. In severe cases, lung consolidation may occur, but pleural effusion is rare .To provide more information about the disease for the treatment in clinics, we collected detailed information about the fatal cases released by official channels. A total of 41 fatal cases were used to reveal the symptoms of the deaths, showing the symptom composition of fever (80.5%), cough (56.1%), short of breath (31.7%), chest tightness/pain (24.4%), fatigue (22.0%), dyspnea (12.2%), dizziness/headache (4.9%), general pain (7.3%) and chills (4.9%) (Figure 2) . A total of 26 cases with fatalities were used to disclose the dangerous comorbidities, showing that the major comorbidities are hypertension (53.8%), diabetes (42.3%), coronary heart disease (19.2%), cerebral infarction (15.4%), chronic bronchitis (19.2%) and Parkinson's disease (7.7%) (Figure 2 ). Chen N et al. reported that, among 99 confirmed cases, the common symptoms are fever (83%), cough (82%), bilateral pneumonia (75%), short of breath (31%), muscle ache (11%), confusion (9%), headache (8%), sore throat (5%), rhinorrhoea (4%), chest pain (2%), diarrhoea (2%), and nausea and vomiting (1%) . Huang C et al. reported that, among 41 confirmed cases, the symptom composition is as follows: pneumonia (100%), fever (98%), cough (76%), lymphopenia (63%), Dyspnea (55%), fatigue (44%), sputum production (28%), headache (8%), hemoptysis (5%), and diarrhea (3%); and the underlying diseases included diabetes (20%), hypertension (15%), and cardiovascular disease (15%) . Though the symptom compositions reported for the confirmed cases are similar to that of the fatality cases we collected, the percentages of hypertension, diabetes, and coronary heart diseases are much higher among the fatality cases than among the The diagnosis was based on a set of clinical criteria recommended by the National Health Commission of the People's Republic of China and the National Administration of Traditional Chinese Medicine .The cases comply with any item of A and any two items of B, or with 3 items of B as follows. (A) Epidemiological history: within two weeks before disease onset, 1. have a history of travel or residence in the district with case report; 2. have contacted the patients positive with nucleic acid detection; 3. have contacted with the patients with fever and respiratory symptoms from the district with case report; 4. disease onset in clustering. (B) Clinical manifestations: 1. Have fever and/or respiratory tract symptoms; 2. Have pneumonia with image characteristics mentioned above; 3. In early stage of the disease, have normal or decreased total number of leukocytes, or decreased lymphocyte count.The unconfirmed cases met the criteria of the suspected cases and are identified positive with SARS-CoV-2 RNA, by real-time RT-PCR or gene sequencing, from the sputum, throat swab, lower respiratory tract secretion, or other samples collected from patients. The diagnosis was based on a set of clinical criteria recommended by the National Health Commission of the People's Republic of China and the National Administration of Traditional Chinese Medicine .The cases comply with any item of A and any two items of B, or with 3 items of B as follows. (A) Epidemiological history: within two weeks before disease onset, 1. have a history of travel or residence in the district with case report; 2. have contacted the patients positive with nucleic acid detection; 3. have contacted with the patients with fever and respiratory symptoms from the district with case report; 4. disease onset in clustering. (B) Clinical manifestations: 1. Have fever and/or respiratory tract symptoms; 2. Have pneumonia with image characteristics mentioned above; 3. In early stage of the disease, have normal or decreased total number of leukocytes, or decreased lymphocyte count.The unconfirmed cases met the criteria of the suspected cases and are identified positive with SARS-CoV-2 RNA, by real-time RT-PCR or gene sequencing, from the sputum, throat swab, lower respiratory tract secretion, or other samples collected from patients.Mild cases: have mild symptoms, no pneumonia manifestation in chest image; Common cases: have fever, respiratory symptoms, and pneumonia manifestation in chest image; Severe cases: comply with any item of the follows. (A) dyspnea, respiratory rate ≥30 times/min; (B) at resting state, finger oxygen saturation ≤ 93%; (C) PaO2/FiO2 ≤ 300 mmHg (1mmHg = 0.133 kPa, PaO 2 : arterial partial pressure of oxygen, FiO 2 : fractional concentration of inspired oxygen);Critical cases: comply with any item of the follows. (A) show respiratory failure and mechanical ventilation is required; (B) present with shock; (C) combine with other organ failure needing Intensive Care Unit (ICU) monitoring and treatment; (D) chest imaging shows multilobe lesions or progress of lesion focus within 48 h ≥ 50%; (E) combine with other clinical conditions requiring hospitalization.Suspected and confirmed cases should be treated in isolation in hospitals with effective isolation and protective conditions. The suspected cases should be isolated in a single room, and the confirmed cases can be accepted in the same room. Critical cases should be treated in ICU as soon as possible.Bed rest, strengthen supportive treatment, ensure sufficient energy; pay attention to water-electrolytes balance and maintain the stability of the internal environment; closely monitor vital signs and finger oxygen saturation, and so on. B.Monitor the blood routine, urine routine, C-reactive protein (CRP) and health indications (liver enzyme, myocardial enzyme, renal function, etc.), coagulation function, arterial blood gas analysis if necessary, and recheck chest imaging. C.According to the change of oxygen saturation, give effective oxygen therapy in time, including oxygen given by nasal catheter or mask. If necessary, apply high flow oxygen therapy via the nose, noninvasive or invasive mechanical ventilation, and so on. D.Antiviral treatment: no effective antiviral drug at present. Treat with IFN-α aerosol inhalation (five million U per time for adults, two times per day), and/or Lopinavir/Ritonavir oral administration (two tablets per time, two times per day). E.Antibiotic treatment: avoid blind and improper use of antibiotics, especially the combination use of broad-spectrum antibiotics. Strengthen bacteriological monitoring. Antibiotics should be used in time in secondary bacterial infection.A. Treatment principle: based on symptomatic treatment, actively prevent and treat complications, treat basic diseases, prevent secondary infection, and timely apply organ function support. B.Respiratory support: apply noninvasive mechanical ventilation for two hours, if the condition is not improved, or the patient is intolerable to noninvasive ventilation, accompanied with increased airway secretions, severe coughing, or unstable hemodynamics, the patient should be transferred to invasive mechanical ventilation in time. The \"lung-protective ventilation strategy\" with low tidal volume should be adopted in invasive mechanical ventilation to reduce ventilator-associated lung injury. If necessary, ventilation in the prone position, recruitment maneuver, or extracorporeal membrane oxygenation (ECMO) can be used. C.Circulation support: improve microcirculation based on full fluid resuscitation, use vasoactive drugs, and apply hemodynamic monitoring if necessary.Others: according to the degree of dyspnea and the progress of chest imaging, use glucocorticoids appropriately for a short time (3-5 days) with the recommended dose no more than what is equivalent to methylprednisolone 1-2 mg/kg·day.As of 26 January 2020, 30 provinces have initiated a level-1 public health response to control COVID-19 . A level-1 response means that during the occurrence of a particularly serious public health emergency, the provincial headquarters shall organize and coordinate the emergency response work within its administrative area according to the decision deployment and unified command of the State Council . Fever observation rooms shall be set up at stations, airports, ports, and so on to detect the body temperature of passengers entering and leaving the area and implement observation/registration for the suspicious patients. The government under its jurisdiction shall, in accordance with the law, take compulsory measures to restrict all kinds of the congregation, and ensure the supply of living resources. They will also ensure the sufficient supply of masks, disinfectants, and other protective articles on the market, and standardize the market order. The strengthening of public health surveillance, hygiene knowledge publicity, and monitoring of public places and key groups is required. Comprehensive medical institutions and some specialized hospitals should be prepared to accept COVID-19 patients to ensure that severe and critical cases can be differentiated, diagnosed, and effectively treated in time. The health administration departments, public health departments, and medical institutions at all (province, city, county, district, township, and street) levels, and social organizations shall function in epidemic prevention and control and provide guidance for patients and close contact families for disease prevention .The responsibilities of the organizations are shown in Table 2 . In accordance with the working principle of \"prevention first, prevention and control combined, scientific guidance and timely treatment\", the prevention and control work shall be carried out in a coordinated and standardized way . Table 2 . Responsibilities for the different organizations at all (province, city, county, district, township, and street) levels in the outbreak of COVID-19.To timely find and report the COVID-19 cases, understand the disease characteristics and possible sources of infection, standardize the management of close contacts, and prevent the spread of the epidemic.Overall guidance of epidemic control, organizing a technical expert group for prevention and control; formulation and improvement of relevant work and technical schemes, and implementation of funds and materials for disease prevention and control; tracking and management of close contacts.Organization, coordination, supervision, and evaluation of the monitoring work; collection, analysis, report, and feedback of the monitoring data; epidemiological investigation; strengthening laboratory testing ability, bio-safety protection awareness, and technical training; carrying out health education and publicity and risk communication to the public.Case detection and report, isolation, diagnosis, and treatment; clinical management and prevention and control of nosocomial infections; sample collection and detection, and training of medical staff in the institution.The recent outbreak of the unknown severe pneumonia in China is caused by a novel coronavirus named 2019-nCoV , later was designated SARS-CoV-2 by the International Committee on Taxonomy of Viruses. This virus and the SARSr-CoV/MERSr-CoV share a common ancestor . Compared with SARSr-CoV and MERSr-CoV, SARS-CoV-2 results in much lower mortality in patients but has a comparable infection ability. From the analysis of the fatal cases of this novel coronavirus pneumonia, the comorbidities of hypertension, diabetes, coronary heart disease, cerebral infarction, and chronic bronchitis were found to be dangerous factors that resulted in death.In 2002 and 2003, the outbreak of SARS brought a disaster to the people of the world, especially the Chinese people [24, 25] . Fortunately, SARS was finally defeated, and Chinese health departments upgraded their disease prevention and control system by summing up their experiences of fighting SARS. Thus, when COVID-19 appeared, the whole country quickly entered a state of fighting against the new infectious disease. Policies led by the National Health Commission have been formulated and implemented efficiently, and Chinese scientists identified the etiology of the disease in no more than a month. However, new cases are increasing every day, showing a trend of spreading to the whole country and across the world.COVID-19 appeared just one month before the Spring Festival of China, and the massive population flow has brought great challenges for disease prevention and control. This virus can be transmitted from human to human and no effective treatment drug has been found. The most effective prevention and control measures are to find suspected patients and close contacts, confirm patients and virus carriers, and block the transmission through isolation, disinfection, and personal protection. Therefore, early detection, isolation, and treatment of patients are the key measures to control the source of infection and reduce the infection rate. It is also crucial to avoid nosocomial infection by strengthening the management of medical staff and patients. Health education on knowledge for disease prevention and control is also important. Finally, if we want to eliminate the threat of this novel coronavirus pneumonia similar to SARS, we need to learn more about the pathogenesis of the virus and develop specific vaccines and therapeutic drugs as soon as possible. \n",
            "\t0.988\tReanalysis of the epidemic curve from the initial cluster of cases with novel coronavirus (2019-nCoV) in December 2019 indicates substantial human-to-human transmission. It is possible that the common exposure history at a seafood market in Wuhan originated from the human-to-human transmission events within the market, and the early, strong emphasis that market exposure indicated animal-to-human transmission was potentially the result of observer bias. To support the hypothesis of zoonotic origin of 2019-nCoV stemming from the Huanan seafood market, the index case should have had exposure history related to the market and the virus should have been identified from animals sold at the market. As these requirements remain unmet, zoonotic spillover at the market must not be overemphasized.The clinical summary of the earliest cases of 2019 novel coronavirus (2019-nCoV) infections in Wuhan, China was recently published , showing the majority of cases were exposed to the Huanan seafood market, which also had wild animals, suggesting the possibility of zoonotic transmission in the market. This suggestion of zoonotic spillover was quoted by international organizations, including the World Health Organization (WHO), and as a result early research focused on zoonotic rather than direct human-to-human transmission of 2019-nCoV. However, the index case had no exposure history related to the seafood market, indicating that Huanan seafood market-related zoonotic spillover may have been an overblown hypothesis. Here, we reanalyze the epidemic data of the initial cluster of cases with 2019-nCoV infections to demonstrate that the epidemic curve is consistent with substantial human-to-human transmission in December 2019.Three important arguments are made here with respect to epidemiological interpretation of the epidemic dataset. First, Figure 1A shows the epidemic curve of cases in Wuhan, distinguishing case generations by color. The index case developed symptoms on 1 December 2019, with cases 2-4 having onset nine days later, and cases 5-6 five days after that. Together, these intervals indicate a possible serial interval (SI)-the time between illness onset in an earlier case to that in a secondary case-with a mean of 7.4 days, consistent with the mean SI of severe acute respiratory syndrome . The latter is also consistent with the mean SI estimate of 7.5 days presented in the preliminary epidemiological study .Although it is possible that the SIs are shorter than quoted here , the epidemic curve is still in agreement with the existence of asymptomatic and unascertained mild cases between diagnosed cases. epidemic curve is still in agreement with the existence of asymptomatic and unascertained mild cases between diagnosed cases. The expected number of cases in each subsequent generation was assumed to follow a Poisson distribution, and the 95% confidence intervals of the reproduction number (whiskers) were derived from the profile likelihood.Second, assuming a constant SI of 8 days, the epidemic curve of cases by the date of illness onset can be transformed to that by generation of cases. The number of cases in each generation is therefore 1, 3, 4, 27, and 6 cases, respectively. These numbers allow for the estimation of generation-dependent reproduction numbers-the average number of secondary cases per primary case for each generation ( Figure 1B ). Assuming that the offspring distribution is Poisson distributed, the reproduction numbers can be estimated at 3.0 (95% confidence interval (CI): 0.75, 7.8), 1.3 (95% CI: 0.4, 3.1), 6.7 (95% CI: 4.5, 9.6), and 0.2 (95% CI: 0.1, 0.5)-broadly in line with preliminary basic reproduction number estimates of 1.5-3.5 quoted by the WHO and presented elsewhere [6, 7] .Third, the common exposure supports secondary transmission events taking place in the market. Although the virus has been identified in market environmental samples , this does not exclude the likelihood of secondary transmission. That is, it is possible that the common exposure history at the Huanan seafood market in Wuhan originated from the human-to-human transmission events within the market.Unfortunately, early emphasis that market exposure implied animal-to-human transmission considerably delayed global recognition of exportation of the virus from Wuhan, especially during the first half of January [9, 10] . The emphasis on market-based zoonotic transmission may have been the result of observer bias-i.e., the bias that originates from having preconceptions or subjective feelings about what is being studied that could influence epidemiological observation and even recording information. For example, the zoonotic origin of another relatively recently emerged coronavirus with predominantly zoonotic transmission-the virus causing Middle East respiratory syndrome (MERS)-may have served as a strong reference for reducing concern about epidemic levels of sustained human-to-human transmission.In conclusion, we believe that zoonotic spillover at the market should not be overemphasized, because the epidemic curve is consistent with substantial human-to-human transmission in December 2019. There are two important take homes for any future investigations that begin with a similar scenario: first, to verify that zoonotic spillover is related to the exposure in question, the index case must be verified to have that exposure history. Second, without identifying the virus in Second, assuming a constant SI of 8 days, the epidemic curve of cases by the date of illness onset can be transformed to that by generation of cases. The number of cases in each generation is therefore 1, 3, 4, 27, and 6 cases, respectively. These numbers allow for the estimation of generation-dependent reproduction numbers-the average number of secondary cases per primary case for each generation ( Figure 1B) . Assuming that the offspring distribution is Poisson distributed, the reproduction numbers can be estimated at 3.0 (95% confidence interval (CI): 0.75, 7.8), 1.3 (95% CI: 0.4, 3.1), 6.7 (95% CI: 4.5, 9.6), and 0.2 (95% CI: 0.1, 0.5)-broadly in line with preliminary basic reproduction number estimates of 1.5-3.5 quoted by the WHO and presented elsewhere [6, 7] .Third, the common exposure supports secondary transmission events taking place in the market. Although the virus has been identified in market environmental samples , this does not exclude the likelihood of secondary transmission. That is, it is possible that the common exposure history at the Huanan seafood market in Wuhan originated from the human-to-human transmission events within the market.Unfortunately, early emphasis that market exposure implied animal-to-human transmission considerably delayed global recognition of exportation of the virus from Wuhan, especially during the first half of January [9, 10] . The emphasis on market-based zoonotic transmission may have been the result of observer bias-i.e., the bias that originates from having preconceptions or subjective feelings about what is being studied that could influence epidemiological observation and even recording information. For example, the zoonotic origin of another relatively recently emerged coronavirus with predominantly zoonotic transmission-the virus causing Middle East respiratory syndrome (MERS)-may have served as a strong reference for reducing concern about epidemic levels of sustained human-to-human transmission.In conclusion, we believe that zoonotic spillover at the market should not be overemphasized, because the epidemic curve is consistent with substantial human-to-human transmission in December 2019. There are two important take homes for any future investigations that begin with a similar scenario: first, to verify that zoonotic spillover is related to the exposure in question, the index case must be verified to have that exposure history. Second, without identifying the virus in animals sold at the market, it is difficult to conclude with certainty that any zoonotic transmission occurred at the market.Author Contributions: H.N. conceived the study, and all authors participated in the study design. H.N. collected the data and A.R.A. and H.N. analyzed the data. All authors jointly drafted the manuscript. All authors gave comments on the earlier versions of the manuscript. All authors have read and agreed to the published version of the manuscript. The authors declare no conflicts of interest.\n",
            "\t0.988\tSince December 2019, China has been experiencing a large outbreak of a novel coronavirus (2019-nCoV) which can cause respiratory disease and severe pneumonia. We estimated the basic reproduction number R0 of 2019-nCoV to be around 2.2 (90% high density interval: 1.4-3.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and pandemic influenza, indicating a risk of global spread.On 31 December 2019, the World Health Organization (WHO) was alerted about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China [1, 2] . Only a few days later, Chinese authorities identified and characterised a novel coronavirus (2019-nCoV) as the causative agent of the outbreak . The outbreak appears to have started from a single or multiple zoonotic transmission events at a wet market in Wuhan where game animals and meat were sold and has resulted in 5,997 confirmed cases in China and 68 confirmed cases in several other countries by 29 January 2020 . Based on the number of exported cases identified in other countries, the actual size of the epidemic in Wuhan has been estimated to be much larger . At this early stage of the outbreak, it is important to gain understanding of the transmission pattern and the potential for sustained human-to-human transmission of 2019-nCoV. Information on the transmission characteristics will help coordinate current screening and containment strategies, support decision making on whether the outbreak constitutes a public health emergency of international concern (PHEIC), and is key for anticipating the risk of pandemic spread of 2019-nCoV. In order to better understand the early transmission pattern of 2019-nCoV, we performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date.Two key properties will determine further spread of 2019-nCoV. Firstly, the basic reproduction number R 0 describes the average number of secondary cases generated by an infectious index case in a fully susceptible population, as was the case during the early phase of the outbreak. If R 0 is above the critical threshold of 1, continuous human-to-human transmission with sustained transmission chains will occur. Secondly, the individual variation in the number of secondary cases provides further information about the expected outbreak dynamics and the potential for superspreading events . If the dispersion of the number of secondary cases is high, a small number of cases may be responsible for a disproportionate number of secondary cases, while a large number of cases will not transmit the pathogen at all. While superspreading always remain a rare event, it can result in a large and explosive transmission event and have a lot of impact on the course of an epidemic. Conversely, low dispersion would lead to a steadier growth of the epidemic, with more homogeneity in the number of secondary cases per index case. This has important implications for control efforts.In a first step, we initialised simulations with one index case. For each primary case, we generated secondary cases according to a negative-binomial offspring distribution with mean R 0 and dispersion k [7, 8] . The dispersion parameter k quantifies the variability in the number of secondary cases, and can be interpreted as a measure of the impact of superspreading events (the lower the value of k, the higher the impact of superspreading). The generation time interval D was assumed to be gamma-distributed with a shape parameter of 2, and a mean that varied between 7 and 14 days. We explored a wide range of parameter combinations (Table) and ran 1,000 stochastic simulations for each individual combination. This corresponds to a total of 3.52 million one-index-case simulations that were run on UBELIX (), the high performance computing cluster at the University of Bern, Switzerland.In a second step, we accounted for the uncertainty regarding the number of index cases n and the date T of the initial zoonotic animal-to-human transmissions at the wet market in Wuhan. An epidemic with several index cases can be considered as the aggregation of several independent epidemics with one index case each. We sampled (with replacement) n of the oneindex-case epidemics, sampled a date of onset for each index case and aggregated the epidemic curves together. The sampling of the date of onset was done uniformly from a 2-week interval around 27 November 2019, in coherence with early phylogenetic analyses of 11 2019-nCoV genomes . This step was repeated 100 times for each combination of R 0 (22 points), k (20 points), D (8 points) and n (6 points) for a total of 2,112,000 full epidemics simulated that included the uncertainty on D, n and T. Finally, we calculated the proportion of stochastic simulations that reached a total number of infected cases within the interval between 1,000 and 9,700 by 18 January 2020, as estimated by Imai et al. . In a process related to approximate Bayesian computation (ABC), the parameter value combinations that led to simulations within that interval were treated as approximations to the posterior distributions of the parameters with uniform prior distributions. Model simulations and analyses were performed in the R software for statistical computing . Code files are available on  order to reach between 1,000 and 9,700 infected cases by 18 January 2020, the early human-to-human transmission of 2019-nCoV was characterised by values of R 0 around 2.2 (median value, with 90% high density interval: 1.4-3.8) (Figure 1 ). The observed data at this point are compatible with a large range of values for the dispersion parameter k (median: 0.54, 90% high density interval: 0.014-6.95). However, our simulations suggest that very low values of k are less likely. These estimates incorporate the uncertainty about the total epidemic size on 18 January 2020 and about the date and scale of the initial zoonotic event (Figure 2 ).Comparison with other emerging coronaviruses in the past allows to put into perspective the available information regarding the transmission patterns of 2019-nCoV. Figure 3 shows the combinations of R 0 and k that are most likely at this stage of the epidemic. Our estimates of R 0 and k are more similar to previous estimates focusing on early human-to-human transmission of SARS-CoV in Beijing and Singapore than of Middle East respiratory syndrome-related coronavirus (MERS-CoV) . The spread of MERS-CoV was characterised by small clusters of transmission following repeated instances of animal-to-human transmission events, mainly driven by the occurrence of superspreading events in hospital settings. MERS-CoV could however not sustain human-to-human transmission beyond a few generations . Conversely, the international spread of SARS-CoV lasted for 9 months and was driven by sustained human-to-human transmission, with occasional superspreading events. It led to more than 8,000 cases around the world and required extensive efforts by public health authorities to be contained . Our assessment of the early transmission of 2019-nCoV suggests that 2019-nCoV might follow a similar path.Our estimates for 2019-nCoV are also compatible with those of 1918 pandemic influenza, for which k was The basic reproduction number R 0 quantifies human-to-human transmission. The dispersion parameter k quantifies the risk of a superspreading event (lower values of k are linked to a higher probability of superspreading). Note that the probability density of k implies a log10 transformation.estimated . Human-to-human transmission of influenza viruses is characterised by R 0 values between 1.5 and 2 and a larger value of k, implying a more steady transmission without superspreading. The emergence of new strains of influenza, for which human populations carried little to no immunity contrary to seasonal influenza, led to pandemics with different severity such as the ones in1918, 1957 1968 and 2009. It is notable that coronaviruses differ from influenza viruses in many aspects, and evidence for the 2019-nCoV with respect to case fatality rate, transmissibility from asymptomatic individuals and speed of transmission is still limited. Without speculating about possible consequences, the values of R 0 and k found here during the early stage of 2019-nCoV emergence and the lack of immunity to 2019-nCoV in the human population leave open the possibility for pandemic circulation of this new virus.The scarcity of available data, especially on case counts by date of disease onset as well as contact tracing, greatly limits the precision of our estimates and does not yet allow for reliable forecasts of epidemic spread. Case counts provided by local authorities in the early stage of an emerging epidemic are notoriously unreliable as reporting rates are unstable and vary with time. This is due to many factors such as the initial lack of proper diagnosis tools, the focus on the more severe cases or the overcrowding of hospitals. We avoided this surveillance bias by relying on an indirect estimate of epidemic size on 18 January, based on cases identified in foreign countries before quarantine measures were implemented on 23 January. This estimated range of epidemic size relies itself on several assumptions, including that all infected individuals who travelled from Wuhan to other countries have been detected . This caveat may lead to an underestimation of transmissibility, especially considering the recent reports about asymptomatic cases . Conversely, our results do not depend on any assumption about the existence of asymptomatic transmission, and only reflect the possible combinations of transmission events that lead to the situation on 18 January. Table. Among these simulated epidemics, 54.3% led to a cumulative incidence between 1,000 and 9,700 on 18 January 2020 (in red).Our analysis, while limited because of the scarcity of data, has two important strengths. Firstly, it is based on the simulation of a wide range of possibilities regarding epidemic parameters and allows for the full propagation on the final estimates of the many remaining uncertainties regarding 2019-nCoV and the situation in Wuhan: on the actual size of the epidemic, on the size of the initial zoonotic event at the wet market, on the date(s) of the initial animal-to-human transmission event(s) and on the generation time interval. As it accounts for all these uncertainties, our analysis provides a summary of the current state of knowledge about the human-to-human transmissibility of 2019-nCoV. Secondly, its focus on the possibility of superspreading events by using negative-binomial offspring distributions appears relevant in the context of emerging coronaviruses [7, 8] . While our estimate of k remains imprecise, the simulations suggest that very low values of k < 0.1 are less likely than higher values < 0.1 that correspond to a more homogeneous transmission pattern. However, values of k in the range of 0.1-0.2 are still compatible with a small risk of occurrence of large superspreading events, especially impactful in hospital settings [15, 16] .Our analysis suggests that the early pattern of humanto-human transmission of 2019-nCoV is reminiscent of SARS-CoV emergence in 2002. International collaboration and coordination will be crucial in order to contain the spread of 2019-nCoV. At this stage, particular attention should be given to the prevention of possible rare but explosive superspreading events, while the establishment of sustained transmission chains from single cases cannot be ruled out. The previous experience with SARS-CoV has shown that established practices of infection control, such as early detection and isolation, contact tracing and the use of personal protective equipment, can stop such an epidemic. Given the existing uncertainty around the case fatality rate This can be interpreted as the combinations of R 0 and k values most compatible with the estimation of epidemic size before quarantine measures were put in place. As a comparison, we show the estimates of R 0 and k for the early human-to-human transmission of SARS-CoV in Singapore and Beijing and of 1918 pandemic influenza [7, 9, 14] . and transmission, our findings confirm the importance of screening, surveillance and control efforts, particularly at airports and other transportation hubs, in order to prevent further international spread of 2019-nCoV.\n",
            "\t0.987\tA novel coronavirus (2019-nCoV) is causing an outbreak of viral pneumonia that started in Wuhan, China. Using the travel history and symptom onset of 88 confirmed cases that were detected outside Wuhan in the early outbreak phase, we estimate the mean incubation period to be 6.4 days (95% credible interval: 5.6-7.7), ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile). These values should help inform 2019-nCoV case definitions and appropriate quarantine durations.Early January 2020, a novel coronavirus (2019-nCoV) was identified as the infectious agent causing an outbreak of viral pneumonia in Wuhan, China, where the first cases had their symptom onset in December 2019 . This newly discovered virus, which causes severe acute respiratory disease, is related to the severe acute respiratory syndrome (SARS) coronavirus and Middle East respiratory syndrome (MERS) coronavirus, but distinct from each of these . The key epidemiological parameters, including incubation period, for this new virus are therefore rapidly being studied from incoming case reports as the epidemic continues. Chief among these key parameters is the incubation period distribution. The range of the values for the incubation period is essential to epidemiological case definitions, and is required to determine the appropriate duration of quarantine. Moreover, knowledge of the incubation period helps to assess the effectiveness of entry screening and contact tracing. The distribution of the incubation period is also used in estimating the size of the epidemic and the transmission potential [6, 7] . In absence of data on the 2019-nCoV incubation period, these studies have assumed incubation periods of SARS or MERS coronaviruses.Here we present the distribution of incubation periods estimated for travellers from Wuhan with confirmed 2019-nCoV infection in the early outbreak phase, using their reported travel histories and symptom onset dates.In January 2020, an increasing number of cases confirmed to be infected with 2019-nCoV were detected outside Wuhan. For 88 cases detected between 20 and 28 January, the travel history (to and) from Wuhan is known, as well as their symptom onset date. Their ages range from 2 to 72 years of age (information missing for four cases); 31 were female and 57 were male. During this initial stage of the epidemic, it is most likely that these travellers were infected in Wuhan. Consequently, their time spent in Wuhan can be taken as the duration of exposure to infection. Of these 88 cases with known travel history, 63 were Wuhan residents who travelled elsewhere and 25 were visitors who stayed in Wuhan for a limited time. By taking the date of symptom onset and travel history together, we inferred the possible incubation period for each of these cases.The data used for this analysis has been translated from Chinese sources such as provincial centres of disease control, and made publicly available . We took the data as available on 29 January 2020 (Supplementary Material S1).Using the duration of stay in Wuhan and the symptom onset date, we obtained a range of possible values for the incubation period of each case. We fitted three parametric forms for the incubation period distribution to these ranges: the Weibull distribution, the gamma distribution and the lognormal distribution. We used a Bayesian approach to fitting that allows for the use of prior knowledge to inform the analysis. We specified strictly positive flat prior probability distributions Exposure to reporting timeline for confirmed 2019 novel coronavirus (2019-nCoV) cases with travel history from Wuhan, sorted by symptom onset date, data 20-28 January 2020 (n = 88)The analysis yields the probability of being infected (dark pink), i.e. the cumulative density function of the estimated infection moments, using the Weibull distribution.for the parameter values of the three distributions (Supplementary Material S2), which ensured our estimates are conservative. Because of the sufficient number of observations, the impact of the priors on the outcome was negligible. We used a uniform prior probability distribution over the exposure interval for the moment of infection for each case. We sampled from the posterior distribution using the rstan package in R software version 3.6.0 (R Foundation, Vienna, Austria) (Supplementary Material S3). Figure 1 shows the exposure to reporting timeline for each case, where the cases without a maximum incubation period lack an unexposed (grey) period. However, the estimated infection times for these cases are close to the end of the exposure window, informed by the cases that do have a maximum incubation period.The Weibull distribution provided the best fit to the data ( Table 1 ). The mean incubation period was estimated to be 6.4 days (95% credible interval (CI): 5.6-7.7). The incubation period ranges from 2.1 to 11.1 days (2.5th to 97.5th percentile) ( Table 2 and Figure 2 ). The results using the gamma distribution provide a slightly poorer description of the data, but a similar range: from 2.4 to 12.5 days. Although the lognormal distribution provides the poorest fit to the data, the incubation period ranging from 2.4 to 15.5 days (2.5th to 97.5th percentile) may be relevant for a conservative choice of quarantine periods.A comparison to the estimated incubation period distribution for MERS (Table 3 and Figure 3 ) shows that the incubation period values are remarkably similar, with mean values differing at most 1 day and 95th percentiles differing at most 2 days. The estimated mean incubation periods for SARS are more variable between studies, including values shorter and longer than those presented here for 2019-nCoV. These findings imply that the findings of previous studies that have assumed incubation period distributions similar to MERS or SARS will not have to be adapted because of a shorter or longer incubation period.We characterised the distribution of incubation periods for travellers from Wuhan infected with 2019-nCoV in Wuhan who were reported as cases between 20 and 28 January 2020. The study provides empirical evidence to back reports on a familial cluster where five family members developed symptoms 3 to 6 days after exposure , and fits within the range for the incubation period of 0 to 14 days assumed by the WHO and of 2 to 12 days assumed by the ECDC . Our estimate of the mean incubation period is longer than the value of 5.2 days based on 10 cases , and 4.8 days (range: 2-11) based on 16 travellers between Wuhan and Guangdong . The latter study is restricted to travellers with a 3-day exposure window. Repeating our analysis with only the 25 visitors to Wuhan who had a closed exposure window, leads to a mean incubation period of 4.5 days (CI: 3.7-5.6) which is more in line with the studies above, but the 95th percentile drops to 8.0 days (CI: 6.3-11.8).In our analysis, we assumed a uniform prior probability of being infected during the period of stay in Wuhan.Since the epidemic was developing during that time period, it is more likely that travellers were infected towards the end rather than the beginning of their stay. This might produce a slight bias towards longer incubation periods, so the estimated upper limit of 11.1 days could be considered conservative.The travellers in this study represent a selective sample of the reported cases. We found travellers to be more often male and younger than the cases reported in . The numbers are too small to detect systematic differences in incubation time with age or sex. Because we only have information on confirmed cases, there is likely a bias towards more severe cases in areas with early awareness and a well-functioning healthcare system. As the epidemic continues, it remains important to collect more information on the incubation periods of 2019-nCoV cases with older ages, with underlying morbidity, who are women or who have mild symptoms. There are various choices one can make about the parametric form of the incubation period distribution.The results with the three often-used forms we report here suggest that there is little impact on the mean and dispersion of the incubation periods. Of these three, the lognormal distribution assigns higher probabilities to longer incubation periods. Although we found that this distribution provided a poorer description of the data than the Weibull and the gamma distributions, it is prudent not to dismiss the possibility of incubation periods up to 14 days at this stage of the epidemic. \n",
            "\n",
            " Title:  nan\n",
            "2 \b. \n",
            "\n",
            "Input question: What is Coronovirus Disease 2019?\n",
            "Results (after 2.053 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.554\tEvery year, during the Lunar New Year, the largest human migration in the world occurs in China. Almost 3 billion passenger-journeys are made as travellers reunite with distant families . This year, things are different. The recent emergence of a novel coronavirus (2019-nCoV), which caused an outbreak of viral pneumonia in Wuhan, China, has led to three Chinese cities; Wuhan, Huanggang, Ezhou, placed under lockdown to curb transmission. The 2019-nCoV has sparked global concern regarding the likelihood of the epidemic turning out like the 2003 Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV), where more than 8000 people were infected with 774 mortalities . Healthcare workers represented 20% of the infected . The 2019-nCoV serves as a reminder of the potential dangers posed by coronaviruses to both patients and doctors alike.Coronaviruses: what are they?Coronaviruses (CoV) belong to the subfamily Coronavirinae, in the family Coronaviridae of the order Nidovirales. There are four genera: Alphacoronavirus, Betacoronavirus, Gammacoronavirus and Deltacoronavirus . It is a single positive-sense RNA virus. Mutation rates of RNA viruses are greater than DNA viruses, suggesting a more efficient adaptation process for survival. The genome codes for at least four main structural proteins: spike (S), membrane (M), envelope (E), nucleocapsid (N) proteins and other accessory proteins which aid the replicative processes and facilitate entry into cells . Figure 1 summarises the coronavirus's structure and the function of the structural proteins. CoVs mainly affect birds and mammals. Prior to 2019, there were only six CoVs that can infect human and cause respiratory diseases: HCoV-229E, HCoV-OC43, HCoV-NL63, HKU1, SARS-CoV, MERS-CoV. The last 2 are capable of causing severe respiratory syndrome in humans.On the 31st of December 2019, the World Health Organisation (WHO) was alerted by Chinese authorities of a series of pneumonia-like cases in the city of Wuhan, a city the size of London with about 11 million people. It was quickly identified that the first human infections likely originated from Huanan Seafood Market in Wuhan . Two weeks later, a group of Chinese scientists, along with WHO announced that a new coronavirus (2019-nCoV), identified through genomic sequencing, was the culprit of the pneumonia . Symptoms of infection included fever, malaise, dry cough, shortness of breath and respiratory distress. While such an effort is a crucial response to tackling the crisis, understanding of the virus's transmission patterns still remain murky. Initially thought to be a virus with mainly animal-human transmission, this was proven to be untrue when the number of cases surged over the weekend of 18th and 19th January and reports of healthcare workers being infected surfaced . As of the 23rd of January 2020, 622 have been infected globally with 17 mortalities all located within China.A timeline of key events up till the 23rd of January is represented in Fig. 2 .While the 2019-nCoV transmission route is still unknown, countries have been preparing measures based on past experiences with coronaviruses namely SARS-CoV and MERS-CoV. These viruses transmit primarily through droplets and other bodily secretions. In the ophthalmology practice, healthcare workers may be particularly susceptible to these infections. Firstly, ophthalmologists are extremely reliant on physical examination during patient consultation. Of particular concern is the proximity between the patient and ophthalmologist during the slit lamp microscope examination. It has been shown that droplets from a cough or sneeze can be propelled for up to 6 m , a range that definitely encompasses the distance between the patient and ophthalmologist.Secondly, during the SARS-CoV epidemic, clinical reports have suggested tears as a medium of infection. In a case series by Loon et al., it was shown that viral RNA of the SARS-CoV can be detected by reverse-transcription polymerase chain reaction (RT-PCR) from the tears of infected individuals . While anecdotal in nature, such accounts highlight the possible infectivity of tears, a fluid which ophthalmologists and instruments come in contact on a daily basis. If true, this represents a crucial need for further development of disinfection and personal protective equipment (PPE) protocols for the ophthalmology clinic. In view of the potential threat in the ophthalmology practice, it may be prudent to revisit the strategies that successfully curbed the transmission of SARS in 2003 . With particular relevance to the ophthalmic practice, it may be beneficial to triage patients according to produced surveillance case definitions . In 2003, the WHO launched a case classification scheme which triaged patients into general, suspect and probable categories. Ophthalmology practices in Hongkong, a country badly hit by SARS, recommended the full PPE for all cases regardless of SARS status. For suspect and probable cases, appointments were recommended to be deferred unless in the event of an ophthalmic emergency. These patients were seen in an isolation ward. An emphasis on hand hygiene measures and stocking up of PPE such as N95 masks, gloves, gowns and googles should also be considered while the mode of transmission is being identified. Decontamination and sterilisation protocols of clinical rooms and equipment should also be improved on as coronaviruses have been found to survive in environments outside the body for a long time . For instance, it is still not established if higher concentrations of dilute bleach (1:10), a chemical used to sterilise the Goldmann applanation tonometer, can be utilised to eliminate coronaviruses . Other shared equipment like the B-scan probe and contact lenses for photocoagulation will also need strict sterilisation protocols. Finally, the reduction of non-urgent ophthalmic operations should also be considered as the risk of viral transmission may outweigh the surgical benefits. For emergency operations, full PPE can be considered to reduce the probability of healthcare transmission.As the WHO Director-General, Tedros Adhanom Ghebreyesus highlighted, \"Make no mistake: this is an emergency in China. But it has not yet become a global health emergency. It may yet become one.\" It is currently very difficult to predict the eventual impact of the 2019-nCoV. However, it will be prudent to utilise the lessons gained from SARS-CoV and prepare for the worst. Until a vaccine is created, public health strategies are likely the best weapons against the enemy.Conflict of interest The authors declare that they have no conflict of interest.Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit . org/licenses/by/4.0/.\n",
            "\t0.547\tThe situation has continued to evolve rapidly since then and just a few weeks later, as at 23 January, 614 laboratory-confirmed cases and 17 deaths have been reported including some cases detected outside mainland China . Meanwhile, on 7 January 2020, the novel coronavirus, currently named 2019-nCoV, was officially announced as the causative agent by Chinese authorities . In order to support public health action, viral genome sequences were released by Chinese researchers on 10 January and 2 days later, four further sequences were also made available on the Global Initiative on Sharing All Influenza Data (GISAID) (). While more cases are being reported on a daily basis and there is evidence for some human-to-human transmission in China, a number of important questions remain unanswered. For example, there is no certainty about the source of the outbreak, the transmissibility of the virus as well as the clinical picture and severity of the disease.In this issue of Eurosurveillance, we are publishing two articles on different aspects of the newly emerged 2019-nCoV. One is a research article by Corman et al. on the development of a diagnostic methodology based on RT-PCR of the E and RdRp genes, without the need for virus material; the assays were validated in five international laboratories . Before this publication, a description of the assays had already been made publically available on a dedicated WHO webpage to support rapid development of laboratory testing capacities. The other is a rapid communication where researchers based in Hong Kong report on their attempt to estimate the severity among hospitalised cases of 2019-nCoV infection through modelling based on publically available information, mainly from Wuhan health authorities . It also points out the need for more detailed information to make an informed evaluation of the situation as basis for public health decision-making.Today, the WHO Director-General Tedros Adhanom Ghebreyesus, taking into consideration the deliberations of the members of the International Health Regulations (IHR) Emergency Committee on 2019-nCoV in their second meeting, decided not to declare a public health emergency of international concern.International health organisations such as the European Centre for Disease Prevention and Control (ECDC) and the WHO are monitoring the situation and provide regular updates. ECDC has set up a dedicated webpage on which updates and risk assessments with focus on Europe are available:  novel-coronavirus-china.\n",
            "\t0.541\tA novel coronavirus (2019-nCoV) originating in Wuhan, China presents a potential respiratory viral pandemic to the world population. Current efforts are focused on containment and quarantine of infected individuals. Ultimately, the outbreak could be controlled with a protective vaccine to prevent 2019-nCoV infection. While vaccine research should be pursued intensely, there exists today no therapy to treat 2019-nCoV upon infection, despite an urgent need to find options to help these patients and preclude potential death. Herein, I review the potential options to treat 2019-nCoV in patients, with an emphasis on the necessity for speed and timeliness in developing new and effective therapies in this outbreak. I consider the options of drug repurposing, developing neutralizing monoclonal antibody therapy, and an oligonucleotide strategy targeting the viral RNA genome, emphasizing the promise and pitfalls of these approaches. Finally, I advocate for the fastest strategy to develop a treatment now, which could be resistant to any mutations the virus may have in the future. The proposal is a biologic that blocks 2019-nCoV entry using a soluble version of the viral receptor, angiotensin-converting enzyme 2 (ACE2), fused to an immunoglobulin Fc domain (ACE2-Fc), providing a neutralizing antibody with maximal breath to avoid any viral escape, while also helping to recruit the immune system to build lasting immunity. The ACE2-Fc therapy would also supplement decreased ACE2 levels in the lungs during infection, thereby directly treating acute respiratory distress pathophysiology as a third mechanism of action. The sequence of the ACE2-Fc protein is provided to investigators, allowing its possible use in recombinant protein expression systems to start producing drug today to treat patients under compassionate use, while formal clinical trials are later undertaken. Such a treatment could help infected patients before a protective vaccine is developed and widely available in the coming months to year(s).A mysterious illness causing pneumonia in December 2019 in Wuhan, China is now growing into a potential pandemic. These pneumonia cases were eventually characterized to be caused by a novel coronavirus (2019-nCoV) 1 , of which Severe Acute Respiratory Syndrome (SARS) 2 and Middle East Respiratory Syndrome (MERS) 3 are members. SARS and MERS famously caused their own outbreak concerns when they were originally identified. SARS caused significant economic damage to Hong Kong and Southern China, before spreading to other countries. Ultimately, SARS infected up to 8,098 people and caused 774 deaths according to the World Health Organization (WHO) 4 .The novel coronavirus, 2019-nCoV, is now quickly spreading across the world after originating in Wuhan 1 . Human-tohuman transmission of 2019-nCoV has been confirmed in familial case cluster reports 5 , as additional cases continue to be identified in different cities in China and countries around the world. Clinical symptoms of 2019-nCoV infection include fever, cough, and myalgia or fatigue with pneumonia demonstrated on chest CT scan imaging 6 . Within China, the city of Wuhan along with several others has been shut down, with individuals not allowed to leave the city in an effort to contain the virus; such efforts are largely unprecedented in a city of this size (). For now, many travelers are being screened for fever (≥38°C) and reported recent history of travel to Wuhan in order to triage diagnostic testing 7 . During those outbreaks, special protocols were put in place to quarantine any infected individuals and identify patient contacts at risk 8 . Healthcare workers were also at risk, and despite extensive personal protective equipment measures, clinical providers did get infected in both outbreaks 9 . There were no specific, antiviral treatments for SARS or Ebola at the time of the outbreaks beyond supportive measures 10,11 , which is a similar situation that healthcare systems are facing with 2019-nCoV.The dire situations facing patients in outbreak scenarios demand quick responses by the healthcare community and the biotech industry. Unfortunately, many of the traditional options that guide drug development are inadequate for outbreaks; a process that takes years can't help patients who are dying today, and economies that are being halted. In these situations, studies have often been conducted on compassionate use, and clinical trial approvals expedited. This was most recently seen in the 2014-2015 Ebola outbreak, where a variety of clinical trial candidates were studied. Many of these therapies failed, but ultimately a vaccine did emerge that was fully protective against the virus 12 . It is important to note that, unlike the current situation with 2019-nCoV, Ebola had already been studied for years and this particular neutralizing vaccine made and tested in preclinical animal models years prior to the outbreak 13 . For 2019-nCoV, beyond knowing the sequence of spike (S) protein of the coronavirus (GenBank: MN908947.3), there are no studies on how immunogenic this particular protein will be beyond surrogate comparisons to SARS and MERS, which limits the potential ability to quickly produce a vaccine. Moreover, while a vaccine would be greatly effective in helping to stop the spread of 2019-nCoV, an effective therapy is also needed for the patients infected with 2019-nCoV today, similar to the situation of Ebola patients needing effective therapies while vaccines were being developed.In this article, I will outline different potential treatment options that could be pursued as a therapy for 2019-nCoV virus, keeping the focus on agents that could be rapidly tested in patients today and broadly effective in spite of limited knowledge of the biology of 2019-nCoV. Simply stated, there is limited time for basic studies of 2019-nCoV in research labs, while patients need effective therapies today. I finally propose the best potential treatment option in my opinion, along with instructions on how to manufacture the therapy for testing in patients today.Developing neutralizing antibodies to 2019-nCoV Coronavirus entry starts with the S protein binding to a target receptor on the cell surface, where after fusion is mediated at the cell membrane, delivering the viral nucleocapsid inside the cell for subsequent replication 14 . The S protein is famous for causing syncytial formation between infected cells and other receptor-bearing cells around them, emphasizing that the S protein does not function in just the virion state alone.A neutralizing antibody targeting the S protein on the surface of 2019-nCoV is likely the first therapy contemplated by biomedical researchers in academia and industry, providing passive immunity to disease 15 . The recently published genome sequence of 2019-nCoV (GenBank: MN908947.3) allows researchers to perform gene synthesis in the lab and consider expressing the S protein as an immunogen. Traditional methods of screening mice or rabbits for neutralizing antibodies may be too slow for this outbreak, but faster methods such as using phage or yeast display libraries that express antibody fragments could be used quickly to identify lead candidates for viral neutralization 16, 17 . The challenge is that any antibody candidate would need to be rigorously tested in cell culture and animal models to confirm that it can neutralize 2019-nCoV and prevent infection. Furthermore, several isolates would need to be tested that are circulating in the population to try to assess if sufficient breadth of coverage is obtained with the neutralizing antibody. Information from other coronaviruses species like SARS would be helpful as to where to target the best epitope in order to produce neutralizing antibodies (the receptorbinding domain in the S protein is a key target) 18 , but again this is a slow and challenging process, which may not yield significant gains for several months. Moreover, ultimately a cocktail of antibodies may be required to ensure full protection for patients, which would add additional complexity for formulation and manufacturing. Like some of the therapeutic options discussed below, the ability to express any lead candidates in lower organisms for protein expression (bacteria, yeast, insect cells) would facilitate faster production of therapy for patients 19 .An alternative strategy of generating neutralizing antibodies against 2019-nCoV S protein would be to immunize large animals (sheep, goat, cow) with the 2019-nCoV S protein, and then purifying polyclonal antibodies from the animals 20 . This strategy may serve an expedited service in the setting of an outbreak and has many advantages such as simplifying production and manufacturing, but has limited guarantees that each animal would produce neutralizing antisera, or what the antibody titer would be in each animal 21 . Moreover, there is also the human immune response against foreign immunoglobulins to other species, which would potentially complicate any treatment scenarios 22 . In a truly desperate scenario, this strategy may be viable for a short-term, but would not easily scale in the 2019-nCoV outbreak, which is already rapidly multiplying.Using oligonucleotides against 2019-nCoV RNA genome Beyond targeting the surface proteins of 2019-nCoV, one could also target the RNA genome itself for degradation. This RNA genome sequence of 2019-nCoV was recently published (Gen-Bank: MN908947.3), and one strategy that could be considered then, is the use of small interfering RNA (siRNA) or antisense oligonucleotides (ASO) to combat the virus by targeting its RNA genome 23 . The challenge with this strategy is multifold. First, the conserved RNA sequence domains of CoV-2019 are not known. Identifying conserved sequences is essential in order to optimize siRNA targeting and avoid viral escape of the oligonucleotide strategy. One could look at genome homology of 2019-nCoV to the SARS virus for comparison of conserved sequences, but this would still be guesswork. A second challenge is how the oligonucleotides would be delivered into the lungs. Advances have been made into delivery vehicles such as lipid nanoparticles that can mediate some delivery into the lungs 24 . It is unknown, however, if enough siRNA's or ASO's would be effectively delivered within the lungs to stop the infection or make a difference in its clinical course. For example, if 25% of alveolar epithelial cells in the lung had siRNA or ASO in them, that efficiency might be a great success for traditional gene therapy, but would hardly make any difference in a viral infection. Such an explanation is also likely why siRNA candidates against Ebola failed in trials 25 , despite significant success in preclinical animal models 26,27 . Lastly, even if one assumed that siRNA was effective clinically, there is a limited ability to scale up manufacturing of siRNA drugs to a large infected population. Current siRNA and ASO therapies are manufactured for rare diseases, and there are no available resources existing to manufacture the medications quickly.Repurposing currently available antiviral medications Ideal agents to fight 2019-nCoV would be approved small molecule drugs that could inhibit different aspects of the viral life cycle, ultimately inhibiting replication. Two classes of potential targets are viral polymerases 28 and protease inhibitors 29 , both of which are components of human immunodeficiency virus (HIV) and hepatitis C virus (HCV) antiviral regimens. Pilot clinical studies are already ensuing by desperate clinicians with various repurposed antiviral medicines. This has been done in every viral outbreak previously with limited success, outside of case reports 30 . Indeed, during the Ebola outbreak, none of the repurposed small molecule drugs were definitively shown to improve the clinical course across all patients 31 . The 2019-nCoV could be different, and there are initial positive reports that lopinavir and ritonavir, which are HIV protease inhibitors, have some clinical efficacy against 2019-nCoV, similar to prior studies using them against SARS 32 . Research should continue to be undertaken to screen other clinically available antivirals in cell culture models of 2019-nCoV, in hopes that a drug candidate would emerge useful against the virus that could be rapidly implemented in the clinic. One promising example could be remdesivir, which interferes with the viral polymerase and has shown efficacy against MERS in mouse models 33 . For further information, reviews of previous drug repurposing efforts for coronaviruses are provided 34,35 . Though these repurposed medications may hold promise, it is still reasonable to pursue novel, 2019-nCoV specific therapies to complement potential repurposed drug candidates.Passive antibody transfer from convalescent patient sera A simple but potentially very effective tool that can be used in infectious outbreaks is to use the serum of patients who have recovered from the virus to treat patients who contract the virus in the future 36 . Patients with resolved viral infection will develop a polyclonal antibody immune response to different viral antigens of 2019-nCoV. Some of these polyclonal antibodies will likely neutralize the virus and prevent new rounds of infection, and the patients with resolved infection should produce 2019-nCoV antibodies in high titer.Patients with resolved cases of 2019-nCoV can simply donate plasma, and then this plasma can be transfused into infected patients 37 . Given that plasma donation is well established, and the transfusion of plasma is also routine medical care, this proposal does not need any new science or medical approvals in order to be put into place. Indeed, the same rationale was used in the treatment of several Ebola patients with convalescent serum during the outbreak in 2014-2015, including two American healthcare workers who became infected 38 .As the outbreak continues, more patients who survived infection will become available to serve as donors to make antisera for 2019-nCoV, and a sizeable stock of antisera could be developed to serve as a treatment for the sickest patients. Unfortunately, the exponential growth of the outbreak would work against this strategy, since the growing number of cases would likely outstrip the ability of previous patients to provide donor plasma as treatment. Moreover, convalescent patient sera would have significant variability in the potency of antiviral effect, making it less ideal 37 . While transfusion medicine services should certainly pursue convalescent patient sera as an option right now for patient treatment, it is ultimately limited in its effective scope of controlling the outbreak.The simplest and most direct approach to combating 2019-nCoV during the outbreak would be one to neutralize the virus from entering cells, the function that antibodies normally perform in the body 39 . For the reasons mentioned above when discussing neutralizing antibodies, it will be difficult to validate a broadly neutralizing antibody quickly, and a challenge to make sure that the mutating RNA virus will not escape its neutralization. A cocktail antibody approach could be undertaken as was explored to treat the Ebola pandemic 40 , but would add complexity to the manufacturing process.However, there is another strategy to pursue in this scenario that does not rely on targeting the viral glycoprotein directly. In this strategy, a neutralizing effect could be obtained by targeting the viral receptor protein on the cell surface, thereby blocking the virus from binding to it and gaining entry. Fortunately, scientists have already uncovered the identity of the viral receptor in cell culture. A recent pre-print publication found that the 2019-nCoV uses the angiotensin-converting enzyme 2 (ACE2) as a receptor for cell entry 41 , which is the same receptor that the SARS coronavirus uses for entry 42 . For both viruses, the coronavirus binds to ACE2 through its S protein on the virion, where after fusion of the viral membrane and cell membrane will occur. Subsequently, the RNA virus will replicate its genome inside the cell, and ultimately make new virions that will be secreted to infect other cells. The coincidence of SARS and 2019-nCoV using ACE2 receptor opens up the possibility of using the extensive research studied on SARS entry and applying it to 2019-nCoV. Based on the SARS literature, several potential blocking strategies could be considered, which were shown to be effective in preventing infection in SARS models.Blocking agents that bind to ACE2 receptor The first strategy would consist of administering to patients an agent that would bind to ACE2. The key advantage here is that the host ACE2 protein will not change, so there is no concern about escape from binding the therapeutic agent. Moreover, the virus will not have the ability to mutate and bind an entirely new host receptor in the time frame of this outbreak; such functional relationships are established by evolution over long periods. By analogy, the influenza virus changes the mutations on its surface to escape antibody neutralization every year, but it always focuses on using sialic acid on the cell surface as an entry receptor 43 .There are two known options for agents to bind to ACE2. The first is using the small receptor-binding domain (RBD) from the SARS S protein that has been shown to be the key domain that binds the ACE2 receptor during entry 44 . Administration of this domain, 193 amino acids in size, has been shown to effectively block the entry of SARS in cell culture 44 . It is well within reason that SARS RBD could be given to patients, thereby binding their ACE2 proteins on target cells, preventing infection (Figure 1 ). There is also the potential for the equivalent RBD of 2019-nCoV to be produced and used as a therapy as well. This strategy assumes SARS and 2019-nCoV share the same binding site on ACE2, which is highly likely given the similar ACE2 binding sites of SARS and NL63 coronavirus The small size of the therapy, similar in size in nanobody domains from camelid antibodies, would enhance the perfusion of the biologic into tissues to more effectively bind to viral entry receptors 45 . In regards to the outbreak situation that is ongoing, the small protein facilitates the rapid production of the therapy in bacteria potentially, which would help production yields 19 . Moreover, bacterial production would allow RBD proteins to be produced in a wide range of production facilities today in China, which already has numerous contract research organization operations 46 . Alternatively, the RBD protein could be attached to an Fc fragment for extended circulation, which was done for an equivalent 212 amino acid domain from MERS. The MERS RBD-Fc fusion demonstrated the ability to block viral infection toward cell receptors, as well as to stimulate an immune response against that specific viral domain in mice 47 . Of note, since the RBD-Fc fusion would bind to normal cells, one would want to eliminate cytotoxic Fc domain functions through mutations that eliminate Fc receptor binding 48 .A second, similar strategy would be to administer an antibody that would bind to ACE2 protein, thereby preventing 2019-nCoV infection (Figure 1 ). This strategy was shown to effectively block SARS entry and replication in experiments 42 . While no ACE2 antibody sequences are published in literature indexes, monoclonal antibodies do exist and the associated hybridoma sequences could be cloned in a matter of days. There would be no concern for any viral escape from an ACE2 binding antibody, which is an advantage over neutralizing approaches against the S protein. There are a couple of design considerations when thinking about how to employ the ACE2 antibody strategy. Any effector functions would need to be removed from the Fc domain 49 , such that inflammation would not be caused in different tissues expressing ACE2. This would retain the long-half life endowed by the Fc domain without any of the side effects. The downside of including the Fc domain is the need to use a more expensive mammalian cell production system to preserve proper glycosylation, which would decrease the turnaround time for getting the drug to patients in the outbreak scenario. Alternatively, one could just administer a single chain variable fragment (scFv) that binds to ACE2. A nanobody or VHH domains from camelids are another option as well 50,51 . These could be produced in bacteria, and its small size would allow for rapid permeation into different tissues. The downside is the shorter half-life of these molecules without the Fc domain.There are several limitations to these two options. Regarding the SARS RBD strategy, the body would likely develop an immune response to the SARS protein eventually, although the key intervention period of infection to combat 2019-nCoV would fall under this window of time, where after an immune response for both viruses would develop. Alternatively, if one were to use the homologous RBD from 2019-nCoV itself, this immune response would likely be very advantageous since it could yield both a blocking effect and a vaccination effect 52 . For both strategies, the dose that would be needed to block ACE2 receptors in the body across different organs is unknown, and as is the percentage of ACE2 receptors that would need to be saturated in order to slow the infection. The number of ACE2 receptors in the body, which are found in lung and gastrointestinal organs along with vascular endothelial cells among other tissues 53 , could ultimately prove prohibitive for this strategy. Moreover, the turnover of the ACE2 receptor on the cell surface would also influence how often the therapeutic protein would need to be administered. To solve this issue, one could increase the concentration of anti-ACE2 therapy at the crucial site of infection in the lungs, via local administration to lungs via nebulization. Lastly, there is the possibility that binding ACE2 directly could paradoxically worsen lung physiology and clinical symptoms. A study found that a fusion protein of SARS RBD to Fc domain bound ACE2 in murine lung tissue after administration, exacerbating alveolar edema via ACE2 interaction, which normally counteracts acute lung injury 54 . This suggests that if one were use an ACE2 binding strategy, it would be best employed early during infection or as a prophylaxis to block the initial viral infection. Ultimately, clinical trials in patients would need to investigate these potential issues.A potentially more promising strategy would be to create an antibody-like molecule that would bind to the coronavirus itself, rather than shielding cells from being infected. For this strategy, it is proposed to use a soluble version of the ACE2 receptor that would bind to the S protein of 2019-nCoV thereby neutralizing the virus (Figure 1) . Again, the research on the SARS virus suggests this strategy is potentially promising. Soluble ACE2 receptor was demonstrated to block the SARS virus from infecting cells in culture 42 . The reported affinity of soluble ACE2 for the SARS S protein was 1.70 nM, which is comparable to the affinities of monoclonal antibodies 55 ; it is likely that 2019-nCoV has similar affinity for ACE2. In order to use ACE2 as a therapy to treat patients, it would be advisable to convert soluble ACE2 into an immunoadhesin format fused to an immunoglobulin Fc domain (ACE2-Fc), thereby extending the lifespan of the circulating molecule, while also recruiting effector functions of the immune system against the virus. While not tested in an animal model, a previous study demonstrated that an ACE2 extracellular domain fused to the human IgG1 domain (ACE2-NN-Ig) was effective in neutralizing SARS coronavirus in vitro, with a 50% inhibitory concentration of 2 nM 56 . This study provides evidence then that ACE2-Fc could similarly inhibit 2019-nCoV in vitro and potentially in patients.An additional advantage of using ACE2 as a 2019-nCoV S protein neutralizing agent is that ACE2 administration could also directly treat the pneumonia pathophysiology. A portion of patients with SARS and 2019-nCoV infection develop pneumonia, which is characterized by pulmonary edema and acute respiratory distress syndrome (ARDS) 1,2 . The viruses may, in part, cause ARDS through viral-induced ACE2 protein shedding and ACE2 protein decreased expression, both of which are mediated by S protein binding 54 . Administration of recombinant ACE2 protein has been shown to improve acute lung injury through decreasing angiotensin II levels and the hormones subsequent binding to angiotensin II type 1a receptor 57 . Recombinant ACE2 can also reduce ARDS in respiratory syncytial virus 58 and H5N1 influenza 59 infection models. Based on these promising preclinical studies, recombinant human ACE2 (rhACE2) was moved into clinical trials in order to treat ARDS in critically ill patients. A phase I trial demonstrated rhACE2 was well tolerated with no effects seen on the cardiovascular system 60 . A phase II trial demonstrated on-target efficacy in reducing Ang1-8 peptide levels, but did not show significant modulation of respiratory parameters 61 . It remains to be seen whether rhACE2 administration has the same clinical benefits in treating ARDS that have been seen in animal models, and whether ACE2-Fc administration could alleviate ARDS in 2019-nCoV patients.The proposed therapy for 2019-nCoV patients would consist of the extracellular domain of the ACE2 protein fused to a human immunoglobulin G Fc domain (Figure 2A) . Studies have shown that the ACE2 amino acids 18 -615 appear to be sufficient for SARS S protein binding 62 , which also covers the peptidase domain necessary for ACE2 enzymatic function. It is possible a smaller portion of the extracellular ACE2 domain would be adequate for S protein binding, although a smaller version would lack enzyme activity beneficial in treating lung injury. Further studies are needed to define the minimal ACE2 domain necessary for 2019-nCoV S protein binding to construct even smaller ACE2-Fc proteins. While we do not know the structure of the 2019-nCoV S protein or how it binds to the ACE2 receptor yet, it is reasonable for now to assume that the same ACE2 protein domains utilized by the SARS virus are also bound by 2019-nCoV to infect cells.The advantage of the Fc domain is endowing a longer-half life of the drug, which could enable healthcare workers to potentially be given drug doses prophylactically before seeing infected patients. Indeed, the half-life of recombinant ACE2 was extended from less than two hours to over one week in mice when formatted as a recombinant ACE2-Fc therapy in a study evaluating treatment for hypertension 63 . One difference from the prior blocking agent strategies is that the effector functions of the Fc domain could be retained in this molecule, allowing recruitment of dendritic cells, macrophages, and natural killer cells through the CD16 receptor against viral particles or infected cells. This may facilitate faster activation of the host antiviral immune response and elimination of the virus, which was illustrated in a SARS mouse model where Fc engaging antibodies were more potent in eliminating SARS via activation of phagocytic cells compared to antibodies that neutralized virus alone 64 . Overall, the ACE2-Fc fusion protein would have many of the same benefits of a traditional neutralizing antibody that would be sought as a treatment for the infection, but represent one with maximal breadth and potency since the 2019-nCoV could not escape its neutralization, given the same protein is also its receptor for cell entry. Indeed, it has been shown that the pathogenicity of SARS versus the more mild human coronavirus NL63 was related to a lower affinity of NL63 for human ACE2 versus SARS, with NL63 S protein reducing ACE2 levels less than SARS S protein 65 . Therefore, if 2019-nCoV were to try to escape ACE2 neutralization via decreasing affinity, it would mutate into a less pathogenic virus. This is similar to the re-emergent SARS virus in 2003-2004, which had lower affinity for human ACE2 and resulted in less severe infection and no secondary transmission 66 . Thus, 2019-nCoV could be presented with an evolutionary trap when faced with potential ACE2-Fc therapy, leading toward a more benign clinical course.To give some additional support to the potential of a receptorimmunoadhesin being a potential antiviral strategy, it should be noted that CD4-Fc or CD4-IgG was one of the early agents developed as a potential HIV medication 67 . The protein contained the first two domains of the CD4 receptor that are known to bind gp120 on the surface of infected HIV cells. CD4-IgG was shown to neutralize HIV in vitro, preventing infection. The protein was also safe when administered in patients, although only limited-to-mild clinical benefit was achieved 68,69 . Updated enhanced versions of CD4-IgG have been developed that additionally have a small peptide derived from the co-receptor, CCR5, enhancing affinity and giving even more potent neutralizing activity, essentially 100% of HIV isolates and making rhesus macaques resistant to multiple simian-human immunodeficiency virus challenges 70,71 . While HIV and 2019-nCoV are very different viruses, with different cell types, kinetics, and clinical courses, the previous results with HIV are encouraging that this could be a therapeutic strategy for 2019-nCoV. If anything, 2019-nCoV is likely more amenable to this neutralizing therapy given that the respiratory virus will only cause an acute infection, unlike HIV, which causes chronic infection in hosts with different cellular reservoirs.One potential limitation of the ACE2-Fc strategy is that the increase in levels of extracellular ACE2 could have unknown effects on the body, particularly when elevated for a prolonged time via Fc domain extended half-life. Small levels of extracellular ACE2 are already secreted by tissues, so the circulation of this extracellular domain would not be unprecedented 72 . Moreover, recombinant ACE2 protein was well-tolerated by healthy patients in a phase I trial, and by patients with lung injury in a phase II trial, suggesting treating 2019-nCoV patients with ACE2-Fc will also tolerated. If investigators are still concerned, critical amino acid(s) for ACE2 peptidase activity could be mutated to abolish the native function of this sequence, while retaining high affinity binding for SARS and 2019-nCoV S protein. Indeed, this possibility was previously investigated in generating an ACE2 and IgG1 fusion protein, which showed that mutation of histidine residues at position 374 and 378 of the ACE2 extracellular domain abolished peptidase activity, but retained high affinity binding to SARS S protein 56 . Of course, ACE2 peptidase mutation would eliminate the beneficial effects from the recombinant protein delivery of ACE2 in treating lung injury, so it is recommended that retaining ACE2 enzyme activity be pursued first. Another potential concern is that receptor binding via an antibody format could inadvertently direct 2019-nCoV toward infecting Fc receptor (CD16) positive cells, which has been shown in vitro for neutralizing antibodies in MERS 73 . It's unclear what clinical significance this would have, and to what extent this would happen in vivo. Ultimately, clinical trials will be needed to delineate any specific side effects of ACE2-Fc treatment.The chief objective of global health efforts against 2019-nCoV remains to effectively quarantine patients and screen individuals who may be infected to limit spread. That objective should continue going forward. What is proposed here is an option to at least give infected patients a medication quickly to help alleviate symptoms and prevent death, while vaccine efforts for 2019-nCoV continue. This could be enabled through ACE2-Fc providing a triple mechanism of action: (1) Treatment of ACE2 deficiency and lung injury, (2) virus neutralization, and (3) immune effector recruitment. Beyond infected patients, ACE2-Fc could provide passive immunity to healthcare works at risk as another benefit. Going forward, it is recommended that physicians, scientists, and biotech industry in China and elsewhere pursue manufacturing an ACE2-Fc biologic agent right now, which can immediately advance into trials. A variety of different protein expression platforms (CHO, insect, yeast) could be utilized, depending on the particular contract manufacturer's expertise. Gene therapy could even be considered to make ACE2-Fc from a DNA or mRNA platform, but would have additional risk of uncertain delivery strategies and ultimately may slow down progress toward treating patients.The goal would be that ACE2-Fc could treat infection in current patients preventing significant morbidities and death, while also serving as a potential prophylactic to give passive immunity to clinical providers on the frontlines, as well as individuals who may have been exposed to the virus. Essentially, ACE2-Fc could be the potent neutralizing antibody that the global health community needs to combat 2019-nCoV, today, while also treating the underlying ARDS pathophysiology causing patient mortality. It could be scaled much more quickly than convalescent patient sera, which would be dependent on infected individuals to make. ACE2-Fc would be resistant to viral escape as well, unlike potential neutralizing monoclonal antibodies that may be developed in the coming weeks to months.While a therapeutic strategy is being outlined here, the longterm goal of 2019-nCoV research would remain developing an effective vaccine to yield neutralizing antibodies, likely based on the S protein and specifically, the RBD protein. Such trials should happen as soon as possible, but may prove to be challenging to get the right level of immunogenicity, antigen presentation, adjuvant addition, and potent antibody stimulation. The virus could continue mutating, foiling different efforts to stimulate protective immunity. By comparison, 2019-nCoV cannot escape the ACE2-Fc treatment strategy, since it leverages its own cognate receptor for infection. As mentioned above, should 2019-nCoV attempt to escape this therapy via reduced ACE2 affinity binding, it would likely become less pathogenic, similar to the comparison of SARS versus human coronavirus NL63 65 . Lastly, scaling the dose of any effective vaccine would also prove to be challenging depending on the vector format (e.g. viral vector versus mRNA versus protein), and even a fully protective vaccine would not help patients who are currently infected with the virus.In an effort to help aide researchers and industry in China to combat 2019-nCoV, the protein sequence of the ACE2-Fc construct is provided ( Figure 2B ). Different human Fc domains (IgG1, IgG2, IgG3, or IgG4) could be contemplated, although IgG1 traditionally has the most potency for triggering anti-microbial responses 49 . Similarly, an ACE2-Fc biologic without active ACE2 peptidase function could be explored as well. Given that gene synthesis of this sequence could happen within a week, the gene could be placed within the protein expression platform of choice shortly thereafter leading to protein production quickly. The availability of protein A columns and other techniques in the industry to purify antibodies would facilitate ACE2-Fc to quickly be repurposed on existing antibody manufacturing infrastructure existing in China.A final benefit of pursuing ACE2-Fc is that it could effectively be used as a therapeutic drug stockpile for future outbreaks of SARS and 2019-nCoV, and any new coronavirus that emerges from a zoonotic reservoir in the future that uses the ACE2 receptor for entry. Moreover, coronaviruses that replicate in animals across China and other countries could be studied in order to assess their entry mechanisms. By understanding entry in these other animals, one could effectively predict a receptor that could be utilized in any zoonotic transmission event, and build a new receptor immunoadhesin molecule in the future. As an example, a similar immunoadhesin, DPP4-Fc, could be envisioned for MERS based on the viral receptor, DPP4, used by that virus 74 . Beyond coronaviruses, this strategy could be utilized for other viruses where the risk of outbreak potential is high. ACE2-Fc could also find use in treating ARDS for other unrelated viruses and causes of acute lung injury, building on the previous clinical trial work 60,61 .In summary, ACE2-Fc has the potential to be the neutralizing antibody that healthcare workers need to treat and prevent 2019-nCoV infection today and could play an important role in the cessation of the outbreak if manufacturing based on an available sequence starts soon. An alternative 2019-nCoV RBD-Fc fusion could also be pursued, if one desired the dual function of receptor blocking and vaccination in one molecule.Underlying data No data are associated with this article. \n",
            "\t0.541\tA novel coronavirus (2019-nCoV) originating in Wuhan, China presents a potential respiratory viral pandemic to the world population. Current efforts are focused on containment and quarantine of infected individuals. Ultimately, the outbreak could be controlled with a protective vaccine to prevent 2019-nCoV infection. While vaccine research should be pursued intensely, there exists today no therapy to treat 2019-nCoV upon infection, despite an urgent need to find options to help these patients and preclude potential death. Herein, I review the potential options to treat 2019-nCoV in patients, with an emphasis on the necessity for speed and timeliness in developing new and effective therapies in this outbreak. I consider the options of drug repurposing, developing neutralizing monoclonal antibody therapy, and an oligonucleotide strategy targeting the viral RNA genome, emphasizing the promise and pitfalls of these approaches. Finally, I advocate for the fastest strategy to develop a treatment now, which could be resistant to any mutations the virus may have in the future. The proposal is a biologic that blocks 2019-nCoV entry using a soluble version of the viral receptor, angiotensin-converting enzyme 2 (ACE2), fused to an immunoglobulin Fc domain (ACE2-Fc), providing a neutralizing antibody with maximal breath to avoid any viral escape, while also helping to recruit the immune system to build lasting immunity. The ACE2-Fc therapy would also supplement decreased ACE2 levels in the lungs during infection, thereby directly treating acute respiratory distress pathophysiology as a third mechanism of action. The sequence of the ACE2-Fc protein is provided to investigators, allowing its possible use in recombinant protein expression systems to start producing drug today to treat patients under compassionate use, while formal clinical trials are later undertaken. Such a treatment could help infected patients before a protective vaccine is developed and widely available in the coming months to year(s).A mysterious illness causing pneumonia in December 2019 in Wuhan, China is now growing into a potential pandemic. These pneumonia cases were eventually characterized to be caused by a novel coronavirus (2019-nCoV) 1 , of which Severe Acute Respiratory Syndrome (SARS) 2 and Middle East Respiratory Syndrome (MERS) 3 are members. SARS and MERS famously caused their own outbreak concerns when they were originally identified. SARS caused significant economic damage to Hong Kong and Southern China, before spreading to other countries. Ultimately, SARS infected up to 8,098 people and caused 774 deaths according to the World Health Organization (WHO) 4 .The novel coronavirus, 2019-nCoV, is now quickly spreading across the world after originating in Wuhan 1 . Human-tohuman transmission of 2019-nCoV has been confirmed in familial case cluster reports 5 , as additional cases continue to be identified in different cities in China and countries around the world. Clinical symptoms of 2019-nCoV infection include fever, cough, and myalgia or fatigue with pneumonia demonstrated on chest CT scan imaging 6 . Within China, the city of Wuhan along with several others has been shut down, with individuals not allowed to leave the city in an effort to contain the virus; such efforts are largely unprecedented in a city of this size (). For now, many travelers are being screened for fever (≥38°C) and reported recent history of travel to Wuhan in order to triage diagnostic testing 7 . During those outbreaks, special protocols were put in place to quarantine any infected individuals and identify patient contacts at risk 8 . Healthcare workers were also at risk, and despite extensive personal protective equipment measures, clinical providers did get infected in both outbreaks 9 . There were no specific, antiviral treatments for SARS or Ebola at the time of the outbreaks beyond supportive measures 10,11 , which is a similar situation that healthcare systems are facing with 2019-nCoV.The dire situations facing patients in outbreak scenarios demand quick responses by the healthcare community and the biotech industry. Unfortunately, many of the traditional options that guide drug development are inadequate for outbreaks; a process that takes years can't help patients who are dying today, and economies that are being halted. In these situations, studies have often been conducted on compassionate use, and clinical trial approvals expedited. This was most recently seen in the 2014-2015 Ebola outbreak, where a variety of clinical trial candidates were studied. Many of these therapies failed, but ultimately a vaccine did emerge that was fully protective against the virus 12 . It is important to note that, unlike the current situation with 2019-nCoV, Ebola had already been studied for years and this particular neutralizing vaccine made and tested in preclinical animal models years prior to the outbreak 13 . For 2019-nCoV, beyond knowing the sequence of spike (S) protein of the coronavirus (GenBank: MN908947.3), there are no studies on how immunogenic this particular protein will be beyond surrogate comparisons to SARS and MERS, which limits the potential ability to quickly produce a vaccine. Moreover, while a vaccine would be greatly effective in helping to stop the spread of 2019-nCoV, an effective therapy is also needed for the patients infected with 2019-nCoV today, similar to the situation of Ebola patients needing effective therapies while vaccines were being developed.In this article, I will outline different potential treatment options that could be pursued as a therapy for 2019-nCoV virus, keeping the focus on agents that could be rapidly tested in patients today and broadly effective in spite of limited knowledge of the biology of 2019-nCoV. Simply stated, there is limited time for basic studies of 2019-nCoV in research labs, while patients need effective therapies today. I finally propose the best potential treatment option in my opinion, along with instructions on how to manufacture the therapy for testing in patients today.Developing neutralizing antibodies to 2019-nCoV Coronavirus entry starts with the S protein binding to a target receptor on the cell surface, where after fusion is mediated at the cell membrane, delivering the viral nucleocapsid inside the cell for subsequent replication 14 . The S protein is famous for causing syncytial formation between infected cells and other receptor-bearing cells around them, emphasizing that the S protein does not function in just the virion state alone.A neutralizing antibody targeting the S protein on the surface of 2019-nCoV is likely the first therapy contemplated by biomedical researchers in academia and industry, providing passive immunity to disease 15 . The recently published genome sequence of 2019-nCoV (GenBank: MN908947.3) allows researchers to perform gene synthesis in the lab and consider expressing the S protein as an immunogen. Traditional methods of screening mice or rabbits for neutralizing antibodies may be too slow for this outbreak, but faster methods such as using phage or yeast display libraries that express antibody fragments could be used quickly to identify lead candidates for viral neutralization 16, 17 . The challenge is that any antibody candidate would need to be rigorously tested in cell culture and animal models to confirm that it can neutralize 2019-nCoV and prevent infection. Furthermore, several isolates would need to be tested that are circulating in the population to try to assess if sufficient breadth of coverage is obtained with the neutralizing antibody. Information from other coronaviruses species like SARS would be helpful as to where to target the best epitope in order to produce neutralizing antibodies (the receptorbinding domain in the S protein is a key target) 18 , but again this is a slow and challenging process, which may not yield significant gains for several months. Moreover, ultimately a cocktail of antibodies may be required to ensure full protection for patients, which would add additional complexity for formulation and manufacturing. Like some of the therapeutic options discussed below, the ability to express any lead candidates in lower organisms for protein expression (bacteria, yeast, insect cells) would facilitate faster production of therapy for patients 19 .An alternative strategy of generating neutralizing antibodies against 2019-nCoV S protein would be to immunize large animals (sheep, goat, cow) with the 2019-nCoV S protein, and then purifying polyclonal antibodies from the animals 20 . This strategy may serve an expedited service in the setting of an outbreak and has many advantages such as simplifying production and manufacturing, but has limited guarantees that each animal would produce neutralizing antisera, or what the antibody titer would be in each animal 21 . Moreover, there is also the human immune response against foreign immunoglobulins to other species, which would potentially complicate any treatment scenarios 22 . In a truly desperate scenario, this strategy may be viable for a short-term, but would not easily scale in the 2019-nCoV outbreak, which is already rapidly multiplying.Using oligonucleotides against 2019-nCoV RNA genome Beyond targeting the surface proteins of 2019-nCoV, one could also target the RNA genome itself for degradation. This RNA genome sequence of 2019-nCoV was recently published (Gen-Bank: MN908947.3), and one strategy that could be considered then, is the use of small interfering RNA (siRNA) or antisense oligonucleotides (ASO) to combat the virus by targeting its RNA genome 23 . The challenge with this strategy is multifold. First, the conserved RNA sequence domains of CoV-2019 are not known. Identifying conserved sequences is essential in order to optimize siRNA targeting and avoid viral escape of the oligonucleotide strategy. One could look at genome homology of 2019-nCoV to the SARS virus for comparison of conserved sequences, but this would still be guesswork. A second challenge is how the oligonucleotides would be delivered into the lungs. Advances have been made into delivery vehicles such as lipid nanoparticles that can mediate some delivery into the lungs 24 . It is unknown, however, if enough siRNA's or ASO's would be effectively delivered within the lungs to stop the infection or make a difference in its clinical course. For example, if 25% of alveolar epithelial cells in the lung had siRNA or ASO in them, that efficiency might be a great success for traditional gene therapy, but would hardly make any difference in a viral infection. Such an explanation is also likely why siRNA candidates against Ebola failed in trials 25 , despite significant success in preclinical animal models 26,27 . Lastly, even if one assumed that siRNA was effective clinically, there is a limited ability to scale up manufacturing of siRNA drugs to a large infected population. Current siRNA and ASO therapies are manufactured for rare diseases, and there are no available resources existing to manufacture the medications quickly.Repurposing currently available antiviral medications Ideal agents to fight 2019-nCoV would be approved small molecule drugs that could inhibit different aspects of the viral life cycle, ultimately inhibiting replication. Two classes of potential targets are viral polymerases 28 and protease inhibitors 29 , both of which are components of human immunodeficiency virus (HIV) and hepatitis C virus (HCV) antiviral regimens. Pilot clinical studies are already ensuing by desperate clinicians with various repurposed antiviral medicines. This has been done in every viral outbreak previously with limited success, outside of case reports 30 . Indeed, during the Ebola outbreak, none of the repurposed small molecule drugs were definitively shown to improve the clinical course across all patients 31 . The 2019-nCoV could be different, and there are initial positive reports that lopinavir and ritonavir, which are HIV protease inhibitors, have some clinical efficacy against 2019-nCoV, similar to prior studies using them against SARS 32 . Research should continue to be undertaken to screen other clinically available antivirals in cell culture models of 2019-nCoV, in hopes that a drug candidate would emerge useful against the virus that could be rapidly implemented in the clinic. One promising example could be remdesivir, which interferes with the viral polymerase and has shown efficacy against MERS in mouse models 33 . For further information, reviews of previous drug repurposing efforts for coronaviruses are provided 34,35 . Though these repurposed medications may hold promise, it is still reasonable to pursue novel, 2019-nCoV specific therapies to complement potential repurposed drug candidates.Passive antibody transfer from convalescent patient sera A simple but potentially very effective tool that can be used in infectious outbreaks is to use the serum of patients who have recovered from the virus to treat patients who contract the virus in the future 36 . Patients with resolved viral infection will develop a polyclonal antibody immune response to different viral antigens of 2019-nCoV. Some of these polyclonal antibodies will likely neutralize the virus and prevent new rounds of infection, and the patients with resolved infection should produce 2019-nCoV antibodies in high titer.Patients with resolved cases of 2019-nCoV can simply donate plasma, and then this plasma can be transfused into infected patients 37 . Given that plasma donation is well established, and the transfusion of plasma is also routine medical care, this proposal does not need any new science or medical approvals in order to be put into place. Indeed, the same rationale was used in the treatment of several Ebola patients with convalescent serum during the outbreak in 2014-2015, including two American healthcare workers who became infected 38 .As the outbreak continues, more patients who survived infection will become available to serve as donors to make antisera for 2019-nCoV, and a sizeable stock of antisera could be developed to serve as a treatment for the sickest patients. Unfortunately, the exponential growth of the outbreak would work against this strategy, since the growing number of cases would likely outstrip the ability of previous patients to provide donor plasma as treatment. Moreover, convalescent patient sera would have significant variability in the potency of antiviral effect, making it less ideal 37 . While transfusion medicine services should certainly pursue convalescent patient sera as an option right now for patient treatment, it is ultimately limited in its effective scope of controlling the outbreak.The simplest and most direct approach to combating 2019-nCoV during the outbreak would be one to neutralize the virus from entering cells, the function that antibodies normally perform in the body 39 . For the reasons mentioned above when discussing neutralizing antibodies, it will be difficult to validate a broadly neutralizing antibody quickly, and a challenge to make sure that the mutating RNA virus will not escape its neutralization. A cocktail antibody approach could be undertaken as was explored to treat the Ebola pandemic 40 , but would add complexity to the manufacturing process.However, there is another strategy to pursue in this scenario that does not rely on targeting the viral glycoprotein directly. In this strategy, a neutralizing effect could be obtained by targeting the viral receptor protein on the cell surface, thereby blocking the virus from binding to it and gaining entry. Fortunately, scientists have already uncovered the identity of the viral receptor in cell culture. A recent pre-print publication found that the 2019-nCoV uses the angiotensin-converting enzyme 2 (ACE2) as a receptor for cell entry 41 , which is the same receptor that the SARS coronavirus uses for entry 42 . For both viruses, the coronavirus binds to ACE2 through its S protein on the virion, where after fusion of the viral membrane and cell membrane will occur. Subsequently, the RNA virus will replicate its genome inside the cell, and ultimately make new virions that will be secreted to infect other cells. The coincidence of SARS and 2019-nCoV using ACE2 receptor opens up the possibility of using the extensive research studied on SARS entry and applying it to 2019-nCoV. Based on the SARS literature, several potential blocking strategies could be considered, which were shown to be effective in preventing infection in SARS models.Blocking agents that bind to ACE2 receptor The first strategy would consist of administering to patients an agent that would bind to ACE2. The key advantage here is that the host ACE2 protein will not change, so there is no concern about escape from binding the therapeutic agent. Moreover, the virus will not have the ability to mutate and bind an entirely new host receptor in the time frame of this outbreak; such functional relationships are established by evolution over long periods. By analogy, the influenza virus changes the mutations on its surface to escape antibody neutralization every year, but it always focuses on using sialic acid on the cell surface as an entry receptor 43 .There are two known options for agents to bind to ACE2. The first is using the small receptor-binding domain (RBD) from the SARS S protein that has been shown to be the key domain that binds the ACE2 receptor during entry 44 . Administration of this domain, 193 amino acids in size, has been shown to effectively block the entry of SARS in cell culture 44 . It is well within reason that SARS RBD could be given to patients, thereby binding their ACE2 proteins on target cells, preventing infection (Figure 1 ). There is also the potential for the equivalent RBD of 2019-nCoV to be produced and used as a therapy as well. This strategy assumes SARS and 2019-nCoV share the same binding site on ACE2, which is highly likely given the similar ACE2 binding sites of SARS and NL63 coronavirus The small size of the therapy, similar in size in nanobody domains from camelid antibodies, would enhance the perfusion of the biologic into tissues to more effectively bind to viral entry receptors 45 . In regards to the outbreak situation that is ongoing, the small protein facilitates the rapid production of the therapy in bacteria potentially, which would help production yields 19 . Moreover, bacterial production would allow RBD proteins to be produced in a wide range of production facilities today in China, which already has numerous contract research organization operations 46 . Alternatively, the RBD protein could be attached to an Fc fragment for extended circulation, which was done for an equivalent 212 amino acid domain from MERS. The MERS RBD-Fc fusion demonstrated the ability to block viral infection toward cell receptors, as well as to stimulate an immune response against that specific viral domain in mice 47 . Of note, since the RBD-Fc fusion would bind to normal cells, one would want to eliminate cytotoxic Fc domain functions through mutations that eliminate Fc receptor binding 48 .A second, similar strategy would be to administer an antibody that would bind to ACE2 protein, thereby preventing 2019-nCoV infection (Figure 1 ). This strategy was shown to effectively block SARS entry and replication in experiments 42 . While no ACE2 antibody sequences are published in literature indexes, monoclonal antibodies do exist and the associated hybridoma sequences could be cloned in a matter of days. There would be no concern for any viral escape from an ACE2 binding antibody, which is an advantage over neutralizing approaches against the S protein. There are a couple of design considerations when thinking about how to employ the ACE2 antibody strategy. Any effector functions would need to be removed from the Fc domain 49 , such that inflammation would not be caused in different tissues expressing ACE2. This would retain the long-half life endowed by the Fc domain without any of the side effects. The downside of including the Fc domain is the need to use a more expensive mammalian cell production system to preserve proper glycosylation, which would decrease the turnaround time for getting the drug to patients in the outbreak scenario. Alternatively, one could just administer a single chain variable fragment (scFv) that binds to ACE2. A nanobody or VHH domains from camelids are another option as well 50,51 . These could be produced in bacteria, and its small size would allow for rapid permeation into different tissues. The downside is the shorter half-life of these molecules without the Fc domain.There are several limitations to these two options. Regarding the SARS RBD strategy, the body would likely develop an immune response to the SARS protein eventually, although the key intervention period of infection to combat 2019-nCoV would fall under this window of time, where after an immune response for both viruses would develop. Alternatively, if one were to use the homologous RBD from 2019-nCoV itself, this immune response would likely be very advantageous since it could yield both a blocking effect and a vaccination effect 52 . For both strategies, the dose that would be needed to block ACE2 receptors in the body across different organs is unknown, and as is the percentage of ACE2 receptors that would need to be saturated in order to slow the infection. The number of ACE2 receptors in the body, which are found in lung and gastrointestinal organs along with vascular endothelial cells among other tissues 53 , could ultimately prove prohibitive for this strategy. Moreover, the turnover of the ACE2 receptor on the cell surface would also influence how often the therapeutic protein would need to be administered. To solve this issue, one could increase the concentration of anti-ACE2 therapy at the crucial site of infection in the lungs, via local administration to lungs via nebulization. Lastly, there is the possibility that binding ACE2 directly could paradoxically worsen lung physiology and clinical symptoms. A study found that a fusion protein of SARS RBD to Fc domain bound ACE2 in murine lung tissue after administration, exacerbating alveolar edema via ACE2 interaction, which normally counteracts acute lung injury 54 . This suggests that if one were use an ACE2 binding strategy, it would be best employed early during infection or as a prophylaxis to block the initial viral infection. Ultimately, clinical trials in patients would need to investigate these potential issues.A potentially more promising strategy would be to create an antibody-like molecule that would bind to the coronavirus itself, rather than shielding cells from being infected. For this strategy, it is proposed to use a soluble version of the ACE2 receptor that would bind to the S protein of 2019-nCoV thereby neutralizing the virus (Figure 1) . Again, the research on the SARS virus suggests this strategy is potentially promising. Soluble ACE2 receptor was demonstrated to block the SARS virus from infecting cells in culture 42 . The reported affinity of soluble ACE2 for the SARS S protein was 1.70 nM, which is comparable to the affinities of monoclonal antibodies 55 ; it is likely that 2019-nCoV has similar affinity for ACE2. In order to use ACE2 as a therapy to treat patients, it would be advisable to convert soluble ACE2 into an immunoadhesin format fused to an immunoglobulin Fc domain (ACE2-Fc), thereby extending the lifespan of the circulating molecule, while also recruiting effector functions of the immune system against the virus. While not tested in an animal model, a previous study demonstrated that an ACE2 extracellular domain fused to the human IgG1 domain (ACE2-NN-Ig) was effective in neutralizing SARS coronavirus in vitro, with a 50% inhibitory concentration of 2 nM 56 . This study provides evidence then that ACE2-Fc could similarly inhibit 2019-nCoV in vitro and potentially in patients.An additional advantage of using ACE2 as a 2019-nCoV S protein neutralizing agent is that ACE2 administration could also directly treat the pneumonia pathophysiology. A portion of patients with SARS and 2019-nCoV infection develop pneumonia, which is characterized by pulmonary edema and acute respiratory distress syndrome (ARDS) 1,2 . The viruses may, in part, cause ARDS through viral-induced ACE2 protein shedding and ACE2 protein decreased expression, both of which are mediated by S protein binding 54 . Administration of recombinant ACE2 protein has been shown to improve acute lung injury through decreasing angiotensin II levels and the hormones subsequent binding to angiotensin II type 1a receptor 57 . Recombinant ACE2 can also reduce ARDS in respiratory syncytial virus 58 and H5N1 influenza 59 infection models. Based on these promising preclinical studies, recombinant human ACE2 (rhACE2) was moved into clinical trials in order to treat ARDS in critically ill patients. A phase I trial demonstrated rhACE2 was well tolerated with no effects seen on the cardiovascular system 60 . A phase II trial demonstrated on-target efficacy in reducing Ang1-8 peptide levels, but did not show significant modulation of respiratory parameters 61 . It remains to be seen whether rhACE2 administration has the same clinical benefits in treating ARDS that have been seen in animal models, and whether ACE2-Fc administration could alleviate ARDS in 2019-nCoV patients.The proposed therapy for 2019-nCoV patients would consist of the extracellular domain of the ACE2 protein fused to a human immunoglobulin G Fc domain (Figure 2A) . Studies have shown that the ACE2 amino acids 18 -615 appear to be sufficient for SARS S protein binding 62 , which also covers the peptidase domain necessary for ACE2 enzymatic function. It is possible a smaller portion of the extracellular ACE2 domain would be adequate for S protein binding, although a smaller version would lack enzyme activity beneficial in treating lung injury. Further studies are needed to define the minimal ACE2 domain necessary for 2019-nCoV S protein binding to construct even smaller ACE2-Fc proteins. While we do not know the structure of the 2019-nCoV S protein or how it binds to the ACE2 receptor yet, it is reasonable for now to assume that the same ACE2 protein domains utilized by the SARS virus are also bound by 2019-nCoV to infect cells.The advantage of the Fc domain is endowing a longer-half life of the drug, which could enable healthcare workers to potentially be given drug doses prophylactically before seeing infected patients. Indeed, the half-life of recombinant ACE2 was extended from less than two hours to over one week in mice when formatted as a recombinant ACE2-Fc therapy in a study evaluating treatment for hypertension 63 . One difference from the prior blocking agent strategies is that the effector functions of the Fc domain could be retained in this molecule, allowing recruitment of dendritic cells, macrophages, and natural killer cells through the CD16 receptor against viral particles or infected cells. This may facilitate faster activation of the host antiviral immune response and elimination of the virus, which was illustrated in a SARS mouse model where Fc engaging antibodies were more potent in eliminating SARS via activation of phagocytic cells compared to antibodies that neutralized virus alone 64 . Overall, the ACE2-Fc fusion protein would have many of the same benefits of a traditional neutralizing antibody that would be sought as a treatment for the infection, but represent one with maximal breadth and potency since the 2019-nCoV could not escape its neutralization, given the same protein is also its receptor for cell entry. Indeed, it has been shown that the pathogenicity of SARS versus the more mild human coronavirus NL63 was related to a lower affinity of NL63 for human ACE2 versus SARS, with NL63 S protein reducing ACE2 levels less than SARS S protein 65 . Therefore, if 2019-nCoV were to try to escape ACE2 neutralization via decreasing affinity, it would mutate into a less pathogenic virus. This is similar to the re-emergent SARS virus in 2003-2004, which had lower affinity for human ACE2 and resulted in less severe infection and no secondary transmission 66 . Thus, 2019-nCoV could be presented with an evolutionary trap when faced with potential ACE2-Fc therapy, leading toward a more benign clinical course.To give some additional support to the potential of a receptorimmunoadhesin being a potential antiviral strategy, it should be noted that CD4-Fc or CD4-IgG was one of the early agents developed as a potential HIV medication 67 . The protein contained the first two domains of the CD4 receptor that are known to bind gp120 on the surface of infected HIV cells. CD4-IgG was shown to neutralize HIV in vitro, preventing infection. The protein was also safe when administered in patients, although only limited-to-mild clinical benefit was achieved 68,69 . Updated enhanced versions of CD4-IgG have been developed that additionally have a small peptide derived from the co-receptor, CCR5, enhancing affinity and giving even more potent neutralizing activity, essentially 100% of HIV isolates and making rhesus macaques resistant to multiple simian-human immunodeficiency virus challenges 70,71 . While HIV and 2019-nCoV are very different viruses, with different cell types, kinetics, and clinical courses, the previous results with HIV are encouraging that this could be a therapeutic strategy for 2019-nCoV. If anything, 2019-nCoV is likely more amenable to this neutralizing therapy given that the respiratory virus will only cause an acute infection, unlike HIV, which causes chronic infection in hosts with different cellular reservoirs.One potential limitation of the ACE2-Fc strategy is that the increase in levels of extracellular ACE2 could have unknown effects on the body, particularly when elevated for a prolonged time via Fc domain extended half-life. Small levels of extracellular ACE2 are already secreted by tissues, so the circulation of this extracellular domain would not be unprecedented 72 . Moreover, recombinant ACE2 protein was well-tolerated by healthy patients in a phase I trial, and by patients with lung injury in a phase II trial, suggesting treating 2019-nCoV patients with ACE2-Fc will also tolerated. If investigators are still concerned, critical amino acid(s) for ACE2 peptidase activity could be mutated to abolish the native function of this sequence, while retaining high affinity binding for SARS and 2019-nCoV S protein. Indeed, this possibility was previously investigated in generating an ACE2 and IgG1 fusion protein, which showed that mutation of histidine residues at position 374 and 378 of the ACE2 extracellular domain abolished peptidase activity, but retained high affinity binding to SARS S protein 56 . Of course, ACE2 peptidase mutation would eliminate the beneficial effects from the recombinant protein delivery of ACE2 in treating lung injury, so it is recommended that retaining ACE2 enzyme activity be pursued first. Another potential concern is that receptor binding via an antibody format could inadvertently direct 2019-nCoV toward infecting Fc receptor (CD16) positive cells, which has been shown in vitro for neutralizing antibodies in MERS 73 . It's unclear what clinical significance this would have, and to what extent this would happen in vivo. Ultimately, clinical trials will be needed to delineate any specific side effects of ACE2-Fc treatment.The chief objective of global health efforts against 2019-nCoV remains to effectively quarantine patients and screen individuals who may be infected to limit spread. That objective should continue going forward. What is proposed here is an option to at least give infected patients a medication quickly to help alleviate symptoms and prevent death, while vaccine efforts for 2019-nCoV continue. This could be enabled through ACE2-Fc providing a triple mechanism of action: (1) Treatment of ACE2 deficiency and lung injury, (2) virus neutralization, and (3) immune effector recruitment. Beyond infected patients, ACE2-Fc could provide passive immunity to healthcare works at risk as another benefit. Going forward, it is recommended that physicians, scientists, and biotech industry in China and elsewhere pursue manufacturing an ACE2-Fc biologic agent right now, which can immediately advance into trials. A variety of different protein expression platforms (CHO, insect, yeast) could be utilized, depending on the particular contract manufacturer's expertise. Gene therapy could even be considered to make ACE2-Fc from a DNA or mRNA platform, but would have additional risk of uncertain delivery strategies and ultimately may slow down progress toward treating patients.The goal would be that ACE2-Fc could treat infection in current patients preventing significant morbidities and death, while also serving as a potential prophylactic to give passive immunity to clinical providers on the frontlines, as well as individuals who may have been exposed to the virus. Essentially, ACE2-Fc could be the potent neutralizing antibody that the global health community needs to combat 2019-nCoV, today, while also treating the underlying ARDS pathophysiology causing patient mortality. It could be scaled much more quickly than convalescent patient sera, which would be dependent on infected individuals to make. ACE2-Fc would be resistant to viral escape as well, unlike potential neutralizing monoclonal antibodies that may be developed in the coming weeks to months.While a therapeutic strategy is being outlined here, the longterm goal of 2019-nCoV research would remain developing an effective vaccine to yield neutralizing antibodies, likely based on the S protein and specifically, the RBD protein. Such trials should happen as soon as possible, but may prove to be challenging to get the right level of immunogenicity, antigen presentation, adjuvant addition, and potent antibody stimulation. The virus could continue mutating, foiling different efforts to stimulate protective immunity. By comparison, 2019-nCoV cannot escape the ACE2-Fc treatment strategy, since it leverages its own cognate receptor for infection. As mentioned above, should 2019-nCoV attempt to escape this therapy via reduced ACE2 affinity binding, it would likely become less pathogenic, similar to the comparison of SARS versus human coronavirus NL63 65 . Lastly, scaling the dose of any effective vaccine would also prove to be challenging depending on the vector format (e.g. viral vector versus mRNA versus protein), and even a fully protective vaccine would not help patients who are currently infected with the virus.In an effort to help aide researchers and industry in China to combat 2019-nCoV, the protein sequence of the ACE2-Fc construct is provided ( Figure 2B ). Different human Fc domains (IgG1, IgG2, IgG3, or IgG4) could be contemplated, although IgG1 traditionally has the most potency for triggering anti-microbial responses 49 . Similarly, an ACE2-Fc biologic without active ACE2 peptidase function could be explored as well. Given that gene synthesis of this sequence could happen within a week, the gene could be placed within the protein expression platform of choice shortly thereafter leading to protein production quickly. The availability of protein A columns and other techniques in the industry to purify antibodies would facilitate ACE2-Fc to quickly be repurposed on existing antibody manufacturing infrastructure existing in China.A final benefit of pursuing ACE2-Fc is that it could effectively be used as a therapeutic drug stockpile for future outbreaks of SARS and 2019-nCoV, and any new coronavirus that emerges from a zoonotic reservoir in the future that uses the ACE2 receptor for entry. Moreover, coronaviruses that replicate in animals across China and other countries could be studied in order to assess their entry mechanisms. By understanding entry in these other animals, one could effectively predict a receptor that could be utilized in any zoonotic transmission event, and build a new receptor immunoadhesin molecule in the future. As an example, a similar immunoadhesin, DPP4-Fc, could be envisioned for MERS based on the viral receptor, DPP4, used by that virus 74 . Beyond coronaviruses, this strategy could be utilized for other viruses where the risk of outbreak potential is high. ACE2-Fc could also find use in treating ARDS for other unrelated viruses and causes of acute lung injury, building on the previous clinical trial work 60,61 .In summary, ACE2-Fc has the potential to be the neutralizing antibody that healthcare workers need to treat and prevent 2019-nCoV infection today and could play an important role in the cessation of the outbreak if manufacturing based on an available sequence starts soon. An alternative 2019-nCoV RBD-Fc fusion could also be pursued, if one desired the dual function of receptor blocking and vaccination in one molecule.Underlying data No data are associated with this article. \n",
            "\t0.533\tThe ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. Aim: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. Methods: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. Results: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive -Global (EVAg), a European Union infrastructure project. Conclusion: The present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.According to the World Health Organization (WHO), the WHO China Country Office was informed of cases of pneumonia of unknown aetiology in Wuhan City, Hubei Province, on 31 December 2019 . A novel coronavirus currently termed 2019-nCoV was officially announced as the causative agent by Chinese authorities on 7 January. A viral genome sequence was released for immediate public health support via the community online resource virological.org on 10 January (Wuhan-Hu-1, GenBank accession number MN908947 ), followed by four other genomes deposited on 12 January in the viral sequence database curated by the Global Initiative on Sharing All Influenza Data (GISAID). The genome sequences suggest presence of a virus closely related to the members of a viral species termed severe acute respiratory syndrome (SARS)-related CoV, a species defined by the agent of the 2002/03 outbreak of SARS in humans [3, 4] . The species also comprises a large number of viruses mostly detected in rhinolophid bats in Asia and Europe.As at 20 January 2019, 282 laboratory-confirmed human cases have been notified to WHO . Confirmed cases in travellers from Wuhan were announced on 13 and 17 January in Thailand as well as on 15 January in Japan and 19 January in Korea. The extent of humanto-human transmission of 2019-nCoV is unclear at the time of writing of this report but there is evidence of some human-to-human transmission.Among the foremost priorities to facilitate public health interventions is reliable laboratory diagnosis. In acute respiratory infection, RT-PCR is routinely used to detect causative viruses from respiratory secretions. We have previously demonstrated the feasibility of introducing robust detection technology based on real-time RT-PCR in public health laboratories during international health emergencies by coordination between public and academic laboratories . In all of these situations, virus isolates were available as the primary substrate for establishing and controlling assays and assay performance.In the present case of 2019-nCoV, virus isolates or samples from infected patients have so far not become available to the international public health community. We report here on the establishment and validation of a diagnostic workflow for 2019-nCoV screening and specific confirmation, designed in absence of available virus isolates or original patient specimens. Design and validation were enabled by the close genetic relatedness to the 2003 SARS-CoV, and aided by the use of synthetic nucleic acid technology.Cell culture supernatants containing typed coronaviruses and other respiratory viruses were provided by Charité and University of Hong Kong research laboratories. Respiratory samples were obtained during 2019 from patients hospitalised at Charité medical centre and tested by the NxTAG respiratory pathogen panel (Luminex, S´Hertogenbosch, The Netherlands) or in cases of MERS-CoV by the MERS-CoV upE assay as published before RNA was extracted from clinical samples with the MagNA Pure 96 system (Roche, Penzberg, Germany) and from cell culture supernatants with the viral RNA mini kit (QIAGEN, Hilden, Germany).A 25 μL reaction contained 5 μL of RNA, 12.5 μL of 2 × reaction buffer provided with the Superscript III one step RT-PCR system with Platinum Taq Polymerase (Invitrogen, Darmstadt, Germany; containing 0.4 mM of each deoxyribont triphosphates (dNTP) and 3.2 mM magnesium sulphate), 1 μL of reverse transcriptase/ Taq mixture from the kit, 0.4 μL of a 50 mM magnesium sulphate solution (Invitrogen), and 1 μg of nonacetylated bovine serum albumin (Roche). Primer and probe sequences, as well as optimised concentrations are shown in Table 1 . All oligonucleotides were synthesised and provided by Tib-Molbiol (Berlin, Laboratories participating in the evaluation used the TaqMan Fast Virus 1-Step Master Mix (Thermo Fisher) with the same oligonucleotide concentrations and cycling conditions. The QIAGEN One-Step RT-PCR Kit was also tested and found to be compatible.The intended cross-reactivity of all assays with viral RNA of SARS-CoV allows us to use the assays without having to rely on external sources of specific 2019-nCoV RNA.For a routine workflow, we recommend the E gene assay as the first-line screening tool, followed by confirmatory testing with the RdRp gene assay. Application of the RdRp gene assay with dual colour technology can discriminate 2019-nCoV (both probes positive) from SARS-CoV RNA if the latter is used as positive control. Alternatively, laboratories may choose to run the RdRp assay with only the 2019-nCoV-specific probe.The internal use of samples for diagnostic workflow optimisation was agreed under the medical ethical rules of each of the participating partners.Before public release of virus sequences from cases of 2019-nCoV, we relied on social media reports announcing detection of a SARS-like virus. We thus assumed that a SARS-related CoV is involved in the outbreak. We downloaded all complete and partial (if > 400 nt) SARS-related virus sequences available in GenBank by 1 January 2020. The list (n = 729 entries) was manually checked and artificial sequences (laboratory-derived, synthetic, etc), as well as sequence duplicates were removed, resulting in a final list of 375 sequences. These sequences were aligned and the alignment was used for assay design (Supplementary Figure S1 ). Upon release of the first 2019-nCoV sequence at virological. org, three assays were selected based on how well they matched to the 2019-nCoV genome ( Figure 1 ). The alignment was complemented by additional sequences released independently on GISAID (. gisaid.org), confirming the good matching of selected primers to all sequences. Alignments of primer binding domains with 2019-nCoV, SARS-CoV as well as selected bat-associated SARS-related CoV are shown in Figure 2 .To obtain a preliminary assessment of analytical sensitivity, we used purified cell culture supernatant containing SARS-CoV strain Frankfurt-1 virions grown on Vero cells. The supernatant was ultrafiltered and thereby concentrated from a ca 20-fold volume of cell culture supernatant. The concentration step simultaneously reduces the relative concentration of background nucleic acids such as not virion-packaged viral RNA. The virion preparation was quantified by realtime RT-PCR using a specific in vitro-transcribed RNA quantification standard as described in Drosten et al. . All assays were subjected to replicate testing in order to determine stochastic detection frequencies at each assay's sensitivity end point ( Figure 3A and B). All assays were highly sensitive, with best results obtained for the E gene and RdRp gene assays (5.2 and 3.8 copies per reaction at 95% detection probability, respectively). These two assays were chosen for further evaluation. One of the laboratories participating in the external evaluation used other basic RT-PCR reagents (TaqMan Fast Virus 1-Step Master Mix) and repeated the sensitivity study, with equivalent results (E gene: 3.2 RNA copies/reaction (95% CI: 2.2-6.8); RdRP: 3.7 RNA copies/reaction (95% CI: 2.8-8.0). Of note, the N gene assay also performed well but was not subjected to intensive further validation because it was slightly less sensitive (Supplementary Figure S2) Although both assays detected 2019-nCoV without polymorphisms at oligonucleotide binding sites ( Figure 2 ), we additionally generated in vitro-transcribed RNA standards that exactly matched the sequence of 2019-nCoV for absolute quantification and studying the limit of detection (LOD). Replicate reactions were done at concentrations around the detection end point determined in preliminary dilution experiments. The resulting LOD from replicate tests was 3.9 copies per reaction for the E gene assay and 3.6 copies per reaction for the RdRp assay ( Figure 3C and D) . These figures were close to the 95% hit rate of 2.9 copies per reaction, according to the Poisson distribution, expected when one RNA molecule is detected.Following the rationale that SARS-CoV RNA can be used as a positive control for the entire laboratory procedure, thus obviating the need to handle 2019-nCoV RNA, we formulated the RdRp assay so that it contains two probes: a broad-range probe reacting with SARS-CoV and 2019-nCoV and an additional probe that reacts only with 2019-nCoV. By limiting dilution experiments, we confirmed that both probes, whether used individually or in combination, provided the same LOD for each target virus. The specific probe RdRP_SARSr-P2 detected only the 2019-nCoV RNA transcript but not the SARS-CoV RNA.At present, the potential exposure to a common environmental source in early reported cases implicates the possibility of independent zoonotic infections with increased sequence variability . To show that the assays can detect other bat-associated SARS-related viruses, we used the E gene assay to test six batderived faecal samples available from Drexler et al. und Muth et al. . These virus-positive samples stemmed from European rhinolophid bats. Detection of these phylogenetic outliers within the SARS-related CoV clade suggests that all Asian viruses are likely to be detected. This would, theoretically, ensure broad sensitivity even in case of multiple independent acquisitions of variant viruses from an animal reservoir.To exclude non-specific reactivity of oligonucleotides among each other, causing artificial fluorescent signals, all assays were tested 120 times in parallel with water and no other nucleic acid except the provided oligonucleotides. In none of these reactions was any positive signal detected.Cell culture supernatants containing all endemic human coronaviruses (HCoV)229E, NL63, OC43 and HKU1 as well as MERS-CoV were tested in duplicate in all three assays (Table 2) . For the non-cultivable HCoV-HKU1, supernatant from human airway culture was used. Viral RNA concentration in all samples was determined by specific real-time RT-PCRs and in vitro-transcribed RNA standards designed for absolute quantification of viral load. Additional undiluted (but not quantified) cell culture supernatants were tested as summarised in Table 2 . These were additionally mixed into negative human sputum samples. None of the tested viruses or virus preparations showed reactivity with any assay.Using the E and RdRp gene assays, we tested a total of 297 clinical samples from patients with respiratory disease from the biobanks of five laboratories that provide diagnostic services (one in Germany, two in the Netherlands, one in Hong Kong, one in the UK). We selected 198 samples from three university medical centres where patients from general and intensive care wards as well as mainly paediatric outpatient departments are seen (Germany, the Netherlands, Hong Kong). The remaining samples were contributed by national public health services performing surveillance studies (RIVM, PHE), with samples mainly submitted by practitioners. The samples contained the broadest range of respiratory agents possible and reflected the general spectrum of virus concentrations encountered in diagnostic laboratories in these countries ( Table 2 ). In total, this testing yielded no false positive outcomes.In four individual test reactions, weak initial reactivity was seen but they were negative upon retesting with the same assay. These signals were not associated with any particular virus, and for each virus with which initial positive reactivity occurred, there were other samples that contained the same virus at a higher concentration but did not test positive. Given the results from the extensive technical qualification described above, it was concluded that this initial reactivity was not due to chemical instability of real-time PCR probes but most probably to handling issues caused by the rapid introduction of new diagnostic tests and controls during this evaluation study.The present report describes the establishment of a diagnostic workflow for detection of an emerging virus in the absence of physical sources of viral genomic nucleic acid. Effective assay design was enabled by the willingness of scientists from China to share genome information before formal publication, as well as the availability of broad sequence knowledge from ca 15 years of investigation of SARS-related viruses in animal reservoirs. The relative ease with which assays could be designed for this virus, in contrast to SARS-CoV in 2003, proves the huge collective value of descriptive studies of disease ecology and viral genome diversity [8, .Real-time RT-PCR is widely deployed in diagnostic virology. In the case of a public health emergency, proficient diagnostic laboratories can rely on this robust technology to establish new diagnostic tests within their routine services before pre-formulated assays become available. In addition to information on Isolated from human airway epithelial culture. d 1 × 10 10 RNA copies/mL, determined by specific real-time RT-PCR of one isolate. The other isolate was not quantified but spiked in human negative-testing sputum. e 4 × 10 9 RNA copies/mL, determined by specific real-time RT-PCR. f 3 × 10 9 RNA copies/mL, determined by specific real-time RT-PCR of one isolate. The other isolate was not quantified spiked in human negative-testing sputum. g 1 × 10 8 RNA copies/mL, determined by specific real-time RT-PCR. reagents, oligonucleotides and positive controls, laboratories working under quality control programmes need to rely on documentation of technical qualification of the assay formulation as well as data from external clinical evaluation tests. The provision of control RNA templates has been effectively implemented by the EVAg project that provides virus-related reagents from academic research collections . SARS-CoV RNA was retrievable from EVAg before the present outbreak; specific products such as RNA transcripts for the here-described assays were first retrievable from the EVAg online catalogue on 14 January 2020 (). Technical qualification data based on cell culture materials and synthetic constructs, as well as results from exclusivity testing on 75 clinical samples, were included in the first version of the diagnostic protocol provided to the WHO on 13 January 2020. Based on efficient collaboration in an informal network of laboratories, these data were augmented within 1 week comprise testing results based on a wide range of respiratory pathogens in clinical samples from natural infections. Comparable evaluation studies during regulatory qualification of in vitro diagnostic assays can take months for organisation, legal implementation and logistics and typically come after the peak of an outbreak has waned. The speed and effectiveness of the present deployment and evaluation effort were enabled by national and European research networks established in response to international health crises in recent years, demonstrating the enormous response capacity that can be released through coordinated action of academic and public laboratories . This laboratory capacity not only supports immediate public health interventions but enables sites to enrol patients during rapid clinical research responses. CD: Planned experiments, conceptualised the laboratory work, conceptualised the overall study, wrote the manuscript draft.\n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.981\tSince December 2019, China has been experiencing a large outbreak of a novel coronavirus (2019-nCoV) which can cause respiratory disease and severe pneumonia. We estimated the basic reproduction number R0 of 2019-nCoV to be around 2.2 (90% high density interval: 1.4-3.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and pandemic influenza, indicating a risk of global spread.On 31 December 2019, the World Health Organization (WHO) was alerted about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China [1, 2] . Only a few days later, Chinese authorities identified and characterised a novel coronavirus (2019-nCoV) as the causative agent of the outbreak . The outbreak appears to have started from a single or multiple zoonotic transmission events at a wet market in Wuhan where game animals and meat were sold and has resulted in 5,997 confirmed cases in China and 68 confirmed cases in several other countries by 29 January 2020 . Based on the number of exported cases identified in other countries, the actual size of the epidemic in Wuhan has been estimated to be much larger . At this early stage of the outbreak, it is important to gain understanding of the transmission pattern and the potential for sustained human-to-human transmission of 2019-nCoV. Information on the transmission characteristics will help coordinate current screening and containment strategies, support decision making on whether the outbreak constitutes a public health emergency of international concern (PHEIC), and is key for anticipating the risk of pandemic spread of 2019-nCoV. In order to better understand the early transmission pattern of 2019-nCoV, we performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date.Two key properties will determine further spread of 2019-nCoV. Firstly, the basic reproduction number R 0 describes the average number of secondary cases generated by an infectious index case in a fully susceptible population, as was the case during the early phase of the outbreak. If R 0 is above the critical threshold of 1, continuous human-to-human transmission with sustained transmission chains will occur. Secondly, the individual variation in the number of secondary cases provides further information about the expected outbreak dynamics and the potential for superspreading events . If the dispersion of the number of secondary cases is high, a small number of cases may be responsible for a disproportionate number of secondary cases, while a large number of cases will not transmit the pathogen at all. While superspreading always remain a rare event, it can result in a large and explosive transmission event and have a lot of impact on the course of an epidemic. Conversely, low dispersion would lead to a steadier growth of the epidemic, with more homogeneity in the number of secondary cases per index case. This has important implications for control efforts.In a first step, we initialised simulations with one index case. For each primary case, we generated secondary cases according to a negative-binomial offspring distribution with mean R 0 and dispersion k [7, 8] . The dispersion parameter k quantifies the variability in the number of secondary cases, and can be interpreted as a measure of the impact of superspreading events (the lower the value of k, the higher the impact of superspreading). The generation time interval D was assumed to be gamma-distributed with a shape parameter of 2, and a mean that varied between 7 and 14 days. We explored a wide range of parameter combinations (Table) and ran 1,000 stochastic simulations for each individual combination. This corresponds to a total of 3.52 million one-index-case simulations that were run on UBELIX (), the high performance computing cluster at the University of Bern, Switzerland.In a second step, we accounted for the uncertainty regarding the number of index cases n and the date T of the initial zoonotic animal-to-human transmissions at the wet market in Wuhan. An epidemic with several index cases can be considered as the aggregation of several independent epidemics with one index case each. We sampled (with replacement) n of the oneindex-case epidemics, sampled a date of onset for each index case and aggregated the epidemic curves together. The sampling of the date of onset was done uniformly from a 2-week interval around 27 November 2019, in coherence with early phylogenetic analyses of 11 2019-nCoV genomes . This step was repeated 100 times for each combination of R 0 (22 points), k (20 points), D (8 points) and n (6 points) for a total of 2,112,000 full epidemics simulated that included the uncertainty on D, n and T. Finally, we calculated the proportion of stochastic simulations that reached a total number of infected cases within the interval between 1,000 and 9,700 by 18 January 2020, as estimated by Imai et al. . In a process related to approximate Bayesian computation (ABC), the parameter value combinations that led to simulations within that interval were treated as approximations to the posterior distributions of the parameters with uniform prior distributions. Model simulations and analyses were performed in the R software for statistical computing . Code files are available on  order to reach between 1,000 and 9,700 infected cases by 18 January 2020, the early human-to-human transmission of 2019-nCoV was characterised by values of R 0 around 2.2 (median value, with 90% high density interval: 1.4-3.8) (Figure 1 ). The observed data at this point are compatible with a large range of values for the dispersion parameter k (median: 0.54, 90% high density interval: 0.014-6.95). However, our simulations suggest that very low values of k are less likely. These estimates incorporate the uncertainty about the total epidemic size on 18 January 2020 and about the date and scale of the initial zoonotic event (Figure 2 ).Comparison with other emerging coronaviruses in the past allows to put into perspective the available information regarding the transmission patterns of 2019-nCoV. Figure 3 shows the combinations of R 0 and k that are most likely at this stage of the epidemic. Our estimates of R 0 and k are more similar to previous estimates focusing on early human-to-human transmission of SARS-CoV in Beijing and Singapore than of Middle East respiratory syndrome-related coronavirus (MERS-CoV) . The spread of MERS-CoV was characterised by small clusters of transmission following repeated instances of animal-to-human transmission events, mainly driven by the occurrence of superspreading events in hospital settings. MERS-CoV could however not sustain human-to-human transmission beyond a few generations . Conversely, the international spread of SARS-CoV lasted for 9 months and was driven by sustained human-to-human transmission, with occasional superspreading events. It led to more than 8,000 cases around the world and required extensive efforts by public health authorities to be contained . Our assessment of the early transmission of 2019-nCoV suggests that 2019-nCoV might follow a similar path.Our estimates for 2019-nCoV are also compatible with those of 1918 pandemic influenza, for which k was The basic reproduction number R 0 quantifies human-to-human transmission. The dispersion parameter k quantifies the risk of a superspreading event (lower values of k are linked to a higher probability of superspreading). Note that the probability density of k implies a log10 transformation.estimated . Human-to-human transmission of influenza viruses is characterised by R 0 values between 1.5 and 2 and a larger value of k, implying a more steady transmission without superspreading. The emergence of new strains of influenza, for which human populations carried little to no immunity contrary to seasonal influenza, led to pandemics with different severity such as the ones in1918, 1957 1968 and 2009. It is notable that coronaviruses differ from influenza viruses in many aspects, and evidence for the 2019-nCoV with respect to case fatality rate, transmissibility from asymptomatic individuals and speed of transmission is still limited. Without speculating about possible consequences, the values of R 0 and k found here during the early stage of 2019-nCoV emergence and the lack of immunity to 2019-nCoV in the human population leave open the possibility for pandemic circulation of this new virus.The scarcity of available data, especially on case counts by date of disease onset as well as contact tracing, greatly limits the precision of our estimates and does not yet allow for reliable forecasts of epidemic spread. Case counts provided by local authorities in the early stage of an emerging epidemic are notoriously unreliable as reporting rates are unstable and vary with time. This is due to many factors such as the initial lack of proper diagnosis tools, the focus on the more severe cases or the overcrowding of hospitals. We avoided this surveillance bias by relying on an indirect estimate of epidemic size on 18 January, based on cases identified in foreign countries before quarantine measures were implemented on 23 January. This estimated range of epidemic size relies itself on several assumptions, including that all infected individuals who travelled from Wuhan to other countries have been detected . This caveat may lead to an underestimation of transmissibility, especially considering the recent reports about asymptomatic cases . Conversely, our results do not depend on any assumption about the existence of asymptomatic transmission, and only reflect the possible combinations of transmission events that lead to the situation on 18 January. Table. Among these simulated epidemics, 54.3% led to a cumulative incidence between 1,000 and 9,700 on 18 January 2020 (in red).Our analysis, while limited because of the scarcity of data, has two important strengths. Firstly, it is based on the simulation of a wide range of possibilities regarding epidemic parameters and allows for the full propagation on the final estimates of the many remaining uncertainties regarding 2019-nCoV and the situation in Wuhan: on the actual size of the epidemic, on the size of the initial zoonotic event at the wet market, on the date(s) of the initial animal-to-human transmission event(s) and on the generation time interval. As it accounts for all these uncertainties, our analysis provides a summary of the current state of knowledge about the human-to-human transmissibility of 2019-nCoV. Secondly, its focus on the possibility of superspreading events by using negative-binomial offspring distributions appears relevant in the context of emerging coronaviruses [7, 8] . While our estimate of k remains imprecise, the simulations suggest that very low values of k < 0.1 are less likely than higher values < 0.1 that correspond to a more homogeneous transmission pattern. However, values of k in the range of 0.1-0.2 are still compatible with a small risk of occurrence of large superspreading events, especially impactful in hospital settings [15, 16] .Our analysis suggests that the early pattern of humanto-human transmission of 2019-nCoV is reminiscent of SARS-CoV emergence in 2002. International collaboration and coordination will be crucial in order to contain the spread of 2019-nCoV. At this stage, particular attention should be given to the prevention of possible rare but explosive superspreading events, while the establishment of sustained transmission chains from single cases cannot be ruled out. The previous experience with SARS-CoV has shown that established practices of infection control, such as early detection and isolation, contact tracing and the use of personal protective equipment, can stop such an epidemic. Given the existing uncertainty around the case fatality rate This can be interpreted as the combinations of R 0 and k values most compatible with the estimation of epidemic size before quarantine measures were put in place. As a comparison, we show the estimates of R 0 and k for the early human-to-human transmission of SARS-CoV in Singapore and Beijing and of 1918 pandemic influenza [7, 9, 14] . and transmission, our findings confirm the importance of screening, surveillance and control efforts, particularly at airports and other transportation hubs, in order to prevent further international spread of 2019-nCoV.\n",
            "\t0.977\t2019-nCoV is a newly identified coronavirus with high similarity to SARS-CoV. We performed a structural analysis of the receptor binding domain (RBD) of spike glycoprotein responsible for entry of coronaviruses into host cells. The RBDs from the two viruses share 72% identity in amino acid sequences, and molecular simulation reveals highly similar ternary structures. However, 2019-nCoV has a distinct loop with flexible glycyl residues replacing rigid prolyl residues in SARS-CoV. Molecular modeling revealed that 2019-nCoV RBD has a stronger interaction with angiotensin converting enzyme 2 (ACE2). A unique phenylalanine F486 in the flexible loop likely plays a major role because its penetration into a deep hydrophobic pocket in ACE2. ACE2 is widely expressed with conserved primary structures throughout the animal kingdom from fish, amphibians, reptiles, birds, to mammals. Structural analysis suggests that ACE2 from these animals can potentially bind RBD of 2019-nCoV, making them all possible natural hosts for the virus. 2019-nCoV is thought to be transmitted through respiratory droplets. However, since ACE2 is predominantly expressed in intestines, testis, and kidney, fecal-oral and other routes of transmission are also possible. Finally, antibodies and small molecular inhibitors that can block the interaction of ACE2 with RBD should be developed to combat the virus.A mysterious pneumonia illness was first reported in late December 2019 in Wuhan, China, and has rapidly spread to a dozen of countries including the United States with thousands of infected individuals and hundreds of deaths within a month . Scientists in China have isolated the virus from patients and determined its genetic code. The pathogen responsible for this epidemic is a new coronavirus designated 2019-nCoV by the World Health Organization. 2019-nCoV belongs to the same family of viruses as the well-known severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV), which have killed hundreds of people in the past 17 years.Coronaviruses consist of a large diverse family of viruses. They can be classified into four genera: Alpha-, Beta-, Gamma-, and Delta coronavirus [2, 3] . Representative alphacoronaviruses include human coronavirus NL63 (HCoV-NL63), while the betacoronaviruses include the best-known SARS-CoV and MERS-CoV. Based on nucleic acid sequence similarity, the newly identified 2019-nCoV is a betacoronavirus. The entry of all coronaviruses into host cells is mediated by spike glycoprotein that gives coronaviruses a crownlike appearance by forming spikes on their surface. The amino acid sequence of spike glycoprotein consists of a large ectodomain, a single-pass transmembrane anchor, and a short C-terminal intracellular tail . The ectodomain contains a receptor-binding unit S1 and a membrane-fusion unit S2. Electron microscopic imaging illustrated that spike glycoprotein forms a clove-shaped spike with three S1 heads and a trimeric S2 stalk. For a virus to enter a host cell, S1 binds to a specific cell surface receptor via its receptorbinding domain (RBD), and S2 fuses the host cell and viral membranes, enabling the entry of viral genomes into host cells. Specific RBD-receptor binding determines if a cell or animal can be infected and also serves as a target for therapeutic inventions to treat diseases caused by coronaviruses. Previous studies have identified angiotensin converting enzyme 2 (ACE2) as a functional receptor for SARS-CoV [4, 5] . In this study, we analyzed the structure of spike glycoprotein RBD of 2019-nCoV and identified a unique feature that potentially allows a high affinity binding to ACE2 in human cells. We further discussed potential candidates for natural hosts of 2019-nCoV, routes of transmission, and strategies to inhibit virus entry for therapeutic applications.The genomic sequence of 2019-nCoV as deposited by Wang et al. was downloaded from the GenBank database (MN908947.3). DNA and protein sequences were compared by using the BLAST program. Multiple sequence alignment was performed by using the Clustal Omega program. Three-dimensional structure was analyzed by using the Cn3D program from the NCBI. Protein structure simulation was performed by using Swiss-model based on the cocrystal structure of human ACE2 with the SARS-CoV spike glycoprotein RBD (6, PDB ID 2AJF). ACE2 and RDB interaction was analyzed by molecular docking using the PatchDock and FireDock programs.By using the initially reported sequence MN908947.3, a BLAST search of the NCBI database revealed 6 inputs for the virus with essentially identical sequences (accession NC_045512.2, MN908947.3, MN975262.1, MN985325.1, MN988713.1, and MN938384.1). The closest homolog of 2019-nCoV is a SARS-like coronavirus isolated from bat (MG772933.1) with a sequence identity of 87.99% at 99% coverage (Fig. 1A) . It also shows 80% sequence identity with SARS coronavirus isolated from human patients or civet with 98% coverage. Throughout the entire 29,903bp genome of 2019-nCoV, the least conserved region encodes for the spike glycoprotein with sequence identity of 74e83%. Spike glycoprotein forms spikes on the surface of coronaviruses and is responsible for entrance of the viruses into the host cells. The RBD in the spike glycoprotein molecule directly binds receptors on the surface of host cells . In the case of SARS-CoV and bat/civet SARS-like CoV, the receptor is ACE2, an exopeptidase that catalyzes the conversion of angiotensin I to the nonapeptide angiotensin1-9 or the conversion of angiotensin II to angiotensin1-7 [3e7]. At the protein level, the whole spike glycoprotein and its RBD share 76% and 72% sequence identity with SARS-CoV, respectively. SARS-CoV spike glycoprotein is known to be glycosylated. A total of 22 predicated N-glycosylation sites is found in spike glycoprotein of 2019-nCoV, which are shared by SARS-CoV except that the latter contains an extra glycosylation site at N370. A detailed sequence alignment of the RBD of SARS-CoV spike glycoprotein with those from closely related coronaviruses at the protein level is shown in Fig. 1B .The crystal structure of SARS-CoV RBD in complex with its receptor, human ACE2, has been solved . By performing molecular simulation, we obtained a ternary structure for RBD of 2019-nCoV that is essentially superimposable with that of SARS-CoV ( Fig. 2AeC) , except for a noted structural variation in a loop (Loop 2). The backbone of the deduced RBD structure consists of 7 beta sheets. Peptide segments involved in the formation of this secondary structure are all highly conserved without the presence of secondary structure breakers. Four cysteinyl residues that form disulfide bonds (corresponding to C366/C418 and C467/C474 of SARS-CoV) are also conserved (see also Fig. 1B) . We furthermore performed molecular docking to examine the binding of RBD with ACE2. The deduced complex structure reveals similar mode of extensive interaction as seen with SARS-CoV with a more favorable binding energy (À21.82 21 vs. À13.38 kcal/mol) (Fig. 2DeF) . The contact between ACE2 and RBD involves two b-sheets and three loops (see Figs. 1B and 2A). There are 16 amino acid residues in SARS-CoV RBD that are directly in contact with ACE2, of which 8 are conserved in 2019-nCoV (see Fig. 1B ). Presumably, the substituted amino acids can either reduce or enhance the interaction.To define the contribution of the variant amino acids to the RBD/ ACE2 interaction, we compare the sequences of RBD from three other SARS-CoV-associated viruses (Fig. 1B) . These include coronaviruses isolated from patients during a short, weak SARS outbreak in 2003e2004 (denoted SARSv here) and from palm civets and bats, possible sources of SARS-CoV found in humans [8e11]. Recombinant proteins containing RBD of these 3 viruses are all known to bind to human ACE2 [11, 12] . In comparison with SARS-CoV (responsible for the major SRAS outbreak during the 2002e2003), binding with SARSv and civet RBDs is substantially weaker , while quantitative binding affinity with bat RBD has been not been determined . Amino acid residues in the RBD/ ACE2 binding interface plays a crucial role in determining the binding affinity. Among the 16 amino acid residues in RBD of SARS that are in contact with ACE2, 14, 14, 7, and 8 are shared by SARSv, civet, bat, and 2019-nCoV, respectively (Fig. 1B) . N479 found in both SARS viruses isolated from human patients is changed to K and R in civet and bat, respectively. An earlier study demonstrated that an N479 to K substitution resulted in significantly lower affinity (30fold increase in Kd values) . Interestingly, this amino acid is substituted by a similar amino acid glutamine (Q493) in 2019-nCoV, which also contains an amide group but at an extended position, which can potentially carry out similar functions. In comparison with SARS-CoV, T487 is changed to asparagine (N501) in 2019-nCoV but alanine or serine in the other viruses. It has been shown that a T487 to S substitution increased Kd by 20-fold, suggesting the methyl group rather than the hydroxyl group in this threonine residue is more important for the interaction . It is hard to predict if N501 with an amide group can confer a better interaction. Hydrophobic amino acid L472 is also important for interaction between RBD and ACE2. Interestingly, it is substituted by proline in SARSv and phenylalanine in 2019-nCoV (corresponding to F486). L472 is located in a loop formed by disulfide bond C467/C474. Interestingly, this loop with CTPPALNC in SARS-CoV is replaced by CNGVEGFNC in 2019-nCoV containing one extra amino acid residue and totally different amino acid compositions. The replacement of two proline residues by two flexible glycine residues converts a rigid structure to a very flexible one. Further examination of the deduced RBD/ACE2 complex structure reveals that this unique phenylalanine F486 in the flexible loop can penetrate deep into a hydrophobic pocket in ACE2 formed by F28, L79, Y83, and L97 (Fig. 2F) . The presence of two aromatic amino acids in the pocket may provide additional binding force via pstacking interactions . Taken together, 2019-nCoV likely has a stronger binding to ACE2 via its spike glycoprotein.Glycosylation may also affect the interaction of RBD with ACE2. Among the 23 glycosylation sites on spike glycoprotein, two are in RBD (Fig. 1B) . Glycosylation has been detected on one of these residues, Asn330 . N330 corresponds to N343 in the spike glycoprotein of 2019-nCoV and is a conserved glycosylation site. Since it is well separated from the RBD/ACE2 interaction interphase, glycosylation at this site is unlikely to interfere with the interaction . It should be noted that another potential glycosylation site corresponding to N357 in SARS-CoV is not conserved in 2019-nCoV because of substitution of T by A in the þ2 position. Lack of this glycosylation is not expected to affect the receptor binding.The 2019-nCoV outbreak is thought to be initiated from a seafood market that also carried many other wild live animals including snakes, birds, and various mammals. Interestingly, a study by Ji et al. suggests that snakes might serve as a likely reservoir for the novel nCoV-2019 based on the observation that the codon usage of nCoV-2019 was more similar to snakes than other potential hosts they investigated . While the data and premise are being debated, we sought to address the problem by analyzing the structure of ACE2 in different animals. ACE2 is widely expressed in the animal kingdom from fish, amphibians, reptiles, birds, to mammals. Remarkably, its structure is highly conserved. Comparison of human ACE2 with that of a civet (Paguma larvata, AAX63775.1), a bat (Rhinolophus sinicus, ADN93475.1), a bird (Nipponia nippon, KFQ92425.1), a snake (Protobothrops mucrosquamatus, XP_029140508.1), a frog (Xenopus laevis, XP_018104311.1), and a fish (Callorhinchus milii, XP_007889845.1) revealed amino acid sequence identity of 83%, 81%, 83%, 61%, 60%, and 59%, respectively. Fig. 3 aligns parts of ACE2 sequences that contain all the interaction sites in contact with SARS-CoV RBD according to the published co-crystal structure . The interaction involved mainly two a-helices of ACE2. Out of 20 amino acid residues involved in the direct interaction, 4 of them are shared by all seven species of animals analyzed in the study, including F28 that supposedly interacts with F486 of spike glycoprotein from 2019-nCoV (Fig. 2F) . Many of the remaining resides in the contact are conserved or replaced by amino acids of similar chemical properties. It is interesting to note that bird ACE2 shares as many conserved contacting amino acid residues as bat and civet ACE2. ACE2 molecules from any of these has the potential to interact with RBD of 2019-nCoV with high affinity. Therefore, it would not be a surprise if any of these wild animals is found to be a primary or secondary host of 2019-nCoV. SARS-CoV-like coronaviruses have been found in many bats that are considered as natural reservoirs for the viruses. They may well be the host for 2019-nCoV. However, the possibility that cold-blooded animals like snakes can serve as a host cannot be ruled out. The flexible interacting loop identified in our study may allow the virus to adapt to both the cold-blooded and warm-blooded hosts.By performing immunostaining, earlies studies have demonstrated the expression of ACE2 in lung alveolar epithelial cells as well as arterial and venous endothelial cells, arterial smooth muscle cells, renal tubular epithelium, and epithelia of the small intestine [15, 16] . The lung expression provides strong support for infection of SARS-CoV and 2019-nCoV through the airways of the lung. However, by searching the Human Protein Atlas database, we found that ACE2 mRNA is mainly detected in small intestine, colon, duodenum, kidney, testis, and gallbladder. Its expression level in the lung is minimal (Fig. 4) . Furthermore, by examining data from two single-cell RNA-seq studies [17, 18] , we only identified 2 out of 4599 and 13 out of 540 lung epithelial cells expressed a detectable level of ACE2 (). This confirms that the overall expression of ACE2 in the lung is low and may also suggest the presence of selected cells with upregulated ACE2 expression under certain conditions.The tissue expression pattern of ACE2 suggests other modes of virus transmission that may involve the functions of intestine, kidney, testis, and other tissues. Particular attention should be paid to the intestines which expressed the highest level of ACE2. Earlier studies have demonstrated that diarrhea was present in up to 70% of patients infected with SARS-CoV . More importantly, a recent case report demonstrated the presence of 2019-nCoV in feces of a patient with an initial diarrhea episode . While this finding has been noted in other reports, tests of feces and urine samples for the presence of 2019-nCoV is warranted, which may help to reveal alternative routes of virus transmission.Since its initial outbreak, the 2019-nCoV infection is much more contagious than it was originally thought. We know that the virus is capable of spreading quickly from human to human and that people can spread the virus even before they become symptomatic . This makes it harder to contain the virus, and many are concerned about the possibility of a new pandemic. Our study suggests unique structural features of the spike glycoprotein RBD of 2019-nCoV that confers potentially higher affinity binding for its receptor than found with SARS-CoV. With a higher affinity binding capability, the number of viruses required to infect a cell is much reduced. This partly explains why 2019-nCoV appears to be more aggressive than SARS-CoV. This also reminds us of a lesser-known coronavirus HCoV-NL63 that also uses ACE2 also as a receptor. HCoV-NL63 was initially isolated from a child with bronchiolitis in the Netherlands . It belongs to the alphacoronavirus subfamily. The RBD of SARS-CoV shares no structural homology with that of SARS-CoV but recognizes the same region in ACE2. However, cocrystal structure reveals that RBD of NL63-CoV has a narrower contact with ACE2, involving fewer amino acids . This presumably results in a weaker interaction. Evidently, NL63-CoV does not spread aggressively and only causes mild to moderate respiratory infections .The exact mode of transmission for 2019-nCoV has not been firmly established. SARS-CoV is thought to be transmitted by respiratory droplets produced when an infected person coughs or sneezes . The respiratory droplets spread can occur only through direct person-to-person contact or at a close distance. Presumably, 2019-nCoV can be transmitted through respiratory droplets. It may also be transmitted more effectively through the air over a long distance (airborne spread) or by other ways. Considering the predominant expression of ACE2 in intestines and kidney, 2019-nCoV may infect cells in these tissues and find its way into feces and urine. This makes transmission through the fecal-oral route and bodyfluids (urine) possible. The presence of 2019-nCoV in feces supports such a notion .Specific RBD-receptor binding determines if a cell or animal can be infected and also serves as a target for therapeutic inventions to treat diseases caused by coronaviruses. By binding directly to ACE2 on the surface of host cells, spike glycoprotein plays an essential role in virus infection. An obvious way to stop the virus infection is to block the RBD and ACE2 interaction. This can be achieved by using antibodies or small molecular inhibitors. Naturally, antibodies and inhibitors that can disrupt the interaction of RBD with ACE2 is of therapeutic importance. By using a molecular docking approach, an earlier study identified N-(2-aminoethyl)-1 aziridineethanamine as a novel ACE2 inhibitor that effectively blocks the SARS-CoV RBD-mediated cell fusion . This has provided a potential candidate and lead compound for further therapeutic drug development. Meanwhile, biochemical and cell-based assays can be established to screen chemical compound libraries to identify novel inhibitors. On the other hand, many ACE inhibitors are currently used to treat hypertension and other cardiovascular diseases . Among them are captopril, perindopril, ramipril, lisinopril, benazepril, and moexipril. Although these drugs primarily target ACE, a homolog of ACE2 with 42% sequence identity and 61% sequence similarity in the catalytic domain, they may be effective toward ACE2 as well . It should be noted that ACE inhibitors bind to the catalytic center rather than RBD binding site. Nonetheless, these enzymatic inhibitors may indirectly alter conformation of the RBD binding site and thereby affect the interaction of ACE2 with RBD. It is certainly worthwhile to test these drugs for their ability to block the RBD/ACE2 interaction. \n",
            "\t0.975\tAn outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV/SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R 0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious, and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of R t . In the first assumption, R t was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with R t = 1.9, 2.6, or 3.1. The number of infections would reach 11,044, 70,258, and 227,989, respectively, by 29 February 2020. In the second assumption, R t was assumed to gradually decrease at different phases from high level of transmission (R t = 3.1, 2.6, and 1.9) to below 1 (R t = 0.9 or 0.5) owing to increasingly implemented public health intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce R t to an ideal level and control the infection.2019 Novel coronavirus (2019-nCoV/SARS-CoV-2) has given rise to an outbreak of viral pneumonia in Wuhan, China since December 2019 1, 2 . World Health Organization (WHO) now has named the disease Coronavirus Disease 2019 (COVID- 19) 3 . Most cases from the initial cluster had an epidemiological link to the Huanan Seafood Wholesale Market 4 . Patients have clinical manifestations, including fever, cough, shortness of breath, muscle ache, confusion, headache, sore throat, rhinorrhoea, chest pain, diarrhea, and nausea and vomiting 5, 6 . As of 17 February 2020, a cumulative total of 72,436 confirmed cases (including 11,741 currently severe cases), 6242 currently suspect cases, a cumulative total of 1868 deaths and 12,552 cases discharged from hospital were reported by National Health Commission of the People's Republic of China (NHC) in mainland China 7 . The significant increases in the number of confirmed cases in China and abroad led to the announcement made by WHO on 30 January that the event has already constituted a Public Health Emergency of International Concern 8 .The reproduction number, R, measures the transmissibility of a virus, representing the average number of new infections generated by each infected person, the initial constant of which is called the basic reproduction number, R 0 9 , and the actual average number of secondary cases per infected case at time t is called effective reproduction number, R t 10-12 . R t shows time-dependent variation with the implementation of control measures. R > 1 indicates that the outbreak is self-sustaining unless effective control measures are implemented, while R < 1 indicates that the number of new cases decreases over time and, eventually, the outbreak will stop 9 . Over the past month, several groups reported estimated R 0 of COVID-19 and generated valuable prediction for the early phase of this outbreak. In particular, Imai et al. 9 provided the first estimation, using R 0 of 2.6 and based on the number of cases in China and those detected in other countries. Other authors estimated R 0 to be 3.8 13 , 6.47 14 , 2.2 15 , and 2.68 16 . These predictions were very alerting and suggestions have been made for very strict public health measures to contain the epidemics.In response to the outbreak of COVID-19, a series of prompt public health measures have been taken. On 1 January, the Huanan Seafood Wholesale Market was closed in the hope of eliminating zoonotic source of the virus 5 . On 11 January, upon isolation of the viral strain for COVID-19 and establishment of its whole-genome sequences 17 , reverse transcription-polymerase chain reaction (RT-PCR) reagents were developed and provided to Wuhan, which ensured the fast ascertainment of infection 15 . On 21 January, Emergency Response System was activated to better provide ongoing support to the COVID-19 response 18 . Ever since the outbreak, the work of intensive surveillance, epidemiological investigations, and isolation of suspect cases gradually improved. Those having had close contacts with infections were asked to receive medical observation and quarantine for 14 days 19 . Travel from and to Wuhan City as well as other mediumsized cities in Hubei Province has been restricted since 23 January 2020 20 .The 2019-nCoV/SARS-CoV-2 has at least 79.5% similarity in genetic sequence to SARS-CoV 5,17 . Riley 21 estimated that 2.7 secondary infections were generated per case on average (R 0 = 2.7) at the start of the SARS epidemic without controlling. After isolating the patients and controlling the infection by the authority, the value of R t for SARS decreased to 0.25 22 . As Li et al. 15 mentioned, it is possible that subsequent control measures in Wuhan, and elsewhere in mainland China, have reduced transmissibility. A new estimation of the epidemic dynamics taking the unprecedentedly strict prevention and control measures in China into consideration is required to better guide the future prevention decisions.In this article, we intended to make phase-adjusted estimation of the epidemic trend for the 2019-nCoV / SARS-CoV-2 infection transmission in Wuhan, China under two assumptions of R t (maintaining high >1 or gradually decreasing to <1). We hope to depict two types of epidemic dynamics to provide potential evaluation standard for the effects of current prevention and control measures, and to provide theoretical basis for future prevention decisions of the current epidemic in China.Estimation of the epidemic trend assuming that the prevention and control measures are insufficient in Wuhan, ChinaAssuming the epidemic continues to develop with R 0 = 1.9, 2.6, and 3.1 9 from 1 December 2019, the number of infections will continue to rise (Fig. 1 ). By the end of February 2020, COVID-19 cases would be 11,044, 70,258, and 227,989 in Wuhan, China with R 0 = 1.9, 2.6, and 3.1, respectively. Detailed calculation process is included in the Materials and methods section.Estimation of the epidemic trend assuming that the prevention and control measures are sufficient in Wuhan, ChinaThe first phase (1 December 2019-23 January 2020): It was the early phase of the epidemic when a few prevention and control measures were implemented. The number of infections in Wuhan, China reached 17,656-25,875 by the end of this phase with R 0 as 3.1.The second phase (24 January 2020-2 February 2020): From 23 January 2020 on, public transportations to and from Wuhan, as well as public transportation within Wuhan were stopped. While gathering events inside Wuhan was banned, quarantine and isolation were gradually established in Wuhan. The number of infections was 32,061-46,905 by the end of this phase as R t decreased to 2.6.The third phase (3 February 2020-15 February 2020): New infectious disease hospitals and mobile cabin hospitals came into service and many medical and public health teams from other provinces and cities in China arrived in Wuhan. The quarantine and isolation at the community level were further enhanced. The number of infections would reach 53,070-77,390 if R t could be reduced sequentially to 1.9.The fourth phase (from 15 February 2020 on): All of the most restrict public health measures may need a longest incubation period to take effect. If R t could be gradually reduced to 0.9 or 0.5 in the fourth phase, the epidemic peaks and inflection points might occur in Wuhan, China on 23 February or 19 February. The number of infections would be 58,077-84,520 or 55,869-81,393 with R t = 0.9 or 0.5, respectively (Figs. 2 and 3).Our model predicted 2323-3381 deaths in Wuhan, China when we assumed R t as 0.9 and the percent of deaths as 4%; 2235-3256 deaths when we assumed R t as 0.5 at the fourth phase. An average of 2279-3318 deaths were also estimated ( Table 1) .When we assumed R t as 0.9 and the percent of deaths 10% based on calculation of case fatality rate (CFR) at early stage of the epidemic 6 , our model predicted 5808-8452 deaths in Wuhan, China; 5587-8139 deaths when we assumed R t as 0.5 at the fourth phase. An average of 5697-8296 deaths were also estimated.Estimations of the transmission risk and the epidemic trend of COVID-19 are of great importance because these can arouse the vigilance of the policy makers, health professionals, and the whole society so that enough resources would be mobilized in a speedy and efficient In all, 55,869 represents the estimated peak number of COVID-19 cases on 19 February 2020 in Wuhan, China with R 0 = 0.5; 58,077 represents the estimated peak number of COVID-19 cases on 23 February 2020 in Wuhan, China with R 0 = 0.9; E: number of exposed cases; I: number of infectious cases; E was assumed to be 20 times of I at baseline. way for both control and treatment. We estimated the number of infections using SEIR (Susceptible, Exposed, Infectious, and Removed) model under two assumptions of R t (R t maintaining to be >1 or R t gradually decreasing to <1) in the purpose of depicting various possible epidemic trends of COVID-19 in Wuhan, China. Two estimations provide an approach for evaluating the sufficiency of the current measures taken in China, depending on whether or not the peak of the number of infections would occur in February 2020. Assuming the current control measures were ineffective and insufficient, the estimated number of infections would continue to increase throughout February without a peak. On the other hand, assuming the current control measures were effective and sufficient, the estimated number of infections would reach the peak in late February 2020.According to Read's research 13 , R 0 for COVID-19 outbreak is much higher compared with other emergent coronaviruses. It might be very difficult to contain or control the spreading of this virus. If the prevention and control measures were not sufficient or some new factors occurred (e.g., a large proportion of cases with mild or none symptoms existed in the community; there were more zoonotic sources), the epidemic might continue to develop at a high speed. Therefore, we depicted first the epidemic dynamics of the relatively unsatisfying circumstance based on the R 0 estimated before the unprecedented efforts of China in the containment of the epidemics occurred and the newest documented parameters. The curve continued to go up throughout February without any indication of dropping, indicating the need for further enhancement of public health measures for containment of the current outbreak.However, as mentioned by WHO in the statement on 30 January, \"it is still possible to interrupt virus spread, provided that countries put in place strong measures to detect disease early, isolate and treat cases, trace contacts, and promote social-distancing measures commensurate with the risk.\" 8 Responding to the outbreak, China has taken a series of unprecedentedly strict measures regardless of economic losses, including daily contact with WHO and comprehensive multi-sectoral approaches to fight against the virus and prevent further spread, showing the sense of responsibility of China to its citizens and the whole world. Epidemic information has been released in an open, transparent, responsible, and timely manner home and abroad. Cooperation has been established with other countries and international organizations. These measures have won full recognition of the international community, including WHO. Specifically in Wuhan, in the early phase from beginning of December 2019 to 23 January 2020, there was no limitation of population flow and gathering. When the human-to-human transmission was confirmed, an important decision was made to isolate Wuhan from other parts of the country. As a result, since 24 January 2020, all public transports from and to Wuhan, as well as public transports and people's gathering events within Wuhan, were stopped. Since 2 February 2020, strict public health measures were taken to prevent population flow among distinct communities, whereas since 9 February 2020, public health interventions including quarantine of each building in the urban area and each village in the rural area were implemented in order to block the transmission chain among the household. Therefore, strong efforts of authorities and people in Wuhan with the support of the central government and people from all over China, as well as the WHO and the international society, may have gradually braked COVID-19 outbreak. R t is therefore assumed to decrease gradually from 3.1 to 0.5 in Wuhan, China in the current study. The trend of the estimated cases is in accordance with the trend of currently confirmed cases. The relatively big difference in number may be due to the possible existence of a large number of mild and asymptomatic cases and the imperfection of current diagnostic measures. According to NHC, before 12 February 2020, the confirmed cases were diagnosed according to contact history, clinical manifestations, chest X-ray, or computer-assisted tomography (CT) and RT-PCR for COVID-19. Since 12 February 2020, the diagnosis has been mainly based on contact history, clinical manifestation, and imaging evidence of pulmonary lesion suggestive of pneumonia, while viral detection with RT-PCR is still being performed in a part of patients 23 . After the diagnosis method was changed, a large number of cases that were previously missed and piled up for testing were reported in Wuhan, which greatly increased the number of existing cases and made it approaching our estimated number. A peak of the estimated number of infections would occur in late February under this assumption. If the peak does occur in February, the very strong measures China has taken may have already received success in controlling the COVID-19 infection in Wuhan.The number of deaths in the current study was estimated based on previously reported CFRs. Chen et al. 6 calculated it to be 11% based on 99 cases at the very early stage of the outbreak. This mortality rate might not be representative of the whole patients' population due to the relatively small sample size and scarce knowledge about the virus at early stage. More recently, Yang et al. 24 estimated the overall adjusted CFR among confirmed patients to be 3.06% with a sample size of 8866. The number of deaths estimated accordingly might be more close to the reality. Our estimation of the number of deaths only provides a possible range based on currently reported CFR. The actual number of deaths might be lower with more mild and asymptomatic cases being detected and the improvement of clinical care and treatment as the epidemic progresses.Hubei Province, of which Wuhan is the capital city, accounts for more than 80% of newly confirmed cases all over the country according to NHC daily report. The current epidemic trend in Hubei Province is similar to that in Wuhan City. Considering the high number of confirmed cases in the province, the currently strict measures should be continuously implemented both in Wuhan and other cities in Hubei Province no matter whether the peak of number of infections would occur or not, in order to reduce R t to an ideal level and to control the epidemic. Owing to the timely transportation restriction in Hubei Province and other measures, the number of newly confirmed cases remains relatively low and has decreased for 13 days in a row in other provinces, autonomous regions, and municipalities in mainland China outside Hubei Province. However, independent self-sustaining human-to-human spread is estimated to already present in multiple major Chinese cities, including Beijing, Shanghai, Guangzhou, and Shenzhen 16 . In addition, pressure on transmission control caused by the population migration after Spring Festival holidays may occur soon, especially in some densely populated cities 25 . Necessary strict measures should still be maintained even when the current measures turn out to be effective.Our study has some limitations. Firstly, the SEIR model was set up based on a number of necessary assumptions. For example, we assumed that no super-spreaders exist in the model, but there is currently no supportive evidence. Secondly, the accuracy of the estimation model depends largely on the accuracy of the parameters it used, such as incubation period. With more precise parameters obtained as the epidemic progresses, our estimation model will also be more precise. Our estimates of the reproductive number from 3.1 to 0.5 are based on previous studies and experience from SARS control. However, this measure may change substantially over the course of this epidemic and as additional data arrives. Besides, using a fixed R t value in each phase may incur potential bias because R t is essentially a dynamic parameter. Thirdly, these estimated data may not be sustained if unforeseeable factors occurred. For example, if some infections were caused by multiple exposures to animals, these estimates will be exposed to a big uncertainty. Fourthly, the epidemic trend shows great difference between Wuhan and Hubei Province and regions in mainland China outside Hubei Province according to the NHC reported data. It is thus inappropriate to generalize the estimations in Wuhan to regions in mainland China outside Hubei Province. The dynamics model for the other locations in mainland China remains to be developed and specific parameters need to be redefined. Lastly, we do not provide model fit information in the current study. SEIR model is a prediction model forecasting the number of infections in the future. The data corresponding to actual situation in the future cannot be determined and this makes model fitting almost impossible during the outbreak. We would carry out model fitting according to the real data in pace with more information and knowledge about the characteristics of COVID-19 and the epidemics in the future.Despite the limitations mentioned above, the current study is the first to provide estimation for epidemic trend after strict prevention and control measures were implemented in China. Whether current prevention and control measures are sufficient or not may be evaluated through the occurrence of the infection number peak in February. Rigorous measures should still be maintained even when the current measures turn out to be effective by the end of February to prevent further spread of the virus.We employed an infectious disease dynamics model (SEIR model) for the purpose of modeling and predicting the number of COVID-19 cases in Wuhan, China. The model is a classic epidemic method to analyze the infectious disease, which has a definite latent period, and has proved to be predictive for a variety of acute infectious diseases in the past such as Ebola and SARS 22, . Application of the mathematical model is of great guiding significance to assess the impact of isolation of symptomatic cases as well as observation of asymptomatic contact cases and to promote evidence-based decisions and policy.We assumed no new transmissions from animals, no differences in individual immunity, the time-scale of the epidemic is much faster than characteristic times for demographic processes (natural birth and death), and no differences in natural births and deaths. In this model, individuals are classified into four types: susceptible (S; at risk of contracting the disease), exposed (E; infected but not yet infectious), infectious (I; capable of transmitting the disease), and removed (R; those who recover or die from the disease). The total population size (N) is given by N = S + E + I + R. It is assumed that susceptible individuals who have been infected first enter a latent (exposed) stage, during which they may have a low level of infectivity. The differential equations of the SEIR model are given as: 32, 33 dS=dt ¼ Àβ S I=N;where β is the transmission rate, σ is the infection rate calculated by the inverse of the mean latent period, and γ is the recovery rate calculated by the inverse of infectious period.R software (version 3.6.2) was applied for all the calculations and estimates in the current study.Estimation of the epidemic trend assuming the prevention and control measures are insufficientWe first estimated the epidemic trend in Wuhan, China assuming the current prevention and control measures are insufficient. In this process, S was assumed to be the population of Wuhan City (11 million) 15, 34 . The initial assumed number of cases caused by zoonotic exposure was 40 (I) according to Imai et al.'s 9 estimation. We proposed E at 20 times of I in accordance with Read et al. 13 . R was set as 0. σ was set as 1/5.2 according to the latest article by Li et al. 15 , which calculated the mean incubation period of COVID-19 to be 5.2 days. Chen et al. 6 calculated the average hospitalization period of 31 discharged patients to be 12.39 ± 4.77 days. Yang et al. 24 calculated the median time from disease onset to diagnosis among confirmed patients to be 5. γ was accordingly set as 1/18 (ceiling of 12.39 ± 5 is 18). R 0 was chosen based on Imai et al.'s 9 estimation 2.6 (1.9-3.1) assuming 4000 (1000-9700) infections as of 18 January.This section discussed the scenario where the current prevention and control measures are sufficient. The set of S, E, I, R, σ, and γ is the same as the first section except that we also explored the model with E at 30 times of I to provide a possible range of number of infections. The absence of fever in COVID-19 cases is more frequent than in SARS-CoV and MERS-CoV infection 35 . Such patients may be missed since the current surveillance case definition focused mainly on fever detection. Accordingly, the possibility of E at 30 times of I cannot be excluded.R 0 in this section was chosen by phases. The first phase ranges from 1 December 2019 to 23 January 2020 and can be regarded as the early phase of the epidemic when a few prevention and control measures were implemented. R 0 was set as 3.1 consistent with Imai et al.'s 9 estimation of high transmission level. On 23 January 2020, airplanes, trains, and other public transportation within the city were restricted and other prevention and control measures such as quarantine and isolation were gradually established in Wuhan 20 . So, the second phase began on 24 January and R t was set as 2.6 consistent with Imai et al.'s 9 estimation of moderate transmission level. Second February was the last day of the extended Spring Festival holiday and Chinese authorities mobilized more medical resources to support Wuhan ever since 36 . The newly constructed hospital \"Huoshenshan\" came into service on this day 37 and \"Leishenshan,\" mobile cabin hospitals several days later 38 . Also, more and more medical teams arrived in Wuhan. So the third phase began on 3 February and R t was set as 1.9 consistent with Imai et al.'s 9 estimation of low transmission level. All of these measures may need one longest incubation period to take effect. So, the last phase began on 16 February and R t was set as 0.9 and 0.5, respectively, assuming the prevention and control measures are sufficient and effective to depict two different levels of effect of the measures in reducing transmission probability.\n",
            "\t0.971\tIn early December 2019 a cluster of cases of pneumonia of unknown cause was identified in Wuhan, a city of 11 million persons in the People's Republic of China. Further investigation revealed these cases to result from infection with a newly identified coronavirus, initially termed 2019-nCoV and subsequently SARS-CoV-2. The infection moved rapidly through China, spread to Thailand and Japan, extended into adjacent countries through infected persons travelling by air, eventually reaching multiple countries and continents. Similar to such other coronaviruses as those causing the Middle East respiratory syndrome (MERS) and severe acute respiratory syndrome (SARS), the new coronavirus was reported to spread via natural aerosols from human-to-human. In the early stages of this epidemic the case fatality rate is estimated to be approximately 2%, with the majority of deaths occurring in special populations. Unfortunately, there is limited experience with coronavirus infections during pregnancy, and it now appears certain that pregnant women have become infected during the present 2019-nCoV epidemic. In order to assess the potential of the Wuhan 2019-nCoV to cause maternal, fetal and neonatal morbidity and other poor obstetrical outcomes, this communication reviews the published data addressing the epidemiological and clinical effects of SARS, MERS, and other coronavirus infections on pregnant women and their infants. Recommendations are also made for the consideration of pregnant women in the design, clinical trials, and implementation of future 2019-nCoV vaccines.Coronaviruses are spherical, enveloped, and the largest of positive-strand RNA viruses. They have a wide host range, including birds, farm animals, pets, camels, and bats, in which they primarily cause respiratory and gastrointestinal disease. Belonging to the order Nidovirales, family Coronaviridae, and the subfamily Orthocoronaviridae there are four genera of coronaviruses-Alphacoronavirus, Betacoronavirus, Deltacorona virus, and Gammacoronavirus .In humans, they are a cause of mild illnesses including the common colds occurring in children and adults, and were believed to be of modest medical importance. However, two zoonotic coronaviruses-including the severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV)-can produce severe lower respiratory In the beginning of December 2019, a cluster of persons with a pneumonia of unknown cause was identified in Wuhan, the capital of Hubei Province and a large city of approximately 11 million persons located in the central region of the People's Republic of China [7, 8] . Between 8 and 18 December 2019 there were 7 cases of pneumonia identified whose clinical features resembled that of a viral pneumonia. The outbreak was initially believed to be linked to the Wuhan Huanan (South China) Seafood Wholesale Market. This market, termed a \"wet\" market, sells a variety of seafood, cuts of meat, and both live and dead animals in over one thousand stalls in constant close contact; however, whether this market was the origin of the outbreak remains unknown . On 31 December 2019, the Chinese Center for Disease Control and Prevention (China CDC) sent a rapid response team to Hubei to work alongside health personnel from the provincial and Wuhan city health departments to conduct an epidemiologic investigation. As the disease was spreading through secondary and tertiary cases, the World Health Organization (WHO) China Country Office was informed on 31 December 2019 of the occurrence of these cases of pneumonia of unknown etiology. During the period from 31 December 2019 to 3 January 2020, 44 patients with pneumonia of unknown etiology were reported by the Chinese authorities to the WHO. On 7 January 2020 investigators in China identified the etiological agent of the epidemic as a previously unknown coronavirus, and it was given the designation 2019-nCoV (for 2019 novel coronavirus) . Analysis of the clinical features of 41 hospitalized patients with laboratory-confirmed 2019-nCoV infection revealed that 30 were men (73%); less than one-half had underlying co-morbid conditions (13; 32%) which included diabetes (8, 20%) , hypertension (6, 15%), and cardiovascular disease (6; 15%); and the average age was 49.0 years old. The most common symptoms at the beginning of their illness included fever (40, 98%) , cough (31, 76%) , and fatigue or myalgia (18, 44%) , sputum production (11, 28%) , and headache (3, 8%) . Among these 41 initial cases of 2019-nCoV infection there were 12 patients (32%) who developed acute respiratory distress syndrome (ARDS), 13 (32%) required intensive care and 6 (15%) died. During the first weeks of January the infection spread rapidly through China and extended to adjacent countries where cases began to appear-13 January in Thailand, 15 January in Japan, 20 January in the Republic of Korea, and Taiwan and the United States on 21 January . Infected travelers, mostly via commercial air travel, are known to have been responsible for introducing the virus outside of Wuhan. The new coronavirus continued to spread throughout multiple countries and continents, and by 9 February 2020 the WHO reported 37,251 confirmed cases in China that resulted in 812 deaths, surpassing the number of deaths that occurred during the 2002-2003 SARS epidemic. An additional 307 cases of 2019-nCoV infection have occurred among 24 other countries outside of China . (Figure 1 ) At the meeting of the Emergency Committee of the WHO on 30 January, the novel coronavirus 2019 epidemic was declared a Public Health Emergency of International Concern (PHEIC) [11, 13] .Viruses 2020, 12, 194 3 of 16 epidemic. An additional 307 cases of 2019-nCoV infection have occurred among 24 other countries outside of China . (Figure 1 ) At the meeting of the Emergency Committee of the WHO on 30 January, the novel coronavirus 2019 epidemic was declared a Public Health Emergency of International Concern (PHEIC) [11, 13] . This newly recognized coronavirus, producing a disease that has been termed COVID-19, is rapidly spreading throughout China, has crossed international borders to infect persons in neighboring countries, and humans infected by the virus are travelling via commercial airlines to other continents. It is certain that 2019-nCoV will infect women who are pregnant, leaving the question open as to whether the novel coronavirus will have a similar or different effect on them compared with SARS-CoV and MERS-CoV. In order to address the potential obstetrical outcomes of infection to both mother and infant, the present communication describes the current state of knowledge regarding the effects of other coronavirus infections in pregnancy.Pneumonia arising from any infectious etiology is an important cause of morbidity and mortality among pregnant women. It is the most prevalent non-obstetric infectious condition that occurs during pregnancy . In one study pneumonia was the 3rd most common cause of indirect maternal death . Approximately 25 percent of pregnant women who develop pneumonia will need to be hospitalized in critical care units and require ventilatory support . Although bacterial pneumonia is a serious disease when it occurs in pregnant women, even when the agent(s) are susceptible to antibiotics, viral pneumonia has even higher levels of morbidity and mortality during pregnancy . As with other infectious diseases, the normal maternal physiologic changes that accompany pregnancy-including altered cell-mediated immunity and changes in pulmonary function-have been hypothesized to affect both susceptibility to and clinical severity of pneumonia . This has been evident historically during previous epidemics. The case fatality rate (CFR) for pregnant women infected with influenza during the 1918-1919 pandemic was 27%-even higher when exposure occurred during the 3rd trimester and upwards of 50% if pneumonia supervened . During the 1957-1958 Asian flu epidemic, 10% of all deaths occurred in pregnant women, and their CFR was twice as high as that of infected women who were not pregnant . The most common adverse obstetrical outcomes associated with maternal pneumonias from all causes include This newly recognized coronavirus, producing a disease that has been termed COVID-19, is rapidly spreading throughout China, has crossed international borders to infect persons in neighboring countries, and humans infected by the virus are travelling via commercial airlines to other continents. It is certain that 2019-nCoV will infect women who are pregnant, leaving the question open as to whether the novel coronavirus will have a similar or different effect on them compared with SARS-CoV and MERS-CoV. In order to address the potential obstetrical outcomes of infection to both mother and infant, the present communication describes the current state of knowledge regarding the effects of other coronavirus infections in pregnancy.Pneumonia arising from any infectious etiology is an important cause of morbidity and mortality among pregnant women. It is the most prevalent non-obstetric infectious condition that occurs during pregnancy . In one study pneumonia was the 3rd most common cause of indirect maternal death . Approximately 25 percent of pregnant women who develop pneumonia will need to be hospitalized in critical care units and require ventilatory support . Although bacterial pneumonia is a serious disease when it occurs in pregnant women, even when the agent(s) are susceptible to antibiotics, viral pneumonia has even higher levels of morbidity and mortality during pregnancy . As with other infectious diseases, the normal maternal physiologic changes that accompany pregnancy-including altered cell-mediated immunity and changes in pulmonary function-have been hypothesized to affect both susceptibility to and clinical severity of pneumonia . This has been evident historically during previous epidemics. The case fatality rate (CFR) for pregnant women infected with influenza during the 1918-1919 pandemic was 27%-even higher when exposure occurred during the 3rd trimester and upwards of 50% if pneumonia supervened . During the 1957-1958 Asian flu epidemic, 10% of all deaths occurred in pregnant women, and their CFR was twice as high as that of infected women who were not pregnant . The most common adverse obstetrical outcomes associated with maternal pneumonias from all causes include premature rupture of membranes (PROM) and preterm labor (PTL), intrauterine fetal demise (IUFD), intrauterine growth restriction (IUGR), and neonatal death .The SARS epidemic began quietly at the turn of the 21st century. In November 2002, a cook in Guangdong Province, China, died from an unidentified illness. He had worked at a restaurant in which meat from wild animals was served. On 27 November 2002 Chinese-language media and internet reports were picked up by Canada's Global Public Health Intelligence Network (GPHIN) that indicated a flu-like illness was occurring in China [25, 26] . Unfortunately, the reports were not translated, and China failed to report the occurrence of this illness to the World Health Organization (WHO) until February 2003. The disease spread to other countries where it primarily infected healthcare workers. One of these was Dr. Carlo Urbani, a WHO physician investigating a patient with the new disease in Hanoi. He recognized that the pneumonia was probably caused by a new, highly infectious agent, and rapidly notified the WHO. He contracted the SARS-CoV while there, became febrile and later died after traveling to Thailand to attend a conference. On 12 March 2003, WHO issued a global alert regarding the disease that was occurring primarily among health care workers in Hanoi, Vietnam and Hong Kong. The disease continued to spread, and by 31 July 2003 there were 8422 probable cases, leading to 916 deaths in 29 countries, with the majority of cases occurring in mainland China and Hong Kong. Approximately 30% of infections occurred in healthcare workers. By the termination of the epidemic the global CFR was 11% .Although there were relatively few documented cases of SARS occurring during pregnancy, several case reports and small clinical studies have described the clinical effects in pregnant women and their infants. In reviewing these reports describing pregnant women with SARS in China it is possible, and perhaps even probable, that some of the same patients were included in more than one publication. However, even if this is the case, there is no doubt that SARS coronavirus infection was found to be associated with severe maternal illness, maternal death, and spontaneous abortion [19, . Martha Anker, an expert in statistics formerly with the WHO and the University of Massachusetts, estimated that more than 100 cases of SARS-CoV infection occurred in pregnant women, which warrants closer inspection .The clinical outcomes among pregnant women with SARS in Hong Kong were worse than those occurring in infected women who were not pregnant . Wong et al. evaluated the obstetrical outcomes from a cohort of pregnant women who developed SARS in Hong Kong during the period of 1 February to 31 July 2003. Four of the 7 women (57%) that presented during the 1st trimester sustained spontaneous miscarriages, likely a result of the hypoxia that was caused by SARS-related acute respiratory distress. Among the 5 women who presented after 24 weeks gestation, 4 had preterm deliveries (80%).A case-control study to determine the effects of SARS on pregnancy compared 10 pregnant and 40 non-pregnant women with the infection at the Princess Margaret Hospital in Hong Kong [27, 33] . There were 3 deaths among the pregnant women with SARS (maternal mortality rate of 30%) and no deaths in the non-pregnant group of infected women (P = 0.006). Renal failure (P = 0.006) and disseminated intravascular coagulopathy (P = 0.006) developed more frequently in pregnant SARS patients when compared with the non-pregnant SARS group. Six pregnant women with SARS required admission to the intensive care unit (ICU) (60%) and 4 required endotracheal intubation (40%), compared with a 12.5% intubation rate (P = 0.065) and 17.5% ICU admission rate (P = 0.012) in the non-pregnant group.Maxwell et al. reported 7 pregnant women infected with SARS-CoV who were followed at a designated SARS unit-2 of the 7 died (CFR of 28%), and 4 (57%) required ICU hospitalization and mechanical ventilation. In contrast, the mortality rate was less than 10% and mechanical ventilation rate less than 20% among non-pregnant, age-matched counterparts who were not infected with SARS-CoV. Two women with SARS recovered and maintained their pregnancy but had infants with IUGR. Among the live newborn infants, none had clinical or laboratory evidence for SARS-CoV infection. The new mothers who had developed SARS were advised not to breastfeed to prevent possible vertical transmission of the virus.Zhang et al. described SARS-CoV infections in 5 primagravidas from Guangzhou, China at the height of the SARS epidemic. Two of the mothers became infected in the 2nd trimester, and 3 developed infection in the 3rd trimester. Two of the pregnant women had hospital-acquired SARS infections, and the other 3 were community-acquired. All 5 pregnant women had fever and abnormal chest radiographs; 4 had cough; 4 developed hypoalbuminemia; 3 had elevated alanine aminotransferase levels (ALT), 3 had chills or rigor, 2 had decreased lymphocytes, and 2 had decreased platelets. One pregnant woman required intensive care, but all recovered and there were no maternal deaths. The 5 infants were clinically evaluated, and none had evidence of SARS.Two pregnant women with SARS were reported from the United States. In a detailed case report, Robertson et al. described a 36-year-old pregnant woman with an intermittent cough of approximately 10 days duration and no fever. While travelling in Hong Kong during the 2003 epidemic, she was exposed at her hotel to a person subsequently known to be infected with SARS-CoV. At 19 weeks gestation she developed fever, anorexia, headache, increasing cough, weakness, and shortness of breath. Upon returning to the United States she was hospitalized with pneumonia. Obstetrical ultrasounds revealed a low-lying placenta (placenta previa) but were otherwise normal. Following her discharge home and clinical recovery, she was found to have antibodies to SARS-CoV. She underwent cesarean section at 38 weeks gestation because of the placenta previa and a healthy baby girl was delivered [35, 36] . The placenta was interpreted as being normal. At 130 days post-maternal illness, maternal serum and whole blood, swabs from maternal nasopharynx and rectum, post-delivery placenta, umbilical cord blood, amniotic fluid, and breast milk were collected for analysis-no viral RNA was detected in specimens tested by reverse transcriptase polymerase chain reaction (RT-PCR). Antibodies to SARS-CoV were detected from maternal serum, umbilical cord blood, and breast milk by enzyme immunoassay (EIA) and indirect immunofluorescence assay. No clinical specimens (except for cord blood) were available for testing from the infant. The second case in the USA occurred in a 38-year-old woman who had travelled to Hong Kong at 7 weeks gestation where she was exposed to SARS-CoV in the same hotel as the aforementioned American woman . Following her return to the United States, her husband developed the clinical onset of SARS, and 6 days later she became ill with fever, myalgia, chills, headache, coryza, and a productive cough with shortness of breath and wheezing. Following her hospitalization for SARS she recovered, serum samples taken on days 28 and 64 post-onset of illness were positive for antibodies to SARS-CoV by enzyme immunoassay and immunofluorescent assays. Her pregnancy continued and was unremarkable except for developing elevated glucose levels. A cesarean section that was performed at 36 weeks gestation due to preterm rupture of membranes and fetal distress resulted in a healthy baby boy. At the time of delivery, the mother's serum samples were positive for antibodies to SARS-CoV, but samples taken of umbilical cord blood and placenta were negative. Breast milk sampled 12 and 30 days after delivery were also negative for SARS-CoV antibodies. Specimens evaluated from maternal blood, stool, and nasopharynx samples, as well as umbilical cord blood of the infant, were all negative for coronavirus RNA by RT-PCR. Neonatal stool samples obtained on days-of-life 12 and 30 were also negative for viral RNA.From Canada, Yudin et al. reported a 33-year-old pregnant woman who was admitted to the hospital at 31 weeks gestation with a fever, dry cough, and abnormal chest radiograph demonstrating patchy infiltrates. She had acquired SARS from contact with an infected family member. Following a 21-day stay in the hospital, during which she did not require ventilatory support, her convalescent antibody titers were positive for coronavirus infection. She had a normal labor and delivery and her newborn girl had no evidence of infection.In a study of 5 liveborn neonates who were delivered to women infected with SARS-CoV during the Hong Kong epidemic, results from multiple tests-including serial RT-PCR assays, viral culture, and paired neonatal serological titers-were negative for SARS-CoV . None of the 5 neonates developed any clinical signs or symptoms of respiratory infection or compromise.Fortunately, there were no cases of vertical transmission identified among pregnant women infected with SARS-CoV during the 2002-2003 Asian epidemic [27, 30, 31, 39, 40] , and with the exception of a small cluster of cases that recurred in late 2003, no new cases of SARS have occurred.In the only reported study of the placental pathology of mothers with SARS, Ng et al. reported the findings from 7 pregnant women infected with SARS-CoV. In the case of 2 women who were convalescing from SARS-CoV infection during the 1st trimester of pregnancy, the placentas were found to be normal. Three placentas were delivered from pregnancies in which the mothers had acute SARS-CoV infection-these were abnormal and demonstrated increased subchorionic and intervillous fibrin, a finding that can be associated with abnormal maternal blood flow to the placenta. In the placentas of 2 women who were convalescing from SARS-CoV infection in the 3rd trimester of pregnancy the placentas were highly abnormal. They showed extensive fetal thrombotic vasculopathy with areas of avascular chorionic villi-chronic findings of fetal vascular malperfusion. These 2 pregnancies also were complicated by oligohydramnios and had poor obstetrical outcomes-both infants had developed IUGR. It is interesting that villitis, the microscopic finding of inflammation of the chorionic villi that is the histologic hallmark of many maternal hematogenous infections that are transmitted through the placenta to the fetus, was not identified in any of these placentas.Similar to other coronavirus infections, SARS-CoV is easily spread from person-to-person via respiratory droplets and secretions as well as through nosocomial contacts [42, 43] . In addition to transmission of SARS-CoV through natural aerosols from infected patients, it was found that in Hong Kong the SARS-CoV could also be transmitted by mechanical aerosols . Environmental factors had an important role when it was discovered that during the Amoy Gardens housing estate outbreak as many as two-thirds of infected persons had diarrhea, SARS-CoV was excreted in their stools, and that aerosols arising from the flushing of toilets could transmit the virus . Healthcare facilities were also an important source of new SARS infections during the 2002-2003 epidemic, and healthcare workers were also at high risk for acquiring the infection.In order to address the safety issues for the obstetrical management and delivery of pregnant women with SARS, guidelines were prepared by the Canadian Task Force on Preventive Health Care and the Society of Obstetricians and Gynaecologists of Canada . These recommendations include:1.\"All hospitals should have infection control systems in place to ensure that alerts regarding changes in exposure risk factors for SARS or other potentially serious communicable diseases are conveyed promptly to clinical units, including the labour and delivery unit.At times of SARS outbreaks, all pregnant patients being assessed or admitted to the hospital should be screened for symptoms of and risk factors for SARS.Upon arrival in the labour triage unit, pregnant patients with suspected and probable SARS should be placed in a negative pressure isolation room with at least 6 air exchanges per hour. All labour and delivery units caring for suspected and probable SARS should have available at least one room in which patients can safely labour and deliver while in need of airborne isolation.If possible, labour and delivery (including operative delivery or Caesarean section) should be managed in a designated negative pressure isolation room, by designated personnel with specialized infection control preparation and protective gear. 5.Either regional or general anaesthesia may be appropriate for delivery of patients with SARS.Neonates of mothers with SARS should be isolated in a designated unit until the infant has been well for 10 days, or until the mother's period of isolation is complete. The mother should not breastfeed during this period. 7.A multidisciplinary team, consisting of obstetricians, nurses, pediatricians, infection control specialists, respiratory therapists, and anaesthesiologists, should be identified in each unit and be responsible for the unit organization and implementation of SARS management protocols. 8.Staff caring for pregnant SARS patients should not care for other pregnant patients. Staff caring for pregnant SARS patients should be actively monitored for fever and other symptoms of SARS. Such individuals should not work in the presence of any SARS symptoms within 10 days of exposure to a SARS patient. 9.All health care personnel, trainees, and support staff should be trained in infection control management and containment to prevent spread of the SARS virus. 10. Regional health authorities in conjunction with hospital staff should consider designating specific facilities or health care units, including primary, secondary, or tertiary health care centers, to care for patients with SARS or similar illnesses.\"Middle East respiratory syndrome (MERS) was first reported in September 2012 in Saudi Arabia, following isolation of MERS-CoV from a male patient who died months earlier from severe pneumonia and multiple organ failure . In the 8 years since then, there have been more than 2494 confirmed cases of MERS resulting in upwards of 858 deaths globally . While 27 countries have reported cases of MERS, approximately 80% of confirmed cases originated in Saudi Arabia . To date, all known cases of MERS can be linked to travel or residence in countries along the Arabian Peninsula-that is, Bahrain; Iraq; Iran; Israel, the West Bank, and Gaza; Jordan; Kuwait; Lebanon; Oman; Qatar, Saudi Arabia; Syria; the United Arab Emirates (UAE); and Yemen . The largest documented outbreak outside of this region occurred in 2015 in the Republic of Korea, in which 186 infections occurred, resulting in 38 deaths . The index case in this outbreak reportedly returned from the Arabian Peninsula just prior to onset of illness .MERS-CoV is characterized by sporadic zoonotic transmission events as well as spread between infected patients and close contacts (i.e., intra-familial transmission) . Nosocomial outbreaks in health care settings-the result of poor infection control and prevention-are widely recognized as the hallmark of MERS . Superspreading events have been recorded in healthcare settings in Jordan, Al Hasa, Jeddah, Abu Dhabi and South Korea [47, . Like other coronaviruses, MERS-CoV can be spread through person-to-person contact, likely via infected respiratory secretions . Transmission dynamics, however, are otherwise poorly understood . Bats are believed to be the natural reservoir of MERS-CoV, and dromedary camels can have the virus and have been suggested as possible intermediary hosts as well as a source of infection to humans [2, 56, 57] .There are no clinical or serological reports of perinatal transmission of MERS, though vertical transmission has been reported for non-coronavirus respiratory viruses including influenza and respiratory syncytial virus (RSV) . Researchers have not yet discovered ongoing transmission of MERS-CoV within communities outside of health care settings.The clinical presentation of MERS varies from asymptomatic to severe pneumonia with acute respiratory distress syndrome (ARDS), septic shock, and multiple organ failure, often resulting in death. Most patients with MERS develop severe acute respiratory illness accompanied by fever, cough, and shortness of breath . Progression to pneumonia is swift-usually within the first week -and at least one-third of patients also present with gastrointestinal symptoms . MERS progresses much more rapidly to respiratory failure and has a higher case fatality rate than SARS . Unlike SARS, however, infection with MERS-CoV is generally mild in healthy individuals but more severe in immunocompromised patients and people with underlying comorbidities . The overall CFR of MERS is approximately 34.4% . Most fatalities have been associated with pre-existing medical conditions like chronic lung disease, diabetes, and renal failure, as well as weakened immune systems , making such individuals high risk. As a result of the immunological changes that occur during pregnancy, women who are pregnant are included in this high-risk group. Pregnant women may develop severe disease and fatal maternal and/or fetal outcomes as a result of MERS-CoV infection; however, little is known of the pathophysiology of this infection during pregnancy.Limited data exists on the prevalence and clinical features of MERS during pregnancy, birth, and the postnatal period. It is likely, however, that the immunological changes that normally occur in pregnancy may alter susceptibility to the MERS-CoV and the severity of clinical illness . Pregnant women infected with SARS-CoV, a related coronavirus, appear to have increased morbidity and mortality when compared to non-pregnant women, suggesting that MERS-CoV could also lead to severe clinical outcomes in pregnancy. To date, however, very few pregnancy-associated cases (n = 11) have been documented, with 91% having adverse clinical outcomes.Between November 2012 and February 2016, there were 1308 cases of MERS reported by the Saudi Arabia Ministry of Health (MoH). Of these, 5 patients were pregnant, according to a retrospective study by Assiri et al. , and all resulted in adverse outcomes. Patient ages ranged from 27 to 34 years, with occurrence of exposure in either the 2nd or 3rd trimester. All 5 cases received intensive care. Two women died and there were 2 cases of perinatal death-1 stillbirth and 1 neonatal death shortly after emergency cesarean section. These instances of severe maternal and perinatal outcomes are consistent with other reports of MERS-CoV infection in pregnant women, as well as outcomes associated with SARS-CoV infection. The authors of the retrospectives study concede that unreported cases of MERS in pregnancy are likely due to lack of routine pregnancy testing . They conclude that pregnancy testing for women of reproductive age should be considered for those who test positive for MERS-CoV, to contribute to overall understanding of pathogenesis and epidemiological risk. Additionally, 2 of the 5 patients were healthcare workers, which corresponds with existing knowledge of higher risk of exposure to MERS-CoV in healthcare settings.In a separate case report of MERS occurring in pregnancy, Alserehi et al. described a 33-year-old critical care nurse who became infected during the 3rd trimester in the midst of a large hospital outbreak. In the days following hospital admission, she developed respiratory failure necessitating mechanical ventilation and administration of dexamethasone as prophylaxis for the fetus. Following an emergency cesarean section at 32 weeks gestation, she was transferred to the intensive care unit (ICU) and later recovered. The preterm but otherwise healthy infant was kept in the neonatal unit for observation and later released along with his mother. In contrast to other reported cases, this patient had a successful outcome, perhaps due to the timing of MERS-CoV exposure, her young age, the use of steroids, and differences in immune response.Alfaraj et al. described 2 cases of maternal infection with MERS-CoV at the Prince Mohammed Bin Abdulaziz Hospital (PMAH) in Saudi Arabia. Maternal infection in both cases was confirmed by nasopharyngeal swab testing by RT-PCR. One patient was a 29-year-old woman at 6 weeks gestation with no underlying medical conditions. The second patient, a 39-year-old at 24 weeks gestation, had several comorbidities, including end stage renal disease, hypertension, and hemodialysis. This woman presented to the hospital after contact with a MERS-CoV-infected person during an active outbreak. Both patients later tested negative for MERS-CoV and were subsequently discharged. The younger patient delivered a healthy, full-term infant. The status of the other delivery is unknown. Neither fetus was tested for MERS-CoV.According to Payne et al. , epidemiologic investigation of the 2012 MERS outbreak in Zarqa, Jordan, revealed that a 2nd trimester stillbirth (5 months gestational age) had occurred as a result of maternal exposure to MERS-CoV. The mother experienced fever, fatigue, headache and cough, concurrently with vaginal bleeding and abdominal pain. On the 7th day of symptoms, she had a fetal death. The mother was confirmed to have antibody to MERS-CoV, and she self-reported having had unprotected contact with family members who later tested positive for the virus. This was the first documented occurrence of stillbirth during maternal infection with MERS-CoV.On 24 November 2013, a 32-year-old pregnant woman in the United Arab Emirates (UAE) developed ARDS following admission to the ICU after suspected community-acquired pneumonia advanced to respiratory failure and hypotension . Later that day, her baby was delivered by caesarean section and subsequent Apgar scores were within healthy range. The next day, RT-PCR evaluation revealed that the mother was positive for MERS-CoV. Despite rigorous intervention, including oral ribavirin-peginterferon-α therapy and ventilator support, the woman continued to deteriorate, developed septic shock, and died. While the outcome for this mother was fatal, Malik et al. noted that virus shedding ceased during therapy with ribavirin and peginterferon-α and radiographic evidence indicated clinical improvement before her death . More research is needed to determine safety, efficacy, and dosage of these therapies in the general population but also in pregnant women. While few data exist on the effects of these treatments in pregnant humans, ribavirin is generally contraindicated during pregnancy .Outside of the Middle East the only confirmed case of MERS in pregnancy occurred in 2015 in South Korea. Jeong et al. reported that a 39-year-old patient was exposed during the 3rd trimester following contact with a patient having MERS. Despite abrupt vaginal bleeding and rupture of membranes, the patient recovered fully and delivered a healthy infant at 37 weeks and 5 days gestation. Subsequent testing of the infant's blood did not detect any IgG, IgM, or IgA antibodies to MERS-CoV.The mean maternal age of the 11 confirmed maternal SARS cases described above was 33.2 years, with a mean gestational age of 26.3 weeks. The source of infection in 2 of the cases was attributed to contact with family members who tested positive for MERS-CoV, unknown in 3 cases, likely due to animal exposure in 1 case, and 6 were healthcare-associated (2 of these patients were healthcare workers). Six patients required intensive care and 3 died. Of those who died, 2 were exposed to MERS-CoV in the 3rd trimester, and 1 was exposed during the 2nd trimester. The infant death rate for all 11 cases was 27%. Fetal survival did not appear to correlate with the timing of maternal infection and gestational age; however, more data are needed to draw conclusions about this relationship. According to Alfaraj et al. , the CFR for the 11 infected women-also 27%-was not statistically different from the overall CFR of MERS in the general population (35%) (P = 0.75). Only 1 case resulted in both maternal and fetal death.Similar to SARS in pregnancy, more research is needed to understand the pathogenesis and epidemiology of MERS in pregnancy including the relationship between the timing of maternal infection, gestational age of the fetus, the effects of comorbid factors, and the occurrence of adverse outcomes. Few studies documented the presence of MERS-CoV antibodies in the umbilical cord or neonatal blood, making it difficult to assess perinatal transmission. As such, future studies should involve the collection of samples from relevant specimens including amniotic fluid, placenta, and umbilical cord .MERS prevention should be high priority for high-risk exposures such as healthcare workers, pregnant women and individuals working with camels, camel meat-milk processors and in abattoirs . Since 2013, the Saudi Arabia MoH has recommended that pregnant women postpone travel to Saudi Arabia for the Hajj and Umrah . To further reduce risk of exposure among pregnant women, additional measures such as avoiding contact with camels and sick persons-particularly in healthcare settings-are also recommended. Pregnant women who present with symptoms of pneumonia, influenza-like illness (ILI), or sepsis on the Arabian Peninsula may also benefit from MERS-CoV screening to expedite early diagnosis and improve disease management .While multiple agents have been used to treat MERS, none have been tested in large clinical studies. Available data are limited to the use of combination therapies of interferon and other agents in case reports and case series . A prospective or randomized study may prove difficult given the sporadic nature of MERS-CoV outbreaks.Due to a gap in research on the treatment of MERS in pregnancy, there are no therapeutic options currently recommended for pregnant women . Therapies under development and testing may be considered inappropriate for pregnant women due to the unknown potential for teratogenic effects. For example, during the 2003 SARS outbreak, ribavirin was administered to pregnant women with severe cases of the disease, but ribavirin therapy has been documented to increase the risk of teratogenic effects in newborns .The Alphacoronaviruses HCoV 229E and NL63, as well as the Betacoronaviruses HKU 1 and OC43, can infect humans and cause the common cold. In order to investigate the potential maternal-fetal transmission of human coronaviruses during pregnancy, Gagneur et al. [64, 65] evaluated 3 types of maternal-infant paired specimens that included maternal vaginal and respiratory specimens that were obtained during labor, as well as gastric samples from the newborn infants. These specimens were evaluated for the presence of HCoV 229E, OC-43, NL63 and HKU 1 using RT-PCR methodology. Between the period from July 2003 to August 2005 the authors examined 159 mother-infant dyads. Human coronaviruses were identified in 12 samples (HCoV 229E: 11; HKU 1 : 1) from 7 mother-child pairs. In 3 mother-infant dyads only maternal respiratory samples were positive; in 2 other pairs all 3 of the samples tested positive for human coronavirus; in 1 case only the maternal vaginal and newborn gastric samples were positive; and in another case the maternal vaginal sample alone was positive. There were no signs of clinical infection in any of the 3 neonates that had positive gastric samples for human coronavirus.It is beyond the scope of this communication to discuss the various technical challenges inherent in developing a safe and efficacious vaccine for coronavirus infections in humans. There are clearly challenges to this endeavor-protective antibodies to coronaviruses are not long-lasting, tissue damage has been reported to occur as a result of exposure to SARS-CoV, development of animal models that closely resemble human infection are limited, and the extensive time and expense necessary to perform clinical trials in humans, to name a few .It is vitally important that pregnant women be considered in the design, clinical trial, and implementation of vaccine candidates for 2019-nCoV. In examining the history of vaccine design, it is clear that the needs of pregnant women have rarely been prioritized in either the preclinical development or the clinical trial phases of production. Today, pregnant women are usually excluded from experimental trial of drugs and vaccines that do not target obstetric conditions . Excluding pregnant women and their infants from participation in vaccine development and implementation undermines ethical principles of justice-fairness, equity, and maximization of benefit-and potentially places their health at risk during outbreaks and other health emergencies .On 23 January 2020 the Coalition for Epidemic Preparedness Innovations (CEPI) announced three programs to develop a vaccine against the novel Wuhan coronavirus. The Chief Executive Officer of CEPI, Richard Hatchett, said :\"Given the rapid global spread of the nCoV-2019 virus the world needs to act quickly and in unity to tackle this disease. Our intention with this work is to leverage our work on the MERS coronavirus and rapid response platforms to speed up vaccine development.\"The novel coronavirus is the first epidemic disease to emerge since the formation of CEPI in Davos in 2017. CEPI was created with the express intent to enable speedy research and development of vaccines against emerging pathogens. In May 2017, WHO released the Target Product Profile (TPP) for MERS-CoV vaccines, following the prioritization of MERS-CoV as one of eight priority pathogens for prevention of epidemics . CEPI and partners aim to use existing platforms-that is, the existing \"backbone\" that can be adapted for use against new pathogens-that are currently in preclinical development for MERS-CoV vaccine candidates. Following the WHO declaration on 30 January that the current 2019-nCoV outbreak is a public health emergency of international concern (PHEIC), global health organizations and researchers will be further mobilized-bolstered by new mechanisms for action and greater resources-to stop the spread of disease.A critical question that must be answered at this stage-with a clear view of the potential deleterious effects of a new coronavirus in pregnancy-is will maternal immunization be a priority in research and development? As of the PHEIC declaration, 12 groups have announced that they are developing new vaccines against 2019-nCoV and seven others announced initiatives to develop new therapies . Safe testing of experimental vaccines in a pregnant population is difficult and, as a result, vaccines are not typically developed with pregnant women in mind. To date, very few clinical trials for vaccines have proactively included pregnant women , and the exclusion of pregnant and lactating women from receiving the rVSV-ZEBOV vaccine through 3 Ebola virus epidemics serves as a recent example . Given the potential severity in pregnancy, as demonstrated by this review of maternal infections of SARS and MERS, women who are pregnant should be considered a priority population in all efforts to prepare for and prevent infection by novel coronaviruses.On 5 February 2020 it was reported by multiple media outlets that a newborn infant delivered during the epidemic in Wuhan had tested positive for 2019-nCoV at the Wuhan Children's Hospital in Hubei Province 30 hours following its birth. According to the official Xinhua news agency, the infant was delivered on 2 February to a mother who had tested positive for the virus. Reports have stated that the infant had stable vital signs, no fever or cough, but had shortness of breath together with abnormal chest radiographs and abnormalities of liver function . Dr. Zeng Lingkong, Chief Physician at the Neonatal Medicine Department of the hospital, said , \"This reminds us to pay attention to mother-to-child being a possible route of coronavirus transmission\"The hospital also provided information about a previous case of a baby that had been delivered on 13 January 2020. Following its birth, the infant's nanny was diagnosed with 2019-nCoV, and the mother was diagnosed days later . On 29 January the baby began to develop symptoms. According to Dr. Zeng Lingkong , \"Whether it was the baby's nanny who passed the virus to the mother who passed it to the baby, we cannot be sure at the moment. But we can confirm that the baby was in close contact with patients infected with the new coronavirus, which says newborns can also be infected\"In considering whether these and future cases of neonatal infection are acquired prior to delivery, it is important to remember that newborn infants can acquire an infection in other ways beyond intrauterine maternal-fetal transmission. In some cases, viral infection can be acquired when the infant passes through the birth canal during a vaginal delivery or through post-partum breast feeding, although these mechanisms would be highly unusual for a respiratory virus. Neonatal infection from respiratory viruses can occur after delivery through such mechanisms as inhalation of the agent through aerosols produced by coughing from the mother, relatives or healthcare workers or other sources in the hospital environment. Based upon past experience with pregnant women who developed MERS and SARS, and realizing that the numbers are limited, there has never been confirmed intrauterine coronavirus transmission from mother to fetus. Discussing the most recent baby to be diagnosed with the 2019-nCoV infection, Dr. Stephen Morse, an epidemiologist at the Mailman School of Public Health at Columbia University stated , \"It's more likely that the baby contracted the virus from the hospital environment, the same way healthcare workers get infected by the patients they treat,\" \"It's quite possible that the baby picked it up very conventionally-by inhaling virus droplets that came from the mother coughing.\"And according to Dr. Paul Hunter, Professor of Medicine at the University of East Anglia , \"As far as I am aware there is currently no evidence that the novel coronavirus can be transmitted in the womb. When a baby is born vaginally it is exposed to the mother's gut microbiome, therefore if a baby does get infected with coronavirus a few days after birth we currently cannot tell if the baby was infected in the womb or during birth.\"There is limited knowledge regarding coronavirus infections that occur during pregnancy-what is known has, for the most part, been the result of epidemics resulting from two different diseases, SARS and MERS. These previous experiences with coronavirus infections in pregnancy indicates that these agents are capable of causing adverse clinical outcomes including life-threatening maternal disease that in some cases requires hospitalization, intensive care and ventilatory support. Both of these coronaviruses can result in maternal death in a small but significant number of cases, but the specific risk factors for a fatal outcome during pregnancy have not been clarified. Coronaviruses can also result in adverse outcomes for the fetus and infant including intrauterine growth restriction, preterm delivery, admission to the ICU, spontaneous abortion and perinatal death. Unlike some viral infections, notably Ebola virus and Zika virus , the likelihood of intrauterine maternal-fetal transmission of coronaviruses is low-there have been no documented cases of vertical transmission occurring with either SARS or MERS. It remains to be seen during the current Wuhan 2019-nCoV epidemic how this newly-emergent coronavirus affects pregnant women and their infants, as well as which factors may modulate obstetrical disease and outcomes including the timing of maternal coronavirus exposure by gestational age, the effects of medications or other treatment regimens, differences in host immune responses, occurrence of coexisting medical and obstetrical conditions, and other covariables. However, pregnant women should be considered to be at high risk for developing severe infection during this current outbreak of 2019-nCoV. Additional clinical research on the treatment of SARS, MERS, and the new coronavirus 2019-nCoV is necessary if we are to understand the potential risks and benefits of novel therapies and new vaccines in pregnancy. This research will be critical in improving the care, and even saving the lives, of pregnant women in the current as well as future outbreaks.\n",
            "\t0.971\tThe novel coronavirus (2019-nCoV, or COVID-19) epidemic first broke out in Wuhan and has been spreading in whole China and the world. The numbers of new infections and deaths in Wuhan are still increasing, which have posed major public health and governance concerns. A series of mandatory actions have been taken by the municipal and provincial governments supported by the central government, such as measures to restrict travels across cities, case detection and contact tracing, quarantine, guidance and information to the public, detection kit development, etc. Challenges such as lacking effective drugs, insufficient hospital services and medical supplies, logistics, etc. have much alleviated with the solidarity of the whole society. The pandemic will definitely be ended with the continuous efforts of both national and international multi-sectoral bodies.Since December 2019, a new type of coronavirus called novel coronavirus (2019-nCoV, or COVID-19) was identified in Wuhan, China. The COVID-19 has then rapidly spread to all over China and the world. It can cause symptoms including fever, difficulty in breathing, cough, and invasive lesions on both lungs of the patients . It can spread to the lower respiratory tract and cause viral pneumonia. In severe cases, patients suffer from dyspnea and respiratory distress syndrome.The pandemic has a big number of infected patients that far exceeded the equivalents of Severe Acute Respiratory Syndromes (SARS) and Middle East respiratory syndrome (MERS), though with a lower fatality rate. According to the surveillance statistics reported by the Chinese government, by February 19, 2020, the number of confirmed infection cases increased to 44,412 for Wuhan and 74,280 for whole China, with 1497 and 2009 deaths respectively. Moreover, the pandemic has caused 919 confirmed infection cases and 3 deaths globally. Therefore, Wuhan city and Hubei Province are the targets for intensive interventions. Otherwise, the spread would have been much faster to all China and the world.Wuhan is a transportation hub of China, it is a highly dense city and has a large population of more than 14 million in 2019. The World Health Organization (WHO) had a meeting on January 30, 2020 and they declared the coronavirus outbreak from China a public health emergency of international concern. Further, there are lots of concerns and debates all over the world, indicating a need for more understanding of China's systems in responding to the outbreak. Therefore, based on our firsthand experience of working with few of the COVID-19 cases, the purpose of this article is to have a brief report of current development, challenges, and future directions of the coronavirus outbreak in Wuhan.It is recognized by the international community that China has made remarkable progress in responding effectively to the outbreak [1, 2] . What made China address the epidemic faster is its ability to finance and mobilize resources combined with its strong governance structure, efficient execution, and solidarity of the whole society. It just took 1 month for China to recognize the existence of a novel coronavirus after the first case was reported, followed with a series mandatory actions in both Wuhan and all over China. In contrast, it took more than 4 months for SARS. On December 31, 2019, delegates of the Chinese Center for Disease Control and Prevention (CDC) went to Wuhan for field investigations, and the sample of new virus was isolated and further identified as a pathogen of unexplained pneumonia on January 6, 2020. The genome-wide sequence of the virus was decoded in the next few days .Population mobility to other cities from Wuhan was restricted on January 23, 2020, and all the inhabitants have been affected to either go out or come back to the city ever since. Vehicle transportations in Wuhan such as city buses and subways were banned and outbound transportations have been canceled (airlines, trains, and long-distance buses). In many places, the Chinese Lunar New Year Festival celebration and other gatherings were cancelled to reduce population concentration. Besides, Wuhan not only imposed a ban on overseas travels for tour purposes, but also suspended selling flight tickets and hotel-booking.After recognizing it as an emergency epidemic on January 22, 2020, strong measures have been adopted immediately by Wuhan local authorities to characterize and control the epidemic, including isolation of suspected cases for treatment, close monitoring of contacts, epidemiological and clinical data collection from patients, and development of diagnostic and treatment procedures. More and more hospitals have been designated by the government to treat infected patients. Thousands of people have been quarantined in the new built hospitals such as Huoshenshan Hospital, Leishenshan Hospital, and Fangcang Hospital to provide care for the confirmed infection patients in Wuhan. In the meantime, patients with different severity are being treated in different hospitals. Thousands of medical professionals nationwide came to Wuhan and other cities in Hubei Province for assistance.Many kinds of guidelines have been developed, and useful information about risk factors and preventive measures are recommended to the public by various means . It is found that the COVID-19 can be transmitted through droplets, contact, aerosol, etc. . A person will not be infected if he washes his hands before touching the conjunctiva. Accordingly, measures such as washing hands, wearing masks and goggles are very effective to prevent potential infections. Further, Wuhan has implemented closed management of communities. Inhabitants are not allowed to go out of their communities and they are very supportive to this regulation.COVID-19 detection kits have been developed and the test results can be generated within 6 h, which is helpful for early diagnosis, treatment and judgment of the treatment effect. Although the number of patients with COVID-19 infections is large in Wuhan, the fatality rate is much lower (3.37%, by February 19, 2020) compared with that of SARS (11%, 2003). By February 19, 2020, 4895 people have been recovered after treatments and most of them are mild cases.It is the first time for the COVID-19 to infect humans and can be transmitted from person to person . The incubation period can be 2 weeks and even longer. Besides, the virus can spread during the incubation period or recessive infection, which makes it difficult to identify those suspected cases without clinical symptoms for prompt control. By far, the numbers of new infections and deaths have already exceeded the equivalent numbers of cases with SARS. If the situation cannot be fully controlled in Wuhan and Hubei province, the situation may deteriorate in other places of China and the world. The government of all levels have been taking strong leadership to combat the outbreak and in recent days there is a decline trend of new cases. However, some challenges still remain and need to be addressed:(1) The large number of confirmed and suspected cases in Wuhan make people staying with them in high risk of getting infected because of the contagiousness of the new virus . This is specially the case for medical professionals. By February 11, 2020, 1716 medical personnel have been infected and six of them died from all over China. Moreover, it is very difficult to identify those people without obvious symptoms, making their families in high risk of getting infected. (2) Hospitals have drastically constrained other services to meet the hospitalization needs of the outbreak in Wuhan. At the early stage, with more people getting infected and less of them recovered, medical facilities, personnel and protective supplies were increasingly insufficient. In many cases, patients cannot be quarantined and treated in time, and many medical staff cannot get fully protected. More efficient logistic services are expected to deliver donated materials from both China and the international society to medical professionals and communities. However, these are all operating problems and the situation has much alleviated with the strong leadership of government of all levels. Wuhan has the competence and will to manage the outbreak supported by the provincial and central governments, together with the solidarity of whole China and the international community. Wuhan has made great contributions to global health. Further global cooperation is required in order to win the pandemic war from local to global. Scientists all over the world need to understand COVID-19 and discover potentially effective antiviral treatments and vaccines; epidemiologists need to conduct intensive contact investigations; It is important to promote protection knowledge for the general public . Finally, Wuhan will not cease the restrictions of population mobility and transportation until the situation is fully under control. With all people working together, it is undoubtedly the COVID-19 can be treated and the pandemic is controllable. However, comprehensive actions from multi-sectoral bodies, both national and international, with supportive attitudes will be important to proceed towards the solution. \n",
            "\n",
            " Title:  Rapid communication\n",
            "3 \b. \n",
            "\n",
            "Input question: What is COVID-19?\n",
            "Results (after 2.192 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.608\tThe virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV .The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host . Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.The symptoms of COVID-19 are fever, dry cough, fatigue, nasal congestion, sore throat and diarrhoea. On February 14th, the Chinese Center for Disease Control and Prevention (China CDC) published the first details of 44,672 confirmed cases, in the biggest study since the outbreak began . Their findings show that COVID-19 was mild for 81% of patients and had an overall case fatality rate of 2.3%. Of those confirmed cases, only 2.2% were under 20 years old. Compared to adults, children generally present with much milder clinical symptoms. It is likely that future serological studies will show much asymptomatic disease in children. As opposed to H1N1, pregnant women do not appear to be at higher risk of severe disease. The severity of the disease appears to be associated with age, with the elderly most at risk; those over 80 years of age had a Case Fatality Rate (CFR) of 14.8%. The CFR was also increased in those with comorbidities including cardiovascular, diabetes, chronic respiratory disease, hypertension, and cancer. The cause of death is respiratory failure, shock or multiple organ failure.There is no proven treatment at this early stage but we will doubtless have more information about this soon. It can be assumed that non-pharmacologic approaches are effective such as fluid support, oxygen and ventilatory support. Most recently the national data suggests that 17.7%, 10.4% and 7.0% of all cases have disease requiring respiratory support in Wuhan, Hubei (not including Wuhan) and the rest of China (respectively). About a quarter of all require ventilation while 75% require oxygen support only. The variation in severity rates probably reflects the outcomes in an overwhelmed health system. Extra Corporeal Membrane Oxygenation (ECMO) is potentially of benefit and we will know more particularly when cities with higher technology health systems become affected and ECMO is truly tested in the most severely ill. ECMO is currently being used in China, however, its effectiveness is yet to be determined. Antiviral drugs as well as a variety of other putative treatments are typically being prescribed for deteriorating patients on a compassionate basis. Clinicians would be well aware of such situations but assurance is required that their safety and efficacy are being scientifically assessed so that meaning is brought to bear quickly. Coordination of clinical trials to avoid duplication and ensure that results are rapidly available will be a challenge but the case numbers should facilitate rapid definitive results. (see What is in the pipeline for vaccine development and/or therapeutics?).As a novel virus newly emerged in humans, the world's population is completely immune-naïve and therefore vulnerable. There is clear human-to-human transmission in family clusters in China and beyond, transmission from close face-to-face social contact, especially in small enclosed spaces, and transmission from failed infection prevention and control measures in health facilities. In addition, the experience in Wuhan shows that transmission can be massive in a short period of time with thousands of new patients diagnosed daily.The current aim of the global response is to flatten the epidemic curve so that transmission is slowed, and to interrupt transmission where possible. While there is clearly a mortality linked to the virus, the most concerning problem will be if a health system is overwhelmed in the wake of rapid transmission so that affected patients cannot receive the care they need. Furthermore, patients with other urgent medical conditions are at risk of not obtaining their necessary care. Countries with vulnerable health systems are particularly of concern.Recently, there have been outbreaks in newly affected countries including Italy and Iran where the index case is unidentified. Furthermore affected countries have very large clusters emerging such as Korea and Japan. There is no reason to believe that the global community is ready for this emerging pandemic; ready in a way that can see drastic public health intervention implemented within days, including aggressive and massive contact tracing, monitoring (or quarantine) and early detection and isolation in an attempt to slow the progression.There is uncertainty regarding transmissibility and severitymore information is emerging about the spectrum of disease, especially mild disease, which is not identified using many current case definitions, and about the ease of transmission from person to person. Health agencies are unsure how to model this and estimates vary depending on the variables being used. For instance, SARS was essentially spread later in the disease from patients with more significant clinical pictures, and it was contained by infection control measures particularly in hospitals. The limited spread to family members of health workers and the community was contained by usual outbreak control measures including early identification and management of persons with infection, tracing of contacts with monitoring for onset of fever and/or symptoms, and active engagement of communities.Most modelling suggests that the severity of illness is more like influenza than SARS, and there is concern among the public health community because the transmissibility of COVID-19 is not yet fully understood, and the potential for it to become endemic like other respiratory pathogens is unknown.Because of the many unknowns, the initial reaction by health agencies is still valid: to brace for the first wave as the virus reaches a completely naïve population and to make maximum effort to interrupt transmission [4, 5] .Experience with managing this outbreak will be very heterogenous across the world. Countries closely connected with China, such as Singapore, will be ahead in this regard. As the outbreak moves across regions, there is opportunity to support those affected later both in terms of readiness and response. Mechanisms available to outbreak response organisations, particularly through the Global Outbreak Alert and Response Network (GOARN), can be valuable in skills-and knowledgesharing. Therefore, there should be a deliberate effort to utilise knowledge from early affected countries in later affected countries.What measures are likely to be successful in curbing its further spread?On January 23rd, Hubei province in Central China was locked down with all movement in and out of the province blocked. Travel across China was discouraged and the number of scheduled flights and train journeys available considerably reduced to perhaps 10% of previous activity. Commercial and social activities became negligible, with schools, restaurants, other entertainment spots and most shops closed. Migrant workers were prevented from returning to work after the extended Chinese New Year Holiday. Frequent hand hygiene when in public and staying at home became the norm. This unprecedented public health effort by China has afforded the rest China and the world a lead time to prepare. The lockdown resulted in a downward trend in national and provincial epidemic curves, however, these measures are not sustainable and eventually there will be a strategy to return to normality. Should this result in a second wave of cases, and more international spread, countries around the world must be prepared. Early identification, and isolation or cohorting of positive cases to designated sites is at the core. In order to achieve this, hospitals, quarantine services, laboratories, together with epidemiology and communication teams will need to be scaled up to provide effective and efficient care.What is in the pipeline for vaccine development and/or therapeutics?It appears that no vaccine will be available for at least one year, likely a little longer. Phase 1 trials for safety and immunogenicity in human populations are likely within 3 months.In terms of therapeutics there is no known effective pharmaceutical agent. There are over 200 registered clinical trials registered in China alone. Putative agents include antivirals; Griffithsin, a spike protein inhibitor, nucleoside analogues eg. remdesivir, ribavirin and protease inhibitors such as lopinavir/ritonavir. Immunomodulatory and other host targeted agents include interferon, chloroquine and immunoglobulins. Corticosteroids will potentially have benefit for immune mediated lung damage late in the course of disease . Much of the theory stems from what we have learnt from limited trials in other corona viruses .\n",
            "\t0.559\tIn December 2019, a novel coronavirus, called COVID-19, was discovered in Wuhan, China, and has spread to different cities in China as well as to 24 other countries. The number of confirmed cases is increasing daily and reached 34,598 on 8 February 2020. In the current study, we present a new forecasting model to estimate and forecast the number of confirmed cases of COVID-19 in the upcoming ten days based on the previously confirmed cases recorded in China. The proposed model is an improved adaptive neuro-fuzzy inference system (ANFIS) using an enhanced flower pollination algorithm (FPA) by using the salp swarm algorithm (SSA). In general, SSA is employed to improve FPA to avoid its drawbacks (i.e., getting trapped at the local optima). The main idea of the proposed model, called FPASSA-ANFIS, is to improve the performance of ANFIS by determining the parameters of ANFIS using FPASSA. The FPASSA-ANFIS model is evaluated using the World Health Organization (WHO) official data of the outbreak of the COVID-19 to forecast the confirmed cases of the upcoming ten days. More so, the FPASSA-ANFIS model is compared to several existing models, and it showed better performance in terms of Mean Absolute Percentage Error (MAPE), Root Mean Squared Relative Error (RMSRE), Root Mean Squared Relative Error (RMSRE), coefficient of determination (R 2 ), and computing time. Furthermore, we tested the proposed model using two different datasets of weekly influenza confirmed cases in two countries, namely the USA and China. The outcomes also showed good performances.A large family of viruses, called coronaviruses, are severe pathogens for human beings, which infect respiratory, hepatic, gastrointestinal, and neurologic diseases. They are distributed among humans, birds, livestock, mice, bats, and other wild animals . The outbreaks of two previous coronaviruses, SARS-CoV and MERS-CoV in 2003 and 2012, respectively, have approved the transmission from animal to animal, and human to human . In December 2019, the World Health Organization (WHO) received notifications from China for many cases of respiratory illness that were linked to some people who had visited a seafood market in Wuhan . Currently, Wuhan city suffers from the spreading of a novel coronavirus, called COVID-19 (previously, it was called 2019-nCoV). In , the authors concluded that COVID-19 likely originated in bats, because it is more similar to two bat-derived coronavirus strains. However, the source of the COVID-19 is not confirmed yet, and it communities, Hong Kong and Toronto, were 1.2 and 1.32, respectively. Ong et al. proposed a monitoring and forecasting model for influenza A (H1N1-2009). Furthermore, Nah et al. proposed a probability-based model to predict the spread of the MERS.The Adaptive Neuro-Fuzzy Inference System (ANFIS) is widely applied in time series prediction and forecasting problems, and it showed good performance in many existing applications. It offers flexibility in determining nonlinearity in the time series data, as well as combining the properties of both artificial neural networks (ANN) and fuzzy logic systems. It has been applied in various forecasting applications, for example, in , a stock price forecasting model was proposed using ANFIS and empirical mode decomposition. Chen et al. proposed a TAIEX time series forecasting model based on a hybrid of ANFIS and ordered weighted averaging (OWA). In , another time series forecasting method was presented for electricity prices based on ANFIS. Svalina et al. proposed an ANFIS based forecasting model for close price indices for a stock market for five days. Ekici and Aksoy presented an ANFIS based building energy consumption forecasting model. More so, ANFIS is also applied to forecast electricity loads . Kumar et al. proposed an ANFIS based model to forecast return products. Ho and Tsai applied ANFIS to forecast product development performance. However, estimating ANFIS parameters is a challenge that needs to be improved. Therefore, in previous studies, some individual swarm intelligence (SI) methods have been applied to the ANFIS parameters to enhance time series forecasting because these parameters have a significant effect on the performance of ANFIS. The SI methods include the particle swarm optimization (PSO) [31, 32] , social-spider optimization , sine-cosine algorithm (SCA) , and multi-verse optimizer (MVO) . For example, in SCA algorithm was applied to improve the ANFIS model to forecast oil consumption in three countries, namely, Canada, Germany, and Japan. In the same context, in , The MVO algorithm was used to enhance the ANFIS model to forecast oil consumption in two countries. In addition, in the PSO was used with ANFIS to predict biochar yield. However, individual SI algorithms may stock at local optima. Therefore, one solution is to apply hybrid SI algorithms to avoid this problem. In , a hybrid of two SI algorithms, namely GA and SSA, was presented to improve the ANFIS model. The proposed new model called GA-SSA-ANFIS was applied to forecast crude oil prices for long-term time series data. However, the previously mentioned methods suffer from some limitations that can affect the performance of the forecasting output such as slow convergence and the ability to balance between exploration and exploitation phases can influence the quality of the final output. This motivated us to propose an alternative forecasting method dependent on the hybridization concept. This concept avoids the limitations of traditional SI techniques by combining the strengths of different techniques, and this produces new SI techniques that are better than traditional ones.In the current study, we propose an improved ANFIS model based on a modified flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). The FPA is an optimization algorithm proposed by Yang , which was inspired by the flow pollination process of the flowering plants. The FPA was employed in various optimization applications, for example to estimate solar PV parameter [39, 40] , solving sudoku puzzles , feature selection , antenna design , and other applications . Moreover, SSA is also an optimization algorithm proposed by Mirjalili et al. inspired by the behavior of salp chains. In recent years, the SSA was utilized to solve different optimization problems, such as feature selection [49, 50] , data classification , image segmentation , and others [53, 54] .The proposed method called FPASSA is a hybrid of FPA and SSA, in which the SSA is applied as a local search method for FPA. The proposed FPASSA starts by receiving the historical COVID-19 dataset. Then a set of solutions is generated where each of them represents the value for the parameters of the ANFIS model. Then the quality of each solution is calculated using the fitness value, and the solution that has the best fitness value is chosen to represent the best solution. Then the probability of each solution is computed. Then the current solution will be updated, either using global or local strategy in FPA. However, in the case of local strategy, the operators of SSA or FPA will be used according to the probability of the fitness value for each solution. The process of updating the solutions is repeated until reaching the stop condition, and the best parameter configurations are used to forecast the number of confirmed cases of COVID-19.The main contribution points of the current study are as follows:1.We propose an efficient forecasting model to forecast the confirmed cases of the COVID-19 in China for the upcoming ten days based on previously confirmed cases.An improved ANFIS model is proposed using a modified FPA algorithm, using SSA.We compare the proposed model with the original ANFIS and existing modified ANFIS models, such as PSO, GA, ABC, and FPA.The rest of this study is organized as follows. The preliminaries of ANFIS, FPA, and SSA are described in Section 2. Section 3 presents the proposed FPASSA, and Section 4 presents the experimental setup and results. We conclude this study in Section 5.The principles of the ANFIS are given in this section. The ANFIS model links the fuzzy logic and neural networks . It generates a mapping between the input and output by applying IF-THEN rules (it is also called Takagi-Sugeno inference model). Figure 1 illustrates the ANFIS model where, y and x define the inputs to Layer 1 whereas, O 1i is its output of node i that is computed as follows:where µ denotes the generalized Gaussian membership functions. A i and B i define the membership values of µ. α i and ρ i denote the premise parameters set. The output of Layer 2 (it is also known as the firing strength of a rule) is calculated as follows:Meanwhile, the output of Layer 3 (it is also known as the normalized firing strength) is calculated as follows:The output of Layer 4 (it is also known as an adaptive node) is calculated as follows:where r i , q i , and p i define the consequent parameters of the node i. Layer 5 contains only one node; its output is computed as: Flower Pollination Algorithm is an optimization method proposed by Yang . It simulates the transfer of flowers' pollen by pollinators in nature. This algorithm utilizes the two types of pollination (i.e., self-pollination and cross-pollination). In self-pollination, the pollination occurs with no pollinators, whereas, in cross-pollination, the pollens are moved between different plants. In more detail, the self-pollination can be represented as a local pollination while the cross-pollination can be called global pollination.The global pollination or cross-pollination can be mathematically formed as follows:where x t i defines the pollen i at iteration t. L denotes the pollination's strength or the step size. F * is the target position or best solution. In some cases, insects can fly with different distance steps for a long space; therefore, Levy fly distribution is applied to simulate this movement.where λ = 1.5. Γ(λ) denotes the gamma function. This distribution is available for large steps s > 0. The self-pollination or local pollination can be mathematically formed as follows:where x t i and x k i represent pollens from different flower in the same plant. in the range [0,1] The process of pollination can be done using cross-pollination or self-pollination. Therefore, the random variable p, in the range [0, 1], is used to determine this process.SSA is an optimization technique introduced by . It simulates the Salps' behavior in nature. This behavior is called salp chain. The mathematical model of SSA begins by splinting its population into a leader group and followers group. The leader is the front salp, whereas, the followers are the other salps. The search space is determined in n-dimensions with n variables. Equation (10) works to update the salps' positions.where x 1 j denotes the leader's position in j-th dimension. F j is the target position. ub j and lb j represent the max and min bounds, respectively. c 2 and c 3 denote random numbers in [0, 1]. c 1 is an important parameter; it balances between the exploration and exploitation phases. It is computed as follows:where the current loop number is t and the max loop' number is t max . Then, the followers' position is updated as follows:where x i j defines the i-th position of the follower in j-th dimension. i > 1.This section explains the proposed FPASSA-ANFIS method. It is a time series method for forecasting the confirmed cases of the COVID-19, as given in Figure 2 . The FPASSA-ANFIS utilizes the improved FPA to train the ANFIS model by optimizing its parameters. The FPASSA-ANFIS contains five layers as the classic ANFIS model. Layer 1 contains the input variables (the historical COVID-19 confirmed cases). Whereas Layer 5 produces the forecasted values. In the learning phase, the FPASSA is used to select the best weights between Layer 4 and Layer 5.The FPASSA-ANFIS starts by formatting the input data in a time series form. In our case, the autocorrelation function (ACF) was considered. ACF is one of the methods applied to find patterns in the data; it presents information about the correlation between points separated by various time lags. Therefore, in this paper, the variables with ACF greater than 0.2 are considered i.e., 5-lags.Besides, the training data contains 75% of the dataset, whereas the testing data contains 25% of them. The number of clusters is defined by the fuzzy c-mean (FCM) method to construct the ANFIS model.The parameters of the ANFIS model are prepared by the FPASSA algorithm. In the training phase, the calculation error (as in Equation (13)) between the real data and the predicted data is used to evaluate the parameters' quality.where T is the real data, and P is the predicted data. N s is the sample length. The smaller values of the objective function indicate good ANFIS's parameter.On the other hand, the updating phase of the followers' positions in the SSA algorithm is applied to improve the global pollination phase in the FPA algorithm. In this improvement, there is a random variable (r) used to switch between both phases. If r > 0.5, then the operators of the SSA is used; otherwise, the operators of the FPA are used. In general, The FPASSA starts by constructing the population (X); afterward, the objective function is calculated for each solution. The solution with the lowest error value is saved to the next iteration. This sequence is repeated until meeting the stop condition, which in this paper, is the maximum number of iterations. Then the best solution is passed to train the parameters of the ANFIS model.After finishing the training phase, the testing phase is started with the best solution to compute the final output. The performance of the proposed method is evaluated by comparing the real data with the predicted data using the performance measures. Finally, the FPASSA produces a foretasted value for confirmed cases of COVID-19 in China in the next day. The steps of the proposed FPASSA are presented in Algorithm 1.Input: Historical COVID-19 dataset, size of population N, total number of iterations t max .Divide the data into training and testing sets.Using Fuzzy c-mean method to determine the number of membership functions.Constructing the ANFIS network.Set the initial value for N solutions (X). Return the best solution that represents the best configuration for ANFIS.Apply the testing set to the best ANFIS model.Forecasting the COVID-19 for the next ten days.This section presents the description of the used dataset, the performance measures, the parameter setting for all methods, the experiment results, and discussions.The main dataset of this study is COVID-19 dataset. It was collected from the WHO website (https: //). It contains the daily confirmed cases in China from 21 January 2020 to 18 February 2020, as shown in Table 1 . We used 75% from the dataset to train the model while the rest is used to test it.Moreover, we evaluated the performance of the proposed method using two datasets of weekly influenza confirmed cases. The first one is called DS1; it was collected from the Centers for Disease Control and Prevention (CDC) (). It starts from week number 40 in 2015 and continues until week number 6 in 2020. Whereas, the second one is called DS2. It was collected from the WHO website (). It contains the data of weekly influenza confirmed cases in China from week number 1 in 2016 to week number 8 in 2020. The quality of the proposed method is evaluated using a set of performance metrics as follows:• Root Mean Square Error (RMSE):where Yp and Y are the predicted and original values, respectively. • Mean Absolute Error (MAE):• Mean Absolute Percentage Error (MAPE):• Root Mean Squared Relative Error (RMSRE):N s represents the sample size of the data. • Coefficient of Determination (R 2 ):where Y represents the average of Y.The lowest value of RMSE, MAE, MAPE, and RMSRE refers to the best method. The higher value of R 2 indicates better correlation for the method.This paper aims to assess the ability of the FPASSA to forecast the COVID-19 by comparing its performance with other methods, namely the ANFIS and the trained ANFIS models using PSO, GA, ABC, FPA, and FPASSA. The parameters' setting for these models is listed in Table 2 .The common parameters, such as population size, are set to 25 and 100 iterations are applied. Besides, each algorithm is performed for 30 independent runs to fair comparisons. The selected parameters are chosen because they produced good behavior in previous experiments, such as [34, 35, 55, 56] . Table 2 . Parameters' setting.Parameters SettingMax. epochs = 100, Error goal = 0, Initial step = 0.01, Decrease rate = 0.9, Increase rate = 1. In this section, the performance of the proposed FPASSA to predict the DS1 and DS2 is discussed. It can be concluded from Table 3 that the performance of FPASSA outperformed the compared methods in all measures, whereas the FPA is ranked second. The results of DS2 indicate that the FPASSA is ranked first in terms of RMSE, MAPE, R 2 , and the CPU time. Whereas, the PSO is ranked second, followed by the FPA, GA, then ABC. These results denote that the proposed method can optimize the parameters of the ANFIS model effectively and produce good results in terms of the performance measures. Comparison results between the proposed FPASSA and other models to forecast COVID-19 are given in Table 4 . It can be concluded that the FPASSA outperforms other models. For example, by analyzing the results of RMSE, MAE, MAPE, RMSRE, and CPU time(s) it can be observed that the FPASSA achieves the smallest value among the comparison algorithms, and this indicates the high quality of the FPASSA. Meanwhile, the FPA allocates the second rank, which provides better results than the rest of the methods.Moreover, the value of R 2 refers to the high correlation between the prediction obtained by the proposed FPASSA method and the original COVID-19, which has nearly 0.97. This can also be noticed from Figure 3 , which depicts the training of the algorithms using the historical data of the COVID-19 as well as their forecasting values for ten days. Table 5 depicts the forecasting value for the confirmed cases of the COVID-19 in China from 19/2/2020 to 28/2/2020. From these results, it can be noticed that the outbreak will reach its highest level on the day 28/2/2020. The average percentage of the increase over the forecasted period is 10%, the highest percentage is 12% on 28/2/2020, and the lowest percentage is 8.7% on 19/2/2020. From the previous results, it can be concluded that the proposed FPASSA-ANFIS has a high ability to forecast the COVID-19 dataset. These results avoid the limitations of traditional ANFIS because of the combination with the modified FPA method. Moreover, the operators of SSA are combined with the local strategy of FPA to enhance their exploitation ability. However, the time computational of the proposed FPASSA method still requires more improvements.This paper proposed a modified version for the flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). This modified version, called FPASSA, is applied to improve the performance of the ANFIS through determining the optimal value for its parameters. The developed FPASSA-ANFIS model is applied as a forecasting technique for a novel coronavirus, called COVID-19, that was discovered in Wuhan, China at the end of last year and January of the current year. The proposed FPASSA-ANFIS model has a high ability to predict the number of confirmed cases within ten days. Besides, FPASSA-ANFIS outperforms other forecasting models in terms of RMSE, MAE, MAPE, RMSRE, and R 2 . Furthermore, two datasets of weekly influenza confirmed cases in the USA and China were used to evaluate the proposed method, and the evaluation outcomes showed its good performance. According to the promising results obtained by the proposed FPASSA-ANFIS, it can be applied in different forecasting applications.\n",
            "\t0.526\tBackground As the outbreak of coronavirus disease 2019 (COVID-19) progresses, epidemiological data are needed to guide situational awareness and intervention strategies. Here we describe efforts to compile and disseminate epidemiological information on COVID-19 from news media and social networks.As the outbreak of coronavirus disease 2019 (COVID- 19) is rapidly expanding in China and beyond, with the potential to become a worldwide pandemic, 1 real-time analyses of epidemiological data are needed to increase situational awareness and inform interventions. 2 Previously, real-time analyses have shed light on the transmissibility, severity, and natural history of an emerging pathogen in the first few weeks of an outbreak, such as with severe acute respiratory syndrome (SARS), the 2009 influenza pandemic, and Ebola. Analyses of detailed line lists of patients are particularly useful to infer key epidemiological parameters, such as the incubation and infectious periods, and delays between infection and detection, isolation, and reporting of cases. 3, 4 However, official individual patient data rarely become publicly available early on in an outbreak, when the information is most needed.Building on our previous experience collating news reports to monitor transmission of Ebola virus, 7 here we present an effort to compile individual patient information and subnational epidemic curves on COVID-19 from a variety of online resources. Data were made publicly available in real time and were used by the infectious disease modelling community to generate and compare epidemiological estimates relevant to interventions. We describe the data generation process and provide an early analysis of age patterns of COVID-19, case counts across China and inter nationally, and delays between symptom onset, admissions to hospital, and reporting, for cases reported until Jan 31, 2020.In this population-level observational study, we used crowdsourced reports from DXY.cn, a social network for Chinese physicians, health-care professionals, pharmacies, and health-care facilities established in 2000. This online platform is providing real-time coverage of the COVID-19 outbreak in China, obtained by collating and curating reports from news media, government television, and national and provincial health agencies. The information reported includes time-stamped cumulative counts of COVID-19 infections, outbreak maps, and realtime streaming of health authority announcements in Chinese (directly or through state media). 8 Every report is linked to an online source, which can be accessed for more detailed information on individual cases.These are publicly available, de-identified patient data reported directly by public health authorities or by state media. No patient consent was needed and no ethics approval was required.We closely monitored updates on DXY.cn between Jan 20, 2020, and Jan 31, 2020, to extract key information on individual patients in near real-time, and reports of daily case counts. For individual-level patient data, we used descriptions from the original source in Chinese to retrieve age, sex, province of identification, travel history, reporting date, dates of symptom onset and seeking care at a hospital or clinic, and discharge status, when available. Individual-level patient data were formatted into a line-list database for further quantitative analysis. Individual-level patient data were entered from DXY.cn by a native Chinese speaker (KS), who also generated an English summary for each patient. Entries were checked by a second person (JC). Since DXY.cn primarily provides Evidence before this study An outbreak of coronavirus disease 2019 (COVID-19) was recognised in early January, 2020, in Wuhan City, Hubei province, China. The new virus is thought to have originated from an animal-to-human spillover event linked to seafood and live-animal markets. The infection has spread locally in Wuhan and elsewhere in China, despite strict intervention measures implemented in the region where the infection originated on Jan 23, 2020. More than 500 patients infected with COVID-19 outside of mainland China have been reported between Jan 1 and Feb 14, 2020. Although laboratory testing for COVID-19 quickly ramped up in China and elsewhere, information on individual patients remains scarce and official datasets have not been made publicly available. Patient-level information is important to estimate key time-to-delay events (such as the incubation period and interval between symptom onset and visit to a hospital), analyse the age profile of infected patients, reconstruct epidemic curves by onset dates, and infer transmission parameters. We searched PubMed for publications between Jan 1, 1990, and Feb 6, 2020, using combinations of the following terms: (\"coronavirus\" OR \"2019-nCoV\") AND (\"line list\" OR \"case description\" OR \"patient data\") AND (\"digital surveillance\" OR \"social media\" OR \"crowd-sourced data\"). The search retrieved one relevant study on Middle East respiratory syndrome coronavirus that mentioned FluTrackers in their discussion, a website that aggregates epidemiological information on emerging pathogens. However, FluTrackers does not report individual-level data on COVID-19.To our knowledge, this is the first study that uses crowdsourced data from social media sources to monitor the COVID-19 outbreak. We searched DXY.cn, a Chinese health-care-oriented social network that broadcasts information from local and national health authorities, to reconstruct patient-level information on COVID-19 in China. We also queried international media sources and national health agency websites to collate data on international exportations of COVID-19. We describe the demographic characteristics, delays between symptom onset, seeking care at a hospital or clinic, and reporting for 507 patients infected with COVID-19 reported until Jan 31, 2020. The overall cumulative progression of the outbreak is consistent between our line list and an official report published by the Chinese national health authorities on Jan 28, 2020. The estimated incubation period in our data aligns with that of previous work. Our dataset was made available in the public domain on Jan 21, 2020.Crowdsourced line-list data can be reconstructed from social media data, especially when a central resource is available to curate relevant information. Public access to line lists is important so that several teams with different expertise can provide their own insights and interpretations of the data, especially in the early phase of an outbreak when little information is available. Publicly available line lists can also increase transparency. The main issue with the quality of patient-level data obtained during health emergencies is the potential lack of information from locations overwhelmed by the outbreak (in this case, Hubei province and other provinces with weaker health infrastructures). Future studies based on larger samples of patients with COVID-19 could explore in more detail the transmission dynamics of the outbreak in different locations, the effectiveness of interventions, and the demographic factors driving transmission.For an example of an online source see  ncovh5/view/pneumonia information on patients reported in China, we also compiled additional information on internationally exported cases of COVID-19. We obtained data for 21 countries outside of mainland China (Australia, Cambodia, Canada, France, Germany, Hong Kong, India, Italy, Japan, Malaysia, Nepal, Russia, Singapore, South Korea, Sri Lanka, Taiwan, Thailand, United Arab Emirates, the UK, the USA, and Vietnam). We gathered and cross-checked data for infected patients outside of China using several sources, including global news media (Kyodo News, Straits Times, and CNN), official press releases from each country's Ministry of Health, and disease control agencies.In addition to detailed information on individual patients, we reconstructed the daily progression of reported patients in each province of China from Jan 13, until Jan 31, 2020. We used the daily outbreak situation reports com municated by provincial health authorities, covered by state television and media, and posted on DXY.cn. All patients in our databases had a laboratory confirmed SARS coronavirus 2 (SARS-CoV-2) infection.Our COVID-19 database was made publicly available as a Google Sheet, disseminated via Twitter on Jan 21, 2020, and posted on the website of Northeastern University, (Boston, MA, USA) on Jan 24, 2020, where it is updated in real time. Data used in this analysis, frozen at Jan 31, 2020, are available online as a spreadsheet.We assessed the age distribution of all patients with COVID-19 by discharge status. We adjusted the age profile of Chinese patients by the population of China. We used 2016 population estimates from the Institute for Health Metrics and Evaluation 9 to calculate the relative risk (RR) of infection with COVID-19 by age group. To calculate the RR, we followed the method used by Lemaitre and colleagues 10 to explore the age profile of influenza, where RR for age group i is defined as where C i is the number of cases in age group i and N i is the population size of age group i.To estimate trends in the strength of case detection and interventions, we analysed delays between symptom onset and visit to a health-care provider, at a hospital or clinic, and from seeking care at a hospital or clinic to reporting, by time period and location. We considered the period before and after Jan 18, 2020, when media attention and awareness of the outbreak became more pronounced. 11 We used non-parametric tests to assess differences in delays between seeking care at a hospital or clinic and reporting between locations (Wilcoxon test to compare two locations and Kruskall-Wallis test to compare three or more locations).We estimated the duration of the incubation period on the basis of our line list data. We analysed a subset of patients returning from Wuhan who had spent less than a week in Wuhan, to ensure a narrowly defined exposure window. The incubation period was estimated as the midpoint between the time spent in Wuhan and the date of symptom onset.We did all analyses in R (version 3.5.3). We considered p values of less than 0·05 to be significant.The funder had no role in study design, data compilation, data analysis, data interpretation, or writing of the report. All authors had access to the data, and had final responsibility for the decision to submit for publication.Our line list comprised 507 patients reported from Jan 13, to Jan 31, 2020, including 364 (72%) from mainland China and 143 (28%) from outside of China (table) The age distribution of COVID-19 cases was skewed towards older age groups with a median age of 45 years (IQR 33-56) for patients who were alive or who had an unknown outcome at the time of reporting (figure 1). The median age of patients who had died at the time of reporting was 70 years (IQR 65-81). Few patients (13 [3%]) were younger than 15 years. Adjustment for the age demographics of China confirmed a deficit of infections among children, with a RR below 0·5 in patients younger than 15 years (figure 1). The RR measure indicated a sharp increase in the likelihood of reported COVID-19 among people aged 30 years and older.A timeline of cases in our crowdsourced patient line list is shown by date of onset in figure 2 , indicating an acceleration of reported cases by Jan 13, 2020. The outbreak progression based on the crowdsourced patient line list was consistent with the timeline published by China Center for Disease Control and Prevention (CDC) on Jan 28, 2020, 12 which is based on a more comprehensive database of more than 6000 patients with COVID-19. Since Jan 23, 2020, the cumulative number of cases has slowed down in the crowdsourced and China CDC curves (figure 2), which probably reflects the delay between disease onset and reporting. The median reporting delay was 5 days (IQR 3-8) in our data.Province-level epidemic curves are shown by reporting date in figure 3 . As of Jan 31, 2020, 16 (52%) of 30 provinces in mainland China had reported more than 100 confirmed cases. The apparent rapid growth of newly reported cases between Jan 18, and Jan 31, 2020, in several provinces outside of Hubei province is consistent with sustained local transmission.Across the study period, the median delay between symptom onset and seeking care at a hospital or clinic was 2 days (IQR 0-5 days) in mainland China ( figure 4 ). This delay decreased from 5 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p=0·0009). Some provinces, such as Tianjin and Yunnan had shorter delays (data by province not shown), while the early cases from Hubei province were characterised by longer delays in seeking care (median 0 days [IQR 0-1]).The median delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 2-5 days) in mainland China and decreased from 9 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p<0·0001; figure 4 ). Similarly to delays in seeking care at a hospital or clinic, reporting was quickest in Tianjin and Yunnan (median 1 day [IQR 0-1]) and slowest in Hubei province (median 12 days [IQR 7-16]).The median delay between symptom onset and seeking care at a hospital or clinic was 1 day (IQR 0-3) for international travellers, and shorter than for patients in Hubei province or the rest of mainland China (Kruskal-Wallis test p<0·0001; figure 4) . Even in the period after Jan 18, 2020, when awareness of the outbreak increased, a shorter delay between symptom onset and seeking care at a hospital or clinic was seen for international patients than for those in mainland China (Wilcoxon test p<0·0001). For international cases, the delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 1-4), also shorter than for mainland China (Wilcoxon test p<0·0001; figure 4) .On the basis of 33 patients with a travel history to Wuhan, we estimated the median incubation period for COVID-19 to be 4·5 days (IQR 3·0-5·5; appendix p 2). Information from patient line lists is crucial but difficult to obtain at the beginning of an outbreak. Here we have shown that careful compilation of crowdsourced reports curated by a long-standing Chinese medical social network provides a valuable picture of the outbreak of COVID-19 in real time. The outbreak timeline is consistent with aggregated case counts provided by health authorities. For comparison, China CDC published the first epidemic curve by symptom onset on Jan 28, 2020. 12 Line lists provide unique information on the delays between symptom onset and detection by the health-care system, reporting delays, and travel histories. This information cannot be extracted from aggregated case counts published by official sources. Line list data can help assess the effectiveness of interventions and the potential for widespread transmission beyond the initial foci of infection. In particular, shorter delays between symptom onset and admission to hospital or seeking care in a hospital or clinic accelerate detection and isolation of cases, effectively shortening the infectious period. A useful feature of our crowdsourced database was the availability of travel histories for patients returning from Wuhan, which, along with dates of symptom onset, allowed for estimation of the incubation period here and in related work. 13, 14 A narrow window of exposure could be defined for a subset of patients who had a short stay in Wuhan, at a time when the epidemic was still localised to Wuhan. Several teams have used our dataset and datasets from others to estimate a mean incubation period for COVID-19 to be 5-6 days (95% CI 2-11). Our own estimate (median 4·5 days [IQR 3·0-5·5]) is consistent with previous work that used other modelling approaches. The incubation period is a useful parameter to guide isolation and contact tracing; based on existing data, the disease status of a contact should be known with near certainty after a period of observation of 14 days. 13 Availability of a public dataset enables independent estimation of important epidemiological parameters by several teams, allowing for confirmation and cross-checking at a time when information can be conflicting and noisy.An interesting finding in our data relates to the age distribution of patients. We found a heavy skew of infection towards older age groups, with substantially fewer children infected. This pattern could indicate agerelated differences in susceptibility to infection, severe outcomes, or behaviour. However, a substantial portion of the patients in our database are travellers, a population that is usually predominantly adults (although does not exclude children). Furthermore, because patient data in our dataset were captured by the health system, they are biased towards the more severe spectrum of the disease, especially for patients from mainland China. Clinical reports have shown that severity of COVID-19 is associated with the presence of chronic conditions, 16, 17 which are more frequent in older age groups. Nevertheless, we would also expect children younger than 5 years to be at risk of severe outcomes and to be reported to the healthcare system, as is seen for other respiratory infections. 18 Biological differences could have a role in shaping these age profiles. A detailed analysis of one of the early COVID-19 clusters by Chan and colleagues 19 revealed symptomatic infections in five adult members of the same household, while a child in the same household aged 10 years was infected but remained asymptomatic, potentially indicating biological differences in the risk of clinical disease driven by age. Previous immunity from infection with a related coronavirus has been speculated to potentially protect children from SARS, 20, 21 and so might also have a role in COVID-19. In any case, if the age distribution of cases reported here was to be confirmed and the epidemic were to progress globally, we would expect an increase in respiratory mortality concentrated among people aged 30 years and older. This mortality pattern would be substantially different from the profile of the 2009 influenza pandemic, for which excess mortality was concentrated in those younger than 65 years. 21 In our dataset, we saw a rapid increase in the number of people infected with COVID-19 in several provinces of China, consistent with local transmission outside of Hubei province. As of Jan 31, 2020, province-level epidemic curves are only available by date of reporting, rather than date of symptom onset, which usually inflates recent case counts if detection has increased. D e c 9 , 2 0 1 9 D e c 1 6 , 2 0 1 9 D e c 2 3 , 2 0 1 9 D e c 3 0 , 2 0 1 9 J a n 6 , 2 0 2 0 J a n 1 3 , 2 0 2 0 J a n 2 0 , 2 0 2 0 J a n 2 7 , 2 0 2 0 Furthermore, province-level data include both returning travellers from Hubei province (ie, importations) and locally acquired cases, which also usually inflate the apparent risk of local transmission. Notably, other lines of evidence suggest that local transmission is now well established outside of Hubei province, because travel increased just before the Chinese New Year on Jan 25, 2020, and before implementation of the travel ban in Wuhan. 22 Accordingly, our own data include evidence of transmission clusters in non-travellers, with, for instance, a second-generation transmission event reported in Shaanxi on Jan 21, 2020.Our study had several limitations, one of which was the data we used. Although all provinces in mainland China provide aggregated information on infections and deaths, individual-level patient descriptions are only available for a subset of provinces. Geographical coverage is heterogeneous in our line list, and we have a notable deficit of cases from Hubei province, the foci of the COVID-19 outbreak. We expect that little patient-level information is shared on social media by province-level and city-level health authorities in Wuhan and Hubei province because health systems are overwhelmed. For similar reasons, provinces with a large total case count at the end of January, 2020, or with a weaker health infrastructure, were under-represented in our line list, with the exception of Beijing. Other limitations in our data include severity (only patients who had severe enough symptoms to seek care were captured) and changes in case definition. A series of epidemiological criteria were required for COVID-19 testing, including travel history to Wuhan within the past 2 weeks; residence in Wuhan within the past 2 weeks; contact with individuals from Wuhan (with fever and respiratory symptoms) within the past 2 weeks; and being part of an established disease cluster. Some of these criteria (eg, relation to Wuhan) were relaxed over time (appendix). As a result, we have an overrepresentation of travel-related cases in our database.The reproduction number is an important quantity for outbreak control. We refrained from estimating this parameter because reporting changes could bias estimates relying on epidemic growth rates. Furthermore, our dataset captured cases all over China and does not reflect transmission patterns in any particular location. A mean reproduction number of 2·5-2·7 has previously been estimated on the basis of the volume of importations of international cases in the pre-intervention period in Wuhan. 11 We recognise that, although our data source is useful and timely, it should not replace official statistics. Manual compilation of detailed line lists from media sources is highly time consuming and is not sustainable when case counts reach several thousands. Here we provide detailed data on 507 patients when the official case count was over 9000 by Jan 31, 2020, representing a sample of approximately 5% of reported cases and a much smaller proportion of the full spectrum of COVID-19 cases, which include mild infections. A crowd sourced system would not be expected to catch all cases, especially if many cases are too mild to be captured by the health-care system, digital surveillance, or social media. Notably, DXY.cn does not generate data outside of traditional surveillance systems but rather provides a channel of rapid communication between the public and health authorities. In turn, our approach has helped extract and repackage information from health authorities into an analytical format, which was not available elsewhere.At the time of writing, efforts are underway to coordinate compilation of COVID-19 data from online sources across several academic teams. Ultimately, we expect that a line list of patients will be shared by government sources with the global community; however, data cleaning and access issues might take a prohibitively long time to resolve. For the west African Ebola outbreak, a similarly coordinated effort to publish a line list took 2 years. 23 Given the progression of the COVID-19 outbreak, such a long delay would be counterproductive.Overall, the novelty of our approach was to rely on a unique source for social media and news reports in China, which aggregated and curated relevant information. This approach facilitated entry of robust and standard data on clinical and demographic information. Reassuringly, DXY.cn maintains a special section dedicated to debunking fake news, myths, and rumours about the COVID-19 outbreak. Looking to the future, collection of patient data in the context of emergencies could include information on whether patients are identified through contact tracing or because they seek care on their own. Furthermore, data interpretability could be improved by gathering more quantitative information on how case definitions are used in practice.In conclusion, crowdsourced epidemiological data can be useful to monitor emerging outbreaks, such as COVID-19 and, as previously, Ebola virus. 7 These efforts can help generate and disseminate detailed information in the early stages of an outbreak when little other data are available, enabling independent estimation of key parameters that affect interventions. Based on our small sample of patients with COVID-19, we note an intriguing age distribution, reminiscent of that of SARS, which warrants further epidemiological and serological studies. We also report early signs that the response is strengthening in China on the basis of a decrease in case detection time, and rapid management of travel-related infections that are identified internationally. This is an early report of a rapidly evolving situation and the parameters discussed here could change quickly. In the coming weeks, we will continue to monitor the epidemiology of this outbreak using data from news reports and official sources.KS and CV contributed to the study design. KS and JC contributed to the data compilation. KS, JC, and CV contributed to data analysis. KS and JC contributed to the design and drawing of figures. KS, JC, and CV contributed to the writing of the manuscript.We declare no competing interests.All data used in this report have been made publicly available on the Laboratory for the Modeling of Biological + Socio-technical systems website of Northeastern University. The available data include daily case counts of COVID-19 by reporting date and Chinese province, and a de-identified line list of patients with COVID-19. The line list includes geographical location (country and province), reporting date, dates of symptom onset and seeking care at a hospital or clinic, relation to Wuhan, discharge status when known, an English summary of the case description from media sources, and a link to the original source of data. Seeking care at hospital or clinic to report Symptom onset to seeking care at hospital or clinic\n",
            "\t0.508\tA.A.Q); (M.R.M.) † These authors contributed equally to this work.The ongoing outbreak of COVID-19 in the Chinese city of Wuhan (Hubei province) and its alarmingly quick transmission to 25 other countries across the world resulted in the World Health Organization (WHO) declaring a global health emergency on 30 January 2020 . This came just one month after the first reported case on 31 December 2019 . WHO, in its first emergency meeting , estimated the fatality rate of COVID-19 to be around 4%. Worldwide collaborative efforts from scientists are underway to understand the novel and rapidly spreading virus that causes this disease, SARS-CoV-2 (originally tentatively named 2019-nCoV), and to develop effective interventions for controlling and preventing it .Coronaviruses are positive-sense single-stranded RNA viruses belonging to the family Coronaviridae. These viruses mostly infect animals, including birds and mammals. In humans, they generally cause mild respiratory infections, such as those observed in the common cold. However, some recent human coronavirus infections have resulted in lethal endemics, which include the SARS (Severe Acute Respiratory Syndrome) and MERS (Middle East Respiratory Syndrome) endemics. Both of these are caused by zoonotic coronaviruses that belong to the genus Betacoronavirus within Coronaviridae. SARS-CoV originated from Southern China and caused an endemic in 2003. A total of 8098 cases of SARS were reported globally, including 774 associated deaths, and an estimated case-fatality rate of 14%-15% . The first case of MERS occurred in Saudi Arabia in 2012. Since then, a total of 2,494 cases of infection have been reported, including 858 associated deaths, and an estimated high case-fatality rate of 34.4% . While no case of SARS-CoV infection has been reported since 2004, MERS-CoV has been around since 2012 and has caused multiple sporadic outbreaks in different countries.Like SARS-CoV and MERS-CoV, the recent SARS-CoV-2 belongs to the Betacoronavirus genus . It has a genome size of~30 kilobases which, like other coronaviruses, encodes for multiple structural and non-structural proteins. The structural proteins include the spike (S) protein, the envelope (E) protein, the membrane (M) protein, and the nucleocapsid (N) protein. With SARS-CoV-2 being discovered very recently, there is currently a lack of immunological information available about the virus (e.g., information about immunogenic epitopes eliciting antibody or T cell responses). Preliminary studies suggest that SARS-CoV-2 is quite similar to SARS-CoV based on the full-length genome phylogenetic analysis [9, 12] , and the putatively similar cell entry mechanism and human cell receptor usage [9, 13, 14] . Due to this apparent similarity between the two viruses, previous research that has provided an understanding of protective immune responses against SARS-CoV may potentially be leveraged to aid vaccine development for SARS-CoV-2.Various reports related to SARS-CoV suggest a protective role of both humoral and cell-mediated immune responses. For the former case, antibody responses generated against the S protein, the most exposed protein of SARS-CoV, have been shown to protect from infection in mouse models . In addition, multiple studies have shown that antibodies generated against the N protein of SARS-CoV, a highly immunogenic and abundantly expressed protein during infection , were particularly prevalent in SARS-CoV-infected patients [19, 20] . While being effective, the antibody response was found to be short-lived in convalescent SARS-CoV patients . In contrast, T cell responses have been shown to provide long-term protection , even up to 11 years post-infection , and thus have also attracted interest for a prospective vaccine against SARS-CoV [reviewed in ]. Among all SARS-CoV proteins, T cell responses against the structural proteins have been found to be the most immunogenic in peripheral blood mononuclear cells of convalescent SARS-CoV patients as compared to the non-structural proteins . Further, of the structural proteins, T cell responses against the S and N proteins have been reported to be the most dominant and long-lasting .Here, by analyzing available experimentally-determined SARS-CoV-derived B cell epitopes (both linear and discontinuous) and T cell epitopes, we identify and report those that are completely identical and comprise no mutation in the available SARS-CoV-2 sequences (as of 21 February 2020). These epitopes have the potential, therefore, to elicit a cross-reactive/effective response against SARS-CoV-2. We focused particularly on the epitopes in the S and N structural proteins due to their dominant and long-lasting immune response previously reported against SARS-CoV. For the identified T cell epitopes, we additionally incorporated the information about the associated MHC alleles to provide a list of epitopes that seek to maximize population coverage globally, as well as in China. Our presented results can potentially narrow down the search for potent targets for an effective vaccine against SARS-CoV-2, and help guide experimental studies focused on vaccine development.A total of 120 whole genome sequences of SARS-CoV-2 were downloaded on 21 February 2020 from the GISAID database () (Table S1 ). We excluded sequences that likely had spurious mutations resulting from sequencing errors, as indicated in the comment field of the GISAID data. These nucleotide sequences were aligned to the GenBank reference sequence (accession ID: NC_045512.2) and then translated into amino acid residues according to the coding sequence positions provided along the reference sequence for SARS-CoV-2 proteins (orf1a, orf1b, S, ORF3a, E, M, ORF6, ORF7a, ORF7b, ORF8, N, and ORF10). These sequences were aligned separately for each protein using the MAFFT multiple sequence alignment program . Reference protein sequences for SARS-CoV and MERS-CoV were obtained following the same procedure from GenBank using the accession IDs NC_004718.3 and NC_019843.3, respectively.SARS-CoV-derived B cell and T cell epitopes were searched on the NIAID Virus Pathogen Database and Analysis Resource (ViPR) (; accessed 21 February 2020) by querying for the virus species name: \"Severe acute respiratory syndrome-related coronavirus\" from \"human\" hosts. We limited our search to include only the experimentally-determined epitopes that were associated with at least one positive assay: (i) Positive B cell assays (e.g., enzyme-linked immunosorbent assay (ELISA)-based qualitative binding) for B cell epitopes; and (ii) either positive T cell assays (such as enzyme-linked immune absorbent spot (ELISPOT) or intracellular cytokine staining (ICS) IFN-γ release), or positive major histocompatibility complex (MHC) binding assays for T cell epitopes. Strictly speaking, the latter set of epitopes, determined using positive MHC binding assays, are antigens which are candidate epitopes, since a T cell response has not been confirmed experimentally. However, for brevity and to be consistent with the terminology used in the ViPR database, we will not make this qualification, and will simply refer to them as epitopes in this study. The number of B cell and T cell epitopes obtained from the database following the above procedure is listed in Table 1 . Population coverages for sets of T cell epitopes were computed using the tool provided by the Immune Epitope Database (IEDB) (; accessed 21 February 2020) . This tool uses the distribution of MHC alleles (with at least 4-digit resolution, e.g., A*02:01) within a defined population (obtained from ) to estimate the population coverage for a set of T cell epitopes. The estimated population coverage represents the percentage of individuals within the population that are likely to elicit an immune response to at least one T cell epitope from the set. To identify the set of epitopes associated with MHC alleles that would maximize the population coverage, we adopted a greedy approach: (i) We first identified the MHC allele with the highest individual population coverage and initialized the set with their associated epitopes, then (ii) we progressively added epitopes associated with other MHC alleles that resulted in the largest increase of the accumulated population coverage. We stopped when no increase in the accumulated population coverage was observed by adding epitopes associated with any of the remaining MHC alleles.We used the publicly available software PASTA v1.6.4 to construct a maximum-likelihood phylogenetic tree of each structural protein using the unique set of sequences in the available data of SARS-CoV, MERS-CoV, and SARS-CoV-2. We additionally included the Zaria Bat coronavirus strain (accession ID: HQ166910.1) to serve as an outgroup. The appropriate parameters for tree estimation are automatically selected in the software based on the provided sequence data. For visualizing the constructed phylogenetic trees, we used the publicly available software Dendroscope v3.6.3 . Each constructed tree was rooted with the outgroup Zaria Bat coronavirus strain, and circular phylogram layout was used.All sequence and immunological data, and all scripts (written in R) for reproducing the results are available online .SARS-CoV-2 has been observed to be close to SARS-CoV-much more so than MERS-CoV-based on full-length genome phylogenetic analysis [9, 12] . We checked whether this is also true at the level of the individual structural proteins (S, E, M, and N). A straightforward reference-sequence-based comparison indeed confirmed this, showing that the M, N, and E proteins of SARS-CoV-2 and SARS-CoV have over 90% genetic similarity, while that of the S protein was notably reduced (but still high) ( Figure 1a ). The similarity between SARS-CoV-2 and MERS-CoV, on the other hand, was substantially lower for all proteins ( Figure 1a ); a feature that was also evident from the corresponding phylogenetic trees (Figure 1b) . We note that while the former analysis ( Figure 1a ) was based on the reference sequence of each coronavirus, it is indeed a good representative of the virus population, since few amino acid mutations have been observed in the corresponding sequence data ( Figure S1 ). It is also noteworthy that while MERS-CoV is the more recent coronavirus to have infected humans, and is comparatively more recurrent (causing outbreaks in 2012, 2015, and 2018) (), SARS-CoV-2 is closer to SARS-CoV, which has not been observed since 2004. Percentage sequence identity with SARS-CoV-2 All trees were constructed based on the available unique sequences using PASTA and rooted with the outgroup Zaria Bat CoV strain (accession ID: HQ166910.1).Given the close genetic similarity between the structural proteins of SARS-CoV and SARS-CoV-2, we attempted to leverage immunological studies of the structural proteins of SARS-CoV to potentially aid vaccine development for SARS-CoV-2. We focused specifically on the S and N proteins as these are known to induce potent and long-lived immune responses in SARS-CoV 19, 20, 25, 27] . We used the available SARS-CoV-derived experimentally-determined epitope data (see Materials and Methods) and searched to identify T cell and B cell epitopes that were identical-and hence potentially cross-reactive-across SARS-CoV and SARS-CoV-2. We first report the analysis for T cell epitopes, which have been shown to provide a long-lasting immune response against SARS-CoV , followed by a discussion of B cell epitopes.The SARS-CoV-derived T cell epitopes used in this study were experimentally-determined from two different types of assays : (i) Positive T cell assays, which tested for a T cell response against epitopes, and (ii) positive MHC binding assays, which tested for epitope-MHC binding. We aligned these T cell epitopes across the SARS-CoV-2 protein sequences. Among the 115 T cell epitopes that were determined by positive T cell assays (Table 1) , we found that 27 epitope-sequences were identical within SARS-CoV-2 proteins and comprised no mutation in the available SARS-CoV-2 sequences (as of 21 February 2020) ( Table 2) . Interestingly, all of these were present in either the N (16) or S (11) protein. MHC binding assays were performed for 19 of these 27 epitopes, and these were reported to be associated with only five distinct MHC alleles (at 4-digit resolution): HLA-A*02:01, HLA-B*40:01, HLA-DRA*01:01, HLA-DRB1*07:01, and HLA-DRB1*04:01. Consequently, the accumulated population coverage of these epitopes (see Materials and Methods for details) is estimated to not be high for the global population (59.76%), and was quite low for China (32.36%). For the remaining 8 epitopes, since the associated MHC alleles are unknown, they could not be used in the population coverage computation. Additional MHC binding tests to identify the MHC alleles that bind to these 8 epitopes may reveal additional distinct alleles, beyond the five determined so far, that may help to improve population coverage. To further expand the search and identify potentially effective T cell targets covering a higher percentage of the population, we next additionally considered the set of T cell epitopes that have been experimentally-determined from positive MHC binding assays (Table 1) , but, unlike the previous epitope set, their ability to induce a T cell response against SARS-CoV was not experimentally determined. Nonetheless, they also present promising candidates for inducing a response against SARS-CoV-2. For the expanded set of epitopes, all of which have at least one positive MHC binding assay, we found that 229 epitope-sequences have an identical match in SARS-CoV-2 proteins and have associated MHC allele information available (listed in Table S2 ). Of these 229 epitopes,~82% were MHC Class I restricted epitopes (Table S3) . Importantly, 102 of the 229 epitopes were derived from either the S (66) or N (36) protein. Mapping all 66 S-derived epitopes onto the resolved crystal structure of the SARS-CoV S protein ( Figure S2 ) revealed that 3 of these (GYQPYRVVVL, QPYRVVVLSF, and PYRVVVLSF) were located entirely in the SARS-CoV receptor-binding motif (), known to be important for virus cell entry .Similar to previous studies on HIV and HCV , we estimated population coverages for various combinations of MHC alleles associated with these 102 epitopes. Our aim was to determine sets of epitopes associated with MHC alleles with maximum population coverage, potentially aiding the development of vaccines against SARS-CoV-2. For selection, we adopted a greedy computational approach (see Materials and Methods), which identified a set of T cell epitopes estimated to maximize global population coverage. This set comprised of multiple T cell epitopes associated with 20 distinct MHC alleles and was estimated to provide an accumulated population coverage of 96.29% (Table 3) . Interestingly, the majority of the T cell epitopes for which a positive immune response has been determined using T cell assays (Table 2) were presented by the globally most-prevalent MHC allele (shown in blue color in Table 3 ). Moreover, the functionally important epitopes located in the SARS-CoV receptor binding motif were associated with the second and third most-prevalent MHC alleles (underlined in Table 3 ). Thus, while the ordering of T cell epitopes in Table 3 is based on the estimated global population coverage of the associated MHC alleles, it is also a natural order in which these epitopes should be tested experimentally for determining their potential to induce a positive immune response against SARS-CoV-2. We also computed the population coverage of this specific set of epitopes in China, the country most affected by the COVID-19 outbreak, which was estimated to be slightly lower (88.11%), as certain MHC alleles (e.g., HLA-A*02:01) associated with some of these epitopes are less frequent in the Chinese population (Table 3) . Repeating the same greedy approach but focusing on the Chinese population, instead of a global population, the maximum population coverage was estimated to be 92.76% (Table S4) . Table 3 . Set of the SARS-CoV-derived spike (S) and nucleocapsid (N) protein T cell epitopes (obtained from positive MHC binding assays) that are identical in SARS-CoV-2 and that maximize estimated population coverage globally (87 distinct epitopes). Due to the promiscuous nature of binding between peptides and MHC alleles, multiple S and N peptides were reported to bind to individual MHC alleles. Thus, while we list all the S and N epitopes that bind to each MHC allele (Table 3) , the estimated maximum population coverage may be achieved by selecting at least one epitope for each listed MHC allele. Likewise, many individual S and N epitopes were found to be presented by multiple alleles and thereby estimated to have varying global population coverage (listed in Table S5 ).Similar to T cell epitopes, we used in our study the SARS-CoV-derived B cell epitopes that have been experimentally-determined from positive B cell assays . These epitopes were classified as: (i) Linear B cell epitopes (antigenic peptides), and (ii) discontinuous B cell epitopes (conformational epitopes with resolved structural determinants).We aligned the 298 linear B cell epitopes (Table 1) across the SARS-CoV-2 proteins and found that 49 epitope-sequences, all derived from structural proteins, have an identical match and comprised no mutation in the available SARS-CoV-2 protein sequences (as of 21 February 2020). Interestingly, a large number (45) of these were derived from either the S (23) or N (22) protein (Table 4) , while the remaining (4) were from the M protein (Table S6) . Table 4 . SARS-CoV-derived linear B cell epitopes from S (23; 20 of which are located in subunit S2) and N (22) proteins that are identical in SARS-CoV-2 (45 epitopes in total). On the other hand, all 6 SARS-CoV-derived discontinuous B cell epitopes obtained from the ViPR database (Table 5) were derived from the S protein. Based on the pairwise alignment between the SARS-CoV and SARS-CoV-2 reference sequences ( Figure S3 ), we found that none of these mapped identically to the SARS-CoV-2 S protein, in contrast to the linear epitopes. For 3 of these discontinuous B cell epitopes (corresponding to antibodies S230, m396, and 80R ), there was a partial mapping, with at least one site having an identical residue at the corresponding site in the SARS-CoV-2 S protein (Table 5) . Table 5 . SARS-CoV-derived discontinuous B cell epitopes (and associated known antibodies ) that have at least one site with an identical amino acid to the corresponding site in SARS-CoV-2. Residues 1,2 910052 S230 G446, P462, D463, Y475 77444 m396 T359, T363, K365, K390, G391, D392, R395, R426, Y436, G482, Y484, T485, T486, T487, G488, I489, G490, Y491, Q492, Y494 77442 80R R426, S432, T433, Y436, N437, K439, Y440, Y442, P469, P470, A471, L472, N473, C474, Y475, W476, L478, N479, D480, Y481, G482, Y484, T485, T486, T487, G488, I489, Y491, Q492 1 Residues are numbered according to the SARS-CoV S protein reference sequence, accession ID: NP_828851.1.; 2 Residues in the epitopes that are identical in the SARS-CoV-2 sequences are underlined.Mapping the residues of the linear and discontinuous B cell epitopes onto the available structure of the SARS-CoV S protein revealed their distinct association with the two functional subunits of the S protein : S1, important for interaction with the host cell receptor, and S2, involved in fusion of the cellular and virus membranes (Figure 2a) . Specifically, 20 of the 23 linear epitopes (Table 4 ) mapped to S2 (Figure 2b) . Thus, the antibodies targeting the identified linear epitopes in the S2 subunit might cross-react and neutralize both SARS-CoV and SARS-CoV-2, as suggested in a very recent study .While S2 is comparatively less exposed than S1, it may be accessible to antibodies during the complex conformational changes involved in viral entry of coronaviruses ; though this remains to be more clearly understood. In contrast, the 3 discontinuous B cell epitopes (Table 5 ) mapped onto the more exposed S1 subunit (Figure 2c, left panel) , which contains the receptor-binding motif of the SARS-CoV S protein . We observed that very few residues of the 3 discontinuous epitopes were identical within SARS-CoV and SARS-CoV-2 (Figure 2c, right panel) . These differences suggest that the SARS-CoV-specific antibodies S230, m396, and 80R known to bind to these epitopes in SARS-CoV might not be able to bind to the same regions in SARS-CoV-2 S protein. Interestingly, while this paper was under review, this has been confirmed experimentally . Further studies are currently under way to identify other SARS-CoV antibodies that may bind to discontinuous epitopes of the SARS-CoV-2 S protein . (a) Subunits S1 and S2 are indicated in purple and green color, respectively. The receptor binding motif lies within the S1 subunit and is indicated in orange color.(b) Residues of the linear B cell epitopes, that were identical in SARS-CoV-2 (Table 4) , are shown in red color. The dark and light shade reflect the surface and buried residues, respectively. (c) Location of discontinuous B cell epitopes that share at least one identical residue with corresponding SARS-CoV-2 sites (Table 5 ). Identical epitope residues are shown in red color, while the remaining epitope residues are shown in blue color. Both the side view (left panel) and the top view (right panel) of the structure are shown.On the T cell side, the identification of SARS-CoV-derived epitopes that map identically to SARS-CoV-2, and the large population that these are expected to cover, is particularly encouraging. It promotes further research in exploring vaccines designed to induce a protective T cell response, which has been shown to provide long term protection in SARS-CoV . On the B cell side, in agreement with very recent experimental studies [47, 48] , our results suggest that SARS-CoV-derived Figure 2 . Location of SARS-CoV S protein subunits and SARS-CoV-derived B cell epitopes on the protein structure (PDB ID: 5XLR). (a) Subunits S1 and S2 are indicated in purple and green color, respectively. The receptor binding motif lies within the S1 subunit and is indicated in orange color. (b) Residues of the linear B cell epitopes, that were identical in SARS-CoV-2 (Table 4) , are shown in red color. The dark and light shade reflect the surface and buried residues, respectively. (c) Location of discontinuous B cell epitopes that share at least one identical residue with corresponding SARS-CoV-2 sites (Table 5 ). Identical epitope residues are shown in red color, while the remaining epitope residues are shown in blue color. Both the side view (left panel) and the top view (right panel) of the structure are shown.The quest for a vaccine against the novel SARS-CoV-2 is recognized as an urgent problem. Effective vaccination could indeed play a significant role in curbing the spread of the virus, and help to eliminate it from the human population. However, scientific efforts to address this challenge are only just beginning. Much remains to be learnt about the virus, its biological properties, epidemiology, etc. At this early stage, there is also a lack of information about specific immune responses against SARS-CoV-2, which presents a challenge for vaccine development.This study has sought to assist with the initial phase of vaccine development by providing recommendations of epitopes that may potentially be considered for incorporation in vaccine designs. Despite having limited understanding of how the human immune system responds naturally to SARS-CoV-2, these epitopes are motivated by responses they have recorded in SARS-CoV (or, for the case of T cell epitopes, to at least confer MHC binding), and the fact that they map identically to SARS-CoV-2, based on the available sequence data (as of 21 February 2020). This important observation should not be taken for granted. Despite the apparent similarity between SARS-CoV and SARS-CoV-2, there is still considerable genetic variation between the two, and it is not obvious a-prior if epitopes that elicit an immune response against SARS-CoV are likely to be effective against SARS-CoV-2. We found that only 23% and 16% of known SARS-CoV T cell and B cell epitopes map identically to SARS-CoV-2, respectively, and with no mutation having been observed in these epitopes among the available SARS-CoV-2 sequences (as of 21 February 2020). This provides a strong indication of their potential for eliciting a robust T cell or antibody response in SARS-CoV-2.On the T cell side, the identification of SARS-CoV-derived epitopes that map identically to SARS-CoV-2, and the large population that these are expected to cover, is particularly encouraging. It promotes further research in exploring vaccines designed to induce a protective T cell response, which has been shown to provide long term protection in SARS-CoV . On the B cell side, in agreement with very recent experimental studies [47, 48] , our results suggest that SARS-CoV-derived antibodies targeting the receptor binding motif in the S1 subunit of the SARS-CoV-2 S protein may not be effective, due to the large genetic mismatches observed in known structural epitopes targeting this domain. Linear SARS-CoV-derived B cell epitopes in the S2 subunit may potentially be more promising candidates for inducing a protective antibody response. Numerous of these epitopes, while being less exposed, are found to map identically to SARS-CoV and SARS-CoV-2, and preliminary results are already emerging which suggest their potential in generating cross-reactive and neutralizing antibodies . Hence, vaccine solutions that attempt to induce antibodies that target the S2 linear epitopes may be effective and should be explored further.Research efforts directed towards the design and development of vaccines for SARS-CoV-2 are increasing, and some related analyses are already being reported in distinct, parallel studies. These studies, like our own, are based on leveraging available data and computational methods, and add to recent work focused on computational analysis and design of vaccines for various different viruses (e.g., . A preliminary analysis of linear SARS-CoV-derived B cell epitopes has been reported online on the ViPR database website ( Corona/2019-nCoV-ViPR-report_24JAN2020.pdf). Different from our study, which is focused on the linear and discontinuous SARS-CoV-derived epitopes, that analysis considered linear B cell epitope data for all Betacoronaviruses from human hosts. While only a summary of the results has been provided so far, preventing direct comparison of the individual epitopes, the number of linear B cell epitopes reported to map identically to SARS-CoV-2 is comparable to our findings.A recent study has also predicted T cell epitopes for SARS-CoV-2 that may be presented by a population from the Asia-Pacific region . Again, there are multiple differences to our work. First, the focus of that study was on MHC Class II epitopes, while here we considered both MHC Class I and II epitopes. Interestingly, while we found a few MHC Class II epitopes using our approach (Table S3) , only one of these (HLA-DRB1*01:01) appeared in our identified epitope set (Table 3 ), due to their comparatively low estimated population coverage. Second, computational tools were used to predict MHC Class II epitopes in , while here we analyzed the SARS-CoV-derived epitopes that have been determined experimentally, using either positive T cell or MHC binding assays, and which match identically with the available SARS-CoV-2 sequences (as of 21 February 2020). Thus, our identified epitopes are seemingly a more rational set of potential targets that can assist in the ongoing search for a SARS-CoV-2 vaccine.We acknowledge that this is a preliminary analysis based on the limited sequence data available for SARS-CoV-2 (as of 21 February 2020). As the virus continues to evolve and as more data is collected, it is expected that additional mutations will be observed. Such mutations will not affect our analysis, provided that they occur outside of the identified epitope regions. If mutations do occur within epitope regions, then these epitopes may be further screened in line with the conservative filtering principle that we have employed, thereby producing a more refined epitope set.Further experimental studies (T cell and B cell assays) are required to determine the potential of the identified epitopes to induce a positive immune response against SARS-CoV-2. This would help to further refine the reported epitope set, based on observed immunogenicity; an important consideration for immunogen design.Overall, as the identified set of SARS-CoV epitopes map identically to SARS-CoV-2, they present potentially useful candidates for guiding experimental efforts towards developing vaccines against SARS-CoV-2. More generally, our study further highlights the potential importance of previous experimental and clinical studies of SARS-CoV, and its use in concert with emerging data for SARS-CoV-2, in searching for effective vaccines to combat the COVID-19 epidemic.Supplementary Materials: The following are available online at , Figure S1 : Fraction of mutations in the observed sequences of the structural proteins of the three coronaviruses, Figure S2 : Location of identified T cell epitopes on the SARS-CoV S protein structure (PDB ID: 5XLR), Figure S3 : Pairwise sequence alignment of the reference sequences of the S proteins of SARS-CoV and SARS-CoV-2 (accession ID: NP_828851.1 and YP_009724390.1, respectively), Table S1 : List of GISAID accession IDs for 120 genomic sequences of SARS-CoV-2, Table S2 : List of all SARS-CoV-derived T cell epitopes determined using positive MHC binding assays (with associated MHC allele information available at 4 digit resolution) and found to be identical in SARS-CoV-2, Table S3 : Distribution of all SARS-CoV-derived T cell epitopes obtained using positive MHC binding assays (with associated MHC allele information available at 4 digit resolution) and that are identical in SARS-CoV-2, Table S4 : Set of SARS-CoV-derived S and N protein T cell epitopes (obtained using positive MHC binding assays) that are identical in SARS-CoV-2 and that maximize estimated population coverage in China (86 distinct epitopes), Table S5 : Estimated global and Chinese population coverages for the individual SARS-CoV-derived S or N protein T cell epitopes (obtained using positive MHC binding assays), that are identical in SARS-CoV-2, Table S6 : SARS-CoV-derived linear B cell epitopes, excluding those in S and N proteins, that are identical in SARS-CoV-2, Table S7 \n",
            "\t0.502\tTraveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015) . Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens. Gostic et al. eLife 2020;9:e55570. DOI:  1 of 18 RESEARCH ADVANCE Gostic et al. eLife 2020;9:e55570.As of February 20, 2020, the 2019 novel coronavirus (now named SARS-CoV-2, causing the disease COVID-19) has caused over 75,000 confirmed cases inside of China and has spread to 25 other countries (World Health Organization, 2020b) . (HCoV-19 has been proposed as an alternate name for the virus; Jiang et al., 2020) . Until now, local transmission remained limited outside of China, but as of this week, new epidemic hotspots have become apparent on multiple continents (World Health Organization, 2020a; Jankowicz, 2020; Sang-Hun, 2020; Schnirring, 2020a) . Many jurisdictions have imposed traveller screening in an effort to prevent importation of COVID-19 cases to unaffected areas. Some high-income countries have escalated control measures beyond screening-based containment policies, and now restrict or quarantine inbound travellers from countries known to be experiencing substantial community transmission. Meanwhile, in many other countries, screening remains the primary barrier to case importation (Guardian reporting team, 2020; Schengen Visa Info, 2020) . Even in countries with the resources to enforce quarantine measures, expanded arrival screening may be the first logical response as the source epidemic expands to regions outside China. Furthermore, symptom screening has become a ubiquitous tool in the effort to contain local spread of COVID-19, in settings from affected cities to cruise ships to quarantines. Our analysis is pertinent to all of these contexts.It is widely recognized that screening is an imperfect barrier to spread (Bitar et al., 2009; Cowling et al., 2010; Gostic et al., 2015; Mabey et al., 2014; Quilty et al., 2020) , due to: the absence of detectable symptoms during the incubation period; variation in the severity and detectability of symptoms once the disease begins to progress; imperfect performance of screening equipment or personnel; or active evasion of screening by travellers. Previously we estimated the effectiveness of traveller screening for a range of pathogens that have emerged in the past, and found that arrival screening would miss 50-75% of infected cases even under optimistic assumptions (Gostic et al., 2015) . Yet the quantitative performance of different policies matters for planning interventions and will influence how public health authorities prioritize different measures as the international and domestic context changes. Here we use a mathematical model to analyse the expected performance of different screening measures for , based on what is currently known about its natural history and epidemiology and on different possible combinations of departure and arrival screening policies.First we assess the probability that any single individual infected with SARS-CoV-2 would be detected by screening, as a function of time since exposure. This individual-level analysis is not a comprehensive measure of program success, but serves to illustrate the various ways in which screening can succeed or fail (and in turn the ways it can or cannot be improved). Further, these analyses emphasize the importance of the incubation period, and the fraction of subclinical cases, as determinants of individual screening outcomes. We define subclinical cases as those too mild to show symptoms detectable in screening (fever or cough) after passing through the incubation period (i.e. once any symptoms have manifested). The true fraction of subclinical COVID-19 cases remains unknown, but anecdotally, many lab-confirmed COVID-19 cases have not shown detectable symptoms on diagnosis (Hoehl et al., 2020; Nishiura et al., 2020; . About 80% of clinically attended cases have been mild (The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020), and clinically attended cases have been conspicuously rare in children and teens The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Huang et al., 2020; , raising the possibility that subclinical cases may be common.Next, we assess the overall effectiveness of a screening program by modeling screening outcomes in a hypothetical population of infected travellers, each with a different time since exposure (and hence a different probability of having progressed through incubation to show detectable symptoms). Crucially, the distribution of times since exposure will depend on the epidemiology of the source population, so this overall measure is not a simple average of the individual-level outcomes. We estimate the fraction of infected travellers detected, breaking down the ways in which screening can succeed or fail. An alternate measure of program success is the extent to which screening delays the first importation of cases into the community, possibly providing additional time to train medical staff, deploy public health responders or refine travel policies (Cowling et al., 2010) . To quantify the potential for screening to delay case importation, we estimate the probability that a given screening program would detect the first n or more imported cases before missing an infected person.Screening will be less effective in a growing epidemic, due to an excess of recently-exposed and not-yet-symptomatic travellers (Gostic et al., 2015) . In the context of COVID-19, we consider both growing and stable epidemic scenarios, but place greater emphasis on the realistic assumption that the COVID-19 epidemic is still growing. Since late January 2020, the Chinese government has imposed strict travel restrictions and surveillance on population centers heavily affected by COVID-19 (BBC News, 2020; Cellan-Jones, 2020) , and numerous other countries have imposed travel and quarantine restrictions on travellers inbound from China. Until about Feb. 20, 2020, these measures had appeared to successfully limit community transmission outside of China, but all the while multiple factors pointed to on-going risk, including evidence that transmission is possible prior to the onset of symptoms , and reports of citizens seeking to elude travel restrictions or leaving before restrictions were in place (Ma and Pinghui, 2020; Mahbubani, 2020) . Now, in the week following Feb. 20, 2020, new source epidemics have appeared on multiple continents (World Health Organization, 2020a) , and the the risk of exportation of cases from beyond the initial travel-restricted area is growing.As the epidemic continues to expand geographically, arrival screening will likely be continued or expanded to prevent importation of cases to areas without established spread. At the same time, there is great concern about potential public health consequences if COVID-19 spreads to developing countries that lack health infrastructure and resources to combat it effectively (de Salazar et al., 2020) . Limited resources also could mean that some countries cannot implement large-scale arrival screening. In this scenario, departure screening implemented elsewhere would be the sole barrierhowever leaky -to new waves of case importation. It is also important to recognize that, owing to the lag time in appearance of symptoms in imported cases, any weaknesses in screening would continue to have an effect on known case importations for up to two weeks, officially considered the maximum incubation period (World Health Organization, 2020c) . Accordingly, we consider scenarios with departure screening only, arrival screening only, or both departure and arrival screening. The model can also consider the consequences when only a fraction of the traveller population is screened, due either to travel from a location not subject to screening, or due to deliberate evasion of screening.Our analysis also has direct bearing on other contexts where symptom screening is being used, beyond international air travel. This includes screening of travelers at rail stations and roadside spot checks, and screening of other at-risk people including people living in affected areas, health-care workers, cruise ship passengers, evacuees and people undergoing quarantine (Hoehl et al., 2020; Japan Ministry of Health, Labor and Welfare, 2020; Nishiura et al., 2020; Schnirring, 2020c) . Below, we chiefly frame our findings in terms of travel screening, but these other screening contexts are also in the scope of our analysis. Any one-off screening effort is equivalent to a departure screen (i.e. a single test with no delay), and our findings on symptom screening effectiveness over the course of infection are directly applicable to longitudinal screening in quarantine or occupational settings.The central aim of our analysis is to assess the expected effectiveness of screening for COVID-19, taking account of current knowledge and uncertainties about the natural history and epidemiology of the virus. We therefore show results using the best estimates currently available, in the hope of informing policy decisions in this fast-changing environment. We also make our model available for public use as a user-friendly online app, so that stakeholders can explore scenarios of particular interest, and results can be updated rapidly as our knowledge of this new viral threat continues to expand.The core model has been described previously (Gostic et al., 2015) , but to summarize briefly, it assumes infected travellers can be detained due to the presence of detectable symptoms (fever or cough), or due to self-reporting of exposure risk via questionnaires or interviews. These assumptions are consistent with WHO traveller screening guidelines (World Health Organization, 2020b; World Health Organization, 2020c). Upon screening, travellers fall into one of four categories: (1) symptomatic but not aware of exposure risk, (2) aware of exposure risk but without detectable symptoms, (3) symptomatic and aware that exposure may have occurred, and (4) neither symptomatic nor aware of exposure risk ( Figure 1) . Travellers in the final category are fundamentally undetectable, and travellers in the second category are only detectable if aware that they have been exposed and willing to self report.In the model, screening for symptoms occurs prior to questionnaire-based screening for exposure risk, and detected cases do not progress to the next stage. This allows us to track the fraction of cases detected using symptom screening or risk screening at arrival or departure. Additionally, building on the four detectability classes explained above, the model keeps track of four ways in which screening can miss infected travellers: (1) due to imperfect sensitivity, symptom screening may fail to detect symptoms in travellers that display symptoms; (2) questionnaires may fail to detect exposure risk in travellers aware they have been exposed, owing to deliberate obfuscation or misunderstanding; (3) screening may fail to detect both symptoms and known exposure risk in travellers who have both and (4) travellers not exhibiting symptoms and with no knowledge of their exposure are fundamentally undetectable. Here, we only consider infected travellers who submit to screening. However, the supplementary app allows users to consider scenarios in which some fraction of infected travellers intentionally evade screening ( Figure 1E ).The probability that an infected person is detectable in a screening program depends on: the incubation period (the time from exposure to onset of detectable symptoms); the proportion of subclinical cases (mild cases that lack fever or cough); the sensitivity of thermal scanners used to detect fever; the fraction of cases aware they have high exposure risk; and the fraction of those cases who would self-report truthfully on a screening questionnaire. Further, the distribution of individual times since exposure affects the probability that any single infected traveller has progressed to the symptomatic stage. If the source epidemic is still growing, the majority of infected cases will have been recently exposed, and will not yet show symptoms. If the source epidemic is no longer growing (stable), times since exposure will be more evenly distributed, meaning that more infected travellers will have progressed through incubation and will show detectable symptoms.We used methods described previously to estimate the distribution of individual times since exposure in a growing or stable epidemic, given various values of the reproductive number R 0 (Gostic et al., 2015) . Briefly, early in the epidemic when the number of cases is still growing, the model draws on epidemiological theory to assume that the fraction of cases who are recently exposed increases with R 0 . The distribution of times since exposure is truncated at a maximum value, which corresponds epidemiologically to the maximum time from exposure to patient isolation, after which point we assume cases will not attempt to travel. (Isolation may occur due to hospitalization, or due to confinement at home in response to escalating symptoms or COVID-19 diagnosis. In the non-travel context, this would correspond to cases that have been hospitalized or otherwise diagnosed and isolated.) Here, we approximate the maximum time from exposure to isolation as the sum of the mean incubation time, and mean time from onset to isolation. To consider the epidemiological context of a stable epidemic in the source population we assume times since exposure follow a uniform distribution across the time period between exposure and isolation.As of February 20, 2020, COVID-19-specific estimates are available for most parameters, but many have been derived from limited or preliminary data and remain subject to considerable uncertainty. Table 1 and the Methods summarize the current state of knowledge. Here, we used two distinct approaches to incorporate parameter uncertainty into our analysis.First, to estimate the probability that an infected individual would be detected or missed we considered a range of plausible values for the mean incubation time, and the fraction of subclinical Table 1 . Parameter values estimated in currently available studies, along with accompanying uncertainties and assumptions. Ranges in the final column reflect confidence interval, credible interval, standard error or range reported by each study referenced.Best estimate (Used in Figure 2 ) Plausible range (Used in Figure 3 )Mean incubation period 5.5 days Sensitivity: 4.5 or 6.5 Clinical data: 83% fever, 67% cough, n = 6 (Chan et al., 2020) 83% fever, 82% cough, n = 99 98% fever, 76% cough, n = 41 43.8% fever at hospital admission, 88.7% fever during hospitalization, n = 1099 Active monitoring after repatriation flights or on cruise ships: % asymptomatic at diagnosis 31.2% (111/355) (Japan Ministry of Health, Labor and Welfare, 2020) 65.2% (5 of 8) (Nishiura et al., 2020) 70.0% (7 of 10) We assume a low percentage, as no specific risk factors have been identified, and known times or sources of exposure are rarely reported in existing line lists.Sensitivity of infrared thermal scanners for fever 70% 60-90% Most studies estimated sensitivity between 60-88% (Bitar et al., 2009; Priest et al., 2011; Tay et al., 2015) . But a handful of studies estimated very low sensitivity (4-30%).In general, sensitivity depended on the device used, body area targeted and ambient temperature.Probability that travellers self-report exposure risk 25% 5-25% 25% is an upper-bound estimate based on outcomes of past screening initiatives. (Gostic et al., 2015) Time from symptom onset to patient isolation (After which we assume travel is not possible) No effect in individual-level analysis.Median 7 days from onset to hospitalization (n = 6) (Chan et al., 2020) Mean 2.9 days onset to patient isolation (n = 164) Median 7 days from onset to hospitalization (n = 41) As awareness increases, times to isolation may decline.* From family cluster. † Parametric distributions fit to cases with known dates of exposure or travel to and from Wuhan. cases. We focus on the incubation period and subclinical fraction of cases because screening outcomes are particularly sensitive to their values. All other parameters were fixed to the best available estimates listed in Table 1 . Second, we considered a population of infected travellers, each with a unique time of exposure, and in turn a unique probability of having progressed to the symptomatic stage. Here, the model used a resampling-based approach to simultaneously consider uncertainty from both stochasticity in any single individual's screening outcome, and uncertainty as to the true underlying natural history parameters driving the epidemic. Details are provided in the methods, but briefly, we constructed 1000 candidate parameter sets, drawn using Latin hypercube sampling from plausible ranges for each parameter (Table 1) . Using each parameter set, we simulated one set of screening outcomes for a population of 30 infected individuals. As of February 20, 2020, 30 approximates the maximum known number of cases imported to any single country (World Health Organization, 2020b), and thus our analysis incorporates a reasonable degree of binomial uncertainty. The actual number of infected travellers passing through screening in any given location may be higher or lower than 30, and will depend on patterns of global connectivity, and on the duration of the source epidemic (Chinazzi et al., 2020; de Salazar et al., 2020) . Finally, we analysed the sensitivity of screening effectiveness (fraction of travellers detected) to each parameter, as measured by the partial rank correlation coefficient (PRCC) (Marino et al., 2008) . The black dashed lines separate detected cases (below) from missed cases (above). Here, we assume flight duration = 24 hr, the probability that an individual is aware of exposure risk is 0.2, the sensitivity of fever scanners is 0.7, and the probability that an individual will truthfully self-report on risk questionnaires is 0.25. Table 1 Table 1 . Dots and vertical line segments show the median and central 95%, respectively. Text above each violin shows the median and central 95% fraction detected. (B) Mean fraction of travellers with each screening outcome. The black dashed lines separate detected cases (below) from missed cases (above). (C) Fraction of simulations in which screening successfully detects at least n cases before the first infected traveller is missed. Figure 3 continued on next page Individual probabilities of detection First, the model estimated the probability that any single infected individual would be detected by screening as a function of the time between exposure and the initiation of travel (Figure 2) . Incubation time is a crucial driver of traveller screening effectiveness; infected people are most likely to travel before the onset of symptoms. Here we considered three mean incubation times, which together span the range of most existing mean estimates: 4.5, 5.5 and 6.5 days ( Table 1) . Additionally, we considered three possible fractions of subclinical cases: 50% represents a worst-case upper bound, 5% represents a best-case lower bound, and 25% represents a plausible middle case. (Table 1 , Materials and methods). Since resubmission, a new delay-adjusted estimate indicates that 34.6% of infections are asymptomatic (Mizumoto et al., 2020) , intermediate between our middle and worst-case scenarios.Even within the narrow range tested, screening outcomes were sensitive to the incubation period mean. For longer incubation periods, we found that larger proportions of departing travellers would not yet be exhibiting symptoms -either at departure or arrival -which in turn reduced the probability that screening would detect these cases, especially since we assume few infected travellers will realize they have been exposed to COVID-19.A second crucial uncertainty is the proportion of subclinical cases, which lack detectable fever or cough even after the onset of symptoms. We considered scenarios in which 5%, 25% and 50% of cases are subclinical, representing a best, middle and worst-case scenario, respectively. The middle and worst-case scenarios have predictable and discouraging consequences for the effectiveness of traveller screening, since they render large fractions of the population undetectable by fever screening ( Figure 2) . Furthermore, subclinical cases who are unaware of their exposure risk are never detectable, by any means. This is manifested as the bright red 'undetectable' region which persists well beyond the mean incubation period. For a screening program combining departure and arrival screening, as shown in Figure 2 , the greatest contributor to case detection is the departure fever screen. The arrival fever screen is the next greatest contributor, with its value arising from two factors: the potential to detect cases whose symptom onset occurred during travel, and the potential to catch cases missed due to imperfect instrument sensitivity in non-contact infrared thermal scanners used in traveller screening (Table 1) . Considering the effectiveness of departure or arrival screening only (Figure 2-figures supplement 1, 2) , we see that fever screening is the dominant contributor in each case, but that the risk of missing infected travellers due to undetected fever is substantially higher when there is no redundancy from two successive screenings.Overall screening effectiveness in a population of infected travellers during a growing or stable epidemic Next we estimated the overall effectiveness of different screening programs, as well as the uncertainties arising from the current partial state of knowledge about this recently-emerged virus. We modeled plausible population-level outcomes by tracking the fraction of 30 infected travellers detained, given a growing or stable epidemic and current uncertainty around parameter values. We separately consider the best, middle and worst-case scenarios for the proportion of infections that are subclinical, and for each scenario we compare the impact of departure screening only (or equivalently, any on-the-spot screening), arrival screening only, or programs that include both.The striking finding is that in a growing epidemic, even under the best-case assumptions, with just one infection in twenty being subclinical and all travellers passing through departure and arrival screening, the median fraction of infected travellers detected is only 0.30, with 95% interval extending from 0.10 up to 0.53 ( Figure 3A) . The total fraction detected is lower for programs with only Source data 1. Source data for Figure 3A . Source data 2. Source data for Figure 3B . Source data 3. Source data for Figure 3C . one layer of screening, with arrival screening preferable to departure screening owing to the possibility of symptom onset during travel. Considering higher proportions of subclinical cases, the overall effectiveness of screening programs is further degraded, with a median of just one in ten infected travellers detected by departure screening in the worst-case scenario. The key driver of these poor outcomes is that even in the best-case scenario, nearly two thirds of infected travellers will not be detectable (as shown by the red regions in Figure 3B ). There are three drivers of this outcome: (1) in a growing epidemic, the majority of travellers will have been recently infected and hence will not yet have progressed to exhibit any symptoms; (2) we assume that a fraction of cases never develop detectable symptoms; and (3) we assume that few people are aware of their exposure risk. As above, the dominant contributor to successful detections is fever screening.In an epidemic that is no longer growing (Figure 3-figures supplement 1) , screening effectiveness is considerably higher, as a lower proportion of travellers will be recently exposed. This is shown by the smaller, red 'undetectable' region in Figure 3 -figures supplement 1B. In a stable epidemic, under the middle-case assumption that 25% of cases are subclinical, we estimate that arrival screening alone would detect roughly one third (17-53%) of infected travelers, and that a combination of arrival and departure screening would detect nearly half (23-63%) of infected travellers. In short, holding all other things equal, screening effectiveness will increase as the source epidemic transitions from growing to stable, owing simply to changes in the distribution of 'infection ages,' or times since exposure.To assess the potential for screening to delay introduction of undiagnosed cases, we evaluated the fraction of simulations in which screening during a growing epidemic would detect the first n or more infected travellers ( Figure 3C ). Depending on the screening strategy (arrival, departure or both) and assumed subclinical fraction (5%, 25%, or 50%), the probability of detecting at least the first two cases ranged from 0.02 to 0.11, and the probability of detecting three or more cases was never better than 0.04 ( Figure 3C ). In all tested scenarios, more than half of simulations failed to detect the first imported case, consistent with probabilities of case detection in Figure 3A . Probabilities of detecting the first n consecutive cases were marginally higher in the stable epidemic context (Figure 3-figures supplement 1) , but still the probability of detecting at least the first three cases was never better than 0.13, and the probability of detecting the first four cases was never better than 0.06 in any tested scenario. Taken together, these results indicate that screening in any context is very unlikely to delay case importation beyond the first 1-3 cases, and often will not delay the first importation at all. What duration of delay this yields will depend on the frequency of infected travellers.In the context of a growing epidemic, sensitivity analysis using the method of Latin hypercube sampling and partial rank correlation (Marino et al., 2008) showed that the fraction of travellers detected was moderately sensitive to all parameters considered -most coefficient estimates fell between 0.1 and 0.3 in absolute value ( Figure 4 ). Sensitivity to R 0 was somewhat higher than sensitivity to other parameters, but the difference was not statistically remarkable. R 0 and the mean incubation period were negatively associated with the fraction of cases detected. An increase in either of these parameters implies an increase in the probability an infected traveller will be undetectable, either because they have been recently exposed (R 0 ), or have not yet progressed to the symptomatic stage (mean incubation time). The positive association between the fraction of cases detected and the sensitivity of thermal scanners, sensitivity of risk questionnaires, or the fraction of travellers aware of exposure risk is intuitive. Finally, the duration from onset to isolation effectively describes the window of time in which we assume a symptomatic individual could initiate travel. Here, a wider window is associated with increased screening effectiveness, because it will lead to a higher proportion of infected travellers who are symptomatic. Figure 4 shows results from the middle case scenario, in which 25% of cases are subclinical. Considering scenarios where more or fewer cases are subclinical, we see increased influence of the factors based on exposure risk (questionnaire sensitivity and the fraction of cases aware of their exposure) as the proportion of cases with detectable symptoms declines (Figure 4-figures supplement 1) .In the context of a stable epidemic, a greater proportion of infected travellers will have progressed to show detectable symptoms, and so screening effectiveness was more sensitive to parameters that impact symptom screening efficacy (thermal scanner sensitivity, and to the time from symptom onset to isolation). Note that by construction, model outcomes are insensitive to parameter R 0 in the stable epidemic context. As a result, R 0 coefficient estimates are very small (non-zero due to stochasticity in simulation outcomes), and never significant. (Figure 4 -figures supplement 2).We have developed an interactive web application using the R package Shiny (Chang et al., 2019) in which users can replicate our analyses using parameter inputs that reflect the most up-do-date information. The supplementary user interface can be accessed at . Please note that while the results in Figures 3 and 4 consider a range of plausible values for each parameter, the outputs of the Shiny app are calculated using fixed, userspecified values only.The international expansion of COVID-19 cases has led to widespread adoption of symptom and risk screening measures, in travel-associated and other contexts, and programs may still be adopted or expanded as source epidemics of COVID-19 emerge in new geographic areas. Using a mathematical model of screening effectiveness, with preliminary estimates of COVID-19 epidemiology and natural history, we estimate that screening will detect less than half of infected travellers in a growing epidemic, and that screening effectiveness will increase marginally as growth of the source epidemic decelerates. We found that two main factors influenced the effectiveness of screening. First, symptom screening depends on the natural history of an infection: individuals are increasingly likely to show detectable symptoms with increasing time since exposure. A fundamental shortcoming of screening is the difficulty of detecting infected individuals during their incubation period, or early after the onset of symptoms, at which point they still feel healthy enough to undertake normal activities or travel. This difficulty is amplified when the incubation period is longer; infected individuals have a longer window in which they may mix socially or travel with low probability of detection. Second, screening depends on whether exposure risk factors exist that would facilitate specific and reasonably sensitive case detection by questionnaire. For COVID-19, there is so far limited evidence for specific risk factors; we therefore assumed that at most 40% of travellers would be aware of a potential exposure. It is plausible that many individuals aware of a potential exposure would voluntarily avoid travel and practice social distancing-if so, the population of infected travellers will be skewed toward those unaware they have been exposed. Furthermore, based on screening outcomes during the 2009 influenza pandemic, we assumed that a minority of infected travellers would selfreport their exposure honestly, which led to limited effectiveness in questionnaire-based screening. The confluence of these two factors led to many infected people being fundamentally undetectable in our model. Even under our most generous assumptions about the natural history of COVID-19, the presence of undetectable cases made the greatest contribution to screening failure. Correctable failures, such as missing an infected person with fever or awareness of their exposure risk, played a more minor role.Our conclusion that screening would detect no more than half of infected travellers in a growing epidemic is consistent with recent studies that have compared country-specific air travel volumes with detected case counts to estimate that roughly two thirds of imported cases remain undetected Bhatia et al., 2020) . Furthermore, the finding that the majority of cases missed by screening are fundamentally undetectable is consistent with observed outcomes so far. Analyzing a line list of 290 cases imported into various countries , we found that symptom onset occurred after the date of inbound travel for 72% (75/104) of cases for whom both dates were available, and a further 14% (15/104) had symptom onset on the date of travel. Even among passengers of repatriation flights, or quarantined on a cruise ship off the coast of Japan (who are all demonstrably at high risk), numerous cases have been undetectable in symptom screening, but have still tested positive for SARS-CoV-2 by PCR Hoehl et al., 2020; Japan Ministry of Health, Labor and Welfare, 2020; Nishiura et al., 2020; . The onset of viral shedding prior to the onset of symptoms, or in cases that remain asymptomatic, is a classic factor that makes infectious disease outbreaks difficult to control (Fraser et al., 2004) .Our results emphasize that the true fraction of subclinical cases (those who lack fever or cough at symptom onset) remains a crucial unknown, and strongly impacts screening effectiveness. Reviewing data from active surveillance of passengers on cruise ships or repatriation flights, we estimate that up to half of cases show no detectable symptoms at the time of diagnosis. To complicate matters further, the fraction of subclinical cases may vary across age groups. Children and young adults have been conspicuously underrepresented, even in very large clinical data sets The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Huang et al., 2020; . Only 2.1% of the first 44,672 confirmed cases were observed in children under 20 years of age (The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020). The possibility cannot be ruled out that large numbers of subclinical cases are occurring in young people. If an age-by-severity interaction does indeed exist, then the mean age of travellers should be taken into account when estimating screening effectiveness.There are some limitations to our analysis. Parameter values for COVID-19 may be affected by bias or censoring, particularly in the early stages of an outbreak when most cases have been recently infected, and when severe or hospitalized cases are overrepresented in the available data. In particular, the tail of the incubation period distribution is difficult to characterize with precision using limited or biased data.As country-specific screening policies can change rapidly in real-time, we focused on a general screening framework rather than specific case studies. We also assumed traveller adherence and no active evasion of screening. However, there are informal reports of people taking antipyretics to beat fever screening (Mahbubani, 2020) , which would further reduce the effectiveness of these methods. With travel restrictions in place, individuals may also take alternative routes (e.g. road rather than air), which would in effect circumvent departure and/or arrival screening as a control measure. Our quantitative findings may overestimate screening effectiveness if many travellers evade screening.Our results have several implications for the design and implementation of traveller screening policies. If the infection is not yet present in a region, then arrival screening could delay the introduction of cases, but consistent with previous analyses, (Cowling et al., 2010) , our results indicate such delays would be minimal. Our findings indicate that for every case detected by travel screening, one or more infected travellers were not caught, and must be found and isolated by other means. We note that even with high R 0 and no control measures in place, a single case importation is not guaranteed to start a sustained chain of transmission (Kucharski et al., 2020; Lloyd-Smith et al., 2005) . This is particularly true for infections that exhibit a tendency toward superspreading events, as increasingly reported for COVID-19, but the flipside is that outbreaks triggered by superspreading are explosive when they do occur (Lloyd-Smith et al., 2005) .We did not analyze second-order benefits from screening, such as potential to raise awareness. Official recommendations emphasize that screening is an opportunity for 'risk communication' in which travellers can be instructed how to proceed responsibly if symptoms develop at the destination (World Health Organization, 2020d). Alongside increased general surveillance/alertness in healthcare settings, such measures could help reduce the risk of local transmission and superspreading, but their quantitative effectiveness is unknown. Once limited local transmission has begun, arrival screening could still have merit as a means to restrict the number of parallel chains of transmission present in a country. Once there is generalized spread which has outpaced contact tracing, departure screening to prevent export of cases to new areas will be more valuable than arrival screening to identify additional incoming cases. Altogether, screening should not be viewed as a definitive barrier to case importation, but used alongside on-the-ground response strategies that help reduce the probability that any single imported case spreads to cause a self-sustaining local epidemic. The cost-benefit tradeoff for any screening policy should be assessed in light of past experiences, where few or no infected travellers have been detected by such programs (Gostic et al., 2015) .While our findings indicate that the majority of screening failures arise from undetectable cases (i.e. those without symptoms or knowledge of their exposure), several factors could potentially strengthen the screening measures described here. With improved efficiency of thermal scanners or other symptom detection technology, we would expect a smaller difference between the effectiveness of arrival-only screening and combined departure and arrival screening in our analysis. Alternatively, the benefits of redundant screening (noted above for programs with departure and arrival screens) could be gained in a single-site screening program by simply having two successive feverscreening stations that travellers pass through (or taking multiple measurements of each traveller at a single station). As risk factors become better known, questionnaires could be refined to identify more potential cases. Alternatively, less stringent definition of high exposure risk (e.g. contact with anyone with respiratory symptoms) would be more sensitive, but at the expense of large numbers of false positives detained, especially during influenza season.The availability of rapid PCR tests would also be beneficial for case identification at arrival, and would help address concerns with false-positive detections by screening. If such tests were fast, there may be potential to test suspected cases in real time based on questionnaire responses, travel origin, or borderline symptoms; at least one PCR test for SARS-CoV-2 claimed to take less than an hour has already been announced (Biomeme, 2020) . However, such measures could prove highly expensive if implemented at scale. There is also scope for new tools to improve the ongoing tracking of travellers who pass through screening, such as smartphone-based self-reporting of temperature or symptoms in incoming cases . Smartphone or diary-based surveillance would be cheaper and more scalable than intense, on-the-ground follow-up, but is likely to be limited by user adherence.Our analysis underscores the reality that respiratory viruses are difficult to detect by symptom and risk screening programs, particularly if a substantial fraction of infected people show mild or indistinct symptoms, if incubation periods are long, and if transmission is possible before the onset of symptoms. Quantitative estimates of screening effectiveness for COVID-19 will improve as more is learned about this recently-emerged virus, and will vary with the precise design of screening programs. However, we present a robust qualitative finding: in any situation where there is widespread epidemic transmission in source populations from which travellers are drawn, travel screening programs can slow (marginally) but not stop the importation of infected cases. Screening programs implemented in other settings will face the same challenges. By decomposing the factors leading to success or failure of screening efforts, our work supports decision-making about program design, and highlights key questions for further research. We hope that these insights may help to mitigate the global impacts of COVID-19 by guiding effective decision-making in both high-and low-resource countries, and may contribute to prospective improvements in screening policy for future emerging infections.The model's structure is summarized above (Figure 1) , and detailed methods have been described previously (Gostic et al., 2015) . Here, we summarize relevant extensions, assumptions and parameter inputs.Our previous model tracked all the ways in which infected travellers can be detected by screening (fever screen, or risk factor screen at arrival or departure). Here, we additionally keep track of the many ways in which infected travellers can be missed (i.e. missed given fever present, missed given exposure risk present, missed given both present, or missed given undetectable). Cases who have not yet passed the incubation period are considered undetectable by fever screening, even if they will eventually develop symptoms in the future. In other words, no traveller is considered 'missed given fever present' until they have passed the incubation period and show detectable symptoms. Infected travellers who progress to symptoms during their journey are considered undetectable by departure screening, but detectable by arrival screening.Additionally, we now provide a supplementary user interface, which allows stakeholders to test input parameters of interest using up-to-date information. Here, in addition to the analyses presented in this study, we implemented the possibility that some fraction of infected travellers deliberately evade screening.Existing point estimates for R 0 span a wide range (2.2-6.47), but most fall between 2.0 and 4.0 ( Table 1) . The vast majority of these estimates are informed by data collected very early in the outbreak, before any control measures were in place. However, several studies already demostrate decreases in the reproductive number over time, as a consequence of social distancing behaviors, and containment measures (Kucharski et al., 2020; Liu et al., 2020) . Realistically, R 0 will vary considerably over time, and across locations, depending on the social context, resource availability, and containment policies. Our analysis considers a plausible range of R 0 values spanning 1.5-4.0, with 4.0 representing a plausible maximum in the absence of any behavioral changes or containment efforts, and 1.5 reflecting a plausible lower bound, given containment measures may already be in place at the time of introduction.To estimate the upper-bound fraction of subclinical cases, we draw on data from active surveillance of passengers quarantined on a cruise ship off the coast of Japan, or passengers of repatriation flights. These data show that 50-70% of cases are asymptomatic at the time of diagnosis Nishiura et al., 2020; Schnirring, 2020c; Schnirring, 2020b) . We estimate that 50% subclinical cases is a reasonable upper bound: due to intensive monitoring, cases in repatriated individuals or in cruise ship passengers will be detected earlier than usual in the course of infection-and possibly before the onset of symptoms. From clinical data (where severe cases are likely overrepresented), we estimate a lower bound of 5%: even among clinically attended cases, 2-15% lack fever or cough, and would be undetectable in symptom screening (Chan et al., 2020; Chen et al., 2020; The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Huang et al., 2020) . In addition to the upper and lower bound scenarios, we consider a plausible middle-case scenario in which 25% of cases are subclinical. A very recent delay-adjusted estimate indicates 30-40% of infections on the cruise ship quarantined off the coast of Japan are asymptomatic, so the truth may fall somewhere between our middle and worst-case scenarios (Mizumoto et al., 2020) .We use a gamma distribution to model individual incubation times. We choose this form over the Weibull and lognormal distribution for ease of interpretation (gamma shape and scale parameters can be transformed easily to mean and standard deviation). So far, best-fit gamma distributions to COVID-19 data have had mean 6.5 and standard deviation 2.6 (Backer et al., 2020) , or mean 5.46 and standard deviation 1.94 (Lauer et al., 2020) . Here, to model uncertainty around the true mean incubation time, we fix the standard deviation to 2.25 (intermediate between the two existing estimates), and allow the mean to vary between 4.5 and 6.5 days (Figure 2-figure supplement 2) . The 95th percentile of the distributions we consider fall between 8.7 and 10.6 days, slightly below the officially accepted maximum incubation time of 14 days, and consistent with existing estimates ( Table 1; Backer et al., 2020; Lauer et al., 2020) .The probability that an infected traveller is detectable using questionnaire-based screening for exposure risk will be highest if risk factors with high sensitivity and specificity are known. Currently, official guidance recommends asking whether travellers have visited a country with epidemic transmission, a healthcare facility with confirmed cases, or had close contact with a confirmed or suspected case (World Health Organization, 2020c). Given the relative anonymity of respiratory transmission, we assume that a minority of infected travellers would realize that they have been exposed before symptoms develop (20% in Figure 2 , range 5-40% in Figure 3 ). Further, relying on a previous upper-bound estimate (Gostic et al., 2015) we assume that only 25% of travellers would self-report truthfully if aware of elevated exposure risk. Table 1 summarizes the state of knowledge about additional parameters, as of February 20, 2020.All code and source data used to perform analyses and generate figures is archived at https:// . (Gostic, 2020 ; copy archived at https:// ). The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication. There are no data inputs into our model. All parameter input values are specified in Table 1 , or in the manuscript text. We provide a link to the github repository containing all code necessary to run the analyses and generate figures (, copy archived at ).\n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.943\tBackground As the outbreak of coronavirus disease 2019 (COVID-19) progresses, epidemiological data are needed to guide situational awareness and intervention strategies. Here we describe efforts to compile and disseminate epidemiological information on COVID-19 from news media and social networks.As the outbreak of coronavirus disease 2019 (COVID- 19) is rapidly expanding in China and beyond, with the potential to become a worldwide pandemic, 1 real-time analyses of epidemiological data are needed to increase situational awareness and inform interventions. 2 Previously, real-time analyses have shed light on the transmissibility, severity, and natural history of an emerging pathogen in the first few weeks of an outbreak, such as with severe acute respiratory syndrome (SARS), the 2009 influenza pandemic, and Ebola. Analyses of detailed line lists of patients are particularly useful to infer key epidemiological parameters, such as the incubation and infectious periods, and delays between infection and detection, isolation, and reporting of cases. 3, 4 However, official individual patient data rarely become publicly available early on in an outbreak, when the information is most needed.Building on our previous experience collating news reports to monitor transmission of Ebola virus, 7 here we present an effort to compile individual patient information and subnational epidemic curves on COVID-19 from a variety of online resources. Data were made publicly available in real time and were used by the infectious disease modelling community to generate and compare epidemiological estimates relevant to interventions. We describe the data generation process and provide an early analysis of age patterns of COVID-19, case counts across China and inter nationally, and delays between symptom onset, admissions to hospital, and reporting, for cases reported until Jan 31, 2020.In this population-level observational study, we used crowdsourced reports from DXY.cn, a social network for Chinese physicians, health-care professionals, pharmacies, and health-care facilities established in 2000. This online platform is providing real-time coverage of the COVID-19 outbreak in China, obtained by collating and curating reports from news media, government television, and national and provincial health agencies. The information reported includes time-stamped cumulative counts of COVID-19 infections, outbreak maps, and realtime streaming of health authority announcements in Chinese (directly or through state media). 8 Every report is linked to an online source, which can be accessed for more detailed information on individual cases.These are publicly available, de-identified patient data reported directly by public health authorities or by state media. No patient consent was needed and no ethics approval was required.We closely monitored updates on DXY.cn between Jan 20, 2020, and Jan 31, 2020, to extract key information on individual patients in near real-time, and reports of daily case counts. For individual-level patient data, we used descriptions from the original source in Chinese to retrieve age, sex, province of identification, travel history, reporting date, dates of symptom onset and seeking care at a hospital or clinic, and discharge status, when available. Individual-level patient data were formatted into a line-list database for further quantitative analysis. Individual-level patient data were entered from DXY.cn by a native Chinese speaker (KS), who also generated an English summary for each patient. Entries were checked by a second person (JC). Since DXY.cn primarily provides Evidence before this study An outbreak of coronavirus disease 2019 (COVID-19) was recognised in early January, 2020, in Wuhan City, Hubei province, China. The new virus is thought to have originated from an animal-to-human spillover event linked to seafood and live-animal markets. The infection has spread locally in Wuhan and elsewhere in China, despite strict intervention measures implemented in the region where the infection originated on Jan 23, 2020. More than 500 patients infected with COVID-19 outside of mainland China have been reported between Jan 1 and Feb 14, 2020. Although laboratory testing for COVID-19 quickly ramped up in China and elsewhere, information on individual patients remains scarce and official datasets have not been made publicly available. Patient-level information is important to estimate key time-to-delay events (such as the incubation period and interval between symptom onset and visit to a hospital), analyse the age profile of infected patients, reconstruct epidemic curves by onset dates, and infer transmission parameters. We searched PubMed for publications between Jan 1, 1990, and Feb 6, 2020, using combinations of the following terms: (\"coronavirus\" OR \"2019-nCoV\") AND (\"line list\" OR \"case description\" OR \"patient data\") AND (\"digital surveillance\" OR \"social media\" OR \"crowd-sourced data\"). The search retrieved one relevant study on Middle East respiratory syndrome coronavirus that mentioned FluTrackers in their discussion, a website that aggregates epidemiological information on emerging pathogens. However, FluTrackers does not report individual-level data on COVID-19.To our knowledge, this is the first study that uses crowdsourced data from social media sources to monitor the COVID-19 outbreak. We searched DXY.cn, a Chinese health-care-oriented social network that broadcasts information from local and national health authorities, to reconstruct patient-level information on COVID-19 in China. We also queried international media sources and national health agency websites to collate data on international exportations of COVID-19. We describe the demographic characteristics, delays between symptom onset, seeking care at a hospital or clinic, and reporting for 507 patients infected with COVID-19 reported until Jan 31, 2020. The overall cumulative progression of the outbreak is consistent between our line list and an official report published by the Chinese national health authorities on Jan 28, 2020. The estimated incubation period in our data aligns with that of previous work. Our dataset was made available in the public domain on Jan 21, 2020.Crowdsourced line-list data can be reconstructed from social media data, especially when a central resource is available to curate relevant information. Public access to line lists is important so that several teams with different expertise can provide their own insights and interpretations of the data, especially in the early phase of an outbreak when little information is available. Publicly available line lists can also increase transparency. The main issue with the quality of patient-level data obtained during health emergencies is the potential lack of information from locations overwhelmed by the outbreak (in this case, Hubei province and other provinces with weaker health infrastructures). Future studies based on larger samples of patients with COVID-19 could explore in more detail the transmission dynamics of the outbreak in different locations, the effectiveness of interventions, and the demographic factors driving transmission.For an example of an online source see  ncovh5/view/pneumonia information on patients reported in China, we also compiled additional information on internationally exported cases of COVID-19. We obtained data for 21 countries outside of mainland China (Australia, Cambodia, Canada, France, Germany, Hong Kong, India, Italy, Japan, Malaysia, Nepal, Russia, Singapore, South Korea, Sri Lanka, Taiwan, Thailand, United Arab Emirates, the UK, the USA, and Vietnam). We gathered and cross-checked data for infected patients outside of China using several sources, including global news media (Kyodo News, Straits Times, and CNN), official press releases from each country's Ministry of Health, and disease control agencies.In addition to detailed information on individual patients, we reconstructed the daily progression of reported patients in each province of China from Jan 13, until Jan 31, 2020. We used the daily outbreak situation reports com municated by provincial health authorities, covered by state television and media, and posted on DXY.cn. All patients in our databases had a laboratory confirmed SARS coronavirus 2 (SARS-CoV-2) infection.Our COVID-19 database was made publicly available as a Google Sheet, disseminated via Twitter on Jan 21, 2020, and posted on the website of Northeastern University, (Boston, MA, USA) on Jan 24, 2020, where it is updated in real time. Data used in this analysis, frozen at Jan 31, 2020, are available online as a spreadsheet.We assessed the age distribution of all patients with COVID-19 by discharge status. We adjusted the age profile of Chinese patients by the population of China. We used 2016 population estimates from the Institute for Health Metrics and Evaluation 9 to calculate the relative risk (RR) of infection with COVID-19 by age group. To calculate the RR, we followed the method used by Lemaitre and colleagues 10 to explore the age profile of influenza, where RR for age group i is defined as where C i is the number of cases in age group i and N i is the population size of age group i.To estimate trends in the strength of case detection and interventions, we analysed delays between symptom onset and visit to a health-care provider, at a hospital or clinic, and from seeking care at a hospital or clinic to reporting, by time period and location. We considered the period before and after Jan 18, 2020, when media attention and awareness of the outbreak became more pronounced. 11 We used non-parametric tests to assess differences in delays between seeking care at a hospital or clinic and reporting between locations (Wilcoxon test to compare two locations and Kruskall-Wallis test to compare three or more locations).We estimated the duration of the incubation period on the basis of our line list data. We analysed a subset of patients returning from Wuhan who had spent less than a week in Wuhan, to ensure a narrowly defined exposure window. The incubation period was estimated as the midpoint between the time spent in Wuhan and the date of symptom onset.We did all analyses in R (version 3.5.3). We considered p values of less than 0·05 to be significant.The funder had no role in study design, data compilation, data analysis, data interpretation, or writing of the report. All authors had access to the data, and had final responsibility for the decision to submit for publication.Our line list comprised 507 patients reported from Jan 13, to Jan 31, 2020, including 364 (72%) from mainland China and 143 (28%) from outside of China (table) The age distribution of COVID-19 cases was skewed towards older age groups with a median age of 45 years (IQR 33-56) for patients who were alive or who had an unknown outcome at the time of reporting (figure 1). The median age of patients who had died at the time of reporting was 70 years (IQR 65-81). Few patients (13 [3%]) were younger than 15 years. Adjustment for the age demographics of China confirmed a deficit of infections among children, with a RR below 0·5 in patients younger than 15 years (figure 1). The RR measure indicated a sharp increase in the likelihood of reported COVID-19 among people aged 30 years and older.A timeline of cases in our crowdsourced patient line list is shown by date of onset in figure 2 , indicating an acceleration of reported cases by Jan 13, 2020. The outbreak progression based on the crowdsourced patient line list was consistent with the timeline published by China Center for Disease Control and Prevention (CDC) on Jan 28, 2020, 12 which is based on a more comprehensive database of more than 6000 patients with COVID-19. Since Jan 23, 2020, the cumulative number of cases has slowed down in the crowdsourced and China CDC curves (figure 2), which probably reflects the delay between disease onset and reporting. The median reporting delay was 5 days (IQR 3-8) in our data.Province-level epidemic curves are shown by reporting date in figure 3 . As of Jan 31, 2020, 16 (52%) of 30 provinces in mainland China had reported more than 100 confirmed cases. The apparent rapid growth of newly reported cases between Jan 18, and Jan 31, 2020, in several provinces outside of Hubei province is consistent with sustained local transmission.Across the study period, the median delay between symptom onset and seeking care at a hospital or clinic was 2 days (IQR 0-5 days) in mainland China ( figure 4 ). This delay decreased from 5 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p=0·0009). Some provinces, such as Tianjin and Yunnan had shorter delays (data by province not shown), while the early cases from Hubei province were characterised by longer delays in seeking care (median 0 days [IQR 0-1]).The median delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 2-5 days) in mainland China and decreased from 9 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p<0·0001; figure 4 ). Similarly to delays in seeking care at a hospital or clinic, reporting was quickest in Tianjin and Yunnan (median 1 day [IQR 0-1]) and slowest in Hubei province (median 12 days [IQR 7-16]).The median delay between symptom onset and seeking care at a hospital or clinic was 1 day (IQR 0-3) for international travellers, and shorter than for patients in Hubei province or the rest of mainland China (Kruskal-Wallis test p<0·0001; figure 4) . Even in the period after Jan 18, 2020, when awareness of the outbreak increased, a shorter delay between symptom onset and seeking care at a hospital or clinic was seen for international patients than for those in mainland China (Wilcoxon test p<0·0001). For international cases, the delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 1-4), also shorter than for mainland China (Wilcoxon test p<0·0001; figure 4) .On the basis of 33 patients with a travel history to Wuhan, we estimated the median incubation period for COVID-19 to be 4·5 days (IQR 3·0-5·5; appendix p 2). Information from patient line lists is crucial but difficult to obtain at the beginning of an outbreak. Here we have shown that careful compilation of crowdsourced reports curated by a long-standing Chinese medical social network provides a valuable picture of the outbreak of COVID-19 in real time. The outbreak timeline is consistent with aggregated case counts provided by health authorities. For comparison, China CDC published the first epidemic curve by symptom onset on Jan 28, 2020. 12 Line lists provide unique information on the delays between symptom onset and detection by the health-care system, reporting delays, and travel histories. This information cannot be extracted from aggregated case counts published by official sources. Line list data can help assess the effectiveness of interventions and the potential for widespread transmission beyond the initial foci of infection. In particular, shorter delays between symptom onset and admission to hospital or seeking care in a hospital or clinic accelerate detection and isolation of cases, effectively shortening the infectious period. A useful feature of our crowdsourced database was the availability of travel histories for patients returning from Wuhan, which, along with dates of symptom onset, allowed for estimation of the incubation period here and in related work. 13, 14 A narrow window of exposure could be defined for a subset of patients who had a short stay in Wuhan, at a time when the epidemic was still localised to Wuhan. Several teams have used our dataset and datasets from others to estimate a mean incubation period for COVID-19 to be 5-6 days (95% CI 2-11). Our own estimate (median 4·5 days [IQR 3·0-5·5]) is consistent with previous work that used other modelling approaches. The incubation period is a useful parameter to guide isolation and contact tracing; based on existing data, the disease status of a contact should be known with near certainty after a period of observation of 14 days. 13 Availability of a public dataset enables independent estimation of important epidemiological parameters by several teams, allowing for confirmation and cross-checking at a time when information can be conflicting and noisy.An interesting finding in our data relates to the age distribution of patients. We found a heavy skew of infection towards older age groups, with substantially fewer children infected. This pattern could indicate agerelated differences in susceptibility to infection, severe outcomes, or behaviour. However, a substantial portion of the patients in our database are travellers, a population that is usually predominantly adults (although does not exclude children). Furthermore, because patient data in our dataset were captured by the health system, they are biased towards the more severe spectrum of the disease, especially for patients from mainland China. Clinical reports have shown that severity of COVID-19 is associated with the presence of chronic conditions, 16, 17 which are more frequent in older age groups. Nevertheless, we would also expect children younger than 5 years to be at risk of severe outcomes and to be reported to the healthcare system, as is seen for other respiratory infections. 18 Biological differences could have a role in shaping these age profiles. A detailed analysis of one of the early COVID-19 clusters by Chan and colleagues 19 revealed symptomatic infections in five adult members of the same household, while a child in the same household aged 10 years was infected but remained asymptomatic, potentially indicating biological differences in the risk of clinical disease driven by age. Previous immunity from infection with a related coronavirus has been speculated to potentially protect children from SARS, 20, 21 and so might also have a role in COVID-19. In any case, if the age distribution of cases reported here was to be confirmed and the epidemic were to progress globally, we would expect an increase in respiratory mortality concentrated among people aged 30 years and older. This mortality pattern would be substantially different from the profile of the 2009 influenza pandemic, for which excess mortality was concentrated in those younger than 65 years. 21 In our dataset, we saw a rapid increase in the number of people infected with COVID-19 in several provinces of China, consistent with local transmission outside of Hubei province. As of Jan 31, 2020, province-level epidemic curves are only available by date of reporting, rather than date of symptom onset, which usually inflates recent case counts if detection has increased. D e c 9 , 2 0 1 9 D e c 1 6 , 2 0 1 9 D e c 2 3 , 2 0 1 9 D e c 3 0 , 2 0 1 9 J a n 6 , 2 0 2 0 J a n 1 3 , 2 0 2 0 J a n 2 0 , 2 0 2 0 J a n 2 7 , 2 0 2 0 Furthermore, province-level data include both returning travellers from Hubei province (ie, importations) and locally acquired cases, which also usually inflate the apparent risk of local transmission. Notably, other lines of evidence suggest that local transmission is now well established outside of Hubei province, because travel increased just before the Chinese New Year on Jan 25, 2020, and before implementation of the travel ban in Wuhan. 22 Accordingly, our own data include evidence of transmission clusters in non-travellers, with, for instance, a second-generation transmission event reported in Shaanxi on Jan 21, 2020.Our study had several limitations, one of which was the data we used. Although all provinces in mainland China provide aggregated information on infections and deaths, individual-level patient descriptions are only available for a subset of provinces. Geographical coverage is heterogeneous in our line list, and we have a notable deficit of cases from Hubei province, the foci of the COVID-19 outbreak. We expect that little patient-level information is shared on social media by province-level and city-level health authorities in Wuhan and Hubei province because health systems are overwhelmed. For similar reasons, provinces with a large total case count at the end of January, 2020, or with a weaker health infrastructure, were under-represented in our line list, with the exception of Beijing. Other limitations in our data include severity (only patients who had severe enough symptoms to seek care were captured) and changes in case definition. A series of epidemiological criteria were required for COVID-19 testing, including travel history to Wuhan within the past 2 weeks; residence in Wuhan within the past 2 weeks; contact with individuals from Wuhan (with fever and respiratory symptoms) within the past 2 weeks; and being part of an established disease cluster. Some of these criteria (eg, relation to Wuhan) were relaxed over time (appendix). As a result, we have an overrepresentation of travel-related cases in our database.The reproduction number is an important quantity for outbreak control. We refrained from estimating this parameter because reporting changes could bias estimates relying on epidemic growth rates. Furthermore, our dataset captured cases all over China and does not reflect transmission patterns in any particular location. A mean reproduction number of 2·5-2·7 has previously been estimated on the basis of the volume of importations of international cases in the pre-intervention period in Wuhan. 11 We recognise that, although our data source is useful and timely, it should not replace official statistics. Manual compilation of detailed line lists from media sources is highly time consuming and is not sustainable when case counts reach several thousands. Here we provide detailed data on 507 patients when the official case count was over 9000 by Jan 31, 2020, representing a sample of approximately 5% of reported cases and a much smaller proportion of the full spectrum of COVID-19 cases, which include mild infections. A crowd sourced system would not be expected to catch all cases, especially if many cases are too mild to be captured by the health-care system, digital surveillance, or social media. Notably, DXY.cn does not generate data outside of traditional surveillance systems but rather provides a channel of rapid communication between the public and health authorities. In turn, our approach has helped extract and repackage information from health authorities into an analytical format, which was not available elsewhere.At the time of writing, efforts are underway to coordinate compilation of COVID-19 data from online sources across several academic teams. Ultimately, we expect that a line list of patients will be shared by government sources with the global community; however, data cleaning and access issues might take a prohibitively long time to resolve. For the west African Ebola outbreak, a similarly coordinated effort to publish a line list took 2 years. 23 Given the progression of the COVID-19 outbreak, such a long delay would be counterproductive.Overall, the novelty of our approach was to rely on a unique source for social media and news reports in China, which aggregated and curated relevant information. This approach facilitated entry of robust and standard data on clinical and demographic information. Reassuringly, DXY.cn maintains a special section dedicated to debunking fake news, myths, and rumours about the COVID-19 outbreak. Looking to the future, collection of patient data in the context of emergencies could include information on whether patients are identified through contact tracing or because they seek care on their own. Furthermore, data interpretability could be improved by gathering more quantitative information on how case definitions are used in practice.In conclusion, crowdsourced epidemiological data can be useful to monitor emerging outbreaks, such as COVID-19 and, as previously, Ebola virus. 7 These efforts can help generate and disseminate detailed information in the early stages of an outbreak when little other data are available, enabling independent estimation of key parameters that affect interventions. Based on our small sample of patients with COVID-19, we note an intriguing age distribution, reminiscent of that of SARS, which warrants further epidemiological and serological studies. We also report early signs that the response is strengthening in China on the basis of a decrease in case detection time, and rapid management of travel-related infections that are identified internationally. This is an early report of a rapidly evolving situation and the parameters discussed here could change quickly. In the coming weeks, we will continue to monitor the epidemiology of this outbreak using data from news reports and official sources.KS and CV contributed to the study design. KS and JC contributed to the data compilation. KS, JC, and CV contributed to data analysis. KS and JC contributed to the design and drawing of figures. KS, JC, and CV contributed to the writing of the manuscript.We declare no competing interests.All data used in this report have been made publicly available on the Laboratory for the Modeling of Biological + Socio-technical systems website of Northeastern University. The available data include daily case counts of COVID-19 by reporting date and Chinese province, and a de-identified line list of patients with COVID-19. The line list includes geographical location (country and province), reporting date, dates of symptom onset and seeking care at a hospital or clinic, relation to Wuhan, discharge status when known, an English summary of the case description from media sources, and a link to the original source of data. Seeking care at hospital or clinic to report Symptom onset to seeking care at hospital or clinic\n",
            "\t0.924\tIn December 2019, a novel coronavirus, called COVID-19, was discovered in Wuhan, China, and has spread to different cities in China as well as to 24 other countries. The number of confirmed cases is increasing daily and reached 34,598 on 8 February 2020. In the current study, we present a new forecasting model to estimate and forecast the number of confirmed cases of COVID-19 in the upcoming ten days based on the previously confirmed cases recorded in China. The proposed model is an improved adaptive neuro-fuzzy inference system (ANFIS) using an enhanced flower pollination algorithm (FPA) by using the salp swarm algorithm (SSA). In general, SSA is employed to improve FPA to avoid its drawbacks (i.e., getting trapped at the local optima). The main idea of the proposed model, called FPASSA-ANFIS, is to improve the performance of ANFIS by determining the parameters of ANFIS using FPASSA. The FPASSA-ANFIS model is evaluated using the World Health Organization (WHO) official data of the outbreak of the COVID-19 to forecast the confirmed cases of the upcoming ten days. More so, the FPASSA-ANFIS model is compared to several existing models, and it showed better performance in terms of Mean Absolute Percentage Error (MAPE), Root Mean Squared Relative Error (RMSRE), Root Mean Squared Relative Error (RMSRE), coefficient of determination (R 2 ), and computing time. Furthermore, we tested the proposed model using two different datasets of weekly influenza confirmed cases in two countries, namely the USA and China. The outcomes also showed good performances.A large family of viruses, called coronaviruses, are severe pathogens for human beings, which infect respiratory, hepatic, gastrointestinal, and neurologic diseases. They are distributed among humans, birds, livestock, mice, bats, and other wild animals . The outbreaks of two previous coronaviruses, SARS-CoV and MERS-CoV in 2003 and 2012, respectively, have approved the transmission from animal to animal, and human to human . In December 2019, the World Health Organization (WHO) received notifications from China for many cases of respiratory illness that were linked to some people who had visited a seafood market in Wuhan . Currently, Wuhan city suffers from the spreading of a novel coronavirus, called COVID-19 (previously, it was called 2019-nCoV). In , the authors concluded that COVID-19 likely originated in bats, because it is more similar to two bat-derived coronavirus strains. However, the source of the COVID-19 is not confirmed yet, and it communities, Hong Kong and Toronto, were 1.2 and 1.32, respectively. Ong et al. proposed a monitoring and forecasting model for influenza A (H1N1-2009). Furthermore, Nah et al. proposed a probability-based model to predict the spread of the MERS.The Adaptive Neuro-Fuzzy Inference System (ANFIS) is widely applied in time series prediction and forecasting problems, and it showed good performance in many existing applications. It offers flexibility in determining nonlinearity in the time series data, as well as combining the properties of both artificial neural networks (ANN) and fuzzy logic systems. It has been applied in various forecasting applications, for example, in , a stock price forecasting model was proposed using ANFIS and empirical mode decomposition. Chen et al. proposed a TAIEX time series forecasting model based on a hybrid of ANFIS and ordered weighted averaging (OWA). In , another time series forecasting method was presented for electricity prices based on ANFIS. Svalina et al. proposed an ANFIS based forecasting model for close price indices for a stock market for five days. Ekici and Aksoy presented an ANFIS based building energy consumption forecasting model. More so, ANFIS is also applied to forecast electricity loads . Kumar et al. proposed an ANFIS based model to forecast return products. Ho and Tsai applied ANFIS to forecast product development performance. However, estimating ANFIS parameters is a challenge that needs to be improved. Therefore, in previous studies, some individual swarm intelligence (SI) methods have been applied to the ANFIS parameters to enhance time series forecasting because these parameters have a significant effect on the performance of ANFIS. The SI methods include the particle swarm optimization (PSO) [31, 32] , social-spider optimization , sine-cosine algorithm (SCA) , and multi-verse optimizer (MVO) . For example, in SCA algorithm was applied to improve the ANFIS model to forecast oil consumption in three countries, namely, Canada, Germany, and Japan. In the same context, in , The MVO algorithm was used to enhance the ANFIS model to forecast oil consumption in two countries. In addition, in the PSO was used with ANFIS to predict biochar yield. However, individual SI algorithms may stock at local optima. Therefore, one solution is to apply hybrid SI algorithms to avoid this problem. In , a hybrid of two SI algorithms, namely GA and SSA, was presented to improve the ANFIS model. The proposed new model called GA-SSA-ANFIS was applied to forecast crude oil prices for long-term time series data. However, the previously mentioned methods suffer from some limitations that can affect the performance of the forecasting output such as slow convergence and the ability to balance between exploration and exploitation phases can influence the quality of the final output. This motivated us to propose an alternative forecasting method dependent on the hybridization concept. This concept avoids the limitations of traditional SI techniques by combining the strengths of different techniques, and this produces new SI techniques that are better than traditional ones.In the current study, we propose an improved ANFIS model based on a modified flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). The FPA is an optimization algorithm proposed by Yang , which was inspired by the flow pollination process of the flowering plants. The FPA was employed in various optimization applications, for example to estimate solar PV parameter [39, 40] , solving sudoku puzzles , feature selection , antenna design , and other applications . Moreover, SSA is also an optimization algorithm proposed by Mirjalili et al. inspired by the behavior of salp chains. In recent years, the SSA was utilized to solve different optimization problems, such as feature selection [49, 50] , data classification , image segmentation , and others [53, 54] .The proposed method called FPASSA is a hybrid of FPA and SSA, in which the SSA is applied as a local search method for FPA. The proposed FPASSA starts by receiving the historical COVID-19 dataset. Then a set of solutions is generated where each of them represents the value for the parameters of the ANFIS model. Then the quality of each solution is calculated using the fitness value, and the solution that has the best fitness value is chosen to represent the best solution. Then the probability of each solution is computed. Then the current solution will be updated, either using global or local strategy in FPA. However, in the case of local strategy, the operators of SSA or FPA will be used according to the probability of the fitness value for each solution. The process of updating the solutions is repeated until reaching the stop condition, and the best parameter configurations are used to forecast the number of confirmed cases of COVID-19.The main contribution points of the current study are as follows:1.We propose an efficient forecasting model to forecast the confirmed cases of the COVID-19 in China for the upcoming ten days based on previously confirmed cases.An improved ANFIS model is proposed using a modified FPA algorithm, using SSA.We compare the proposed model with the original ANFIS and existing modified ANFIS models, such as PSO, GA, ABC, and FPA.The rest of this study is organized as follows. The preliminaries of ANFIS, FPA, and SSA are described in Section 2. Section 3 presents the proposed FPASSA, and Section 4 presents the experimental setup and results. We conclude this study in Section 5.The principles of the ANFIS are given in this section. The ANFIS model links the fuzzy logic and neural networks . It generates a mapping between the input and output by applying IF-THEN rules (it is also called Takagi-Sugeno inference model). Figure 1 illustrates the ANFIS model where, y and x define the inputs to Layer 1 whereas, O 1i is its output of node i that is computed as follows:where µ denotes the generalized Gaussian membership functions. A i and B i define the membership values of µ. α i and ρ i denote the premise parameters set. The output of Layer 2 (it is also known as the firing strength of a rule) is calculated as follows:Meanwhile, the output of Layer 3 (it is also known as the normalized firing strength) is calculated as follows:The output of Layer 4 (it is also known as an adaptive node) is calculated as follows:where r i , q i , and p i define the consequent parameters of the node i. Layer 5 contains only one node; its output is computed as: Flower Pollination Algorithm is an optimization method proposed by Yang . It simulates the transfer of flowers' pollen by pollinators in nature. This algorithm utilizes the two types of pollination (i.e., self-pollination and cross-pollination). In self-pollination, the pollination occurs with no pollinators, whereas, in cross-pollination, the pollens are moved between different plants. In more detail, the self-pollination can be represented as a local pollination while the cross-pollination can be called global pollination.The global pollination or cross-pollination can be mathematically formed as follows:where x t i defines the pollen i at iteration t. L denotes the pollination's strength or the step size. F * is the target position or best solution. In some cases, insects can fly with different distance steps for a long space; therefore, Levy fly distribution is applied to simulate this movement.where λ = 1.5. Γ(λ) denotes the gamma function. This distribution is available for large steps s > 0. The self-pollination or local pollination can be mathematically formed as follows:where x t i and x k i represent pollens from different flower in the same plant. in the range [0,1] The process of pollination can be done using cross-pollination or self-pollination. Therefore, the random variable p, in the range [0, 1], is used to determine this process.SSA is an optimization technique introduced by . It simulates the Salps' behavior in nature. This behavior is called salp chain. The mathematical model of SSA begins by splinting its population into a leader group and followers group. The leader is the front salp, whereas, the followers are the other salps. The search space is determined in n-dimensions with n variables. Equation (10) works to update the salps' positions.where x 1 j denotes the leader's position in j-th dimension. F j is the target position. ub j and lb j represent the max and min bounds, respectively. c 2 and c 3 denote random numbers in [0, 1]. c 1 is an important parameter; it balances between the exploration and exploitation phases. It is computed as follows:where the current loop number is t and the max loop' number is t max . Then, the followers' position is updated as follows:where x i j defines the i-th position of the follower in j-th dimension. i > 1.This section explains the proposed FPASSA-ANFIS method. It is a time series method for forecasting the confirmed cases of the COVID-19, as given in Figure 2 . The FPASSA-ANFIS utilizes the improved FPA to train the ANFIS model by optimizing its parameters. The FPASSA-ANFIS contains five layers as the classic ANFIS model. Layer 1 contains the input variables (the historical COVID-19 confirmed cases). Whereas Layer 5 produces the forecasted values. In the learning phase, the FPASSA is used to select the best weights between Layer 4 and Layer 5.The FPASSA-ANFIS starts by formatting the input data in a time series form. In our case, the autocorrelation function (ACF) was considered. ACF is one of the methods applied to find patterns in the data; it presents information about the correlation between points separated by various time lags. Therefore, in this paper, the variables with ACF greater than 0.2 are considered i.e., 5-lags.Besides, the training data contains 75% of the dataset, whereas the testing data contains 25% of them. The number of clusters is defined by the fuzzy c-mean (FCM) method to construct the ANFIS model.The parameters of the ANFIS model are prepared by the FPASSA algorithm. In the training phase, the calculation error (as in Equation (13)) between the real data and the predicted data is used to evaluate the parameters' quality.where T is the real data, and P is the predicted data. N s is the sample length. The smaller values of the objective function indicate good ANFIS's parameter.On the other hand, the updating phase of the followers' positions in the SSA algorithm is applied to improve the global pollination phase in the FPA algorithm. In this improvement, there is a random variable (r) used to switch between both phases. If r > 0.5, then the operators of the SSA is used; otherwise, the operators of the FPA are used. In general, The FPASSA starts by constructing the population (X); afterward, the objective function is calculated for each solution. The solution with the lowest error value is saved to the next iteration. This sequence is repeated until meeting the stop condition, which in this paper, is the maximum number of iterations. Then the best solution is passed to train the parameters of the ANFIS model.After finishing the training phase, the testing phase is started with the best solution to compute the final output. The performance of the proposed method is evaluated by comparing the real data with the predicted data using the performance measures. Finally, the FPASSA produces a foretasted value for confirmed cases of COVID-19 in China in the next day. The steps of the proposed FPASSA are presented in Algorithm 1.Input: Historical COVID-19 dataset, size of population N, total number of iterations t max .Divide the data into training and testing sets.Using Fuzzy c-mean method to determine the number of membership functions.Constructing the ANFIS network.Set the initial value for N solutions (X). Return the best solution that represents the best configuration for ANFIS.Apply the testing set to the best ANFIS model.Forecasting the COVID-19 for the next ten days.This section presents the description of the used dataset, the performance measures, the parameter setting for all methods, the experiment results, and discussions.The main dataset of this study is COVID-19 dataset. It was collected from the WHO website (https: //). It contains the daily confirmed cases in China from 21 January 2020 to 18 February 2020, as shown in Table 1 . We used 75% from the dataset to train the model while the rest is used to test it.Moreover, we evaluated the performance of the proposed method using two datasets of weekly influenza confirmed cases. The first one is called DS1; it was collected from the Centers for Disease Control and Prevention (CDC) (). It starts from week number 40 in 2015 and continues until week number 6 in 2020. Whereas, the second one is called DS2. It was collected from the WHO website (). It contains the data of weekly influenza confirmed cases in China from week number 1 in 2016 to week number 8 in 2020. The quality of the proposed method is evaluated using a set of performance metrics as follows:• Root Mean Square Error (RMSE):where Yp and Y are the predicted and original values, respectively. • Mean Absolute Error (MAE):• Mean Absolute Percentage Error (MAPE):• Root Mean Squared Relative Error (RMSRE):N s represents the sample size of the data. • Coefficient of Determination (R 2 ):where Y represents the average of Y.The lowest value of RMSE, MAE, MAPE, and RMSRE refers to the best method. The higher value of R 2 indicates better correlation for the method.This paper aims to assess the ability of the FPASSA to forecast the COVID-19 by comparing its performance with other methods, namely the ANFIS and the trained ANFIS models using PSO, GA, ABC, FPA, and FPASSA. The parameters' setting for these models is listed in Table 2 .The common parameters, such as population size, are set to 25 and 100 iterations are applied. Besides, each algorithm is performed for 30 independent runs to fair comparisons. The selected parameters are chosen because they produced good behavior in previous experiments, such as [34, 35, 55, 56] . Table 2 . Parameters' setting.Parameters SettingMax. epochs = 100, Error goal = 0, Initial step = 0.01, Decrease rate = 0.9, Increase rate = 1. In this section, the performance of the proposed FPASSA to predict the DS1 and DS2 is discussed. It can be concluded from Table 3 that the performance of FPASSA outperformed the compared methods in all measures, whereas the FPA is ranked second. The results of DS2 indicate that the FPASSA is ranked first in terms of RMSE, MAPE, R 2 , and the CPU time. Whereas, the PSO is ranked second, followed by the FPA, GA, then ABC. These results denote that the proposed method can optimize the parameters of the ANFIS model effectively and produce good results in terms of the performance measures. Comparison results between the proposed FPASSA and other models to forecast COVID-19 are given in Table 4 . It can be concluded that the FPASSA outperforms other models. For example, by analyzing the results of RMSE, MAE, MAPE, RMSRE, and CPU time(s) it can be observed that the FPASSA achieves the smallest value among the comparison algorithms, and this indicates the high quality of the FPASSA. Meanwhile, the FPA allocates the second rank, which provides better results than the rest of the methods.Moreover, the value of R 2 refers to the high correlation between the prediction obtained by the proposed FPASSA method and the original COVID-19, which has nearly 0.97. This can also be noticed from Figure 3 , which depicts the training of the algorithms using the historical data of the COVID-19 as well as their forecasting values for ten days. Table 5 depicts the forecasting value for the confirmed cases of the COVID-19 in China from 19/2/2020 to 28/2/2020. From these results, it can be noticed that the outbreak will reach its highest level on the day 28/2/2020. The average percentage of the increase over the forecasted period is 10%, the highest percentage is 12% on 28/2/2020, and the lowest percentage is 8.7% on 19/2/2020. From the previous results, it can be concluded that the proposed FPASSA-ANFIS has a high ability to forecast the COVID-19 dataset. These results avoid the limitations of traditional ANFIS because of the combination with the modified FPA method. Moreover, the operators of SSA are combined with the local strategy of FPA to enhance their exploitation ability. However, the time computational of the proposed FPASSA method still requires more improvements.This paper proposed a modified version for the flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). This modified version, called FPASSA, is applied to improve the performance of the ANFIS through determining the optimal value for its parameters. The developed FPASSA-ANFIS model is applied as a forecasting technique for a novel coronavirus, called COVID-19, that was discovered in Wuhan, China at the end of last year and January of the current year. The proposed FPASSA-ANFIS model has a high ability to predict the number of confirmed cases within ten days. Besides, FPASSA-ANFIS outperforms other forecasting models in terms of RMSE, MAE, MAPE, RMSRE, and R 2 . Furthermore, two datasets of weekly influenza confirmed cases in the USA and China were used to evaluate the proposed method, and the evaluation outcomes showed its good performance. According to the promising results obtained by the proposed FPASSA-ANFIS, it can be applied in different forecasting applications.\n",
            "\t0.900\tThe ongoing coronavirus disease 2019 (COVID-19) outbreak is giving rise to worldwide anxieties, rumours, and online misinformation. But it offers an opportunity to put into practice some lessons learned in studies of social media during epidemics, particularly with respect to the dynamics of online heroisation and blame.Epidemics such as the H1N1 influenza pandemic, severe acute respiratory syndrome, and Ebola take place in a complex world, with many disasters (human-caused and natural) and a host of social, cultural, economic, political, and religious concerns. Responding to such concerns is not usually part of public health approaches to epidemic communications, which emphasise biomedical and epidemiological information.This approach is short-sighted. Research on the Ebola and H1N1 influenza epidemics 1-4 suggests that gathering online data on local perceptions has the potential to help public authorities mount more robust responses and better targeted health communications. Particularly fruitful paths to investigate, we believe, are the dynamics of heroisation and the creation of so-called figures of blame.Searching for someone to blame is part of the process of making sense of any disaster, akin to the phenomenon of moral panic. 5 Conspiracy theories and misinformation are already circulating in traditional and social media about COVID-19. 6, 7 Thus, it is important to track the evolving dynamics of blame in real time, both to correct inaccurate information and to respond to online scapegoating.The other side of the coin is heroisation, the investment of hope and trust in a context of risk and unease. Analyses of blame and heroisation during the 2014-15 Ebola epidemic, using Twitter and Facebook posts in French and English, 2,8 suggest that heroic status was widely conferred on ordinary individuals and insiders rather than altruistic foreigners, as in other crises. The term local hero is not an empty phrase: identification of local heroes as they emerge, and working with them (online and offline), could have a strong pay-off in communication campaigns. What constitutes a hero during a time of crisis is nuanced and context-specific, however, and needs careful qualitative work to understand. Heroes can include, for example, whistle-blowers (who put their careers on the line to alert the public) and health workers (who generate essential information while doing their work). All these figures can be seen emerging during the COVID-19 outbreak.Qualitative analysis can also identify issues in authorities' handling of crises, which crystallise around transparency. For instance, discussions can coalesce not only around conspiracy theories but also around real uncertainties and blind spots in health authorities' communications. 9, 10 In times of crisis, public authorities tend to focus their concern on avoiding panic and filtering the information they provide to the public. But trust is a crucial support to public health systems. It is during crises such as the COVID-19 outbreak that this trust is put to the test.Tracking heroisation and blame dynamics in realtime, as epidemics unfold, can help health authorities to understand public attitudes both to the threats posed by epidemics and the hope offered by health interventions, to fine-tune targeted health communication strategies accordingly, to identify and amplify local and international heroes, to identify and counter attempts to blame, scapegoat, and spread misinformation, and to improve crisis management practices for the future. Such an approach can bring to the surface what we propose to call complex geographies of hope and blame, which public health authorities need to be aware of while planning responses to the epidemic.\n",
            "\t0.889\tRapid data sharing is the basis for public health action. The report from the 30 January 2020 International Health Regulations (2005) Emergency Committee regarding the outbreak of novel coronavirus (COVID-19) stressed the importance of the continued sharing of full data with the World Health Organization (WHO). The information disseminated through peer-reviewed journals and accompanying online data sets is vital for decision-makers. For example, the release of full viral genome sequences through a public access platform and the polymerase chain reaction assay protocols that were developed as a result made it possible to accurately diagnose infections early in the current emergency.Deficiencies in data-sharing mechanisms -highlighted during the 2013-2016 Ebola virus disease outbreak in west Africa -brought the question of data access to the forefront of the global health agenda. 2 In September 2015, agreement was reached on the need for open sharing of data and results, especially in public health emergencies. 2 Subsequently, the International Committee of Medical Journal Editors confirmed that pre-publication dissemination of information critical to public health will not prejudice journal publication in the context of health emergencies declared by WHO. 4 Furthermore the committee stated that information critical for public health is to be shared with WHO before publication 5 -a commitment echoed by several leading journals in the context of the COVID-19 response.Efforts for expedited data and results reporting should not be limited to clinical trials, but should include observational studies, operational research, routine surveillance and information on the virus and its genetic sequences, as well as the monitoring of disease control programmes.To improve timely access to data in the context of the COVID-19 emergency the Bulletin of the World Health Organization will implement an \"COVID-19 Open\" data sharing and reporting protocol, which will apply during the current COVID-19 emergency.On submission to the Bulletin, all research manuscripts relevant to the coronavirus emergency will be assigned a digital object identifier and posted online in the \"COVID-19 Open\" collection within 24 hours while undergoing peer review. The data in these papers will thus be attributed to the authors while being freely available for unrestricted use, distribution and reproduction in any medium, provided that the original work is properly cited as indicated by the Creative Commons Attribution 3.0 Intergovernmental Organizations license (CC BY IGO 3.0). Should a paper be accepted by the Bulletin following peer review, this open access review period will be reported in the final publication. If a paper does not meet the journal's requirements after peer review, authors will be free to seek publication elsewhere. If the authors of any paper posted with the Bulletin in this context are unable to obtain acceptance with a suitable journal, WHO undertakes to publish these papers in its institutional repository as citable working papers, independently of the Bulletin. The choice of a pre-print platform remains the sole discretion of the author. This early access to research manuscripts at WHO builds on examples of other rapid information access platforms such as PROMED and F1000Research. 5, 6 Given the many unanswered questions on the reservoir, transmission, consequences and manifestations of COVID-19 infection and associated disease, our goal is to encourage all researchers to share their data as quickly and widely as possible. With this protocol for immediate online posting, we are providing another means to achieve immediate global access to relevant data. By submitting their studies to \"COVID-19 Open,\" researchers can share their data while meeting their need to retain authorship, document precedence and facilitate international scientific cooperation in the response to this emergency. ■\n",
            "\t0.860\tThe virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV .The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host . Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.The symptoms of COVID-19 are fever, dry cough, fatigue, nasal congestion, sore throat and diarrhoea. On February 14th, the Chinese Center for Disease Control and Prevention (China CDC) published the first details of 44,672 confirmed cases, in the biggest study since the outbreak began . Their findings show that COVID-19 was mild for 81% of patients and had an overall case fatality rate of 2.3%. Of those confirmed cases, only 2.2% were under 20 years old. Compared to adults, children generally present with much milder clinical symptoms. It is likely that future serological studies will show much asymptomatic disease in children. As opposed to H1N1, pregnant women do not appear to be at higher risk of severe disease. The severity of the disease appears to be associated with age, with the elderly most at risk; those over 80 years of age had a Case Fatality Rate (CFR) of 14.8%. The CFR was also increased in those with comorbidities including cardiovascular, diabetes, chronic respiratory disease, hypertension, and cancer. The cause of death is respiratory failure, shock or multiple organ failure.There is no proven treatment at this early stage but we will doubtless have more information about this soon. It can be assumed that non-pharmacologic approaches are effective such as fluid support, oxygen and ventilatory support. Most recently the national data suggests that 17.7%, 10.4% and 7.0% of all cases have disease requiring respiratory support in Wuhan, Hubei (not including Wuhan) and the rest of China (respectively). About a quarter of all require ventilation while 75% require oxygen support only. The variation in severity rates probably reflects the outcomes in an overwhelmed health system. Extra Corporeal Membrane Oxygenation (ECMO) is potentially of benefit and we will know more particularly when cities with higher technology health systems become affected and ECMO is truly tested in the most severely ill. ECMO is currently being used in China, however, its effectiveness is yet to be determined. Antiviral drugs as well as a variety of other putative treatments are typically being prescribed for deteriorating patients on a compassionate basis. Clinicians would be well aware of such situations but assurance is required that their safety and efficacy are being scientifically assessed so that meaning is brought to bear quickly. Coordination of clinical trials to avoid duplication and ensure that results are rapidly available will be a challenge but the case numbers should facilitate rapid definitive results. (see What is in the pipeline for vaccine development and/or therapeutics?).As a novel virus newly emerged in humans, the world's population is completely immune-naïve and therefore vulnerable. There is clear human-to-human transmission in family clusters in China and beyond, transmission from close face-to-face social contact, especially in small enclosed spaces, and transmission from failed infection prevention and control measures in health facilities. In addition, the experience in Wuhan shows that transmission can be massive in a short period of time with thousands of new patients diagnosed daily.The current aim of the global response is to flatten the epidemic curve so that transmission is slowed, and to interrupt transmission where possible. While there is clearly a mortality linked to the virus, the most concerning problem will be if a health system is overwhelmed in the wake of rapid transmission so that affected patients cannot receive the care they need. Furthermore, patients with other urgent medical conditions are at risk of not obtaining their necessary care. Countries with vulnerable health systems are particularly of concern.Recently, there have been outbreaks in newly affected countries including Italy and Iran where the index case is unidentified. Furthermore affected countries have very large clusters emerging such as Korea and Japan. There is no reason to believe that the global community is ready for this emerging pandemic; ready in a way that can see drastic public health intervention implemented within days, including aggressive and massive contact tracing, monitoring (or quarantine) and early detection and isolation in an attempt to slow the progression.There is uncertainty regarding transmissibility and severitymore information is emerging about the spectrum of disease, especially mild disease, which is not identified using many current case definitions, and about the ease of transmission from person to person. Health agencies are unsure how to model this and estimates vary depending on the variables being used. For instance, SARS was essentially spread later in the disease from patients with more significant clinical pictures, and it was contained by infection control measures particularly in hospitals. The limited spread to family members of health workers and the community was contained by usual outbreak control measures including early identification and management of persons with infection, tracing of contacts with monitoring for onset of fever and/or symptoms, and active engagement of communities.Most modelling suggests that the severity of illness is more like influenza than SARS, and there is concern among the public health community because the transmissibility of COVID-19 is not yet fully understood, and the potential for it to become endemic like other respiratory pathogens is unknown.Because of the many unknowns, the initial reaction by health agencies is still valid: to brace for the first wave as the virus reaches a completely naïve population and to make maximum effort to interrupt transmission [4, 5] .Experience with managing this outbreak will be very heterogenous across the world. Countries closely connected with China, such as Singapore, will be ahead in this regard. As the outbreak moves across regions, there is opportunity to support those affected later both in terms of readiness and response. Mechanisms available to outbreak response organisations, particularly through the Global Outbreak Alert and Response Network (GOARN), can be valuable in skills-and knowledgesharing. Therefore, there should be a deliberate effort to utilise knowledge from early affected countries in later affected countries.What measures are likely to be successful in curbing its further spread?On January 23rd, Hubei province in Central China was locked down with all movement in and out of the province blocked. Travel across China was discouraged and the number of scheduled flights and train journeys available considerably reduced to perhaps 10% of previous activity. Commercial and social activities became negligible, with schools, restaurants, other entertainment spots and most shops closed. Migrant workers were prevented from returning to work after the extended Chinese New Year Holiday. Frequent hand hygiene when in public and staying at home became the norm. This unprecedented public health effort by China has afforded the rest China and the world a lead time to prepare. The lockdown resulted in a downward trend in national and provincial epidemic curves, however, these measures are not sustainable and eventually there will be a strategy to return to normality. Should this result in a second wave of cases, and more international spread, countries around the world must be prepared. Early identification, and isolation or cohorting of positive cases to designated sites is at the core. In order to achieve this, hospitals, quarantine services, laboratories, together with epidemiology and communication teams will need to be scaled up to provide effective and efficient care.What is in the pipeline for vaccine development and/or therapeutics?It appears that no vaccine will be available for at least one year, likely a little longer. Phase 1 trials for safety and immunogenicity in human populations are likely within 3 months.In terms of therapeutics there is no known effective pharmaceutical agent. There are over 200 registered clinical trials registered in China alone. Putative agents include antivirals; Griffithsin, a spike protein inhibitor, nucleoside analogues eg. remdesivir, ribavirin and protease inhibitors such as lopinavir/ritonavir. Immunomodulatory and other host targeted agents include interferon, chloroquine and immunoglobulins. Corticosteroids will potentially have benefit for immune mediated lung damage late in the course of disease . Much of the theory stems from what we have learnt from limited trials in other corona viruses .\n",
            "\n",
            " Title:  Early epidemiological analysis of the coronavirus disease 2019 outbreak based on crowdsourced data: a population- level observational study\n",
            "4 \b. \n",
            "\n",
            "Input question: What is caused by SARS-COV2?\n",
            "Results (after 2.207 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.721\tSevere acute respiratory syndrome (SARS), caused by the coronavirus SARS-CoV, is an acute infectious disease with significant mortality. A typical clinical feature associated with SARS is pulmonary fibrosis and associated lung failure. In the aftermath of the SARS epidemic, although significant progress towards understanding the underlying molecular mechanism of the infection has been made, a large gap still remains in our knowledge regarding how SARS-CoV interacts with the host cell at the onset of infection. The rapidly changing viral genome adds another variable to this equation. We have focused on a novel concept of microRNA (miRNA)-mediated host-virus interactions in bronchoalveolar stem cells (BASCs) at the onset of infection by correlating the ''BASC-microRNome'' with their targets within BASCs and viral genome. This work encompasses miRNA array data analysis, target prediction, and miRNA-mRNA enrichment analysis and develops a complex interaction map among disease-related factors, miRNAs, and BASCs in SARS pathway, which will provide some clues for diagnostic markers to view an overall interplay leading to disease progression. Our observation reveals the BASCs (Sca-1+ CD34+ CD45-Pecam-), a subset of Oct-4+ ACE2+ epithelial colony cells at the broncho-alveolar duct junction, to be the prime target cells of SARS-CoV infection. Upregulated BASC miRNAs-17*, -574-5p, and -214 are co-opted by SARS-CoV to suppress its own replication and evade immune elimination until successful transmission takes place. Viral Nucleocapsid and Spike protein targets seem to co-opt downregulated miR-223 and miR-98 respectively within BASCs to control the various stages of BASC differentiation, activation of inflammatory chemokines, and downregulation of ACE2. All these effectively accounts for a successful viral transmission and replication within BASCs causing continued deterioration of lung tissues and apparent loss of capacity for lung repair. Overall, this investigation reveals another mode of exploitation of cellular miRNA machinery by virus to their own advantage.Severe Acute Respiratory Syndrome (SARS) is a new fulminant atypical pneumonia which emerged as a regional and global threat in 2002-2003 with a high mortality rate resulting from acute lung failure . The disease causing agent has been identified as a novel coronavirus termed as SARS-associated coronavirus (SARS-CoV) [2, 3] . The SARS-CoV is an enveloped virus containing a single stranded, positive-sense RNA genome which encodes 14 putative open reading frames encoding 28 potential proteins [4, 5] . These include four structural proteins, spike (S) glycoprotein, matrix (M) protein, small envelope (E) protein, and nucleocapsid (N) protein . These proteins have various roles in aiding the virus to enter the host and spread infection. While the incidence of new cases of SARS waned in 2003-2004, many aspects of SARS disease pathogenesis and host-pathogen interactions remain unsolved. Limited pathologic studies reveal that the major site of SARS-CoV infection and morbidity is the respiratory tract. The target organ of SARS is mainly lungs . Among the various animal models that have been used to study the pathogenesis of SARS-CoV infection, the monkey model mimics the clinical course of SARS to a certain degree . The cellular tropism of SARS-CoV in mouse lung has also been investigated by Ling et al., 2006 . But still today, very little is known regarding the mode of SARS-CoV interaction with host cells at the onset of infection in the lungs at a molecular level, and also the cell types in which the primary viral infection and replication occurs.The lung is an extremely complex, conditionally renewing organ which contains anatomically and functionally distinct epithelial stem cell populations which reside in distinct anatomical locations. The basal cells , Clara cells and type-II pneumocytes are the candidate stem/progenitor cells which can repair the injured lungs and contribute to local needs in times of tissue damage. Recently, Kim et al. have isolated a regional pulmonary stem cell population termed as bronchoalveolar stem cells (BASCs) residing at the bronchoalveolar duct junction of adult lungs . These have been identified and characterized as CD34 + Sca-1 + CD45 -PE-CAM 2 cells expressing both cytoplasmic Clara cell secretion protein (CCSP) and prosurfactant protein-C proteins, which are markers for Clara cells and type-II pneumocytes respectively . We have searched for the evidence to establish BASC as the prime target cells of SARS infection initiation and replication.To date, miRNA-mediated RNA interference is reported to be an essential tool to understand the regulatory pathways at molecular level that underlie infection biology [13, 14] . They not only participate in executive decisions but also perform much of the grunt work to micromanage protein output. Inspite of being such potential regulatory elements, miRNAs have so far missed attention in the pathogenesis of SARS-CoV infection.MiRNAs are small (19-25 nucleotides) endogenous noncoding RNAs that have been shown to influence the abundance and translational efficiency of cognate mRNAs [15, 16] . Viruses, which typically employ many components of the host gene expression machinery, also encode miRNAs. On the contrary, miRNAbiogenesis pathway poses some serious problem for RNA viruses and a group of DNA viruses (poxviruses) to encode miRNAs . SARS-CoV being an RNA virus is assumed not to encode its own miRNAs. But in due course of evolution, the virus might have developed certain highly sophisticated molecular mechanisms to exploit the cellular biosynthetic machinery of host cells and elude the cellular defense mechanisms. It is most likely that in certain situations SARS-CoV uses the cellular miRNA transcripts in order to foster their own agenda.Given the proposed involvement of BASCs in SARS disease , elucidation of the miRNA-mediated regulatory mechanisms responsible for making BASCs the prime targets of SARS infection might provide new avenues to explore the underlying mechanisms of SARS infection.The entry of SARS-CoV into cells is mediated through interaction between spike (S) glycoprotein of the virus and angiotensin-converting enzyme 2 (ACE2), the primary receptor of SARS-CoV on the host cell . ACE2 is a homolog of angiotensin-converting enzyme (ACE), which plays an important role in the renin-angiotensin system for blood pressure homeostasis. Recent studies of Li et al. have shown a protective effect of ACE2 against experimental lung fibrosis through its ability to degrade local tissue angiotensin II (ANG II) in response to bleomycin. Furthermore, in the pathogenesis of lung fibrosis, they have proposed the involvement of the down regulation of ACE2 as an integral component of the sequence of events leading to lung collagen deposition. The molecular mechanisms responsible for the loss of ACE2 in lung fibrosis are currently under investigation. A typical clinical feature associated with SARS is also pulmonary fibrosis and associated lung failure whose underlying mechanism still remains elusive . Investigations leading to the underlying molecular mechanisms responsible for the loss of ACE2 in pulmonary fibrosis might provide a clue to the reason behind fibrosis associated lung injury in SARS.Several reports suggest that there is a close interaction between SARS-CoV and respiratory epithelia which play an important role in the genesis of SARS . Jia et al. have investigated interactions between SARS-CoV and human airway epithelia and indicated that the state of cell differentiation and ACE2 expression levels are important determinants of the susceptibility of human airway epithelia to infection. To date, the molecular switches regulating the state of cellular differentiation of an infected cell and ACE2 expression have not been studied properly. In this work, we have proposed the role of miRNA in modulating the expression of ACE2 along with the stage specific cellular differentiation at the site of infection initiation. Further we have tried to focus on the role of SARS-CoV which dictates these events for its successful transmission and replication.The POU-homeodomain transcription factor Oct-4 (Pou5f1) plays a central role in self-renewal, pluripotency, and lineage commitment. Alterations in Oct-4 expression promote differentiation and leads to the specification of ectodermal, endodermal or mesodermal primitive progenitors . A small number of Oct-4expressing cells have been observed at the bronchoalveolar junction of the neonatal lung . We have investigated miRNAmediated modulatory role on Oct-4+ BASCs and have correlated their contribution to SARS infection.Our analysis establishes a complex interplay between miR-NAs, BASCs and certain essential factors related to the disease which might provide a better understanding of SARS pathogenesis. The multifarious relationships shared by genes related to the disease-system pathway, modulatory effects of miRNAs within the prime target cells of infection in response to indefinite cues impose the need of an interaction map. This study develops a complex interaction map between disease-related factors, miRNAs and BASCs in SARS disease pathway, which will provide some clues for diagnostic markers to view an overall interplay leading to disease progression. Further, our investigation towards investigating the therapeutic potential of miRNAmediated RNAi as an effective antiviral agent against SARS might unravel some of the efficient therapeutic measures against the deadly disease in future.We have investigated the cellular tropism of SARS-CoV (Table 1) in the BASCs present in lungs and attempted to establish the role of BASC-miRNAs towards understanding the pathogenesis of SARS in lung fibrosis. We describe the role of BASCs as the target cells of SARS infection and show how a repertoire of BASC miRNAs act as molecular switches to contribute to this host-pathogen interaction.Which is the primary site of SARS infection and how do these corona virus interacts with those cells to enter the host and eventually invades the host immune system to spread itself -is a pertinent question which prevails and is a debated issue. In order to throw light on this very important question, we looked upon at the SARS + cell markers and BASC markers (specific to each of its developmental stages from multipotent state to fully differentiated state).It has been reported that SARS-infected cells are unlikely to be pneumocytes (both type I or type II) or cytokeratin + epithelial cells and it is quite obvious since SARS-CoV cannot infect or replicate in a fully differentiated cell . Further, SARS + cells are distinct from cells expressing the macrophage/monocyte specific marker CD68. But SARS + cells are found to express the functional receptor ACE2 as well as the stem/progenitor cell marker CD34 and Oct-4 . Chen et al have also shown that SARS + cells are CD45 2 . Hence it has been confirmed that the SARS + cells in the infected lung were a subset of putative stem/ progenitor cells expressing CD34, Oct-4 and ACE2. Further, it is unlikely for SARS + cells to produce differentiated cell markers. The presence of SARS + cells within the multiple cell types at the bronchiolar lining layers makes it difficult to isolate SARS + cells. Some of the SARS + cells remains overlapped with adjacent cytokeratin + cells which might be mistakenly interpreted as colocalized cells. In such a situation BASC (Sca-1 + CD34 + CD45 2 Pecam 2 ) in terminal bronchioles located exclusively at the broncho-alveolar duct junction (BADJ) stands a good selection to be investigated as the prime target cells of SARS-CoV infection.A percentage of the Sca-1 + CD4 2 Pecam 2 cell populations at BADJ are CD34 + . This Sca-1 + CD34 + CD45 2 Pecam 2 population is enriched for BASCs became evident from the fact that it contained no ciliated cells. In addition to clonal colony formation, these cells exhibited extensive self-renewal in culture. They also had a greater capacity for differentiation compared to other lung epithelial cells. Sca-1 + CD34 + CD45 2 Pecam 2 BASC cultures further confirmed the multilineage differentiation capacity of BASCs. Within seven to ten days they show up positive for differentiated cell markers as they get differentiated into Clara-like cells (CCA + SP-C 2 ), AT2-like cells (SP-C + CCA) and AT1-like cells (AQ5 + ).Within the Sca-1 + CD4 2 Pecam 2 cells the other percentage of cells showed CD34 2 and contained ciliated cells. The Oct-4 + colony cells which are Sca-1 + CD34 2 SSEA-1 + cytokeratin + were shown to be succeptible to SARS-CoV infection by Ling et al. . These might be a subset of Sca-1 + CD4 2 Pecam 2 CD34 2 population ( Figure 1 ). Later studies have shown that cytokeratin + cells do not express the functional receptor ACE2 . Further, SARS + cells do not express cytokeratin and are CD34 + SSEA-1 2 . Phagocytosis is a common mode of virus entry within surrounding cells in SARS infection . Hence, presence of SARS antigen within the subset of Oct-4 + colony cells which are Sca-1 + CD34 2 SSEA-1 + cytokeratin + is most likely due to phagocytosis. Thereby, it is evident that these Oct-4 cells (Sca-1 + CD34 2 SSEA-1 + cytokeratin + ) are not the prime target cells for SARS-CoV infection.On the contrary, the BASCs (Sca-1 + CD34 + CD45 2 Pecam 2 ) must be another subset of these Oct-4 + ACE2 + epithelial colony cells (Figure 1 ) which provides a more favorable environment for SARS-CoV entry and replication. Absence of ciliated cells and CD34 + makes them the most probable targets of SARS-CoV at the onset of infection.Since SARS-CoV cannot infect or replicate in a fully differentiated cell, it is indeed essential for the virus to control the differentiation stages in an infected BASC cells so that they do not reach a fully differentiated state until the virus has undergone successful transmission and replication within this primary site of infection. Hence, it is important to monitor the expression levels of the different developmental stage specific markers of SARS infected BASC and also that of the corresponding regulatory switches that control their level of expression.MiRNAs are widespread agents of post-transcriptional gene silencing and have been strongly linked with stem cells [25, 26] . They exhibit a high degree of stage-and tissue-specificity, and therefore it is important to scan the profiling data for those miRNAs that operate during the narrow windows of development of a BASC in order to understand their regulatory impact on the expression levels of essential marker genes and transcription factors within the cell. In order to model the regulatory mechanisms within a SARS + cell, we have tried to understand the underlying regulatory mechanisms within a normal BASC cells mediated by the molecular switches-the miRNAs.It has been established that the virus exploits the host-miRNA milieu to foster their own agenda of evading the host immune system, thereby establish a consistent infection and continue replication.We undertook an intensive transcriptome-wide search for candidate mh-miRNAs (BASC-miRNAs which are homologous between mouse-human whole genomes) ( Table 2 ) targeting the developmental stage specific transcriptional factors and marker genes of BASCs. Table 3 shows the list of the miRNAs targeting the potential stage specific factors of BASCs. Further, we have also elucidated the mh-miRNAs targeting ACE2. The target factors have further been mapped with the expression levels of their corresponding miRNAs within BASCs which shows their integrated correlation contributing towards each developmental stage of BASCs in a healthy individual ( Figure 2 ). SARS-CoV definitely interferes with these miRNA mediated regulation to dictate the cell fate and continue its replication. Hence as a future advancement of this work, it will be worthy designing benchwork to detect the correlation of these miRNA-mRNA expressions within a SARS-CoV infected cell.We have searched for the host miRNA targets within the virulent genes of SARS-CoV. Table 4 includes the BASC-miRNA candidates targeting the most important virulent proteins responsible for viral infection. Interestingly, miRNAs-17*, -574- Normally presence of such host miRNA targets in SARS-CoV genome indicates that the corresponding miRNAs are a part of the host's innate antiviral defense. On the contrary, the RNA virus coopt host-miRNAs to suppress their own replication to evade immune elimination and establish a strong infection. It has been found that human infecting single-stranded RNA viruses are enriched for target sequences of human miRNAs . Or else rapidly evolving RNA viruses might evolve targets that match host miRNAs in order to increase their host specificity. It is very likely that the rapidly evolving virus take advantage of slowly evolving host miRNAs to increase their survival in the host, analogous to the manner in which viruses take advantage of other cellular processes. Co-option of host miRNAs by SARS-CoV to suppress their own replication will allow it to be in a latent state and escape the host immune system at the initial time point of transmission and infection. Once successful transmission within the host system takes place, antiviral miRNA target sequences in the virus can be expected to mutate rapidly to maximize mismatches and thus minimize the impact of 'antiviral' miRNAs. Rapid mutation in SARS-CoV is supported by the fact that SARS-CoV lack proof reading activity in their polymerases. RNA-dependent polymerase in SARS-CoV misincorporates 8.26 bases per million .While investigating the seed sequence conservation of the viralspecific host miRNAs ( Figure 3 ) we found that there is a strong bias towards G, Y(pYrimidine) and R (PuRine) bases at the 3 rd , 4 th and 5 th position respectively of the seed region. This implies that a single mutation within the complementary regions of this conserved 3-mer motif within the viral targets will disrupt the miRNA seed-target match and will enable the virus to escape the antiviral effect of host-miRNAs after successful transmission within the host. Such host-miRNA sequence conservation helps the virus to mutate the target regions with more precision and at ease so as to escape the effect of all the 4 miRNAs together.The miRNA expression profiles of three miRNAs-17*, -574-5p and -214 targeting S, N, Orf1a, M and E shows that they are upregulated within uninfected BASCs. Presumably at the onset of SARS infection the expression of these targets is suppressed which is likely to control viral replication and consequently help the virus to evade the host immune system. But the targets for downregulated miR-148a within ORF1a, M, S and E might escape repression and maintain replication at a lower rate. Further, the exclusive miRNAs viz. miR-223 and -98 for N and S protein respectively are highly downregulated (8.7 fold and 5.6 fold respectively) in BASCs. N and S proteins are important for viral transmission in host. This provides a clue towards the fact that the S protein and N protein (having targets corresponding to these miRNAs) of SARS-CoV takes advantage of this and escapes miRNA-mediated repression which is a rescue to the virus for effective transmission at the initial stage of viral infection.Key miRNA-mRNA pairs correlating with virus entry, replication, and host-virus interaction ACE2-S interaction. It has been shown that proinflammatory and fibrogenic cytokine pathways are activated within the first 24-48 hr following pulmonary insult resulting from SARS-CoV infection . High initial levels of these activities are associated with persistent pulmonary damage and increased risks of subsequent pulmonary fibrosis and poor outcome in diffuse [17, 18] . Studies have also been performed which support that differential regulation of host cell mRNA transcription and protein synthesis occurs in a SARS-CoV infected cells . SARS-CoV S protein is a multifunctional protein which plays pivotal roles in the biology and pathogenesis of SARS-CoV. It has been shown that S protein mediates viral infection by binding to cellular receptor ACE2 and thus induces membrane fusion. SARS-CoV infections and the Spike protein of the SARS-CoV reduce ACE2 expression . Further a recent work of Imai et al. showed that on injecting SARS-CoV Spike into mice worsens acute lung failure in vivo, which can be attenuated by blocking the renin-angiotensin pathway. This suggests that the S-protein mediated activation of the pulmonary RAS influences the pathogenesis of ALI/ARDS and SARS . But the molecular mechanism by which S protein down regulates ACE2 in lung cells following SARS injury is not well addressed till to date.In our work we observed that the miRNAs-miR-186,-93 and -200b targeting ACE2 are found to be highly down regulated in normal BASCs thereby showing that they do not interfere with the expression level of ACE2 and thus prevents lung fibrogenesis by limiting the local accumulation of the profibrotic peptide ANG II in an uninfected state. Considering the expression of S-protein, we find that its target corresponding to highly downregulated miR-98 probably escapes miRNA-mediated host RNAi defense at the onset of infection and interferes with ACE2 expression. Figure 4 shows how the virus takes advantage of the microRNA-mediated complex interplay between the host and viral factors within BASCs. The question still remains regarding the molecular mechanism by which this S-protein represses the expression of ACE2.S-proteins have been shown to act as transcription factor activating the promoters of GRP94/78 genes inducing ER stress in a SARS-CoV infected cell . The role of S protein as a transcription factor has obliged us to think that they might play a role to activate the miRNA promoters of miR-186,-93 and -200b targeting ACE2 to enhance their expression resulting in a reduced expression of ACE2 in an infected host. Whether this decrease in ACE2 expression is a result of miRNA mediated gene silencing controlled by S-protein will be an interesting and compelling question for further study.Oct-4-N interaction. Oct-4 plays a crucial role towards regulating the chromatin structure in a state consistent with selfrenewal and later facilitating the expression of genes that keeps the cell poised to respond to cues that lead to differentiation. Hence this is one of the essential factors which control the state of differentiation in BASCs. Furthermore, it has a significant role in apoptosis. miR-146b-5p is seen to target Oct-4 in normal BASCs. The downregulated expression of this miRNA explains for the uninterrupted expression of Oct-4 resulting its normal function in an uninfected cell. We have harnessed the high possibility of a complicated crosstalk between the ability of N-protein to control the expression of Oct-4 in an infected cell.N protein is known to be the most abundantly expressed protein of SARS-CoV. The most unique and significant property of it is revealed in its ability to act as a sequence specific DNA binding factor. It has been reported to bind NFkB response element of COX2 promoter and enhance COX2 gene expression which effectively is beneficial for its existence within the host. This reveals the power of the N-protein to manipulate the entire gene expression programme of the infected cell. The common miRNAs, miR-214,-574-5p and -17* having targets in N protein are upregulated in BASCs. But miR-223, the unique miRNA targeting N-protein is highly downregulated. This signifies that the Nprotein targets corresponding to miR-223 is going to escape miRNA-mediated downregulation at the onset of infection (See Figure 4) . Further it seems that N-protein plays a role in interfering with the expression modulation of Oct-4 by binding to the promoter of miR-146-5p and controls the different stages of BASC differentiation as well as prevents apoptosis of SARS infected cells. Stem Cell Antigen-1 is identified as a potential marker for BASCs located at the BADJ [12, 27] .OCT4 (POU5F1) miR-146b-5p Octamer-binding transcription factor 4+ (Oct-4) is a stem cell marker protein. It acts as a master switch in differentiation by regulating cells that have pluripotent potential.Recently, it has been found to be expressed in pulmonary cells which are a target for SARS infection [8, 23] .CD34 let-7g let-7d let-7cThese are membrane-bound stem cell markers for BASCs. This is also expressed in SARS-infected lung cells [23, 28] .Marker protein of alveolar type-2(AT2-like) cells which is a subset of the differentiated BASC population. Hence is quiescent in normal lung but proliferate in response to bronchiolar and alveolar epithelial injury [29, 30] .This is the Clara cell marker protein. These are also quiescent in normal lung and proliferate in response to bronchiolar and alveolar epithelial injury .Aquaporin5 miR-331-3p miR-497Marker protein of AT1-like cells which is another subset of the differentiated BASC population. Aquaporin5 plays a critical role in the maintenance of normal lung water homeostasis [30, 31] .Angiotensin-converting enzyme 2 (ACE2), the primary receptor of SARS-CoV on the host cell. It is a negative regulator of the renninangiotensin system (RAS) in the setting of acute lung injury and in response to pulmonary infection with the SARS-CoV [17, 32] .Inflammatory chemokine receptor for CCL3 and CCL5 & responsible for lung fibrosis [33, 34] .We have predicted the differentially expressed mh-miRNAs which operate within the narrow windows of development of BASCs by targeting a set of host encoded mRNAs. Class A specifies the host mRNAs corresponding to BASC marker proteins which designates its undifferentiated state as well as the state of proliferation and/initiation of differentiation. Class B specifies the host mRNAs corresponding to BASC marker proteins as it reaches a fully differentiated state. Class C specifies the host mRNAs corresponding to the receptor proteins which participates in SARS-CoV pathogenesis. These miRNA-mRNA pairs specific to BASC are important to understand hostvirus interaction network. doi:10.1371/journal.pone.0007837.t003Immunity-related issues-induction of inflammatory chemokines in SARS-CoV-infected cells might be an advantage for the virus to replicate All respiratory virus disease results from two concurrent pathological components: ongoing virus replication and the resulting inflammatory response. Even if antivirals clearly inhibit virus replication, the biochemical and cellular inflammatory responses to the initial infection related events continue despite diminished virus titer . While acute inflammatory responses are generally beneficial in nature and have been shown to limit virus replication in situ, prolonged, uncontrolled inflammation has been recognized as a significant component contributing to the pathological sequelae to the morbidity and mortality of severe acute respiratory syndrome coronavirus infection . A histopathologic analysis of SARS-CoV infected lung in mice showed that there has been an induction of significant changes in the expression levels of the chemokine receptor CCR1 which acts as receptors for CCL3 and CCL5. The expression levels of these inflammatory chemokines were increased by viral infection of the lung . It has been shown that CCR1 play an essential role in the pathogenesis of pulmonary fibrosis in lung-injury models . Further, previous work suggests that antagonism of signaling via the chemokine receptor CCR1 is a potent strategy that attenuates leukocyte recruitment in response to severe respiratory virus infection .Although there has been a subsequent increase in inflammatory cytokines related to innate immune response in response to viral infection, there has been a lack of immunoregulatory cytokines (T or B lymphocytes) in the lung during infection which confirms the fact that the expression of viral replication proteins remains at a quiescent state so as to avoid viral clearance by the activation of immunoregulatory cytokines by adaptive immune responses at the onset of the infection. On the contrary, activation of inflammatory chemokines might be acting as a beneficial phenomenon for the replication of various viruses [51, 52] .Emerging studies suggest that miRNAs provide an added layer in orchestrating immune responses . miRNAs function in shaping immunity by regulating the repertoire of genes expressed in immune cells and the magnitude and duration of responses to particular pathogens . Hence we investigated the underlying miRNA induced molecular mechanisms responsible for these changes in the lung and we found that miR-223 has its target in the 39-UTR of CCR1. Previously, miR-223 was experimentally identified in chorioamnionitis-related inflammation and its predicted target genes include several genes involved in inflammation and immune responses. Further, miR-223 is known to promote granulocyte differentiation and is a part of a regulatory loop that involves C/EBP and NFI-A .Interestingly miR-223 has its target in N protein of SARS-CoV too and it is one of the miRNAs which probably helps the virus to escape the RNAi mediated silencing at the onset of infection. Or in another way we may say that the virus encodes the target of miR-223 to escape the RNAi mediated repression and at the same time takes advantage of the effect of induction of the inflammatory chemokine responses to accelerate lung fibrosis. But the controlled expression of target CCR1 in an uninfected cell inspite of the significantly low expression of its corresponding miR-223 in BASCs suggest that there must be a cascaded fine tuned interaction masterminded by the virus which is going on within the infected cell. It is known that N protein activates NF-KB in Vero E6 cells . CCR1 is one of the target genes of NF-KB which is activated at a later stage . Hence, we propose that within BASCs, reduced expression of miR-223 helps N protein to enter the host cell. On successful entry, N protein might be using miR-223 once again to activate CCR1 via NF-KB at its replicative stage to enhance lung fibrosis and at the same time gets empowered to manipulate the entire gene expression of the host cell by controlling the expression of Oct-4. The exact mechanism of such miRNA-mediated events needs further investigation. Further, checking the expression levels of N-protein, CCR1 as well as miR-223 in an infected BASC will provide a more compact conclusion.Based on these findings, we propose that BASCs (Sca-1 + CD34 + CD45 2 Pecam -) are a subset of Oct-4 + ACE2 + cells and are the chief targets of SARS-CoV infection. SARS-CoV enters the host; infects the BASCs of the lungs and controls its developmental stages via the molecular switches-miRNAs. Co-option of host miRNAs by viruses reveal their intelligent plan to control their replication in order to evade immune elimination until they undergoes successful transmission and establish a strong infection. Thereafter, they undergo rapid mutation to maximize the target-miRNA mismatches and enhance their replication before the cell reaches at a fully differentiated state. On successful replication, the virus infects resident, infiltrating, and circulating immune cells. The circulating immune cells carry the virus to other organs and causes damages to the immune cells of spleen, peripheral and central lymph nodes and other lymphoid tissues. The immune defense being weakened significantly, leads to rapid deterioration of the pneumonia. It is worth to test these cells for the expression of Oct-4 and ACE2 for further confirmatory conclusion.The proposed host miRNA-dependent mechanism probably acts in concert with the host and viral factors for establishing a strong infection. Our proposal shifts away from the simplistic notion that host miRNAs with specific sequence complementarity to viruses are indicative of a bonafide innate antiviral immune mechanism to an established viral pathogen.Although the immediate threat of SARS is over, we intend to continue with its therapeutic measures partly to guard for its possible return. Moreover, this will help us to transform our knowledge from SARS to other emerging viral diseases specially causing lung injuries which most certainly will be a deadly threat in future. Lot of attempts have been made and is still going on to design effective antiviral agents against this deadly disease . While some of these approaches showed partial efficiency to combat SARS infection, the others need further clinical investigations to be used as a proper therapeutic reagent.This era has witnessed an increase in the development of exogenous siRNA therapeutics against viruses. The potential therapeutic effect of modulating host-miRNA levels is also worth considering. This emerging picture of miRNA regulation in SARS-CoV infected individual will make the host-virus interaction far richer and more complex than the crisp linear pathways of the previous decade, with miRNAs participating in executive decisions at the interesting and vulnerable nodes in regulatory networks. By boosting the level of the host virus-specific miRNAs, it might be possible to turn what was a desirable outcome for the virus into a desirable outcome for the patient. Under the current hypothesis, we would predict that decreasing the level of host virus-specific miRNAs would result in promotion of viral replication, leading to immunological recognition and clearance of the virus by the host immune system. Further, increasing the level of miRNAs exclusive for the N and S protein might also block viral entry. However, manipulating the level of host miRNAs could have unintended consequences because the physiological functions of the miRNAs might be altered or viral pathology might be enhanced. Further silencing of miRNAs with antagomirs specifically within the infected cell could become another therapeutic strategy where miRNAs participate in disease aetiology. Nevertheless, these potential interventions merit further evaluation.To identify the miRNAs involved in the cellular tropism of SARS-CoV in lungs, we undertook an intensive search for potential miRNA candidates in the BASCGAP (Broncho-alveolar Our search strategy was motivated by the need to apply purehomology based selection of miRNAs so that the results represent a closer approach to mimic the molecular mechanisms that underlie disease pathway behaviors and response in a human host. Further the overall miRNA expression profile of mouse and human lungs are similar . From the BASCGAP sequence data of 116 miRNAs, we screened out the BASC-miRNAs lying at the intersection of mouse-human whole genomes. This filter is based on a homology search (SSEARCH), performed with miRBase (release 12.0) and is applied on the mature candidates from the BASCGAP sequence data. E-value cutoff considered for SSEARCH was ,10.0. Mature miRNA sequence homology was one of the stringent parameters considered in our search. Table 2 represents the screened output set of 95 mature candidates termed as mh-miRNAs.We have considered the developmental stage specific marker proteins of BASCs, which includes the transcription factor Pou5f1; ACE2-the most important receptor of SARS-CoV (Table 3 ). In addition to these, we have chosen the virulent protein coding genes, viz. Nucleocapsid (N), Spike(S), Envelope (E), Membrane (M) and Orf1a from 30 strains of SARS-CoV (Table 1 and Table 4 ) for our investigation. Annotated UTR sequences of the target genes were retrieved from UTRdb .We adopted RNAhybrid , that predicts the most favorable hybridization sites between miRNA and UTR regions and generates energy minimized duplex structures using the Dynamic Programming technique. The resultant miRNA-mRNA duplexes are consistent with the following structural and energy constraints at the first step of screening the targets: (i) Binding energy cutoff , = 220 kcal/mol ; (ii) complementarity of minimum seven bases to the 59 end of the miRNA; (iii) minimization of GU base pair within the miRNA-target duplex, a maximum of only one GU base pair between 2 and 7 position of the miRNA is allowed; (iv) minimization of loop size throughout the duplex.Although, currently available target prediction methods are diverse, both in approach and performance and all have room for further improvement, perfect seed pairing centered on nucleotides 2-7 is considered as the most important criteria for prediction reliability. Furthermore, in order to screen out the most favorable target in the case of miRNAs having multiple targets we incorporated the constraint of site efficacy (hierarchy of site efficacy follows-8 mer.7 mer-m8.7 mer-A1.6 mer) following the work of Grimson et al. and Friedman et al. [65, 66] . 39 compensatory pairing of miRNA to its target have been considered to compensate for imperfect/weak seed matches . We have further checked the effect of cooperativity as well as the UTR binding location.For bantam miRNAs, good minimum free energies (MFEs) can occur frequently by chance. The longer a putative target sequence, the better such random energies will be. Hence, statistical significance of predicted targets is assessed with an extreme value statistics of length normalized minimum free energies and a Poisson approximation of multiple binding sites.Microarray data analysis in BASCs miRNA expression profile of 2610 6 mouse BASCs (CD45 2 CD31 2 CD34 + Sca-1 + ) and 4610 6 control cells(CD45 2 CD31 2 CD34 2 Sca-1 2 ) are determined using miRNA microarray. The data considered for our analysis is obtained by first subtracting the background and then normalizing the signals using a LOWESS filter (locally weighted regression). Background is determined using a regression-based background mapping method . The miRNA dataset was filtered according to the standard procedure to exclude spots with minimum intensity and size. Further for the dual-sample experiments, the ratio of the two sets of detected signals (log2 transformed, balanced) are calculated. Thereafter, t-values are calculated for each miRNA between the test and the control groups, and P-values are computed from the theoretical t-distribution. miRNAs with P-values below a critical P-value (typically 0.01) are Figure 5 . miRNA-mRNA prediction pipelines. This flow chart summarizes the steps followed and yields from BASCGAP consortium sequences to predict the miRNA-mRNA pairs involved in SARS pathogenesis. doi:10.1371/journal.pone.0007837.g005 the differentially detected signals and were selected as our global miRNA expression profile data set (of mouse BASCs) in BASCGAP. The flowchart of methodology is provided in Figure 5 .\n",
            "\t0.719\tThe SARS (severe acute respiratory syndrome) outbreak was caused by a coronavirus (CoV) named the SARS-CoV. SARS pathology is propagated both by direct cytotoxic effects of the virus and aberrant activation of the innate immune response. Here, we identify several mechanisms by which a SARS-CoV open reading frame (ORF) activates intracellular stress pathways and targets the innate immune response. We show that ORF8b forms insoluble intracellular aggregates dependent on a valine at residue 77. Aggregated ORF8b induces endoplasmic reticulum (ER) stress, lysosomal damage, and subsequent activation of the master regulator of the autophagy and lysosome machinery, Transcription factor EB (TFEB). ORF8b causes cell death in epithelial cells, which is partially rescued by reducing its ability to aggregate. In macrophages, ORF8b robustly activates the NLRP3 inflammasome by providing a potent signal 2 required for activation. Mechanistically, ORF8b interacts directly with the Leucine Rich Repeat domain of NLRP3 and localizes with NLRP3 and ASC in cytosolic dot-like structures. ORF8b triggers cell death consistent with pyroptotic cell death in macrophages. While in those cells lacking NLRP3 accumulating ORF8b cytosolic aggregates cause ER stress, mitochondrial dysfunction, and caspase-independent cell death.In 2002-2003 the SARS-CoV caused a severe respiratory illness affecting more than 8000 individuals with a mortality rate near 10% 1,2 . The subsequent identification of a large pool of coronaviruses circulating in bats and other animals portended the appearance of other highly pathogenic CoVs 3 , and in 2012 the Middle East Respiratory Syndrome-CoV caused a similar outbreak with a higher mortality rate 4 . Severe SARS-CoV infection manifests clinically as acute lung injury associated with high initial virus titers, macrophage/neutrophil accumulation in the lungs. and elevated proinflammatory serum cytokines (IL-1, IL-18, IL-6, IL-8, and MCP-1) . Recent advances have implicated inflammatory monocyte-macrophages (IMMs) in the lungs as critical mediators of SARS-CoV pathology, as a delayed type I interferon response promotes high initial virus titers and aberrant IMM recruitment 9 . Host disease is likely a combination of direct viral damage and consequences of an aberrant immune response promoted by IMMs 9 , underscoring the importance of determining the mechanisms by which the virus targets innate immunity.The SARS-CoV is an enveloped, positive-strand RNA virus that encodes a set of accessory proteins, several of which target the innate immune response. Open reading frame (ORF) 8a and ORF9b trigger cellular apoptosis; ORF7a activates nuclear factor-κB (NF-κB); ORF3b upregulates the expression of several cytokines and chemokines; ORF6 limits interferon production; ORF3a induces necrotic cell death; and ORF8b induces cellular DNA synthesis and suppresses the expression of the viral envelope protein 10, 11 . Recently, we found that ORF9b localizes to mitochondrial membranes and reduces mitochondrial-associated adapter molecule MAVS, severely limiting the interferon response 12 . Of note, ORF8b is of interest as ORF8 encoded a single polypeptide during the early phase of the SARS epidemic, while in the later stages a 29-nucleotide deletion split it into two ORFs, ORF8a and ORF8b. They encode 39-and 84-residue polypeptides respectively. The splitting of ORF8 is thought to confer evolutionary advantage to the virus, and accordingly virus expressing ORF8b is better able to replicate in the presence of interferon 10, 13 .Initially, two studies had difficulty expressing the ORF8b protein, suggesting it may be rapidly degraded in cells 14, 15 . However, antibodies raised specifically against the ORF8b protein detect low levels of expression as early as 8 h (h) post infection, while expression is prominent by 24 h 16 . Immunostaining ORF8b after expression in Vero E6 cells localized it to cytosolic punctate vesicle-like structures reminiscent of intracellular aggregates 17 . Here, we investigate the cellular mechanisms by which ORF8b contributes to SARS-CoV pathology. We report that ORF8b forms intracellular aggregates dependent on a valine at residue 77, which contributes to the induction of lysosomal stress, autophagy, and eventual cell death. Given the importance of IMMs in SARS-CoV pathology, the link between intracellular aggregates and inflammasome activation 18 , and the observation that SARS-CoV infected patients have elevated serum levels of IL-1 and IL-18 19 , we tested whether ORF8b might affect inflammasome activation in macrophages and lung epithelial cells. We found that ORF8b triggers robust NLRP3 inflammasome activation and IL-1β release. NLRP3 activation was accompanied by direct binding of ORF8b to the LRR domain of NLRP3. In sum, this study suggests that ORF8b activates cell stress pathways by forming intracellular aggregates and may be important in the aberrant activation of macrophages during SARS-CoV infection.As previous studies have shown that ORF8b forms vesicle-like puncta in the cytosol, we used the bioinformatics aggregation prediction server PASTA2 (http:// ) 20 to investigate the intrinsic aggregation propensity of ORF8b. Analysis of the ORF8b sequence predicted a tendency to aggregate based on a carboxy-terminal VLVVL motif (aa 75-79), while reanalyzing the ORF8b sequence with V77 mutated to K (V77K) reduced its predicted propensity to aggregate (Fig. 1a) . To empirically test the aggregation tendency of ORF8b, we expressed GFP-ORF8b (GFP-8b) in Hela cells overnight and imaged the transfected cells by confocal microscopy. While some cells had a homogenous pattern of expression, other GFP-8b expressing cells showed intracellular patch and fiber aggregation patterns (Fig. 1b) . Quantification showed a considerably higher percentage of cells with intracellular aggregates in GFP-8b expressing cells than GFP vector (GFP) expressing cells (Fig. 1c) . Presumably, the homogenous expression of GFP-8b precedes the development of intracellular aggregates. These findings were replicated in A549 cells, a human lung epithelial cell line (Fig. 1b) .Aggregated proteins often form detergent insoluble deposits. Thus, we separated the triton soluble and insoluble fractions of HeLa cells expressing either GFP, GFP-ORF8b, or mutant GFP-8b V77K to determine if ORF8b forms insoluble aggregates. To separate these fractions, we lysed the cells in a Triton-X based buffer and spun down the resultant lysates at 14,000 rpm for 10 min (min). The supernatant was collected and considered the triton soluble fraction, while the pellet was reconstituted in an LDS based buffer and heated at 100°C for 30 min. While preparing the triton insoluble fraction, we consistently observed that addition of LDS and heating for 30 min was unable to completely solubilize the pellet from GFP-8b expressing cells, but completely solubilized the pellet from GFP and GFP-8b V77K expressing cells. Accordingly, analysis of each fraction by SDS-PAGE followed by immunoblotting for GFP showed considerable monomeric and oligomerized GFP-8b in the insoluble fraction, while significantly less monomeric and almost no oligomerized protein was seen in the GFP-8b V77K insoluble fraction (Fig. 1d) . The control GFP was undetectable in the insoluble fraction and seen only in the soluble fraction (Fig. 1d) . Moreover, we observed consistently less GFP-8b V77K expression in the soluble fraction compared to GFP-8b, indicating that GFP-8b's ability to aggregate may be protective against degradation (Fig. 1d) .Intracellular protein aggregates and misfolded proteins cause ER stress and induction of the unfolded protein response (UPR), which can be quantified by detecting the upregulation of the UPR protein CHOP 21, 22 . Confocal imaging showed that aggregated GFP-8b caused ER dispersion as assessed by immunostaining for ERp72 (Fig. 2a) . The expression of GFP did not affect the usual juxta nuclear localization of the ER while expressing GFP-8b disrupted the usual ERp72 localization. Immunoblotting showed greater CHOP induction after transfection of GFP-8b than either GFP or GFP-8b V77K (Fig. 2b) , indicating that GFP-8b forms protein aggregates dependent on its valine 77. As intracellular protein aggregation is toxic once the aggregates overwhelm cellular homeostatic mechanisms, we looked at cell death by Trypan blue uptake following expression of GFP, GFP-8b, and GFP-8b V77K. Expression of GFP-8b induced significantly more cell death than did GFP, while the GFP-8b V77K was partially protective (Fig. 2c) . Taken together, this data suggests that GFP-8b forms insoluble protein aggregates and that can cause cell death, which at least partially depends on its ability to form intracellular aggregates.ORF8b induces lysosomal stress, the nuclear translocation of TFEB, and the activation of Transcription factor EB target genes Aggregated proteins are typically cleared from the cytosol by the autophagy-lysosome pathway, which engulfs bulk cargo in the cytoplasm and delivers it to the lysosome for degradation 23, 24 . Formation of intracellular protein aggregates can disrupt this pathway, as aggregated proteins are poorly degraded by lysosomes. Lysosomal protein buildup then results in lysosomal stress and compensatory activation of transcription factor EB (TFEB), the master transcriptional regulator of autophagic and lysosomal machinery 25, 26 . To determine if ORF8b disrupts autophagic and lysosomal homeostasis, we expressed GFP-8b and mCherry-galectin 3 in HeLa cells followed by imaging via confocal microscopy. Under homeostatic conditions galectin-3 is localized diffusely throughout the cytosol, but rapidly forms puncta on lysosomes following lysosomal stress/damage 27 . Accordingly, GFP expressing cells showed diffuse cytosolic galectin-3 distribution, while many GFP-8b expressing cells showed clear galectin-3 puncta formation, some of which co-localized with GFP-8b (Fig. 3a) .Correspondingly, expression of GFP-8b with mCherry-TFEB showed prominent TFEB nuclear localization compared to the control (Fig. 3b) . Immunoblotting endogenous TFEB after separation of the nuclear and cytosolic fractions in GFP-8b expressing HeLa cells showed significantly more TFEB in the nucleus than control cells, and levels of the TFEB target gene LAMP1 were consistently increased (Fig. 3c) . Confocal microscopy of GFP-8b expressing cells showed large, prominent lysosomes as assessed by LAMP1 immunofluorescence (Fig. 3d) . However, we noted only moderate overlap between the LAMP1 immunostaining and the GFP-8b aggregates. TFEB nuclear translocation following both ER and lysosomal stress has been reported to depend upon dephosphorylation of its S211 residue by the phosphatase calcineurin (PPP3CB) . To determine the mechanism of TFEB activation in GFP-8b overexpressing cells, we immunoblotted endogenous TFEB following nuclear fractionation of GFP, GFP-8b, and GFP-8b V77K expressing cells. The GFP-8b expressing cells were treated or not with the calcineurin inhibitor cyclosporin A. Again, GFP-8b induced TFEB nuclear translocation and increased LAMP1 expression, both of which were reduced by treatment with cyclosporin A. Expression of GFP-8b V77K did not increase nuclear TFEB levels or increase the level of LAMP1 in the cytosolic fraction (Fig. 3e) . These data indicate that ORF8b aggregates causes ER and lysosomal stress, which leads to the nuclear translocation of TFEB, and the upregulation of TFEB target genes.Activation of TFEB increases both lysosomal biogenesis and autophagic flux 31 , thus we checked to see whether GFP-8b affects autophagy downstream of TFEB. Expression of GFP-8b induced clear formation of RFP-LC3 puncta, while GFP did not (Fig. 4a) . In some cells the RFP-LC3 defined autophagosomes strongly co-localized with the GFP-8b aggregates while in other cells many of the aggregates did not (Fig. 4a ). An increase in LC3 puncta can be due to an increase in autophagosome formation or a decrease in autophagosome/lysosome fusion 32 . During autophagosome formation, LC3 is lipidated and converted from LC3-I to LC3-II. We expressed GFP or GPF-8b in the presence or absence of Bafilomycin A1, which blocks autophagosome-lysosome fusion, and immunoblotted the cell lysates for LC3. We found that the cell lysates from the GFP-8b and GFP expressing cells had similar LC3-II Fig. 2 ORF8b induces ER stress and cell death. a Confocal microscopy images of HeLa cells transiently transfected with either GFP or GFP-8b and immunostained for the endoplasmic reticulum marker ERp72 (red). Nuclei were counterstained with DAPI and cells were fixed prior to imaging. Individual and merged images are shown, and the scale bar is 5 μm. b Immunoblot looking at the accumulation of the ER stress marker CHOP after transient transfection of GFP, GFP-8b, or GFP-8b V77K in HeLa cells. c Quantification of cell death by Tryptan Blue uptake after overnight transfection of HeLa cells with the indicated proteins. A minimum of three fields with >50 cells per field were quantified. Statistical significance evaluated with Student's t-test (*p < 0.05, **p < 0.01). Each experiment repeated a minimum of three times levels. However, bafilomycin A1 treatment increased the LC3-II levels in the GFP-8b expressing cells more than it did in the control cells (Fig. 4b) . These results indicate that GFP-8b induces autophagic flux. Treatment with bafilomycin A1 increased the GFP-8b levels, but not GFP levels suggesting that ORF8b aggregates are partially degraded by the autophagy-lysosome pathway. Taken together, the GFP-8b intracellular aggregates trigger cellular stress. This results in a calcineurin dependent activation of TFEB and TFEB target genes, which leads to an increase in autophagic flux.Intracellular protein aggregation and lysosomal stress have previously been mechanistically linked to NLRP3 activation 33, 34 . As SARS patients show increased serum IL-1β and IL-18 7,8 , we investigated whether ORF8b affects inflammasome activation in macrophages. To do so, we first attempted to generate a permanent cell line expressing GFP-8b in the human macrophage cell line THP-1 cells. While we easily established GFP expressing THP-1 cells, no cells survived the selection process when we attempted to express GFP-8b. This is likely secondary to the toxicity of the protein, as even attempts at using an inducible promoter failed to generate a useable cell line. Therefore, we switched to a transient transfection system. Evaluating inflammasome activation in macrophage cell lines following transient DNA transfection can be problematic as cytosolic DNA activates DNA sensing inflammasomes. However, we found that that by carefully titering the amount of DNA transfected, we could express the GFP vector and yet trigger minimal Il-1β production. Therefore, we compared IL-1β production by THP-1 cells transiently transfected with low amounts of the vectors producing GFP, GFP-8b, 8b-GFP, or GFP-8b V77K. We Confocal microscopy images of GFP or GFP-8b co-transfected with RFP-galectin 3 (a) or mCherry-TFEB (b). Individual and merged images are shown, and the scale bar is 5 μm. c The percentage of cells with predominantly nuclear TFEB from part b was quantified for each condition (three fields, minimum of 50 cells/field). **p < 0.01 with Student's t-test. d Immunoblot to assess LAMP1 and TFEB expression in nuclear and cytosolic fractions separated by SDS-PAGE. HeLa cells were transiently transfected with GFP or GFP-8b. e Representative confocal images from HeLa cell expressing GFP or GFP-8b and immunostained for LAMP1 expression. Individual and merged images are shown, and the scale bar is 5 μm. f Immunoblot of cytosol and nuclear extracts from GFP, GFP-8b, or GFP-8b V77K transfected HeLa cells treated overnight with or without the calcineurin inhibitor cyclosporin A (CsA-400 nM). Images and blots are representative data from experiments repeated a minimum of 3 times found that both GFP-8b and 8b-GFP increased the amount of IL-1β present in the cell supernatant compared to GFP alone (Fig. 5a) . Surprisingly, the GFP-8b V77K construct also enhanced IL-1β production suggesting that the mechanism did not solely depend upon lysosomal stress (Fig. 5a) . We also checked whether an alternatively tagged version could trigger IL-1β production by expressing Flag tagged version of ORF8b. Similar to the results with the GFP-tagged ORF8b, we found that 8b-Flag induced 3-fold more Il-1β than did the Flag vector (Fig. 5b) . A complementary approach to assess inflammasome activity is to reconstitute the requisite components in HEK293T cells. Therefore, we transfected HEK293T cells with expression vectors for NLRP3, caspase-1, ASC, and full-length IL-1β in the presence of GFP or GFP-8b. A similar experiment was performed using FLAG or 8b-FLAG. In both instances we found an increase in IL-1β in the supernatant and a decrease in fulllength IL-1β in the cell lysate in the presence of the ORF8b constructs versus the controls (Fig. 5c) . These results suggest that ORF8b activates NLRP3 inflammasomes in macrophages and likely in monocytes.In NLRP3 expressing cells, signals that activate the NLRP3 inflammasomes cause the disassembly of the trans-Golgi network (TGN) and the recruitment of NLRP3 to the dispersed TGN, which serves as a scaffold for NLRP3 aggregation 35 . This leads to the polymerization of the adapter protein ASC and recruitment of caspase-1.To assess the impact of GFP-8b on the localization of NLRP3, we expressed NLRP3-DsRed in HeLa cells in the presence or absence of GFP-8b. In the absence of GFP-8b NLRP3 expressed homogenously throughout the cytosol of transfected HeLa cells, however the presence of GFP-8b dramatically altered NLRP3 localization (Fig. 5d) . We found that both proteins co-localized in the cytosol in small cytoplasmic dots as well as in a large perinuclear aggregate. We performed a similar experiment using HEK 293T cells (Fig. 5e ). In the left panel the images show a HEK 293T cell expressing NLRP3 alone and one expressing both proteins, while the right panel shows another cell co-expressing NLRP3 and GFP-8b. We found that GFP-8b forms large aggregates in HEK 293T cells and that its expression disturbed the homogenous cytosolic pattern of NLRP3, and as we had observed with HeLa cells both proteins showed an overlapping expression pattern. We also checked the distribution of the trans-Golgi network in HeLa cells co-expressing GFP-8b and NLRP3-DsRed by immunostaining for TGN38 (Fig. 5f ). We found partial co-localization between the NLRP3-DsRed and GFP-8b cytoplasmic dots and TGN38.In macrophages, NLRP3 also adopts a homogenous pattern until activating signals rapidly trigger the dispersion of the TGN and recruitment of NLRP3 in a punctate pattern. As macrophages also express ASC, the punctate NLRP3 distribution rapidly redistributes into a single speck that contains NLRP3 and ASC. When we expressed GFP-8b in LPS activated THP-1 cells we found that a major portion of the GFP-8b localized in a single speck that co-localized with endogenous NLRP3 and ASC (Fig. 5g) . These results also suggested that ORF8b targets NLRP3.Enterovirus 71 (EV71) 3D protein stimulates NLRP3 inflammasome activation by interacting with NLRP3 to facilitate inflammasome complex assembly. EV71 3D interacted with NLRP3 through the NACHT and LRR domains. To test whether ORF8b might also interact with NLRP3, we transiently expressed 8b-Flag in PMA differentiated and LPS stimulated THP-1 cells and tested whether we could detect an association with endogenous NLRP3. We found that immunoprecipitated 8b-Flag pulled down endogenous NLRP3, while control immunoprecipitates did not (Fig. 6a) . To characterize the region in NLRP3 domain responsible for the interaction, we examined the binding of Myc-tagged truncated NLRP3 proteins to 8b-Flag. In addition to full-length NLRP3, we used constructs that express NLRP3 with a deleted leucine-rich repeat (LRR) domain (NLRP3ΔLRR), a deleted Pyrin domain (NLRP3ΔPYD), or deleted NACHT and LRR domains (NLRP3ΔLRRΔNACHT). Following transient expression, 8b-Flag interacted with both fulllength NLRP3 and the PYD domain deleted protein, but not the LRR and LRR/NACHT domains deleted proteins (Fig. 6b) To directly show that ORF8b interacts with the LRR domain of NLRP3, we expressed the NLRP3 LRR domain as a Myc tagged protein (amino acids 537-991) and collected Myc immunoprecipitates following GFP-8b or GFP expression. We found GFP-8b, but not GFP in the Myc-immunoprecipitates (Fig. 6c) . Next, we expressed the non-functional NLRP3 LRR domain and determined whether its presence affected the induction of mature IL-1β secretion by ORF8b. We found that expression of the NLRP3 LRR domain impaired the production of mature IL-1β after GFP-8b expression in inflammasome reconstituted HEK 293T cells (Fig. 5e) , confirming the functional importance of this interaction. In summary, our data indicates that ORF8b activates the NLRP3 inflammasome and directly targets the NRLP3 LRR domain.Though the exact mechanistic role of ORF8b in SARS-CoV pathogenesis remains unclear, it is well established that the splitting of ORF8 corresponds with the shift from the early to the middle stage of the SARS epidemic, with the majority of affected patients being infected by virus containing the split ORF 36 . Recent advances in mouse models have shown that SARS pathogenesis is driven by high initial virus titers resulting from a late interferon response, which drives aberrant recruitment of IMMs and activation of the innate immune response resulting in cytotoxicity 9 . Given the likelihood that ORF8b is evolutionarily adaptive for the virus 10 , it follows that ORF8b may contribute to SARS-CoV pathogenesis by impairing the host interferon response and/or targeting the innate immune system. While the former has been previously shown 13 , here we show that ORF8b activates intracellular stress pathways upon forming insoluble aggregates and directly targets innate immunity by activation of the NLRP3 inflammasome.Intracellular protein aggregation contributes to the pathogenesis of a variety of diseases and is both propagated by and contributes to inflammation 37, 38 . Protein aggregates have long been observed after viral infection; viruses are known to use aggregates as scaffolds for replication and/or for shielding from host degradation 39 . In our study, ORF8b appears to be degraded by the autophagy-lysosome system, as treatment with bafilomycin A1 increases ORF8b levels. The aggregation ability of ORF8b may be protective against degradation, as we consistently observed that the anti-aggregation V77K mutant is expressed at lower levels than ORF8b. It has previously been reported that ORF8ab and ORF8b induce ER stress 40 , which is consistent with our findings showing accumulation of CHOP by ORF8b expression dependent on V77. We also find induction of lysosomal stress and activation of TFEB by ORF8b. Both ER stress and TFEB have been shown to propagate the inflammatory response, with TFEB directly promoting the transcription of inflammatory cytokines 26, 41, 42 . This may be a cellular mechanism by which ORF8b activates the innate immune response in macrophages and contributes to SARS-CoV pathogenesis.Finally, RNA or DNA virus infection is known to trigger NLRP3 or AIM2 inflammasome activation 43, 44 . The released IL-1β and IL-18 functions to promote host defense, but aberrant inflammasome activation can result in excessive inflammation and enhanced disease. Examples include the influenza encoded proton-selective ion channel M2 which provides signal 2 for NLRP3 inflammasome activation 45 , the encephalomyocarditis virus viroporin 2B which promotes the release of intracellular Ca 2+ from intracellular stores also providing signal 2 46 , and the pathogenic influenza A viruses which use PB1-F2 to activate NLRP3 inflammasomes 47 . Our findings indicate the SARS-CoV can activate NLRP3 inflammasome in macrophages via ORF8b. While SARS-CoV abortively infects macrophages/monocytes, enough ORF8b may be present to impact lysosome integrity, autophagy pathways, and NLRP3 inflammasomes. In contrast to macrophages, the SARS-CoV productively replicates in lung epithelial cells. These cells also express NLRP3 and can assemble NLRP3 inflammasomes. In humans infected with the SARS-CoV the full impact of ORF-8b on the pathways we delineated in this study are likely in the lung epithelium. ORF8b may contribute to the cytokine storm and inflammasome activation that occurs during severe SARS-CoV infection. Moving forward, live virus deletion studies are required to assess the effects of ORF8b mediated intracellular aggregates and ORF8b mediated NLRP3 activation. However, here we identify novel mechanisms through which ORF8b may contribute to SARS pathogenesis.The following primary Abs were used for immuno Alexa 568 conjugated polyclonal antimouse (Thermo Fisher, A11004) and anti-rabbit (Thermo Fisher, A11011) were used as secondary Abs prior to fluorescent imaging. For immunoprecipitation experiments, we used GFP antibodies coupled to magnetic beads (MBL, D153-9) and Flag antibodies coupled to agarose (Sigma, A2220) to pull down tagged proteins prior to immunoblotting. Cyclosporin A (R&D Systems, 1101) and Z-VAD-FMK (Sigma, V116) were used at 10 μM overnight, and Bafilomycin A1 (Sigma, B1793) at 100 nM for 4 h.THP-1, HEK293, A549, and HeLa cells were obtained from the American Type Culture Collection (ATCC) and maintained following ATCC's recommendations. THP-1 monocytes were differentiated into macrophages by treating with PMA (50 nM) for 3 h. To generate stable THP-1 cell lines, constructs expressing 8b-GFP or GFP were transfected followed by G418 (200 μg/ml) selection for 6 weeks. GFP-positive cells were FACS sorted twice and expanded. SARS-CoV ORF8b cDNA was generated by PCR from K14 and J17 cDNA clones by Dr. H.Y. Qi and Dr. James Shelhamer (Critical Care Medicine Department, National Institutes of Health), which were produced from the SARS-CoV genome Tor2 isolate (Michael Smith Genome Sciences Centre, Vancouver, Canada) 48 . PCR primers for cDNA generation were derived from the SARS-CoV accessory gene sequences in accession number NC_004718 (NCBI). Constructs expressing N and C terminal GFP-tagged SARS-CoV ORF8b were made using the pEGFP-N1 and C1 vectors (Clontech). PCR mutagenesis of GFP-8b was used to generate the V77K GFP-8b point mutant, and ORF8b-Flag was made by replacing the 8b-GFP c-terminal GFP with a 3x Flag tag. All constructs were verified by DNA sequencing.Myc tagged NLRP3 plasmids were kindly provided by Dr. Yong-Jun Liu (Baylor). The Myc tagged NLRP3 LRR domain construct spanned amino acids 537-991 in NLRP3. The pcDNA3 NLRP3-Flag construct was kindly provided by Dr. Gabriel Nunez (University of Michigan). To construct the NLRP3 Ds-Red construct, the Ds-Red coding region was PCRed from the pDs-Red-Monomer fluorescent vector (Clontech) and inserted into Flag tagged NLRP3 construct after removal of the c-terminal Flag coding sequence. The TFEB-GFP plasmid was bought from Addgene, (38119) and the TFEB-mCherry plasmid made by moving the TFEB cDNA into the mCherry N1 vector (ClonTech). The ptf-Galectin 3 plasmid was also purchased from Addgene (64149), and the mCherry-Galectin 3 plasmid made by removing the GFP from the ptf-Galectin 3 vector. Pooled siRNAs targeting NLRP3 (sc-45469), ASC (sc-37281), GFP (sc-45924), and a scrambled control (sc-37007) were purchased from Santa Cruz Biotechnology. Plasmids and siRNA were transfected into cells using X-tremeGENE-HP (Roche, 6366236001) following the manufacture's protocol.For standard immunoblotting, cells were lysed in Buffer A containing 20 mM HEPES (pH 7.4), 50 mM β-glycerophosphate, 1% (v/v) Triton X-100, 2 mM EGTA, and cOmplete protease inhibitor cocktail (Sigma, 11836170001) plus PhosStop (Sigma, 04906837001) phosphatase inhibitor tablets for 30 min. Lysates were cleared by centrifugation at 14,000 rpm for 10 min and the supernatant collected. For separation of Triton soluble and insoluble fractions, cells were lysed in Buffer A and centrifuged as before. The supernatant was considered the Triton soluble fraction, while the pellet was directly mixed with 1x NuPage LDS Sample Buffer (Invitrogen, NP0008) and heated at 100°C for 30 min (Triton insoluble fraction). For nuclear fractionation, cells were lysed with Buffer B, which is Buffer A substituting 0.5 (v/v) NP-40 for Triton X-100. After centrifugation, the supernatant was considered the cytosolic fraction. The resultant pellet was washed 4-6 times with Buffer B and then lysed in Buffer B + 0.5% (v/v) SDS for 30 min, this was considered the nuclear fraction. Inflammasome activation was measured by immunoblotting cell culture supernatants for mature IL-1β and cleaved caspase-1 as previously described 49 . For immunoprecipitations, cells were lysed with Buffer C, which is Buffer A + 0.5% (w/v) CHAPS and incubated with either GFP or FLAG beads for 3 h. Immunoprecipitates were washed 4-6 times and resuspended in 1x NuPage LDS buffer to release captured proteins prior to immunoblotting.All lysates were mixed with 4x NuPage LDS Sample Buffer and heated at 100°C for 10 min, followed by separation on a 4-20% Tris-Gylcine Gel (Invitrogen) and transfer to a nitrocellulose membrane using the iBlot Gel Transfer System (Invitrogen). The membrane was blocked with 5% nonfat milk (or 5% BSA) in TBST (25 mM Tris-HCl, 150 mM NaCl, 0.1% Tween-20) for 1 h and incubated at 4°C overnight with the primary Ab in TBST with 5% BSA. The appropriate secondary Abs conjugated to HRP were used to detect the protein of interest via ECL. When necessary, membranes were stripped using Restore Plus Western Blot Stripping Buffer (Thermo, 46430) following the manufacturers protocol, re-blocked, and reblotted. Images were acquired either by exposure on film (Amersham Hyperfilm ECL) or using the iBright 1000FL (Invitrogen). Blots were scanned and imported into Photoshop as unmodified tagged image files; quantification of band intensity was performed using standard methods on ImageJ (NIH) and presented relative to the appropriate housekeeping gene.Immunofluorescence THP-1 cells transiently transfected to express GFP-8b or GFP were seeded on glass-bottom collagen coated microwell dishes (MatTek), differentiated with PMA, and incubated overnight with LPS (50 ng/ml) and a caspase-1 inhibitor (Z-VAD-FMK, 10 μM). The following day, the cells were washed, fixed with 4% paraformaldehyde, permeabilized with 0.1% Triton X-100 in PBS, and blocked with 1% bovine serum albumin for 15 min. For immunostaining, the cells were incubated for 4 h at room temperature or overnight at 4°C with primary antibodies diluted in 1% bovine serum albumin/ PBS. Fluorescent images were collected after secondary antibody incubation using a Leica TCS-SP5-X-WLL confocal microscope equipped with an argon and white light laser (Leica Microsystems) at either 63X or 100X oil immersion objective (NA 1.4). The samples were excited with 488-or 561-nm laser lines. Image analysis was performed using Imaris 8.0.0 (Bitplane AG) and Adobe Photoshop CS3 (Adobe Systems). HeLa cells transiently transfected to express NLRP3-Ds-Red plus or minus GFP-8b were washed 6 h after transfection and plated in glass-bottom microwell dishes overnight. Plated cells were processed and imaged as outlined above the following day. For DNA staining, the cells were incubated with 30 nM DAPI (Sigma) in PBS for 30 min and washed prior to imaging.HEK293 cells were seeded onto 24-well plates in standard media. The following day, the media was changed to OPTI-MEM (Gibco) and the cells transfected with constructs expressing NLRP3 Flag (15 ng), Casp-1 Flag (5 ng), ASC (5 ng), full-length IL-1β Flag (150 ng) and either a control plasmid (GFP or Flag) or GFP-8b or 8b-Flag. Cell supernatants and lysates were immunoblotted for fulllength and mature IL-1β the next day.Cell death assay 0.4% Tryptan Blue (Sigma T8154) was added to cell culture plates and incubated at room temperature for 2 min. Percentage of non-viable cells (cells that had taken up Tryptan Blue) was determined manually by light microscopy and recorded; a minimum of three fields with >50 cells per field were quantified.All immunoblots, and immunofluorescence images shown are representative of reproducible experiments. Differences between groups were determined using Student's t-test (p < 0.05 was considered significant), and all statistical analysis was performed on GraphPad Prism 6.0. All experiments were performed a minimum of three times.\n",
            "\t0.715\tBackground: Severe acute respiratory syndrome (SARS)-CoV is a newly emerging virus that causes SARS with high mortality rate in infected people. Successful control of the global SARS epidemic will require rapid and sensitive diagnostic tests to monitor its spread, as well as, the development of vaccines and new antiviral compounds including neutralizing antibodies that effectively prevent or treat this disease.The widespread diffusion and mortality of severe acute respiratory syndrome (SARS), caused by a new Coronavirus (SARS-CoV) has threatened the entire world and has urged the scientific community to develop molecular and serological tests which can assist for rapid detection of SARS cases and implementation of control measures . Effective prophylaxis and antiviral therapies are urgently needed in the event of re-emergence of the highly contagious and often fatal SARS infection.Like other known Coronaviruses, SARS-CoV is an enveloped virus containing four structural proteins, namely the membrane (M), envelope (E), spike (S) and the nucleocapsid (N) proteins . The N protein is a 423 amino acid, predicted phospho-protein of 46 kDa, which shares little homology with other members of the Coronavirus genus [3, 4] . Several studies have found that N protein is highly immunogenic, thus antibody response in patients with SARS is directed most frequently and predominantly to the nucleocapsid. It has also been found that anti N antibodies are detected early and with high specificity during infection . Therefore, the generation of N-specific reagents, in particular monoclonal antibodies (mAbs) could be of assistance in studies of viral replication and antiviral activity, as well as in the diagnosis of SARS-CoV infection at various disease stages. It has also been found that SARS patients show clinical improvement if they are treated with serum from previously infected patients suggesting that passive immunotherapy could be developed for the treatment of this infectious disease . However, the traditional approach to generate mAbs to SARS-CoV has presented difficulties for several reasons including safety concerns in handling SARS particles . To overcome these limitations, we applied a very effective and safe in in vitro procedure based on human scFvs selection from the large synthetic ETH-2 phage antibody library .Here, we report the identification, production, and epitope characterization of human scFv antibodies specifically recognizing distinct N protein domains. These recombinant mAbs were found to bind selectively and with good affinity to distinct N protein epitopes, and were suitable to specifically identify SARS particles in Vero infected cells.Nucleocapsid (N) protein of SARS-CoV (residues: 1-422) and its fragments N1 (residues: 1-117), N2 (residues: 110-340) and N3 (residues: 333-422) expressed in E. coli by DNA recombinant technology as reported by Carattoli, Di Bonito et al . These polypeptides have proved to react with specific sera of SARS-CoV infected patients.The synthetic human recombinant antibodies library (ETH-2) consists of a large array (more than 10 9 antibody combination) of scFv polypeptides displayed on the surface of M13 phage. It was built by random mutagenesis of the CDR3 of only three antibody germline gene segments (DP47 for the heavy chain, DPK22 and DPL16 for the light chain). Diversity of the heavy chain was created by randomizing four to six positions replacing the pre-exist-ing positions 95 to 98 of the CDR3. The diversity of the light chain was created by randomizing six positions (91 to 96) in the CDR3 .An aliquot of the ETH-2 library, containing 10 13 cfu phage, was used to isolate specific human antibodies in scFv format to SARS-CoV N protein. Immunotubes (Nunc Maxisorp; Denmark) were coated overnight (ON) at room temperature (RT) with 10 µg/ml of recombinant purified SARS-CoV N protein in PBS. After panning, phages were eluted with 1 ml of 100 mM triethylamine and the solution was immediately neutralized by adding 0.5 ml of 1 M Tris-HCl pH 7.4. Eluted phages were used to infect log phase TG1 E. coli bacteria and amplified for the next round of selection. Briefly, 50 ml of 2 × YT with 100 µg/ ml ampicillin (2 × YTA medium) and glucose 1% were inoculated with enough bacterial suspension to yield an OD 600 nm ≅ 0.05-0.1. The culture was grown up to OD 600 nm 0.4-0.5 and infected with M13 K07 helper phage in a ratio of around 20:1 phage/bacteria. The rescued phages were concentrated by precipitation with PEG 6000 and used for next round of panning (usually three to recover antigen-specific antibody phages from the ETH-2 library). For soluble scFv preparation, individual colonies were grown in 96 flat bottomed wells (Nunc) for 2 hours at 37°C in 180 µl 2 × YTA medium and glucose 0.1% in 96 well plates and inducted with 50 µl 2 × YTA medium and 6 mM IPTG. The following day the plates were spinned down at 1800 g for 10 minutes and the supernatants containing soluble scFvs were recovered and tested for specificity.ELISA 96 well ELISA-plates (Nunc Maxisorp) were coated ON at RT with 0.5 µg of antigen (N protein or its N1, N2, N3 fragments) in PBS. The following day a blocking solution, consisting of 2% non fat dry milk in PBS (MPBS), was added; plates were washed with PBS containing 0.05% Tween 20 (TPBS) and incubated for 2 hours at RT with 50 µl of supernatants containing soluble scFv antibody, anti-Flag M2 antibody (1.6 µg/ml Sigma-Aldrich; MO, USA) and anti-mouse HRP conjugated antibody (1.6 µg/ml Dako; Denmark). The reaction was developed using 3, 3 1 -5, 5 1 -tetramethylbenzidin BM blue, POD-substrate soluble (Roche Diagnostics; IN, USA) and stopped by adding 50 µl of 1 M sulfidric acid. The reaction was detected with an ELISA reader (Biorad; CA, USA), and the results were expressed as A (absorbance) = A(450 nm)-A(620 nm).1 µg of SARS-CoV N, N1, N2 and N3 proteins were loaded onto 12% SDS-PAGE and transferred to a nitrocellulose membrane using standard procedures. The membrane was blocked in 4% MPBS ON at RT. Blotted proteins were incubated for 2 hours with supernatant containing soluble scFvs, washed with 0,05% TPBS and incubated again with 5 µg/ml anti-Flag M2 mouse antibody (Sigma-Aldrich) in 2% MPBS. After an additional incubation for 1 hour at RT in presence of 5 µg/ml of a goat anti-mouse antibody HRP-conjugated (Dako), the reaction was developed and visualized with a chemiluminescence detection kit (Pierce; IL, USA).BIOCHIP slides (Euroimmun, Luebeck, Germany) coated with SARS-CoV infected Vero cells were used for immunocytochemistry determination of virus particles by phage or scFv specific antibodies. Slides were treated for 10 minutes with peroxidase block solution (Dako En Vision System HRP), washed in TPBS buffer and incubated for 1 hour at RT with 10 12 phage antibody particles (resuspended in 25 µl of TPBS) or scFv containing supernatant (25 µl). As control, BIOCHIP slides were incubated with an irrelevant anti-glucose oxidase phage or scFv antibodies . After extensive washings with TPBS, the slides were incubated for 1 hour at RT with an anti-M13 secondary mouse antibody (5 µg/ml in 20 µl TPBS) (Amersham) or with an anti-FLAG secondary mouse antibody (5 µg/ml in 20 µl TPBS) (Sigma-Aldrich) for labelling phage and scFv reactive antibodies, respectively. After washings, the slides were treated for 30 minutes at RT with cromogen solutions (Dako) and inspected with a light microscope. To isolate specific scFv antibodies, an aliquot of the human synthetic ETH-2 library containing approximately 1 × 10 12 cfu phages was introduced for panning into Maxisorp immunotubes coated with purified SARS-CoV N protein as antigen. Non-specifically absorbed phages were removed by washing. Bound phages were eluted, amplified and used for the next cycle of panning as described elsewhere [8, 10] . By using this protocol , phage antibody populations specifically recognizing the N protein of SARS-CoV were isolated after only three rounds of selection. Plating on agar of TG1 phage antibody-infected cells, allowed growth of individual phagemid clones. Soluble scFvs derived from IPTG inducted colonies, were screened by ELISA and several of them proved to be specific for N protein (Figure 1 ).N-positive scFv clones were analysed by ELISA for reactivity with N1, N2 and N3 protein fragments ( Figure 2 ). Three distinct classes of scFv antibodies recognizing either the intact N protein only or also N2 or N3 fragments were identified. ScFv antibodies recognizing N protein only but none of its fragments (for instance the MA2D5 scFv) were likely directed against epitopes encompassing two adjacent protein fragments or conformational epitopes expressed only on the integral protein. The absence of scFv antibodies reacting with the N1 fragment somewhat matches the lower antigenicity of this polypeptide, as compared to immunodominant N2 and N3 fragments, , despite the reported reactivity of several linear synthetic epitopes of the N1 region with SARS sera . However, a relatively low number of scFv clones were here tested for a sound conclusion on this aspect.Three of the most reactive scFv antibody clones named MA2.D5, MA2.D7 and MA2.E3 each one representing the different classes of epitope recognition in the N protein domains were further assessed to verify their biochemical patterns, reactivity with SARS-CoV infected cells, and DNA sequences of their encoding genes.Western blotting analysis (Figure 2) shows that all three antibodies react with a 46 kDa band corresponding to the predicted molecular mass of the intact N protein . As expected by previous ELISA indications, the scFvs MA2.E3 and MA2.D7 recognize also a 28-30 kDa and 12-14 kDa bands respectively, corresponding to the predicted molecular mass of N protein fragments N2 and N3 .The molecular analysis of the phage antibodies MA2.D5, MA2.D7 and MA2.E3 shows scFv antibodies with a molecular weight of about 27 kDa and the integrity of the genes encoding for the cognate scFvs displayed on M13 phage (data not show). VH and VL gene sequences, shows that each single scFv antibody clone possess a unique DNA sequence encoding for the CDR3 region ( Figure 3) ; this uniqueness is consistent with the distinct epitope recognition patterns of the three scFvs noted above. Immunocytochemical investigations show that MA2.D5 antibody specifically recognizes SARS-CoV particles in infected Vero cells (Figure 4) . The other two antibodies MA2.D7 and MA2.E3, though genuinely reacting with N protein and its N2 and N3 fragments in ELISA and western blotting (see above) did not prove to be useful for SARS-CoV determination in infected cells.The usefulness of the library as a tool for generating monoclonal antibodies against viral pathogens including SARS-CoV [15, 16] has been tested and showing that phage antibodies may recognize viral proteins used as antigens. In particular, a rodent library has proved to be very effective for the isolation of murine scFvs exhibiting high specificity against SARS-CoV E protein, which is recognized as a 31 kDa protein in western blot studies . Interestingly, one of these clones (A17) isolated by selecting the rodent library against the 31 kDa E protein crossreacts in ELISA with a well-defined fragment of the 46-48 kDa N protein. Recently, human mAbs were identified by biopanning of a synthetic library on SARS-CoV lysate; the isolated scFvs expressed on phages have proved to recognize N protein only in ELISA while no evidences for N protein recognition in western blot studies or in SARS infected cells were shown . The efficient display of recombinant antibodies on filamentous phage has been also assessed by our findings showing that soluble human scFvs selected from the ETH-2 library may yield agents capable of specifically recognizing SARS-CoV particles in infected Vero cells as well as the intact 46-48 kDa SARS-CoV N protein and its N2 (28-30 kDa) and N3 (12-14 kDa) fragments in western blot and ELISA investigations (Figure 2 and 4) . To our knowledge, the human scFvs directed against N protein, by us isolated and characterized, are the first antibodies so far described which are able to detect SARS-CoV N protein in different routinary laboratory techniques such as ELISA, Western blot and immunocytochemistry. Hence, these human mAbs represent excellent candidates for future development of serological diagnosis of SARS-CoV infections since high functionality (all clones express soluble antibodies in bacteria), specificity (all clones recognize the 46-48 kDa N protein) and quality (all clones give good signals in ELISA, in western blot as well and stain SARS-CoV infected cells). scFv phage sequenced, thus facilitating various molecular approaches including site direct mutagenesis to maturate binding affinity and construct whole recombinant immunoglobulins for SARS-CoV studies and applications. To this latter regard, another immediate use of human scFv directed to N protein is through intracellular expression as a novel strategy of gene therapy aimed at knockout the replicative cycle of SARS-CoV in infected cells. It is thought that intracellular expression of scFv antibodies may be superior over alternative methods for gene inactivation such as anti-sense RNA and dominant negative mutants because of its high specificity [17, 18] .Nevertheless, phage display approach make feasible several strategies for the isolation of neutralizing human mAbs to provide an immediate treatment for emergency prophylaxis of SARS-CoV while vaccines and new drugs are underway. For example, immune library may be constructed and screened on SARS-CoV proteins, using Bcells from convalescent SARS patients as a genetic source of specific VH and VL. In alternative, synthetic peptides mimicking immunodominant epitopes in S or M SARS-CoV proteins may be used as a substrate antigens for the identification of specific scFvs from synthetic human phage antibody library.SARS, Severe Acute Respiratory Syndrome; CoV, Coronavirus; N, Nucleocapsid; scFv, single chain fragment variable; ETH-2, synthetic scFv antibody phage library; CDR, complementarity determining regions; mAbs, monoclonal antibodies.The author(s) declare that they have not competing interest. \n",
            "\t0.709\tBackground: Severe acute respiratory syndrome coronavirus (SARS-CoV) caused a global panic due to its high morbidity and mortality during 2002 and 2003. Soon after the deadly disease outbreak, the angiotensin-converting enzyme 2 (ACE2) was identified as a functional cellular receptor in vitro and in vivo for SARS-CoV spike protein.Severe acute respiratory syndrome (SARS), an atypical pneumonia, had emerged in late 2002 and spread to more than two dozen countries within Asia, South and North America, and Europe in the spring of 2003 . During this period, the SARS epidemic had been considered as a significant threat to populations as its mortality rate was approximately 10 %, responsible for around 800 deaths out of approximately 8000 patients . SARScoronavirus (SARS-CoV), a novel coronavirus classified as a member of the Coronaviridae family was soon identified as the causative pathogen [3, 4] . The RNA genome of SARS-CoV consists of 14 potential major open reading frames that encode the viral non-structural proteins, accesory proteins, and structural proteins including spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins . Angiotensin-converting enzyme 2 (ACE2), the type I integral transmembrane protein, was identified to be the functional receptor for SARS-CoV both in vitro [5, 6] and in vivo . Studies on the expression of ACE2 protein and the tissue tropism and cellular distributions of SARS-CoV provided new insight into the mechanism of pathogenesis . Nevertheless, certain ACE2-expressing endothelial cells and human intestinal cell lines failed to be infected by SARS-CoV [9, 10] . In contrast, cells without a detectable expression level of ACE2 such as hepatocytes could be infected by SARS-CoV . In addition, the presence of ACE2 alone is not sufficient for maintaining viral infection . Altogether, these observations indicate that different virus receptors or co-receptors may be utilized in the infection of SARS-CoV in various tissues. Indeed, DC-SIGN, a c-type lectin receptor expressed on dendritic cells, and a DC-SIGN-related molecule, L-SIGN (also named DC-SIGNR and CD209L) have been indicated to interact with the SARS-CoV spike protein and to facilitate the virus dissemination [11, 12] .Vimentin is the major component of the type III intermediate filament protein which aims mainly to maintain the architecture of cytoplasm, it can also be secreted under certain conditions . Vimentin participates in cell adhesion, migration, and cellular signaling [14, 15] . In addition, vimentin has been reported to play roles in viral multiplication. Rearrangement of cytosolic vimentin and formation of vimentin cages around the viral factories were observed during the infection of vaccinia virus and African swine fever virus [16, 17] . Studies on mammalian porcine reproductive and respiratory syndrome virus, Japanese encephalitis virus, and cowpea mosaic virus also provided evidence that binding of virus to surface vimentin can facilitate internalization and infection . Blocking the expression and binding of surface vimentin inhibited viral entry.In this study, intermediate filament vimentin was identified as a cellular factor abundantly present in the SARS-CoV spike protein-ACE2 complexes. Incubating Vero E6 cells with SARS-CoV virus-like particles (VLPs) enhanced the expression level of the surface form of vimentin. Co-localization of vimentin and the SARS-CoV spike protein was observed in a short time period soon after incubation. Further studies indicate that vimentin directly binds to the SARS-CoV spike protein and is involved in the entry of SARS-CoV. These results suggest that vimentin serves as a putative co-receptor for coordinately interacting with ACE2 during SARS-CoV infection. The study provides a new target for drug development against SARS-CoV infection.Sf9 (Spodoptera frugiperda) insect cells were maintained at 27°C while being supplied with Sf-900IISFM medium (Gibco). Vero E6 cells (African green monkey kidney epithelial cells) were maintained at 37°C with 5 % CO 2 in Dulbecco's modified Eagle's medium (Gibco), supplemented with heat-inactivated fetal bovine serum (8 %), penicillin (100 units/ml), and streptomycin (100 μg/ml).Production and purification of SARS-CoV VLPs and SARS-CoV spike protein were carried out as described previously . In brief, Sf9 cells were coinfected with three recombinant baculoviruses expressing V5-tagged SARS-CoV E, M, and S proteins individually. Sf-900IISFM conditional medium containing properly assembled VLPs was harvested at 4 days post-infection. SARS-CoV VLPs were then purified by sucrose gradient centrifugation at 26,700 rpm for 3.5 h at 4°C using an SW41 rotor in a Beckman L9-M Ultracentrifuge. SARS-CoV VLPs were analyzed by immunogold electron microscopy (data not shown). In addition, the number of SARS-CoV VLPs was calculated by atomic force microscopy as described previously . To obtain the SARS-CoV spike protein, a culture medium of Sf9 cells previously infected with the recombinant baculoviruses expressing the C-terminal V5and His-tagged full-length SARS-CoV spike protein were collected at 4 days post-infection and subjected to the purification of the spike protein by using a Ni 2+ Sepharose purification system.Extracellular chemical cross-linking of Vero E6 cells pre-incubated with SARS-CoV VLPs at VLP-to-cell ratio 1000:1 was performed at 4°C for 2 h with 5 mM membrane-impermeant primary amine-reactive crosslinker, bis(sulfo-succinimidyl) suberate (BS 3 , Pierce) or the thiol-cleavable reagent 3,3'-dithiobis(sulfo-succinimidylpropionate) (DTSSP, Pierce) in the reaction buffer (20 mM Na 3 PO 4 and 0.15 M NaCl in PBS, pH 8.0). The reaction mixtures were then quenched with 20 mM Tris buffer (pH 7.5) for 15 min at room temperature and subjected to cell lysis for further identification of ACE2-associated proteins.Immunoprecipitation, silver staining, and mass spectrometry Following extracellular chemical cross-linking, the SARS-CoV VLP-pre-incubated Vero E6 cells were lysed with Empigen BB lysis buffer (50 mM Tris-HCl, pH 7.4, 0.05 % sodium deoxycholate, 150 mM NaCl, 1 mM EDTA, and 0.3 % Empigen BB) supplemented with protease inhibitor cocktail (Roche) and phenylmethylsulfonyl fluoride. The protein lysates were subjected to immunoprecipitation followed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), silver staining, and LC-MS/ MS analysis as described previously [23, 24] .Flow cytometry analysis was performed for quantitative determination of cell surface vimentin and VLP-positive or spike-positive cells. For the detection of surface vimentin, Vero E6 cells were pre-incubated with SARS-CoV VLPs at VLP-to-cell ratio 100:1 for 10 min. Cells were subsequently fixed with 4 % formaldehyde in FACS buffer (5 mM EDTA in PBS, pH 7.4) for 20 min followed by incubation with mouse anti-vimentin antibodies (V5255, Sigma) and the goat anti-mouse AlexaFluor 488-conjugated antibodies. For quantitative determination of VLP-positive cells and spike-positive cells, Vero E6 cells pre-incubated with SARS-CoV VLPs or purified spike protein were fixed in FACS buffer with 4 % formaldehyde, permeabilized in FACS buffer with 0.1 % Triton X-100, and subjected to immunofluorescence labeling with goat anti-mouse AlexaFluor 594-conjugated antibodies following incubating with anti-V5 epitope antibodies (Invitrogen) and with goat anti-rabbit AlexaFluor 488-conjugated antibodies following incubating with rabbit anti-6-His antibody (Bethyl Laboratories Inc.), respectively. The immunofluorescence stained cells were subjected to LSR-II Digital Flow Cytometer (BD Biosciences) and the data was analyzed using FlowJo software (Tree Star Inc.). In antibody neutralization experiments, rabbit monoclonal anti-vimentin antibody (RP4002, Immuno Bioscience Corp.), mouse monoclonal anti-vimentin antibodies (V5255, Sigma) and goat polyclonal anti-ACE2 antibodies (AF933, R&D Systems) were applied prior to the treatment of SARS-CoV VLPs or spike protein and subsequent flow cytometry. In vimentin knockdown experiments, Vero E6 cells were infected by lentiviruses carrying shRNAs specific to vimentin prior to the treatment of spike protein and subsequent flow cytometry.Cells grown on 18 mm round glass coverslips were fixed using 4 % formaldehyde in PBS for 12 min. Then the cells were rinsed with PBS buffer, subsequently an appropriate amount of 1:1 of acetone/ methanol solution was added and incubated for 2 min to permeabilize the cell membranes. PBS buffer was immediately applied followed by quick rinse using PBS buffer. In the study for monitoring the proteins on cellular membranes, the permeabilization step was skipped. The cells were treated with 4 % BSA in PBS buffer to block for 1 h, then 1:10,000 of primary antibodies in PBS buffer containing 1 % BSA at 4°C overnight. Next day, the cells were rinsed with PBS, and subsequently incubated with secondary antibodies at 1:5000 in PBS buffer containing 1 % BSA for 1 h at room temperature. To stain the nuclei, 1:500 of Hoechst in PBS buffer containing 1 % BSA was added at the same time as the addition of secondary antibodies. The coverslips were then treated with mounting fluids and sealed for observation. For certain groups of experiments aiming to examine surface vimentin upon SARS-CoV VLPs binding, Vero E6 cells were pre-incubated with SARS-CoV VLPs at VLP-to-cell ratio 100:1. The images were acquired on a Leica TCS SP5 Confocal Microscopy. For immunofluorescence staining assay, Vero E6 cells pre-incubated with SARS-CoV VLPs were fixed and subjected to analysis as described previously using mouse anti-vimentin antibodies (Sigma) and rabbit anti-SARS-CoV spike protein antibodies (IMGE-NEX) as the primary antibodies, and AlexaFluor 488conjugated goat anti-mouse IgG antibody and AlexaFluor 594-conjugated goat anti-rabbit IgG antibody, respectively, as the secondary antibodies.For far-Western blot analysis, cell lysates resolved by SDS-PAGE were transferred onto a PVDF membrane. Following a blocking with 3 % BSA in PBS buffer, the membrane was then incubated with purified SARS-CoV spike protein (10 μg/ml) at 4°C overnight in PBS buffer containing 1 % BSA. Detection of the binding of the purified spike protein, along with its interacting cellular proteins resolved on the membrane were detected by Western blot analysis as described previously .The lentivirus vector pLKO.1 carrying the small hairpin RNA (shRNA) targeting vimentin (shVim-A: 5'-GCATCACGATGACCTTGAATA-3', TRCN0000089830; shVim-B: 5'-GCTAACTACCAAGACACTATT-3', TRCN 0000029119; shVim-C: 5'-GCGCAAGATAGATTTGGAA TA-3', TRCN0000089828) and targeting luciferase (shLuc: 5'-CAAATCACAGAATCGTCGTAT-3', TRCN0000072 246) from the National RNAi Core Facility, Academia Sinica, Taiwan were individually cotransfected with pMD.G and pCMVΔR8.91 to HEK293T cells by using T-Pro NTR II transfection reagent (T-Pro Biotechnology). The culture supernatants containing lentiviruses carrying individual shRNAs were then harvested and used to knockdown vimentin expression in Vero E6 cells following infection. In addition, polybrene (Sigma) was added at 8 μg/ml to increase efficiency of infection. The lentivirus carrying shRNA against luciferase served as a negative control in vimentin knockdown experiments.To examine the putative SARS-CoV spike proteininteracting host factors, SARS-CoV-permissive Vero E6 cells were incubated with SARS-CoV VLPs in the presence of membrane-impermeant BS 3 that triggered a chemical cross-link of the N-terminal domain of SARS-CoV spike protein to its functional receptor ACE2 as well as other components appear in the spike-receptor complexes. While Vero E6 cell lysates were immunoprecipitated by anti-ACE2 antibodies, those pre-treated with purified SARS-CoV spike proteins were shown to form various complexes with molecular sizes larger than 200 kDa (data not shown). This is an indication of numerous host factors associated with viral spike-ACE2 complexes upon binding. To further dissect which proteins were involved in the complexes, the reversible crosslinker DTSSP was used to link interacting partners within 12 Å. Subsequently, DTSSP-linked proteins were cleaved on the central disulfide bonds of the cross-linkers by adding the reducing agent β-mercaptoethanol prior to SDS-PAGE followed by silver staining. As a result, protein bands seen on the gel referred to the molecular weight of each component in the spike-ACE2 complex. By comparing the Vero E6 cell lysates pre-incubated with SARS-CoV VLPs in a time course from 10, 30, to 60 min with those treated VLPs for 0 min as a reference, seven major protein bands, named S1 to S7 (indicated by black arrowheads), showed differential abundance (Fig. 1) . The bands were then cut off the gel and sent for LC-MS/MS analysis. The resulting spectra of protein S3 demonstrated that 32 fragments were matched to a fraction of vimentin protein sequence, suggesting that a cytoskeletal protein vimentin is one of the putative ACE2-interacting proteins (inset in Fig. 1 ).To further confirm the interaction between vimentin and the ACE2 protein, cell lysates prepared from SARS-CoV VLP-treated Vero E6 cells were subjected to co-immunoprecipitation with anti-ACE2 antibodies and detected with anti-vimentin antibodies. The negative control for this experiment was Vero E6 cells without the treatment of SARS-CoV VLPs. As shown in Fig. 2 , vimentin formed complex with ACE2 in Vero E6 cells pretreated with the SARS-CoV VLPs for 10 min. Nevertheless, the interaction between vimentin and spike-ACE2 complex on the membrane is transient as co-immunoprecipitation of vimentin was no longer detected after 10 min. These results suggest an involvement of the cell surface vimentin in the virus entry via an interaction, for a relatively short time period soon after exposing to the SARS-CoV VLPs, with the spike-ACE2 complexes. Speculation regarding the conflicts about whether vimentin interacted with ACE2 at 30 min (Fig. 1) or 10 min (Fig. 2) will be given in the Discussion section.To further examine the biological role of vimentin in association with the spike-ACE2 complexes, the expression of vimentin and its cellular distribution were examined by immunofluorescence assay following a pretreatment of Vero E6 cells with the SARS-CoV VLPs for 10 min. As shown in Fig. 3 , vimentin colocalized with the SARS-CoV spike protein on the cell surface (indicated by white arrows) under a non-permeable fixation condition that allowed only the detection of the protein level on the extracellular side. For the negative control, Vero E6 cells without pretreatment with the SARS-CoV VLPs were shown no detectable signals referring to spike proteins on the membrane. Although some background noise appeared when the anti-vimentin antibody was applied to detect surface vimentin, no co-localization of spike and vimentin can be observed. In addition, HEK293T cell line was used to ensure that, when there was a lack of ACE2, the cells cannot accommodate the binding of spike proteins on the membranes; moreover, vimentin showed no correlation with spike-ACE2 complex Fig. 1 Vimentin as a cross-linked protein in association with the SARS-CoV spike protein-ACE2 complex. Vero E6 cells at 8 × 10 6 were pre-incubated with SARS-CoV spike protein (1 μg/10 6 cells) at 4°C for various time periods as indicated followed by a treatment with the thiol-cleavable cross-linker DTSSP. Cell lysates were harvested and subjected to immunoprecipitation with anti-ACE2 antibodies. Vero E6 cells being treated by SARS-CoV VLPs for 0 min was used as the negative control. The immunoprecipitates were incubated with protein sample buffer with the addition of 5 % β-mercaptoethenol and subjected to SDS-PAGE. Proteins separated on the polyacrylamide gel were visualized by silver staining. The protein bands marked S1 to S7 were collected for LC-MS/MS analysis and results were shown. The number of matched spectra of the protein fragments compared to the published data was stated in parentheses. The detailed analysis of S3 was shown in the inset, where sequence comparison with data banks indicated the S3 protein to be vimentin. Underlines show the peptide sequences of 32 spectra in terms of the distributions (Fig. 3) . When Vero E6 cells without pretreatment of SARS-CoV VLPs were permeabilized by 1:1 acetone/methanol, vimentin protein was detected in the cytosols.To quantify the amount of surface vimentin induced by SARS-CoV VLPs treatment, flow cytometry was utilized. Using Vero E6 cells permeabilized with 0.1 % Triton X-100 for the detection of intracellular vimentin as a positive control (light blue peak), an increase of 16.5 % on the cell surface vimentin upon a 10-min application of SARS-CoV VLPs (magenta curve) was detected in comparison with those without the VLPpretreatment (grey peak) (Fig. 4) .Knowing that vimentin is present in the viral spike protein-ACE2 complex and potentially important for the cell entry of SARS-CoV VLPs, it is vital to understand if the interaction between surface vimentin and spike or ACE2 is direct or indirect. By conducting far-Western blot analysis, a direct interaction between the viral spike protein and vimentin was then concluded from Fig. 5 . The purified V5-tagged spike protein colocalized with the 57 kDa vimentin blotted on the membrane, indicating a direct binding of the spike protein to vimentin ( Fig. 5a left panel) . Besides, while the membrane was pre-treated with anti-vimentin antibodies prior to spike protein incubation, vimentin cannot be detected via far-Western blotting (Fig. 5a right panel) . Western blot analysis with antibodies to vimentin was performed in parallel as a positive control (Fig. 5b) . In addition, a Coomassie Blue staining was shown in Fig. 5c as an internal control with the aim of ensuring there are no Fig. 3 Co-localization of the spike protein with vimentin in Vero E6 cells pre-treated with SARS-CoV VLPs. Vero E6 cells pre-treated with SARS-CoV VLPs for 10 min were fixed for immunofluorescence assay with anti-spike antibodies followed by AlexaFluor 594-conjugated goat anti-rabbit IgG and anti-vimentin antibodies followed by AlexaFluor 488-conjugated goat anti-mouse IgG antibodies as shown in red and green colors, respectively. Vero E6 cells without the pretreatment with SARS-CoV VLPs and ACE2-negative HEK293T cells treated with SARS-CoV VLPs for 10 min were used as controls. Hoechst staining was performed in parallel to localize cell nuclei in the field and the distribution of vimentin in cytosol was examined using permeable Vero E6 cells The specificity of the surface vimentin involving in the entry of SARS-CoV VLPs was further elucidated by antibody neutralization experiments. Vero E6 cells were pre-incubated with anti-vimentin antibodies (RP), anti-ACE2 antibodies or control antibodies prior to the treatment of SARS-CoV VLPs. Intracellular VLPs were monitored by flow cytometry using anti-V5 antibodies that detected membrane-permeabilized SARS-CoV VLPpositive Vero E6 cells. Results demonstrated a 43.4 % reduction (15.41 ± 0.083 vs. 8.73 ± 2.127 %) on the number of cells that uptook SARS-CoV VLPs when the cells were pre-treated with anti-vimentin antibodies, while cells pre-treated with anti-ACE2 antibodies showed a 51.6 % decline (8.86 ± 1.135 vs. 4.25 ± 0.541 %) (Fig. 6a) . The results suggested that anti-vimentin antibodies successfully blocked SARS-CoV VLPs entering into Vero E6 cells and its neutralization efficiency was comparable to that of the anti-ACE2 antibody. An inhibitory effect of anti-vimentin antibodies was also observed on the uptake of SARS-CoV spike protein. When Vero E6 cells were pre-treated with anti-vimentin antibodies RP at two different doses, a 78.0 and 38.6 % reduction on the number of cells that uptook SARS-CoV spike protein were observed, respectively; while cells pre-treated with different doses of anti-vimentin antibodies V showed a 47.0 and 91.6 % decline, respectively (Fig. 6b) . The results further confirmed that vimentin is critical for the entry of SARS-CoV. The role of vimentin in the cell entry of SARS-CoV was also evident by knockdown experiments. Vero E6 cells infected by lentiviruses carrying shRNA specific to vimentin (shVim-A, shVim-B and shVim-C) displayed remarkable reductions on the protein expression of vimentin (Fig. 7a) . When cells were preinfected by lentiviruses carrying shVim-C, which reduced expression of vimentin protein to an undetectable level, a 27.2 % reduction on the uptake of SARS-CoV spike protein was observed (Fig. 7b) . Altogether, surface vimentin interacts directly with the viral spike protein in a spike-ACE2 complex on Vero E6 cell membrane and is involved in the Previous studies demonstrated that cell fusion occurred at 37°C at roughly 1 h after coculturing the viral spike-expressing 293 T cells and cellular receptor ACE2-expressing Vero E6 cells . By using membraneinsoluble cross-linkers BS 3 and DTSSP to capture interacting proteins in close proximity (within 12 Å) in the current study, vimentin was detected at 30 min after the treatment of Vero E6 cells with purified viral spike protein (Fig. 1) . The follow-up experiment demonstrated the interaction of SARS-CoV VLPs with the receptor ACE2 within 10 min of incubation (Fig. 2) . Since the structure of SARS-CoV VLPs is similar to that of SARS-CoV with a crown-shaped appearance and about 100 nm in diameter , it is proposed that binding topology of the conformation-mimicking spike protein of VLPs to its ACE2 receptor is similar to the topology of SARS-CoV spike protein and requires a shorter period of time than the partially purified spike protein. Apart from the aforementioned reason, the different experimental conditions might also be playing a significant role on when vimentin proteins appear on cellular membranes since the cells treated with the purified SARS-CoV spike protein have a reacting temperature of 4°C (Fig. 1) , whereas those treated with SARS-CoV VLPs were conducted at 37°C (Fig. 2) .Previous pathological studies indicated that alveolar epithelial cells are the essential cell type involved in SARS infection, and subsequent respiratory failure. Since the alveolar epithelium cells can further differentiate into type I and type II cells, both cell types are theoretical targets for SARS-CoV infections. Nevertheless, ACE2 had been proven to be largely expressed in the type II cells of the human lung which was thus believed to be the primary target for SARS-CoV [30, 31] . This proposal was supported by another in vitro study in which human alveolar type II cells were demonstrated to be susceptible to the SARS nucleocapsid (SARS-N) protein, even though immunofluorescence staining failed to detect vimentin expression in these type II cells . In the above mentioned study, infection of SARS-CoV was also detected in the primary culture of type I-like cells, where replication could not be successfully carried out. In addition, the studies on pulmonary stem/progenitor cells demonstrated their capability of differentiating into alveolar type II-and type I-like pneumocytes sequentially, both cell types were susceptible to SARS-CoV infection . On the other hand, clinical and in vivo studies using macaque monkey models indicated that the target cells of SARS-CoV are primarily type I pneumocytes . These studies stated above reveal that the susceptibility of either type I or type II alveolar cells to SARS-CoV infection is still controversial among different species. On top of this, the mechanism of stem/progenitor cell differentiation to the two types of alveolar cells is not yet clear. Nevertheless, our studies support the role of vimentin in the cell entry of SARS-CoV.Our data also provided the evidence that SARS-CoV VLPs enhanced the expression of vimentin on the cell membrane. This is the first observation towards the impact of vimentin on SARS-CoV cell entry, although the potential extracellular and surface-associated functions of vimentin have been reported previously in the Mycobacterium tuberculosis-infected monocytes, where vimentin acts as a ligand for the NKp46 receptor on natural killer cells . Besides, one previous study demonstrated that the tail region of surface vimentin was the endothelial target for Cowpea mosaic virus (CPMV) during infection , supporting the hypothesis that vimentin is functionally important in the viral pathogenesis. Furthermore, the association of vimentin with the outer capsid protein VP2 was also considered essential for bluetongue virus egress . One more example in African swine fever virus provided more details in the mechanism of vimentin during viral pathogenesis, suggesting that phosphorylation and rearrangement of vimentin occurred after virus infection. A vimentin cage was then subsequently formed around the virus assembly site during the later stages of infection . In this research, vimentin is one of the interacting partners of the ACE2 receptor and has direct contact with SARS-CoV spike protein. Yet, which domain of vimentin plays the role of interacting with viral spike protein remains unclear. Moreover, whether vimentin binds to ACE2 directly or they only arrange in close proximity will need to be further confirmed. As mentioned, the tail region of vimentin with a unique conformation was shown to be important for CPMV entry and a potential association with the membrane proteins. Hence, it is hypothesized that a putative SARS-CoV spike protein-binding domain may reside in the tail region of vimentin.Despite the fact that vimentin was known as intracellular filamentous cytoskeletal protein [37, 38] , it has recently been indicated to be secreted from blood endothelial cells and activated macrophages through the Golgi apparatus and present on the plasma membrane . Previous studies also suggested that vimentin may be localized on the cell membrane due to its structure Fig. 8 Simplified diagram of the involvement of vimentin within SARS-CoV spike-ACE2 complex during cell entry. The dashed circle indicates the spike-ACE2 complex on the cellular membrane, and the blue dotted arrow demonstrates the possible pathway when vimentin translocates from cytosol to membrane with highly positively charged amino-terminal domain and a Tyr-X-X-Y motif (where X is any amino acid, and Y is a bulky hydrophobic amino acid) followed by a diacidic motif (Asp-X-Glu)-containing carboxy terminus . Furthermore, phosphorylation of vimentin by several protein kinases has been identified [40, 41] . The secreted vimentin has a higher level of phosphorylation and a lower apparent molar mass than the intracellular form . The evidence stated above implies that secreted and intracellular vimentin may have differential biological functions.It has previously been demonstrated in our laboratory that the SARS-CoV spike protein stimulates the activation of Ras-ERK-AP-1 signaling pathway and the expression of fibrosis-associated CCL2 through the phosphorylation of ACE2 . Also, vimentin was identified as a β-adrenergic receptor (βAR)-interacting protein, which involved in the Src to ERK signaling pathway . It is proposed that the rearrangement of vimentin from intracellular compartments to the cell surface might be resulted from the phosphorylation of vimentin through the activation of Ras-ERK signaling pathway induced by the interaction between SARS-CoV spike protein and ACE2. The cell surface vimentin further participates in the viral entry by associating with the spike-ACE2 complexes. However, whether vimentin is involved in the activation of ACE2 downstream signaling pathways needs to be further investigated.In this study, we have demonstrated that cytoplasmic vimentin was translocated to the cell surface and interacted with the spike-ACE2 complex, revealing a putative entry pathway for SARS-CoV (Fig. 8 ). This study is the first report indicating that vimentin interacts directly with SARS-CoV spike protein during the spike-ACE2 binding process and serves as a putative co-receptor involved in the entry of SARS-CoV. Ultimately, the results may help in the development of new drugs for preventing the infection of SARS-CoV or reducing the susceptibility to SARS-CoV infection.\n",
            "\t0.702\tA respiratory disease caused by a novel coronavirus, termed the severe acute respiratory syndrome coronavirus (SARS-CoV), was first reported in China in late 2002. The subsequent efficient human-to-human transmission of this virus eventually affected more than 30 countries worldwide, resulting in a mortality rate of ~10% of infected individuals. The spread of the virus was ultimately controlled by isolation of infected individuals and there has been no infections reported since April 2004. However, the natural reservoir of the virus was never identified and it is not known if this virus will re-emerge and, therefore, research on this virus continues. The SARS-CoV genome is about 30 kb in length and is predicted to contain 14 functional open reading frames (ORFs). The genome encodes for proteins that are homologous to known coronavirus proteins, such as the replicase proteins (ORFs 1a and 1b) and the four major structural proteins: nucleocapsid (N), spike (S), membrane (M) and envelope (E). SARS-CoV also encodes for eight unique proteins, called accessory proteins, with no known homologues. This review will summarize the current knowledge on SARS-CoV accessory proteins and will include: (i) expression and processing; (ii) the effects on cellular processes; and (iii) functional studies.Prior to the outbreak of severe acute respiratory syndrome (SARS) caused by SARS human coronavirus (HCoV) in 2003 , only two human coronaviruses, HCoV-OC43 and HCoV-229E, were known [4, 5] . In the following two years, two additional coronaviruses were identified, namely HCoV-NL63 and HKU1 . Then, very recently, in September 2012 another novel coronavirus was identified in two patients, of which one died . Not much is known about this latest coronavirus and studies are underway to understand the virus better. Only HCoV-OC43, HCoV-229E, HCoV-NL63 and HKU1 are known to continuously circulate in the human population . The International Committee for Taxonomy of Viruses (ICTV) recently reported that the three traditional coronavirus groups have been replaced by four genera, namely Alpha-, Beta-, Gamma-and Deltacoronaviruses .SARS eventually spread to more than 30 countries in 2003, infecting ~8000 people with a mortality rate of ~10% . The etiological agent was eventually identified as the SARS human coronavirus (SARS-CoV) . The virus has been identified as an enveloped, positive-sense RNA virus containing a ~27 kbs genome that encodes for proteins expressed from full-length and subgenomic (sg) mRNAs . In addition to the replicase and structural open reading frames (ORFs), the SARS-CoV genome contains eight ORFs encoding for accessory proteins with no known homologues ( Figure 1 ). These group-specific accessory proteins are interspersed among the structural genes at the 3'-end of the genome. The eight accessory open reading frames (ORFs) (grey shaded boxes) encode for ORFs 3a, 3b, 6, 7a, 7b, 8a, 8b and 9b. The ORFs encoding for the replicase genes (ORF 1a and ORF 1b) and structural genes, spike (S), membrane (M), envelope (E) and nucleocapsid (N) are also shown (not drawn to scale). Coronavirus accessory proteins have been shown to be dispensable for growth in vivo and in vitro . On the other hand, deletion of the nonessential genes from the mouse hepatitis virus (MHV) genome results in attenuation of the virus when inoculated into the natural hosts , indicating a possible in vivo function. Continuous passage of infectious bronchitis virus (IBV) in cell culture, results in mutations in the IBV 3b gene that confers a growth advantage to the virus when cultured in cells and chicken embryos. These mutations also increase the virulence of the IBV in the chicken embryos . Reports to date make it clear that the CoV accessory proteins are not essential for virus replication in vitro, but since these genes are maintained in the virus genomes under selective pressures, they have to confer a biological advantage to the virus in the natural environment of the infected host .Although recent evidence shows that the SARS-CoV accessory genes are expressed in the host during infection, their functions remain somewhat obscure. Currently, there are a range of proposed functions for these accessory proteins, including modulation of viral pathogenicity and replication, as well as acting as cell death inducers and interferon (IFN) antagonists, to name a few. In this review the current knowledge on SARS-CoV accessory proteins (ORFs 3a, 3b, 6, 7a, 7b, 8a, 8b and 9b) will be summarized and will include: (i) expression and processing; (ii) the effects on cellular processes; and (iii) functional studies.SARS-CoV 3a protein, also referred to as ORF3 , X1 , ORF3a and U274 , was the first of the SARS-CoV accessory proteins to be characterized. ORF3a, located at the 3´-end of the SARS-CoV genome between the S and E genes, encodes for a 274 amino acid (a.a.), 31 kDa accessory protein [21, 22, 25, 26] . The ORF3a N-terminal ectodomain (a.a. 1-34), contains three transmembrane regions (a.a. 35-56; 77-99; 103-125) , and is separated from the C-terminal endodomain (a.a. 209-264) by a central region containing a cysteine-rich domain (a.a 127-133), a YxxΦ domain (a.a. 160-163; where x is any a.a. and Φ is an a.a. with a bulky hydrophobic side-chain) and a diacidic domain (a.a. 171-173) .Although indirect immunofluorescence detects ORF3a partly in the rough endoplasmic reticulum , it is mainly found perinuclearly in the Golgi complex [24, 27, 28] . Antibodies to ORF3a also stain the plasma membrane [24, 27, 29] , where it is either secreted out of the cell [30, 31] or undergoes endocytosis . Following ORF3a translation, the diacidic domain ensures that the protein is efficiently exported from the endoplasmic reticulum to the Golgi apparatus, where ORF3a undergoes posttranslational modifications in the form of O-glycosylation before being inserted into the plasma membrane. Transport of ORF3a to the cell surface requires two protein sorting and tracking signals, the centrally located YxxΦ and diacidic motifs . Once in the plasma membrane, the side chains of cysteine in the cysteine rich domain allow ORF3a to form homo-and heterotetramers, which cluster together to form a potassium-permeable ion channel . In transfected cells, co-immunoprecipitation experiments have shown that membrane-bound ORF3a interacts with structural proteins S, M and E, as well as with SARS-CoV ORF7a, another accessory protein [24, 27, 35, 36] .The presence of ORF3a in the cytoplasm of infected cells has been confirmed through immunohistochemical analysis of tissue biopsies and/or autopsies of SARS-CoV-infected patients. In lung sections, the expression of ORF3a is restricted to the cytoplasm of pneumocytes and to multinucleated giant cells, a rare feature of the lung pathology in SARS . Moreover, sections taken from the small intestine show ORF3a localizes to the cytoplasm of surface enterocytes . The co-localization of ORF3a together with the SARS-CoV genome in the cytoplasm of pulmonary pneumocytes and small intestinal enterocytes is not totally unexpected as the lungs and small intestine are primary targets of SARS-CoV infection. The lung and small intestines are, however, not the only tissue targets for SARS-CoV. One of the cellular receptors for SARS-CoV, angiotensin-converting enzyme 2 (ACE2) , is also expressed by a certain population of peripheral CD14+ monocytes, from which osteoclasts are derived . Interestingly, the expression of ORF3a in a murine macrophage osteoclast precursor cell line enhances osteoclast differentiation, and may explain why many recovered SARS patients suffer from reduced bone density .The first attempts to elucidate the functions of coronaviral group-specific genes involved the generation of recombinant MHV from which the group-specific genes had been deleted. These recombinant viruses could still replicate, demonstrating that these genes are dispensable for virus growth in vitro. The mutant viruses did, however, still lead to a significant attenuation of the MHV when inoculated into mice, suggesting that coronaviral group-specific genes play a role in virus-host interactions and therefore contribute to viral fitness . Subsequently, much effort has been placed on elucidating the role of ORF3a in the life cycle of the SARS-CoV. The biological functions of ORF3a may be closely linked to those of the structural proteins with which they interact. The S protein is involved in viral assembly and packaging of coronaviruses (reviewed by ) and so the formation of inter-chain disulfide bonds between ORF3a and the S protein in the interior of the viral envelope may suggest that ORF3a is integrated into viral particles (reviewed by ). Indeed, ORF3a is incorporated into newly packaged mature SARS-CoV virions, suggesting that ORF3a is a structural protein [29, 30, 43] . Furthermore, the co-localization of ORF3a with M and E proteins, which are essential for viral assembly , suggests that ORF3a is important in SARS-CoV assembly or budding .There seems to be some conflicting data in the literature as to the role that ORF3a plays in viral replication. In Vero E6 cells transfected with a SARS-CoV deletion mutant lacking the 3a gene, the virus still replicates as efficiently as the wild-type, suggesting a non-essential function of ORF3a in viral assembly in vitro . Moreover, in a study that utilized small interference RNAs (siRNAs) to knockdown the expression of ORF3a in FRhK-4 cells, viral replication was also unaffected . However, the transfection of Vero E6 cells with siRNAs specific to ORF3a significantly reduced the yield of progeny SARS-CoV, signifying an inhibition of SARS-CoV replication cycle in these cells . This could have been a combinational effect due to the concurrent knockdown of ORF3b. In addition to a possible role in viral replication, ORF3a is likely to modulate viral release, as a study has shown that there is a significant reduction in viral release in FRhK-4 cells transfected with ORF3a specific siRNAs . Taking into account that ORF3a forms a potassium-sensitive ion channel in the plasma membrane [29, 34] , and ion channels for viral proteins control virus entry or release [46, 47] , the mechanism whereby ORF3a influences viral release warrants further investigation .ORF3a has also been shown to regulate various host cellular responses. Overexpression of ORF3a in several different cell culture models has demonstrated that it induces apoptosis [29, , upregulates mRNA and protein levels of fibrinogen in lung epithelial cells , activates C-Jun N-terminal kinase (JNK) and the transcription factor nuclear factor kappa B (NF-kappaB), which is involved in the activation of pro-inflammatory genes [20, 41, 52] and downstream from that, upregulates the production of pro-inflammatory cytokines and chemokines such as interleukin 8 (IL-8) and RANTES (CCL5) . Taken together, these changes to host cellular homeostasis imply a role for ORF3a in the pathogenesis of SARS.The occurrence of both apoptosis and necrosis in host cells during SARS-CoV infection suggests that the regulation of cell death is important for viral replication and/or pathogenesis (reviewed by ). The pro-apoptotic function of ORF3a is reliant on its ion channel activity and is induced via caspase-8 and -9 dependent pathways, i.e. through both the death receptor-and mitochondria-mediated pathways, respectively [23, 29, 48] . In addition, it has recently been reported that Bax, p53 and p38 MAP kinase also play roles in ORF3a-induced apoptosis . The fact that ORF3a utilizes more than one caspase pathway to trigger cell death may explain why tissue responses to SARS-CoV infection are distinct in different organs; for instance whereas the lung pathology is dominated by diffuse alveolar damage, the small intestine remains relatively intact .Even though the natural animal reservoir for SARS-CoV has not been identified, the threat of another SARS outbreak is still a disturbing possibility as bats have been shown to be natural reservoirs for different SARS-like coronaviruses (SL-CoV) . Like all accessory proteins, ORF3a is unique to the SARS-CoV, and shares an 83% homology to SL-CoV , making it a good candidate for the development of diagnostic assays, vaccines, and drugs. The immunogenic properties of ORF3a in SARS-CoV-infected patients has been shown to vary depending on: (i) whether or not the sera was collected from convalescent or recovered patients; (ii) whether or not full-length or truncated fusion proteins were used to detect immunoglobulin G (IgGs) to ORF3a; and (iii) the method used to detect these antibodies to ORF3a, namely western blot, ELISA or protein microarrays. To date, in general, IgGs to ORF3a could be detected in only ~60-73% of patient sera screened [35, . This could be due to frame-shift mutations within the 3a gene, resulting in a heterogeneous population of sgRNA3 transcripts in patients with acute SARS-CoV infection containing copies of both wild-type and mutant 3a genes. [24, 60] . These mutant 3a genes encode for proteins with distinctively shorter N-termini than the wild-type forms .The neutralizing activity of anti-3a sera has subsequently been assessed in vitro. The sera from rabbits immunized with recombinant ORF3a (a.a. 125-274) does not exhibit any significant neutralizing activity on SARS-CoV infected Vero E6 cells . In another study, however, sera from rabbits immunized with a different recombinant ORF3a (a.a. 15-28) contains neutralizing antibodies that do inhibit SARS-CoV infection in Vero E6 cells . These contrasting results could be explained by differences in the immunogenicity of the central region and C-terminal domain compared to the N-terminal ectodomain, respectively. Indeed, ectodomain-specific antibodies (a.a. 15-28) from convalescent-phase patient plasma recognize and destroy SARS-CoV-ORF3a-expressing cells . A more recent study has showed that mice immunized with full-length 3a DNA elicit strong humoral and Th1-based cellular immune responses. The vaccine stimulated IFN-γ production mainly by CD8+ T cells and interleukin-2 (IL-2) mainly by CD4+ T cells . Whether or not ORF3a will be useful as a subunit vaccine for humans, still remains to be seen though.ORF3b is predicted to express the 154 amino-acid 3b protein, also referred to as ORF4 , X2 and U154 , via an internal ribosomal entry mechanism [21, 22, 25] . SARS-CoV ORF3b has been described as a multi-localized protein due to its shuttling behavior . Although the protein initially accumulates in the nucleolus , it subsequently translocates to the outer membrane of mitochondria [63, 65] . Nuclear export of other accessory proteins, namely SARS-CoV ORF9b, has also been previously reported but how this spatio-temporal distribution impacts on ORF3b behavior and function is still unclear . In fact, one study has reported that ORF3b is localized primarily in the nucleus with no evidence of mitochondrial localization .Immunohistochemical analysis of tissue biopsies and/or autopsies of SARS-CoV-infected patients has failed to demonstrate the presence of ORF3b in vivo . The antigenicity of ORF3b has also come into question as IgGs to ORF3b are detectable in SARS patient sera in some studies [69, 70] , but not in others . To date, the presence of ORF3b in SARS-CoV-infected Vero E6 cells is the only evidence for the expression of this protein .In comparison to ORF3a protein, much less is known about the function(s) of ORF3b protein. SARS-CoV deletion mutants lacking the 3b gene replicate to levels similar to those of wild-type virus in several tissue culture cell types , suggesting that, like ORF3a, ORF3b is dispensable for viral replication in vitro. Although ORF3b may not be involved in viral replication, a role for this protein in viral pathogenicity has been inferred through its ability to modulate the host innate immune response to SARS-CoV infection . ORF3b has been shown to inhibit the synthesis of type 1 IFNs (IFN-alpha/beta), major components of the innate immune response; as well as with IFN signaling [67, 68] . Considering that SARS-CoV-infected cells do not produce IFN [71, 72] , it is possible that ORF3b provides the SARS-CoV with a mechanism to subvert the IFN response, and therefore the innate immune response . However, when mice are infected with mutant SARS-CoV lacking 3b genes, the deletion viruses grow to levels similar to those of wild-type virus in the lungs of BALB/c mice at day 2 post infection . This demonstrates that SARS-CoV is therefore able to inhibit the host IFN response without the 3b gene and emphasizes the limitations of transfection studies. Recently, SARS-CoV ORF3b was shown to modulate the transcriptional activity of the host factor, RUNX1b, indicating a physiological role in upregulating cytokines and chemokine levels during SARS-CoV infection . ORF3b also induces AP-1 transcriptional activity through activation of JNK and ERK pathways .SARS-CoV ORF3b has also been shown to affect cell cycle and apoptosis-regulation but again, there are discrepancies within the literature. When overexpressed in Vero E6 cells, ORF3b induces apoptosis and necrosis with an increase in caspase-3 activity . Similarly, expression of ORF3b causes cell growth arrest in GO/G1 phase, inducing cell-or tissue-specific apoptosis . Since data show massive necrosis in the lungs, spleen and lymph nodes, as well as increased apoptosis in cells from the spleen, liver, lung and lymph nodes of SARS patients , it may suggest that ORF3b contributes to these clinical observations.ORF6, also referred to as p6 , is a small 63 amino acid membrane protein with no known homologues. This relatively small peptide has a ~44 amino acid residue amphipathic N-terminus and ~20 residue polar C-terminus ; the C-terminus contains intracellular protein sorting motifs . The protein is found on cytoplasmic membranes in the endoplasmic reticulum and Golgi-complex of cells overexpressing ORF6 [82, 83] . The membrane topology of this protein is N-endo C-endo, with a membrane-spanning region ~36 a.a. in length [79, 84] . Using immunohistochemical staining, ORF6 was detected in both SARS-CoV infected cells, as well as in the pneumocytes of the lung and surface enterocytes of the terminal ileum from 8 SARS autopsy cases ; the latter corresponds to the tissue-sites of infection reported by another group .Recent work has hinted at a few possible functions for SARS-CoV ORF6. Sucrose gradient purification of SARS-CoV particles and virus capture assays show that ORF6 is associated with purified virions, with the presence of SARS-CoV proteins E, M and S sufficient for incorporation of ORF6 into virus-like particles (VLPs). When overexpressed alone in cells [79, 85, 86] , as well as in virus infected cells , ORF6 is released in membrane-associated vesicular structures, most likely a subpopulation of endosomal/lysosomal vesicles . The C-terminus portion ORF6 contains the sequence motif YSEL, which has been shown to target proteins for internalization from the cell membrane to endosomal/lysosomal vesicles . Several groups have previously shown that coronaviruses use similar vesicular structures within infected cells, suggesting that these vesicles have a role in the viral replication cycle [86, . Recombinant studies with SARS-CoV lacking ORF6 remained viable in vitro and in vivo at a high multiplicity of infection (moi) . In contrast, an enhancement of replication of the virus is observed at low moi, which is consistent with previous findings using recombinant studies . Recombinant studies using attenuated MHV expressing SARS-CoV ORF6, show accelerated MHV replication and lead to higher viral titers in both cell culture and the mouse central nervous system [83, 93] . In fact, the presence of ORF6 leads to increased virulence of normally sub-lethal infections in mice . The recombinant MHV studies by Tangudu and colleagues also showed that ORF6 associated with viral RNA, co-localized with replication viral RNA on cytoplasmic vesicles and increased MHV protein synthesis [20, 92] . Lastly, not only did ORF6 partially co-localize with nonstructural protein 3 (nsp3), a marker for virus replication complexes , it also interacted with nsp8, another replicase protein , pointing to a putative role for ORF6 in replication. Collectively, these results provide strong evidence that ORF6 plays a role in enhancing SARS-CoV replication and pathogenicity by interacting with membrane-bound replication and assembly machinery .Interestingly, overexpression of ORF6 in CHO cells induces DNA synthesis, of which the biological significance is still not clear . However, this was done in the absence of other viral proteins and needs to be verified in the context of the whole viral genome. On the other hand, ORF6 suppresses the expression levels of co-transfected expression constructs. The ability of ORF6 to interfere with the expression of co-expressed constructs has previously been linked to the blockage of nuclear translocation of these proteins [80, 93] . Gunalan and co-workers also found that, even though ORF6 did not have a global effect on protein synthesis, it down-regulated the mRNA level of co-transfected nsp8. However, they linked the N-terminus diacidic cluster motif, DDEE which is similar to the putative diacidic motif DxE, which governs ER export and localization to different membranous compartments [32, 86] , to this suppression .ORF6 is a type I IFN antagonist, and its expression suppresses the induction of both IFN as well as the IFN signaling pathways [68, 94] . The protein inhibits IFN signaling by interacting via its C-terminal region to the nuclear import protein, karyopherin α2, thereby sequestering it in the cytoplasm and indirectly preventing movement of signal transducer and activator of transcription factor 1 (STAT1) to the nucleus ; the 18 a.a. N-terminus is also required for maximal inhibition and this inhibition probably plays a role in the immune evasion of SARS-CoV . ORF9b immunoprecipitated with ORF6 from extracts of SARS-CoV infected cells and subsequent mass spectrometry analysis reveal an interaction of the two in biologically relevant conditions. This interaction has been validated by co-localization of both proteins in the cytoplasm of SARS-CoV infected cells . Taken together, findings support the hypothesis that ORF6 could be a minor structural protein and could be an important virulence factor during SARS-CoV infection in vivo.The SARS-CoV ORF7a protein, also known as ORF8 , U122 or X4 [22, 97] , is encoded by the first ORF of sgRNA7 [25, 26] . The protein is a 122 a.a. long, type 1 transmembrane protein with an N terminal signal peptide sequence of 15 aa, an ectodomain of 81 aa, and a C terminus transmembrane domain containing 21 aa residues followed by a short cytoplasmic tail of 5 aa residues . The ORF7a shares a 26% sequence homology with human CD62L protein, an adhesive molecule that plays important roles in immune and inflammatory responses, suggesting that ORF7a may adhere to other cells and influence their functions, such as leukocyte adhesion, migration and the inflammatory response . In fact, the ectodomain bears high structural and similar sequence homology to members of the immunoglobulin superfamily, namely the D1 domains of ICAM-1 and ICAM-2 [98, 99] . The ectodomain of ORF7a binds to the alpha integrin I domain of human lymphocyte function-related antigen 1 (LFA-1) on Jurkat cells, suggesting that LFA-1 may be an attachment factor or even the receptor for SARS-CoV on human leukocytes . The C-terminus contains a motif (KRKTE) that typically serves as an ER retrieval signal but in the case of ORF7a, serves as a dibasic ER export signal that allows for the ER exit of ORF7a to the Golgi apparatus .Expression of ORF7a in SARS-CoV-infected cells and lung tissue of SARS patients has been confirmed , but there are discrepancies in the literature as to the exact subcellular localization of the protein. It seems to localize predominantly to the intermediate compartments between the ER and Golgi apparatus [13, 100] , partially in the mitochondria , as well as in the cytoplasm . The export of ORF7a from the ER to the Golgi apparatus is regulated via the ER export motif [96, 102] . Therefore the presence of ORF7a in the ER-Golgi intermediate compartment, where coronaviruses are known to assemble , could explain how ORF7a becomes incorporated into purified SARS-CoV particles . The interaction between ORF7a and the viral structural proteins M, E and S [31, 96] , which are important for viral assembly, implies that ORF7a is important in the viral replication cycle and that virion-associated ORF7a protein may exert its function early in infection . ORF7a also interacts with the accessory ORF3a which itself interacts with M, E and S, suggesting that these viral proteins may form complexes in SARS-CoV-infected cells .The incorporation of ORF7a in VLPs occurs however in the absence of S or ORF3a protein interaction . The presence of 7a protein in the cytoplasm could explain why ORF7a is able to inhibit protein translation . How this would be beneficial to the viral life cycle is unclear as the SARS-CoV relies on host cell protein synthesis to replicate viral proteins. Expressed ORF7a also interacts with human small glutamine-rich tetratricopeptide protein , human lymphocyte functionassociated antigen 1 (LFA-1) on Jurkat cells and with human Ap4A-hydrolase .ORF7b is encoded by the second ORF of sgRNA7 and may be expressed by \"leaky scanning\" of ribosomes . This protein has been partially characterized and is a 44 amino-acid long integral membrane protein that is localized to the Golgi compartment in SARS-CoV-infected cells . The protein is found in association with intracellular virus particles and also in purified virions .To date, there is very little experimental evidence to support a role for ORF7a or ORF7b in the replication of SARS-CoV. Monkey Vero E6 cells transfected with SARS-CoV deletion mutants lacking the 7a and 7b genes still release virions at levels similar to those of wild-type virus [102, 106] and virion morphology remains unchanged . In airway epithelial cultures generated from Golden Syrian hamster tracheas, SARS-CoV replication titers are comparable to infected human cultures . Transgenic mice expressing the SARS-CoV receptor ACE2, one of the receptors for SARS-CoV, also still release virions when infected with mutant SARS-CoV . Moreover, studies in immunosuppressed Syrian golden hamsters, an animal model for the study of SARS-CoV, have shown that infection with mutant SARS-CoV bearing deletions in gene 7 does not result in significant changes to replication kinetics, tissue tropism, morbidity or mortality . One study utilizing siRNAs targeting specifically ORF7a and ORF7b has, however, demonstrated that the silencing of these proteins significantly reduces the yield of progeny virus . The 70% decrease in progeny virus is not due to a disruption of the full-length genomic mRNA but reflects inhibition of both sgRNA 7 and 8, resulting in the subsequent knockdown of ORF7a, -7b, -8a and -8b . Moreover, the limitations of in vitro studies must be taken into consideration. For example, although the 7b gene of feline coronavirus is easily lost upon virus adaptation to cell culture, it is strictly maintained in naturally occurring strains and its loss is correlated with reduced virulence . The importance of ORF7a and ORF7b during the replication cycle of SARS-CoV therefore still remains uncertain.ORF7a may be important for viral host interactions and therefore contribute to viral pathogenesis in vivo by inducing inflammatory responses in SARS. Like ORF3a, ORF7a activates NF-kappaB and the IL-8 promotor , and downstream from that, augments the production of pro-inflammatory chemokines such as IL-8 and RANTES . ORF7a may also play a role in the pathogenesis of SARS-CoV by way of the protein's contribution to virus-induced apoptosis. Several studies have shown that overexpression of ORF7a induces apoptosis in a caspase-3-dependent manner [101, 109] as well as independently via p38 MAPK . Although deletion of gene 7 does not alter markers of early apoptosis or activation of caspase 3, viruses with gene 7 disruptions are not as efficient as wild-type virus in inducing DNA fragmentation, an apoptotic feature in later stages of infection . Overexpression of Bcl-XL protein, a member of the pro-survival Bcl-2 family, blocks ORF7a-induced apoptosis in 293T cells, suggesting that ORF7a regulates host cell apoptosis at or upstream of the Bcl-2 family in the apoptotic pathway . In addition, ORF7a forms complexes with Bcl-XL protein and binds to other pro-survival Bcl-2 family members, indicating that it may induce apoptosis by interfering with the pro-survival function of Bcl-XL protein . It must be kept in mind though that ORF7a is not the only apoptosis-inducing factor during SARS-CoV infection (review by ) and that the actual levels at which ORF7a is expressed in infected tissues is still unknown bringing into the question the validity of the overexpression data in the in vivo context.Although ORF7a is expressed in SARS-CoV-infected cells [96, 97] and in lung tissue of SARS patients, specifically in the bronchial epithelial cells, peripheral blood erythrocytes and leukocytes , it does not display strong immunogenicity. Protein microarrays fail to detect antibodies to ORF7a in SARS patient sera and SARS patient sera has low to non-specific antibody responses to ORF7a epitopes when tested on a peptide chip platform . The selected ORF7a epitopes are therefore poorly immunogenic, and so ORF7a is not likely to serve as a potential marker for SARS-CoV infection or be important for vaccine development. The ORF7b protein is likely to be expressed in vivo because anti-7b antibody has been detected in SARS convalescent patient serum .Whereas the SARS-CoV isolated from humans infected during the peak of the SARS epidemic, encodes two accessory proteins termed ORF8a and ORF8b, the SARS-CoV isolated from animals and some early human isolates, encodes for a single 122 a.a. protein, ORF8ab. Interestingly, this difference is due to a 29 nucleotide deletion in the mRNA8 of SARS-CoV from some animal species , that splits ORF 8 into ORF8a and ORF8b, encoding 39-and 84-residue polypeptides, respectively . In fact, larger deletions were reported in sequenced ORF8 genes from a cluster of viruses that were isolated from humans later during the SARS epidemic [111, 115] .Vaccinia virus expressed SARS-CoV ORF8ab, contains a cleavable signal sequence, which directs the movement of the precursor protein to the ER and then mediates its translocation into the ER lumen, where it is N-glycosylated on the N81 residue [110, 116] and subsequently assembled into disulfidelinked homo-multimeric complexes, which remains stably in the ER . This retention and accumulation of ORF8ab in the ER occurs in the absence of a recognized ER retention signal [110, 117] . Not only do in vitro and in vivo studies show that ORF8ab is ubiquitinated, but both ORF8a and ORF8b also binds to monoubiquitin and polyubiquitin, suggesting the potential involvement of these proteins in the pathogenesis of SARS-CoV . ORF8ab interacts with SARS-CoV proteins S, M, ORF3a and ORF7a , all of which have been shown to be structural proteins.Since ORF8a is too small for its signal sequence to function, it is most likely directly released from the ribosome and remains in the cytosol in its precursor form . Antibodies to ORF8a have been detected in a small number of SARS patients, proving that the protein is indeed expressed during virus infection . This group also reported that ORF8a not only enhances viral replication, but also induces apoptosis through a mitochondria-dependent pathway .When overexpressed, the 9.6 kDa ORF8b localizes to both the nucleus and cytoplasm of Vero E6 and CHO cells . In another study, however, ORF8b was not detected in SARS-CoV-infected cells or when expressed from mRNA's mimicking mRNA8. However, after it was cloned immediately downstream of a T7 promoter, a soluble, unmodified and monomeric ORF8b protein was expressed in the cytoplasm, which was extremely unstable and rapidly degraded . Similarly, in the absence of the ORF8a region, ORF8b undergoes rapid degradation by proteasomes . ORF8b down-regulates the expression of SARS-CoV E, a process that could be inhibited by the addition of proteasome inhibitors . For this reason, E is not detectable in SARS-CoV-infected cells that are expressing high levels of ORF8b . Similarly to ORF6, overexpression of ORF8b in mammalian cells also induces DNA synthesis. However, co-expression of SARS 8b and SARS 6 did not elicit synergistic effects on DNA synthesis . The biological function of this induction is still to be elucidated.It is obvious that the 29-nucleotide deletion disrupts the proper expression of the SARS-CoV ORF8. Even though this mutation does not appear to adversely affect the survival of the virus, it has become clear that ORF8ab, -8a and -8b have different stabilities and varying functions, and could contribute differently to viral replication and pathogenesis in vivo. Taken together these findings suggest that ORF8ab may modulate viral replication and/or pathogenesis in a yet unknown manner.Bi-cistronic mRNA9 encodes for ORF9b which is translated via a leaky ribosomal scanning mechanism . This 98 amino acid peptide is synthesized from an alternative complete reading frame within the N gene [37, 121] . It has no homology with known proteins and, as yet, its function has not been confirmed. Both anti-9b antibody and ORF9b antigen have been detected in clinical samples from SARS patients [37, 59] , with the latter also detected in virus infected cells . Analysis, using fluorescence microscopy of live transfected cells and indirect immunofluorescence of transfected fixed cells, shows that ORF9b is exported outside of the cell nucleus and localizes to the endoplasmic reticulum [66, 122] ; the authors suggest that the 46-LRLGSQLSL-54 amino acid sequence motif of ORF9b functions as a nuclear export signal (NES) . More recently, it was confirmed that ORF9b interacts with the cellular protein Crm1 and gets exported out of the nucleus using an active NES; this nucleocytoplasmic export of ORF9b is linked to apoptosis . Cellular CRM1 (Exportin1) protein functions as a nuclear export receptor for proteins carrying a Rev-like NES in a process that also requires the GTP bound form of the Ran GTPase . The authors concluded that the interaction of ORF9b with Crm1 is essential for the proper degradation of ORF9b and blocking the nuclear export of the latter induces apoptosis. Also, ORF9b was shown to enter the nucleus in the absence of any nuclear localization signal (NLS) by passive transport, independent of the cell cycle .The resolved crystal structure of ORF9b shows that the protein has a novel fold, a dimeric tent-like β structure and a central hydrophobic cavity involved in lipid binding. Since the protein also co-localizes with intracellular vesicles, it has been speculated that the protein has a role in virus assemble via membrane association . Interestingly, recent evidence shows that ORF9b is a structural protein incorporated into both purified virions and VLPs, with efficient incorporation dependent on the co-expression of viral structural proteins E and M . The interaction of ORF9b with another SARS-CoV accessory protein, ORF6, was discussed earlier in this review.Viruses generally encode for: (i) proteins functioning in the replication and transcription of the viral genome; (ii) structural proteins of the virion; and (iii) accessory proteins that enable, facilitate or modulate the infection process. These accessory proteins normally act by interfering with cellular processes or by modulating virus-host interactions at the level of the organism (Table 1 ). It has also been shown that these proteins are often dispensable for virus replication in vitro, but required for optimal replication and virulence in the natural host . SARS-CoV is the prototype human CoV and has been studied extensively. In addition, SARS-CoV contains eight encoded accessory genes interspersed amongst the viral structural genes [22, 25, 126] , the most of the known CoVs. Interestingly, these accessory genes are present in SARS-CoVs isolated from bats, civet cats, raccoon dogs, and humans, suggesting important accessory functions in a variety of hosts [55, 127, 128] . Evidence confirms that these accessory proteins are indeed expressed during SARS-CoV infection, but for many, their functions are yet to be determined. Five of the SARS-CoV accessory proteins (ORF3a, -6, -7a, -7b and -9b) have been identified as minor viral structural components, and others induce ion channels, act as cell death inducers and IFN antagonists ( Table 1 ). It would be interesting to determine whether the high SARS-CoV accessory gene number correlate with the increased virulence of SARS-CoV when compared to the other known human CoVs. For this reason, the role of the accessory proteins in pathogenesis must be elucidated. Table 1 . Summary of the effects of the SARS-CoV accessory proteins on cellular processes.Enhances osteoclast differentiation in murine macrophage osteoclast precursor cell line Induces apoptosis via caspase-8 and -9 dependent pathways [29, ; Bax, p53 and p38MAP kinase also plays role in ORF3a apoptosis Upregulates fibrinogen levels in lungs Activates JNK and NK-kappaB, which is involved in the activation of pro-inflammatory genes [20, 41, 52] Upregulates the production of pro-inflammatory cytokines and chemokines such as IL-8 and RANTES [20, 41, 52] ORF3b Upregulates cytokines and chemokines by modulating the transcriptional activity of RUNX1b AP1 transcriptional activity through activation of JNK and ERK pathways ORF6 Induces DNA synthesis Suppresses the induction of type 1 IFN and IFN signaling pathways [68, 94] ORF7a Induces inflammatory response by activating NF-kappaB and the IL-8 promotor Augments the production of pro-inflammatory chemokines such as IL-8 and RANTES Induces apoptosis [13, 101, 109] ORF7b ND #Induces apoptosis through mitochondria-dependent pathway ORF8b Induces DNA synthesis ORF9bInduces apoptosis ¥ : The effects of the accessory proteins on cellular functions in cells transfected with DNA constructs (for expressing individual accessory proteins), or in cells infected with SARS-CoV or recombinant viruses expressing the individual accessory proteins, are reported. # : Not Determined.\n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.978\tSevere acute respiratory syndrome (SARS) is a new infectious disease caused by a novel coronavirus that leads to deleterious pulmonary pathological features. Due to its high morbidity and mortality and widespread occurrence, SARS has evolved as an important respiratory disease which may be encountered everywhere in the world. The virus was identified as the causative agent of SARS due to the efforts of a WHO-led laboratory network. The potential mutability of the SARS-CoV genome may lead to new SARS outbreaks and several regions of the viral genomes open reading frames have been identified which may contribute to the severe virulence of the virus. With regard to the pathogenesis of SARS, several mechanisms involving both direct effects on target cells and indirect effects via the immune system may exist. Vaccination would offer the most attractive approach to prevent new epidemics of SARS, but the development of vaccines is difficult due to missing data on the role of immune system-virus interactions and the potential mutability of the virus. Even in a situation of no new infections, SARS remains a major health hazard, as new epidemics may arise. Therefore, further experimental and clinical research is required to control the disease.Severe acute respiratory syndrome (SARS) is the first new infectious disease of this millennium. SARS has originated from Southern China at the end of 2002 and has a high mortality and morbidity. Within a period of six months beginning at the end of 2002, the disease has affected more than 8,000 people and killed nearly 800 . The disease poses a new threat for respiratory medicine and rep-resents a challenge for antiviral drug development and administration [2, 3] .SARS is caused by a novel, SARS-associated coronavirus (SARS-CoV) which has been identified by a World Health Organization (WHO)-led global laboratory network. The first cases of SARS were reported from a hospital in Hanoi, Vietnam, by Carlo Urbani, a WHO scientist who himself died from the disease . After reports from health authorities in Hong Kong on the outbreak of a new form of epidemical atypical pneumonia in public hospitals, the WHO issued a global alert on the disease. During this period, cases of SARS were also reported from China, other Asian countries and even other continents including America (Canada, U.S.A.) and Europe (Germany).Shortly after the initial global alert, the WHO initiated a collaborative multi-center research project on SARS diagnosis, led by eleven principal laboratories in nine countries . Using modern communication technologies to optimize the analysis of SARS tissue samples, it was soon shown that a novel coronavirus is the causative agent of SARS (SARS-CoV) . Due to the death of Carlo Urbani who first identified the new disease, the first isolate of the virus was proposed to be named Urbani strain of SARSassociated coronavirus, but a final terminology has not been proposed so far . Since Koch's principles have been shown to be fulfilled by the new pathogen [10, 11] , it is not necessary to call the virus SARS-associated and the general agreement is now to call it SARS coronavirus (SARS-CoV).Parallel to the progress made in the epidemiology and clinical diagnosis which has recently been demonstrated by numerous case reports, clinical studies and definitions , scientists have also revealed basic mechanisms of the underlying causative agent, the SARS coronavirus. As it is crucial for future strategies that SARS is detected in its earliest stages and that therapeutic options are optimized, insights into the molecular mechanism of SARS have to be used to develop new therapeutic strategies and vaccines.While other reviews have focused on the epidemiology, clinical presentation and potential treatment of SARS, the present overview aims to analyze and present the currently available data on molecular mechanisms of SARS. In this respect, it is important to underline that in the present state of no specific drug or vaccine being available, research on molecular mechanism is crucial to identify potential treatment targets.Prior to the development of therapeutic regimes based on molecular mechanisms of the disease, the causative agent had to be isolated and analysed. Soon after the fast establishment of the international WHO laboratory network, rapid progress was made in the identification process of the causative agent, and it was reported that SARS is most probably caused by a novel strain of the family of coronaviruses . These viruses are commonly known to cause respiratory and gastrointestinal diseases of humans and domestic animals [12, 13] . The group of coronaviruses is classified as a member of the order nidovirales, which represents a group of enveloped positive-sense RNA viruses consisting of coronaviridae and arteriviridae . Viruses of this group are known to synthesize a 3' co-terminal set of subgenomic mRNAs in the infected cells .Soon after the identification of a new coronavirus as the causative agent of SARS and of a southern Chinese province as the first area of occurrence, animal species of this area have been speculated to be the origin of the SARS-CoV. As analysis of the SARS-CoV genetic sequence revealed large differences to any other currently known coronaviruses in humans or domestic animals [16, 17] , it was hypothesized that the new virus might originate from wild animals. This hypothesis was supported by a search for coronaviruses in wild animals sold on markets in southern China, which identified the presence of a coronavirus in civet cats. This animal coronavirus was shown to have a sequence identity of more than 99% to the SARS coronavirus with only a limited number of deletions and mutations between both viruses. SARS-CoV has a deletion of 29 nucleotides relative to the civet cat virus, indicating that if there was direct transmission, it went from the animal to man, because deletions occur probably more easily than insertions. Recent reports indicate that SARS-CoV is distinct from the civet cat virus and it has not been answered so far if the civet cat virus is the origin of the SARS-CoV or if civet cats were also infected from other species . Therefore, there are no data available on the possibility of horizontal transmission between animals, and the question whether the jump of the virus from an animal to humans was a single accident or may frequently occur in future with the animals as dangerous reservoirs for future SARS epidemics remains unanswered. So far, the SARS-CoV has been reported to be able to infect not only humans but also macaque monkeys , domestic cats, and ferrets . However, transmission of the virus from the domestic cat to man has not been shown. The ability of the SARS-CoV to infect other animal species could point to potential natural reservoirs of the virus. In this respect, coronaviruses are known to relatively easily jump to other species. I.e., the human coronavirus OC43 shares a high degree of genetic sequence homology to bovine coronavirus (BCoV) and it is commonly assumed that it has jumped from one species to the other [21, 22] . In the same way, BCoV has been reported to be able to infect humans and cause diarrhea . Whereas the precise mechanisms of these species jumps remain unclear, it is most likely that they represent the results of mutations and epidemiological studies of coronavirus infections in wild animals will therefore be crucial for future understanding and control of new SARS outbreaks.Until the identification of the new SARS-CoV, the coronaviruses have been divided into three subgroups, which differ with respect to their genome . The first group consists of viruses such as the human coronavirus 229E (HCoV-229E), porcine respiratory coronavirus (PRCV), porcine transmissible gastroenteritis virus (TGEV), feline infectious peritonitis virus (FIPV) and feline enteritis virus (FEV) or the canine coronavirus (CCoV). The second group comprises human coronavirus OC43 (HCoV-OC43), bovine coronavirus (BCoV), and mouse hepatitis virus (MHV), and the third group mainly consists of avian species such as the chicken infectious bronchitis virus (IBV). Whereas the SARS-CoV has been shown to crossreact with some group I coronavirus antibodies , its genetic sequence does not belong to this group. Within the nucleic acid or protein sequence phylogenetic trees of the coronavirus family, the SARS-CoV has first been located at an equal distance from the second and third group, irrespective of which SARS-CoV RNA region is used for analysis [6, 16, 17] . Therefore, the SARS-CoV may represent the first member of a new group of coronaviruses ( Figure 1 ). However, the taxonomy is still no clear [19, 25] , and recent studies that focused on the N-terminal domain of the spike protein and on poorly conserved proteins such as Nsp1, matrix protein, or nucleocapsid, have suggested a relation to group II viruses . A similar conclusion can be drawn if the polymerase gene is examined, pointing to an early split-off from the coronavirus group 2 lineage .Despite the fact that this new virus most likely jumped to humans from wild animal species, it has remarkably well adapted to the human organism as shown by its high person-to-person transmissibility.The structure of the SARS viral RNA is organized in open reading frames (ORF) and contains a total of approximately 30,000 nucleotides [6, 16, 17] .Recently, 61 SARS-CoV sequences derived from the early, middle, and late phases of the SARS epidemic together with two viral sequences from palm civets were analyzed . Genotypes characteristic of each phase were discovered, and it was found that the neutral mutation rate of the viral genome was constant but the amino acid substitution rate of the coding sequences slowed during the course of the epidemic. The spike protein showed the strongest initial responses to positive selection pressures .Only ORFs exceeding fifty amino acids in translational capacity are considered relevant as they contain the sequences for the structural and functional properties of the virus and are therefore of potential interest for the development for future therapeutic strategies. The comparison of the different SARS-CoV ORFs with those of other coronaviruses reveals a familiar pattern of structural gene arrangement with replicase and protease genes (gene 1a-1b) and the spike (S), envelope (E), membrane (M) and nucleocapsid (N) genes in a typical 5'-to 3' order of appearance . The proteins encoded by these genes may be targets for novel treatments. Between these wellknown genes, a series of ORFs of unknown function was found: There are two ORFs situated between the spike and the envelope genes and three to five ORFs between the membrane and nucleocapsid genes. Comparison of this gene organization with other known coronaviruses does not indicate a closest proximity to group II coronaviruses. Also, the SARS-CoV genomic sequence does not contain a gene for hemagglutinin-esterase (HE) protein, which is present in the majority of group II coronaviruses.Two-thirds of the SARS RNA is organized in the gene 1a-1b. The sequence of this gene is highly conserved among all coronaviruses . ORFs 1a and 1b encode two polyproteins, pp1a and pp1ab, the latter through a ribosomal frameshifting mechanism. These polyproteins are processed by virus-encoded proteinases, to yield 16 individual proteins. Most potential gene 1a-1b products are fairly well conserved between SARS-CoV and other coronaviruses [17, 29] . Many of their functions are unknown but it is suggested that they participate in viral RNA replication, making them potential targets for the development of antiviral compounds. Therefore, research efforts will focus on these proteins. One exception from the overall conservation of SARS-CoV gene 1a-1b is the lack of a sequence coding for PL1 pro , one of the two papain-like proteinases operating on cleavage sites at the N-terminus of the polyproteins (Figure 2 ). The main proteinase (M pro ), also called 3C-like protease (3CL pro ), is responsible for the cleavage of all the remaining proteins encoded by gene 1a-1b [29, 30] .Apart from gene 1, coronavirus genes are known to be usually expressed from subgenomic mRNAs. They share a common leader sequence at the 5'-end and initiate at different places in the genome extending toward the 3'-end of the virus genome . Some ORFs may also be unconventionally translated from a single mRNA. As these uncommon translation mechanisms are not very Whereas the ORFs between the structural protein genes are very heterogeneous among the different coronaviruses and not essential for viral replication, recent studies suggested that deletion of non-essential ORFs may result in a reduced virulence . In agreement with this, some of these non-essential ORFs of the new SARS-CoV genome may be responsible for the high SARS-CoV virulence.So far, five to eight subgenomic mRNAs were found in SARS-CoV-infected cells [17, 27] . Thiel and colleagues performed the first detailed study on mechanisms and enzymes involved in SARS-CoV genome expression (Figure 2 ) . They determined the sequence of the SARS-CoV isolate Frankfurt 1 and characterized the major RNA elements and protein functions involved in the genome expression by characterizing regulatory mechanisms such as the discontinuous synthesis of eight subgenomic mRNAs, ribosomal frameshifting and post-translational proteolytic processing. Also, the activities of SARS-CoV enzymes such as the helicase or the two cysteine proteinases (PL2 pro and M pro ) were addressed as they are involved in replication, transcription or post-translational polyprotein processing .In conclusion, research in the area of coronavirus gene expression is important to delineate components which directly affect SARS-CoV virulence.The structural proteins of the new SARS-CoV are potential targets for new treatment options. The new SARS-CoV only contains the three envelope proteins, spike (S), envelope (E), and membrane (M) but not the hemagglutinin-esterase (HE) protein, which is present in some coronaviruses of the second group.The spike glycoprotein is responsible for the characteristic spikes of the SARS-CoV ( Figure 3 ). Intra-and extracellular proteases often cleave the S protein into S1 and S2 domains, with the cleavage process often increasing infectivity of the virus. Molecular modelling has been performed for the S1 and S2 units of the SARS-CoV spike protein [33, 34] . The spike proteins of coronaviruses are reported to bind to receptors on their target cells and the domains responsible for receptor-binding are commonly situated in the N-terminal region of S1 . The spikes consist of oligomeric structures, that are formed by heptad repeats of the S2 domain which also represent a fusion peptide sequence. This peptide is responsible for the coronavirus fusion activity.The SARS-CoV has also been reported to cause the formation of syncytia in vivo, but so far only under the condition of cultured Vero cells . The SARS-CoV S protein seems to have most of its characteristics in common with the S proteins of other coronaviruses, but it will be important for the understanding of the SARS-CoV pathogenic properties to identify the exact conditions of membrane fusion, i.e. pH dependency and protease sensitivity, which can increase the infectivity. The envelope and membrane proteins are integral membrane proteins and required for virus assembly . In the case of the murine coronavirus MHV-A59 the coexpression of the E and M proteins but A B not the S or N proteins is needed for the release of viruslike particles (VLP) . The nucleocapsid and viral core of the SARS-CoV are likely to be formed by the N protein.An interesting feature of the SARS-CoV and other coronaviruses is the resistance against the gastrointestinal fluids despite the lipid composition of their envelope. It has been reported that the SARS-CoV can survive in diarrheal stool for four days and also, patients with SARS often suffer from gastrointestinal symptoms with the virus to be detected in the stool . As the molecular basis for the envelope's resistance against acidic environments and gastrointestinal enzymes is unclear, further research has to be carried out in this area which is important for the control of future SARS outbreaks.It is unclear when and how novel pathogens such as the SARS-CoV cross the barriers between their natural reservoirs and human populations, leading to the epidemic spread of novel infectious diseases . As with the SARS-CoV, new pathogens are believed to emerge from animal reservoirs and a variety of molecular mechanisms may contribute to the evolution of the viruses or bacteria. Due to the estimated error frequency of 1 × 10 -4 for RNAdependent RNA polymerases , RNA viruses such as the SARS-CoV can undergo mutation at a high frequency. The SARS-CoV seems to be relatively genetically stable as the RNA sequences from different SARS patients were quite homogeneous. Even the entire genomic sequences of virus isolates from different continental areas did not differ by more than ten amino acids and it seems that two lineages of the virus can be traced . This obvious contradiction to the high potential error rate of the RNAdependent RNA polymerase suggests the presence of some proofreading mechanism connected with this enzyme. In fact, a detailed analysis of the SARS-CoV genome by bioinformatics indicates the presence of an exonuclease activity .Next to mutations, a further threat of the SARS-CoV is based on the ability of coronaviruses to undergo RNA recombination at a high frequency . For a variety of other coronaviruses, both recombination and mutation in natural infections have been shown to contribute to the diversification of the coronaviruses. Because of the demonstrated ability of coronaviruses to recombinate, the question whether the SARS-CoV will show a higher frequency of mutations within possible future seasonal changes or in respond to drug treatment is an issue of major concern. It was reported that in the initial phases of the SARS epidemic, the mutation rate was high in the gene for the spike protein, but this stabilized during the middle and final stages of the 2003 epidemy . Thus, the virus had experienced great pressure to adapt to the new host after crossing the species barrier, but has then been optimized .Although human coronaviruses are characteristically causing self-limiting short diseases, the question of potential chronic SARS infections is of major importance for a future disease control. If the SARS-CoV is able to cause a chronic persistent infection, chronic carriers may serve as sources for new SARS outbreaks. However, the detection of SARS-CoV in stool of patients for longer periods than 6 weeks after hospital discharge has not been reported so far. Therefore, the danger of chronic carriers may not be relevant. In contrast to common human coronavirus infections with short durations, most animal coronaviruses cause persistent infections. As an example, the feline coronavirus FIPV infects animals which then continue to shed virus for periods reaching up to seven months after infection without carrying disease symptoms . Also, TGEV and MHV tend to cause chronic infections as these viruses may be found in the airways and small intestine (TGEV) or the nervous system (MHV) several months after infection [47, 48] . Although the SARS-CoV has jumped to humans it may still have this property of inducing chronic infections. Thus, SARS-CoV RNA was found in patients' stool specimen more than 30 days after the infections.The mean incubation period of SARS was estimated to be 6.4 days (95% confidence interval, 5.2 to 7.7). The mean reported time from the onset of clinical symptoms to the hospital admission varied between three and five days .Main clinical features of the disease are in the initial period common symptoms such as persistent fever, myalgia, chills, dry cough, dizziness, and headache. Further, although less common symptoms are sore throat, sputum production, coryza, vomiting or nausea, and diarrhea [50, 51] . Special attention has been paid to the symptom of diarrhea: Watery diarrhea has also been reported in a subgroup of patients one week after the initial symptoms .The clinical course of the disease seems to follow a bi-or triphasic pattern. In the first phase viral replication and an increasing viral load, fever, myalgia, and other systemic symptoms can be found. These symptoms generally improve after a few days. In the second phase representing an immunopathologic imbalance, major clinical findings are oxygen desaturation, a recurrence of fever, and clinical and radiological progression of acute pneumonia. This second phase is concomitant with a fall in the viral load. The majority of patients is known to respond in the second phase to treatment. However, about 20% of patients may progress to the third and critical phase. This phase is characterized by the development of an acute respiratory distress syndrome (ARDS) commonly necessitating mechanical ventilation.Rapid progress has been made in understanding the clinical presentation of SARS in adults and children .In comparison to adults, SARS seems to be less aggressive in younger children, with no children in one case series requiring supplementary oxygen while in adults, systemic infection as well as respiratory infection may be the rule. SARS is much milder with non-specific cold-like symptoms in children younger than 12 years than it is in adolescents and adults . The reason for the milder clinical presentation of SARS in children is most likely due to differences in developmental stage of the immune system.The course of the disease in teenagers more likely resembles adults in concerning clinical presentation and disease progression . SARS may also develop severe illness requiring intensive care and assisted ventilation in these adolescent patients. The common presenting features are fever, malaise, coryza, cough, chills or rigor, headache, myalgia, leucopaenia, thrombocytopaenia, lymphopaenia, elevated lactate dehydrogenase levels and mildly prolonged activated partial thromboplastin times . The radiographic findings are non-specific: However, high-resolution computed chest tomography in clinically suspected cases may prove to be an early diagnostic aid when initial chest radiographs appeared normal. While rapid diagnosis with the first-generation RT-PCR assay was not satisfactory, improved RT-PCR assays may help to diagnose SARS in early stages. In this respect, a sensitivity approaching 80% in the first 3 days of illness when performed on nasopharyngeal aspirates may be achieved. The best treatment strategy for SARS among children still has to be determined while no case fatality has been reported in children. In comparison to the prognosis in adults, there is a relatively good short-to medium-term outcome. However, it is crucial to emphasize that continued monitoring for long-term complications due to the disease or its treatment is of major importance .Coronaviruses are known to exert their effects by cytocidal and immune-mediated mechanisms. In vitro studies using cell culture assays have shown that coronavirus infection commonly results in cytopathic effects such as cellular lysis or apoptosis . Also, the virus can cause cellular fusion leading to the formation of syncytia. These cytopathic effects are caused by steps of the viral replica-tion such as the mobilisation of vesicles to form the viral replication complex , leading to the disruption of Golgi complexes . Parallel to results on other coronaviruses, SARS-CoV has been shown to cause cytopathic effects in Vero cells and the formation of syncytia in lung tissues. A further similarity with other coronaviruses seems to be the potential of the SARS-CoV to cause tissue fibrosis . As molecular mechanism for this fibrosis which has been reported for infections with the coronavirus MHV, the N protein has been demonstrated to induce promoter activity of the prothrombinase gene that correlates with fibrin deposition .Next to cytocidal effects, also immune-mediated mechanisms of both the innate and adaptive immune system seem to contribute to the pathogenesis of SARS-CoV infections. In this respect, it has been shown that in MHV infection, T cells and cytokines play an important role in development of the disease . Also, humoral antibodies have been reported to be crucial in infections caused by coronaviruses such as FIPV. Herein, antibodies against the spike protein were shown to be related to the induction of peritonitis .For SARS-CoV infections, it has been reported that there seems to be an inflammatory cell influx consisting in particular of macrophages in the airways, and a massive release of cytokines during the peak of the infection [67, 68] . It is therefore crucial that these immune mechanisms are further analysed on the molecular level as it seems appropriate that not only antiviral but also antiinflammatory strategies are evaluated for a use in the clinical management of future SARS cases.The pharmacotherapy for SARS with anti-inflammatory steroids is controversial and largely anecdotal . It was reported that the initial use of pulse methylprednisolone therapy appears to be more efficacious and equally safe when compared with regimens with lower dosage and should therefore be considered as the preferred steroid regimen in the treatment of SARS, pending data from future randomized controlled trials . A further preliminary, uncontrolled study of patients with SARS, reported that the use of interferon alfacon-1 plus steroids was associated with reduced disease-associated impaired oxygen saturation and more rapid resolution of radiographic lung abnormalities .The most obvious gene which is likely to be a key modifier of SARS pathomechanisms is the spike (S) protein gene. As known for other coronaviruses, it does not only affect viral pathogenesis by determining the target cell specificity but also by other mechanisms. In this respect, a single mutation in the S gene of MHV has significant effects on the viral virulence and tissue tropism . Also, mutations in the S gene led to the emergence of the weakly virulent PRCV from the virulent enteric TGEV . Further potentially important genes are the 'non-essential' ORFs which show a significant divergence between SARS-CoV and other coronaviruses. In this respect, it was reported that the civet cat coronavirus has a 29-nucleotide deletion leading to a fusion of two non-essential ORFs into one new ORF in the SARS-CoV . It was shown that deletion mutants of 'non-essential' ORFs of the group 2 mouse hepatitis virus (MHV) leads to a lower virulence without an impact on viral replication . It has to be established if this also applies to 'non-essential' ORFs of SARS-CoV. Also, other viral gene products such as the M or E proteins may have an impact on the pathogenesis of the disease as they may induce interferon production or apoptosis [75, 76] .The primary target cells of SARS-CoV infection are respiratory epithelial cells. As the virus can also be detected in stool specimen and patients with SARS often also have gastrointestinal symptoms, epithelial cells of the gastrointestinal tract also seem to be major target cells. Next to these epithelial cells, the SARS-CoV has also been found in macrophages and many other cells as it has been detected in not only in the respiratory tract and stool specimen but also in the blood, liver, kidney and urine . In this respect, pathological examination did not only show changes in the respiratory tract, but also in splenic lymphoid tissues and lymph nodes. Furthermore, signs of a systemic vasculitis were found which included edema, localized fibrinoid necrosis, and infiltration of monocytes, lymphocytes, and plasma cells into vessel walls in the heart, lung, liver, kidney, adrenal gland, and the stroma of striated muscles. There was also thrombosis present in veins. Systemic toxic changes included necrosis and degeneration of parenchymal cells of the lung, heart, liver, kidney, and adrenal gland . It may therefore be concluded that SARS can induce a systemic disease and thereby injuring many other organs apart from the respiratory tract.The SARS-CoV target cell specificity is determined by the spike protein affinity to cellular receptors. In contrast to the all group III coronaviruses and the SARS-CoV for which the receptors have not been finally analyzed, it is known that group I coronaviruses bind to aminopeptidase N (CD13) as receptors , while group II coronavirus such as MHV use carcinoembryonic antigen (CEA) as receptor .Recently, it was shown that a metallopeptidase, angiotensin-converting enzyme 2 (ACE2), efficiently binds the S1 domain of the SARS-CoV S protein. SARS-CoV replicated efficiently on ACE2-transfected but not mocktransfected 293T cells. Also, anti-ACE2 but not anti-ACE1 antibodies blocked viral replication on Vero E6 cells, indicating that ACE2 is a functional receptor for SARS-CoV which was also identified by a further study .Recently, the C-type lectin CD209L (also called L-SIGN) was discovered to be a further human cellular glycoprotein that can serve as an alternative receptor for SARS-CoV . The interruption of virus-receptor interactions could be a potential target for future therapeutic strategies (Figure 4) . In this respect, the receptor-binding S1 domain of the SARS-CoV S protein represents a possible target for new SARS antiviral drugs. Also, antibodies against ACE2, but not inhibitors binding to the active site of ACE2 may be useful for the development of therapeutic strategies.After binding to the receptor, the next molecular step of potential use for the development of anti-SARS drugs is the virus entry into the cells. While most coronaviruses enter their target cells via plasma membrane fusion, a further entry mechanism may be acidic pH-dependent endocytosis . Focusing on these mechanisms, it will be crucial to gain further knowledge about SARS-CoV fusion activity. As a drug development candidate, a putative fusion peptide has good potential ( Figure 4 ).After the binding to a host cell receptor and entry into the cells, the molecular steps of transcription, translation and protein processing display further potential targets for new therapeutic strategies. In this respect, the RNAdependent RNA polymerases (SARS-CoV RdRp) may be a potential target for a future anti-SARS therapy. A recent study located its conserved motifs and built a threedimensional model of the catalytic domain . The authors suggested that potential anti-SARS-CoV RdRp nucleotide-analog inhibitors should feature a hydrogenbonding capability for the 2' and 3' groups of the sugar ring and C3' endo sugar puckering. Also, the absence of a hydrophobic binding pocket for non-nucleoside analog inhibitors similar to those observed in hepatitis C virus RdRp and human immunodeficiency virus type 1 reverse transcriptase seems to be crucial .Also, protease activity is crucial for SARS-CoV RNA replication and protein processing [29, 85] , and the inhibition of protease function leads to an immediate stop of viral RNA synthesis. Most of the coronaviruses express one major cysteine proteinase, called the main proteinase (M pro ) or the 3C-like proteinase (3CL pro ), and two Potential target sites for therapeutic strategies auxiliary, papain-like proteinases (PL1 pro and PL2 pro ). The latter two are responsible for the cleavage of the viral polyproteins, pp1a and pp1ab, at three sites near the aminoterminus, while the M pro processes these proteins at as many as 11 additional sites. Interestingly, SARS-CoV lacks the PL1 pro [16, 17] , but it can be assumed that its action is taken over by the PL2 pro . This is conceivable since operation of the PL2 pro on PL1 pro cleavage sites has been shown in IBV and HCoV . Roughly at the position of the PL1 pro gene in other coronavirus genomes, SARS-CoV displays a domain within ORF1a that lacks any detectable sequence homology and has therefore been named the SARS-unique domain (SUD) . It is not known whether the SUD protein is ever expressed in the life cycle of SARS-CoV but if it is, it may be connected to the high pathogenicity of SARS-CoV compared to other human coronaviruses and, therefore, it may constitute an attractive target for therapeutic intervention.Crystal structures have been determined for the M pro s of TGEV , HCoV 229E , and, more recently, SARS-CoV . They all show a similar overall architecture for the 34 kD enzyme which forms a dimer in the crystals and also at intermediate and high concentrations in solution.The monomer consists of three domains of which the first two are β-barrels with an overall similarity to the 3C proteinases of picornaviruses and to the serine proteinase, chymotrypsin. The third domain is α-helical and was shown to be essential for dimerization [85, 87, 88] . The active site of the enzyme is located in a cleft between domains I and II and comprises a catalytic dyad of Cys...His, rather than the catalytic triad common for cysteine and serine proteinases. Anand et al. have synthesized a substrate-analogous hexapeptidyl chloromethylketone inhibitor and bound it to TGEV M pro in the crystalline state. The X-ray structure of the complex revealed binding of the P1 glutamine, P2 leucine, and P4 threonine side chains of this compound to the respective subsites in the substrate-binding cleft, in agreement with the pronounced specificity for cleavage by the M pro after the substrate sequence (Thr, Val, Ser)-Xaa-Leu-Gln. The structure also showed the expected covalent attachment of the methyl ketone group at P1 of the inhibitor to the catalytic cysteine of the enzyme.In spite of 40% and 44% sequence identity, respectively, to the M pro s of HCoV 229E and TGEV, the crystal structure of the SARS-CoV M pro revealed some surprises .Within the dimer, one molecule was in the active conformation seen in the other structures, whereas the other one adopted a catalytically incompetent conformation. This enzyme had been crystallized at a pH value of <6, which in one of the monomers apparently led to the protonation of a histidine residue at the bottom of the S1 specificity pocket. This resulted in major conformational rearrange-ments leading to the collapse of this binding site for the P1 glutamine residue of the substrate and to a catalytically incompetent conformation of the oxyanion-binding loop. However, when the crystals were equilibrated at higher pH values, their X-ray structures revealed the active conformation for both monomers in the dimer. This pHdependent activation mechanism allows interesting conclusions to be made for the self-activation of the M pro from the viral polyprotein, which probably involves a pHdependent step.The same hexapeptidyl chloromethylketone inhibitor used by Anand et al. in their crystallographic study of the TGEV M pro was employed by Yang et al. to characterize the interaction of the SARS-CoV enzyme with substrate. This was performed by soaking the inhibitor into crystals grown at the low pH. In spite of the inactive conformation of one of the two monomers in the dimer being preserved, the compound was found to bind to it, but with its P1 glutamine side chain pointing towards bulk solvent rather than into the S1 binding site, because of the collapse of the latter. The binding mode of the inhibitor to the active monomer was also somewhat unusual and is not fully understood at present.On the basis of their crystallographic work, Anand et al. found that the binding mode of their hexapeptidyl chloromethylketone inhibitor to the TGEV M pro resembled that of AG7088 in complex with its target, the 3C proteinase of human rhinovirus , even though the respective target enzymes displayed large structural differences except in the immediate neighbourhood of the active site. AG7088 is in phase II/III clinical studies as an inhalation treatment for the common cold as caused by human rhinovirus. Anand et al. therefore proposed that AG7088 should be a good starting point for the design of anti-SARS drugs, and indeed, the manufacturer of AG7088 confirmed only a few days after their proposal had appeared on-line that the compound was effective against SARS coronavirus in cell culture. AG7088 is now the subject of intensive optimization efforts .Other studies used molecular dynamics simulations of the M pro and screened 29 approved and experimental drugs against a model of the SARS CoV proteinase as well as the experimental structure of the transmissible gastroenteritis virus (TGEV) proteinase . It was suggested that existing HIV-1 protease inhibitors, L-700,417 for instance, may have high binding affinities and may therefore provide another good starting point for the future design of SARS-CoV proteinase inhibitors . However, this has to be proved experimentally.Further potential targets are the E and M proteins ( Figure 4 ) as they represent the minimum essential components for the assembly of coronaviruses which form the viruslike particles [41, 42] . Ultrastructurally, the process of SARS-CoV assembly is most likely localised to the ER-Golgi intermediate compartment . Together with strategies that may focus on the inhibition of virus assembly, the virus exit through secretory pathways is also of interest for the development of new antiviral compounds.With regard to the multitude of potential epithelial target cells, specific endogenous drug delivery systems may also be of relevance. In this respect, the family of peptide transporters consisting of PEPT1 and PEPT2 which are differentially expressed in potentially infected cells of the respiratory tract [94, 95] , small intestine , kidneys [97, 98] , nervous system and other organs , may serve a target for the rational drug design of antiviral drugs. So far, a variety of antiviral drugs or prodrugs such as valacyclovir , valganciclovir or the valyl ester of zidovudine have been shown to be transported via these systems and minimal structure requirements for substrate transport have been determined . A further tool which may be used to approach antiviral therapies is the technique of small interfering RNAs (siRNAs). SiRNAs are double-stranded RNAs which lead to a sequence-specific degradation of mRNAs . Recent in vitro studies used six 21-mer siRNAs that were targeted to different sites of the replicase 1A region of SARS-CoV . Monkey kidney cells (FRhk-4) were infected with the SARS-CoV GZ50 strain and transfected eight hours later with the siRNAs. Three of the six siRNAs led to a marked inhibition of virus cytopathic effects and a reduction of virus copies between 85 and 92 %, indicating that siRNAs may have a potency as antiviral treatment options and that the 1A region displays a promising region to suppress virus replication .As most patients develop an immunity against the SARS-CoV and survive the infection, the possibility of creating an effective and safe vaccine seems to exist . There are several options to develop vaccines against the SARS-CoV .Live-attenuated coronavirus vaccines can be generated by deletions in \"group-specific genes\". The deletions of these genes do not change replication properties but attenuate the virus . Examples for the use of live-attenuated vaccines to prevent coronavirus infections are live attenuated IBV vaccines which are used in broiler chickens . For the animal coronavirus infections, live attenuated vaccines have been proven to be significantly more effective than whole killed vaccines, indicating that cellmediated immunity is a crucial defence mechanism. However, the great threat remains that a vaccine strain can recombine with a circulating wild type strain and without evidence that recombination and reversion of a live-attenuated SARS-CoV to virulence can not occur, it is unlikely that a live attenuated SARS-CoV vaccines will be developed and used.Whole killed vaccines are generally safe and easy to generate. In fact, this technique has been applied in veterinary medicine to generate vaccines for BoCV and IBV . Also, an inactivated canine coronavirus vaccine has been produced . A SARS inactivated vaccine was recently developed using the SARS coronavirus (SARS-CoV) strain F69 treated with formaldehyde and mixed with Al(OH)(3) . However, killed vaccines may not protect against different strains of coronaviruses, and live attenuated vaccines have been shown to be more effective than whole killed vaccines in preventing coronavirus animal infections .Using molecular biology techniques to generate large quantities of recombinant viral proteins, recombinant subunit vaccines, e.g. against the spike protein, are expected to be created relatively easy as shown by two recent studies [116, 117] . Eight recombinant human single-chain variable region fragments (scFvs) against the S1 domain of spike (S) protein of the SARS-CoV from two nonimmune human antibody libraries were screened and one scFv 80R efficiently neutralized SARS-CoV and inhibited syncytia formation between cells expressing the S protein and those expressing the SARS-CoV receptor angiotensin-converting enzyme 2 (ACE2) . A recent study used the SARS-CoV spike protein receptor binding domain (aa 318-510) for immunization, which resulted in the induction of effective neutralizing antibodies . However, recombinant subunit vaccines may have a limited ability to protect against SARS-CoV infections in view of the variations which may arise in the viral genome in future outbreaks. Therefore, the approach of recombinant subunit vaccines may have to be supplemented by further vaccine strategies which focus on cell-mediated immunity.An approach using recombinant vectored vaccines with DNA or a viral vector could be a promising target. The DNA prime and adenovirus or MVA boost approach which is currently analysed for a potential use in the development of HIV vaccines, may also offer a strategy to prevent SARS infections. In this respect, a multi-valent approach which induces both humoral and T cell-mediated host responses seems to be the most attractive strategy.From the field of veterinary medicine, data on this approach are already available: A recombinant fowlpox with the S1 gene of IBV was demonstrated to be relatively protective against IBV . Also, a DNA vaccine was developed which contains the nucleocapsid protein gene of porcine transmissible gastroenteritis virus (PTGV). This vaccine was shown to initiate both humoral and cellmediated immune host responses . Recently, three murine studies demonstrated that DNA vaccines encoding different SARS-CoV antigens are capable of generating humoral and cellular immunity and may potentially be useful for control of infection with SARS-CoV . However, it was also shown that immunization with modified vaccinia virus Ankara-based recombinant vaccine against SARS is associated with enhanced hepatitis in ferrets .A further strategy is based on the use of epitopes which can be delivered using viral or DNA vectors. Such an epitope-based strategy for coronavirus vaccination has already been reported and the major advantages is the prevention of a possible vaccine reversion to virulence. A further benefit of this technique is the possibility to eliminate any regions of the viral genomic sequence which be associated with a potential autoimmune effects. The limitation of this approach is mainly based on potential variations. In this respect, epitopes which frequently undergo mutations will not protect against the SARS-CoV infections if used in epitope-based vaccines. If the SARS-CoV evolves as a highly variable virus, it will be crucial to identify highly conserved epitopes of the virus.In summary, the important development of SARS vaccines can be approached using several techniques which should ideally encompass the induction of both humoral and cell-mediated mechanisms. As coronavirus vaccines in animals have partly been reported to cause an enhancement of viral infections , a cautious approach has to be followed. A first study has investigated the ability of adenoviral delivery of a codon-optimised SARS-CoV spike protein S1 fragment, membrane protein, and nucleocapsid protein to induce immunity in rhesus macaques. The immunization with a combination of these three Ad5-SARS-CoV vectors and a booster vaccination on day 28 demonstrated antibody responses against the spike protein S1 fragment. Also T-cell responses against the nucleocapsid protein were found and all vaccinated animals displayed strong neutralising antibody responses in vitro. These results indicated that an adenoviral-based vaccine can induce SARS-CoV-specific immune responses in monkeys.In summary, the onset of the SARS epidemic in different continents has led to the formation of a successful laboratory network to identify the molecular mechanisms underlying the SARS infection. Next to the development of early diagnostic tests and effective treatment strategies, it is most important to orchestrate research activities which lead to the development of vaccines and antiviral agents, as there is no established therapy to date. Even now in a situation of only a handful of new cases, SARS remains a major global health hazard which may reappear.\n",
            "\t0.976\tSevere acute respiratory syndrome (SARS), caused by the coronavirus SARS-CoV, is an acute infectious disease with significant mortality. A typical clinical feature associated with SARS is pulmonary fibrosis and associated lung failure. In the aftermath of the SARS epidemic, although significant progress towards understanding the underlying molecular mechanism of the infection has been made, a large gap still remains in our knowledge regarding how SARS-CoV interacts with the host cell at the onset of infection. The rapidly changing viral genome adds another variable to this equation. We have focused on a novel concept of microRNA (miRNA)-mediated host-virus interactions in bronchoalveolar stem cells (BASCs) at the onset of infection by correlating the ''BASC-microRNome'' with their targets within BASCs and viral genome. This work encompasses miRNA array data analysis, target prediction, and miRNA-mRNA enrichment analysis and develops a complex interaction map among disease-related factors, miRNAs, and BASCs in SARS pathway, which will provide some clues for diagnostic markers to view an overall interplay leading to disease progression. Our observation reveals the BASCs (Sca-1+ CD34+ CD45-Pecam-), a subset of Oct-4+ ACE2+ epithelial colony cells at the broncho-alveolar duct junction, to be the prime target cells of SARS-CoV infection. Upregulated BASC miRNAs-17*, -574-5p, and -214 are co-opted by SARS-CoV to suppress its own replication and evade immune elimination until successful transmission takes place. Viral Nucleocapsid and Spike protein targets seem to co-opt downregulated miR-223 and miR-98 respectively within BASCs to control the various stages of BASC differentiation, activation of inflammatory chemokines, and downregulation of ACE2. All these effectively accounts for a successful viral transmission and replication within BASCs causing continued deterioration of lung tissues and apparent loss of capacity for lung repair. Overall, this investigation reveals another mode of exploitation of cellular miRNA machinery by virus to their own advantage.Severe Acute Respiratory Syndrome (SARS) is a new fulminant atypical pneumonia which emerged as a regional and global threat in 2002-2003 with a high mortality rate resulting from acute lung failure . The disease causing agent has been identified as a novel coronavirus termed as SARS-associated coronavirus (SARS-CoV) [2, 3] . The SARS-CoV is an enveloped virus containing a single stranded, positive-sense RNA genome which encodes 14 putative open reading frames encoding 28 potential proteins [4, 5] . These include four structural proteins, spike (S) glycoprotein, matrix (M) protein, small envelope (E) protein, and nucleocapsid (N) protein . These proteins have various roles in aiding the virus to enter the host and spread infection. While the incidence of new cases of SARS waned in 2003-2004, many aspects of SARS disease pathogenesis and host-pathogen interactions remain unsolved. Limited pathologic studies reveal that the major site of SARS-CoV infection and morbidity is the respiratory tract. The target organ of SARS is mainly lungs . Among the various animal models that have been used to study the pathogenesis of SARS-CoV infection, the monkey model mimics the clinical course of SARS to a certain degree . The cellular tropism of SARS-CoV in mouse lung has also been investigated by Ling et al., 2006 . But still today, very little is known regarding the mode of SARS-CoV interaction with host cells at the onset of infection in the lungs at a molecular level, and also the cell types in which the primary viral infection and replication occurs.The lung is an extremely complex, conditionally renewing organ which contains anatomically and functionally distinct epithelial stem cell populations which reside in distinct anatomical locations. The basal cells , Clara cells and type-II pneumocytes are the candidate stem/progenitor cells which can repair the injured lungs and contribute to local needs in times of tissue damage. Recently, Kim et al. have isolated a regional pulmonary stem cell population termed as bronchoalveolar stem cells (BASCs) residing at the bronchoalveolar duct junction of adult lungs . These have been identified and characterized as CD34 + Sca-1 + CD45 -PE-CAM 2 cells expressing both cytoplasmic Clara cell secretion protein (CCSP) and prosurfactant protein-C proteins, which are markers for Clara cells and type-II pneumocytes respectively . We have searched for the evidence to establish BASC as the prime target cells of SARS infection initiation and replication.To date, miRNA-mediated RNA interference is reported to be an essential tool to understand the regulatory pathways at molecular level that underlie infection biology [13, 14] . They not only participate in executive decisions but also perform much of the grunt work to micromanage protein output. Inspite of being such potential regulatory elements, miRNAs have so far missed attention in the pathogenesis of SARS-CoV infection.MiRNAs are small (19-25 nucleotides) endogenous noncoding RNAs that have been shown to influence the abundance and translational efficiency of cognate mRNAs [15, 16] . Viruses, which typically employ many components of the host gene expression machinery, also encode miRNAs. On the contrary, miRNAbiogenesis pathway poses some serious problem for RNA viruses and a group of DNA viruses (poxviruses) to encode miRNAs . SARS-CoV being an RNA virus is assumed not to encode its own miRNAs. But in due course of evolution, the virus might have developed certain highly sophisticated molecular mechanisms to exploit the cellular biosynthetic machinery of host cells and elude the cellular defense mechanisms. It is most likely that in certain situations SARS-CoV uses the cellular miRNA transcripts in order to foster their own agenda.Given the proposed involvement of BASCs in SARS disease , elucidation of the miRNA-mediated regulatory mechanisms responsible for making BASCs the prime targets of SARS infection might provide new avenues to explore the underlying mechanisms of SARS infection.The entry of SARS-CoV into cells is mediated through interaction between spike (S) glycoprotein of the virus and angiotensin-converting enzyme 2 (ACE2), the primary receptor of SARS-CoV on the host cell . ACE2 is a homolog of angiotensin-converting enzyme (ACE), which plays an important role in the renin-angiotensin system for blood pressure homeostasis. Recent studies of Li et al. have shown a protective effect of ACE2 against experimental lung fibrosis through its ability to degrade local tissue angiotensin II (ANG II) in response to bleomycin. Furthermore, in the pathogenesis of lung fibrosis, they have proposed the involvement of the down regulation of ACE2 as an integral component of the sequence of events leading to lung collagen deposition. The molecular mechanisms responsible for the loss of ACE2 in lung fibrosis are currently under investigation. A typical clinical feature associated with SARS is also pulmonary fibrosis and associated lung failure whose underlying mechanism still remains elusive . Investigations leading to the underlying molecular mechanisms responsible for the loss of ACE2 in pulmonary fibrosis might provide a clue to the reason behind fibrosis associated lung injury in SARS.Several reports suggest that there is a close interaction between SARS-CoV and respiratory epithelia which play an important role in the genesis of SARS . Jia et al. have investigated interactions between SARS-CoV and human airway epithelia and indicated that the state of cell differentiation and ACE2 expression levels are important determinants of the susceptibility of human airway epithelia to infection. To date, the molecular switches regulating the state of cellular differentiation of an infected cell and ACE2 expression have not been studied properly. In this work, we have proposed the role of miRNA in modulating the expression of ACE2 along with the stage specific cellular differentiation at the site of infection initiation. Further we have tried to focus on the role of SARS-CoV which dictates these events for its successful transmission and replication.The POU-homeodomain transcription factor Oct-4 (Pou5f1) plays a central role in self-renewal, pluripotency, and lineage commitment. Alterations in Oct-4 expression promote differentiation and leads to the specification of ectodermal, endodermal or mesodermal primitive progenitors . A small number of Oct-4expressing cells have been observed at the bronchoalveolar junction of the neonatal lung . We have investigated miRNAmediated modulatory role on Oct-4+ BASCs and have correlated their contribution to SARS infection.Our analysis establishes a complex interplay between miR-NAs, BASCs and certain essential factors related to the disease which might provide a better understanding of SARS pathogenesis. The multifarious relationships shared by genes related to the disease-system pathway, modulatory effects of miRNAs within the prime target cells of infection in response to indefinite cues impose the need of an interaction map. This study develops a complex interaction map between disease-related factors, miRNAs and BASCs in SARS disease pathway, which will provide some clues for diagnostic markers to view an overall interplay leading to disease progression. Further, our investigation towards investigating the therapeutic potential of miRNAmediated RNAi as an effective antiviral agent against SARS might unravel some of the efficient therapeutic measures against the deadly disease in future.We have investigated the cellular tropism of SARS-CoV (Table 1) in the BASCs present in lungs and attempted to establish the role of BASC-miRNAs towards understanding the pathogenesis of SARS in lung fibrosis. We describe the role of BASCs as the target cells of SARS infection and show how a repertoire of BASC miRNAs act as molecular switches to contribute to this host-pathogen interaction.Which is the primary site of SARS infection and how do these corona virus interacts with those cells to enter the host and eventually invades the host immune system to spread itself -is a pertinent question which prevails and is a debated issue. In order to throw light on this very important question, we looked upon at the SARS + cell markers and BASC markers (specific to each of its developmental stages from multipotent state to fully differentiated state).It has been reported that SARS-infected cells are unlikely to be pneumocytes (both type I or type II) or cytokeratin + epithelial cells and it is quite obvious since SARS-CoV cannot infect or replicate in a fully differentiated cell . Further, SARS + cells are distinct from cells expressing the macrophage/monocyte specific marker CD68. But SARS + cells are found to express the functional receptor ACE2 as well as the stem/progenitor cell marker CD34 and Oct-4 . Chen et al have also shown that SARS + cells are CD45 2 . Hence it has been confirmed that the SARS + cells in the infected lung were a subset of putative stem/ progenitor cells expressing CD34, Oct-4 and ACE2. Further, it is unlikely for SARS + cells to produce differentiated cell markers. The presence of SARS + cells within the multiple cell types at the bronchiolar lining layers makes it difficult to isolate SARS + cells. Some of the SARS + cells remains overlapped with adjacent cytokeratin + cells which might be mistakenly interpreted as colocalized cells. In such a situation BASC (Sca-1 + CD34 + CD45 2 Pecam 2 ) in terminal bronchioles located exclusively at the broncho-alveolar duct junction (BADJ) stands a good selection to be investigated as the prime target cells of SARS-CoV infection.A percentage of the Sca-1 + CD4 2 Pecam 2 cell populations at BADJ are CD34 + . This Sca-1 + CD34 + CD45 2 Pecam 2 population is enriched for BASCs became evident from the fact that it contained no ciliated cells. In addition to clonal colony formation, these cells exhibited extensive self-renewal in culture. They also had a greater capacity for differentiation compared to other lung epithelial cells. Sca-1 + CD34 + CD45 2 Pecam 2 BASC cultures further confirmed the multilineage differentiation capacity of BASCs. Within seven to ten days they show up positive for differentiated cell markers as they get differentiated into Clara-like cells (CCA + SP-C 2 ), AT2-like cells (SP-C + CCA) and AT1-like cells (AQ5 + ).Within the Sca-1 + CD4 2 Pecam 2 cells the other percentage of cells showed CD34 2 and contained ciliated cells. The Oct-4 + colony cells which are Sca-1 + CD34 2 SSEA-1 + cytokeratin + were shown to be succeptible to SARS-CoV infection by Ling et al. . These might be a subset of Sca-1 + CD4 2 Pecam 2 CD34 2 population ( Figure 1 ). Later studies have shown that cytokeratin + cells do not express the functional receptor ACE2 . Further, SARS + cells do not express cytokeratin and are CD34 + SSEA-1 2 . Phagocytosis is a common mode of virus entry within surrounding cells in SARS infection . Hence, presence of SARS antigen within the subset of Oct-4 + colony cells which are Sca-1 + CD34 2 SSEA-1 + cytokeratin + is most likely due to phagocytosis. Thereby, it is evident that these Oct-4 cells (Sca-1 + CD34 2 SSEA-1 + cytokeratin + ) are not the prime target cells for SARS-CoV infection.On the contrary, the BASCs (Sca-1 + CD34 + CD45 2 Pecam 2 ) must be another subset of these Oct-4 + ACE2 + epithelial colony cells (Figure 1 ) which provides a more favorable environment for SARS-CoV entry and replication. Absence of ciliated cells and CD34 + makes them the most probable targets of SARS-CoV at the onset of infection.Since SARS-CoV cannot infect or replicate in a fully differentiated cell, it is indeed essential for the virus to control the differentiation stages in an infected BASC cells so that they do not reach a fully differentiated state until the virus has undergone successful transmission and replication within this primary site of infection. Hence, it is important to monitor the expression levels of the different developmental stage specific markers of SARS infected BASC and also that of the corresponding regulatory switches that control their level of expression.MiRNAs are widespread agents of post-transcriptional gene silencing and have been strongly linked with stem cells [25, 26] . They exhibit a high degree of stage-and tissue-specificity, and therefore it is important to scan the profiling data for those miRNAs that operate during the narrow windows of development of a BASC in order to understand their regulatory impact on the expression levels of essential marker genes and transcription factors within the cell. In order to model the regulatory mechanisms within a SARS + cell, we have tried to understand the underlying regulatory mechanisms within a normal BASC cells mediated by the molecular switches-the miRNAs.It has been established that the virus exploits the host-miRNA milieu to foster their own agenda of evading the host immune system, thereby establish a consistent infection and continue replication.We undertook an intensive transcriptome-wide search for candidate mh-miRNAs (BASC-miRNAs which are homologous between mouse-human whole genomes) ( Table 2 ) targeting the developmental stage specific transcriptional factors and marker genes of BASCs. Table 3 shows the list of the miRNAs targeting the potential stage specific factors of BASCs. Further, we have also elucidated the mh-miRNAs targeting ACE2. The target factors have further been mapped with the expression levels of their corresponding miRNAs within BASCs which shows their integrated correlation contributing towards each developmental stage of BASCs in a healthy individual ( Figure 2 ). SARS-CoV definitely interferes with these miRNA mediated regulation to dictate the cell fate and continue its replication. Hence as a future advancement of this work, it will be worthy designing benchwork to detect the correlation of these miRNA-mRNA expressions within a SARS-CoV infected cell.We have searched for the host miRNA targets within the virulent genes of SARS-CoV. Table 4 includes the BASC-miRNA candidates targeting the most important virulent proteins responsible for viral infection. Interestingly, miRNAs-17*, -574- Normally presence of such host miRNA targets in SARS-CoV genome indicates that the corresponding miRNAs are a part of the host's innate antiviral defense. On the contrary, the RNA virus coopt host-miRNAs to suppress their own replication to evade immune elimination and establish a strong infection. It has been found that human infecting single-stranded RNA viruses are enriched for target sequences of human miRNAs . Or else rapidly evolving RNA viruses might evolve targets that match host miRNAs in order to increase their host specificity. It is very likely that the rapidly evolving virus take advantage of slowly evolving host miRNAs to increase their survival in the host, analogous to the manner in which viruses take advantage of other cellular processes. Co-option of host miRNAs by SARS-CoV to suppress their own replication will allow it to be in a latent state and escape the host immune system at the initial time point of transmission and infection. Once successful transmission within the host system takes place, antiviral miRNA target sequences in the virus can be expected to mutate rapidly to maximize mismatches and thus minimize the impact of 'antiviral' miRNAs. Rapid mutation in SARS-CoV is supported by the fact that SARS-CoV lack proof reading activity in their polymerases. RNA-dependent polymerase in SARS-CoV misincorporates 8.26 bases per million .While investigating the seed sequence conservation of the viralspecific host miRNAs ( Figure 3 ) we found that there is a strong bias towards G, Y(pYrimidine) and R (PuRine) bases at the 3 rd , 4 th and 5 th position respectively of the seed region. This implies that a single mutation within the complementary regions of this conserved 3-mer motif within the viral targets will disrupt the miRNA seed-target match and will enable the virus to escape the antiviral effect of host-miRNAs after successful transmission within the host. Such host-miRNA sequence conservation helps the virus to mutate the target regions with more precision and at ease so as to escape the effect of all the 4 miRNAs together.The miRNA expression profiles of three miRNAs-17*, -574-5p and -214 targeting S, N, Orf1a, M and E shows that they are upregulated within uninfected BASCs. Presumably at the onset of SARS infection the expression of these targets is suppressed which is likely to control viral replication and consequently help the virus to evade the host immune system. But the targets for downregulated miR-148a within ORF1a, M, S and E might escape repression and maintain replication at a lower rate. Further, the exclusive miRNAs viz. miR-223 and -98 for N and S protein respectively are highly downregulated (8.7 fold and 5.6 fold respectively) in BASCs. N and S proteins are important for viral transmission in host. This provides a clue towards the fact that the S protein and N protein (having targets corresponding to these miRNAs) of SARS-CoV takes advantage of this and escapes miRNA-mediated repression which is a rescue to the virus for effective transmission at the initial stage of viral infection.Key miRNA-mRNA pairs correlating with virus entry, replication, and host-virus interaction ACE2-S interaction. It has been shown that proinflammatory and fibrogenic cytokine pathways are activated within the first 24-48 hr following pulmonary insult resulting from SARS-CoV infection . High initial levels of these activities are associated with persistent pulmonary damage and increased risks of subsequent pulmonary fibrosis and poor outcome in diffuse [17, 18] . Studies have also been performed which support that differential regulation of host cell mRNA transcription and protein synthesis occurs in a SARS-CoV infected cells . SARS-CoV S protein is a multifunctional protein which plays pivotal roles in the biology and pathogenesis of SARS-CoV. It has been shown that S protein mediates viral infection by binding to cellular receptor ACE2 and thus induces membrane fusion. SARS-CoV infections and the Spike protein of the SARS-CoV reduce ACE2 expression . Further a recent work of Imai et al. showed that on injecting SARS-CoV Spike into mice worsens acute lung failure in vivo, which can be attenuated by blocking the renin-angiotensin pathway. This suggests that the S-protein mediated activation of the pulmonary RAS influences the pathogenesis of ALI/ARDS and SARS . But the molecular mechanism by which S protein down regulates ACE2 in lung cells following SARS injury is not well addressed till to date.In our work we observed that the miRNAs-miR-186,-93 and -200b targeting ACE2 are found to be highly down regulated in normal BASCs thereby showing that they do not interfere with the expression level of ACE2 and thus prevents lung fibrogenesis by limiting the local accumulation of the profibrotic peptide ANG II in an uninfected state. Considering the expression of S-protein, we find that its target corresponding to highly downregulated miR-98 probably escapes miRNA-mediated host RNAi defense at the onset of infection and interferes with ACE2 expression. Figure 4 shows how the virus takes advantage of the microRNA-mediated complex interplay between the host and viral factors within BASCs. The question still remains regarding the molecular mechanism by which this S-protein represses the expression of ACE2.S-proteins have been shown to act as transcription factor activating the promoters of GRP94/78 genes inducing ER stress in a SARS-CoV infected cell . The role of S protein as a transcription factor has obliged us to think that they might play a role to activate the miRNA promoters of miR-186,-93 and -200b targeting ACE2 to enhance their expression resulting in a reduced expression of ACE2 in an infected host. Whether this decrease in ACE2 expression is a result of miRNA mediated gene silencing controlled by S-protein will be an interesting and compelling question for further study.Oct-4-N interaction. Oct-4 plays a crucial role towards regulating the chromatin structure in a state consistent with selfrenewal and later facilitating the expression of genes that keeps the cell poised to respond to cues that lead to differentiation. Hence this is one of the essential factors which control the state of differentiation in BASCs. Furthermore, it has a significant role in apoptosis. miR-146b-5p is seen to target Oct-4 in normal BASCs. The downregulated expression of this miRNA explains for the uninterrupted expression of Oct-4 resulting its normal function in an uninfected cell. We have harnessed the high possibility of a complicated crosstalk between the ability of N-protein to control the expression of Oct-4 in an infected cell.N protein is known to be the most abundantly expressed protein of SARS-CoV. The most unique and significant property of it is revealed in its ability to act as a sequence specific DNA binding factor. It has been reported to bind NFkB response element of COX2 promoter and enhance COX2 gene expression which effectively is beneficial for its existence within the host. This reveals the power of the N-protein to manipulate the entire gene expression programme of the infected cell. The common miRNAs, miR-214,-574-5p and -17* having targets in N protein are upregulated in BASCs. But miR-223, the unique miRNA targeting N-protein is highly downregulated. This signifies that the Nprotein targets corresponding to miR-223 is going to escape miRNA-mediated downregulation at the onset of infection (See Figure 4) . Further it seems that N-protein plays a role in interfering with the expression modulation of Oct-4 by binding to the promoter of miR-146-5p and controls the different stages of BASC differentiation as well as prevents apoptosis of SARS infected cells. Stem Cell Antigen-1 is identified as a potential marker for BASCs located at the BADJ [12, 27] .OCT4 (POU5F1) miR-146b-5p Octamer-binding transcription factor 4+ (Oct-4) is a stem cell marker protein. It acts as a master switch in differentiation by regulating cells that have pluripotent potential.Recently, it has been found to be expressed in pulmonary cells which are a target for SARS infection [8, 23] .CD34 let-7g let-7d let-7cThese are membrane-bound stem cell markers for BASCs. This is also expressed in SARS-infected lung cells [23, 28] .Marker protein of alveolar type-2(AT2-like) cells which is a subset of the differentiated BASC population. Hence is quiescent in normal lung but proliferate in response to bronchiolar and alveolar epithelial injury [29, 30] .This is the Clara cell marker protein. These are also quiescent in normal lung and proliferate in response to bronchiolar and alveolar epithelial injury .Aquaporin5 miR-331-3p miR-497Marker protein of AT1-like cells which is another subset of the differentiated BASC population. Aquaporin5 plays a critical role in the maintenance of normal lung water homeostasis [30, 31] .Angiotensin-converting enzyme 2 (ACE2), the primary receptor of SARS-CoV on the host cell. It is a negative regulator of the renninangiotensin system (RAS) in the setting of acute lung injury and in response to pulmonary infection with the SARS-CoV [17, 32] .Inflammatory chemokine receptor for CCL3 and CCL5 & responsible for lung fibrosis [33, 34] .We have predicted the differentially expressed mh-miRNAs which operate within the narrow windows of development of BASCs by targeting a set of host encoded mRNAs. Class A specifies the host mRNAs corresponding to BASC marker proteins which designates its undifferentiated state as well as the state of proliferation and/initiation of differentiation. Class B specifies the host mRNAs corresponding to BASC marker proteins as it reaches a fully differentiated state. Class C specifies the host mRNAs corresponding to the receptor proteins which participates in SARS-CoV pathogenesis. These miRNA-mRNA pairs specific to BASC are important to understand hostvirus interaction network. doi:10.1371/journal.pone.0007837.t003Immunity-related issues-induction of inflammatory chemokines in SARS-CoV-infected cells might be an advantage for the virus to replicate All respiratory virus disease results from two concurrent pathological components: ongoing virus replication and the resulting inflammatory response. Even if antivirals clearly inhibit virus replication, the biochemical and cellular inflammatory responses to the initial infection related events continue despite diminished virus titer . While acute inflammatory responses are generally beneficial in nature and have been shown to limit virus replication in situ, prolonged, uncontrolled inflammation has been recognized as a significant component contributing to the pathological sequelae to the morbidity and mortality of severe acute respiratory syndrome coronavirus infection . A histopathologic analysis of SARS-CoV infected lung in mice showed that there has been an induction of significant changes in the expression levels of the chemokine receptor CCR1 which acts as receptors for CCL3 and CCL5. The expression levels of these inflammatory chemokines were increased by viral infection of the lung . It has been shown that CCR1 play an essential role in the pathogenesis of pulmonary fibrosis in lung-injury models . Further, previous work suggests that antagonism of signaling via the chemokine receptor CCR1 is a potent strategy that attenuates leukocyte recruitment in response to severe respiratory virus infection .Although there has been a subsequent increase in inflammatory cytokines related to innate immune response in response to viral infection, there has been a lack of immunoregulatory cytokines (T or B lymphocytes) in the lung during infection which confirms the fact that the expression of viral replication proteins remains at a quiescent state so as to avoid viral clearance by the activation of immunoregulatory cytokines by adaptive immune responses at the onset of the infection. On the contrary, activation of inflammatory chemokines might be acting as a beneficial phenomenon for the replication of various viruses [51, 52] .Emerging studies suggest that miRNAs provide an added layer in orchestrating immune responses . miRNAs function in shaping immunity by regulating the repertoire of genes expressed in immune cells and the magnitude and duration of responses to particular pathogens . Hence we investigated the underlying miRNA induced molecular mechanisms responsible for these changes in the lung and we found that miR-223 has its target in the 39-UTR of CCR1. Previously, miR-223 was experimentally identified in chorioamnionitis-related inflammation and its predicted target genes include several genes involved in inflammation and immune responses. Further, miR-223 is known to promote granulocyte differentiation and is a part of a regulatory loop that involves C/EBP and NFI-A .Interestingly miR-223 has its target in N protein of SARS-CoV too and it is one of the miRNAs which probably helps the virus to escape the RNAi mediated silencing at the onset of infection. Or in another way we may say that the virus encodes the target of miR-223 to escape the RNAi mediated repression and at the same time takes advantage of the effect of induction of the inflammatory chemokine responses to accelerate lung fibrosis. But the controlled expression of target CCR1 in an uninfected cell inspite of the significantly low expression of its corresponding miR-223 in BASCs suggest that there must be a cascaded fine tuned interaction masterminded by the virus which is going on within the infected cell. It is known that N protein activates NF-KB in Vero E6 cells . CCR1 is one of the target genes of NF-KB which is activated at a later stage . Hence, we propose that within BASCs, reduced expression of miR-223 helps N protein to enter the host cell. On successful entry, N protein might be using miR-223 once again to activate CCR1 via NF-KB at its replicative stage to enhance lung fibrosis and at the same time gets empowered to manipulate the entire gene expression of the host cell by controlling the expression of Oct-4. The exact mechanism of such miRNA-mediated events needs further investigation. Further, checking the expression levels of N-protein, CCR1 as well as miR-223 in an infected BASC will provide a more compact conclusion.Based on these findings, we propose that BASCs (Sca-1 + CD34 + CD45 2 Pecam -) are a subset of Oct-4 + ACE2 + cells and are the chief targets of SARS-CoV infection. SARS-CoV enters the host; infects the BASCs of the lungs and controls its developmental stages via the molecular switches-miRNAs. Co-option of host miRNAs by viruses reveal their intelligent plan to control their replication in order to evade immune elimination until they undergoes successful transmission and establish a strong infection. Thereafter, they undergo rapid mutation to maximize the target-miRNA mismatches and enhance their replication before the cell reaches at a fully differentiated state. On successful replication, the virus infects resident, infiltrating, and circulating immune cells. The circulating immune cells carry the virus to other organs and causes damages to the immune cells of spleen, peripheral and central lymph nodes and other lymphoid tissues. The immune defense being weakened significantly, leads to rapid deterioration of the pneumonia. It is worth to test these cells for the expression of Oct-4 and ACE2 for further confirmatory conclusion.The proposed host miRNA-dependent mechanism probably acts in concert with the host and viral factors for establishing a strong infection. Our proposal shifts away from the simplistic notion that host miRNAs with specific sequence complementarity to viruses are indicative of a bonafide innate antiviral immune mechanism to an established viral pathogen.Although the immediate threat of SARS is over, we intend to continue with its therapeutic measures partly to guard for its possible return. Moreover, this will help us to transform our knowledge from SARS to other emerging viral diseases specially causing lung injuries which most certainly will be a deadly threat in future. Lot of attempts have been made and is still going on to design effective antiviral agents against this deadly disease . While some of these approaches showed partial efficiency to combat SARS infection, the others need further clinical investigations to be used as a proper therapeutic reagent.This era has witnessed an increase in the development of exogenous siRNA therapeutics against viruses. The potential therapeutic effect of modulating host-miRNA levels is also worth considering. This emerging picture of miRNA regulation in SARS-CoV infected individual will make the host-virus interaction far richer and more complex than the crisp linear pathways of the previous decade, with miRNAs participating in executive decisions at the interesting and vulnerable nodes in regulatory networks. By boosting the level of the host virus-specific miRNAs, it might be possible to turn what was a desirable outcome for the virus into a desirable outcome for the patient. Under the current hypothesis, we would predict that decreasing the level of host virus-specific miRNAs would result in promotion of viral replication, leading to immunological recognition and clearance of the virus by the host immune system. Further, increasing the level of miRNAs exclusive for the N and S protein might also block viral entry. However, manipulating the level of host miRNAs could have unintended consequences because the physiological functions of the miRNAs might be altered or viral pathology might be enhanced. Further silencing of miRNAs with antagomirs specifically within the infected cell could become another therapeutic strategy where miRNAs participate in disease aetiology. Nevertheless, these potential interventions merit further evaluation.To identify the miRNAs involved in the cellular tropism of SARS-CoV in lungs, we undertook an intensive search for potential miRNA candidates in the BASCGAP (Broncho-alveolar Our search strategy was motivated by the need to apply purehomology based selection of miRNAs so that the results represent a closer approach to mimic the molecular mechanisms that underlie disease pathway behaviors and response in a human host. Further the overall miRNA expression profile of mouse and human lungs are similar . From the BASCGAP sequence data of 116 miRNAs, we screened out the BASC-miRNAs lying at the intersection of mouse-human whole genomes. This filter is based on a homology search (SSEARCH), performed with miRBase (release 12.0) and is applied on the mature candidates from the BASCGAP sequence data. E-value cutoff considered for SSEARCH was ,10.0. Mature miRNA sequence homology was one of the stringent parameters considered in our search. Table 2 represents the screened output set of 95 mature candidates termed as mh-miRNAs.We have considered the developmental stage specific marker proteins of BASCs, which includes the transcription factor Pou5f1; ACE2-the most important receptor of SARS-CoV (Table 3 ). In addition to these, we have chosen the virulent protein coding genes, viz. Nucleocapsid (N), Spike(S), Envelope (E), Membrane (M) and Orf1a from 30 strains of SARS-CoV (Table 1 and Table 4 ) for our investigation. Annotated UTR sequences of the target genes were retrieved from UTRdb .We adopted RNAhybrid , that predicts the most favorable hybridization sites between miRNA and UTR regions and generates energy minimized duplex structures using the Dynamic Programming technique. The resultant miRNA-mRNA duplexes are consistent with the following structural and energy constraints at the first step of screening the targets: (i) Binding energy cutoff , = 220 kcal/mol ; (ii) complementarity of minimum seven bases to the 59 end of the miRNA; (iii) minimization of GU base pair within the miRNA-target duplex, a maximum of only one GU base pair between 2 and 7 position of the miRNA is allowed; (iv) minimization of loop size throughout the duplex.Although, currently available target prediction methods are diverse, both in approach and performance and all have room for further improvement, perfect seed pairing centered on nucleotides 2-7 is considered as the most important criteria for prediction reliability. Furthermore, in order to screen out the most favorable target in the case of miRNAs having multiple targets we incorporated the constraint of site efficacy (hierarchy of site efficacy follows-8 mer.7 mer-m8.7 mer-A1.6 mer) following the work of Grimson et al. and Friedman et al. [65, 66] . 39 compensatory pairing of miRNA to its target have been considered to compensate for imperfect/weak seed matches . We have further checked the effect of cooperativity as well as the UTR binding location.For bantam miRNAs, good minimum free energies (MFEs) can occur frequently by chance. The longer a putative target sequence, the better such random energies will be. Hence, statistical significance of predicted targets is assessed with an extreme value statistics of length normalized minimum free energies and a Poisson approximation of multiple binding sites.Microarray data analysis in BASCs miRNA expression profile of 2610 6 mouse BASCs (CD45 2 CD31 2 CD34 + Sca-1 + ) and 4610 6 control cells(CD45 2 CD31 2 CD34 2 Sca-1 2 ) are determined using miRNA microarray. The data considered for our analysis is obtained by first subtracting the background and then normalizing the signals using a LOWESS filter (locally weighted regression). Background is determined using a regression-based background mapping method . The miRNA dataset was filtered according to the standard procedure to exclude spots with minimum intensity and size. Further for the dual-sample experiments, the ratio of the two sets of detected signals (log2 transformed, balanced) are calculated. Thereafter, t-values are calculated for each miRNA between the test and the control groups, and P-values are computed from the theoretical t-distribution. miRNAs with P-values below a critical P-value (typically 0.01) are Figure 5 . miRNA-mRNA prediction pipelines. This flow chart summarizes the steps followed and yields from BASCGAP consortium sequences to predict the miRNA-mRNA pairs involved in SARS pathogenesis. doi:10.1371/journal.pone.0007837.g005 the differentially detected signals and were selected as our global miRNA expression profile data set (of mouse BASCs) in BASCGAP. The flowchart of methodology is provided in Figure 5 .\n",
            "\t0.976\tBackground: During the SARS epidemic, healthcare utilization and medical services decreased significantly. However, the long-term impact of SARS on hospital performance needs to be further discussed.Severe Acute Respiratory Syndrome (SARS) is a viral respiratory disease caused by a coronavirus (SARS-CoV). SARS has caused a significant impact on psychosocial and legislative regulation . SARS brought about not only relatively discernable economic losses , but also observable damage to healthcare organizations, and this has resulted in a lower healthcare utilization rate [6, 7] .During the SARS epidemic, there were many reports that looked into healthcare utilization and decreases in medical service volume . However, most of them explored only one department of the hospital or over a very short period of time. No reports have studied the influence on whole hospital performance and followed the long-term impact on the recovery. A municipal hospital in Taipei was shut down for a month due to SARS , and afterwards became the designated SARS and infectious disease hospital for the city in addition to its general regional hospital's character. This study collects the service volumes of all departments in this hospital from one year before and for three years after the SARS outbreak. No similar study has been published previously.The municipal hospital in Taipei City studied here is a general and teaching hospital with 450 beds. The departments in this hospital include internal medicine, surgery, OBS/GYN, pediatrics, radiology and pathology. The hospital underwent a SARS outbreak during April 2003. Fiftynine hospital workers were infected and this led to seven deaths [1, 3] . To halt the outbreak of SARS infection, the hospital was shut down to allow evacuation of hospitalized patients and workers gradually over 10 days. Twentyfive days after that, the hospital was fully vacated and sterilization carried out. Outpatient services were first reopened after 35 days of quarantine. Inpatient and emergency services were reopened after 65 days. Subsequently, in addition to its general regional hospital function, the hospital became the infectious disease designated hospital for the Taipei area, and receives infectious disease patients, undergoes preparedness exercises and admits tuberculosis patients.In order to compare the recovery situation after SARS, this study follows the numbers of outpatients, emergency patients and inpatients from 2002 to 2006. The patient numbers for emergency and outpatient services, and number of patient days for inpatient service were used as quantitative evaluations and are expressed as the average monthly services amount ± standard deviation. The hospital was shut down during April 2003, and therefore the study collected the services volume from every April to next March. Year 2002 was selected as the base year of this study. The service volume of 2003, which is when SARS was occurred, and of the two years after that, were compared to the 2002 data. The study was approved by the ethics committee of the Taipei City Hospital Hoping branch.Statistically analysis by repeat measurement ANOVA (SPSS for Windows 15.0) was utilized in this study to compare the variation over 4 years. If the service volume for a specific area and year was less than the base year, and showed a significant difference (p < 0.05), then this year was considered to be still recovering. Based on this recovery situation, all the departments can be divided into four groups; these are departments that recovered in the year SARS happened, departments that recovered in the first year after SARS, departments that recovered in the second year after SARS, and finally departments still not yet recovered. The autoregressive integrated moving average (ARIMA) model of outpatient volume in the pre-SARS year was developed. Various permutations of the order of correlation and order of integration (I) were computed to choose the optimal combination of parameters based on the mean square error. Partial correlogram graphs and correlograms were used to select the order of moving average (MA) and autoregressive (AR) terms in the model. Based on the best model selected, the mean absolute percentage error (MAPE) was used to measure the quality of fit of the service volume of outpatients during the SARS, and two years after SARS period .During the four years of this study, the economic aspects that affected the hospital's performances remained the same, include beds and medical specialties. Furthermore, the number of similar hospitals in local area did not change. The organization of the healthcare system and its reimbursement still come from the Taiwan National Health Insurance (NHI) system . In addition, there were also no significant policy changes with respect to NHI payment policy in these four years. Moreover, the neighborhood population of this hospital underwent no obvious change (the neighborhood population was 173,637 in 2002 and had become 171,092 in 2006) . However, in addition to the general regional hospital, the hospital was also now designated as an infectious disease hospital.The average monthly number of outpatient visits at base year was 52317 ± 4204. The numbers for the year SARS that happened and the following two years were 55% (28950 ± 11731), 82% (42906 ± 4047) and 84% (43715 ± 2758) of the base year respectively (Table 1) . Average emergency numbers was 4382 ± 356 visits per month at base year. This number decreased to 45% (1975 ± 1242) of the base year for the year SARS happened, and came back to 77% (3395 ± 345) during the first year after SARS, and 87% (3824 ± 346) of the base year during the second year after SARS. The average inpatient service volume was 8520 ± 909 inpatient days per month at base year. The number decreased to 43% (3678 ± 2350) of base year for the year SARS happened, and came back to 81% (6863 ± 486) and 87% (7371 ± 471) of the base year for the following two years. Only the emergency service volumes recovered to the level of non-significant difference at the second year after SARS. Services volumes for outpatients and inpatients were still significant different (p < 0.05). Using the pre-SARS year as the training dataset in the ARIMA model, the ARIMA IAR (2,1,0) model was the most suitable. Similar result about the delayed recovery of outpatient services is noted (figure 1). The MAPE of the ARIMA (2,1,0) model for the pre-SARS year was 6.94%. The MAPE for the year of SARS was 43.24%, improving to 10.58% for the 1 st post-SARS year, and to 6.20% for the 2 nd post-SARS year.On examining the various outpatient departments, family medicine, metabolism and nephrology recovered during the year that SARS happened. Family medicine was even higher than the base year during the first and second years after SARS. Nephrology was also higher than the base year during the second year after SARS. The departments that recovered during the first year after SARS included neurology, cardiology, infectious diseases, neurosurgery, urology, plastic surgery, dentistry and psychiatry. Among these, psychiatry was even higher than the base year. Pediatrics recovered during the second year after SARS. The departments that still have not recovered include general surgery, ophthalmology, orthopedics, ENT, internal medicine, pulmonary medicine, gastroenterology, OBS/GYN, dermatology, rehabilitation and Chinese medicine ( Table 2 ). The numbers of physicians were 165 for the base year, 162 for the SARS year, 170 for the first year after SARS, and 173 for the second year after SARS. The numbers of physicians in each department were also similar for these years with differences ranging from 0 to 2 physicians.The SARS crisis at this municipal hospital during April 2003 resulted in a reduction in outpatient, emergency and inpatient services to 45%, 55% and 57% respectively,The ARIMA (2,1,0) model was the most suitable by using the pre-SARS year as the training dataset compared to the previous year. The delayed recovery of outpatient service was also noted during and in the first year after SARS by the ARIMA model. After the SARS epidemic, the hospital became the infectious disease designated hospital for the Taipei area in additional to retaining its character as a general regional hospital. Many factors caused the changes in hospital performance, such as patient transfer to other hospitals, reduction of healthcare manpower due to death, sequelae to staff infection and staff turnover. In addition, the publicity related to the change in the hospital status to an infectious disease unit influenced performance. In the few years since SARS, hospital performance has improved continuously. However, only emergency services had recovered by the second year after SARS. Outpatient and inpatient services had still not recovered by 2006.If we consider outpatient services, in the neighboring countries, certain reports have indicated a 20% to 59% decrease during the SARS epidemic only . Vlantis reported that the weekly outpatient clinic attendance showed a decline of 59% and the daily admission rate by 84% for the division of otorhinolaryngology head and neck surgery at an academic tertiary referral hospital in Hong Kong . None of the reported hospitals were shut down during the event. The outpatient service volume in There were also huge recovery differences between the different departments even though the physicians' numbers are similar over these four years. Family medicine, metabolism and nephrology departments have recovered quickly. Moreover, patient visits to family medicine and nephrology were even higher than before SARS. This indicates that chronic patients who need long-term treatment tend to go back to their former local hospital to receive their treatments. The patient visits to psychiatry decreased during the year SARS happened, and were obviously increased the year after SARS. This suggests that people may have needed to rely on psychiatry treatment after the SARS epidemic. Outpatient visits to surgery recovered slower. Only urology, plastic surgery and neurosurgery recovered in the year after SARS, and all others have not recovered as yet. Thus, the general public may not wish to receive surgery from an infectious disease designated hospital that once underwent an emergency shut down. As a result, the recovery of inpatient services is definitely affected. Inpatient services usually come from outpatient and emergency transfers. Therefore, recovery of the inpatient service volume is much slower than outpatient and emergency services.We observe recovery differences across different departments. However, the limitation of this study is that we are unable to explain the failure of certain departments to return to normal service levels in terms of the quantitative and statistical measurements of some variables that may affecting the performance of the hospital. A population based survey of patient's willingness and physicians' attitude such as the KAP (knowledge, attitude, practice) might help in this.Emergency department (ED) visits during SARS were decreased worldwide when the available literature is considered [6, 8, 13, 15] . The duration of the impact on each hospital as described in these papers is not the same, with one decreased for three months , another had recovered by the end of year 2003 after SARS , and another had recovered during the second year after SARS . In this study, ED visits only recovered during the second year after SARS. There was no change in the ED numbers in the local area. Some papers have suggested that the decrease in ED visits can have different impacts on different subdivisions. For example, non-critical ED patient visits may decrease more than the critical patient visits [17, 19] . One possible reason for the delay in ED service recovery at this hospital is that non-critical patient chose to go to other hospitals, and this needs further research. Alternatively, a patient response to the emergency whole hospital shut down may have occurred or there may be a mixture of both factors.The impacts caused by SARS were very serious, especially on global economics and health care. In order to control the epidemic, the Department of Health, Executive Yuan, Taiwan, designated some public hospitals to admit major infectious disease patients. This measure had a positive effect on controlling the epidemic , and reminds people of the importance that public hospitals have in public health, preventive medicine, and the prevention of infectious diseases. Even three years after SARS, the nation still maintains the ideology of a designated hospital, and has expanded the prevention network national wide. Under the threat of new epidemics, an infectious disease designated hospital has to face the issue of losses in its operating performance. The medical standards and ideology of the designated hospital are always affected.At present, medical expenditure is increasing dramatically everywhere in the world. It is very important for an infectious disease designated hospital to maintain its self-sufficient operation when the public health budget is under threat. Self-sufficient operation can reduce the need for a government subsidy and allow the maintenance of an adequate number of physicians as well as the quality of medical services at such a general hospital. Although the outpatient and inpatient services did not completely recovered from SARS during first year, the hospital had reached 84% to 86% of the baseline service volume during the second year after SARS. Therefore, the hospital should be capable of self-sufficiency as a designated hospital. In particular, the family medicine, metabolism and nephrology departments were hardly influenced by the shut down. In addition, some other departments had recovered by the second year. Therefore, these departments were able to maintain a sufficient number of physicians in the absence of an infection emergency. If another outbreak of major infectious disease occurs, the physicians in the above mentioned departments would be able to participate directly in prevention tasks. An adequate number of physicians are essential for the success of an infectious disease designated hospital. Thus, the findings of this study should provide a direction for other infectious disease designated hospitals to consider when deciding what subspecialties should be included in their makeup. If an infectious disease designated hospital includes a certain subspecialty and the physicians can operate self-sufficiently during ordinary times, the government only has to subside such a hospital during the year of the epidemic and the year after that. The hospital can become self-sufficient again quickly once everyday operations return.In conclusion, this study demonstrates that if the whole hospital is shut down during an infectious disease outbreak, the impact is much longer than other studies have shown. The outpatient and inpatient services in this example had not completely recovered by the second year after SARS. Emergency services were the fastest recovering department, but even this unit did not recover until the second year after SARS. Among outpatient services, family medicine, metabolism and nephrology recovered better than the others. Surgery needs a longer recovery period. This study provides the decision maker with information regards choices when managing an infectious disease designated hospital and is a reference resource for future public policy crisis management decisions during acute infectious disease outbreaks.\n",
            "\t0.975\tBackground: Severe acute respiratory syndrome (SARS) is caused by a newly discovered coronavirus (SARS-CoV). No effective prophylactic or post-exposure therapy is currently available.Severe acute respiratory syndrome (SARS) is an emerging disease that was first reported in Guangdong Province, China, in late 2002. The disease rapidly spread to at least 30 countries within months of its first appearance, and concerted worldwide efforts led to the identification of the etiological agent as SARS coronavirus (SARS-CoV), a novel member of the family Coronaviridae . Complete genome sequencing of SARS-CoV [2, 3] confirmed that this pathogen is not closely related to any of the previously established coronavirus groups. Budding of the SARS-CoV occurs in the Golgi apparatus and results in the incorporation of the envelope spike glycoprotein into the virion. The spike glycoprotein is a type I membrane protein that facilitates viral attachment to the cellular receptor and initiation of infection, and angiotensin-converting enzyme-2 (ACE2) has been identified as a functional cellular receptor of SARS-CoV . We have recently shown that the processing of the spike protein was effected by furin-like convertases and that inhibition of this cleavage by a specific inhibitor abrogated cytopathicity and significantly reduced the virus titer of SARS-CoV .Due to the severity of SARS-CoV infection, the potential for rapid spread of the disease, and the absence of proven effective and safe in vivo inhibitors of the virus, it is important to identify drugs that can effectively be used to treat or prevent potential SARS-CoV infections. Many novel therapeutic approaches have been evaluated in laboratory studies of SARS-CoV: notable among these approaches are those using siRNA , passive antibody transfer , DNA vaccination , vaccinia or parainfluenza virus expressing the spike protein [10, 11] , interferons [12, 13] , and monoclonal antibody to the S1-subunit of the spike glycoprotein that blocks receptor binding . In this report, we describe the identification of chloroquine as an effective pre-and post-infection antiviral agent for SARS-CoV. Chloroquine, a 9-aminoquinoline that was identified in 1934, is a weak base that increases the pH of acidic vesicles. When added extracellularly, the non-protonated portion of chloroquine enters the cell, where it becomes protonated and concentrated in acidic, low-pH organelles, such as endosomes, Golgi vesicles, and lysosomes. Chloroquine can affect virus infection in many ways, and the antiviral effect depends in part on the extent to which the virus utilizes endosomes for entry. Chloroquine has been widely used to treat human diseases, such as malaria, amoebiosis, HIV, and autoimmune diseases, without significant detrimental side effects . Together with data presented here, showing virus inhibition in cell culture by chloroquine doses compatible with patient treatment, these features suggest that further evaluation of chloroquine in animal models of SARS-CoV infection would be warranted as we progress toward finding effective antivirals for prevention or treatment of the disease.In order to investigate if chloroquine might prevent SARS-CoV infection, permissive Vero E6 cells were pretreated with various concentrations of chloroquine (0.1-10 µM) for 20-24 h prior to virus infection. Cells were then infected with SARS-CoV, and virus antigens were vis-ualized by indirect immunofluorescence as described in Materials and Methods. Microscopic examination (Fig. 1A ) of the control cells (untreated, infected) revealed extensive SARS-CoV-specific immunostaining of the monolayer. A dose-dependant decrease in virus antigen-positive cells was observed starting at 0.1 µM chloroquine, and concentrations of 10 µM completely abolished SARS-CoV infection. For quantitative purposes, we counted the number of cells stained positive from three random locations on a slide. The average number of positively stained control cells was scored as 100% and was compared with the number of positive cells observed under various chloroquine concentrations (Fig. 1B) . Pretreatment with 0.1, 1, and 10 µM chloroquine reduced infectivity by 28%, 53%, and 100%, respectively. Reproducible results were obtained from three independent experiments. These data demonstrated that pretreatment of Vero E6 cells with chloroquine rendered these cells refractory to SARS-CoV infection.In order to investigate the antiviral properties of chloroquine on SARS-CoV after the initiation of infection, Vero E6 cells were infected with the virus and fresh medium supplemented with various concentrations of chloroquine was added immediately after virus adsorption. Infected cells were incubated for an additional 16-18 h, after which the presence of virus antigens was analyzed by indirect immunofluorescence analysis. When chloroquine was added after the initiation of infection, there was a dramatic dose-dependant decrease in the number of virus antigen-positive cells ( Fig. 2A) . As little as 0.1-1 µM chloroquine reduced the infection by 50% and up to 90-94% inhibition was observed with 33-100 µM concentrations ( Fig. 2B ). At concentrations of chloroquine in excess of 1 µM, only a small number of individual cells were initially infected, and the spread of the infection to adjacent cells was all but eliminated. A half-maximal inhibitory effect was estimated to occur at 4.4 ± 1.0 µM chloroquine (Fig. 2C ). These data clearly show that addition of chloroquine can effectively reduce the establishment of infection and spread of SARS-CoV if the drug is added immediately following virus adsorption.Electron microscopic analysis indicated the appearance of significant amounts of extracellular virus particles 5-6 h after infection . Since we observed antiviral effects by chloroquine immediately after virus adsorption, we further extended the analysis by adding chloroquine 3 and 5 h after virus adsorption and examined for the presence of virus antigens after 20 h. We found that chloroquine was still significantly effective even when added 5 h after infection (Fig. 3) ; however, to obtain equivalent antiviral effect, a higher concentration of chloroquine was required if the drug was added 3 or 5 h after adsorption.Since chloroquine inhibited SARS-CoV infection when added before or after infection, we hypothesized that another common lysosomotropic agent, NH 4 Cl, might also function in a similar manner. Ammonium chloride has been widely used in studies addressing endosomemediated virus entry. Coincidently, NH 4 Cl was recently shown to reduce the transduction of pseudotype viruses decorated with SARS-CoV spike protein [17, 18] . In an attempt to examine if NH 4 Cl functions similarly to chloroquine, we performed infection analyses in Vero E6 cells before (Fig. 4A ) and after (Fig. 4B ) they were treated with various concentrations of NH 4 Cl. In both cases, we observed a 93-99% inhibition with NH 4 Cl at ≥ 5 mM. These data indicated that NH 4 Cl (≥ 5 mM) and chloroquine (≥ 10 µM) are very effective in reducing SARS-CoV infection. These results suggest that effects of chloroquine and NH 4 Cl in controlling SARS CoV infection and spread might be mediated by similar mechanism(s).We performed additional experiments to elucidate the mechanism of SARS-CoV inhibition by chloroquine and NH 4 Cl. Since intra-vesicular acidic pH regulates cellular functions, including N-glycosylation trimming, cellular trafficking, and various enzymatic activities, it was of interest to characterize the effect of both drugs on the processing, glycosylation, and cellular sorting of SARS-CoV spike glycoprotein and its receptor, ACE2. Flow cytometry analysis was performed on Vero E6 cells that were either untreated or treated with highly effective anti-SARS-CoV concentrations of chloroquine or NH 4 Cl. The results revealed that neither drug caused a significant change in the levels of cell-surface ACE2, indicating that the observed inhibitory effects on SARS-CoV infection are not due to the lack of available cell-surface ACE2 (Fig. 5A ). We next analyzed the molecular forms of endog-Prophylactic effect of chloroquine enous ACE2 in untreated Vero E6 cells and in cells that were pre-incubated for 1 h with various concentrations of either NH 4 Cl (2.5-10 mM) or chloroquine (1 and 10 µM) and labeled with 35 S-(Met) for 3 h in the presence or absence of the drugs (Fig. 5B and 5C ). Under normal conditions, we observed two immunoreactive ACE2 forms, migrating at ~105 and ~113 kDa, respectively (Fig. 5B , lane 1). The ~105-kDa protein is endoglycosidase H sensitive, suggesting that it represents the endoplasmic reticulum (ER) localized form, whereas the ~113-kDa protein is endoglycosidase H resistant and represents the Golgimodified form of ACE2 . The specificity of the antibody was confirmed by displacing the immunoreactive protein bands with excess cold-soluble human recombinant ACE2 (+ rhACE2; Fig. 5B , lane 2). When we analyzed ACE2 forms in the presence of NH 4 Cl, a clear stepwise increase in the migration of the ~113-kDa protein was observed with increasing concentrations of NH 4 Cl, with a maximal effect observed at 10 mM NH 4 Cl, resulting in only the ER form of ACE2 being visible on the gel (Fig. 5B , compare lanes . This suggested that the trimming and/or terminal modifications of the N-glycosylated chains of ACE2 were affected by NH 4 Cl treatment.In addition, at 10 mM NH 4 Cl, the ER form of ACE2 migrated with slightly faster mobility, indicating that NH 4 Cl at that concentration might also affect core glyco- sylation. We also examined the terminal glycosylation status of ACE2 when the cells were treated with chloroquine (Fig. 5C ). Similar to NH 4 Cl, a stepwise increase in the electrophoretic mobility of ACE2 was observed with increasing concentrations of chloroquine. At 25 µM chloroquine, the faster electrophoretic mobility of the Golgimodified form of ACE2 was clearly evident. On the basis of the flow cytometry and immunoprecipitation analyses, it can be inferred that NH 4 Cl and chloroquine both impaired the terminal glycosylation of ACE2, while NH 4 Cl resulted in a more dramatic effect. Although ACE2 is expressed in similar quantities at the cell surface, the variations in its glycosylation status might render the ACE2-SARS-CoV interaction less efficient and inhibit virus entry when the cells are treated with NH 4 Cl and chloroquine.To confirm that ACE2 undergoes terminal sugar modifications and that the terminal glycosylation is affected by NH 4 Cl or chloroquine treatment, we performed immunopreipitation of 35 S-labeled ACE2 and subjected the immu-Effect of lysomotropic agents on the cell-surface expression and biosynthesis of ACE2 noprecipitates to neuraminidase digestion. Proteins were resolved using SDS-PAGE (Fig 5D) . It is evident from the slightly faster mobility of the Golgi form of ACE2 after neuraminidase treatment (Fig 5D, compare lanes 1 and 2) , that ACE2 undergoes terminal glycosylation; however, the ER form of ACE2 was not affected by neuraminidase. Cells treated with 10 µM chloroquine did not result in a significant shift; whereas 25 µM chloroquine caused the Golgi form of ACE2 to resolve similar to the neuraminidase-treated ACE2 (Fig 5D, compare lanes 5 and 6) . These data provide evidence that ACE2 undergoes terminal glycosylation and that chloroquine at anti-SARS-CoV concentrations abrogates the process.We next addressed whether the lysosomotropic drugs (NH 4 Cl and chloroquine) affect the biosynthesis, glyco-sylation, and/or trafficking of the SARS-CoV spike glycoprotein. For this purpose, Vero E6 cells were infected with SARS-CoV for 18 h. Chloroquine or ammonium chloride was added to these cells during while they were being starved (1 h), labeled (30 min) or chased (3 h). The cell lysates were analyzed by immunoprecipitation with the SARS-specific polyclonal antibody (HMAF). The 30-min pulse results indicated that pro-spike (proS) was synthesized as a ~190-kDa precursor (proS-ER) and processed into ~125-, ~105-, and ~80-kDa proteins (Fig. 6A, lane 2) , a result identical to that in our previous analysis . Except for the 100 µM chloroquine (Fig. 6A, lane 3) , there was no significant difference in the biosynthesis or processing of the virus spike protein in untreated or chloroquine-treated cells (Fig. 6A, lanes 4-6) . It should be noted that chloroquine at 100 µM resulted in an overall decrease in biosynthesis and in the levels of processed virus glycoprotein. In view of the lack of reduction in the Effects of NH 4 Cl and chloroquine (CQ) on the biosynthesis, processing, and glycosylation of SARS-CoV spike protein Figure 6 Effects of NH 4 Cl and chloroquine (CQ) on the biosynthesis, processing, and glycosylation of SARS-CoV spike protein. Vero E6 cells were infected with SARS-CoV as described in Fig. 2 . CQ or NH 4 Cl was added during the periods of starvation (1 h) and labeling (30 min) with 35 S-Cys and followed by chase for 3 h in the presence of unlabeled medium. Cells were lysed in RIPA buffer and immunoprecipitated with HMAF. Virus proteins were resolved using 3-8% NuPAGE gel (Invitrogen). The cells presented were labeled for 30 min (A) and chased for 3 h (B). The migration positions of the various spike molecular forms are indicated at the right side, and those of the molecular standards are shown to the left side. proS-ER and proS-Golgi are the pro-spike of SARS-Co in the ER and Golgi compartments, respectively and proS-ungly is the unglycosylated pro-spike ER.biosynthesis and processing of the spike glycoprotein in the presence of chloroquine concentrations (10 and 50 µM) that caused large reductions in SARS-CoV replication and spread, we conclude that the antiviral effect is probably not due to alteration of virus glycoprotein biosynthesis and processing. Similar analyses were performed with NH 4 Cl, and the data suggested that the biosynthesis and processing of the spike protein were also not negatively affected by NH 4 Cl (Fig. 6A, lanes 7-12) . Consistent with our previous analysis , we observed the presence of a larger protein, which is referred to here as oligomers. Recently, Song et al. provided evidence that these are homotrimers of the SARS-CoV spike protein and were incorporated into the virions. Interestingly, the levels of the homotrimers in cells treated with 100 µM chloroquine and 40 and 20 mM NH 4 Cl (Fig. 6A, lanes 3, 9, and 10 ) were slightly lower than in control cells or cells treated with lower drug concentrations.The data obtained from a 30-min pulse followed by a 3-h chase (Fig. 6B , lanes 2 and 8) confirmed our earlier observation that the SARS-CoV spike protein precursor (proS-ER) acquires Golgi-specific modifications (proS-Golgi) resulting in a ~210-kDa protein . Chloroquine at 10, 25, and 50 µM had no substantial negative impact on the appearance of the Golgi form (Fig. 6B , compare lane 2 to lanes 4-6). Only at 100 µM chloroquine was a reduction in the level of the Golgi-modified pro-spike observed (lane 3). On the other hand, NH 4 Cl abrogated the appearance of Golgi-modified forms at ≥10 mM (compare lane 8 with 9-11) and had a milder effect at 1 mM (lane 12). These data clearly demonstrate that the biosynthesis and proteolytic processing of SARS-CoV spike protein are not affected at chloroquine (25 and 50 µM) and NH 4 Cl (1 mM) doses that cause virus inhibitory effects. In addition, with 40, 20, and 10 mM NH 4 Cl, there was an increased accumulation of proS-ER with a concomitant decrease in the amount of oligomers (Fig. 6B, lanes 9-11) . When we examined the homotrimers, we found that chloroquine at 100 µM and NH 4 Cl at 40 and 20 mM resulted in slightly faster mobility of the trimers (Fig. 6B, lanes 3, 9, and 10 ), but lower drug doses, which did exhibit significant antiviral effects, did not result in appreciable differences. These data suggest that the newly synthesized intracellular spike protein may not be a major target for chloroquine and NH 4 Cl antiviral action. The faster mobility of the trimer at certain higher concentration of the drugs might be due the effect of these drugs on the terminal glycosylation of the trimers.We have identified chloroquine as an effective antiviral agent for SARS-CoV in cell culture conditions, as evidenced by its inhibitory effect when the drug was added prior to infection or after the initiation and establishment of infection. The fact that chloroquine exerts an antiviral effect during pre-and post-infection conditions suggest that it is likely to have both prophylactic and therapeutic advantages. Recently, Keyaerts et al. reported the antiviral properties of chloroquine and identified that the drug affects SARS-CoV replication in cell culture, as evidenced by quantitative RT-PCR. Taken together with the findings of Keyaerts et al. , our analysis provides further evidence that chloroquine is effective against SARS-CoV Frankfurt and Urbani strains. We have provided evidence that chloroquine is effective in preventing SARS-CoV infection in cell culture if the drug is added to the cells 24 h prior to infection. In addition, chloroquine was significantly effective even when the drug was added 3-5 h after infection, suggesting an antiviral effect even after the establishment of infection. Since similar results were obtained by NH 4 Cl treatment of Vero E6 cells, the underlying mechanism(s) of action of these drugs might be similar.Apart from the probable role of chloroquine on SARS-CoV replication, the mechanisms of action of chloroquine on SARS-CoV are not fully understood. Previous studies have suggested the elevation of pH as a mechanism by which chloroquine reduces the transduction of SARS-CoV pseudotype viruses [17, 18] . We examined the effect of chloroquine and NH 4 Cl on the SARS-CoV spike proteins and on its receptor, ACE2. Immunoprecipitation results of ACE2 clearly demonstrated that effective anti-SARS-CoV concentrations of chloroquine and NH 4 Cl also impaired the terminal glycosylation of ACE2. However, the flow cytometry data demonstrated that there are no significant differences in the cell surface expression of ACE2 in cells treated with chloroquine or NH 4 Cl. On the basis of these results, it is reasonable to suggest that the pre-treatment with NH 4 Cl or chloroquine has possibly resulted in the surface expression of the under-glycosylated ACE2. In the case of chloroquine treatment prior to infection, the impairment of terminal glycosylation of ACE2 may result in reduced binding affinities between ACE2 and SARS-CoV spike protein and negatively influence the initiation of SARS-CoV infection. Since the biosynthesis, processing, Golgi modification, and oligomerization of the newly synthesized spike protein were not appreciably affected by anti-SARS-CoV concentrations of either chloroquine or NH 4 Cl, we conclude that these events occur in the cell independent of the presence of the drugs. The potential contribution of these drugs in the elevation of endosomal pH and its impact on subsequent virus entry or exit could not be ruled out. A decrease in SARS-CoV pseudotype transduction in the presence of NH 4 Cl was observed and was attributed to the effect on intracellular pH [17, 18] . When chloroquine or NH 4 Cl are added after infection, these agents can rapidly raise the pH and subvert on-going fusion events between virus and endosomes, thus inhibiting the infection.In addition, the mechanism of action of NH 4 Cl and chloroquine might depend on when they were added to the cells. When added after the initiation of infection, these drugs might affect the endosome-mediated fusion, subsequent virus replication, or assembly and release. Previous studies of chloroquine have demonstrated that it has multiple effects on mammalian cells in addition to the elevation of endosomal pH, including the prevention of terminal glycosyaltion of immunoglobulins . When added to virus-infected cells, chloroquine inhibited later stages in vesicular stomatitis virus maturation by inhibiting the glycoprotein expression at the cell surface , and it inhibited the production of infectious HIV-1 particles by interfering with terminal glycosylation of the glycoprotein [24, 25] . On the basis of these properties, we suggest that the cell surface expression of under-glycosylated ACE2 and its poor affinity to SARS-CoV spike protein may be the primary mechanism by which infection is prevented by drug pretreatment of cells prior to infection. On the other hand, rapid elevation of endosomal pH and abrogation of virus-endosome fusion may be the primary mechanism by which virus infection is prevented under post-treatment conditions. More detailed SARS CoV spike-ACE2 binding assays in the presence or absence of chloroquine will be performed to confirm our findings.Our studies indicate that the impact of NH 4 Cl and chloroquine on the ACE2 and spike protein profiles are significantly different. NH 4 Cl exhibits a more pronounced effect than does chloroquine on terminal glycosylation, highlighting the novel intricate differences between chloroquine and ammonium chloride in affecting the protein transport or glycosylation of SARS-CoV spike protein and its receptor, ACE2, despite their well-established similar effects of endosomal pH elevation.The infectivity of coronaviruses other than SARS-CoV are also affected by chloroquine, as exemplified by the human CoV-229E . The inhibitory effects observed on SARS-CoV infectivity and cell spread occurred in the presence of 1-10 µM chloroquine, which are plasma concentrations achievable during the prophylaxis and treatment of malaria (varying from 1.6-12.5 µM) and hence are well tolerated by patients. It recently was speculated that chloroquine might be effective against SARS and the authors suggested that this compound might block the production of TNFα, IL6, or IFNγ . Our data provide evidence for the possibility of using the well-established drug chloroquine in the clinical management of SARS.Chloroquine, a relatively safe, effective and cheap drug used for treating many human diseases including malaria, amoebiosis and human immunodeficiency virus is effective in inhibiting the infection and spread of SARS CoV in cell culture. The fact that the drug has significant inhibitory antiviral effect when the susceptible cells were treated either prior to or after infection suggests a possible prophylactic and therapeutic use.Vero E6 cells (an African green monkey kidney cell line) were infected with SARS-CoV (Urbani strain) at a multiplicity of infection of 0.5 for 1 h. The cells were washed with PBS and then incubated in OPTI-MEM (Invitrogen) medium with or without various concentrations of either chloroquine or NH 4 Cl (both from Sigma). Immunofluorescence staining was performed with SARS-CoV-specific hyperimmune mouse ascitic fluid (HMAF) followed by anti-mouse fluorescein-coupled antibody.Eighteen hours after infection, the virus-containing supernatants were removed, and the cells were pulsed with 35 S-(Cys) for 30 min and chased for 3 h before lysis in RIPA buffer. Clarified cell lysates and media were incubated with HMAF, and immunoprecipitated proteins were separated by 3-8% NuPAGE gel (Invitrogen); proteins were visualized by autoradiography. In some experiments, cells were chased for 3 h with isotope-free medium. Clarified cell supernatants were also immunoprecipitated with SARS-CoV-specific HMAF.Vero E6 cells were seeded in Dulbecco's modified Eagle medium (Invitrogen) supplemented with 10% fetal bovine serum. The next day, the cells were incubated in Opti-MEM (Invitrogen) in the presence or absence of 10 µM chloroquine or 20 mM NH 4 Cl. To analyze the levels of ACE2 at the cell surface, cells were incubated on ice with 10 µg/mL affinity-purified goat anti-ACE2 antibody (R&D Systems) and then incubated with FITC-labeled swine anti-goat IgG antibody (Caltag Laboratories). Labeled cells were analyzed by flow cytometry with a FAC-SCalibur flow cytometer (BD Biosciences). For ACE2 biosynthesis studies, Vero E6 cells were pulsed with 250 µCi 35 S-(Met) (Perkin Elmer) for 3 h with the indicated concentrations of chloroquine or NH 4 Cl and then lysed in RIPA buffer. Clarified lysates were immunoprecipitated with an affinity-purified goat anti-ACE2 antibody (R&D systems), and the immunoprecipitated proteins were separated by SDS-polyacrylamide gel electrophoresis.The author(s) declare that they have no competing interests.\n",
            "\t0.973\tBackground: Severe acute respiratory syndrome-associated coronavirus (SARS-CoV) spreads rapidly and has a high case-mortality rate. The nucleocapsid protein (NP) of SARS-CoV may be critical for pathogenicity. This study sought to discover the host proteins that interact with SARS-CoV NP.The outbreak of severe acute respiratory syndrome (SARS), which began in the Guangdong Province of China, spread rapidly to more than 30 countries during 2003. SARS has an acute onset, is highly transmissible and has a high case-mortality rate (approximately 10%) [1, 2] . During SARS infection, three phases of viral replication result in respiratory tract pathological changes and an over-exuberant host immune response. This mediates immunopathological damage of the lungs and other organs, and pulmonary fibrosis. SARS mortality is caused primarily by extensive lung damage and severe lymphopenia . Approximately 10% of individuals (6.8% of patients younger and 55% of patients older than 60 years of age) with clinical symptoms died as a consequence of immunopathological lung damage, caused by a hyperactive antiviral immune response .The mechanism of the serious damage to the respiratory system caused by SARS-CoV remains unclear. At least two possibilities exist: (i) direct damage to cells and tissues by the SARS-CoV and (ii) indirect damage, mediated primarily by the cellular immune response and cytokines.SARS-CoV nucleocapsid protein (SARS-CoV NP) is an extensively phosphorylated, highly basic, vital structural protein the primary function of which is to form a helical ribonucleoprotein complex with viral RNA (vRNA). This complex comprises the core structure of the SARS-CoV virion. A variety of functions have been ascribed to SARS-CoV NP, including packaging, transcription, and replication. However, these are based solely on known functions of the NP of other coronaviruses . SARS-CoV NP shows intrinsic multimerization and interacts with M protein, suggesting that NP is both critical to formation of the viral nucleocapsid core and is involved in virion assembly [6, 7] . Sequence analysis indicates that the RNA-binding domain of SARS-CoV NP may be located at residues 178-205 . Motif scanning predicted a bipartite nuclear localization signal, located at residues 373-390, suggesting that this protein may play a role in the pathogenicity of SARS-CoV .SARS-CoV NP is highly immunogenic. Antibodies against the nucleocapsid protein are longer lived and occur in greater abundance in SARS patients than antibodies against other viral components such as the spike, membrane and envelope proteins . This may be due to the presence of higher levels of nucleocapsid protein, compared with other viral proteins, after SARS-CoV infection . These data suggest that the SARS-CoV NP is strongly antigenic and so may play an important role in generation of the host immune response and immunopathological damage.In this study, SPR/BIACORE, MALID-TOF MS, the GST-fusion expression pulldown technique, and cell colocalization were used to investigate the interactions of SARS-CoV NP with host cell proteins. In this way, we sought to further elucidate the molecular pathogenic mechanisms of SARS-CoV. This, in turn, will allow development of novel therapeutics effective against this debilitating infection.Plasmid pET22b-SNP22b was constructed by cloning the SARS-CoV NP (SNP22b) gene by reverse transcriptase PCR (RT-PCR) using vRNA from SARS-CoV SCV-8 (isolated from a SARS patient in Beijing, China) with the following primers: forward: 5'-GAAGGATCCGATGT CTGATAATGGACCCCAATCAA-3', reverse: 5'-GCTG AATTCTTAATGGTGATGGTGATGGTGTGCCTGA GTTGAATCAGCAGAAGC-3'. PCR products were purified and inserted into the pET22b plasmid using BamHI/ EcoRI. The p42 gene was amplified by RT-PCR using mRNA from 2BS cells with the following primers: p42 forward: 5'-GATGAATTCATGGCGGACCCTAGAGA-TAAGG-3', reverse: 5'-GATCTCGAGTTACACAG-GTTTGTAGTCCAATTTAG-3'. PCR products were purified and cloned into the pGEX-5X-1 plasmid (Pharmacia, GE) using XhoI/EcoRI, to generate pGEX-5X-1-p42. The plasmid pcDNA3.0-SNP22b was constructed by subcloning the SNP gene, released from pET22b-SNP22b by BamHI/EcoRI, into pcDNA3.0 (Invitrogen). All of the recombinant clones were confirmed by sequencing. Escherichia coli DH5α and BL21 (DE3) were obtained from Invitrogen (USA).Human fetal lung diploid fibroblast (2BS) cells (SARS-CoV susceptible), were obtained from ATCC (USA) and maintained in DMEM (Invitrogen), containing 10% fetal bovine serum (FBS) and gentamicin (5 μg/mL). Transfection of 2BS cells was carried out using Lipofectamine 2000 according to the manufacturer's protocol. All restriction endonucleases were purchased from New England Biolabs (UK). Horse anti-SARS-CoV NP polyclonal antibody was a gift from the Academy of Military Medical Sciences. The anti-SARS-CoV NP mAb was prepared in our laboratory, while goat anti-mouse HRP-IgG and goat anti-rabbit IgG-FITC conjugates were purchased from Sigma (USA).Microtiter plates were coated with anti-SARS-CoV NP mAb (5 μg/mL) in bicarbonate buffer (15 mM Na 2 CO 3 , 35 mM NaHCO 3 , pH 9.6) overnight at 4°C. Plates were washed with PBST (100 mM NaCl, 10 mM Na 2 HPO 4 , 3 mM KH 2 PO 4 , 0.05% (v/v) Tween 20, pH 7.2) and blocked using 5% (w/v) fat-free milk in PBST for 1 h at 37°C. Plates were then washed with PBST and increasing concentrations of SARS-CoV NP added. SARS-CoV NP, bovine serum albumin (BSA), and PBS were the positive, negative, and blank controls, respectively. All plates were then incubated at 37°C for 1 h, followed by washing six times with PBST. All wells were incubated with anti-SARS-CoV NP horse polyclonal antibody-HRP conjugate for 1 h at 37°C. Plates were subsequently washed six times with PBST; TMB chromogenic substrate solution and stop solution were then added and the A 450 was determined.Samples were lysed in 1× loading buffer (0.08 M Tris, 2.0% (w/v) SDS, 10% (v/v) glycerol, 0.1 M dithiothreitol, 0.2% (w/v) bromophenol blue, pH 6.8). Samples were boiled for 10 min and resolved by one-dimensional SDS-PAGE. Proteins were transferred onto nitrocellulose membranes and the membranes were probed with the appropriate primary antibody. Secondary antibodies were alkaline phosphatase-conjugated anti-human, anti-rabbit, anti-mouse, or anti-goat IgG (Jackson Immunoresearch, Inc.). Gels were stained using 5-bromo-4-chloro-3-indolyl phosphate (BCIP) and nitro blue tetrazolium (NBT) solutions (Sigma).Plasmid pET22b-SNP22b was transformed into E. coli BL21. The fusion protein SNP-His was expressed under 1 mM IPTG at 22°C for 12 h. Bacteria were harvested and lyzed in lysis buffer (1 mg/mL lysozyme (Sigma), 1% (v/v) Triton X-100, 5 μg/mL DNAse, 5 μg/mL RNAse) at 4°C for 30 min. Lysates were harvested by centrifugation (3,000 × g, 30 min, 4°C). SARS-CoV NP levels in superna-tants were determined by Western blotting using anti-SARS-CoV NP horse polyclonal IgG as the primary antibody.SARS-CoV NP was purified by affinity chromatography using His. Bind ® Resin (Novagen) and ion exchange chromatography, using the Econo pac High CM cartridge (Bio-Rad). Purified NP was desalted using a PD-10 desalting column (Amersham) and concentrated by ultrafiltration using Centricon™ centrifugal filters (10 kDa MWCO, Millipore). Purified NP was suspended in HBS-EP buffer (pH 7.4, appropriate for BIAcore). SARS-CoV NP concentration was determined using a BCA protein assay reagent kit (Pierce). SARS-CoV NP activity was detected by indirect ELISA with anti-SARS-CoV NP horse polyclonal antibody.2BS cells were washed twice in cold 1× PBS and harvested. Pellets were lysed in 700 μL lysis buffer (1% Nonidet P40 and 20 μL protease inhibitor cocktail set III (Calbiochem) in 1 mL HBS-EP Buffer (pH 7.4, BIAcore), followed by freezing at -70°C and thawing at room temperature three times. Cells were harvested by centrifugation (12,000 × g, 20 min, 4°C). Supernatant was collected, aliquoted, and stored at -70°C.Host cell protein capturing was performed on a Sensor Chip CM5 (Amersham Bioscience) using BIAcore3000 and the amine coupling direct capture method, as described by the manufacturer. Briefly, SARS-CoV NP was diluted in 10 mM NaAc (pH 5.5). A Sensor Chip CM5 flow cell was activated with 70 μL of a mixture of EDC/NHS (Amine Coupling Kit, Amersham Bioscience), SARS-CoV NP (70 μL, 80 μg/mL) injected, and the flow cell was blocked using ethanolamine (70 μL, 1 M). Host cell lysate was then injected and allowed to flow through the cell containing immobilized SARS-CoV NP. A flow cell immobilized with HBS-EP buffer through which an identical volume of host cell lysate flowed represented the negative control. Lysis buffer, injected into a flow cell immobilized with SARS-CoV NP, functioned as the blank control. The reaction unit (RU) of SARS-CoV NP is obtained thus: RU reaction -RU blank control -RU negative control = RU SARS-CoV NP Captured proteins were precipitated by addition of a three-fold volume of cold acetone at -20°C overnight. Pellets were dissolved in HBS-EP buffer (10 μL) and 2× loading buffer (10 μL) and then resolved by 1-D SDS-PAGE on a 12% gel. The gel was stained using modified Neuhoff's colloidal Coomassie blue G-250 stain solution (0.12% (w/v) Coomassie Blue G-250 dye, 10% (w/v) ammonium sulfate, 10% (v/v) phosphoric acid, 20% (v/v) methanol) .Protein bands of interest were cut out of the gel and rinsed twice in 50% (v/v) methanol (HPLC grade). In-gel digestion was performed using sequencing-grade modified trypsin (Promega). Extracted peptides were analyzed by matrix-assisted laser desorption ionization time-offlight (MALDI-TOF) mass spectrometry by the Life Science Academy of Beijing University, China using an Ultraflex™ MALDI-TOF/TOF (Bruker) and Data Explorer (v. 4.0). Proteins were identified by comparison of their monoisotopic masses with those in the NCBI nonredundant or SwissProt databases using the MS-Fit search engine of ProteinProspector.Pulldown assay E. coli BL21 were transformed with the pGEX-5X-1-p42 plasmid. The fusion protein p42-GST was expressed by addition of IPTG (1 mM) at 22°C for 12 h. E. coli were lysed in lysis buffer (20 mM Tris-HCl (pH 8.0), 200 mM NaCl, 1 mM EDTA (pH 8.0), 0.5% NP40). SDS-PAGE and immunoblotting (anti-GST mAb, Pharmacia) were used to identify soluble expression of the p42-GST fusion protein.Glutathione Sepharose 4B beads (Pharmacia, GE) were suspended in lysis buffer to a concentration of 50% (w/v). Lysates of E. coli expressing the p42-GST fusion protein (1 mL), lysates of E. coli expressing GST alone, and lysis buffer represented the test sample, negative control, and blank control, respectively. Glutathione Sepharose 4B beads and bacterial lysates were added into three tubes, mixed for 60 min at 4°C, and 40 μL of this mixture was saved for SDS-PAGE analysis. The remaining mixture was washed with lysis buffer three times. Pellets were suspended in lysis buffer (200 μL), to which was added purified SARS-CoV NP (100 μL) and the mixture was further incubated at 4°C. After 3 h, the reaction mixture was washed three times in cold lysis buffer and eluted by boiling in 2× loading buffer. Eluted materials were subsequently analyzed by immunoblotting.2BS cells were cultured on coverslips in 6-well plates to 70-80% confluence and transfected with plasmid pcDNA3.0-SNP22b using the lipofectAMINE™ 2000 transfection kit (Invitrogen). At 48 h after transfection, coverslips were washed in 1× PBS and fixed in 1% paraformaldehyde. Coverslips were then blocked with 5% (w/ v) skimmed milk at 37°C for 60 min and washed six times in 1× PBS. Following incubation with a 1:100 dilution of mouse anti-SARS-CoV NP mAb and a 1:100 dilution of anti-p42 polyclonal antibody (Biomol, USA) at 37°C for 60 min, coverslips were washed six times with 1× PBS. After incubation with a 1:80 dilution of goat anti-rabbit IgG-FITC (Sigma) and a 1:200 dilution of goat antimouse IgG-TRITC (Sigma) conjugates at 37°C for 45 min, coverslips were again washed six times in distilled water and air-dried before being mounted on a slide with interspaces containing 50% (v/v) glycerol. Photomicrographs were taken using a confocal microscope (Confocal Microscope FluoView™ FV1000).A plasmid expression vector, pET22b-SNP22b, was constructed to express SARS-CoV NP in E. coli BL21. Expression of SARS-CoV NP was confirmed by SDS-PAGE (Fig. 1A) and Western blotting using a horse polyclonal antibody against SARS-CoV NP (Fig. 1B) . When expression was induced using isopropyl β-D-thiogalactopyranoside (IPTG), cells transfected with pET22b-SNP22b produced large amounts of SARS NP, amounting to up to 2% of total soluble protein ( Fig. 1A; lanes 4 and 8) .Purification of SARS-CoV NP was achieved first by Ni 2+ chelate affinity chromatography, where SARS-CoV NP was eluted using wash buffer containing imidazole (250 mM), and then further purified by cation exchange chromatography. The purity of SARS-CoV NP achieved by means of this procedure was greater than 90%. SARS-CoV NP was desalted, dissolved in HBS-EP buffer, concentrated, and quantified. Indirect ELISA data suggested that purified SARS-CoV NP had a specific antibody binding activity similar to the native form (data not shown).SARS-CoV NP was diluted to 140 μg/mL in 10 mM NaAc (pH 5.5) for immobilization. Regeneration was achieved using NaOH (50 mM). Immobilization of SARS-CoV NP in a sensor chip CM5 flow cell resulted in an RU value of 12355.1. This increased to 31 after HBS-EP buffer was immobilized in an adjacent flow cell of the same chip and increased further upon addition of 2BS cell lysates to the flow cell containing SARS-CoV NP. A stable level (1120 RU) was achieved after a total of 70 μL 2BS cell lysate was injected. The flow cell was then washed six times and NaOH (5 μL, 50 mM) was injected to elute proteins bound to SARS-CoV NP. The RU values of the blank and negative controls were 23 RU and 41 RU, respectively. Thus, the specific reaction of SARS-CoV NP and 2BS cell lysate was quantified as 1056 RU (calculated by 1120RU-23RU-41RU), suggesting that SARS-CoV NP had bound to at least one protein present in 2BS cell lysate ( Fig. 2A) . Captured proteins were recovered and collected by repeating the direct capture and blank control procedures.The protein(s) captured by SARS-CoV NP were precipitated, resolved by 1D SDS-PAGE and visualized using modified Neuhoff's colloidal Coomassie blue G-250 staining. SARS-CoV NP was found to have bound three proteins from 2BS cell lysate, the molecular weights of which ranged from 43 to 66 kDa (Fig. 2B, lane 3) . No protein was captured from the blank or negative controls (Fig. 2B, lanes 1, 2) . Analysis by MALID-TOF mass spectrometry identified the proteins as: 26S proteasome regulatory subunit S10B (proteasome subunit p42 or proteasome 26S subunit ATPase 6, P62333; Fig. 2B ), cytokeratin 1 (CK1, P04264), and cytoskeletal protein 10 (CK10, P13645). Cytokeratins are a subfamily of intermediate filament proteins. Cytokertin 1 (CK1) has the highest molecular weight and the highest isoelectric point of the family, while cytokeratin 10 has the lowest molecular weight and a low isoelectric point. These proteins are expressed in combinations that vary according to the type of epithelium within which they are found. Proteasome subunit p42 was selected for further assessment; the roles of CK1 and CK10 will be considered in a subsequent study.To examine further the interaction between SARS-CoV NP and proteasome subunit p42 in vitro, GST and the p42-GST fusion protein were produced, purified, and identified by 1D SDS-PAGE (Fig. 3A) and Western blotting, using an anti-GST mAb (Fig. 3B) . Interaction of SARS-CoV NP with the proteasome subunit p42 in vitro was identified using the p42-GST fusion protein pulldown technique. Data suggested that the SARS-CoV NP was pulled down by p42-GST fusion protein (Fig. 4, lanes 6, 8) , but not by either of the negative controls: GST alone (Fig. 4, lanes 1, 3, 5, 7) or glutathione Sepharose 4B beads ( Fig. 4, lanes 2, 9) . Thus, these data suggest that SARS-CoV NP exhibits a specific interaction with the proteasome subunit p42 in vitro.To analyze the interaction of SARS-CoV NP and proteasome subunit p42 within host cells, SARS-CoV NP was expressed in 2BS cells and identified by indirect immunofluorescence using an anti-SARS-CoV NP mAb (Fig. 5A ).After transfection of 2BS cells with SNP22b-pcDNA3.0, the proteasome subunit p42 was detected by means of an anti-p42 antibody and anti-rabbit IgG-FITC conjugate (Fig. 5B , lane 1; excitation wavelength 488 nm). Expression of SARS-CoV NP in 2BS cells was confirmed using an anti-SARS-CoV NP mAb and anti-mouse IgG-TRITC conjugate (Fig. 5 , lane 2; excitation wavelength 568 nm). Thus, the co-localization signals (Fig. 5 , lane 3; excitation SARS is an emerging infectious disease that has become a global public health concern in the 21 st century. However, the pathogenesis of SARS remains unclear. It has been suggested that the cytokine storm observed during the early stages of SARS in most patients contributes to the progression of systemic inflammatory response syndrome . The SARS-CoV NP has been shown to be highly immunogenic and the predominant target for the humoral immune response during infection with SARS-CoV . This process may trigger cytokine production and so induce apoptosis in host cells .Law suggested that SARS-CoV, during infection of dendritic cells, evades the immune response by down-regulating expression of the anti-viral chemokines IFN-α, β and γ and IL-12p40, while simultaneously up-regulating that of others: for example, TNF-α, IL-6, MIP-1α, and IP-10 . Additionally, it seems likely that proinflammatory cytokines released by macrophages in pulmonary alveoli play an important role in the pathogenesis of SARS.SARS-CoV is capable of inducing apoptosis of Vero E6 cells in vitro . Apoptosis of T-and B-lymphocytes in the immune organs and pulmonary alveoli and mononuclear inflammatory infiltration in the lungs, lymph nodes, and spleens of SARS-CoV infected individuals has been observed. Some SARS-CoV proteins, for example, E protein, NP, ORF7a, ORF3a, and ORF3b, have been shown to induce apoptosis in vitro [13, . Thus, the rapid apoptosis induced by SARS-CoV may be one of the mechanisms responsible for the damage to the lungs and immune organs observed in SARS patients.Interaction of viral and host proteins may contribute to the processes involved in viral infection, replication, and assembly and so increasing our knowledge of these interactions may lead to increased insight into the pathology, clinical manifestations, and pathogenesis of SARS. Li et al. reported in 2003 that angiotensin-converting enzyme 2 acted as a functional receptor for the SARS-CoV by binding to SARS-CoV S protein . To further examine the role of such interactions in the pathogenesis of SARS-CoV infection, we investigated the ability of SARS-CoV nucleocapsid protein to bind to host cell proteins. The prokaryotic expression vector pET22b was used for soluble expression of SARS-CoV NP. This vector contains the PeIB signal peptide to assure correct conformation of NP in E. coli. Because the SARS-CoV NP, unlike other SARS-CoV proteins, contains no glycosylation site, the recombinant form shows identical immune reactivity to that of the native protein .Data suggested that the proteasome subunit p42 of 2BS cells interacted with SARS-CoV NP. To our knowledge, this is the first report of such an interaction. Proteasome subunit p42 (also called the 26S protease regulatory subunit S10B) is a subunit of PA700 in the ubiquitin-proteasome pathway (UPP). This pathway is one of a number of intracellular proteolytic systems in eukaryotes and mediates ATP-dependent degradation of ubiquitinated proteins. The 26S ATP-dependent proteasome, formed from the 20S proteasome and 19S regulatory particle (PA700), is a multi-protein complex. The 20S component has proteolytic activity, while the PA700 component binds ubiquitinated proteins and promotes their degradation by the 20S component. Consistent with its ATPase activity, the proteasome subunit p42 contains an ATP binding region in a conserved 200 aa domain. The proteasome subunit p42, together with the five other subunits, TBP1, MSS1, S4, p45, and TBP7, assemble to form the PA700 component. 26S proteasomes are distributed in the cytoplasm and nucleolus [22, 23] . The ubiquitin-proteasome pathway mediates turnover of intracellular proteins. It plays a central role in the degradation of short-lived and regulatory proteins that are themselves vital for correct functioning of a variety of basic cellular processes including regulation of the cell cycle, modulation of cell surface receptor and ion channel expression, and antigen processing and presentation .While proteasomes do hydrolyze endogenous proteins, they are also capable of degrading exogenous proteins: for example, viral peptides. In this way, these complexes are involved in defenses against viral infection. Interactions between viruses and UPP have recently been reported. Hepatitis B virus X protein (HBX) has been demonstrated to target the proteasome complex after viral entry into the cell . Hu revealed that HBX is both a substrate and a potential inhibitor of the proteasome complex. The authors suggested that HBX may inhibit proteasomemediated proteolysis by binding two subunits (Sub4, Sub7) of the 26S proteasome . HBX expression inhibited turnover of c-Jun and ubiquitin-arginine-β-galactosidase, both known substrates of UPP . HIV1 Tat protein was shown to inhibit the degradative activity of 20S proteasomes, by interacting with the S6a subunit of the 26S component . Adenovirus E1A protein enhanced degradation of topoisomerase II alpha protein, via interaction with the S8 subunit of the 26S component . Evidence to date suggests that viruses employ two strategies for proteasome inhibition; first, interference with the antigen presentation activity of the proteasome, thus promoting immune evasion. Second, stimulation of the degradative activity of the 26S proteasome causes an acceleration of G1/S turnover, thus promoting viral replication by interference with the cell cycle . During viral infection, UPP hydrolyzes viral proteins into short peptides, which are then presented to CTL via MHC-1. Activated CTL then attack virally infected cells. The transcription factor nuclear factor-κB (NF-κB) is involved in regulation of immunity and inflammation; the UPP plays a central role in the regulation of NF-κB activation. When a cell is infected by a virus, IκB is phosphorylated and hydrolyzed by UPP, thus releasing NF-κB, which enters the nucleus and activates transcription of genes encoding inflammatory responses . Thus, the UPP may play a vital role in infection, and the host response to infection, by a variety of viral pathogens. Development of a specific inhibitor of UPP may have potential not only as an antiviral, but also as an anti-tumor and anti-inflammatory agent. Study of the interactions of components of the UPP with viral proteins may provide useful information relating to the pathogenesis of a number of diseases .Determining potential mechanisms of the interaction of SARS-CoV NP with proteasome subunit p42 was outside the scope of this study. However, such information may enhance our understanding of the activities of proteasomes in eukaryotic cells. We assumed that the interaction between SARS-CoV NP and proteasome subunit p42 inhibited the proteolytic activity of proteasomes. The actions of SARS-CoV NP and some other viral proteins may act to increase the deleterious effect of SARS-CoV infection, resulting in increased permeability, inflammatory infiltration, and mononuclear cell soakage, perhaps even accelerating pulmonary fibrosis. Moreover, this interaction may impair proteolysis of viral proteins and their presentation to CTLs and thus aid SARS-CoV evasion of immune effectors. Furthermore, such activity may affect inflammatory processes, resulting in the immunopathological damage so common in SARS. SARS-CoV NP is post-translationally modified by covalent attachment to small ubiquitin-like modifiers. Sumoylation of NP aids in promoting homo-oligomerization and assists in interference with host cell division .Whether SARS-CoV NP interacts with proteasome subunit p42 in vivo remains unknown, and so further studies are required to fully determine the nature of this interaction and its effect in vivo during SARS-CoV infection. Furthermore, more research on the downstream effect of this interaction may lead to the development of novel antiviral agents effective against this debilitating and often fatal infection.In this study, we demonstrated for the first time that SARS-CoV NP interacts with the proteasome subunit p42 in vitro. This finding may lead to a new direction in research focusing on the molecular mechanisms of SARS-CoV infection and the pathogenesis of SARS. Whether the interaction of SARS-CoV NP and p42 impacts presentation of viral antigens and assists in viral evasion of CTLs and/or promotes enhanced inflammatory responses resulting in immunopathological damage remain to be determined.\n",
            "\n",
            " Title:  Respiratory Research Molecular mechanisms of severe acute respiratory syndrome (SARS)\n",
            "5 \b. \n",
            "\n",
            "Input question: Where was COVID-19 discovered?\n",
            "Results (after 2.066 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.527\tIn December 2019, a novel coronavirus, called COVID-19, was discovered in Wuhan, China, and has spread to different cities in China as well as to 24 other countries. The number of confirmed cases is increasing daily and reached 34,598 on 8 February 2020. In the current study, we present a new forecasting model to estimate and forecast the number of confirmed cases of COVID-19 in the upcoming ten days based on the previously confirmed cases recorded in China. The proposed model is an improved adaptive neuro-fuzzy inference system (ANFIS) using an enhanced flower pollination algorithm (FPA) by using the salp swarm algorithm (SSA). In general, SSA is employed to improve FPA to avoid its drawbacks (i.e., getting trapped at the local optima). The main idea of the proposed model, called FPASSA-ANFIS, is to improve the performance of ANFIS by determining the parameters of ANFIS using FPASSA. The FPASSA-ANFIS model is evaluated using the World Health Organization (WHO) official data of the outbreak of the COVID-19 to forecast the confirmed cases of the upcoming ten days. More so, the FPASSA-ANFIS model is compared to several existing models, and it showed better performance in terms of Mean Absolute Percentage Error (MAPE), Root Mean Squared Relative Error (RMSRE), Root Mean Squared Relative Error (RMSRE), coefficient of determination (R 2 ), and computing time. Furthermore, we tested the proposed model using two different datasets of weekly influenza confirmed cases in two countries, namely the USA and China. The outcomes also showed good performances.A large family of viruses, called coronaviruses, are severe pathogens for human beings, which infect respiratory, hepatic, gastrointestinal, and neurologic diseases. They are distributed among humans, birds, livestock, mice, bats, and other wild animals . The outbreaks of two previous coronaviruses, SARS-CoV and MERS-CoV in 2003 and 2012, respectively, have approved the transmission from animal to animal, and human to human . In December 2019, the World Health Organization (WHO) received notifications from China for many cases of respiratory illness that were linked to some people who had visited a seafood market in Wuhan . Currently, Wuhan city suffers from the spreading of a novel coronavirus, called COVID-19 (previously, it was called 2019-nCoV). In , the authors concluded that COVID-19 likely originated in bats, because it is more similar to two bat-derived coronavirus strains. However, the source of the COVID-19 is not confirmed yet, and it communities, Hong Kong and Toronto, were 1.2 and 1.32, respectively. Ong et al. proposed a monitoring and forecasting model for influenza A (H1N1-2009). Furthermore, Nah et al. proposed a probability-based model to predict the spread of the MERS.The Adaptive Neuro-Fuzzy Inference System (ANFIS) is widely applied in time series prediction and forecasting problems, and it showed good performance in many existing applications. It offers flexibility in determining nonlinearity in the time series data, as well as combining the properties of both artificial neural networks (ANN) and fuzzy logic systems. It has been applied in various forecasting applications, for example, in , a stock price forecasting model was proposed using ANFIS and empirical mode decomposition. Chen et al. proposed a TAIEX time series forecasting model based on a hybrid of ANFIS and ordered weighted averaging (OWA). In , another time series forecasting method was presented for electricity prices based on ANFIS. Svalina et al. proposed an ANFIS based forecasting model for close price indices for a stock market for five days. Ekici and Aksoy presented an ANFIS based building energy consumption forecasting model. More so, ANFIS is also applied to forecast electricity loads . Kumar et al. proposed an ANFIS based model to forecast return products. Ho and Tsai applied ANFIS to forecast product development performance. However, estimating ANFIS parameters is a challenge that needs to be improved. Therefore, in previous studies, some individual swarm intelligence (SI) methods have been applied to the ANFIS parameters to enhance time series forecasting because these parameters have a significant effect on the performance of ANFIS. The SI methods include the particle swarm optimization (PSO) [31, 32] , social-spider optimization , sine-cosine algorithm (SCA) , and multi-verse optimizer (MVO) . For example, in SCA algorithm was applied to improve the ANFIS model to forecast oil consumption in three countries, namely, Canada, Germany, and Japan. In the same context, in , The MVO algorithm was used to enhance the ANFIS model to forecast oil consumption in two countries. In addition, in the PSO was used with ANFIS to predict biochar yield. However, individual SI algorithms may stock at local optima. Therefore, one solution is to apply hybrid SI algorithms to avoid this problem. In , a hybrid of two SI algorithms, namely GA and SSA, was presented to improve the ANFIS model. The proposed new model called GA-SSA-ANFIS was applied to forecast crude oil prices for long-term time series data. However, the previously mentioned methods suffer from some limitations that can affect the performance of the forecasting output such as slow convergence and the ability to balance between exploration and exploitation phases can influence the quality of the final output. This motivated us to propose an alternative forecasting method dependent on the hybridization concept. This concept avoids the limitations of traditional SI techniques by combining the strengths of different techniques, and this produces new SI techniques that are better than traditional ones.In the current study, we propose an improved ANFIS model based on a modified flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). The FPA is an optimization algorithm proposed by Yang , which was inspired by the flow pollination process of the flowering plants. The FPA was employed in various optimization applications, for example to estimate solar PV parameter [39, 40] , solving sudoku puzzles , feature selection , antenna design , and other applications . Moreover, SSA is also an optimization algorithm proposed by Mirjalili et al. inspired by the behavior of salp chains. In recent years, the SSA was utilized to solve different optimization problems, such as feature selection [49, 50] , data classification , image segmentation , and others [53, 54] .The proposed method called FPASSA is a hybrid of FPA and SSA, in which the SSA is applied as a local search method for FPA. The proposed FPASSA starts by receiving the historical COVID-19 dataset. Then a set of solutions is generated where each of them represents the value for the parameters of the ANFIS model. Then the quality of each solution is calculated using the fitness value, and the solution that has the best fitness value is chosen to represent the best solution. Then the probability of each solution is computed. Then the current solution will be updated, either using global or local strategy in FPA. However, in the case of local strategy, the operators of SSA or FPA will be used according to the probability of the fitness value for each solution. The process of updating the solutions is repeated until reaching the stop condition, and the best parameter configurations are used to forecast the number of confirmed cases of COVID-19.The main contribution points of the current study are as follows:1.We propose an efficient forecasting model to forecast the confirmed cases of the COVID-19 in China for the upcoming ten days based on previously confirmed cases.An improved ANFIS model is proposed using a modified FPA algorithm, using SSA.We compare the proposed model with the original ANFIS and existing modified ANFIS models, such as PSO, GA, ABC, and FPA.The rest of this study is organized as follows. The preliminaries of ANFIS, FPA, and SSA are described in Section 2. Section 3 presents the proposed FPASSA, and Section 4 presents the experimental setup and results. We conclude this study in Section 5.The principles of the ANFIS are given in this section. The ANFIS model links the fuzzy logic and neural networks . It generates a mapping between the input and output by applying IF-THEN rules (it is also called Takagi-Sugeno inference model). Figure 1 illustrates the ANFIS model where, y and x define the inputs to Layer 1 whereas, O 1i is its output of node i that is computed as follows:where µ denotes the generalized Gaussian membership functions. A i and B i define the membership values of µ. α i and ρ i denote the premise parameters set. The output of Layer 2 (it is also known as the firing strength of a rule) is calculated as follows:Meanwhile, the output of Layer 3 (it is also known as the normalized firing strength) is calculated as follows:The output of Layer 4 (it is also known as an adaptive node) is calculated as follows:where r i , q i , and p i define the consequent parameters of the node i. Layer 5 contains only one node; its output is computed as: Flower Pollination Algorithm is an optimization method proposed by Yang . It simulates the transfer of flowers' pollen by pollinators in nature. This algorithm utilizes the two types of pollination (i.e., self-pollination and cross-pollination). In self-pollination, the pollination occurs with no pollinators, whereas, in cross-pollination, the pollens are moved between different plants. In more detail, the self-pollination can be represented as a local pollination while the cross-pollination can be called global pollination.The global pollination or cross-pollination can be mathematically formed as follows:where x t i defines the pollen i at iteration t. L denotes the pollination's strength or the step size. F * is the target position or best solution. In some cases, insects can fly with different distance steps for a long space; therefore, Levy fly distribution is applied to simulate this movement.where λ = 1.5. Γ(λ) denotes the gamma function. This distribution is available for large steps s > 0. The self-pollination or local pollination can be mathematically formed as follows:where x t i and x k i represent pollens from different flower in the same plant. in the range [0,1] The process of pollination can be done using cross-pollination or self-pollination. Therefore, the random variable p, in the range [0, 1], is used to determine this process.SSA is an optimization technique introduced by . It simulates the Salps' behavior in nature. This behavior is called salp chain. The mathematical model of SSA begins by splinting its population into a leader group and followers group. The leader is the front salp, whereas, the followers are the other salps. The search space is determined in n-dimensions with n variables. Equation (10) works to update the salps' positions.where x 1 j denotes the leader's position in j-th dimension. F j is the target position. ub j and lb j represent the max and min bounds, respectively. c 2 and c 3 denote random numbers in [0, 1]. c 1 is an important parameter; it balances between the exploration and exploitation phases. It is computed as follows:where the current loop number is t and the max loop' number is t max . Then, the followers' position is updated as follows:where x i j defines the i-th position of the follower in j-th dimension. i > 1.This section explains the proposed FPASSA-ANFIS method. It is a time series method for forecasting the confirmed cases of the COVID-19, as given in Figure 2 . The FPASSA-ANFIS utilizes the improved FPA to train the ANFIS model by optimizing its parameters. The FPASSA-ANFIS contains five layers as the classic ANFIS model. Layer 1 contains the input variables (the historical COVID-19 confirmed cases). Whereas Layer 5 produces the forecasted values. In the learning phase, the FPASSA is used to select the best weights between Layer 4 and Layer 5.The FPASSA-ANFIS starts by formatting the input data in a time series form. In our case, the autocorrelation function (ACF) was considered. ACF is one of the methods applied to find patterns in the data; it presents information about the correlation between points separated by various time lags. Therefore, in this paper, the variables with ACF greater than 0.2 are considered i.e., 5-lags.Besides, the training data contains 75% of the dataset, whereas the testing data contains 25% of them. The number of clusters is defined by the fuzzy c-mean (FCM) method to construct the ANFIS model.The parameters of the ANFIS model are prepared by the FPASSA algorithm. In the training phase, the calculation error (as in Equation (13)) between the real data and the predicted data is used to evaluate the parameters' quality.where T is the real data, and P is the predicted data. N s is the sample length. The smaller values of the objective function indicate good ANFIS's parameter.On the other hand, the updating phase of the followers' positions in the SSA algorithm is applied to improve the global pollination phase in the FPA algorithm. In this improvement, there is a random variable (r) used to switch between both phases. If r > 0.5, then the operators of the SSA is used; otherwise, the operators of the FPA are used. In general, The FPASSA starts by constructing the population (X); afterward, the objective function is calculated for each solution. The solution with the lowest error value is saved to the next iteration. This sequence is repeated until meeting the stop condition, which in this paper, is the maximum number of iterations. Then the best solution is passed to train the parameters of the ANFIS model.After finishing the training phase, the testing phase is started with the best solution to compute the final output. The performance of the proposed method is evaluated by comparing the real data with the predicted data using the performance measures. Finally, the FPASSA produces a foretasted value for confirmed cases of COVID-19 in China in the next day. The steps of the proposed FPASSA are presented in Algorithm 1.Input: Historical COVID-19 dataset, size of population N, total number of iterations t max .Divide the data into training and testing sets.Using Fuzzy c-mean method to determine the number of membership functions.Constructing the ANFIS network.Set the initial value for N solutions (X). Return the best solution that represents the best configuration for ANFIS.Apply the testing set to the best ANFIS model.Forecasting the COVID-19 for the next ten days.This section presents the description of the used dataset, the performance measures, the parameter setting for all methods, the experiment results, and discussions.The main dataset of this study is COVID-19 dataset. It was collected from the WHO website (https: //). It contains the daily confirmed cases in China from 21 January 2020 to 18 February 2020, as shown in Table 1 . We used 75% from the dataset to train the model while the rest is used to test it.Moreover, we evaluated the performance of the proposed method using two datasets of weekly influenza confirmed cases. The first one is called DS1; it was collected from the Centers for Disease Control and Prevention (CDC) (). It starts from week number 40 in 2015 and continues until week number 6 in 2020. Whereas, the second one is called DS2. It was collected from the WHO website (). It contains the data of weekly influenza confirmed cases in China from week number 1 in 2016 to week number 8 in 2020. The quality of the proposed method is evaluated using a set of performance metrics as follows:• Root Mean Square Error (RMSE):where Yp and Y are the predicted and original values, respectively. • Mean Absolute Error (MAE):• Mean Absolute Percentage Error (MAPE):• Root Mean Squared Relative Error (RMSRE):N s represents the sample size of the data. • Coefficient of Determination (R 2 ):where Y represents the average of Y.The lowest value of RMSE, MAE, MAPE, and RMSRE refers to the best method. The higher value of R 2 indicates better correlation for the method.This paper aims to assess the ability of the FPASSA to forecast the COVID-19 by comparing its performance with other methods, namely the ANFIS and the trained ANFIS models using PSO, GA, ABC, FPA, and FPASSA. The parameters' setting for these models is listed in Table 2 .The common parameters, such as population size, are set to 25 and 100 iterations are applied. Besides, each algorithm is performed for 30 independent runs to fair comparisons. The selected parameters are chosen because they produced good behavior in previous experiments, such as [34, 35, 55, 56] . Table 2 . Parameters' setting.Parameters SettingMax. epochs = 100, Error goal = 0, Initial step = 0.01, Decrease rate = 0.9, Increase rate = 1. In this section, the performance of the proposed FPASSA to predict the DS1 and DS2 is discussed. It can be concluded from Table 3 that the performance of FPASSA outperformed the compared methods in all measures, whereas the FPA is ranked second. The results of DS2 indicate that the FPASSA is ranked first in terms of RMSE, MAPE, R 2 , and the CPU time. Whereas, the PSO is ranked second, followed by the FPA, GA, then ABC. These results denote that the proposed method can optimize the parameters of the ANFIS model effectively and produce good results in terms of the performance measures. Comparison results between the proposed FPASSA and other models to forecast COVID-19 are given in Table 4 . It can be concluded that the FPASSA outperforms other models. For example, by analyzing the results of RMSE, MAE, MAPE, RMSRE, and CPU time(s) it can be observed that the FPASSA achieves the smallest value among the comparison algorithms, and this indicates the high quality of the FPASSA. Meanwhile, the FPA allocates the second rank, which provides better results than the rest of the methods.Moreover, the value of R 2 refers to the high correlation between the prediction obtained by the proposed FPASSA method and the original COVID-19, which has nearly 0.97. This can also be noticed from Figure 3 , which depicts the training of the algorithms using the historical data of the COVID-19 as well as their forecasting values for ten days. Table 5 depicts the forecasting value for the confirmed cases of the COVID-19 in China from 19/2/2020 to 28/2/2020. From these results, it can be noticed that the outbreak will reach its highest level on the day 28/2/2020. The average percentage of the increase over the forecasted period is 10%, the highest percentage is 12% on 28/2/2020, and the lowest percentage is 8.7% on 19/2/2020. From the previous results, it can be concluded that the proposed FPASSA-ANFIS has a high ability to forecast the COVID-19 dataset. These results avoid the limitations of traditional ANFIS because of the combination with the modified FPA method. Moreover, the operators of SSA are combined with the local strategy of FPA to enhance their exploitation ability. However, the time computational of the proposed FPASSA method still requires more improvements.This paper proposed a modified version for the flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). This modified version, called FPASSA, is applied to improve the performance of the ANFIS through determining the optimal value for its parameters. The developed FPASSA-ANFIS model is applied as a forecasting technique for a novel coronavirus, called COVID-19, that was discovered in Wuhan, China at the end of last year and January of the current year. The proposed FPASSA-ANFIS model has a high ability to predict the number of confirmed cases within ten days. Besides, FPASSA-ANFIS outperforms other forecasting models in terms of RMSE, MAE, MAPE, RMSRE, and R 2 . Furthermore, two datasets of weekly influenza confirmed cases in the USA and China were used to evaluate the proposed method, and the evaluation outcomes showed its good performance. According to the promising results obtained by the proposed FPASSA-ANFIS, it can be applied in different forecasting applications.\n",
            "\t0.503\tThe virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV .The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host . Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.The symptoms of COVID-19 are fever, dry cough, fatigue, nasal congestion, sore throat and diarrhoea. On February 14th, the Chinese Center for Disease Control and Prevention (China CDC) published the first details of 44,672 confirmed cases, in the biggest study since the outbreak began . Their findings show that COVID-19 was mild for 81% of patients and had an overall case fatality rate of 2.3%. Of those confirmed cases, only 2.2% were under 20 years old. Compared to adults, children generally present with much milder clinical symptoms. It is likely that future serological studies will show much asymptomatic disease in children. As opposed to H1N1, pregnant women do not appear to be at higher risk of severe disease. The severity of the disease appears to be associated with age, with the elderly most at risk; those over 80 years of age had a Case Fatality Rate (CFR) of 14.8%. The CFR was also increased in those with comorbidities including cardiovascular, diabetes, chronic respiratory disease, hypertension, and cancer. The cause of death is respiratory failure, shock or multiple organ failure.There is no proven treatment at this early stage but we will doubtless have more information about this soon. It can be assumed that non-pharmacologic approaches are effective such as fluid support, oxygen and ventilatory support. Most recently the national data suggests that 17.7%, 10.4% and 7.0% of all cases have disease requiring respiratory support in Wuhan, Hubei (not including Wuhan) and the rest of China (respectively). About a quarter of all require ventilation while 75% require oxygen support only. The variation in severity rates probably reflects the outcomes in an overwhelmed health system. Extra Corporeal Membrane Oxygenation (ECMO) is potentially of benefit and we will know more particularly when cities with higher technology health systems become affected and ECMO is truly tested in the most severely ill. ECMO is currently being used in China, however, its effectiveness is yet to be determined. Antiviral drugs as well as a variety of other putative treatments are typically being prescribed for deteriorating patients on a compassionate basis. Clinicians would be well aware of such situations but assurance is required that their safety and efficacy are being scientifically assessed so that meaning is brought to bear quickly. Coordination of clinical trials to avoid duplication and ensure that results are rapidly available will be a challenge but the case numbers should facilitate rapid definitive results. (see What is in the pipeline for vaccine development and/or therapeutics?).As a novel virus newly emerged in humans, the world's population is completely immune-naïve and therefore vulnerable. There is clear human-to-human transmission in family clusters in China and beyond, transmission from close face-to-face social contact, especially in small enclosed spaces, and transmission from failed infection prevention and control measures in health facilities. In addition, the experience in Wuhan shows that transmission can be massive in a short period of time with thousands of new patients diagnosed daily.The current aim of the global response is to flatten the epidemic curve so that transmission is slowed, and to interrupt transmission where possible. While there is clearly a mortality linked to the virus, the most concerning problem will be if a health system is overwhelmed in the wake of rapid transmission so that affected patients cannot receive the care they need. Furthermore, patients with other urgent medical conditions are at risk of not obtaining their necessary care. Countries with vulnerable health systems are particularly of concern.Recently, there have been outbreaks in newly affected countries including Italy and Iran where the index case is unidentified. Furthermore affected countries have very large clusters emerging such as Korea and Japan. There is no reason to believe that the global community is ready for this emerging pandemic; ready in a way that can see drastic public health intervention implemented within days, including aggressive and massive contact tracing, monitoring (or quarantine) and early detection and isolation in an attempt to slow the progression.There is uncertainty regarding transmissibility and severitymore information is emerging about the spectrum of disease, especially mild disease, which is not identified using many current case definitions, and about the ease of transmission from person to person. Health agencies are unsure how to model this and estimates vary depending on the variables being used. For instance, SARS was essentially spread later in the disease from patients with more significant clinical pictures, and it was contained by infection control measures particularly in hospitals. The limited spread to family members of health workers and the community was contained by usual outbreak control measures including early identification and management of persons with infection, tracing of contacts with monitoring for onset of fever and/or symptoms, and active engagement of communities.Most modelling suggests that the severity of illness is more like influenza than SARS, and there is concern among the public health community because the transmissibility of COVID-19 is not yet fully understood, and the potential for it to become endemic like other respiratory pathogens is unknown.Because of the many unknowns, the initial reaction by health agencies is still valid: to brace for the first wave as the virus reaches a completely naïve population and to make maximum effort to interrupt transmission [4, 5] .Experience with managing this outbreak will be very heterogenous across the world. Countries closely connected with China, such as Singapore, will be ahead in this regard. As the outbreak moves across regions, there is opportunity to support those affected later both in terms of readiness and response. Mechanisms available to outbreak response organisations, particularly through the Global Outbreak Alert and Response Network (GOARN), can be valuable in skills-and knowledgesharing. Therefore, there should be a deliberate effort to utilise knowledge from early affected countries in later affected countries.What measures are likely to be successful in curbing its further spread?On January 23rd, Hubei province in Central China was locked down with all movement in and out of the province blocked. Travel across China was discouraged and the number of scheduled flights and train journeys available considerably reduced to perhaps 10% of previous activity. Commercial and social activities became negligible, with schools, restaurants, other entertainment spots and most shops closed. Migrant workers were prevented from returning to work after the extended Chinese New Year Holiday. Frequent hand hygiene when in public and staying at home became the norm. This unprecedented public health effort by China has afforded the rest China and the world a lead time to prepare. The lockdown resulted in a downward trend in national and provincial epidemic curves, however, these measures are not sustainable and eventually there will be a strategy to return to normality. Should this result in a second wave of cases, and more international spread, countries around the world must be prepared. Early identification, and isolation or cohorting of positive cases to designated sites is at the core. In order to achieve this, hospitals, quarantine services, laboratories, together with epidemiology and communication teams will need to be scaled up to provide effective and efficient care.What is in the pipeline for vaccine development and/or therapeutics?It appears that no vaccine will be available for at least one year, likely a little longer. Phase 1 trials for safety and immunogenicity in human populations are likely within 3 months.In terms of therapeutics there is no known effective pharmaceutical agent. There are over 200 registered clinical trials registered in China alone. Putative agents include antivirals; Griffithsin, a spike protein inhibitor, nucleoside analogues eg. remdesivir, ribavirin and protease inhibitors such as lopinavir/ritonavir. Immunomodulatory and other host targeted agents include interferon, chloroquine and immunoglobulins. Corticosteroids will potentially have benefit for immune mediated lung damage late in the course of disease . Much of the theory stems from what we have learnt from limited trials in other corona viruses .\n",
            "\t0.491\tTraveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015) . Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens. Gostic et al. eLife 2020;9:e55570. DOI:  1 of 18 RESEARCH ADVANCE Gostic et al. eLife 2020;9:e55570.As of February 20, 2020, the 2019 novel coronavirus (now named SARS-CoV-2, causing the disease COVID-19) has caused over 75,000 confirmed cases inside of China and has spread to 25 other countries (World Health Organization, 2020b) . (HCoV-19 has been proposed as an alternate name for the virus; Jiang et al., 2020) . Until now, local transmission remained limited outside of China, but as of this week, new epidemic hotspots have become apparent on multiple continents (World Health Organization, 2020a; Jankowicz, 2020; Sang-Hun, 2020; Schnirring, 2020a) . Many jurisdictions have imposed traveller screening in an effort to prevent importation of COVID-19 cases to unaffected areas. Some high-income countries have escalated control measures beyond screening-based containment policies, and now restrict or quarantine inbound travellers from countries known to be experiencing substantial community transmission. Meanwhile, in many other countries, screening remains the primary barrier to case importation (Guardian reporting team, 2020; Schengen Visa Info, 2020) . Even in countries with the resources to enforce quarantine measures, expanded arrival screening may be the first logical response as the source epidemic expands to regions outside China. Furthermore, symptom screening has become a ubiquitous tool in the effort to contain local spread of COVID-19, in settings from affected cities to cruise ships to quarantines. Our analysis is pertinent to all of these contexts.It is widely recognized that screening is an imperfect barrier to spread (Bitar et al., 2009; Cowling et al., 2010; Gostic et al., 2015; Mabey et al., 2014; Quilty et al., 2020) , due to: the absence of detectable symptoms during the incubation period; variation in the severity and detectability of symptoms once the disease begins to progress; imperfect performance of screening equipment or personnel; or active evasion of screening by travellers. Previously we estimated the effectiveness of traveller screening for a range of pathogens that have emerged in the past, and found that arrival screening would miss 50-75% of infected cases even under optimistic assumptions (Gostic et al., 2015) . Yet the quantitative performance of different policies matters for planning interventions and will influence how public health authorities prioritize different measures as the international and domestic context changes. Here we use a mathematical model to analyse the expected performance of different screening measures for , based on what is currently known about its natural history and epidemiology and on different possible combinations of departure and arrival screening policies.First we assess the probability that any single individual infected with SARS-CoV-2 would be detected by screening, as a function of time since exposure. This individual-level analysis is not a comprehensive measure of program success, but serves to illustrate the various ways in which screening can succeed or fail (and in turn the ways it can or cannot be improved). Further, these analyses emphasize the importance of the incubation period, and the fraction of subclinical cases, as determinants of individual screening outcomes. We define subclinical cases as those too mild to show symptoms detectable in screening (fever or cough) after passing through the incubation period (i.e. once any symptoms have manifested). The true fraction of subclinical COVID-19 cases remains unknown, but anecdotally, many lab-confirmed COVID-19 cases have not shown detectable symptoms on diagnosis (Hoehl et al., 2020; Nishiura et al., 2020; . About 80% of clinically attended cases have been mild (The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020), and clinically attended cases have been conspicuously rare in children and teens The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Huang et al., 2020; , raising the possibility that subclinical cases may be common.Next, we assess the overall effectiveness of a screening program by modeling screening outcomes in a hypothetical population of infected travellers, each with a different time since exposure (and hence a different probability of having progressed through incubation to show detectable symptoms). Crucially, the distribution of times since exposure will depend on the epidemiology of the source population, so this overall measure is not a simple average of the individual-level outcomes. We estimate the fraction of infected travellers detected, breaking down the ways in which screening can succeed or fail. An alternate measure of program success is the extent to which screening delays the first importation of cases into the community, possibly providing additional time to train medical staff, deploy public health responders or refine travel policies (Cowling et al., 2010) . To quantify the potential for screening to delay case importation, we estimate the probability that a given screening program would detect the first n or more imported cases before missing an infected person.Screening will be less effective in a growing epidemic, due to an excess of recently-exposed and not-yet-symptomatic travellers (Gostic et al., 2015) . In the context of COVID-19, we consider both growing and stable epidemic scenarios, but place greater emphasis on the realistic assumption that the COVID-19 epidemic is still growing. Since late January 2020, the Chinese government has imposed strict travel restrictions and surveillance on population centers heavily affected by COVID-19 (BBC News, 2020; Cellan-Jones, 2020) , and numerous other countries have imposed travel and quarantine restrictions on travellers inbound from China. Until about Feb. 20, 2020, these measures had appeared to successfully limit community transmission outside of China, but all the while multiple factors pointed to on-going risk, including evidence that transmission is possible prior to the onset of symptoms , and reports of citizens seeking to elude travel restrictions or leaving before restrictions were in place (Ma and Pinghui, 2020; Mahbubani, 2020) . Now, in the week following Feb. 20, 2020, new source epidemics have appeared on multiple continents (World Health Organization, 2020a) , and the the risk of exportation of cases from beyond the initial travel-restricted area is growing.As the epidemic continues to expand geographically, arrival screening will likely be continued or expanded to prevent importation of cases to areas without established spread. At the same time, there is great concern about potential public health consequences if COVID-19 spreads to developing countries that lack health infrastructure and resources to combat it effectively (de Salazar et al., 2020) . Limited resources also could mean that some countries cannot implement large-scale arrival screening. In this scenario, departure screening implemented elsewhere would be the sole barrierhowever leaky -to new waves of case importation. It is also important to recognize that, owing to the lag time in appearance of symptoms in imported cases, any weaknesses in screening would continue to have an effect on known case importations for up to two weeks, officially considered the maximum incubation period (World Health Organization, 2020c) . Accordingly, we consider scenarios with departure screening only, arrival screening only, or both departure and arrival screening. The model can also consider the consequences when only a fraction of the traveller population is screened, due either to travel from a location not subject to screening, or due to deliberate evasion of screening.Our analysis also has direct bearing on other contexts where symptom screening is being used, beyond international air travel. This includes screening of travelers at rail stations and roadside spot checks, and screening of other at-risk people including people living in affected areas, health-care workers, cruise ship passengers, evacuees and people undergoing quarantine (Hoehl et al., 2020; Japan Ministry of Health, Labor and Welfare, 2020; Nishiura et al., 2020; Schnirring, 2020c) . Below, we chiefly frame our findings in terms of travel screening, but these other screening contexts are also in the scope of our analysis. Any one-off screening effort is equivalent to a departure screen (i.e. a single test with no delay), and our findings on symptom screening effectiveness over the course of infection are directly applicable to longitudinal screening in quarantine or occupational settings.The central aim of our analysis is to assess the expected effectiveness of screening for COVID-19, taking account of current knowledge and uncertainties about the natural history and epidemiology of the virus. We therefore show results using the best estimates currently available, in the hope of informing policy decisions in this fast-changing environment. We also make our model available for public use as a user-friendly online app, so that stakeholders can explore scenarios of particular interest, and results can be updated rapidly as our knowledge of this new viral threat continues to expand.The core model has been described previously (Gostic et al., 2015) , but to summarize briefly, it assumes infected travellers can be detained due to the presence of detectable symptoms (fever or cough), or due to self-reporting of exposure risk via questionnaires or interviews. These assumptions are consistent with WHO traveller screening guidelines (World Health Organization, 2020b; World Health Organization, 2020c). Upon screening, travellers fall into one of four categories: (1) symptomatic but not aware of exposure risk, (2) aware of exposure risk but without detectable symptoms, (3) symptomatic and aware that exposure may have occurred, and (4) neither symptomatic nor aware of exposure risk ( Figure 1) . Travellers in the final category are fundamentally undetectable, and travellers in the second category are only detectable if aware that they have been exposed and willing to self report.In the model, screening for symptoms occurs prior to questionnaire-based screening for exposure risk, and detected cases do not progress to the next stage. This allows us to track the fraction of cases detected using symptom screening or risk screening at arrival or departure. Additionally, building on the four detectability classes explained above, the model keeps track of four ways in which screening can miss infected travellers: (1) due to imperfect sensitivity, symptom screening may fail to detect symptoms in travellers that display symptoms; (2) questionnaires may fail to detect exposure risk in travellers aware they have been exposed, owing to deliberate obfuscation or misunderstanding; (3) screening may fail to detect both symptoms and known exposure risk in travellers who have both and (4) travellers not exhibiting symptoms and with no knowledge of their exposure are fundamentally undetectable. Here, we only consider infected travellers who submit to screening. However, the supplementary app allows users to consider scenarios in which some fraction of infected travellers intentionally evade screening ( Figure 1E ).The probability that an infected person is detectable in a screening program depends on: the incubation period (the time from exposure to onset of detectable symptoms); the proportion of subclinical cases (mild cases that lack fever or cough); the sensitivity of thermal scanners used to detect fever; the fraction of cases aware they have high exposure risk; and the fraction of those cases who would self-report truthfully on a screening questionnaire. Further, the distribution of individual times since exposure affects the probability that any single infected traveller has progressed to the symptomatic stage. If the source epidemic is still growing, the majority of infected cases will have been recently exposed, and will not yet show symptoms. If the source epidemic is no longer growing (stable), times since exposure will be more evenly distributed, meaning that more infected travellers will have progressed through incubation and will show detectable symptoms.We used methods described previously to estimate the distribution of individual times since exposure in a growing or stable epidemic, given various values of the reproductive number R 0 (Gostic et al., 2015) . Briefly, early in the epidemic when the number of cases is still growing, the model draws on epidemiological theory to assume that the fraction of cases who are recently exposed increases with R 0 . The distribution of times since exposure is truncated at a maximum value, which corresponds epidemiologically to the maximum time from exposure to patient isolation, after which point we assume cases will not attempt to travel. (Isolation may occur due to hospitalization, or due to confinement at home in response to escalating symptoms or COVID-19 diagnosis. In the non-travel context, this would correspond to cases that have been hospitalized or otherwise diagnosed and isolated.) Here, we approximate the maximum time from exposure to isolation as the sum of the mean incubation time, and mean time from onset to isolation. To consider the epidemiological context of a stable epidemic in the source population we assume times since exposure follow a uniform distribution across the time period between exposure and isolation.As of February 20, 2020, COVID-19-specific estimates are available for most parameters, but many have been derived from limited or preliminary data and remain subject to considerable uncertainty. Table 1 and the Methods summarize the current state of knowledge. Here, we used two distinct approaches to incorporate parameter uncertainty into our analysis.First, to estimate the probability that an infected individual would be detected or missed we considered a range of plausible values for the mean incubation time, and the fraction of subclinical Table 1 . Parameter values estimated in currently available studies, along with accompanying uncertainties and assumptions. Ranges in the final column reflect confidence interval, credible interval, standard error or range reported by each study referenced.Best estimate (Used in Figure 2 ) Plausible range (Used in Figure 3 )Mean incubation period 5.5 days Sensitivity: 4.5 or 6.5 Clinical data: 83% fever, 67% cough, n = 6 (Chan et al., 2020) 83% fever, 82% cough, n = 99 98% fever, 76% cough, n = 41 43.8% fever at hospital admission, 88.7% fever during hospitalization, n = 1099 Active monitoring after repatriation flights or on cruise ships: % asymptomatic at diagnosis 31.2% (111/355) (Japan Ministry of Health, Labor and Welfare, 2020) 65.2% (5 of 8) (Nishiura et al., 2020) 70.0% (7 of 10) We assume a low percentage, as no specific risk factors have been identified, and known times or sources of exposure are rarely reported in existing line lists.Sensitivity of infrared thermal scanners for fever 70% 60-90% Most studies estimated sensitivity between 60-88% (Bitar et al., 2009; Priest et al., 2011; Tay et al., 2015) . But a handful of studies estimated very low sensitivity (4-30%).In general, sensitivity depended on the device used, body area targeted and ambient temperature.Probability that travellers self-report exposure risk 25% 5-25% 25% is an upper-bound estimate based on outcomes of past screening initiatives. (Gostic et al., 2015) Time from symptom onset to patient isolation (After which we assume travel is not possible) No effect in individual-level analysis.Median 7 days from onset to hospitalization (n = 6) (Chan et al., 2020) Mean 2.9 days onset to patient isolation (n = 164) Median 7 days from onset to hospitalization (n = 41) As awareness increases, times to isolation may decline.* From family cluster. † Parametric distributions fit to cases with known dates of exposure or travel to and from Wuhan. cases. We focus on the incubation period and subclinical fraction of cases because screening outcomes are particularly sensitive to their values. All other parameters were fixed to the best available estimates listed in Table 1 . Second, we considered a population of infected travellers, each with a unique time of exposure, and in turn a unique probability of having progressed to the symptomatic stage. Here, the model used a resampling-based approach to simultaneously consider uncertainty from both stochasticity in any single individual's screening outcome, and uncertainty as to the true underlying natural history parameters driving the epidemic. Details are provided in the methods, but briefly, we constructed 1000 candidate parameter sets, drawn using Latin hypercube sampling from plausible ranges for each parameter (Table 1) . Using each parameter set, we simulated one set of screening outcomes for a population of 30 infected individuals. As of February 20, 2020, 30 approximates the maximum known number of cases imported to any single country (World Health Organization, 2020b), and thus our analysis incorporates a reasonable degree of binomial uncertainty. The actual number of infected travellers passing through screening in any given location may be higher or lower than 30, and will depend on patterns of global connectivity, and on the duration of the source epidemic (Chinazzi et al., 2020; de Salazar et al., 2020) . Finally, we analysed the sensitivity of screening effectiveness (fraction of travellers detected) to each parameter, as measured by the partial rank correlation coefficient (PRCC) (Marino et al., 2008) . The black dashed lines separate detected cases (below) from missed cases (above). Here, we assume flight duration = 24 hr, the probability that an individual is aware of exposure risk is 0.2, the sensitivity of fever scanners is 0.7, and the probability that an individual will truthfully self-report on risk questionnaires is 0.25. Table 1 Table 1 . Dots and vertical line segments show the median and central 95%, respectively. Text above each violin shows the median and central 95% fraction detected. (B) Mean fraction of travellers with each screening outcome. The black dashed lines separate detected cases (below) from missed cases (above). (C) Fraction of simulations in which screening successfully detects at least n cases before the first infected traveller is missed. Figure 3 continued on next page Individual probabilities of detection First, the model estimated the probability that any single infected individual would be detected by screening as a function of the time between exposure and the initiation of travel (Figure 2) . Incubation time is a crucial driver of traveller screening effectiveness; infected people are most likely to travel before the onset of symptoms. Here we considered three mean incubation times, which together span the range of most existing mean estimates: 4.5, 5.5 and 6.5 days ( Table 1) . Additionally, we considered three possible fractions of subclinical cases: 50% represents a worst-case upper bound, 5% represents a best-case lower bound, and 25% represents a plausible middle case. (Table 1 , Materials and methods). Since resubmission, a new delay-adjusted estimate indicates that 34.6% of infections are asymptomatic (Mizumoto et al., 2020) , intermediate between our middle and worst-case scenarios.Even within the narrow range tested, screening outcomes were sensitive to the incubation period mean. For longer incubation periods, we found that larger proportions of departing travellers would not yet be exhibiting symptoms -either at departure or arrival -which in turn reduced the probability that screening would detect these cases, especially since we assume few infected travellers will realize they have been exposed to COVID-19.A second crucial uncertainty is the proportion of subclinical cases, which lack detectable fever or cough even after the onset of symptoms. We considered scenarios in which 5%, 25% and 50% of cases are subclinical, representing a best, middle and worst-case scenario, respectively. The middle and worst-case scenarios have predictable and discouraging consequences for the effectiveness of traveller screening, since they render large fractions of the population undetectable by fever screening ( Figure 2) . Furthermore, subclinical cases who are unaware of their exposure risk are never detectable, by any means. This is manifested as the bright red 'undetectable' region which persists well beyond the mean incubation period. For a screening program combining departure and arrival screening, as shown in Figure 2 , the greatest contributor to case detection is the departure fever screen. The arrival fever screen is the next greatest contributor, with its value arising from two factors: the potential to detect cases whose symptom onset occurred during travel, and the potential to catch cases missed due to imperfect instrument sensitivity in non-contact infrared thermal scanners used in traveller screening (Table 1) . Considering the effectiveness of departure or arrival screening only (Figure 2-figures supplement 1, 2) , we see that fever screening is the dominant contributor in each case, but that the risk of missing infected travellers due to undetected fever is substantially higher when there is no redundancy from two successive screenings.Overall screening effectiveness in a population of infected travellers during a growing or stable epidemic Next we estimated the overall effectiveness of different screening programs, as well as the uncertainties arising from the current partial state of knowledge about this recently-emerged virus. We modeled plausible population-level outcomes by tracking the fraction of 30 infected travellers detained, given a growing or stable epidemic and current uncertainty around parameter values. We separately consider the best, middle and worst-case scenarios for the proportion of infections that are subclinical, and for each scenario we compare the impact of departure screening only (or equivalently, any on-the-spot screening), arrival screening only, or programs that include both.The striking finding is that in a growing epidemic, even under the best-case assumptions, with just one infection in twenty being subclinical and all travellers passing through departure and arrival screening, the median fraction of infected travellers detected is only 0.30, with 95% interval extending from 0.10 up to 0.53 ( Figure 3A) . The total fraction detected is lower for programs with only Source data 1. Source data for Figure 3A . Source data 2. Source data for Figure 3B . Source data 3. Source data for Figure 3C . one layer of screening, with arrival screening preferable to departure screening owing to the possibility of symptom onset during travel. Considering higher proportions of subclinical cases, the overall effectiveness of screening programs is further degraded, with a median of just one in ten infected travellers detected by departure screening in the worst-case scenario. The key driver of these poor outcomes is that even in the best-case scenario, nearly two thirds of infected travellers will not be detectable (as shown by the red regions in Figure 3B ). There are three drivers of this outcome: (1) in a growing epidemic, the majority of travellers will have been recently infected and hence will not yet have progressed to exhibit any symptoms; (2) we assume that a fraction of cases never develop detectable symptoms; and (3) we assume that few people are aware of their exposure risk. As above, the dominant contributor to successful detections is fever screening.In an epidemic that is no longer growing (Figure 3-figures supplement 1) , screening effectiveness is considerably higher, as a lower proportion of travellers will be recently exposed. This is shown by the smaller, red 'undetectable' region in Figure 3 -figures supplement 1B. In a stable epidemic, under the middle-case assumption that 25% of cases are subclinical, we estimate that arrival screening alone would detect roughly one third (17-53%) of infected travelers, and that a combination of arrival and departure screening would detect nearly half (23-63%) of infected travellers. In short, holding all other things equal, screening effectiveness will increase as the source epidemic transitions from growing to stable, owing simply to changes in the distribution of 'infection ages,' or times since exposure.To assess the potential for screening to delay introduction of undiagnosed cases, we evaluated the fraction of simulations in which screening during a growing epidemic would detect the first n or more infected travellers ( Figure 3C ). Depending on the screening strategy (arrival, departure or both) and assumed subclinical fraction (5%, 25%, or 50%), the probability of detecting at least the first two cases ranged from 0.02 to 0.11, and the probability of detecting three or more cases was never better than 0.04 ( Figure 3C ). In all tested scenarios, more than half of simulations failed to detect the first imported case, consistent with probabilities of case detection in Figure 3A . Probabilities of detecting the first n consecutive cases were marginally higher in the stable epidemic context (Figure 3-figures supplement 1) , but still the probability of detecting at least the first three cases was never better than 0.13, and the probability of detecting the first four cases was never better than 0.06 in any tested scenario. Taken together, these results indicate that screening in any context is very unlikely to delay case importation beyond the first 1-3 cases, and often will not delay the first importation at all. What duration of delay this yields will depend on the frequency of infected travellers.In the context of a growing epidemic, sensitivity analysis using the method of Latin hypercube sampling and partial rank correlation (Marino et al., 2008) showed that the fraction of travellers detected was moderately sensitive to all parameters considered -most coefficient estimates fell between 0.1 and 0.3 in absolute value ( Figure 4 ). Sensitivity to R 0 was somewhat higher than sensitivity to other parameters, but the difference was not statistically remarkable. R 0 and the mean incubation period were negatively associated with the fraction of cases detected. An increase in either of these parameters implies an increase in the probability an infected traveller will be undetectable, either because they have been recently exposed (R 0 ), or have not yet progressed to the symptomatic stage (mean incubation time). The positive association between the fraction of cases detected and the sensitivity of thermal scanners, sensitivity of risk questionnaires, or the fraction of travellers aware of exposure risk is intuitive. Finally, the duration from onset to isolation effectively describes the window of time in which we assume a symptomatic individual could initiate travel. Here, a wider window is associated with increased screening effectiveness, because it will lead to a higher proportion of infected travellers who are symptomatic. Figure 4 shows results from the middle case scenario, in which 25% of cases are subclinical. Considering scenarios where more or fewer cases are subclinical, we see increased influence of the factors based on exposure risk (questionnaire sensitivity and the fraction of cases aware of their exposure) as the proportion of cases with detectable symptoms declines (Figure 4-figures supplement 1) .In the context of a stable epidemic, a greater proportion of infected travellers will have progressed to show detectable symptoms, and so screening effectiveness was more sensitive to parameters that impact symptom screening efficacy (thermal scanner sensitivity, and to the time from symptom onset to isolation). Note that by construction, model outcomes are insensitive to parameter R 0 in the stable epidemic context. As a result, R 0 coefficient estimates are very small (non-zero due to stochasticity in simulation outcomes), and never significant. (Figure 4 -figures supplement 2).We have developed an interactive web application using the R package Shiny (Chang et al., 2019) in which users can replicate our analyses using parameter inputs that reflect the most up-do-date information. The supplementary user interface can be accessed at . Please note that while the results in Figures 3 and 4 consider a range of plausible values for each parameter, the outputs of the Shiny app are calculated using fixed, userspecified values only.The international expansion of COVID-19 cases has led to widespread adoption of symptom and risk screening measures, in travel-associated and other contexts, and programs may still be adopted or expanded as source epidemics of COVID-19 emerge in new geographic areas. Using a mathematical model of screening effectiveness, with preliminary estimates of COVID-19 epidemiology and natural history, we estimate that screening will detect less than half of infected travellers in a growing epidemic, and that screening effectiveness will increase marginally as growth of the source epidemic decelerates. We found that two main factors influenced the effectiveness of screening. First, symptom screening depends on the natural history of an infection: individuals are increasingly likely to show detectable symptoms with increasing time since exposure. A fundamental shortcoming of screening is the difficulty of detecting infected individuals during their incubation period, or early after the onset of symptoms, at which point they still feel healthy enough to undertake normal activities or travel. This difficulty is amplified when the incubation period is longer; infected individuals have a longer window in which they may mix socially or travel with low probability of detection. Second, screening depends on whether exposure risk factors exist that would facilitate specific and reasonably sensitive case detection by questionnaire. For COVID-19, there is so far limited evidence for specific risk factors; we therefore assumed that at most 40% of travellers would be aware of a potential exposure. It is plausible that many individuals aware of a potential exposure would voluntarily avoid travel and practice social distancing-if so, the population of infected travellers will be skewed toward those unaware they have been exposed. Furthermore, based on screening outcomes during the 2009 influenza pandemic, we assumed that a minority of infected travellers would selfreport their exposure honestly, which led to limited effectiveness in questionnaire-based screening. The confluence of these two factors led to many infected people being fundamentally undetectable in our model. Even under our most generous assumptions about the natural history of COVID-19, the presence of undetectable cases made the greatest contribution to screening failure. Correctable failures, such as missing an infected person with fever or awareness of their exposure risk, played a more minor role.Our conclusion that screening would detect no more than half of infected travellers in a growing epidemic is consistent with recent studies that have compared country-specific air travel volumes with detected case counts to estimate that roughly two thirds of imported cases remain undetected Bhatia et al., 2020) . Furthermore, the finding that the majority of cases missed by screening are fundamentally undetectable is consistent with observed outcomes so far. Analyzing a line list of 290 cases imported into various countries , we found that symptom onset occurred after the date of inbound travel for 72% (75/104) of cases for whom both dates were available, and a further 14% (15/104) had symptom onset on the date of travel. Even among passengers of repatriation flights, or quarantined on a cruise ship off the coast of Japan (who are all demonstrably at high risk), numerous cases have been undetectable in symptom screening, but have still tested positive for SARS-CoV-2 by PCR Hoehl et al., 2020; Japan Ministry of Health, Labor and Welfare, 2020; Nishiura et al., 2020; . The onset of viral shedding prior to the onset of symptoms, or in cases that remain asymptomatic, is a classic factor that makes infectious disease outbreaks difficult to control (Fraser et al., 2004) .Our results emphasize that the true fraction of subclinical cases (those who lack fever or cough at symptom onset) remains a crucial unknown, and strongly impacts screening effectiveness. Reviewing data from active surveillance of passengers on cruise ships or repatriation flights, we estimate that up to half of cases show no detectable symptoms at the time of diagnosis. To complicate matters further, the fraction of subclinical cases may vary across age groups. Children and young adults have been conspicuously underrepresented, even in very large clinical data sets The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Huang et al., 2020; . Only 2.1% of the first 44,672 confirmed cases were observed in children under 20 years of age (The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020). The possibility cannot be ruled out that large numbers of subclinical cases are occurring in young people. If an age-by-severity interaction does indeed exist, then the mean age of travellers should be taken into account when estimating screening effectiveness.There are some limitations to our analysis. Parameter values for COVID-19 may be affected by bias or censoring, particularly in the early stages of an outbreak when most cases have been recently infected, and when severe or hospitalized cases are overrepresented in the available data. In particular, the tail of the incubation period distribution is difficult to characterize with precision using limited or biased data.As country-specific screening policies can change rapidly in real-time, we focused on a general screening framework rather than specific case studies. We also assumed traveller adherence and no active evasion of screening. However, there are informal reports of people taking antipyretics to beat fever screening (Mahbubani, 2020) , which would further reduce the effectiveness of these methods. With travel restrictions in place, individuals may also take alternative routes (e.g. road rather than air), which would in effect circumvent departure and/or arrival screening as a control measure. Our quantitative findings may overestimate screening effectiveness if many travellers evade screening.Our results have several implications for the design and implementation of traveller screening policies. If the infection is not yet present in a region, then arrival screening could delay the introduction of cases, but consistent with previous analyses, (Cowling et al., 2010) , our results indicate such delays would be minimal. Our findings indicate that for every case detected by travel screening, one or more infected travellers were not caught, and must be found and isolated by other means. We note that even with high R 0 and no control measures in place, a single case importation is not guaranteed to start a sustained chain of transmission (Kucharski et al., 2020; Lloyd-Smith et al., 2005) . This is particularly true for infections that exhibit a tendency toward superspreading events, as increasingly reported for COVID-19, but the flipside is that outbreaks triggered by superspreading are explosive when they do occur (Lloyd-Smith et al., 2005) .We did not analyze second-order benefits from screening, such as potential to raise awareness. Official recommendations emphasize that screening is an opportunity for 'risk communication' in which travellers can be instructed how to proceed responsibly if symptoms develop at the destination (World Health Organization, 2020d). Alongside increased general surveillance/alertness in healthcare settings, such measures could help reduce the risk of local transmission and superspreading, but their quantitative effectiveness is unknown. Once limited local transmission has begun, arrival screening could still have merit as a means to restrict the number of parallel chains of transmission present in a country. Once there is generalized spread which has outpaced contact tracing, departure screening to prevent export of cases to new areas will be more valuable than arrival screening to identify additional incoming cases. Altogether, screening should not be viewed as a definitive barrier to case importation, but used alongside on-the-ground response strategies that help reduce the probability that any single imported case spreads to cause a self-sustaining local epidemic. The cost-benefit tradeoff for any screening policy should be assessed in light of past experiences, where few or no infected travellers have been detected by such programs (Gostic et al., 2015) .While our findings indicate that the majority of screening failures arise from undetectable cases (i.e. those without symptoms or knowledge of their exposure), several factors could potentially strengthen the screening measures described here. With improved efficiency of thermal scanners or other symptom detection technology, we would expect a smaller difference between the effectiveness of arrival-only screening and combined departure and arrival screening in our analysis. Alternatively, the benefits of redundant screening (noted above for programs with departure and arrival screens) could be gained in a single-site screening program by simply having two successive feverscreening stations that travellers pass through (or taking multiple measurements of each traveller at a single station). As risk factors become better known, questionnaires could be refined to identify more potential cases. Alternatively, less stringent definition of high exposure risk (e.g. contact with anyone with respiratory symptoms) would be more sensitive, but at the expense of large numbers of false positives detained, especially during influenza season.The availability of rapid PCR tests would also be beneficial for case identification at arrival, and would help address concerns with false-positive detections by screening. If such tests were fast, there may be potential to test suspected cases in real time based on questionnaire responses, travel origin, or borderline symptoms; at least one PCR test for SARS-CoV-2 claimed to take less than an hour has already been announced (Biomeme, 2020) . However, such measures could prove highly expensive if implemented at scale. There is also scope for new tools to improve the ongoing tracking of travellers who pass through screening, such as smartphone-based self-reporting of temperature or symptoms in incoming cases . Smartphone or diary-based surveillance would be cheaper and more scalable than intense, on-the-ground follow-up, but is likely to be limited by user adherence.Our analysis underscores the reality that respiratory viruses are difficult to detect by symptom and risk screening programs, particularly if a substantial fraction of infected people show mild or indistinct symptoms, if incubation periods are long, and if transmission is possible before the onset of symptoms. Quantitative estimates of screening effectiveness for COVID-19 will improve as more is learned about this recently-emerged virus, and will vary with the precise design of screening programs. However, we present a robust qualitative finding: in any situation where there is widespread epidemic transmission in source populations from which travellers are drawn, travel screening programs can slow (marginally) but not stop the importation of infected cases. Screening programs implemented in other settings will face the same challenges. By decomposing the factors leading to success or failure of screening efforts, our work supports decision-making about program design, and highlights key questions for further research. We hope that these insights may help to mitigate the global impacts of COVID-19 by guiding effective decision-making in both high-and low-resource countries, and may contribute to prospective improvements in screening policy for future emerging infections.The model's structure is summarized above (Figure 1) , and detailed methods have been described previously (Gostic et al., 2015) . Here, we summarize relevant extensions, assumptions and parameter inputs.Our previous model tracked all the ways in which infected travellers can be detected by screening (fever screen, or risk factor screen at arrival or departure). Here, we additionally keep track of the many ways in which infected travellers can be missed (i.e. missed given fever present, missed given exposure risk present, missed given both present, or missed given undetectable). Cases who have not yet passed the incubation period are considered undetectable by fever screening, even if they will eventually develop symptoms in the future. In other words, no traveller is considered 'missed given fever present' until they have passed the incubation period and show detectable symptoms. Infected travellers who progress to symptoms during their journey are considered undetectable by departure screening, but detectable by arrival screening.Additionally, we now provide a supplementary user interface, which allows stakeholders to test input parameters of interest using up-to-date information. Here, in addition to the analyses presented in this study, we implemented the possibility that some fraction of infected travellers deliberately evade screening.Existing point estimates for R 0 span a wide range (2.2-6.47), but most fall between 2.0 and 4.0 ( Table 1) . The vast majority of these estimates are informed by data collected very early in the outbreak, before any control measures were in place. However, several studies already demostrate decreases in the reproductive number over time, as a consequence of social distancing behaviors, and containment measures (Kucharski et al., 2020; Liu et al., 2020) . Realistically, R 0 will vary considerably over time, and across locations, depending on the social context, resource availability, and containment policies. Our analysis considers a plausible range of R 0 values spanning 1.5-4.0, with 4.0 representing a plausible maximum in the absence of any behavioral changes or containment efforts, and 1.5 reflecting a plausible lower bound, given containment measures may already be in place at the time of introduction.To estimate the upper-bound fraction of subclinical cases, we draw on data from active surveillance of passengers quarantined on a cruise ship off the coast of Japan, or passengers of repatriation flights. These data show that 50-70% of cases are asymptomatic at the time of diagnosis Nishiura et al., 2020; Schnirring, 2020c; Schnirring, 2020b) . We estimate that 50% subclinical cases is a reasonable upper bound: due to intensive monitoring, cases in repatriated individuals or in cruise ship passengers will be detected earlier than usual in the course of infection-and possibly before the onset of symptoms. From clinical data (where severe cases are likely overrepresented), we estimate a lower bound of 5%: even among clinically attended cases, 2-15% lack fever or cough, and would be undetectable in symptom screening (Chan et al., 2020; Chen et al., 2020; The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Huang et al., 2020) . In addition to the upper and lower bound scenarios, we consider a plausible middle-case scenario in which 25% of cases are subclinical. A very recent delay-adjusted estimate indicates 30-40% of infections on the cruise ship quarantined off the coast of Japan are asymptomatic, so the truth may fall somewhere between our middle and worst-case scenarios (Mizumoto et al., 2020) .We use a gamma distribution to model individual incubation times. We choose this form over the Weibull and lognormal distribution for ease of interpretation (gamma shape and scale parameters can be transformed easily to mean and standard deviation). So far, best-fit gamma distributions to COVID-19 data have had mean 6.5 and standard deviation 2.6 (Backer et al., 2020) , or mean 5.46 and standard deviation 1.94 (Lauer et al., 2020) . Here, to model uncertainty around the true mean incubation time, we fix the standard deviation to 2.25 (intermediate between the two existing estimates), and allow the mean to vary between 4.5 and 6.5 days (Figure 2-figure supplement 2) . The 95th percentile of the distributions we consider fall between 8.7 and 10.6 days, slightly below the officially accepted maximum incubation time of 14 days, and consistent with existing estimates ( Table 1; Backer et al., 2020; Lauer et al., 2020) .The probability that an infected traveller is detectable using questionnaire-based screening for exposure risk will be highest if risk factors with high sensitivity and specificity are known. Currently, official guidance recommends asking whether travellers have visited a country with epidemic transmission, a healthcare facility with confirmed cases, or had close contact with a confirmed or suspected case (World Health Organization, 2020c). Given the relative anonymity of respiratory transmission, we assume that a minority of infected travellers would realize that they have been exposed before symptoms develop (20% in Figure 2 , range 5-40% in Figure 3 ). Further, relying on a previous upper-bound estimate (Gostic et al., 2015) we assume that only 25% of travellers would self-report truthfully if aware of elevated exposure risk. Table 1 summarizes the state of knowledge about additional parameters, as of February 20, 2020.All code and source data used to perform analyses and generate figures is archived at https:// . (Gostic, 2020 ; copy archived at https:// ). The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication. There are no data inputs into our model. All parameter input values are specified in Table 1 , or in the manuscript text. We provide a link to the github repository containing all code necessary to run the analyses and generate figures (, copy archived at ).\n",
            "\t0.472\tBackground As the outbreak of coronavirus disease 2019 (COVID-19) progresses, epidemiological data are needed to guide situational awareness and intervention strategies. Here we describe efforts to compile and disseminate epidemiological information on COVID-19 from news media and social networks.As the outbreak of coronavirus disease 2019 (COVID- 19) is rapidly expanding in China and beyond, with the potential to become a worldwide pandemic, 1 real-time analyses of epidemiological data are needed to increase situational awareness and inform interventions. 2 Previously, real-time analyses have shed light on the transmissibility, severity, and natural history of an emerging pathogen in the first few weeks of an outbreak, such as with severe acute respiratory syndrome (SARS), the 2009 influenza pandemic, and Ebola. Analyses of detailed line lists of patients are particularly useful to infer key epidemiological parameters, such as the incubation and infectious periods, and delays between infection and detection, isolation, and reporting of cases. 3, 4 However, official individual patient data rarely become publicly available early on in an outbreak, when the information is most needed.Building on our previous experience collating news reports to monitor transmission of Ebola virus, 7 here we present an effort to compile individual patient information and subnational epidemic curves on COVID-19 from a variety of online resources. Data were made publicly available in real time and were used by the infectious disease modelling community to generate and compare epidemiological estimates relevant to interventions. We describe the data generation process and provide an early analysis of age patterns of COVID-19, case counts across China and inter nationally, and delays between symptom onset, admissions to hospital, and reporting, for cases reported until Jan 31, 2020.In this population-level observational study, we used crowdsourced reports from DXY.cn, a social network for Chinese physicians, health-care professionals, pharmacies, and health-care facilities established in 2000. This online platform is providing real-time coverage of the COVID-19 outbreak in China, obtained by collating and curating reports from news media, government television, and national and provincial health agencies. The information reported includes time-stamped cumulative counts of COVID-19 infections, outbreak maps, and realtime streaming of health authority announcements in Chinese (directly or through state media). 8 Every report is linked to an online source, which can be accessed for more detailed information on individual cases.These are publicly available, de-identified patient data reported directly by public health authorities or by state media. No patient consent was needed and no ethics approval was required.We closely monitored updates on DXY.cn between Jan 20, 2020, and Jan 31, 2020, to extract key information on individual patients in near real-time, and reports of daily case counts. For individual-level patient data, we used descriptions from the original source in Chinese to retrieve age, sex, province of identification, travel history, reporting date, dates of symptom onset and seeking care at a hospital or clinic, and discharge status, when available. Individual-level patient data were formatted into a line-list database for further quantitative analysis. Individual-level patient data were entered from DXY.cn by a native Chinese speaker (KS), who also generated an English summary for each patient. Entries were checked by a second person (JC). Since DXY.cn primarily provides Evidence before this study An outbreak of coronavirus disease 2019 (COVID-19) was recognised in early January, 2020, in Wuhan City, Hubei province, China. The new virus is thought to have originated from an animal-to-human spillover event linked to seafood and live-animal markets. The infection has spread locally in Wuhan and elsewhere in China, despite strict intervention measures implemented in the region where the infection originated on Jan 23, 2020. More than 500 patients infected with COVID-19 outside of mainland China have been reported between Jan 1 and Feb 14, 2020. Although laboratory testing for COVID-19 quickly ramped up in China and elsewhere, information on individual patients remains scarce and official datasets have not been made publicly available. Patient-level information is important to estimate key time-to-delay events (such as the incubation period and interval between symptom onset and visit to a hospital), analyse the age profile of infected patients, reconstruct epidemic curves by onset dates, and infer transmission parameters. We searched PubMed for publications between Jan 1, 1990, and Feb 6, 2020, using combinations of the following terms: (\"coronavirus\" OR \"2019-nCoV\") AND (\"line list\" OR \"case description\" OR \"patient data\") AND (\"digital surveillance\" OR \"social media\" OR \"crowd-sourced data\"). The search retrieved one relevant study on Middle East respiratory syndrome coronavirus that mentioned FluTrackers in their discussion, a website that aggregates epidemiological information on emerging pathogens. However, FluTrackers does not report individual-level data on COVID-19.To our knowledge, this is the first study that uses crowdsourced data from social media sources to monitor the COVID-19 outbreak. We searched DXY.cn, a Chinese health-care-oriented social network that broadcasts information from local and national health authorities, to reconstruct patient-level information on COVID-19 in China. We also queried international media sources and national health agency websites to collate data on international exportations of COVID-19. We describe the demographic characteristics, delays between symptom onset, seeking care at a hospital or clinic, and reporting for 507 patients infected with COVID-19 reported until Jan 31, 2020. The overall cumulative progression of the outbreak is consistent between our line list and an official report published by the Chinese national health authorities on Jan 28, 2020. The estimated incubation period in our data aligns with that of previous work. Our dataset was made available in the public domain on Jan 21, 2020.Crowdsourced line-list data can be reconstructed from social media data, especially when a central resource is available to curate relevant information. Public access to line lists is important so that several teams with different expertise can provide their own insights and interpretations of the data, especially in the early phase of an outbreak when little information is available. Publicly available line lists can also increase transparency. The main issue with the quality of patient-level data obtained during health emergencies is the potential lack of information from locations overwhelmed by the outbreak (in this case, Hubei province and other provinces with weaker health infrastructures). Future studies based on larger samples of patients with COVID-19 could explore in more detail the transmission dynamics of the outbreak in different locations, the effectiveness of interventions, and the demographic factors driving transmission.For an example of an online source see  ncovh5/view/pneumonia information on patients reported in China, we also compiled additional information on internationally exported cases of COVID-19. We obtained data for 21 countries outside of mainland China (Australia, Cambodia, Canada, France, Germany, Hong Kong, India, Italy, Japan, Malaysia, Nepal, Russia, Singapore, South Korea, Sri Lanka, Taiwan, Thailand, United Arab Emirates, the UK, the USA, and Vietnam). We gathered and cross-checked data for infected patients outside of China using several sources, including global news media (Kyodo News, Straits Times, and CNN), official press releases from each country's Ministry of Health, and disease control agencies.In addition to detailed information on individual patients, we reconstructed the daily progression of reported patients in each province of China from Jan 13, until Jan 31, 2020. We used the daily outbreak situation reports com municated by provincial health authorities, covered by state television and media, and posted on DXY.cn. All patients in our databases had a laboratory confirmed SARS coronavirus 2 (SARS-CoV-2) infection.Our COVID-19 database was made publicly available as a Google Sheet, disseminated via Twitter on Jan 21, 2020, and posted on the website of Northeastern University, (Boston, MA, USA) on Jan 24, 2020, where it is updated in real time. Data used in this analysis, frozen at Jan 31, 2020, are available online as a spreadsheet.We assessed the age distribution of all patients with COVID-19 by discharge status. We adjusted the age profile of Chinese patients by the population of China. We used 2016 population estimates from the Institute for Health Metrics and Evaluation 9 to calculate the relative risk (RR) of infection with COVID-19 by age group. To calculate the RR, we followed the method used by Lemaitre and colleagues 10 to explore the age profile of influenza, where RR for age group i is defined as where C i is the number of cases in age group i and N i is the population size of age group i.To estimate trends in the strength of case detection and interventions, we analysed delays between symptom onset and visit to a health-care provider, at a hospital or clinic, and from seeking care at a hospital or clinic to reporting, by time period and location. We considered the period before and after Jan 18, 2020, when media attention and awareness of the outbreak became more pronounced. 11 We used non-parametric tests to assess differences in delays between seeking care at a hospital or clinic and reporting between locations (Wilcoxon test to compare two locations and Kruskall-Wallis test to compare three or more locations).We estimated the duration of the incubation period on the basis of our line list data. We analysed a subset of patients returning from Wuhan who had spent less than a week in Wuhan, to ensure a narrowly defined exposure window. The incubation period was estimated as the midpoint between the time spent in Wuhan and the date of symptom onset.We did all analyses in R (version 3.5.3). We considered p values of less than 0·05 to be significant.The funder had no role in study design, data compilation, data analysis, data interpretation, or writing of the report. All authors had access to the data, and had final responsibility for the decision to submit for publication.Our line list comprised 507 patients reported from Jan 13, to Jan 31, 2020, including 364 (72%) from mainland China and 143 (28%) from outside of China (table) The age distribution of COVID-19 cases was skewed towards older age groups with a median age of 45 years (IQR 33-56) for patients who were alive or who had an unknown outcome at the time of reporting (figure 1). The median age of patients who had died at the time of reporting was 70 years (IQR 65-81). Few patients (13 [3%]) were younger than 15 years. Adjustment for the age demographics of China confirmed a deficit of infections among children, with a RR below 0·5 in patients younger than 15 years (figure 1). The RR measure indicated a sharp increase in the likelihood of reported COVID-19 among people aged 30 years and older.A timeline of cases in our crowdsourced patient line list is shown by date of onset in figure 2 , indicating an acceleration of reported cases by Jan 13, 2020. The outbreak progression based on the crowdsourced patient line list was consistent with the timeline published by China Center for Disease Control and Prevention (CDC) on Jan 28, 2020, 12 which is based on a more comprehensive database of more than 6000 patients with COVID-19. Since Jan 23, 2020, the cumulative number of cases has slowed down in the crowdsourced and China CDC curves (figure 2), which probably reflects the delay between disease onset and reporting. The median reporting delay was 5 days (IQR 3-8) in our data.Province-level epidemic curves are shown by reporting date in figure 3 . As of Jan 31, 2020, 16 (52%) of 30 provinces in mainland China had reported more than 100 confirmed cases. The apparent rapid growth of newly reported cases between Jan 18, and Jan 31, 2020, in several provinces outside of Hubei province is consistent with sustained local transmission.Across the study period, the median delay between symptom onset and seeking care at a hospital or clinic was 2 days (IQR 0-5 days) in mainland China ( figure 4 ). This delay decreased from 5 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p=0·0009). Some provinces, such as Tianjin and Yunnan had shorter delays (data by province not shown), while the early cases from Hubei province were characterised by longer delays in seeking care (median 0 days [IQR 0-1]).The median delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 2-5 days) in mainland China and decreased from 9 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p<0·0001; figure 4 ). Similarly to delays in seeking care at a hospital or clinic, reporting was quickest in Tianjin and Yunnan (median 1 day [IQR 0-1]) and slowest in Hubei province (median 12 days [IQR 7-16]).The median delay between symptom onset and seeking care at a hospital or clinic was 1 day (IQR 0-3) for international travellers, and shorter than for patients in Hubei province or the rest of mainland China (Kruskal-Wallis test p<0·0001; figure 4) . Even in the period after Jan 18, 2020, when awareness of the outbreak increased, a shorter delay between symptom onset and seeking care at a hospital or clinic was seen for international patients than for those in mainland China (Wilcoxon test p<0·0001). For international cases, the delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 1-4), also shorter than for mainland China (Wilcoxon test p<0·0001; figure 4) .On the basis of 33 patients with a travel history to Wuhan, we estimated the median incubation period for COVID-19 to be 4·5 days (IQR 3·0-5·5; appendix p 2). Information from patient line lists is crucial but difficult to obtain at the beginning of an outbreak. Here we have shown that careful compilation of crowdsourced reports curated by a long-standing Chinese medical social network provides a valuable picture of the outbreak of COVID-19 in real time. The outbreak timeline is consistent with aggregated case counts provided by health authorities. For comparison, China CDC published the first epidemic curve by symptom onset on Jan 28, 2020. 12 Line lists provide unique information on the delays between symptom onset and detection by the health-care system, reporting delays, and travel histories. This information cannot be extracted from aggregated case counts published by official sources. Line list data can help assess the effectiveness of interventions and the potential for widespread transmission beyond the initial foci of infection. In particular, shorter delays between symptom onset and admission to hospital or seeking care in a hospital or clinic accelerate detection and isolation of cases, effectively shortening the infectious period. A useful feature of our crowdsourced database was the availability of travel histories for patients returning from Wuhan, which, along with dates of symptom onset, allowed for estimation of the incubation period here and in related work. 13, 14 A narrow window of exposure could be defined for a subset of patients who had a short stay in Wuhan, at a time when the epidemic was still localised to Wuhan. Several teams have used our dataset and datasets from others to estimate a mean incubation period for COVID-19 to be 5-6 days (95% CI 2-11). Our own estimate (median 4·5 days [IQR 3·0-5·5]) is consistent with previous work that used other modelling approaches. The incubation period is a useful parameter to guide isolation and contact tracing; based on existing data, the disease status of a contact should be known with near certainty after a period of observation of 14 days. 13 Availability of a public dataset enables independent estimation of important epidemiological parameters by several teams, allowing for confirmation and cross-checking at a time when information can be conflicting and noisy.An interesting finding in our data relates to the age distribution of patients. We found a heavy skew of infection towards older age groups, with substantially fewer children infected. This pattern could indicate agerelated differences in susceptibility to infection, severe outcomes, or behaviour. However, a substantial portion of the patients in our database are travellers, a population that is usually predominantly adults (although does not exclude children). Furthermore, because patient data in our dataset were captured by the health system, they are biased towards the more severe spectrum of the disease, especially for patients from mainland China. Clinical reports have shown that severity of COVID-19 is associated with the presence of chronic conditions, 16, 17 which are more frequent in older age groups. Nevertheless, we would also expect children younger than 5 years to be at risk of severe outcomes and to be reported to the healthcare system, as is seen for other respiratory infections. 18 Biological differences could have a role in shaping these age profiles. A detailed analysis of one of the early COVID-19 clusters by Chan and colleagues 19 revealed symptomatic infections in five adult members of the same household, while a child in the same household aged 10 years was infected but remained asymptomatic, potentially indicating biological differences in the risk of clinical disease driven by age. Previous immunity from infection with a related coronavirus has been speculated to potentially protect children from SARS, 20, 21 and so might also have a role in COVID-19. In any case, if the age distribution of cases reported here was to be confirmed and the epidemic were to progress globally, we would expect an increase in respiratory mortality concentrated among people aged 30 years and older. This mortality pattern would be substantially different from the profile of the 2009 influenza pandemic, for which excess mortality was concentrated in those younger than 65 years. 21 In our dataset, we saw a rapid increase in the number of people infected with COVID-19 in several provinces of China, consistent with local transmission outside of Hubei province. As of Jan 31, 2020, province-level epidemic curves are only available by date of reporting, rather than date of symptom onset, which usually inflates recent case counts if detection has increased. D e c 9 , 2 0 1 9 D e c 1 6 , 2 0 1 9 D e c 2 3 , 2 0 1 9 D e c 3 0 , 2 0 1 9 J a n 6 , 2 0 2 0 J a n 1 3 , 2 0 2 0 J a n 2 0 , 2 0 2 0 J a n 2 7 , 2 0 2 0 Furthermore, province-level data include both returning travellers from Hubei province (ie, importations) and locally acquired cases, which also usually inflate the apparent risk of local transmission. Notably, other lines of evidence suggest that local transmission is now well established outside of Hubei province, because travel increased just before the Chinese New Year on Jan 25, 2020, and before implementation of the travel ban in Wuhan. 22 Accordingly, our own data include evidence of transmission clusters in non-travellers, with, for instance, a second-generation transmission event reported in Shaanxi on Jan 21, 2020.Our study had several limitations, one of which was the data we used. Although all provinces in mainland China provide aggregated information on infections and deaths, individual-level patient descriptions are only available for a subset of provinces. Geographical coverage is heterogeneous in our line list, and we have a notable deficit of cases from Hubei province, the foci of the COVID-19 outbreak. We expect that little patient-level information is shared on social media by province-level and city-level health authorities in Wuhan and Hubei province because health systems are overwhelmed. For similar reasons, provinces with a large total case count at the end of January, 2020, or with a weaker health infrastructure, were under-represented in our line list, with the exception of Beijing. Other limitations in our data include severity (only patients who had severe enough symptoms to seek care were captured) and changes in case definition. A series of epidemiological criteria were required for COVID-19 testing, including travel history to Wuhan within the past 2 weeks; residence in Wuhan within the past 2 weeks; contact with individuals from Wuhan (with fever and respiratory symptoms) within the past 2 weeks; and being part of an established disease cluster. Some of these criteria (eg, relation to Wuhan) were relaxed over time (appendix). As a result, we have an overrepresentation of travel-related cases in our database.The reproduction number is an important quantity for outbreak control. We refrained from estimating this parameter because reporting changes could bias estimates relying on epidemic growth rates. Furthermore, our dataset captured cases all over China and does not reflect transmission patterns in any particular location. A mean reproduction number of 2·5-2·7 has previously been estimated on the basis of the volume of importations of international cases in the pre-intervention period in Wuhan. 11 We recognise that, although our data source is useful and timely, it should not replace official statistics. Manual compilation of detailed line lists from media sources is highly time consuming and is not sustainable when case counts reach several thousands. Here we provide detailed data on 507 patients when the official case count was over 9000 by Jan 31, 2020, representing a sample of approximately 5% of reported cases and a much smaller proportion of the full spectrum of COVID-19 cases, which include mild infections. A crowd sourced system would not be expected to catch all cases, especially if many cases are too mild to be captured by the health-care system, digital surveillance, or social media. Notably, DXY.cn does not generate data outside of traditional surveillance systems but rather provides a channel of rapid communication between the public and health authorities. In turn, our approach has helped extract and repackage information from health authorities into an analytical format, which was not available elsewhere.At the time of writing, efforts are underway to coordinate compilation of COVID-19 data from online sources across several academic teams. Ultimately, we expect that a line list of patients will be shared by government sources with the global community; however, data cleaning and access issues might take a prohibitively long time to resolve. For the west African Ebola outbreak, a similarly coordinated effort to publish a line list took 2 years. 23 Given the progression of the COVID-19 outbreak, such a long delay would be counterproductive.Overall, the novelty of our approach was to rely on a unique source for social media and news reports in China, which aggregated and curated relevant information. This approach facilitated entry of robust and standard data on clinical and demographic information. Reassuringly, DXY.cn maintains a special section dedicated to debunking fake news, myths, and rumours about the COVID-19 outbreak. Looking to the future, collection of patient data in the context of emergencies could include information on whether patients are identified through contact tracing or because they seek care on their own. Furthermore, data interpretability could be improved by gathering more quantitative information on how case definitions are used in practice.In conclusion, crowdsourced epidemiological data can be useful to monitor emerging outbreaks, such as COVID-19 and, as previously, Ebola virus. 7 These efforts can help generate and disseminate detailed information in the early stages of an outbreak when little other data are available, enabling independent estimation of key parameters that affect interventions. Based on our small sample of patients with COVID-19, we note an intriguing age distribution, reminiscent of that of SARS, which warrants further epidemiological and serological studies. We also report early signs that the response is strengthening in China on the basis of a decrease in case detection time, and rapid management of travel-related infections that are identified internationally. This is an early report of a rapidly evolving situation and the parameters discussed here could change quickly. In the coming weeks, we will continue to monitor the epidemiology of this outbreak using data from news reports and official sources.KS and CV contributed to the study design. KS and JC contributed to the data compilation. KS, JC, and CV contributed to data analysis. KS and JC contributed to the design and drawing of figures. KS, JC, and CV contributed to the writing of the manuscript.We declare no competing interests.All data used in this report have been made publicly available on the Laboratory for the Modeling of Biological + Socio-technical systems website of Northeastern University. The available data include daily case counts of COVID-19 by reporting date and Chinese province, and a de-identified line list of patients with COVID-19. The line list includes geographical location (country and province), reporting date, dates of symptom onset and seeking care at a hospital or clinic, relation to Wuhan, discharge status when known, an English summary of the case description from media sources, and a link to the original source of data. Seeking care at hospital or clinic to report Symptom onset to seeking care at hospital or clinic\n",
            "\t0.454\tA.A.Q); (M.R.M.) † These authors contributed equally to this work.The ongoing outbreak of COVID-19 in the Chinese city of Wuhan (Hubei province) and its alarmingly quick transmission to 25 other countries across the world resulted in the World Health Organization (WHO) declaring a global health emergency on 30 January 2020 . This came just one month after the first reported case on 31 December 2019 . WHO, in its first emergency meeting , estimated the fatality rate of COVID-19 to be around 4%. Worldwide collaborative efforts from scientists are underway to understand the novel and rapidly spreading virus that causes this disease, SARS-CoV-2 (originally tentatively named 2019-nCoV), and to develop effective interventions for controlling and preventing it .Coronaviruses are positive-sense single-stranded RNA viruses belonging to the family Coronaviridae. These viruses mostly infect animals, including birds and mammals. In humans, they generally cause mild respiratory infections, such as those observed in the common cold. However, some recent human coronavirus infections have resulted in lethal endemics, which include the SARS (Severe Acute Respiratory Syndrome) and MERS (Middle East Respiratory Syndrome) endemics. Both of these are caused by zoonotic coronaviruses that belong to the genus Betacoronavirus within Coronaviridae. SARS-CoV originated from Southern China and caused an endemic in 2003. A total of 8098 cases of SARS were reported globally, including 774 associated deaths, and an estimated case-fatality rate of 14%-15% . The first case of MERS occurred in Saudi Arabia in 2012. Since then, a total of 2,494 cases of infection have been reported, including 858 associated deaths, and an estimated high case-fatality rate of 34.4% . While no case of SARS-CoV infection has been reported since 2004, MERS-CoV has been around since 2012 and has caused multiple sporadic outbreaks in different countries.Like SARS-CoV and MERS-CoV, the recent SARS-CoV-2 belongs to the Betacoronavirus genus . It has a genome size of~30 kilobases which, like other coronaviruses, encodes for multiple structural and non-structural proteins. The structural proteins include the spike (S) protein, the envelope (E) protein, the membrane (M) protein, and the nucleocapsid (N) protein. With SARS-CoV-2 being discovered very recently, there is currently a lack of immunological information available about the virus (e.g., information about immunogenic epitopes eliciting antibody or T cell responses). Preliminary studies suggest that SARS-CoV-2 is quite similar to SARS-CoV based on the full-length genome phylogenetic analysis [9, 12] , and the putatively similar cell entry mechanism and human cell receptor usage [9, 13, 14] . Due to this apparent similarity between the two viruses, previous research that has provided an understanding of protective immune responses against SARS-CoV may potentially be leveraged to aid vaccine development for SARS-CoV-2.Various reports related to SARS-CoV suggest a protective role of both humoral and cell-mediated immune responses. For the former case, antibody responses generated against the S protein, the most exposed protein of SARS-CoV, have been shown to protect from infection in mouse models . In addition, multiple studies have shown that antibodies generated against the N protein of SARS-CoV, a highly immunogenic and abundantly expressed protein during infection , were particularly prevalent in SARS-CoV-infected patients [19, 20] . While being effective, the antibody response was found to be short-lived in convalescent SARS-CoV patients . In contrast, T cell responses have been shown to provide long-term protection , even up to 11 years post-infection , and thus have also attracted interest for a prospective vaccine against SARS-CoV [reviewed in ]. Among all SARS-CoV proteins, T cell responses against the structural proteins have been found to be the most immunogenic in peripheral blood mononuclear cells of convalescent SARS-CoV patients as compared to the non-structural proteins . Further, of the structural proteins, T cell responses against the S and N proteins have been reported to be the most dominant and long-lasting .Here, by analyzing available experimentally-determined SARS-CoV-derived B cell epitopes (both linear and discontinuous) and T cell epitopes, we identify and report those that are completely identical and comprise no mutation in the available SARS-CoV-2 sequences (as of 21 February 2020). These epitopes have the potential, therefore, to elicit a cross-reactive/effective response against SARS-CoV-2. We focused particularly on the epitopes in the S and N structural proteins due to their dominant and long-lasting immune response previously reported against SARS-CoV. For the identified T cell epitopes, we additionally incorporated the information about the associated MHC alleles to provide a list of epitopes that seek to maximize population coverage globally, as well as in China. Our presented results can potentially narrow down the search for potent targets for an effective vaccine against SARS-CoV-2, and help guide experimental studies focused on vaccine development.A total of 120 whole genome sequences of SARS-CoV-2 were downloaded on 21 February 2020 from the GISAID database () (Table S1 ). We excluded sequences that likely had spurious mutations resulting from sequencing errors, as indicated in the comment field of the GISAID data. These nucleotide sequences were aligned to the GenBank reference sequence (accession ID: NC_045512.2) and then translated into amino acid residues according to the coding sequence positions provided along the reference sequence for SARS-CoV-2 proteins (orf1a, orf1b, S, ORF3a, E, M, ORF6, ORF7a, ORF7b, ORF8, N, and ORF10). These sequences were aligned separately for each protein using the MAFFT multiple sequence alignment program . Reference protein sequences for SARS-CoV and MERS-CoV were obtained following the same procedure from GenBank using the accession IDs NC_004718.3 and NC_019843.3, respectively.SARS-CoV-derived B cell and T cell epitopes were searched on the NIAID Virus Pathogen Database and Analysis Resource (ViPR) (; accessed 21 February 2020) by querying for the virus species name: \"Severe acute respiratory syndrome-related coronavirus\" from \"human\" hosts. We limited our search to include only the experimentally-determined epitopes that were associated with at least one positive assay: (i) Positive B cell assays (e.g., enzyme-linked immunosorbent assay (ELISA)-based qualitative binding) for B cell epitopes; and (ii) either positive T cell assays (such as enzyme-linked immune absorbent spot (ELISPOT) or intracellular cytokine staining (ICS) IFN-γ release), or positive major histocompatibility complex (MHC) binding assays for T cell epitopes. Strictly speaking, the latter set of epitopes, determined using positive MHC binding assays, are antigens which are candidate epitopes, since a T cell response has not been confirmed experimentally. However, for brevity and to be consistent with the terminology used in the ViPR database, we will not make this qualification, and will simply refer to them as epitopes in this study. The number of B cell and T cell epitopes obtained from the database following the above procedure is listed in Table 1 . Population coverages for sets of T cell epitopes were computed using the tool provided by the Immune Epitope Database (IEDB) (; accessed 21 February 2020) . This tool uses the distribution of MHC alleles (with at least 4-digit resolution, e.g., A*02:01) within a defined population (obtained from ) to estimate the population coverage for a set of T cell epitopes. The estimated population coverage represents the percentage of individuals within the population that are likely to elicit an immune response to at least one T cell epitope from the set. To identify the set of epitopes associated with MHC alleles that would maximize the population coverage, we adopted a greedy approach: (i) We first identified the MHC allele with the highest individual population coverage and initialized the set with their associated epitopes, then (ii) we progressively added epitopes associated with other MHC alleles that resulted in the largest increase of the accumulated population coverage. We stopped when no increase in the accumulated population coverage was observed by adding epitopes associated with any of the remaining MHC alleles.We used the publicly available software PASTA v1.6.4 to construct a maximum-likelihood phylogenetic tree of each structural protein using the unique set of sequences in the available data of SARS-CoV, MERS-CoV, and SARS-CoV-2. We additionally included the Zaria Bat coronavirus strain (accession ID: HQ166910.1) to serve as an outgroup. The appropriate parameters for tree estimation are automatically selected in the software based on the provided sequence data. For visualizing the constructed phylogenetic trees, we used the publicly available software Dendroscope v3.6.3 . Each constructed tree was rooted with the outgroup Zaria Bat coronavirus strain, and circular phylogram layout was used.All sequence and immunological data, and all scripts (written in R) for reproducing the results are available online .SARS-CoV-2 has been observed to be close to SARS-CoV-much more so than MERS-CoV-based on full-length genome phylogenetic analysis [9, 12] . We checked whether this is also true at the level of the individual structural proteins (S, E, M, and N). A straightforward reference-sequence-based comparison indeed confirmed this, showing that the M, N, and E proteins of SARS-CoV-2 and SARS-CoV have over 90% genetic similarity, while that of the S protein was notably reduced (but still high) ( Figure 1a ). The similarity between SARS-CoV-2 and MERS-CoV, on the other hand, was substantially lower for all proteins ( Figure 1a ); a feature that was also evident from the corresponding phylogenetic trees (Figure 1b) . We note that while the former analysis ( Figure 1a ) was based on the reference sequence of each coronavirus, it is indeed a good representative of the virus population, since few amino acid mutations have been observed in the corresponding sequence data ( Figure S1 ). It is also noteworthy that while MERS-CoV is the more recent coronavirus to have infected humans, and is comparatively more recurrent (causing outbreaks in 2012, 2015, and 2018) (), SARS-CoV-2 is closer to SARS-CoV, which has not been observed since 2004. Percentage sequence identity with SARS-CoV-2 All trees were constructed based on the available unique sequences using PASTA and rooted with the outgroup Zaria Bat CoV strain (accession ID: HQ166910.1).Given the close genetic similarity between the structural proteins of SARS-CoV and SARS-CoV-2, we attempted to leverage immunological studies of the structural proteins of SARS-CoV to potentially aid vaccine development for SARS-CoV-2. We focused specifically on the S and N proteins as these are known to induce potent and long-lived immune responses in SARS-CoV 19, 20, 25, 27] . We used the available SARS-CoV-derived experimentally-determined epitope data (see Materials and Methods) and searched to identify T cell and B cell epitopes that were identical-and hence potentially cross-reactive-across SARS-CoV and SARS-CoV-2. We first report the analysis for T cell epitopes, which have been shown to provide a long-lasting immune response against SARS-CoV , followed by a discussion of B cell epitopes.The SARS-CoV-derived T cell epitopes used in this study were experimentally-determined from two different types of assays : (i) Positive T cell assays, which tested for a T cell response against epitopes, and (ii) positive MHC binding assays, which tested for epitope-MHC binding. We aligned these T cell epitopes across the SARS-CoV-2 protein sequences. Among the 115 T cell epitopes that were determined by positive T cell assays (Table 1) , we found that 27 epitope-sequences were identical within SARS-CoV-2 proteins and comprised no mutation in the available SARS-CoV-2 sequences (as of 21 February 2020) ( Table 2) . Interestingly, all of these were present in either the N (16) or S (11) protein. MHC binding assays were performed for 19 of these 27 epitopes, and these were reported to be associated with only five distinct MHC alleles (at 4-digit resolution): HLA-A*02:01, HLA-B*40:01, HLA-DRA*01:01, HLA-DRB1*07:01, and HLA-DRB1*04:01. Consequently, the accumulated population coverage of these epitopes (see Materials and Methods for details) is estimated to not be high for the global population (59.76%), and was quite low for China (32.36%). For the remaining 8 epitopes, since the associated MHC alleles are unknown, they could not be used in the population coverage computation. Additional MHC binding tests to identify the MHC alleles that bind to these 8 epitopes may reveal additional distinct alleles, beyond the five determined so far, that may help to improve population coverage. To further expand the search and identify potentially effective T cell targets covering a higher percentage of the population, we next additionally considered the set of T cell epitopes that have been experimentally-determined from positive MHC binding assays (Table 1) , but, unlike the previous epitope set, their ability to induce a T cell response against SARS-CoV was not experimentally determined. Nonetheless, they also present promising candidates for inducing a response against SARS-CoV-2. For the expanded set of epitopes, all of which have at least one positive MHC binding assay, we found that 229 epitope-sequences have an identical match in SARS-CoV-2 proteins and have associated MHC allele information available (listed in Table S2 ). Of these 229 epitopes,~82% were MHC Class I restricted epitopes (Table S3) . Importantly, 102 of the 229 epitopes were derived from either the S (66) or N (36) protein. Mapping all 66 S-derived epitopes onto the resolved crystal structure of the SARS-CoV S protein ( Figure S2 ) revealed that 3 of these (GYQPYRVVVL, QPYRVVVLSF, and PYRVVVLSF) were located entirely in the SARS-CoV receptor-binding motif (), known to be important for virus cell entry .Similar to previous studies on HIV and HCV , we estimated population coverages for various combinations of MHC alleles associated with these 102 epitopes. Our aim was to determine sets of epitopes associated with MHC alleles with maximum population coverage, potentially aiding the development of vaccines against SARS-CoV-2. For selection, we adopted a greedy computational approach (see Materials and Methods), which identified a set of T cell epitopes estimated to maximize global population coverage. This set comprised of multiple T cell epitopes associated with 20 distinct MHC alleles and was estimated to provide an accumulated population coverage of 96.29% (Table 3) . Interestingly, the majority of the T cell epitopes for which a positive immune response has been determined using T cell assays (Table 2) were presented by the globally most-prevalent MHC allele (shown in blue color in Table 3 ). Moreover, the functionally important epitopes located in the SARS-CoV receptor binding motif were associated with the second and third most-prevalent MHC alleles (underlined in Table 3 ). Thus, while the ordering of T cell epitopes in Table 3 is based on the estimated global population coverage of the associated MHC alleles, it is also a natural order in which these epitopes should be tested experimentally for determining their potential to induce a positive immune response against SARS-CoV-2. We also computed the population coverage of this specific set of epitopes in China, the country most affected by the COVID-19 outbreak, which was estimated to be slightly lower (88.11%), as certain MHC alleles (e.g., HLA-A*02:01) associated with some of these epitopes are less frequent in the Chinese population (Table 3) . Repeating the same greedy approach but focusing on the Chinese population, instead of a global population, the maximum population coverage was estimated to be 92.76% (Table S4) . Table 3 . Set of the SARS-CoV-derived spike (S) and nucleocapsid (N) protein T cell epitopes (obtained from positive MHC binding assays) that are identical in SARS-CoV-2 and that maximize estimated population coverage globally (87 distinct epitopes). Due to the promiscuous nature of binding between peptides and MHC alleles, multiple S and N peptides were reported to bind to individual MHC alleles. Thus, while we list all the S and N epitopes that bind to each MHC allele (Table 3) , the estimated maximum population coverage may be achieved by selecting at least one epitope for each listed MHC allele. Likewise, many individual S and N epitopes were found to be presented by multiple alleles and thereby estimated to have varying global population coverage (listed in Table S5 ).Similar to T cell epitopes, we used in our study the SARS-CoV-derived B cell epitopes that have been experimentally-determined from positive B cell assays . These epitopes were classified as: (i) Linear B cell epitopes (antigenic peptides), and (ii) discontinuous B cell epitopes (conformational epitopes with resolved structural determinants).We aligned the 298 linear B cell epitopes (Table 1) across the SARS-CoV-2 proteins and found that 49 epitope-sequences, all derived from structural proteins, have an identical match and comprised no mutation in the available SARS-CoV-2 protein sequences (as of 21 February 2020). Interestingly, a large number (45) of these were derived from either the S (23) or N (22) protein (Table 4) , while the remaining (4) were from the M protein (Table S6) . Table 4 . SARS-CoV-derived linear B cell epitopes from S (23; 20 of which are located in subunit S2) and N (22) proteins that are identical in SARS-CoV-2 (45 epitopes in total). On the other hand, all 6 SARS-CoV-derived discontinuous B cell epitopes obtained from the ViPR database (Table 5) were derived from the S protein. Based on the pairwise alignment between the SARS-CoV and SARS-CoV-2 reference sequences ( Figure S3 ), we found that none of these mapped identically to the SARS-CoV-2 S protein, in contrast to the linear epitopes. For 3 of these discontinuous B cell epitopes (corresponding to antibodies S230, m396, and 80R ), there was a partial mapping, with at least one site having an identical residue at the corresponding site in the SARS-CoV-2 S protein (Table 5) . Table 5 . SARS-CoV-derived discontinuous B cell epitopes (and associated known antibodies ) that have at least one site with an identical amino acid to the corresponding site in SARS-CoV-2. Residues 1,2 910052 S230 G446, P462, D463, Y475 77444 m396 T359, T363, K365, K390, G391, D392, R395, R426, Y436, G482, Y484, T485, T486, T487, G488, I489, G490, Y491, Q492, Y494 77442 80R R426, S432, T433, Y436, N437, K439, Y440, Y442, P469, P470, A471, L472, N473, C474, Y475, W476, L478, N479, D480, Y481, G482, Y484, T485, T486, T487, G488, I489, Y491, Q492 1 Residues are numbered according to the SARS-CoV S protein reference sequence, accession ID: NP_828851.1.; 2 Residues in the epitopes that are identical in the SARS-CoV-2 sequences are underlined.Mapping the residues of the linear and discontinuous B cell epitopes onto the available structure of the SARS-CoV S protein revealed their distinct association with the two functional subunits of the S protein : S1, important for interaction with the host cell receptor, and S2, involved in fusion of the cellular and virus membranes (Figure 2a) . Specifically, 20 of the 23 linear epitopes (Table 4 ) mapped to S2 (Figure 2b) . Thus, the antibodies targeting the identified linear epitopes in the S2 subunit might cross-react and neutralize both SARS-CoV and SARS-CoV-2, as suggested in a very recent study .While S2 is comparatively less exposed than S1, it may be accessible to antibodies during the complex conformational changes involved in viral entry of coronaviruses ; though this remains to be more clearly understood. In contrast, the 3 discontinuous B cell epitopes (Table 5 ) mapped onto the more exposed S1 subunit (Figure 2c, left panel) , which contains the receptor-binding motif of the SARS-CoV S protein . We observed that very few residues of the 3 discontinuous epitopes were identical within SARS-CoV and SARS-CoV-2 (Figure 2c, right panel) . These differences suggest that the SARS-CoV-specific antibodies S230, m396, and 80R known to bind to these epitopes in SARS-CoV might not be able to bind to the same regions in SARS-CoV-2 S protein. Interestingly, while this paper was under review, this has been confirmed experimentally . Further studies are currently under way to identify other SARS-CoV antibodies that may bind to discontinuous epitopes of the SARS-CoV-2 S protein . (a) Subunits S1 and S2 are indicated in purple and green color, respectively. The receptor binding motif lies within the S1 subunit and is indicated in orange color.(b) Residues of the linear B cell epitopes, that were identical in SARS-CoV-2 (Table 4) , are shown in red color. The dark and light shade reflect the surface and buried residues, respectively. (c) Location of discontinuous B cell epitopes that share at least one identical residue with corresponding SARS-CoV-2 sites (Table 5 ). Identical epitope residues are shown in red color, while the remaining epitope residues are shown in blue color. Both the side view (left panel) and the top view (right panel) of the structure are shown.On the T cell side, the identification of SARS-CoV-derived epitopes that map identically to SARS-CoV-2, and the large population that these are expected to cover, is particularly encouraging. It promotes further research in exploring vaccines designed to induce a protective T cell response, which has been shown to provide long term protection in SARS-CoV . On the B cell side, in agreement with very recent experimental studies [47, 48] , our results suggest that SARS-CoV-derived Figure 2 . Location of SARS-CoV S protein subunits and SARS-CoV-derived B cell epitopes on the protein structure (PDB ID: 5XLR). (a) Subunits S1 and S2 are indicated in purple and green color, respectively. The receptor binding motif lies within the S1 subunit and is indicated in orange color. (b) Residues of the linear B cell epitopes, that were identical in SARS-CoV-2 (Table 4) , are shown in red color. The dark and light shade reflect the surface and buried residues, respectively. (c) Location of discontinuous B cell epitopes that share at least one identical residue with corresponding SARS-CoV-2 sites (Table 5 ). Identical epitope residues are shown in red color, while the remaining epitope residues are shown in blue color. Both the side view (left panel) and the top view (right panel) of the structure are shown.The quest for a vaccine against the novel SARS-CoV-2 is recognized as an urgent problem. Effective vaccination could indeed play a significant role in curbing the spread of the virus, and help to eliminate it from the human population. However, scientific efforts to address this challenge are only just beginning. Much remains to be learnt about the virus, its biological properties, epidemiology, etc. At this early stage, there is also a lack of information about specific immune responses against SARS-CoV-2, which presents a challenge for vaccine development.This study has sought to assist with the initial phase of vaccine development by providing recommendations of epitopes that may potentially be considered for incorporation in vaccine designs. Despite having limited understanding of how the human immune system responds naturally to SARS-CoV-2, these epitopes are motivated by responses they have recorded in SARS-CoV (or, for the case of T cell epitopes, to at least confer MHC binding), and the fact that they map identically to SARS-CoV-2, based on the available sequence data (as of 21 February 2020). This important observation should not be taken for granted. Despite the apparent similarity between SARS-CoV and SARS-CoV-2, there is still considerable genetic variation between the two, and it is not obvious a-prior if epitopes that elicit an immune response against SARS-CoV are likely to be effective against SARS-CoV-2. We found that only 23% and 16% of known SARS-CoV T cell and B cell epitopes map identically to SARS-CoV-2, respectively, and with no mutation having been observed in these epitopes among the available SARS-CoV-2 sequences (as of 21 February 2020). This provides a strong indication of their potential for eliciting a robust T cell or antibody response in SARS-CoV-2.On the T cell side, the identification of SARS-CoV-derived epitopes that map identically to SARS-CoV-2, and the large population that these are expected to cover, is particularly encouraging. It promotes further research in exploring vaccines designed to induce a protective T cell response, which has been shown to provide long term protection in SARS-CoV . On the B cell side, in agreement with very recent experimental studies [47, 48] , our results suggest that SARS-CoV-derived antibodies targeting the receptor binding motif in the S1 subunit of the SARS-CoV-2 S protein may not be effective, due to the large genetic mismatches observed in known structural epitopes targeting this domain. Linear SARS-CoV-derived B cell epitopes in the S2 subunit may potentially be more promising candidates for inducing a protective antibody response. Numerous of these epitopes, while being less exposed, are found to map identically to SARS-CoV and SARS-CoV-2, and preliminary results are already emerging which suggest their potential in generating cross-reactive and neutralizing antibodies . Hence, vaccine solutions that attempt to induce antibodies that target the S2 linear epitopes may be effective and should be explored further.Research efforts directed towards the design and development of vaccines for SARS-CoV-2 are increasing, and some related analyses are already being reported in distinct, parallel studies. These studies, like our own, are based on leveraging available data and computational methods, and add to recent work focused on computational analysis and design of vaccines for various different viruses (e.g., . A preliminary analysis of linear SARS-CoV-derived B cell epitopes has been reported online on the ViPR database website ( Corona/2019-nCoV-ViPR-report_24JAN2020.pdf). Different from our study, which is focused on the linear and discontinuous SARS-CoV-derived epitopes, that analysis considered linear B cell epitope data for all Betacoronaviruses from human hosts. While only a summary of the results has been provided so far, preventing direct comparison of the individual epitopes, the number of linear B cell epitopes reported to map identically to SARS-CoV-2 is comparable to our findings.A recent study has also predicted T cell epitopes for SARS-CoV-2 that may be presented by a population from the Asia-Pacific region . Again, there are multiple differences to our work. First, the focus of that study was on MHC Class II epitopes, while here we considered both MHC Class I and II epitopes. Interestingly, while we found a few MHC Class II epitopes using our approach (Table S3) , only one of these (HLA-DRB1*01:01) appeared in our identified epitope set (Table 3 ), due to their comparatively low estimated population coverage. Second, computational tools were used to predict MHC Class II epitopes in , while here we analyzed the SARS-CoV-derived epitopes that have been determined experimentally, using either positive T cell or MHC binding assays, and which match identically with the available SARS-CoV-2 sequences (as of 21 February 2020). Thus, our identified epitopes are seemingly a more rational set of potential targets that can assist in the ongoing search for a SARS-CoV-2 vaccine.We acknowledge that this is a preliminary analysis based on the limited sequence data available for SARS-CoV-2 (as of 21 February 2020). As the virus continues to evolve and as more data is collected, it is expected that additional mutations will be observed. Such mutations will not affect our analysis, provided that they occur outside of the identified epitope regions. If mutations do occur within epitope regions, then these epitopes may be further screened in line with the conservative filtering principle that we have employed, thereby producing a more refined epitope set.Further experimental studies (T cell and B cell assays) are required to determine the potential of the identified epitopes to induce a positive immune response against SARS-CoV-2. This would help to further refine the reported epitope set, based on observed immunogenicity; an important consideration for immunogen design.Overall, as the identified set of SARS-CoV epitopes map identically to SARS-CoV-2, they present potentially useful candidates for guiding experimental efforts towards developing vaccines against SARS-CoV-2. More generally, our study further highlights the potential importance of previous experimental and clinical studies of SARS-CoV, and its use in concert with emerging data for SARS-CoV-2, in searching for effective vaccines to combat the COVID-19 epidemic.Supplementary Materials: The following are available online at , Figure S1 : Fraction of mutations in the observed sequences of the structural proteins of the three coronaviruses, Figure S2 : Location of identified T cell epitopes on the SARS-CoV S protein structure (PDB ID: 5XLR), Figure S3 : Pairwise sequence alignment of the reference sequences of the S proteins of SARS-CoV and SARS-CoV-2 (accession ID: NP_828851.1 and YP_009724390.1, respectively), Table S1 : List of GISAID accession IDs for 120 genomic sequences of SARS-CoV-2, Table S2 : List of all SARS-CoV-derived T cell epitopes determined using positive MHC binding assays (with associated MHC allele information available at 4 digit resolution) and found to be identical in SARS-CoV-2, Table S3 : Distribution of all SARS-CoV-derived T cell epitopes obtained using positive MHC binding assays (with associated MHC allele information available at 4 digit resolution) and that are identical in SARS-CoV-2, Table S4 : Set of SARS-CoV-derived S and N protein T cell epitopes (obtained using positive MHC binding assays) that are identical in SARS-CoV-2 and that maximize estimated population coverage in China (86 distinct epitopes), Table S5 : Estimated global and Chinese population coverages for the individual SARS-CoV-derived S or N protein T cell epitopes (obtained using positive MHC binding assays), that are identical in SARS-CoV-2, Table S6 : SARS-CoV-derived linear B cell epitopes, excluding those in S and N proteins, that are identical in SARS-CoV-2, Table S7 \n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.979\tIn December 2019, a novel coronavirus, called COVID-19, was discovered in Wuhan, China, and has spread to different cities in China as well as to 24 other countries. The number of confirmed cases is increasing daily and reached 34,598 on 8 February 2020. In the current study, we present a new forecasting model to estimate and forecast the number of confirmed cases of COVID-19 in the upcoming ten days based on the previously confirmed cases recorded in China. The proposed model is an improved adaptive neuro-fuzzy inference system (ANFIS) using an enhanced flower pollination algorithm (FPA) by using the salp swarm algorithm (SSA). In general, SSA is employed to improve FPA to avoid its drawbacks (i.e., getting trapped at the local optima). The main idea of the proposed model, called FPASSA-ANFIS, is to improve the performance of ANFIS by determining the parameters of ANFIS using FPASSA. The FPASSA-ANFIS model is evaluated using the World Health Organization (WHO) official data of the outbreak of the COVID-19 to forecast the confirmed cases of the upcoming ten days. More so, the FPASSA-ANFIS model is compared to several existing models, and it showed better performance in terms of Mean Absolute Percentage Error (MAPE), Root Mean Squared Relative Error (RMSRE), Root Mean Squared Relative Error (RMSRE), coefficient of determination (R 2 ), and computing time. Furthermore, we tested the proposed model using two different datasets of weekly influenza confirmed cases in two countries, namely the USA and China. The outcomes also showed good performances.A large family of viruses, called coronaviruses, are severe pathogens for human beings, which infect respiratory, hepatic, gastrointestinal, and neurologic diseases. They are distributed among humans, birds, livestock, mice, bats, and other wild animals . The outbreaks of two previous coronaviruses, SARS-CoV and MERS-CoV in 2003 and 2012, respectively, have approved the transmission from animal to animal, and human to human . In December 2019, the World Health Organization (WHO) received notifications from China for many cases of respiratory illness that were linked to some people who had visited a seafood market in Wuhan . Currently, Wuhan city suffers from the spreading of a novel coronavirus, called COVID-19 (previously, it was called 2019-nCoV). In , the authors concluded that COVID-19 likely originated in bats, because it is more similar to two bat-derived coronavirus strains. However, the source of the COVID-19 is not confirmed yet, and it communities, Hong Kong and Toronto, were 1.2 and 1.32, respectively. Ong et al. proposed a monitoring and forecasting model for influenza A (H1N1-2009). Furthermore, Nah et al. proposed a probability-based model to predict the spread of the MERS.The Adaptive Neuro-Fuzzy Inference System (ANFIS) is widely applied in time series prediction and forecasting problems, and it showed good performance in many existing applications. It offers flexibility in determining nonlinearity in the time series data, as well as combining the properties of both artificial neural networks (ANN) and fuzzy logic systems. It has been applied in various forecasting applications, for example, in , a stock price forecasting model was proposed using ANFIS and empirical mode decomposition. Chen et al. proposed a TAIEX time series forecasting model based on a hybrid of ANFIS and ordered weighted averaging (OWA). In , another time series forecasting method was presented for electricity prices based on ANFIS. Svalina et al. proposed an ANFIS based forecasting model for close price indices for a stock market for five days. Ekici and Aksoy presented an ANFIS based building energy consumption forecasting model. More so, ANFIS is also applied to forecast electricity loads . Kumar et al. proposed an ANFIS based model to forecast return products. Ho and Tsai applied ANFIS to forecast product development performance. However, estimating ANFIS parameters is a challenge that needs to be improved. Therefore, in previous studies, some individual swarm intelligence (SI) methods have been applied to the ANFIS parameters to enhance time series forecasting because these parameters have a significant effect on the performance of ANFIS. The SI methods include the particle swarm optimization (PSO) [31, 32] , social-spider optimization , sine-cosine algorithm (SCA) , and multi-verse optimizer (MVO) . For example, in SCA algorithm was applied to improve the ANFIS model to forecast oil consumption in three countries, namely, Canada, Germany, and Japan. In the same context, in , The MVO algorithm was used to enhance the ANFIS model to forecast oil consumption in two countries. In addition, in the PSO was used with ANFIS to predict biochar yield. However, individual SI algorithms may stock at local optima. Therefore, one solution is to apply hybrid SI algorithms to avoid this problem. In , a hybrid of two SI algorithms, namely GA and SSA, was presented to improve the ANFIS model. The proposed new model called GA-SSA-ANFIS was applied to forecast crude oil prices for long-term time series data. However, the previously mentioned methods suffer from some limitations that can affect the performance of the forecasting output such as slow convergence and the ability to balance between exploration and exploitation phases can influence the quality of the final output. This motivated us to propose an alternative forecasting method dependent on the hybridization concept. This concept avoids the limitations of traditional SI techniques by combining the strengths of different techniques, and this produces new SI techniques that are better than traditional ones.In the current study, we propose an improved ANFIS model based on a modified flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). The FPA is an optimization algorithm proposed by Yang , which was inspired by the flow pollination process of the flowering plants. The FPA was employed in various optimization applications, for example to estimate solar PV parameter [39, 40] , solving sudoku puzzles , feature selection , antenna design , and other applications . Moreover, SSA is also an optimization algorithm proposed by Mirjalili et al. inspired by the behavior of salp chains. In recent years, the SSA was utilized to solve different optimization problems, such as feature selection [49, 50] , data classification , image segmentation , and others [53, 54] .The proposed method called FPASSA is a hybrid of FPA and SSA, in which the SSA is applied as a local search method for FPA. The proposed FPASSA starts by receiving the historical COVID-19 dataset. Then a set of solutions is generated where each of them represents the value for the parameters of the ANFIS model. Then the quality of each solution is calculated using the fitness value, and the solution that has the best fitness value is chosen to represent the best solution. Then the probability of each solution is computed. Then the current solution will be updated, either using global or local strategy in FPA. However, in the case of local strategy, the operators of SSA or FPA will be used according to the probability of the fitness value for each solution. The process of updating the solutions is repeated until reaching the stop condition, and the best parameter configurations are used to forecast the number of confirmed cases of COVID-19.The main contribution points of the current study are as follows:1.We propose an efficient forecasting model to forecast the confirmed cases of the COVID-19 in China for the upcoming ten days based on previously confirmed cases.An improved ANFIS model is proposed using a modified FPA algorithm, using SSA.We compare the proposed model with the original ANFIS and existing modified ANFIS models, such as PSO, GA, ABC, and FPA.The rest of this study is organized as follows. The preliminaries of ANFIS, FPA, and SSA are described in Section 2. Section 3 presents the proposed FPASSA, and Section 4 presents the experimental setup and results. We conclude this study in Section 5.The principles of the ANFIS are given in this section. The ANFIS model links the fuzzy logic and neural networks . It generates a mapping between the input and output by applying IF-THEN rules (it is also called Takagi-Sugeno inference model). Figure 1 illustrates the ANFIS model where, y and x define the inputs to Layer 1 whereas, O 1i is its output of node i that is computed as follows:where µ denotes the generalized Gaussian membership functions. A i and B i define the membership values of µ. α i and ρ i denote the premise parameters set. The output of Layer 2 (it is also known as the firing strength of a rule) is calculated as follows:Meanwhile, the output of Layer 3 (it is also known as the normalized firing strength) is calculated as follows:The output of Layer 4 (it is also known as an adaptive node) is calculated as follows:where r i , q i , and p i define the consequent parameters of the node i. Layer 5 contains only one node; its output is computed as: Flower Pollination Algorithm is an optimization method proposed by Yang . It simulates the transfer of flowers' pollen by pollinators in nature. This algorithm utilizes the two types of pollination (i.e., self-pollination and cross-pollination). In self-pollination, the pollination occurs with no pollinators, whereas, in cross-pollination, the pollens are moved between different plants. In more detail, the self-pollination can be represented as a local pollination while the cross-pollination can be called global pollination.The global pollination or cross-pollination can be mathematically formed as follows:where x t i defines the pollen i at iteration t. L denotes the pollination's strength or the step size. F * is the target position or best solution. In some cases, insects can fly with different distance steps for a long space; therefore, Levy fly distribution is applied to simulate this movement.where λ = 1.5. Γ(λ) denotes the gamma function. This distribution is available for large steps s > 0. The self-pollination or local pollination can be mathematically formed as follows:where x t i and x k i represent pollens from different flower in the same plant. in the range [0,1] The process of pollination can be done using cross-pollination or self-pollination. Therefore, the random variable p, in the range [0, 1], is used to determine this process.SSA is an optimization technique introduced by . It simulates the Salps' behavior in nature. This behavior is called salp chain. The mathematical model of SSA begins by splinting its population into a leader group and followers group. The leader is the front salp, whereas, the followers are the other salps. The search space is determined in n-dimensions with n variables. Equation (10) works to update the salps' positions.where x 1 j denotes the leader's position in j-th dimension. F j is the target position. ub j and lb j represent the max and min bounds, respectively. c 2 and c 3 denote random numbers in [0, 1]. c 1 is an important parameter; it balances between the exploration and exploitation phases. It is computed as follows:where the current loop number is t and the max loop' number is t max . Then, the followers' position is updated as follows:where x i j defines the i-th position of the follower in j-th dimension. i > 1.This section explains the proposed FPASSA-ANFIS method. It is a time series method for forecasting the confirmed cases of the COVID-19, as given in Figure 2 . The FPASSA-ANFIS utilizes the improved FPA to train the ANFIS model by optimizing its parameters. The FPASSA-ANFIS contains five layers as the classic ANFIS model. Layer 1 contains the input variables (the historical COVID-19 confirmed cases). Whereas Layer 5 produces the forecasted values. In the learning phase, the FPASSA is used to select the best weights between Layer 4 and Layer 5.The FPASSA-ANFIS starts by formatting the input data in a time series form. In our case, the autocorrelation function (ACF) was considered. ACF is one of the methods applied to find patterns in the data; it presents information about the correlation between points separated by various time lags. Therefore, in this paper, the variables with ACF greater than 0.2 are considered i.e., 5-lags.Besides, the training data contains 75% of the dataset, whereas the testing data contains 25% of them. The number of clusters is defined by the fuzzy c-mean (FCM) method to construct the ANFIS model.The parameters of the ANFIS model are prepared by the FPASSA algorithm. In the training phase, the calculation error (as in Equation (13)) between the real data and the predicted data is used to evaluate the parameters' quality.where T is the real data, and P is the predicted data. N s is the sample length. The smaller values of the objective function indicate good ANFIS's parameter.On the other hand, the updating phase of the followers' positions in the SSA algorithm is applied to improve the global pollination phase in the FPA algorithm. In this improvement, there is a random variable (r) used to switch between both phases. If r > 0.5, then the operators of the SSA is used; otherwise, the operators of the FPA are used. In general, The FPASSA starts by constructing the population (X); afterward, the objective function is calculated for each solution. The solution with the lowest error value is saved to the next iteration. This sequence is repeated until meeting the stop condition, which in this paper, is the maximum number of iterations. Then the best solution is passed to train the parameters of the ANFIS model.After finishing the training phase, the testing phase is started with the best solution to compute the final output. The performance of the proposed method is evaluated by comparing the real data with the predicted data using the performance measures. Finally, the FPASSA produces a foretasted value for confirmed cases of COVID-19 in China in the next day. The steps of the proposed FPASSA are presented in Algorithm 1.Input: Historical COVID-19 dataset, size of population N, total number of iterations t max .Divide the data into training and testing sets.Using Fuzzy c-mean method to determine the number of membership functions.Constructing the ANFIS network.Set the initial value for N solutions (X). Return the best solution that represents the best configuration for ANFIS.Apply the testing set to the best ANFIS model.Forecasting the COVID-19 for the next ten days.This section presents the description of the used dataset, the performance measures, the parameter setting for all methods, the experiment results, and discussions.The main dataset of this study is COVID-19 dataset. It was collected from the WHO website (https: //). It contains the daily confirmed cases in China from 21 January 2020 to 18 February 2020, as shown in Table 1 . We used 75% from the dataset to train the model while the rest is used to test it.Moreover, we evaluated the performance of the proposed method using two datasets of weekly influenza confirmed cases. The first one is called DS1; it was collected from the Centers for Disease Control and Prevention (CDC) (). It starts from week number 40 in 2015 and continues until week number 6 in 2020. Whereas, the second one is called DS2. It was collected from the WHO website (). It contains the data of weekly influenza confirmed cases in China from week number 1 in 2016 to week number 8 in 2020. The quality of the proposed method is evaluated using a set of performance metrics as follows:• Root Mean Square Error (RMSE):where Yp and Y are the predicted and original values, respectively. • Mean Absolute Error (MAE):• Mean Absolute Percentage Error (MAPE):• Root Mean Squared Relative Error (RMSRE):N s represents the sample size of the data. • Coefficient of Determination (R 2 ):where Y represents the average of Y.The lowest value of RMSE, MAE, MAPE, and RMSRE refers to the best method. The higher value of R 2 indicates better correlation for the method.This paper aims to assess the ability of the FPASSA to forecast the COVID-19 by comparing its performance with other methods, namely the ANFIS and the trained ANFIS models using PSO, GA, ABC, FPA, and FPASSA. The parameters' setting for these models is listed in Table 2 .The common parameters, such as population size, are set to 25 and 100 iterations are applied. Besides, each algorithm is performed for 30 independent runs to fair comparisons. The selected parameters are chosen because they produced good behavior in previous experiments, such as [34, 35, 55, 56] . Table 2 . Parameters' setting.Parameters SettingMax. epochs = 100, Error goal = 0, Initial step = 0.01, Decrease rate = 0.9, Increase rate = 1. In this section, the performance of the proposed FPASSA to predict the DS1 and DS2 is discussed. It can be concluded from Table 3 that the performance of FPASSA outperformed the compared methods in all measures, whereas the FPA is ranked second. The results of DS2 indicate that the FPASSA is ranked first in terms of RMSE, MAPE, R 2 , and the CPU time. Whereas, the PSO is ranked second, followed by the FPA, GA, then ABC. These results denote that the proposed method can optimize the parameters of the ANFIS model effectively and produce good results in terms of the performance measures. Comparison results between the proposed FPASSA and other models to forecast COVID-19 are given in Table 4 . It can be concluded that the FPASSA outperforms other models. For example, by analyzing the results of RMSE, MAE, MAPE, RMSRE, and CPU time(s) it can be observed that the FPASSA achieves the smallest value among the comparison algorithms, and this indicates the high quality of the FPASSA. Meanwhile, the FPA allocates the second rank, which provides better results than the rest of the methods.Moreover, the value of R 2 refers to the high correlation between the prediction obtained by the proposed FPASSA method and the original COVID-19, which has nearly 0.97. This can also be noticed from Figure 3 , which depicts the training of the algorithms using the historical data of the COVID-19 as well as their forecasting values for ten days. Table 5 depicts the forecasting value for the confirmed cases of the COVID-19 in China from 19/2/2020 to 28/2/2020. From these results, it can be noticed that the outbreak will reach its highest level on the day 28/2/2020. The average percentage of the increase over the forecasted period is 10%, the highest percentage is 12% on 28/2/2020, and the lowest percentage is 8.7% on 19/2/2020. From the previous results, it can be concluded that the proposed FPASSA-ANFIS has a high ability to forecast the COVID-19 dataset. These results avoid the limitations of traditional ANFIS because of the combination with the modified FPA method. Moreover, the operators of SSA are combined with the local strategy of FPA to enhance their exploitation ability. However, the time computational of the proposed FPASSA method still requires more improvements.This paper proposed a modified version for the flower pollination algorithm (FPA) using the salp swarm algorithm (SSA). This modified version, called FPASSA, is applied to improve the performance of the ANFIS through determining the optimal value for its parameters. The developed FPASSA-ANFIS model is applied as a forecasting technique for a novel coronavirus, called COVID-19, that was discovered in Wuhan, China at the end of last year and January of the current year. The proposed FPASSA-ANFIS model has a high ability to predict the number of confirmed cases within ten days. Besides, FPASSA-ANFIS outperforms other forecasting models in terms of RMSE, MAE, MAPE, RMSRE, and R 2 . Furthermore, two datasets of weekly influenza confirmed cases in the USA and China were used to evaluate the proposed method, and the evaluation outcomes showed its good performance. According to the promising results obtained by the proposed FPASSA-ANFIS, it can be applied in different forecasting applications.\n",
            "\t0.886\tBackground As the outbreak of coronavirus disease 2019 (COVID-19) progresses, epidemiological data are needed to guide situational awareness and intervention strategies. Here we describe efforts to compile and disseminate epidemiological information on COVID-19 from news media and social networks.As the outbreak of coronavirus disease 2019 (COVID- 19) is rapidly expanding in China and beyond, with the potential to become a worldwide pandemic, 1 real-time analyses of epidemiological data are needed to increase situational awareness and inform interventions. 2 Previously, real-time analyses have shed light on the transmissibility, severity, and natural history of an emerging pathogen in the first few weeks of an outbreak, such as with severe acute respiratory syndrome (SARS), the 2009 influenza pandemic, and Ebola. Analyses of detailed line lists of patients are particularly useful to infer key epidemiological parameters, such as the incubation and infectious periods, and delays between infection and detection, isolation, and reporting of cases. 3, 4 However, official individual patient data rarely become publicly available early on in an outbreak, when the information is most needed.Building on our previous experience collating news reports to monitor transmission of Ebola virus, 7 here we present an effort to compile individual patient information and subnational epidemic curves on COVID-19 from a variety of online resources. Data were made publicly available in real time and were used by the infectious disease modelling community to generate and compare epidemiological estimates relevant to interventions. We describe the data generation process and provide an early analysis of age patterns of COVID-19, case counts across China and inter nationally, and delays between symptom onset, admissions to hospital, and reporting, for cases reported until Jan 31, 2020.In this population-level observational study, we used crowdsourced reports from DXY.cn, a social network for Chinese physicians, health-care professionals, pharmacies, and health-care facilities established in 2000. This online platform is providing real-time coverage of the COVID-19 outbreak in China, obtained by collating and curating reports from news media, government television, and national and provincial health agencies. The information reported includes time-stamped cumulative counts of COVID-19 infections, outbreak maps, and realtime streaming of health authority announcements in Chinese (directly or through state media). 8 Every report is linked to an online source, which can be accessed for more detailed information on individual cases.These are publicly available, de-identified patient data reported directly by public health authorities or by state media. No patient consent was needed and no ethics approval was required.We closely monitored updates on DXY.cn between Jan 20, 2020, and Jan 31, 2020, to extract key information on individual patients in near real-time, and reports of daily case counts. For individual-level patient data, we used descriptions from the original source in Chinese to retrieve age, sex, province of identification, travel history, reporting date, dates of symptom onset and seeking care at a hospital or clinic, and discharge status, when available. Individual-level patient data were formatted into a line-list database for further quantitative analysis. Individual-level patient data were entered from DXY.cn by a native Chinese speaker (KS), who also generated an English summary for each patient. Entries were checked by a second person (JC). Since DXY.cn primarily provides Evidence before this study An outbreak of coronavirus disease 2019 (COVID-19) was recognised in early January, 2020, in Wuhan City, Hubei province, China. The new virus is thought to have originated from an animal-to-human spillover event linked to seafood and live-animal markets. The infection has spread locally in Wuhan and elsewhere in China, despite strict intervention measures implemented in the region where the infection originated on Jan 23, 2020. More than 500 patients infected with COVID-19 outside of mainland China have been reported between Jan 1 and Feb 14, 2020. Although laboratory testing for COVID-19 quickly ramped up in China and elsewhere, information on individual patients remains scarce and official datasets have not been made publicly available. Patient-level information is important to estimate key time-to-delay events (such as the incubation period and interval between symptom onset and visit to a hospital), analyse the age profile of infected patients, reconstruct epidemic curves by onset dates, and infer transmission parameters. We searched PubMed for publications between Jan 1, 1990, and Feb 6, 2020, using combinations of the following terms: (\"coronavirus\" OR \"2019-nCoV\") AND (\"line list\" OR \"case description\" OR \"patient data\") AND (\"digital surveillance\" OR \"social media\" OR \"crowd-sourced data\"). The search retrieved one relevant study on Middle East respiratory syndrome coronavirus that mentioned FluTrackers in their discussion, a website that aggregates epidemiological information on emerging pathogens. However, FluTrackers does not report individual-level data on COVID-19.To our knowledge, this is the first study that uses crowdsourced data from social media sources to monitor the COVID-19 outbreak. We searched DXY.cn, a Chinese health-care-oriented social network that broadcasts information from local and national health authorities, to reconstruct patient-level information on COVID-19 in China. We also queried international media sources and national health agency websites to collate data on international exportations of COVID-19. We describe the demographic characteristics, delays between symptom onset, seeking care at a hospital or clinic, and reporting for 507 patients infected with COVID-19 reported until Jan 31, 2020. The overall cumulative progression of the outbreak is consistent between our line list and an official report published by the Chinese national health authorities on Jan 28, 2020. The estimated incubation period in our data aligns with that of previous work. Our dataset was made available in the public domain on Jan 21, 2020.Crowdsourced line-list data can be reconstructed from social media data, especially when a central resource is available to curate relevant information. Public access to line lists is important so that several teams with different expertise can provide their own insights and interpretations of the data, especially in the early phase of an outbreak when little information is available. Publicly available line lists can also increase transparency. The main issue with the quality of patient-level data obtained during health emergencies is the potential lack of information from locations overwhelmed by the outbreak (in this case, Hubei province and other provinces with weaker health infrastructures). Future studies based on larger samples of patients with COVID-19 could explore in more detail the transmission dynamics of the outbreak in different locations, the effectiveness of interventions, and the demographic factors driving transmission.For an example of an online source see  ncovh5/view/pneumonia information on patients reported in China, we also compiled additional information on internationally exported cases of COVID-19. We obtained data for 21 countries outside of mainland China (Australia, Cambodia, Canada, France, Germany, Hong Kong, India, Italy, Japan, Malaysia, Nepal, Russia, Singapore, South Korea, Sri Lanka, Taiwan, Thailand, United Arab Emirates, the UK, the USA, and Vietnam). We gathered and cross-checked data for infected patients outside of China using several sources, including global news media (Kyodo News, Straits Times, and CNN), official press releases from each country's Ministry of Health, and disease control agencies.In addition to detailed information on individual patients, we reconstructed the daily progression of reported patients in each province of China from Jan 13, until Jan 31, 2020. We used the daily outbreak situation reports com municated by provincial health authorities, covered by state television and media, and posted on DXY.cn. All patients in our databases had a laboratory confirmed SARS coronavirus 2 (SARS-CoV-2) infection.Our COVID-19 database was made publicly available as a Google Sheet, disseminated via Twitter on Jan 21, 2020, and posted on the website of Northeastern University, (Boston, MA, USA) on Jan 24, 2020, where it is updated in real time. Data used in this analysis, frozen at Jan 31, 2020, are available online as a spreadsheet.We assessed the age distribution of all patients with COVID-19 by discharge status. We adjusted the age profile of Chinese patients by the population of China. We used 2016 population estimates from the Institute for Health Metrics and Evaluation 9 to calculate the relative risk (RR) of infection with COVID-19 by age group. To calculate the RR, we followed the method used by Lemaitre and colleagues 10 to explore the age profile of influenza, where RR for age group i is defined as where C i is the number of cases in age group i and N i is the population size of age group i.To estimate trends in the strength of case detection and interventions, we analysed delays between symptom onset and visit to a health-care provider, at a hospital or clinic, and from seeking care at a hospital or clinic to reporting, by time period and location. We considered the period before and after Jan 18, 2020, when media attention and awareness of the outbreak became more pronounced. 11 We used non-parametric tests to assess differences in delays between seeking care at a hospital or clinic and reporting between locations (Wilcoxon test to compare two locations and Kruskall-Wallis test to compare three or more locations).We estimated the duration of the incubation period on the basis of our line list data. We analysed a subset of patients returning from Wuhan who had spent less than a week in Wuhan, to ensure a narrowly defined exposure window. The incubation period was estimated as the midpoint between the time spent in Wuhan and the date of symptom onset.We did all analyses in R (version 3.5.3). We considered p values of less than 0·05 to be significant.The funder had no role in study design, data compilation, data analysis, data interpretation, or writing of the report. All authors had access to the data, and had final responsibility for the decision to submit for publication.Our line list comprised 507 patients reported from Jan 13, to Jan 31, 2020, including 364 (72%) from mainland China and 143 (28%) from outside of China (table) The age distribution of COVID-19 cases was skewed towards older age groups with a median age of 45 years (IQR 33-56) for patients who were alive or who had an unknown outcome at the time of reporting (figure 1). The median age of patients who had died at the time of reporting was 70 years (IQR 65-81). Few patients (13 [3%]) were younger than 15 years. Adjustment for the age demographics of China confirmed a deficit of infections among children, with a RR below 0·5 in patients younger than 15 years (figure 1). The RR measure indicated a sharp increase in the likelihood of reported COVID-19 among people aged 30 years and older.A timeline of cases in our crowdsourced patient line list is shown by date of onset in figure 2 , indicating an acceleration of reported cases by Jan 13, 2020. The outbreak progression based on the crowdsourced patient line list was consistent with the timeline published by China Center for Disease Control and Prevention (CDC) on Jan 28, 2020, 12 which is based on a more comprehensive database of more than 6000 patients with COVID-19. Since Jan 23, 2020, the cumulative number of cases has slowed down in the crowdsourced and China CDC curves (figure 2), which probably reflects the delay between disease onset and reporting. The median reporting delay was 5 days (IQR 3-8) in our data.Province-level epidemic curves are shown by reporting date in figure 3 . As of Jan 31, 2020, 16 (52%) of 30 provinces in mainland China had reported more than 100 confirmed cases. The apparent rapid growth of newly reported cases between Jan 18, and Jan 31, 2020, in several provinces outside of Hubei province is consistent with sustained local transmission.Across the study period, the median delay between symptom onset and seeking care at a hospital or clinic was 2 days (IQR 0-5 days) in mainland China ( figure 4 ). This delay decreased from 5 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p=0·0009). Some provinces, such as Tianjin and Yunnan had shorter delays (data by province not shown), while the early cases from Hubei province were characterised by longer delays in seeking care (median 0 days [IQR 0-1]).The median delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 2-5 days) in mainland China and decreased from 9 days before Jan 18, 2020, to 2 days thereafter (Wilcoxon test p<0·0001; figure 4 ). Similarly to delays in seeking care at a hospital or clinic, reporting was quickest in Tianjin and Yunnan (median 1 day [IQR 0-1]) and slowest in Hubei province (median 12 days [IQR 7-16]).The median delay between symptom onset and seeking care at a hospital or clinic was 1 day (IQR 0-3) for international travellers, and shorter than for patients in Hubei province or the rest of mainland China (Kruskal-Wallis test p<0·0001; figure 4) . Even in the period after Jan 18, 2020, when awareness of the outbreak increased, a shorter delay between symptom onset and seeking care at a hospital or clinic was seen for international patients than for those in mainland China (Wilcoxon test p<0·0001). For international cases, the delay between seeking care at a hospital or clinic and reporting was 2 days (IQR 1-4), also shorter than for mainland China (Wilcoxon test p<0·0001; figure 4) .On the basis of 33 patients with a travel history to Wuhan, we estimated the median incubation period for COVID-19 to be 4·5 days (IQR 3·0-5·5; appendix p 2). Information from patient line lists is crucial but difficult to obtain at the beginning of an outbreak. Here we have shown that careful compilation of crowdsourced reports curated by a long-standing Chinese medical social network provides a valuable picture of the outbreak of COVID-19 in real time. The outbreak timeline is consistent with aggregated case counts provided by health authorities. For comparison, China CDC published the first epidemic curve by symptom onset on Jan 28, 2020. 12 Line lists provide unique information on the delays between symptom onset and detection by the health-care system, reporting delays, and travel histories. This information cannot be extracted from aggregated case counts published by official sources. Line list data can help assess the effectiveness of interventions and the potential for widespread transmission beyond the initial foci of infection. In particular, shorter delays between symptom onset and admission to hospital or seeking care in a hospital or clinic accelerate detection and isolation of cases, effectively shortening the infectious period. A useful feature of our crowdsourced database was the availability of travel histories for patients returning from Wuhan, which, along with dates of symptom onset, allowed for estimation of the incubation period here and in related work. 13, 14 A narrow window of exposure could be defined for a subset of patients who had a short stay in Wuhan, at a time when the epidemic was still localised to Wuhan. Several teams have used our dataset and datasets from others to estimate a mean incubation period for COVID-19 to be 5-6 days (95% CI 2-11). Our own estimate (median 4·5 days [IQR 3·0-5·5]) is consistent with previous work that used other modelling approaches. The incubation period is a useful parameter to guide isolation and contact tracing; based on existing data, the disease status of a contact should be known with near certainty after a period of observation of 14 days. 13 Availability of a public dataset enables independent estimation of important epidemiological parameters by several teams, allowing for confirmation and cross-checking at a time when information can be conflicting and noisy.An interesting finding in our data relates to the age distribution of patients. We found a heavy skew of infection towards older age groups, with substantially fewer children infected. This pattern could indicate agerelated differences in susceptibility to infection, severe outcomes, or behaviour. However, a substantial portion of the patients in our database are travellers, a population that is usually predominantly adults (although does not exclude children). Furthermore, because patient data in our dataset were captured by the health system, they are biased towards the more severe spectrum of the disease, especially for patients from mainland China. Clinical reports have shown that severity of COVID-19 is associated with the presence of chronic conditions, 16, 17 which are more frequent in older age groups. Nevertheless, we would also expect children younger than 5 years to be at risk of severe outcomes and to be reported to the healthcare system, as is seen for other respiratory infections. 18 Biological differences could have a role in shaping these age profiles. A detailed analysis of one of the early COVID-19 clusters by Chan and colleagues 19 revealed symptomatic infections in five adult members of the same household, while a child in the same household aged 10 years was infected but remained asymptomatic, potentially indicating biological differences in the risk of clinical disease driven by age. Previous immunity from infection with a related coronavirus has been speculated to potentially protect children from SARS, 20, 21 and so might also have a role in COVID-19. In any case, if the age distribution of cases reported here was to be confirmed and the epidemic were to progress globally, we would expect an increase in respiratory mortality concentrated among people aged 30 years and older. This mortality pattern would be substantially different from the profile of the 2009 influenza pandemic, for which excess mortality was concentrated in those younger than 65 years. 21 In our dataset, we saw a rapid increase in the number of people infected with COVID-19 in several provinces of China, consistent with local transmission outside of Hubei province. As of Jan 31, 2020, province-level epidemic curves are only available by date of reporting, rather than date of symptom onset, which usually inflates recent case counts if detection has increased. D e c 9 , 2 0 1 9 D e c 1 6 , 2 0 1 9 D e c 2 3 , 2 0 1 9 D e c 3 0 , 2 0 1 9 J a n 6 , 2 0 2 0 J a n 1 3 , 2 0 2 0 J a n 2 0 , 2 0 2 0 J a n 2 7 , 2 0 2 0 Furthermore, province-level data include both returning travellers from Hubei province (ie, importations) and locally acquired cases, which also usually inflate the apparent risk of local transmission. Notably, other lines of evidence suggest that local transmission is now well established outside of Hubei province, because travel increased just before the Chinese New Year on Jan 25, 2020, and before implementation of the travel ban in Wuhan. 22 Accordingly, our own data include evidence of transmission clusters in non-travellers, with, for instance, a second-generation transmission event reported in Shaanxi on Jan 21, 2020.Our study had several limitations, one of which was the data we used. Although all provinces in mainland China provide aggregated information on infections and deaths, individual-level patient descriptions are only available for a subset of provinces. Geographical coverage is heterogeneous in our line list, and we have a notable deficit of cases from Hubei province, the foci of the COVID-19 outbreak. We expect that little patient-level information is shared on social media by province-level and city-level health authorities in Wuhan and Hubei province because health systems are overwhelmed. For similar reasons, provinces with a large total case count at the end of January, 2020, or with a weaker health infrastructure, were under-represented in our line list, with the exception of Beijing. Other limitations in our data include severity (only patients who had severe enough symptoms to seek care were captured) and changes in case definition. A series of epidemiological criteria were required for COVID-19 testing, including travel history to Wuhan within the past 2 weeks; residence in Wuhan within the past 2 weeks; contact with individuals from Wuhan (with fever and respiratory symptoms) within the past 2 weeks; and being part of an established disease cluster. Some of these criteria (eg, relation to Wuhan) were relaxed over time (appendix). As a result, we have an overrepresentation of travel-related cases in our database.The reproduction number is an important quantity for outbreak control. We refrained from estimating this parameter because reporting changes could bias estimates relying on epidemic growth rates. Furthermore, our dataset captured cases all over China and does not reflect transmission patterns in any particular location. A mean reproduction number of 2·5-2·7 has previously been estimated on the basis of the volume of importations of international cases in the pre-intervention period in Wuhan. 11 We recognise that, although our data source is useful and timely, it should not replace official statistics. Manual compilation of detailed line lists from media sources is highly time consuming and is not sustainable when case counts reach several thousands. Here we provide detailed data on 507 patients when the official case count was over 9000 by Jan 31, 2020, representing a sample of approximately 5% of reported cases and a much smaller proportion of the full spectrum of COVID-19 cases, which include mild infections. A crowd sourced system would not be expected to catch all cases, especially if many cases are too mild to be captured by the health-care system, digital surveillance, or social media. Notably, DXY.cn does not generate data outside of traditional surveillance systems but rather provides a channel of rapid communication between the public and health authorities. In turn, our approach has helped extract and repackage information from health authorities into an analytical format, which was not available elsewhere.At the time of writing, efforts are underway to coordinate compilation of COVID-19 data from online sources across several academic teams. Ultimately, we expect that a line list of patients will be shared by government sources with the global community; however, data cleaning and access issues might take a prohibitively long time to resolve. For the west African Ebola outbreak, a similarly coordinated effort to publish a line list took 2 years. 23 Given the progression of the COVID-19 outbreak, such a long delay would be counterproductive.Overall, the novelty of our approach was to rely on a unique source for social media and news reports in China, which aggregated and curated relevant information. This approach facilitated entry of robust and standard data on clinical and demographic information. Reassuringly, DXY.cn maintains a special section dedicated to debunking fake news, myths, and rumours about the COVID-19 outbreak. Looking to the future, collection of patient data in the context of emergencies could include information on whether patients are identified through contact tracing or because they seek care on their own. Furthermore, data interpretability could be improved by gathering more quantitative information on how case definitions are used in practice.In conclusion, crowdsourced epidemiological data can be useful to monitor emerging outbreaks, such as COVID-19 and, as previously, Ebola virus. 7 These efforts can help generate and disseminate detailed information in the early stages of an outbreak when little other data are available, enabling independent estimation of key parameters that affect interventions. Based on our small sample of patients with COVID-19, we note an intriguing age distribution, reminiscent of that of SARS, which warrants further epidemiological and serological studies. We also report early signs that the response is strengthening in China on the basis of a decrease in case detection time, and rapid management of travel-related infections that are identified internationally. This is an early report of a rapidly evolving situation and the parameters discussed here could change quickly. In the coming weeks, we will continue to monitor the epidemiology of this outbreak using data from news reports and official sources.KS and CV contributed to the study design. KS and JC contributed to the data compilation. KS, JC, and CV contributed to data analysis. KS and JC contributed to the design and drawing of figures. KS, JC, and CV contributed to the writing of the manuscript.We declare no competing interests.All data used in this report have been made publicly available on the Laboratory for the Modeling of Biological + Socio-technical systems website of Northeastern University. The available data include daily case counts of COVID-19 by reporting date and Chinese province, and a de-identified line list of patients with COVID-19. The line list includes geographical location (country and province), reporting date, dates of symptom onset and seeking care at a hospital or clinic, relation to Wuhan, discharge status when known, an English summary of the case description from media sources, and a link to the original source of data. Seeking care at hospital or clinic to report Symptom onset to seeking care at hospital or clinic\n",
            "\t0.792\tSince December 2019, an increasing number of patients with pneumonia occurred in Wuhan, Hubei province, China, which attracted much attention not only within China but across the world 1,2 . The novel pneumonia was named as Corona Virus by World Health Organization (WHO) (), the common symptoms of COVID-19 at illness onset were fever, fatigue, dry cough, myalgia, and dyspnea 3 . In addition, some patients might suffer from headache, dizziness, abdominal pain, diarrhea, nausea, and vomiting 3 . Onset of disease may lead to progressive respiratory failure due to alveolar damage and even death 4 .Scientists then isolated a novel coronavirus from human airway epithelial cells, which was named 2019-nCoV 5 . Lu et al. 6 found that 2019-nCoV was closer to bat-SL-CoVZC45 and bat-SL-CoVZXC21 at the whole-genome level, and the external subdomain of the 2019-nCoV receptor-binding domain (RBD) was more similar to that of severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV). Study of Zhou et al. 4 indicated that the angiotensin-converting enzyme II (ACE2) is likely the cell receptor of 2019-nCoV, which were also the receptor for SARS-CoV and HCoV-NL63 7,8 . Zhou et al. 4 also proved that 2019-nCoV does not use other coronavirus receptors, aminopeptidase N, and dipeptidyl peptidase 4. The study of Xu et al. 9 found that the RBD domain of the 2019-nCoV S-protein supports strong interaction with human ACE2 molecules. These findings suggest that the ACE2 plays an important role in cellular entry, thus ACE2-expressing cells may act as target cells and are susceptible to 2019-nCoV infection 10 .The expression and distribution of the ACE2 in human body may indicate the potential infection routes of 2019-nCoV. Through the developed single-cell RNA sequencing (scRNA-Seq) technique and single-cell transcriptomes based on the public database, researchers analyzed the ACE2 RNA expression profile at single-cell resolution. High ACE2 expression was identified in type II alveolar cells (AT2) of lung , esophagus upper and stratified epithelial cells, absorptive enterocytes from ileum and colon 12 , cholangiocytes 13 , myocardial cells, kidney proximal tubule cells, and bladder urothelial cells 10 . These findings indicated that those organs with high ACE2-expressing cells should be considered as potential high risk for 2019-nCoV infection 10 .In order to investigated the potential routes of 2019-nCov infection on the mucosa of oral cavity, we explored whether the ACE2 is expressed and the ACE2-expressing cell composition and proportion in oral cavity based on the public bulk RNA-seq profiles from two public databases and single-cell transcriptomes from an independent data generated in-house. The result showed that the ACE2 could be expressed in the oral cavity, and was highly enriched in epithelial cells. Moreover, among different oral sites, ACE2 expression was higher in tongue than buccal and gingival tissues. These findings indicate that the mucosa of oral cavity may be a potentially high risk route of 2019-nCov infection.Public bulk RNA-seq dataset analysis NA-seq profile data of 13 organs including 695 para-carcinoma normal tissues as control from public TCGA were obtained for our analysis, and Fig. 1a showed that ACE2 could be expressed in various organs, the mean expression of different organs could be found in Table 1 . According to the mean expression of ACE2, the mucosa of oral cavity could express ACE2, and the results were validated by the data of normal tissues from the FANTOM5 CAGE dataset (Fig. 1b) .To investigate the ACE2 expression on mucosa of oral cavity, we looked into the ACE2 expression in different oral sites. According to the site information provided by the TCGA, among the 32 adjacent normal tissues, 13 tissues located in the oral tongue, 2 tissues located in the base of tongue, 3 tissues located in the floor of mouse, and 14 tissues did not definite the site and were just put into the category of oral cavity. The mean expression distribution of different sites was shown in Fig. 1c . When we combined the base of tongue, floor of mouth and oral cavity as other sites, and compared them with oral tongue, we found the obvious tendency that the mean expression of ACE2 was higher in oral tongue (13 tissues) than others (19 tissues) (Fig. 1d ), while may due to the limitation of the sample size, the p value was not significant (P = 0.062).Single cell RNA-seq analysis of oral tissues Single cell RNA-seq was utilized for four oral tissues, and the data was analyzed to confirm the above results and assess the cell type-specific expression of ACE2. After the data preprocessing (shown in section \"Materials and methods\"), 22 969 cells were acquired and 7 cell types were identified (Fig. 2a) , including epithelial cells (marker genes including SFN, KRT6A, and KRT10), fibroblasts (marker genes including FAP, PDPN, COL1A2, DCN, COL3A1, COL6A1), T cells (marker genes including CD2, CD3D, CD3E, and CD3G), macrophages (marker genes including CD163, CSF1R, CD68, and FCGR2A), mast cells (marker genes including CMA1, MS4A2, TPSAB1, TPSB2), B cells (marker genes including SLAMF7, FCRL5, and CD79A) and endothelial cells (marker genes including PECAM1, VWF, and ENG). The heatmap of main cell markers across the cell types can be found in Fig. 2b .According to Fig. 2c , d, we confirmed the ACE2 was expressed in oral tissues (0.52% ACE2-positive cells), and higher in oral tongue than buccal and gingival tissues (95.86% ACE2-positive cells located in oral tongue). Figure 2e shows that the ACE2-positive cells could be found in oral tissues including epithelial cells (1.19% ACE2-positive cells), T cells (<0.5%), B cells (<0.5%), and fibroblast (<0.5%), and the ACE2 was highly enriched in epithelial cells, of which 93.38% ACE2positive cells belong to epithelial cells (Fig. 2f) . The above results indicated that the ACE2 could be expressed on the epithelial cells of the oral mucosa and highly enriched in tongue epithelial cells.In the last two decades, coronavirus has caused two large-scale pandemics, SARS in 2002 and the Middle East respiratory syndrome In this study, the analysis of public bulk-seq RNA datasets showed that the mucosa of oral cavity could express the ACE2 and was higher in tongue than other oral sites. The results of this study were consistent with the study of Zou et al. 10 in general, many organs with higher expression of ACE2 than lung, such as intestine, heart, and kidney. According to the study of Zhao et al. 11 , the ACE2 expression in lung is concentrated in a small population of type II alveolar cells (AT2), that may cause the relatively low ACE2 expression of lung in bulk-seq RNA datasets analysis. Even though, the result of Zou et al. indicated that the respiratory tract should also be considered as a vulnerable target to 2019-nCoV infection 10 .The results of our single cell RNA-seq profiles validated the ACE2 expression in oral cavity, and the level of ACE2 expression in oral tissues was higher in tongue than buccal or gingival tissues. Furthermore, we have also demonstrated that the ACE2-positive cells were enriched in epithelial cells, which was also reported by previous study 16 . These findings indicated that oral cavity could be regarded as potentially high risk for 2019-nCov infectious susceptibility.Interestingly, we found that the ACE2 also expressed in lymphocytes within oral mucosa, and similar results were found in various organs of the digestive system and in lungs 11, 12 . Whether those facts have reminded the 2019-nCoV attacks the lymphocytes and leads to the severe illness of patients needs more in vitro and in vivo evidence and validations, though the proportion of ACE2-positive lymphocytes is quite small.Previous studies have investigated the ACE2 mRNA and protein expression in various tissues by bulk samples 17, 18 , however, the distribution of ACE2 through bulk data could not indicate the cell type-specific expression of ACE2. Recently developed single-cell RNAsequencing technology enabled the generation of vast amounts of the transcriptomic data at cellular resolution 19 . The ACE2 expression profile in various organs, tissues, and cell types, provides the bioinformatics evidence for the potential infection routes of 2019-nCov, which might also be associated with presented symptoms.Although studies have reported multiple symptoms of hospitalized patients with 2019-nCoV infection 3, 20 , some cases at home might be asymptomatic. It is worth noting that, a previous study showed that 99% of the patients had no clinical manifestation of oral human papillomavirus (HPV), but HPV DNA was detected in 81% of oral mucosa samples, and anti-HPV IgA was detected in the saliva of 44% of the patients 21 . Likewise, although 2019-ncov infection hardly presented oral symptoms, the ACE2 expression in the oral cavity indicated that the oral infection route of 2019-nCoV cannot be excluded. Moreover, a latest pilot experiment showed that 4 out of 62 stool specimens tested positive to 2019-nCoV, and another four patients in a separate cohort who tested positive to rectal swabs had the 2019-nCoV being detected in the gastrointestinal tract, saliva, or urine 20 . Thus, our results support that in addition to the respiratory droplets and direct contact, fecal-oral transmission might also be the route of transmission of 2019-nCoV.Our results are mainly based on public datasets and single cell RNA-sequencing data of in-house oral tissues with minimal diseased lesion which from our previous project found no significant expression difference among the common epithelial markers in our past study and other previous study . It is warrant that further histological methods are used to confirm our results and enhance the persuasion of the conclusion.The ACE2-expressing cells in oral tissues, especially in epithelial cells of tongue, might provide possible routes of entry for the 2019-nCov, which indicate oral cavity might be a potential risk route of 2019-nCov infection. Those preliminary findings have explained the basic mechanism that the oral cavity is a potentially high risk for 2019-nCoV infectious susceptibility and provide a piece of evidence for the future prevention strategy in clinical practice as well as daily life.Bulk RNA-seq data of para-carcinoma normal tissues which were taken as control tissues in the studies were downloaded from The Cancer Genome Atlas (TCGA; ), and 695 para-carcinoma normal tissues distributed in different organs were obtained for this study, which included intestine (51 tissues), kidney (129 tissues), stomach (35 tissues), bile duct (9 tissues), liver (50 tissues), oral cavity (32 tissues), lung (110 tissues), thyroid (59 tissues), esophagus (11 tissues), bladder (19 tissues), breast (113 tissues), uterus (25 tissues), and prostate (52 tissues). The RNA-seq data were batch effects normalized and log2-transformed for the subsequent analysis. Violin plot was used to show the distribution of ACE2 expression among different organs, t test was performed to compare the ACE2 expression between two different groups shown in boxplot.Besides, Bulk RNA-seq data of normal tissues were downloaded from Functional Annotation of The Mammalian Genome Cap Analysis of Gene Expression (FANTOM5 CAGE) dataset 25 , as only two samples of this dataset which owns 60 samples in total located in the tongue, we just downloaded 14 organ types to validate the ACE2 expression in oral cavity with bar plot, including colon, ovary, breast, cerebellum, epididymis, esophagus, gallbladder, heart muscle, kidney, liver, lung, pancreas, prostate, and tongue.All analyses were performed in R (R version 3.6.0) and significant level was set as 0.05.Oral tissues of in-house cohort Due to our previous project about oral potential malignant disorders, four tissues of oral mucosa were obtained from patients after informed consent and ethical approval from West China Hospital of Stomatology, Sichuan University. These oral tissues had been sent for single cell RNA sequence. The four were taken from three patients with an average age of 50, which were all diagnosed as hyperkeratosis without dysplasia by pathologists, just showing an increase in cell number in the spinous layer and/or in the basal/ parabasal cell layers without cellular atypia, and its genetic profiles would be much more closed to normal tissue than malignant Single cell RNA-sequencing library preparation To generate single-cell Gel Beads-in-Emulsion (GEMs), the 10× chromium platform was used to capture and barcode cells. Cells were partitioned into GEMs along with gel beads coated with oligonucleotides and cDNAs with both barcodes were amplified, and a library was constructed using the 10× Genomics Chromium Single Cell Kit (v3 chemistry) for each sample. The resulting libraries were sequenced on an Illumina NovaSeq 6000 System.Single cell RNA-seq data preprocessing and analysis The FASTQ files were analyzed with the Cell Ranger Software Suite (version 3.1; 10× Genomics). The Seurat (version 3.0) was applied to read the gene-barcode matrix of four tissues. To control quality, we removed cells with <200 genes and 500 UMI counts, and as well as the cells with mitochondrial content higher than 5%. Besides, the genes detected in <3 cells were filtered out. The \"sctransform\" wrapper in Seurat was applied to normalize the data and remove confounding sources of variation, the \"IntegrateData\" was used for integrated the Seurat objects from four tissues. The uniform manifold approximation and projection (UMAP) was used for dimensionality reduction and clustering the cells, cell types were assigned based on their canonical markers. UMAP plots, heatmap, and violin plots were generated with Seurat in R.\n",
            "\t0.792\tSince December 2019, an increasing number of patients with pneumonia occurred in Wuhan, Hubei province, China, which attracted much attention not only within China but across the world 1,2 . The novel pneumonia was named as Corona Virus by World Health Organization (WHO) (), the common symptoms of COVID-19 at illness onset were fever, fatigue, dry cough, myalgia, and dyspnea 3 . In addition, some patients might suffer from headache, dizziness, abdominal pain, diarrhea, nausea, and vomiting 3 . Onset of disease may lead to progressive respiratory failure due to alveolar damage and even death 4 .Scientists then isolated a novel coronavirus from human airway epithelial cells, which was named 2019-nCoV 5 . Lu et al. 6 found that 2019-nCoV was closer to bat-SL-CoVZC45 and bat-SL-CoVZXC21 at the whole-genome level, and the external subdomain of the 2019-nCoV receptor-binding domain (RBD) was more similar to that of severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV). Study of Zhou et al. 4 indicated that the angiotensin-converting enzyme II (ACE2) is likely the cell receptor of 2019-nCoV, which were also the receptor for SARS-CoV and HCoV-NL63 7,8 . Zhou et al. 4 also proved that 2019-nCoV does not use other coronavirus receptors, aminopeptidase N, and dipeptidyl peptidase 4. The study of Xu et al. 9 found that the RBD domain of the 2019-nCoV S-protein supports strong interaction with human ACE2 molecules. These findings suggest that the ACE2 plays an important role in cellular entry, thus ACE2-expressing cells may act as target cells and are susceptible to 2019-nCoV infection 10 .The expression and distribution of the ACE2 in human body may indicate the potential infection routes of 2019-nCoV. Through the developed single-cell RNA sequencing (scRNA-Seq) technique and single-cell transcriptomes based on the public database, researchers analyzed the ACE2 RNA expression profile at single-cell resolution. High ACE2 expression was identified in type II alveolar cells (AT2) of lung , esophagus upper and stratified epithelial cells, absorptive enterocytes from ileum and colon 12 , cholangiocytes 13 , myocardial cells, kidney proximal tubule cells, and bladder urothelial cells 10 . These findings indicated that those organs with high ACE2-expressing cells should be considered as potential high risk for 2019-nCoV infection 10 .In order to investigated the potential routes of 2019-nCov infection on the mucosa of oral cavity, we explored whether the ACE2 is expressed and the ACE2-expressing cell composition and proportion in oral cavity based on the public bulk RNA-seq profiles from two public databases and single-cell transcriptomes from an independent data generated in-house. The result showed that the ACE2 could be expressed in the oral cavity, and was highly enriched in epithelial cells. Moreover, among different oral sites, ACE2 expression was higher in tongue than buccal and gingival tissues. These findings indicate that the mucosa of oral cavity may be a potentially high risk route of 2019-nCov infection.Public bulk RNA-seq dataset analysis NA-seq profile data of 13 organs including 695 para-carcinoma normal tissues as control from public TCGA were obtained for our analysis, and Fig. 1a showed that ACE2 could be expressed in various organs, the mean expression of different organs could be found in Table 1 . According to the mean expression of ACE2, the mucosa of oral cavity could express ACE2, and the results were validated by the data of normal tissues from the FANTOM5 CAGE dataset (Fig. 1b) .To investigate the ACE2 expression on mucosa of oral cavity, we looked into the ACE2 expression in different oral sites. According to the site information provided by the TCGA, among the 32 adjacent normal tissues, 13 tissues located in the oral tongue, 2 tissues located in the base of tongue, 3 tissues located in the floor of mouse, and 14 tissues did not definite the site and were just put into the category of oral cavity. The mean expression distribution of different sites was shown in Fig. 1c . When we combined the base of tongue, floor of mouth and oral cavity as other sites, and compared them with oral tongue, we found the obvious tendency that the mean expression of ACE2 was higher in oral tongue (13 tissues) than others (19 tissues) (Fig. 1d ), while may due to the limitation of the sample size, the p value was not significant (P = 0.062).Single cell RNA-seq analysis of oral tissues Single cell RNA-seq was utilized for four oral tissues, and the data was analyzed to confirm the above results and assess the cell type-specific expression of ACE2. After the data preprocessing (shown in section \"Materials and methods\"), 22 969 cells were acquired and 7 cell types were identified (Fig. 2a) , including epithelial cells (marker genes including SFN, KRT6A, and KRT10), fibroblasts (marker genes including FAP, PDPN, COL1A2, DCN, COL3A1, COL6A1), T cells (marker genes including CD2, CD3D, CD3E, and CD3G), macrophages (marker genes including CD163, CSF1R, CD68, and FCGR2A), mast cells (marker genes including CMA1, MS4A2, TPSAB1, TPSB2), B cells (marker genes including SLAMF7, FCRL5, and CD79A) and endothelial cells (marker genes including PECAM1, VWF, and ENG). The heatmap of main cell markers across the cell types can be found in Fig. 2b .According to Fig. 2c , d, we confirmed the ACE2 was expressed in oral tissues (0.52% ACE2-positive cells), and higher in oral tongue than buccal and gingival tissues (95.86% ACE2-positive cells located in oral tongue). Figure 2e shows that the ACE2-positive cells could be found in oral tissues including epithelial cells (1.19% ACE2-positive cells), T cells (<0.5%), B cells (<0.5%), and fibroblast (<0.5%), and the ACE2 was highly enriched in epithelial cells, of which 93.38% ACE2positive cells belong to epithelial cells (Fig. 2f) . The above results indicated that the ACE2 could be expressed on the epithelial cells of the oral mucosa and highly enriched in tongue epithelial cells.In the last two decades, coronavirus has caused two large-scale pandemics, SARS in 2002 and the Middle East respiratory syndrome In this study, the analysis of public bulk-seq RNA datasets showed that the mucosa of oral cavity could express the ACE2 and was higher in tongue than other oral sites. The results of this study were consistent with the study of Zou et al. 10 in general, many organs with higher expression of ACE2 than lung, such as intestine, heart, and kidney. According to the study of Zhao et al. 11 , the ACE2 expression in lung is concentrated in a small population of type II alveolar cells (AT2), that may cause the relatively low ACE2 expression of lung in bulk-seq RNA datasets analysis. Even though, the result of Zou et al. indicated that the respiratory tract should also be considered as a vulnerable target to 2019-nCoV infection 10 .The results of our single cell RNA-seq profiles validated the ACE2 expression in oral cavity, and the level of ACE2 expression in oral tissues was higher in tongue than buccal or gingival tissues. Furthermore, we have also demonstrated that the ACE2-positive cells were enriched in epithelial cells, which was also reported by previous study 16 . These findings indicated that oral cavity could be regarded as potentially high risk for 2019-nCov infectious susceptibility.Interestingly, we found that the ACE2 also expressed in lymphocytes within oral mucosa, and similar results were found in various organs of the digestive system and in lungs 11, 12 . Whether those facts have reminded the 2019-nCoV attacks the lymphocytes and leads to the severe illness of patients needs more in vitro and in vivo evidence and validations, though the proportion of ACE2-positive lymphocytes is quite small.Previous studies have investigated the ACE2 mRNA and protein expression in various tissues by bulk samples 17, 18 , however, the distribution of ACE2 through bulk data could not indicate the cell type-specific expression of ACE2. Recently developed single-cell RNAsequencing technology enabled the generation of vast amounts of the transcriptomic data at cellular resolution 19 . The ACE2 expression profile in various organs, tissues, and cell types, provides the bioinformatics evidence for the potential infection routes of 2019-nCov, which might also be associated with presented symptoms.Although studies have reported multiple symptoms of hospitalized patients with 2019-nCoV infection 3, 20 , some cases at home might be asymptomatic. It is worth noting that, a previous study showed that 99% of the patients had no clinical manifestation of oral human papillomavirus (HPV), but HPV DNA was detected in 81% of oral mucosa samples, and anti-HPV IgA was detected in the saliva of 44% of the patients 21 . Likewise, although 2019-ncov infection hardly presented oral symptoms, the ACE2 expression in the oral cavity indicated that the oral infection route of 2019-nCoV cannot be excluded. Moreover, a latest pilot experiment showed that 4 out of 62 stool specimens tested positive to 2019-nCoV, and another four patients in a separate cohort who tested positive to rectal swabs had the 2019-nCoV being detected in the gastrointestinal tract, saliva, or urine 20 . Thus, our results support that in addition to the respiratory droplets and direct contact, fecal-oral transmission might also be the route of transmission of 2019-nCoV.Our results are mainly based on public datasets and single cell RNA-sequencing data of in-house oral tissues with minimal diseased lesion which from our previous project found no significant expression difference among the common epithelial markers in our past study and other previous study . It is warrant that further histological methods are used to confirm our results and enhance the persuasion of the conclusion.The ACE2-expressing cells in oral tissues, especially in epithelial cells of tongue, might provide possible routes of entry for the 2019-nCov, which indicate oral cavity might be a potential risk route of 2019-nCov infection. Those preliminary findings have explained the basic mechanism that the oral cavity is a potentially high risk for 2019-nCoV infectious susceptibility and provide a piece of evidence for the future prevention strategy in clinical practice as well as daily life.Bulk RNA-seq data of para-carcinoma normal tissues which were taken as control tissues in the studies were downloaded from The Cancer Genome Atlas (TCGA; ), and 695 para-carcinoma normal tissues distributed in different organs were obtained for this study, which included intestine (51 tissues), kidney (129 tissues), stomach (35 tissues), bile duct (9 tissues), liver (50 tissues), oral cavity (32 tissues), lung (110 tissues), thyroid (59 tissues), esophagus (11 tissues), bladder (19 tissues), breast (113 tissues), uterus (25 tissues), and prostate (52 tissues). The RNA-seq data were batch effects normalized and log2-transformed for the subsequent analysis. Violin plot was used to show the distribution of ACE2 expression among different organs, t test was performed to compare the ACE2 expression between two different groups shown in boxplot.Besides, Bulk RNA-seq data of normal tissues were downloaded from Functional Annotation of The Mammalian Genome Cap Analysis of Gene Expression (FANTOM5 CAGE) dataset 25 , as only two samples of this dataset which owns 60 samples in total located in the tongue, we just downloaded 14 organ types to validate the ACE2 expression in oral cavity with bar plot, including colon, ovary, breast, cerebellum, epididymis, esophagus, gallbladder, heart muscle, kidney, liver, lung, pancreas, prostate, and tongue.All analyses were performed in R (R version 3.6.0) and significant level was set as 0.05.Oral tissues of in-house cohort Due to our previous project about oral potential malignant disorders, four tissues of oral mucosa were obtained from patients after informed consent and ethical approval from West China Hospital of Stomatology, Sichuan University. These oral tissues had been sent for single cell RNA sequence. The four were taken from three patients with an average age of 50, which were all diagnosed as hyperkeratosis without dysplasia by pathologists, just showing an increase in cell number in the spinous layer and/or in the basal/ parabasal cell layers without cellular atypia, and its genetic profiles would be much more closed to normal tissue than malignant Single cell RNA-sequencing library preparation To generate single-cell Gel Beads-in-Emulsion (GEMs), the 10× chromium platform was used to capture and barcode cells. Cells were partitioned into GEMs along with gel beads coated with oligonucleotides and cDNAs with both barcodes were amplified, and a library was constructed using the 10× Genomics Chromium Single Cell Kit (v3 chemistry) for each sample. The resulting libraries were sequenced on an Illumina NovaSeq 6000 System.Single cell RNA-seq data preprocessing and analysis The FASTQ files were analyzed with the Cell Ranger Software Suite (version 3.1; 10× Genomics). The Seurat (version 3.0) was applied to read the gene-barcode matrix of four tissues. To control quality, we removed cells with <200 genes and 500 UMI counts, and as well as the cells with mitochondrial content higher than 5%. Besides, the genes detected in <3 cells were filtered out. The \"sctransform\" wrapper in Seurat was applied to normalize the data and remove confounding sources of variation, the \"IntegrateData\" was used for integrated the Seurat objects from four tissues. The uniform manifold approximation and projection (UMAP) was used for dimensionality reduction and clustering the cells, cell types were assigned based on their canonical markers. UMAP plots, heatmap, and violin plots were generated with Seurat in R.\n",
            "\t0.686\tThe geographic spread of 2019 novel coronavirus infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.As of 31 January 2020, mainland China reported 11,791 confirmed cases of novel coronavirus infections, causing 259 deaths . Initially, these infections were thought to result from zoonotic (animal-to-human) transmission; however, recently published evidence and the exponential growth of case incidence show compelling evidence of human-to-human secondary transmission fueled by travel, with many cases detected in other parts of the world . This geographic expansion beyond the initial epicenter of Wuhan provides an opportunity to study the natural history of COVID-19 infection, as these migration events limit the risk of infection to the time during which an individual traveled to an area where exposure could occur .The incubation period is defined as the time from infection to illness onset. Knowledge of the incubation period of a directly transmitted infectious disease is critical to determine the time period required for monitoring and restricting the movement of healthy individuals (i.e., the quarantine period) [5, 6] . The incubation period also aids in understanding the relative infectiousness of COVID-19 and can be used to estimate the epidemic size .Time-delay distributions including dates of hospital admission (for treatment and/or isolation) and death also inform the temporal dynamics of epidemics. A published clinical study on the COVID-19 epidemic has already shown that the average time delay from illness onset to hospital admission is approximately 7 days , but this distribution has yet to be explicitly estimated. The time from hospital admission to death is also critical to the avoidance of underestimation when calculating case fatality risk . Using publicly available data from the ongoing epidemic with known case event dates, the present study aimed to estimate the incubation period and other time intervals that govern the interpretation of epidemiological dynamics of COVID-19 infections.We retrieved information on cases with confirmed COVID-19 infection and diagnosis outside of the epicenter of Wuhan, China, based on official reports from governmental institutes, as well as reports on deceased cases from both in and outside of Wuhan. We aggregated the data directly from government websites or from news sites that quoted government statements. The data were collected in real time, and thus may have been updated as more details on cases became available. The arranged data include a selection of cases reported through 31 January 2020 and are available as Supplementary Tables S1 and S2 .Specifically, we collected the dates of exposure (entry and/or exit from Wuhan or dates of close contact with a Wuhan resident/known epidemic case), illness onset, earliest healthcare seeking related to infection, hospital admission (for treatment and/or isolation), and death. Cases included both residents from other locations who travelled to Wuhan, as well as individuals who lived, worked, or studied in Wuhan (hereafter: Wuhan residents) but who were diagnosed outside of Wuhan and reported by the governments of the locations where their infection was detected. We thus estimated the incubation period by (i) excluding Wuhan residents and (ii) including Wuhan residents. The former may be more precise in defining the interval of exposure, but the sample size is greater for the latter. More detailed information about the criteria used for the estimation of each defined time interval and the data used are described in Supplementary Text S1.We used the dates of three critical points in the course of infection-illness onset, hospital admission, and death-to calculate four time intervals: the time from (a) exposure to illness onset (i.e., the incubation period), (b) illness onset to hospital admission, (c) illness onset to death, and (d) hospital admission to death. We used a doubly interval-censored likelihood function to estimate the parameter values for these intervals, written as:Here, in the case of (a) g(.) is the probability density function (PDF) of exposure following a uniform distribution, and f (.) is the PDF of the incubation period independent of g(.). D represents a dataset among all observed cases i, where exposure and symptom onset fall within the lower and upper bounds (E L , E R ) and (S L , S R ). We fit the PDF f (.) to lognormal, Weibull, and gamma distributions.To address the selection bias in the dataset due to the continued growth of the outbreak (i.e., cases with shorter incubation periods are more likely to be included in the dataset), we also accounted for right truncation using the formula:Here, r is the exponential growth rate (estimated at 0.14 ), T is the latest time of observation (31 January 2020), and F(.) is the cumulative density function of f (.).In both cases, we used Bayesian methods to infer parameter estimates and obtain credible intervals. We selected the best fit model by using the widely applicable information criterion (WAIC). We also verified that the Bayesian estimates were in line with pointwise estimates derived by maximum likelihood estimation (MLE). As the formulation of the likelihood with right truncation (1)- (2) contained the function f and was dependent on both the time interval (s − e) and time of exposure e, we generalized a previously obtained result for doubly interval-censored likelihood with f (s − e, e) ≡ f (s − e) .The data were processed using R version 3.6.2 , MLE was computed using Julia version 1.3 , and the Markov chain Monte Carlo (MCMC) simulations were performed in Stan (cmdStan version 2.22.1 ). All code is freely available at the github repository:  WuhanIncubationPeriod2020.The ratio of male to female cases among living cases resembled , at 58%, with most 30-59 years of age (information missing for 9 cases). The deceased cases were more predominantly male (70%) and older (85% were 60 years of age or older). Table 1 shows estimates for the various time intervals without right truncation. For the incubation period estimates, the lognormal distribution provided the best fit to the data, both when excluding and including Wuhan residents. The mean incubation period was estimated at 5.0 days (95% credible interval [CI]: 4.2, 6.0) when excluding Wuhan residents (n = 52) and 5.6 days (95% CI: 5.0, 6.3) when including Wuhan residents (n = 158).The median time from illness onset to hospital admission was estimated at 3.3 days (95% CI: 2.7, 4.0) among living cases and 6.5 days (95% CI: 5.2, 8.0) among deceased cases using the gamma distribution, which provided the best fit for both sets of data. Figure 1A shows the corresponding PDFs. Data from the time from illness onset and hospital admission to death best fit lognormal and Weibull distributions, respectively, as presented in Figure 1B , C. The mean time from illness onset to death was 15.0 days (95% CI: 12.8, 17.5) and from hospital admission to death was 8.8 days (95% CI: 7.2, 10.8). Table 2 shows estimates for the fit of the lognormal distribution for each interval when accounting for right truncation. The mean incubation period was 5.6 days (95% CI: 4.4, 7.4) when excluding Wuhan residents-slightly larger than the estimate without right truncation. The mean estimate for illness onset to hospital admission was 9.7 days (95% CI: 5.4, 17.0) for living cases and 6.6 days (95% CI: 5.2, 8.8) for deceased cases, with the former nearly 2.5 times the length of its untruncated version. Illness onset to death and hospital admission to death were likewise longer than their non-truncated counterparts, at 20.2 days (95% CI: 15.1, 29.5) and 13.0 days (95% CI: 8.7, 20.9), respectively. Figure 2 shows the cumulative distribution function of the incubation period with and without right truncation. The 5th and 95th percentiles are shown in addition to the median. The 95th percentiles were estimated at 10.6 days (95% CI: 8.5, 14.1) and 10.8 days (95% CI: 9.3, 12.9) for non-truncated data excluding and including Wuhan residents and 12.3 days (95% CI: 9.1, 19.8) when applying right truncation and excluding Wuhan residents. The respective median values for these CDFs were 4.3 days (95% CI: 3.5, 5.1), 5.0 days (95% CI: 4.4, 5.6), and 4.6 days (95% CI: 3.7, 5.7). The present study advances the public discussion on COVID-19 infections by presenting explicit estimations of the incubation period and other epidemiologic characteristics using publicly available data. Our estimated mean incubation period of approximately 5 days is comparable to known mean values of the incubation period for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) [9, , as well as other recent estimates of the incubation period for COVID-19 . In addition to empirically showing the comparability of COVID-19 to other disease-causing coronaviruses, the present study has also shown that the 95th percentile of the incubation period is around 10-14 days, indicating that a 14-day quarantine period would largely ensure the absence of disease among healthy exposed individuals.Wuhan residents have a less precisely defined exposure period compared to travelers and secondary cases from known human to human transmission events. However, our calculations have shown that adding more cases to the dataset even with uncertainty reduces both the variance The present study advances the public discussion on COVID-19 infections by presenting explicit estimations of the incubation period and other epidemiologic characteristics using publicly available data. Our estimated mean incubation period of approximately 5 days is comparable to known mean values of the incubation period for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) [9, , as well as other recent estimates of the incubation period for COVID-19 . In addition to empirically showing the comparability of COVID-19 to other disease-causing coronaviruses, the present study has also shown that the 95th percentile of the incubation period is around 10-14 days, indicating that a 14-day quarantine period would largely ensure the absence of disease among healthy exposed individuals.Wuhan residents have a less precisely defined exposure period compared to travelers and secondary cases from known human to human transmission events. However, our calculations have shown that adding more cases to the dataset even with uncertainty reduces both the variance The present study advances the public discussion on COVID-19 infections by presenting explicit estimations of the incubation period and other epidemiologic characteristics using publicly available data. Our estimated mean incubation period of approximately 5 days is comparable to known mean values of the incubation period for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) [9, , as well as other recent estimates of the incubation period for COVID-19 . In addition to empirically showing the comparability of COVID-19 to other disease-causing coronaviruses, the present study has also shown that the 95th percentile of the incubation period is around 10-14 days, indicating that a 14-day quarantine period would largely ensure the absence of disease among healthy exposed individuals.Wuhan residents have a less precisely defined exposure period compared to travelers and secondary cases from known human to human transmission events. However, our calculations have shown that adding more cases to the dataset even with uncertainty reduces both the variance of the estimates and selection bias, improving the fit of the distribution mean. Our estimates are in agreement with the report of Li et al. . A recent study by Backer et al. noted a similar finding in their analysis of the incubation period for 88 cases (including 63 Wuhan residents). However, the estimates of Backer et al. for the model that included Wuhan residents were subject to overestimation as the lower bounds for Wuhan residents-who had unknown left exposure dates-were fixed in their analysis. In contrast, we considered the left exposure dates for Wuhan residents as parameters to be fitted-see for details. Notably, our results demonstrated the overall benefit of using additional case data, even when some of exposure values were not precisely known.The time from the illness onset to death is also comparable to SARS , and the 15-20-day mean delay indicates that a crude estimation of the ratio of the cumulative number of deaths to that of confirmed cases will tend to result in an underestimation of the case fatality risk, especially during the early stage of epidemic spread [18, 19] . During the SARS epidemic in Hong Kong, 2003, the time from illness onset to hospital admission was shown to have shortened as a function of the calendar time, gradually reflecting the effects of contact tracing . It remains to be seen if this will be the case for COVID-19 as well. The time delay distribution between illness onset and hospital admission may also be negatively associated with the basic reproduction number, i.e., the average number of secondary cases generated by a single primary case in a fully susceptible population .The median time from illness onset to hospital admission was approximately 4 days among cases not known to be deceased at the time of the case report, and 6 days among cases reported as deceased. The reasons for this difference are not altogether clear. However, the living cases include persons who were isolated-in some cases more for reducing transmission than for treatment purposes-while all deceased cases were admitted for treatment. In addition, deceased cases for whom information was available had onset dates closer to the beginning of the outbreak compared to the living cases, who mostly had onset in the latter two-thirds of January 2020. The time delay distributions from illness onset to hospital admission for cases reported later in the epidemic, when there was a more widespread recognition of the virus and a more prevalent social imperative for those with symptoms to seek healthcare, may differ from those of early cases .Several limitations of the present study exist. First, the dataset relies on publicly available information that is not uniformly distributed (i.e., collected from various sources), and therefore the availability of dates relevant to our analyses is limited to a small, selective sample that is not necessarily generalizable to all confirmed cases. Moreover, given the novelty of the COVID-19 pneumonia, it is possible that illness onset and other event data were handled differently between jurisdictions (e.g., was illness onset the date of fever or date of dyspnea?). Second, our data include very coarse date intervals with some proxy dates used to determine the left and/or right hand dates of some intervals. Third, as the sample size was limited, the variance is likely to be biased. Fourth, we were not able to examine the heterogeneity of estimates by different attributes of cases (e.g., severity of disease) . Lastly, as we only have information on confirmed cases, there is a bias towards more severe disease-particularly for earlier cases.This study presents the estimates of epidemiological characteristics of COVID-19 infections that are key parameter for studies on incidence, case fatality, and epidemic final size, among other possibilities [7, 11] . From the 95th percentile estimate of the incubation period we found that the length of quarantine should be at least 14 days, and we stress that the 17-24-day time delay from illness onset to death must be addressed when estimating COVID-19 case fatality risk. This study was made possible only through open sharing of case data from China and other countries where cases were diagnosed. Continued communication of dates and other details related to exposure and infection is crucial to furthering scientific understanding of the virus, the infections it causes, and preventive measures that can be used to contain and mitigate epidemic spread.Supplementary Materials: The following are available online at , Table S1 : Event dates for exported cases included in the analysis, Table S2 : Event dates for deceased cases included in the analysis, Text S1, Estimation of the time interval distribution using doubly interval-censored likelihood, estimation of the time interval distributions using Bayesian framework, and data cleaning rules implemented for the various time intervals. The authors declare no conflicts of interest.\n",
            "\n",
            " Title:  Clinical Medicine Optimization Method for Forecasting Confirmed Cases of COVID-19 in China\n",
            "6 \b. \n",
            "\n",
            "Input question: How does coronavirus spread?\n",
            "Results (after 2.230 seconds):\n",
            "Top-5 Bi-Encoder Retrieval hits \n",
            "\n",
            "\t0.651\tCorona viruses cause common cold, and infections caused by corona viruses are generally self-resolving. During the last 4 years, corona viruses have become the most important viruses worldwide because of the occurrence of several recent deaths caused by corona viruses in Saudi Arabia. Spread of the infection occurred worldwide; however, most cases of mortality have occurred in the Middle East. Owing to the predominance of outbreaks in the Middle Eastern countries, the virus was renamed a Middle East respiratory syndrome corona virus (MERS-CoV) by the Corona virus Study Group. The Center for Diseases Control and Prevention and World Health Organization maintain a website that is updated frequently with new cases of MERS-CoV infection. In this review, we describe the history and epidemiology of this novel virus. Studies of the genetics and molecular mechanisms of this virus are expected to facilitate the development of vaccines in the future.publications and the risk of bias of included studies, and extracted data (Table 1) .The authors searched the topic through the following steps a) Search in Google. b) Search in PubMed c) Evaluated the papers, divided them depending on the inclusion and exclusion criteria. d) Evaluated the articles and analyzed them.We searched through Google, Pub Med, Pub Med Central, CAS, Citebase, DOAJ, Embase, Embiology, MED-LINE, OAIster, SCImago, Scopus, SOCOLAR and Zetoc by using the keywords.We used the ROBIS tool and guidance which was available in the website () and (). The risk of bias was low ( Table 3) .Corona virus was first identified as a cause of the common cold in 1960. In one study carried out in Canada in 2001, more than 500 patients presented with flu-like symptoms. Virological analyses showed that 3.6% of these cases were positive for the HCoV-NL63 strain by polymerase chain reaction (PCR). Until 2002, corona virus was considered a relatively simple, nonfatal virus; however, an outbreak in in Guangdong province in China, which resulted in spread to many other countries, including Thailand, Vietnam, Taiwan, Hong Kong Singapore, and the United States of America, caused severe acute respiratory syndrome (SARS) and high mortality rates in over 1000 patients. After this outbreak, microbiologists and infectious disease experts focused on the understanding the pathogenesis of the disease and discovered that this infection was caused by a new form of corona virus. A total of 8096 individuals were infected with this virus, resulting in 774 deaths; thus, in 2004, the Centers for Disease Control and Prevention (CDC) and World Health Organization (WHO) declared a state of emergency . In another report from Hong Kong, 50 patients presented with SARS, and more than 60% of these patients were positive for corona virus . The evolution of this virus demonstrated that coronavirus is not a stable virus and can adapt to become more virulent, even lethal, to humans. Indeed, another outbreak in Saudi Arabia in 2012 resulted in many deaths and spread first to other countries in the Middle East and then worldwide, resulting in renewed interest in studies of this new form of coronavirus ( Fig. 1) .Corona virus is a single-stranded, enveloped RNA virus 1 that is spherical or pleomorphic in shape with bear's club-shaped glycoprotein projections. There are subtypes of corona virus, alpha corona virus, beta corona virus, gamma corona virus, and delta corona virus, and each subtype has many serotypes. For example, OC43-like and 229E-like have been shown to affect humans, whereas the other types mainly affect animals (Fig. 2) . Corona viruses are transmitted via airborne zoonotic droplets, and viral replication occurs in the ciliated epithelium, resulting in cellular damage and inflammatory reactions at the site of infection [3, 4] . In addition to humans, corona viruses are also found in bats, whales, pigs, birds, cats, dogs, and mice . The median incubation period for MERS-CoV is 5.2-12 days, and adults are mostly affected. However, several pediatric cases have been reported in Saudi Arabia. The clinical presentation of MERS-CoV ranges from flu-like symptoms, i.e., fever and cough in 87% of patients, chills, rigor, rhinorrhea, myalgia, and fatigue, to more severe symptoms, including shortness of breath in 48% of patients and respiratory failure, resulting in the requirement for intubation and ventilation. Gastrointestinal symptoms, including nausea, vomiting, diarrhea, and abdominal pain, have also been reported in about 35% of cases, and acute renal failure necessitating hemodialysis has been seen. On physical examination, patients generally present with fever and pulmonary findings, such as rhonchi and crepitation. Laboratory investigation may revealed lymphopenia, thrombocytopenia, disseminated intravascular coagulation (DIC), and multi-organ failure, which can result in death. The CDC recommends collecting multiple specimens at different times from different sites; for example, samples should be collected from oropharyngeal and nasopharyngeal swabs, sputum, blood, and stool and rectal swabs, as well as the lower respiratory tract, which is most frequently positive for the virus.Typically, samples will be processed by real-time reverse transcription (rRT)-PCR to analyze the following sequences: upstream of the E protein gene (upE), open reading frame 1b (ORF 1b), and open reading frame (ORF 1a), which is the most sensitive sequence for identification of the specific virus. RNA dependent RNA polymerase (RdRp) and N genes are also target sites in the MERS-CoV genome. To confirm the diagnosis, one of the following criteria must be present: (1) a positive PCR result for at least two different specific targets in the MERS-CoV genome or (2) one positive PCR result for a specific target on the MERS-CoV genome and an additional different PCR product confirming a known sequence of MERS-CoV. The algorithm for diagnosis is as follows: (A) upE-specific rRT-PCRif positive, confirm with ORF 1a rRT-PCR analysis, and, if positive, the case is confirmed; (B) upE-specific rRT-PCRif positive, confirm with sequencing of one of the two target sites (RdRp or N assay), and, if positive, the case is confirmed. False negative results can occur due to issues with specimen collection, including early or late collection or shipping/handling problems .In June 13, 2012, the first reported case of MERS-CoV 1 occurred in Jeddah, Saudi Arabia. This outbreak resulted in many cases of infection, particularly among health care workers who were in direct contact with the patients, indicating that the virus could be transmitted from human to human via air droplets. This prompted strict contact precautions, such as isolation and the use of personal protective equipment, i.e., gloves, N 95-typemasks, and gowns. In Saudi Arabia, many cases of MERS-CoV were reported in almost all provinces, with Jeddah, Makka, Riyadh, and Al-Hassa being the most commonly affected cities. Additionally, many outbreaks were reported in hospitals, causing affected hospitals to close down and not to accept any infected patients. Most of the patients who died were immunocompromised, having conditions such as chronic renal failure, congestive heart failure, and diabetes or having recently received organ transplants (e.g., kidney transplants). After the initial outbreak in Saudi Arabia, MERS-CoV was reported in several other countries, including Qatar, Bahrain, Kuwait, Jordan, and Tunisia. According to the Minister of Health in Saudi Arabia, from June 13, 2012 until December 2015, a total of 1227 cases of MERS-CoV have been reported, with 728 recovered, one still under treatment, and 549 expired due to MERS-CoV-related symptoms. The Minister of Health, primary healthcare facilities, and public health officials immediately sought to educate the population in Saudi Arabia through internet, lectures, and brochures and continued to carefully record each new case in order to improve knowledge and therapeutic strategies for this virus. The latest report from the WHO on December 7, 2015 showed that MERS-COV has been identified in 26 countries, with 1621 confirmed cases and 584 deaths globally. Notably, the highest numbers of both Corona virus as known to be a zoonotic virus; however, the MERS-CoV is a novel virus, and whether zoonotic transmission occurs is not clear yet. International studies carried out from 2012 to 2014 in Mexico, European countries (i.e., Germany, Ukraine, the Netherlands, and Romania), Ghana, and South Africa have examined whether bats may be carriers of MERS-CoV. These studies have tested bats mainly for the 329-bp fragment of RdRp using blood, fecal, and oral samples ( Table 2 ). The bat species that were tested in these studies included Pipistrelluspipistrellus, P. nathusii, P. pygmaeus, Nycteris, and Neoromiciazuluensis, and 5.3-24.9% were found to be positive for MERS-CoV, with most positive results (> 70%) being identified in fecal samples with high viral loads . Thus, it may be possible for transmission to occur via bats; however, in Saudi Arabia, the species of bats that patients may have come in contact with are different from those tested, including Rhinopomahardwickii, Rhinopomamicrophyllum, Taphozousperforatus, P. kuhlii, Eptesicusbottae, Eidolon helvum, and Rosettusaegyptiacus. Thus, although there was a positive association between bats and corona virus infection, there was no association between bats and MERS-CoV. Therefore, these data have suggested that MERS-CoV is not transmitted through bats .Researchers have also examined whether camels may be linked to the outbreak of MERS-CoV in Saudi Arabia. Studies have been carried out in many Middle Eastern countries, including Saudi Arabia, Qatar, Egypt, United Arab Emirates, and Oman, using samples from lung, nasal, and rectal swabs. Positivity for MERS-CoV by RT-PCR for the RdRpwas observed in 1.6-61.5% of samples, mostly lung and nasal swabs (Table 3) . Analyses using anti-MERS-CoV antibodies have shown that 98-100% of camels are positive for MERS-CoV; consistent with this, the incidence of MERS-CoV in humans is 15 times higher in camel shepherds and 23 times higher in slaughterhouse workers than in the general population. Therefore, these data supported that the main route of transmission from camels to humans is through the respiratory system .The main treatment strategy for typical corona virus infection is supportive therapy, in deeding administration of antipyretics and analgesics, maintenance of hydration, respiratory support by either mechanical ventilation or extracorporeal membrane oxygenation (ECMO), and treatment with antibiotics in the case of bacterial super infections. However, such treatments may not be sufficient for MERS-CoV infections, which may be more severe. Ribavirin and interferon alpha have been shown to have synergistic effects and are more beneficial when The main aim of this study was to report MERS-CoV antibodies; however, the virus itself was not tested (range: 98-100%)\n",
            "\t0.618\tCoronaviruses are enveloped positive-stranded RNA viruses that replicate in the cytoplasm. To deliver their nucleocapsid into the host cell, they rely on the fusion of their envelope with the host cell membrane. The spike glycoprotein (S) mediates virus entry and is a primary determinant of cell tropism and pathogenesis. It is classified as a class I fusion protein, and is responsible for binding to the receptor on the host cell as well as mediating the fusion of host and viral membranes-A process driven by major conformational changes of the S protein. This review discusses coronavirus entry mechanisms focusing on the different triggers used by coronaviruses to initiate the conformational change of the S protein: receptor binding, low pH exposure and proteolytic activation. We also highlight commonalities between coronavirus S proteins and other class I viral fusion proteins, as well as distinctive features that confer distinct tropism, pathogenicity and host interspecies transmission characteristics to coronaviruses.Although the first member of the coronavirus family was discovered in the 1930s coronaviruses gained particular notoriety when the severe acute respiratory syndrome (SARS) outbreak shook the world in 2002-2003. Interest in this family of viruses grew in the aftermath of this epidemic, leading to the identification of many new family members. This episode also shed light on the capabilities of coronaviruses to jump across species. Before gaining importance for public health in 2003, the diseases associated with coronaviruses were mainly of veterinary interest. Coronaviruses infect a wide variety of mammals and birds, causing respiratory and enteric diseases and, in some rarer cases, hepatitis and neurologic disease. Infection can be acute or persistent .Coronaviruses are classified in four different genera, historically based on serological analysis and now on genetic studies: alpha-, beta-, gamma-, and delta-CoV (Table 1) . Coronaviruses belong to the Coronavirinae subfamily that together with Torovirinae form the Coronaviridae family in the Nidovirales order.Coronaviruses are enveloped, spherical or pleiomorphic viruses, with typical sizes ranging from 80 to 120 nm. They possess a 5' capped, single-strand positive sense RNA genome, with a length between 26.2 and 31.7 kb, the longest amongst all RNA viruses. The genome is composed of six to ten open reading frames (ORFs). The first ORF comprises two-thirds of the genome and encodes the replicase proteins, whereas the last third contains the structural protein genes in a fixed order: (HE)-S-E-M-N ( Figure 1A ). Variable numbers of ORF encoding accessory proteins are present between these genes. The genome is packaged into a helical nucleocapsid surrounded by a host-derived lipid bilayer. The virion envelope contains at least three viral proteins, the spike protein (S), the membrane protein (M) and the envelope protein (E) ( Figure 1B ). In addition, some coronaviruses also contain a hemagglutinin esterase (HE). Whereas the M and E proteins are involved in virus assembly, the spike protein is the leading mediator of viral entry. The spike protein is also the principal player in determining host range [3, 4] .Viral entry relies on a fine interplay between the virion and the host cell. Infection is initiated by interaction of the viral particle with specific proteins on the cell surface. After initial binding of the receptor, enveloped viruses need to fuse their envelope with the host cell membrane to deliver their nucleocapsid to the target cell. The spike protein plays a dual role in entry by mediating receptor binding and membrane fusion. The fusion process involves large conformational changes of the spike protein. Coronaviruses use a variety of receptors and triggers to activate fusion, however fundamental aspects that enable this initial step of the viral life cycle are conserved. In this review, we will address entry strategies of coronaviruses and how these mechanisms are related to host tropism and pathogenicity. The spike protein is a large type I transmembrane protein ranging from 1,160 amino acids for avian infectious bronchitis virus (IBV) and up to 1,400 amino acids for feline coronavirus (FCoV). In addition, this protein is highly glycosylated as it contains 21 to 35 N-glycosylation sites. Spike proteins assemble into trimers on the virion surface to form the distinctive \"corona\", or crown-like appearance. The ectodomain of all CoV spike proteins share the same organization in two domains: a N-terminal domain named S1 that is responsible for receptor binding and a C-terminal S2 domain responsible for fusion (Figures 2 and 3) . A notable distinction between the spike proteins of different coronaviruses is whether it is cleaved or not during assembly and exocytosis of virions. With some exceptions, in most alphacoronaviruses and the betacoronavirus SARS-CoV, the virions harbor a spike protein that is uncleaved, whereas in some beta-and all gammacoronaviruses the protein is found cleaved between the S1 and S2 domains, typically by furin, a Golgi-resident host protease ( Figure 2 ). Interestingly, within the betacoronavirus mouse hepatitis virus (MHV) species, different strains, such as MHV-2 and MHV-A59 display different cleavage requirements. This has important consequences on their fusogenicity, as detailed in Section 4. The S2 subunit is the most conserved region of the protein, whereas the S1 subunit diverges in sequence even among species of a single coronavirus. The S1 contains two subdomains, a N-terminal domain (NTD) and a C-terminal domain (CTD). Both are able to function as receptor binding domains (RBDs) and bind variety of proteins and sugars.The coronavirus spike protein is a class I fusion protein . The formation of an α-helical coiled-coil structure is characteristic of this class of fusion protein, which contain in their C-terminal part regions predicted to have an α-helical secondary structure and to form coiled-coils. Influenza hemagglutinin protein HA is the prototypical member of the class I fusion protein family and one of the best characterized so far . HA is synthesized as a HA0 precursor and assembles into trimers. The protein becomes fusion competent by cleavage of HA0 into HA1 and HA2. The fusion peptide, a very conserved hydrophobic sequence, is located at the N-terminus of HA2. In the pre-fusion conformation, the central coiled-coil of the trimer is formed by three long helices with three shorter helices packed around them. In this conformation, the fusion peptide is protected, buried within the trimer interface. Two major conformation changes occur during fusion. Upon endosomal acidification, an unstructured linker becomes helical allowing formation of a long helix in the N-terminal part. In this conformation, called a prehairpin, the fusion peptide is projected towards the target membrane where it is then embedded, connecting the viral and target cell membranes. The second conformational change consists of the inversion of the C-helix that packs into the grooves of the N-terminal trimeric coiled-coils forming a six-helix bundle (6HB). In the resulting conformation, the transmembrane domain and the fusion peptide anchored into the target membrane are brought in close proximity facilitating merging of viral and cell membranes.Coronavirus spike proteins contain two heptad repeats in their S2 domain, a feature typical of a class I viral fusion proteins. Heptad repeats comprise a repetitive heptapeptide abcdefg with a and d being hydrophobic residues characteristic of the formation of coiled-coil that participate in the fusion process. For SARS-CoV and MHV, the post-fusion structures of the HR have been solved; they form the characteristic six-helix bundle [7, 8] . The functional role of MHV and SARS-CoV HR was confirmed by mutating key residues and by inhibition experiments using HR2 peptides [9, 10] .Severe acute respiratory syndrome (SARS)-CoV spike protein schematic. The spike protein ectodomain consists of the S1 and S2 domains. The S1 domain contains the receptor binding domain and is responsible for recognition and binding to the host cell receptor. The S2 domain, responsible for fusion, contains the putative fusion peptide (blue) and the heptad repeat HR1 (orange) and HR2 (brown). The transmembrane domain is represented in purple. Cleavage sites are indicated with arrows.The important role of the spike protein in cell tropism has been demonstrated with chimeric viruses. There are many strains of mouse hepatitis virus (MHV), viruses that infect mainly the brain and liver. Because of the different patterns of disease associated with the various strains of MHV, involvement of their spike protein in tissue tropism has been extensively studied. The strain JHM is highly virulent causing severe encephalitis that is often lethal, but is poorly hepatotropic. The strain MHV-A59 causes hepatitis and mild encephalitis. MHV-2 is highly hepatotropic. By using chimeric viruses between these different strains, it has been shown that the S protein is linked to the tropism and pathogenesis of MHV. Introduction of JHM or MHV-2 S genes in the MHV-A59 background increases the recombinant virus' neurovirulence and hepatotropism respectively [11, 12] . However, replacement of JHM S protein sequence with MHV-A59 S gene in the JHM background does not confer hepatotropism suggesting that other factors modulate virus tropism. A mutant MHV-A59 strain exhibiting altered tropism was isolated from persistently infected microglial cells . The single mutation Q159L in the S1 domain is responsible for reduced replication in the liver and low hepatotropism of the virus . The important role of the spike protein in tropism has also been shown for other coronaviruses. IBV is an important domestic fowl pathogen that replicates in the respiratory tract but also in epithelial cells from the kidney, the oviduct and the gut. In vitro, clinical strains of IBV infect only chicken embryo kidney cells and grow on embryonated eggs. IBV Beaudette strain is an attenuated strain that was obtained by serial passage of IBV on eggs. IBV Beaudette, in addition to chicken embryo kidney cells, also infects CEF, BHK-21 and Vero cells. Substitution of the S gene in the Beaudette background with that of the IBV M41 strain restricts the tropism of the virus to primary chicken cells . However, in vivo this chimeric virus has the attenuated phenotype of Beaudette. These data show that change in tropism of Beaudette in cell culture is mainly determined by the S protein though the avirulence also results from attenuating mutations in other genes .Feline coronaviruses (FCoV) provide a fascinating example of the critical involvement of the spike protein in tropism and pathogenesis . Within this alphacoronavirus species, there are two known serotypes, 1 and 2, based on serological and genetic characteristics of their spike. Furthermore, there are two biotypes within each serotype, both of which are associated with extremely contrasting pathological potential. Cats get commonly infected with the feline enteric coronavirus (FECV), a biotype that gives rise to usually asymptomatic to mild enteric tract infections and may establish persistence in the host. In contrast, some FCoV-infected cats sporadically develop an invariably fatal immune-mediated disease called feline infectious peritonitis (FIP). In this case, the causative agent is called feline infectious peritonitis virus (FIPV). A striking characteristic of FIPVs that sets them apart from FECVs is their ability to efficiently replicate in monocytes and macrophages . It is thought that this switch in tropism, from gut epithelium to motile monocytes/macrophages cells, is a crucial tipping point towards the development of FIP as it allows for viral dissemination throughout the host.The current understanding is that mutations in FECV in a persistently infected host cause it to change into the virulent FIPV . While it has been hypothesized that mutations or deletions in certain genes, such as 3c and 7b, may be associated with the emergence of FIPV , the causative mutations for the biotype switch are still unknown. There is, however, evidence that mutations in the spike gene may play key role in the transition of tropism from gut epithelium to macrophages. Rottier and colleagues have focused on the genetically close and laboratory-adapted type 2 FECV 79-1683 and FIPV 79-1146 pair . While both viruses have similar growth characteristics in established feline epithelial cells, only FIPV 79-1146 but not FECV 79-1683 has the ability to efficiently infect and replicate in macrophages. Using a targeted RNA recombination system the authors were able to generate recombinant chimeric virus to determine regions of the genome that are important for infection of bone marrow derived macrophages. They found that the exchange of the FIPV 79-1146 S gene with that of FECV 79-1683 in the FIPV 79-1146 genetic background strongly reduced the chimeric recombinant virus' capacity to infect macrophages compared to the recombinant wild type FIPV 79-1146. Furthermore, additional chimeras were generated to more precisely map the regions of spike that are important for macrophage tropism. Surprisingly, the C-terminal region of the spike (from residue 874 to the C-terminus) but not the N-terminal region (which contains the S1 receptor binding domain) was found to be responsible for the macrophage tropism in this system. A total of ten amino acid substitution differentiates the C-terminal regions of FECV 79-1683 and FIPV 79-1146, however the precise mutation(s) that cause the tropism switch remain(s) to be determined .While serotype 2 FCoV have been studied in a relatively detailed manner, in particular because they propagate more easily in vitro, serotype 1 FCoV, which are more relevant clinically as they are more prevalent, are less well understood. And although it can be assumed that viruses of both serotypes behave in similar ways for most of their life cycle, it remains to be investigated whether the same or different set of mutations would account for the biotype switch in the two serotypes. Thus, more efforts are needed to study serotype 1 FCoV. Such efforts would shed light on the basis of FCoV pathogenesis.The difference in tropism mediated by S proteins results from different mechanisms linked to the two main functions of the protein: receptor binding and fusion, which will be further discussed.The first coronavirus receptor identified was the MHV receptor, in 1991 . MHV binds to the adhesion molecule CEACAM1 (Carcinoembryonic antigen-cell adhesion molecule) to infect cells. CEACAM1 is a type I transmembrane protein belonging to the immunoglobulin superfamily. CEACAM1 is a multifunctional protein that has roles in adhesion and cell signaling, among others. The CEACAM1 ectodomain contains four Ig constant region like domains, N, A1, B and A2. The N-terminal domain N of CEACAM1 is involved in MHV binding [23, 24] . There are two allelic forms of CEACAM1, CEACAM1a and CEACAM1b. They can both function as receptors, however, binding by CEACAM1a is much more efficient . The involvement of receptor usage and tropism of MHV strains have been studied. It has been shown that neurovirulence of JHM is associated with rapid spread of the virus in the brain that is partly independent of CEACAM1. In vitro, MHV-JHM requires CEACAM1 for entry, however, in vivo, JHM is able to infect ceacam−/− mice but with a 100-fold higher lethal dose . As a consequence, it has been suggested that JHM, in the absence of CEACAM1, uses an alternative, less effective and yet to be determined receptor in order to initiate infection. After primary infection, the virus could propagate very rapidly by using cell-cell fusion independently of the receptor (receptor independent spread) . In vivo, MHV-A59 is strictly dependent on CEACAM1 for infection , but persistent infection of murine cells leads to the emergence of viruses with extended tropism . The MHV/BHK virus infects cells in a heparan sulfate-dependent and CEACAM1-independent manner because of the acquisition of two heparan sulfate binding sites in the S protein [29, 30] . It has been shown that both binding sites are required to acquire the CEACAM1-independent phenotype .For JHM strain, many isolates exist that differ in their neurovirulence levels. The virulence is correlated to the length of a hypervariable region present within S1. The isolate MHV-4 of JHM contains the longest region and it is associated with independent CEACAM1 cell-cell fusion and spread . It has been suggested that conformational changes of the spike protein are facilitated by a less stable association of the S1 and S2 subunits . This suggests that the higher the fusogenic potential of the spike protein is, the less the virus depends on its receptor for entry.Among alphacoronaviruses two human viruses (HCoV-229E and HCoV-NL63) can be found along with viruses that infect animals and can be responsible of severe illness: transmissible gastroenteritis CoV (TGEV) and canine CoV (CCoV) cause enteric disease in pigs and dogs respectively while feline coronaviruses cause enteric and systemic disease in cats.HCoV-229E, TGEV, serotype 2 FCoV and CCoV all use the aminopeptidase N (APN) protein of their natural host as receptor. Interestingly, in addition to their specific host APN, these viruses are able to bind the feline APN. It has been suggested that these viruses may have originated from a common ancestor coronavirus infecting felines that used APN as a receptor . APN, also known as CD13, is a type II transmembrane protein expressed on the apical domain of epithelial cells of respiratory and enteric tracts. APN is a Zn 2+ dependent protease that preferentially degrades peptides or proteins with a N-terminal neutral amino acid. It has been shown that tropism differences of these viruses are due to the ability of their spike proteins to recognize small species-specific amino acid differences in APN . Spike proteins of HCoV-229E, TGEV, FCoV and CCoV present a high homology, however, binding domains are located in non-homologous regions.TGEV infects epithelial cells from the small intestine but is also able to infect cells from the respiratory tract. In the mid-1980s, an attenuated variant of TGEV, porcine respiratory coronavirus (PRCoV) was isolated in Belgium. This virus provides an example of altered tissue tropism due to a deletion occurring in the spike gene . Unlike TGEV, PRCoV infects only pulmonary epithelial cells. Both spike proteins bind porcine APN, the receptor binding domain being located between residues 522 and 744 of TGEV S protein. The spike protein of TGEV has a hemagglutinating activity that is absent in PRCoV as this activity is contained in the deleted N-terminal part of the protein . One of the consequences of this lack of activity is the inability of PRCoV to replicate in the gut. The hemagglutinating activity was mapped to the residues 145-155 of TGEV spike protein and it has been shown that a mutation abrogating this activity reduced the enteropathogenicity of the virus . In addition, a study has shown that the sialic acid binding activity of TGEV is responsible for binding of an additional protein designated as mucin-like glycoprotein (MPG) in brush border membranes . It has been suggested that this binding might shield the virus from the action of gut emulsifiers . The role of the NTD and carbohydrate binding in TGEV provides interesting insights into coronavirus enterotropism, a property that generally is attributed to non-enveloped viruses. The role of the NTD in other enteric alphacoronaviruses such as FCoV and canine coronavirus (CCoV) is still unknown.Other coronaviruses have sialic acid binding activity, in particular bovine coronavirus (BCoV) and human HCoV-OC43 . The ability of betacoronaviruses to bind carbohydrates has been mapped to a galectin fold-like structure present in the S1 NTD . So far, besides the binding of Neu5,9Ac2 conjugates, no other specific receptors have been identified for these viruses. They belong to the betacoronavirus group and contain HE proteins, so they resemble influenza virus as they have a receptor-destroying enzyme. However, the exact role of HE during coronavirus entry remains unclear. IBV also exhibits sialic acid binding activity but the role of such activity in pathogenicity is not known. For IBV, extended host range of Beaudette strains in cell culture has been linked to the presence of a heparin binding site in the spike protein .Another example of heparan sulfate binding is found with type 1 FCoV spike. By incubating viruses with heparin-agarose beads (heparin has a very similar structure to heparan sulfate, and is used in binding assays) de Haan and colleagues have demonstrated by quantification of bead-associated viral RNA that the cell-culture-adapted type 1 FIPV UCD1 strain can bind heparin . Very interestingly, the putative heparin binding motif proposed by the authors resides in a defective furin cleavage site at the boundary between the S1 and S2 domains. The type 2 FIPV 79-1146 as well as the UCD1-related type 1 FIPV UCD, which harbors a functional furin cleavage site, were not able to bind the heparin beads. This lends support to the notion that an uncleaved heparan sulfate recognition motif is required for binding activity. Furthermore, the authors found that inoculation of UCD1 to FCWF cells in the presence of competing heparin severely diminished infection. Infection by type 2 FIPV 79-1146 was not affected by this heparin competition assay.As mentioned above, the coronavirus spike protein/receptor pairing is a key determinant of tropism. To infect a new host species, coronaviruses must adapt to the receptor of their new host either by mutation or by recombination with a coronavirus infecting their new host. In the case of SARS-CoV, the virus appeared in 2002 in live animal retail markets in China. Related viruses were isolated from Himalayan palm civets, raccoon dogs and Chinese ferrets; however, it is believed that these animals were not the reservoir of the virus, but intermediate hosts during the species-jumping event. The receptor of the SARS-CoV is the angiotensin-converting enzyme 2 (ACE2) . ACE2 is a type I integral membrane protein abundantly expressed in lung tissue; it is a mono-carboxypeptidase that hydrolyses angiotensin II. Human and Himalayan palm civet coronavirus receptor usage analyses have shown that human SARS-CoV can bind both human and palm civet ACE2 whereas the palm civet virus cannot bind hACE2. It has been shown that adaptation of the virus to humans was due to two point mutations, K479N and S487T, in the binding domain of the SARS-CoV S protein . Further characterization by Wu et al. of adaptive mutations of the RBD led to the identification of mutations that strengthen the interaction with either human or palm civet ACE2 . SARS-CoV-like viruses have been isolated in bats. In this case, entry does not occur via ACE2 and their receptor(s) is/are unknown; however, replacement of the amino acid sequence found between residues 323 and 505 with the corresponding sequence of the SARS-CoV RBD is sufficient to allow human ACE2 receptor usage .Coronaviruses are able to exploit many cell surface molecules-proteins and carbohydrates alikein order to gain entry into target cells. Host calcium dependent (C-type) lectins have been recognized to play a role in infection by SARS-CoV, IBV, and FCoV. Dendritic cell-specific intercellular adhesion molecule-3-grabbing non-integrin (DC-SIGN) is a C-type lectin expressed on macrophages and dendritic cells. Its function is to recognize high mannose glycosylation patterns commonly found on viral and bacterial pathogens. Viral exploitation of DC-SIGN is best documented in HIV-1, which attaches via N-glycosylated residues on the surface of the virus. HIV-1 uses DC-SIGN to subvert the host immune defenses by entering and initiating infection of dendritic cells or macrophages directly (in-cis), or by traveling with the cell to lymph nodes where the virus is transferred to T-cells at the immunological synapse (in-trans). Like HIV-1 gp120, the coronavirus spike is heavily glycosylated providing the virus with the opportunity to interact with host lectins such as DC/L-SIGN. L-SIGN, which is expressed on endothelial cells of the liver as well as in the lung, has been reported to be an alternate receptor for SARS-CoV and HCoV-229E [47, 48] . In-trans transmission of SARS-CoV by dendritic cells to susceptible target cells has been documented. Although the dendritic cells studied were capable of transferring infectious virions via a synapse-like structure, in-cis infection was not observed . Site directed mutagenesis has identified glycosylation at the asparagine residues 109, 118, 119, 158, 227, 589, and 699 as critical for L-SIGN/DC-SIGN mediated entry .FIPV is an example of a coronavirus that targets immune cells-specifically monocytes and macrophages-to achieve systemic spread. Infection of non-permissive cell types was achieved through exogenous expression of DC-SIGN, demonstrating that both type 1 and type 2 FIPVs use DC-SIGN as a co-receptor or as an alternative receptor to fAPN, respectively [51, 52] . In the case of IBV, experiments have demonstrated that DC-SIGN and the closely related L-SIGN enhance infection of otherwise non-permissive cells, in a sialic acid-independent fashion . The role of lectins in IBV infection in vivo is undetermined.Enveloped virus entry can occur directly at the cell surface after binding to the receptor or after internalization via endocytosis with fusion taking place in the endosomal compartment. Fusion of viral membranes with host membranes is driven by large conformational changes of the spike protein. Over time, coronaviruses have modified their spike proteins, leading to the diversity of triggers used to activate their fusion. These conformational changes can be initiated by receptor binding but may need additional triggers such as pH acidification or proteolytic activation. The mechanisms of coronavirus entry are complex and differ between coronavirus species and strains. For example, depending on the MHV strain, fusion can occur directly at the cell surface after receptor binding or after endocytosis. JHM strain MHV-4 fuses at neutral pH, but the virus was also detected in endosomal vesicles . It is likely that MHV-4 is capable of entering directly at the cell surface or through the endosomal pathway. The choice of entry mechanism may depend on the cell type. In both cases, fusion is solely triggered by receptor binding. Indeed, it has been shown that incubation of JHM spike protein with a soluble form of the receptor CEACAM1 induces modifications in S hydrophobicity and conformational change of the S2 region . In addition, JHM is able to spread from DBT cells to BHK cells that do not express the MHV receptor. Incubation with soluble receptor increases this receptor-independent propagation . The capacity of the spike protein to fuse at neutral pH relies on the properties of the fusion machinery. A variant of MHV-4 isolated from persistently infected cells requires low pH exposure for productive infection. The difference of pH requirement for fusion between the mutant and the wild type virus was attributed to three point mutations in the heptad repeat regions . Concerning MHV-A59 entry mechanisms, contradictory results have been reported. Qui et al. reported that parental strain of MHV-A59 was insensitive to lysomotropic agent whereas the recombinant strain containing the MHV-2 spike protein relies on low pH for entry . Indeed, MHV-2 entry requires low-pH-activated endosomal proteases (cathepsin B and L) and, if a cleavage site is introduced in MHV-2 spike protein, virus entry no longer requires these proteases; these data are in favor of pH-independent fusion induced by receptor binding. Eifart et al. challenged this scenario. Combining different approaches of infection and microscopy, the authors have shown that MHV-A59 infection is sensitive to lysomotropic agent and that the virus is internalized allowing initiation of infection, suggesting that pH acidification is required to trigger viral fusion . It is likely that receptor binding is a key determinant of MHV entry, however the requirement for an additional fusion trigger remains unclear. For MHV-2, the endocytosis mechanism was further characterized: the virus is internalized by a clathrin-dependent pathway that does not depend on the eps15 adaptor .Endosomal pH acidification is a fusion trigger for many viruses such as influenza virus and vesicular stomatitis virus (VSV). For many years, it was believed that IBV fusion occurs at neutral pH as infected cells form large syncytia at neutral pH. However, it has been shown by Chu et al. that infection is blocked by lysomotropic agent and that IBV fusion process was activated by low pH .The authors directly assessed the pH dependence of IBV fusion in fluorescence dequenching assays. They showed that fusion occurs at acidic pH, with a half-maximal fusion rate occurring at pH 5.5. In order to infect cells, IBV enters by endocytosis. Inhibitory drugs of the clathrin-mediated pathway such as chlorpromazine also abolished IBV infection. IBV virions harbor cleaved spike proteins, with the cleavage occurring between the S1 and S2 domains. The IBV Beaudette strain has the peculiar feature to contain a second furin cleavage site in the S2 domain of the spike. Infection and syncytia formation is inhibited by the presence of a furin inhibitor. Mutation or deletion of the S1-S2 cleavage site in Beaudette spike protein delays virus propagation but does not abolish syncytia formation. These mutants are still sensitive to furin inhibition. Mutants containing a minimal cleavage site XXXR690/S are infectious, however they are dependent on serine proteases for productive infection .The relationship between the cleavability of the coronavirus spike protein and its fusogenicity has been a source of debate for researchers for many years. When viral fusion proteins are expressed at the cell surface, activation of viral fusion results in cell-cell fusion and formation of giant multinucleated cells named syncytia. It is generally believed that syncytia formation, which involves cell membranes fusing together, reflects the fusion process between viral and host cell membranes. However, it now appears that cell-cell and virus-cell fusion mechanisms may differ. Indeed, because of differences in factors such as membrane curvature and/or density of viral envelope glycoproteins, the processes involved in cell-cell and virus-cell fusion may considerably differ mechanistically. Furthermore, formation of syncytia in infected cells is not observed with all coronaviruses. Cleavage of the fusion protein is a common characteristic of class I viral fusion proteins. For influenza, the nature of the cleavage site of HA is of great importance in virus pathogenicity. The cleavage event required to prime the protein for fusion can occur in the secretion pathway by furin or during infection by host proteases of the respiratory tract. Influenza virus strains that are cleaved by host furin are highly pathogenic as they cause systemic infection . For coronaviruses, the relationship between cleavage of the S protein and its cell-cell fusion capability has been well established. Cleaved proteins show a higher propensity for cell-cell fusion. Introduction of a mutation H716D in the spike protein of MHV-A59 strongly impaired the cleavage of the protein and delayed cell-cell fusion . MHV-2 strain spike protein is not cleaved and mutation of the sequence of MHV-A59 cleavage site with the corresponding sequence of MHV-2 S protein delays cell-cell fusion. Introduction of a cleavage site in MHV-2 spike protein induces the formation of syncytia at neutral pH. In vitro spike protein cleavage plays an important role in fusogenicity, however the role in virus-fusion and pathogenicity is less clear. For example, MHV-A59 bearing the mutation H716D is very similar to the wild type virus in terms of pathogenicity . MHV-A59 viral stock produced in presence of a furin inhibitor to block spike cleavage enters cells with kinetics similar to the wild type virus . Moreover, a study by Hingley and collaborators has shown that the spike of virus purified from liver homogenates of MHV-A59infected mice was not cleaved, further adding evidence that proteolytic processing of spike is not essential for entry and spread in vivo . For coronaviruses that have a spike that is cleaved by furin, it is important to note that this protease belongs to a family of enzymes called proprotein convertases (PCs), which has nine members . Along with furin, six other PCs proteases, PC1, PC2, PC4, PC5, PACE4 and PC7, share the same basic recognition motif: R/K-[X] 0,2,4,6 -R/K (where X is any amino acid) . It remains to be determined whether other PCs that are related to furin can also recognize and cleave coronavirus spike proteins.For some coronaviruses that harbor a non-cleaved spike protein on their surface, such as MHV-2 and SARS-CoV, it has been shown that they rely on endosomal proteases for productive entry. Indeed, MHV-2 entry depends on host cell cathepsin L and B . This dependence is abolished by the introduction of a furin cleavage site between the S1 and S2 domains. For SARS-CoV, the link between cleavage and fusion is more complex ( Table 2) . It has been shown that SARS-CoV infection is inhibited by lysomotropic agents because of the inhibition of the low-pH-actived protease cathepsin L . In addition cell-cell and virus-cell fusion can be triggered by trypsin treatment . This led to the hypothesis that SARS-CoV fusion was triggered by proteolytic processing of the spike protein. It has been shown that different proteases enhance SARS-CoV infection in vitro: trypsin, thermolysin and elastase . Analysis of SARS-CoV spike protein processing by trypsin and elastase had shed light on SARS-CoV fusion [75, 76] . It was shown that trypsin activates fusion by sequential cleavage of the spike protein at two discrete sites. The first cleavage event at the S1-S2 boundary (R667) probably facilitates the second cleavage event at the position R797 (S2' region) that is responsible for fusion activation [75, 77] . The second cleavage occurs directly at the N-terminal extremity of the fusion peptide. Cleavage at R667 is dispensable for fusion activation but enhances cell-cell or virus-cell fusion. Elastase mediates cleavage at the residue T795, not directly next to the fusion peptide. SARS-CoV spike protein shows a certain degree of plasticity in the position of the cleavage site for priming of fusion. Conversely, when influenza HA is cleaved by Pseudomonas elastase the cleavage position is shifted by one amino acid, which leads to fusion incompetency . SARS-CoV S residue 795 is probably less accessible for cleavage than residue 797 and the fusion induced by cleavage at position 797 is more efficient. The difference of fusion efficacy may also result from its location at the N-terminus of the fusion peptide. These data suggest that fusion is modulated by spatial regulation of the cleavage site. It has been shown that cathepsin L cleaves the SARS-CoV spike protein in the S1-S2 boundary region at residue T678, however, so far, cleavage in the S2' region has yet to be conclusively demonstrated .SARS-CoV is able to fuse directly at the cell surface in the presence of relevant exogenous protease. It is believed that this route of entry is 100-to 1000-fold more efficient than the endosomal pathway . The availability of proteases in the extracellular milieu is a key factor of tropism. SARS-CoV is a respiratory pathogen and it has been known for a long time that proteases from the respiratory tract such as members of the transmembrane protease/serine subfamily (TMPRSS), TMPRSS2 or HAT (TMPRSS11d) are able to cleave influenza HA . Indeed, TMPRSS2 and HAT (or TMPRSS11d) are both able to induce SARS-CoV fusion . Infection of target cells with SARS-CoV S-pseudotyped virions is less sensitive to cathepsin inhibitors when target cells express TMPRSS2 [83, 84] . SARS-CoV S-pseudotyped virions produced in cells expressing TMPRSS2 still rely on endosomal cathepsin for entry, while they are less sensitive to neutralizing antibodies. This effect was attributed to the release of spike fragments in the supernatant that lure antibodies and may be of great importance for the spread of the virus . It has been shown that processing of the spike protein by HAT and TMPRSS2 may differ: HAT cleaves the SARS-CoV S protein mainly at R667 whereas TMPRSS2 cleaves the protein at multiple sites, notably in a region near S2', although the precise locations of the cleavage sites for this protease remain to be determined . Expression of HAT in target cells does not confer NH 4 Cl or cathepsin inhibitor resistance to SARS-CoV S-pseudotyped virion entry . It is likely that spatial and temporal modulation of activation by proteases play important roles. Cleavage of incoming virions before their binding to the cell would probably abort infection by inactivating the virion. Interestingly, TMPRSS2 is associated with ACE2, the SARS-CoV receptor . TMPRSS2 likely plays a key role in the initial infection and spread of the virus; however the importance of this protease on SARS-CoV infection results from a fine balance between two antagonist effects on infection: shedding of the receptor and fusion activation. For FCoV, a well-studied case for the role of proteolytic activation of spike comes from research on the type 2 FCoV pair FECV 79-1683 and FIPV 79-1146 . By using specific cathepsin inhibitors, the authors have shown that the two strains differ substantially in their use of activating proteases used during entry. While the FECV strain 79-1683 was found to rely on both cathepsin B and L as well as on an acidic endosomal environment, the FIPV strain 79-1146 was dependent on cathepsin B activity only. This was further confirmed by a biochemical assay that found that FECV 79-1683 can be cleaved by cathepsin B and L, whereas FIPV 79-1146 could only be cleaved by cathepsin B. Based on the molecular weights of the cathepsin cleavage products, it was hypothesized that the cleavage site did not reside in the boundary region between the S1 and S2 domain, but in a region located in the C-terminal part of S2 .A critical feature of any viral fusion protein is the so-called \"fusion peptide\", which is a relatively apolar region of 15-25 amino acids that interacts with membranes and plays an essential role in the fusion reaction [6, 87, 88] . Fusion peptides of class I viral fusion proteins are typically classified as \"external\" or \"internal\" depending on their location relative to the proteolytic cleavage site . One key feature of viral fusion peptides is that within a particular virus family, there is high conservation of amino acid residues; however, there is little similarity between fusion peptides of different virus families . In the case of influenza HA, which is a classic example of an \"external\" fusion peptide, the N-and C-terminal parts of the fusion peptide (which are α-helical) penetrate the outer leaflet of the target membrane, with a kink at the phospholipid surface. The inside of the kink contains hydrophobic amino acids, with charged residues on the outer face . Internal fusion peptides (such as the one found in the Ebola virus GP) often consist of loops, but also require a mixture of hydrophobic and flexible residues similar to N-terminal fusion peptides [87, 92] . It is important to note that despite the presence of key hydrophobic residues, viral fusion peptides often do not display extensive stretches of hydrophobicity, and can contain one or more charged residues .To date, the exact location and sequence of the coronavirus fusion peptide is not known , however, by analogy with other class I viral fusion proteins, it is predicted to be in the S2 domain. The location of the fusion peptide has been most extensively studied for SARS-CoV. Three membranotropic regions in SARS-CoV S2 were originally suggested as potential fusion peptides [95, 96] . Based on sequence analysis and a hydrophobicity analysis of the S protein using the Wimley-White (WW) interfacial hydrophobic interface scale, initial indications were that the SARS-CoV fusion peptide resided in the N-terminal part of HR1 [5, 97] , which is conserved across the Coronaviridae. Mutagenesis of this predicted fusion peptide inhibited fusion in syncytia assays of S-expressing cells . This region of SARS-CoV has also been analyzed by other groups in biochemical assays [96, 99, 100] and was defined as the WW II region (residues 864-886)-although Sainz et al. actually identified another, less conserved and less hydrophobic, region (WW I, residues 770-778) as being most important for fusion. Peptides corresponding to this region have also been studied in biochemical assays by other groups . In addition, a third, aromatic region adjacent to the transmembrane domain (the membrane-proximal domain) has been shown to be important in SARS-CoV fusion . This membrane-proximal domain likely acts in concert with a fusion peptide in the S2 ectodomain to mediate final bilayer fusion once conformational changes have exposed the fusion peptide in the ectodomain.Based on the finding that SARS-CoV S can be proteolytically cleaved at a downstream position in S2, at residue 797 , further investigations were carried out to determine whether cleavage at this internal position in S2 might expose a domain with properties of a viral fusion peptide. A mutagenesis study of SARS-CoV S residues 798-815, a stretch located in between the WW I and WW II regions, combined with lipid-mixing and structural studies of an isolated peptide, showed the importance of this region as a novel fusion peptide for SARS-CoV. The sequence immediately C-terminal to the R797 cleavage site of SARS-CoV S is SFIEDLLFNKVTLADAGF, and it is notable that both R797 and this downstream sequence are highly conserved across the Coronaviridae. In particular, the IEDLLF motif showed only minimal divergence, with occasional conservative substitutions .Examination of the proposed IEDLLF fusion peptide in the context of a structural model of the SARS-CoV S homotrimer shows that it is externally positioned mid-way down the trimer, and as such, would appear to be appropriately located to function as a fusion peptide. Notably, L803, L804 and F805 are the initial residues of a major antigenic determinant of SARS-CoV S (Leu 803-Ala 828) that is capable of inducing neutralizing antibodies . This SARS-CoV epitope is also homologous to an immunodominant neutralizing domain (the 5B19 epitope) on the MHV S2 subunit . While a crystal structure of the SARS-CoV S ectodomain has not yet be solved, a predictive model of the quaternary structure is available: PDB entry 1T7G . In the context of this model, the novel S2 fusion peptide is mainly helical (especially the conserved residues SxIEDLLF), with a short central unstructured region, and is in a relatively exposed position mid-way down the trimeric spike protein complex (Figure 3 ). This structure and position within the S trimer is consistent with its function as a viral fusion peptide. Comparisons with other fusion proteins reveal some similarities to the \"internal\" fusion peptides of Ebola virus and avian leukosis virus. Like these viruses, the coronavirus fusion peptide is exposed by proteolytic cleavage [6, 89, 110 ], yet is not a classic \"external\" fusion peptide like influenza HA. Figure 3 . SARS-CoV spike protein three-dimensional predicted structure. This representation, shows the trimeric (inset) and monomeric forms of the protein and is based on the three-dimensional predicted model found in the PDB database (PDB entry 1T7G, ). Note that the predicted model does not include residues 681-736 of the protein. The S1 and S2 domains as well as the cleavage sites and putative fusion peptide are highlighted.In the past ten years, many new coronaviruses have been identified. They infect a wide range of hosts from mammals to birds. Closely related coronaviruses have been identified in distantly related animals suggesting recent interspecies jumps. Coronavirus diversity is fundamentally due to the low fidelity of the virally encoded RNA-dependent-RNA-polymerase that generates around 10 −3 to 10 −5 substitutions per site per year. The large size and replication strategy of the coronavirus genome also allows for frequent homologous recombination, a process that enables exchange of genetic material during co-infection. Persistent infection leads to the accumulation of adaptive mutations. The consequences of coronavirus species barrier jumping can be devastating and result in severe disease and mortality, as exemplified by the SARS outbreak. The spike protein is the major determinant of coronaviruses tropism. Modification of the spike can alter cell and tissue tropism and, in some cases, in association with other viral and host factors, may lead to change of virus pathogenicity. Zoonoses constitute a real risk for human health. In the past, coronaviruses have often demonstrated their propensity to infect new hosts, highlighting the capacity for viral evolution and the need for surveillance. Great progress has been made in the understanding of spike protein functions, however it remains impossible to predict the effect of mutations that a virus might acquire.\n",
            "\t0.616\tPathogens of bacterial and viral origin hijack pathways operating in eukaryotic cells in many ways in order to gain access into the host, to establish themselves and to eventually produce their progeny. The detailed molecular characterization of the subversion mechanisms devised by pathogens to infect host cells is crucial to generate targets for therapeutic intervention. Here we review recent data indicating that coronaviruses probably co-opt membranous carriers derived from the endoplasmic reticulum, which contain proteins that regulate disposal of misfolded polypeptides, for their replication. In addition, we also present models describing potential mechanisms that coronaviruses could employ for this hijacking.Coronaviruses (CoV) are enveloped viruses with plus-strand RNA genomes belonging to the family Coronaviridae, which together with the Roni-and the Arteriviridae, form the order Nidovirales . CoV infect a large range of birds and mammals, including humans, and their pathogenesis has been intensively studied since the 1960s. CoV generally cause respiratory and/or intestinal infections, although some may spread systemically. Human CoV (HCoV)-OC43 and HCoV-229E, for example, cause mild upper respiratory tract infections , although they are occasionally associated with severe pulmonary diseases in newborns and immuno-compromised people . In spring 2003, a new human CoV became infamously notorious due to an outbreak in South East Asia and Canada [4, 5] . The accused virus was rapidly identified as the SARS-CoV. Unlike HCoV-OC43 and -229E, the SARS-CoV causes a severe respiratory disease , with nearly 10% mortality and it also spreads systemically . Since 2003, two additional new human CoV have been characterized, i.e., HCoV-NL63 and HCoV-HKU-1, which are also mainly associated with mild upper respiratory tract infections .Several animal CoV are economically important pathogens . For example, transmissible gastroenteritis virus (TGEV) causes diarrhea in pigs , feline infectious peritonitis virus (FIPV) leads to a fatal systemic disease in cats , the bovine coronavirus (BCoV) causes respiratory tract diseases and diarrhea in cattle [14, 18] and the avian infectious bronchitis virus (IBV) is the etiological agent of severe respiratory tract and kidney diseases in chickens [14, 19] . The mouse hepatitis virus (MHV) has been extensively used to study the replication and assembly of CoV in cell culture models as well as in whole animals and is thus considered as a prototype CoV [11, 14] .CoV virions are spherical enveloped particles, with a diameter of 70-120 nm . All CoV particles contain a common set of 4 structural proteins, i.e., the spike (S), the membrane (M) and the envelope (E) transmembrane proteins, and the nucleocapsid (N) protein. The large genomic RNA is encapsidated by multiple copies of the N protein forming the helical nucleocapsid. The CoV envelope, which is composed of a lipid bilayer derived from host intracellular membranes, accommodates the 3 transmembrane proteins. The S protein forms the peplomers that radiate from the virion surface. The M protein is the most abundant protein in the virus particle while the E protein is only present in small amounts [21, 22] .The first step in the CoV infection cycle is the association of the virion with the host cell surface through the binding of the S proteins to specific receptors. After endocytic uptake, conformational changes in the S protein result in fusion of the viral envelope with a limiting cellular membrane . Upon virus-cell fusion, the virions disassemble and release their genomic RNA in the cytoplasm. The viral RNA is directly translated into two very long polypeptides of approximately 400 and 800 kDa, called pp1a and pp1b, respectively [11, 28, 29] . During and after their synthesis, pp1a and pp1b are cleaved by viral proteinase activities contained in their sequence and this processing leads to the generation of 16 non-structural proteins (nsp's) [11, 30] . These factors probably trigger the rearrangement of host cellular membranes, resulting in the formation of a reticulovesicular network of double membrane vesicles (DMVs) and convoluted membranes ( Figure 1 ) and form replicationtranscription complexes (RTCs) that are associated with the rearranged membranes . The RTCs copy the genomic RNA either continuously into genome-length or discontinuously into various subgenome-length minus-strand templates. The minus strands are in turn used as templates for the synthesis of genomic and subgenomic mRNA [35, 36] . These latter products comprise a nested set of overlapping species of mRNAs that extend for different lengths from a common 3' terminus [35, 36] .Replication and transcription of plus-strand RNA viruses result in the formation of double-stranded RNA (dsRNA) molecules that are generally assumed to function as intermediates in the RNA synthesis [36, 38] . The exact location of viral RNA synthesis remains unknown at present. Although it has been shown that dsRNA accumulates in the interior of DMVs, the significance of this phenomenon remains unclear . The dsRNA generated by viruses are potent inducers of antiviral interferon signaling pathways [39, 40] and therefore its sequestration into DMVs could prevent the activation of these innate immune responses [31, 38] .The structural proteins are synthesized from the subgenomic RNAs and the ones with transmembrane segments are subsequently inserted in the limiting membrane of the endoplasmic reticulum (ER) . The S protein is found along the secretory pathway and at the plasma membrane. The M protein, in contrast, localizes predominantly in the Golgi compartments while the E protein is detected in the ER, in the Golgi, and in the ER-to-Golgi intermediate compartment (ERGIC). Nonetheless, these proteins and the helical nucleocapsids cooperatively assemble into virions via lateral interactions that induce the invagination and luminal pinching off of the limiting membrane of the ERGIC. The resulting luminal virions subsequently reach the extracellular environment following the conventional secretory pathway.All plus-strand RNA viruses synthesize their viral RNA in association with extensive virus-induced rearrangements of specific intracellular membranes. Different plus-strand RNA viruses may target different membrane compartments, such as the ER, endosomes, mitochondria or chloroplasts, thereby giving rise to membrane invaginations, (clusters of) DMVs or single-membrane vesicles, membranebound vesicle packets, convoluted membranes and other structures or combinations thereof (for reviews see [38, 44] ). For a long time, the subcellular compartment from where the membranes composing the CoV-induced DMVs are derived has remained mysterious. The major difficulty in solving this issue has been the lack or the undetectable levels of marker proteins of subcellular organelles [33, , the detection of which could have provided insight into the origin of these structures. Even so, several pieces of evidence has indicated that the ER is the most probable source for the lipid bilayers composing the CoV-induced DMVs. First, nsp3 and nsp4, two nonstructural proteins with transmembrane segments, which are very likely part of the RTCs, become N-glycosylated , a co-translational modification that occurs in the ER. Second, nsp4 localizes to the ER when ectopically expressed and moves to the DMVs upon viral infection . Third, a block of the early transport steps of the secretory pathway inhibits CoV replication . Fourth, ultrastructural studies where electron tomography was applied, demonstrated that the CoV-induced DMVs are interconnected via their outer membranes and are part of a membranous reticulovesicular network, which also includes convoluted membranes and is connected to the ER . Fifth, Sec61α, which is a subunit of the ER translocon, redistributes upon SARS-CoV infection and localizes to the rearranged membranes . Finally, the DMVs induced in SARS-CoV infected cells contain ribosomes on their outer membranes , although this has not been observed in MHV-infected cells [33, 45] .Even though the ER is the likely (possible) source of the membranes composing the CoV-induced DMVs, the lack of ER, ERGIC or Golgi protein markers, as well as the absence of coatomer proteins at their limiting membranes [31, 33, 46, 47] question a model in which CoV co-opt ER-derived secretory vesicles for DMVs formation. Based on available data, it seems more plausible that DMVs might result from extensive modifications of ER membranes or of ER-derived vesicles that regulate ERAD tuning, i.e., the export from the ER of EDEM1, OS-9 and other regulators of ER-associated degradation (ERAD) . In the next sections, we will mainly review this latter scenario.The ER is the site of maturation for secretory and membrane proteins in eukaryotic cells. Proteins that fail to attain the native structure must efficiently be removed from the ER lumen to protect cells from stress conditions eventually leading to cell death. Thus, besides molecular chaperones and folding enzymes that assist maturation of newly synthesized proteins, the ER also contains ERAD factors that recognize non-native proteins, extract them from the folding machineries and ensure their transport across the ER membrane for proteasomal degradation [52, 53] . The intraluminal concentration of ERAD factors must be tightly regulated. When present in excess, ERAD factors might interfere with ongoing folding programs and trigger inappropriate degradation of not-yet-native folding intermediates. This might eventually compromise the ER capacity to efficiently produce functional polypeptides and may correlate, for example, with the enhancement of the metastatic potential of tumor cells caused by the inappropriate degradation of the newly synthesized metastatic repressor KAI1 occurring in cells with elevated concentration of the E3 ubiquitin ligase GP78 .Cumulating data hint at the important role for maintenance of ER homeostasis played by the post-translational regulation of ERAD factors content in the ER lumen (reviewed in ). This regulation has been named ERAD tuning and consists in the rapid and selective removal from the ER lumen of ERAD regulators. Unlike conventional folding chaperones and enzymes, several ERAD regulators such as ERManI [56, 57] , EDEM1 [45, 58, 59] , OS-9 , XTP3-B , HERP [61, 62] and SEL1L are in fact rapidly removed from the ER lumen in unstressed cells. Some of them are degraded by the proteasome ; others, such as the ERManI [56, 57] , EDEM1 [45, 58, 59] and OS-9 are degraded by endo-lysosomal enzymes.In 2007, Juergen Roth's group revealed that EDEM1 is selectively released from the ER in vesicles . More recently, we have shown that these carriers, which we have named EDEMosomes and display LC3/Atg8 at their limiting membrane, remove EDEM1 and other ERAD factors such as OS-9 from the ER lumen and transport them to endo-lysosomes for disposal ( Figure 2) [45, 58] . LC3 is a cytosolic ubiquitin-like protein that plays a crucial regulatory role in macroautophagy. Upon activation of this catabolic process that targets cellular components to lysosomes for degradation [65, 66] , LC3-I is converted into LC3-II by covalent conjugation to the membrane lipid phosphatidylethanolamine [67, 68] . The covalent association of LC3-II to lipid bilayers appears to be essential to promote the elongation of the autophagosome membrane . Interestingly, and unlike autophagosomes, the LC3-positive EDEMosomes are not decorated with ectopically expressed GFP-LC3 and do not contain LC3-II . Rather, LC3-I is non-covalently associated to their limiting membrane and therefore this protein can be removed by carbonate extraction . These results led us to propose an unconventional, autophagy-independent use of LC3 for the vesicle-mediated removal of short-living chaperones from the ER lumen . Significantly, the DMVs in cells infected with MHV or SARS-CoV share several analogies with the EDEMosomes since they also derive from the ER and display LC3-I, but not ectopically expressed GFP-LC3 or LC3-II, at their limiting membranes [31, 45, 72] . In agreement with these results, a non-lipidable form of LC3 associates with the MHV-induced DMVs . The finding that in MHV-infected cells the turnover of EDEM1 and OS-9 is essentially stopped and that these two proteins accumulate in the DMVs (they actually are the only cellular factors that distinctly co-localize with MHV-induced DMVs in immuno-fluorescence analyses ) led us to propose that the mechanism regulating the formation of the EDEMosomes is co-opted by CoV to ensure their efficient replication . In keeping with this assumption, as previously reported for ERAD tuning , an autophagy-independent function of LC3 supports CoV infection. In fact, while the macroautophagy machinery regulating the covalent association of LC3 to the autophagosomal membranes is dispensable for infection as demonstrated by normal replication of MHV in atg7 −/− mouse embryonic fibroblasts (MEFs), the reduction of the intracellular levels of LC3 by siRNA efficiently interferes with MHV replication . Interestingly, when endogenous LC3 is replaced with a non-lipidable form of this protein, which cannot sustain autophagy, MHV infection is restored further supporting the notion of an unconventional use of LC3 in CoV replication.These data explain previous observations. Originally, it was reported that the ATG5 gene is essential for MHV replication in MEFs . Careful reassessment of these findings in low passage atg5 −/− MEFs and bone marrow derived macrophages lacking ATG5 by virtue of a Cre recombinase mediated gene deletion, revealed that an intact autophagy pathway is not required for MHV life cycle . In addition, these data also reconcile previous contrasting reports about LC3 association with CoV-induced DMVs. The works describing a co-localization were analyzing endogenous LC3, while those affirming the contrary used ectopically expressed GFP-LC3 .The rapid disposal of ERAD factors through the ERAD tuning and the hijacking of the ERAD tuning pathway by CoV are recent discoveries . As a result, numerous questions still remain open. One scenario is that CoV anchor their replication and transcription complexes to the membranes of either EDEMosomes or \"modified\" EDEMosomes, whose fusion with a degradative endo-lysosomal compartment would be inhibited as a consequence of the infection. This would explain the defective EDEM1 and OS-9 turnover observed in infected cells and the enrichment of these two ERAD factors in the DMVs . However, the molecular principles of the biogenesis of the EDEMosomes and DMVs are poorly understood (see above, [55, 58, 78, 79] ). In particular, the role of LC3-I in the formation of both EDEMosomes and CoV-induced DMVs remains unknown. One speculative idea is that LC3-I acts as a vesicle coat protein . In such a scenario and similar to other vesicular transport pathways, one or more still elusive EDEMosome cargo receptors would bind EDEM1 and OS-9 in the ER lumen to segregate these short living ERAD factors from conventional and long-living molecular chaperones and folding enzymes. The cytosolic domain of this putative cargo receptor would then recruit cytosolic LC3-I. This latter step will be the key event required for the coat-driven formation of a carrier vesicle. Thus, one possible way for CoV to exploit the ERAD tuning machinery for generating their replicative DMVs would be to hijack one of the EDEMosome cargo receptors, perhaps by using one or more of their transmembrane non-structural proteins (i.e., nsp3, nsp4 and/or nsp6). Alternatively, these nsp's could act more directly by recruiting LC3-I and other vesicle coating factors. The first option contemplates that EDEM1 and OS-9 end up in the DMVs through their association with the EDEMosome cargo receptor. The second does not explain the peculiar distribution of these two chaperones in the MHV-induced DMVs, but it could be consistent with a model claiming that CoV may actively sequester EDEMosome cargo proteins such as EDEM1 and OS-9 into the DMVs in order to weaken the ERAD capacity in the ER lumen of the host cell. At the peak of its replication, CoV induce ER stress due to a sustained high production of viral components , including the 3 integral membrane nsps and the 3 structural membrane proteins that are initially inserted in the ER lipid bilayer. One of the consequences of the induction of ER stress is the enhancement of ERAD. As this would hamper CoV replication by degrading viral products, sequestering EDEM1 and OS-9, two positive regulators of the ERAD process, could limit this cellular response that would interfere with viral replication.LC3 could also play a role in ERAD tuning and/or viral replication by linking EDEMosomes and the CoV-induced DMVs to the microtubule network, a notion suggested by the original full-length name of this protein, i.e., microtubule-associated protein 1 light chain 3 (MAP1-LC3). The initial studies, revealed in fact that LC3 belongs to a family of microtubule-associated proteins and that it interacts with MAP1A or MAP1B to form a complex that binds and modulates the shape of microtubules . Autophagosomes are mostly formed randomly at the periphery of the cell and redistribute in a microtubule-dependent way at the perinuclear region around the microtubuleorganizing center (MTOC) where the majority of late endosomes and lysosomes are concentrated . Recently, one of the molecular bases that could regulate this trafficking event has been revealed by showing that the N-terminus of LC3 interacts with FYCO1 (FYVE and coiled-coil [CC] domain containing 1), which in turn could interact with kinesin(s) . Depletion of FYCO1 or antibodies against the N-terminus of LC3 blocks the subcellular redistribution of autophagosomes after completion [87, 91] . Similarly, LC3 could link the EDEMosomes and the CoV-induced DMVs to the microtubule network. Interestingly, the MHV replicative structures, when visualized by using a GFP-tagged version of nsp2, were shown to be transported along microtubules. Upon disruption of this cytoskeletal scaffold, these structures remain dispersed throughout the cytoplasm and fail to concentrate to the perinuclear region . Connecting CoV-induced DMVs with microtubules, however, cannot be the only function of this protein during an infection because while microtubules are dispensable for MHV replication in culture cells , depletion of LC3 affects MHV replication .Whatever scenario holds true, it remains unclear how the DMVs are shaped from the singlemembrane EDEMosome vesicles . Furthermore, as EDEMosomes are probably derived from the smooth ER, it is not yet clear how to reconcile this with the observation that the surface of DMVs induced in SARS-CoV-infected cells is decorated with ribosomes , although ribosomes have not been observed on MHV-induced DMVs [33, 45, 79] . In addition, it remains to be solved how the alleged subversion of EDEMosomes by CoV results in the formation of a reticulovesicular network as observed in SARS-CoV infected cells .Viruses are dependent on the host cell for virtually every step of the infection cycle and are able to subvert cellular processes to their own advantage. They can also be regarded as unique tools in cell biology research and have been used in the past decades to characterize a vast array of cellular machineries and pathways. Studying CoV replication and the biogenesis of the coronaviral replicative structures is therefore expected to increase our knowledge about the (re)shaping of cellular membranes and the formation of EDEMosomes, while the opposite also holds true. In addition, insight into virus-host interactions not only provides information into the cellular pathways hijacked by viruses, but also opens new avenues for therapeutic intervention. The absence of efficient anti-viral therapies emphasizes the necessity to further study and understand in detail the molecular mechanisms that regulate the CoV life cycle, including the role of LC3 in virus replication. This is desirable, as CoV are not only pathogens of veterinary importance, but a threat to mankind as well, as revealed by the emergence of the SARS-CoV. Furthermore, targeting of host rather than of viral proteins may, at least in some cases, be advantageous as it is less likely to result in viral escape mutants, as in contrast to the viral proteins, cellular factors are not prone to rapid mutations.\n",
            "\t0.602\tCoronaviruses are single stranded, positive sense RNA viruses, which induce the rearrangement of cellular membranes upon infection of a host cell. This provides the virus with a platform for the assembly of viral replication complexes, improving efficiency of RNA synthesis. The membranes observed in coronavirus infected cells include double membrane vesicles. By nature of their double membrane, these vesicles resemble cellular autophagosomes, generated during the cellular autophagy pathway. In addition, coronavirus infection has been demonstrated to induce autophagy. Here we review current knowledge of coronavirus induced membrane rearrangements and the involvement of autophagy or autophagy protein microtubule associated protein 1B light chain 3 (LC3) in coronavirus replication.Coronaviruses are single stranded positive sense RNA viruses belonging to the order Nidovirales, and are known to infect a variety of hosts. Several human coronaviruses have been identified, causing mainly mild respiratory infections, with the exception of severe acute respiratory syndrome coronavirus (SARS-CoV). In addition, coronavirus infections have an economic impact on livestock industries worldwide. The avian coronavirus, infectious bronchitis virus (IBV), causes infectious OPEN ACCESS bronchitis (IB), a mild respiratory infection, but as a consequence is responsible for serious effects on the global poultry industries due to poor weight gain in broiler chickens as well as reduced egg production and egg quality in layers. In addition, some strains of IBV are nephropathogenic whilst others result in severe pathology in the reproductive organs. Bovine coronavirus (BCoV) causes respiratory infection and diarrhoea in cattle, transmissible gastroenteritis virus (TGEV) and porcine epidemic diarrhoea virus (PEDV) cause diarrhoea in pigs and porcine haemagglutinating encephalomyelitis virus (PHEV) causes vomiting and wasting disease in pigs.Following attachment of coronavirus particles to virus specific receptors on the host cell and uptake of the virus into the cytoplasm, viral genomic RNA is released. This genomic RNA is recognised directly by the host cell translation machinery and two large polyproteins, pp1a and pp1ab, of approximately 400 and 800 kDa are translated. The two polyproteins encode the 15 (IBV) or 16 (all other coronaviruses) non-structural proteins (nsps), which are generated by co-or post-translational cleavage by virally encoded proteases. These proteins assemble into viral replication-transcription complexes (RTCs), providing the virus with the enzymes required for viral RNA transcription and replication, as well as proof-reading and capping of new viral transcripts . In addition, expression of the nsps triggers the rearrangement of host cell membranes, presumed to provide a platform for the assembly of RTCs. It is likely that expression of the membrane associated nsps 3, 4 and 6 is responsible for inducing these rearrangements. Data from closely related arterivirus, equine arterivirus (EAV), demonstrated that expression of nsps 2 and 3 (homologues of nsps 3 and 4 in coronaviruses), in the absence of other viral proteins, was sufficient for the induction of membrane rearrangements and mutation of nsp3 blocked this function [2, 3] . Non-structural protein 4 from MHV is known to play a role in formation of rearranged membranes because viruses containing mutant nsp4s show defects in membrane rearrangements and a reduction in virus replication [4, 5] . Furthermore, co-expression of nsp4 with the C-terminus of nsp3 from MHV resulted in the relocation of both proteins from diffuse in the cytoplasm to punctate . The authors hypothesised that interaction between these proteins may result in the rearrangement of host membranes . However, the precise mechanism by which the nsps induce membrane rearrangements remains to be elucidated.The nature of membrane rearrangements has been well studied in both SARS-CoV and MHV infected cells by conventional transmission electron microscopy or by electron tomography. Both viruses have been shown to induce double membrane vesicles (DMVs) as well as regions of convoluted membranes . Electron tomography data showed that the convoluted membranes in SARS-CoV infected cells were derived from and joined to the rough ER. In addition, the outer membranes of the DMVs were interlinked, studded with ribosomes and were also joined to the convoluted membranes and ER. Openings between the interior of the DMVs and the cytoplasm were not observed . In contrast, ribosomes were not seen on the membranes of MHV induced structures . Preliminary work presented here ( Figure 1 ) shows for the first time that IBV infection of mammalian Vero cells also results in the induction of DMVs. However, a detailed analysis of the membrane rearrangements triggered by IBV infection of mammalian and avian cells remains to be performed. The site of assembly of coronavirus RTCs is currently unclear. During the process of transcription and replication of the viral genome, both positive and negative sense RNAs are synthesised. As a result, dsRNA can form, possibly as a replicative intermediate. This dsRNA has been used as a marker for sites of viral RNA synthesis. However, in SARS-CoV infected cells, dsRNA was found to predominantly locate on the interior of DMVs while the majority of the nsps were found to locate on the convoluted membranes. High resolution immunofluorescence microscopy also demonstrated a separation in the signals for nsps and dsRNA. In addition, due to the lack of any connecting channels between the interior of DMVs and the cytoplasm, questions were raised about transport of RNA to sites of virus assembly at the ERGIC . Subsequent work using 5-ethynyl uridine (EU) to label nascent RNA demonstrated a degree of co-localisation between EU and dsRNA signals at earlier time points of infection but this co-localisation was significantly reduced at later time points . Therefore, further evidence is required to determine the precise location of coronavirus RTCs and the site of RNA synthesis.Autophagy is a cellular pathway for self-degradation. The pathway allows a cell to degrade long-lived proteins, aggregated proteins and organelles during periods of starvation to provide nutrients for continued cellular processes, as well as playing an important role in cellular homeostasis, ageing and development . In addition, dysregulation of autophagy plays an important role in the development of some cancers . During autophagy, regions of the cytoplasm become engulfed into double membrane bound vesicles termed autophagosomes. These vesicles then fuse with late endosomes/lysosomes, where the contents are degraded by lysosomal proteases (Figure 2 ). For detailed reviews of autophagy signaling, see [17, 18] . The major control complex for autophagy is MTOR (mammalian target of rapamycin) which, when active, inhibits initiation of the pathway. MTOR senses amino acid levels, as well as levels of growth factors and glucose and genotoxic and ER stress. Under resting conditions, MTOR phosphorylates and inactivates the ULK complex, comprising unc-51-like kinase 1/2 (ULK1/2), focal adhesion kinase family-interacting protein of 200 kDa (FIP200) and mammalian ATG13. Under stimulatory conditions, MTOR is inactivated; the ULK complex becomes hypophosphorylated and relocates to the site of formation of the autophagosome, the phagophore. Formation of the autophagosome proceeds by addition of new membrane to the phagophore, as opposed to budding from an existing membrane, in a poorly understood process. However, a number of proteins are known to be required. Recruitment of these proteins occurs via the generation of phosphatidylinositol 3-phosphate (PI3P) by the class III phosphatidylinositol 3-kinase complex (PI3K), including BECN1 (Beclin 1). Inhibition of PI3K activity using drugs such as wortmannin and 3-methyladenine (3-MA), and sequestration of BECN1 by antiapoptotic protein B-cell lymphoma/leukemia-2 (Bcl2) all inhibit autophagy.Elongation of the autophagosome membrane and formation of the complete autophagosome requires the recruitment of 2 ubiquitin-like (Ubl) conjugation systems. In the first, ATG12 becomes conjugated to ATG5 in a process requiring the E1-like enzyme ATG7 and the E2-like enzyme ATG10. ATG12-ATG5 then binds to ATG16L and forms a large complex known as the ATG16L complex. This complex localises to the phagophore and can determine the site of conjugation of the second Ubl system. In this second system, microtubule associated protein 1B light chain 3 (LC3) is initially cleaved by ATG4 near the C-terminus at position G120 to generate cytoplasmic LC3-I. This subsequently becomes lipidated with phosphatidylethanolamine (PE) in a process requiring ATG7 and another E2-like enzyme ATG3 to generate membrane tethered LC3-II . LC3-II is inserted into both the inner and outer membranes of the autophagosome, and as such, remains associated with the autophagosome throughout the pathway [20, 21] . As a result, LC3 has become an extremely valuable marker protein for studying autophagy. Figure 2 . Schematic of mammalian autophagy pathway. MTOR is the major control complex for autophagy. MTOR senses levels of amino acids, glucose and growth factors, as well as genotoxic and ER stress. Upon stimulatory signals, MTOR becomes inactivated and the ULK complex becomes hypophosphorylated and relocalises to the phagophore, along with PIP3, produced by class III PI3K complexes. The Atg16L complex and LC3II are also recruited to the growing autophagosome, allowing expansion of the membrane and fusion to give a complete autophagosome, engulfing organelles, aggregated proteins and intracellular pathogens. The autophagosome then fuses with a lysosome, resulting in the degradation of the contents and recycling of nutrients into the cytoplasm.In addition to its role in cellular homeostasis, autophagy has been shown to have a function in innate immunity by degrading intracellular pathogens, including viruses. Furthermore, autophagy plays a role in presenting pathogen components to the immune system . Inhibition of autophagy has a positive effect on the replication or virulence of herpes simplex virus 1 (HSV1) [26, 27] , Sindbis virus [28, 29] and vesicular stomatitis virus (VSV) . In addition, the capsid protein of Sindbis virus was found to be specifically targeted to the autophagosome via an interaction with the autophagy cargo receptor, p62 . However, many viruses have evolved mechanisms to evade autophagy by inhibiting the pathway, or diverting the process to benefit virus replication. Many viral proteins have been identified that inhibit formation of autophagosomes. For example, HSV1 protein ICP34.5 binds to BECN1 and inhibits autophagosome formation and Kaposi's sarcoma associated herpesvirus (KSHV) and murine γ-herpesvirus (MHV-68) encode Bcl2 homologues to bind to and inhibit BECN1 [26, 31, 32] . KSHV also encodes another protein to block LC3 processing by inhibiting ATG3 . Other viruses have developed mechanisms to inhibit fusion of autophagosomes with lysosomes. Human immunodeficiency virus (HIV) encoded protein Nef interacts with BECN1, inhibiting lysosomal fusion . Influenza A virus protein M2 has also been shown to induce accumulation of autophagosomes as a result of inhibition of lysosomal fusion, possibly via an interaction with BECN1 . Moreover, numerous viruses have been identified which require autophagy for optimal replication, including hepatitis B virus (HBV) , poliovirus , coxsackievirus, HIV-1 , hepatitis C virus (HCV) , Dengue virus [40, 41] and Japanese encephalitis virus . Finally, poliovirus subverts autophagy in order to generate membranous structures required for assembly of viral replication complexes [37, 43] . For more detailed information about the role of autophagy in the replication cycles of viruses, see reviews .The presence of DMVs in coronavirus infected cells suggested that this group of viruses might, like other positive sense RNA viruses, utilise the autophagy pathway to generate the membrane structures required for replication. Initial work showed that MHV infection induced autophagy . Nsp8 co-localised with LC3 throughout infection and the nucleocapsid protein N co-localised with LC3 early in infection, but this decreased over time. In addition, MHV replication was markedly reduced in ATG5 −/− embryonic stem cell lines but virus titre was rescued in the presence of an ATG5 expressing plasmid . Further work using SARS-CoV again showed co-localisation between nsp8 and LC3 . However, work by others using bone marrow derived macrophages lacking ATG5 or ATG5 −/− primary murine embryonic fibroblasts (MEFs), showed that MHV replication does not require either an intact autophagy pathway or conversion of LC3I to LC3II. They did however confirm that SARS-CoV nsps co-localised with LC3 . Recently, Schneider et al. demonstrated that SARS-CoV replication could also occur in ATG5 −/− MEFs, indicating no requirement for a complete autophagy pathway . Interestingly, it was observed that virus replication was unaffected by the induction of autophagy in wild type MEFs . Finally, de Haan et al. were unable to show co-localisation between MHV nsp8 and GFP-LC3 , and Snijder et al. were unable to show co-localisation between SARS-CoV nsp3 and either endogenous LC3 or GFP-LC3A, GFP-LC3B or GFP-LC3C . Despite the lack of clarity with regard to the requirement for autophagy during coronavirus infection, all studies did indicate that LC3 became punctate upon coronavirus infection, suggesting an induction of autophagy [9, 54] .Consistent with previous observations, recent experiments performed in mammalian cells using IBV showed that this avian coronavirus is also capable of inducing autophagy during infection .Here, further work was performed and individual expression of viral nsp6 was also capable of inducing autophagy, whereas nsp4 and nsp10 were not. Nsp6 induced autophagosomes fused with LAMP1 labelled lysosomes and autophagosomes were susceptible to wortmannin treatment. This indicates that bone fide autophagy was induced. Interestingly, nsp6 homologues from SARS-CoV, MHV and arterivirus porcine reproductive and respiratory syndrome virus (PRRSV), also induced autophagy.The mechanism for the induction of autophagy was not determined, although it was shown not to be by induction of ER stress, MTOR inhibition or via sirtuin 1. In addition, it was not confirmed whether nsp6 was responsible for the induction of autophagy in virus infected cells. However, in agreement with previous work, virus infection was not inhibited by the knockdown of ATG5 expression .In other work, it has been suggested that whilst endogenous LC3 could co-localise with MHV RTCs, GFP-LC3 showed significantly reduced co-localisation . This may provide an explanation for some of the earlier experimental discrepancies. Further work in this study demonstrated that LC3 puncta were still observed in MHV infected ATG7 −/− cells, where LC3I to LC3II conversion cannot occur and also in cells expressing LC3 that cannot be processed to LC3II (LC3 G120A). In addition, although the absence of the complete autophagy pathway due to a lack of ATG7 did not alter MHV replication, reduced expression of LC3 by RNAi did significantly reduce virus replication. Furthermore, viral protein expression could be rescued in the presence of LC3 G120A . This demonstrated that MHV replication does not require the complete autophagy pathway, or conversion of LC3I to LC3II, in agreement with previous work, but instead required LC3I . Interestingly, MHV RTCs co-localise with markers for cellular EDEMosomes [56, 57] , vesicles involved in ER associated protein degradation (ERAD) tuning and knockdown of EDEMosome cargo receptor SEL1L reduced virus replication [57, 58] . In the ERAD pathway, unfolded proteins are removed from the ER and targeted for degradation. However, under normal conditions, the ERAD machinery must be regulated to prevent premature removal of proteins before they have been folded . During ERAD tuning, parts of the ERAD machinery are removed from the ER in EDEMosomes to down regulate the pathway [57, 58, 60] . LC3I is recruited to EDEMosomes by transmembrane protein SEL1L and possibly acts as a coat protein . This work has highlighted a role for LC3 as part of the ERAD tuning pathway, but not autophagy, in the replication of MHV. Whether this pathway is involved in the replication cycles of other coronaviruses remains to be determined.Although mounting evidence suggests that autophagy is unlikely to play a role in the replication of coronaviruses and the generation of coronavirus replicative structures, several questions remain unanswered. Work performed so far focusses mainly of members of the betacoronaviruses, MHV and SARS-CoV. Detailed characterisation of the membrane structures induced in cells infected with alphacoronaviruses, like transmissible gastroenteritis virus (TGEV), or gammacoronaviruses, like IBV, needs to be performed. Although these viruses are related, it is possible that there are differences in the mechanisms of membrane rearrangement and the types of structures induced. In addition, the location of RTC assembly and the site of viral RNA transcription and replication need to be identified. The membranes induced in MHV and SARS-CoV infected cells are complex and how the different structures play a role in virus replication is currently unknown. Furthermore, the mechanism by which these rearrangements are generated is not understood. Whether MHV induced membranes are altered upon inhibition of ERAD tuning and which proteins might be involved in hijacking the pathway remains unknown. Moreover, whether this pathway is important for the replication of other coronaviruses is also unknown.Recently the replication of IBV has been shown to induce autophagy in mammalian cells . Furthermore, individual expression of IBV nsp6, as well as nsp6 homologues from other viruses in the Nidovirales order, has been shown to induce autophagy . However, the mechanism by which this induction occurs is unknown. Whether nsp6 is responsible for inducing autophagy in the context of whole virus would also be interesting to discover. In addition, as IBV is restricted to avian species, would similar observations be made in a more natural host model? Finally, several studies have shown that coronavirus infection induces autophagy [9, 54, 55] . However, the pathway does not appear to be required for virus replication [52, 53, 55, 56] . Therefore, is autophagy acting as a cellular defence to virus infection and does the virus have mechanisms to control this response?\n",
            "\t0.602\tCoronaviruses are closely monitored in the context of emerging diseases and, as illustrated with Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome-coronavirus (MERS-CoV), are known to cross the species barrier and eventually to move from wildlife to humans. Knowledge of the diversity of coronaviruses in wildlife is therefore essential to better understand and prevent emergence events. This study explored the presence of coronaviruses in four wild mammal orders in France: Bats, rodents, lagomorphs, and hedgehogs. Betacoronavirus and Alphacoronavirus genera were identified. The results obtained suggest the circulation of potentially evolving virus strains, with the potential to cross the species barrier.Coronaviruses (CoVs) infect a wide variety of animals and are common throughout the world. They cause respiratory, enteric, hepatic, and neurological diseases with variable severity, from asymptomatic to severe. Coronaviruses that infect mammals (except pigs) belong mainly to two genetic and serologic groups: The Alphaand Betacoronavirus genera.Along with Paramyxoviridae and the influenza viruses, the Coronaviridae family is one of the three viral families closely monitored in the context of emerging diseases. With Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome (MERS-CoV), coronaviruses have shown their ability to move from wildlife to domestic animals or to humans. In this context, describing the coronaviruses circulating in wildlife in a specific area is the first step to being readily responsive in the event of emergence. However, data concerning wildlife species harboring coronaviruses in Europe and especially France are currently very limited. Bat coronaviruses have been studied because of the SARS-CoV and MERS-CoV epidemics. In Europe, both beta-CoVs and alpha-CoVs have been found in bats [1, 2] . To our knowledge, only four bat species among the 35 species living in France have been screened for Coronaviridae .Since all these novel alpha-and beta-CoVs were found in insectivorous bats, the search in Western European hedgehogs (Erinaceus europaeus), as an insectivorous mammal belonging to a related order of Chiroptera, resulted in the identification of a novel Betacoronavirus, Eri-CoV, detected from animals raised in an animal shelter in northern Germany . Other wildlife species appear to be relevant to identify the potential presence of Coronavirinae given, for instance, the recent description in wild rodents of infection by Alphacoronavirus . Another species of interest is wild rabbits (Oryctolagus cuniculus) because of beta-CoV identification in domestic rabbits in China , which have wild European rabbits as ancestors.The aim of this study was to explore the presence of Coronaviridae in specific relevant species of wildlife in France, in order to produce an overview of the reservoir of potential emerging Coronaviridae from wildlife and of the species that host them.Samples of intestinal content were used for the detection of CoVs in bats, rodents, and rabbits. Regarding studies about the detection of CoVs, intestinal samples were tissues with the highest CoV RNA concentration in hedgehogs and were successfully used to detect CoV in bats and rabbits . Intestinal samples were collected from carcasses of these animals as follows: Figure 1 ). Intestinal samples were taken from animals killed by hunters during rabbit hunting. Therefore, no wild animal was killed specifically for the purpose of this study.Intestinal samples were stored at −20 • C without additives. Samples of hedgehogs (E. europaeus) were fecal samples collected in three animal shelters in 2015 and 2016 ( Figure 1 ). Hedgehog droppings were sampled on the first or second day after their arrival at the animal shelter. Fecal samples were stored at −20 • C in UTM-RT™ (Universal Transport Medium, Copan diagnostics, Murrieta, CA, USA). A Nucleospin Viral RNA isolation kit (Macherey-Nagel, Hoerdt, France) was used for RNA extraction from stools or from intestinal homogenate. Reverse transcription was performed from 8 µL of RNA with RT Superscript III and random hexamer primers. Amplification by PCR was carried out as described previously , by using a semi-nested PCR targeting the RdRp conserved coding region using 5 µL of cDNA in a final volume of 50 µL. PCR products were sent for sequencing by private companies (GenoScreen and Genewizz, Lille, France). Nucleic sequences of 125 bases corresponding to the RdRp gene were recovered from all positive samples.The CoV sequences found in this study were deposited in the DNA Data Bank of Japan (DDBJ) under accession numbersLC334347 to LC334385 and LC334392 to LC334408. Sequence similarity to known sequences was determined using Basic Local Alignment Search Tool (BLAST) nucleotide analysis against the GenBank database. The phylogenetic tree ( Figure 2 ) was inferred using sequences obtained from positive samples aligned with coronavirus sequences representing the whole diversity of the group. Sequence multi-alignments were performed in SEAVIEW v4.6 . Phylogenetic reconstruction using maximum likelihood was done using PhyML v3.0 . The GTR+G+I (General Time Reversible) substitution model was selected as the optimal model by a function implemented in MEGA7 and a discrete gamma distribution with four categories was used for rate heterogeneity. Trees were also inferred using the Bayesian approach implemented in the Beast package, with both GTR (General Time Reversible) and TN93 (Tamura-Neï 1993) optimum models, gamma distribution, and invariable sites. The coalescent constant size model was used and the clock parameters were set to the uncorrelated lognormal relaxed clock. The length of the chain was then set to 30 million iterations in order to produce ESS (Effective Sample Size) superior to 200. The maximum credibility tree with the branch length in number of substitutions was defined from ten thousand trees after a discard of 10% and was edited using FigTree (version 1.4.3 on ). A Nucleospin Viral RNA isolation kit (Macherey-Nagel, Hoerdt, France) was used for RNA extraction from stools or from intestinal homogenate. Reverse transcription was performed from 8 µL of RNA with RT Superscript III and random hexamer primers. Amplification by PCR was carried out as described previously , by using a semi-nested PCR targeting the RdRp conserved coding region using 5 µL of cDNA in a final volume of 50 µL. PCR products were sent for sequencing by private companies (GenoScreen and Genewizz, Lille, France). Nucleic sequences of 125 bases corresponding to the RdRp gene were recovered from all positive samples.The CoV sequences found in this study were deposited in the DNA Data Bank of Japan (DDBJ) under accession numbersLC334347 to LC334385 and LC334392 to LC334408. Sequence similarity to known sequences was determined using Basic Local Alignment Search Tool (BLAST) nucleotide analysis against the GenBank database. The phylogenetic tree ( Figure 2 ) was inferred using sequences obtained from positive samples aligned with coronavirus sequences representing the whole diversity of the group. Sequence multi-alignments were performed in SEAVIEW v4.6 . Phylogenetic reconstruction using maximum likelihood was done using PhyML v3.0 . The GTR+G+I (General Time Reversible) substitution model was selected as the optimal model by a function implemented in MEGA7 and a discrete gamma distribution with four categories was used for rate heterogeneity. Trees were also inferred using the Bayesian approach implemented in the Beast package, with both GTR (General Time Reversible) and TN93 (Tamura-Neï 1993) optimum models, gamma distribution, and invariable sites. The coalescent constant size model was used and the clock parameters were set to the uncorrelated lognormal relaxed clock. The length of the chain was then set to 30 million iterations in order to produce ESS (Effective Sample Size) superior to 200. The maximum credibility tree with the branch length in number of substitutions was defined from ten thousand trees after a discard of 10% and was edited using FigTree (version 1.4.3 on ). Bayesian phylogeny of 50 genetic sequences of viruses detected in the study with 127 representative sequences of the whole diversity of the group of Coronavirinae. Selected coronavirus RdRp sequences, including some extracted from full genomes, were aligned with original sequences from this study in SEAVIEW v4.6. . Statistical support (posterior probability) of nodes are depicted using a gradual color code of the tree, with green indicating significant posterior probability values (>0.95). Genbank identification numbers, strain names, and the main information are written in taxa labels. Taxa labels of Alphacoronavirus, Betacoronavirus and of the clade grouping Gammacoronavirus and Deltacoronavirus are in purple, blue, and orange, respectively. Viruses detected in this study are depicted in red (HE: Sequences from hedgehogs, R: Sequences from rodents, CS: Sequences from bats; and L: Sequences from rabbits).Confidence intervals for seroprevalence were estimated using the Wilson method in the Epitools calculator ().Intestinal samples from 504 bats were analyzed (Table 1) . Twelve bats were found to be infected by a coronavirus. Five sequences from Pipistrellus pipistrellus and two from Pipistrellus sp. had a 94% to 99% nucleic acid sequence identity with alpha-CoVs of Pipistrellus from France and Germany (GenBank KT345295, KT345296 and EU375868). Two Myotis emarginatus were found to be infected by coronaviruses with an identity of 100% with sequences of alpha-CoVs found in Myotis emarginatus in Luxembourg (GenBank KY502401). The coronavirus found in one Myotis nattereri exhibited a 95% identity with an unclassified coronavirus found in Myotis nattereri in Hungary (GenBank KJ652333). Concerning Miniopterus schreibersii, the nucleic sequences of the two infected animals had an identity of 99% and 98% with coronaviruses found in the same species in Bulgaria (GenBank GU190247 and GU190241). Fecal samples from 74 hedgehogs were analyzed and 37 samples were positive. The estimated prevalence for all the animal shelters was 50% (CI95%: 38.89; 61.11), with no statistical difference between the three animal shelters.All sequences were strongly homogeneous with a maximum of four differences at the nucleotide level. The nucleic sequences detected from hedgehogs had an identity from 96% to 98% with the sequence of coronavirus (KC545383) found in the hedgehogs in Germany.Of the 330 trapped rodents ( Table 2) , most of the animals belonged to Apodemus spp. (n = 206) and Myodes glareolus (n = 80), and were collected at four sites (Bitche, Forbach, La-Petite-Pierre, and Murbach) in mixed coniferous and deciduous forests. Other species of rodents were Arvicola terrestris (n = 35) and Microtus spp. (n = 9), and were mainly collected in an apple orchard in Wissembourg (Bas-Rhin).Twenty-one samples were positive, represented by five Myodes glareolus (6.25% (CI 95%: 2.70; 13.81)) and 16 Apodemus spp. A total of 291 intestinal samples from rabbits were analyzed. Twenty-two animals were found to be infected, corresponding to a global estimated prevalence of 7.56% (CI 95%: 5.05; 11.18), which differed from one department to another from 0% (CI 95%: 0.00; 29.91) to 16.66% (CI 95%: 8.32; 30.60).Seventeen sequences were very close (with identities from 94% to 98%) to the beta-CoV RbCoV HKU14 (5) and to an alpha-CoV identified in hares (Lepus sp.) and rabbits in Spain (with identities from 97% to 98%) (GenBank Nos. KM888169 to KM888181). Five sequences were very close (with identities from 96% to 97%) to the alpha-CoV identified by Tsoleridis et al. in rodents (KU739072). Both betaand alpha-CoVs were identified in rabbits from the same geographical area and collected the same year. For each genus, nucleic sequences showed strong homogeneity, with a maximum of five differences at the nucleotide level.Phylogenetic analyses (Figure 2 ) of the partial RdRp gene revealed that all coronavirus sequences from bats belonged to the Alphacoronavirus genus. Sequences of alpha-CoVs detected in France from Miniopterus schreibersii (CS131010) clustered with sequences of alpha-CoVs detected in the same species in Germany. Likewise, sequences of alpha-CoVs from Myotis emarginatus from France (CS130613) clustered with sequences of alpha-CoVs detected in the same species in Luxembourg. Similarly, sequences of alpha-CoVs from Myotis nattereri in France (CS130938) clustered with sequences from the same species in Germany and England. Sequences of alpha-CoVs from Pipistrellus sp. in France (CS130412, CS130747, CS130894, CS130459, CS130860, and CS130786) were found in two distinct groups, well supported by Likelihood Ratio Test analysis and posterior probabilities. The other sequences of European alpha-CoVs from Pipistrellus sp. (KT345294, KT345295, and KT345296 in France, KF500945 in Italy, and GQ259964 in the Netherlands) also clustered into these two distinct groups.Nucleic sequences from hedgehogs (HE on Figure 2 ) were highly homogeneous and clustered with one of the other sequences of beta-CoV detected from hedgehogs in Germany. The sublineage of hedgehog sequences, human MERS-CoV, camel MERS-CoV-like, and bat MERS CoV-like sister clade is different from the other beta-CoVs, as shown by its well-supported Likelihood Ratio Test analysis and posterior probabilities.All sequences of beta-CoV detected from rodents in France were in the same group of sequences as beta-CoV from wild rabbits detected in the same country (L103, L118, L139, L172, L276) and domestic rabbits in China (GenBank No. JN874559). Sequences of beta-CoV from Apodemus sp. in France (R49, R86, R90, R150, R152) were in the same subgroup, whereas sequences from Myodes glareolus were found in three distinct subgroups: Two (R42, R91, R214, R218) with sequences of beta-CoV from Myodes glareolus from Germany (KM888128, KM888131 and KM888139); and the third with a sequence (R75) from Microtus arvalis from the Netherlands (KM888142).The two sequences of alpha-CoV from rabbits from France (L8, L16) were in the same group of alpha-CoVs from rodents in the United Kingdom (GenBank Nos. KU739072, KU739073, KU739074, and KU739071).This study described the presence of coronavirus infection in four wild mammal orders in France. Our results suggest that coronaviruses seem to be relatively common in these species, regardless of the place and time of sampling. The sequence length in our study was short because samples originated from other works for other purposes (except hedgehogs); however, interesting conclusions can be suggested. Studies about coronaviruses in bats are numerous in comparison with other wild mammals. In Europe, alpha-CoVs have been found in Pipistrellus sp. (France, Hungary, Romania, Germany, Spain, Italy, and the Netherlands), Myotis sp. (Hungary, Germany, Spain, the United Kingdom, and the Netherlands), Rhinolophus sp. (Bulgaria and Hungary), and Nyctalus sp. (Spain and Bulgaria) . In this study, seven Pipistrellus (six P. pipistrellus and one Pipistrellus sp.) were found to be positive in several geographically distant parts of the country (East, West and North of France). Two Myotis emarginatus in the same area in the East of France, one Myotis nattereri in the North of France, and two Miniopterus schreibersii were found in two areas in the South and East of the country. The number of positive samples by site (except for Miniopterus with two positive samples out of five) was highly dependent on the number of collected samples by site. The collection concerned 23 species of bats, out of the 35 known in mainland France. The collection of dead bats through the French network for lyssaviruses surveillance is performed by volunteers with no minimum sample sizes in each department, explaining the heterogeneous distribution of samples and the respective abundance of populations in biotopes close to humans. The use of this network allowed for better distribution of the dead animal samples across the national territory, compared to colony by colony collection. Nevertheless, the detection of the RNA sequence of RdRp in the infected animals was highly dependent on the state of the carcasses when discovered, and on storage before being dispatched to the laboratory.The overall prevalence in bats of 2.4% (CI 95%: 1.37; 4.12) was consistent with other studies in European countries, which have reported figures from 0% ( in Belgium) to 17.1% ( in the UK).This was the second report of Myotis emarginatus infection by coronavirus inside the usual range of this bat; the first was in Luxembourg . Interestingly, both cases reported here came from a department very close to Luxembourg. This species was rarely (about 20 samples) concerned in previous studies , but it would be relevant to study coronaviruses in these bats from other parts of its range to confirm the existence of a novel RdRp-grouping unit (RGU), as suggested in a recent study . In addition, our new nucleic sequences of coronavirus identified from Myotis emarginatus belong to Alphacoronavirus, like all sequences previously detected from other Myotis bats in European countries [13, , except in one more recent study that focused on this species and found alpha-CoV and beta-CoV shed by bats. In the same way, alpha-CoVs were also identified in Miniopterus schreibersii, as described in other studies [15, 16] , but no beta-CoV was found, probably because of the small number of fecal samples tested (one in , two in , and five in our study).The situation is more contrasted with Pipistrellus pipistrellus. Our results reinforce the findings of the other study carried out in France with the identification of alpha-CoV and a prevalence of 2.44% (CI 95%: 1.05; 5.58) in our study versus 5.4% (CI 95%: 0.6; 7.9) in Goffard et al. . However, Pipistrellus pipistrellus may be infected by beta-CoV as reported in Romania , while both betaand alpha-CoVs were found in this species in the Netherlands , but also when considering the Pipistrellus genus in general [1, 6, 21] .This study demonstrated the circulation of beta-and alpha-CoVsin wild rabbits. Most of the detected nucleic sequences were very close to the beta-CoV of domestic rabbits in China , and the others were close to an alpha-CoV KU739072 detected from rodents in the UK . Unfortunately, the lack of other studies targeting wild rabbits in its historical range prevents us from carrying out other comparisons. Continued surveillance of coronaviruses in rabbits would be useful to supplement our knowledge on infection in this species. Bearing in mind that wild rabbits are the ancestors of domestic rabbits, the study of coronaviruses in wild and domestic rabbits could be a useful model to better understand viral adaptive strategies. Indeed, human subtype OC43-related beta-CoV, Beta-CoV1 (lineage A), as identified is this study, is thought to have low host-specificity and to evolve by host-switching and recombination [5, 14, 23] . By contrast, alpha-CoVs are thought to be more specific to their hosts, at least in bats . The identification in intestinal rabbit samples from twos department of alpha-CoVs very close to those identified previously (KU739072) in rodents is not in favor of host specificity. Two hypotheses are possible. First, there is the circulation of non-specific alpha-CoVs within rodents, very close to viruses found in rabbits. The proximity of the Rodentia order to the Lagomorpha order, together forming the Glires clade , corroborates this hypothesis since rodents and rabbits share the same environment. Second, our sequences were short and were not sufficiently discriminant.All sequences from rodents in this study were identified as beta-CoV, whereas in the study in the UK and Poland , all identified coronaviruses from rodents belonged to alpha-CoVs.Species that tested CoV-positive in the 2016 study were Apodemus sylvaticus, Myodes glareolus, Microtus agrestis, and Rattus norvegicus. The CoV-positive species in our study were Myodes glareolus and Apodemus flavicollis. No Microtus sp. tested were positive, and no Rattus norvegicus were sampled. As for bats and rabbits, the circulation of both coronaviruses should be expected in rodents in Europe, but further investigations are needed to confirm this hypothesis.In addition, genetic segregation between coronaviruses from Apodemus and from Myodes was observed and suggested co-evolution with the host family . The identification of one Apodemus infected by a coronavirus very close to that observed in Myodes may indicate potential host switching.The situation seemed to be very different for hedgehogs with the observation of only beta-CoV. The prevalence in hedgehogs was higher than in other species. The only other study to our knowledge concerning coronaviruses in hedgehogs in Europe found results for Germany that are very similar to those in our study, with a beta-CoV close to those that identified in this study (96% to 98% identity), and with a prevalence in fecal samples of 58.9%. By contrast, in the same animal shelter, two groups of coronaviruses separated by a 3.1% to 3.4% nucleotide distance in an 816-nt RdRp fragment were found, but all viruses were in the same clade. In our study, our short nucleic sequences of the RdRp gene were highly homogeneous, despite the significant distance between the three animal shelters in France. Hedgehogs are territorial; they do not travel distances as long as those between the shelters and they do not migrate. The similarity between sequences could be the result of co-evolution between viruses and hedgehogs.This preliminary investigation of the presence of coronavirus in different wildlife species in France highlighted the presence of both alpha-and beta-CoVs in rodents, rabbits, and bats. The circulation of a range of coronaviruses within the same order or the same genus of mammals suggests the emergence of new coronaviruses by host switching, which is one of the main evolutionary mechanisms for coronaviruses .In addition, our study highlighted observations in French wildlife compatible with the second evolutionary mechanism for coronaviruses reported by Anthony et al. : Co-speciation, like for hedgehogs, Apodemus spp. and Myodes glareolus.Our results showed the existence of potentially evolving virus strains, with possible crossing of the species barrier. Further investigations are needed in these species and other species of wildlife to assess the potential risk for domestic animals and humans of widely distributed CoVs by studying the spike region involved in the crossing of the species barrier [26, 27] .\n",
            "Top-5 Cross-Encoder Re-ranker hits \n",
            "\n",
            "\t0.929\tAs the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in 'lockdown' directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation's economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and advances how smart city networks should work towards enhancing standardization protocols for increased data sharing in the event of outbreaks or disasters, leading to better global understanding and management of the same.The novel Coronavirus outbreak, (previously known as the 2019-nCoV and later renamed COVID-19 during the writing of this manuscript) is leading to the closure of entire cities in China, and causing stringent measures to be taken in others. While in distant different continents, far from China where the virus was first reported, places are being placed on high alert. In Wuhan, where the virus broke, schools, roads and markets have been shut down . The same is true in Hong Kong, Beijing and Hubei Province amongst surrounding areas, as precautionary measures are being emphasized to ensure that the spread of the virus is minimized, and complete and accurate information on the virus is being obtained . However, the rate of spread of the virus and the uncertainties surrounding the entire situation has led the World Health Organization (WHO) on 30 January 2019 to declare the Coronavirus outbreak a 'Global Public Health Emergency'. WHO determined, however, not to declare the outbreak a 'Public Health Emergency of International Concern' (PHEIC) which is a higher level of declaration. A PHEIC is defined as \"an extraordinary event which is determined to constitute a public health risk to other States through the international spread of disease and to potentially require a coordinated international response\" whose scope may include: serious, sudden, unusual or unexpected; carries implications for public health beyond the affected State's national border; and may require immediate international action .With the world having experienced some notable influenza pandemics in the past, a Global Initiative on Sharing All Influenza Data (GISAID) platform was established and was instrumental in the rapid sharing of information by the Chinese scientists regarding the emergence of the COVID-19 virus. Through this platform, scientists from other regions were observed to gain access to information and are, subsequently, able to act in a much faster capacity; like in the case of scientists from the Virus Identification Laboratory based at Doherty Institute, Australia, who managed to grow a similar virus in the laboratory after accessing the data shared by the Chinese scientists .Beyond the aspect of pandemic preparedness and response, the case of COVID-19 virus and its spread provide a fascinating case study for the thematics of urban health. Here, as technological tools and laboratories around the world share data and collectively work to devise tools and cures, similar efforts should be considered between smart city professionals on how collaborative strategies could allow for the maximization of public safety on such and similar scenarios. This is valid as smart cities host a rich array of technological products [6, 7] that can assist in early detection of outbreaks; either through thermal cameras or Internet of Things (IoT) sensors, and early discussions could render efforts towards better management of similar situations in case of future potential outbreaks, and to improve the health fabric of cities generally. While thermal cameras are not sufficient on their own for the detection of pandemics -like the case of the COVID-19, the integration of such products with artificial intelligence (AI) can provide added benefits. The fact that initial screenings of temperature is being pursued for the case of the COVID-19 at airports and in areas of mass convergence is a testament to its potential in an automated fashion. Kamel Boulos et al. supports that data from various technological products can help enrich health databases, provide more accurate, efficient, comprehensive and real-time information on outbreaks and their dispersal, thus aiding in the provision of better urban fabric risk management decisions.The above improvements in the healthcare sector can only be achieved if different smart city products are fashioned to support standardized protocols that would allow for seamless communication between themselves. Weber and Podnar Žarko suggest that IoT devices in use should support open protocols, and at the same time, the device provider should ensure that those fashioned uphold data integrity and safety during communication and transmission. Unfortunately, this has not been the case and, as Vermesan and Friess explain, most smart city products use proprietary solutions that are only understood by the service providers. This situation often creates unnecessary fragmentation of information rendering only a partial integrated view on the dynamics of the urban realm. With restricted knowledge on emergent trends, urban managers cannot effectively take decisions to contain outbreaks and adequately act without compromising the social and economic integrity of their city. This paper, inspired by the case of the COVID-19 virus, explores how urban resilience can be further achieved, and outlines the importance of seeking standardization of communication across and between smart cities.With the advent of the digital age and the plethora of Internet of Things (IoT) devices it brings, there has been a substantial rise in the amount of data gathered by these devices in different sectors like transport, environment, entertainment, sport and health sectors, amongst others . To put this into perspective, it is believed that by the end of 2020, over 2314 exabytes (1 exabyte = 1 billion gigabytes) of data will be generated globally from the health sector. Stanford Medicine acknowledges that this increase, especially in the medical field, is witnessing a proportional increase due to the increase in sources of data that are not limited to hospital records. Rather, the increase is being underpinned by drawing upon a myriad and increasing number of IoT smart devices, that are projected to exponentially increase the global healthcare market to a value of more than USD $543.3 billion by 2025 . However, while the potential for the data market is understood, such issues like privacy of information, data protection and sharing, and obligatory requirements of healthcare management and monitoring, among others, are critical. Moreover, in the present case of the Coronavirus outbreak, this ought to be handled with care to avoid jeopardizing efforts already in place to combat the pandemic. On the foremost, since these cut across different countries, which are part of the global community and have their unique laws and regulations concerning issues mentioned above, it is paramount to observe them as per the dictate of their source country's laws and regulations; hence, underlining the importance of working towards not only the promoting of data through its usage but also the need for standardized and universally agreed protocols.While the significance of such data in advancing efficiency, productivity and processes in different sectors is being lauded, there are criticisms arising as to the nature of data collection, storage, management and accessibility by only a small group of users. The latter particularly includes select ICT corporations that are also located in specific geographies [6, . These criticisms are justified, as in recent years, big data is seen as the new 'gold rush' of the 21st century and limiting its access means higher economic returns and increased influence and control at various scales to those who control data. These associated benefits with big data are clearly influencing geopolitical standings, in both corporate and conventional governance realms, and there is increased competition between powerful economies to ensure that they have the maximum control of big data. As case in point is the amount of 'push and pull' that has arisen from Huawei's 5G internet planned rollout . Though the latter service offers unprecedented opportunities to increase internet speeds, and thereby influence the handling of big data, countries like the U.S. and some European countries that are key proponents and players in global political, economic and health landscapes, are against this rollout, arguing that it is a deceptive way of gathering private data under the guise of espionage. On this, it has been noted that the issue of data control and handling by a few corporations accords with their principles of nationalism, and that these work for their own wellbeing as well as to benefit the territories they are registered in. Therefore, geopolitical issues are expected on the technological front as most large data-rich corporations are located in powerful countries that have influence both economically, health-wise and politically . Such are deemed prized tokens on the international landscape, and it is expected that these economies will continue to work towards their predominant control as much as possible. On the health sector, the same approach is being upheld where critical information and data are not freely shared between economies as that would be seen to be benefiting other in-competition economies, whereas different economies would cherish the maximization of benefits from such data collections.In addition to the obvious deep-rooted social issues related to nationalism, other challenges include the increasing movement of people globally that is being enhanced by reduced costs and higher speed. In particular, these challenges are more pronounced when it comes to public health. This is because most of the health-related data collected not only can compromise local nations, but also captures those of travelers. In such cases, in a bid to improve the health status of a nation, it becomes paramount to factor in data from other regions necessitating unhindered sharing of this data.Such data-sharing truth is emphasized in situations like the recent case of Coronavirus outbreak threatening the global health environment, facilitated by air transportation. The virus was first reported in Wuhan, China, and in a matter of three weeks (by 17th January 2020) over 300 cases were confirmed in that region, and 10 days later (26th January 2020), a total of 2014 cases of Coronavirus have been reported, with 684 of those being confirmed, and with 29 reported outside China. The fatalities from the virus stands at 56 as of 26th January 2020 . The virus had then been confirmed in various countries including Taiwan, South Korea, Japan, Thailand, France, the United States, Singapore and Vietnam .In the above case, though major cities are known to prepare themselves for potential outbreaks, their health policies and protocols are observed to diverge from one another. Thus, without a global collaborative approach, progress towards working for a cure and universally acceptable policy approach can take longer. Such fears, of a lack of international collaboration, were highlighted by the World Health Organization (WHO) during an emergency meeting in Geneva on 22nd January 2020 to determine whether the virus outbreak had reached a level warranting international emergency concern. However, WHO was satisfied that China was being proactive in this case, unlike in 2002, when China withheld information on the outbreak for far too long, causing delays in addressing the epidemic . As in this instance, it is the opinion in this paper that if there was seamless collaboration and seamless sharing of data between different cities, it would not warrant such a high-level meeting to result in action, and instead, a decision could have been made much earlier. On this, the saddest part is that some global cities are less prepared to handle the challenges posed by this type of outbreak for lack of information on issues like symptoms of the virus, the protective measures to be taken, and the treatment procedures that an infected person should be processed through, amongst other issues.The timely response by stakeholders in regard to this new outbreak are commendable compared to previous cases. The latter includes the Severe Acute Respiratory Syndrome (SARS) outbreak in 2002 that took substantial time (from November 2002 to April 2003) to identify and be dealt with ; the Ebola outbreak in West Africa in 2013 that took months to determine; and the Zika Virus that was first reported in 2014 before being successfully identified in 2015.With the Coronavirus (COVID-19) , it took only 17 days (31st December 2019 to 17th January 2020) to be identified. The sharing of data has also been quicker, as immediately after the virus' genetic sequence was discovered, Chinese scientists were able to share the information with the WHO, thus helping in its identification and enabling the auctioning of precautionary measures in other countries. Latest technological tools have also allowed for the receipt of information in realtime, in contrast to traditional epidemiological approaches that would have required months to identify the outbreak type . Similarly, though substantial data and information on the disease has been shared, Wetsman acknowledges that there is a lack of some vital information, like the ease of spread of the virus from person-to-person, and this is a key to containing the disease as interactions between people from different parts of the globe are still active. This hindrance can be made further possible as many cities advance in their smart and safe city model implementation towards constructing sufficient soft and hard urban infrastructures equipped with, for example, thermal imagery sensors to allow for early detections. However, while that is the case, data access to many is a challenge because the information is often seen as being sensitive for national security reasons, whilst at the same time, acknowledging that a virus outbreak is an equal threat to both national security and the economy.The outbreak of any disease has significant impacts on local economies across the globe. For instance, when SARS (Severe Acute Respiratory Syndrome) (SARS-CoV) broke in China in 2002, it was estimated, that the Asian region incurred tremendous negative impacts socially, health-wise and economically, potentially amounting to Asian regional economy losses of between USD $12-18 billion from tourism, travel and retail sales industries alone . The Zika virus outbreak, spread by daytime-active Aedes mosquitoes, is estimated to have cost equator-belt local economies in affected areas between USD $7 and USD $18 billion . The Ebola virus (or Ebola hemorrhagic fever (EHF)) caused an estimated loss of USD $2.2 billion in GDP in three West African economies (Guinea, Liberia and Sierra Leone) in 2015 alone . In regard to the current epidemic of Coronavirus, though it is too early to quantify or project its impacts on the global economy, there are fears that it may take the precedent of other outbreaks where billions of dollars will be lost. The foundations for this escalating loss can be witnessed in the rapid growth of travel bans being enacted by some countries and their international airports, especially specifically restricting people from visiting the affected regions in China and their growth into general non-Chinese travel movements. On this, noting that the outbreak came almost on the eve of the Lunar New Year celebrations, and that it had been estimated that over 400 million people were expected to travel in different parts of the world and China to observe this festivity, the majority have had to reconsider their options as to flights, hotels and entertainment events due to service provider cancellations . Those who had already booked their flights are expected to receive their refunds following the directive by the Civil Aviation Administration of China, however, this move has already affected the share value of Chinese airline companies .The above impacts demonstrate that the issues of virus outbreaks transcend urban safety and impacts upon all other facets of our urban fabric. Therefore, it becomes paramount to ensure that the measures taken to contain a virus transcend nationalist agendas where data and information sharing is normally restricted, to a more global agenda where humanity and global order are encouraged. With such an approach, it would be easier to share urban health data across geographies to better monitor emerging health threats in order to provide more economic stability, thereby ensuring no disruptions on such sectors like tourism and travel industries, amongst others. This is possible by ensuring collaborative, proactive measures to control outbreak spread and thus, human movements. This would remove fears on travelers, and would have positive impacts upon the tourism industry, that has been seen to bear the economic brunt whenever such outbreaks occur. This can be achieved by ensuring that protocols on data sharing are calibrated to remove all hurdles pertaining to sharing of information. On this, Lawpoolsri et al. posits that such issues, like transparency, timelessness of sharing and access and quality of data, should be upheld so that continuous monitoring and assessment can be pursued.Virus outbreaks in recent years have shown that, in the urban realm, data, including health data, can be sourced from diverse places. Presently, in the case of Coronavirus (COVID-19) outbreak, data is being collected from airports through screening and monitoring, through the use of smart sensors installed in airport infrastructures and from personnel working in those air/seaports. For instance, it has been reported that in the U.S.A., screening is being carried out at 20 different airports to ensure that possible affected people are intercepted for quarantine at the point of entry. Beside airports, as reported by Buckley and May , data is also being collected at bus terminals, market places (in Wuhan), subways, and also in health facilities where patients are taken for further medical attention. Such is prevalent especially in China, and other Asian regions where cases of the virus have been recorded and confirmed.In addition to these methods, other smart city data sources include the application of terminal tracking systems that are mostly emphasized in Safe City concepts, where, at the point of entry or departure, relevant data is collected and analyzed. Li et al. highlights that sensors installed in such locations have the potential to receive and distribute data in real-time to digital infrastructures within the network, and their interconnectedness in the network renders them extremely efficient in providing real-time updates on different issues. Urban areas are also known to be amassed with numerous Urban Health sensors, some of which are wearable. Though these are not specifically fashioned to track the present case of virus outbreak, they are able to track other related parameters like heartbeat, blood pressure, body temperature and others variables, that when analyzed can offer valuable insights. Loncar-Turukalo et al. hail these devices for their role in transforming the health care sector especially by allowing for Connected Health (CH) care, where data collected from them can be analyzed and provide insightful information on the health scenario in any given area. Vashist et al. further highlight how emerging features such as spatiotemporal mapping, remote monitoring and management, and enhanced cloud computing capabilities can emanate from such endeavours, leading to better urban management potential.While it is true that the basic source of medical data is generally sourced from general practitioners or medical laboratories-a fact that has also been affirmed in the case of the current epidemic-this paper explores how data sourced from an urban perspective can contribute to the medical narrative. The conviction to dwell on the urban realm in this manuscript is based on the fact that the current epidemic (COVID-19) is transmitted majorly through human-to-human contact, and in most cases, especially where the spread is reported in a different country, the first point of contact is an urban area, where large groups of people convene, like airports or subway stations. In most cases, such facilities, which are mostly based in urban areas, are observed to have installed surveillance technologies to ensure that anyone showing any symptoms of the disease are identified and quarantined. However, even in such cases, as underlined in the present manuscript, the need for anonymizing medical data is emphasized to ensure that the use of current technologies does not breach data privacy and security requirements, across different geographies. In this case, novel technologies like Blockchain technologies and quantum cryptography can aid in the discussion and be made to integrate with data collecting technologies. This would render an increased wealth of data from both the medical field and smart city operators, while ensuring privacy and security; hence, aiding in providing relevant information for better informed decisions.However, despite the indisputable roles that installed devices play in providing relevant health information, their data communication aspect needs to be reviewed. First, communications are seen to be geography-restricted (restricted to a given location), such that they seldom expand or communicate with their like, installed beyond their restricted areas. Secondly, these devices are usually sourced and installed by separate corporations that maintain unique and specific standards for data processing and sharing, and accordingly, tying cities to the sole usage of their product(s). Such strategies are adopted as private corporations try to maximize their economic gains, since the digital solution market is a lucrative one and is expected to continue growing and expanding [6, 7] .For its current application, the standardization of protocols as elaborated in this manuscript need to be pursued to ensure that there is seamless sharing of information and data. By doing this, it is expected that issues like burdens of collecting data, accuracy and other complexity that are experienced (when systems are fragmented) are reduced or eliminated altogether. The standardization can be achieved by, for example, ensuring that all the devices and systems are linked into a single network, like was done in the U.S., where all the surveillance of healthcare were combined into the National Healthcare Safety Network (NHSH) . The fact that cities are increasingly tuning on the concept of Smart Cities and boasting an increased adoption rate of technological and connected products, existing surveillance networks can be re-calibrated to make use of those new sets of databases. Appropriate protocols however have to be drafted to ensure effective actions while ensuring privacy and security of data and people.With scenarios like the present Coronavirus (COVID-19) outbreak, that not only impacts upon the economic status of cities, but also affects their social standing, it becomes imperative to emphasize the adoption of universal standards for data sharing. Such a move could have far reaching impact across cities and territories especially in positively combating outbreaks and disasters in a quicker, safer and standardized way, such that when the cure is discovered, the results can be replicated in various parts of the globe. With a collaborated data sharing protocol, it would be possible to have a larger dataset resulting in increased processing capabilities especially with technologies that are powered by artificial intelligence (AI) tools. Through this way, as noted by Jiang et al. and Allam , it would be possible to facilitate early detection, achieve better diagnosis and provide better urban management decisions for increased efficiency for virus containment.An example of how beneficial collaboration and sharing of data can be occurred during the 2014 Ebola outbreak in West Africa where scientists, health workers and clinicians, amongst other stakeholders from around the world, openly worked together and were able to contain the spread of this pandemic . On this front, Boué et al. highlight that levels of trust and transparency need to be reviewed and enhanced to facilitate unfettered data generation and sharing. Such could lead to an even earlier detection scenario of future virus outbreaks, and in the better curative management of the same, without minimal compromise on urban functions and on an urban economy.Furthermore, in cases of emergencies like the current outbreak of COVID-19 and any other, the need for observance of regulatory practices and international healthcare guidelines are paramount. This would ensure that both healthcare professionals and the general populace are informed, protected and remain within the prescribed rules and regulations. As noted by the WHO , the healthcare guidelines and regulatory practices are advanced to also ensure that the health risk in question is reduced together with its consequences. In the current era of technological advancement, such regulations and guidelines are paramount as they have potential to lead to positive or negative outcomes. The position of this paper is to advance that it now possible to integrate technologies like the use of smart devices through IoT networks and wearable devices, data from mobile apps and others to help users to share information with accredited and certified health professionals, and in this case, improve the outcomes for better cross disciplinary and more resilient protocols and policies.\n",
            "\t0.917\tThe Middle East respiratory syndrome coronavirus (MERS-CoV) has spread through 27 countries and infected more than 2,200 people since its first outbreak in Saudi Arabia in 2012. The high fatality rate (35.4%) of this novel coronavirus and its persistent wide spread infectiousness in animal reservoirs have generated tremendous global public health concern. However, no licensed therapeutic agents or vaccines against MERS-CoV are currently available and only a limited few have entered clinical trials. Among all the potential targets of MERS-CoV, the spike glycoprotein (S) has been the most well-studied due to its critical role in mediating viral entry and in inducing a protective antibody response in infected individuals. The most notable studies include the recent discoveries of monoclonal antibodies and development of candidate vaccines against the S glycoprotein. Structural characterization of MERS-CoV S protein bound with these monoclonal antibodies has provided insights into the mechanisms of humoral immune responses against MERS-CoV infection. The current review aims to highlight these developments and discuss possible hurdles and strategies to translate these discoveries into ultimate medical interventions against MERS-CoV infection.The rapid emergence and dissemination of infectious diseases has taken a heavy toll on humans since the beginning of the twenty-first century. One of the most well-known examples was the outbreak of severe acute respiratory syndrome (SARS) in the winter of 2002 and 2003, caused by a novel coronavirus (SARS-CoV) [1, 2] . In distinct contrast to the mild human coronaviruses HCoV-229E , HCoV-OC43 , HCoV-NL63 , and HCoV-HKU1 , infection with SARS-CoV frequently resulted in severe symptoms including fever, dry cough, shortness of breath and pneumonia. Transmission of SARS-CoV was primarily from person to person and most cases occurred in health care settings lacking adequate infection control precautions . The SARS outbreak had severe consequences in 29 countries and regions, infecting 8096 people worldwide with a fatality rate of approximately 10% . There are still no vaccines or therapeutics specific to SARS-CoV available 16 years after the SARS outbreak. It is not hard to imagine how catastrophic it would be if SARS-CoV were to hit the human community again.While SARS-CoV remains a mystery and a loose cannon, another novel coronavirus emerged in Saudi Arabia in 2012, later known as the Middle East respiratory syndrome coronavirus (MERS-CoV) . The fatality rate of MERS-CoV infection is approximately 35 .4%, and new cases as well as associated deaths continue to arise to date . Despite that most cases have been attributed to human-to-human transmission, MERS-CoV does not appear to transmit efficiently among humans unless there is close contact. The exact source of MERS-CoV and its routes of transmission to humans still remain uncertain. Dromedary camels are believed to be the animal reservoir for MERS-CoV because isolates from camels are almost identical to those from human, and that many domestic camels are seropositive for MERS-CoV (reviewed in [10, 11] ). Furthermore, current evidence strongly suggests that bats are the original source for MERS-CoV, as many coronaviruses phylogenetically related to MERS-CoV originate in bats, including BatCoV-HKU4, BatCoV-HKU5 and other MERS-related coronaviruses . The BatCoV-HKU4 was also shown to be able to engage the cellular receptor of MERS-CoV, adding evidence to the bat origin theory . However, there has not yet been direct evidence for isolating MERS-CoV from bats (reviewed in [10, 11, 17] ).Great efforts have been made to develop preventive and therapeutic interventions against MERS-CoV infection. In particular, monoclonal antibodies and vaccines targeting the Spike glycoprotein are major areas of focus due to its critical role in mediating viral entry, and its potential in inducing protective antibody responses in infected individuals. So far, more than twenty monoclonal antibodies with nanomolar neutralizing activities have been reported and many vaccine candidates are underway in preclinical and clinical studies. In this review, we aim to capture the current advances and discuss possible strategies to translate these discoveries into an ultimate medical intervention against MERS-CoV infection.MERS-CoV belongs to the genus betacoronavirus of the coronaviridae family . It is an enveloped, single-stranded, positive-sense RNA virus with a helical capsid structure (Figure 1(A) ). The genome of MERS-CoV is around 30 kb (30,119nt) long and encodes 4 structural proteins (Spike, Envelope, Membrane, and Nucleocapsid) and 16 nonstructural proteins (Figure 1 (C)) . Like other coronaviruses, the MERS-CoV uses its spike (S) glycoprotein to interact with cellular receptors and enter into the target cell . As a unique structural component of the virion membrane, the S glycoprotein assembles into trimers and forms large protruding spikes on the surface of the virion . The S glycoprotein is a typical type I membrane glycoprotein consisting of a globular S1 domain at the N-terminal, followed by a membrane-proximal S2 domain and a transmembrane (TM) domain . The S1 domain mediates viral attachment and contains the RBD (receptor binding domain), which determines the host range and cellular tropism for MERS-CoV . Similar to other coronaviruses, the S2 domain of MERS-CoV, mediating membrane fusion, contains the hydrophobic fusion peptide (FP) at the N-terminus as well as two heptad repeats designated as HR1 and HR2 (Figure 1 (C)) . Through co-purification with the MERS-CoV S1 domain, Raj and colleagues identified that dipeptidyl peptidase 4 (DPP4, also known as CD26) functions as a cellular receptor for MERS-CoV .The MERS-CoV virion enters the host airway cells in the respiratory tract through fusion with either the plasma or endosomal membrane . Binding between RBD and the cell receptor triggers a cascade of conformational changes that lead to the formation of a pre-hairpin intermediate of S2, in which the hydrophobic HR1 is exposed and allows the fusion peptide to insert into the target cell membrane. This transient S2 intermediate then refolds with HR2 into a stabilized trimer of hairpins, also called six-helix bundle structure (6-HB), which brings the target cell membrane into close proximity of the viral envelope, resulting in the completion of the fusion process and initiation of the virus life cycle (Figure 1(B) ). Structure-based design of various peptides able to block the formation of 6-HB have demonstrated potent inhibition on MERS-CoV replication and spike-mediated cell-cell fusion, showing great promise for further development into effective viral fusion inhibitors for treating MERS-CoV infection [26, . Among them, the peptide EK1 is effective to multiple human coronaviruses apart from MERS-CoV and therefore serves as a potential pan-coronavirus fusion inhibitor .Recently, structural studies on the prefusion state spike protein of MERS-CoV and SARS-CoV have provided more insights into the spike-mediated membrane fusion process . The MERS-CoV spike protein trimerizes and folds into a metastable prefusion conformation on the virion surface, in which three S1 domains fold into a steady trimer structure and sit on top to stabilize the coiled S2 domains (Figure 2(A-B) ). We and others have identified that the RBD of SARS-CoV and MERS-CoV can be found either buried ('down' position) or exposed ('up' position) in the spike trimer structure [31, . The two conformational states of RBD may have distinct roles during receptor binding and membrane fusion: only the RBDs in 'up' position, but not those in 'down' position, can bind to the cell receptor DPP4 (Figure 2 (C-D)). Great steric clash was observed between DPP4 and neighboring spike protomers when we mapped it to the RBD in 'down' position ( Figure 2(C-D) ). Transformation of the RBD from the buried to the exposed state is therefore a prerequisite for receptor binding (Figure 2(G-H) ). On the other hand, this conformational change also seems to open up the stable cap structure sitting above the S2 cores ( Figure 2 (E-F)). This may lead to disassociation of S1 trimer and exposure of the fusion apparatus, triggering the membrane fusion process.To gain a better understanding of MERS-CoV interaction with cellular receptors at atomic levels, we and others have determined the crystal structure of MERS-CoV RBD bound to the extracellular domain of its cellular receptor dipeptidyl peptidase 4 (DPP4) [23, 24] . We showed that MERS-CoV RBD consists of a core and a receptor binding subdomain. MERS-CoV RBD and the related SARS-CoV RBD share a high degree of structural similarity in their core subdomains, but are notably divergent in the receptor binding subdomains . The receptor binding subdomain of MERS-CoV RBD directly interacts with blades 4 and 5 of DPP4 propeller instead of its intrinsic hydrolase domain. The interface consists of a buried surface of ∼2550 Å 2 involving 14 residues in receptor binding subdomain interacting with 15 residues in DPP4. The actual binding forces are mediated through two major binding patches. Patch 1 represents 49% of buried surface and forms between the C-terminal end of the long loop connecting the β6 and β7 strands and blade 4 of DPP4. Patch 2 occupies 51% of buried surface and forms a slightly concaved outer surface at the far end of the MERS-CoV receptor binding subdomain and a linker containing a short helix between blade 4 and blade 5 of DPP4. The concaved outer surface is made by the short β6 strand, C-terminal parts of β5 and β7 strands, N-terminal part of β8 strand and the β5-β6 linking loop. It is hoped that better understanding of the atomic details of the spike glycoprotein, as well as the interface between MERS-CoV RBD and DPP4 will provide the structural basis for rational design and development of therapeutics and vaccines against MERS-CoV infection. Neutralizing antibodies are a major component of protective immunity against viral infection in humans. Polyclonal by nature, the antibody response in vivo mobilizes a dynamic and complex mixture of monoclonal antibodies (mAbs) that work in concert to target various antigenic domains on the viral envelope glycoprotein. Identifying the neutralizing mAbs that constitute the neutralizing activity of polyclonal response and their recognized antigenic domains has therefore become the first crucial step towards gaining a better understanding of the protective antibody response, developing clinical intervention methods, and designing immunogens capable of eliciting neutralizing antibodies.Great achievements have been made in the isolation of neutralizing mAbs in the past few years using various technology platforms ( Figure 3 ). Up till now, more than 20 mAbs, most of which are human or humanized antibodies, have been described by scientists from all over the world. These antibodies are listed in chronological order of publication in Table 1 , together with their unique biochemical and antiviral properties against MERS-CoV infection observed in cell culture and experimental animal models.It is apparent that the single chain fragment variable (scFv) library approach allows rapid discovery of mAb, without time constraints from immunizing experimental animals or approaching convalescent individuals of MERS-CoV infection. The earliest mAbs reported in 2014 were identified through screening non-immune human scFv libraries with the ectodomain of S glycoprotein (mAb 3B11) or soluble RBD from S glycoprotein (MERS-4, MERS-27 and the m336 panel) [37, 42] as bait protein (Figure 3(A) ). These antibodies all demonstrated high neutralizing activities and therefore were widely used as reference antibodies in later studies.Antibodies have also been generated from immunized animals (Figure 3 (B)). Several groups have reported mAbs isolated from either wild-type inbred mice or transgenic mice expressing human antibodyvariable heavy chains and κ light chains. Mersmab-1 (known as hMS-1 after humanization) was isolated from mice immunized subcutaneously with chimeric S1-Fc [47, 48] . The mAbs 2E6 and 4C2 (humanized form 4C2 h) were isolated in mice immunized with recombinant RBD produced in insect cells . Furthermore, two human-like mAbs, REGN3048 and REGN3051, were directly cloned from transgenic mice expressing human versions of the antibody after immunization with DNA encoding S glycoprotein and purified recombinant S glycoprotein . Both mAbs have been tested in humanized mice models and in non-human primates [51, 52] . The authors indicated that the advantages of their system not only lay in the human component of their antibodies but also in the quick speed associated with isolation and production, since no humanization or optimization step was required. Currently, REGN3048 and REGN3051 have entered phase I clinical trials.Most of the mAbs reported so far target the RBD region of S glycoprotein, but RBD does not seem to be the only target for anti-MERS-CoV antibody responses. Recently, a mAb targeting the S1 N-terminus domain (NTD) region, which does not contain RBD, was isolated from mice immunized with S glycoprotein . This antibody, 5F9, was shown to successfully block virus entry in cell culture models and the efficacy was comparable to other mAbs in IC 50 . Further, the mAb panel D12, F11, G2 and G4 were generated by priming Similarly, 3B11 and m336 were isolated from non-immunized human scFv phage libraries with MERS-CoV S protein or RBD protein as bait protein, respectively. (B) Monoclonal antibodies sorted from immunized animals. The antibodies 5F9, hMS-1 (Mersmab-1), D12, F11, G2, G4, 4C2h (4C2), REGN3048 and REGN3051 were isolated from mice immunized with the indicated vaccines labelled in the colour-coded boxes, each representing a different immunogen; the bait or target protein for antibody selection were also listed. The mice from which REGN3048 and REGN3051 were isolated were given the pale blue colour to indicate that they express human immunoglobulin genes. NbMS10-Fc, JC57-11, JC57-13, JC57-14, F1B-H1 and HCAb-83 were isolated from larger animal including llama, rhesus macaque and camels as indicated. The vaccines and selection criteria were also shown. NTD (N-terminal domain), Fc (fragment constant). (C) Monoclonal antibodies isolated from human survivors recovered from MERS-CoV infection. MERS-GD27, MERS-GD33, LCA60, CDC-C2, CDC-C5, CDC-A2 and CDC-A10 were generated by culturing B cells sorted from the patient and screening for MERS-CoV-specific antibodies. MCA1 was produced by constructing a phage library displaying scFv cloned from a convalescent patient. mice with DNA encoding the full-length S glycoprotein and boosting them with S1 protein. Among them are two mAbs that target the non-RBD S1 (mAb G2) and S2 region (mAb G4), respectively . These non-RBD-binding antibodies potently neutralized pseudoand live MERS-CoV in cell culture and were also protective in mouse models [49, 50] . Together, the development of these antibodies elucidates that RBD may not be the single target for anti-viral antibody response. More studies are needed to elaborate the detailed mechanisms for these antibodies.Apart from the traditional approach of isolating mAbs from immunized mice, several groups have turned to larger animal models for antibody isolation. One group immunized rhesus macaques with combined DNA and protein vaccines and isolated a panel of mAbs, including JC57-11, JC57-13, JC57-14, and FIB-H1, targeting both RBD and non-RBD S1 region of the S glycoprotein, all with potent neutralizing activities . Another group immunized llama with recombinant RBD and screened the nanobody library for high-affinity single heavy chain antibody . These nanobody-derived mAbs are smaller in molecular weight and more stable than traditional antibodies, and may provide a new option for future antibody isolation.In terms of closeness to authentic human antibodies, no approach can compete with those based on direct B cell cloning from convalescent individuals. One such mAb LCA60 was isolated from memory B cells of human survivors of MERS-CoV infection and was among the most potent mAbs reported in neutralizing pseudo-and live viruses . More mAbs isolated from human survivors were described as more convalescent blood samples became available, including MCA1 , CDC-C2, CDC-C5, CDC-A2, CDC-A10 , MERS-GD27, and MERS-GD33 [58, 59] (Figure 3 (C)), all with potent neutralizing activities against MERS-CoV. The mAbs LCA60, CDC-C2, MCA1, and MERS-GD27 were also tested to be protective in animal models.As MERS-CoV research progressed quickly in the past few years, many mAbs have been tested for prophylactic or therapeutic protection efficacy in human DPP4 transgenic / transduced mice models, and a few have entered large animal model trials such as in rabbits or non-human primates (NHPs). However, as different animal models were established among labs worldwide with slightly different evaluation end points, it is difficult to make a direct comparison among these mAbs. This is also true for in vitro evaluation of neutralizing activitiessince different cell lines, pseudoviruses, and neutralizing assay techniques are utilized, the published IC 50 values can only serve as indirect reference for comparison. Head to head comparison in the same experimental system would be required to identify the most protective mAb or combination of mAbs against MERS-CoV infection in order to proceed to clinical trials.We and others have carried out structural studies of MERS-CoV neutralizing antibodies in complex with MERS-RBD to understand neutralizing mechanism at atomic levels ( Figure 4) . Based on the epitopes revealed by structural studies, MERS-CoV antibodies targeting RBD can be classified into three groups (Figure 4(B) , Table 1 ).The first group consists of antibodies MERS-27, D12, 4C2 and JC57-14, which interact with the Cterminal segment of the β6-β7 loop and β7 strand of RBD by both heavy and light chains (Figure 4(B) ) [38, 49, 50, 55] . Their common epitopes on the RBD include residues Val527, Ser528, Ile529, Val530, Pro531, Ser532, Trp535, Glu536 and Asp539 in the β6-β7 loop. The residues Trp535, Glu536 and Asp539 also happen to be within the DPP4-binding site patch 1 of MERS-CoV RBD , mediating interaction with Lys267 and the carbohydrate moiety linked to Asn229 of DPP4 . Therefore, the Group 1 antibodies would directly compete with DPP4 in binding to RBD by interfering with both protein-protein and protein-carbohydrate interactions between RBD and DPP4. Structural super-impositions also showed that these four antibodies and DPP4 would have steric clashes between the variable domain of the heavy chain and the propeller domain of DPP4 if they simultaneously bind to RBD (Figure 4(C) ).The second group consists of antibodies m336, MCA1, CDC2-C2 and MERS-GD27, which interact with the β5-β8 strands, β5-β6 loop and β6-β7 loop in RBD mainly by the heavy chain (Figure 4(B) ) [43, 50, 56, 58] . Their common epitope consists of Phe/ Leu506, Asp510, Trp535, Glu536, Asp539, Tyr540, Tyr541, Arg542, and Trp553. Although antibodies in both Group 1 and Group 2 share the binding residues Trp535, Glu536 and Asp539, their approaching angles to the RBD are significantly different. As shown in Figure 4 (C), the approaching angle of Group 2 antibodies is closer to that of DPP4 by rotating approximately 90 degrees anti-clockwise from that of Group 1 antibodies, thereby generating more steric clashes with DPP4. This is also evidenced by a larger overlap between the common epitope of Group 2 antibodies and DPP4-binding site on RBD . As a representative of Group 2 antibodies, m336 exhibits very potent neutralizing activity by not only mimicking critical interactions between RBD and DPP4 but also adopting an approaching angle similar to that of DPP4 (Figure 4(C) ).The third group consists of antibody MERS-4 and its variant MERS-4V2 with four residue replacements in the HCDR3 (Figure 4(B) ) . By structural determination, it was shown that MERS-4 Fab and MERS-4V2 scFv share the same mode of binding to the RBD (Figure 4(B) ) . Analysis of the RBD/MERS-4V2 complex structure showed that the antibody contacts with the β5-β6, β6-β7 and β7-β8 loops of the receptor-binding subdomain in RBD . The epitope involves Leu507, Ser508, Gln516, Asn519, Asn521, Gln522, Tyr523, Pro525, Lys543, Leu545, and Gly550 . To be note, the MERS-4 epitope has no overlap with DPP4-binding site (Figure 4(C) ). By approaching the RBD outside the DPP4-binding site, MERS-4 recognizes a unique epitope different from all previously reported RBD-targeting antibodies. Comparisons of RBD in DPP4-bound and MERS-4-bound states revealed that binding of MERS-4 induces or fixes the β5-β6 loop into a conformation in which it folds into a shallow groove on the RBD interface critical for accommodating a short helix of DPP4, thereby indirectly disrupting the interaction between RBD and DPP4 (Figure 4(C) ). Such different epitope and mechanism enable MERS-4 to synergize with other antibodies including RBD-targeting MERS-27 and m336 in neutralization, which provides valuable addition for the combined use of antibodies against MERS-CoV infection .In addition to the aforementioned ten antibodies targeting RBD, the near atomic resolution cryo-EM structures of the trimeric MERS-CoV spike and its complex with antibody G4 were also determined ( Figure 4(D) ) . G4 is the first reported S2-targeting antibody and its epitope consists of a glycosylated, solvent-exposed loop residing in a connector domain between the HR1 and HR2 of the S2 subunit. In the unbound spike trimer structure this loop is largely disordered, whereas it extends out from two β-strands and is surrounded by all six CDRs (complementarity determining regions) of the mAb G4 upon antibody recognition (Figure 4(E) ). The specific spike-G4 interaction may stabilize the loop and further impede conformational changes of S2 subunit essential for membrane fusion after DPP4 binding. The binding epitope for G4 in S2 subunit is more conserved than RBD among MERS-CoV isolates, shedding light on G4 as a potential broad-spectrum neutralizing antibody for MERS-CoV. Yet this loop between HR1 and HR2 is variable in sequence and length among different viruses even in lineage C betacoronaviruses , limiting its application to other coronaviruses. In terms of pan-coronavirus medical countermeasures (MCMs), the recently developed fusion inhibitor peptide EK1 is a potential candidate. The peptide EK1 was designed to target the more conserved HR1 region of the S2 stem, and was shown to block cell-cell fusion induced by spike protein from multiple human coronaviruses .In general, most reported MERS-CoV neutralizing antibodies recognize the RBD in the S1 subunit, and these antibodies are highly potent in neutralization. These facts show that the RBD in the S1 subunit is a major vulnerable site for antibody recognition and neutralization. To be note, the RBD is also the region where most naturally occurring mutations of the S glycoprotein occur. Currently, the comprehensively studied antibodies targeting the non-RBD region of the spike glycoprotein also include mAb 5F9 targeting the N-terminal domain (NTD) of the S1 subunit , as well as mAbs G2, CDC2-A2, CDC2-A10, JC57-13 and FIB-H1 targeting the non-RBD region of the S1 subunit [33, 50] . However, the detailed epitopes and specific mechanisms are still unclear for these antibodies. We expect that more antibodies with new neutralizing epitopes and/or mechanisms would be important for the combined use of antibodies against MERS-CoV infection.Although monoclonal antibodies show promising antiviral effects in both cell culture and animal models against MERS-CoV infection, their roles are still limited in large-scale disease prevention in MERS-CoV high risk areas, as the therapeutic window is generally narrow for mAbs and mass-scale production is timeand resource-consuming. Vaccines still remain the best choice for MERS-CoV prevention.Given its critical role in mediating viral entry and as major targets for neutralizing antibodies, S glycoprotein and its RBD have become the prime targets for MERS-CoV immunogen design and vaccine development. Various approaches have been applied and more than twenty vaccine candidates have been reported in the past few years, including vaccines based on inactivated virions [62, 63] , virus-like particles , recombinant viral vectors , DNA [49, 81, 82] , recombinant protein subunits [33, 49, , and nanoparticles [80, 93, 94] . Table 2 summarizes the critical features of these approaches and their protective potentials in experimental animal models.Up till now, only two vaccine candidates, GLS-5300 and MERS001, have entered human clinical trials. The vaccine GLS-5300 was the first to be tested in healthy human volunteers. It is a DNA plasmid encoding the MERS-CoV S glycoprotein, requiring two-to-three injections delivered by electroporation . The phase I clinical trial was started in 2016 at the Walter Reed Army Institute, and another phase I/II clinical trial is being conducted in Korea to test dosage safety and immunogenicity. Another vaccine candidate, MERS001, is a replication-deficient chimpanzee adenovirus (ChAdOx1) containing the MERS-CoV S glycoprotein antigen [70, 71] . This vaccine only requires one-time administration of 5×10 9 -5×10 10 virus particles via intramuscular route, and the local adverse events as well as immunogenicity will be evaluated in the phase I clinical trial conducted at the University of Oxford. In addition, one more candidate vaccine has been tested in dromedary camels either for potential human use or straight into veterinary use. It explores a modified vaccinia virus Ankara (MVA) as a vector to express MERS-CoV S glycoprotein . The regimen involves immunization through intranasal as well as intramuscular routes twice at a 4-week interval. The vaccinated camels demonstrated a significant reduction of excreted infectious virus and viral RNA transcripts in vaccinated animals upon MERS-CoV challenge. Protection against MERS-CoV infection correlated with the presence of serum neutralizing antibodies to MERS-CoV. As MVA has established a reasonably good safety profile in humans and induced desirable protective immunity in camels, it represents one of the potential candidates to be further evaluated in humans in the near future.The remaining vaccine candidates are all in the stages of preclinical or laboratory development and invariably target the S glycoprotein or RBD critical for viral entry ( Table 2) . Vaccines based on inactivated [62, 63] or virus-like particles have historical precedence in inducing protective immune responses in humans. Whether the same strategies are applicable to MERS-CoV requires further studies, particularly when it comes to possible safety concerns .Apart from MERS001 and the MVA-based vaccine tested in dromedary camels, other vector-based approaches are also being actively pursued, including adenovirus [68, 69, 72, 73, 80] , measles virus [74, 75] , VEEV replicon particle [76, 77] , vesicular stomatitis virus , and rabies virus . All recombinant viruses encoding the MERS-CoV S or S1 antigen demonstrated strong immunogenicity in mice or non-human primate models, and some were shown to confer protection in MERS-CoV challenge mouse models (Table 2) . However, concerns remain regarding the pre-existing immunity against these viral vectors from natural infection, because it would diminish the vaccine potency . To overcome the issue of preexisting immunity against human adenoviruses while preserving their advantages such as high yields and strong immunogenicity, rare serotypes of chimpanzee adenovirus of low human seroprevalence may be adopted as viral vectors [70, 73] . Our group recently developed a vaccine candidate with replication-defective chimpanzee adenovirus C68 (AdC68) vector expressing full length MERS-CoV S glycoprotein. Seroprevalence of AdC68 is around 2% in human population, much lower than that of the commonly used human adenovirus 5 (HuAd5) vector (>60%) [96, 97] . One intra-nasal administration of 2 × 10 9 viral particles completely protected human DPP4 knock-in (hDPP4-KI) mice from lethal MERS-CoV challenge, and passive transfer of AdC68-S immune sera conferred survival advantage in lethal challenge mouse models . Further, the safety profiles of these vectors have yet to be extensively tested in humans. Recently, Hashem and colleagues showed that the adenovirus-based S1 vaccine may pose potential safety concerns because it may induce pulmonary perivascular hemorrhage in a MERS-CoV challenge mouse model, regardless of the its full protection upon lethal viral infection. They also showed that the pulmonary pathology can be mitigated by incorporating CD40L, an immune-modulator therefore potential molecular adjuvant, into the recombinant adenovirus-based vaccine . Whether this vaccine-associated pathology is related to residual infectious viruses or unbalanced immune responses awaits further investigation. With this in mind, all future MERS-CoV vaccine candidate designs should take extra cautions on safety evaluation. Furthermore, recombinant-protein-based vaccines are widely pursued. Strategies to solubilize the MERS-CoV S glycoprotein in order to form stable immunogens include forming nanoparticles and using soluble protein truncations. In particular, both nanoparticles formed with full length MERS-CoV S glycoprotein [93, 94] and subunit RBD-based vaccines have been shown to induce virus neutralizing antibodies and to protect mice when challenged with MERS-CoV. One RBD subunit vaccine also conferred protection in rhesus macaques . This indicates that RBD alone as antigen may be sufficient for protective immunity to develop against the virus. Along with the finding that mAb targeting NTD is able to neutralize MERS-CoV, Lan et al showed that three doses of intramuscularly administered recombinant NTD protein also induced protective immunity against live MERS-CoV in human DPP4 transduced mouse model (Ad5-hDPP4 mice) . More recently, with the structural insights into the spike glycoprotein, Pallesen et al developed a prefusion-stabilized S trimer vaccine by substituting proline residues into the S2 domain . The introduction of proline disfavours the refolding of the linker between HR1 and the central helix, thus preventing the transition of spike into the post-fusion state. This rationally designed antigen, MERS S-2P, was shown to induce broader and more potent neutralizing activity than wild type spike trimer protein .Finally, a prime-boost strategy based on a full-length S glycoprotein DNA vaccine followed by an S1-glycoprotein boost was able to induce virus-neutralizing antibodies and confer protection against the clinical severity of diseases in non-human primate models . Compared with the protein-only regimen, the combination of DNA and protein induced a more functionally diverse antibody repertoire and stronger Th1 immune response. It was suggested that the native S glycoprotein conformation, formed on the cell surface after DNA vaccination, helped induce more diverse antibodies against MERS-CoV.As summarized in Table 2 , most of the aforementioned strategies require multiple immunizations which may pose additional logistic hurdles at the end point use. It is unclear whether these immunization strategies were empirically designed or due to relatively # Abbreviations for animal models: human DPP4 transgenic (hDPP4-Tg) mice with global/epithelial hDPP4 expression, human DPP4 knock-in (hDPP4-KI) mice with hDPP4 replacing mDPP4 in situ, mice transduced with human adenovirus 5 vector expressing hDPP4 (Ad5-hDPP4 mice), and non-human primates (NHPs).* Efficacy in the specific animal model listed in the previous column. If studies are conducted in multi-animal models, only results in the highest-level model are shown. Neutralizing antibody (nAb).↑ indicates more, while ↓ indicates less.poor immunogenicity of candidate vaccines. For practical and compliant purposes, a single immunization with the highest immunogenicity in animals and humans will be preferred.The outbreak of MERS-CoV in Saudi Arabia in 2012 reminded us of the 2003 SARS-CoV outbreak in China. Despite the differences in geographic location, epidemiology and immediate animal reservoirs, these two viruses share remarkable similarity in causing severe respiratory syndrome, leading to high fatality in humans and trigger serious public health concerns. With the advent of modern techniques in virology, immunology and vaccinology, we have gained substantial insights into the biology of MERS-CoV, and its pathogenesis with unprecedented speed and accuracy. As summarized in the current review, tremendous progress has been made in understanding (1) the entry process of MERS-CoV into target cells, (2) the structure and function of S glycoprotein and cellular receptor DPP4 in mediating viral entry, (3) antibody response during natural infection and isolation of broad and potent neutralizing mAbs, and (4) design and development of vaccine candidates using various innovative technologies. However, our progress in translating these discoveries into clinical application has been slow. Only two vaccine candidates and one mAb panel have entered phase I clinical trials for safety. Ironically, no vaccines and treatment strategies have been approved for SARS-CoV infection even after more than a decade of outbreak. We could not imagine how catastrophic it would be should SARS-CoV hit again or MERS-CoV continues to probe and gain strong capacity in transmission to and among humans.We are facing a difficult predicament when it comes to public health challenges in the new era of emerging and re-emerging infectious diseases. On one hand, the human population is becoming ever mobile and exposed to an increasing number of pathogens. On the other hand, translating basic discoveries into preventative and treatment applications has been exceedingly slow. Among many plausible reasons, a lack of incentives in financial returns perhaps stands the tallest. The deadlock is not just happening to MERS-CoV and SARS-CoV but also to many other infectious pathogens such as Ebola, Marburg, Lassa, highly pathogenic avian influenza, HIV-1, and so on. Fundamental and drastic changes have to be made in the entire research and development system before we can truly prepare and position ourselves ahead of deadly epidemic and pandemic. Only then, can our speed and accuracy in basic discovery be timely translated into clinical and public health needs. The time to act is now.\n",
            "\t0.912\tOn the last day of 2019, a cluster of cases of a pneumonia with unknown cause were reported by the Chinese authorities to the World Health Organization (WHO), believed to be connected to a seafood market in Wuhan, China. This market was closed the following day. On 7 January 2020, a novel coronavirus was isolated, and known pathogens were ruled out. 1 Coronaviruses usually cause respiratory illness ranging from the common cold to severe acute respiratory syndrome (SARS). Clinical symptoms and signs of the Wuhan coronavirus include fever, with some sufferers experiencing difficulty breathing and bilateral pulmonary infiltrates seen on chest X-ray. WHO are referring to it as '2019-nCov'.At the time of writing, there have been over 4,500 confirmed cases and 106 deaths, including among healthcare workers. Over 98% of these cases are within mainland China, but cases have also been confirmed in tens of other countries.There is international concern regarding the potential spread of the virus, especially given Wuhan's status as a major domestic and international hub, its large population of 11 million, and current Chinese New Year celebrations. To curb the spread, all public transport and air travel to and from Wuhan has been stopped, with other Chinese cities following suit. International exit screening of travellers with health information dissemination has begun, including at Heathrow airport. Contacts of confirmed cases are recommended to remain under medical observation, avoiding travel within the viral incubation period of 14 days. 2 Whilst the mortality and infectivity of the 2019-nCoV are still unknown, 3 there is understandable interest and concern regarding the potential for a pandemic as experienced in 2009 with H1N1 ('swine flu'). However, the WHO decided against declaring a public health emergency of international concern on 22 January. Current data suggests the 2019-nCoV has a lower mortality than SARS (another coronavirus), which killed nearly 800 people globally before being contained in 2003.Despite current reassurances about the severity of 2019-nCoV, it will be incredibly difficult to further contain the cross-border spread of the virus due to the sheer volume of global travel. It will be necessary to coordinate a unified, global response across many differing geopolitical boundaries, political settings, cultures, and health system contexts. The fact that incubating cases may have left Wuhan before the quarantine began complicates matters further, as does the emerging possibility that some carriers may be asymptomatic. WHO has asked all countries to prepare for cases including through surveillance, tracing, treatment and isolation practices, and by sharing data. 4 The resources needed to do this effectively may not be routinely available, however, particularly in low-resource settings.In the past decade, there has been an increased Chinese political focus on health, both nationally and internationally. Examples of a growing role in global health include unprecedented involvement in managing the Ebola outbreak and the ambitious 'One Belt One Road' 64-country development initiative, which encompasses 66% of the world's population and a third of global gross domestic product. 5 Chinese authorities were quick to close the Wuhan market, conduct epidemiological assessments, and notify the global health community. 6 The genetic sequence of the virus was shared by Chinese authorities on the 12 January to aid international development of effective diagnostics. 1 President Xi Jinping is personally leading a special cross-departmental taskforce, involving the heads of the health, security, traffic, education, and commerce departments.The WHO exists to coordinate and lead in global health, and for the past week the Strategic Health Operations Centre (SHOC) has been holding hourly meetings to keep abreast of nCoV developments. The SHOC coordinates information and responses through a network of WHO teams, member states, and partner organisations; providing advice, tracking cases, and monitoring essential resources in the field. An Emergency Committee advises the Director General, who makes the ultimate decision on when and whether to declare a public health emergency of international concern. Thus far, escalation to this status has not been made, but the committee has planned to reconvene only days after their initial meeting to review this decision, closely following the rapid development of events. 4The UK government is tracing all those who have arrived from Wuhan within the past 14 days. To date, 31 completed tests on suspected cases have been returned negative and Public Health England (PHE) is advising that the risk to the public remains low. Due to international travel, as well as widespread media and public interest, the number of suspected cases is anticipated to rise and confirmed cases in the UK may yet be seen.It is reasonable to expect that the widespread attention will impact on the demand for appointments in GP practices in coming weeks, as well as anxiety for members of the public. PHE has advised that any person coughing, feeling short of breath, or experiencing fever should seek medical attention if they have recently travelled to Wuhan. Interim guidance has been issued specifically for primary care by PHE. 7 In summary, unwell patients with a relevant travel history should isolate themselves and consult remotely. If they come to the practice, they should be isolated on entrance to the GP surgery. If the consultation has already begun, the healthcare practitioner should leave the room, close the door, and wash their hands thoroughly with soap and water. Clinical history can be conducted by telephone, and physical examination should be avoided. If possible, arrangements should be made for any further assessment to occur in secondary care, after first discussing the differential with the hospital (and ambulance team if required). The local PHE health protection team must be informed. Generally, coronaviruses show more severe symptoms in those with chronic diseases, immunosuppression, or who are older, so these individuals may be more at risk of severe progression.Whilst strong epidemiology and surveillance systems are indispensable tools for the detection and monitoring of outbreaks and public health emergencies, strong primary care systems form the foundation of any emergency response. In the UK, primary care handles over 95% of all health system activity. WHO member states have repeatedly affirmed their commitment to developing their primary care systems with a view to training up community-based health professionals who are able to provide care across the spectrum of prevention, preparedness, response, and recovery. As the 'front door' of the health system, primary care professionals should be involved in planning and action for health emergency risk management. WONCA (the global professional body for family medicine) has actively championed the ways in which primary care can be supported to deliver care during population emergencies. National primary care bodies can coordinate with public health leads to cascade information to practitioners, communicate with the public, and collate health intelligence from the frontline primary care. 8 The Ebola crisis taught us a valuable lesson about what happens when an outbreak takes health workers away from core functions to focus on crisis response; the number of people who died from reduced access to usual care probably exceeded the number killed by the virus. 9 Strong health systems built on comprehensive primary care are able to integrate both functions, disseminating the emergency response resources and information required to community-level staff who have the breadth of training required to manage new suspected cases alongside routine family medicine. Decent access to primary health care is essential in health emergencies, and its infrastructure crucial for containment, 10 just as good access to high-quality primary care is at the foundation of any strong health system. 11No funding was received for this article.Commissioned; not externally peer reviewed.\n",
            "\t0.850\tBackground: Middle Eastern respiratory syndrome coronavirus (MERS-CoV) has spread rapidly across much of the Middle East, but no quantitative mapping of transmission risk has been developed to date. Moreover, details of the transmission cycle of the virus remain unclear, particularly regarding the role of camels as a reservoir host for human infections.Middle East respiratory syndrome coronavirus (MERS-CoV) has caused disease in more than 1300 persons across the Middle East . Most cases have been in Saudi Arabia , although the virus is known from other countries in the Middle East and North Africa ; cases exported from the Middle East to Europe and North America , and a recent pulse of cases across South Korea, have brought renewed attention to the disease . Clearly, MERS-CoV represents a significant global threat to public health, such that detailed analyses from diverse perspectives are needed.As the public health community has rushed to understand the etiology and natural history of this disease over the past several years, insights have begun to emerge. A first point is that bats appear to play some sort of ultimate role in the long-term hosting of a diverse community of coronaviruses [e.g., ], including virus lineages closely allied to that found in MERS-affected humans [9, 10] . This bat origin and long-term hosting coincides with the apparent conclusions regarding the identity of the long-term host of the preceding SARS-CoV virus that caused disease across East Asia .*Correspondence: 1 Biodiversity Institute, University of Kansas, Lawrence, KS 66045, USA Full list of author information is available at the end of the article For MERS-CoV, however, increasing evidence indicates that both camels and humans may play intermediate roles, as both disease victims and reservoirs for further transmission; indeed, no robust evidence indicates direct transmission from bats to humans [see partial and controversial evidence in 9]. Human-to-human transmission has been documented amply ; in the Korean situation, for example, transmission appears to be entirely from humans to humans . Contact of various types with camels, such as consumption of raw milk, butchering and cleaning meat, and visiting live animals, has been identified as a significant risk factor, such that camels are seen as a significant reservoir host for MERS-CoV in its transmission to humans (e.g., ).All studies to date of the host ecology and transmission biology of MERS-CoV have been based on case reports, serological studies, and molecular sequence data. The geographic pattern of case occurrences, particularly with respect to the environments manifested at those sites, however, holds potentially rich information about the transmission biology and spatial distribution of transmission risk of diseases such as this one . As such, this paper explores the geographic and environmental circumstances under which MERS-CoV has been transmitted from camels to humans, and where those highrisk circumstances are manifested (i.e., developing a risk map of transmission probabilities). In particular, we test the hypothesis that MERS-CoV cases derived from camel exposure (i.e., as opposed to human-to-human transmission) will have a narrower set of associated environments (i.e., niche breadth), which would be indicative of environmental influences on transmission that can be translated into a map of transmission risk.We conducted an exhaustive search of all mentions of MERS-CoV (search terms \"MERS\" and \"Middle Eastern Respiratory Syndrome\", in ProMED-mail, . promedmail.org/), covering from 2012 through 31 May 2015. The initial number of ProMED posts was 10,248, and each was read carefully by one person (TR), and a form filled out to capture the following information: ProMed archive number, report date, country, state/province, locality, age of person, sex of person, whether person is a healthcare worker, whether the person had other existing co-morbidities, whether the person reported exposure to domestic animals (particularly camels), whether the person reported contact with other MERS cases, and remarks. Each report had to be read carefully to extract information for as many of the fields as possible, and we worked hard not to include duplicate records of cases. Although other, country-specific sources are available regarding MERS (e.g., ), we opted for the more consistent, regionwide coverage provided by the global resource.We next embarked on a long series of steps designed to clean the data, detecting and fixing errors whenever possible, removing any remaining duplicate records, and adding geographic references. These steps involved repeated searches and manipulations of the data set in spreadsheets, as well as use of Microsoft Access to encounter pairs of records with identical information (e.g., for country, city, age, and date) that would signal duplication. These initial cleaning steps reduced the data matrix to 1171 records of MERS-CoV cases.We then extracted from the data matrix all unique combinations of country, state or province, and locality (93 sites), and assigned latitude-longitude pairs in WGS 1984 geographic coordinates to them via consultation of various online gazetteers (Google Earth, . ; Global Gazetteer version 2.3, http:// ). In each case, georeferencing was by TR; AMS, a native Arabic speaker, checked all interpretations carefully. We then plotted the geographic coordinates in a GIS (QGIS, version 2.4) to explore further any localities that had suspicious or uncertain georeferences. Of the 93 localities, some did not provide sufficient information or could not be localized with confidence, such that 67 localities and 1113 case reports could be used in our analyses.We used the WorldClim, version 1.4, 'bioclimatic' variable set at 10′ (~17 km) spatial resolution to summarize environments associated with MERS-CoV cases. We used this coarse spatial resolution, and did not take advantage of more information-rich remotely-sensed data resources, in light of the coarse spatial resolution of the input occurrence data-occurrences were generally referenced to city names only, and nothing more specific. We used bioclimatic variables 1-7 and 10-17, omitting variables 8-9 and 18-19, in light of known spatial artifacts in those four variables. Because the remaining 15 variables show considerable intercorrelation and non-independence , we used principal components analysis to reduce dimensionality and reduce correlations among variables.Specifically, we first clipped the global rasters to the area within 10° (~1100 km) of known MERS-CoV cases (excluding exported cases in Korea, Europe, and North America), in light of ample evidence that MERS-CoV transmission to humans has concentrated in the Middle East. Next, we 'stacked' the 15 bioclimatic raster data layers, and applied a principal components analysis. Because the first five components summarized 99.99 % of the total variance in the original data, we focused on those components only in our analyses. The first three components were plotted in red-green-blue color space for visualization of overall patterns of variation in climate across the region (Fig. 1) . To permit exploratory analyses, we used the point sampling tool in QGIS to extract raster data values to each known occurrence point.To explore patterns of environmental associations, and their implications for the potential geography of MERS-CoV transmission , we developed ecological niche models for the disease, creating models for all cases (N = 1113), and for only those cases with reported camel contact (N = 83). Models were calibrated across the region within 10° of known MERS-CoV cases, as a rough hypothesis of the accessible area for the virus . Using Maxent version 3.3.3k , we developed 10 replicate bootstrap analyses for each occurrence data set, and used the median across the 10 as the output (regularization parameter = 1, prevalence = 0.5). We estimated the uncertainty associated with these outputs, pixel by pixel, as the difference between the maximum and minimum values among the 10 random replicate analyses.To compare niche breadths estimated in the two niche models, we plotted 10,000 random points across the broader study area (of which 7759 fell on land and were included in analyses), and extracted grid values of each of the original bioclimatic variables for each of the points using the point sampling tool in QGIS. We then thresholded each of the niche models using a least training presence thresholding approach, modified to permit an omission rate of E = 0.1 , such that we identified the highest raw suitability level that included 90 % of the data with which the model was calibrated in the suitable area. We then discarded all random points not identified as suitable in either of the models, leaving 5936 points for analysis. Finally, as fewer points were identified as suitable in the model based on camel exposures (4811 points), we rarefied the larger set of points identified as suitable in the model based on all MERS-CoV cases (5707 points) down to the number of points suitable in the camel model, and calculated the standard deviation of each environmental variable associated with points in each data set. Based on direct counts of the observed standard deviation from the camel-based model with those among the 100 random replicates drawn from the unrestricted model, we tested whether the niche estimated based on camel exposures was narrower than that based on all cases.Our MERS-CoV case-occurrence data set reflected a clear concentration of cases in Saudi Arabia (that is, among Middle Eastern cases only; Fig. 1 ), although a few cases have been detected in neighboring countries (United Arab Emirates, Qatar, Oman, Jordan, Iran). This concentration was reflected in the ecological niche model outputs (Fig. 2) : the model based on reported camel exposure showed highest suitability in the northeastern and southwestern parts of the Arabian Peninsula, as well as adjacent areas of southwestern Asia and northeastern Africa. The model based on all occurrences, on the other hand, identified a broader suitable area across the central portion of the Arabian Peninsula and adjacent areas of Asia and Africa. High uncertainty in these predictions, in both cases, coincided with areas of high modeled suitability (Fig. 2) , such that confidence in model predictions is not uniformly high.The outputs of the two ecological niche models contrasted in their spatial predictions across the study region. Figure 3 shows the difference between the two outputs: the model based on camel exposures emphasized the northern and southern coastal parts of the Arabian Peninsula, and de-emphasized the central and western parts. Clearly, the two models emphasize different regions, which reflects different environmental Fig. 2 Visualization of ecological niche model outputs in terms of median prediction of suitability for MERS-CoV transmission to humans (left column; light orange low suitability, dark orange high suitability), and the uncertainty associated with those predictions estimated as the range of suitability values (pixel by pixel) among the 10 bootstrapped random replicate models (right column; white low uncertainty, black high uncertainty). Dots represent camel-exposed cases; X's represent any case occurrence regimes underlying the occurrence data that drive the model outputs.To explore relative niche breadths of models based on reported camel exposure versus the broader suite of possible exposure regimes that could produce MERS-CoV cases, we developed detailed comparisons of ecological niche breadths. That is, we applied 10 % omission thresholds of 0.05357 for models based on camel exposures, and 0.02958 for models based on all occurrences (note that these thresholds are driven by the relationship between calibration data and model outputs) to convert the two to binary models. We then related these two predictions to annual mean temperature and annual precipitation via the random points across the study region. The rarefied points for annual mean temperature in the camel-based model had a standard deviation of 40.5, whereas those for the model based on any occurrence ranged 49.0-50.2; for annual precipitation, the camel-based model had a standard deviation of 94.7, whereas those for the anyoccurrence model ranged 236.2-253.1 (Fig. 4) . Hence, with respect to both temperature and precipitation, the diversity of environments indicated as suitable based on camel-based exposures was significantly less than the diversity of environments so identified based on all occurrences (P < 0.01, in both cases).MERS-CoV has had a dramatic, if brief, history, in which it has emerged in the Middle East, and spread to at least four continents in just 3 years. Although many analyses have been published regarding various aspects of the distribution [3, 5] , epidemiology , evolution , and phylogeny , major and significant knowledge gaps exist regarding this disease . This contribution offers a novel approach: we tested and corroborated the hypothesis that transmission from camels to humans occurs under a quite-restricted set of environmental circumstances. We also produce a first view of regionwide suitability for MERS-CoV transmission to humans, focusing in the Middle East, as that is the region in which the disease has the longest history.Our ecological niche models represent and reconstruct the relationship between the occurrence data and the environmental data; although we are intrigued by Fig. 3 Map of difference between the two ecological niche models (one based on reported camel exposure versus one based on all reported cases). Red areas emphasized in the camel-exposed analysis, blue areas emphasized in the any-case analysis. Dots represent camel-exposed cases; X's represent any case occurrence the spatial and environmental patterns that we detected, we are nonetheless cognizant of several limitations of this approach. First of all, the occurrence data are quite crude in their geographic localization: because MERS-CoV exposure and infection are rather coarse and diffuse, the occurrence data cannot be georeferenced more finely than multiple kilometers, which limits the spatial detail in our predictions, and constrains us from using remotesensing data to achieve improved spatial resolution .Second, reporting of exposure patterns in MERS-CoV cases is rather uneven and unreliable. That is, fewer than 10 % of the overall suite of cases reported contact with camels; this small number either reflects a dominant role of human-to-human transmission in the etiology of this disease, or lack of recognition and reporting of camel contacts among other cases. On the other hand, some portion of the camel contacts reported may represent 'red herrings': the contact reported may not have been the source of the infection.Given these shortcomings in the data, we were impressed by the clarity of the results that we obtained. Our analyses pointed clearly to a restricted set of environmental conditions under which MERS-CoV is transmitted from camels to humans: niche breadth in the camel-exposed models was strikingly restricted relative to that manifested in the any-occurrence models. This better definition of a 'transmission niche' may reflect a narrower suite of environmental conditions under which camel-to-human MERS-CoV transmission can occur.This well-defined ecological niche can in turn be used to explore the potential geography of the transmission phenomenon . We documented an intriguing contrast between the overall transmission geography and that associated with camel exposure. The implication, then, is that circumscribed portions of the Arabian Peninsula can be identified in which camel-to-human MERS-CoV transmission may be particularly likely.The results of this study also point to a number of research and data gaps in the knowledge of the disease. That is, we note considerable variation and incompleteness of reporting of cases: much more could be learned about this disease (and others) were data reporting to be made more standard and uniform in quality. Detailed geographic sampling of bat, camel, and human populations across the study region would enrich the environmental picture considerably, in addition to permitting detection of reporting filters and other biases affecting the picture of the disease's geography and ecology ; we also see considerable need for sampling and testing more broadly across North Africa and Sub-Saharan Africa . These improvements would offer greatly improved insights into the questions on which this analysis centers.The data set supporting the results of this article is available in the LabArchives repository, DOI: 10.6070/ H4DR2SHR. \n",
            "\t0.797\tA novel coronavirus (2019-nCoV) causing severe acute respiratory disease emerged recently in Wuhan, China. Information on reported cases strongly indicates human-to-human spread, and the most recent information is increasingly indicative of sustained human-tohuman transmission. While the overall severity profile among cases may change as more mild cases are identified, we estimate a risk of fatality among hospitalised cases at 14% (95% confidence interval: 3.9-32%).Four strains of coronaviruses are known to spread easily in humans, causing generally-mild acute respiratory illnesses known as the common cold . A much larger number of coronaviruses have been detected in animals, particularly in bats, but have not been found in humans . Prior to December 2019 when clusters of pneumonia cases with unknown aetiology were detected in Wuhan, China, only two additional strains of coronaviruses had caused outbreaks of severe acute respiratory disease around the world . In the 2003 outbreaks of severe acute respiratory syndrome coronavirus (SARS-CoV) infections in mainland China, Hong Kong and a number of other locations, there were more than 8,000 documented cases and 774 deaths . Since 2012, outbreaks of Middle East respiratory syndrome coronavirus (MERS-CoV) infection have occurred in the Middle East , and in 2015, there was a large outbreak in South Korea [6, 7] . Super-spreading events have contributed to large outbreaks of SARS-CoV and MERS-CoV .On 9 January 2020, a novel coronavirus, 2019-nCoV, was officially identified as the cause of an outbreak of viral pneumonia in Wuhan, China . Wuhan is a large city of more than 11 million people located in central China around 1,200 km south of Beijing. As of 22 January, there have been 440 confirmed 2019-nCoV infections reported in 13 provinces and municipalities in mainland China and five other countries and regions overseas, with an increasing number of cases reported in recent days ( Figure 1 ). Here, we describe the preliminary epidemiological characteristics of 2019-nCoV infections based on publicly-available information (including some media reports) before an official 'line list' of confirmed cases becomes available.On 31 December 2019, the Wuhan Municipal Health Commission announced a cluster of cases of viral pneumonia of unexplained aetiology. The Southern China Seafood Wholesale Market in Wuhan was suspected to be related to the first 27 pneumonia cases without identified pathogenic agents that were reported in late December 2019 . Most of the early cases were reportedly either shop owners, largely in the West District of the Southern China Seafood Wholesale Market, or people who visited the market before symptom onset. This market is a large open complex of 50,000 square metres including sections selling seafood, fresh meat, produce, other perishable goods, and a very wide variety of live wild animals for consumption. While wet markets selling such perishable food products are common in China, they usually do not sell such a wide variety of wild animals. Environmental disinfection of the Southern China Seafood Wholesale Market was initiated on 30 December 2019 and the market was closed on 1 January 2020 . Contact tracing was initiated, and by 5 January, more than 160 close contacts of these cases were under medical surveillance, and none of them had an infection .On 9 January 2020, a novel coronavirus, 2019-nCoV, was officially identified as the cause of the outbreak of pneumonia in Wuhan. Following the official announcement of the genetic sequence of the virus, on 11 January, 41 laboratory-confirmed cases of 2019-nCov infection with pneumonia were reported in Wuhan . While the case definition for laboratory-confirmed cases has not been officially published, our understanding is that the initial case definition required (i) fever, (ii) x-ray evidence of pneumonia, (iii) white blood cell count normal or low or low lymphocyte count, (iv) antibiotic treatment for 3 days without improvement, in addition to (v) one or more recent visits to Wuhan or direct or indirect exposure to a wet market in Wuhan, and (vi) a respiratory specimen positive for 2019-nCoV and confirmed as 2019-nCoV by whole genome sequencing. The earliest known case had illness onset on 8 December 2019.Between 10 and 12 January when there were no new cases reported, the Wuhan Municipal Health Commission announced that no new cases of 2019-nCoV infection were identified with illness onset after 3 January. As at 11 January, more than 700 close contacts were under medical surveillance, more than half being healthcare workers (HCWs), with no infections being identified . Among the first 41 confirmed cases, approximately 70% reported exposure to the Southern China Seafood Wholesale Market . Using the method described by Cauchemez et al. , assuming that the market was the only source of zoonotic infections, we estimated R 0 to be 0.3 (95% confidence interval (95% CI): 0.17-0.44).On 13 January 2020, Thai health authorities reported an imported case in a person in their 60s who had travelled from Wuhan. This person had onset of an upper respiratory illness on 5 January and landed in Bangkok on 8 January, and was screened out by the thermal scanners at the airport. They did not visit the Southern China Seafood Wholesale Market, but reported visiting another wet market in Wuhan. On 16 January, Japanese health authorities reported a confirmed imported case in a person in their 30s who had travelled from Wuhan and landed in Kanagawa on 6 January. Illness onset was on 3 January and the person has since recovered. The person had not visited any wet market in Wuhan, but had visited a close relative who was in hospital in Wuhan with pneumonia. A third case was reported by Thai health authorities on 17 January. This person was in their 70s and had landed in Bangkok on 13 January, reporting an illness onset on 6 January and unclear history of exposure to the market in Wuhan. On 20 January, the fourth exported case from Wuhan, a person in their 30s visiting South Korea, was identified in Seoul. This person reported no previous visit to wet markets in Wuhan nor contact with any other case within 2 weeks before illness onset. The lack of exposure history to wet markets in Wuhan in two of four generally-mild exported cases indicated that there might be a larger number of undetected infections in Wuhan.Two family clusters involving five people were reported from 15 to 16 January. One cluster of three close relatives all with illness onset on the same day, were thought to have occurred through a common exposure since they all lived together and worked in the same stall in the Southern China Seafood Wholesale Market. The other cluster involved a couple and may have occurred through human-to-human transmission with a serial interval of 5 days .The basic reproduction number R 0 is defined as the expected number of secondary cases produced by a typical single infection in a completely susceptible population. If there had been only one case infected by human-to-human transmission among the first 41 identified cases by that date, it implies R 0 was 0.02 (i.e. 1/41) .On 19 January 2020, a task force under the National Health Commission, created on 18 January, visitedWuhan. On 20 January, 136 new infections in Wuhan, including 100 mild cases, 33 severe cases and three critically ill cases, were reported by the Wuhan Municipal Health Commission. Three traveller cases from Wuhan were also identified in Beijing and Shenzhen, possibly suggesting a large number of infections in Wuhan itself. However, further information on the most recently confirmed 2019-nCoV infections is required to determine the transmissibility of the virus and the severity profile of infections.On 20 January, the task force reported three possible episodes of human-to-human transmission. This included, in two different families, family members who had not recently visited Wuhan being laboratoryconfirmed with 2019-nCoV infection after other family members came back from Wuhan and were confirmed cases. A possible super-spreading event was also speculated by the task force given that there had been 15 HCWs infected with the virus in Wuhan until 20 January . On 21 January, additional exported cases were reported in Taiwan and the United States, with illness onset on 11 and 19 January respectively.We estimated the hospital fatality risk, i.e. the risk of fatality among hospitalised cases using the formula (fatal cases)/(fatal cases + recovered cases), which provides a more accurate early estimate of the hospital fatality risk compared with (fatal cases)/(all cases) . We estimated the associated 95% CI for the hospital fatality risk using the binomial distribution. According to the update on 21 January 2020 when information on deaths and recoveries were reported, four cases had died while 25 had recovered, and our estimate of the hospital fatality risk is therefore 14% (95% CI: 3.9-32%). The estimate of the hospital fatality risk remained fairly stable over the 10 day period since the first death was announced on 11 January ( Figure 2 ). If deaths continue to be reported without any corresponding increase in reported recoveries, the formula will overestimate the risk of fatality.To date, as at 22 January, official information has not been sufficient to determine the source of infection and risk of human-to-human transmission for the newly emerged 2019-nCoV. Two competing hypotheses are therefore proposed and compared as potential explanations for the characteristics of the current outbreak of 2019-nCoV infections within and outside mainland China. Scenario 1 comprises a large zoonotic spillover event starting in early December 2019, perhaps over a number of days or weeks, and very limited human-to-human transmission subsequently. Scenario 2 comprises a small zoonotic spillover event in early December 2019 followed by efficient human-to-human transmission. Evidence consistent or inconsistent with each of these two scenarios is outlined in the Table. In this article, we describe a preliminary assessment of the outbreak of infections with the newly identified 2019-nCoV. This assessment is based on the cases of infection reported over time by health authorities in Wuhan and then at the national level, as well as the media in China and other countries. One of the most urgent priorities is to determine the degree of humanto-human transmissibility of the novel pathogen, and accordingly, this is where information is most urgently needed. We outline two possible scenarios in the Table and find that the early evidence was most consistent with limited human-to-human transmissibility, however more recent data seem to be increasingly more compatible with scenario 2 in which sustained human-to-human transmission has been occurring. Determining the exposure profile among the recently confirmed cases would directly contribute to this assessment. Additional information on approaches to case identification and laboratory testing protocols in Wuhan and in other cities in China would also be informative. A separate priority is to identify the animal reservoir of this novel pathogen and any intermediary hosts, including potential supply chains of wild or game meat.It is challenging to judge severity from the information available to date. We estimated the risk of death among hospitalised cases of around 14% (Figure 2 ). For both SARS-CoV and MERS-CoV infections, the risk of severe disease increases substantially with age and with the presence of underlying conditions . One other caveat with estimating severity is that there can be long delays between hospitalisation and death for infections that are ultimately fatal. For SARS in Hong Kong, the average time from illness to death for fatal cases was 24 days . This means that early estimates of the case fatality risk that ignore the potential outcomes of cases still in hospital are typically underestimates of the final severity profile . We accounted for that by only including cases that either died or recovered in our estimate of the hospital fatality risk. Given that the cases reported outside Wuhan have mostly not been severe, it would be reasonable to infer that there might be a large number of undetected relatively mild infections in Wuhan and that the infection fatality risk is below 1% or even below 0.1%.There are a number of limitations to our analyses. Most importantly, they are only based on data in the public domain to date. Detailed information has not yet been released by authoritative sources on the most recently reported cases. In our analysis in the \n",
            "\n",
            " Title:  On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XkQjeuauHXt1"
      },
      "source": [
        "Compared to the previous sentence embedding approach, this is far better.\n",
        "\n",
        "The returned answers we're getting, are way more better than the previous ones. Also, the time for each query is around ~3seconds.\n",
        "\n",
        " We are going to contiinue with this approach, and we'll save in the array **indexes**, each index of the best returned paper. So, below this cell we are going to try return the answer **it-self** (passage).\n",
        "\n",
        "For this task we are using an other notebook for minimizing the resources as much as possible. We only are going to need the indexes."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "iXxCMBkVj6Ru",
        "outputId": "d9f4815a-4e06-44ea-b53f-d65a8d7d5bc5"
      },
      "source": [
        "indexes"
      ],
      "execution_count": 27,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[8093, 1988, 2384, 2596, 3533, 7823, 4815]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 27
        }
      ]
    }
  ]
}